{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Jupyter notebook with code for importing demultiplexed data from the 18S marker. For each library (9 total, a-i) 1) import 2) trim primers 3) summarize and view trimmed data 4) generate ESVs with quality trimming parameters based on (3) 5) summarize and view ESV table 6) tabulate and view ESV sequences\n",
    "Then: 7) merge ESV table and sequences for all libraries 8) tabulate and view sequences, download as .fasta (GUI download) 9) summarize and view OTU table 10) export OTU table as biom 11) convert biom to tsv, json, and hdf formats"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "a: P1604 Library 1 b: P1604 Library 2 c: P1604 Library 3 d: BB Library 1 e: Hirai Library 1 f: Hirai Library 2 g: P2107 Library 1 h: ENP Library 1 i: GAK Library 1 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#0: import the qiime2 python module to allow for inline visualizations\n",
    "import qiime2 as q2\n",
    "#which version of qiime2 and python are we using?\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib2-zhan_manifest_run1_grid.csv as PairedEndFastqManifestPhred33 to a-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S124_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S124_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S124_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S124_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S124_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S124_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S124_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S124_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        67,912 reads  @     91.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 6.22 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,912\n",
      "  Read 1 with adapter:                  67,897 (100.0%)\n",
      "  Read 2 with adapter:                  67,576 (99.5%)\n",
      "Pairs written (passing filters):        67,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,065,443 bp\n",
      "  Read 1:    20,033,305 bp\n",
      "  Read 2:    20,032,138 bp\n",
      "Total written (filtered):     37,426,257 bp (93.4%)\n",
      "  Read 1:    18,676,745 bp\n",
      "  Read 2:    18,749,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t68\t0.0\t1\t0 0 68\n",
      "19\t1429\t0.0\t1\t0 1381 48\n",
      "20\t66252\t0.0\t2\t64654 1581 17\n",
      "21\t111\t0.0\t2\t0 105 6\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1447\t0.0\t1\t0 1447\n",
      "19\t66000\t0.0\t1\t64304 1696\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S125_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S125_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S125_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S125_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S125_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S125_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S125_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S125_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,744 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.34 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,744\n",
      "  Read 1 with adapter:                  49,730 (100.0%)\n",
      "  Read 2 with adapter:                  49,443 (99.4%)\n",
      "Pairs written (passing filters):        49,744 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,347,422 bp\n",
      "  Read 1:    14,673,323 bp\n",
      "  Read 2:    14,674,099 bp\n",
      "Total written (filtered):     27,415,283 bp (93.4%)\n",
      "  Read 1:    13,679,784 bp\n",
      "  Read 2:    13,735,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t47\t0.0\t1\t0 0 47\n",
      "19\t1083\t0.0\t1\t0 1057 26\n",
      "20\t48509\t0.0\t2\t47260 1233 16\n",
      "21\t66\t0.0\t2\t0 63 3\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t915\t0.0\t1\t0 915\n",
      "19\t48430\t0.0\t1\t46983 1447\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S126_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S126_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S126_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S126_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S126_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S126_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S126_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S126_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        54,118 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.97 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,118\n",
      "  Read 1 with adapter:                  54,106 (100.0%)\n",
      "  Read 2 with adapter:                  53,606 (99.1%)\n",
      "Pairs written (passing filters):        54,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,927,906 bp\n",
      "  Read 1:    15,964,212 bp\n",
      "  Read 2:    15,963,694 bp\n",
      "Total written (filtered):     29,829,258 bp (93.4%)\n",
      "  Read 1:    14,883,157 bp\n",
      "  Read 2:    14,946,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1098\t0.0\t1\t0 1061 37\n",
      "20\t52870\t0.0\t2\t51547 1297 26\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1020\t0.0\t1\t0 1020\n",
      "19\t52487\t0.0\t1\t50768 1719\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S127_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S127_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S127_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S127_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S127_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S127_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S127_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S127_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        60,020 reads  @    113.6 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.82 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,020\n",
      "  Read 1 with adapter:                  60,005 (100.0%)\n",
      "  Read 2 with adapter:                  59,765 (99.6%)\n",
      "Pairs written (passing filters):        60,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,803 bp\n",
      "  Read 1:    17,704,936 bp\n",
      "  Read 2:    17,705,867 bp\n",
      "Total written (filtered):     33,077,353 bp (93.4%)\n",
      "  Read 1:    16,506,054 bp\n",
      "  Read 2:    16,571,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60005 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t54\t0.0\t1\t0 0 54\n",
      "19\t1246\t0.0\t1\t0 1204 42\n",
      "20\t58590\t0.0\t2\t57053 1522 15\n",
      "21\t94\t0.0\t2\t0 94\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59765 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1095\t0.0\t1\t0 1095\n",
      "19\t58542\t0.0\t1\t56981 1561\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S128_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S128_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S128_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S128_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S128_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S128_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S128_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S128_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        51,242 reads  @    109.5 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.62 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,242\n",
      "  Read 1 with adapter:                  51,232 (100.0%)\n",
      "  Read 2 with adapter:                  50,999 (99.5%)\n",
      "Pairs written (passing filters):        51,242 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,231,296 bp\n",
      "  Read 1:    15,116,131 bp\n",
      "  Read 2:    15,115,165 bp\n",
      "Total written (filtered):     28,239,587 bp (93.4%)\n",
      "  Read 1:    14,092,615 bp\n",
      "  Read 2:    14,146,972 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1145\t0.0\t1\t0 1118 27\n",
      "20\t49934\t0.0\t2\t48735 1176 23\n",
      "21\t89\t0.0\t2\t0 85 4\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t881\t0.0\t1\t0 881\n",
      "19\t50025\t0.0\t1\t48672 1353\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S129_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S129_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S129_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S129_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S129_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S129_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S129_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S129_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,537 reads  @     92.7 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.50 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,537\n",
      "  Read 1 with adapter:                  48,523 (100.0%)\n",
      "  Read 2 with adapter:                  48,218 (99.3%)\n",
      "Pairs written (passing filters):        48,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,623,448 bp\n",
      "  Read 1:    14,317,401 bp\n",
      "  Read 2:    14,306,047 bp\n",
      "Total written (filtered):     26,738,717 bp (93.4%)\n",
      "  Read 1:    13,347,908 bp\n",
      "  Read 2:    13,390,809 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48523 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t58\t0.0\t1\t0 0 58\n",
      "19\t978\t0.0\t1\t0 950 28\n",
      "20\t47387\t0.0\t2\t46139 1234 14\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t989\t0.0\t1\t0 989\n",
      "19\t47144\t0.0\t1\t45859 1285\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S130_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S130_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S130_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S130_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S130_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S130_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S130_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S130_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        60,486 reads  @    100.2 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.07 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,486\n",
      "  Read 1 with adapter:                  60,472 (100.0%)\n",
      "  Read 2 with adapter:                  60,055 (99.3%)\n",
      "Pairs written (passing filters):        60,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,664,535 bp\n",
      "  Read 1:    17,843,224 bp\n",
      "  Read 2:    17,821,311 bp\n",
      "Total written (filtered):     33,316,132 bp (93.4%)\n",
      "  Read 1:    16,634,985 bp\n",
      "  Read 2:    16,681,147 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60472 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t1246\t0.0\t1\t0 1200 46\n",
      "20\t59076\t0.0\t2\t57483 1564 29\n",
      "21\t79\t0.0\t2\t0 77 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t973\t0.0\t1\t0 973\n",
      "19\t58990\t0.0\t1\t57546 1444\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S131_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S131_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S131_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S131_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S131_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S131_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S131_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S131_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        69,839 reads  @    111.8 µs/read;   0.54 M reads/minute\n",
      "Finished in 7.81 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,839\n",
      "  Read 1 with adapter:                  69,830 (100.0%)\n",
      "  Read 2 with adapter:                  67,848 (97.1%)\n",
      "Pairs written (passing filters):        69,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,092,988 bp\n",
      "  Read 1:    20,601,721 bp\n",
      "  Read 2:    20,491,267 bp\n",
      "Total written (filtered):     38,409,198 bp (93.5%)\n",
      "  Read 1:    19,206,506 bp\n",
      "  Read 2:    19,202,692 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69830 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1444\t0.0\t1\t0 1385 59\n",
      "20\t68208\t0.0\t2\t66447 1737 24\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t67091\t0.0\t1\t65644 1447\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S132_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S132_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S132_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S132_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S132_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S132_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S132_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S132_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        68,047 reads  @    109.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 7.42 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,047\n",
      "  Read 1 with adapter:                  68,033 (100.0%)\n",
      "  Read 2 with adapter:                  67,727 (99.5%)\n",
      "Pairs written (passing filters):        68,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,144,011 bp\n",
      "  Read 1:    20,073,426 bp\n",
      "  Read 2:    20,070,585 bp\n",
      "Total written (filtered):     37,498,770 bp (93.4%)\n",
      "  Read 1:    18,713,734 bp\n",
      "  Read 2:    18,785,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 68033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1064\t0.0\t1\t0 1041 23\n",
      "20\t66783\t0.0\t2\t65001 1755 27\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67727 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1409\t0.0\t1\t0 1409\n",
      "19\t66173\t0.0\t1\t64532 1641\n",
      "20\t145\t0.0\t1\t0 145\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S133_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S133_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S133_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S133_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S133_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S133_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S133_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S133_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        69,695 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.67 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,695\n",
      "  Read 1 with adapter:                  69,690 (100.0%)\n",
      "  Read 2 with adapter:                  69,316 (99.5%)\n",
      "Pairs written (passing filters):        69,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,119,345 bp\n",
      "  Read 1:    20,559,999 bp\n",
      "  Read 2:    20,559,346 bp\n",
      "Total written (filtered):     38,410,807 bp (93.4%)\n",
      "  Read 1:    19,167,293 bp\n",
      "  Read 2:    19,243,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1159\t0.0\t1\t0 1118 41\n",
      "20\t68349\t0.0\t2\t66615 1712 22\n",
      "21\t91\t0.0\t2\t0 88 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1282\t0.0\t1\t0 1282\n",
      "19\t67924\t0.0\t1\t66152 1772\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S134_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S134_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S134_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S134_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S134_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S134_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S134_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S134_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        80,095 reads  @     94.1 µs/read;   0.64 M reads/minute\n",
      "Finished in 7.54 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,095\n",
      "  Read 1 with adapter:                  80,079 (100.0%)\n",
      "  Read 2 with adapter:                  79,737 (99.6%)\n",
      "Pairs written (passing filters):        80,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,255,407 bp\n",
      "  Read 1:    23,627,804 bp\n",
      "  Read 2:    23,627,603 bp\n",
      "Total written (filtered):     44,141,514 bp (93.4%)\n",
      "  Read 1:    22,027,438 bp\n",
      "  Read 2:    22,114,076 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1314\t0.0\t1\t0 1288 26\n",
      "20\t78553\t0.0\t2\t76498 2026 29\n",
      "21\t116\t0.0\t2\t0 115 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1641\t0.0\t1\t0 1641\n",
      "19\t77931\t0.0\t1\t76056 1875\n",
      "20\t165\t0.0\t1\t0 165\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S135_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S135_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S135_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S135_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S135_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S135_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S135_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S135_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        81,314 reads  @    100.5 µs/read;   0.60 M reads/minute\n",
      "Finished in 8.18 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,314\n",
      "  Read 1 with adapter:                  81,300 (100.0%)\n",
      "  Read 2 with adapter:                  81,008 (99.6%)\n",
      "Pairs written (passing filters):        81,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,974,606 bp\n",
      "  Read 1:    23,987,034 bp\n",
      "  Read 2:    23,987,572 bp\n",
      "Total written (filtered):     44,812,090 bp (93.4%)\n",
      "  Read 1:    22,362,356 bp\n",
      "  Read 2:    22,449,734 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1418\t0.0\t1\t0 1377 41\n",
      "20\t79671\t0.0\t2\t77587 2053 31\n",
      "21\t106\t0.0\t2\t0 104 2\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1482\t0.0\t1\t0 1482\n",
      "19\t79358\t0.0\t1\t77512 1846\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S136_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S136_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S136_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S136_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S136_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S136_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S136_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S136_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,544 reads  @    105.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.89 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,544\n",
      "  Read 1 with adapter:                  46,532 (100.0%)\n",
      "  Read 2 with adapter:                  46,368 (99.6%)\n",
      "Pairs written (passing filters):        46,543 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,459,035 bp\n",
      "  Read 1:    13,729,821 bp\n",
      "  Read 2:    13,729,214 bp\n",
      "Total written (filtered):     25,648,572 bp (93.4%)\n",
      "  Read 1:    12,799,612 bp\n",
      "  Read 2:    12,848,960 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t765\t0.0\t1\t0 738 27\n",
      "20\t45646\t0.0\t2\t44561 1079 6\n",
      "21\t59\t0.0\t2\t0 57 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t821\t0.0\t1\t0 821\n",
      "19\t45464\t0.0\t1\t44391 1073\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S137_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S137_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S137_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S137_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S137_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S137_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S137_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S137_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        74,550 reads  @     97.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 7.30 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,550\n",
      "  Read 1 with adapter:                  74,528 (100.0%)\n",
      "  Read 2 with adapter:                  74,148 (99.5%)\n",
      "Pairs written (passing filters):        74,549 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,966,112 bp\n",
      "  Read 1:    21,991,857 bp\n",
      "  Read 2:    21,974,255 bp\n",
      "Total written (filtered):     41,069,114 bp (93.4%)\n",
      "  Read 1:    20,502,233 bp\n",
      "  Read 2:    20,566,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t60\t0.0\t1\t0 0 60\n",
      "19\t1298\t0.0\t1\t0 1263 35\n",
      "20\t73005\t0.0\t2\t71027 1950 28\n",
      "21\t123\t0.0\t2\t0 120 3\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74148 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1567\t0.0\t1\t0 1567\n",
      "19\t72452\t0.0\t1\t70694 1758\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S138_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S138_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S138_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S138_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S138_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S138_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S138_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S138_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:13       146,234 reads  @     95.4 µs/read;   0.63 M reads/minute\n",
      "Finished in 13.96 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,234\n",
      "  Read 1 with adapter:                 146,199 (100.0%)\n",
      "  Read 2 with adapter:                 145,342 (99.4%)\n",
      "Pairs written (passing filters):       146,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,226,364 bp\n",
      "  Read 1:    43,138,552 bp\n",
      "  Read 2:    43,087,812 bp\n",
      "Total written (filtered):     80,545,108 bp (93.4%)\n",
      "  Read 1:    40,216,550 bp\n",
      "  Read 2:    40,328,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 146199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t94\t0.0\t1\t0 0 94\n",
      "19\t2388\t0.0\t1\t0 2306 82\n",
      "20\t143461\t0.0\t2\t139614 3787 60\n",
      "21\t189\t0.0\t2\t0 176 13\n",
      "22\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 145342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2493\t0.0\t1\t0 2493\n",
      "19\t142600\t0.0\t1\t139439 3161\n",
      "20\t249\t0.0\t1\t0 249\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S139_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S139_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S139_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S139_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S139_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S139_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S139_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S139_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        53,802 reads  @    104.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.61 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,802\n",
      "  Read 1 with adapter:                  53,786 (100.0%)\n",
      "  Read 2 with adapter:                  52,314 (97.2%)\n",
      "Pairs written (passing filters):        53,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,658,865 bp\n",
      "  Read 1:    15,870,808 bp\n",
      "  Read 2:    15,788,057 bp\n",
      "Total written (filtered):     29,590,393 bp (93.5%)\n",
      "  Read 1:    14,795,912 bp\n",
      "  Read 2:    14,794,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53786 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t890\t0.0\t1\t0 871 19\n",
      "20\t52731\t0.0\t2\t51329 1375 27\n",
      "21\t88\t0.0\t2\t0 82 6\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t465\t0.0\t1\t0 465\n",
      "19\t51774\t0.0\t1\t50705 1069\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S140_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S140_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S140_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S140_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S140_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S140_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S140_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S140_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,232 reads  @     91.4 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.88 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,232\n",
      "  Read 1 with adapter:                  64,216 (100.0%)\n",
      "  Read 2 with adapter:                  63,918 (99.5%)\n",
      "Pairs written (passing filters):        64,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,895,298 bp\n",
      "  Read 1:    18,947,569 bp\n",
      "  Read 2:    18,947,729 bp\n",
      "Total written (filtered):     35,398,308 bp (93.4%)\n",
      "  Read 1:    17,663,725 bp\n",
      "  Read 2:    17,734,583 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t600\t0.0\t1\t0 584 16\n",
      "20\t63457\t0.0\t2\t61876 1561 20\n",
      "21\t90\t0.0\t2\t0 88 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63918 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1434\t0.0\t1\t0 1434\n",
      "19\t62346\t0.0\t1\t60605 1741\n",
      "20\t138\t0.0\t1\t0 138\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S141_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S141_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S141_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S141_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S141_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S141_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S141_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S141_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        58,139 reads  @     95.4 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.55 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,139\n",
      "  Read 1 with adapter:                  58,133 (100.0%)\n",
      "  Read 2 with adapter:                  57,793 (99.4%)\n",
      "Pairs written (passing filters):        58,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,301,543 bp\n",
      "  Read 1:    17,150,746 bp\n",
      "  Read 2:    17,150,797 bp\n",
      "Total written (filtered):     32,042,320 bp (93.4%)\n",
      "  Read 1:    15,988,599 bp\n",
      "  Read 2:    16,053,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t599\t0.0\t1\t0 581 18\n",
      "20\t57416\t0.0\t2\t55950 1445 21\n",
      "21\t78\t0.0\t2\t0 74 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1093\t0.0\t1\t0 1093\n",
      "19\t56598\t0.0\t1\t54804 1794\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S142_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S142_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S142_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S142_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S142_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S142_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S142_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S142_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,305 reads  @     94.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.24 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,305\n",
      "  Read 1 with adapter:                  55,291 (100.0%)\n",
      "  Read 2 with adapter:                  55,017 (99.5%)\n",
      "Pairs written (passing filters):        55,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,628,817 bp\n",
      "  Read 1:    16,314,843 bp\n",
      "  Read 2:    16,313,974 bp\n",
      "Total written (filtered):     30,479,015 bp (93.4%)\n",
      "  Read 1:    15,209,448 bp\n",
      "  Read 2:    15,269,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t535\t0.0\t1\t0 527 8\n",
      "20\t54619\t0.0\t2\t53254 1341 24\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1027\t0.0\t1\t0 1027\n",
      "19\t53879\t0.0\t1\t52407 1472\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S143_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S143_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S143_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S143_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S143_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S143_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S143_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S143_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        52,684 reads  @     99.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.25 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,684\n",
      "  Read 1 with adapter:                  52,668 (100.0%)\n",
      "  Read 2 with adapter:                  52,404 (99.5%)\n",
      "Pairs written (passing filters):        52,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,083,180 bp\n",
      "  Read 1:    15,541,505 bp\n",
      "  Read 2:    15,541,675 bp\n",
      "Total written (filtered):     29,035,475 bp (93.4%)\n",
      "  Read 1:    14,488,589 bp\n",
      "  Read 2:    14,546,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52668 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t523\t0.0\t1\t0 508 15\n",
      "20\t52044\t0.0\t2\t50759 1267 18\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t993\t0.0\t1\t0 993\n",
      "19\t51305\t0.0\t1\t49851 1454\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S144_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S144_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S144_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S144_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S144_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S144_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S144_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S144_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       103,391 reads  @     92.1 µs/read;   0.65 M reads/minute\n",
      "Finished in 9.53 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,391\n",
      "  Read 1 with adapter:                 103,372 (100.0%)\n",
      "  Read 2 with adapter:                 102,829 (99.5%)\n",
      "Pairs written (passing filters):       103,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,996,827 bp\n",
      "  Read 1:    30,500,168 bp\n",
      "  Read 2:    30,496,659 bp\n",
      "Total written (filtered):     56,977,920 bp (93.4%)\n",
      "  Read 1:    28,433,451 bp\n",
      "  Read 2:    28,544,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 103372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t905\t0.0\t1\t0 876 29\n",
      "20\t102233\t0.0\t2\t99783 2420 30\n",
      "21\t142\t0.0\t2\t0 140 2\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 102829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1727\t0.0\t1\t0 1727\n",
      "19\t100936\t0.0\t1\t98001 2935\n",
      "20\t166\t0.0\t1\t0 166\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S145_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S145_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S145_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S145_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S145_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S145_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S145_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S145_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        58,289 reads  @     98.7 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.76 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,289\n",
      "  Read 1 with adapter:                  58,269 (100.0%)\n",
      "  Read 2 with adapter:                  57,874 (99.3%)\n",
      "Pairs written (passing filters):        58,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,377,277 bp\n",
      "  Read 1:    17,194,485 bp\n",
      "  Read 2:    17,182,792 bp\n",
      "Total written (filtered):     32,113,851 bp (93.4%)\n",
      "  Read 1:    16,029,541 bp\n",
      "  Read 2:    16,084,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t526\t0.0\t1\t0 512 14\n",
      "20\t57608\t0.0\t2\t56215 1373 20\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1220\t0.0\t1\t0 1220\n",
      "19\t56558\t0.0\t1\t54945 1613\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S146_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S146_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S146_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S146_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S146_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S146_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S146_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S146_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        55,325 reads  @     93.1 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.15 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,325\n",
      "  Read 1 with adapter:                  55,316 (100.0%)\n",
      "  Read 2 with adapter:                  54,910 (99.2%)\n",
      "Pairs written (passing filters):        55,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,617,822 bp\n",
      "  Read 1:    16,320,614 bp\n",
      "  Read 2:    16,297,208 bp\n",
      "Total written (filtered):     30,469,470 bp (93.4%)\n",
      "  Read 1:    15,214,777 bp\n",
      "  Read 2:    15,254,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t560\t0.0\t1\t0 538 22\n",
      "20\t54630\t0.0\t2\t53268 1349 13\n",
      "21\t75\t0.0\t2\t0 72 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t865\t0.0\t1\t0 865\n",
      "19\t53955\t0.0\t1\t52614 1341\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S147_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S147_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S147_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S147_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S147_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S147_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S147_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S147_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,196 reads  @     86.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.57 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,196\n",
      "  Read 1 with adapter:                  64,176 (100.0%)\n",
      "  Read 2 with adapter:                  61,755 (96.2%)\n",
      "Pairs written (passing filters):        64,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,737,260 bp\n",
      "  Read 1:    18,937,674 bp\n",
      "  Read 2:    18,799,586 bp\n",
      "Total written (filtered):     35,281,359 bp (93.5%)\n",
      "  Read 1:    17,654,614 bp\n",
      "  Read 2:    17,626,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t573\t0.0\t1\t0 550 23\n",
      "20\t63458\t0.0\t2\t61786 1646 26\n",
      "21\t85\t0.0\t2\t0 82 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t587\t0.0\t1\t0 587\n",
      "19\t61085\t0.0\t1\t59565 1520\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S148_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S148_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S148_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S148_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S148_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S148_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S148_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S148_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        70,857 reads  @    118.6 µs/read;   0.51 M reads/minute\n",
      "Finished in 8.41 s (119 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,857\n",
      "  Read 1 with adapter:                  70,835 (100.0%)\n",
      "  Read 2 with adapter:                  70,532 (99.5%)\n",
      "Pairs written (passing filters):        70,856 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,802,816 bp\n",
      "  Read 1:    20,901,800 bp\n",
      "  Read 2:    20,901,016 bp\n",
      "Total written (filtered):     39,048,450 bp (93.4%)\n",
      "  Read 1:    19,486,222 bp\n",
      "  Read 2:    19,562,228 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t65\t0.0\t1\t0 0 65\n",
      "19\t1495\t0.0\t1\t0 1459 36\n",
      "20\t69104\t0.0\t2\t67653 1428 23\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1462\t0.0\t1\t0 1462\n",
      "19\t68928\t0.0\t1\t67193 1735\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S149_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S149_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S149_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S149_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S149_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S149_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S149_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S149_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        49,411 reads  @    105.3 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.21 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,411\n",
      "  Read 1 with adapter:                  49,399 (100.0%)\n",
      "  Read 2 with adapter:                  49,190 (99.6%)\n",
      "Pairs written (passing filters):        49,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,152,007 bp\n",
      "  Read 1:    14,576,157 bp\n",
      "  Read 2:    14,575,850 bp\n",
      "Total written (filtered):     27,231,215 bp (93.4%)\n",
      "  Read 1:    13,589,222 bp\n",
      "  Read 2:    13,641,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1060\t0.0\t1\t0 1042 18\n",
      "20\t48179\t0.0\t2\t47233 936 10\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49190 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t827\t0.0\t1\t0 827\n",
      "19\t48289\t0.0\t1\t47080 1209\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S150_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S150_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S150_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S150_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S150_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S150_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S150_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S150_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        58,244 reads  @     92.1 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.37 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,244\n",
      "  Read 1 with adapter:                  58,223 (100.0%)\n",
      "  Read 2 with adapter:                  58,022 (99.6%)\n",
      "Pairs written (passing filters):        58,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,363,637 bp\n",
      "  Read 1:    17,181,959 bp\n",
      "  Read 2:    17,181,678 bp\n",
      "Total written (filtered):     32,099,031 bp (93.4%)\n",
      "  Read 1:    16,018,698 bp\n",
      "  Read 2:    16,080,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1253\t0.0\t1\t0 1220 33\n",
      "20\t56788\t0.0\t2\t55700 1073 15\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1182\t0.0\t1\t0 1182\n",
      "19\t56731\t0.0\t1\t55426 1305\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S151_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S151_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S151_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S151_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S151_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S151_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S151_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S151_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:06        76,114 reads  @     84.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 6.40 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,114\n",
      "  Read 1 with adapter:                  76,094 (100.0%)\n",
      "  Read 2 with adapter:                  75,813 (99.6%)\n",
      "Pairs written (passing filters):        76,114 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,906,674 bp\n",
      "  Read 1:    22,453,352 bp\n",
      "  Read 2:    22,453,322 bp\n",
      "Total written (filtered):     41,947,106 bp (93.4%)\n",
      "  Read 1:    20,932,967 bp\n",
      "  Read 2:    21,014,139 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 76094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1609\t0.0\t1\t0 1574 35\n",
      "20\t74250\t0.0\t2\t72765 1462 23\n",
      "21\t128\t0.0\t2\t0 127 1\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75813 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1446\t0.0\t1\t0 1446\n",
      "19\t74185\t0.0\t1\t72400 1785\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S152_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S152_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S152_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S152_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S152_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S152_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S152_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S152_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,705 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.45 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,705\n",
      "  Read 1 with adapter:                  65,688 (100.0%)\n",
      "  Read 2 with adapter:                  65,458 (99.6%)\n",
      "Pairs written (passing filters):        65,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,764,910 bp\n",
      "  Read 1:    19,382,696 bp\n",
      "  Read 2:    19,382,214 bp\n",
      "Total written (filtered):     36,209,786 bp (93.4%)\n",
      "  Read 1:    18,070,259 bp\n",
      "  Read 2:    18,139,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65688 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t64\t0.0\t1\t0 0 64\n",
      "19\t1375\t0.0\t1\t0 1341 34\n",
      "20\t64106\t0.0\t2\t62825 1272 9\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1111\t0.0\t1\t0 1111\n",
      "19\t64251\t0.0\t1\t62711 1540\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S153_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S153_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S153_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S153_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S153_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S153_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S153_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S153_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        82,326 reads  @    100.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 8.29 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,326\n",
      "  Read 1 with adapter:                  82,304 (100.0%)\n",
      "  Read 2 with adapter:                  81,930 (99.5%)\n",
      "Pairs written (passing filters):        82,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,558,155 bp\n",
      "  Read 1:    24,286,138 bp\n",
      "  Read 2:    24,272,017 bp\n",
      "Total written (filtered):     45,358,658 bp (93.4%)\n",
      "  Read 1:    22,641,695 bp\n",
      "  Read 2:    22,716,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 82304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t72\t0.0\t1\t0 0 72\n",
      "19\t1748\t0.0\t1\t0 1698 50\n",
      "20\t80300\t0.0\t2\t78783 1501 16\n",
      "21\t113\t0.0\t2\t0 109 4\n",
      "22\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1774\t0.0\t1\t0 1774\n",
      "19\t79998\t0.0\t1\t78152 1846\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S154_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S154_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S154_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S154_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S154_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S154_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S154_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S154_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        63,655 reads  @     94.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 6.00 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,655\n",
      "  Read 1 with adapter:                  63,635 (100.0%)\n",
      "  Read 2 with adapter:                  63,213 (99.3%)\n",
      "Pairs written (passing filters):        63,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,538,390 bp\n",
      "  Read 1:    18,778,195 bp\n",
      "  Read 2:    18,760,195 bp\n",
      "Total written (filtered):     35,066,896 bp (93.4%)\n",
      "  Read 1:    17,506,812 bp\n",
      "  Read 2:    17,560,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t71\t0.0\t1\t0 0 71\n",
      "19\t1318\t0.0\t1\t0 1281 37\n",
      "20\t62128\t0.0\t2\t60902 1210 16\n",
      "21\t93\t0.0\t2\t0 89 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1038\t0.0\t1\t0 1038\n",
      "19\t62073\t0.0\t1\t60647 1426\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S155_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S155_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S155_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S155_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S155_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S155_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S155_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S155_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        62,580 reads  @     93.1 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.83 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,580\n",
      "  Read 1 with adapter:                  62,565 (100.0%)\n",
      "  Read 2 with adapter:                  60,926 (97.4%)\n",
      "Pairs written (passing filters):        62,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,831,752 bp\n",
      "  Read 1:    18,460,673 bp\n",
      "  Read 2:    18,371,079 bp\n",
      "Total written (filtered):     34,424,665 bp (93.5%)\n",
      "  Read 1:    17,210,693 bp\n",
      "  Read 2:    17,213,972 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1368\t0.0\t1\t0 1325 43\n",
      "20\t61001\t0.0\t2\t59766 1225 10\n",
      "21\t96\t0.0\t2\t0 94 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t571\t0.0\t1\t0 571\n",
      "19\t60271\t0.0\t1\t58982 1289\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S156_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S156_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S156_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S156_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S156_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S156_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S156_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S156_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        65,196 reads  @     91.0 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.93 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,196\n",
      "  Read 1 with adapter:                  65,175 (100.0%)\n",
      "  Read 2 with adapter:                  64,848 (99.5%)\n",
      "Pairs written (passing filters):        65,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,462,301 bp\n",
      "  Read 1:    19,231,322 bp\n",
      "  Read 2:    19,230,979 bp\n",
      "Total written (filtered):     35,928,864 bp (93.4%)\n",
      "  Read 1:    17,928,755 bp\n",
      "  Read 2:    18,000,109 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65175 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t1044\t0.0\t1\t0 1014 30\n",
      "20\t63960\t0.0\t2\t62389 1551 20\n",
      "21\t103\t0.0\t2\t0 98 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1376\t0.0\t1\t0 1376\n",
      "19\t63338\t0.0\t1\t61660 1678\n",
      "20\t134\t0.0\t1\t0 134\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S157_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S157_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S157_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S157_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S157_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S157_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S157_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S157_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        60,461 reads  @     92.7 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.61 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,461\n",
      "  Read 1 with adapter:                  60,448 (100.0%)\n",
      "  Read 2 with adapter:                  60,167 (99.5%)\n",
      "Pairs written (passing filters):        60,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,670,867 bp\n",
      "  Read 1:    17,835,431 bp\n",
      "  Read 2:    17,835,436 bp\n",
      "Total written (filtered):     33,320,623 bp (93.4%)\n",
      "  Read 1:    16,627,379 bp\n",
      "  Read 2:    16,693,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t984\t0.0\t1\t0 953 31\n",
      "20\t59308\t0.0\t2\t57919 1374 15\n",
      "21\t72\t0.0\t2\t0 68 4\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1077\t0.0\t1\t0 1077\n",
      "19\t58994\t0.0\t1\t57445 1549\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S158_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S158_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S158_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S158_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S158_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S158_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S158_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S158_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,249 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.65 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,249\n",
      "  Read 1 with adapter:                  62,240 (100.0%)\n",
      "  Read 2 with adapter:                  61,970 (99.6%)\n",
      "Pairs written (passing filters):        62,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,726,439 bp\n",
      "  Read 1:    18,363,153 bp\n",
      "  Read 2:    18,363,286 bp\n",
      "Total written (filtered):     34,306,242 bp (93.4%)\n",
      "  Read 1:    17,119,271 bp\n",
      "  Read 2:    17,186,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62240 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t994\t0.0\t1\t0 967 27\n",
      "20\t61075\t0.0\t2\t59580 1474 21\n",
      "21\t106\t0.0\t2\t0 102 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1248\t0.0\t1\t0 1248\n",
      "19\t60589\t0.0\t1\t59073 1516\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S159_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S159_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S159_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S159_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S159_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S159_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S159_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S159_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,090 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.93 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,090\n",
      "  Read 1 with adapter:                  62,079 (100.0%)\n",
      "  Read 2 with adapter:                  61,796 (99.5%)\n",
      "Pairs written (passing filters):        62,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,632,071 bp\n",
      "  Read 1:    18,315,889 bp\n",
      "  Read 2:    18,316,182 bp\n",
      "Total written (filtered):     34,218,354 bp (93.4%)\n",
      "  Read 1:    17,075,233 bp\n",
      "  Read 2:    17,143,121 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t1024\t0.0\t1\t0 993 31\n",
      "20\t60877\t0.0\t2\t59385 1477 15\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61796 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1199\t0.0\t1\t0 1199\n",
      "19\t60461\t0.0\t1\t58939 1522\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S160_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S160_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S160_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S160_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S160_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S160_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S160_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S160_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,683 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.72 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,683\n",
      "  Read 1 with adapter:                  54,678 (100.0%)\n",
      "  Read 2 with adapter:                  54,451 (99.6%)\n",
      "Pairs written (passing filters):        54,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,261,717 bp\n",
      "  Read 1:    16,131,130 bp\n",
      "  Read 2:    16,130,587 bp\n",
      "Total written (filtered):     30,135,361 bp (93.4%)\n",
      "  Read 1:    15,038,454 bp\n",
      "  Read 2:    15,096,907 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t948\t0.0\t1\t0 917 31\n",
      "20\t53601\t0.0\t2\t52303 1281 17\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54451 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t985\t0.0\t1\t0 985\n",
      "19\t53370\t0.0\t1\t52052 1318\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S161_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S161_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S161_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S161_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S161_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S161_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S161_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S161_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,169 reads  @     84.8 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.00 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,169\n",
      "  Read 1 with adapter:                  47,161 (100.0%)\n",
      "  Read 2 with adapter:                  46,896 (99.4%)\n",
      "Pairs written (passing filters):        47,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,821,323 bp\n",
      "  Read 1:    13,914,559 bp\n",
      "  Read 2:    13,906,764 bp\n",
      "Total written (filtered):     25,988,708 bp (93.4%)\n",
      "  Read 1:    12,972,041 bp\n",
      "  Read 2:    13,016,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t762\t0.0\t1\t0 744 18\n",
      "20\t46265\t0.0\t2\t45158 1099 8\n",
      "21\t72\t0.0\t2\t0 70 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1011\t0.0\t1\t0 1011\n",
      "19\t45801\t0.0\t1\t44633 1168\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S162_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S162_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S162_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S162_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S162_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S162_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S162_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S162_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,075 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.84 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,075\n",
      "  Read 1 with adapter:                  46,067 (100.0%)\n",
      "  Read 2 with adapter:                  45,795 (99.4%)\n",
      "Pairs written (passing filters):        46,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,170,567 bp\n",
      "  Read 1:    13,592,104 bp\n",
      "  Read 2:    13,578,463 bp\n",
      "Total written (filtered):     25,380,221 bp (93.4%)\n",
      "  Read 1:    12,671,171 bp\n",
      "  Read 2:    12,709,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t738\t0.0\t1\t0 722 16\n",
      "20\t45236\t0.0\t2\t44144 1073 19\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45795 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t768\t0.0\t1\t0 768\n",
      "19\t44951\t0.0\t1\t43957 994\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S163_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S163_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S163_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S163_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S163_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S163_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S163_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S163_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,034 reads  @    156.0 µs/read;   0.38 M reads/minute\n",
      "Finished in 0.16 s (158 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,034\n",
      "  Read 1 with adapter:                   1,026 (99.2%)\n",
      "  Read 2 with adapter:                   1,004 (97.1%)\n",
      "Pairs written (passing filters):         1,034 (100.0%)\n",
      "\n",
      "Total basepairs processed:       608,726 bp\n",
      "  Read 1:       304,014 bp\n",
      "  Read 2:       304,712 bp\n",
      "Total written (filtered):        569,157 bp (93.5%)\n",
      "  Read 1:       283,513 bp\n",
      "  Read 2:       285,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 1026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t21\t0.0\t1\t0 21\n",
      "20\t1003\t0.0\t2\t978 25\n",
      "21\t2\t0.0\t2\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 1004 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 12\n",
      "19\t988\t0.0\t1\t957 31\n",
      "20\t4\t0.0\t1\t0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S165_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S165_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S165_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S165_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S165_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S165_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S165_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S165_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        15,435 reads  @    182.6 µs/read;   0.33 M reads/minute\n",
      "Finished in 2.82 s (183 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,435\n",
      "  Read 1 with adapter:                  15,391 (99.7%)\n",
      "  Read 2 with adapter:                  15,316 (99.2%)\n",
      "Pairs written (passing filters):        15,435 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,102,935 bp\n",
      "  Read 1:     4,552,104 bp\n",
      "  Read 2:     4,550,831 bp\n",
      "Total written (filtered):      8,504,575 bp (93.4%)\n",
      "  Read 1:     4,244,395 bp\n",
      "  Read 2:     4,260,180 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 15391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t155\t0.0\t1\t0 151 4\n",
      "20\t15179\t0.0\t2\t14696 478 5\n",
      "21\t50\t0.0\t2\t0 48 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t393\t0.0\t1\t0 393\n",
      "19\t14883\t0.0\t1\t14412 471\n",
      "20\t40\t0.0\t1\t0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S167_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S167_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S167_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S167_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S167_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S167_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S167_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S167_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,252 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.43 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,252\n",
      "  Read 1 with adapter:                  39,239 (100.0%)\n",
      "  Read 2 with adapter:                  39,025 (99.4%)\n",
      "Pairs written (passing filters):        39,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,156,843 bp\n",
      "  Read 1:    11,578,285 bp\n",
      "  Read 2:    11,578,558 bp\n",
      "Total written (filtered):     21,631,988 bp (93.4%)\n",
      "  Read 1:    10,794,129 bp\n",
      "  Read 2:    10,837,859 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t723\t0.0\t1\t0 704 19\n",
      "20\t38400\t0.0\t2\t37458 930 12\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t846\t0.0\t1\t0 846\n",
      "19\t38109\t0.0\t1\t36979 1130\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S168_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S168_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S168_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S168_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S168_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S168_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S168_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S168_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,421 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.31 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,421\n",
      "  Read 1 with adapter:                  38,413 (100.0%)\n",
      "  Read 2 with adapter:                  38,210 (99.5%)\n",
      "Pairs written (passing filters):        38,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,667,347 bp\n",
      "  Read 1:    11,334,034 bp\n",
      "  Read 2:    11,333,313 bp\n",
      "Total written (filtered):     21,174,254 bp (93.4%)\n",
      "  Read 1:    10,566,321 bp\n",
      "  Read 2:    10,607,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t626\t0.0\t1\t0 607 19\n",
      "20\t37657\t0.0\t2\t36737 907 13\n",
      "21\t65\t0.0\t2\t0 62 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t675\t0.0\t1\t0 675\n",
      "19\t37470\t0.0\t1\t36382 1088\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S169_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S169_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S169_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S169_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S169_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S169_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S169_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S169_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:05        57,932 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 5.14 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,932\n",
      "  Read 1 with adapter:                  57,919 (100.0%)\n",
      "  Read 2 with adapter:                  57,618 (99.5%)\n",
      "Pairs written (passing filters):        57,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,179,072 bp\n",
      "  Read 1:    17,089,771 bp\n",
      "  Read 2:    17,089,301 bp\n",
      "Total written (filtered):     31,927,928 bp (93.4%)\n",
      "  Read 1:    15,932,312 bp\n",
      "  Read 2:    15,995,616 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57919 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1016\t0.0\t1\t0 982 34\n",
      "20\t56715\t0.0\t2\t55234 1448 33\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1167\t0.0\t1\t0 1167\n",
      "19\t56341\t0.0\t1\t54674 1667\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S170_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S170_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S170_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S170_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S170_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S170_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S170_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S170_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,551 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.49 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,551\n",
      "  Read 1 with adapter:                  63,521 (100.0%)\n",
      "  Read 2 with adapter:                  63,186 (99.4%)\n",
      "Pairs written (passing filters):        63,551 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,493,855 bp\n",
      "  Read 1:    18,746,473 bp\n",
      "  Read 2:    18,747,382 bp\n",
      "Total written (filtered):     35,024,931 bp (93.4%)\n",
      "  Read 1:    17,477,017 bp\n",
      "  Read 2:    17,547,914 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63521 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t1098\t0.0\t1\t0 1058 40\n",
      "20\t62264\t0.0\t2\t60599 1641 24\n",
      "21\t88\t0.0\t2\t0 86 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1208\t0.0\t1\t0 1208\n",
      "19\t61836\t0.0\t1\t60061 1775\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S171_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S171_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S171_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S171_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S171_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S171_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S171_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S171_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        64,652 reads  @     97.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 6.29 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,652\n",
      "  Read 1 with adapter:                  64,621 (100.0%)\n",
      "  Read 2 with adapter:                  64,341 (99.5%)\n",
      "Pairs written (passing filters):        64,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,139,806 bp\n",
      "  Read 1:    19,071,327 bp\n",
      "  Read 2:    19,068,479 bp\n",
      "Total written (filtered):     35,626,645 bp (93.4%)\n",
      "  Read 1:    17,779,629 bp\n",
      "  Read 2:    17,847,016 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64621 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t1109\t0.0\t1\t0 1074 35\n",
      "20\t63309\t0.0\t2\t61623 1664 22\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64341 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1131\t0.0\t1\t0 1131\n",
      "19\t63095\t0.0\t1\t61243 1852\n",
      "20\t115\t0.0\t1\t0 115\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S172_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S172_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S172_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S172_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S172_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S172_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S172_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S172_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,322 reads  @     88.2 µs/read;   0.68 M reads/minute\n",
      "Finished in 5.68 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,322\n",
      "  Read 1 with adapter:                  64,301 (100.0%)\n",
      "  Read 2 with adapter:                  63,934 (99.4%)\n",
      "Pairs written (passing filters):        64,322 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,937,106 bp\n",
      "  Read 1:    18,974,582 bp\n",
      "  Read 2:    18,962,524 bp\n",
      "Total written (filtered):     35,438,588 bp (93.4%)\n",
      "  Read 1:    17,689,555 bp\n",
      "  Read 2:    17,749,033 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1113\t0.0\t1\t0 1085 28\n",
      "20\t62996\t0.0\t2\t61389 1582 25\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63934 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1389\t0.0\t1\t0 1389\n",
      "19\t62411\t0.0\t1\t60589 1822\n",
      "20\t134\t0.0\t1\t0 134\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S173_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S173_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S173_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S173_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S173_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S173_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S173_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S173_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,837 reads  @     86.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.70 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,837\n",
      "  Read 1 with adapter:                  65,811 (100.0%)\n",
      "  Read 2 with adapter:                  65,413 (99.4%)\n",
      "Pairs written (passing filters):        65,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,819,287 bp\n",
      "  Read 1:    19,421,148 bp\n",
      "  Read 2:    19,398,139 bp\n",
      "Total written (filtered):     36,262,317 bp (93.4%)\n",
      "  Read 1:    18,105,985 bp\n",
      "  Read 2:    18,156,332 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t1146\t0.0\t1\t0 1100 46\n",
      "20\t64487\t0.0\t2\t62836 1634 17\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1130\t0.0\t1\t0 1130\n",
      "19\t64193\t0.0\t1\t62588 1605\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S174_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S174_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S174_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S174_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S174_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S174_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S174_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S174_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        69,412 reads  @    101.7 µs/read;   0.59 M reads/minute\n",
      "Finished in 7.06 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,412\n",
      "  Read 1 with adapter:                  69,402 (100.0%)\n",
      "  Read 2 with adapter:                  67,801 (97.7%)\n",
      "Pairs written (passing filters):        69,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,866,038 bp\n",
      "  Read 1:    20,476,296 bp\n",
      "  Read 2:    20,389,742 bp\n",
      "Total written (filtered):     38,191,431 bp (93.5%)\n",
      "  Read 1:    19,089,357 bp\n",
      "  Read 2:    19,102,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t45\t0.0\t1\t0 0 45\n",
      "19\t1224\t0.0\t1\t0 1176 48\n",
      "20\t67970\t0.0\t2\t66094 1849 27\n",
      "21\t113\t0.0\t2\t0 110 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67801 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t641\t0.0\t1\t0 641\n",
      "19\t67070\t0.0\t1\t65588 1482\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S175_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S175_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S175_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S175_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S175_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S175_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S175_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S175_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        33,226 reads  @     96.7 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.22 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,226\n",
      "  Read 1 with adapter:                  33,218 (100.0%)\n",
      "  Read 2 with adapter:                  33,017 (99.4%)\n",
      "Pairs written (passing filters):        33,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,600,462 bp\n",
      "  Read 1:     9,801,617 bp\n",
      "  Read 2:     9,798,845 bp\n",
      "Total written (filtered):     18,309,746 bp (93.4%)\n",
      "  Read 1:     9,137,623 bp\n",
      "  Read 2:     9,172,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t423\t0.0\t1\t0 415 8\n",
      "20\t32695\t0.0\t2\t31981 703 11\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t670\t0.0\t1\t0 670\n",
      "19\t32278\t0.0\t1\t31476 802\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S176_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S176_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S176_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S176_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S176_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S176_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S176_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S176_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,491 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.20 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,491\n",
      "  Read 1 with adapter:                  55,481 (100.0%)\n",
      "  Read 2 with adapter:                  55,199 (99.5%)\n",
      "Pairs written (passing filters):        55,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,738,191 bp\n",
      "  Read 1:    16,368,807 bp\n",
      "  Read 2:    16,369,384 bp\n",
      "Total written (filtered):     30,581,374 bp (93.4%)\n",
      "  Read 1:    15,259,847 bp\n",
      "  Read 2:    15,321,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t765\t0.0\t1\t0 742 23\n",
      "20\t54577\t0.0\t2\t53181 1368 28\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1007\t0.0\t1\t0 1007\n",
      "19\t54109\t0.0\t1\t52686 1423\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S177_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S177_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S177_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S177_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S177_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S177_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S177_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S177_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,846 reads  @     80.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.86 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,846\n",
      "  Read 1 with adapter:                  47,835 (100.0%)\n",
      "  Read 2 with adapter:                  47,599 (99.5%)\n",
      "Pairs written (passing filters):        47,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,229,003 bp\n",
      "  Read 1:    14,114,549 bp\n",
      "  Read 2:    14,114,454 bp\n",
      "Total written (filtered):     26,369,391 bp (93.4%)\n",
      "  Read 1:    13,158,487 bp\n",
      "  Read 2:    13,210,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t693\t0.0\t1\t0 673 20\n",
      "20\t47020\t0.0\t2\t45833 1171 16\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47599 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t919\t0.0\t1\t0 919\n",
      "19\t46592\t0.0\t1\t45343 1249\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S178_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S178_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S178_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S178_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S178_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S178_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S178_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S178_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,024 reads  @     93.9 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.39 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,024\n",
      "  Read 1 with adapter:                  36,017 (100.0%)\n",
      "  Read 2 with adapter:                  35,803 (99.4%)\n",
      "Pairs written (passing filters):        36,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,253,563 bp\n",
      "  Read 1:    10,626,832 bp\n",
      "  Read 2:    10,626,731 bp\n",
      "Total written (filtered):     19,854,050 bp (93.4%)\n",
      "  Read 1:     9,906,963 bp\n",
      "  Read 2:     9,947,087 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t521\t0.0\t1\t0 499 22\n",
      "20\t35396\t0.0\t2\t34462 919 15\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t700\t0.0\t1\t0 700\n",
      "19\t35016\t0.0\t1\t34026 990\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S179_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S179_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S179_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S179_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S179_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S179_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S179_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S179_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,784 reads  @    100.1 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.59 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,784\n",
      "  Read 1 with adapter:                  55,776 (100.0%)\n",
      "  Read 2 with adapter:                  55,509 (99.5%)\n",
      "Pairs written (passing filters):        55,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,910,588 bp\n",
      "  Read 1:    16,456,180 bp\n",
      "  Read 2:    16,454,408 bp\n",
      "Total written (filtered):     30,741,371 bp (93.4%)\n",
      "  Read 1:    15,340,711 bp\n",
      "  Read 2:    15,400,660 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t709\t0.0\t1\t0 690 19\n",
      "20\t54928\t0.0\t2\t53493 1414 21\n",
      "21\t102\t0.0\t2\t0 99 3\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1021\t0.0\t1\t0 1021\n",
      "19\t54390\t0.0\t1\t52911 1479\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S180_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S180_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S180_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S180_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S180_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S180_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S180_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S180_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,635 reads  @     95.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.90 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,635\n",
      "  Read 1 with adapter:                  61,624 (100.0%)\n",
      "  Read 2 with adapter:                  61,274 (99.4%)\n",
      "Pairs written (passing filters):        61,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,348,608 bp\n",
      "  Read 1:    18,182,217 bp\n",
      "  Read 2:    18,166,391 bp\n",
      "Total written (filtered):     33,953,881 bp (93.4%)\n",
      "  Read 1:    16,950,457 bp\n",
      "  Read 2:    17,003,424 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61624 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t856\t0.0\t1\t0 833 23\n",
      "20\t60564\t0.0\t2\t59022 1521 21\n",
      "21\t120\t0.0\t2\t0 118 2\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1348\t0.0\t1\t0 1348\n",
      "19\t59817\t0.0\t1\t58217 1600\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S181_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S181_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S181_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S181_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S181_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S181_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S181_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S181_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        51,181 reads  @     92.5 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.74 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,181\n",
      "  Read 1 with adapter:                  51,169 (100.0%)\n",
      "  Read 2 with adapter:                  50,828 (99.3%)\n",
      "Pairs written (passing filters):        51,181 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,176,605 bp\n",
      "  Read 1:    15,098,234 bp\n",
      "  Read 2:    15,078,371 bp\n",
      "Total written (filtered):     28,188,917 bp (93.4%)\n",
      "  Read 1:    14,075,438 bp\n",
      "  Read 2:    14,113,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t659\t0.0\t1\t0 632 27\n",
      "20\t50375\t0.0\t2\t48984 1376 15\n",
      "21\t79\t0.0\t2\t0 79\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t49854\t0.0\t1\t48595 1259\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S182_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S182_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S182_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S182_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S182_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S182_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S182_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S182_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,955 reads  @     83.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.37 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,955\n",
      "  Read 1 with adapter:                  63,936 (100.0%)\n",
      "  Read 2 with adapter:                  62,274 (97.4%)\n",
      "Pairs written (passing filters):        63,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,637,405 bp\n",
      "  Read 1:    18,866,142 bp\n",
      "  Read 2:    18,771,263 bp\n",
      "Total written (filtered):     35,176,750 bp (93.5%)\n",
      "  Read 1:    17,588,174 bp\n",
      "  Read 2:    17,588,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63936 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t892\t0.0\t1\t0 867 25\n",
      "20\t62841\t0.0\t2\t61116 1691 34\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t600\t0.0\t1\t0 600\n",
      "19\t61593\t0.0\t1\t60185 1408\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S183_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S183_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S183_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S183_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S183_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S183_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S183_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S183_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        29,642 reads  @    108.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 3.20 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,642\n",
      "  Read 1 with adapter:                  29,635 (100.0%)\n",
      "  Read 2 with adapter:                  29,457 (99.4%)\n",
      "Pairs written (passing filters):        29,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,488,554 bp\n",
      "  Read 1:     8,744,206 bp\n",
      "  Read 2:     8,744,348 bp\n",
      "Total written (filtered):     16,337,146 bp (93.4%)\n",
      "  Read 1:     8,151,967 bp\n",
      "  Read 2:     8,185,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t512\t0.0\t1\t0 490 22\n",
      "20\t29045\t0.0\t2\t28256 779 10\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t580\t0.0\t1\t0 580\n",
      "19\t28811\t0.0\t1\t27987 824\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S184_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S184_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S184_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S184_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S184_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S184_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S184_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S184_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        71,086 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 6.21 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,086\n",
      "  Read 1 with adapter:                  71,063 (100.0%)\n",
      "  Read 2 with adapter:                  70,632 (99.4%)\n",
      "Pairs written (passing filters):        71,086 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,939,150 bp\n",
      "  Read 1:    20,969,071 bp\n",
      "  Read 2:    20,970,079 bp\n",
      "Total written (filtered):     39,178,233 bp (93.4%)\n",
      "  Read 1:    19,548,994 bp\n",
      "  Read 2:    19,629,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 71063 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t1255\t0.0\t1\t0 1209 46\n",
      "20\t69625\t0.0\t2\t67662 1930 33\n",
      "21\t102\t0.0\t2\t0 98 4\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1268\t0.0\t1\t0 1268\n",
      "19\t69264\t0.0\t1\t67265 1999\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S185_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S185_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S185_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S185_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S185_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S185_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S185_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S185_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        43,018 reads  @     98.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.26 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,018\n",
      "  Read 1 with adapter:                  43,010 (100.0%)\n",
      "  Read 2 with adapter:                  42,778 (99.4%)\n",
      "Pairs written (passing filters):        43,018 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,380,383 bp\n",
      "  Read 1:    12,690,159 bp\n",
      "  Read 2:    12,690,224 bp\n",
      "Total written (filtered):     23,708,848 bp (93.4%)\n",
      "  Read 1:    11,830,636 bp\n",
      "  Read 2:    11,878,212 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43010 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t726\t0.0\t1\t0 708 18\n",
      "20\t42168\t0.0\t2\t41005 1144 19\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t844\t0.0\t1\t0 844\n",
      "19\t41860\t0.0\t1\t40717 1143\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S186_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S186_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S186_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S186_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S186_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S186_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S186_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S186_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        50,822 reads  @    100.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.12 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,822\n",
      "  Read 1 with adapter:                  50,808 (100.0%)\n",
      "  Read 2 with adapter:                  50,426 (99.2%)\n",
      "Pairs written (passing filters):        50,822 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,984,403 bp\n",
      "  Read 1:    14,991,966 bp\n",
      "  Read 2:    14,992,437 bp\n",
      "Total written (filtered):     28,011,842 bp (93.4%)\n",
      "  Read 1:    13,976,616 bp\n",
      "  Read 2:    14,035,226 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t856\t0.0\t1\t0 814 42\n",
      "20\t49822\t0.0\t2\t48347 1457 18\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t980\t0.0\t1\t0 980\n",
      "19\t49349\t0.0\t1\t47759 1590\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S187_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S187_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S187_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S187_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S187_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S187_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S187_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S187_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        50,657 reads  @    113.7 µs/read;   0.53 M reads/minute\n",
      "Finished in 5.76 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,657\n",
      "  Read 1 with adapter:                  50,641 (100.0%)\n",
      "  Read 2 with adapter:                  50,262 (99.2%)\n",
      "Pairs written (passing filters):        50,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,884,202 bp\n",
      "  Read 1:    14,943,421 bp\n",
      "  Read 2:    14,940,781 bp\n",
      "Total written (filtered):     27,917,428 bp (93.4%)\n",
      "  Read 1:    13,930,833 bp\n",
      "  Read 2:    13,986,595 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t854\t0.0\t1\t0 827 27\n",
      "20\t49651\t0.0\t2\t48291 1340 20\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t885\t0.0\t1\t0 885\n",
      "19\t49284\t0.0\t1\t47715 1569\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S188_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S188_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S188_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S188_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S188_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S188_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S188_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S188_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        63,776 reads  @    106.2 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.78 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,776\n",
      "  Read 1 with adapter:                  63,759 (100.0%)\n",
      "  Read 2 with adapter:                  63,283 (99.2%)\n",
      "Pairs written (passing filters):        63,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,612,015 bp\n",
      "  Read 1:    18,812,231 bp\n",
      "  Read 2:    18,799,784 bp\n",
      "Total written (filtered):     35,136,698 bp (93.4%)\n",
      "  Read 1:    17,538,044 bp\n",
      "  Read 2:    17,598,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63759 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t1088\t0.0\t1\t0 1045 43\n",
      "20\t62484\t0.0\t2\t60788 1663 33\n",
      "21\t107\t0.0\t2\t0 104 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1333\t0.0\t1\t0 1333\n",
      "19\t61864\t0.0\t1\t60114 1750\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S189_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S189_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S189_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S189_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S189_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S189_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S189_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S189_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        55,204 reads  @    115.0 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.35 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,204\n",
      "  Read 1 with adapter:                  55,189 (100.0%)\n",
      "  Read 2 with adapter:                  54,749 (99.2%)\n",
      "Pairs written (passing filters):        55,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,548,946 bp\n",
      "  Read 1:    16,283,487 bp\n",
      "  Read 2:    16,265,459 bp\n",
      "Total written (filtered):     30,406,749 bp (93.4%)\n",
      "  Read 1:    15,180,657 bp\n",
      "  Read 2:    15,226,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1001\t0.0\t1\t0 968 33\n",
      "20\t54028\t0.0\t2\t52364 1637 27\n",
      "21\t85\t0.0\t2\t0 80 5\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t943\t0.0\t1\t0 943\n",
      "19\t53727\t0.0\t1\t52271 1456\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S190_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S190_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S190_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S190_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S190_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S190_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S190_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S190_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        74,229 reads  @    102.6 µs/read;   0.58 M reads/minute\n",
      "Finished in 7.62 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,229\n",
      "  Read 1 with adapter:                  74,206 (100.0%)\n",
      "  Read 2 with adapter:                  71,965 (96.9%)\n",
      "Pairs written (passing filters):        74,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,664,560 bp\n",
      "  Read 1:    21,895,884 bp\n",
      "  Read 2:    21,768,676 bp\n",
      "Total written (filtered):     40,814,876 bp (93.5%)\n",
      "  Read 1:    20,412,933 bp\n",
      "  Read 2:    20,401,943 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1247\t0.0\t1\t0 1204 43\n",
      "20\t72758\t0.0\t2\t70640 2077 41\n",
      "21\t112\t0.0\t2\t0 111 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 71965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t698\t0.0\t1\t0 698\n",
      "19\t71171\t0.0\t1\t69418 1753\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S191_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S191_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S191_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S191_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S191_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S191_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S191_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S191_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        59,950 reads  @     96.0 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.76 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,950\n",
      "  Read 1 with adapter:                  59,935 (100.0%)\n",
      "  Read 2 with adapter:                  59,608 (99.4%)\n",
      "Pairs written (passing filters):        59,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,367,015 bp\n",
      "  Read 1:    17,684,249 bp\n",
      "  Read 2:    17,682,766 bp\n",
      "Total written (filtered):     33,037,533 bp (93.4%)\n",
      "  Read 1:    16,486,194 bp\n",
      "  Read 2:    16,551,339 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59935 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t731\t0.0\t1\t0 712 19\n",
      "20\t59065\t0.0\t2\t57436 1610 19\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1253\t0.0\t1\t0 1253\n",
      "19\t58227\t0.0\t1\t56659 1568\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S192_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S192_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S192_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S192_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S192_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S192_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S192_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S192_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        69,485 reads  @    103.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 7.17 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,485\n",
      "  Read 1 with adapter:                  69,456 (100.0%)\n",
      "  Read 2 with adapter:                  69,174 (99.6%)\n",
      "Pairs written (passing filters):        69,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,993,447 bp\n",
      "  Read 1:    20,495,559 bp\n",
      "  Read 2:    20,497,888 bp\n",
      "Total written (filtered):     38,291,850 bp (93.4%)\n",
      "  Read 1:    19,107,144 bp\n",
      "  Read 2:    19,184,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t791\t0.0\t1\t0 765 26\n",
      "20\t68499\t0.0\t2\t66641 1830 28\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1249\t0.0\t1\t0 1249\n",
      "19\t67800\t0.0\t1\t66065 1735\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S193_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S193_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S193_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S193_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S193_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S193_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S193_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S193_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        59,955 reads  @     99.9 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.00 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,955\n",
      "  Read 1 with adapter:                  59,933 (100.0%)\n",
      "  Read 2 with adapter:                  59,692 (99.6%)\n",
      "Pairs written (passing filters):        59,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,371,476 bp\n",
      "  Read 1:    17,685,914 bp\n",
      "  Read 2:    17,685,562 bp\n",
      "Total written (filtered):     33,040,374 bp (93.4%)\n",
      "  Read 1:    16,487,875 bp\n",
      "  Read 2:    16,552,499 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t701\t0.0\t1\t0 683 18\n",
      "20\t59093\t0.0\t2\t57493 1575 25\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1192\t0.0\t1\t0 1192\n",
      "19\t58393\t0.0\t1\t56910 1483\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S194_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S194_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S194_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S194_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S194_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S194_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S194_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S194_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        53,825 reads  @    106.4 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.73 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,825\n",
      "  Read 1 with adapter:                  53,807 (100.0%)\n",
      "  Read 2 with adapter:                  53,606 (99.6%)\n",
      "Pairs written (passing filters):        53,825 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,756,042 bp\n",
      "  Read 1:    15,877,711 bp\n",
      "  Read 2:    15,878,331 bp\n",
      "Total written (filtered):     29,662,891 bp (93.4%)\n",
      "  Read 1:    14,802,121 bp\n",
      "  Read 2:    14,860,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t624\t0.0\t1\t0 608 16\n",
      "20\t53063\t0.0\t2\t51569 1458 36\n",
      "21\t74\t0.0\t2\t0 72 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1067\t0.0\t1\t0 1067\n",
      "19\t52425\t0.0\t1\t51015 1410\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S195_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S195_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S195_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S195_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S195_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S195_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S195_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S195_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09        91,084 reads  @    104.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 9.50 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,084\n",
      "  Read 1 with adapter:                  91,052 (100.0%)\n",
      "  Read 2 with adapter:                  90,718 (99.6%)\n",
      "Pairs written (passing filters):        91,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,736,283 bp\n",
      "  Read 1:    26,868,108 bp\n",
      "  Read 2:    26,868,175 bp\n",
      "Total written (filtered):     50,194,063 bp (93.4%)\n",
      "  Read 1:    25,048,059 bp\n",
      "  Read 2:    25,146,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 91052 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t52\t0.0\t1\t0 0 52\n",
      "19\t1083\t0.0\t1\t0 1044 39\n",
      "20\t89760\t0.0\t2\t87141 2575 44\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 90718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1607\t0.0\t1\t0 1607\n",
      "19\t88975\t0.0\t1\t86749 2226\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S196_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S196_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S196_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S196_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S196_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S196_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S196_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S196_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        60,175 reads  @     97.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.89 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,175\n",
      "  Read 1 with adapter:                  60,159 (100.0%)\n",
      "  Read 2 with adapter:                  59,890 (99.5%)\n",
      "Pairs written (passing filters):        60,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,489,132 bp\n",
      "  Read 1:    17,750,526 bp\n",
      "  Read 2:    17,738,606 bp\n",
      "Total written (filtered):     33,149,970 bp (93.4%)\n",
      "  Read 1:    16,548,065 bp\n",
      "  Read 2:    16,601,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t768\t0.0\t1\t0 743 25\n",
      "20\t59226\t0.0\t2\t57494 1704 28\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59890 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1312\t0.0\t1\t0 1312\n",
      "19\t58475\t0.0\t1\t57090 1385\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S197_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S197_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S197_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S197_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S197_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S197_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S197_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S197_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,439 reads  @    101.3 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.61 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,439\n",
      "  Read 1 with adapter:                  45,425 (100.0%)\n",
      "  Read 2 with adapter:                  45,168 (99.4%)\n",
      "Pairs written (passing filters):        45,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,793,318 bp\n",
      "  Read 1:    13,403,449 bp\n",
      "  Read 2:    13,389,869 bp\n",
      "Total written (filtered):     25,027,805 bp (93.4%)\n",
      "  Read 1:    12,495,431 bp\n",
      "  Read 2:    12,532,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45425 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t516\t0.0\t1\t0 491 25\n",
      "20\t44816\t0.0\t2\t43512 1279 25\n",
      "21\t64\t0.0\t2\t0 62 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45168 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t763\t0.0\t1\t0 763\n",
      "19\t44339\t0.0\t1\t43393 946\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S198_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S198_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S198_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S198_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S198_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S198_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S198_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S198_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        62,513 reads  @    106.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.68 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,513\n",
      "  Read 1 with adapter:                  62,488 (100.0%)\n",
      "  Read 2 with adapter:                  60,903 (97.4%)\n",
      "Pairs written (passing filters):        62,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,787,153 bp\n",
      "  Read 1:    18,438,883 bp\n",
      "  Read 2:    18,348,270 bp\n",
      "Total written (filtered):     34,381,392 bp (93.5%)\n",
      "  Read 1:    17,189,788 bp\n",
      "  Read 2:    17,191,604 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t743\t0.0\t1\t0 721 22\n",
      "20\t61591\t0.0\t2\t59802 1754 35\n",
      "21\t94\t0.0\t2\t0 89 5\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t569\t0.0\t1\t0 569\n",
      "19\t60256\t0.0\t1\t59063 1193\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S199_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S199_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S199_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S199_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S199_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S199_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S199_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S199_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        24,643 reads  @     82.2 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.03 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,643\n",
      "  Read 1 with adapter:                  24,638 (100.0%)\n",
      "  Read 2 with adapter:                  24,535 (99.6%)\n",
      "Pairs written (passing filters):        24,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,539,220 bp\n",
      "  Read 1:     7,269,595 bp\n",
      "  Read 2:     7,269,625 bp\n",
      "Total written (filtered):     13,580,916 bp (93.4%)\n",
      "  Read 1:     6,776,962 bp\n",
      "  Read 2:     6,803,954 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24638 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t217\t0.0\t1\t0 209 8\n",
      "20\t24332\t0.0\t2\t23857 471 4\n",
      "21\t52\t0.0\t2\t0 51 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t540\t0.0\t1\t0 540\n",
      "19\t23949\t0.0\t1\t23336 613\n",
      "20\t46\t0.0\t1\t0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S200_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S200_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S200_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S200_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S200_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S200_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S200_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S200_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        58,495 reads  @     98.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.74 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,495\n",
      "  Read 1 with adapter:                  58,485 (100.0%)\n",
      "  Read 2 with adapter:                  58,262 (99.6%)\n",
      "Pairs written (passing filters):        58,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,510,936 bp\n",
      "  Read 1:    17,255,716 bp\n",
      "  Read 2:    17,255,220 bp\n",
      "Total written (filtered):     32,235,515 bp (93.4%)\n",
      "  Read 1:    16,086,333 bp\n",
      "  Read 2:    16,149,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t526\t0.0\t1\t0 515 11\n",
      "20\t57754\t0.0\t2\t56535 1206 13\n",
      "21\t109\t0.0\t2\t0 106 3\n",
      "22\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1043\t0.0\t1\t0 1043\n",
      "19\t57116\t0.0\t1\t55652 1464\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S201_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S201_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S201_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S201_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S201_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S201_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S201_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S201_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        63,299 reads  @     91.1 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.77 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,299\n",
      "  Read 1 with adapter:                  63,291 (100.0%)\n",
      "  Read 2 with adapter:                  63,050 (99.6%)\n",
      "Pairs written (passing filters):        63,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,345,746 bp\n",
      "  Read 1:    18,672,625 bp\n",
      "  Read 2:    18,673,121 bp\n",
      "Total written (filtered):     34,883,508 bp (93.4%)\n",
      "  Read 1:    17,407,218 bp\n",
      "  Read 2:    17,476,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t567\t0.0\t1\t0 547 20\n",
      "20\t62520\t0.0\t2\t61223 1283 14\n",
      "21\t130\t0.0\t2\t0 127 3\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1241\t0.0\t1\t0 1241\n",
      "19\t61687\t0.0\t1\t60171 1516\n",
      "20\t122\t0.0\t1\t0 122\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S202_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S202_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S202_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S202_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S202_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S202_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S202_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S202_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        49,705 reads  @     93.0 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.62 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,705\n",
      "  Read 1 with adapter:                  49,693 (100.0%)\n",
      "  Read 2 with adapter:                  49,469 (99.5%)\n",
      "Pairs written (passing filters):        49,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,325,458 bp\n",
      "  Read 1:    14,662,673 bp\n",
      "  Read 2:    14,662,785 bp\n",
      "Total written (filtered):     27,392,860 bp (93.4%)\n",
      "  Read 1:    13,669,108 bp\n",
      "  Read 2:    13,723,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t436\t0.0\t1\t0 422 14\n",
      "20\t49083\t0.0\t2\t47984 1091 8\n",
      "21\t87\t0.0\t2\t0 84 3\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t986\t0.0\t1\t0 986\n",
      "19\t48375\t0.0\t1\t47093 1282\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S203_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S203_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S203_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S203_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S203_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S203_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S203_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S203_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        52,858 reads  @     89.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.71 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,858\n",
      "  Read 1 with adapter:                  52,848 (100.0%)\n",
      "  Read 2 with adapter:                  52,632 (99.6%)\n",
      "Pairs written (passing filters):        52,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,183,336 bp\n",
      "  Read 1:    15,592,511 bp\n",
      "  Read 2:    15,590,825 bp\n",
      "Total written (filtered):     29,127,473 bp (93.4%)\n",
      "  Read 1:    14,535,826 bp\n",
      "  Read 2:    14,591,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52848 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t468\t0.0\t1\t0 456 12\n",
      "20\t52196\t0.0\t2\t51078 1105 13\n",
      "21\t99\t0.0\t2\t0 95 4\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t926\t0.0\t1\t0 926\n",
      "19\t51610\t0.0\t1\t50256 1354\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S204_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S204_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S204_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S204_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S204_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S204_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S204_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S204_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        48,770 reads  @     91.0 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.44 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,770\n",
      "  Read 1 with adapter:                  48,762 (100.0%)\n",
      "  Read 2 with adapter:                  48,490 (99.4%)\n",
      "Pairs written (passing filters):        48,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,764,865 bp\n",
      "  Read 1:    14,386,990 bp\n",
      "  Read 2:    14,377,875 bp\n",
      "Total written (filtered):     26,869,555 bp (93.4%)\n",
      "  Read 1:    13,412,030 bp\n",
      "  Read 2:    13,457,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t429\t0.0\t1\t0 419 10\n",
      "20\t48175\t0.0\t2\t47151 1016 8\n",
      "21\t91\t0.0\t2\t0 87 4\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1062\t0.0\t1\t0 1062\n",
      "19\t47326\t0.0\t1\t46063 1263\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S205_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S205_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S205_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S205_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S205_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S205_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S205_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S205_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,180 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.26 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,180\n",
      "  Read 1 with adapter:                  49,164 (100.0%)\n",
      "  Read 2 with adapter:                  48,912 (99.5%)\n",
      "Pairs written (passing filters):        49,180 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,002,443 bp\n",
      "  Read 1:    14,507,742 bp\n",
      "  Read 2:    14,494,701 bp\n",
      "Total written (filtered):     27,090,877 bp (93.4%)\n",
      "  Read 1:    13,524,751 bp\n",
      "  Read 2:    13,566,126 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t446\t0.0\t1\t0 431 15\n",
      "20\t48559\t0.0\t2\t47468 1079 12\n",
      "21\t105\t0.0\t2\t0 102 3\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48912 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t823\t0.0\t1\t0 823\n",
      "19\t48019\t0.0\t1\t46954 1065\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S208_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S208_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S208_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S208_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S208_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S208_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S208_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S208_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        54,371 reads  @    100.1 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.45 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,371\n",
      "  Read 1 with adapter:                  54,361 (100.0%)\n",
      "  Read 2 with adapter:                  54,031 (99.4%)\n",
      "Pairs written (passing filters):        54,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,063,312 bp\n",
      "  Read 1:    16,038,444 bp\n",
      "  Read 2:    16,024,868 bp\n",
      "Total written (filtered):     29,951,364 bp (93.4%)\n",
      "  Read 1:    14,952,306 bp\n",
      "  Read 2:    14,999,058 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1119\t0.0\t1\t0 1091 28\n",
      "20\t53075\t0.0\t2\t51661 1398 16\n",
      "21\t97\t0.0\t2\t0 92 5\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t860\t0.0\t1\t0 860\n",
      "19\t53090\t0.0\t1\t51757 1333\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S209_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S209_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S209_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S209_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S209_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S209_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S209_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S209_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        58,380 reads  @    107.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.25 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,380\n",
      "  Read 1 with adapter:                  58,361 (100.0%)\n",
      "  Read 2 with adapter:                  58,046 (99.4%)\n",
      "Pairs written (passing filters):        58,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,442,966 bp\n",
      "  Read 1:    17,221,844 bp\n",
      "  Read 2:    17,221,122 bp\n",
      "Total written (filtered):     32,174,968 bp (93.4%)\n",
      "  Read 1:    16,055,888 bp\n",
      "  Read 2:    16,119,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t67\t0.0\t1\t0 0 67\n",
      "19\t1277\t0.0\t1\t0 1240 37\n",
      "20\t56897\t0.0\t2\t55374 1499 24\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58046 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t919\t0.0\t1\t0 919\n",
      "19\t57040\t0.0\t1\t55423 1617\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S210_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S210_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S210_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S210_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S210_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S210_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S210_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S210_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        30,358 reads  @     94.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 2.88 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,358\n",
      "  Read 1 with adapter:                  30,347 (100.0%)\n",
      "  Read 2 with adapter:                  30,209 (99.5%)\n",
      "Pairs written (passing filters):        30,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,909,544 bp\n",
      "  Read 1:     8,955,318 bp\n",
      "  Read 2:     8,954,226 bp\n",
      "Total written (filtered):     16,729,583 bp (93.4%)\n",
      "  Read 1:     8,349,047 bp\n",
      "  Read 2:     8,380,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30347 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t674\t0.0\t1\t0 659 15\n",
      "20\t29586\t0.0\t2\t28882 696 8\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t313\t0.0\t1\t0 313\n",
      "19\t29864\t0.0\t1\t29116 748\n",
      "20\t32\t0.0\t1\t0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S212_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S212_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S212_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S212_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S212_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S212_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S212_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S212_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        63,554 reads  @     95.0 µs/read;   0.63 M reads/minute\n",
      "Finished in 6.04 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,554\n",
      "  Read 1 with adapter:                  63,543 (100.0%)\n",
      "  Read 2 with adapter:                  63,235 (99.5%)\n",
      "Pairs written (passing filters):        63,554 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,483,992 bp\n",
      "  Read 1:    18,748,019 bp\n",
      "  Read 2:    18,735,973 bp\n",
      "Total written (filtered):     35,013,710 bp (93.4%)\n",
      "  Read 1:    17,478,244 bp\n",
      "  Read 2:    17,535,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63543 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t61\t0.0\t1\t0 0 61\n",
      "19\t1124\t0.0\t1\t0 1091 33\n",
      "20\t62240\t0.0\t2\t60528 1684 28\n",
      "21\t75\t0.0\t2\t0 71 4\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1058\t0.0\t1\t0 1058\n",
      "19\t62077\t0.0\t1\t60516 1561\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S213_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S213_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S213_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S213_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S213_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S213_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S213_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S213_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,449 reads  @     95.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.45 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,449\n",
      "  Read 1 with adapter:                  46,435 (100.0%)\n",
      "  Read 2 with adapter:                  46,255 (99.6%)\n",
      "Pairs written (passing filters):        46,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,402,623 bp\n",
      "  Read 1:    13,700,541 bp\n",
      "  Read 2:    13,702,082 bp\n",
      "Total written (filtered):     25,596,410 bp (93.4%)\n",
      "  Read 1:    12,772,598 bp\n",
      "  Read 2:    12,823,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t834\t0.0\t1\t0 804 30\n",
      "20\t45473\t0.0\t2\t44124 1325 24\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t654\t0.0\t1\t0 654\n",
      "19\t45522\t0.0\t1\t44356 1166\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S214_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S214_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S214_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S214_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S214_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S214_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S214_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S214_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10        93,624 reads  @    117.2 µs/read;   0.51 M reads/minute\n",
      "Finished in 10.98 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,624\n",
      "  Read 1 with adapter:                  93,593 (100.0%)\n",
      "  Read 2 with adapter:                  93,346 (99.7%)\n",
      "Pairs written (passing filters):        93,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,233,050 bp\n",
      "  Read 1:    27,617,065 bp\n",
      "  Read 2:    27,615,985 bp\n",
      "Total written (filtered):     51,590,218 bp (93.4%)\n",
      "  Read 1:    25,746,818 bp\n",
      "  Read 2:    25,843,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93593 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t71\t0.0\t1\t0 0 71\n",
      "19\t1746\t0.0\t1\t0 1693 53\n",
      "20\t91567\t0.0\t2\t88860 2664 43\n",
      "21\t143\t0.0\t2\t0 139 4\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 93346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1097\t0.0\t1\t0 1097\n",
      "19\t92141\t0.0\t1\t89782 2359\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S216_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S216_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S216_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S216_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        58,307 reads  @    106.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.23 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,307\n",
      "  Read 1 with adapter:                  58,290 (100.0%)\n",
      "  Read 2 with adapter:                  57,980 (99.4%)\n",
      "Pairs written (passing filters):        58,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,387,364 bp\n",
      "  Read 1:    17,200,034 bp\n",
      "  Read 2:    17,187,330 bp\n",
      "Total written (filtered):     32,121,307 bp (93.4%)\n",
      "  Read 1:    16,034,714 bp\n",
      "  Read 2:    16,086,593 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t586\t0.0\t1\t0 568 18\n",
      "20\t57560\t0.0\t2\t56036 1506 18\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t989\t0.0\t1\t0 989\n",
      "19\t56885\t0.0\t1\t55534 1351\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S217_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S217_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S217_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S217_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        53,490 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.42 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,490\n",
      "  Read 1 with adapter:                  53,483 (100.0%)\n",
      "  Read 2 with adapter:                  53,283 (99.6%)\n",
      "Pairs written (passing filters):        53,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,558,643 bp\n",
      "  Read 1:    15,779,204 bp\n",
      "  Read 2:    15,779,439 bp\n",
      "Total written (filtered):     29,477,803 bp (93.4%)\n",
      "  Read 1:    14,709,959 bp\n",
      "  Read 2:    14,767,844 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53483 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t490\t0.0\t1\t0 473 17\n",
      "20\t52873\t0.0\t2\t51531 1324 18\n",
      "21\t85\t0.0\t2\t0 82 3\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t845\t0.0\t1\t0 845\n",
      "19\t52375\t0.0\t1\t51130 1245\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S218_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S218_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S218_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S218_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        48,007 reads  @     91.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.40 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,007\n",
      "  Read 1 with adapter:                  47,988 (100.0%)\n",
      "  Read 2 with adapter:                  47,815 (99.6%)\n",
      "Pairs written (passing filters):        48,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,321,504 bp\n",
      "  Read 1:    14,160,824 bp\n",
      "  Read 2:    14,160,680 bp\n",
      "Total written (filtered):     26,454,138 bp (93.4%)\n",
      "  Read 1:    13,201,456 bp\n",
      "  Read 2:    13,252,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t459\t0.0\t1\t0 443 16\n",
      "20\t47413\t0.0\t2\t46184 1211 18\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t542\t0.0\t1\t0 542\n",
      "19\t47218\t0.0\t1\t46067 1151\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S220_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S220_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S220_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S220_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S220_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S220_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S220_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S220_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,931 reads  @     96.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.61 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,931\n",
      "  Read 1 with adapter:                  47,918 (100.0%)\n",
      "  Read 2 with adapter:                  47,714 (99.5%)\n",
      "Pairs written (passing filters):        47,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,270,707 bp\n",
      "  Read 1:    14,139,584 bp\n",
      "  Read 2:    14,131,123 bp\n",
      "Total written (filtered):     26,407,465 bp (93.4%)\n",
      "  Read 1:    13,182,186 bp\n",
      "  Read 2:    13,225,279 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47918 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t47\t0.0\t1\t0 0 47\n",
      "19\t1008\t0.0\t1\t0 975 33\n",
      "20\t46753\t0.0\t2\t45722 1023 8\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47714 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t810\t0.0\t1\t0 810\n",
      "19\t46816\t0.0\t1\t45749 1067\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S221_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S221_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S221_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S221_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S221_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S221_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S221_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S221_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        42,221 reads  @    126.0 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.32 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,221\n",
      "  Read 1 with adapter:                  42,215 (100.0%)\n",
      "  Read 2 with adapter:                  42,087 (99.7%)\n",
      "Pairs written (passing filters):        42,221 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,909,770 bp\n",
      "  Read 1:    12,455,043 bp\n",
      "  Read 2:    12,454,727 bp\n",
      "Total written (filtered):     23,267,210 bp (93.4%)\n",
      "  Read 1:    11,611,582 bp\n",
      "  Read 2:    11,655,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t870\t0.0\t1\t0 847 23\n",
      "20\t41202\t0.0\t2\t40327 863 12\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t613\t0.0\t1\t0 613\n",
      "19\t41415\t0.0\t1\t40419 996\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S222_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S222_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S222_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S222_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S222_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S222_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S222_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S222_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        50,268 reads  @    104.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.26 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,268\n",
      "  Read 1 with adapter:                  50,260 (100.0%)\n",
      "  Read 2 with adapter:                  50,096 (99.7%)\n",
      "Pairs written (passing filters):        50,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,656,482 bp\n",
      "  Read 1:    14,829,037 bp\n",
      "  Read 2:    14,827,445 bp\n",
      "Total written (filtered):     27,701,039 bp (93.4%)\n",
      "  Read 1:    13,824,852 bp\n",
      "  Read 2:    13,876,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50260 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1067\t0.0\t1\t0 1041 26\n",
      "20\t49029\t0.0\t2\t47829 1187 13\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t622\t0.0\t1\t0 622\n",
      "19\t49418\t0.0\t1\t48182 1236\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S224_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S224_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S224_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S224_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S224_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S224_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S224_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S224_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        56,302 reads  @    100.2 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.64 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,302\n",
      "  Read 1 with adapter:                  56,290 (100.0%)\n",
      "  Read 2 with adapter:                  55,932 (99.3%)\n",
      "Pairs written (passing filters):        56,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,203,735 bp\n",
      "  Read 1:    16,608,171 bp\n",
      "  Read 2:    16,595,564 bp\n",
      "Total written (filtered):     31,017,014 bp (93.4%)\n",
      "  Read 1:    15,483,287 bp\n",
      "  Read 2:    15,533,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t988\t0.0\t1\t0 948 40\n",
      "20\t55158\t0.0\t2\t53541 1589 28\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55932 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t946\t0.0\t1\t0 946\n",
      "19\t54911\t0.0\t1\t53323 1588\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S225_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S225_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S225_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S225_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S225_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S225_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S225_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S225_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        57,631 reads  @    109.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 6.29 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,631\n",
      "  Read 1 with adapter:                  57,619 (100.0%)\n",
      "  Read 2 with adapter:                  57,346 (99.5%)\n",
      "Pairs written (passing filters):        57,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,000,851 bp\n",
      "  Read 1:    17,000,512 bp\n",
      "  Read 2:    17,000,339 bp\n",
      "Total written (filtered):     31,760,548 bp (93.4%)\n",
      "  Read 1:    15,849,022 bp\n",
      "  Read 2:    15,911,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57619 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t965\t0.0\t1\t0 931 34\n",
      "20\t56518\t0.0\t2\t54976 1530 12\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t841\t0.0\t1\t0 841\n",
      "19\t56425\t0.0\t1\t54805 1620\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S226_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S226_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S226_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S226_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S226_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S226_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S226_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S226_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        41,267 reads  @    122.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.08 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,267\n",
      "  Read 1 with adapter:                  41,263 (100.0%)\n",
      "  Read 2 with adapter:                  41,044 (99.5%)\n",
      "Pairs written (passing filters):        41,267 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,346,142 bp\n",
      "  Read 1:    12,173,475 bp\n",
      "  Read 2:    12,172,667 bp\n",
      "Total written (filtered):     22,742,056 bp (93.4%)\n",
      "  Read 1:    11,348,810 bp\n",
      "  Read 2:    11,393,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t665\t0.0\t1\t0 647 18\n",
      "20\t40486\t0.0\t2\t39440 1030 16\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t471\t0.0\t1\t0 471\n",
      "19\t40517\t0.0\t1\t39279 1238\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S228_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S228_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S228_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S228_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S228_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S228_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S228_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S228_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        41,252 reads  @    105.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.34 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,252\n",
      "  Read 1 with adapter:                  41,238 (100.0%)\n",
      "  Read 2 with adapter:                  40,915 (99.2%)\n",
      "Pairs written (passing filters):        41,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,325,738 bp\n",
      "  Read 1:    12,168,602 bp\n",
      "  Read 2:    12,157,136 bp\n",
      "Total written (filtered):     22,724,812 bp (93.4%)\n",
      "  Read 1:    11,344,464 bp\n",
      "  Read 2:    11,380,348 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t706\t0.0\t1\t0 686 20\n",
      "20\t40409\t0.0\t2\t39349 1043 17\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t40218\t0.0\t1\t39021 1197\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S229_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S229_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S229_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S229_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S229_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S229_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S229_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S229_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,350 reads  @    108.2 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.91 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,350\n",
      "  Read 1 with adapter:                  45,319 (99.9%)\n",
      "  Read 2 with adapter:                  45,155 (99.6%)\n",
      "Pairs written (passing filters):        45,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,753,174 bp\n",
      "  Read 1:    13,375,247 bp\n",
      "  Read 2:    13,377,927 bp\n",
      "Total written (filtered):     24,990,149 bp (93.4%)\n",
      "  Read 1:    12,469,574 bp\n",
      "  Read 2:    12,520,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t777\t0.0\t1\t0 747 30\n",
      "20\t44407\t0.0\t2\t43281 1108 18\n",
      "21\t76\t0.0\t2\t0 75 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45155 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t677\t0.0\t1\t0 677\n",
      "19\t44394\t0.0\t1\t43148 1246\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S230_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S230_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S230_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S230_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S230_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S230_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S230_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S230_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,483 reads  @     97.7 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.64 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,483\n",
      "  Read 1 with adapter:                  47,232 (99.5%)\n",
      "  Read 2 with adapter:                  47,268 (99.5%)\n",
      "Pairs written (passing filters):        47,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,978,267 bp\n",
      "  Read 1:    13,973,060 bp\n",
      "  Read 2:    14,005,207 bp\n",
      "Total written (filtered):     26,136,810 bp (93.4%)\n",
      "  Read 1:    13,029,153 bp\n",
      "  Read 2:    13,107,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t812\t0.0\t1\t0 785 27\n",
      "20\t46288\t0.0\t2\t44949 1320 19\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t588\t0.0\t1\t0 588\n",
      "19\t46634\t0.0\t1\t45292 1342\n",
      "20\t46\t0.0\t1\t0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S232_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S232_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S232_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S232_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S232_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S232_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S232_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S232_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,551 reads  @     93.8 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.22 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,551\n",
      "  Read 1 with adapter:                  55,538 (100.0%)\n",
      "  Read 2 with adapter:                  55,223 (99.4%)\n",
      "Pairs written (passing filters):        55,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,761,926 bp\n",
      "  Read 1:    16,387,477 bp\n",
      "  Read 2:    16,374,449 bp\n",
      "Total written (filtered):     30,603,103 bp (93.4%)\n",
      "  Read 1:    15,277,079 bp\n",
      "  Read 2:    15,326,024 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55538 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t744\t0.0\t1\t0 732 12\n",
      "20\t54656\t0.0\t2\t53289 1348 19\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t892\t0.0\t1\t0 892\n",
      "19\t54251\t0.0\t1\t52993 1258\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S233_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S233_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S233_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S233_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S233_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S233_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S233_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S233_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        64,202 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 6.34 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,202\n",
      "  Read 1 with adapter:                  64,191 (100.0%)\n",
      "  Read 2 with adapter:                  63,902 (99.5%)\n",
      "Pairs written (passing filters):        64,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,878,599 bp\n",
      "  Read 1:    18,939,565 bp\n",
      "  Read 2:    18,939,034 bp\n",
      "Total written (filtered):     35,382,311 bp (93.4%)\n",
      "  Read 1:    17,656,566 bp\n",
      "  Read 2:    17,725,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t902\t0.0\t1\t0 876 26\n",
      "20\t63122\t0.0\t2\t61566 1529 27\n",
      "21\t109\t0.0\t2\t0 106 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63902 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t945\t0.0\t1\t0 945\n",
      "19\t62861\t0.0\t1\t61256 1605\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S234_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S234_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S234_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S234_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S234_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S234_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S234_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S234_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        43,030 reads  @     97.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.18 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,030\n",
      "  Read 1 with adapter:                  43,022 (100.0%)\n",
      "  Read 2 with adapter:                  42,855 (99.6%)\n",
      "Pairs written (passing filters):        43,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,386,775 bp\n",
      "  Read 1:    12,693,601 bp\n",
      "  Read 2:    12,693,174 bp\n",
      "Total written (filtered):     23,713,103 bp (93.4%)\n",
      "  Read 1:    11,833,737 bp\n",
      "  Read 2:    11,879,366 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t629\t0.0\t1\t0 604 25\n",
      "20\t42266\t0.0\t2\t41250 1006 10\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t488\t0.0\t1\t0 488\n",
      "19\t42316\t0.0\t1\t41246 1070\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S236_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S236_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S236_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S236_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S236_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S236_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S236_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S236_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,057 reads  @     84.6 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.66 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,057\n",
      "  Read 1 with adapter:                  55,042 (100.0%)\n",
      "  Read 2 with adapter:                  54,843 (99.6%)\n",
      "Pairs written (passing filters):        55,057 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,476,738 bp\n",
      "  Read 1:    16,241,794 bp\n",
      "  Read 2:    16,234,944 bp\n",
      "Total written (filtered):     30,335,637 bp (93.4%)\n",
      "  Read 1:    15,141,839 bp\n",
      "  Read 2:    15,193,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55042 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t934\t0.0\t1\t0 904 30\n",
      "20\t53973\t0.0\t2\t52483 1468 22\n",
      "21\t81\t0.0\t2\t0 79 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54843 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t951\t0.0\t1\t0 951\n",
      "19\t53812\t0.0\t1\t52674 1138\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S237_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S237_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S237_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S237_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S237_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S237_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S237_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S237_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,040 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.83 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,040\n",
      "  Read 1 with adapter:                  53,030 (100.0%)\n",
      "  Read 2 with adapter:                  52,812 (99.6%)\n",
      "Pairs written (passing filters):        53,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,293,111 bp\n",
      "  Read 1:    15,646,613 bp\n",
      "  Read 2:    15,646,498 bp\n",
      "Total written (filtered):     29,230,658 bp (93.4%)\n",
      "  Read 1:    14,586,860 bp\n",
      "  Read 2:    14,643,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53030 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t914\t0.0\t1\t0 879 35\n",
      "20\t51965\t0.0\t2\t50462 1479 24\n",
      "21\t79\t0.0\t2\t0 78 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t796\t0.0\t1\t0 796\n",
      "19\t51948\t0.0\t1\t50705 1243\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S238_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S238_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S238_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S238_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S238_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S238_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S238_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S238_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        61,373 reads  @    105.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.47 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,373\n",
      "  Read 1 with adapter:                  61,361 (100.0%)\n",
      "  Read 2 with adapter:                  61,151 (99.6%)\n",
      "Pairs written (passing filters):        61,373 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,208,011 bp\n",
      "  Read 1:    18,104,521 bp\n",
      "  Read 2:    18,103,490 bp\n",
      "Total written (filtered):     33,820,607 bp (93.4%)\n",
      "  Read 1:    16,878,269 bp\n",
      "  Read 2:    16,942,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1033\t0.0\t1\t0 1005 28\n",
      "20\t60143\t0.0\t2\t58468 1658 17\n",
      "21\t121\t0.0\t2\t0 119 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61151 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t773\t0.0\t1\t0 773\n",
      "19\t60322\t0.0\t1\t58784 1538\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S240_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S240_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S240_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S240_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S240_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S240_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S240_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S240_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        53,679 reads  @    101.8 µs/read;   0.59 M reads/minute\n",
      "Finished in 5.47 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,679\n",
      "  Read 1 with adapter:                  53,657 (100.0%)\n",
      "  Read 2 with adapter:                  53,364 (99.4%)\n",
      "Pairs written (passing filters):        53,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,655,363 bp\n",
      "  Read 1:    15,833,956 bp\n",
      "  Read 2:    15,821,407 bp\n",
      "Total written (filtered):     29,569,651 bp (93.4%)\n",
      "  Read 1:    14,761,390 bp\n",
      "  Read 2:    14,808,261 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t657\t0.0\t1\t0 637 20\n",
      "20\t52876\t0.0\t2\t51412 1450 14\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53364 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t849\t0.0\t1\t0 849\n",
      "19\t52436\t0.0\t1\t51167 1269\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S241_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S241_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S241_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S241_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S241_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S241_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S241_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S241_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,890 reads  @     79.9 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.07 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,890\n",
      "  Read 1 with adapter:                  50,869 (100.0%)\n",
      "  Read 2 with adapter:                  50,678 (99.6%)\n",
      "Pairs written (passing filters):        50,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,021,824 bp\n",
      "  Read 1:    15,010,184 bp\n",
      "  Read 2:    15,011,640 bp\n",
      "Total written (filtered):     28,042,798 bp (93.4%)\n",
      "  Read 1:    13,993,368 bp\n",
      "  Read 2:    14,049,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50869 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t623\t0.0\t1\t0 602 21\n",
      "20\t50129\t0.0\t2\t48791 1317 21\n",
      "21\t87\t0.0\t2\t0 80 7\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t738\t0.0\t1\t0 738\n",
      "19\t49874\t0.0\t1\t48616 1258\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S242_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S242_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S242_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S242_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S242_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S242_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S242_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S242_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        42,536 reads  @    121.2 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.17 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,536\n",
      "  Read 1 with adapter:                  42,517 (100.0%)\n",
      "  Read 2 with adapter:                  42,338 (99.5%)\n",
      "Pairs written (passing filters):        42,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,092,809 bp\n",
      "  Read 1:    12,545,823 bp\n",
      "  Read 2:    12,546,986 bp\n",
      "Total written (filtered):     23,438,983 bp (93.4%)\n",
      "  Read 1:    11,695,942 bp\n",
      "  Read 2:    11,743,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t490\t0.0\t1\t0 480 10\n",
      "20\t41937\t0.0\t2\t40777 1145 15\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t515\t0.0\t1\t0 515\n",
      "19\t41785\t0.0\t1\t40678 1107\n",
      "20\t38\t0.0\t1\t0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S244_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S244_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S244_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S244_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S244_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S244_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S244_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S244_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,032 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.03 s (81 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,032\n",
      "  Read 1 with adapter:                  50,022 (100.0%)\n",
      "  Read 2 with adapter:                  49,770 (99.5%)\n",
      "Pairs written (passing filters):        50,032 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,506,277 bp\n",
      "  Read 1:    14,759,239 bp\n",
      "  Read 2:    14,747,038 bp\n",
      "Total written (filtered):     27,561,257 bp (93.4%)\n",
      "  Read 1:    13,759,122 bp\n",
      "  Read 2:    13,802,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t460\t0.0\t1\t0 449 11\n",
      "20\t49427\t0.0\t2\t48390 1033 4\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49770 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t799\t0.0\t1\t0 799\n",
      "19\t48899\t0.0\t1\t47690 1209\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S245_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S245_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S245_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S245_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S245_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7akqjq4t/S245_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S245_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rhqc4_5f/f1aa4b7d-641c-400d-a181-cac170924afb/data/S245_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        49,651 reads  @     97.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.86 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,651\n",
      "  Read 1 with adapter:                  49,641 (100.0%)\n",
      "  Read 2 with adapter:                  49,471 (99.6%)\n",
      "Pairs written (passing filters):        49,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,293,572 bp\n",
      "  Read 1:    14,646,904 bp\n",
      "  Read 2:    14,646,668 bp\n",
      "Total written (filtered):     27,361,772 bp (93.4%)\n",
      "  Read 1:    13,654,398 bp\n",
      "  Read 2:    13,707,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t435\t0.0\t1\t0 425 10\n",
      "20\t49067\t0.0\t2\t48154 905 8\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t717\t0.0\t1\t0 717\n",
      "19\t48692\t0.0\t1\t47608 1084\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: a-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: a-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1y6kglcw')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 8ffe9ec5-6576-42c4-a176-9848ce31cdc1>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.a import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib2-zhan_manifest_run1_grid.csv \\\n",
    "  --output-path a-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.a trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences a-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences a-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.a summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data a-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization a-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('a-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: a-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: a-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: a-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.a Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs a-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 273 \\\n",
    "  --p-trunc-len-r 216 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table a-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats a-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: a-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-turq0oq6')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1e900444-6d18-4a40-8531-897ec67fab4a>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.a view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table a-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization a-table.qzv \n",
    "q2.Visualization.load('a-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: a-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.a view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization a-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib3-zhan_manifest_run2_v1.csv as PairedEndFastqManifestPhred33 to b-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S371_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S371_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S371_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S371_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S371_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S371_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S371_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S371_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        34,922 reads  @     89.2 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.12 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,922\n",
      "  Read 1 with adapter:                  34,886 (99.9%)\n",
      "  Read 2 with adapter:                  34,740 (99.5%)\n",
      "Pairs written (passing filters):        34,922 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,601,497 bp\n",
      "  Read 1:    10,301,808 bp\n",
      "  Read 2:    10,299,689 bp\n",
      "Total written (filtered):     19,244,877 bp (93.4%)\n",
      "  Read 1:     9,604,674 bp\n",
      "  Read 2:     9,640,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34886 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t605\t0.0\t1\t0 575 30\n",
      "20\t34185\t0.0\t2\t32471 1668 46\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t658\t0.0\t1\t0 658\n",
      "19\t33998\t0.0\t1\t33093 905\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S372_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S372_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S372_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S372_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S372_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S372_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S372_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S372_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        43,680 reads  @    102.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.48 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,680\n",
      "  Read 1 with adapter:                  43,645 (99.9%)\n",
      "  Read 2 with adapter:                  43,430 (99.4%)\n",
      "Pairs written (passing filters):        43,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,770,322 bp\n",
      "  Read 1:    12,885,302 bp\n",
      "  Read 2:    12,885,020 bp\n",
      "Total written (filtered):     24,073,677 bp (93.4%)\n",
      "  Read 1:    12,013,201 bp\n",
      "  Read 2:    12,060,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t844\t0.0\t1\t0 814 30\n",
      "20\t42685\t0.0\t2\t40476 2169 40\n",
      "21\t67\t0.0\t2\t0 67\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t699\t0.0\t1\t0 699\n",
      "19\t42658\t0.0\t1\t41325 1333\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S373_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S373_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S373_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S373_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S373_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S373_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S373_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S373_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        42,852 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.26 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,852\n",
      "  Read 1 with adapter:                  42,818 (99.9%)\n",
      "  Read 2 with adapter:                  42,544 (99.3%)\n",
      "Pairs written (passing filters):        42,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,281,156 bp\n",
      "  Read 1:    12,641,091 bp\n",
      "  Read 2:    12,640,065 bp\n",
      "Total written (filtered):     23,617,787 bp (93.4%)\n",
      "  Read 1:    11,785,521 bp\n",
      "  Read 2:    11,832,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42818 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t833\t0.0\t1\t0 802 31\n",
      "20\t41850\t0.0\t2\t39563 2222 65\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t615\t0.0\t1\t0 615\n",
      "19\t41851\t0.0\t1\t40551 1300\n",
      "20\t78\t0.0\t1\t0 78\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S374_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S374_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S374_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S374_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S374_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S374_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S374_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S374_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,712 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.61 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,712\n",
      "  Read 1 with adapter:                  56,663 (99.9%)\n",
      "  Read 2 with adapter:                  56,488 (99.6%)\n",
      "Pairs written (passing filters):        56,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,458,926 bp\n",
      "  Read 1:    16,729,117 bp\n",
      "  Read 2:    16,729,809 bp\n",
      "Total written (filtered):     31,254,037 bp (93.4%)\n",
      "  Read 1:    15,596,821 bp\n",
      "  Read 2:    15,657,216 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56663 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t1010\t0.0\t1\t0 962 48\n",
      "20\t55495\t0.0\t2\t52671 2747 77\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t795\t0.0\t1\t0 795\n",
      "19\t55577\t0.0\t1\t54093 1484\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S375_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S375_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S375_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S375_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S375_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S375_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S375_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S375_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,436 reads  @     95.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.53 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,436\n",
      "  Read 1 with adapter:                  47,402 (99.9%)\n",
      "  Read 2 with adapter:                  47,238 (99.6%)\n",
      "Pairs written (passing filters):        47,436 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,986,206 bp\n",
      "  Read 1:    13,993,131 bp\n",
      "  Read 2:    13,993,075 bp\n",
      "Total written (filtered):     26,142,057 bp (93.4%)\n",
      "  Read 1:    13,045,914 bp\n",
      "  Read 2:    13,096,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t873\t0.0\t1\t0 829 44\n",
      "20\t46395\t0.0\t2\t43970 2369 56\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t667\t0.0\t1\t0 667\n",
      "19\t46494\t0.0\t1\t45195 1299\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S376_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S376_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S376_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S376_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S376_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S376_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S376_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S376_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        54,090 reads  @    102.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 5.53 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,090\n",
      "  Read 1 with adapter:                  54,035 (99.9%)\n",
      "  Read 2 with adapter:                  53,716 (99.3%)\n",
      "Pairs written (passing filters):        54,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,898,141 bp\n",
      "  Read 1:    15,955,934 bp\n",
      "  Read 2:    15,942,207 bp\n",
      "Total written (filtered):     29,798,829 bp (93.4%)\n",
      "  Read 1:    14,876,281 bp\n",
      "  Read 2:    14,922,548 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54035 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t1123\t0.0\t1\t0 1057 66\n",
      "20\t52724\t0.0\t2\t50032 2602 90\n",
      "21\t104\t0.0\t2\t0 100 4\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1040\t0.0\t1\t0 1040\n",
      "19\t52581\t0.0\t1\t51057 1524\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S377_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S377_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S377_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S377_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S377_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S377_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S377_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S377_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        50,747 reads  @    100.8 µs/read;   0.59 M reads/minute\n",
      "Finished in 5.12 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,747\n",
      "  Read 1 with adapter:                  50,683 (99.9%)\n",
      "  Read 2 with adapter:                  50,401 (99.3%)\n",
      "Pairs written (passing filters):        50,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,918,145 bp\n",
      "  Read 1:    14,968,367 bp\n",
      "  Read 2:    14,949,778 bp\n",
      "Total written (filtered):     27,948,467 bp (93.4%)\n",
      "  Read 1:    13,955,626 bp\n",
      "  Read 2:    13,992,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t958\t0.0\t1\t0 907 51\n",
      "20\t49578\t0.0\t2\t46988 2524 66\n",
      "21\t71\t0.0\t2\t0 69 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t740\t0.0\t1\t0 740\n",
      "19\t49603\t0.0\t1\t48387 1216\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S378_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S378_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S378_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S378_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S378_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S378_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S378_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S378_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        49,798 reads  @    108.2 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.39 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,798\n",
      "  Read 1 with adapter:                  49,758 (99.9%)\n",
      "  Read 2 with adapter:                  48,699 (97.8%)\n",
      "Pairs written (passing filters):        49,798 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,321,258 bp\n",
      "  Read 1:    14,690,392 bp\n",
      "  Read 2:    14,630,866 bp\n",
      "Total written (filtered):     27,401,925 bp (93.5%)\n",
      "  Read 1:    13,696,030 bp\n",
      "  Read 2:    13,705,895 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49758 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t853\t0.0\t1\t0 810 43\n",
      "20\t48760\t0.0\t2\t46412 2297 51\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48699 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t367\t0.0\t1\t0 367\n",
      "19\t48275\t0.0\t1\t47206 1069\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S379_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S379_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S379_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S379_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S379_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S379_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S379_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S379_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,104 reads  @     88.0 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.59 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,104\n",
      "  Read 1 with adapter:                  52,059 (99.9%)\n",
      "  Read 2 with adapter:                  51,881 (99.6%)\n",
      "Pairs written (passing filters):        52,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,740,591 bp\n",
      "  Read 1:    15,370,657 bp\n",
      "  Read 2:    15,369,934 bp\n",
      "Total written (filtered):     28,715,406 bp (93.4%)\n",
      "  Read 1:    14,330,230 bp\n",
      "  Read 2:    14,385,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t809\t0.0\t1\t0 771 38\n",
      "20\t51111\t0.0\t2\t48360 2684 67\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1080\t0.0\t1\t0 1080\n",
      "19\t50702\t0.0\t1\t49440 1262\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S380_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S380_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S380_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S380_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S380_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S380_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S380_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S380_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        66,880 reads  @     90.9 µs/read;   0.66 M reads/minute\n",
      "Finished in 6.08 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,880\n",
      "  Read 1 with adapter:                  66,822 (99.9%)\n",
      "  Read 2 with adapter:                  66,590 (99.6%)\n",
      "Pairs written (passing filters):        66,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,458,391 bp\n",
      "  Read 1:    19,729,376 bp\n",
      "  Read 2:    19,729,015 bp\n",
      "Total written (filtered):     36,858,743 bp (93.4%)\n",
      "  Read 1:    18,393,909 bp\n",
      "  Read 2:    18,464,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t59\t0.0\t1\t0 0 59\n",
      "19\t1064\t0.0\t1\t0 1012 52\n",
      "20\t65543\t0.0\t2\t62010 3446 87\n",
      "21\t103\t0.0\t2\t0 103\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1139\t0.0\t1\t0 1139\n",
      "19\t65341\t0.0\t1\t63639 1702\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S381_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S381_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S381_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S381_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S381_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S381_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S381_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S381_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,257 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.01 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,257\n",
      "  Read 1 with adapter:                  42,214 (99.9%)\n",
      "  Read 2 with adapter:                  42,102 (99.6%)\n",
      "Pairs written (passing filters):        42,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,931,146 bp\n",
      "  Read 1:    12,465,790 bp\n",
      "  Read 2:    12,465,356 bp\n",
      "Total written (filtered):     23,288,175 bp (93.4%)\n",
      "  Read 1:    11,622,120 bp\n",
      "  Read 2:    11,666,055 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t673\t0.0\t1\t0 635 38\n",
      "20\t41426\t0.0\t2\t39115 2267 44\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t728\t0.0\t1\t0 728\n",
      "19\t41283\t0.0\t1\t40300 983\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S382_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S382_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S382_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S382_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S382_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S382_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S382_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S382_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,865 reads  @     67.3 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.90 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,865\n",
      "  Read 1 with adapter:                  57,810 (99.9%)\n",
      "  Read 2 with adapter:                  57,659 (99.6%)\n",
      "Pairs written (passing filters):        57,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,138,047 bp\n",
      "  Read 1:    17,069,202 bp\n",
      "  Read 2:    17,068,845 bp\n",
      "Total written (filtered):     31,888,018 bp (93.4%)\n",
      "  Read 1:    15,913,888 bp\n",
      "  Read 2:    15,974,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t967\t0.0\t1\t0 913 54\n",
      "20\t56697\t0.0\t2\t53736 2891 70\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t938\t0.0\t1\t0 938\n",
      "19\t56589\t0.0\t1\t55296 1293\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S383_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S383_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S383_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S383_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S383_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S383_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S383_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S383_25_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,168 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.74 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,168\n",
      "  Read 1 with adapter:                  52,112 (99.9%)\n",
      "  Read 2 with adapter:                  52,015 (99.7%)\n",
      "Pairs written (passing filters):        52,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,777,913 bp\n",
      "  Read 1:    15,389,543 bp\n",
      "  Read 2:    15,388,370 bp\n",
      "Total written (filtered):     28,748,854 bp (93.4%)\n",
      "  Read 1:    14,348,105 bp\n",
      "  Read 2:    14,400,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t856\t0.0\t1\t0 811 45\n",
      "20\t51113\t0.0\t2\t48454 2592 67\n",
      "21\t76\t0.0\t2\t0 74 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52015 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t758\t0.0\t1\t0 758\n",
      "19\t51163\t0.0\t1\t50001 1162\n",
      "20\t94\t0.0\t1\t0 94\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S384_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S384_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S384_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S384_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S384_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S384_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S384_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S384_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,804 reads  @     73.7 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.34 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,804\n",
      "  Read 1 with adapter:                  58,748 (99.9%)\n",
      "  Read 2 with adapter:                  58,519 (99.5%)\n",
      "Pairs written (passing filters):        58,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,681,419 bp\n",
      "  Read 1:    17,346,959 bp\n",
      "  Read 2:    17,334,460 bp\n",
      "Total written (filtered):     32,396,604 bp (93.4%)\n",
      "  Read 1:    16,172,914 bp\n",
      "  Read 2:    16,223,690 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t1007\t0.0\t1\t0 959 48\n",
      "20\t57592\t0.0\t2\t54523 2992 77\n",
      "21\t78\t0.0\t2\t0 78\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58519 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1187\t0.0\t1\t0 1187\n",
      "19\t57236\t0.0\t1\t55866 1370\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S385_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S385_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S385_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S385_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S385_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S385_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S385_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S385_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,315 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.01 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,315\n",
      "  Read 1 with adapter:                  54,274 (99.9%)\n",
      "  Read 2 with adapter:                  53,938 (99.3%)\n",
      "Pairs written (passing filters):        54,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,021,825 bp\n",
      "  Read 1:    16,022,903 bp\n",
      "  Read 2:    15,998,922 bp\n",
      "Total written (filtered):     29,913,101 bp (93.4%)\n",
      "  Read 1:    14,938,249 bp\n",
      "  Read 2:    14,974,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t892\t0.0\t1\t0 850 42\n",
      "20\t53230\t0.0\t2\t50308 2853 69\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53938 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t820\t0.0\t1\t0 820\n",
      "19\t53050\t0.0\t1\t51775 1275\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S386_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S386_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S386_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S386_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S386_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S386_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S386_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S386_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        54,277 reads  @     78.8 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.28 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,277\n",
      "  Read 1 with adapter:                  54,231 (99.9%)\n",
      "  Read 2 with adapter:                  53,132 (97.9%)\n",
      "Pairs written (passing filters):        54,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,960,502 bp\n",
      "  Read 1:    16,011,201 bp\n",
      "  Read 2:    15,949,301 bp\n",
      "Total written (filtered):     29,867,461 bp (93.5%)\n",
      "  Read 1:    14,927,296 bp\n",
      "  Read 2:    14,940,165 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t807\t0.0\t1\t0 768 39\n",
      "20\t53282\t0.0\t2\t50403 2801 78\n",
      "21\t84\t0.0\t2\t0 82 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t427\t0.0\t1\t0 427\n",
      "19\t52650\t0.0\t1\t51611 1039\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S387_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S387_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S387_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S387_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S387_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S387_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S387_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S387_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,492 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 4.00 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,492\n",
      "  Read 1 with adapter:                  56,434 (99.9%)\n",
      "  Read 2 with adapter:                  56,179 (99.4%)\n",
      "Pairs written (passing filters):        56,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,325,713 bp\n",
      "  Read 1:    16,664,885 bp\n",
      "  Read 2:    16,660,828 bp\n",
      "Total written (filtered):     31,131,060 bp (93.4%)\n",
      "  Read 1:    15,536,591 bp\n",
      "  Read 2:    15,594,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t486\t0.0\t1\t0 467 19\n",
      "20\t55827\t0.0\t2\t53048 2718 61\n",
      "21\t70\t0.0\t2\t0 69 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1167\t0.0\t1\t0 1167\n",
      "19\t54887\t0.0\t1\t53391 1496\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S388_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S388_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S388_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S388_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S388_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S388_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S388_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S388_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,210 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.66 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,210\n",
      "  Read 1 with adapter:                  50,152 (99.9%)\n",
      "  Read 2 with adapter:                  49,920 (99.4%)\n",
      "Pairs written (passing filters):        50,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,623,381 bp\n",
      "  Read 1:    14,811,935 bp\n",
      "  Read 2:    14,811,446 bp\n",
      "Total written (filtered):     27,673,014 bp (93.4%)\n",
      "  Read 1:    13,809,226 bp\n",
      "  Read 2:    13,863,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t430\t0.0\t1\t0 411 19\n",
      "20\t49609\t0.0\t2\t46997 2562 50\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t895\t0.0\t1\t0 895\n",
      "19\t48952\t0.0\t1\t47286 1666\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S389_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S389_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S389_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S389_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S389_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S389_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S389_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S389_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,850 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.27 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,850\n",
      "  Read 1 with adapter:                  45,789 (99.9%)\n",
      "  Read 2 with adapter:                  45,566 (99.4%)\n",
      "Pairs written (passing filters):        45,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,050,254 bp\n",
      "  Read 1:    13,525,498 bp\n",
      "  Read 2:    13,524,756 bp\n",
      "Total written (filtered):     25,269,709 bp (93.4%)\n",
      "  Read 1:    12,610,032 bp\n",
      "  Read 2:    12,659,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45789 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t378\t0.0\t1\t0 363 15\n",
      "20\t45311\t0.0\t2\t42876 2367 68\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45566 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t755\t0.0\t1\t0 755\n",
      "19\t44731\t0.0\t1\t43466 1265\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S390_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S390_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S390_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S390_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S390_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S390_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S390_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S390_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,374 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.90 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,374\n",
      "  Read 1 with adapter:                  57,318 (99.9%)\n",
      "  Read 2 with adapter:                  57,086 (99.5%)\n",
      "Pairs written (passing filters):        57,374 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,850,210 bp\n",
      "  Read 1:    16,924,985 bp\n",
      "  Read 2:    16,925,225 bp\n",
      "Total written (filtered):     31,620,453 bp (93.4%)\n",
      "  Read 1:    15,778,992 bp\n",
      "  Read 2:    15,841,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t469\t0.0\t1\t0 450 19\n",
      "20\t56711\t0.0\t2\t53779 2863 69\n",
      "21\t70\t0.0\t2\t0 67 3\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t981\t0.0\t1\t0 981\n",
      "19\t55994\t0.0\t1\t54359 1635\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S391_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S391_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S391_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S391_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S391_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S391_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S391_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S391_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,652 reads  @     65.9 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.88 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,652\n",
      "  Read 1 with adapter:                  43,601 (99.9%)\n",
      "  Read 2 with adapter:                  43,463 (99.6%)\n",
      "Pairs written (passing filters):        43,651 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,752,816 bp\n",
      "  Read 1:    12,877,327 bp\n",
      "  Read 2:    12,875,489 bp\n",
      "Total written (filtered):     24,055,574 bp (93.4%)\n",
      "  Read 1:    12,005,300 bp\n",
      "  Read 2:    12,050,274 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t335\t0.0\t1\t0 323 12\n",
      "20\t43166\t0.0\t2\t40978 2132 56\n",
      "21\t53\t0.0\t2\t0 52 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43463 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t649\t0.0\t1\t0 649\n",
      "19\t42747\t0.0\t1\t41496 1251\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S392_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S392_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S392_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S392_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S392_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S392_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S392_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S392_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        59,667 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 4.20 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,667\n",
      "  Read 1 with adapter:                  59,611 (99.9%)\n",
      "  Read 2 with adapter:                  59,255 (99.3%)\n",
      "Pairs written (passing filters):        59,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,191,654 bp\n",
      "  Read 1:    17,601,742 bp\n",
      "  Read 2:    17,589,912 bp\n",
      "Total written (filtered):     32,875,069 bp (93.4%)\n",
      "  Read 1:    16,409,921 bp\n",
      "  Read 2:    16,465,148 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59611 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t520\t0.0\t1\t0 505 15\n",
      "20\t58955\t0.0\t2\t55847 3031 77\n",
      "21\t71\t0.0\t2\t0 71\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1158\t0.0\t1\t0 1158\n",
      "19\t58020\t0.0\t1\t56319 1701\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S393_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S393_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S393_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S393_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S393_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S393_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S393_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S393_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,235 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.99 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,235\n",
      "  Read 1 with adapter:                  57,182 (99.9%)\n",
      "  Read 2 with adapter:                  56,799 (99.2%)\n",
      "Pairs written (passing filters):        57,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,741,321 bp\n",
      "  Read 1:    16,884,116 bp\n",
      "  Read 2:    16,857,205 bp\n",
      "Total written (filtered):     31,519,760 bp (93.4%)\n",
      "  Read 1:    15,740,940 bp\n",
      "  Read 2:    15,778,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t538\t0.0\t1\t0 511 27\n",
      "20\t56525\t0.0\t2\t53697 2762 66\n",
      "21\t72\t0.0\t2\t0 69 3\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t871\t0.0\t1\t0 871\n",
      "19\t55853\t0.0\t1\t54437 1416\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S394_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S394_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S394_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S394_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S394_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S394_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S394_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S394_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,880 reads  @     72.4 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.12 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,880\n",
      "  Read 1 with adapter:                  56,828 (99.9%)\n",
      "  Read 2 with adapter:                  55,006 (96.7%)\n",
      "Pairs written (passing filters):        56,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,455,116 bp\n",
      "  Read 1:    16,779,478 bp\n",
      "  Read 2:    16,675,638 bp\n",
      "Total written (filtered):     31,274,198 bp (93.5%)\n",
      "  Read 1:    15,643,283 bp\n",
      "  Read 2:    15,630,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t453\t0.0\t1\t0 434 19\n",
      "20\t56248\t0.0\t2\t53410 2761 77\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t464\t0.0\t1\t0 464\n",
      "19\t54469\t0.0\t1\t53010 1459\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S395_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S395_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S395_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S395_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S395_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S395_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S395_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S395_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,921 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.85 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,921\n",
      "  Read 1 with adapter:                  55,870 (99.9%)\n",
      "  Read 2 with adapter:                  55,704 (99.6%)\n",
      "Pairs written (passing filters):        55,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,991,690 bp\n",
      "  Read 1:    16,496,684 bp\n",
      "  Read 2:    16,495,006 bp\n",
      "Total written (filtered):     30,817,742 bp (93.4%)\n",
      "  Read 1:    15,380,148 bp\n",
      "  Read 2:    15,437,594 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55870 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t904\t0.0\t1\t0 865 39\n",
      "20\t54843\t0.0\t2\t52363 2431 49\n",
      "21\t66\t0.0\t2\t0 64 2\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1066\t0.0\t1\t0 1066\n",
      "19\t54536\t0.0\t1\t53259 1277\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S396_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S396_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S396_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S396_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S396_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S396_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S396_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S396_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,357 reads  @     66.6 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.95 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,357\n",
      "  Read 1 with adapter:                  59,320 (99.9%)\n",
      "  Read 2 with adapter:                  59,125 (99.6%)\n",
      "Pairs written (passing filters):        59,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,019,715 bp\n",
      "  Read 1:    17,510,040 bp\n",
      "  Read 2:    17,509,675 bp\n",
      "Total written (filtered):     32,711,847 bp (93.4%)\n",
      "  Read 1:    16,324,665 bp\n",
      "  Read 2:    16,387,182 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t1117\t0.0\t1\t0 1075 42\n",
      "20\t58008\t0.0\t2\t55251 2690 67\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t979\t0.0\t1\t0 979\n",
      "19\t58049\t0.0\t1\t56677 1372\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S397_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S397_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S397_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S397_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S397_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S397_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S397_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S397_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,035 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.60 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,035\n",
      "  Read 1 with adapter:                  52,995 (99.9%)\n",
      "  Read 2 with adapter:                  52,856 (99.7%)\n",
      "Pairs written (passing filters):        53,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,290,042 bp\n",
      "  Read 1:    15,645,309 bp\n",
      "  Read 2:    15,644,733 bp\n",
      "Total written (filtered):     29,227,700 bp (93.4%)\n",
      "  Read 1:    14,586,436 bp\n",
      "  Read 2:    14,641,264 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52995 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1049\t0.0\t1\t0 980 69\n",
      "20\t51769\t0.0\t2\t49299 2407 63\n",
      "21\t84\t0.0\t2\t0 81 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t51837\t0.0\t1\t50630 1207\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S398_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S398_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S398_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S398_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S398_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S398_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S398_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S398_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        63,940 reads  @     66.8 µs/read;   0.90 M reads/minute\n",
      "Finished in 4.28 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,940\n",
      "  Read 1 with adapter:                  63,900 (99.9%)\n",
      "  Read 2 with adapter:                  63,692 (99.6%)\n",
      "Pairs written (passing filters):        63,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,724,095 bp\n",
      "  Read 1:    18,862,277 bp\n",
      "  Read 2:    18,861,818 bp\n",
      "Total written (filtered):     35,238,017 bp (93.4%)\n",
      "  Read 1:    17,585,415 bp\n",
      "  Read 2:    17,652,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1218\t0.0\t1\t0 1162 56\n",
      "20\t62483\t0.0\t2\t59491 2927 65\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1055\t0.0\t1\t0 1055\n",
      "19\t62514\t0.0\t1\t61044 1470\n",
      "20\t123\t0.0\t1\t0 123\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S399_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S399_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S399_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S399_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S399_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S399_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S399_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S399_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,818 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.53 s (66 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,818\n",
      "  Read 1 with adapter:                  53,776 (99.9%)\n",
      "  Read 2 with adapter:                  53,656 (99.7%)\n",
      "Pairs written (passing filters):        53,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,751,106 bp\n",
      "  Read 1:    15,876,287 bp\n",
      "  Read 2:    15,874,819 bp\n",
      "Total written (filtered):     29,657,732 bp (93.4%)\n",
      "  Read 1:    14,801,693 bp\n",
      "  Read 2:    14,856,039 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t994\t0.0\t1\t0 944 50\n",
      "20\t52632\t0.0\t2\t50189 2390 53\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t761\t0.0\t1\t0 761\n",
      "19\t52818\t0.0\t1\t51671 1147\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S400_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S400_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S400_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S400_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S400_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S400_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S400_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S400_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        59,452 reads  @     66.8 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.98 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,452\n",
      "  Read 1 with adapter:                  59,402 (99.9%)\n",
      "  Read 2 with adapter:                  59,199 (99.6%)\n",
      "Pairs written (passing filters):        59,452 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,070,275 bp\n",
      "  Read 1:    17,538,313 bp\n",
      "  Read 2:    17,531,962 bp\n",
      "Total written (filtered):     32,759,537 bp (93.4%)\n",
      "  Read 1:    16,351,375 bp\n",
      "  Read 2:    16,408,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1153\t0.0\t1\t0 1118 35\n",
      "20\t58064\t0.0\t2\t55359 2638 67\n",
      "21\t91\t0.0\t2\t0 90 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1076\t0.0\t1\t0 1076\n",
      "19\t58028\t0.0\t1\t56687 1341\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S401_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S401_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S401_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S401_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S401_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S401_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S401_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S401_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,699 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 4.09 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,699\n",
      "  Read 1 with adapter:                  61,646 (99.9%)\n",
      "  Read 2 with adapter:                  61,342 (99.4%)\n",
      "Pairs written (passing filters):        61,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,380,670 bp\n",
      "  Read 1:    18,201,099 bp\n",
      "  Read 2:    18,179,571 bp\n",
      "Total written (filtered):     33,984,296 bp (93.4%)\n",
      "  Read 1:    16,969,381 bp\n",
      "  Read 2:    17,014,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t63\t0.0\t1\t0 0 63\n",
      "19\t1229\t0.0\t1\t0 1176 53\n",
      "20\t60232\t0.0\t2\t57250 2921 61\n",
      "21\t91\t0.0\t2\t0 91\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t927\t0.0\t1\t0 927\n",
      "19\t60330\t0.0\t1\t58944 1386\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S402_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S402_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S402_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S402_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S402_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S402_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S402_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S402_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,496 reads  @     65.0 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.74 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,496\n",
      "  Read 1 with adapter:                  57,456 (99.9%)\n",
      "  Read 2 with adapter:                  56,224 (97.8%)\n",
      "Pairs written (passing filters):        57,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,852,633 bp\n",
      "  Read 1:    16,960,831 bp\n",
      "  Read 2:    16,891,802 bp\n",
      "Total written (filtered):     31,636,577 bp (93.5%)\n",
      "  Read 1:    15,812,648 bp\n",
      "  Read 2:    15,823,929 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t969\t0.0\t1\t0 936 33\n",
      "20\t56335\t0.0\t2\t53701 2573 61\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56224 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t466\t0.0\t1\t0 466\n",
      "19\t55675\t0.0\t1\t54590 1085\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S403_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S403_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S403_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S403_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S403_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S403_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S403_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S403_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,935 reads  @     65.3 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.53 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,935\n",
      "  Read 1 with adapter:                  53,887 (99.9%)\n",
      "  Read 2 with adapter:                  53,678 (99.5%)\n",
      "Pairs written (passing filters):        53,935 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,819,583 bp\n",
      "  Read 1:    15,909,389 bp\n",
      "  Read 2:    15,910,194 bp\n",
      "Total written (filtered):     29,723,614 bp (93.4%)\n",
      "  Read 1:    14,832,351 bp\n",
      "  Read 2:    14,891,263 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t776\t0.0\t1\t0 747 29\n",
      "20\t52993\t0.0\t2\t50394 2539 60\n",
      "21\t82\t0.0\t2\t0 78 4\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1056\t0.0\t1\t0 1056\n",
      "19\t52517\t0.0\t1\t51199 1318\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S404_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S404_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S404_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S404_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S404_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S404_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S404_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S404_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,421 reads  @     70.2 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.40 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,421\n",
      "  Read 1 with adapter:                  48,378 (99.9%)\n",
      "  Read 2 with adapter:                  48,201 (99.5%)\n",
      "Pairs written (passing filters):        48,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,567,768 bp\n",
      "  Read 1:    14,284,170 bp\n",
      "  Read 2:    14,283,598 bp\n",
      "Total written (filtered):     26,685,734 bp (93.4%)\n",
      "  Read 1:    13,317,241 bp\n",
      "  Read 2:    13,368,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t735\t0.0\t1\t0 702 33\n",
      "20\t47509\t0.0\t2\t45122 2325 62\n",
      "21\t76\t0.0\t2\t0 76\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t801\t0.0\t1\t0 801\n",
      "19\t47313\t0.0\t1\t46056 1257\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S405_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S405_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S405_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S405_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S405_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S405_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S405_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S405_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,890 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.08 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,890\n",
      "  Read 1 with adapter:                  48,846 (99.9%)\n",
      "  Read 2 with adapter:                  48,659 (99.5%)\n",
      "Pairs written (passing filters):        48,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,844,407 bp\n",
      "  Read 1:    14,422,524 bp\n",
      "  Read 2:    14,421,883 bp\n",
      "Total written (filtered):     26,944,497 bp (93.4%)\n",
      "  Read 1:    13,446,343 bp\n",
      "  Read 2:    13,498,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48846 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t764\t0.0\t1\t0 727 37\n",
      "20\t47929\t0.0\t2\t45385 2490 54\n",
      "21\t85\t0.0\t2\t0 83 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t888\t0.0\t1\t0 888\n",
      "19\t47675\t0.0\t1\t46521 1154\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S406_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S406_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S406_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S406_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S406_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S406_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S406_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S406_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,139 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.38 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,139\n",
      "  Read 1 with adapter:                  33,115 (99.9%)\n",
      "  Read 2 with adapter:                  32,971 (99.5%)\n",
      "Pairs written (passing filters):        33,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,551,891 bp\n",
      "  Read 1:     9,775,992 bp\n",
      "  Read 2:     9,775,899 bp\n",
      "Total written (filtered):     18,264,176 bp (93.4%)\n",
      "  Read 1:     9,114,159 bp\n",
      "  Read 2:     9,150,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t556\t0.0\t1\t0 528 28\n",
      "20\t32442\t0.0\t2\t30505 1893 44\n",
      "21\t45\t0.0\t2\t0 45\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32971 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t639\t0.0\t1\t0 639\n",
      "19\t32260\t0.0\t1\t31178 1082\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S407_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S407_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S407_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S407_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S407_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S407_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S407_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S407_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,919 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.53 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,919\n",
      "  Read 1 with adapter:                  50,880 (99.9%)\n",
      "  Read 2 with adapter:                  50,722 (99.6%)\n",
      "Pairs written (passing filters):        50,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,040,806 bp\n",
      "  Read 1:    15,021,078 bp\n",
      "  Read 2:    15,019,728 bp\n",
      "Total written (filtered):     28,060,965 bp (93.4%)\n",
      "  Read 1:    14,004,281 bp\n",
      "  Read 2:    14,056,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50880 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t861\t0.0\t1\t0 816 45\n",
      "20\t49863\t0.0\t2\t47285 2516 62\n",
      "21\t82\t0.0\t2\t0 80 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t774\t0.0\t1\t0 774\n",
      "19\t49848\t0.0\t1\t48628 1220\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S408_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S408_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S408_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S408_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S408_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S408_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S408_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S408_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,605 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.63 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,605\n",
      "  Read 1 with adapter:                  67,554 (99.9%)\n",
      "  Read 2 with adapter:                  67,134 (99.3%)\n",
      "Pairs written (passing filters):        67,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,870,745 bp\n",
      "  Read 1:    19,942,924 bp\n",
      "  Read 2:    19,927,821 bp\n",
      "Total written (filtered):     37,246,268 bp (93.4%)\n",
      "  Read 1:    18,592,830 bp\n",
      "  Read 2:    18,653,438 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67554 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t1091\t0.0\t1\t0 1038 53\n",
      "20\t66285\t0.0\t2\t62674 3540 71\n",
      "21\t87\t0.0\t2\t0 83 4\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67134 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1268\t0.0\t1\t0 1268\n",
      "19\t65761\t0.0\t1\t64016 1745\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S409_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S409_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S409_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S409_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S409_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S409_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S409_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S409_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,533 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.12 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,533\n",
      "  Read 1 with adapter:                  59,491 (99.9%)\n",
      "  Read 2 with adapter:                  59,094 (99.3%)\n",
      "Pairs written (passing filters):        59,533 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,095,733 bp\n",
      "  Read 1:    17,561,772 bp\n",
      "  Read 2:    17,533,961 bp\n",
      "Total written (filtered):     32,784,883 bp (93.4%)\n",
      "  Read 1:    16,372,863 bp\n",
      "  Read 2:    16,412,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t51\t0.0\t1\t0 0 51\n",
      "19\t1001\t0.0\t1\t0 956 45\n",
      "20\t58289\t0.0\t2\t55222 2993 74\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t932\t0.0\t1\t0 932\n",
      "19\t58075\t0.0\t1\t56730 1345\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S410_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S410_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S410_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S410_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S410_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S410_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S410_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S410_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           137 reads  @    393.1 µs/read;   0.15 M reads/minute\n",
      "Finished in 0.06 s (414 us/read; 0.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                137\n",
      "  Read 1 with adapter:                     132 (96.4%)\n",
      "  Read 2 with adapter:                     131 (95.6%)\n",
      "Pairs written (passing filters):           137 (100.0%)\n",
      "\n",
      "Total basepairs processed:        80,074 bp\n",
      "  Read 1:        39,940 bp\n",
      "  Read 2:        40,134 bp\n",
      "Total written (filtered):         74,949 bp (93.6%)\n",
      "  Read 1:        37,303 bp\n",
      "  Read 2:        37,646 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t3\t0.0\t1\t0 3\n",
      "20\t129\t0.0\t2\t125 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t130\t0.0\t1\t128 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S411_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S411_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S411_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S411_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S411_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S411_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S411_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S411_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,124 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.19 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,124\n",
      "  Read 1 with adapter:                  45,031 (99.8%)\n",
      "  Read 2 with adapter:                  44,856 (99.4%)\n",
      "Pairs written (passing filters):        45,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,613,946 bp\n",
      "  Read 1:    13,303,456 bp\n",
      "  Read 2:    13,310,490 bp\n",
      "Total written (filtered):     24,862,426 bp (93.4%)\n",
      "  Read 1:    12,403,377 bp\n",
      "  Read 2:    12,459,049 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t614\t0.0\t1\t0 594 20\n",
      "20\t44300\t0.0\t2\t42123 2131 46\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t911\t0.0\t1\t0 911\n",
      "19\t43857\t0.0\t1\t42626 1231\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S412_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S412_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S412_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S412_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S412_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S412_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S412_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S412_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,593 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.29 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,593\n",
      "  Read 1 with adapter:                  49,544 (99.9%)\n",
      "  Read 2 with adapter:                  49,284 (99.4%)\n",
      "Pairs written (passing filters):        49,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,258,913 bp\n",
      "  Read 1:    14,629,621 bp\n",
      "  Read 2:    14,629,292 bp\n",
      "Total written (filtered):     27,332,938 bp (93.4%)\n",
      "  Read 1:    13,639,317 bp\n",
      "  Read 2:    13,693,621 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t661\t0.0\t1\t0 636 25\n",
      "20\t48770\t0.0\t2\t46410 2297 63\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49284 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t832\t0.0\t1\t0 832\n",
      "19\t48345\t0.0\t1\t46957 1388\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S413_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S413_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S413_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S413_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S413_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S413_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S413_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S413_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,699 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.54 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,699\n",
      "  Read 1 with adapter:                  35,672 (99.9%)\n",
      "  Read 2 with adapter:                  35,489 (99.4%)\n",
      "Pairs written (passing filters):        35,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,060,207 bp\n",
      "  Read 1:    10,530,821 bp\n",
      "  Read 2:    10,529,386 bp\n",
      "Total written (filtered):     19,673,538 bp (93.4%)\n",
      "  Read 1:     9,817,871 bp\n",
      "  Read 2:     9,855,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35672 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t538\t0.0\t1\t0 514 24\n",
      "20\t35044\t0.0\t2\t33207 1798 39\n",
      "21\t48\t0.0\t2\t0 47 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t631\t0.0\t1\t0 631\n",
      "19\t34799\t0.0\t1\t33795 1004\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S414_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S414_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S414_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S414_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S414_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S414_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S414_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S414_87_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,110 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.45 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,110\n",
      "  Read 1 with adapter:                  35,088 (99.9%)\n",
      "  Read 2 with adapter:                  34,930 (99.5%)\n",
      "Pairs written (passing filters):        35,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,714,421 bp\n",
      "  Read 1:    10,357,291 bp\n",
      "  Read 2:    10,357,130 bp\n",
      "Total written (filtered):     19,349,975 bp (93.4%)\n",
      "  Read 1:     9,655,974 bp\n",
      "  Read 2:     9,694,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35088 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t512\t0.0\t1\t0 482 30\n",
      "20\t34478\t0.0\t2\t32689 1755 34\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t617\t0.0\t1\t0 617\n",
      "19\t34237\t0.0\t1\t33206 1031\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S415_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S415_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S415_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S415_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S415_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S415_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S415_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S415_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,605 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.81 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,605\n",
      "  Read 1 with adapter:                  55,563 (99.9%)\n",
      "  Read 2 with adapter:                  55,368 (99.6%)\n",
      "Pairs written (passing filters):        55,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,804,784 bp\n",
      "  Read 1:    16,403,335 bp\n",
      "  Read 2:    16,401,449 bp\n",
      "Total written (filtered):     30,642,538 bp (93.4%)\n",
      "  Read 1:    15,292,253 bp\n",
      "  Read 2:    15,350,285 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55563 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t847\t0.0\t1\t0 815 32\n",
      "20\t54545\t0.0\t2\t51790 2693 62\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t936\t0.0\t1\t0 936\n",
      "19\t54324\t0.0\t1\t52678 1646\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S416_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S416_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S416_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S416_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S416_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S416_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S416_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S416_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,743 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.61 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,743\n",
      "  Read 1 with adapter:                  51,693 (99.9%)\n",
      "  Read 2 with adapter:                  51,383 (99.3%)\n",
      "Pairs written (passing filters):        51,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,515,553 bp\n",
      "  Read 1:    15,263,550 bp\n",
      "  Read 2:    15,252,003 bp\n",
      "Total written (filtered):     28,507,206 bp (93.4%)\n",
      "  Read 1:    14,230,498 bp\n",
      "  Read 2:    14,276,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t852\t0.0\t1\t0 804 48\n",
      "20\t50702\t0.0\t2\t48041 2590 71\n",
      "21\t50\t0.0\t2\t0 48 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1066\t0.0\t1\t0 1066\n",
      "19\t50233\t0.0\t1\t48785 1448\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S417_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S417_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S417_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S417_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S417_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S417_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S417_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S417_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,262 reads  @     71.9 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.61 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,262\n",
      "  Read 1 with adapter:                  50,223 (99.9%)\n",
      "  Read 2 with adapter:                  49,911 (99.3%)\n",
      "Pairs written (passing filters):        50,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,631,194 bp\n",
      "  Read 1:    14,827,132 bp\n",
      "  Read 2:    14,804,062 bp\n",
      "Total written (filtered):     27,679,963 bp (93.4%)\n",
      "  Read 1:    13,823,456 bp\n",
      "  Read 2:    13,856,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t858\t0.0\t1\t0 825 33\n",
      "20\t49217\t0.0\t2\t46683 2462 72\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49911 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t816\t0.0\t1\t0 816\n",
      "19\t49033\t0.0\t1\t47864 1169\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S418_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S418_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S418_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S418_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S418_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S418_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S418_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S418_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,084 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.36 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,084\n",
      "  Read 1 with adapter:                  31,058 (99.9%)\n",
      "  Read 2 with adapter:                  30,385 (97.8%)\n",
      "Pairs written (passing filters):        31,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,302,634 bp\n",
      "  Read 1:     9,169,578 bp\n",
      "  Read 2:     9,133,056 bp\n",
      "Total written (filtered):     17,104,823 bp (93.5%)\n",
      "  Read 1:     8,548,865 bp\n",
      "  Read 2:     8,555,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t491\t0.0\t1\t0 475 16\n",
      "20\t30481\t0.0\t2\t28915 1528 38\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t30086\t0.0\t1\t29418 668\n",
      "20\t41\t0.0\t1\t0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S419_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S419_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S419_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S419_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S419_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S419_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S419_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S419_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,802 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.77 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,802\n",
      "  Read 1 with adapter:                  54,753 (99.9%)\n",
      "  Read 2 with adapter:                  54,543 (99.5%)\n",
      "Pairs written (passing filters):        54,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,332,502 bp\n",
      "  Read 1:    16,166,574 bp\n",
      "  Read 2:    16,165,928 bp\n",
      "Total written (filtered):     30,202,623 bp (93.4%)\n",
      "  Read 1:    15,072,085 bp\n",
      "  Read 2:    15,130,538 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t641\t0.0\t1\t0 607 34\n",
      "20\t53959\t0.0\t2\t51274 2638 47\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54543 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1023\t0.0\t1\t0 1023\n",
      "19\t53424\t0.0\t1\t52129 1295\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S420_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S420_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S420_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S420_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S420_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S420_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S420_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S420_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,933 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.37 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,933\n",
      "  Read 1 with adapter:                  34,897 (99.9%)\n",
      "  Read 2 with adapter:                  34,756 (99.5%)\n",
      "Pairs written (passing filters):        34,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,609,637 bp\n",
      "  Read 1:    10,304,924 bp\n",
      "  Read 2:    10,304,713 bp\n",
      "Total written (filtered):     19,252,275 bp (93.4%)\n",
      "  Read 1:     9,607,386 bp\n",
      "  Read 2:     9,644,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t446\t0.0\t1\t0 432 14\n",
      "20\t34358\t0.0\t2\t32551 1763 44\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t590\t0.0\t1\t0 590\n",
      "19\t34116\t0.0\t1\t33234 882\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S421_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S421_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S421_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S421_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S421_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S421_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S421_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S421_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,153 reads  @     66.5 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.21 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,153\n",
      "  Read 1 with adapter:                  48,104 (99.9%)\n",
      "  Read 2 with adapter:                  47,914 (99.5%)\n",
      "Pairs written (passing filters):        48,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,409,798 bp\n",
      "  Read 1:    14,204,965 bp\n",
      "  Read 2:    14,204,833 bp\n",
      "Total written (filtered):     26,538,666 bp (93.4%)\n",
      "  Read 1:    13,243,430 bp\n",
      "  Read 2:    13,295,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48104 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t619\t0.0\t1\t0 582 37\n",
      "20\t47350\t0.0\t2\t44776 2512 62\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t853\t0.0\t1\t0 853\n",
      "19\t46977\t0.0\t1\t45795 1182\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S422_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S422_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S422_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S422_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S422_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S422_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S422_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S422_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        28,975 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.98 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,975\n",
      "  Read 1 with adapter:                  28,949 (99.9%)\n",
      "  Read 2 with adapter:                  28,836 (99.5%)\n",
      "Pairs written (passing filters):        28,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,094,652 bp\n",
      "  Read 1:     8,547,615 bp\n",
      "  Read 2:     8,547,037 bp\n",
      "Total written (filtered):     15,968,523 bp (93.4%)\n",
      "  Read 1:     7,968,971 bp\n",
      "  Read 2:     7,999,552 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t383\t0.0\t1\t0 367 16\n",
      "20\t28478\t0.0\t2\t27015 1420 43\n",
      "21\t49\t0.0\t2\t0 48 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t470\t0.0\t1\t0 470\n",
      "19\t28295\t0.0\t1\t27497 798\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S423_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S423_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S423_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S423_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S423_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S423_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S423_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S423_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        49,352 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.41 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,352\n",
      "  Read 1 with adapter:                  49,307 (99.9%)\n",
      "  Read 2 with adapter:                  49,135 (99.6%)\n",
      "Pairs written (passing filters):        49,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,115,438 bp\n",
      "  Read 1:    14,558,822 bp\n",
      "  Read 2:    14,556,616 bp\n",
      "Total written (filtered):     27,197,058 bp (93.4%)\n",
      "  Read 1:    13,573,276 bp\n",
      "  Read 2:    13,623,782 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49307 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t651\t0.0\t1\t0 608 43\n",
      "20\t48508\t0.0\t2\t46002 2441 65\n",
      "21\t71\t0.0\t2\t0 68 3\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t802\t0.0\t1\t0 802\n",
      "19\t48262\t0.0\t1\t47014 1248\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S424_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S424_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S424_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S424_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S424_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S424_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S424_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S424_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,457 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.65 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,457\n",
      "  Read 1 with adapter:                  36,408 (99.9%)\n",
      "  Read 2 with adapter:                  36,258 (99.5%)\n",
      "Pairs written (passing filters):        36,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,503,117 bp\n",
      "  Read 1:    10,754,802 bp\n",
      "  Read 2:    10,748,315 bp\n",
      "Total written (filtered):     20,087,084 bp (93.4%)\n",
      "  Read 1:    10,027,005 bp\n",
      "  Read 2:    10,060,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t433\t0.0\t1\t0 419 14\n",
      "20\t35853\t0.0\t2\t33974 1833 46\n",
      "21\t78\t0.0\t2\t0 76 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36258 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t730\t0.0\t1\t0 730\n",
      "19\t35464\t0.0\t1\t34507 957\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S425_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S425_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S425_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S425_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S425_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S425_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S425_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S425_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        52,354 reads  @    102.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.38 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,354\n",
      "  Read 1 with adapter:                  52,304 (99.9%)\n",
      "  Read 2 with adapter:                  51,933 (99.2%)\n",
      "Pairs written (passing filters):        52,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,860,012 bp\n",
      "  Read 1:    15,444,077 bp\n",
      "  Read 2:    15,415,935 bp\n",
      "Total written (filtered):     28,828,559 bp (93.4%)\n",
      "  Read 1:    14,398,608 bp\n",
      "  Read 2:    14,429,951 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t682\t0.0\t1\t0 648 34\n",
      "20\t51477\t0.0\t2\t48830 2577 70\n",
      "21\t91\t0.0\t2\t0 85 6\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t798\t0.0\t1\t0 798\n",
      "19\t51080\t0.0\t1\t49935 1145\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S426_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S426_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S426_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S426_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S426_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S426_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S426_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S426_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        55,457 reads  @    138.9 µs/read;   0.43 M reads/minute\n",
      "Finished in 7.71 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,457\n",
      "  Read 1 with adapter:                  55,421 (99.9%)\n",
      "  Read 2 with adapter:                  54,159 (97.7%)\n",
      "Pairs written (passing filters):        55,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,650,700 bp\n",
      "  Read 1:    16,359,361 bp\n",
      "  Read 2:    16,291,339 bp\n",
      "Total written (filtered):     30,514,207 bp (93.5%)\n",
      "  Read 1:    15,251,553 bp\n",
      "  Read 2:    15,262,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55421 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t686\t0.0\t1\t0 655 31\n",
      "20\t54595\t0.0\t2\t51826 2694 75\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t408\t0.0\t1\t0 408\n",
      "19\t53679\t0.0\t1\t52566 1113\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S427_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S427_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S427_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S427_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S427_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S427_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S427_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S427_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        64,700 reads  @    124.1 µs/read;   0.48 M reads/minute\n",
      "Finished in 8.03 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,700\n",
      "  Read 1 with adapter:                  64,658 (99.9%)\n",
      "  Read 2 with adapter:                  64,312 (99.4%)\n",
      "Pairs written (passing filters):        64,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,168,691 bp\n",
      "  Read 1:    19,086,354 bp\n",
      "  Read 2:    19,082,337 bp\n",
      "Total written (filtered):     35,655,493 bp (93.4%)\n",
      "  Read 1:    17,794,008 bp\n",
      "  Read 2:    17,861,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t874\t0.0\t1\t0 836 38\n",
      "20\t63648\t0.0\t2\t60522 3060 66\n",
      "21\t80\t0.0\t2\t0 80\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64312 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1209\t0.0\t1\t0 1209\n",
      "19\t62970\t0.0\t1\t61362 1608\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S428_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S428_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S428_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S428_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S428_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S428_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S428_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S428_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,801 reads  @    120.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 5.26 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,801\n",
      "  Read 1 with adapter:                  43,771 (99.9%)\n",
      "  Read 2 with adapter:                  43,561 (99.5%)\n",
      "Pairs written (passing filters):        43,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,841,990 bp\n",
      "  Read 1:    12,921,042 bp\n",
      "  Read 2:    12,920,948 bp\n",
      "Total written (filtered):     24,140,041 bp (93.4%)\n",
      "  Read 1:    12,046,067 bp\n",
      "  Read 2:    12,093,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t531\t0.0\t1\t0 510 21\n",
      "20\t43123\t0.0\t2\t40898 2179 46\n",
      "21\t68\t0.0\t2\t0 68\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t745\t0.0\t1\t0 745\n",
      "19\t42756\t0.0\t1\t41621 1135\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S429_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S429_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S429_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S429_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S429_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S429_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S429_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S429_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06        50,363 reads  @    120.4 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.07 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,363\n",
      "  Read 1 with adapter:                  50,338 (100.0%)\n",
      "  Read 2 with adapter:                  50,131 (99.5%)\n",
      "Pairs written (passing filters):        50,363 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,713,272 bp\n",
      "  Read 1:    14,856,863 bp\n",
      "  Read 2:    14,856,409 bp\n",
      "Total written (filtered):     27,755,467 bp (93.4%)\n",
      "  Read 1:    13,850,749 bp\n",
      "  Read 2:    13,904,718 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t699\t0.0\t1\t0 657 42\n",
      "20\t49522\t0.0\t2\t46866 2574 82\n",
      "21\t69\t0.0\t2\t0 67 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t904\t0.0\t1\t0 904\n",
      "19\t49121\t0.0\t1\t47878 1243\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S430_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S430_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S430_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S430_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S430_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S430_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S430_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S430_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        45,285 reads  @    121.5 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.51 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,285\n",
      "  Read 1 with adapter:                  45,262 (99.9%)\n",
      "  Read 2 with adapter:                  45,066 (99.5%)\n",
      "Pairs written (passing filters):        45,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,707 bp\n",
      "  Read 1:    13,358,707 bp\n",
      "  Read 2:    13,359,000 bp\n",
      "Total written (filtered):     24,957,527 bp (93.4%)\n",
      "  Read 1:    12,454,059 bp\n",
      "  Read 2:    12,503,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t664\t0.0\t1\t0 630 34\n",
      "20\t44498\t0.0\t2\t42161 2267 70\n",
      "21\t64\t0.0\t2\t0 62 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45066 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t811\t0.0\t1\t0 811\n",
      "19\t44166\t0.0\t1\t42938 1228\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S431_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S431_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S431_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S431_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S431_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S431_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S431_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S431_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        29,627 reads  @    125.7 µs/read;   0.48 M reads/minute\n",
      "Finished in 3.73 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,627\n",
      "  Read 1 with adapter:                  29,604 (99.9%)\n",
      "  Read 2 with adapter:                  29,471 (99.5%)\n",
      "Pairs written (passing filters):        29,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,477,872 bp\n",
      "  Read 1:     8,739,224 bp\n",
      "  Read 2:     8,738,648 bp\n",
      "Total written (filtered):     16,326,334 bp (93.4%)\n",
      "  Read 1:     8,147,256 bp\n",
      "  Read 2:     8,179,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29604 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t427\t0.0\t1\t0 406 21\n",
      "20\t29101\t0.0\t2\t27553 1516 32\n",
      "21\t40\t0.0\t2\t0 40\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t438\t0.0\t1\t0 438\n",
      "19\t28974\t0.0\t1\t28207 767\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S432_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S432_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S432_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S432_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S432_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S432_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S432_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S432_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        73,229 reads  @    123.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 9.07 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,229\n",
      "  Read 1 with adapter:                  73,184 (99.9%)\n",
      "  Read 2 with adapter:                  72,802 (99.4%)\n",
      "Pairs written (passing filters):        73,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,191,336 bp\n",
      "  Read 1:    21,602,077 bp\n",
      "  Read 2:    21,589,259 bp\n",
      "Total written (filtered):     40,346,688 bp (93.4%)\n",
      "  Read 1:    20,139,400 bp\n",
      "  Read 2:    20,207,288 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t1109\t0.0\t1\t0 1069 40\n",
      "20\t71859\t0.0\t2\t67954 3814 91\n",
      "21\t114\t0.0\t2\t0 112 2\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1387\t0.0\t1\t0 1387\n",
      "19\t71295\t0.0\t1\t69374 1921\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S433_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S433_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S433_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S433_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S433_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S433_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S433_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S433_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        38,117 reads  @    126.0 µs/read;   0.48 M reads/minute\n",
      "Finished in 4.81 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,117\n",
      "  Read 1 with adapter:                  38,095 (99.9%)\n",
      "  Read 2 with adapter:                  37,882 (99.4%)\n",
      "Pairs written (passing filters):        38,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,475,571 bp\n",
      "  Read 1:    11,244,500 bp\n",
      "  Read 2:    11,231,071 bp\n",
      "Total written (filtered):     20,994,989 bp (93.4%)\n",
      "  Read 1:    10,483,148 bp\n",
      "  Read 2:    10,511,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38095 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t596\t0.0\t1\t0 569 27\n",
      "20\t37407\t0.0\t2\t35440 1919 48\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37882 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t576\t0.0\t1\t0 576\n",
      "19\t37258\t0.0\t1\t36391 867\n",
      "20\t48\t0.0\t1\t0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S434_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S434_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S434_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S434_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S434_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S434_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S434_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S434_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        56,593 reads  @    128.3 µs/read;   0.47 M reads/minute\n",
      "Finished in 7.27 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,593\n",
      "  Read 1 with adapter:                  56,564 (99.9%)\n",
      "  Read 2 with adapter:                  55,124 (97.4%)\n",
      "Pairs written (passing filters):        56,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,306,528 bp\n",
      "  Read 1:    16,694,914 bp\n",
      "  Read 2:    16,611,614 bp\n",
      "Total written (filtered):     31,128,997 bp (93.5%)\n",
      "  Read 1:    15,564,365 bp\n",
      "  Read 2:    15,564,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t776\t0.0\t1\t0 739 37\n",
      "20\t55643\t0.0\t2\t52826 2744 73\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55124 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t458\t0.0\t1\t0 458\n",
      "19\t54582\t0.0\t1\t53453 1129\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S435_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S435_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S435_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S435_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S435_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S435_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S435_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S435_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[    8<------] 00:00:10        80,131 reads  @    125.3 µs/read;   0.48 M reads/minute\n",
      "Finished in 10.04 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,131\n",
      "  Read 1 with adapter:                  80,042 (99.9%)\n",
      "  Read 2 with adapter:                  79,771 (99.6%)\n",
      "Pairs written (passing filters):        80,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,273,584 bp\n",
      "  Read 1:    23,636,959 bp\n",
      "  Read 2:    23,636,625 bp\n",
      "Total written (filtered):     44,159,032 bp (93.4%)\n",
      "  Read 1:    22,036,722 bp\n",
      "  Read 2:    22,122,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80042 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t709\t0.0\t1\t0 675 34\n",
      "20\t79146\t0.0\t2\t75326 3716 104\n",
      "21\t120\t0.0\t2\t0 118 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1466\t0.0\t1\t0 1466\n",
      "19\t78173\t0.0\t1\t76344 1829\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S436_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S436_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S436_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S436_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S436_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S436_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S436_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S436_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        52,702 reads  @    121.8 µs/read;   0.49 M reads/minute\n",
      "Finished in 6.43 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,702\n",
      "  Read 1 with adapter:                  52,654 (99.9%)\n",
      "  Read 2 with adapter:                  52,477 (99.6%)\n",
      "Pairs written (passing filters):        52,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,093,486 bp\n",
      "  Read 1:    15,546,594 bp\n",
      "  Read 2:    15,546,892 bp\n",
      "Total written (filtered):     29,044,657 bp (93.4%)\n",
      "  Read 1:    14,494,025 bp\n",
      "  Read 2:    14,550,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t562\t0.0\t1\t0 532 30\n",
      "20\t51976\t0.0\t2\t49191 2716 69\n",
      "21\t73\t0.0\t2\t0 71 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52477 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t895\t0.0\t1\t0 895\n",
      "19\t51490\t0.0\t1\t50178 1312\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S437_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S437_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S437_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S437_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S437_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S437_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S437_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S437_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,924 reads  @    123.0 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.16 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,924\n",
      "  Read 1 with adapter:                  41,884 (99.9%)\n",
      "  Read 2 with adapter:                  41,747 (99.6%)\n",
      "Pairs written (passing filters):        41,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,733,460 bp\n",
      "  Read 1:    12,366,937 bp\n",
      "  Read 2:    12,366,523 bp\n",
      "Total written (filtered):     23,103,647 bp (93.4%)\n",
      "  Read 1:    11,529,632 bp\n",
      "  Read 2:    11,574,015 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t415\t0.0\t1\t0 401 14\n",
      "20\t41360\t0.0\t2\t39063 2235 62\n",
      "21\t62\t0.0\t2\t0 59 3\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t764\t0.0\t1\t0 764\n",
      "19\t40904\t0.0\t1\t39916 988\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S438_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S438_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S438_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S438_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S438_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S438_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S438_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S438_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,025 reads  @    122.4 µs/read;   0.49 M reads/minute\n",
      "Finished in 3.92 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,025\n",
      "  Read 1 with adapter:                  31,991 (99.9%)\n",
      "  Read 2 with adapter:                  31,887 (99.6%)\n",
      "Pairs written (passing filters):        32,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,894,402 bp\n",
      "  Read 1:     9,447,093 bp\n",
      "  Read 2:     9,447,309 bp\n",
      "Total written (filtered):     17,649,461 bp (93.4%)\n",
      "  Read 1:     8,807,546 bp\n",
      "  Read 2:     8,841,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t327\t0.0\t1\t0 309 18\n",
      "20\t31591\t0.0\t2\t29939 1614 38\n",
      "21\t44\t0.0\t2\t0 42 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t523\t0.0\t1\t0 523\n",
      "19\t31300\t0.0\t1\t30504 796\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S439_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S439_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S439_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S439_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S439_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S439_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S439_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S439_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,341 reads  @    123.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.12 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,341\n",
      "  Read 1 with adapter:                  41,292 (99.9%)\n",
      "  Read 2 with adapter:                  41,192 (99.6%)\n",
      "Pairs written (passing filters):        41,341 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,388,913 bp\n",
      "  Read 1:    12,194,844 bp\n",
      "  Read 2:    12,194,069 bp\n",
      "Total written (filtered):     22,781,391 bp (93.4%)\n",
      "  Read 1:    11,369,369 bp\n",
      "  Read 2:    11,412,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t413\t0.0\t1\t0 395 18\n",
      "20\t40778\t0.0\t2\t38606 2118 54\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t669\t0.0\t1\t0 669\n",
      "19\t40455\t0.0\t1\t39514 941\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S440_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S440_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S440_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S440_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S440_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S440_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S440_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S440_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        33,917 reads  @    126.1 µs/read;   0.48 M reads/minute\n",
      "Finished in 4.28 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,917\n",
      "  Read 1 with adapter:                  33,884 (99.9%)\n",
      "  Read 2 with adapter:                  33,754 (99.5%)\n",
      "Pairs written (passing filters):        33,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,003,753 bp\n",
      "  Read 1:    10,004,952 bp\n",
      "  Read 2:     9,998,801 bp\n",
      "Total written (filtered):     18,685,657 bp (93.4%)\n",
      "  Read 1:     9,327,593 bp\n",
      "  Read 2:     9,358,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t349\t0.0\t1\t0 332 17\n",
      "20\t33454\t0.0\t2\t31703 1706 45\n",
      "21\t38\t0.0\t2\t0 37 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33754 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t634\t0.0\t1\t0 634\n",
      "19\t33075\t0.0\t1\t32335 740\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S441_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S441_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S441_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S441_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S441_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S441_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S441_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S441_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:06        50,737 reads  @    123.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 6.26 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,737\n",
      "  Read 1 with adapter:                  50,672 (99.9%)\n",
      "  Read 2 with adapter:                  50,396 (99.3%)\n",
      "Pairs written (passing filters):        50,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,912,864 bp\n",
      "  Read 1:    14,966,065 bp\n",
      "  Read 2:    14,946,799 bp\n",
      "Total written (filtered):     27,943,082 bp (93.4%)\n",
      "  Read 1:    13,953,087 bp\n",
      "  Read 2:    13,989,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50672 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t513\t0.0\t1\t0 497 16\n",
      "20\t50049\t0.0\t2\t47331 2651 67\n",
      "21\t65\t0.0\t2\t0 61 4\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50396 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t793\t0.0\t1\t0 793\n",
      "19\t49530\t0.0\t1\t48466 1064\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S442_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S442_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S442_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S442_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S442_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S442_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S442_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S442_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        50,723 reads  @    119.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.05 s (119 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,723\n",
      "  Read 1 with adapter:                  50,659 (99.9%)\n",
      "  Read 2 with adapter:                  49,568 (97.7%)\n",
      "Pairs written (passing filters):        50,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,857,782 bp\n",
      "  Read 1:    14,960,957 bp\n",
      "  Read 2:    14,896,825 bp\n",
      "Total written (filtered):     27,903,631 bp (93.5%)\n",
      "  Read 1:    13,948,257 bp\n",
      "  Read 2:    13,955,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t524\t0.0\t1\t0 502 22\n",
      "20\t50036\t0.0\t2\t47436 2537 63\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t398\t0.0\t1\t0 398\n",
      "19\t49113\t0.0\t1\t48121 992\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S443_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S443_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S443_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S443_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S443_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S443_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S443_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S443_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        53,664 reads  @    116.9 µs/read;   0.51 M reads/minute\n",
      "Finished in 6.28 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,664\n",
      "  Read 1 with adapter:                  53,618 (99.9%)\n",
      "  Read 2 with adapter:                  53,423 (99.6%)\n",
      "Pairs written (passing filters):        53,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,661,244 bp\n",
      "  Read 1:    15,830,669 bp\n",
      "  Read 2:    15,830,575 bp\n",
      "Total written (filtered):     29,575,085 bp (93.4%)\n",
      "  Read 1:    14,758,570 bp\n",
      "  Read 2:    14,816,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t378\t0.0\t1\t0 360 18\n",
      "20\t53113\t0.0\t2\t50699 2371 43\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1089\t0.0\t1\t0 1089\n",
      "19\t52222\t0.0\t1\t50910 1312\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S444_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S444_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S444_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S444_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S444_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S444_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S444_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S444_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        52,375 reads  @    115.2 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.04 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,375\n",
      "  Read 1 with adapter:                  52,338 (99.9%)\n",
      "  Read 2 with adapter:                  52,172 (99.6%)\n",
      "Pairs written (passing filters):        52,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,900,207 bp\n",
      "  Read 1:    15,450,052 bp\n",
      "  Read 2:    15,450,155 bp\n",
      "Total written (filtered):     28,863,228 bp (93.4%)\n",
      "  Read 1:    14,403,534 bp\n",
      "  Read 2:    14,459,694 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t423\t0.0\t1\t0 406 17\n",
      "20\t51728\t0.0\t2\t49278 2408 42\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52172 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t906\t0.0\t1\t0 906\n",
      "19\t51167\t0.0\t1\t49866 1301\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S445_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S445_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S445_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S445_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S445_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S445_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S445_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S445_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        19,954 reads  @    114.5 µs/read;   0.52 M reads/minute\n",
      "Finished in 2.29 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,954\n",
      "  Read 1 with adapter:                  19,939 (99.9%)\n",
      "  Read 2 with adapter:                  19,874 (99.6%)\n",
      "Pairs written (passing filters):        19,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,772,659 bp\n",
      "  Read 1:     5,886,425 bp\n",
      "  Read 2:     5,886,234 bp\n",
      "Total written (filtered):     10,996,675 bp (93.4%)\n",
      "  Read 1:     5,487,744 bp\n",
      "  Read 2:     5,508,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 19939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t161\t0.0\t1\t0 155 6\n",
      "20\t19718\t0.0\t2\t18809 888 21\n",
      "21\t30\t0.0\t2\t0 29 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 19874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t336\t0.0\t1\t0 336\n",
      "19\t19505\t0.0\t1\t19021 484\n",
      "20\t33\t0.0\t1\t0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S446_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S446_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S446_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S446_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S446_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S446_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S446_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S446_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        34,756 reads  @    122.0 µs/read;   0.49 M reads/minute\n",
      "Finished in 4.24 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,756\n",
      "  Read 1 with adapter:                  34,725 (99.9%)\n",
      "  Read 2 with adapter:                  34,570 (99.5%)\n",
      "Pairs written (passing filters):        34,756 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,505,531 bp\n",
      "  Read 1:    10,252,529 bp\n",
      "  Read 2:    10,253,002 bp\n",
      "Total written (filtered):     19,154,795 bp (93.4%)\n",
      "  Read 1:     9,558,058 bp\n",
      "  Read 2:     9,596,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t312\t0.0\t1\t0 297 15\n",
      "20\t34211\t0.0\t2\t32356 1803 52\n",
      "21\t53\t0.0\t2\t0 50 3\n",
      "22\t132\t0.0\t2\t0 0 132\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34570 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t647\t0.0\t1\t0 647\n",
      "19\t33841\t0.0\t1\t32742 1099\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S447_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S447_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S447_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S447_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S447_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S447_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S447_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S447_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        41,375 reads  @    121.2 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.02 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,375\n",
      "  Read 1 with adapter:                  41,342 (99.9%)\n",
      "  Read 2 with adapter:                  41,233 (99.7%)\n",
      "Pairs written (passing filters):        41,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,410,011 bp\n",
      "  Read 1:    12,205,096 bp\n",
      "  Read 2:    12,204,915 bp\n",
      "Total written (filtered):     22,800,588 bp (93.4%)\n",
      "  Read 1:    11,378,486 bp\n",
      "  Read 2:    11,422,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t376\t0.0\t1\t0 358 18\n",
      "20\t40822\t0.0\t2\t38864 1925 33\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t682\t0.0\t1\t0 682\n",
      "19\t40483\t0.0\t1\t39460 1023\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S448_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S448_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S448_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S448_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S448_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S448_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S448_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S448_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        52,688 reads  @    118.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.27 s (119 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,688\n",
      "  Read 1 with adapter:                  52,650 (99.9%)\n",
      "  Read 2 with adapter:                  52,411 (99.5%)\n",
      "Pairs written (passing filters):        52,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,077,550 bp\n",
      "  Read 1:    15,542,894 bp\n",
      "  Read 2:    15,534,656 bp\n",
      "Total written (filtered):     29,029,974 bp (93.4%)\n",
      "  Read 1:    14,490,186 bp\n",
      "  Read 2:    14,539,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t475\t0.0\t1\t0 457 18\n",
      "20\t51971\t0.0\t2\t49574 2347 50\n",
      "21\t105\t0.0\t2\t0 103 2\n",
      "22\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1026\t0.0\t1\t0 1026\n",
      "19\t51300\t0.0\t1\t49996 1304\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S449_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S449_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S449_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S449_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S449_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S449_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S449_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S449_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        64,264 reads  @     95.4 µs/read;   0.63 M reads/minute\n",
      "Finished in 6.14 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,264\n",
      "  Read 1 with adapter:                  64,213 (99.9%)\n",
      "  Read 2 with adapter:                  63,864 (99.4%)\n",
      "Pairs written (passing filters):        64,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,885,747 bp\n",
      "  Read 1:    18,957,568 bp\n",
      "  Read 2:    18,928,179 bp\n",
      "Total written (filtered):     35,389,390 bp (93.4%)\n",
      "  Read 1:    17,673,694 bp\n",
      "  Read 2:    17,715,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t581\t0.0\t1\t0 551 30\n",
      "20\t63410\t0.0\t2\t60439 2910 61\n",
      "21\t137\t0.0\t2\t0 135 2\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1020\t0.0\t1\t0 1020\n",
      "19\t62757\t0.0\t1\t61344 1413\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S450_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S450_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S450_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S450_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S450_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S450_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S450_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S450_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           269 reads  @    217.4 µs/read;   0.28 M reads/minute\n",
      "Finished in 0.06 s (227 us/read; 0.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                269\n",
      "  Read 1 with adapter:                     266 (98.9%)\n",
      "  Read 2 with adapter:                     261 (97.0%)\n",
      "Pairs written (passing filters):           269 (100.0%)\n",
      "\n",
      "Total basepairs processed:       158,454 bp\n",
      "  Read 1:        79,355 bp\n",
      "  Read 2:        79,099 bp\n",
      "Total written (filtered):        148,176 bp (93.5%)\n",
      "  Read 1:        74,035 bp\n",
      "  Read 2:        74,141 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t3\t0.0\t1\t0 2 1\n",
      "20\t261\t0.0\t2\t256 5\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 2\n",
      "19\t258\t0.0\t1\t251 7\n",
      "20\t1\t0.0\t1\t0 1\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S451_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S451_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S451_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S451_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S451_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S451_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S451_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S451_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,664 reads  @     69.1 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.26 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,664\n",
      "  Read 1 with adapter:                  32,632 (99.9%)\n",
      "  Read 2 with adapter:                  32,474 (99.4%)\n",
      "Pairs written (passing filters):        32,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,270,405 bp\n",
      "  Read 1:     9,635,173 bp\n",
      "  Read 2:     9,635,232 bp\n",
      "Total written (filtered):     18,001,407 bp (93.4%)\n",
      "  Read 1:     8,982,670 bp\n",
      "  Read 2:     9,018,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t210\t0.0\t1\t0 202 8\n",
      "20\t32321\t0.0\t2\t30808 1480 33\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32474 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t591\t0.0\t1\t0 591\n",
      "19\t31803\t0.0\t1\t30916 887\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S452_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S452_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S452_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S452_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S452_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S452_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S452_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S452_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,066 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.85 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,066\n",
      "  Read 1 with adapter:                  57,008 (99.9%)\n",
      "  Read 2 with adapter:                  56,707 (99.4%)\n",
      "Pairs written (passing filters):        57,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,647,990 bp\n",
      "  Read 1:    16,833,862 bp\n",
      "  Read 2:    16,814,128 bp\n",
      "Total written (filtered):     31,432,151 bp (93.4%)\n",
      "  Read 1:    15,694,761 bp\n",
      "  Read 2:    15,737,390 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1060\t0.0\t1\t0 1001 59\n",
      "20\t55806\t0.0\t2\t52821 2920 65\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t770\t0.0\t1\t0 770\n",
      "19\t55862\t0.0\t1\t54404 1458\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S453_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S453_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S453_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S453_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S453_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S453_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S453_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S453_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        23,439 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.59 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,439\n",
      "  Read 1 with adapter:                  23,417 (99.9%)\n",
      "  Read 2 with adapter:                  23,305 (99.4%)\n",
      "Pairs written (passing filters):        23,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,828,599 bp\n",
      "  Read 1:     6,914,407 bp\n",
      "  Read 2:     6,914,192 bp\n",
      "Total written (filtered):     12,918,136 bp (93.4%)\n",
      "  Read 1:     6,446,522 bp\n",
      "  Read 2:     6,471,614 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 23417 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t460\t0.0\t1\t0 435 25\n",
      "20\t22886\t0.0\t2\t21670 1176 40\n",
      "21\t37\t0.0\t2\t0 37\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t23052\t0.0\t1\t22447 605\n",
      "20\t18\t0.0\t1\t0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S454_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S454_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S454_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S454_167_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S454_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S454_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S454_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S454_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,918 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.65 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,918\n",
      "  Read 1 with adapter:                  54,865 (99.9%)\n",
      "  Read 2 with adapter:                  54,690 (99.6%)\n",
      "Pairs written (passing filters):        54,918 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,400,072 bp\n",
      "  Read 1:    16,200,322 bp\n",
      "  Read 2:    16,199,750 bp\n",
      "Total written (filtered):     30,265,210 bp (93.4%)\n",
      "  Read 1:    15,104,056 bp\n",
      "  Read 2:    15,161,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54865 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1050\t0.0\t1\t0 995 55\n",
      "20\t53658\t0.0\t2\t50831 2748 79\n",
      "21\t78\t0.0\t2\t0 78\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t580\t0.0\t1\t0 580\n",
      "19\t54044\t0.0\t1\t52481 1563\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S455_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S455_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S455_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S455_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S455_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S455_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S455_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S455_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,829 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.14 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,829\n",
      "  Read 1 with adapter:                  47,772 (99.9%)\n",
      "  Read 2 with adapter:                  47,471 (99.3%)\n",
      "Pairs written (passing filters):        47,829 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,201,978 bp\n",
      "  Read 1:    14,106,261 bp\n",
      "  Read 2:    14,095,717 bp\n",
      "Total written (filtered):     26,345,689 bp (93.4%)\n",
      "  Read 1:    13,151,046 bp\n",
      "  Read 2:    13,194,643 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t350\t0.0\t1\t0 336 14\n",
      "20\t47303\t0.0\t2\t44957 2293 53\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t951\t0.0\t1\t0 951\n",
      "19\t46444\t0.0\t1\t45187 1257\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S456_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S456_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S456_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S456_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S456_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S456_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S456_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S456_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,080 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 1.67 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,080\n",
      "  Read 1 with adapter:                  24,061 (99.9%)\n",
      "  Read 2 with adapter:                  23,917 (99.3%)\n",
      "Pairs written (passing filters):        24,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,199,805 bp\n",
      "  Read 1:     7,103,589 bp\n",
      "  Read 2:     7,096,216 bp\n",
      "Total written (filtered):     13,264,823 bp (93.4%)\n",
      "  Read 1:     6,622,682 bp\n",
      "  Read 2:     6,642,141 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t359\t0.0\t1\t0 346 13\n",
      "20\t23618\t0.0\t2\t22311 1276 31\n",
      "21\t42\t0.0\t2\t0 38 4\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t384\t0.0\t1\t0 384\n",
      "19\t23497\t0.0\t1\t22887 610\n",
      "20\t36\t0.0\t1\t0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S457_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S457_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S457_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S457_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S457_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S457_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S457_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S457_173_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,314 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 1.81 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,314\n",
      "  Read 1 with adapter:                  25,290 (99.9%)\n",
      "  Read 2 with adapter:                  25,191 (99.5%)\n",
      "Pairs written (passing filters):        25,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,934,839 bp\n",
      "  Read 1:     7,467,431 bp\n",
      "  Read 2:     7,467,408 bp\n",
      "Total written (filtered):     13,951,083 bp (93.4%)\n",
      "  Read 1:     6,961,995 bp\n",
      "  Read 2:     6,989,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 25290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t422\t0.0\t1\t0 407 15\n",
      "20\t24791\t0.0\t2\t23499 1261 31\n",
      "21\t44\t0.0\t2\t0 43 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t343\t0.0\t1\t0 343\n",
      "19\t24814\t0.0\t1\t24131 683\n",
      "20\t34\t0.0\t1\t0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S458_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S458_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S458_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S458_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S458_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S458_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S458_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S458_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,026 reads  @     76.1 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.66 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,026\n",
      "  Read 1 with adapter:                  47,980 (99.9%)\n",
      "  Read 2 with adapter:                  47,855 (99.6%)\n",
      "Pairs written (passing filters):        48,026 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,331,681 bp\n",
      "  Read 1:    14,164,906 bp\n",
      "  Read 2:    14,166,775 bp\n",
      "Total written (filtered):     26,463,966 bp (93.4%)\n",
      "  Read 1:    13,205,971 bp\n",
      "  Read 2:    13,257,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t740\t0.0\t1\t0 701 39\n",
      "20\t47119\t0.0\t2\t44659 2401 59\n",
      "21\t67\t0.0\t2\t0 64 3\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t537\t0.0\t1\t0 537\n",
      "19\t47246\t0.0\t1\t45984 1262\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S459_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S459_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S459_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S459_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S459_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S459_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S459_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S459_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        57,590 reads  @    131.6 µs/read;   0.46 M reads/minute\n",
      "Finished in 7.58 s (132 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,590\n",
      "  Read 1 with adapter:                  57,528 (99.9%)\n",
      "  Read 2 with adapter:                  57,283 (99.5%)\n",
      "Pairs written (passing filters):        57,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,974,104 bp\n",
      "  Read 1:    16,986,243 bp\n",
      "  Read 2:    16,987,861 bp\n",
      "Total written (filtered):     31,736,285 bp (93.4%)\n",
      "  Read 1:    15,835,907 bp\n",
      "  Read 2:    15,900,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t376\t0.0\t1\t0 365 11\n",
      "20\t56982\t0.0\t2\t54194 2724 64\n",
      "21\t100\t0.0\t2\t0 97 3\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t993\t0.0\t1\t0 993\n",
      "19\t56191\t0.0\t1\t54799 1392\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S460_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S460_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S460_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S460_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S460_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S460_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S460_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S460_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8<-------] 00:00:06        60,568 reads  @    113.3 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.87 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,568\n",
      "  Read 1 with adapter:                  60,520 (99.9%)\n",
      "  Read 2 with adapter:                  60,257 (99.5%)\n",
      "Pairs written (passing filters):        60,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,720,578 bp\n",
      "  Read 1:    17,867,550 bp\n",
      "  Read 2:    17,853,028 bp\n",
      "Total written (filtered):     33,366,542 bp (93.4%)\n",
      "  Read 1:    16,657,601 bp\n",
      "  Read 2:    16,708,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60520 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t556\t0.0\t1\t0 528 28\n",
      "20\t59833\t0.0\t2\t56645 3120 68\n",
      "21\t73\t0.0\t2\t0 70 3\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60257 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t907\t0.0\t1\t0 907\n",
      "19\t59239\t0.0\t1\t57891 1348\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S461_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S461_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S461_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S461_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S461_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S461_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S461_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S461_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        61,276 reads  @    122.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 7.50 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,276\n",
      "  Read 1 with adapter:                  61,222 (99.9%)\n",
      "  Read 2 with adapter:                  61,055 (99.6%)\n",
      "Pairs written (passing filters):        61,276 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,152,031 bp\n",
      "  Read 1:    18,075,969 bp\n",
      "  Read 2:    18,076,062 bp\n",
      "Total written (filtered):     33,768,542 bp (93.4%)\n",
      "  Read 1:    16,851,911 bp\n",
      "  Read 2:    16,916,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61222 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t503\t0.0\t1\t0 484 19\n",
      "20\t60561\t0.0\t2\t57420 3058 83\n",
      "21\t103\t0.0\t2\t0 101 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t701\t0.0\t1\t0 701\n",
      "19\t60267\t0.0\t1\t58962 1305\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S462_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S462_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S462_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S462_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S462_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S462_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S462_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S462_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,983 reads  @    121.6 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.35 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,983\n",
      "  Read 1 with adapter:                  43,940 (99.9%)\n",
      "  Read 2 with adapter:                  43,812 (99.6%)\n",
      "Pairs written (passing filters):        43,983 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,949,409 bp\n",
      "  Read 1:    12,974,734 bp\n",
      "  Read 2:    12,974,675 bp\n",
      "Total written (filtered):     24,238,948 bp (93.4%)\n",
      "  Read 1:    12,096,277 bp\n",
      "  Read 2:    12,142,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t422\t0.0\t1\t0 393 29\n",
      "20\t43423\t0.0\t2\t41143 2232 48\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t480\t0.0\t1\t0 480\n",
      "19\t43276\t0.0\t1\t42241 1035\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S463_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S463_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S463_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S463_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S463_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S463_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S463_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S463_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        65,890 reads  @    125.7 µs/read;   0.48 M reads/minute\n",
      "Finished in 8.29 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,890\n",
      "  Read 1 with adapter:                  65,824 (99.9%)\n",
      "  Read 2 with adapter:                  65,452 (99.3%)\n",
      "Pairs written (passing filters):        65,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,842,127 bp\n",
      "  Read 1:    19,435,185 bp\n",
      "  Read 2:    19,406,942 bp\n",
      "Total written (filtered):     36,283,217 bp (93.4%)\n",
      "  Read 1:    18,118,939 bp\n",
      "  Read 2:    18,164,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65824 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t399\t0.0\t1\t0 378 21\n",
      "20\t65248\t0.0\t2\t62026 3151 71\n",
      "21\t121\t0.0\t2\t0 118 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1001\t0.0\t1\t0 1001\n",
      "19\t64374\t0.0\t1\t62951 1423\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S464_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S464_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S464_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S464_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S464_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S464_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S464_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S464_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        60,330 reads  @    135.4 µs/read;   0.44 M reads/minute\n",
      "Finished in 8.18 s (136 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,330\n",
      "  Read 1 with adapter:                  60,278 (99.9%)\n",
      "  Read 2 with adapter:                  59,980 (99.4%)\n",
      "Pairs written (passing filters):        60,330 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,577,871 bp\n",
      "  Read 1:    17,797,223 bp\n",
      "  Read 2:    17,780,648 bp\n",
      "Total written (filtered):     33,234,691 bp (93.4%)\n",
      "  Read 1:    16,592,780 bp\n",
      "  Read 2:    16,641,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t56\t0.0\t1\t0 0 56\n",
      "19\t1173\t0.0\t1\t0 1128 45\n",
      "20\t58920\t0.0\t2\t56144 2719 57\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t976\t0.0\t1\t0 976\n",
      "19\t58911\t0.0\t1\t57538 1373\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S465_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S465_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S465_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S465_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S465_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S465_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S465_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S465_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        48,628 reads  @    104.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.07 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,628\n",
      "  Read 1 with adapter:                  48,601 (99.9%)\n",
      "  Read 2 with adapter:                  48,423 (99.6%)\n",
      "Pairs written (passing filters):        48,628 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,690,448 bp\n",
      "  Read 1:    14,345,248 bp\n",
      "  Read 2:    14,345,200 bp\n",
      "Total written (filtered):     26,799,847 bp (93.4%)\n",
      "  Read 1:    13,374,097 bp\n",
      "  Read 2:    13,425,750 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t959\t0.0\t1\t0 926 33\n",
      "20\t47475\t0.0\t2\t45343 2095 37\n",
      "21\t88\t0.0\t2\t0 86 2\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t655\t0.0\t1\t0 655\n",
      "19\t47700\t0.0\t1\t46610 1090\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S466_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S466_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S466_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S466_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S466_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S466_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S466_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S466_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        51,672 reads  @     84.2 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.35 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,672\n",
      "  Read 1 with adapter:                  51,638 (99.9%)\n",
      "  Read 2 with adapter:                  51,452 (99.6%)\n",
      "Pairs written (passing filters):        51,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,480,750 bp\n",
      "  Read 1:    15,243,227 bp\n",
      "  Read 2:    15,237,523 bp\n",
      "Total written (filtered):     28,471,959 bp (93.4%)\n",
      "  Read 1:    14,211,483 bp\n",
      "  Read 2:    14,260,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51638 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t57\t0.0\t1\t0 0 57\n",
      "19\t1082\t0.0\t1\t0 1029 53\n",
      "20\t50365\t0.0\t2\t47966 2330 69\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t607\t0.0\t1\t0 607\n",
      "19\t50779\t0.0\t1\t49588 1191\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S467_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S467_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S467_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S467_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S467_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S467_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S467_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S467_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,128 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.79 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,128\n",
      "  Read 1 with adapter:                  66,065 (99.9%)\n",
      "  Read 2 with adapter:                  65,657 (99.3%)\n",
      "Pairs written (passing filters):        66,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,011,614 bp\n",
      "  Read 1:    19,506,791 bp\n",
      "  Read 2:    19,504,823 bp\n",
      "Total written (filtered):     36,444,197 bp (93.4%)\n",
      "  Read 1:    18,185,784 bp\n",
      "  Read 2:    18,258,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t468\t0.0\t1\t0 447 21\n",
      "20\t65423\t0.0\t2\t62067 3283 73\n",
      "21\t117\t0.0\t2\t0 115 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1175\t0.0\t1\t0 1175\n",
      "19\t64380\t0.0\t1\t62448 1932\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S468_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S468_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S468_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S468_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S468_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S468_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S468_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S468_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        55,723 reads  @    105.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.88 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,723\n",
      "  Read 1 with adapter:                  55,679 (99.9%)\n",
      "  Read 2 with adapter:                  55,328 (99.3%)\n",
      "Pairs written (passing filters):        55,723 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,857,054 bp\n",
      "  Read 1:    16,438,265 bp\n",
      "  Read 2:    16,418,789 bp\n",
      "Total written (filtered):     30,693,849 bp (93.4%)\n",
      "  Read 1:    15,325,511 bp\n",
      "  Read 2:    15,368,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t903\t0.0\t1\t0 868 35\n",
      "20\t54609\t0.0\t2\t51596 2933 80\n",
      "21\t93\t0.0\t2\t0 89 4\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t870\t0.0\t1\t0 870\n",
      "19\t54369\t0.0\t1\t52846 1523\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S469_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S469_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S469_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S469_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S469_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S469_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S469_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S469_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        56,804 reads  @     87.1 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.95 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,804\n",
      "  Read 1 with adapter:                  56,753 (99.9%)\n",
      "  Read 2 with adapter:                  56,556 (99.6%)\n",
      "Pairs written (passing filters):        56,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,513,974 bp\n",
      "  Read 1:    16,757,076 bp\n",
      "  Read 2:    16,756,898 bp\n",
      "Total written (filtered):     31,305,774 bp (93.4%)\n",
      "  Read 1:    15,622,899 bp\n",
      "  Read 2:    15,682,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t978\t0.0\t1\t0 929 49\n",
      "20\t55598\t0.0\t2\t52600 2917 81\n",
      "21\t103\t0.0\t2\t0 100 3\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56556 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t624\t0.0\t1\t0 624\n",
      "19\t55849\t0.0\t1\t54341 1508\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S470_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S470_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S470_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S470_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S470_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S470_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S470_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S470_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        61,681 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.85 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,681\n",
      "  Read 1 with adapter:                  61,639 (99.9%)\n",
      "  Read 2 with adapter:                  61,377 (99.5%)\n",
      "Pairs written (passing filters):        61,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,390,075 bp\n",
      "  Read 1:    18,195,653 bp\n",
      "  Read 2:    18,194,422 bp\n",
      "Total written (filtered):     33,992,724 bp (93.4%)\n",
      "  Read 1:    16,963,840 bp\n",
      "  Read 2:    17,028,884 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t1053\t0.0\t1\t0 996 57\n",
      "20\t60379\t0.0\t2\t57059 3244 76\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61377 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t703\t0.0\t1\t0 703\n",
      "19\t60596\t0.0\t1\t58797 1799\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S471_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S471_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S471_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S471_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S471_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S471_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S471_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S471_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        67,892 reads  @     97.5 µs/read;   0.62 M reads/minute\n",
      "Finished in 6.63 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,892\n",
      "  Read 1 with adapter:                  67,822 (99.9%)\n",
      "  Read 2 with adapter:                  65,765 (96.9%)\n",
      "Pairs written (passing filters):        67,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,933,723 bp\n",
      "  Read 1:    20,026,579 bp\n",
      "  Read 2:    19,907,144 bp\n",
      "Total written (filtered):     37,328,479 bp (93.5%)\n",
      "  Read 1:    18,670,399 bp\n",
      "  Read 2:    18,658,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t439\t0.0\t1\t0 419 20\n",
      "20\t67201\t0.0\t2\t63803 3316 82\n",
      "21\t117\t0.0\t2\t0 115 2\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65765 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t559\t0.0\t1\t0 559\n",
      "19\t65118\t0.0\t1\t63390 1728\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S472_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S472_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S472_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S472_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S472_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S472_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S472_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S472_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        47,730 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.18 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,730\n",
      "  Read 1 with adapter:                  47,644 (99.8%)\n",
      "  Read 2 with adapter:                  47,314 (99.1%)\n",
      "Pairs written (passing filters):        47,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,130,658 bp\n",
      "  Read 1:    14,072,627 bp\n",
      "  Read 2:    14,058,031 bp\n",
      "Total written (filtered):     26,280,144 bp (93.4%)\n",
      "  Read 1:    13,120,370 bp\n",
      "  Read 2:    13,159,774 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t706\t0.0\t1\t0 675 31\n",
      "20\t46798\t0.0\t2\t44364 2369 65\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t772\t0.0\t1\t0 772\n",
      "19\t46479\t0.0\t1\t45044 1435\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S473_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S473_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S473_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S473_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S473_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S473_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S473_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S473_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        51,927 reads  @     89.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.63 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,927\n",
      "  Read 1 with adapter:                  51,863 (99.9%)\n",
      "  Read 2 with adapter:                  51,706 (99.6%)\n",
      "Pairs written (passing filters):        51,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,634,460 bp\n",
      "  Read 1:    15,316,213 bp\n",
      "  Read 2:    15,318,247 bp\n",
      "Total written (filtered):     28,616,257 bp (93.4%)\n",
      "  Read 1:    14,279,714 bp\n",
      "  Read 2:    14,336,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t850\t0.0\t1\t0 813 37\n",
      "20\t50842\t0.0\t2\t48310 2465 67\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t780\t0.0\t1\t0 780\n",
      "19\t50856\t0.0\t1\t49447 1409\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S474_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S474_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S474_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S474_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S474_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S474_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S474_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S474_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        70,404 reads  @     87.0 µs/read;   0.69 M reads/minute\n",
      "Finished in 6.13 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,404\n",
      "  Read 1 with adapter:                  69,493 (98.7%)\n",
      "  Read 2 with adapter:                  70,061 (99.5%)\n",
      "Pairs written (passing filters):        70,404 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,409,967 bp\n",
      "  Read 1:    20,646,407 bp\n",
      "  Read 2:    20,763,560 bp\n",
      "Total written (filtered):     38,690,741 bp (93.4%)\n",
      "  Read 1:    19,257,610 bp\n",
      "  Read 2:    19,433,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t62\t0.0\t1\t0 0 62\n",
      "19\t1142\t0.0\t1\t0 1079 63\n",
      "20\t68137\t0.0\t2\t64543 3498 96\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t807\t0.0\t1\t0 807\n",
      "19\t69177\t0.0\t1\t67032 2145\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S475_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S475_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S475_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S475_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S475_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S475_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S475_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S475_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        60,916 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.07 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,916\n",
      "  Read 1 with adapter:                  60,866 (99.9%)\n",
      "  Read 2 with adapter:                  60,664 (99.6%)\n",
      "Pairs written (passing filters):        60,916 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,940,343 bp\n",
      "  Read 1:    17,970,194 bp\n",
      "  Read 2:    17,970,149 bp\n",
      "Total written (filtered):     33,571,551 bp (93.4%)\n",
      "  Read 1:    16,753,132 bp\n",
      "  Read 2:    16,818,419 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60866 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t415\t0.0\t1\t0 397 18\n",
      "20\t60291\t0.0\t2\t57245 2979 67\n",
      "21\t103\t0.0\t2\t0 103\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1017\t0.0\t1\t0 1017\n",
      "19\t59516\t0.0\t1\t57987 1529\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S476_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S476_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S476_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S476_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S476_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S476_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S476_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S476_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        60,780 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.08 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,780\n",
      "  Read 1 with adapter:                  60,740 (99.9%)\n",
      "  Read 2 with adapter:                  60,508 (99.6%)\n",
      "Pairs written (passing filters):        60,779 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,847,690 bp\n",
      "  Read 1:    17,929,980 bp\n",
      "  Read 2:    17,917,710 bp\n",
      "Total written (filtered):     33,484,484 bp (93.4%)\n",
      "  Read 1:    16,715,557 bp\n",
      "  Read 2:    16,768,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t750\t0.0\t1\t0 720 30\n",
      "20\t59820\t0.0\t2\t56736 2989 95\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t971\t0.0\t1\t0 971\n",
      "19\t59435\t0.0\t1\t58146 1289\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S477_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S477_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S477_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S477_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S477_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S477_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S477_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S477_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,614 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.86 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,614\n",
      "  Read 1 with adapter:                  50,573 (99.9%)\n",
      "  Read 2 with adapter:                  50,402 (99.6%)\n",
      "Pairs written (passing filters):        50,614 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,861,997 bp\n",
      "  Read 1:    14,931,012 bp\n",
      "  Read 2:    14,930,985 bp\n",
      "Total written (filtered):     27,894,085 bp (93.4%)\n",
      "  Read 1:    13,920,147 bp\n",
      "  Read 2:    13,973,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t711\t0.0\t1\t0 669 42\n",
      "20\t49704\t0.0\t2\t47069 2564 71\n",
      "21\t96\t0.0\t2\t0 91 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t658\t0.0\t1\t0 658\n",
      "19\t49677\t0.0\t1\t48507 1170\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S478_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S478_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S478_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S478_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S478_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S478_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S478_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S478_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        49,249 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.07 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,249\n",
      "  Read 1 with adapter:                  49,197 (99.9%)\n",
      "  Read 2 with adapter:                  49,105 (99.7%)\n",
      "Pairs written (passing filters):        49,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,055,528 bp\n",
      "  Read 1:    14,528,424 bp\n",
      "  Read 2:    14,527,104 bp\n",
      "Total written (filtered):     27,139,660 bp (93.4%)\n",
      "  Read 1:    13,545,072 bp\n",
      "  Read 2:    13,594,588 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t688\t0.0\t1\t0 649 39\n",
      "20\t48377\t0.0\t2\t45886 2434 57\n",
      "21\t84\t0.0\t2\t0 81 3\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49105 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t542\t0.0\t1\t0 542\n",
      "19\t48500\t0.0\t1\t47410 1090\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S479_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S479_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S479_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S479_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S479_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S479_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S479_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S479_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,189 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.10 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,189\n",
      "  Read 1 with adapter:                  45,140 (99.9%)\n",
      "  Read 2 with adapter:                  44,948 (99.5%)\n",
      "Pairs written (passing filters):        45,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,642,227 bp\n",
      "  Read 1:    13,329,570 bp\n",
      "  Read 2:    13,312,657 bp\n",
      "Total written (filtered):     24,886,282 bp (93.4%)\n",
      "  Read 1:    12,427,005 bp\n",
      "  Read 2:    12,459,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t306\t0.0\t1\t0 289 17\n",
      "20\t44728\t0.0\t2\t42471 2200 57\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t699\t0.0\t1\t0 699\n",
      "19\t44182\t0.0\t1\t43257 925\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S480_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S480_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S480_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S480_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S480_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S480_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S480_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S480_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        50,736 reads  @     80.8 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.10 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,736\n",
      "  Read 1 with adapter:                  50,694 (99.9%)\n",
      "  Read 2 with adapter:                  50,496 (99.5%)\n",
      "Pairs written (passing filters):        50,736 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,918,398 bp\n",
      "  Read 1:    14,966,843 bp\n",
      "  Read 2:    14,951,555 bp\n",
      "Total written (filtered):     27,946,408 bp (93.4%)\n",
      "  Read 1:    13,953,583 bp\n",
      "  Read 2:    13,992,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t697\t0.0\t1\t0 665 32\n",
      "20\t49844\t0.0\t2\t47278 2508 58\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t756\t0.0\t1\t0 756\n",
      "19\t49678\t0.0\t1\t48615 1063\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S481_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S481_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S481_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S481_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S481_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S481_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S481_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S481_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,030 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.23 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,030\n",
      "  Read 1 with adapter:                  50,986 (99.9%)\n",
      "  Read 2 with adapter:                  50,831 (99.6%)\n",
      "Pairs written (passing filters):        51,030 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,107,380 bp\n",
      "  Read 1:    15,053,717 bp\n",
      "  Read 2:    15,053,663 bp\n",
      "Total written (filtered):     28,123,297 bp (93.4%)\n",
      "  Read 1:    14,034,740 bp\n",
      "  Read 2:    14,088,557 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50986 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t785\t0.0\t1\t0 759 26\n",
      "20\t50076\t0.0\t2\t47409 2589 78\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t758\t0.0\t1\t0 758\n",
      "19\t49998\t0.0\t1\t48811 1187\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S482_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S482_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S482_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S482_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S482_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S482_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S482_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S482_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,001 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.25 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,001\n",
      "  Read 1 with adapter:                  42,970 (99.9%)\n",
      "  Read 2 with adapter:                  42,860 (99.7%)\n",
      "Pairs written (passing filters):        43,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,367,936 bp\n",
      "  Read 1:    12,684,459 bp\n",
      "  Read 2:    12,683,477 bp\n",
      "Total written (filtered):     23,695,242 bp (93.4%)\n",
      "  Read 1:    11,825,666 bp\n",
      "  Read 2:    11,869,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t658\t0.0\t1\t0 634 24\n",
      "20\t42186\t0.0\t2\t40009 2118 59\n",
      "21\t73\t0.0\t2\t0 72 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t497\t0.0\t1\t0 497\n",
      "19\t42305\t0.0\t1\t41320 985\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S483_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S483_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S483_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S483_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S483_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S483_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S483_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S483_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,979 reads  @     83.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.71 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,979\n",
      "  Read 1 with adapter:                  67,922 (99.9%)\n",
      "  Read 2 with adapter:                  67,676 (99.6%)\n",
      "Pairs written (passing filters):        67,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,104,489 bp\n",
      "  Read 1:    20,052,875 bp\n",
      "  Read 2:    20,051,614 bp\n",
      "Total written (filtered):     37,460,979 bp (93.4%)\n",
      "  Read 1:    18,694,199 bp\n",
      "  Read 2:    18,766,780 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67922 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t481\t0.0\t1\t0 459 22\n",
      "20\t67256\t0.0\t2\t63954 3214 88\n",
      "21\t119\t0.0\t2\t0 119\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1131\t0.0\t1\t0 1131\n",
      "19\t66424\t0.0\t1\t64678 1746\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S484_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S484_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S484_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S484_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S484_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S484_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S484_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S484_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        15,565 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 1.37 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,565\n",
      "  Read 1 with adapter:                  15,549 (99.9%)\n",
      "  Read 2 with adapter:                  15,469 (99.4%)\n",
      "Pairs written (passing filters):        15,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,176,152 bp\n",
      "  Read 1:     4,591,097 bp\n",
      "  Read 2:     4,585,055 bp\n",
      "Total written (filtered):      8,571,640 bp (93.4%)\n",
      "  Read 1:     4,280,253 bp\n",
      "  Read 2:     4,291,387 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 15549 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t149\t0.0\t1\t0 143 6\n",
      "20\t15360\t0.0\t2\t14468 868 24\n",
      "21\t23\t0.0\t2\t0 23\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t270\t0.0\t1\t0 270\n",
      "19\t15172\t0.0\t1\t14830 342\n",
      "20\t27\t0.0\t1\t0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S485_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S485_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S485_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S485_229_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S485_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S485_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S485_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S485_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,760 reads  @     81.9 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.65 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,760\n",
      "  Read 1 with adapter:                  56,693 (99.9%)\n",
      "  Read 2 with adapter:                  56,476 (99.5%)\n",
      "Pairs written (passing filters):        56,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,484,588 bp\n",
      "  Read 1:    16,740,898 bp\n",
      "  Read 2:    16,743,690 bp\n",
      "Total written (filtered):     31,278,862 bp (93.4%)\n",
      "  Read 1:    15,607,545 bp\n",
      "  Read 2:    15,671,317 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t581\t0.0\t1\t0 550 31\n",
      "20\t55984\t0.0\t2\t52955 2947 82\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56476 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t759\t0.0\t1\t0 759\n",
      "19\t55629\t0.0\t1\t54204 1425\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S486_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S486_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S486_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S486_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S486_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S486_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S486_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S486_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        74,811 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 6.44 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,811\n",
      "  Read 1 with adapter:                  74,442 (99.5%)\n",
      "  Read 2 with adapter:                  74,205 (99.2%)\n",
      "Pairs written (passing filters):        74,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,135,376 bp\n",
      "  Read 1:    22,067,800 bp\n",
      "  Read 2:    22,067,576 bp\n",
      "Total written (filtered):     41,238,078 bp (93.4%)\n",
      "  Read 1:    20,579,553 bp\n",
      "  Read 2:    20,658,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t815\t0.0\t1\t0 768 47\n",
      "20\t73390\t0.0\t2\t68935 4335 120\n",
      "21\t96\t0.0\t2\t0 93 3\n",
      "22\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t927\t0.0\t1\t0 927\n",
      "19\t73195\t0.0\t1\t70828 2367\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S487_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S487_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S487_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S487_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S487_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S487_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S487_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S487_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,242 reads  @     93.8 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.19 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,242\n",
      "  Read 1 with adapter:                  55,176 (99.9%)\n",
      "  Read 2 with adapter:                  54,954 (99.5%)\n",
      "Pairs written (passing filters):        55,242 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,586,688 bp\n",
      "  Read 1:    16,290,917 bp\n",
      "  Read 2:    16,295,771 bp\n",
      "Total written (filtered):     30,439,964 bp (93.4%)\n",
      "  Read 1:    15,187,607 bp\n",
      "  Read 2:    15,252,357 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t355\t0.0\t1\t0 339 16\n",
      "20\t54651\t0.0\t2\t51875 2709 67\n",
      "21\t107\t0.0\t2\t0 107\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t791\t0.0\t1\t0 791\n",
      "19\t54084\t0.0\t1\t52712 1372\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S488_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S488_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S488_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S488_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S488_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S488_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S488_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S488_235_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        66,067 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.57 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,067\n",
      "  Read 1 with adapter:                  66,016 (99.9%)\n",
      "  Read 2 with adapter:                  65,602 (99.3%)\n",
      "Pairs written (passing filters):        66,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,956,896 bp\n",
      "  Read 1:    19,489,462 bp\n",
      "  Read 2:    19,467,434 bp\n",
      "Total written (filtered):     36,391,197 bp (93.4%)\n",
      "  Read 1:    18,169,260 bp\n",
      "  Read 2:    18,221,937 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t579\t0.0\t1\t0 542 37\n",
      "20\t65214\t0.0\t2\t62059 3078 77\n",
      "21\t128\t0.0\t2\t0 125 3\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1047\t0.0\t1\t0 1047\n",
      "19\t64449\t0.0\t1\t62839 1610\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S489_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S489_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S489_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S489_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S489_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7z31ns6z/S489_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S489_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rr0nehp6/8a36fe07-fc65-4b35-a7fa-463e5d9ec5e5/data/S489_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        14,325 reads  @    103.0 µs/read;   0.58 M reads/minute\n",
      "Finished in 1.48 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,325\n",
      "  Read 1 with adapter:                  14,310 (99.9%)\n",
      "  Read 2 with adapter:                  14,268 (99.6%)\n",
      "Pairs written (passing filters):        14,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,451,705 bp\n",
      "  Read 1:     4,225,872 bp\n",
      "  Read 2:     4,225,833 bp\n",
      "Total written (filtered):      7,894,703 bp (93.4%)\n",
      "  Read 1:     3,939,729 bp\n",
      "  Read 2:     3,954,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 14310 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t121\t0.0\t1\t0 116 5\n",
      "20\t14137\t0.0\t2\t13497 628 12\n",
      "21\t32\t0.0\t2\t0 31 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 14268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t250\t0.0\t1\t0 250\n",
      "19\t14001\t0.0\t1\t13660 341\n",
      "20\t17\t0.0\t1\t0 17\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: b-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: b-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-68hnv13s')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: cf2a3ce9-f9ee-42b5-9e07-feaa6b15a816>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.b import the fastq files for library b into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib3-zhan_manifest_run2_v1.csv \\\n",
    "  --output-path b-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.b trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences b-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences b-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.b summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data b-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization b-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('b-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: b-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: b-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: b-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.b Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs b-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 273 \\\n",
    "  --p-trunc-len-r 213 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table b-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats b-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: b-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-lvcg_h0m')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1ceb737c-18a1-44c2-b184-a149b7691bd9>"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.b view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table b-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization b-table.qzv \n",
    "q2.Visualization.load('b-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: b-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.b view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization b-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib1-zhan_manifest_r3.csv as PairedEndFastqManifestPhred33 to c-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s501_136_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s501_136_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s501_136_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s501_136_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,154 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 2.99 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,154\n",
      "  Read 1 with adapter:                  34,142 (100.0%)\n",
      "  Read 2 with adapter:                  34,086 (99.8%)\n",
      "Pairs written (passing filters):        34,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,150,240 bp\n",
      "  Read 1:    10,075,422 bp\n",
      "  Read 2:    10,074,818 bp\n",
      "Total written (filtered):     18,819,993 bp (93.4%)\n",
      "  Read 1:     9,392,678 bp\n",
      "  Read 2:     9,427,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t173\t0.0\t1\t0 167 6\n",
      "20\t33887\t0.0\t2\t33337 542 8\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t182\t0.0\t1\t0 182\n",
      "19\t33853\t0.0\t1\t33425 428\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s502_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s502_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s502_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s502_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s502_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s502_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s502_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s502_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,465 reads  @     75.2 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.35 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,465\n",
      "  Read 1 with adapter:                  44,442 (99.9%)\n",
      "  Read 2 with adapter:                  44,313 (99.7%)\n",
      "Pairs written (passing filters):        44,465 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,233,081 bp\n",
      "  Read 1:    13,116,271 bp\n",
      "  Read 2:    13,116,810 bp\n",
      "Total written (filtered):     24,502,610 bp (93.4%)\n",
      "  Read 1:    12,227,610 bp\n",
      "  Read 2:    12,275,000 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t277\t0.0\t1\t0 270 7\n",
      "20\t44037\t0.0\t2\t43339 689 9\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t44014\t0.0\t1\t43089 925\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s503_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s503_138_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s503_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s503_138_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s503_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s503_138_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s503_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s503_138_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,415 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.24 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,415\n",
      "  Read 1 with adapter:                  33,399 (100.0%)\n",
      "  Read 2 with adapter:                  33,295 (99.6%)\n",
      "Pairs written (passing filters):        33,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,714,196 bp\n",
      "  Read 1:     9,857,177 bp\n",
      "  Read 2:     9,857,019 bp\n",
      "Total written (filtered):     18,413,824 bp (93.4%)\n",
      "  Read 1:     9,189,318 bp\n",
      "  Read 2:     9,224,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t188\t0.0\t1\t0 184 4\n",
      "20\t33125\t0.0\t2\t32660 462 3\n",
      "21\t49\t0.0\t2\t0 49\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33295 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t33067\t0.0\t1\t32478 589\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s504_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s504_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s504_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s504_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s504_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s504_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s504_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s504_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,418 reads  @     74.7 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.65 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,418\n",
      "  Read 1 with adapter:                  35,407 (100.0%)\n",
      "  Read 2 with adapter:                  35,183 (99.3%)\n",
      "Pairs written (passing filters):        35,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,896,348 bp\n",
      "  Read 1:    10,448,302 bp\n",
      "  Read 2:    10,448,046 bp\n",
      "Total written (filtered):     19,519,971 bp (93.4%)\n",
      "  Read 1:     9,740,275 bp\n",
      "  Read 2:     9,779,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35407 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t199\t0.0\t1\t0 195 4\n",
      "20\t35121\t0.0\t2\t34608 507 6\n",
      "21\t60\t0.0\t2\t0 57 3\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35183 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t34890\t0.0\t1\t33865 1025\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s506_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s506_140_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s506_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s506_140_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s506_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s506_140_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s506_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s506_140_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,587 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.10 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,587\n",
      "  Read 1 with adapter:                  44,576 (100.0%)\n",
      "  Read 2 with adapter:                  44,473 (99.7%)\n",
      "Pairs written (passing filters):        44,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,302,598 bp\n",
      "  Read 1:    13,152,825 bp\n",
      "  Read 2:    13,149,773 bp\n",
      "Total written (filtered):     24,566,321 bp (93.4%)\n",
      "  Read 1:    12,261,444 bp\n",
      "  Read 2:    12,304,877 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t264\t0.0\t1\t0 262 2\n",
      "20\t44175\t0.0\t2\t43437 735 3\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t44198\t0.0\t1\t43419 779\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s507_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s507_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s507_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s507_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s507_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s507_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s507_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s507_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,773 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.91 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,773\n",
      "  Read 1 with adapter:                  40,755 (100.0%)\n",
      "  Read 2 with adapter:                  40,648 (99.7%)\n",
      "Pairs written (passing filters):        40,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,051,910 bp\n",
      "  Read 1:    12,027,692 bp\n",
      "  Read 2:    12,024,218 bp\n",
      "Total written (filtered):     22,464,746 bp (93.4%)\n",
      "  Read 1:    11,212,749 bp\n",
      "  Read 2:    11,251,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t216\t0.0\t1\t0 214 2\n",
      "20\t40433\t0.0\t2\t39859 570 4\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40648 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t40391\t0.0\t1\t39748 643\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s508_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s508_142_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s508_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s508_142_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s508_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s508_142_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s508_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s508_142_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        37,196 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.67 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,196\n",
      "  Read 1 with adapter:                  37,184 (100.0%)\n",
      "  Read 2 with adapter:                  36,748 (98.8%)\n",
      "Pairs written (passing filters):        37,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,922,151 bp\n",
      "  Read 1:    10,972,813 bp\n",
      "  Read 2:    10,949,338 bp\n",
      "Total written (filtered):     20,480,495 bp (93.4%)\n",
      "  Read 1:    10,229,263 bp\n",
      "  Read 2:    10,251,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t229\t0.0\t1\t0 227 2\n",
      "20\t36847\t0.0\t2\t36200 636 11\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t36510\t0.0\t1\t35929 581\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s509_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s509_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s509_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s509_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s509_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s509_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s509_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s509_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        26,553 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.01 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,553\n",
      "  Read 1 with adapter:                  26,546 (100.0%)\n",
      "  Read 2 with adapter:                  26,509 (99.8%)\n",
      "Pairs written (passing filters):        26,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,665,865 bp\n",
      "  Read 1:     7,833,129 bp\n",
      "  Read 2:     7,832,736 bp\n",
      "Total written (filtered):     14,631,449 bp (93.4%)\n",
      "  Read 1:     7,302,306 bp\n",
      "  Read 2:     7,329,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26546 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t166\t0.0\t1\t0 164 2\n",
      "20\t26306\t0.0\t2\t25833 469 4\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t128\t0.0\t1\t0 128\n",
      "19\t26331\t0.0\t1\t25953 378\n",
      "20\t50\t0.0\t1\t0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s510_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s510_144_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s510_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s510_144_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s510_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s510_144_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s510_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s510_144_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,361 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.23 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,361\n",
      "  Read 1 with adapter:                  32,349 (100.0%)\n",
      "  Read 2 with adapter:                  32,280 (99.7%)\n",
      "Pairs written (passing filters):        32,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,092,532 bp\n",
      "  Read 1:     9,546,491 bp\n",
      "  Read 2:     9,546,041 bp\n",
      "Total written (filtered):     17,832,433 bp (93.4%)\n",
      "  Read 1:     8,899,598 bp\n",
      "  Read 2:     8,932,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32349 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t32090\t0.0\t2\t31608 476 6\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32280 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t32050\t0.0\t1\t31472 578\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s511_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s511_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s511_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s511_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s511_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s511_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s511_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s511_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        26,380 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 1.94 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,380\n",
      "  Read 1 with adapter:                  26,374 (100.0%)\n",
      "  Read 2 with adapter:                  26,309 (99.7%)\n",
      "Pairs written (passing filters):        26,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,563,558 bp\n",
      "  Read 1:     7,782,090 bp\n",
      "  Read 2:     7,781,468 bp\n",
      "Total written (filtered):     14,536,372 bp (93.4%)\n",
      "  Read 1:     7,254,700 bp\n",
      "  Read 2:     7,281,672 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26374 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t166\t0.0\t1\t0 162 4\n",
      "20\t26137\t0.0\t2\t25732 401 4\n",
      "21\t46\t0.0\t2\t0 46\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t122\t0.0\t1\t0 122\n",
      "19\t26140\t0.0\t1\t25702 438\n",
      "20\t47\t0.0\t1\t0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s512_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s512_146_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s512_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s512_146_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s512_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s512_146_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s512_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s512_146_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,575 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.32 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,575\n",
      "  Read 1 with adapter:                  30,568 (100.0%)\n",
      "  Read 2 with adapter:                  30,395 (99.4%)\n",
      "Pairs written (passing filters):        30,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,038,528 bp\n",
      "  Read 1:     9,019,616 bp\n",
      "  Read 2:     9,018,912 bp\n",
      "Total written (filtered):     16,849,909 bp (93.4%)\n",
      "  Read 1:     8,408,380 bp\n",
      "  Read 2:     8,441,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t198\t0.0\t1\t0 198\n",
      "20\t30277\t0.0\t2\t29834 440 3\n",
      "21\t52\t0.0\t2\t0 50 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30395 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t30133\t0.0\t1\t29360 773\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s514_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s514_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s514_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s514_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s514_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s514_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s514_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s514_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,975 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.20 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,975\n",
      "  Read 1 with adapter:                  31,960 (100.0%)\n",
      "  Read 2 with adapter:                  31,894 (99.7%)\n",
      "Pairs written (passing filters):        31,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,862,555 bp\n",
      "  Read 1:     9,432,608 bp\n",
      "  Read 2:     9,429,947 bp\n",
      "Total written (filtered):     17,617,527 bp (93.4%)\n",
      "  Read 1:     8,793,484 bp\n",
      "  Read 2:     8,824,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t181\t0.0\t1\t0 178 3\n",
      "20\t31675\t0.0\t2\t31195 474 6\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31894 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t137\t0.0\t1\t0 137\n",
      "19\t31702\t0.0\t1\t31209 493\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s515_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s515_148_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s515_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s515_148_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s515_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s515_148_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s515_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s515_148_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        33,598 reads  @     71.0 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.39 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,598\n",
      "  Read 1 with adapter:                  33,587 (100.0%)\n",
      "  Read 2 with adapter:                  33,508 (99.7%)\n",
      "Pairs written (passing filters):        33,598 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,820,347 bp\n",
      "  Read 1:     9,911,110 bp\n",
      "  Read 2:     9,909,237 bp\n",
      "Total written (filtered):     18,512,167 bp (93.4%)\n",
      "  Read 1:     9,239,513 bp\n",
      "  Read 2:     9,272,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t212\t0.0\t1\t0 209 3\n",
      "20\t33304\t0.0\t2\t32842 453 9\n",
      "21\t45\t0.0\t2\t0 44 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t137\t0.0\t1\t0 137\n",
      "19\t33303\t0.0\t1\t32796 507\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s516_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s516_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s516_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s516_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s516_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s516_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s516_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s516_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,616 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.57 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,616\n",
      "  Read 1 with adapter:                  36,602 (100.0%)\n",
      "  Read 2 with adapter:                  36,205 (98.9%)\n",
      "Pairs written (passing filters):        36,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,581,137 bp\n",
      "  Read 1:    10,801,712 bp\n",
      "  Read 2:    10,779,425 bp\n",
      "Total written (filtered):     20,161,392 bp (93.4%)\n",
      "  Read 1:    10,069,752 bp\n",
      "  Read 2:    10,091,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t194\t0.0\t1\t0 193 1\n",
      "20\t36299\t0.0\t2\t35746 549 4\n",
      "21\t60\t0.0\t2\t0 59 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t35969\t0.0\t1\t35425 544\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s517_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s517_150_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s517_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s517_150_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s517_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s517_150_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s517_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s517_150_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,298 reads  @     72.5 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.92 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,298\n",
      "  Read 1 with adapter:                  40,278 (100.0%)\n",
      "  Read 2 with adapter:                  40,221 (99.8%)\n",
      "Pairs written (passing filters):        40,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,775,788 bp\n",
      "  Read 1:    11,887,894 bp\n",
      "  Read 2:    11,887,894 bp\n",
      "Total written (filtered):     22,206,230 bp (93.4%)\n",
      "  Read 1:    11,082,401 bp\n",
      "  Read 2:    11,123,829 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t192\t0.0\t1\t0 184 8\n",
      "20\t39961\t0.0\t2\t39358 599 4\n",
      "21\t89\t0.0\t2\t0 84 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40221 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t39937\t0.0\t1\t39320 617\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s518_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s518_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s518_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s518_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s518_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s518_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s518_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s518_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,636 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.78 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,636\n",
      "  Read 1 with adapter:                  39,626 (100.0%)\n",
      "  Read 2 with adapter:                  39,541 (99.8%)\n",
      "Pairs written (passing filters):        39,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,384,904 bp\n",
      "  Read 1:    11,692,612 bp\n",
      "  Read 2:    11,692,292 bp\n",
      "Total written (filtered):     21,841,328 bp (93.4%)\n",
      "  Read 1:    10,900,176 bp\n",
      "  Read 2:    10,941,152 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t203\t0.0\t1\t0 201 2\n",
      "20\t39291\t0.0\t2\t38715 572 4\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39541 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t0 200\n",
      "19\t39280\t0.0\t1\t38538 742\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s519_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s519_152_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s519_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s519_152_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s519_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s519_152_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s519_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s519_152_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,023 reads  @     66.8 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.21 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,023\n",
      "  Read 1 with adapter:                  33,014 (100.0%)\n",
      "  Read 2 with adapter:                  32,929 (99.7%)\n",
      "Pairs written (passing filters):        33,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,483,206 bp\n",
      "  Read 1:     9,741,750 bp\n",
      "  Read 2:     9,741,456 bp\n",
      "Total written (filtered):     18,197,474 bp (93.4%)\n",
      "  Read 1:     9,081,577 bp\n",
      "  Read 2:     9,115,897 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33014 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t175\t0.0\t1\t0 172 3\n",
      "20\t32750\t0.0\t2\t32239 508 3\n",
      "21\t58\t0.0\t2\t0 55 3\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t32701\t0.0\t1\t32156 545\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s520_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s520_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s520_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s520_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s520_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s520_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s520_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s520_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,583 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.43 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,583\n",
      "  Read 1 with adapter:                  32,571 (100.0%)\n",
      "  Read 2 with adapter:                  32,309 (99.2%)\n",
      "Pairs written (passing filters):        32,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,223,806 bp\n",
      "  Read 1:     9,611,981 bp\n",
      "  Read 2:     9,611,825 bp\n",
      "Total written (filtered):     17,958,721 bp (93.4%)\n",
      "  Read 1:     8,960,647 bp\n",
      "  Read 2:     8,998,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t164\t0.0\t1\t0 161 3\n",
      "20\t32323\t0.0\t2\t31810 509 4\n",
      "21\t54\t0.0\t2\t0 52 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t190\t0.0\t1\t0 190\n",
      "19\t32049\t0.0\t1\t31178 871\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s522_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s522_154_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s522_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s522_154_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s522_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s522_154_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s522_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s522_154_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        42,642 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.04 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,642\n",
      "  Read 1 with adapter:                  42,626 (100.0%)\n",
      "  Read 2 with adapter:                  42,510 (99.7%)\n",
      "Pairs written (passing filters):        42,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,154,142 bp\n",
      "  Read 1:    12,579,368 bp\n",
      "  Read 2:    12,574,774 bp\n",
      "Total written (filtered):     23,494,199 bp (93.4%)\n",
      "  Read 1:    11,727,006 bp\n",
      "  Read 2:    11,767,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t234\t0.0\t1\t0 227 7\n",
      "20\t42277\t0.0\t2\t41596 671 10\n",
      "21\t62\t0.0\t2\t0 60 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t42229\t0.0\t1\t41468 761\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s523_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s523_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s523_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s523_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s523_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s523_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s523_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s523_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,391 reads  @     80.8 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.59 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,391\n",
      "  Read 1 with adapter:                  44,368 (99.9%)\n",
      "  Read 2 with adapter:                  44,236 (99.7%)\n",
      "Pairs written (passing filters):        44,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,184,543 bp\n",
      "  Read 1:    13,095,233 bp\n",
      "  Read 2:    13,089,310 bp\n",
      "Total written (filtered):     24,456,916 bp (93.4%)\n",
      "  Read 1:    12,208,005 bp\n",
      "  Read 2:    12,248,911 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t249\t0.0\t1\t0 246 3\n",
      "20\t43990\t0.0\t2\t43301 682 7\n",
      "21\t93\t0.0\t2\t0 90 3\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t43959\t0.0\t1\t43200 759\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s524_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s524_156_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s524_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s524_156_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s524_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s524_156_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s524_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s524_156_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,307 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.44 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,307\n",
      "  Read 1 with adapter:                  44,290 (100.0%)\n",
      "  Read 2 with adapter:                  43,485 (98.1%)\n",
      "Pairs written (passing filters):        44,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,096,540 bp\n",
      "  Read 1:    13,070,555 bp\n",
      "  Read 2:    13,025,985 bp\n",
      "Total written (filtered):     24,384,777 bp (93.4%)\n",
      "  Read 1:    12,184,872 bp\n",
      "  Read 2:    12,199,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t242\t0.0\t1\t0 241 1\n",
      "20\t43917\t0.0\t2\t43161 748 8\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t43210\t0.0\t1\t42461 749\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s525_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s525_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s525_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s525_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s525_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s525_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s525_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s525_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,821 reads  @     88.3 µs/read;   0.68 M reads/minute\n",
      "Finished in 2.81 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,821\n",
      "  Read 1 with adapter:                  31,814 (100.0%)\n",
      "  Read 2 with adapter:                  31,763 (99.8%)\n",
      "Pairs written (passing filters):        31,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,772,154 bp\n",
      "  Read 1:     9,387,190 bp\n",
      "  Read 2:     9,384,964 bp\n",
      "Total written (filtered):     17,532,575 bp (93.4%)\n",
      "  Read 1:     8,751,021 bp\n",
      "  Read 2:     8,781,554 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31814 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t31569\t0.0\t2\t30833 722 14\n",
      "21\t51\t0.0\t2\t0 50 1\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t151\t0.0\t1\t0 151\n",
      "19\t31548\t0.0\t1\t31173 375\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s526_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s526_158_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s526_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s526_158_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s526_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s526_158_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s526_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s526_158_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,776 reads  @    109.8 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.92 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,776\n",
      "  Read 1 with adapter:                  44,766 (100.0%)\n",
      "  Read 2 with adapter:                  44,681 (99.8%)\n",
      "Pairs written (passing filters):        44,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,417,757 bp\n",
      "  Read 1:    13,208,901 bp\n",
      "  Read 2:    13,208,856 bp\n",
      "Total written (filtered):     24,673,765 bp (93.4%)\n",
      "  Read 1:    12,313,774 bp\n",
      "  Read 2:    12,359,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t292\t0.0\t1\t0 289 3\n",
      "20\t44361\t0.0\t2\t43698 657 6\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t176\t0.0\t1\t0 176\n",
      "19\t44403\t0.0\t1\t43724 679\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s527_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s527_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s527_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s527_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s527_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s527_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s527_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s527_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        28,192 reads  @    131.8 µs/read;   0.46 M reads/minute\n",
      "Finished in 3.72 s (132 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,192\n",
      "  Read 1 with adapter:                  28,186 (100.0%)\n",
      "  Read 2 with adapter:                  28,119 (99.7%)\n",
      "Pairs written (passing filters):        28,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,631,560 bp\n",
      "  Read 1:     8,316,637 bp\n",
      "  Read 2:     8,314,923 bp\n",
      "Total written (filtered):     15,533,745 bp (93.4%)\n",
      "  Read 1:     7,753,021 bp\n",
      "  Read 2:     7,780,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t183\t0.0\t1\t0 181 2\n",
      "20\t27921\t0.0\t2\t27479 436 6\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28119 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t124\t0.0\t1\t0 124\n",
      "19\t27933\t0.0\t1\t27523 410\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s528_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s528_160_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s528_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s528_160_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s528_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s528_160_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s528_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s528_160_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        31,058 reads  @    150.6 µs/read;   0.40 M reads/minute\n",
      "Finished in 4.68 s (151 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,058\n",
      "  Read 1 with adapter:                  31,047 (100.0%)\n",
      "  Read 2 with adapter:                  30,898 (99.5%)\n",
      "Pairs written (passing filters):        31,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,322,534 bp\n",
      "  Read 1:     9,162,084 bp\n",
      "  Read 2:     9,160,450 bp\n",
      "Total written (filtered):     17,114,823 bp (93.4%)\n",
      "  Read 1:     8,541,307 bp\n",
      "  Read 2:     8,573,516 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31047 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t215\t0.0\t1\t0 214 1\n",
      "20\t30762\t0.0\t2\t30285 472 5\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t30622\t0.0\t1\t29850 772\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s530_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s530_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s530_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s530_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s530_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s530_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s530_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s530_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        44,271 reads  @    137.5 µs/read;   0.44 M reads/minute\n",
      "Finished in 6.09 s (138 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,271\n",
      "  Read 1 with adapter:                  44,252 (100.0%)\n",
      "  Read 2 with adapter:                  44,135 (99.7%)\n",
      "Pairs written (passing filters):        44,271 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,115,281 bp\n",
      "  Read 1:    13,059,932 bp\n",
      "  Read 2:    13,055,349 bp\n",
      "Total written (filtered):     24,391,922 bp (93.4%)\n",
      "  Read 1:    12,175,024 bp\n",
      "  Read 2:    12,216,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t267\t0.0\t1\t0 264 3\n",
      "20\t43851\t0.0\t2\t43212 630 9\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t43867\t0.0\t1\t43167 700\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s531_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s531_162_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s531_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s531_162_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s531_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s531_162_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s531_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s531_162_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        28,288 reads  @    177.8 µs/read;   0.34 M reads/minute\n",
      "Finished in 5.04 s (178 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,288\n",
      "  Read 1 with adapter:                  28,276 (100.0%)\n",
      "  Read 2 with adapter:                  27,703 (97.9%)\n",
      "Pairs written (passing filters):        28,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,670,830 bp\n",
      "  Read 1:     8,344,462 bp\n",
      "  Read 2:     8,326,368 bp\n",
      "Total written (filtered):     15,579,145 bp (93.5%)\n",
      "  Read 1:     7,779,064 bp\n",
      "  Read 2:     7,800,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28276 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t186\t0.0\t1\t0 184 2\n",
      "20\t28018\t0.0\t2\t27445 561 12\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t124\t0.0\t1\t0 124\n",
      "19\t27525\t0.0\t1\t25791 1734\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s532_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s532_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s532_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s532_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s532_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s532_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s532_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s532_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        46,330 reads  @    140.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 6.53 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,330\n",
      "  Read 1 with adapter:                  46,314 (100.0%)\n",
      "  Read 2 with adapter:                  45,702 (98.6%)\n",
      "Pairs written (passing filters):        46,330 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,299,453 bp\n",
      "  Read 1:    13,667,344 bp\n",
      "  Read 2:    13,632,109 bp\n",
      "Total written (filtered):     25,505,210 bp (93.4%)\n",
      "  Read 1:    12,741,269 bp\n",
      "  Read 2:    12,763,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t294\t0.0\t1\t0 292 2\n",
      "20\t45904\t0.0\t2\t45103 793 8\n",
      "21\t71\t0.0\t2\t0 70 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t45402\t0.0\t1\t44716 686\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s533_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s533_164_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s533_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s533_164_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s533_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s533_164_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s533_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s533_164_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,821 reads  @    147.1 µs/read;   0.41 M reads/minute\n",
      "Finished in 5.42 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,821\n",
      "  Read 1 with adapter:                  36,807 (100.0%)\n",
      "  Read 2 with adapter:                  36,756 (99.8%)\n",
      "Pairs written (passing filters):        36,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,723,796 bp\n",
      "  Read 1:    10,862,175 bp\n",
      "  Read 2:    10,861,621 bp\n",
      "Total written (filtered):     20,289,574 bp (93.4%)\n",
      "  Read 1:    10,126,183 bp\n",
      "  Read 2:    10,163,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t235\t0.0\t1\t0 231 4\n",
      "20\t36485\t0.0\t2\t35839 635 11\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t0 200\n",
      "19\t36490\t0.0\t1\t36011 479\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s534_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s534_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s534_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s534_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s534_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s534_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s534_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s534_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,627 reads  @    137.1 µs/read;   0.44 M reads/minute\n",
      "Finished in 5.71 s (137 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,627\n",
      "  Read 1 with adapter:                  41,620 (100.0%)\n",
      "  Read 2 with adapter:                  41,526 (99.8%)\n",
      "Pairs written (passing filters):        41,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,558,967 bp\n",
      "  Read 1:    12,279,582 bp\n",
      "  Read 2:    12,279,385 bp\n",
      "Total written (filtered):     22,937,845 bp (93.4%)\n",
      "  Read 1:    11,447,335 bp\n",
      "  Read 2:    11,490,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41620 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t252\t0.0\t1\t0 249 3\n",
      "20\t41264\t0.0\t2\t40603 650 11\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t41259\t0.0\t1\t40550 709\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s535_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s535_166_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s535_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s535_166_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s535_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s535_166_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s535_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s535_166_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        32,208 reads  @    138.2 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.46 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,208\n",
      "  Read 1 with adapter:                  32,193 (100.0%)\n",
      "  Read 2 with adapter:                  32,126 (99.7%)\n",
      "Pairs written (passing filters):        32,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,998,709 bp\n",
      "  Read 1:     9,500,561 bp\n",
      "  Read 2:     9,498,148 bp\n",
      "Total written (filtered):     17,744,643 bp (93.4%)\n",
      "  Read 1:     8,856,797 bp\n",
      "  Read 2:     8,887,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t170\t0.0\t1\t0 169 1\n",
      "20\t31944\t0.0\t2\t31437 505 2\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t150\t0.0\t1\t0 150\n",
      "19\t31918\t0.0\t1\t31414 504\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s536_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s536_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s536_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s536_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s536_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s536_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s536_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s536_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        31,955 reads  @    141.1 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.52 s (141 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,955\n",
      "  Read 1 with adapter:                  31,942 (100.0%)\n",
      "  Read 2 with adapter:                  31,762 (99.4%)\n",
      "Pairs written (passing filters):        31,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,852,754 bp\n",
      "  Read 1:     9,426,293 bp\n",
      "  Read 2:     9,426,461 bp\n",
      "Total written (filtered):     17,610,689 bp (93.4%)\n",
      "  Read 1:     8,787,587 bp\n",
      "  Read 2:     8,823,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t219\t0.0\t1\t0 216 3\n",
      "20\t31633\t0.0\t2\t31069 557 7\n",
      "21\t63\t0.0\t2\t0 61 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t31459\t0.0\t1\t30647 812\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s538_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s538_168_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s538_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s538_168_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s538_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s538_168_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s538_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s538_168_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,133 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.48 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,133\n",
      "  Read 1 with adapter:                  50,106 (99.9%)\n",
      "  Read 2 with adapter:                  50,016 (99.8%)\n",
      "Pairs written (passing filters):        50,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,572,499 bp\n",
      "  Read 1:    14,787,342 bp\n",
      "  Read 2:    14,785,157 bp\n",
      "Total written (filtered):     27,620,381 bp (93.4%)\n",
      "  Read 1:    13,785,414 bp\n",
      "  Read 2:    13,834,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t308\t0.0\t1\t0 301 7\n",
      "20\t49665\t0.0\t2\t48777 882 6\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t49702\t0.0\t1\t48916 786\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s539_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s539_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s539_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s539_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s539_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s539_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s539_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s539_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        12,448 reads  @    108.5 µs/read;   0.55 M reads/minute\n",
      "Finished in 1.35 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,448\n",
      "  Read 1 with adapter:                  12,438 (99.9%)\n",
      "  Read 2 with adapter:                  12,381 (99.5%)\n",
      "Pairs written (passing filters):        12,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,339,259 bp\n",
      "  Read 1:     3,672,153 bp\n",
      "  Read 2:     3,667,106 bp\n",
      "Total written (filtered):      6,855,334 bp (93.4%)\n",
      "  Read 1:     3,423,418 bp\n",
      "  Read 2:     3,431,916 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 12438 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t80\t0.0\t1\t0 77 3\n",
      "20\t12310\t0.0\t2\t12038 270 2\n",
      "21\t29\t0.0\t2\t0 28 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12381 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t79\t0.0\t1\t0 79\n",
      "19\t12272\t0.0\t1\t11981 291\n",
      "20\t30\t0.0\t1\t0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s540_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s540_170_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s540_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s540_170_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s540_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s540_170_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s540_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s540_170_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,138 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.41 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,138\n",
      "  Read 1 with adapter:                  49,122 (100.0%)\n",
      "  Read 2 with adapter:                  48,516 (98.7%)\n",
      "Pairs written (passing filters):        49,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,957,223 bp\n",
      "  Read 1:    14,495,684 bp\n",
      "  Read 2:    14,461,539 bp\n",
      "Total written (filtered):     27,053,247 bp (93.4%)\n",
      "  Read 1:    13,513,394 bp\n",
      "  Read 2:    13,539,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t252\t0.0\t1\t0 239 13\n",
      "20\t48760\t0.0\t2\t47862 882 16\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t48212\t0.0\t1\t47410 802\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s541_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s541_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s541_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s541_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s541_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s541_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s541_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s541_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,570 reads  @     77.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.37 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,570\n",
      "  Read 1 with adapter:                  30,565 (100.0%)\n",
      "  Read 2 with adapter:                  30,537 (99.9%)\n",
      "Pairs written (passing filters):        30,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,035,991 bp\n",
      "  Read 1:     9,018,139 bp\n",
      "  Read 2:     9,017,852 bp\n",
      "Total written (filtered):     16,844,622 bp (93.4%)\n",
      "  Read 1:     8,406,884 bp\n",
      "  Read 2:     8,437,738 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t146\t0.0\t1\t0 139 7\n",
      "20\t30312\t0.0\t2\t29748 559 5\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30537 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t146\t0.0\t1\t0 146\n",
      "19\t30334\t0.0\t1\t29958 376\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s542_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s542_172_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s542_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s542_172_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s542_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s542_172_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s542_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s542_172_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        35,995 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.44 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,995\n",
      "  Read 1 with adapter:                  35,985 (100.0%)\n",
      "  Read 2 with adapter:                  35,881 (99.7%)\n",
      "Pairs written (passing filters):        35,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,236,774 bp\n",
      "  Read 1:    10,618,514 bp\n",
      "  Read 2:    10,618,260 bp\n",
      "Total written (filtered):     19,835,516 bp (93.4%)\n",
      "  Read 1:     9,898,893 bp\n",
      "  Read 2:     9,936,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t203\t0.0\t1\t0 201 2\n",
      "20\t35668\t0.0\t2\t35133 529 6\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t35647\t0.0\t1\t34973 674\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s543_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s543_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s543_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s543_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s543_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s543_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s543_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s543_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,701 reads  @     65.3 µs/read;   0.92 M reads/minute\n",
      "Finished in 1.81 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,701\n",
      "  Read 1 with adapter:                  27,693 (100.0%)\n",
      "  Read 2 with adapter:                  27,633 (99.8%)\n",
      "Pairs written (passing filters):        27,701 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,343,228 bp\n",
      "  Read 1:     8,171,785 bp\n",
      "  Read 2:     8,171,443 bp\n",
      "Total written (filtered):     15,264,533 bp (93.4%)\n",
      "  Read 1:     7,618,035 bp\n",
      "  Read 2:     7,646,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t160\t0.0\t1\t0 157 3\n",
      "20\t27453\t0.0\t2\t27068 381 4\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27633 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t130\t0.0\t1\t0 130\n",
      "19\t27455\t0.0\t1\t26976 479\n",
      "20\t48\t0.0\t1\t0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s544_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s544_174_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s544_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s544_174_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s544_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s544_174_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s544_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s544_174_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,240 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.15 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,240\n",
      "  Read 1 with adapter:                  31,228 (100.0%)\n",
      "  Read 2 with adapter:                  31,023 (99.3%)\n",
      "Pairs written (passing filters):        31,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,431,210 bp\n",
      "  Read 1:     9,215,701 bp\n",
      "  Read 2:     9,215,509 bp\n",
      "Total written (filtered):     17,217,438 bp (93.4%)\n",
      "  Read 1:     8,591,239 bp\n",
      "  Read 2:     8,626,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t186\t0.0\t1\t0 182 4\n",
      "20\t30949\t0.0\t2\t30443 496 10\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t30768\t0.0\t1\t29857 911\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s546_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s546_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s546_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s546_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s546_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s546_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s546_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s546_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,713 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.38 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,713\n",
      "  Read 1 with adapter:                  36,702 (100.0%)\n",
      "  Read 2 with adapter:                  36,608 (99.7%)\n",
      "Pairs written (passing filters):        36,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,656,405 bp\n",
      "  Read 1:    10,830,244 bp\n",
      "  Read 2:    10,826,161 bp\n",
      "Total written (filtered):     20,226,957 bp (93.4%)\n",
      "  Read 1:    10,096,270 bp\n",
      "  Read 2:    10,130,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t164\t0.0\t1\t0 162 2\n",
      "20\t36430\t0.0\t2\t35854 571 5\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t36362\t0.0\t1\t35688 674\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s547_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s547_176_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s547_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s547_176_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s547_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s547_176_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s547_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s547_176_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,494 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.58 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,494\n",
      "  Read 1 with adapter:                  37,478 (100.0%)\n",
      "  Read 2 with adapter:                  37,401 (99.8%)\n",
      "Pairs written (passing filters):        37,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,116,801 bp\n",
      "  Read 1:    11,060,697 bp\n",
      "  Read 2:    11,056,104 bp\n",
      "Total written (filtered):     20,656,802 bp (93.4%)\n",
      "  Read 1:    10,311,221 bp\n",
      "  Read 2:    10,345,581 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37478 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t197\t0.0\t1\t0 195 2\n",
      "20\t37160\t0.0\t2\t36592 564 4\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t169\t0.0\t1\t0 169\n",
      "19\t37159\t0.0\t1\t36539 620\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s548_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s548_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s548_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s548_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s548_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s548_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s548_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s548_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,488 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.40 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,488\n",
      "  Read 1 with adapter:                  35,473 (100.0%)\n",
      "  Read 2 with adapter:                  34,972 (98.5%)\n",
      "Pairs written (passing filters):        35,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,908,907 bp\n",
      "  Read 1:    10,468,948 bp\n",
      "  Read 2:    10,439,959 bp\n",
      "Total written (filtered):     19,535,185 bp (93.4%)\n",
      "  Read 1:     9,759,587 bp\n",
      "  Read 2:     9,775,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t204\t0.0\t1\t0 200 4\n",
      "20\t35155\t0.0\t2\t34608 539 8\n",
      "21\t79\t0.0\t2\t0 79\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t34737\t0.0\t1\t34183 554\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s549_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s549_178_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s549_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s549_178_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s549_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s549_178_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s549_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s549_178_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        19,695 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 1.50 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,695\n",
      "  Read 1 with adapter:                  19,693 (100.0%)\n",
      "  Read 2 with adapter:                  19,662 (99.8%)\n",
      "Pairs written (passing filters):        19,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,620,022 bp\n",
      "  Read 1:     5,810,018 bp\n",
      "  Read 2:     5,810,004 bp\n",
      "Total written (filtered):     10,852,733 bp (93.4%)\n",
      "  Read 1:     5,416,228 bp\n",
      "  Read 2:     5,436,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 19693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t110\t0.0\t1\t0 107 3\n",
      "20\t19525\t0.0\t2\t18937 578 10\n",
      "21\t40\t0.0\t2\t0 37 3\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 19662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t103\t0.0\t1\t0 103\n",
      "19\t19535\t0.0\t1\t19275 260\n",
      "20\t24\t0.0\t1\t0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s550_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s550_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s550_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s550_179_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s550_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s550_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s550_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s550_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,727 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.98 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,727\n",
      "  Read 1 with adapter:                  33,915 (79.4%)\n",
      "  Read 2 with adapter:                  42,473 (99.4%)\n",
      "Pairs written (passing filters):        42,727 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,121,084 bp\n",
      "  Read 1:    10,517,341 bp\n",
      "  Read 2:    12,603,743 bp\n",
      "Total written (filtered):     21,636,047 bp (93.6%)\n",
      "  Read 1:     9,839,178 bp\n",
      "  Read 2:    11,796,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t210\t0.0\t1\t0 207 3\n",
      "20\t33600\t0.0\t2\t32936 657 7\n",
      "21\t81\t0.0\t2\t0 79 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t42200\t0.0\t1\t41032 1168\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s551_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s551_180_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s551_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s551_180_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s551_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s551_180_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s551_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s551_180_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,328 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.74 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,328\n",
      "  Read 1 with adapter:                  37,311 (100.0%)\n",
      "  Read 2 with adapter:                  37,278 (99.9%)\n",
      "Pairs written (passing filters):        37,328 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,023,209 bp\n",
      "  Read 1:    11,011,747 bp\n",
      "  Read 2:    11,011,462 bp\n",
      "Total written (filtered):     20,568,993 bp (93.4%)\n",
      "  Read 1:    10,265,697 bp\n",
      "  Read 2:    10,303,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37311 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t256\t0.0\t1\t0 252 4\n",
      "20\t36950\t0.0\t2\t36244 700 6\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t37024\t0.0\t1\t36556 468\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s552_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s552_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s552_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s552_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s552_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s552_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s552_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s552_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,354 reads  @     67.1 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.91 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,354\n",
      "  Read 1 with adapter:                  43,329 (99.9%)\n",
      "  Read 2 with adapter:                  43,238 (99.7%)\n",
      "Pairs written (passing filters):        43,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,578,425 bp\n",
      "  Read 1:    12,789,169 bp\n",
      "  Read 2:    12,789,256 bp\n",
      "Total written (filtered):     23,890,627 bp (93.4%)\n",
      "  Read 1:    11,922,752 bp\n",
      "  Read 2:    11,967,875 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43329 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t267\t0.0\t1\t0 263 4\n",
      "20\t42942\t0.0\t2\t42213 722 7\n",
      "21\t80\t0.0\t2\t0 77 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t42947\t0.0\t1\t42187 760\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s553_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s553_182_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s553_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s553_182_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s553_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s553_182_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s553_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s553_182_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,703 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.20 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,703\n",
      "  Read 1 with adapter:                  34,689 (100.0%)\n",
      "  Read 2 with adapter:                  34,617 (99.8%)\n",
      "Pairs written (passing filters):        34,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,474,263 bp\n",
      "  Read 1:    10,237,373 bp\n",
      "  Read 2:    10,236,890 bp\n",
      "Total written (filtered):     19,122,966 bp (93.4%)\n",
      "  Read 1:     9,543,698 bp\n",
      "  Read 2:     9,579,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34689 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t210\t0.0\t1\t0 207 3\n",
      "20\t34379\t0.0\t2\t33806 565 8\n",
      "21\t59\t0.0\t2\t0 59\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34617 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t164\t0.0\t1\t0 164\n",
      "19\t34390\t0.0\t1\t33816 574\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s554_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s554_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s554_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s554_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s554_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s554_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s554_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s554_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,058 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.33 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,058\n",
      "  Read 1 with adapter:                  35,049 (100.0%)\n",
      "  Read 2 with adapter:                  34,873 (99.5%)\n",
      "Pairs written (passing filters):        35,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,683,280 bp\n",
      "  Read 1:    10,341,831 bp\n",
      "  Read 2:    10,341,449 bp\n",
      "Total written (filtered):     19,319,967 bp (93.4%)\n",
      "  Read 1:     9,640,971 bp\n",
      "  Read 2:     9,678,996 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t217\t0.0\t1\t0 214 3\n",
      "20\t34733\t0.0\t2\t34120 606 7\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34873 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t226\t0.0\t1\t0 226\n",
      "19\t34555\t0.0\t1\t33741 814\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s556_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s556_184_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s556_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s556_184_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s556_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s556_184_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s556_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s556_184_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,701 reads  @     63.7 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.91 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,701\n",
      "  Read 1 with adapter:                  45,673 (99.9%)\n",
      "  Read 2 with adapter:                  45,597 (99.8%)\n",
      "Pairs written (passing filters):        45,701 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,958,495 bp\n",
      "  Read 1:    13,481,733 bp\n",
      "  Read 2:    13,476,762 bp\n",
      "Total written (filtered):     25,178,977 bp (93.4%)\n",
      "  Read 1:    12,568,443 bp\n",
      "  Read 2:    12,610,534 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45673 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t294\t0.0\t1\t0 289 5\n",
      "20\t45256\t0.0\t2\t44554 691 11\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t45296\t0.0\t1\t44553 743\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s557_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s557_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s557_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s557_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s557_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s557_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s557_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s557_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        41,555 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.65 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,555\n",
      "  Read 1 with adapter:                  41,531 (99.9%)\n",
      "  Read 2 with adapter:                  41,449 (99.7%)\n",
      "Pairs written (passing filters):        41,555 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,513,148 bp\n",
      "  Read 1:    12,258,718 bp\n",
      "  Read 2:    12,254,430 bp\n",
      "Total written (filtered):     22,895,266 bp (93.4%)\n",
      "  Read 1:    11,428,273 bp\n",
      "  Read 2:    11,466,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t278\t0.0\t1\t0 275 3\n",
      "20\t41132\t0.0\t2\t40456 671 5\n",
      "21\t95\t0.0\t2\t0 95\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41449 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t41165\t0.0\t1\t40549 616\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s558_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s558_186_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s558_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s558_186_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s558_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s558_186_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s558_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s558_186_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        46,898 reads  @     63.2 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.96 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,898\n",
      "  Read 1 with adapter:                  46,872 (99.9%)\n",
      "  Read 2 with adapter:                  46,315 (98.8%)\n",
      "Pairs written (passing filters):        46,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,638,601 bp\n",
      "  Read 1:    13,834,900 bp\n",
      "  Read 2:    13,803,701 bp\n",
      "Total written (filtered):     25,821,493 bp (93.4%)\n",
      "  Read 1:    12,897,624 bp\n",
      "  Read 2:    12,923,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46872 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t290\t0.0\t1\t0 287 3\n",
      "20\t46448\t0.0\t2\t45618 817 13\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t46008\t0.0\t1\t45336 672\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s559_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s559_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s559_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s559_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s559_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s559_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s559_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s559_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,635 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.46 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,635\n",
      "  Read 1 with adapter:                  36,626 (100.0%)\n",
      "  Read 2 with adapter:                  36,582 (99.9%)\n",
      "Pairs written (passing filters):        36,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,614,373 bp\n",
      "  Read 1:    10,807,311 bp\n",
      "  Read 2:    10,807,062 bp\n",
      "Total written (filtered):     20,187,149 bp (93.4%)\n",
      "  Read 1:    10,075,050 bp\n",
      "  Read 2:    10,112,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t344\t0.0\t1\t0 339 5\n",
      "20\t36177\t0.0\t2\t35568 600 9\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36582 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t161\t0.0\t1\t0 161\n",
      "19\t36355\t0.0\t1\t35762 593\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s560_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s560_188_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s560_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s560_188_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s560_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s560_188_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s560_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s560_188_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,699 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.14 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,699\n",
      "  Read 1 with adapter:                  31,690 (100.0%)\n",
      "  Read 2 with adapter:                  31,623 (99.8%)\n",
      "Pairs written (passing filters):        31,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,702,188 bp\n",
      "  Read 1:     9,351,095 bp\n",
      "  Read 2:     9,351,093 bp\n",
      "Total written (filtered):     17,467,844 bp (93.4%)\n",
      "  Read 1:     8,717,525 bp\n",
      "  Read 2:     8,750,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t295\t0.0\t1\t0 289 6\n",
      "20\t31316\t0.0\t2\t30802 504 10\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31623 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t134\t0.0\t1\t0 134\n",
      "19\t31418\t0.0\t1\t30770 648\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s561_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s561_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s561_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s561_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s561_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s561_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s561_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s561_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        30,333 reads  @     64.1 µs/read;   0.94 M reads/minute\n",
      "Finished in 1.95 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,333\n",
      "  Read 1 with adapter:                  30,324 (100.0%)\n",
      "  Read 2 with adapter:                  30,221 (99.6%)\n",
      "Pairs written (passing filters):        30,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,896,028 bp\n",
      "  Read 1:     8,948,226 bp\n",
      "  Read 2:     8,947,802 bp\n",
      "Total written (filtered):     16,715,616 bp (93.4%)\n",
      "  Read 1:     8,341,942 bp\n",
      "  Read 2:     8,373,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t290\t0.0\t1\t0 287 3\n",
      "20\t29945\t0.0\t2\t29475 467 3\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30221 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t126\t0.0\t1\t0 126\n",
      "19\t30040\t0.0\t1\t29456 584\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s562_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s562_190_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s562_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s562_190_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s562_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s562_190_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s562_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s562_190_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,275 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.15 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,275\n",
      "  Read 1 with adapter:                  31,263 (100.0%)\n",
      "  Read 2 with adapter:                  31,069 (99.3%)\n",
      "Pairs written (passing filters):        31,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,452,081 bp\n",
      "  Read 1:     9,226,115 bp\n",
      "  Read 2:     9,225,966 bp\n",
      "Total written (filtered):     17,236,802 bp (93.4%)\n",
      "  Read 1:     8,601,029 bp\n",
      "  Read 2:     8,635,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t278\t0.0\t1\t0 275 3\n",
      "20\t30872\t0.0\t2\t30392 470 10\n",
      "21\t78\t0.0\t2\t0 77 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31069 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t30825\t0.0\t1\t29804 1021\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s564_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s564_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s564_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s564_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s564_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s564_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s564_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s564_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        41,220 reads  @     64.0 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.64 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,220\n",
      "  Read 1 with adapter:                  41,209 (100.0%)\n",
      "  Read 2 with adapter:                  41,079 (99.7%)\n",
      "Pairs written (passing filters):        41,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,316,163 bp\n",
      "  Read 1:    12,159,887 bp\n",
      "  Read 2:    12,156,276 bp\n",
      "Total written (filtered):     22,711,851 bp (93.4%)\n",
      "  Read 1:    11,335,970 bp\n",
      "  Read 2:    11,375,881 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t383\t0.0\t1\t0 379 4\n",
      "20\t40709\t0.0\t2\t40038 669 2\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t40813\t0.0\t1\t39980 833\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s565_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s565_192_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s565_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s565_192_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s565_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s565_192_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s565_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s565_192_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,514 reads  @     63.1 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.43 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,514\n",
      "  Read 1 with adapter:                  38,504 (100.0%)\n",
      "  Read 2 with adapter:                  38,397 (99.7%)\n",
      "Pairs written (passing filters):        38,514 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,719,596 bp\n",
      "  Read 1:    11,361,619 bp\n",
      "  Read 2:    11,357,977 bp\n",
      "Total written (filtered):     21,220,350 bp (93.4%)\n",
      "  Read 1:    10,591,784 bp\n",
      "  Read 2:    10,628,566 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t345\t0.0\t1\t0 341 4\n",
      "20\t38053\t0.0\t2\t37462 587 4\n",
      "21\t76\t0.0\t2\t0 72 4\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t38143\t0.0\t1\t37450 693\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s566_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s566_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s566_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s566_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s566_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s566_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s566_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s566_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,470 reads  @     65.0 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.96 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,470\n",
      "  Read 1 with adapter:                  45,458 (100.0%)\n",
      "  Read 2 with adapter:                  44,777 (98.5%)\n",
      "Pairs written (passing filters):        45,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,789,040 bp\n",
      "  Read 1:    13,413,642 bp\n",
      "  Read 2:    13,375,398 bp\n",
      "Total written (filtered):     25,029,624 bp (93.4%)\n",
      "  Read 1:    12,504,845 bp\n",
      "  Read 2:    12,524,779 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t446\t0.0\t1\t0 432 14\n",
      "20\t44892\t0.0\t2\t44147 733 12\n",
      "21\t77\t0.0\t2\t0 77\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t44497\t0.0\t1\t43743 754\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s567_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s567_194_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s567_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s567_194_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s567_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s567_194_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s567_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s567_194_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,304 reads  @     65.4 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.51 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,304\n",
      "  Read 1 with adapter:                  38,283 (99.9%)\n",
      "  Read 2 with adapter:                  38,232 (99.8%)\n",
      "Pairs written (passing filters):        38,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,597,973 bp\n",
      "  Read 1:    11,298,319 bp\n",
      "  Read 2:    11,299,654 bp\n",
      "Total written (filtered):     21,106,213 bp (93.4%)\n",
      "  Read 1:    10,532,840 bp\n",
      "  Read 2:    10,573,373 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t242\t0.0\t1\t0 235 7\n",
      "20\t37948\t0.0\t2\t37309 634 5\n",
      "21\t61\t0.0\t2\t0 61\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t37949\t0.0\t1\t37383 566\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s568_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s568_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s568_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s568_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s568_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s568_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s568_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s568_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,842 reads  @     63.0 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.51 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,842\n",
      "  Read 1 with adapter:                  39,825 (100.0%)\n",
      "  Read 2 with adapter:                  39,766 (99.8%)\n",
      "Pairs written (passing filters):        39,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,505,772 bp\n",
      "  Read 1:    11,752,646 bp\n",
      "  Read 2:    11,753,126 bp\n",
      "Total written (filtered):     21,953,988 bp (93.4%)\n",
      "  Read 1:    10,956,288 bp\n",
      "  Read 2:    10,997,700 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t252\t0.0\t1\t0 248 4\n",
      "20\t39458\t0.0\t2\t38833 620 5\n",
      "21\t84\t0.0\t2\t0 84\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t179\t0.0\t1\t0 179\n",
      "19\t39536\t0.0\t1\t38946 590\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s569_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s569_196_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s569_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s569_196_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s569_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s569_196_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s569_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s569_196_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,092 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.01 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,092\n",
      "  Read 1 with adapter:                  32,075 (99.9%)\n",
      "  Read 2 with adapter:                  32,041 (99.8%)\n",
      "Pairs written (passing filters):        32,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,933,789 bp\n",
      "  Read 1:     9,466,807 bp\n",
      "  Read 2:     9,466,982 bp\n",
      "Total written (filtered):     17,683,750 bp (93.4%)\n",
      "  Read 1:     8,825,452 bp\n",
      "  Read 2:     8,858,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t209\t0.0\t1\t0 206 3\n",
      "20\t31783\t0.0\t2\t31264 517 2\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32041 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t163\t0.0\t1\t0 163\n",
      "19\t31810\t0.0\t1\t31311 499\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s570_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s570_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s570_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s570_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s570_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s570_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s570_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s570_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        28,679 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.96 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,679\n",
      "  Read 1 with adapter:                  28,664 (99.9%)\n",
      "  Read 2 with adapter:                  28,529 (99.5%)\n",
      "Pairs written (passing filters):        28,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,919,348 bp\n",
      "  Read 1:     8,460,015 bp\n",
      "  Read 2:     8,459,333 bp\n",
      "Total written (filtered):     15,804,259 bp (93.4%)\n",
      "  Read 1:     7,886,844 bp\n",
      "  Read 2:     7,917,415 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t172\t0.0\t1\t0 170 2\n",
      "20\t28412\t0.0\t2\t27937 467 8\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28529 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t28266\t0.0\t1\t27637 629\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s572_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s572_198_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s572_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s572_198_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s572_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s572_198_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s572_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s572_198_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,865 reads  @     62.1 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.79 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,865\n",
      "  Read 1 with adapter:                  44,837 (99.9%)\n",
      "  Read 2 with adapter:                  44,767 (99.8%)\n",
      "Pairs written (passing filters):        44,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,466,156 bp\n",
      "  Read 1:    13,233,847 bp\n",
      "  Read 2:    13,232,309 bp\n",
      "Total written (filtered):     24,719,137 bp (93.4%)\n",
      "  Read 1:    12,337,289 bp\n",
      "  Read 2:    12,381,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44837 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t314\t0.0\t1\t0 304 10\n",
      "20\t44386\t0.0\t2\t43544 831 11\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t44463\t0.0\t1\t43765 698\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s573_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s573_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s573_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s573_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s573_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s573_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s573_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s573_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,389 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.92 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,389\n",
      "  Read 1 with adapter:                  41,371 (100.0%)\n",
      "  Read 2 with adapter:                  41,287 (99.8%)\n",
      "Pairs written (passing filters):        41,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,416,377 bp\n",
      "  Read 1:    12,209,396 bp\n",
      "  Read 2:    12,206,981 bp\n",
      "Total written (filtered):     22,804,838 bp (93.4%)\n",
      "  Read 1:    11,382,194 bp\n",
      "  Read 2:    11,422,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41371 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t278\t0.0\t1\t0 274 4\n",
      "20\t41006\t0.0\t2\t40306 693 7\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t40997\t0.0\t1\t40348 649\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s574_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s574_200_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s574_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s574_200_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s574_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s574_200_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s574_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s574_200_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,224 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.29 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,224\n",
      "  Read 1 with adapter:                  48,201 (100.0%)\n",
      "  Read 2 with adapter:                  47,622 (98.8%)\n",
      "Pairs written (passing filters):        48,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,418,381 bp\n",
      "  Read 1:    14,225,403 bp\n",
      "  Read 2:    14,192,978 bp\n",
      "Total written (filtered):     26,549,924 bp (93.4%)\n",
      "  Read 1:    13,261,591 bp\n",
      "  Read 2:    13,288,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t307\t0.0\t1\t0 298 9\n",
      "20\t47746\t0.0\t2\t46903 831 12\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t257\t0.0\t1\t0 257\n",
      "19\t47281\t0.0\t1\t46578 703\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s575_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s575_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s575_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s575_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s575_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s575_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s575_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s575_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,393 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.51 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,393\n",
      "  Read 1 with adapter:                  33,377 (100.0%)\n",
      "  Read 2 with adapter:                  33,318 (99.8%)\n",
      "Pairs written (passing filters):        33,393 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,698,594 bp\n",
      "  Read 1:     9,849,845 bp\n",
      "  Read 2:     9,848,749 bp\n",
      "Total written (filtered):     18,398,217 bp (93.4%)\n",
      "  Read 1:     9,182,402 bp\n",
      "  Read 2:     9,215,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33377 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t184\t0.0\t1\t0 179 5\n",
      "20\t33106\t0.0\t2\t32585 521\n",
      "21\t59\t0.0\t2\t0 58 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t33080\t0.0\t1\t32620 460\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s576_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s576_202_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s576_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s576_202_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s576_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s576_202_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s576_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s576_202_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,840 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.15 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,840\n",
      "  Read 1 with adapter:                  31,828 (100.0%)\n",
      "  Read 2 with adapter:                  31,781 (99.8%)\n",
      "Pairs written (passing filters):        31,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,785,253 bp\n",
      "  Read 1:     9,392,549 bp\n",
      "  Read 2:     9,392,704 bp\n",
      "Total written (filtered):     17,545,030 bp (93.4%)\n",
      "  Read 1:     8,756,066 bp\n",
      "  Read 2:     8,788,964 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31828 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t182\t0.0\t1\t0 179 3\n",
      "20\t31548\t0.0\t2\t31041 504 3\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31781 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t144\t0.0\t1\t0 144\n",
      "19\t31592\t0.0\t1\t31101 491\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s577_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s577_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s577_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s577_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s577_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s577_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s577_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s577_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        26,042 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 1.87 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,042\n",
      "  Read 1 with adapter:                  26,034 (100.0%)\n",
      "  Read 2 with adapter:                  25,958 (99.7%)\n",
      "Pairs written (passing filters):        26,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,361,036 bp\n",
      "  Read 1:     7,682,385 bp\n",
      "  Read 2:     7,678,651 bp\n",
      "Total written (filtered):     14,347,300 bp (93.4%)\n",
      "  Read 1:     7,161,782 bp\n",
      "  Read 2:     7,185,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t157\t0.0\t1\t0 154 3\n",
      "20\t25798\t0.0\t2\t25382 408 8\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25958 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t118\t0.0\t1\t0 118\n",
      "19\t25791\t0.0\t1\t25379 412\n",
      "20\t49\t0.0\t1\t0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s578_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s578_204_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s578_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s578_204_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s578_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s578_204_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s578_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s578_204_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,448 reads  @     78.8 µs/read;   0.76 M reads/minute\n",
      "Finished in 2.40 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,448\n",
      "  Read 1 with adapter:                  30,436 (100.0%)\n",
      "  Read 2 with adapter:                  30,255 (99.4%)\n",
      "Pairs written (passing filters):        30,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,963,598 bp\n",
      "  Read 1:     8,981,962 bp\n",
      "  Read 2:     8,981,636 bp\n",
      "Total written (filtered):     16,780,232 bp (93.4%)\n",
      "  Read 1:     8,373,334 bp\n",
      "  Read 2:     8,406,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t178\t0.0\t1\t0 175 3\n",
      "20\t30161\t0.0\t2\t29664 492 5\n",
      "21\t48\t0.0\t2\t0 48\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t30020\t0.0\t1\t29198 822\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s580_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s580_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s580_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s580_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s580_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s580_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s580_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s580_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,113 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.90 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,113\n",
      "  Read 1 with adapter:                  39,102 (100.0%)\n",
      "  Read 2 with adapter:                  39,000 (99.7%)\n",
      "Pairs written (passing filters):        39,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,070,856 bp\n",
      "  Read 1:    11,538,205 bp\n",
      "  Read 2:    11,532,651 bp\n",
      "Total written (filtered):     21,548,042 bp (93.4%)\n",
      "  Read 1:    10,756,285 bp\n",
      "  Read 2:    10,791,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t231\t0.0\t1\t0 224 7\n",
      "20\t38753\t0.0\t2\t38109 637 7\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t177\t0.0\t1\t0 177\n",
      "19\t38752\t0.0\t1\t38047 705\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s581_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s581_206_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s581_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s581_206_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s581_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s581_206_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s581_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s581_206_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,849 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.87 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,849\n",
      "  Read 1 with adapter:                  35,834 (100.0%)\n",
      "  Read 2 with adapter:                  35,732 (99.7%)\n",
      "Pairs written (passing filters):        35,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,145,033 bp\n",
      "  Read 1:    10,574,867 bp\n",
      "  Read 2:    10,570,166 bp\n",
      "Total written (filtered):     19,749,674 bp (93.4%)\n",
      "  Read 1:     9,858,315 bp\n",
      "  Read 2:     9,891,359 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t207\t0.0\t1\t0 204 3\n",
      "20\t35545\t0.0\t2\t35020 521 4\n",
      "21\t57\t0.0\t2\t0 55 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t35463\t0.0\t1\t34928 535\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s582_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s582_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s582_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s582_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s582_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s582_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s582_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s582_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,024 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.78 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,024\n",
      "  Read 1 with adapter:                  38,007 (100.0%)\n",
      "  Read 2 with adapter:                  37,509 (98.6%)\n",
      "Pairs written (passing filters):        38,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,406,016 bp\n",
      "  Read 1:    11,216,749 bp\n",
      "  Read 2:    11,189,267 bp\n",
      "Total written (filtered):     20,933,420 bp (93.4%)\n",
      "  Read 1:    10,456,736 bp\n",
      "  Read 2:    10,476,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t231\t0.0\t1\t0 225 6\n",
      "20\t37671\t0.0\t2\t37101 563 7\n",
      "21\t74\t0.0\t2\t0 74\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t163\t0.0\t1\t0 163\n",
      "19\t37271\t0.0\t1\t36739 532\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s583_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s583_208_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s583_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s583_208_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s583_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s583_208_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s583_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s583_208_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,428 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 2.88 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,428\n",
      "  Read 1 with adapter:                  37,408 (99.9%)\n",
      "  Read 2 with adapter:                  37,357 (99.8%)\n",
      "Pairs written (passing filters):        37,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,080,568 bp\n",
      "  Read 1:    11,039,615 bp\n",
      "  Read 2:    11,040,953 bp\n",
      "Total written (filtered):     20,622,863 bp (93.4%)\n",
      "  Read 1:    10,291,598 bp\n",
      "  Read 2:    10,331,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t249\t0.0\t1\t0 244 5\n",
      "20\t37052\t0.0\t2\t36465 577 10\n",
      "21\t72\t0.0\t2\t0 69 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37357 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t176\t0.0\t1\t0 176\n",
      "19\t37100\t0.0\t1\t36574 526\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s584_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s584_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s584_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s584_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s584_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s584_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s584_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s584_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        40,583 reads  @     72.5 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.95 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,583\n",
      "  Read 1 with adapter:                  40,557 (99.9%)\n",
      "  Read 2 with adapter:                  40,478 (99.7%)\n",
      "Pairs written (passing filters):        40,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,941,757 bp\n",
      "  Read 1:    11,970,333 bp\n",
      "  Read 2:    11,971,424 bp\n",
      "Total written (filtered):     22,361,758 bp (93.4%)\n",
      "  Read 1:    11,159,311 bp\n",
      "  Read 2:    11,202,447 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40557 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t229\t0.0\t1\t0 227 2\n",
      "20\t40209\t0.0\t2\t39550 651 8\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40478 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t40247\t0.0\t1\t39595 652\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s585_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s585_210_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s585_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s585_210_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s585_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s585_210_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s585_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s585_210_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,268 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.36 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,268\n",
      "  Read 1 with adapter:                  32,250 (99.9%)\n",
      "  Read 2 with adapter:                  32,115 (99.5%)\n",
      "Pairs written (passing filters):        32,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,032,154 bp\n",
      "  Read 1:     9,517,451 bp\n",
      "  Read 2:     9,514,703 bp\n",
      "Total written (filtered):     17,777,145 bp (93.4%)\n",
      "  Read 1:     8,872,539 bp\n",
      "  Read 2:     8,904,606 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32250 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t201\t0.0\t1\t0 201\n",
      "20\t31949\t0.0\t2\t31360 579 10\n",
      "21\t59\t0.0\t2\t0 58 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t31879\t0.0\t1\t31256 623\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s586_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s586_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s586_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s586_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s586_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s586_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s586_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s586_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,415 reads  @     74.7 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.35 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,415\n",
      "  Read 1 with adapter:                  31,399 (99.9%)\n",
      "  Read 2 with adapter:                  31,193 (99.3%)\n",
      "Pairs written (passing filters):        31,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,532,161 bp\n",
      "  Read 1:     9,265,487 bp\n",
      "  Read 2:     9,266,674 bp\n",
      "Total written (filtered):     17,311,727 bp (93.4%)\n",
      "  Read 1:     8,637,590 bp\n",
      "  Read 2:     8,674,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31399 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t31131\t0.0\t2\t30635 492 4\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t30915\t0.0\t1\t29934 981\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s588_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s588_212_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s588_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s588_212_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s588_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s588_212_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s588_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s588_212_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,475 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.42 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,475\n",
      "  Read 1 with adapter:                  44,450 (99.9%)\n",
      "  Read 2 with adapter:                  44,313 (99.6%)\n",
      "Pairs written (passing filters):        44,475 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,231,823 bp\n",
      "  Read 1:    13,118,022 bp\n",
      "  Read 2:    13,113,801 bp\n",
      "Total written (filtered):     24,501,165 bp (93.4%)\n",
      "  Read 1:    12,229,174 bp\n",
      "  Read 2:    12,271,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44450 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t281\t0.0\t1\t0 272 9\n",
      "20\t44046\t0.0\t2\t43270 769 7\n",
      "21\t77\t0.0\t2\t0 77\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t44010\t0.0\t1\t43176 834\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s589_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s589_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s589_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s589_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s589_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s589_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s589_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s589_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         8,721 reads  @    110.8 µs/read;   0.54 M reads/minute\n",
      "Finished in 0.97 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,721\n",
      "  Read 1 with adapter:                   8,715 (99.9%)\n",
      "  Read 2 with adapter:                   8,637 (99.0%)\n",
      "Pairs written (passing filters):         8,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,139,005 bp\n",
      "  Read 1:     2,571,935 bp\n",
      "  Read 2:     2,567,070 bp\n",
      "Total written (filtered):      4,800,651 bp (93.4%)\n",
      "  Read 1:     2,397,647 bp\n",
      "  Read 2:     2,403,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 8715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t50\t0.0\t1\t0 49 1\n",
      "20\t8622\t0.0\t2\t8416 203 3\n",
      "21\t28\t0.0\t2\t0 26 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 8637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t65\t0.0\t1\t0 65\n",
      "19\t8544\t0.0\t1\t8353 191\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s590_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s590_214_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s590_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s590_214_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s590_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s590_214_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s590_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s590_214_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,082 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.91 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,082\n",
      "  Read 1 with adapter:                  45,064 (100.0%)\n",
      "  Read 2 with adapter:                  44,458 (98.6%)\n",
      "Pairs written (passing filters):        45,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,564,663 bp\n",
      "  Read 1:    13,298,921 bp\n",
      "  Read 2:    13,265,742 bp\n",
      "Total written (filtered):     24,818,914 bp (93.4%)\n",
      "  Read 1:    12,397,770 bp\n",
      "  Read 2:    12,421,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t268\t0.0\t1\t0 260 8\n",
      "20\t44646\t0.0\t2\t43939 699 8\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44458 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t44176\t0.0\t1\t43472 704\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s591_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s591_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s591_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s591_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s591_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s591_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s591_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s591_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        30,785 reads  @     74.1 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.28 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,785\n",
      "  Read 1 with adapter:                  30,775 (100.0%)\n",
      "  Read 2 with adapter:                  30,733 (99.8%)\n",
      "Pairs written (passing filters):        30,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,162,243 bp\n",
      "  Read 1:     9,080,679 bp\n",
      "  Read 2:     9,081,564 bp\n",
      "Total written (filtered):     16,963,006 bp (93.4%)\n",
      "  Read 1:     8,465,269 bp\n",
      "  Read 2:     8,497,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30775 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t162\t0.0\t1\t0 159 3\n",
      "20\t30542\t0.0\t2\t29937 592 13\n",
      "21\t46\t0.0\t2\t0 44 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t144\t0.0\t1\t0 144\n",
      "19\t30545\t0.0\t1\t30177 368\n",
      "20\t44\t0.0\t1\t0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s592_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s592_216_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s592_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s592_216_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s592_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s592_216_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s592_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s592_216_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,568 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.89 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,568\n",
      "  Read 1 with adapter:                  43,548 (100.0%)\n",
      "  Read 2 with adapter:                  43,452 (99.7%)\n",
      "Pairs written (passing filters):        43,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,703,982 bp\n",
      "  Read 1:    12,851,687 bp\n",
      "  Read 2:    12,852,295 bp\n",
      "Total written (filtered):     24,007,741 bp (93.4%)\n",
      "  Read 1:    11,980,923 bp\n",
      "  Read 2:    12,026,818 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43548 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t277\t0.0\t1\t0 271 6\n",
      "20\t43161\t0.0\t2\t42501 657 3\n",
      "21\t67\t0.0\t2\t0 67\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t43181\t0.0\t1\t42352 829\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s593_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s593_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s593_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s593_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s593_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s593_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s593_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s593_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,656 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.19 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,656\n",
      "  Read 1 with adapter:                  31,639 (99.9%)\n",
      "  Read 2 with adapter:                  31,540 (99.6%)\n",
      "Pairs written (passing filters):        31,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,675,117 bp\n",
      "  Read 1:     9,338,082 bp\n",
      "  Read 2:     9,337,035 bp\n",
      "Total written (filtered):     17,443,261 bp (93.4%)\n",
      "  Read 1:     8,705,415 bp\n",
      "  Read 2:     8,737,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t177\t0.0\t1\t0 172 5\n",
      "20\t31385\t0.0\t2\t30879 503 3\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31540 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t139\t0.0\t1\t0 139\n",
      "19\t31333\t0.0\t1\t30710 623\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s594_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s594_218_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s594_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s594_218_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s594_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s594_218_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s594_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s594_218_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,777 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.43 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,777\n",
      "  Read 1 with adapter:                  34,758 (99.9%)\n",
      "  Read 2 with adapter:                  34,576 (99.4%)\n",
      "Pairs written (passing filters):        34,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,517,520 bp\n",
      "  Read 1:    10,258,843 bp\n",
      "  Read 2:    10,258,677 bp\n",
      "Total written (filtered):     19,165,637 bp (93.4%)\n",
      "  Read 1:     9,563,790 bp\n",
      "  Read 2:     9,601,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34758 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t217\t0.0\t1\t0 215 2\n",
      "20\t34431\t0.0\t2\t33831 588 12\n",
      "21\t70\t0.0\t2\t0 67 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t34282\t0.0\t1\t33315 967\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s596_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s596_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s596_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s596_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s596_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s596_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s596_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s596_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,541 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.34 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,541\n",
      "  Read 1 with adapter:                  49,522 (100.0%)\n",
      "  Read 2 with adapter:                  49,413 (99.7%)\n",
      "Pairs written (passing filters):        49,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,224,228 bp\n",
      "  Read 1:    14,613,753 bp\n",
      "  Read 2:    14,610,475 bp\n",
      "Total written (filtered):     27,295,226 bp (93.4%)\n",
      "  Read 1:    13,623,453 bp\n",
      "  Read 2:    13,671,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49522 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t290\t0.0\t1\t0 288 2\n",
      "20\t49084\t0.0\t2\t48277 798 9\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t49148\t0.0\t1\t48329 819\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s597_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s597_220_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s597_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s597_220_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s597_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s597_220_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s597_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s597_220_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,152 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.78 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,152\n",
      "  Read 1 with adapter:                  43,142 (100.0%)\n",
      "  Read 2 with adapter:                  43,032 (99.7%)\n",
      "Pairs written (passing filters):        43,152 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,453,990 bp\n",
      "  Read 1:    12,729,742 bp\n",
      "  Read 2:    12,724,248 bp\n",
      "Total written (filtered):     23,773,806 bp (93.4%)\n",
      "  Read 1:    11,867,049 bp\n",
      "  Read 2:    11,906,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t259\t0.0\t1\t0 257 2\n",
      "20\t42763\t0.0\t2\t42114 643 6\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43032 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t205\t0.0\t1\t0 205\n",
      "19\t42739\t0.0\t1\t42064 675\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s598_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s598_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s598_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s598_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s598_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s598_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s598_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s598_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        47,654 reads  @     65.3 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.11 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,654\n",
      "  Read 1 with adapter:                  47,635 (100.0%)\n",
      "  Read 2 with adapter:                  46,974 (98.6%)\n",
      "Pairs written (passing filters):        47,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,077,500 bp\n",
      "  Read 1:    14,057,569 bp\n",
      "  Read 2:    14,019,931 bp\n",
      "Total written (filtered):     26,232,597 bp (93.4%)\n",
      "  Read 1:    13,105,046 bp\n",
      "  Read 2:    13,127,551 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t299\t0.0\t1\t0 290 9\n",
      "20\t47197\t0.0\t2\t46394 795 8\n",
      "21\t92\t0.0\t2\t0 90 2\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t46680\t0.0\t1\t45911 769\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s599_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s599_222_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s599_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s599_222_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s599_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s599_222_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s599_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s599_222_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,852 reads  @     66.6 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.79 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,852\n",
      "  Read 1 with adapter:                  41,832 (100.0%)\n",
      "  Read 2 with adapter:                  41,773 (99.8%)\n",
      "Pairs written (passing filters):        41,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,691,606 bp\n",
      "  Read 1:    12,345,519 bp\n",
      "  Read 2:    12,346,087 bp\n",
      "Total written (filtered):     23,061,542 bp (93.4%)\n",
      "  Read 1:    11,508,987 bp\n",
      "  Read 2:    11,552,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t214\t0.0\t1\t0 212 2\n",
      "20\t41496\t0.0\t2\t40719 774 3\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t41460\t0.0\t1\t40721 739\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s601_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s601_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s601_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s601_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s601_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s601_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s601_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s601_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,050 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.04 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,050\n",
      "  Read 1 with adapter:                  45,040 (100.0%)\n",
      "  Read 2 with adapter:                  44,899 (99.7%)\n",
      "Pairs written (passing filters):        45,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,573,317 bp\n",
      "  Read 1:    13,289,470 bp\n",
      "  Read 2:    13,283,847 bp\n",
      "Total written (filtered):     24,819,696 bp (93.4%)\n",
      "  Read 1:    12,388,810 bp\n",
      "  Read 2:    12,430,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t240\t0.0\t1\t0 237 3\n",
      "20\t44709\t0.0\t2\t43976 727 6\n",
      "21\t62\t0.0\t2\t0 60 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t44555\t0.0\t1\t43673 882\n",
      "20\t112\t0.0\t1\t0 112\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s602_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s602_224_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s602_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s602_224_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s602_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s602_224_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s602_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s602_224_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,874 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.25 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,874\n",
      "  Read 1 with adapter:                  46,859 (100.0%)\n",
      "  Read 2 with adapter:                  46,661 (99.5%)\n",
      "Pairs written (passing filters):        46,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,648,178 bp\n",
      "  Read 1:    13,827,280 bp\n",
      "  Read 2:    13,820,898 bp\n",
      "Total written (filtered):     25,824,781 bp (93.4%)\n",
      "  Read 1:    12,890,300 bp\n",
      "  Read 2:    12,934,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t281\t0.0\t1\t0 271 10\n",
      "20\t46460\t0.0\t2\t45561 895 4\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t46337\t0.0\t1\t45330 1007\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s603_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s603_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s603_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s603_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s603_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s603_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s603_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s603_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,697 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.21 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,697\n",
      "  Read 1 with adapter:                  49,682 (100.0%)\n",
      "  Read 2 with adapter:                  49,507 (99.6%)\n",
      "Pairs written (passing filters):        49,697 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,321,113 bp\n",
      "  Read 1:    14,660,562 bp\n",
      "  Read 2:    14,660,551 bp\n",
      "Total written (filtered):     27,387,134 bp (93.4%)\n",
      "  Read 1:    13,667,082 bp\n",
      "  Read 2:    13,720,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49682 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t287\t0.0\t1\t0 284 3\n",
      "20\t49238\t0.0\t2\t48501 725 12\n",
      "21\t99\t0.0\t2\t0 98 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49507 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t49181\t0.0\t1\t47980 1201\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s604_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s604_226_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s604_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s604_226_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s604_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s604_226_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s604_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s604_226_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,767 reads  @     64.0 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.13 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,767\n",
      "  Read 1 with adapter:                  48,742 (99.9%)\n",
      "  Read 2 with adapter:                  48,659 (99.8%)\n",
      "Pairs written (passing filters):        48,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,771,335 bp\n",
      "  Read 1:    14,385,707 bp\n",
      "  Read 2:    14,385,628 bp\n",
      "Total written (filtered):     26,872,261 bp (93.4%)\n",
      "  Read 1:    13,411,020 bp\n",
      "  Read 2:    13,461,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48742 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t295\t0.0\t1\t0 292 3\n",
      "20\t48301\t0.0\t2\t47495 794 12\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t265\t0.0\t1\t0 265\n",
      "19\t48263\t0.0\t1\t47423 840\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s605_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s605_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s605_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s605_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s605_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s605_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s605_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s605_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,588 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.08 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,588\n",
      "  Read 1 with adapter:                  48,572 (100.0%)\n",
      "  Read 2 with adapter:                  48,443 (99.7%)\n",
      "Pairs written (passing filters):        48,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,657,312 bp\n",
      "  Read 1:    14,333,445 bp\n",
      "  Read 2:    14,323,867 bp\n",
      "Total written (filtered):     26,765,801 bp (93.4%)\n",
      "  Read 1:    13,362,184 bp\n",
      "  Read 2:    13,403,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t298\t0.0\t1\t0 290 8\n",
      "20\t48148\t0.0\t2\t47380 763 5\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t48090\t0.0\t1\t47146 944\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s606_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s606_228_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s606_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s606_228_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s606_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s606_228_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s606_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s606_228_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,945 reads  @     62.0 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.04 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,945\n",
      "  Read 1 with adapter:                  48,931 (100.0%)\n",
      "  Read 2 with adapter:                  48,791 (99.7%)\n",
      "Pairs written (passing filters):        48,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,869,199 bp\n",
      "  Read 1:    14,438,756 bp\n",
      "  Read 2:    14,430,443 bp\n",
      "Total written (filtered):     26,963,910 bp (93.4%)\n",
      "  Read 1:    13,460,340 bp\n",
      "  Read 2:    13,503,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48931 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t311\t0.0\t1\t0 310 1\n",
      "20\t48491\t0.0\t2\t47786 698 7\n",
      "21\t91\t0.0\t2\t0 90 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t250\t0.0\t1\t0 250\n",
      "19\t48447\t0.0\t1\t47594 853\n",
      "20\t94\t0.0\t1\t0 94\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s607_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s607_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s607_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s607_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s607_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s607_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s607_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s607_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,878 reads  @     62.4 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.12 s (62 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,878\n",
      "  Read 1 with adapter:                  49,858 (100.0%)\n",
      "  Read 2 with adapter:                  49,766 (99.8%)\n",
      "Pairs written (passing filters):        49,878 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,427,766 bp\n",
      "  Read 1:    14,713,824 bp\n",
      "  Read 2:    14,713,942 bp\n",
      "Total written (filtered):     27,485,364 bp (93.4%)\n",
      "  Read 1:    13,716,847 bp\n",
      "  Read 2:    13,768,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t293\t0.0\t1\t0 289 4\n",
      "20\t49429\t0.0\t2\t48691 730 8\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t49459\t0.0\t1\t48627 832\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s608_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s608_230_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s608_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s608_230_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s608_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s608_230_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s608_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s608_230_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,795 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.86 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,795\n",
      "  Read 1 with adapter:                  43,783 (100.0%)\n",
      "  Read 2 with adapter:                  43,691 (99.8%)\n",
      "Pairs written (passing filters):        43,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,838,808 bp\n",
      "  Read 1:    12,919,516 bp\n",
      "  Read 2:    12,919,292 bp\n",
      "Total written (filtered):     24,133,292 bp (93.4%)\n",
      "  Read 1:    12,044,046 bp\n",
      "  Read 2:    12,089,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t291\t0.0\t1\t0 285 6\n",
      "20\t43365\t0.0\t2\t42680 680 5\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43691 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t43344\t0.0\t1\t42558 786\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s609_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s609_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s609_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s609_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s609_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s609_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s609_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s609_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,069 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.29 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,069\n",
      "  Read 1 with adapter:                  36,055 (100.0%)\n",
      "  Read 2 with adapter:                  35,987 (99.8%)\n",
      "Pairs written (passing filters):        36,069 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,274,816 bp\n",
      "  Read 1:    10,640,344 bp\n",
      "  Read 2:    10,634,472 bp\n",
      "Total written (filtered):     19,870,175 bp (93.4%)\n",
      "  Read 1:     9,919,343 bp\n",
      "  Read 2:     9,950,832 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t183\t0.0\t1\t0 183\n",
      "20\t35795\t0.0\t2\t35218 571 6\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t35694\t0.0\t1\t35125 569\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s610_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s610_232_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s610_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s610_232_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s610_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s610_232_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s610_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s610_232_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,808 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.18 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,808\n",
      "  Read 1 with adapter:                  34,798 (100.0%)\n",
      "  Read 2 with adapter:                  34,715 (99.7%)\n",
      "Pairs written (passing filters):        34,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,532,948 bp\n",
      "  Read 1:    10,268,352 bp\n",
      "  Read 2:    10,264,596 bp\n",
      "Total written (filtered):     19,177,630 bp (93.4%)\n",
      "  Read 1:     9,572,496 bp\n",
      "  Read 2:     9,605,134 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34798 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t184\t0.0\t1\t0 182 2\n",
      "20\t34523\t0.0\t2\t33901 617 5\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t34452\t0.0\t1\t33836 616\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s611_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s611_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s611_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s611_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s611_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s611_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s611_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s611_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        39,852 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.62 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,852\n",
      "  Read 1 with adapter:                  39,838 (100.0%)\n",
      "  Read 2 with adapter:                  39,776 (99.8%)\n",
      "Pairs written (passing filters):        39,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,512,641 bp\n",
      "  Read 1:    11,756,318 bp\n",
      "  Read 2:    11,756,323 bp\n",
      "Total written (filtered):     21,960,412 bp (93.4%)\n",
      "  Read 1:    10,959,701 bp\n",
      "  Read 2:    11,000,711 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t234\t0.0\t1\t0 232 2\n",
      "20\t39499\t0.0\t2\t38780 713 6\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t39526\t0.0\t1\t38828 698\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s612_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s612_234_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s612_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s612_234_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s612_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s612_234_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s612_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s612_234_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,166 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.36 s (63 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,166\n",
      "  Read 1 with adapter:                  37,149 (100.0%)\n",
      "  Read 2 with adapter:                  37,096 (99.8%)\n",
      "Pairs written (passing filters):        37,166 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,927,741 bp\n",
      "  Read 1:    10,963,960 bp\n",
      "  Read 2:    10,963,781 bp\n",
      "Total written (filtered):     20,480,096 bp (93.4%)\n",
      "  Read 1:    10,221,044 bp\n",
      "  Read 2:    10,259,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t198\t0.0\t1\t0 191 7\n",
      "20\t36823\t0.0\t2\t36231 582 10\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t36823\t0.0\t1\t36299 524\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s613_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s613_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s613_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s613_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s613_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s613_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s613_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s613_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,828 reads  @     62.6 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.31 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,828\n",
      "  Read 1 with adapter:                  36,821 (100.0%)\n",
      "  Read 2 with adapter:                  36,710 (99.7%)\n",
      "Pairs written (passing filters):        36,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,722,399 bp\n",
      "  Read 1:    10,864,246 bp\n",
      "  Read 2:    10,858,153 bp\n",
      "Total written (filtered):     20,288,772 bp (93.4%)\n",
      "  Read 1:    10,127,961 bp\n",
      "  Read 2:    10,160,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t236\t0.0\t1\t0 232 4\n",
      "20\t36475\t0.0\t2\t35896 571 8\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t36420\t0.0\t1\t35770 650\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s614_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s614_236_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s614_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s614_236_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s614_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s614_236_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s614_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s614_236_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,946 reads  @     61.7 µs/read;   0.97 M reads/minute\n",
      "Finished in 2.47 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,946\n",
      "  Read 1 with adapter:                  39,930 (100.0%)\n",
      "  Read 2 with adapter:                  39,824 (99.7%)\n",
      "Pairs written (passing filters):        39,946 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,562,491 bp\n",
      "  Read 1:    11,784,062 bp\n",
      "  Read 2:    11,778,429 bp\n",
      "Total written (filtered):     22,007,510 bp (93.4%)\n",
      "  Read 1:    10,985,645 bp\n",
      "  Read 2:    11,021,865 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t270\t0.0\t1\t0 267 3\n",
      "20\t39556\t0.0\t2\t38967 587 2\n",
      "21\t69\t0.0\t2\t0 69\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39824 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t39534\t0.0\t1\t38901 633\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s615_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s615_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s615_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s615_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s615_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s615_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s615_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s615_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,613 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.60 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,613\n",
      "  Read 1 with adapter:                  40,596 (100.0%)\n",
      "  Read 2 with adapter:                  40,501 (99.7%)\n",
      "Pairs written (passing filters):        40,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,961,642 bp\n",
      "  Read 1:    11,980,826 bp\n",
      "  Read 2:    11,980,816 bp\n",
      "Total written (filtered):     22,380,457 bp (93.4%)\n",
      "  Read 1:    11,169,058 bp\n",
      "  Read 2:    11,211,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t40240\t0.0\t2\t39639 588 13\n",
      "21\t66\t0.0\t2\t0 64 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40501 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t187\t0.0\t1\t0 187\n",
      "19\t40229\t0.0\t1\t39544 685\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s616_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s616_238_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s616_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s616_238_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s616_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s616_238_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s616_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s616_238_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,665 reads  @     62.7 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.18 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,665\n",
      "  Read 1 with adapter:                  34,656 (100.0%)\n",
      "  Read 2 with adapter:                  34,572 (99.7%)\n",
      "Pairs written (passing filters):        34,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,452,321 bp\n",
      "  Read 1:    10,226,167 bp\n",
      "  Read 2:    10,226,154 bp\n",
      "Total written (filtered):     19,102,546 bp (93.4%)\n",
      "  Read 1:     9,533,174 bp\n",
      "  Read 2:     9,569,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t185\t0.0\t1\t0 183 2\n",
      "20\t34381\t0.0\t2\t33821 554 6\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t34280\t0.0\t1\t33680 600\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s617_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s617_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s617_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s617_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s617_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s617_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s617_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s617_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        43,773 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.80 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,773\n",
      "  Read 1 with adapter:                  43,760 (100.0%)\n",
      "  Read 2 with adapter:                  43,658 (99.7%)\n",
      "Pairs written (passing filters):        43,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,818,843 bp\n",
      "  Read 1:    12,912,802 bp\n",
      "  Read 2:    12,906,041 bp\n",
      "Total written (filtered):     24,114,367 bp (93.4%)\n",
      "  Read 1:    12,037,709 bp\n",
      "  Read 2:    12,076,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43760 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t208\t0.0\t1\t0 200 8\n",
      "20\t43431\t0.0\t2\t42701 725 5\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t43307\t0.0\t1\t42529 778\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s618_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s618_240_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s618_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s618_240_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s618_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s618_240_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s618_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s618_240_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,187 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.82 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,187\n",
      "  Read 1 with adapter:                  44,171 (100.0%)\n",
      "  Read 2 with adapter:                  44,030 (99.6%)\n",
      "Pairs written (passing filters):        44,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,064,335 bp\n",
      "  Read 1:    13,035,152 bp\n",
      "  Read 2:    13,029,183 bp\n",
      "Total written (filtered):     24,344,704 bp (93.4%)\n",
      "  Read 1:    12,151,938 bp\n",
      "  Read 2:    12,192,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t280\t0.0\t1\t0 274 6\n",
      "20\t43784\t0.0\t2\t43032 745 7\n",
      "21\t72\t0.0\t2\t0 68 4\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44030 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t223\t0.0\t1\t0 223\n",
      "19\t43737\t0.0\t1\t42927 810\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s619_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s619_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s619_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s619_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s619_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s619_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s619_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s619_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,679 reads  @     62.8 µs/read;   0.95 M reads/minute\n",
      "Finished in 3.25 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,679\n",
      "  Read 1 with adapter:                  51,662 (100.0%)\n",
      "  Read 2 with adapter:                  51,547 (99.7%)\n",
      "Pairs written (passing filters):        51,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,490,564 bp\n",
      "  Read 1:    15,245,283 bp\n",
      "  Read 2:    15,245,281 bp\n",
      "Total written (filtered):     28,478,236 bp (93.4%)\n",
      "  Read 1:    14,212,219 bp\n",
      "  Read 2:    14,266,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t300\t0.0\t1\t0 297 3\n",
      "20\t51198\t0.0\t2\t50359 831 8\n",
      "21\t112\t0.0\t2\t0 110 2\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t51238\t0.0\t1\t50312 926\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s620_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s620_242_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s620_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s620_242_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s620_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s620_242_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s620_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s620_242_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,082 reads  @     62.7 µs/read;   0.96 M reads/minute\n",
      "Finished in 3.08 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,082\n",
      "  Read 1 with adapter:                  49,058 (100.0%)\n",
      "  Read 2 with adapter:                  48,943 (99.7%)\n",
      "Pairs written (passing filters):        49,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,957,872 bp\n",
      "  Read 1:    14,478,997 bp\n",
      "  Read 2:    14,478,875 bp\n",
      "Total written (filtered):     27,047,104 bp (93.4%)\n",
      "  Read 1:    13,498,003 bp\n",
      "  Read 2:    13,549,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t304\t0.0\t1\t0 295 9\n",
      "20\t48591\t0.0\t2\t47700 884 7\n",
      "21\t96\t0.0\t2\t0 96\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48943 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t284\t0.0\t1\t0 284\n",
      "19\t48518\t0.0\t1\t47687 831\n",
      "20\t141\t0.0\t1\t0 141\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s621_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s621_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s621_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s621_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s621_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s621_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s621_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s621_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        47,801 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 2.99 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,801\n",
      "  Read 1 with adapter:                  47,782 (100.0%)\n",
      "  Read 2 with adapter:                  47,627 (99.6%)\n",
      "Pairs written (passing filters):        47,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,193,511 bp\n",
      "  Read 1:    14,101,265 bp\n",
      "  Read 2:    14,092,246 bp\n",
      "Total written (filtered):     26,333,233 bp (93.4%)\n",
      "  Read 1:    13,145,765 bp\n",
      "  Read 2:    13,187,468 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t259\t0.0\t1\t0 252 7\n",
      "20\t47378\t0.0\t2\t46608 761 9\n",
      "21\t111\t0.0\t2\t0 111\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47627 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t47294\t0.0\t1\t46395 899\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s622_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s622_244_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s622_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s622_244_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s622_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s622_244_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s622_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s622_244_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        45,295 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.85 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,295\n",
      "  Read 1 with adapter:                  45,280 (100.0%)\n",
      "  Read 2 with adapter:                  45,158 (99.7%)\n",
      "Pairs written (passing filters):        45,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,841 bp\n",
      "  Read 1:    13,362,012 bp\n",
      "  Read 2:    13,355,829 bp\n",
      "Total written (filtered):     24,954,516 bp (93.4%)\n",
      "  Read 1:    12,456,525 bp\n",
      "  Read 2:    12,497,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45280 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t249\t0.0\t1\t0 244 5\n",
      "20\t44900\t0.0\t2\t44202 690 8\n",
      "21\t102\t0.0\t2\t0 99 3\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t44820\t0.0\t1\t44084 736\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s623_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s623_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s623_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s623_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s623_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s623_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s623_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s623_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        52,474 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.62 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,474\n",
      "  Read 1 with adapter:                  52,445 (99.9%)\n",
      "  Read 2 with adapter:                  52,337 (99.7%)\n",
      "Pairs written (passing filters):        52,474 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,957,643 bp\n",
      "  Read 1:    15,477,958 bp\n",
      "  Read 2:    15,479,685 bp\n",
      "Total written (filtered):     28,914,589 bp (93.4%)\n",
      "  Read 1:    14,429,180 bp\n",
      "  Read 2:    14,485,409 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t297\t0.0\t1\t0 291 6\n",
      "20\t51968\t0.0\t2\t51134 818 16\n",
      "21\t129\t0.0\t2\t0 129\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t52008\t0.0\t1\t51068 940\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s624_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s624_246_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s624_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s624_246_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s624_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s624_246_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s624_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s624_246_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,165 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.45 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,165\n",
      "  Read 1 with adapter:                  48,135 (99.9%)\n",
      "  Read 2 with adapter:                  48,033 (99.7%)\n",
      "Pairs written (passing filters):        48,165 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,416,128 bp\n",
      "  Read 1:    14,207,790 bp\n",
      "  Read 2:    14,208,338 bp\n",
      "Total written (filtered):     26,541,042 bp (93.4%)\n",
      "  Read 1:    13,245,184 bp\n",
      "  Read 2:    13,295,858 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t272\t0.0\t1\t0 271 1\n",
      "20\t47679\t0.0\t2\t46884 790 5\n",
      "21\t134\t0.0\t2\t0 134\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t274\t0.0\t1\t0 274\n",
      "19\t47632\t0.0\t1\t46803 829\n",
      "20\t127\t0.0\t1\t0 127\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s625_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s625_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s625_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s625_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s625_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s625_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s625_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s625_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,329 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.22 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,329\n",
      "  Read 1 with adapter:                  46,308 (100.0%)\n",
      "  Read 2 with adapter:                  46,197 (99.7%)\n",
      "Pairs written (passing filters):        46,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,329,125 bp\n",
      "  Read 1:    13,667,048 bp\n",
      "  Read 2:    13,662,077 bp\n",
      "Total written (filtered):     25,525,489 bp (93.4%)\n",
      "  Read 1:    12,741,028 bp\n",
      "  Read 2:    12,784,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46308 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t264\t0.0\t1\t0 259 5\n",
      "20\t45914\t0.0\t2\t45123 785 6\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t243\t0.0\t1\t0 243\n",
      "19\t45838\t0.0\t1\t45104 734\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s626_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s626_248_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s626_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s626_248_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s626_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s626_248_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s626_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s626_248_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,010 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.82 s (66 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,010\n",
      "  Read 1 with adapter:                  42,990 (100.0%)\n",
      "  Read 2 with adapter:                  42,896 (99.7%)\n",
      "Pairs written (passing filters):        43,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,372,686 bp\n",
      "  Read 1:    12,687,929 bp\n",
      "  Read 2:    12,684,757 bp\n",
      "Total written (filtered):     23,698,122 bp (93.4%)\n",
      "  Read 1:    11,828,237 bp\n",
      "  Read 2:    11,869,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t42582\t0.0\t2\t41839 734 9\n",
      "21\t110\t0.0\t2\t0 109 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t234\t0.0\t1\t0 234\n",
      "19\t42580\t0.0\t1\t41845 735\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s627_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s627_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s627_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s627_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s627_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s627_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s627_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s627_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,210 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.25 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,210\n",
      "  Read 1 with adapter:                  47,187 (100.0%)\n",
      "  Read 2 with adapter:                  47,124 (99.8%)\n",
      "Pairs written (passing filters):        47,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,853,860 bp\n",
      "  Read 1:    13,926,935 bp\n",
      "  Read 2:    13,926,925 bp\n",
      "Total written (filtered):     26,015,031 bp (93.4%)\n",
      "  Read 1:    12,983,360 bp\n",
      "  Read 2:    13,031,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t301\t0.0\t1\t0 296 5\n",
      "20\t46733\t0.0\t2\t45939 784 10\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47124 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t46854\t0.0\t1\t46085 769\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s628_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s628_250_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s628_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s628_250_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s628_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s628_250_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s628_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s628_250_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,384 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.55 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,384\n",
      "  Read 1 with adapter:                  50,346 (99.9%)\n",
      "  Read 2 with adapter:                  50,278 (99.8%)\n",
      "Pairs written (passing filters):        50,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,725,694 bp\n",
      "  Read 1:    14,862,844 bp\n",
      "  Read 2:    14,862,850 bp\n",
      "Total written (filtered):     27,763,857 bp (93.4%)\n",
      "  Read 1:    13,856,140 bp\n",
      "  Read 2:    13,907,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t359\t0.0\t1\t0 351 8\n",
      "20\t49820\t0.0\t2\t48882 923 15\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t49889\t0.0\t1\t49066 823\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s629_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s629_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s629_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s629_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s629_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s629_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s629_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s629_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        43,063 reads  @     71.1 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.07 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,063\n",
      "  Read 1 with adapter:                  43,049 (100.0%)\n",
      "  Read 2 with adapter:                  42,958 (99.8%)\n",
      "Pairs written (passing filters):        43,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,404,310 bp\n",
      "  Read 1:    12,703,340 bp\n",
      "  Read 2:    12,700,970 bp\n",
      "Total written (filtered):     23,727,578 bp (93.4%)\n",
      "  Read 1:    11,842,677 bp\n",
      "  Read 2:    11,884,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t402\t0.0\t1\t0 394 8\n",
      "20\t42532\t0.0\t2\t41810 712 10\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42958 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t238\t0.0\t1\t0 238\n",
      "19\t42615\t0.0\t1\t41933 682\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s630_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s630_252_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s630_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s630_252_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s630_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s630_252_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s630_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s630_252_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,590 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.35 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,590\n",
      "  Read 1 with adapter:                  48,571 (100.0%)\n",
      "  Read 2 with adapter:                  48,464 (99.7%)\n",
      "Pairs written (passing filters):        48,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,664,407 bp\n",
      "  Read 1:    14,333,793 bp\n",
      "  Read 2:    14,330,614 bp\n",
      "Total written (filtered):     26,772,638 bp (93.4%)\n",
      "  Read 1:    13,362,704 bp\n",
      "  Read 2:    13,409,934 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t491\t0.0\t1\t0 484 7\n",
      "20\t47912\t0.0\t2\t47113 791 8\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48464 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t48136\t0.0\t1\t47356 780\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s631_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s631_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s631_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s631_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s631_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s631_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s631_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s631_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,528 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.49 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,528\n",
      "  Read 1 with adapter:                  47,506 (100.0%)\n",
      "  Read 2 with adapter:                  47,448 (99.8%)\n",
      "Pairs written (passing filters):        47,528 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,041,198 bp\n",
      "  Read 1:    14,020,505 bp\n",
      "  Read 2:    14,020,693 bp\n",
      "Total written (filtered):     26,190,014 bp (93.4%)\n",
      "  Read 1:    13,070,692 bp\n",
      "  Read 2:    13,119,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t431\t0.0\t1\t0 426 5\n",
      "20\t46930\t0.0\t2\t46149 773 8\n",
      "21\t102\t0.0\t2\t0 102\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t47131\t0.0\t1\t46397 734\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s632_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s632_254_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s632_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s632_254_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s632_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s632_254_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s632_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s632_254_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,692 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.78 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,692\n",
      "  Read 1 with adapter:                  38,680 (100.0%)\n",
      "  Read 2 with adapter:                  38,612 (99.8%)\n",
      "Pairs written (passing filters):        38,692 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,828,025 bp\n",
      "  Read 1:    11,414,125 bp\n",
      "  Read 2:    11,413,900 bp\n",
      "Total written (filtered):     21,321,197 bp (93.4%)\n",
      "  Read 1:    10,640,821 bp\n",
      "  Read 2:    10,680,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38680 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t372\t0.0\t1\t0 362 10\n",
      "20\t38213\t0.0\t2\t37602 603 8\n",
      "21\t62\t0.0\t2\t0 62\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38612 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t38310\t0.0\t1\t37734 576\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s633_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s633_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s633_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s633_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s633_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s633_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s633_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s633_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,505 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.81 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,505\n",
      "  Read 1 with adapter:                  37,485 (99.9%)\n",
      "  Read 2 with adapter:                  37,400 (99.7%)\n",
      "Pairs written (passing filters):        37,505 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,123,730 bp\n",
      "  Read 1:    11,063,684 bp\n",
      "  Read 2:    11,060,046 bp\n",
      "Total written (filtered):     20,663,684 bp (93.4%)\n",
      "  Read 1:    10,314,137 bp\n",
      "  Read 2:    10,349,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t219\t0.0\t1\t0 213 6\n",
      "20\t37171\t0.0\t2\t36479 686 6\n",
      "21\t68\t0.0\t2\t0 68\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t37119\t0.0\t1\t36522 597\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s634_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s634_256_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s634_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s634_256_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s634_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s634_256_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s634_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s634_256_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,058 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.22 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,058\n",
      "  Read 1 with adapter:                  44,036 (100.0%)\n",
      "  Read 2 with adapter:                  43,903 (99.6%)\n",
      "Pairs written (passing filters):        44,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,986,020 bp\n",
      "  Read 1:    12,995,833 bp\n",
      "  Read 2:    12,990,187 bp\n",
      "Total written (filtered):     24,271,611 bp (93.4%)\n",
      "  Read 1:    12,115,449 bp\n",
      "  Read 2:    12,156,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t75\t0.0\t1\t0 0 75\n",
      "19\t311\t0.0\t1\t0 306 5\n",
      "20\t43544\t0.0\t2\t42751 786 7\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t223\t0.0\t1\t0 223\n",
      "19\t43589\t0.0\t1\t42758 831\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s635_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s635_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s635_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s635_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s635_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s635_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s635_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s635_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,428 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.27 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,428\n",
      "  Read 1 with adapter:                  46,401 (99.9%)\n",
      "  Read 2 with adapter:                  46,334 (99.8%)\n",
      "Pairs written (passing filters):        46,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,392,022 bp\n",
      "  Read 1:    13,695,787 bp\n",
      "  Read 2:    13,696,235 bp\n",
      "Total written (filtered):     25,584,034 bp (93.4%)\n",
      "  Read 1:    12,768,013 bp\n",
      "  Read 2:    12,816,021 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t350\t0.0\t1\t0 343 7\n",
      "20\t45913\t0.0\t2\t45095 808 10\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t46008\t0.0\t1\t45239 769\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s636_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s636_258_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s636_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s636_258_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s636_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s636_258_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s636_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s636_258_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,154 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.72 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,154\n",
      "  Read 1 with adapter:                  54,121 (99.9%)\n",
      "  Read 2 with adapter:                  54,007 (99.7%)\n",
      "Pairs written (passing filters):        54,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,949,675 bp\n",
      "  Read 1:    15,975,320 bp\n",
      "  Read 2:    15,974,355 bp\n",
      "Total written (filtered):     29,841,508 bp (93.4%)\n",
      "  Read 1:    14,893,120 bp\n",
      "  Read 2:    14,948,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54121 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t344\t0.0\t1\t0 339 5\n",
      "20\t53619\t0.0\t2\t52659 948 12\n",
      "21\t112\t0.0\t2\t0 109 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t304\t0.0\t1\t0 304\n",
      "19\t53565\t0.0\t1\t52689 876\n",
      "20\t138\t0.0\t1\t0 138\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s637_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s637_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s637_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s637_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s637_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s637_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s637_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s637_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,547 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.09 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,547\n",
      "  Read 1 with adapter:                  44,535 (100.0%)\n",
      "  Read 2 with adapter:                  44,358 (99.6%)\n",
      "Pairs written (passing filters):        44,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,276,144 bp\n",
      "  Read 1:    13,140,465 bp\n",
      "  Read 2:    13,135,679 bp\n",
      "Total written (filtered):     24,542,965 bp (93.4%)\n",
      "  Read 1:    12,249,943 bp\n",
      "  Read 2:    12,293,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t282\t0.0\t1\t0 278 4\n",
      "20\t44141\t0.0\t2\t43429 700 12\n",
      "21\t64\t0.0\t2\t0 63 1\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t255\t0.0\t1\t0 255\n",
      "19\t43993\t0.0\t1\t43165 828\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s638_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s638_260_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s638_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s638_260_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s638_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s638_260_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s638_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s638_260_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,206 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.51 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,206\n",
      "  Read 1 with adapter:                  48,191 (100.0%)\n",
      "  Read 2 with adapter:                  48,075 (99.7%)\n",
      "Pairs written (passing filters):        48,206 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,436,376 bp\n",
      "  Read 1:    14,220,445 bp\n",
      "  Read 2:    14,215,931 bp\n",
      "Total written (filtered):     26,559,458 bp (93.4%)\n",
      "  Read 1:    13,256,808 bp\n",
      "  Read 2:    13,302,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t301\t0.0\t1\t0 295 6\n",
      "20\t47747\t0.0\t2\t46929 814 4\n",
      "21\t84\t0.0\t2\t0 79 5\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t262\t0.0\t1\t0 262\n",
      "19\t47695\t0.0\t1\t46925 770\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s639_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s639_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s639_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s639_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s639_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s639_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s639_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s639_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         9,323 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 0.68 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,323\n",
      "  Read 1 with adapter:                   9,318 (99.9%)\n",
      "  Read 2 with adapter:                   9,302 (99.8%)\n",
      "Pairs written (passing filters):         9,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,500,138 bp\n",
      "  Read 1:     2,749,855 bp\n",
      "  Read 2:     2,750,283 bp\n",
      "Total written (filtered):      5,137,076 bp (93.4%)\n",
      "  Read 1:     2,563,492 bp\n",
      "  Read 2:     2,573,584 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 9318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t45\t0.0\t1\t0 43 2\n",
      "20\t9233\t0.0\t2\t9006 225 2\n",
      "21\t20\t0.0\t2\t0 18 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 9302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t63\t0.0\t1\t0 63\n",
      "19\t9215\t0.0\t1\t9013 202\n",
      "20\t24\t0.0\t1\t0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s640_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s640_262_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s640_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s640_262_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s640_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s640_262_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s640_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s640_262_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,793 reads  @     64.6 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.70 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,793\n",
      "  Read 1 with adapter:                  41,771 (99.9%)\n",
      "  Read 2 with adapter:                  41,707 (99.8%)\n",
      "Pairs written (passing filters):        41,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,657,644 bp\n",
      "  Read 1:    12,328,736 bp\n",
      "  Read 2:    12,328,908 bp\n",
      "Total written (filtered):     23,030,038 bp (93.4%)\n",
      "  Read 1:    11,493,461 bp\n",
      "  Read 2:    11,536,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t270\t0.0\t1\t0 264 6\n",
      "20\t41376\t0.0\t2\t40654 716 6\n",
      "21\t65\t0.0\t2\t0 63 2\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t41367\t0.0\t1\t40720 647\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s641_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s641_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s641_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s641_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s641_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s641_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s641_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s641_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        36,706 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.42 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,706\n",
      "  Read 1 with adapter:                  36,693 (100.0%)\n",
      "  Read 2 with adapter:                  36,605 (99.7%)\n",
      "Pairs written (passing filters):        36,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,650,521 bp\n",
      "  Read 1:    10,828,124 bp\n",
      "  Read 2:    10,822,397 bp\n",
      "Total written (filtered):     20,221,395 bp (93.4%)\n",
      "  Read 1:    10,094,401 bp\n",
      "  Read 2:    10,126,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t199\t0.0\t1\t0 196 3\n",
      "20\t36403\t0.0\t2\t35829 564 10\n",
      "21\t64\t0.0\t2\t0 63 1\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t36353\t0.0\t1\t35620 733\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s642_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s642_264_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s642_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s642_264_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s642_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s642_264_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s642_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s642_264_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,912 reads  @     67.1 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.62 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,912\n",
      "  Read 1 with adapter:                  38,898 (100.0%)\n",
      "  Read 2 with adapter:                  38,791 (99.7%)\n",
      "Pairs written (passing filters):        38,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,953,164 bp\n",
      "  Read 1:    11,479,027 bp\n",
      "  Read 2:    11,474,137 bp\n",
      "Total written (filtered):     21,438,475 bp (93.4%)\n",
      "  Read 1:    10,701,232 bp\n",
      "  Read 2:    10,737,243 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t247\t0.0\t1\t0 245 2\n",
      "20\t38538\t0.0\t2\t37888 648 2\n",
      "21\t72\t0.0\t2\t0 70 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t197\t0.0\t1\t0 197\n",
      "19\t38532\t0.0\t1\t37777 755\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s643_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s643_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s643_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s643_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s643_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s643_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s643_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s643_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,509 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.89 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,509\n",
      "  Read 1 with adapter:                  41,489 (100.0%)\n",
      "  Read 2 with adapter:                  41,322 (99.5%)\n",
      "Pairs written (passing filters):        41,509 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,489,761 bp\n",
      "  Read 1:    12,244,902 bp\n",
      "  Read 2:    12,244,859 bp\n",
      "Total written (filtered):     22,875,141 bp (93.4%)\n",
      "  Read 1:    11,415,288 bp\n",
      "  Read 2:    11,459,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t279\t0.0\t1\t0 272 7\n",
      "20\t41088\t0.0\t2\t40349 727 12\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t41066\t0.0\t1\t40083 983\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s644_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s644_266_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s644_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s644_266_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s644_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s644_266_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s644_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s644_266_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,542 reads  @     66.4 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.63 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,542\n",
      "  Read 1 with adapter:                  39,528 (100.0%)\n",
      "  Read 2 with adapter:                  39,457 (99.8%)\n",
      "Pairs written (passing filters):        39,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,329,141 bp\n",
      "  Read 1:    11,664,643 bp\n",
      "  Read 2:    11,664,498 bp\n",
      "Total written (filtered):     21,789,087 bp (93.4%)\n",
      "  Read 1:    10,874,186 bp\n",
      "  Read 2:    10,914,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t226\t0.0\t1\t0 224 2\n",
      "20\t39178\t0.0\t2\t38531 642 5\n",
      "21\t81\t0.0\t2\t0 81\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t194\t0.0\t1\t0 194\n",
      "19\t39155\t0.0\t1\t38462 693\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s645_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s645_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s645_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s645_267_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s645_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s645_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s645_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s645_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,214 reads  @     63.5 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.37 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,214\n",
      "  Read 1 with adapter:                  37,205 (100.0%)\n",
      "  Read 2 with adapter:                  37,098 (99.7%)\n",
      "Pairs written (passing filters):        37,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,950,787 bp\n",
      "  Read 1:    10,977,891 bp\n",
      "  Read 2:    10,972,896 bp\n",
      "Total written (filtered):     20,502,052 bp (93.4%)\n",
      "  Read 1:    10,233,938 bp\n",
      "  Read 2:    10,268,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t225\t0.0\t1\t0 222 3\n",
      "20\t36888\t0.0\t2\t36297 588 3\n",
      "21\t78\t0.0\t2\t0 75 3\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37098 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t177\t0.0\t1\t0 177\n",
      "19\t36824\t0.0\t1\t36076 748\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s646_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s646_268_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s646_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s646_268_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s646_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s646_268_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s646_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s646_268_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,033 reads  @     64.5 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.58 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,033\n",
      "  Read 1 with adapter:                  40,019 (100.0%)\n",
      "  Read 2 with adapter:                  39,926 (99.7%)\n",
      "Pairs written (passing filters):        40,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,614,345 bp\n",
      "  Read 1:    11,809,053 bp\n",
      "  Read 2:    11,805,292 bp\n",
      "Total written (filtered):     22,055,607 bp (93.4%)\n",
      "  Read 1:    11,008,786 bp\n",
      "  Read 2:    11,046,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40019 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t218\t0.0\t1\t0 213 5\n",
      "20\t39706\t0.0\t2\t39058 643 5\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t39629\t0.0\t1\t38918 711\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s647_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s647_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s647_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s647_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s647_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s647_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s647_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s647_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        42,040 reads  @     62.9 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.65 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,040\n",
      "  Read 1 with adapter:                  42,020 (100.0%)\n",
      "  Read 2 with adapter:                  41,953 (99.8%)\n",
      "Pairs written (passing filters):        42,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,802,963 bp\n",
      "  Read 1:    12,401,229 bp\n",
      "  Read 2:    12,401,734 bp\n",
      "Total written (filtered):     23,165,720 bp (93.4%)\n",
      "  Read 1:    11,560,981 bp\n",
      "  Read 2:    11,604,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42020 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t253\t0.0\t1\t0 250 3\n",
      "20\t41647\t0.0\t2\t40927 714 6\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41953 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t41657\t0.0\t1\t40877 780\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s648_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s648_270_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s648_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s648_270_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s648_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s648_270_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s648_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s648_270_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,871 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.46 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,871\n",
      "  Read 1 with adapter:                  37,859 (100.0%)\n",
      "  Read 2 with adapter:                  37,792 (99.8%)\n",
      "Pairs written (passing filters):        37,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,343,525 bp\n",
      "  Read 1:    11,171,632 bp\n",
      "  Read 2:    11,171,893 bp\n",
      "Total written (filtered):     20,868,523 bp (93.4%)\n",
      "  Read 1:    10,414,604 bp\n",
      "  Read 2:    10,453,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t239\t0.0\t1\t0 237 2\n",
      "20\t37523\t0.0\t2\t36863 648 12\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37792 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t37518\t0.0\t1\t36816 702\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s649_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s649_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s649_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s649_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s649_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-7tae0n68/s649_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s649_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7w5j2o9q/bc77c955-e777-49dc-b909-ee680ea57962/data/s649_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,986 reads  @     64.0 µs/read;   0.94 M reads/minute\n",
      "Finished in 1.80 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,986\n",
      "  Read 1 with adapter:                  27,970 (99.9%)\n",
      "  Read 2 with adapter:                  27,930 (99.8%)\n",
      "Pairs written (passing filters):        27,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,511,482 bp\n",
      "  Read 1:     8,255,665 bp\n",
      "  Read 2:     8,255,817 bp\n",
      "Total written (filtered):     15,421,652 bp (93.4%)\n",
      "  Read 1:     7,696,359 bp\n",
      "  Read 2:     7,725,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t170\t0.0\t1\t0 166 4\n",
      "20\t27713\t0.0\t2\t27179 529 5\n",
      "21\t66\t0.0\t2\t0 66\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t27680\t0.0\t1\t27169 511\n",
      "20\t52\t0.0\t1\t0 52\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: c-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: c-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-tx4y6r3s')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 53abfc32-f2f6-4e00-bccc-1ce9df9ac9aa>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.c import the fastq files for library c into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P1604-Lib1-zhan_manifest_r3.csv \\\n",
    "  --output-path c-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.c trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences c-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences c-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.c summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data c-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization c-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('c-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: c-paired-end-demux_table-dada2.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: c-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: c-paired-end-demux_stats-dada2.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#4.c Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs c-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 240 \\\n",
    "  --p-trunc-len-r 199 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table c-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats c-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: c-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-qrmmw0iq')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: ba9365b5-f9df-408f-b076-d2ba3b052c96>"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.c view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table c-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization c-table.qzv \n",
    "q2.Visualization.load('c-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: c-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.c view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization c-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/BB-Lib1_zhan_manifest_runbb_v2_grid.csv as PairedEndFastqManifestPhred33 to d-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1000_294_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1000_295_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1000_294_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1000_295_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1000_294_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1000_295_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1000_294_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1000_295_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,434 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.28 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,434\n",
      "  Read 1 with adapter:                  55,287 (99.7%)\n",
      "  Read 2 with adapter:                  54,939 (99.1%)\n",
      "Pairs written (passing filters):        55,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,675,552 bp\n",
      "  Read 1:    16,334,861 bp\n",
      "  Read 2:    16,340,691 bp\n",
      "Total written (filtered):     30,526,069 bp (93.4%)\n",
      "  Read 1:    15,229,081 bp\n",
      "  Read 2:    15,296,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t384\t0.0\t1\t0 376 8\n",
      "20\t54749\t0.0\t2\t53804 932 13\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t54575\t0.0\t1\t51773 2802\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1001_296_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1001_297_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1001_296_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1001_297_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1001_296_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1001_297_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1001_296_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1001_297_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,332 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.01 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,332\n",
      "  Read 1 with adapter:                  51,299 (99.9%)\n",
      "  Read 2 with adapter:                  51,027 (99.4%)\n",
      "Pairs written (passing filters):        51,332 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,285,302 bp\n",
      "  Read 1:    15,142,696 bp\n",
      "  Read 2:    15,142,606 bp\n",
      "Total written (filtered):     28,290,129 bp (93.4%)\n",
      "  Read 1:    14,116,925 bp\n",
      "  Read 2:    14,173,204 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51299 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t342\t0.0\t1\t0 338 4\n",
      "20\t50814\t0.0\t2\t49945 855 14\n",
      "21\t101\t0.0\t2\t0 100 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51027 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t50720\t0.0\t1\t48040 2680\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1002_298_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1002_299_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1002_298_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1002_299_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1002_298_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1002_299_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1002_298_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1002_299_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,859 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.99 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,859\n",
      "  Read 1 with adapter:                  64,821 (99.9%)\n",
      "  Read 2 with adapter:                  64,294 (99.1%)\n",
      "Pairs written (passing filters):        64,859 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,261,814 bp\n",
      "  Read 1:    19,132,538 bp\n",
      "  Read 2:    19,129,276 bp\n",
      "Total written (filtered):     35,744,220 bp (93.4%)\n",
      "  Read 1:    17,836,356 bp\n",
      "  Read 2:    17,907,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t412\t0.0\t1\t0 405 7\n",
      "20\t64219\t0.0\t2\t63154 1051 14\n",
      "21\t138\t0.0\t2\t0 134 4\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64294 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t332\t0.0\t1\t0 332\n",
      "19\t63804\t0.0\t1\t60370 3434\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1003_300_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1003_301_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1003_300_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1003_301_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1003_300_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S1003_301_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1003_300_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S1003_301_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,653 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.72 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,653\n",
      "  Read 1 with adapter:                  47,628 (99.9%)\n",
      "  Read 2 with adapter:                  47,057 (98.7%)\n",
      "Pairs written (passing filters):        47,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,115,199 bp\n",
      "  Read 1:    14,057,621 bp\n",
      "  Read 2:    14,057,578 bp\n",
      "Total written (filtered):     26,268,925 bp (93.4%)\n",
      "  Read 1:    13,105,259 bp\n",
      "  Read 2:    13,163,666 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t293\t0.0\t1\t0 286 7\n",
      "20\t47204\t0.0\t2\t46308 883 13\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t46772\t0.0\t1\t43848 2924\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S853_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S853_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S853_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S853_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S853_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S853_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S853_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S853_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,425 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.85 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,425\n",
      "  Read 1 with adapter:                  50,405 (100.0%)\n",
      "  Read 2 with adapter:                  49,973 (99.1%)\n",
      "Pairs written (passing filters):        50,425 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,750,028 bp\n",
      "  Read 1:    14,874,875 bp\n",
      "  Read 2:    14,875,153 bp\n",
      "Total written (filtered):     27,792,726 bp (93.4%)\n",
      "  Read 1:    13,866,920 bp\n",
      "  Read 2:    13,925,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50405 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t305\t0.0\t1\t0 301 4\n",
      "20\t49946\t0.0\t2\t49205 735 6\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49973 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t49631\t0.0\t1\t47106 2525\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S854_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S854_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S854_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S854_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S854_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S854_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S854_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S854_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,362 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.47 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,362\n",
      "  Read 1 with adapter:                  47,334 (99.9%)\n",
      "  Read 2 with adapter:                  46,926 (99.1%)\n",
      "Pairs written (passing filters):        47,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,941,778 bp\n",
      "  Read 1:    13,971,294 bp\n",
      "  Read 2:    13,970,484 bp\n",
      "Total written (filtered):     26,103,843 bp (93.4%)\n",
      "  Read 1:    13,024,815 bp\n",
      "  Read 2:    13,079,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t324\t0.0\t1\t0 315 9\n",
      "20\t46876\t0.0\t2\t46085 783 8\n",
      "21\t93\t0.0\t2\t0 92 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t217\t0.0\t1\t0 217\n",
      "19\t46630\t0.0\t1\t43776 2854\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S855_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S855_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S855_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S855_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S855_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S855_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S855_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S855_5_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,691 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.71 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,691\n",
      "  Read 1 with adapter:                  51,666 (100.0%)\n",
      "  Read 2 with adapter:                  51,140 (98.9%)\n",
      "Pairs written (passing filters):        51,691 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,494,970 bp\n",
      "  Read 1:    15,248,830 bp\n",
      "  Read 2:    15,246,140 bp\n",
      "Total written (filtered):     28,490,337 bp (93.4%)\n",
      "  Read 1:    14,215,732 bp\n",
      "  Read 2:    14,274,605 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t329\t0.0\t1\t0 318 11\n",
      "20\t51211\t0.0\t2\t50239 963 9\n",
      "21\t89\t0.0\t2\t0 85 4\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t225\t0.0\t1\t0 225\n",
      "19\t50815\t0.0\t1\t47733 3082\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S856_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S856_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S856_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S856_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S856_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S856_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S856_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S856_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,278 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.43 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,278\n",
      "  Read 1 with adapter:                  64,248 (100.0%)\n",
      "  Read 2 with adapter:                  63,736 (99.2%)\n",
      "Pairs written (passing filters):        64,278 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,923,968 bp\n",
      "  Read 1:    18,961,990 bp\n",
      "  Read 2:    18,961,978 bp\n",
      "Total written (filtered):     35,428,515 bp (93.4%)\n",
      "  Read 1:    17,677,313 bp\n",
      "  Read 2:    17,751,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64248 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t412\t0.0\t1\t0 407 5\n",
      "20\t63653\t0.0\t2\t62640 996 17\n",
      "21\t129\t0.0\t2\t0 127 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63736 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t356\t0.0\t1\t0 356\n",
      "19\t63232\t0.0\t1\t59395 3837\n",
      "20\t148\t0.0\t1\t0 148\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S857_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S857_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S857_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S857_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S857_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S857_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S857_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S857_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,507 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.31 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,507\n",
      "  Read 1 with adapter:                  48,480 (99.9%)\n",
      "  Read 2 with adapter:                  48,078 (99.1%)\n",
      "Pairs written (passing filters):        48,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,617,445 bp\n",
      "  Read 1:    14,309,553 bp\n",
      "  Read 2:    14,307,892 bp\n",
      "Total written (filtered):     26,734,712 bp (93.4%)\n",
      "  Read 1:    13,340,188 bp\n",
      "  Read 2:    13,394,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t324\t0.0\t1\t0 317 7\n",
      "20\t48018\t0.0\t2\t47248 755 15\n",
      "21\t99\t0.0\t2\t0 97 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t47770\t0.0\t1\t44693 3077\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S858_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S858_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S858_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S858_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S858_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S858_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S858_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S858_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,725 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.56 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,725\n",
      "  Read 1 with adapter:                  51,692 (99.9%)\n",
      "  Read 2 with adapter:                  51,215 (99.0%)\n",
      "Pairs written (passing filters):        51,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,506,550 bp\n",
      "  Read 1:    15,258,773 bp\n",
      "  Read 2:    15,247,777 bp\n",
      "Total written (filtered):     28,499,926 bp (93.4%)\n",
      "  Read 1:    14,225,136 bp\n",
      "  Read 2:    14,274,790 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t312\t0.0\t1\t0 309 3\n",
      "20\t51231\t0.0\t2\t50405 814 12\n",
      "21\t97\t0.0\t2\t0 97\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t50891\t0.0\t1\t47861 3030\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S859_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S859_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S859_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S859_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S859_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S859_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S859_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S859_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,674 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.55 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,674\n",
      "  Read 1 with adapter:                  50,654 (100.0%)\n",
      "  Read 2 with adapter:                  50,308 (99.3%)\n",
      "Pairs written (passing filters):        50,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,882,477 bp\n",
      "  Read 1:    14,948,342 bp\n",
      "  Read 2:    14,934,135 bp\n",
      "Total written (filtered):     27,913,359 bp (93.4%)\n",
      "  Read 1:    13,934,943 bp\n",
      "  Read 2:    13,978,416 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t314\t0.0\t1\t0 308 6\n",
      "20\t50208\t0.0\t2\t49403 792 13\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50308 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t49939\t0.0\t1\t47412 2527\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S860_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S860_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S860_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S860_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S860_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S860_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S860_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S860_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,203 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.88 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,203\n",
      "  Read 1 with adapter:                  53,176 (99.9%)\n",
      "  Read 2 with adapter:                  52,317 (98.3%)\n",
      "Pairs written (passing filters):        53,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,348,177 bp\n",
      "  Read 1:    15,694,868 bp\n",
      "  Read 2:    15,653,309 bp\n",
      "Total written (filtered):     29,290,986 bp (93.4%)\n",
      "  Read 1:    14,631,561 bp\n",
      "  Read 2:    14,659,425 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t345\t0.0\t1\t0 342 3\n",
      "20\t52677\t0.0\t2\t51802 862 13\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52317 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t255\t0.0\t1\t0 255\n",
      "19\t51946\t0.0\t1\t49519 2427\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S861_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S861_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S861_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S861_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S861_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S861_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S861_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S861_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        43,558 reads  @     73.5 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.21 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,558\n",
      "  Read 1 with adapter:                  43,540 (100.0%)\n",
      "  Read 2 with adapter:                  43,292 (99.4%)\n",
      "Pairs written (passing filters):        43,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,698,658 bp\n",
      "  Read 1:    12,849,594 bp\n",
      "  Read 2:    12,849,064 bp\n",
      "Total written (filtered):     24,005,615 bp (93.4%)\n",
      "  Read 1:    11,978,975 bp\n",
      "  Read 2:    12,026,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43540 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t295\t0.0\t1\t0 287 8\n",
      "20\t43130\t0.0\t2\t42424 700 6\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t206\t0.0\t1\t0 206\n",
      "19\t43004\t0.0\t1\t40855 2149\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S862_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S862_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S862_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S862_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S862_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S862_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S862_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S862_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,065 reads  @     74.1 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.71 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,065\n",
      "  Read 1 with adapter:                  50,033 (99.9%)\n",
      "  Read 2 with adapter:                  49,750 (99.4%)\n",
      "Pairs written (passing filters):        50,065 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,537,746 bp\n",
      "  Read 1:    14,769,036 bp\n",
      "  Read 2:    14,768,710 bp\n",
      "Total written (filtered):     27,592,198 bp (93.4%)\n",
      "  Read 1:    13,768,567 bp\n",
      "  Read 2:    13,823,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t306\t0.0\t1\t0 301 5\n",
      "20\t49596\t0.0\t2\t48818 772 6\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49750 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t262\t0.0\t1\t0 262\n",
      "19\t49397\t0.0\t1\t46832 2565\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S863_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S863_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S863_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S863_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S863_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S863_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S863_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S863_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,927 reads  @     73.7 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.24 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,927\n",
      "  Read 1 with adapter:                  43,897 (99.9%)\n",
      "  Read 2 with adapter:                  43,596 (99.2%)\n",
      "Pairs written (passing filters):        43,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,913,031 bp\n",
      "  Read 1:    12,958,457 bp\n",
      "  Read 2:    12,954,574 bp\n",
      "Total written (filtered):     24,207,100 bp (93.4%)\n",
      "  Read 1:    12,080,709 bp\n",
      "  Read 2:    12,126,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t285\t0.0\t1\t0 279 6\n",
      "20\t43512\t0.0\t2\t42837 668 7\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t222\t0.0\t1\t0 222\n",
      "19\t43293\t0.0\t1\t41015 2278\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S864_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S864_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S864_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S864_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S864_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S864_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S864_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S864_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,385 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.67 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,385\n",
      "  Read 1 with adapter:                  38,367 (100.0%)\n",
      "  Read 2 with adapter:                  38,125 (99.3%)\n",
      "Pairs written (passing filters):        38,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,647,121 bp\n",
      "  Read 1:    11,323,566 bp\n",
      "  Read 2:    11,323,555 bp\n",
      "Total written (filtered):     21,155,703 bp (93.4%)\n",
      "  Read 1:    10,556,360 bp\n",
      "  Read 2:    10,599,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38367 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t228\t0.0\t1\t0 226 2\n",
      "20\t38042\t0.0\t2\t37380 656 6\n",
      "21\t60\t0.0\t2\t0 57 3\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t245\t0.0\t1\t0 245\n",
      "19\t37798\t0.0\t1\t35757 2041\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S865_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S865_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S865_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S865_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S865_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S865_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S865_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S865_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,080 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.38 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,080\n",
      "  Read 1 with adapter:                  46,051 (99.9%)\n",
      "  Read 2 with adapter:                  45,778 (99.3%)\n",
      "Pairs written (passing filters):        46,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,186,022 bp\n",
      "  Read 1:    13,593,584 bp\n",
      "  Read 2:    13,592,438 bp\n",
      "Total written (filtered):     25,395,537 bp (93.4%)\n",
      "  Read 1:    12,672,722 bp\n",
      "  Read 2:    12,722,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t270\t0.0\t1\t0 263 7\n",
      "20\t45648\t0.0\t2\t44937 698 13\n",
      "21\t90\t0.0\t2\t0 89 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t237\t0.0\t1\t0 237\n",
      "19\t45463\t0.0\t1\t43031 2432\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S866_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S866_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S866_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S866_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S866_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S866_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S866_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S866_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,399 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.11 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,399\n",
      "  Read 1 with adapter:                  44,368 (99.9%)\n",
      "  Read 2 with adapter:                  44,034 (99.2%)\n",
      "Pairs written (passing filters):        44,399 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,183,694 bp\n",
      "  Read 1:    13,097,624 bp\n",
      "  Read 2:    13,086,070 bp\n",
      "Total written (filtered):     24,459,938 bp (93.4%)\n",
      "  Read 1:    12,210,426 bp\n",
      "  Read 2:    12,249,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t43992\t0.0\t2\t43296 686 10\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t179\t0.0\t1\t0 179\n",
      "19\t43764\t0.0\t1\t41519 2245\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S867_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S867_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S867_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S867_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S867_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S867_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S867_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S867_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        42,058 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.86 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,058\n",
      "  Read 1 with adapter:                  42,037 (100.0%)\n",
      "  Read 2 with adapter:                  41,821 (99.4%)\n",
      "Pairs written (passing filters):        42,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,802,216 bp\n",
      "  Read 1:    12,406,853 bp\n",
      "  Read 2:    12,395,363 bp\n",
      "Total written (filtered):     23,167,228 bp (93.4%)\n",
      "  Read 1:    11,566,336 bp\n",
      "  Read 2:    11,600,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t281\t0.0\t1\t0 275 6\n",
      "20\t41671\t0.0\t2\t40964 703 4\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t41533\t0.0\t1\t39583 1950\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S868_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S868_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S868_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S868_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S868_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S868_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S868_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S868_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,684 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.34 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,684\n",
      "  Read 1 with adapter:                  43,665 (100.0%)\n",
      "  Read 2 with adapter:                  42,888 (98.2%)\n",
      "Pairs written (passing filters):        43,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,735,953 bp\n",
      "  Read 1:    12,886,470 bp\n",
      "  Read 2:    12,849,483 bp\n",
      "Total written (filtered):     24,048,021 bp (93.4%)\n",
      "  Read 1:    12,013,325 bp\n",
      "  Read 2:    12,034,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t253\t0.0\t1\t0 246 7\n",
      "20\t43288\t0.0\t2\t42541 733 14\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t42625\t0.0\t1\t40660 1965\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S869_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S869_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S869_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S869_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S869_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S869_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S869_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S869_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,861 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.41 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,861\n",
      "  Read 1 with adapter:                  46,840 (100.0%)\n",
      "  Read 2 with adapter:                  46,397 (99.0%)\n",
      "Pairs written (passing filters):        46,861 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,645,214 bp\n",
      "  Read 1:    13,823,985 bp\n",
      "  Read 2:    13,821,229 bp\n",
      "Total written (filtered):     25,827,185 bp (93.4%)\n",
      "  Read 1:    12,887,337 bp\n",
      "  Read 2:    12,939,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46840 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t269\t0.0\t1\t0 265 4\n",
      "20\t46464\t0.0\t2\t45652 799 13\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t46077\t0.0\t1\t43546 2531\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S870_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S870_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S870_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S870_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S870_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S870_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S870_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S870_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,851 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.03 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,851\n",
      "  Read 1 with adapter:                  28,832 (99.9%)\n",
      "  Read 2 with adapter:                  28,494 (98.8%)\n",
      "Pairs written (passing filters):        28,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,018,359 bp\n",
      "  Read 1:     8,509,742 bp\n",
      "  Read 2:     8,508,617 bp\n",
      "Total written (filtered):     15,900,474 bp (93.4%)\n",
      "  Read 1:     7,933,193 bp\n",
      "  Read 2:     7,967,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t156\t0.0\t1\t0 152 4\n",
      "20\t28608\t0.0\t2\t28072 530 6\n",
      "21\t47\t0.0\t2\t0 45 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28494 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t111\t0.0\t1\t0 111\n",
      "19\t28322\t0.0\t1\t26613 1709\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S871_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S871_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S871_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S871_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S871_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S871_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S871_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S871_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,900 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.72 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,900\n",
      "  Read 1 with adapter:                  54,866 (99.9%)\n",
      "  Read 2 with adapter:                  54,315 (98.9%)\n",
      "Pairs written (passing filters):        54,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,389,798 bp\n",
      "  Read 1:    16,195,298 bp\n",
      "  Read 2:    16,194,500 bp\n",
      "Total written (filtered):     30,260,788 bp (93.4%)\n",
      "  Read 1:    15,098,147 bp\n",
      "  Read 2:    15,162,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54866 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t305\t0.0\t1\t0 300 5\n",
      "20\t54418\t0.0\t2\t53473 925 20\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t53985\t0.0\t1\t50877 3108\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S872_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S872_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S872_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S872_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S872_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S872_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S872_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S872_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,529 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.06 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,529\n",
      "  Read 1 with adapter:                  43,508 (100.0%)\n",
      "  Read 2 with adapter:                  43,045 (98.9%)\n",
      "Pairs written (passing filters):        43,529 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,681,765 bp\n",
      "  Read 1:    12,840,903 bp\n",
      "  Read 2:    12,840,862 bp\n",
      "Total written (filtered):     23,994,063 bp (93.4%)\n",
      "  Read 1:    11,970,896 bp\n",
      "  Read 2:    12,023,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t247\t0.0\t1\t0 242 5\n",
      "20\t43141\t0.0\t2\t42390 747 4\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t42689\t0.0\t1\t40168 2521\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S873_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S873_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S873_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S873_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S873_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S873_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S873_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S873_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:07        98,667 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 7.18 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,667\n",
      "  Read 1 with adapter:                  98,613 (99.9%)\n",
      "  Read 2 with adapter:                  97,484 (98.8%)\n",
      "Pairs written (passing filters):        98,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,206,055 bp\n",
      "  Read 1:    29,106,734 bp\n",
      "  Read 2:    29,099,321 bp\n",
      "Total written (filtered):     54,382,169 bp (93.4%)\n",
      "  Read 1:    27,134,768 bp\n",
      "  Read 2:    27,247,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 98613 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t521\t0.0\t1\t0 508 13\n",
      "20\t97834\t0.0\t2\t96108 1708 18\n",
      "21\t181\t0.0\t2\t0 178 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 97484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t461\t0.0\t1\t0 461\n",
      "19\t96838\t0.0\t1\t90609 6229\n",
      "20\t185\t0.0\t1\t0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S874_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S874_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S874_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S874_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S874_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S874_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S874_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S874_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,478 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.72 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,478\n",
      "  Read 1 with adapter:                  63,436 (99.9%)\n",
      "  Read 2 with adapter:                  62,700 (98.8%)\n",
      "Pairs written (passing filters):        63,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,426,813 bp\n",
      "  Read 1:    18,725,756 bp\n",
      "  Read 2:    18,701,057 bp\n",
      "Total written (filtered):     34,967,135 bp (93.4%)\n",
      "  Read 1:    17,457,197 bp\n",
      "  Read 2:    17,509,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t321\t0.0\t1\t0 314 7\n",
      "20\t62943\t0.0\t2\t61838 1091 14\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62700 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t281\t0.0\t1\t0 281\n",
      "19\t62319\t0.0\t1\t58827 3492\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S875_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S875_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S875_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S875_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S875_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S875_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S875_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S875_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,386 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.00 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,386\n",
      "  Read 1 with adapter:                  53,356 (99.9%)\n",
      "  Read 2 with adapter:                  52,780 (98.9%)\n",
      "Pairs written (passing filters):        53,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,480,250 bp\n",
      "  Read 1:    15,748,732 bp\n",
      "  Read 2:    15,731,518 bp\n",
      "Total written (filtered):     29,410,651 bp (93.4%)\n",
      "  Read 1:    14,681,841 bp\n",
      "  Read 2:    14,728,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t333\t0.0\t1\t0 324 9\n",
      "20\t52878\t0.0\t2\t51938 933 7\n",
      "21\t106\t0.0\t2\t0 106\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t217\t0.0\t1\t0 217\n",
      "19\t52458\t0.0\t1\t49686 2772\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S876_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S876_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S876_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S876_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S876_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S876_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S876_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S876_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,154 reads  @     78.5 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.80 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,154\n",
      "  Read 1 with adapter:                  61,127 (100.0%)\n",
      "  Read 2 with adapter:                  59,296 (97.0%)\n",
      "Pairs written (passing filters):        61,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,999,231 bp\n",
      "  Read 1:    18,040,230 bp\n",
      "  Read 2:    17,959,001 bp\n",
      "Total written (filtered):     33,650,408 bp (93.5%)\n",
      "  Read 1:    16,817,885 bp\n",
      "  Read 2:    16,832,523 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61127 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t345\t0.0\t1\t0 341 4\n",
      "20\t60603\t0.0\t2\t59514 1076 13\n",
      "21\t124\t0.0\t2\t0 124\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t58940\t0.0\t1\t55648 3292\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S877_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S877_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S877_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S877_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S877_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S877_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S877_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S877_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,672 reads  @     73.5 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.58 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,672\n",
      "  Read 1 with adapter:                  48,647 (99.9%)\n",
      "  Read 2 with adapter:                  48,451 (99.5%)\n",
      "Pairs written (passing filters):        48,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,715,267 bp\n",
      "  Read 1:    14,358,228 bp\n",
      "  Read 2:    14,357,039 bp\n",
      "Total written (filtered):     26,822,090 bp (93.4%)\n",
      "  Read 1:    13,385,448 bp\n",
      "  Read 2:    13,436,642 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t297\t0.0\t1\t0 293 4\n",
      "20\t48218\t0.0\t2\t47290 917 11\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48451 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t48085\t0.0\t1\t45871 2214\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S878_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S878_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S878_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S878_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S878_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S878_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S878_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S878_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,892 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.31 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,892\n",
      "  Read 1 with adapter:                  48,867 (99.9%)\n",
      "  Read 2 with adapter:                  48,675 (99.6%)\n",
      "Pairs written (passing filters):        48,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,845,717 bp\n",
      "  Read 1:    14,423,134 bp\n",
      "  Read 2:    14,422,583 bp\n",
      "Total written (filtered):     26,943,906 bp (93.4%)\n",
      "  Read 1:    13,446,000 bp\n",
      "  Read 2:    13,497,906 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48867 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t295\t0.0\t1\t0 291 4\n",
      "20\t48455\t0.0\t2\t47663 781 11\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t48353\t0.0\t1\t46144 2209\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S879_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S879_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S879_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S879_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S879_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S879_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S879_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S879_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,745 reads  @     66.2 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.23 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,745\n",
      "  Read 1 with adapter:                  48,722 (100.0%)\n",
      "  Read 2 with adapter:                  48,551 (99.6%)\n",
      "Pairs written (passing filters):        48,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,759,443 bp\n",
      "  Read 1:    14,379,770 bp\n",
      "  Read 2:    14,379,673 bp\n",
      "Total written (filtered):     26,862,844 bp (93.4%)\n",
      "  Read 1:    13,405,525 bp\n",
      "  Read 2:    13,457,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t333\t0.0\t1\t0 327 6\n",
      "20\t48246\t0.0\t2\t47466 763 17\n",
      "21\t100\t0.0\t2\t0 96 4\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48551 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t48236\t0.0\t1\t46167 2069\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S880_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S880_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S880_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S880_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S880_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S880_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S880_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S880_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,139 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.26 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,139\n",
      "  Read 1 with adapter:                  48,122 (100.0%)\n",
      "  Read 2 with adapter:                  47,938 (99.6%)\n",
      "Pairs written (passing filters):        48,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,401,700 bp\n",
      "  Read 1:    14,200,783 bp\n",
      "  Read 2:    14,200,917 bp\n",
      "Total written (filtered):     26,528,816 bp (93.4%)\n",
      "  Read 1:    13,238,547 bp\n",
      "  Read 2:    13,290,269 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t306\t0.0\t1\t0 297 9\n",
      "20\t47698\t0.0\t2\t46956 732 10\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47938 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t281\t0.0\t1\t0 281\n",
      "19\t47550\t0.0\t1\t45473 2077\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S881_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S881_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S881_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S881_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S881_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S881_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S881_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S881_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,061 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.79 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,061\n",
      "  Read 1 with adapter:                  53,036 (100.0%)\n",
      "  Read 2 with adapter:                  52,859 (99.6%)\n",
      "Pairs written (passing filters):        53,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,304,901 bp\n",
      "  Read 1:    15,652,986 bp\n",
      "  Read 2:    15,651,915 bp\n",
      "Total written (filtered):     29,240,243 bp (93.4%)\n",
      "  Read 1:    14,592,491 bp\n",
      "  Read 2:    14,647,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t362\t0.0\t1\t0 358 4\n",
      "20\t52550\t0.0\t2\t51640 896 14\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t253\t0.0\t1\t0 253\n",
      "19\t52511\t0.0\t1\t50249 2262\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S882_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S882_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S882_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S882_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S882_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S882_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S882_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S882_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,625 reads  @     74.0 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.19 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,625\n",
      "  Read 1 with adapter:                  56,587 (99.9%)\n",
      "  Read 2 with adapter:                  56,381 (99.6%)\n",
      "Pairs written (passing filters):        56,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,398,424 bp\n",
      "  Read 1:    16,704,338 bp\n",
      "  Read 2:    16,694,086 bp\n",
      "Total written (filtered):     31,195,887 bp (93.4%)\n",
      "  Read 1:    15,572,878 bp\n",
      "  Read 2:    15,623,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t382\t0.0\t1\t0 377 5\n",
      "20\t56070\t0.0\t2\t55167 897 6\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56381 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t56023\t0.0\t1\t53490 2533\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S883_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S883_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S883_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S883_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S883_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S883_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S883_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S883_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,690 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.95 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,690\n",
      "  Read 1 with adapter:                  43,674 (100.0%)\n",
      "  Read 2 with adapter:                  43,519 (99.6%)\n",
      "Pairs written (passing filters):        43,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,771,279 bp\n",
      "  Read 1:    12,888,541 bp\n",
      "  Read 2:    12,882,738 bp\n",
      "Total written (filtered):     24,071,250 bp (93.4%)\n",
      "  Read 1:    12,015,263 bp\n",
      "  Read 2:    12,055,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43674 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t293\t0.0\t1\t0 291 2\n",
      "20\t43262\t0.0\t2\t42539 713 10\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43519 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t43239\t0.0\t1\t41456 1783\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S884_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S884_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S884_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S884_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S884_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S884_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S884_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S884_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,463 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.55 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,463\n",
      "  Read 1 with adapter:                  48,448 (100.0%)\n",
      "  Read 2 with adapter:                  47,662 (98.3%)\n",
      "Pairs written (passing filters):        48,463 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,549,900 bp\n",
      "  Read 1:    14,296,567 bp\n",
      "  Read 2:    14,253,333 bp\n",
      "Total written (filtered):     26,675,736 bp (93.4%)\n",
      "  Read 1:    13,327,814 bp\n",
      "  Read 2:    13,347,922 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t311\t0.0\t1\t0 309 2\n",
      "20\t48010\t0.0\t2\t47204 792 14\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t237\t0.0\t1\t0 237\n",
      "19\t47355\t0.0\t1\t45478 1877\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S885_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S885_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S885_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S885_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S885_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S885_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S885_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S885_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,369 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.30 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,369\n",
      "  Read 1 with adapter:                  48,353 (100.0%)\n",
      "  Read 2 with adapter:                  48,114 (99.5%)\n",
      "Pairs written (passing filters):        48,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,535,647 bp\n",
      "  Read 1:    14,268,842 bp\n",
      "  Read 2:    14,266,805 bp\n",
      "Total written (filtered):     26,654,746 bp (93.4%)\n",
      "  Read 1:    13,301,967 bp\n",
      "  Read 2:    13,352,779 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48353 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t301\t0.0\t1\t0 295 6\n",
      "20\t47907\t0.0\t2\t47054 843 10\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48114 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t47790\t0.0\t1\t45668 2122\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S886_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S886_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S886_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S886_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S886_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S886_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S886_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S886_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,752 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.66 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,752\n",
      "  Read 1 with adapter:                  54,725 (100.0%)\n",
      "  Read 2 with adapter:                  54,492 (99.5%)\n",
      "Pairs written (passing filters):        54,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,302,283 bp\n",
      "  Read 1:    16,151,605 bp\n",
      "  Read 2:    16,150,678 bp\n",
      "Total written (filtered):     30,172,731 bp (93.4%)\n",
      "  Read 1:    15,057,269 bp\n",
      "  Read 2:    15,115,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t294\t0.0\t1\t0 285 9\n",
      "20\t54283\t0.0\t2\t53350 922 11\n",
      "21\t98\t0.0\t2\t0 93 5\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t54108\t0.0\t1\t51545 2563\n",
      "20\t126\t0.0\t1\t0 126\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S887_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S887_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S887_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S887_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S887_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S887_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S887_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S887_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,711 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.85 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,711\n",
      "  Read 1 with adapter:                  52,690 (100.0%)\n",
      "  Read 2 with adapter:                  52,418 (99.4%)\n",
      "Pairs written (passing filters):        52,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,097,653 bp\n",
      "  Read 1:    15,549,734 bp\n",
      "  Read 2:    15,547,919 bp\n",
      "Total written (filtered):     29,048,184 bp (93.4%)\n",
      "  Read 1:    14,496,086 bp\n",
      "  Read 2:    14,552,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t291\t0.0\t1\t0 283 8\n",
      "20\t52259\t0.0\t2\t51364 886 9\n",
      "21\t89\t0.0\t2\t0 87 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52418 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t52085\t0.0\t1\t49597 2488\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S888_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S888_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S888_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S888_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S888_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S888_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S888_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S888_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,764 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.55 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,764\n",
      "  Read 1 with adapter:                  45,741 (99.9%)\n",
      "  Read 2 with adapter:                  45,484 (99.4%)\n",
      "Pairs written (passing filters):        45,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,999,790 bp\n",
      "  Read 1:    13,500,177 bp\n",
      "  Read 2:    13,499,613 bp\n",
      "Total written (filtered):     25,221,074 bp (93.4%)\n",
      "  Read 1:    12,585,518 bp\n",
      "  Read 2:    12,635,556 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45741 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t277\t0.0\t1\t0 270 7\n",
      "20\t45326\t0.0\t2\t44568 746 12\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t247\t0.0\t1\t0 247\n",
      "19\t45129\t0.0\t1\t42964 2165\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S889_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S889_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S889_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S889_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S889_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S889_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S889_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S889_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,718 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.40 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,718\n",
      "  Read 1 with adapter:                  54,701 (100.0%)\n",
      "  Read 2 with adapter:                  54,412 (99.4%)\n",
      "Pairs written (passing filters):        54,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,280,907 bp\n",
      "  Read 1:    16,141,793 bp\n",
      "  Read 2:    16,139,114 bp\n",
      "Total written (filtered):     30,153,416 bp (93.4%)\n",
      "  Read 1:    15,047,953 bp\n",
      "  Read 2:    15,105,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54701 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t326\t0.0\t1\t0 318 8\n",
      "20\t54222\t0.0\t2\t53252 952 18\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t276\t0.0\t1\t0 276\n",
      "19\t54037\t0.0\t1\t51423 2614\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S890_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S890_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S890_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S890_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S890_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S890_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S890_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S890_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,483 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.71 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,483\n",
      "  Read 1 with adapter:                  49,452 (99.9%)\n",
      "  Read 2 with adapter:                  49,226 (99.5%)\n",
      "Pairs written (passing filters):        49,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,189,021 bp\n",
      "  Read 1:    14,597,249 bp\n",
      "  Read 2:    14,591,772 bp\n",
      "Total written (filtered):     27,264,983 bp (93.4%)\n",
      "  Read 1:    13,608,381 bp\n",
      "  Read 2:    13,656,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t282\t0.0\t1\t0 278 4\n",
      "20\t49045\t0.0\t2\t48238 798 9\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49226 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t48922\t0.0\t1\t46689 2233\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S891_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S891_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S891_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S891_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S891_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S891_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S891_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S891_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,636 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.92 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,636\n",
      "  Read 1 with adapter:                  50,616 (100.0%)\n",
      "  Read 2 with adapter:                  50,414 (99.6%)\n",
      "Pairs written (passing filters):        50,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,865,349 bp\n",
      "  Read 1:    14,937,413 bp\n",
      "  Read 2:    14,927,936 bp\n",
      "Total written (filtered):     27,895,534 bp (93.4%)\n",
      "  Read 1:    13,925,287 bp\n",
      "  Read 2:    13,970,247 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t302\t0.0\t1\t0 294 8\n",
      "20\t50190\t0.0\t2\t49226 957 7\n",
      "21\t84\t0.0\t2\t0 82 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50414 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t50053\t0.0\t1\t47902 2151\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S892_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S892_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S892_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S892_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S892_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S892_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S892_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S892_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,129 reads  @     79.6 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.15 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,129\n",
      "  Read 1 with adapter:                  52,097 (99.9%)\n",
      "  Read 2 with adapter:                  51,340 (98.5%)\n",
      "Pairs written (passing filters):        52,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,716,366 bp\n",
      "  Read 1:    15,377,279 bp\n",
      "  Read 2:    15,339,087 bp\n",
      "Total written (filtered):     28,699,312 bp (93.4%)\n",
      "  Read 1:    14,335,536 bp\n",
      "  Read 2:    14,363,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52097 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t316\t0.0\t1\t0 309 7\n",
      "20\t51644\t0.0\t2\t50788 851 5\n",
      "21\t95\t0.0\t2\t0 93 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51340 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t248\t0.0\t1\t0 248\n",
      "19\t50993\t0.0\t1\t48749 2244\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S893_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S893_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S893_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S893_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S893_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S893_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S893_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S893_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,977 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.05 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,977\n",
      "  Read 1 with adapter:                  43,954 (99.9%)\n",
      "  Read 2 with adapter:                  43,661 (99.3%)\n",
      "Pairs written (passing filters):        43,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,945,556 bp\n",
      "  Read 1:    12,973,076 bp\n",
      "  Read 2:    12,972,480 bp\n",
      "Total written (filtered):     24,237,141 bp (93.4%)\n",
      "  Read 1:    12,094,081 bp\n",
      "  Read 2:    12,143,060 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t224\t0.0\t1\t0 218 6\n",
      "20\t43577\t0.0\t2\t42911 661 5\n",
      "21\t103\t0.0\t2\t0 99 4\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t43358\t0.0\t1\t41243 2115\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S894_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S894_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S894_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S894_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S894_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S894_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S894_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S894_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,656 reads  @     71.9 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.07 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,656\n",
      "  Read 1 with adapter:                  42,632 (99.9%)\n",
      "  Read 2 with adapter:                  42,336 (99.2%)\n",
      "Pairs written (passing filters):        42,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,165,464 bp\n",
      "  Read 1:    12,582,642 bp\n",
      "  Read 2:    12,582,822 bp\n",
      "Total written (filtered):     23,508,675 bp (93.4%)\n",
      "  Read 1:    11,730,133 bp\n",
      "  Read 2:    11,778,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t42246\t0.0\t2\t41611 627 8\n",
      "21\t105\t0.0\t2\t0 104 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t42074\t0.0\t1\t39742 2332\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S895_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S895_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S895_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S895_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S895_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S895_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S895_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S895_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,959 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.16 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,959\n",
      "  Read 1 with adapter:                  61,920 (99.9%)\n",
      "  Read 2 with adapter:                  61,233 (98.8%)\n",
      "Pairs written (passing filters):        61,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,552,341 bp\n",
      "  Read 1:    18,276,018 bp\n",
      "  Read 2:    18,276,323 bp\n",
      "Total written (filtered):     34,150,865 bp (93.4%)\n",
      "  Read 1:    17,037,754 bp\n",
      "  Read 2:    17,113,111 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t319\t0.0\t1\t0 316 3\n",
      "20\t61409\t0.0\t2\t60441 953 15\n",
      "21\t141\t0.0\t2\t0 140 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t324\t0.0\t1\t0 324\n",
      "19\t60800\t0.0\t1\t56916 3884\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S896_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S896_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S896_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S896_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S896_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S896_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S896_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S896_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,231 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.34 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,231\n",
      "  Read 1 with adapter:                  43,200 (99.9%)\n",
      "  Read 2 with adapter:                  42,942 (99.3%)\n",
      "Pairs written (passing filters):        43,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,506,096 bp\n",
      "  Read 1:    12,753,060 bp\n",
      "  Read 2:    12,753,036 bp\n",
      "Total written (filtered):     23,826,513 bp (93.4%)\n",
      "  Read 1:    11,889,180 bp\n",
      "  Read 2:    11,937,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t244\t0.0\t1\t0 240 4\n",
      "20\t42814\t0.0\t2\t42101 701 12\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t283\t0.0\t1\t0 283\n",
      "19\t42571\t0.0\t1\t40219 2352\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S897_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S897_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S897_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S897_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S897_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S897_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S897_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S897_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,464 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.63 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,464\n",
      "  Read 1 with adapter:                  46,428 (99.9%)\n",
      "  Read 2 with adapter:                  46,152 (99.3%)\n",
      "Pairs written (passing filters):        46,464 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,410,724 bp\n",
      "  Read 1:    13,706,003 bp\n",
      "  Read 2:    13,704,721 bp\n",
      "Total written (filtered):     25,605,533 bp (93.4%)\n",
      "  Read 1:    12,777,581 bp\n",
      "  Read 2:    12,827,952 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t261\t0.0\t1\t0 255 6\n",
      "20\t46014\t0.0\t2\t45248 759 7\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t45851\t0.0\t1\t43280 2571\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S898_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S898_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S898_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S898_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S898_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S898_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S898_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S898_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,994 reads  @     78.8 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.33 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,994\n",
      "  Read 1 with adapter:                  54,956 (99.9%)\n",
      "  Read 2 with adapter:                  54,569 (99.2%)\n",
      "Pairs written (passing filters):        54,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,435,677 bp\n",
      "  Read 1:    16,222,260 bp\n",
      "  Read 2:    16,213,417 bp\n",
      "Total written (filtered):     30,300,081 bp (93.4%)\n",
      "  Read 1:    15,123,317 bp\n",
      "  Read 2:    15,176,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t327\t0.0\t1\t0 324 3\n",
      "20\t54460\t0.0\t2\t53559 893 8\n",
      "21\t120\t0.0\t2\t0 115 5\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t54211\t0.0\t1\t51350 2861\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S899_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S899_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S899_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S899_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S899_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S899_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S899_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S899_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,325 reads  @     78.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.32 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,325\n",
      "  Read 1 with adapter:                  42,295 (99.9%)\n",
      "  Read 2 with adapter:                  42,096 (99.5%)\n",
      "Pairs written (passing filters):        42,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,960,966 bp\n",
      "  Read 1:    12,485,548 bp\n",
      "  Read 2:    12,475,418 bp\n",
      "Total written (filtered):     23,315,186 bp (93.4%)\n",
      "  Read 1:    11,639,482 bp\n",
      "  Read 2:    11,675,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42295 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t223\t0.0\t1\t0 219 4\n",
      "20\t41928\t0.0\t2\t41254 665 9\n",
      "21\t114\t0.0\t2\t0 113 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t41824\t0.0\t1\t39773 2051\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S900_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S900_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S900_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S900_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S900_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S900_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S900_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S900_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,679 reads  @     78.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.05 s (78 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,679\n",
      "  Read 1 with adapter:                  51,645 (99.9%)\n",
      "  Read 2 with adapter:                  50,637 (98.0%)\n",
      "Pairs written (passing filters):        51,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,435,406 bp\n",
      "  Read 1:    15,244,441 bp\n",
      "  Read 2:    15,190,965 bp\n",
      "Total written (filtered):     28,440,678 bp (93.4%)\n",
      "  Read 1:    14,211,670 bp\n",
      "  Read 2:    14,229,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t260\t0.0\t1\t0 254 6\n",
      "20\t51224\t0.0\t2\t50326 880 18\n",
      "21\t123\t0.0\t2\t0 120 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t50347\t0.0\t1\t48063 2284\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S901_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S901_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S901_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S901_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S901_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S901_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S901_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S901_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           178 reads  @    239.3 µs/read;   0.25 M reads/minute\n",
      "Finished in 0.05 s (258 us/read; 0.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                178\n",
      "  Read 1 with adapter:                     178 (100.0%)\n",
      "  Read 2 with adapter:                      13 (7.3%)\n",
      "Pairs written (passing filters):           178 (100.0%)\n",
      "\n",
      "Total basepairs processed:        60,423 bp\n",
      "  Read 1:        52,510 bp\n",
      "  Read 2:         7,913 bp\n",
      "Total written (filtered):         56,603 bp (93.7%)\n",
      "  Read 1:        48,950 bp\n",
      "  Read 2:         7,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 178 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t176\t0.0\t2\t175 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S902_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S902_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S902_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S902_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S902_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S902_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S902_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S902_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,957 reads  @     74.5 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.02 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,957\n",
      "  Read 1 with adapter:                  53,928 (99.9%)\n",
      "  Read 2 with adapter:                  53,666 (99.5%)\n",
      "Pairs written (passing filters):        53,957 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,832,354 bp\n",
      "  Read 1:    15,917,037 bp\n",
      "  Read 2:    15,915,317 bp\n",
      "Total written (filtered):     29,734,528 bp (93.4%)\n",
      "  Read 1:    14,838,673 bp\n",
      "  Read 2:    14,895,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t334\t0.0\t1\t0 332 2\n",
      "20\t53449\t0.0\t2\t52267 1163 19\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t323\t0.0\t1\t0 323\n",
      "19\t53212\t0.0\t1\t50658 2554\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S903_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S903_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S903_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S903_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S903_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S903_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S903_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S903_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,017 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 0.88 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,017\n",
      "  Read 1 with adapter:                  11,013 (100.0%)\n",
      "  Read 2 with adapter:                  10,855 (98.5%)\n",
      "Pairs written (passing filters):        11,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,499,798 bp\n",
      "  Read 1:     3,249,788 bp\n",
      "  Read 2:     3,250,010 bp\n",
      "Total written (filtered):      6,073,336 bp (93.4%)\n",
      "  Read 1:     3,029,532 bp\n",
      "  Read 2:     3,043,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 11013 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t63\t0.0\t1\t0 62 1\n",
      "20\t10907\t0.0\t2\t10601 303 3\n",
      "21\t15\t0.0\t2\t0 15\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 10855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t69\t0.0\t1\t0 69\n",
      "19\t10756\t0.0\t1\t9999 757\n",
      "20\t30\t0.0\t1\t0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S904_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S904_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S904_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S904_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S904_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S904_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S904_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S904_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,184 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.07 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,184\n",
      "  Read 1 with adapter:                  41,150 (99.9%)\n",
      "  Read 2 with adapter:                  40,988 (99.5%)\n",
      "Pairs written (passing filters):        41,184 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,297,843 bp\n",
      "  Read 1:    12,149,231 bp\n",
      "  Read 2:    12,148,612 bp\n",
      "Total written (filtered):     22,696,352 bp (93.4%)\n",
      "  Read 1:    11,326,381 bp\n",
      "  Read 2:    11,369,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41150 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t276\t0.0\t1\t0 272 4\n",
      "20\t40744\t0.0\t2\t40025 708 11\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t199\t0.0\t1\t0 199\n",
      "19\t40721\t0.0\t1\t38881 1840\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S905_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S905_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S905_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S905_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S905_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S905_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S905_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S905_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,654 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.63 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,654\n",
      "  Read 1 with adapter:                  49,628 (99.9%)\n",
      "  Read 2 with adapter:                  49,423 (99.5%)\n",
      "Pairs written (passing filters):        49,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,295,376 bp\n",
      "  Read 1:    14,647,907 bp\n",
      "  Read 2:    14,647,469 bp\n",
      "Total written (filtered):     27,364,093 bp (93.4%)\n",
      "  Read 1:    13,655,534 bp\n",
      "  Read 2:    13,708,559 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t333\t0.0\t1\t0 326 7\n",
      "20\t49130\t0.0\t2\t48228 890 12\n",
      "21\t112\t0.0\t2\t0 112\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t49120\t0.0\t1\t46853 2267\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S906_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S906_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S906_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S906_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S906_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S906_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S906_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S906_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,862 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.39 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,862\n",
      "  Read 1 with adapter:                  47,835 (99.9%)\n",
      "  Read 2 with adapter:                  47,645 (99.5%)\n",
      "Pairs written (passing filters):        47,862 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,237,752 bp\n",
      "  Read 1:    14,119,243 bp\n",
      "  Read 2:    14,118,509 bp\n",
      "Total written (filtered):     26,376,133 bp (93.4%)\n",
      "  Read 1:    13,162,744 bp\n",
      "  Read 2:    13,213,389 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t322\t0.0\t1\t0 319 3\n",
      "20\t47365\t0.0\t2\t46531 821 13\n",
      "21\t107\t0.0\t2\t0 103 4\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t47314\t0.0\t1\t45074 2240\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S907_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S907_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S907_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S907_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S907_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S907_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S907_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S907_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        68,049 reads  @     73.5 µs/read;   0.82 M reads/minute\n",
      "Finished in 5.01 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,049\n",
      "  Read 1 with adapter:                  68,011 (99.9%)\n",
      "  Read 2 with adapter:                  67,738 (99.5%)\n",
      "Pairs written (passing filters):        68,049 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,148,612 bp\n",
      "  Read 1:    20,074,206 bp\n",
      "  Read 2:    20,074,406 bp\n",
      "Total written (filtered):     37,501,867 bp (93.4%)\n",
      "  Read 1:    18,714,240 bp\n",
      "  Read 2:    18,787,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 68011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t459\t0.0\t1\t0 456 3\n",
      "20\t67346\t0.0\t2\t66188 1135 23\n",
      "21\t147\t0.0\t2\t0 146 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t386\t0.0\t1\t0 386\n",
      "19\t67209\t0.0\t1\t64052 3157\n",
      "20\t143\t0.0\t1\t0 143\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S908_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S908_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S908_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S908_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S908_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S908_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S908_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S908_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,397 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.39 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,397\n",
      "  Read 1 with adapter:                  50,363 (99.9%)\n",
      "  Read 2 with adapter:                  50,142 (99.5%)\n",
      "Pairs written (passing filters):        50,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,733,428 bp\n",
      "  Read 1:    14,867,102 bp\n",
      "  Read 2:    14,866,326 bp\n",
      "Total written (filtered):     27,773,875 bp (93.4%)\n",
      "  Read 1:    13,860,094 bp\n",
      "  Read 2:    13,913,781 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t367\t0.0\t1\t0 358 9\n",
      "20\t49839\t0.0\t2\t48968 851 20\n",
      "21\t95\t0.0\t2\t0 90 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t254\t0.0\t1\t0 254\n",
      "19\t49787\t0.0\t1\t47501 2286\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S909_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S909_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S909_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S909_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S909_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S909_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S909_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S909_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        93,108 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 6.18 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,108\n",
      "  Read 1 with adapter:                  93,040 (99.9%)\n",
      "  Read 2 with adapter:                  92,680 (99.5%)\n",
      "Pairs written (passing filters):        93,107 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,916,871 bp\n",
      "  Read 1:    27,466,829 bp\n",
      "  Read 2:    27,450,042 bp\n",
      "Total written (filtered):     51,295,441 bp (93.4%)\n",
      "  Read 1:    25,606,089 bp\n",
      "  Read 2:    25,689,352 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t617\t0.0\t1\t0 610 7\n",
      "20\t92105\t0.0\t2\t90440 1634 31\n",
      "21\t214\t0.0\t2\t0 212 2\n",
      "22\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 92680 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t405\t0.0\t1\t0 405\n",
      "19\t92100\t0.0\t1\t88042 4058\n",
      "20\t175\t0.0\t1\t0 175\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S910_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S910_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S910_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S910_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S910_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S910_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S910_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S910_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,178 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.84 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,178\n",
      "  Read 1 with adapter:                  42,144 (99.9%)\n",
      "  Read 2 with adapter:                  42,017 (99.6%)\n",
      "Pairs written (passing filters):        42,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,877,252 bp\n",
      "  Read 1:    12,442,489 bp\n",
      "  Read 2:    12,434,763 bp\n",
      "Total written (filtered):     23,236,333 bp (93.4%)\n",
      "  Read 1:    11,599,775 bp\n",
      "  Read 2:    11,636,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42144 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t271\t0.0\t1\t0 263 8\n",
      "20\t41761\t0.0\t2\t40996 757 8\n",
      "21\t79\t0.0\t2\t0 78 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t41731\t0.0\t1\t39989 1742\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S911_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S911_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S911_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S911_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S911_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S911_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S911_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S911_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,298 reads  @     72.4 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.72 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,298\n",
      "  Read 1 with adapter:                  51,272 (99.9%)\n",
      "  Read 2 with adapter:                  50,655 (98.7%)\n",
      "Pairs written (passing filters):        51,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,230,895 bp\n",
      "  Read 1:    15,132,897 bp\n",
      "  Read 2:    15,097,998 bp\n",
      "Total written (filtered):     28,243,303 bp (93.4%)\n",
      "  Read 1:    14,107,630 bp\n",
      "  Read 2:    14,135,673 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51272 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t324\t0.0\t1\t0 320 4\n",
      "20\t50785\t0.0\t2\t49877 895 13\n",
      "21\t119\t0.0\t2\t0 116 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t207\t0.0\t1\t0 207\n",
      "19\t50361\t0.0\t1\t48284 2077\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S912_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S912_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S912_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S912_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S912_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S912_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S912_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S912_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,105 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 5.90 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,105\n",
      "  Read 1 with adapter:                  80,064 (99.9%)\n",
      "  Read 2 with adapter:                  79,491 (99.2%)\n",
      "Pairs written (passing filters):        80,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,260,029 bp\n",
      "  Read 1:    23,630,249 bp\n",
      "  Read 2:    23,629,780 bp\n",
      "Total written (filtered):     44,149,186 bp (93.4%)\n",
      "  Read 1:    22,029,515 bp\n",
      "  Read 2:    22,119,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t721\t0.0\t1\t0 713 8\n",
      "20\t79144\t0.0\t2\t77842 1283 19\n",
      "21\t135\t0.0\t2\t0 133 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t375\t0.0\t1\t0 375\n",
      "19\t78961\t0.0\t1\t74799 4162\n",
      "20\t155\t0.0\t1\t0 155\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S913_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S913_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S913_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S913_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S913_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S913_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S913_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S913_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,954 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.51 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,954\n",
      "  Read 1 with adapter:                  46,929 (99.9%)\n",
      "  Read 2 with adapter:                  46,497 (99.0%)\n",
      "Pairs written (passing filters):        46,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,701,141 bp\n",
      "  Read 1:    13,850,918 bp\n",
      "  Read 2:    13,850,223 bp\n",
      "Total written (filtered):     25,879,597 bp (93.4%)\n",
      "  Read 1:    12,912,653 bp\n",
      "  Read 2:    12,966,944 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t430\t0.0\t1\t0 424 6\n",
      "20\t46386\t0.0\t2\t45630 741 15\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46497 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t46191\t0.0\t1\t43341 2850\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S914_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S914_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S914_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S914_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S914_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S914_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S914_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S914_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,452 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.51 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,452\n",
      "  Read 1 with adapter:                  48,430 (100.0%)\n",
      "  Read 2 with adapter:                  48,018 (99.1%)\n",
      "Pairs written (passing filters):        48,452 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,582,760 bp\n",
      "  Read 1:    14,292,892 bp\n",
      "  Read 2:    14,289,868 bp\n",
      "Total written (filtered):     26,702,250 bp (93.4%)\n",
      "  Read 1:    13,324,599 bp\n",
      "  Read 2:    13,377,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t406\t0.0\t1\t0 402 4\n",
      "20\t47920\t0.0\t2\t47104 808 8\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48018 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t47719\t0.0\t1\t44921 2798\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S915_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S915_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S915_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S915_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S915_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S915_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S915_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S915_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,816 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.07 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,816\n",
      "  Read 1 with adapter:                  44,806 (100.0%)\n",
      "  Read 2 with adapter:                  44,445 (99.2%)\n",
      "Pairs written (passing filters):        44,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,441,406 bp\n",
      "  Read 1:    13,220,713 bp\n",
      "  Read 2:    13,220,693 bp\n",
      "Total written (filtered):     24,701,330 bp (93.4%)\n",
      "  Read 1:    12,324,919 bp\n",
      "  Read 2:    12,376,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t409\t0.0\t1\t0 403 6\n",
      "20\t44283\t0.0\t2\t43530 748 5\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t44098\t0.0\t1\t41447 2651\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S916_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S916_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S916_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S916_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S916_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S916_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S916_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S916_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,887 reads  @     72.5 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.84 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,887\n",
      "  Read 1 with adapter:                  52,857 (99.9%)\n",
      "  Read 2 with adapter:                  52,403 (99.1%)\n",
      "Pairs written (passing filters):        52,886 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,200,445 bp\n",
      "  Read 1:    15,600,763 bp\n",
      "  Read 2:    15,599,682 bp\n",
      "Total written (filtered):     29,147,942 bp (93.4%)\n",
      "  Read 1:    14,543,749 bp\n",
      "  Read 2:    14,604,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52857 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t511\t0.0\t1\t0 503 8\n",
      "20\t52186\t0.0\t2\t51367 809 10\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t259\t0.0\t1\t0 259\n",
      "19\t52053\t0.0\t1\t48660 3393\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S917_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S917_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S917_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S917_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S917_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S917_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S917_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S917_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        84,986 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 6.11 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,986\n",
      "  Read 1 with adapter:                  84,939 (99.9%)\n",
      "  Read 2 with adapter:                  83,829 (98.6%)\n",
      "Pairs written (passing filters):        84,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,113,912 bp\n",
      "  Read 1:    25,069,709 bp\n",
      "  Read 2:    25,044,203 bp\n",
      "Total written (filtered):     46,823,196 bp (93.4%)\n",
      "  Read 1:    23,371,540 bp\n",
      "  Read 2:    23,451,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 84939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t806\t0.0\t1\t0 790 16\n",
      "20\t83906\t0.0\t2\t82406 1480 20\n",
      "21\t147\t0.0\t2\t0 146 1\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 83829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t361\t0.0\t1\t0 361\n",
      "19\t83311\t0.0\t1\t77729 5582\n",
      "20\t157\t0.0\t1\t0 157\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S918_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S918_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S918_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S918_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S918_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S918_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S918_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S918_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,497 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.56 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,497\n",
      "  Read 1 with adapter:                  52,470 (99.9%)\n",
      "  Read 2 with adapter:                  52,122 (99.3%)\n",
      "Pairs written (passing filters):        52,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,961,606 bp\n",
      "  Read 1:    15,486,424 bp\n",
      "  Read 2:    15,475,182 bp\n",
      "Total written (filtered):     28,922,418 bp (93.4%)\n",
      "  Read 1:    14,437,429 bp\n",
      "  Read 2:    14,484,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52470 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t492\t0.0\t1\t0 489 3\n",
      "20\t51834\t0.0\t2\t50900 918 16\n",
      "21\t101\t0.0\t2\t0 101\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t242\t0.0\t1\t0 242\n",
      "19\t51763\t0.0\t1\t49135 2628\n",
      "20\t117\t0.0\t1\t0 117\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S919_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S919_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S919_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S919_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S919_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S919_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S919_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S919_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        43,229 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.15 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,229\n",
      "  Read 1 with adapter:                  43,214 (100.0%)\n",
      "  Read 2 with adapter:                  42,188 (97.6%)\n",
      "Pairs written (passing filters):        43,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,453,323 bp\n",
      "  Read 1:    12,752,191 bp\n",
      "  Read 2:    12,701,132 bp\n",
      "Total written (filtered):     23,787,881 bp (93.5%)\n",
      "  Read 1:    11,888,207 bp\n",
      "  Read 2:    11,899,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t386\t0.0\t1\t0 383 3\n",
      "20\t42710\t0.0\t2\t41974 730 6\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42188 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t41936\t0.0\t1\t39815 2121\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S920_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S920_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S920_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S920_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S920_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S920_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S920_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S920_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,657 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.93 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,657\n",
      "  Read 1 with adapter:                  42,628 (99.9%)\n",
      "  Read 2 with adapter:                  42,473 (99.6%)\n",
      "Pairs written (passing filters):        42,657 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,166,367 bp\n",
      "  Read 1:    12,583,306 bp\n",
      "  Read 2:    12,583,061 bp\n",
      "Total written (filtered):     23,507,107 bp (93.4%)\n",
      "  Read 1:    11,730,938 bp\n",
      "  Read 2:    11,776,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t289\t0.0\t1\t0 285 4\n",
      "20\t42217\t0.0\t2\t41553 656 8\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t42222\t0.0\t1\t40286 1936\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S921_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S921_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S921_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S921_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S921_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S921_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S921_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S921_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,570 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.15 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,570\n",
      "  Read 1 with adapter:                  46,531 (99.9%)\n",
      "  Read 2 with adapter:                  46,354 (99.5%)\n",
      "Pairs written (passing filters):        46,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,474,945 bp\n",
      "  Read 1:    13,737,253 bp\n",
      "  Read 2:    13,737,692 bp\n",
      "Total written (filtered):     25,663,931 bp (93.4%)\n",
      "  Read 1:    12,806,821 bp\n",
      "  Read 2:    12,857,110 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t280\t0.0\t1\t0 274 6\n",
      "20\t46128\t0.0\t2\t45369 748 11\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46354 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t46056\t0.0\t1\t43934 2122\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S922_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S922_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S922_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S922_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S922_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S922_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S922_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S922_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,250 reads  @     66.0 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.92 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,250\n",
      "  Read 1 with adapter:                  44,213 (99.9%)\n",
      "  Read 2 with adapter:                  44,059 (99.6%)\n",
      "Pairs written (passing filters):        44,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,107,096 bp\n",
      "  Read 1:    13,053,394 bp\n",
      "  Read 2:    13,053,702 bp\n",
      "Total written (filtered):     24,386,049 bp (93.4%)\n",
      "  Read 1:    12,169,365 bp\n",
      "  Read 2:    12,216,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t320\t0.0\t1\t0 311 9\n",
      "20\t43781\t0.0\t2\t43018 752 11\n",
      "21\t71\t0.0\t2\t0 69 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t43776\t0.0\t1\t41741 2035\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S923_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S923_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S923_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S923_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S923_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S923_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S923_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S923_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,788 reads  @     66.8 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.59 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,788\n",
      "  Read 1 with adapter:                  38,768 (99.9%)\n",
      "  Read 2 with adapter:                  38,622 (99.6%)\n",
      "Pairs written (passing filters):        38,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,884,739 bp\n",
      "  Read 1:    11,442,394 bp\n",
      "  Read 2:    11,442,345 bp\n",
      "Total written (filtered):     21,375,903 bp (93.4%)\n",
      "  Read 1:    10,667,184 bp\n",
      "  Read 2:    10,708,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t231\t0.0\t1\t0 226 5\n",
      "20\t38429\t0.0\t2\t37779 645 5\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t265\t0.0\t1\t0 265\n",
      "19\t38284\t0.0\t1\t36542 1742\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S924_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S924_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S924_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S924_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S924_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S924_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S924_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S924_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,659 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.88 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,659\n",
      "  Read 1 with adapter:                  40,630 (99.9%)\n",
      "  Read 2 with adapter:                  40,493 (99.6%)\n",
      "Pairs written (passing filters):        40,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,987,753 bp\n",
      "  Read 1:    11,994,244 bp\n",
      "  Read 2:    11,993,509 bp\n",
      "Total written (filtered):     22,406,079 bp (93.4%)\n",
      "  Read 1:    11,181,806 bp\n",
      "  Read 2:    11,224,273 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t239\t0.0\t1\t0 235 4\n",
      "20\t40275\t0.0\t2\t39579 686 10\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t40218\t0.0\t1\t38419 1799\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S925_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S925_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S925_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S925_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S925_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S925_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S925_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S925_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        44,441 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.09 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,441\n",
      "  Read 1 with adapter:                  44,406 (99.9%)\n",
      "  Read 2 with adapter:                  44,215 (99.5%)\n",
      "Pairs written (passing filters):        44,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,214,200 bp\n",
      "  Read 1:    13,109,766 bp\n",
      "  Read 2:    13,104,434 bp\n",
      "Total written (filtered):     24,486,304 bp (93.4%)\n",
      "  Read 1:    12,221,830 bp\n",
      "  Read 2:    12,264,474 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t296\t0.0\t1\t0 290 6\n",
      "20\t43976\t0.0\t2\t43182 783 11\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t201\t0.0\t1\t0 201\n",
      "19\t43938\t0.0\t1\t42059 1879\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S926_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S926_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S926_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S926_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S926_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S926_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S926_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S926_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,807 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.41 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,807\n",
      "  Read 1 with adapter:                  47,763 (99.9%)\n",
      "  Read 2 with adapter:                  47,590 (99.5%)\n",
      "Pairs written (passing filters):        47,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,197,833 bp\n",
      "  Read 1:    14,102,439 bp\n",
      "  Read 2:    14,095,394 bp\n",
      "Total written (filtered):     26,338,743 bp (93.4%)\n",
      "  Read 1:    13,147,424 bp\n",
      "  Read 2:    13,191,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t319\t0.0\t1\t0 312 7\n",
      "20\t47312\t0.0\t2\t46432 860 20\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t47271\t0.0\t1\t45217 2054\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S927_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S927_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S927_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S927_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S927_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S927_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S927_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S927_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,117 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.38 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,117\n",
      "  Read 1 with adapter:                  47,090 (99.9%)\n",
      "  Read 2 with adapter:                  46,571 (98.8%)\n",
      "Pairs written (passing filters):        47,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,775,252 bp\n",
      "  Read 1:    13,899,242 bp\n",
      "  Read 2:    13,876,010 bp\n",
      "Total written (filtered):     25,948,927 bp (93.4%)\n",
      "  Read 1:    12,957,625 bp\n",
      "  Read 2:    12,991,302 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47090 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t298\t0.0\t1\t0 292 6\n",
      "20\t46650\t0.0\t2\t45836 796 18\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t46284\t0.0\t1\t44228 2056\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S928_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S928_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S928_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S928_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S928_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S928_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S928_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S928_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,229 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.37 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,229\n",
      "  Read 1 with adapter:                  44,205 (99.9%)\n",
      "  Read 2 with adapter:                  43,951 (99.4%)\n",
      "Pairs written (passing filters):        44,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,094,190 bp\n",
      "  Read 1:    13,047,347 bp\n",
      "  Read 2:    13,046,843 bp\n",
      "Total written (filtered):     24,375,257 bp (93.4%)\n",
      "  Read 1:    12,163,379 bp\n",
      "  Read 2:    12,211,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t241\t0.0\t1\t0 239 2\n",
      "20\t43842\t0.0\t2\t43041 786 15\n",
      "21\t75\t0.0\t2\t0 72 3\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t43633\t0.0\t1\t41521 2112\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S929_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S929_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S929_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S929_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S929_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S929_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S929_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S929_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,019 reads  @     67.1 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.89 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,019\n",
      "  Read 1 with adapter:                  42,994 (99.9%)\n",
      "  Read 2 with adapter:                  42,756 (99.4%)\n",
      "Pairs written (passing filters):        43,019 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,380,099 bp\n",
      "  Read 1:    12,690,329 bp\n",
      "  Read 2:    12,689,770 bp\n",
      "Total written (filtered):     23,708,107 bp (93.4%)\n",
      "  Read 1:    11,830,592 bp\n",
      "  Read 2:    11,877,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t274\t0.0\t1\t0 271 3\n",
      "20\t42576\t0.0\t2\t41903 667 6\n",
      "21\t101\t0.0\t2\t0 101\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t42487\t0.0\t1\t40423 2064\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S930_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S930_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S930_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S930_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S930_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S930_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S930_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S930_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,014 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.69 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,014\n",
      "  Read 1 with adapter:                  39,994 (100.0%)\n",
      "  Read 2 with adapter:                  39,776 (99.4%)\n",
      "Pairs written (passing filters):        40,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,607,471 bp\n",
      "  Read 1:    11,804,121 bp\n",
      "  Read 2:    11,803,350 bp\n",
      "Total written (filtered):     22,052,071 bp (93.4%)\n",
      "  Read 1:    11,004,346 bp\n",
      "  Read 2:    11,047,725 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t214\t0.0\t1\t0 208 6\n",
      "20\t39674\t0.0\t2\t39016 652 6\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t39517\t0.0\t1\t37600 1917\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S931_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S931_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S931_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S931_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S931_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S931_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S931_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S931_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        39,197 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.75 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,197\n",
      "  Read 1 with adapter:                  39,181 (100.0%)\n",
      "  Read 2 with adapter:                  38,974 (99.4%)\n",
      "Pairs written (passing filters):        39,197 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,125,737 bp\n",
      "  Read 1:    11,562,765 bp\n",
      "  Read 2:    11,562,972 bp\n",
      "Total written (filtered):     21,601,901 bp (93.4%)\n",
      "  Read 1:    10,779,261 bp\n",
      "  Read 2:    10,822,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t218\t0.0\t1\t0 214 4\n",
      "20\t38857\t0.0\t2\t38233 609 15\n",
      "21\t74\t0.0\t2\t0 72 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t254\t0.0\t1\t0 254\n",
      "19\t38640\t0.0\t1\t36615 2025\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S932_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S932_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S932_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S932_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S932_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S932_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S932_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S932_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,684 reads  @     71.0 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.74 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,684\n",
      "  Read 1 with adapter:                  66,651 (100.0%)\n",
      "  Read 2 with adapter:                  66,279 (99.4%)\n",
      "Pairs written (passing filters):        66,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,339,641 bp\n",
      "  Read 1:    19,671,456 bp\n",
      "  Read 2:    19,668,185 bp\n",
      "Total written (filtered):     36,747,494 bp (93.4%)\n",
      "  Read 1:    18,338,435 bp\n",
      "  Read 2:    18,409,059 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t406\t0.0\t1\t0 400 6\n",
      "20\t66065\t0.0\t2\t65002 1046 17\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t305\t0.0\t1\t0 305\n",
      "19\t65844\t0.0\t1\t62465 3379\n",
      "20\t130\t0.0\t1\t0 130\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S933_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S933_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S933_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S933_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S933_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S933_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S933_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S933_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,455 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 5.72 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,455\n",
      "  Read 1 with adapter:                  81,416 (100.0%)\n",
      "  Read 2 with adapter:                  80,823 (99.2%)\n",
      "Pairs written (passing filters):        81,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,036,073 bp\n",
      "  Read 1:    24,029,207 bp\n",
      "  Read 2:    24,006,866 bp\n",
      "Total written (filtered):     44,872,552 bp (93.4%)\n",
      "  Read 1:    22,401,151 bp\n",
      "  Read 2:    22,471,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t479\t0.0\t1\t0 473 6\n",
      "20\t80724\t0.0\t2\t79336 1364 24\n",
      "21\t131\t0.0\t2\t0 130 1\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t340\t0.0\t1\t0 340\n",
      "19\t80315\t0.0\t1\t76443 3872\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S934_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S934_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S934_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S934_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S934_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S934_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S934_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S934_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,407 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.87 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,407\n",
      "  Read 1 with adapter:                  42,376 (99.9%)\n",
      "  Read 2 with adapter:                  42,175 (99.5%)\n",
      "Pairs written (passing filters):        42,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,009,270 bp\n",
      "  Read 1:    12,509,622 bp\n",
      "  Read 2:    12,499,648 bp\n",
      "Total written (filtered):     23,360,358 bp (93.4%)\n",
      "  Read 1:    11,661,938 bp\n",
      "  Read 2:    11,698,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t211\t0.0\t1\t0 207 4\n",
      "20\t42051\t0.0\t2\t41344 698 9\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42175 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t182\t0.0\t1\t0 182\n",
      "19\t41908\t0.0\t1\t40004 1904\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S935_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S935_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S935_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S935_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S935_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S935_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S935_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S935_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,823 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.04 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,823\n",
      "  Read 1 with adapter:                  41,810 (100.0%)\n",
      "  Read 2 with adapter:                  41,083 (98.2%)\n",
      "Pairs written (passing filters):        41,823 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,638,835 bp\n",
      "  Read 1:    12,337,676 bp\n",
      "  Read 2:    12,301,159 bp\n",
      "Total written (filtered):     23,022,298 bp (93.4%)\n",
      "  Read 1:    11,501,591 bp\n",
      "  Read 2:    11,520,707 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t227\t0.0\t1\t0 227\n",
      "20\t41454\t0.0\t2\t40755 690 9\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41083 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t40830\t0.0\t1\t39029 1801\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S936_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S936_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S936_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S936_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S936_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S936_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S936_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S936_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,009 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.98 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,009\n",
      "  Read 1 with adapter:                  41,985 (99.9%)\n",
      "  Read 2 with adapter:                  41,670 (99.2%)\n",
      "Pairs written (passing filters):        42,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,780,180 bp\n",
      "  Read 1:    12,390,676 bp\n",
      "  Read 2:    12,389,504 bp\n",
      "Total written (filtered):     23,149,047 bp (93.4%)\n",
      "  Read 1:    11,551,142 bp\n",
      "  Read 2:    11,597,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t262\t0.0\t1\t0 259 3\n",
      "20\t41617\t0.0\t2\t40925 686 6\n",
      "21\t72\t0.0\t2\t0 72\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41670 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t41397\t0.0\t1\t39337 2060\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S937_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S937_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S937_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S937_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S937_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S937_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S937_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S937_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,022 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.30 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,022\n",
      "  Read 1 with adapter:                  47,982 (99.9%)\n",
      "  Read 2 with adapter:                  47,695 (99.3%)\n",
      "Pairs written (passing filters):        48,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,329,962 bp\n",
      "  Read 1:    14,164,124 bp\n",
      "  Read 2:    14,165,838 bp\n",
      "Total written (filtered):     26,464,447 bp (93.4%)\n",
      "  Read 1:    13,204,679 bp\n",
      "  Read 2:    13,259,768 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t301\t0.0\t1\t0 297 4\n",
      "20\t47545\t0.0\t2\t46734 797 14\n",
      "21\t90\t0.0\t2\t0 88 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47695 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t47392\t0.0\t1\t44709 2683\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S938_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S938_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S938_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S938_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S938_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S938_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S938_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S938_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,281 reads  @     73.5 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.77 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,281\n",
      "  Read 1 with adapter:                  51,247 (99.9%)\n",
      "  Read 2 with adapter:                  50,820 (99.1%)\n",
      "Pairs written (passing filters):        51,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,252,214 bp\n",
      "  Read 1:    15,126,470 bp\n",
      "  Read 2:    15,125,744 bp\n",
      "Total written (filtered):     28,262,011 bp (93.4%)\n",
      "  Read 1:    14,101,706 bp\n",
      "  Read 2:    14,160,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51247 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t341\t0.0\t1\t0 338 3\n",
      "20\t50744\t0.0\t2\t49880 855 9\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t239\t0.0\t1\t0 239\n",
      "19\t50483\t0.0\t1\t47689 2794\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S939_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S939_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S939_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S939_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S939_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S939_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S939_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S939_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,458 reads  @     78.5 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.57 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,458\n",
      "  Read 1 with adapter:                  45,411 (99.9%)\n",
      "  Read 2 with adapter:                  45,102 (99.2%)\n",
      "Pairs written (passing filters):        45,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,815,314 bp\n",
      "  Read 1:    13,405,607 bp\n",
      "  Read 2:    13,409,707 bp\n",
      "Total written (filtered):     25,050,481 bp (93.4%)\n",
      "  Read 1:    12,497,512 bp\n",
      "  Read 2:    12,552,969 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t263\t0.0\t1\t0 262 1\n",
      "20\t45011\t0.0\t2\t44270 730 11\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t300\t0.0\t1\t0 300\n",
      "19\t44702\t0.0\t1\t42158 2544\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S940_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S940_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S940_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S940_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S940_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S940_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S940_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S940_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        96,159 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 7.73 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,159\n",
      "  Read 1 with adapter:                  95,954 (99.8%)\n",
      "  Read 2 with adapter:                  95,388 (99.2%)\n",
      "Pairs written (passing filters):        96,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,691,814 bp\n",
      "  Read 1:    28,332,248 bp\n",
      "  Read 2:    28,359,566 bp\n",
      "Total written (filtered):     52,961,049 bp (93.4%)\n",
      "  Read 1:    26,413,542 bp\n",
      "  Read 2:    26,547,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 95954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t602\t0.0\t1\t0 590 12\n",
      "20\t95070\t0.0\t2\t93410 1640 20\n",
      "21\t180\t0.0\t2\t0 177 3\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 95388 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t482\t0.0\t1\t0 482\n",
      "19\t94737\t0.0\t1\t89177 5560\n",
      "20\t169\t0.0\t1\t0 169\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S941_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S941_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S941_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S941_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S941_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S941_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S941_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S941_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,796 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.14 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,796\n",
      "  Read 1 with adapter:                  54,739 (99.9%)\n",
      "  Read 2 with adapter:                  54,410 (99.3%)\n",
      "Pairs written (passing filters):        54,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,314,001 bp\n",
      "  Read 1:    16,159,165 bp\n",
      "  Read 2:    16,154,836 bp\n",
      "Total written (filtered):     30,185,776 bp (93.4%)\n",
      "  Read 1:    15,064,593 bp\n",
      "  Read 2:    15,121,183 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t342\t0.0\t1\t0 335 7\n",
      "20\t54243\t0.0\t2\t53369 862 12\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54410 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t54089\t0.0\t1\t51252 2837\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S942_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S942_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S942_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S942_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S942_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S942_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S942_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S942_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,630 reads  @     76.3 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.64 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,630\n",
      "  Read 1 with adapter:                  47,589 (99.9%)\n",
      "  Read 2 with adapter:                  47,330 (99.4%)\n",
      "Pairs written (passing filters):        47,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,091,541 bp\n",
      "  Read 1:    14,049,171 bp\n",
      "  Read 2:    14,042,370 bp\n",
      "Total written (filtered):     26,240,799 bp (93.4%)\n",
      "  Read 1:    13,097,585 bp\n",
      "  Read 2:    13,143,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t296\t0.0\t1\t0 289 7\n",
      "20\t47173\t0.0\t2\t46388 772 13\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47330 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t47026\t0.0\t1\t44711 2315\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S943_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S943_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S943_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S943_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S943_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S943_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S943_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S943_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        45,781 reads  @     82.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.80 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,781\n",
      "  Read 1 with adapter:                  45,749 (99.9%)\n",
      "  Read 2 with adapter:                  44,965 (98.2%)\n",
      "Pairs written (passing filters):        45,781 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,969,324 bp\n",
      "  Read 1:    13,503,763 bp\n",
      "  Read 2:    13,465,561 bp\n",
      "Total written (filtered):     25,200,296 bp (93.4%)\n",
      "  Read 1:    12,588,965 bp\n",
      "  Read 2:    12,611,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t302\t0.0\t1\t0 299 3\n",
      "20\t45300\t0.0\t2\t44524 766 10\n",
      "21\t98\t0.0\t2\t0 97 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t44688\t0.0\t1\t42677 2011\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S944_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S944_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S944_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S944_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S944_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S944_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S944_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S944_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,070 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.24 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,070\n",
      "  Read 1 with adapter:                  46,963 (99.8%)\n",
      "  Read 2 with adapter:                  46,675 (99.2%)\n",
      "Pairs written (passing filters):        47,070 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,759,605 bp\n",
      "  Read 1:    13,874,517 bp\n",
      "  Read 2:    13,885,088 bp\n",
      "Total written (filtered):     25,933,810 bp (93.4%)\n",
      "  Read 1:    12,935,405 bp\n",
      "  Read 2:    12,998,405 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46963 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t291\t0.0\t1\t0 281 10\n",
      "20\t46536\t0.0\t2\t45755 776 5\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t46353\t0.0\t1\t44000 2353\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S945_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S945_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S945_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S945_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S945_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S945_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S945_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S945_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,631 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.50 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,631\n",
      "  Read 1 with adapter:                  48,516 (99.8%)\n",
      "  Read 2 with adapter:                  48,184 (99.1%)\n",
      "Pairs written (passing filters):        48,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,680,117 bp\n",
      "  Read 1:    14,334,301 bp\n",
      "  Read 2:    14,345,816 bp\n",
      "Total written (filtered):     26,794,660 bp (93.4%)\n",
      "  Read 1:    13,364,185 bp\n",
      "  Read 2:    13,430,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t331\t0.0\t1\t0 326 5\n",
      "20\t48040\t0.0\t2\t47269 760 11\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t47879\t0.0\t1\t45175 2704\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S946_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S946_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S946_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S946_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S946_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S946_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S946_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S946_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,153 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.84 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,153\n",
      "  Read 1 with adapter:                  51,093 (99.9%)\n",
      "  Read 2 with adapter:                  50,647 (99.0%)\n",
      "Pairs written (passing filters):        51,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,171,467 bp\n",
      "  Read 1:    15,085,135 bp\n",
      "  Read 2:    15,086,332 bp\n",
      "Total written (filtered):     28,187,625 bp (93.4%)\n",
      "  Read 1:    14,063,468 bp\n",
      "  Read 2:    14,124,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t325\t0.0\t1\t0 319 6\n",
      "20\t50619\t0.0\t2\t49751 854 14\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t50333\t0.0\t1\t47550 2783\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S947_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S947_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S947_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S947_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S947_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S947_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S947_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S947_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,568 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.76 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,568\n",
      "  Read 1 with adapter:                  48,547 (100.0%)\n",
      "  Read 2 with adapter:                  48,142 (99.1%)\n",
      "Pairs written (passing filters):        48,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,654,310 bp\n",
      "  Read 1:    14,326,790 bp\n",
      "  Read 2:    14,327,520 bp\n",
      "Total written (filtered):     26,769,039 bp (93.4%)\n",
      "  Read 1:    13,356,036 bp\n",
      "  Read 2:    13,413,003 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t299\t0.0\t1\t0 295 4\n",
      "20\t48119\t0.0\t2\t47314 792 13\n",
      "21\t97\t0.0\t2\t0 95 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t297\t0.0\t1\t0 297\n",
      "19\t47729\t0.0\t1\t45043 2686\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S948_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S948_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S948_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S948_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S948_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S948_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S948_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S948_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,157 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.18 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,157\n",
      "  Read 1 with adapter:                  52,118 (99.9%)\n",
      "  Read 2 with adapter:                  51,657 (99.0%)\n",
      "Pairs written (passing filters):        52,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,767,094 bp\n",
      "  Read 1:    15,385,282 bp\n",
      "  Read 2:    15,381,812 bp\n",
      "Total written (filtered):     28,743,613 bp (93.4%)\n",
      "  Read 1:    14,343,113 bp\n",
      "  Read 2:    14,400,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t337\t0.0\t1\t0 334 3\n",
      "20\t51626\t0.0\t2\t50747 867 12\n",
      "21\t112\t0.0\t2\t0 109 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t271\t0.0\t1\t0 271\n",
      "19\t51286\t0.0\t1\t48284 3002\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S949_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S949_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S949_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S949_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S949_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S949_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S949_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S949_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        59,125 reads  @     79.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.70 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,125\n",
      "  Read 1 with adapter:                  59,078 (99.9%)\n",
      "  Read 2 with adapter:                  58,498 (98.9%)\n",
      "Pairs written (passing filters):        59,125 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,866,577 bp\n",
      "  Read 1:    17,440,599 bp\n",
      "  Read 2:    17,425,978 bp\n",
      "Total written (filtered):     32,573,972 bp (93.4%)\n",
      "  Read 1:    16,259,280 bp\n",
      "  Read 2:    16,314,692 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t414\t0.0\t1\t0 406 8\n",
      "20\t58483\t0.0\t2\t57397 1072 14\n",
      "21\t125\t0.0\t2\t0 119 6\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58498 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t278\t0.0\t1\t0 278\n",
      "19\t58118\t0.0\t1\t54863 3255\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S950_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S950_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S950_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S950_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S950_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S950_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S950_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S950_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,786 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.95 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,786\n",
      "  Read 1 with adapter:                  59,748 (99.9%)\n",
      "  Read 2 with adapter:                  59,422 (99.4%)\n",
      "Pairs written (passing filters):        59,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,257,595 bp\n",
      "  Read 1:    17,635,656 bp\n",
      "  Read 2:    17,621,939 bp\n",
      "Total written (filtered):     32,933,990 bp (93.4%)\n",
      "  Read 1:    16,440,910 bp\n",
      "  Read 2:    16,493,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t367\t0.0\t1\t0 363 4\n",
      "20\t59223\t0.0\t2\t58177 1028 18\n",
      "21\t119\t0.0\t2\t0 116 3\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59422 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t270\t0.0\t1\t0 270\n",
      "19\t59041\t0.0\t1\t56202 2839\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S951_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S951_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S951_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S951_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S951_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S951_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S951_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S951_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,308 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.02 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,308\n",
      "  Read 1 with adapter:                  49,275 (99.9%)\n",
      "  Read 2 with adapter:                  48,368 (98.1%)\n",
      "Pairs written (passing filters):        49,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,043,225 bp\n",
      "  Read 1:    14,545,409 bp\n",
      "  Read 2:    14,497,816 bp\n",
      "Total written (filtered):     27,139,097 bp (93.4%)\n",
      "  Read 1:    13,560,115 bp\n",
      "  Read 2:    13,578,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49275 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t322\t0.0\t1\t0 317 5\n",
      "20\t48816\t0.0\t2\t47913 892 11\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t48060\t0.0\t1\t45951 2109\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S952_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S952_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S952_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S952_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S952_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S952_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S952_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S952_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,287 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.06 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,287\n",
      "  Read 1 with adapter:                  40,264 (99.9%)\n",
      "  Read 2 with adapter:                  39,942 (99.1%)\n",
      "Pairs written (passing filters):        40,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,756,932 bp\n",
      "  Read 1:    11,884,360 bp\n",
      "  Read 2:    11,872,572 bp\n",
      "Total written (filtered):     22,192,762 bp (93.4%)\n",
      "  Read 1:    11,078,945 bp\n",
      "  Read 2:    11,113,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t240\t0.0\t1\t0 233 7\n",
      "20\t39910\t0.0\t2\t39126 778 6\n",
      "21\t76\t0.0\t2\t0 76\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t39679\t0.0\t1\t37485 2194\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S953_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S953_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S953_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S953_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S953_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S953_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S953_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S953_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           269 reads  @    209.3 µs/read;   0.29 M reads/minute\n",
      "Finished in 0.06 s (221 us/read; 0.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                269\n",
      "  Read 1 with adapter:                     269 (100.0%)\n",
      "  Read 2 with adapter:                      22 (8.2%)\n",
      "Pairs written (passing filters):           269 (100.0%)\n",
      "\n",
      "Total basepairs processed:        90,565 bp\n",
      "  Read 1:        79,355 bp\n",
      "  Read 2:        11,210 bp\n",
      "Total written (filtered):         84,746 bp (93.6%)\n",
      "  Read 1:        73,976 bp\n",
      "  Read 2:        10,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t268\t0.0\t2\t263 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t22\t0.0\t1\t0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S954_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S954_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S954_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S954_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S954_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S954_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S954_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S954_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        12,115 reads  @     84.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 1.02 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,115\n",
      "  Read 1 with adapter:                  12,102 (99.9%)\n",
      "  Read 2 with adapter:                  11,675 (96.4%)\n",
      "Pairs written (passing filters):        12,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,145,992 bp\n",
      "  Read 1:     3,573,456 bp\n",
      "  Read 2:     3,572,536 bp\n",
      "Total written (filtered):      6,682,198 bp (93.5%)\n",
      "  Read 1:     3,331,433 bp\n",
      "  Read 2:     3,350,765 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 12102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t94\t0.0\t1\t0 91 3\n",
      "20\t11950\t0.0\t2\t11675 271 4\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 11675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t85\t0.0\t1\t0 85\n",
      "19\t11559\t0.0\t1\t10378 1181\n",
      "20\t31\t0.0\t1\t0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S955_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S955_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S955_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S955_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S955_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S955_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S955_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S955_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           108 reads  @    420.8 µs/read;   0.14 M reads/minute\n",
      "Finished in 0.05 s (451 us/read; 0.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                108\n",
      "  Read 1 with adapter:                     108 (100.0%)\n",
      "  Read 2 with adapter:                      12 (11.1%)\n",
      "Pairs written (passing filters):           108 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,530 bp\n",
      "  Read 1:        31,860 bp\n",
      "  Read 2:         7,670 bp\n",
      "Total written (filtered):         37,133 bp (93.9%)\n",
      "  Read 1:        29,703 bp\n",
      "  Read 2:         7,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 108 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t106\t0.0\t2\t102 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t12\t0.0\t1\t0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S956_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S956_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S956_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S956_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S956_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S956_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S956_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S956_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,512 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.10 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,512\n",
      "  Read 1 with adapter:                  42,492 (100.0%)\n",
      "  Read 2 with adapter:                  42,266 (99.4%)\n",
      "Pairs written (passing filters):        42,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,076,476 bp\n",
      "  Read 1:    12,540,786 bp\n",
      "  Read 2:    12,535,690 bp\n",
      "Total written (filtered):     23,423,856 bp (93.4%)\n",
      "  Read 1:    11,691,116 bp\n",
      "  Read 2:    11,732,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t245\t0.0\t1\t0 239 6\n",
      "20\t42123\t0.0\t2\t41386 732 5\n",
      "21\t73\t0.0\t2\t0 71 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t42000\t0.0\t1\t40114 1886\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S957_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S957_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S957_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S957_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S957_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S957_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S957_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S957_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,095 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.97 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,095\n",
      "  Read 1 with adapter:                  43,080 (100.0%)\n",
      "  Read 2 with adapter:                  42,955 (99.7%)\n",
      "Pairs written (passing filters):        43,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,425,391 bp\n",
      "  Read 1:    12,712,434 bp\n",
      "  Read 2:    12,712,957 bp\n",
      "Total written (filtered):     23,747,917 bp (93.4%)\n",
      "  Read 1:    11,850,984 bp\n",
      "  Read 2:    11,896,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t266\t0.0\t1\t0 260 6\n",
      "20\t42680\t0.0\t2\t41937 732 11\n",
      "21\t92\t0.0\t2\t0 89 3\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42955 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t188\t0.0\t1\t0 188\n",
      "19\t42700\t0.0\t1\t40833 1867\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S958_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S958_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S958_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S958_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S958_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S958_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S958_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S958_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,486 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.09 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,486\n",
      "  Read 1 with adapter:                  45,457 (99.9%)\n",
      "  Read 2 with adapter:                  45,318 (99.6%)\n",
      "Pairs written (passing filters):        45,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,836,057 bp\n",
      "  Read 1:    13,418,146 bp\n",
      "  Read 2:    13,417,911 bp\n",
      "Total written (filtered):     25,066,167 bp (93.4%)\n",
      "  Read 1:    12,509,180 bp\n",
      "  Read 2:    12,556,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t265\t0.0\t1\t0 259 6\n",
      "20\t45078\t0.0\t2\t44308 757 13\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t224\t0.0\t1\t0 224\n",
      "19\t44988\t0.0\t1\t42969 2019\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S959_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S959_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S959_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S959_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S959_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S959_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S959_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S959_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00           101 reads  @    414.5 µs/read;   0.14 M reads/minute\n",
      "Finished in 0.05 s (446 us/read; 0.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                101\n",
      "  Read 1 with adapter:                     100 (99.0%)\n",
      "  Read 2 with adapter:                      21 (20.8%)\n",
      "Pairs written (passing filters):           101 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,118 bp\n",
      "  Read 1:        29,795 bp\n",
      "  Read 2:         9,323 bp\n",
      "Total written (filtered):         36,698 bp (93.8%)\n",
      "  Read 1:        27,795 bp\n",
      "  Read 2:         8,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t100\t0.0\t2\t98 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t21\t0.0\t1\t0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S960_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S960_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S960_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S960_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S960_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S960_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S960_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S960_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,617 reads  @     64.1 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.80 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,617\n",
      "  Read 1 with adapter:                  43,589 (99.9%)\n",
      "  Read 2 with adapter:                  43,436 (99.6%)\n",
      "Pairs written (passing filters):        43,617 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,730,171 bp\n",
      "  Read 1:    12,867,000 bp\n",
      "  Read 2:    12,863,171 bp\n",
      "Total written (filtered):     24,033,387 bp (93.4%)\n",
      "  Read 1:    11,995,381 bp\n",
      "  Read 2:    12,038,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t256\t0.0\t1\t0 254 2\n",
      "20\t43217\t0.0\t2\t42469 738 10\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t194\t0.0\t1\t0 194\n",
      "19\t43167\t0.0\t1\t41208 1959\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S961_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S961_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S961_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S961_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S961_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S961_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S961_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S961_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,752 reads  @     65.8 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.42 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,752\n",
      "  Read 1 with adapter:                  36,739 (100.0%)\n",
      "  Read 2 with adapter:                  36,626 (99.7%)\n",
      "Pairs written (passing filters):        36,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,683,624 bp\n",
      "  Read 1:    10,841,808 bp\n",
      "  Read 2:    10,841,816 bp\n",
      "Total written (filtered):     20,253,258 bp (93.4%)\n",
      "  Read 1:    10,107,211 bp\n",
      "  Read 2:    10,146,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t268\t0.0\t1\t0 262 6\n",
      "20\t36385\t0.0\t2\t35787 588 10\n",
      "21\t51\t0.0\t2\t0 50 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t36389\t0.0\t1\t34751 1638\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S962_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S962_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S962_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S962_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S962_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S962_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S962_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S962_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,138 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.24 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,138\n",
      "  Read 1 with adapter:                  47,115 (100.0%)\n",
      "  Read 2 with adapter:                  46,987 (99.7%)\n",
      "Pairs written (passing filters):        47,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,809,655 bp\n",
      "  Read 1:    13,905,500 bp\n",
      "  Read 2:    13,904,155 bp\n",
      "Total written (filtered):     25,974,904 bp (93.4%)\n",
      "  Read 1:    12,963,374 bp\n",
      "  Read 2:    13,011,530 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t295\t0.0\t1\t0 284 11\n",
      "20\t46687\t0.0\t2\t45929 751 7\n",
      "21\t85\t0.0\t2\t0 83 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t244\t0.0\t1\t0 244\n",
      "19\t46627\t0.0\t1\t44525 2102\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S963_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S963_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S963_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S963_221_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S963_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S963_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S963_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S963_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           113 reads  @    371.6 µs/read;   0.16 M reads/minute\n",
      "Finished in 0.05 s (400 us/read; 0.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                113\n",
      "  Read 1 with adapter:                     113 (100.0%)\n",
      "  Read 2 with adapter:                      10 (8.8%)\n",
      "Pairs written (passing filters):           113 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,019 bp\n",
      "  Read 1:        33,335 bp\n",
      "  Read 2:         7,684 bp\n",
      "Total written (filtered):         38,561 bp (94.0%)\n",
      "  Read 1:        31,077 bp\n",
      "  Read 2:         7,484 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t111\t0.0\t2\t111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t10\t0.0\t1\t0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S964_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S964_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S964_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S964_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S964_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S964_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S964_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S964_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,033 reads  @     81.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.62 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,033\n",
      "  Read 1 with adapter:                  32,012 (99.9%)\n",
      "  Read 2 with adapter:                  31,709 (99.0%)\n",
      "Pairs written (passing filters):        32,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,890,097 bp\n",
      "  Read 1:     9,449,722 bp\n",
      "  Read 2:     9,440,375 bp\n",
      "Total written (filtered):     17,647,561 bp (93.4%)\n",
      "  Read 1:     8,809,549 bp\n",
      "  Read 2:     8,838,012 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32012 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t154\t0.0\t1\t0 151 3\n",
      "20\t31777\t0.0\t2\t31247 523 7\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t31493\t0.0\t1\t29491 2002\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S965_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S965_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S965_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S965_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S965_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S965_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S965_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S965_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,888 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.43 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,888\n",
      "  Read 1 with adapter:                  47,856 (99.9%)\n",
      "  Read 2 with adapter:                  47,547 (99.3%)\n",
      "Pairs written (passing filters):        47,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,253,688 bp\n",
      "  Read 1:    14,126,827 bp\n",
      "  Read 2:    14,126,861 bp\n",
      "Total written (filtered):     26,393,496 bp (93.4%)\n",
      "  Read 1:    13,169,905 bp\n",
      "  Read 2:    13,223,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t288\t0.0\t1\t0 284 4\n",
      "20\t47458\t0.0\t2\t46571 874 13\n",
      "21\t78\t0.0\t2\t0 75 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t47238\t0.0\t1\t44501 2737\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S966_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S966_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S966_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S966_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S966_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S966_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S966_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S966_227_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,711 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.63 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,711\n",
      "  Read 1 with adapter:                  38,684 (99.9%)\n",
      "  Read 2 with adapter:                  38,471 (99.4%)\n",
      "Pairs written (passing filters):        38,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,838,579 bp\n",
      "  Read 1:    11,419,693 bp\n",
      "  Read 2:    11,418,886 bp\n",
      "Total written (filtered):     21,334,227 bp (93.4%)\n",
      "  Read 1:    10,646,160 bp\n",
      "  Read 2:    10,688,067 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38684 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t216\t0.0\t1\t0 215 1\n",
      "20\t38373\t0.0\t2\t37660 699 14\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t38161\t0.0\t1\t36009 2152\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S967_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S967_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S967_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S967_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S967_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S967_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S967_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S967_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           424 reads  @    157.2 µs/read;   0.38 M reads/minute\n",
      "Finished in 0.07 s (165 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                424\n",
      "  Read 1 with adapter:                     424 (100.0%)\n",
      "  Read 2 with adapter:                      28 (6.6%)\n",
      "Pairs written (passing filters):           424 (100.0%)\n",
      "\n",
      "Total basepairs processed:       141,664 bp\n",
      "  Read 1:       125,080 bp\n",
      "  Read 2:        16,584 bp\n",
      "Total written (filtered):        132,629 bp (93.6%)\n",
      "  Read 1:       116,605 bp\n",
      "  Read 2:        16,024 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t6\t0.0\t1\t0 6\n",
      "20\t417\t0.0\t2\t410 7\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S968_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S968_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S968_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S968_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S968_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S968_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S968_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S968_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,747 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.24 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,747\n",
      "  Read 1 with adapter:                  47,725 (100.0%)\n",
      "  Read 2 with adapter:                  47,384 (99.2%)\n",
      "Pairs written (passing filters):        47,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,162,566 bp\n",
      "  Read 1:    14,085,355 bp\n",
      "  Read 2:    14,077,211 bp\n",
      "Total written (filtered):     26,308,101 bp (93.4%)\n",
      "  Read 1:    13,131,054 bp\n",
      "  Read 2:    13,177,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t313\t0.0\t1\t0 308 5\n",
      "20\t47287\t0.0\t2\t46534 742 11\n",
      "21\t88\t0.0\t2\t0 85 3\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47384 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t47078\t0.0\t1\t44437 2641\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S969_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S969_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S969_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S969_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S969_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S969_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S969_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S969_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,659 reads  @     65.2 µs/read;   0.92 M reads/minute\n",
      "Finished in 5.33 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,659\n",
      "  Read 1 with adapter:                  81,623 (100.0%)\n",
      "  Read 2 with adapter:                  81,104 (99.3%)\n",
      "Pairs written (passing filters):        81,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,178,405 bp\n",
      "  Read 1:    24,089,117 bp\n",
      "  Read 2:    24,089,288 bp\n",
      "Total written (filtered):     45,005,525 bp (93.4%)\n",
      "  Read 1:    22,456,990 bp\n",
      "  Read 2:    22,548,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81623 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t525\t0.0\t1\t0 520 5\n",
      "20\t80886\t0.0\t2\t79508 1360 18\n",
      "21\t160\t0.0\t2\t0 155 5\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81104 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t358\t0.0\t1\t0 358\n",
      "19\t80611\t0.0\t1\t76106 4505\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S970_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S970_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S970_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S970_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S970_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S970_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S970_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S970_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,009 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.21 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,009\n",
      "  Read 1 with adapter:                  44,990 (100.0%)\n",
      "  Read 2 with adapter:                  44,584 (99.1%)\n",
      "Pairs written (passing filters):        45,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,553,091 bp\n",
      "  Read 1:    13,277,643 bp\n",
      "  Read 2:    13,275,448 bp\n",
      "Total written (filtered):     24,806,510 bp (93.4%)\n",
      "  Read 1:    12,378,052 bp\n",
      "  Read 2:    12,428,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t307\t0.0\t1\t0 301 6\n",
      "20\t44556\t0.0\t2\t43781 764 11\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44584 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t44250\t0.0\t1\t41670 2580\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S971_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S971_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S971_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S971_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S971_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S971_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S971_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S971_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           120 reads  @    355.6 µs/read;   0.17 M reads/minute\n",
      "Finished in 0.05 s (381 us/read; 0.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                120\n",
      "  Read 1 with adapter:                     120 (100.0%)\n",
      "  Read 2 with adapter:                       9 (7.5%)\n",
      "Pairs written (passing filters):           120 (100.0%)\n",
      "\n",
      "Total basepairs processed:        42,869 bp\n",
      "  Read 1:        35,400 bp\n",
      "  Read 2:         7,469 bp\n",
      "Total written (filtered):         40,289 bp (94.0%)\n",
      "  Read 1:        33,000 bp\n",
      "  Read 2:         7,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 120 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t120\t0.0\t2\t118 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t9\t0.0\t1\t0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S972_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S972_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S972_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S972_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S972_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S972_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S972_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S972_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,385 reads  @     65.7 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.13 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,385\n",
      "  Read 1 with adapter:                  32,367 (99.9%)\n",
      "  Read 2 with adapter:                  32,256 (99.6%)\n",
      "Pairs written (passing filters):        32,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,104,890 bp\n",
      "  Read 1:     9,553,562 bp\n",
      "  Read 2:     9,551,328 bp\n",
      "Total written (filtered):     17,844,920 bp (93.4%)\n",
      "  Read 1:     8,906,349 bp\n",
      "  Read 2:     8,938,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32367 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t187\t0.0\t1\t0 184 3\n",
      "20\t32111\t0.0\t2\t31594 513 4\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32256 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t32027\t0.0\t1\t30656 1371\n",
      "20\t61\t0.0\t1\t0 61\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S973_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S973_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S973_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S973_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S973_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S973_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S973_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S973_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,783 reads  @     62.5 µs/read;   0.96 M reads/minute\n",
      "Finished in 4.05 s (63 us/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,783\n",
      "  Read 1 with adapter:                  64,743 (99.9%)\n",
      "  Read 2 with adapter:                  64,602 (99.7%)\n",
      "Pairs written (passing filters):        64,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,221,696 bp\n",
      "  Read 1:    19,110,960 bp\n",
      "  Read 2:    19,110,736 bp\n",
      "Total written (filtered):     35,699,827 bp (93.4%)\n",
      "  Read 1:    17,816,346 bp\n",
      "  Read 2:    17,883,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64743 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t377\t0.0\t1\t0 372 5\n",
      "20\t64194\t0.0\t2\t63142 1034 18\n",
      "21\t117\t0.0\t2\t0 114 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t297\t0.0\t1\t0 297\n",
      "19\t64191\t0.0\t1\t61450 2741\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S974_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S974_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S974_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S974_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S974_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S974_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S974_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S974_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,499 reads  @     67.0 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.52 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,499\n",
      "  Read 1 with adapter:                  52,466 (99.9%)\n",
      "  Read 2 with adapter:                  52,318 (99.7%)\n",
      "Pairs written (passing filters):        52,499 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,973,971 bp\n",
      "  Read 1:    15,487,185 bp\n",
      "  Read 2:    15,486,786 bp\n",
      "Total written (filtered):     28,930,892 bp (93.4%)\n",
      "  Read 1:    14,438,025 bp\n",
      "  Read 2:    14,492,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t294\t0.0\t1\t0 290 4\n",
      "20\t52025\t0.0\t2\t51185 833 7\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t243\t0.0\t1\t0 243\n",
      "19\t51955\t0.0\t1\t49704 2251\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S975_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S975_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S975_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S975_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S975_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S975_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S975_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S975_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           167 reads  @    309.2 µs/read;   0.19 M reads/minute\n",
      "Finished in 0.05 s (329 us/read; 0.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                167\n",
      "  Read 1 with adapter:                     167 (100.0%)\n",
      "  Read 2 with adapter:                      35 (21.0%)\n",
      "Pairs written (passing filters):           167 (100.0%)\n",
      "\n",
      "Total basepairs processed:        64,457 bp\n",
      "  Read 1:        49,265 bp\n",
      "  Read 2:        15,192 bp\n",
      "Total written (filtered):         60,418 bp (93.7%)\n",
      "  Read 1:        45,926 bp\n",
      "  Read 2:        14,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t166\t0.0\t2\t164 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t35\t0.0\t1\t0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S976_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S976_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S976_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S976_247_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S976_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S976_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S976_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S976_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,647 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.89 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,647\n",
      "  Read 1 with adapter:                  55,604 (99.9%)\n",
      "  Read 2 with adapter:                  55,416 (99.6%)\n",
      "Pairs written (passing filters):        55,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,824,368 bp\n",
      "  Read 1:    16,415,448 bp\n",
      "  Read 2:    16,408,920 bp\n",
      "Total written (filtered):     30,659,617 bp (93.4%)\n",
      "  Read 1:    15,303,457 bp\n",
      "  Read 2:    15,356,160 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55604 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t270\t0.0\t1\t0 267 3\n",
      "20\t55128\t0.0\t2\t54207 907 14\n",
      "21\t151\t0.0\t2\t0 151\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t253\t0.0\t1\t0 253\n",
      "19\t55054\t0.0\t1\t52687 2367\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S977_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S977_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S977_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S977_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S977_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S977_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S977_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S977_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,654 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.05 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,654\n",
      "  Read 1 with adapter:                  44,626 (99.9%)\n",
      "  Read 2 with adapter:                  44,515 (99.7%)\n",
      "Pairs written (passing filters):        44,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,345,486 bp\n",
      "  Read 1:    13,172,904 bp\n",
      "  Read 2:    13,172,582 bp\n",
      "Total written (filtered):     24,607,423 bp (93.4%)\n",
      "  Read 1:    12,280,470 bp\n",
      "  Read 2:    12,326,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t234\t0.0\t1\t0 230 4\n",
      "20\t44234\t0.0\t2\t43532 693 9\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44515 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t44207\t0.0\t1\t42312 1895\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S978_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S978_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S978_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S978_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S978_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S978_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S978_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S978_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,457 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 5.61 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,457\n",
      "  Read 1 with adapter:                  80,423 (100.0%)\n",
      "  Read 2 with adapter:                  80,133 (99.6%)\n",
      "Pairs written (passing filters):        80,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,466,792 bp\n",
      "  Read 1:    23,734,790 bp\n",
      "  Read 2:    23,732,002 bp\n",
      "Total written (filtered):     44,336,239 bp (93.4%)\n",
      "  Read 1:    22,126,523 bp\n",
      "  Read 2:    22,209,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t440\t0.0\t1\t0 436 4\n",
      "20\t79701\t0.0\t2\t78326 1353 22\n",
      "21\t201\t0.0\t2\t0 201\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t423\t0.0\t1\t0 423\n",
      "19\t79528\t0.0\t1\t75871 3657\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S979_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S979_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S979_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S979_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S979_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S979_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S979_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S979_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           138 reads  @    321.9 µs/read;   0.19 M reads/minute\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.05 s (346 us/read; 0.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                138\n",
      "  Read 1 with adapter:                     138 (100.0%)\n",
      "  Read 2 with adapter:                      13 (9.4%)\n",
      "Pairs written (passing filters):           138 (100.0%)\n",
      "\n",
      "Total basepairs processed:        50,498 bp\n",
      "  Read 1:        40,710 bp\n",
      "  Read 2:         9,788 bp\n",
      "Total written (filtered):         47,478 bp (94.0%)\n",
      "  Read 1:        37,950 bp\n",
      "  Read 2:         9,528 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t136\t0.0\t2\t132 4\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S980_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S980_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S980_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S980_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S980_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S980_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S980_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S980_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,695 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.30 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,695\n",
      "  Read 1 with adapter:                  33,671 (99.9%)\n",
      "  Read 2 with adapter:                  33,490 (99.4%)\n",
      "Pairs written (passing filters):        33,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,874,930 bp\n",
      "  Read 1:     9,939,733 bp\n",
      "  Read 2:     9,935,197 bp\n",
      "Total written (filtered):     18,565,437 bp (93.4%)\n",
      "  Read 1:     9,266,454 bp\n",
      "  Read 2:     9,298,983 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33671 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t230\t0.0\t1\t0 223 7\n",
      "20\t33346\t0.0\t2\t32792 546 8\n",
      "21\t61\t0.0\t2\t0 59 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t155\t0.0\t1\t0 155\n",
      "19\t33276\t0.0\t1\t31651 1625\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S981_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S981_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S981_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S981_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S981_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S981_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S981_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S981_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,186 reads  @     65.4 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.04 s (66 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,186\n",
      "  Read 1 with adapter:                  31,167 (99.9%)\n",
      "  Read 2 with adapter:                  31,036 (99.5%)\n",
      "Pairs written (passing filters):        31,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,399,591 bp\n",
      "  Read 1:     9,199,858 bp\n",
      "  Read 2:     9,199,733 bp\n",
      "Total written (filtered):     17,186,751 bp (93.4%)\n",
      "  Read 1:     8,576,615 bp\n",
      "  Read 2:     8,610,136 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t197\t0.0\t1\t0 196 1\n",
      "20\t30858\t0.0\t2\t30391 457 10\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t141\t0.0\t1\t0 141\n",
      "19\t30841\t0.0\t1\t29453 1388\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S982_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S982_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S982_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S982_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S982_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S982_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S982_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S982_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,850 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.39 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,850\n",
      "  Read 1 with adapter:                  35,833 (100.0%)\n",
      "  Read 2 with adapter:                  35,645 (99.4%)\n",
      "Pairs written (passing filters):        35,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,151,269 bp\n",
      "  Read 1:    10,575,739 bp\n",
      "  Read 2:    10,575,530 bp\n",
      "Total written (filtered):     19,757,612 bp (93.4%)\n",
      "  Read 1:     9,859,224 bp\n",
      "  Read 2:     9,898,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35833 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t257\t0.0\t1\t0 251 6\n",
      "20\t35455\t0.0\t2\t34849 599 7\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t35374\t0.0\t1\t33631 1743\n",
      "20\t79\t0.0\t1\t0 79\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S983_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S983_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S983_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S983_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S983_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S983_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S983_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S983_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           387 reads  @    143.6 µs/read;   0.42 M reads/minute\n",
      "Finished in 0.06 s (151 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                387\n",
      "  Read 1 with adapter:                     387 (100.0%)\n",
      "  Read 2 with adapter:                      14 (3.6%)\n",
      "Pairs written (passing filters):           387 (100.0%)\n",
      "\n",
      "Total basepairs processed:       124,299 bp\n",
      "  Read 1:       114,165 bp\n",
      "  Read 2:        10,134 bp\n",
      "Total written (filtered):        116,280 bp (93.5%)\n",
      "  Read 1:       106,426 bp\n",
      "  Read 2:         9,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t381\t0.0\t2\t372 9\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t14\t0.0\t1\t0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S984_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S984_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S984_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S984_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S984_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S984_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S984_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S984_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,630 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.12 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,630\n",
      "  Read 1 with adapter:                  45,607 (99.9%)\n",
      "  Read 2 with adapter:                  45,390 (99.5%)\n",
      "Pairs written (passing filters):        45,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,913,800 bp\n",
      "  Read 1:    13,460,836 bp\n",
      "  Read 2:    13,452,964 bp\n",
      "Total written (filtered):     25,139,695 bp (93.4%)\n",
      "  Read 1:    12,549,028 bp\n",
      "  Read 2:    12,590,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t428\t0.0\t1\t0 423 5\n",
      "20\t45047\t0.0\t2\t44248 788 11\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45390 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t45113\t0.0\t1\t42938 2175\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S985_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S985_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S985_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S985_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S985_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S985_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S985_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S985_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,146 reads  @     64.1 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.64 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,146\n",
      "  Read 1 with adapter:                  41,129 (100.0%)\n",
      "  Read 2 with adapter:                  40,960 (99.5%)\n",
      "Pairs written (passing filters):        41,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,275,489 bp\n",
      "  Read 1:    12,138,056 bp\n",
      "  Read 2:    12,137,433 bp\n",
      "Total written (filtered):     22,675,054 bp (93.4%)\n",
      "  Read 1:    11,315,742 bp\n",
      "  Read 2:    11,359,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t385\t0.0\t1\t0 381 4\n",
      "20\t40624\t0.0\t2\t39934 677 13\n",
      "21\t89\t0.0\t2\t0 87 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t40683\t0.0\t1\t38696 1987\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S986_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S986_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S986_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S986_267_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S986_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S986_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S986_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S986_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,724 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.06 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,724\n",
      "  Read 1 with adapter:                  40,704 (100.0%)\n",
      "  Read 2 with adapter:                  40,434 (99.3%)\n",
      "Pairs written (passing filters):        40,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,025,633 bp\n",
      "  Read 1:    12,013,565 bp\n",
      "  Read 2:    12,012,068 bp\n",
      "Total written (filtered):     22,443,682 bp (93.4%)\n",
      "  Read 1:    11,199,749 bp\n",
      "  Read 2:    11,243,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t340\t0.0\t1\t0 338 2\n",
      "20\t40265\t0.0\t2\t39592 660 13\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t40125\t0.0\t1\t38082 2043\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S987_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S987_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S987_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S987_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S987_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S987_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S987_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S987_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            79 reads  @    527.6 µs/read;   0.11 M reads/minute\n",
      "Finished in 0.04 s (566 us/read; 0.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 79\n",
      "  Read 1 with adapter:                      78 (98.7%)\n",
      "  Read 2 with adapter:                      12 (15.2%)\n",
      "Pairs written (passing filters):            79 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,339 bp\n",
      "  Read 1:        23,305 bp\n",
      "  Read 2:         6,034 bp\n",
      "Total written (filtered):         27,540 bp (93.9%)\n",
      "  Read 1:        21,746 bp\n",
      "  Read 2:         5,794 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 78 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t77\t0.0\t2\t75 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t12\t0.0\t1\t0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S988_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S988_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S988_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S988_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S988_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S988_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S988_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S988_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,142 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.54 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,142\n",
      "  Read 1 with adapter:                  34,076 (99.8%)\n",
      "  Read 2 with adapter:                  33,859 (99.2%)\n",
      "Pairs written (passing filters):        34,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,140,186 bp\n",
      "  Read 1:    10,071,780 bp\n",
      "  Read 2:    10,068,406 bp\n",
      "Total written (filtered):     18,815,552 bp (93.4%)\n",
      "  Read 1:     9,390,375 bp\n",
      "  Read 2:     9,425,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t193\t0.0\t1\t0 191 2\n",
      "20\t33788\t0.0\t2\t33191 589 8\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t153\t0.0\t1\t0 153\n",
      "19\t33645\t0.0\t1\t31845 1800\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S989_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S989_273_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S989_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S989_273_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S989_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S989_273_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S989_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S989_273_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,965 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.94 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,965\n",
      "  Read 1 with adapter:                  52,902 (99.9%)\n",
      "  Read 2 with adapter:                  52,612 (99.3%)\n",
      "Pairs written (passing filters):        52,965 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,247,747 bp\n",
      "  Read 1:    15,623,457 bp\n",
      "  Read 2:    15,624,290 bp\n",
      "Total written (filtered):     29,190,470 bp (93.4%)\n",
      "  Read 1:    14,565,668 bp\n",
      "  Read 2:    14,624,802 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52902 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t347\t0.0\t1\t0 336 11\n",
      "20\t52395\t0.0\t2\t51470 915 10\n",
      "21\t104\t0.0\t2\t0 104\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52612 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t52310\t0.0\t1\t49481 2829\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S990_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S990_275_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S990_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S990_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S990_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S990_275_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S990_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S990_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,229 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.09 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,229\n",
      "  Read 1 with adapter:                  40,170 (99.9%)\n",
      "  Read 2 with adapter:                  39,991 (99.4%)\n",
      "Pairs written (passing filters):        40,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,733,390 bp\n",
      "  Read 1:    11,867,438 bp\n",
      "  Read 2:    11,865,952 bp\n",
      "Total written (filtered):     22,170,460 bp (93.4%)\n",
      "  Read 1:    11,064,242 bp\n",
      "  Read 2:    11,106,218 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40170 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t272\t0.0\t1\t0 268 4\n",
      "20\t39806\t0.0\t2\t39032 765 9\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t39668\t0.0\t1\t37696 1972\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S991_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S991_277_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S991_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S991_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S991_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S991_277_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S991_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S991_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           334 reads  @    176.9 µs/read;   0.34 M reads/minute\n",
      "Finished in 0.06 s (187 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                334\n",
      "  Read 1 with adapter:                     334 (100.0%)\n",
      "  Read 2 with adapter:                      28 (8.4%)\n",
      "Pairs written (passing filters):           333 (99.7%)\n",
      "\n",
      "Total basepairs processed:       115,836 bp\n",
      "  Read 1:        98,530 bp\n",
      "  Read 2:        17,306 bp\n",
      "Total written (filtered):        108,322 bp (93.5%)\n",
      "  Read 1:        91,576 bp\n",
      "  Read 2:        16,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t333\t0.0\t2\t327 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S992_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S992_279_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S992_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S992_279_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S992_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S992_279_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S992_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S992_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,286 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.10 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,286\n",
      "  Read 1 with adapter:                  54,259 (100.0%)\n",
      "  Read 2 with adapter:                  53,920 (99.3%)\n",
      "Pairs written (passing filters):        54,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,019,376 bp\n",
      "  Read 1:    16,014,208 bp\n",
      "  Read 2:    16,005,168 bp\n",
      "Total written (filtered):     29,910,078 bp (93.4%)\n",
      "  Read 1:    14,929,238 bp\n",
      "  Read 2:    14,980,840 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54259 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t353\t0.0\t1\t0 347 6\n",
      "20\t53750\t0.0\t2\t52755 988 7\n",
      "21\t109\t0.0\t2\t0 108 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t252\t0.0\t1\t0 252\n",
      "19\t53568\t0.0\t1\t50771 2797\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S993_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S993_281_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S993_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S993_281_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S993_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S993_281_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S993_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S993_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,093 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.42 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,093\n",
      "  Read 1 with adapter:                  46,073 (100.0%)\n",
      "  Read 2 with adapter:                  45,840 (99.5%)\n",
      "Pairs written (passing filters):        46,093 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,194,420 bp\n",
      "  Read 1:    13,597,055 bp\n",
      "  Read 2:    13,597,365 bp\n",
      "Total written (filtered):     25,402,243 bp (93.4%)\n",
      "  Read 1:    12,675,714 bp\n",
      "  Read 2:    12,726,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46073 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t45687\t0.0\t2\t44917 753 17\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45840 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t45556\t0.0\t1\t43150 2406\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S994_282_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S994_283_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S994_282_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S994_283_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S994_282_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S994_283_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S994_282_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S994_283_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,880 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.56 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,880\n",
      "  Read 1 with adapter:                  43,850 (99.9%)\n",
      "  Read 2 with adapter:                  43,590 (99.3%)\n",
      "Pairs written (passing filters):        43,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,885,306 bp\n",
      "  Read 1:    12,944,106 bp\n",
      "  Read 2:    12,941,200 bp\n",
      "Total written (filtered):     24,180,388 bp (93.4%)\n",
      "  Read 1:    12,067,285 bp\n",
      "  Read 2:    12,113,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t264\t0.0\t1\t0 260 4\n",
      "20\t43481\t0.0\t2\t42718 755 8\n",
      "21\t61\t0.0\t2\t0 61\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t43263\t0.0\t1\t40989 2274\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S995_284_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S995_285_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S995_284_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S995_285_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S995_284_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S995_285_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S995_284_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S995_285_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           112 reads  @    369.8 µs/read;   0.16 M reads/minute\n",
      "Finished in 0.04 s (401 us/read; 0.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                112\n",
      "  Read 1 with adapter:                     112 (100.0%)\n",
      "  Read 2 with adapter:                      15 (13.4%)\n",
      "Pairs written (passing filters):           112 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,661 bp\n",
      "  Read 1:        33,040 bp\n",
      "  Read 2:         8,621 bp\n",
      "Total written (filtered):         39,123 bp (93.9%)\n",
      "  Read 1:        30,802 bp\n",
      "  Read 2:         8,321 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t110\t0.0\t2\t108 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t15\t0.0\t1\t0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S996_286_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S996_287_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S996_286_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S996_287_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S996_286_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S996_287_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S996_286_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S996_287_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,220 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.66 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,220\n",
      "  Read 1 with adapter:                  32,186 (99.9%)\n",
      "  Read 2 with adapter:                  31,929 (99.1%)\n",
      "Pairs written (passing filters):        32,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,994,378 bp\n",
      "  Read 1:     9,501,314 bp\n",
      "  Read 2:     9,493,064 bp\n",
      "Total written (filtered):     17,743,925 bp (93.4%)\n",
      "  Read 1:     8,857,425 bp\n",
      "  Read 2:     8,886,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t190\t0.0\t1\t0 185 5\n",
      "20\t31901\t0.0\t2\t31348 541 12\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t145\t0.0\t1\t0 145\n",
      "19\t31726\t0.0\t1\t29687 2039\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S997_288_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S997_289_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S997_288_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S997_289_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S997_288_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S997_289_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S997_288_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S997_289_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        86,033 reads  @     74.7 µs/read;   0.80 M reads/minute\n",
      "Finished in 6.43 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,033\n",
      "  Read 1 with adapter:                  85,948 (99.9%)\n",
      "  Read 2 with adapter:                  85,427 (99.3%)\n",
      "Pairs written (passing filters):        86,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,748,133 bp\n",
      "  Read 1:    25,368,700 bp\n",
      "  Read 2:    25,379,433 bp\n",
      "Total written (filtered):     47,406,595 bp (93.4%)\n",
      "  Read 1:    23,650,091 bp\n",
      "  Read 2:    23,756,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 85948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t556\t0.0\t1\t0 545 11\n",
      "20\t85135\t0.0\t2\t83668 1447 20\n",
      "21\t177\t0.0\t2\t0 174 3\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85427 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t356\t0.0\t1\t0 356\n",
      "19\t84899\t0.0\t1\t80452 4447\n",
      "20\t172\t0.0\t1\t0 172\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S998_290_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S998_291_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S998_290_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S998_291_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S998_290_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S998_291_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S998_290_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S998_291_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,416 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.66 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,416\n",
      "  Read 1 with adapter:                  47,358 (99.9%)\n",
      "  Read 2 with adapter:                  47,077 (99.3%)\n",
      "Pairs written (passing filters):        47,416 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,965,229 bp\n",
      "  Read 1:    13,979,278 bp\n",
      "  Read 2:    13,985,951 bp\n",
      "Total written (filtered):     26,123,903 bp (93.4%)\n",
      "  Read 1:    13,032,285 bp\n",
      "  Read 2:    13,091,618 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t264\t0.0\t1\t0 261 3\n",
      "20\t46973\t0.0\t2\t46185 773 15\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t46753\t0.0\t1\t44288 2465\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S999_292_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S999_293_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S999_292_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S999_293_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S999_292_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-d80wmkbg/S999_293_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S999_292_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mq8t5xa/cda73272-6cd2-4bad-aea4-d19e70c1634a/data/S999_293_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           389 reads  @    161.1 µs/read;   0.37 M reads/minute\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.07 s (169 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                389\n",
      "  Read 1 with adapter:                     389 (100.0%)\n",
      "  Read 2 with adapter:                      40 (10.3%)\n",
      "Pairs written (passing filters):           389 (100.0%)\n",
      "\n",
      "Total basepairs processed:       134,804 bp\n",
      "  Read 1:       114,755 bp\n",
      "  Read 2:        20,049 bp\n",
      "Total written (filtered):        126,224 bp (93.6%)\n",
      "  Read 1:       106,975 bp\n",
      "  Read 2:        19,249 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 389 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t386\t0.0\t2\t378 8\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t40\t0.0\t1\t0 40\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: d-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: d-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-o1e6c533')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 27c274cf-bfa0-4eb2-bdf3-c12c1b7b92f2>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.d import the fastq files for library d into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/BB-Lib1_zhan_manifest_runbb_v2_grid.csv \\\n",
    "  --output-path d-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.d trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences d-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences d-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.d summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data d-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization d-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('d-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: d-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: d-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: d-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.d Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs d-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 256 \\\n",
    "  --p-trunc-len-r 184 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table d-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats d-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: d-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-3rot9j_q')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 2af53363-23a5-4c24-be60-1ef5367cf6ec>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.d view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table d-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization d-table.qzv \n",
    "q2.Visualization.load('d-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: d-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.d view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization d-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/Hirai-Lib1-manifest_18S.csv as PairedEndFastqManifestPhred33 to e-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1244_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1244_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1244_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1244_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1244_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1244_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1244_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1244_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        79,106 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 6.57 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,106\n",
      "  Read 1 with adapter:                  76,230 (96.4%)\n",
      "  Read 2 with adapter:                  75,789 (95.8%)\n",
      "Pairs written (passing filters):        79,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,617,525 bp\n",
      "  Read 1:    23,334,085 bp\n",
      "  Read 2:    23,283,440 bp\n",
      "Total written (filtered):     43,653,483 bp (93.6%)\n",
      "  Read 1:    21,809,795 bp\n",
      "  Read 2:    21,843,688 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 76230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t437\t0.0\t1\t0 431 6\n",
      "20\t75636\t0.0\t2\t74022 1600 14\n",
      "21\t119\t0.0\t2\t0 119\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75789 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t362\t0.0\t1\t0 362\n",
      "19\t75304\t0.0\t1\t73327 1977\n",
      "20\t123\t0.0\t1\t0 123\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1245_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1245_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1245_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1245_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1245_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1245_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1245_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1245_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,309 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.46 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,309\n",
      "  Read 1 with adapter:                  62,568 (93.0%)\n",
      "  Read 2 with adapter:                  62,268 (92.5%)\n",
      "Pairs written (passing filters):        67,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,259,388 bp\n",
      "  Read 1:    19,855,398 bp\n",
      "  Read 2:    19,403,990 bp\n",
      "Total written (filtered):     36,825,303 bp (93.8%)\n",
      "  Read 1:    18,604,281 bp\n",
      "  Read 2:    18,221,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t352\t0.0\t1\t0 341 11\n",
      "20\t62072\t0.0\t2\t60781 1283 8\n",
      "21\t95\t0.0\t2\t0 94 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t245\t0.0\t1\t0 245\n",
      "19\t61902\t0.0\t1\t60181 1721\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1246_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1246_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1246_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1246_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1246_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1246_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1246_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1246_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        78,227 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 6.01 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,227\n",
      "  Read 1 with adapter:                  77,992 (99.7%)\n",
      "  Read 2 with adapter:                  77,614 (99.2%)\n",
      "Pairs written (passing filters):        78,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,102,056 bp\n",
      "  Read 1:    23,075,899 bp\n",
      "  Read 2:    23,026,157 bp\n",
      "Total written (filtered):     43,067,471 bp (93.4%)\n",
      "  Read 1:    21,515,734 bp\n",
      "  Read 2:    21,551,737 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t438\t0.0\t1\t0 431 7\n",
      "20\t77375\t0.0\t2\t75764 1585 26\n",
      "21\t133\t0.0\t2\t0 133\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77614 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t362\t0.0\t1\t0 362\n",
      "19\t77136\t0.0\t1\t75210 1926\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1247_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1247_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1247_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1247_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1247_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1247_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1247_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1247_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       110,338 reads  @     83.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 9.25 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,338\n",
      "  Read 1 with adapter:                 110,287 (100.0%)\n",
      "  Read 2 with adapter:                 109,231 (99.0%)\n",
      "Pairs written (passing filters):       110,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,098,185 bp\n",
      "  Read 1:    32,549,342 bp\n",
      "  Read 2:    32,548,843 bp\n",
      "Total written (filtered):     60,817,985 bp (93.4%)\n",
      "  Read 1:    30,343,967 bp\n",
      "  Read 2:    30,474,018 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 110287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t747\t0.0\t1\t0 722 25\n",
      "20\t109198\t0.0\t2\t106473 2697 28\n",
      "21\t166\t0.0\t2\t0 161 5\n",
      "22\t142\t0.0\t2\t0 0 142\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t840\t0.0\t1\t0 840\n",
      "19\t108115\t0.0\t1\t103993 4122\n",
      "20\t276\t0.0\t1\t0 276\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1248_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1248_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1248_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1248_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1248_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1248_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1248_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1248_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,418 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 6.36 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,418\n",
      "  Read 1 with adapter:                  81,380 (100.0%)\n",
      "  Read 2 with adapter:                  81,043 (99.5%)\n",
      "Pairs written (passing filters):        81,416 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,028,209 bp\n",
      "  Read 1:    24,016,281 bp\n",
      "  Read 2:    24,011,928 bp\n",
      "Total written (filtered):     44,860,829 bp (93.4%)\n",
      "  Read 1:    22,388,441 bp\n",
      "  Read 2:    22,472,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t462\t0.0\t1\t0 452 10\n",
      "20\t80740\t0.0\t2\t79176 1544 20\n",
      "21\t116\t0.0\t2\t0 112 4\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81043 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t409\t0.0\t1\t0 409\n",
      "19\t80502\t0.0\t1\t77877 2625\n",
      "20\t132\t0.0\t1\t0 132\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1249_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1249_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1249_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1249_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1249_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1249_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1249_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1249_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,507 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 5.90 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,507\n",
      "  Read 1 with adapter:                  80,456 (99.9%)\n",
      "  Read 2 with adapter:                  80,087 (99.5%)\n",
      "Pairs written (passing filters):        80,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,485,849 bp\n",
      "  Read 1:    23,746,564 bp\n",
      "  Read 2:    23,739,285 bp\n",
      "Total written (filtered):     44,355,550 bp (93.4%)\n",
      "  Read 1:    22,137,703 bp\n",
      "  Read 2:    22,217,847 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t443\t0.0\t1\t0 435 8\n",
      "20\t79810\t0.0\t2\t78243 1545 22\n",
      "21\t146\t0.0\t2\t0 142 4\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t359\t0.0\t1\t0 359\n",
      "19\t79584\t0.0\t1\t77336 2248\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1250_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1250_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1250_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1250_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1250_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1250_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1250_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1250_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,851 reads  @     80.2 µs/read;   0.75 M reads/minute\n",
      "Finished in 6.57 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,851\n",
      "  Read 1 with adapter:                  81,817 (100.0%)\n",
      "  Read 2 with adapter:                  81,430 (99.5%)\n",
      "Pairs written (passing filters):        81,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,270,740 bp\n",
      "  Read 1:    24,145,121 bp\n",
      "  Read 2:    24,125,619 bp\n",
      "Total written (filtered):     45,087,787 bp (93.4%)\n",
      "  Read 1:    22,509,064 bp\n",
      "  Read 2:    22,578,723 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t463\t0.0\t1\t0 454 9\n",
      "20\t81175\t0.0\t2\t79126 2020 29\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t411\t0.0\t1\t0 411\n",
      "19\t80882\t0.0\t1\t78516 2366\n",
      "20\t137\t0.0\t1\t0 137\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1251_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1251_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1251_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1251_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1251_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1251_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1251_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1251_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        70,245 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.60 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,245\n",
      "  Read 1 with adapter:                  70,210 (100.0%)\n",
      "  Read 2 with adapter:                  68,893 (98.1%)\n",
      "Pairs written (passing filters):        70,245 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,382,136 bp\n",
      "  Read 1:    20,719,624 bp\n",
      "  Read 2:    20,662,512 bp\n",
      "Total written (filtered):     38,669,388 bp (93.4%)\n",
      "  Read 1:    19,315,648 bp\n",
      "  Read 2:    19,353,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t353\t0.0\t1\t0 349 4\n",
      "20\t69710\t0.0\t2\t68188 1500 22\n",
      "21\t97\t0.0\t2\t0 95 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 68893 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t319\t0.0\t1\t0 319\n",
      "19\t68450\t0.0\t1\t66940 1510\n",
      "20\t124\t0.0\t1\t0 124\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1252_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1252_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1252_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1252_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1252_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1252_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1252_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1252_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       120,936 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 10.42 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,936\n",
      "  Read 1 with adapter:                 120,860 (99.9%)\n",
      "  Read 2 with adapter:                 120,388 (99.5%)\n",
      "Pairs written (passing filters):       120,936 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,340,073 bp\n",
      "  Read 1:    35,668,296 bp\n",
      "  Read 2:    35,671,777 bp\n",
      "Total written (filtered):     66,636,310 bp (93.4%)\n",
      "  Read 1:    33,251,583 bp\n",
      "  Read 2:    33,384,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 120860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t732\t0.0\t1\t0 719 13\n",
      "20\t119869\t0.0\t2\t117397 2449 23\n",
      "21\t169\t0.0\t2\t0 165 4\n",
      "22\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 120388 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t579\t0.0\t1\t0 579\n",
      "19\t119552\t0.0\t1\t116704 2848\n",
      "20\t257\t0.0\t1\t0 257\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1253_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1253_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1253_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1253_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1253_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1253_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1253_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1253_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:06        71,323 reads  @     85.4 µs/read;   0.70 M reads/minute\n",
      "Finished in 6.09 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,323\n",
      "  Read 1 with adapter:                  71,278 (99.9%)\n",
      "  Read 2 with adapter:                  70,981 (99.5%)\n",
      "Pairs written (passing filters):        71,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,073,917 bp\n",
      "  Read 1:    21,037,279 bp\n",
      "  Read 2:    21,036,638 bp\n",
      "Total written (filtered):     39,300,280 bp (93.4%)\n",
      "  Read 1:    19,612,014 bp\n",
      "  Read 2:    19,688,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 71278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t510\t0.0\t1\t0 484 26\n",
      "20\t70553\t0.0\t2\t68888 1638 27\n",
      "21\t119\t0.0\t2\t0 115 4\n",
      "22\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t408\t0.0\t1\t0 408\n",
      "19\t70432\t0.0\t1\t68411 2021\n",
      "20\t141\t0.0\t1\t0 141\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1254_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1254_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1254_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1254_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1254_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1254_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1254_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1254_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        73,090 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 5.63 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,090\n",
      "  Read 1 with adapter:                  73,054 (100.0%)\n",
      "  Read 2 with adapter:                  72,684 (99.4%)\n",
      "Pairs written (passing filters):        73,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,099,358 bp\n",
      "  Read 1:    21,561,535 bp\n",
      "  Read 2:    21,537,823 bp\n",
      "Total written (filtered):     40,257,801 bp (93.4%)\n",
      "  Read 1:    20,100,744 bp\n",
      "  Read 2:    20,157,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73054 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t443\t0.0\t1\t0 427 16\n",
      "20\t72425\t0.0\t2\t70968 1438 19\n",
      "21\t110\t0.0\t2\t0 107 3\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72684 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t352\t0.0\t1\t0 352\n",
      "19\t72210\t0.0\t1\t70432 1778\n",
      "20\t122\t0.0\t1\t0 122\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1255_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1255_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1255_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1255_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1255_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1255_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1255_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1255_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,903 reads  @     76.9 µs/read;   0.78 M reads/minute\n",
      "Finished in 5.38 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,903\n",
      "  Read 1 with adapter:                  69,876 (100.0%)\n",
      "  Read 2 with adapter:                  69,564 (99.5%)\n",
      "Pairs written (passing filters):        69,903 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,242,167 bp\n",
      "  Read 1:    20,621,366 bp\n",
      "  Read 2:    20,620,801 bp\n",
      "Total written (filtered):     38,523,455 bp (93.4%)\n",
      "  Read 1:    19,224,109 bp\n",
      "  Read 2:    19,299,346 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t418\t0.0\t1\t0 410 8\n",
      "20\t69297\t0.0\t2\t67928 1348 21\n",
      "21\t95\t0.0\t2\t0 94 1\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t405\t0.0\t1\t0 405\n",
      "19\t69015\t0.0\t1\t67255 1760\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1256_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1256_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1256_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1256_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1256_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1256_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1256_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1256_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,165 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 5.06 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,165\n",
      "  Read 1 with adapter:                  64,130 (99.9%)\n",
      "  Read 2 with adapter:                  63,860 (99.5%)\n",
      "Pairs written (passing filters):        64,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,854,623 bp\n",
      "  Read 1:    18,928,505 bp\n",
      "  Read 2:    18,926,118 bp\n",
      "Total written (filtered):     35,358,590 bp (93.4%)\n",
      "  Read 1:    17,645,593 bp\n",
      "  Read 2:    17,712,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64130 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t386\t0.0\t1\t0 378 8\n",
      "20\t63576\t0.0\t2\t62275 1290 11\n",
      "21\t116\t0.0\t2\t0 116\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t338\t0.0\t1\t0 338\n",
      "19\t63403\t0.0\t1\t61547 1856\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1257_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1257_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1257_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1257_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1257_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1257_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1257_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1257_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,100 reads  @     73.7 µs/read;   0.81 M reads/minute\n",
      "Finished in 5.98 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,100\n",
      "  Read 1 with adapter:                  81,051 (99.9%)\n",
      "  Read 2 with adapter:                  80,616 (99.4%)\n",
      "Pairs written (passing filters):        81,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,831,678 bp\n",
      "  Read 1:    23,923,397 bp\n",
      "  Read 2:    23,908,281 bp\n",
      "Total written (filtered):     44,679,498 bp (93.4%)\n",
      "  Read 1:    22,302,700 bp\n",
      "  Read 2:    22,376,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t496\t0.0\t1\t0 487 9\n",
      "20\t80336\t0.0\t2\t78714 1609 13\n",
      "21\t137\t0.0\t2\t0 136 1\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t371\t0.0\t1\t0 371\n",
      "19\t80095\t0.0\t1\t78156 1939\n",
      "20\t150\t0.0\t1\t0 150\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1258_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1258_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1258_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1258_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1258_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1258_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1258_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1258_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,470 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 5.63 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,470\n",
      "  Read 1 with adapter:                  80,436 (100.0%)\n",
      "  Read 2 with adapter:                  80,091 (99.5%)\n",
      "Pairs written (passing filters):        80,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,465,969 bp\n",
      "  Read 1:    23,738,343 bp\n",
      "  Read 2:    23,727,626 bp\n",
      "Total written (filtered):     44,336,095 bp (93.4%)\n",
      "  Read 1:    22,129,945 bp\n",
      "  Read 2:    22,206,150 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t504\t0.0\t1\t0 495 9\n",
      "20\t79726\t0.0\t2\t78131 1581 14\n",
      "21\t138\t0.0\t2\t0 134 4\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80091 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t401\t0.0\t1\t0 401\n",
      "19\t79542\t0.0\t1\t77914 1628\n",
      "20\t148\t0.0\t1\t0 148\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1259_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1259_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1259_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1259_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1259_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1259_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1259_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1259_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        57,929 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.14 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,929\n",
      "  Read 1 with adapter:                  57,911 (100.0%)\n",
      "  Read 2 with adapter:                  57,013 (98.4%)\n",
      "Pairs written (passing filters):        57,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,129,258 bp\n",
      "  Read 1:    17,089,041 bp\n",
      "  Read 2:    17,040,217 bp\n",
      "Total written (filtered):     31,888,265 bp (93.4%)\n",
      "  Read 1:    15,931,075 bp\n",
      "  Read 2:    15,957,190 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57911 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t372\t0.0\t1\t0 368 4\n",
      "20\t57392\t0.0\t2\t56156 1222 14\n",
      "21\t100\t0.0\t2\t0 96 4\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57013 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t310\t0.0\t1\t0 310\n",
      "19\t56613\t0.0\t1\t55434 1179\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1260_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1260_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1260_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1260_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1260_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1260_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1260_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1260_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        74,979 reads  @     75.6 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.67 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,979\n",
      "  Read 1 with adapter:                  74,665 (99.6%)\n",
      "  Read 2 with adapter:                  74,234 (99.0%)\n",
      "Pairs written (passing filters):        74,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,194,853 bp\n",
      "  Read 1:    22,081,739 bp\n",
      "  Read 2:    22,113,114 bp\n",
      "Total written (filtered):     41,291,539 bp (93.4%)\n",
      "  Read 1:    20,588,642 bp\n",
      "  Read 2:    20,702,897 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t359\t0.0\t1\t0 352 7\n",
      "20\t74142\t0.0\t2\t72610 1513 19\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74234 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t365\t0.0\t1\t0 365\n",
      "19\t73733\t0.0\t1\t71820 1913\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1261_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1261_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1261_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1261_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1261_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1261_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1261_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1261_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       116,609 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 8.25 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,609\n",
      "  Read 1 with adapter:                 116,457 (99.9%)\n",
      "  Read 2 with adapter:                 115,812 (99.3%)\n",
      "Pairs written (passing filters):       116,609 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,773,645 bp\n",
      "  Read 1:    34,385,821 bp\n",
      "  Read 2:    34,387,824 bp\n",
      "Total written (filtered):     64,244,777 bp (93.4%)\n",
      "  Read 1:    32,057,031 bp\n",
      "  Read 2:    32,187,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 116457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t613\t0.0\t1\t0 596 17\n",
      "20\t115566\t0.0\t2\t113295 2248 23\n",
      "21\t189\t0.0\t2\t0 187 2\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 115812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t586\t0.0\t1\t0 586\n",
      "19\t114990\t0.0\t1\t111876 3114\n",
      "20\t236\t0.0\t1\t0 236\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1262_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1262_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1262_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1262_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1262_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1262_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1262_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1262_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        94,696 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 6.51 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,696\n",
      "  Read 1 with adapter:                  94,650 (100.0%)\n",
      "  Read 2 with adapter:                  93,690 (98.9%)\n",
      "Pairs written (passing filters):        94,696 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,811,674 bp\n",
      "  Read 1:    27,935,294 bp\n",
      "  Read 2:    27,876,380 bp\n",
      "Total written (filtered):     52,139,212 bp (93.4%)\n",
      "  Read 1:    26,042,662 bp\n",
      "  Read 2:    26,096,550 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 94650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t564\t0.0\t1\t0 553 11\n",
      "20\t93857\t0.0\t2\t91981 1850 26\n",
      "21\t142\t0.0\t2\t0 136 6\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 93690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t461\t0.0\t1\t0 461\n",
      "19\t93048\t0.0\t1\t90675 2373\n",
      "20\t181\t0.0\t1\t0 181\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1263_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1263_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1263_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1263_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1263_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1263_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1263_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1263_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        74,144 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 5.21 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,144\n",
      "  Read 1 with adapter:                  73,991 (99.8%)\n",
      "  Read 2 with adapter:                  73,579 (99.2%)\n",
      "Pairs written (passing filters):        74,144 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,698,950 bp\n",
      "  Read 1:    21,858,570 bp\n",
      "  Read 2:    21,840,380 bp\n",
      "Total written (filtered):     40,821,691 bp (93.4%)\n",
      "  Read 1:    20,378,986 bp\n",
      "  Read 2:    20,442,705 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t430\t0.0\t1\t0 423 7\n",
      "20\t73363\t0.0\t2\t71921 1415 27\n",
      "21\t130\t0.0\t2\t0 130\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 73579 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t479\t0.0\t1\t0 479\n",
      "19\t72947\t0.0\t1\t70967 1980\n",
      "20\t153\t0.0\t1\t0 153\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1264_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1264_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1264_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1264_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1264_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1264_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1264_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1264_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        75,473 reads  @     71.1 µs/read;   0.84 M reads/minute\n",
      "Finished in 5.37 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,473\n",
      "  Read 1 with adapter:                  75,430 (99.9%)\n",
      "  Read 2 with adapter:                  74,961 (99.3%)\n",
      "Pairs written (passing filters):        75,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,521,176 bp\n",
      "  Read 1:    22,262,930 bp\n",
      "  Read 2:    22,258,246 bp\n",
      "Total written (filtered):     41,588,558 bp (93.4%)\n",
      "  Read 1:    20,754,319 bp\n",
      "  Read 2:    20,834,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t450\t0.0\t1\t0 438 12\n",
      "20\t74778\t0.0\t2\t73206 1548 24\n",
      "21\t122\t0.0\t2\t0 118 4\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t388\t0.0\t1\t0 388\n",
      "19\t74437\t0.0\t1\t72111 2326\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1265_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1265_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1265_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1265_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1265_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1265_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1265_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1265_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        77,302 reads  @     64.2 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.97 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,302\n",
      "  Read 1 with adapter:                  77,056 (99.7%)\n",
      "  Read 2 with adapter:                  76,567 (99.0%)\n",
      "Pairs written (passing filters):        77,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,525,818 bp\n",
      "  Read 1:    22,780,137 bp\n",
      "  Read 2:    22,745,681 bp\n",
      "Total written (filtered):     42,530,376 bp (93.4%)\n",
      "  Read 1:    21,239,252 bp\n",
      "  Read 2:    21,291,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77056 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t430\t0.0\t1\t0 419 11\n",
      "20\t76411\t0.0\t2\t74869 1526 16\n",
      "21\t143\t0.0\t2\t0 141 2\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 76567 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t363\t0.0\t1\t0 363\n",
      "19\t76057\t0.0\t1\t73999 2058\n",
      "20\t147\t0.0\t1\t0 147\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1266_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1266_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1266_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1266_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1266_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1266_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1266_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1266_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        85,488 reads  @     65.9 µs/read;   0.91 M reads/minute\n",
      "Finished in 5.64 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,488\n",
      "  Read 1 with adapter:                  85,383 (99.9%)\n",
      "  Read 2 with adapter:                  84,915 (99.3%)\n",
      "Pairs written (passing filters):        85,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,396,597 bp\n",
      "  Read 1:    25,212,127 bp\n",
      "  Read 2:    25,184,470 bp\n",
      "Total written (filtered):     47,076,096 bp (93.4%)\n",
      "  Read 1:    23,504,778 bp\n",
      "  Read 2:    23,571,318 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 85383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t480\t0.0\t1\t0 472 8\n",
      "20\t84696\t0.0\t2\t82924 1752 20\n",
      "21\t137\t0.0\t2\t0 131 6\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 84915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t421\t0.0\t1\t0 421\n",
      "19\t84306\t0.0\t1\t82279 2027\n",
      "20\t188\t0.0\t1\t0 188\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1267_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1267_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1267_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1267_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1267_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1267_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1267_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1267_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       112,844 reads  @     72.5 µs/read;   0.83 M reads/minute\n",
      "Finished in 8.19 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,844\n",
      "  Read 1 with adapter:                 112,780 (99.9%)\n",
      "  Read 2 with adapter:                 109,582 (97.1%)\n",
      "Pairs written (passing filters):       112,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,402,037 bp\n",
      "  Read 1:    33,287,443 bp\n",
      "  Read 2:    33,114,594 bp\n",
      "Total written (filtered):     62,065,063 bp (93.5%)\n",
      "  Read 1:    31,032,159 bp\n",
      "  Read 2:    31,032,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 112780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t628\t0.0\t1\t0 611 17\n",
      "20\t111837\t0.0\t2\t109441 2365 31\n",
      "21\t194\t0.0\t2\t0 189 5\n",
      "22\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109582 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t584\t0.0\t1\t0 584\n",
      "19\t108782\t0.0\t1\t106035 2747\n",
      "20\t216\t0.0\t1\t0 216\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1268_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1268_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1268_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1268_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1268_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1268_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1268_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1268_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        73,288 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 4.95 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,288\n",
      "  Read 1 with adapter:                  73,233 (99.9%)\n",
      "  Read 2 with adapter:                  72,972 (99.6%)\n",
      "Pairs written (passing filters):        73,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,234,403 bp\n",
      "  Read 1:    21,615,133 bp\n",
      "  Read 2:    21,619,270 bp\n",
      "Total written (filtered):     40,383,758 bp (93.4%)\n",
      "  Read 1:    20,150,757 bp\n",
      "  Read 2:    20,233,001 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t440\t0.0\t1\t0 430 10\n",
      "20\t72629\t0.0\t2\t71247 1369 13\n",
      "21\t120\t0.0\t2\t0 117 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t328\t0.0\t1\t0 328\n",
      "19\t72515\t0.0\t1\t70873 1642\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1269_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1269_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1269_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1269_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1269_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1269_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1269_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1269_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        77,494 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 5.35 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,494\n",
      "  Read 1 with adapter:                  77,459 (100.0%)\n",
      "  Read 2 with adapter:                  77,174 (99.6%)\n",
      "Pairs written (passing filters):        77,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,719,795 bp\n",
      "  Read 1:    22,860,450 bp\n",
      "  Read 2:    22,859,345 bp\n",
      "Total written (filtered):     42,704,911 bp (93.4%)\n",
      "  Read 1:    21,311,622 bp\n",
      "  Read 2:    21,393,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t553\t0.0\t1\t0 542 11\n",
      "20\t76709\t0.0\t2\t75215 1477 17\n",
      "21\t125\t0.0\t2\t0 122 3\n",
      "22\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t402\t0.0\t1\t0 402\n",
      "19\t76620\t0.0\t1\t75030 1590\n",
      "20\t152\t0.0\t1\t0 152\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1270_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1270_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1270_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1270_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1270_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1270_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1270_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1270_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        78,254 reads  @     66.5 µs/read;   0.90 M reads/minute\n",
      "Finished in 5.21 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,254\n",
      "  Read 1 with adapter:                  78,217 (100.0%)\n",
      "  Read 2 with adapter:                  78,006 (99.7%)\n",
      "Pairs written (passing filters):        78,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,166,939 bp\n",
      "  Read 1:    23,084,369 bp\n",
      "  Read 2:    23,082,570 bp\n",
      "Total written (filtered):     43,121,061 bp (93.4%)\n",
      "  Read 1:    21,520,349 bp\n",
      "  Read 2:    21,600,712 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 78217 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t515\t0.0\t1\t0 499 16\n",
      "20\t77498\t0.0\t2\t76033 1453 12\n",
      "21\t133\t0.0\t2\t0 130 3\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 78006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t408\t0.0\t1\t0 408\n",
      "19\t77446\t0.0\t1\t75910 1536\n",
      "20\t152\t0.0\t1\t0 152\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1271_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1271_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1271_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1271_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1271_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1271_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1271_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1271_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:08       110,604 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 8.76 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,604\n",
      "  Read 1 with adapter:                 110,518 (99.9%)\n",
      "  Read 2 with adapter:                 110,147 (99.6%)\n",
      "Pairs written (passing filters):       110,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,250,844 bp\n",
      "  Read 1:    32,625,876 bp\n",
      "  Read 2:    32,624,968 bp\n",
      "Total written (filtered):     60,948,662 bp (93.4%)\n",
      "  Read 1:    30,416,009 bp\n",
      "  Read 2:    30,532,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 110518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t734\t0.0\t1\t0 721 13\n",
      "20\t109487\t0.0\t2\t107384 2081 22\n",
      "21\t197\t0.0\t2\t0 194 3\n",
      "22\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 110147 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t745\t0.0\t1\t0 745\n",
      "19\t109135\t0.0\t1\t106679 2456\n",
      "20\t267\t0.0\t1\t0 267\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1272_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1272_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1272_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1272_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1272_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1272_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1272_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1272_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        78,496 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.95 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,496\n",
      "  Read 1 with adapter:                  78,461 (100.0%)\n",
      "  Read 2 with adapter:                  78,215 (99.6%)\n",
      "Pairs written (passing filters):        78,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,308,631 bp\n",
      "  Read 1:    23,155,693 bp\n",
      "  Read 2:    23,152,938 bp\n",
      "Total written (filtered):     43,253,116 bp (93.4%)\n",
      "  Read 1:    21,586,004 bp\n",
      "  Read 2:    21,667,112 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 78461 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t531\t0.0\t1\t0 518 13\n",
      "20\t77750\t0.0\t2\t76155 1570 25\n",
      "21\t125\t0.0\t2\t0 124 1\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 78215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t403\t0.0\t1\t0 403\n",
      "19\t77668\t0.0\t1\t75944 1724\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1273_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1273_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1273_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1273_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1273_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1273_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1273_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1273_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        79,434 reads  @     79.0 µs/read;   0.76 M reads/minute\n",
      "Finished in 6.28 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,434\n",
      "  Read 1 with adapter:                  79,372 (99.9%)\n",
      "  Read 2 with adapter:                  78,797 (99.2%)\n",
      "Pairs written (passing filters):        79,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,774,430 bp\n",
      "  Read 1:    23,427,489 bp\n",
      "  Read 2:    23,346,941 bp\n",
      "Total written (filtered):     43,690,464 bp (93.4%)\n",
      "  Read 1:    21,840,453 bp\n",
      "  Read 2:    21,850,011 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 79372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t575\t0.0\t1\t0 559 16\n",
      "20\t78588\t0.0\t2\t76899 1664 25\n",
      "21\t143\t0.0\t2\t0 140 3\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 78797 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t378\t0.0\t1\t0 378\n",
      "19\t78254\t0.0\t1\t76602 1652\n",
      "20\t165\t0.0\t1\t0 165\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1274_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1274_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1274_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1274_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1274_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1274_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1274_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1274_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        73,035 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.54 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,035\n",
      "  Read 1 with adapter:                  73,004 (100.0%)\n",
      "  Read 2 with adapter:                  72,784 (99.7%)\n",
      "Pairs written (passing filters):        73,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,077,813 bp\n",
      "  Read 1:    21,545,304 bp\n",
      "  Read 2:    21,532,509 bp\n",
      "Total written (filtered):     40,235,434 bp (93.4%)\n",
      "  Read 1:    20,085,583 bp\n",
      "  Read 2:    20,149,851 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73004 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t484\t0.0\t1\t0 472 12\n",
      "20\t72361\t0.0\t2\t70825 1521 15\n",
      "21\t105\t0.0\t2\t0 103 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72784 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t380\t0.0\t1\t0 380\n",
      "19\t72262\t0.0\t1\t70842 1420\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1275_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1275_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1275_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1275_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1275_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1275_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1275_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1275_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        70,704 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.37 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,704\n",
      "  Read 1 with adapter:                  70,673 (100.0%)\n",
      "  Read 2 with adapter:                  69,369 (98.1%)\n",
      "Pairs written (passing filters):        70,704 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,638,825 bp\n",
      "  Read 1:    20,857,670 bp\n",
      "  Read 2:    20,781,155 bp\n",
      "Total written (filtered):     38,907,898 bp (93.4%)\n",
      "  Read 1:    19,444,544 bp\n",
      "  Read 2:    19,463,354 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70673 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t491\t0.0\t1\t0 479 12\n",
      "20\t70016\t0.0\t2\t68537 1462 17\n",
      "21\t111\t0.0\t2\t0 109 2\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69369 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t334\t0.0\t1\t0 334\n",
      "19\t68911\t0.0\t1\t67574 1337\n",
      "20\t124\t0.0\t1\t0 124\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1276_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1276_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1276_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1276_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1276_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1276_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1276_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1276_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        93,783 reads  @     74.0 µs/read;   0.81 M reads/minute\n",
      "Finished in 6.95 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,783\n",
      "  Read 1 with adapter:                  93,647 (99.9%)\n",
      "  Read 2 with adapter:                  92,285 (98.4%)\n",
      "Pairs written (passing filters):        93,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,309,427 bp\n",
      "  Read 1:    27,658,629 bp\n",
      "  Read 2:    27,650,798 bp\n",
      "Total written (filtered):     51,683,693 bp (93.4%)\n",
      "  Read 1:    25,785,858 bp\n",
      "  Read 2:    25,897,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t591\t0.0\t1\t0 573 18\n",
      "20\t92708\t0.0\t2\t90310 2359 39\n",
      "21\t162\t0.0\t2\t0 157 5\n",
      "22\t158\t0.0\t2\t0 0 158\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 92285 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t659\t0.0\t1\t0 659\n",
      "19\t91419\t0.0\t1\t87798 3621\n",
      "20\t207\t0.0\t1\t0 207\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1292_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1292_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1292_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1292_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1292_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1292_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1292_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1292_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        64,285 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 5.01 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,285\n",
      "  Read 1 with adapter:                  64,214 (99.9%)\n",
      "  Read 2 with adapter:                  63,991 (99.5%)\n",
      "Pairs written (passing filters):        64,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,919,256 bp\n",
      "  Read 1:    18,962,649 bp\n",
      "  Read 2:    18,956,607 bp\n",
      "Total written (filtered):     35,419,499 bp (93.4%)\n",
      "  Read 1:    17,678,531 bp\n",
      "  Read 2:    17,740,968 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t350\t0.0\t1\t0 346 4\n",
      "20\t63664\t0.0\t2\t62332 1312 20\n",
      "21\t136\t0.0\t2\t0 132 4\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t310\t0.0\t1\t0 310\n",
      "19\t63561\t0.0\t1\t62083 1478\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1293_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1293_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1293_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1293_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1293_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1293_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1293_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1293_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,129 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.74 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,129\n",
      "  Read 1 with adapter:                  62,065 (99.9%)\n",
      "  Read 2 with adapter:                  61,864 (99.6%)\n",
      "Pairs written (passing filters):        62,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,651,385 bp\n",
      "  Read 1:    18,326,168 bp\n",
      "  Read 2:    18,325,217 bp\n",
      "Total written (filtered):     34,235,056 bp (93.4%)\n",
      "  Read 1:    17,085,090 bp\n",
      "  Read 2:    17,149,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t407\t0.0\t1\t0 401 6\n",
      "20\t61455\t0.0\t2\t60168 1271 16\n",
      "21\t137\t0.0\t2\t0 134 3\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t276\t0.0\t1\t0 276\n",
      "19\t61477\t0.0\t1\t60017 1460\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1294_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1294_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1294_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1294_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1294_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1294_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1294_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1294_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,390 reads  @     76.1 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.60 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,390\n",
      "  Read 1 with adapter:                  60,325 (99.9%)\n",
      "  Read 2 with adapter:                  60,128 (99.6%)\n",
      "Pairs written (passing filters):        60,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,624,176 bp\n",
      "  Read 1:    17,814,206 bp\n",
      "  Read 2:    17,809,970 bp\n",
      "Total written (filtered):     33,275,695 bp (93.4%)\n",
      "  Read 1:    16,607,937 bp\n",
      "  Read 2:    16,667,758 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60325 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t377\t0.0\t1\t0 366 11\n",
      "20\t59751\t0.0\t2\t58468 1270 13\n",
      "21\t132\t0.0\t2\t0 129 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60128 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t326\t0.0\t1\t0 326\n",
      "19\t59696\t0.0\t1\t58322 1374\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1295_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1295_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1295_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1295_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1295_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1295_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1295_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1295_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,882 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 2.48 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,882\n",
      "  Read 1 with adapter:                  29,791 (99.7%)\n",
      "  Read 2 with adapter:                  29,544 (98.9%)\n",
      "Pairs written (passing filters):        29,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,630,105 bp\n",
      "  Read 1:     8,815,184 bp\n",
      "  Read 2:     8,814,921 bp\n",
      "Total written (filtered):     16,473,236 bp (93.4%)\n",
      "  Read 1:     8,219,443 bp\n",
      "  Read 2:     8,253,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t243\t0.0\t1\t0 236 7\n",
      "20\t29401\t0.0\t2\t28617 775 9\n",
      "21\t62\t0.0\t2\t0 59 3\n",
      "22\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t280\t0.0\t1\t0 280\n",
      "19\t29192\t0.0\t1\t28185 1007\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1296_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1296_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1296_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1296_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1296_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1296_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1296_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1296_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,404 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.38 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,404\n",
      "  Read 1 with adapter:                  58,375 (100.0%)\n",
      "  Read 2 with adapter:                  58,121 (99.5%)\n",
      "Pairs written (passing filters):        58,398 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,450,166 bp\n",
      "  Read 1:    17,228,718 bp\n",
      "  Read 2:    17,221,448 bp\n",
      "Total written (filtered):     32,177,138 bp (93.4%)\n",
      "  Read 1:    16,059,796 bp\n",
      "  Read 2:    16,117,342 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58375 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t367\t0.0\t1\t0 358 9\n",
      "20\t57832\t0.0\t2\t56620 1193 19\n",
      "21\t101\t0.0\t2\t0 98 3\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58121 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t298\t0.0\t1\t0 298\n",
      "19\t57718\t0.0\t1\t56229 1489\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1297_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1297_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1297_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1297_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1297_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1297_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1297_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1297_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,773 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.00 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,773\n",
      "  Read 1 with adapter:                  54,734 (99.9%)\n",
      "  Read 2 with adapter:                  54,466 (99.4%)\n",
      "Pairs written (passing filters):        54,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,289,342 bp\n",
      "  Read 1:    16,156,564 bp\n",
      "  Read 2:    16,132,778 bp\n",
      "Total written (filtered):     30,160,160 bp (93.4%)\n",
      "  Read 1:    15,062,085 bp\n",
      "  Read 2:    15,098,075 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54734 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t377\t0.0\t1\t0 368 9\n",
      "20\t54167\t0.0\t2\t53058 1095 14\n",
      "21\t136\t0.0\t2\t0 132 4\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t247\t0.0\t1\t0 247\n",
      "19\t54123\t0.0\t1\t53016 1107\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1298_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1298_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1298_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1298_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1298_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1298_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1298_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1298_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:03        39,377 reads  @     83.4 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.29 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,377\n",
      "  Read 1 with adapter:                  39,075 (99.2%)\n",
      "  Read 2 with adapter:                  38,913 (98.8%)\n",
      "Pairs written (passing filters):        39,376 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,214,597 bp\n",
      "  Read 1:    11,615,432 bp\n",
      "  Read 2:    11,599,165 bp\n",
      "Total written (filtered):     21,693,719 bp (93.4%)\n",
      "  Read 1:    10,833,788 bp\n",
      "  Read 2:    10,859,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t239\t0.0\t1\t0 233 6\n",
      "20\t38716\t0.0\t2\t37800 904 12\n",
      "21\t72\t0.0\t2\t0 72\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38913 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t172\t0.0\t1\t0 172\n",
      "19\t38682\t0.0\t1\t37786 896\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1299_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1299_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1299_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1299_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1299_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1299_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1299_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1299_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,264 reads  @     79.0 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.90 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,264\n",
      "  Read 1 with adapter:                  49,204 (99.9%)\n",
      "  Read 2 with adapter:                  40,975 (83.2%)\n",
      "Pairs written (passing filters):        49,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,536,869 bp\n",
      "  Read 1:    14,529,631 bp\n",
      "  Read 2:    14,007,238 bp\n",
      "Total written (filtered):     26,774,599 bp (93.8%)\n",
      "  Read 1:    13,545,745 bp\n",
      "  Read 2:    13,228,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t329\t0.0\t1\t0 324 5\n",
      "20\t48719\t0.0\t2\t47624 1083 12\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40975 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t200\t0.0\t1\t0 200\n",
      "19\t40716\t0.0\t1\t39990 726\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1300_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1300_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1300_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1300_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1300_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1300_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1300_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1300_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,982 reads  @     77.4 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.34 s (77 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,982\n",
      "  Read 1 with adapter:                  55,915 (99.9%)\n",
      "  Read 2 with adapter:                  55,620 (99.4%)\n",
      "Pairs written (passing filters):        55,982 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,017,994 bp\n",
      "  Read 1:    16,508,149 bp\n",
      "  Read 2:    16,509,845 bp\n",
      "Total written (filtered):     30,843,518 bp (93.4%)\n",
      "  Read 1:    15,390,268 bp\n",
      "  Read 2:    15,453,250 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t545\t0.0\t1\t0 531 14\n",
      "20\t55218\t0.0\t2\t54030 1177 11\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55620 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t292\t0.0\t1\t0 292\n",
      "19\t55221\t0.0\t1\t53800 1421\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1301_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1301_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1301_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1301_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1301_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1301_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1301_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1301_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        24,308 reads  @    101.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 2.46 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,308\n",
      "  Read 1 with adapter:                  24,284 (99.9%)\n",
      "  Read 2 with adapter:                  24,193 (99.5%)\n",
      "Pairs written (passing filters):        24,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,337,926 bp\n",
      "  Read 1:     7,168,819 bp\n",
      "  Read 2:     7,169,107 bp\n",
      "Total written (filtered):     13,392,874 bp (93.4%)\n",
      "  Read 1:     6,683,333 bp\n",
      "  Read 2:     6,709,541 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24284 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t259\t0.0\t1\t0 250 9\n",
      "20\t23963\t0.0\t2\t23304 649 10\n",
      "21\t39\t0.0\t2\t0 39\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t164\t0.0\t1\t0 164\n",
      "19\t23966\t0.0\t1\t23192 774\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1302_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1302_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1302_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1302_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1302_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1302_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1302_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1302_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,746 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.99 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,746\n",
      "  Read 1 with adapter:                  64,687 (99.9%)\n",
      "  Read 2 with adapter:                  64,210 (99.2%)\n",
      "Pairs written (passing filters):        64,746 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,150,159 bp\n",
      "  Read 1:    19,095,908 bp\n",
      "  Read 2:    19,054,251 bp\n",
      "Total written (filtered):     35,637,047 bp (93.4%)\n",
      "  Read 1:    17,802,604 bp\n",
      "  Read 2:    17,834,443 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t555\t0.0\t1\t0 545 10\n",
      "20\t63986\t0.0\t2\t62721 1248 17\n",
      "21\t93\t0.0\t2\t0 92 1\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t312\t0.0\t1\t0 312\n",
      "19\t63768\t0.0\t1\t62072 1696\n",
      "20\t130\t0.0\t1\t0 130\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1303_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1303_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1303_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1303_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1303_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1303_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1303_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1303_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        75,293 reads  @     78.3 µs/read;   0.77 M reads/minute\n",
      "Finished in 5.90 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,293\n",
      "  Read 1 with adapter:                  75,218 (99.9%)\n",
      "  Read 2 with adapter:                  74,863 (99.4%)\n",
      "Pairs written (passing filters):        75,293 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,416,219 bp\n",
      "  Read 1:    22,204,848 bp\n",
      "  Read 2:    22,211,371 bp\n",
      "Total written (filtered):     41,490,301 bp (93.4%)\n",
      "  Read 1:    20,700,997 bp\n",
      "  Read 2:    20,789,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t714\t0.0\t1\t0 707 7\n",
      "20\t74291\t0.0\t2\t72779 1488 24\n",
      "21\t141\t0.0\t2\t0 140 1\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t486\t0.0\t1\t0 486\n",
      "19\t74221\t0.0\t1\t71952 2269\n",
      "20\t156\t0.0\t1\t0 156\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1304_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1304_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1304_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1304_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1304_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1304_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1304_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1304_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:06        72,905 reads  @     83.3 µs/read;   0.72 M reads/minute\n",
      "Finished in 6.08 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,905\n",
      "  Read 1 with adapter:                  72,853 (99.9%)\n",
      "  Read 2 with adapter:                  72,370 (99.3%)\n",
      "Pairs written (passing filters):        72,903 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,002,125 bp\n",
      "  Read 1:    21,503,987 bp\n",
      "  Read 2:    21,498,138 bp\n",
      "Total written (filtered):     40,170,240 bp (93.4%)\n",
      "  Read 1:    20,046,873 bp\n",
      "  Read 2:    20,123,367 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 72853 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t671\t0.0\t1\t0 657 14\n",
      "20\t71977\t0.0\t2\t70362 1598 17\n",
      "21\t147\t0.0\t2\t0 144 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72370 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t399\t0.0\t1\t0 399\n",
      "19\t71831\t0.0\t1\t68881 2950\n",
      "20\t140\t0.0\t1\t0 140\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1305_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1305_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1305_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1305_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1305_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1305_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1305_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1305_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,292 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.74 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,292\n",
      "  Read 1 with adapter:                  62,252 (99.9%)\n",
      "  Read 2 with adapter:                  61,598 (98.9%)\n",
      "Pairs written (passing filters):        62,292 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,710,976 bp\n",
      "  Read 1:    18,373,134 bp\n",
      "  Read 2:    18,337,842 bp\n",
      "Total written (filtered):     34,296,196 bp (93.4%)\n",
      "  Read 1:    17,128,547 bp\n",
      "  Read 2:    17,167,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t594\t0.0\t1\t0 578 16\n",
      "20\t61499\t0.0\t2\t60208 1271 20\n",
      "21\t97\t0.0\t2\t0 96 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t283\t0.0\t1\t0 283\n",
      "19\t61201\t0.0\t1\t59111 2090\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1306_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1306_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1306_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1306_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1306_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1306_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1306_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1306_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,799 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.01 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,799\n",
      "  Read 1 with adapter:                  54,755 (99.9%)\n",
      "  Read 2 with adapter:                  54,424 (99.3%)\n",
      "Pairs written (passing filters):        54,798 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,307,248 bp\n",
      "  Read 1:    16,163,986 bp\n",
      "  Read 2:    16,143,262 bp\n",
      "Total written (filtered):     30,178,418 bp (93.4%)\n",
      "  Read 1:    15,069,040 bp\n",
      "  Read 2:    15,109,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t544\t0.0\t1\t0 531 13\n",
      "20\t54065\t0.0\t2\t52843 1210 12\n",
      "21\t117\t0.0\t2\t0 117\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t274\t0.0\t1\t0 274\n",
      "19\t54048\t0.0\t1\t52693 1355\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1307_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1307_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1307_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1307_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1307_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1307_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1307_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1307_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,001 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.80 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,001\n",
      "  Read 1 with adapter:                  51,911 (99.8%)\n",
      "  Read 2 with adapter:                  45,815 (88.1%)\n",
      "Pairs written (passing filters):        52,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,285,421 bp\n",
      "  Read 1:    15,329,910 bp\n",
      "  Read 2:    14,955,511 bp\n",
      "Total written (filtered):     28,377,154 bp (93.7%)\n",
      "  Read 1:    14,292,003 bp\n",
      "  Read 2:    14,085,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51911 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t433\t0.0\t1\t0 420 13\n",
      "20\t51344\t0.0\t2\t50317 1013 14\n",
      "21\t92\t0.0\t2\t0 86 6\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t45508\t0.0\t1\t44466 1042\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1308_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1308_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1308_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1308_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1308_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1308_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1308_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1308_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,500 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 5.12 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,500\n",
      "  Read 1 with adapter:                  66,375 (99.8%)\n",
      "  Read 2 with adapter:                  66,173 (99.5%)\n",
      "Pairs written (passing filters):        66,500 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,222,042 bp\n",
      "  Read 1:    19,609,570 bp\n",
      "  Read 2:    19,612,472 bp\n",
      "Total written (filtered):     36,637,688 bp (93.4%)\n",
      "  Read 1:    18,282,271 bp\n",
      "  Read 2:    18,355,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66375 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t352\t0.0\t1\t0 347 5\n",
      "20\t65855\t0.0\t2\t64537 1302 16\n",
      "21\t117\t0.0\t2\t0 113 4\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66173 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t325\t0.0\t1\t0 325\n",
      "19\t65755\t0.0\t1\t64248 1507\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1309_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1309_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1309_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1309_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1309_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1309_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1309_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1309_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,171 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.14 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,171\n",
      "  Read 1 with adapter:                  56,113 (99.9%)\n",
      "  Read 2 with adapter:                  55,919 (99.6%)\n",
      "Pairs written (passing filters):        56,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,137,450 bp\n",
      "  Read 1:    16,568,232 bp\n",
      "  Read 2:    16,569,218 bp\n",
      "Total written (filtered):     30,953,191 bp (93.4%)\n",
      "  Read 1:    15,446,266 bp\n",
      "  Read 2:    15,506,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t378\t0.0\t1\t0 373 5\n",
      "20\t55607\t0.0\t2\t54484 1113 10\n",
      "21\t80\t0.0\t2\t0 76 4\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55919 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t268\t0.0\t1\t0 268\n",
      "19\t55551\t0.0\t1\t54303 1248\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1310_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1310_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1310_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1310_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1310_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1310_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1310_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1310_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,400 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.59 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,400\n",
      "  Read 1 with adapter:                  50,353 (99.9%)\n",
      "  Read 2 with adapter:                  49,665 (98.5%)\n",
      "Pairs written (passing filters):        50,400 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,671,912 bp\n",
      "  Read 1:    14,866,297 bp\n",
      "  Read 2:    14,805,615 bp\n",
      "Total written (filtered):     27,721,607 bp (93.4%)\n",
      "  Read 1:    13,859,475 bp\n",
      "  Read 2:    13,862,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50353 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t344\t0.0\t1\t0 335 9\n",
      "20\t49871\t0.0\t2\t48848 1009 14\n",
      "21\t82\t0.0\t2\t0 79 3\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t252\t0.0\t1\t0 252\n",
      "19\t49313\t0.0\t1\t48263 1050\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1311_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1311_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1311_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1311_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1311_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1311_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1311_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1311_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,932 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.78 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,932\n",
      "  Read 1 with adapter:                  53,883 (99.9%)\n",
      "  Read 2 with adapter:                  53,662 (99.5%)\n",
      "Pairs written (passing filters):        53,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,817,707 bp\n",
      "  Read 1:    15,908,289 bp\n",
      "  Read 2:    15,909,418 bp\n",
      "Total written (filtered):     29,720,938 bp (93.4%)\n",
      "  Read 1:    14,830,863 bp\n",
      "  Read 2:    14,890,075 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53883 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t344\t0.0\t1\t0 336 8\n",
      "20\t53392\t0.0\t2\t52260 1112 20\n",
      "21\t96\t0.0\t2\t0 94 2\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t361\t0.0\t1\t0 361\n",
      "19\t53175\t0.0\t1\t51960 1215\n",
      "20\t126\t0.0\t1\t0 126\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1312_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1312_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1312_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1312_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1312_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1312_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1312_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1312_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,961 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.22 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,961\n",
      "  Read 1 with adapter:                  57,887 (99.9%)\n",
      "  Read 2 with adapter:                  57,720 (99.6%)\n",
      "Pairs written (passing filters):        57,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,186,906 bp\n",
      "  Read 1:    17,094,804 bp\n",
      "  Read 2:    17,092,102 bp\n",
      "Total written (filtered):     31,931,076 bp (93.4%)\n",
      "  Read 1:    15,935,419 bp\n",
      "  Read 2:    15,995,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t388\t0.0\t1\t0 380 8\n",
      "20\t57346\t0.0\t2\t56109 1227 10\n",
      "21\t110\t0.0\t2\t0 110\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57720 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t354\t0.0\t1\t0 354\n",
      "19\t57247\t0.0\t1\t55963 1284\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1313_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1313_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1313_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1313_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1313_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1313_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1313_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1313_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:07       110,141 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 7.35 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,141\n",
      "  Read 1 with adapter:                 110,026 (99.9%)\n",
      "  Read 2 with adapter:                 109,464 (99.4%)\n",
      "Pairs written (passing filters):       110,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,935,117 bp\n",
      "  Read 1:    32,486,329 bp\n",
      "  Read 2:    32,448,788 bp\n",
      "Total written (filtered):     60,655,245 bp (93.4%)\n",
      "  Read 1:    30,285,979 bp\n",
      "  Read 2:    30,369,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 110026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t716\t0.0\t1\t0 702 14\n",
      "20\t109012\t0.0\t2\t106876 2118 18\n",
      "21\t205\t0.0\t2\t0 203 2\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109464 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t491\t0.0\t1\t0 491\n",
      "19\t108776\t0.0\t1\t106432 2344\n",
      "20\t197\t0.0\t1\t0 197\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1314_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1314_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1314_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1314_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1314_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1314_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1314_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1314_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,524 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.26 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,524\n",
      "  Read 1 with adapter:                  48,480 (99.9%)\n",
      "  Read 2 with adapter:                  48,184 (99.3%)\n",
      "Pairs written (passing filters):        48,524 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,598,926 bp\n",
      "  Read 1:    14,312,903 bp\n",
      "  Read 2:    14,286,023 bp\n",
      "Total written (filtered):     26,714,161 bp (93.4%)\n",
      "  Read 1:    13,343,509 bp\n",
      "  Read 2:    13,370,652 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t285\t0.0\t1\t0 275 10\n",
      "20\t48077\t0.0\t2\t47156 909 12\n",
      "21\t93\t0.0\t2\t0 92 1\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t222\t0.0\t1\t0 222\n",
      "19\t47865\t0.0\t1\t46935 930\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1315_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1315_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1315_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1315_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1315_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1315_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1315_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1315_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        73,752 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 5.36 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,752\n",
      "  Read 1 with adapter:                  73,460 (99.6%)\n",
      "  Read 2 with adapter:                  68,953 (93.5%)\n",
      "Pairs written (passing filters):        73,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,207,746 bp\n",
      "  Read 1:    21,739,310 bp\n",
      "  Read 2:    21,468,436 bp\n",
      "Total written (filtered):     40,428,927 bp (93.6%)\n",
      "  Read 1:    20,270,374 bp\n",
      "  Read 2:    20,158,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73460 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t431\t0.0\t1\t0 420 11\n",
      "20\t72833\t0.0\t2\t71247 1572 14\n",
      "21\t125\t0.0\t2\t0 123 2\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 68953 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t342\t0.0\t1\t0 342\n",
      "19\t68493\t0.0\t1\t67160 1333\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1316_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1316_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1316_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1316_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1316_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1316_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1316_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1316_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        56,744 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.90 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,744\n",
      "  Read 1 with adapter:                  56,705 (99.9%)\n",
      "  Read 2 with adapter:                  56,438 (99.5%)\n",
      "Pairs written (passing filters):        56,744 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,473,540 bp\n",
      "  Read 1:    16,737,108 bp\n",
      "  Read 2:    16,736,432 bp\n",
      "Total written (filtered):     31,267,484 bp (93.4%)\n",
      "  Read 1:    15,603,191 bp\n",
      "  Read 2:    15,664,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56705 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t293\t0.0\t1\t0 287 6\n",
      "20\t56305\t0.0\t2\t55276 1019 10\n",
      "21\t80\t0.0\t2\t0 80\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56438 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t273\t0.0\t1\t0 273\n",
      "19\t56075\t0.0\t1\t54716 1359\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1317_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1317_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1317_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1317_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1317_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1317_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1317_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1317_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        94,727 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 6.49 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,727\n",
      "  Read 1 with adapter:                  94,643 (99.9%)\n",
      "  Read 2 with adapter:                  94,286 (99.5%)\n",
      "Pairs written (passing filters):        94,727 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,880,206 bp\n",
      "  Read 1:    27,938,894 bp\n",
      "  Read 2:    27,941,312 bp\n",
      "Total written (filtered):     52,196,568 bp (93.4%)\n",
      "  Read 1:    26,046,376 bp\n",
      "  Read 2:    26,150,192 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 94643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t587\t0.0\t1\t0 576 11\n",
      "20\t93798\t0.0\t2\t92029 1755 14\n",
      "21\t171\t0.0\t2\t0 166 5\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94286 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t489\t0.0\t1\t0 489\n",
      "19\t93622\t0.0\t1\t91429 2193\n",
      "20\t175\t0.0\t1\t0 175\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1318_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1318_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1318_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1318_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1318_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1318_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1318_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1318_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,371 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.08 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,371\n",
      "  Read 1 with adapter:                  58,327 (99.9%)\n",
      "  Read 2 with adapter:                  58,123 (99.6%)\n",
      "Pairs written (passing filters):        58,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,433,673 bp\n",
      "  Read 1:    17,216,705 bp\n",
      "  Read 2:    17,216,968 bp\n",
      "Total written (filtered):     32,163,215 bp (93.4%)\n",
      "  Read 1:    16,050,384 bp\n",
      "  Read 2:    16,112,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t342\t0.0\t1\t0 333 9\n",
      "20\t57849\t0.0\t2\t56726 1107 16\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t301\t0.0\t1\t0 301\n",
      "19\t57721\t0.0\t1\t56414 1307\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1319_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1319_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1319_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1319_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1319_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1319_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1319_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1319_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        41,970 reads  @     83.3 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.50 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,970\n",
      "  Read 1 with adapter:                  41,895 (99.8%)\n",
      "  Read 2 with adapter:                  41,785 (99.6%)\n",
      "Pairs written (passing filters):        41,970 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,755,405 bp\n",
      "  Read 1:    12,375,108 bp\n",
      "  Read 2:    12,380,297 bp\n",
      "Total written (filtered):     23,123,909 bp (93.4%)\n",
      "  Read 1:    11,537,353 bp\n",
      "  Read 2:    11,586,556 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41895 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t273\t0.0\t1\t0 267 6\n",
      "20\t41499\t0.0\t2\t40688 794 17\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t275\t0.0\t1\t0 275\n",
      "19\t41409\t0.0\t1\t40379 1030\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1320_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1320_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1320_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1320_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1320_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1320_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1320_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1320_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,664 reads  @     78.6 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.91 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,664\n",
      "  Read 1 with adapter:                  49,622 (99.9%)\n",
      "  Read 2 with adapter:                  49,435 (99.5%)\n",
      "Pairs written (passing filters):        49,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,290,789 bp\n",
      "  Read 1:    14,647,433 bp\n",
      "  Read 2:    14,643,356 bp\n",
      "Total written (filtered):     27,358,630 bp (93.4%)\n",
      "  Read 1:    13,654,365 bp\n",
      "  Read 2:    13,704,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t311\t0.0\t1\t0 301 10\n",
      "20\t49187\t0.0\t2\t48170 1000 17\n",
      "21\t86\t0.0\t2\t0 83 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t257\t0.0\t1\t0 257\n",
      "19\t49095\t0.0\t1\t47757 1338\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1321_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1321_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1321_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1321_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1321_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1321_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1321_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1321_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,594 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.40 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,594\n",
      "  Read 1 with adapter:                  59,529 (99.9%)\n",
      "  Read 2 with adapter:                  56,736 (95.2%)\n",
      "Pairs written (passing filters):        59,594 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,600,399 bp\n",
      "  Read 1:    17,577,052 bp\n",
      "  Read 2:    17,023,347 bp\n",
      "Total written (filtered):     32,332,231 bp (93.4%)\n",
      "  Read 1:    16,386,706 bp\n",
      "  Read 2:    15,945,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59529 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t369\t0.0\t1\t0 357 12\n",
      "20\t59005\t0.0\t2\t57789 1203 13\n",
      "21\t103\t0.0\t2\t0 100 3\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56736 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t266\t0.0\t1\t0 266\n",
      "19\t56366\t0.0\t1\t55095 1271\n",
      "20\t104\t0.0\t1\t0 104\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1322_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1322_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1322_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1322_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1322_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1322_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1322_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1322_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        52,592 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.95 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,592\n",
      "  Read 1 with adapter:                  52,549 (99.9%)\n",
      "  Read 2 with adapter:                  52,236 (99.3%)\n",
      "Pairs written (passing filters):        52,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,994,459 bp\n",
      "  Read 1:    15,512,429 bp\n",
      "  Read 2:    15,482,030 bp\n",
      "Total written (filtered):     28,951,338 bp (93.4%)\n",
      "  Read 1:    14,461,634 bp\n",
      "  Read 2:    14,489,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52549 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t295\t0.0\t1\t0 289 6\n",
      "20\t52120\t0.0\t2\t51049 1058 13\n",
      "21\t82\t0.0\t2\t0 80 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t289\t0.0\t1\t0 289\n",
      "19\t51816\t0.0\t1\t50635 1181\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1323_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1323_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1323_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1323_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1323_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1323_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1323_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1323_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,155 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 5.16 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,155\n",
      "  Read 1 with adapter:                  65,113 (99.9%)\n",
      "  Read 2 with adapter:                  63,063 (96.8%)\n",
      "Pairs written (passing filters):        65,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,316,817 bp\n",
      "  Read 1:    19,218,045 bp\n",
      "  Read 2:    19,098,772 bp\n",
      "Total written (filtered):     35,816,801 bp (93.5%)\n",
      "  Read 1:    17,916,005 bp\n",
      "  Read 2:    17,900,796 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t371\t0.0\t1\t0 362 9\n",
      "20\t64578\t0.0\t2\t63264 1298 16\n",
      "21\t101\t0.0\t2\t0 99 2\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63063 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t328\t0.0\t1\t0 328\n",
      "19\t62628\t0.0\t1\t61278 1350\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1324_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1324_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1324_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1324_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1324_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1324_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1324_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1324_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,747 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.97 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,747\n",
      "  Read 1 with adapter:                  61,703 (99.9%)\n",
      "  Read 2 with adapter:                  60,383 (97.8%)\n",
      "Pairs written (passing filters):        61,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,415,934 bp\n",
      "  Read 1:    18,215,156 bp\n",
      "  Read 2:    18,200,778 bp\n",
      "Total written (filtered):     34,034,970 bp (93.5%)\n",
      "  Read 1:    16,981,158 bp\n",
      "  Read 2:    17,053,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t422\t0.0\t1\t0 398 24\n",
      "20\t60997\t0.0\t2\t59375 1595 27\n",
      "21\t124\t0.0\t2\t0 104 20\n",
      "22\t139\t0.0\t2\t0 0 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t436\t0.0\t1\t0 436\n",
      "19\t59822\t0.0\t1\t56950 2872\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1340_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1340_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1340_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1340_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1340_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1340_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1340_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1340_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           254 reads  @    173.8 µs/read;   0.35 M reads/minute\n",
      "Finished in 0.05 s (186 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                254\n",
      "  Read 1 with adapter:                     254 (100.0%)\n",
      "  Read 2 with adapter:                      23 (9.1%)\n",
      "Pairs written (passing filters):           254 (100.0%)\n",
      "\n",
      "Total basepairs processed:        88,544 bp\n",
      "  Read 1:        74,930 bp\n",
      "  Read 2:        13,614 bp\n",
      "Total written (filtered):         83,002 bp (93.7%)\n",
      "  Read 1:        69,848 bp\n",
      "  Read 2:        13,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 254 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t253\t0.0\t2\t246 7\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t23\t0.0\t1\t0 23\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1341_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1341_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1341_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1341_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1341_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1341_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1341_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1341_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,597 reads  @     80.8 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.63 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,597\n",
      "  Read 1 with adapter:                  69,531 (99.9%)\n",
      "  Read 2 with adapter:                  68,978 (99.1%)\n",
      "Pairs written (passing filters):        69,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,986,627 bp\n",
      "  Read 1:    20,524,806 bp\n",
      "  Read 2:    20,461,821 bp\n",
      "Total written (filtered):     38,285,693 bp (93.4%)\n",
      "  Read 1:    19,134,242 bp\n",
      "  Read 2:    19,151,451 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t476\t0.0\t1\t0 462 14\n",
      "20\t68878\t0.0\t2\t67528 1336 14\n",
      "21\t121\t0.0\t2\t0 119 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 68978 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t355\t0.0\t1\t0 355\n",
      "19\t68480\t0.0\t1\t66827 1653\n",
      "20\t143\t0.0\t1\t0 143\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1342_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1342_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1342_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1342_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1342_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1342_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1342_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1342_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,029 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.83 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,029\n",
      "  Read 1 with adapter:                  61,977 (99.9%)\n",
      "  Read 2 with adapter:                  61,772 (99.6%)\n",
      "Pairs written (passing filters):        62,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,579,259 bp\n",
      "  Read 1:    18,292,015 bp\n",
      "  Read 2:    18,287,244 bp\n",
      "Total written (filtered):     34,166,447 bp (93.4%)\n",
      "  Read 1:    17,052,703 bp\n",
      "  Read 2:    17,113,744 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61977 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t381\t0.0\t1\t0 368 13\n",
      "20\t61431\t0.0\t2\t60201 1220 10\n",
      "21\t113\t0.0\t2\t0 113\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t275\t0.0\t1\t0 275\n",
      "19\t61390\t0.0\t1\t60010 1380\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1343_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1343_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1343_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1343_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1343_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1343_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1343_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1343_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,762 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.81 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,762\n",
      "  Read 1 with adapter:                  57,739 (100.0%)\n",
      "  Read 2 with adapter:                  57,296 (99.2%)\n",
      "Pairs written (passing filters):        57,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,073,138 bp\n",
      "  Read 1:    17,039,382 bp\n",
      "  Read 2:    17,033,756 bp\n",
      "Total written (filtered):     31,830,089 bp (93.4%)\n",
      "  Read 1:    15,884,728 bp\n",
      "  Read 2:    15,945,361 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t402\t0.0\t1\t0 390 12\n",
      "20\t57085\t0.0\t2\t55667 1406 12\n",
      "21\t120\t0.0\t2\t0 118 2\n",
      "22\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t388\t0.0\t1\t0 388\n",
      "19\t56749\t0.0\t1\t55063 1686\n",
      "20\t159\t0.0\t1\t0 159\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1344_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1344_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1344_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1344_141_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1344_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1344_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1344_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1344_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           172 reads  @    304.9 µs/read;   0.20 M reads/minute\n",
      "Finished in 0.06 s (323 us/read; 0.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                172\n",
      "  Read 1 with adapter:                     172 (100.0%)\n",
      "  Read 2 with adapter:                      13 (7.6%)\n",
      "Pairs written (passing filters):           172 (100.0%)\n",
      "\n",
      "Total basepairs processed:        59,677 bp\n",
      "  Read 1:        50,740 bp\n",
      "  Read 2:         8,937 bp\n",
      "Total written (filtered):         55,977 bp (93.8%)\n",
      "  Read 1:        47,300 bp\n",
      "  Read 2:         8,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 172 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t172\t0.0\t2\t169 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1345_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1345_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1345_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1345_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1345_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1345_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1345_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1345_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,908 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.11 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,908\n",
      "  Read 1 with adapter:                  56,877 (99.9%)\n",
      "  Read 2 with adapter:                  56,498 (99.3%)\n",
      "Pairs written (passing filters):        56,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,522,988 bp\n",
      "  Read 1:    16,786,895 bp\n",
      "  Read 2:    16,736,093 bp\n",
      "Total written (filtered):     31,312,379 bp (93.4%)\n",
      "  Read 1:    15,649,557 bp\n",
      "  Read 2:    15,662,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56877 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t344\t0.0\t1\t0 342 2\n",
      "20\t56369\t0.0\t2\t55327 1032 10\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56498 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t277\t0.0\t1\t0 277\n",
      "19\t56135\t0.0\t1\t54973 1162\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1346_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1346_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1346_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1346_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1346_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1346_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1346_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1346_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,913 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.14 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,913\n",
      "  Read 1 with adapter:                  63,876 (99.9%)\n",
      "  Read 2 with adapter:                  63,714 (99.7%)\n",
      "Pairs written (passing filters):        63,913 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,705,051 bp\n",
      "  Read 1:    18,851,162 bp\n",
      "  Read 2:    18,853,889 bp\n",
      "Total written (filtered):     35,217,372 bp (93.4%)\n",
      "  Read 1:    17,573,847 bp\n",
      "  Read 2:    17,643,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t380\t0.0\t1\t0 370 10\n",
      "20\t63338\t0.0\t2\t61955 1366 17\n",
      "21\t105\t0.0\t2\t0 103 2\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63714 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t321\t0.0\t1\t0 321\n",
      "19\t63274\t0.0\t1\t61740 1534\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1347_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1347_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1347_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1347_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1347_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1347_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1347_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1347_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        54,521 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.16 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,521\n",
      "  Read 1 with adapter:                  54,496 (100.0%)\n",
      "  Read 2 with adapter:                  54,343 (99.7%)\n",
      "Pairs written (passing filters):        54,521 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,161,358 bp\n",
      "  Read 1:    16,083,066 bp\n",
      "  Read 2:    16,078,292 bp\n",
      "Total written (filtered):     30,039,325 bp (93.4%)\n",
      "  Read 1:    14,993,379 bp\n",
      "  Read 2:    15,045,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t369\t0.0\t1\t0 358 11\n",
      "20\t53988\t0.0\t2\t52988 985 15\n",
      "21\t90\t0.0\t2\t0 89 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54343 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t306\t0.0\t1\t0 306\n",
      "19\t53902\t0.0\t1\t52888 1014\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1348_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1348_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1348_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1348_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1348_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1348_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1348_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1348_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           340 reads  @    202.9 µs/read;   0.30 M reads/minute\n",
      "Finished in 0.07 s (213 us/read; 0.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                340\n",
      "  Read 1 with adapter:                     292 (85.9%)\n",
      "  Read 2 with adapter:                      18 (5.3%)\n",
      "Pairs written (passing filters):           339 (99.7%)\n",
      "\n",
      "Total basepairs processed:       124,240 bp\n",
      "  Read 1:       100,300 bp\n",
      "  Read 2:        23,940 bp\n",
      "Total written (filtered):        117,761 bp (94.8%)\n",
      "  Read 1:        94,181 bp\n",
      "  Read 2:        23,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t288\t0.0\t2\t281 7\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t18\t0.0\t1\t0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1349_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1349_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1349_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1349_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1349_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1349_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1349_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1349_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        20,159 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 2.13 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,159\n",
      "  Read 1 with adapter:                  20,132 (99.9%)\n",
      "  Read 2 with adapter:                  20,065 (99.5%)\n",
      "Pairs written (passing filters):        20,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,885,943 bp\n",
      "  Read 1:     5,945,042 bp\n",
      "  Read 2:     5,940,901 bp\n",
      "Total written (filtered):     11,102,235 bp (93.4%)\n",
      "  Read 1:     5,542,460 bp\n",
      "  Read 2:     5,559,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 20132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t135\t0.0\t1\t0 133 2\n",
      "20\t19936\t0.0\t2\t19355 574 7\n",
      "21\t33\t0.0\t2\t0 30 3\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 20065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t161\t0.0\t1\t0 161\n",
      "19\t19852\t0.0\t1\t19286 566\n",
      "20\t52\t0.0\t1\t0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1350_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1350_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1350_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1350_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1350_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1350_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1350_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1350_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        93,085 reads  @     96.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 8.95 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,085\n",
      "  Read 1 with adapter:                  70,008 (75.2%)\n",
      "  Read 2 with adapter:                  69,757 (74.9%)\n",
      "Pairs written (passing filters):        93,085 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,911,427 bp\n",
      "  Read 1:    27,455,340 bp\n",
      "  Read 2:    27,456,087 bp\n",
      "Total written (filtered):     52,186,330 bp (95.0%)\n",
      "  Read 1:    26,055,412 bp\n",
      "  Read 2:    26,130,918 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t384\t0.0\t1\t0 380 4\n",
      "20\t69455\t0.0\t2\t68166 1272 17\n",
      "21\t110\t0.0\t2\t0 106 4\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69757 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t334\t0.0\t1\t0 334\n",
      "19\t69303\t0.0\t1\t67323 1980\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1351_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1351_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1351_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1351_155_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1351_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1351_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1351_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1351_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        67,033 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 6.03 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,033\n",
      "  Read 1 with adapter:                  52,188 (77.9%)\n",
      "  Read 2 with adapter:                  52,000 (77.6%)\n",
      "Pairs written (passing filters):        67,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,537,832 bp\n",
      "  Read 1:    19,767,914 bp\n",
      "  Read 2:    19,769,918 bp\n",
      "Total written (filtered):     37,506,349 bp (94.9%)\n",
      "  Read 1:    18,724,306 bp\n",
      "  Read 2:    18,782,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52188 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t280\t0.0\t1\t0 274 6\n",
      "20\t51777\t0.0\t2\t50687 1080 10\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t270\t0.0\t1\t0 270\n",
      "19\t51585\t0.0\t1\t50289 1296\n",
      "20\t145\t0.0\t1\t0 145\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1352_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1352_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1352_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1352_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1352_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1352_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1352_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1352_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           371 reads  @    156.5 µs/read;   0.38 M reads/minute\n",
      "Finished in 0.06 s (165 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                371\n",
      "  Read 1 with adapter:                     318 (85.7%)\n",
      "  Read 2 with adapter:                      21 (5.7%)\n",
      "Pairs written (passing filters):           371 (100.0%)\n",
      "\n",
      "Total basepairs processed:       126,961 bp\n",
      "  Read 1:       109,177 bp\n",
      "  Read 2:        17,784 bp\n",
      "Total written (filtered):        120,183 bp (94.7%)\n",
      "  Read 1:       102,818 bp\n",
      "  Read 2:        17,365 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t315\t0.0\t2\t309 6\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t20\t0.0\t1\t0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1353_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1353_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1353_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1353_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1353_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1353_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1353_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1353_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        70,616 reads  @     94.4 µs/read;   0.64 M reads/minute\n",
      "Finished in 6.67 s (94 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,616\n",
      "  Read 1 with adapter:                  54,086 (76.6%)\n",
      "  Read 2 with adapter:                  53,653 (76.0%)\n",
      "Pairs written (passing filters):        70,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,623,276 bp\n",
      "  Read 1:    20,830,099 bp\n",
      "  Read 2:    20,793,177 bp\n",
      "Total written (filtered):     39,522,566 bp (95.0%)\n",
      "  Read 1:    19,748,646 bp\n",
      "  Read 2:    19,773,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t396\t0.0\t1\t0 394 2\n",
      "20\t53545\t0.0\t2\t52551 986 8\n",
      "21\t97\t0.0\t2\t0 97\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53653 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t256\t0.0\t1\t0 256\n",
      "19\t53291\t0.0\t1\t51830 1461\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1354_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1354_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1354_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1354_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1354_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1354_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1354_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1354_161_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,051 reads  @     82.3 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.87 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,051\n",
      "  Read 1 with adapter:                  49,554 (83.9%)\n",
      "  Read 2 with adapter:                  49,360 (83.6%)\n",
      "Pairs written (passing filters):        59,051 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,838,603 bp\n",
      "  Read 1:    17,419,547 bp\n",
      "  Read 2:    17,419,056 bp\n",
      "Total written (filtered):     32,910,056 bp (94.5%)\n",
      "  Read 1:    16,428,705 bp\n",
      "  Read 2:    16,481,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49554 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t346\t0.0\t1\t0 345 1\n",
      "20\t49080\t0.0\t2\t48167 903 10\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49360 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t49013\t0.0\t1\t47679 1334\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1355_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1355_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1355_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1355_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1355_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1355_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1355_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1355_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,229 reads  @     83.4 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.44 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,229\n",
      "  Read 1 with adapter:                  43,719 (82.1%)\n",
      "  Read 2 with adapter:                  43,452 (81.6%)\n",
      "Pairs written (passing filters):        53,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,396,528 bp\n",
      "  Read 1:    15,702,328 bp\n",
      "  Read 2:    15,694,200 bp\n",
      "Total written (filtered):     29,696,917 bp (94.6%)\n",
      "  Read 1:    14,828,159 bp\n",
      "  Read 2:    14,868,758 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43719 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t300\t0.0\t1\t0 290 10\n",
      "20\t43310\t0.0\t2\t42504 796 10\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t259\t0.0\t1\t0 259\n",
      "19\t43080\t0.0\t1\t41984 1096\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1356_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1356_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1356_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1356_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1356_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1356_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1356_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1356_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           206 reads  @    202.1 µs/read;   0.30 M reads/minute\n",
      "Finished in 0.04 s (216 us/read; 0.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                206\n",
      "  Read 1 with adapter:                     202 (98.1%)\n",
      "  Read 2 with adapter:                      18 (8.7%)\n",
      "Pairs written (passing filters):           206 (100.0%)\n",
      "\n",
      "Total basepairs processed:        70,400 bp\n",
      "  Read 1:        60,435 bp\n",
      "  Read 2:         9,965 bp\n",
      "Total written (filtered):         65,999 bp (93.7%)\n",
      "  Read 1:        56,394 bp\n",
      "  Read 2:         9,605 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 202 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t199\t0.0\t2\t194 5\n",
      "21\t2\t0.0\t2\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 18 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t18\t0.0\t1\t0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1357_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1357_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1357_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1357_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1357_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1357_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1357_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1357_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,155 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.49 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,155\n",
      "  Read 1 with adapter:                  48,134 (100.0%)\n",
      "  Read 2 with adapter:                  47,796 (99.3%)\n",
      "Pairs written (passing filters):        48,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,369,087 bp\n",
      "  Read 1:    14,205,451 bp\n",
      "  Read 2:    14,163,636 bp\n",
      "Total written (filtered):     26,498,605 bp (93.4%)\n",
      "  Read 1:    13,242,971 bp\n",
      "  Read 2:    13,255,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48134 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t286\t0.0\t1\t0 279 7\n",
      "20\t47739\t0.0\t2\t46855 880 4\n",
      "21\t76\t0.0\t2\t0 75 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47796 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t47494\t0.0\t1\t46534 960\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1358_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1358_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1358_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1358_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1358_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1358_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1358_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1358_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,773 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.41 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,773\n",
      "  Read 1 with adapter:                  49,744 (99.9%)\n",
      "  Read 2 with adapter:                  49,616 (99.7%)\n",
      "Pairs written (passing filters):        49,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,365,083 bp\n",
      "  Read 1:    14,682,449 bp\n",
      "  Read 2:    14,682,634 bp\n",
      "Total written (filtered):     27,427,858 bp (93.4%)\n",
      "  Read 1:    13,687,763 bp\n",
      "  Read 2:    13,740,095 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t293\t0.0\t1\t0 291 2\n",
      "20\t49333\t0.0\t2\t48365 945 23\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t250\t0.0\t1\t0 250\n",
      "19\t49281\t0.0\t1\t48279 1002\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1359_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1359_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1359_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1359_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1359_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1359_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1359_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1359_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,118 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.20 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,118\n",
      "  Read 1 with adapter:                  47,084 (99.9%)\n",
      "  Read 2 with adapter:                  46,972 (99.7%)\n",
      "Pairs written (passing filters):        47,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,795,155 bp\n",
      "  Read 1:    13,898,001 bp\n",
      "  Read 2:    13,897,154 bp\n",
      "Total written (filtered):     25,961,298 bp (93.4%)\n",
      "  Read 1:    12,956,507 bp\n",
      "  Read 2:    13,004,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47084 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t283\t0.0\t1\t0 279 4\n",
      "20\t46667\t0.0\t2\t45816 838 13\n",
      "21\t79\t0.0\t2\t0 76 3\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t231\t0.0\t1\t0 231\n",
      "19\t46615\t0.0\t1\t45781 834\n",
      "20\t126\t0.0\t1\t0 126\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1360_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1360_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1360_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1360_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1360_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1360_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1360_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1360_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           153 reads  @    258.0 µs/read;   0.23 M reads/minute\n",
      "Finished in 0.04 s (279 us/read; 0.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                153\n",
      "  Read 1 with adapter:                     138 (90.2%)\n",
      "  Read 2 with adapter:                      21 (13.7%)\n",
      "Pairs written (passing filters):           153 (100.0%)\n",
      "\n",
      "Total basepairs processed:        56,944 bp\n",
      "  Read 1:        43,507 bp\n",
      "  Read 2:        13,437 bp\n",
      "Total written (filtered):         53,764 bp (94.4%)\n",
      "  Read 1:        40,747 bp\n",
      "  Read 2:        13,017 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t138\t0.0\t2\t135 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t21\t0.0\t1\t0 21\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1361_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1361_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1361_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1361_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1361_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1361_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1361_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1361_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        65,804 reads  @     64.2 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.23 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,804\n",
      "  Read 1 with adapter:                  65,718 (99.9%)\n",
      "  Read 2 with adapter:                  65,329 (99.3%)\n",
      "Pairs written (passing filters):        65,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,747,987 bp\n",
      "  Read 1:    19,405,447 bp\n",
      "  Read 2:    19,342,540 bp\n",
      "Total written (filtered):     36,192,777 bp (93.4%)\n",
      "  Read 1:    18,091,265 bp\n",
      "  Read 2:    18,101,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t329\t0.0\t1\t0 323 6\n",
      "20\t65211\t0.0\t2\t64098 1104 9\n",
      "21\t129\t0.0\t2\t0 127 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65329 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t332\t0.0\t1\t0 332\n",
      "19\t64888\t0.0\t1\t63653 1235\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1362_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1362_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1362_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1362_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1362_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1362_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1362_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1362_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,297 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 4.08 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,297\n",
      "  Read 1 with adapter:                  63,254 (99.9%)\n",
      "  Read 2 with adapter:                  63,072 (99.6%)\n",
      "Pairs written (passing filters):        63,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,343,518 bp\n",
      "  Read 1:    18,672,043 bp\n",
      "  Read 2:    18,671,475 bp\n",
      "Total written (filtered):     34,880,436 bp (93.4%)\n",
      "  Read 1:    17,407,137 bp\n",
      "  Read 2:    17,473,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63254 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t345\t0.0\t1\t0 339 6\n",
      "20\t62720\t0.0\t2\t61592 1115 13\n",
      "21\t131\t0.0\t2\t0 129 2\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63072 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t302\t0.0\t1\t0 302\n",
      "19\t62660\t0.0\t1\t61456 1204\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1363_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1363_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1363_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1363_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1363_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1363_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1363_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1363_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,183 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.05 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,183\n",
      "  Read 1 with adapter:                  57,788 (99.3%)\n",
      "  Read 2 with adapter:                  57,998 (99.7%)\n",
      "Pairs written (passing filters):        58,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,269,546 bp\n",
      "  Read 1:    17,112,134 bp\n",
      "  Read 2:    17,157,412 bp\n",
      "Total written (filtered):     32,012,085 bp (93.4%)\n",
      "  Read 1:    15,956,504 bp\n",
      "  Read 2:    16,055,581 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57788 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t364\t0.0\t1\t0 355 9\n",
      "20\t57200\t0.0\t2\t56066 1119 15\n",
      "21\t154\t0.0\t2\t0 150 4\n",
      "22\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57998 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t289\t0.0\t1\t0 289\n",
      "19\t57551\t0.0\t1\t56408 1143\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1364_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1364_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1364_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1364_181_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1364_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1364_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1364_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1364_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            82 reads  @    499.4 µs/read;   0.12 M reads/minute\n",
      "Finished in 0.04 s (540 us/read; 0.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 82\n",
      "  Read 1 with adapter:                      82 (100.0%)\n",
      "  Read 2 with adapter:                      13 (15.9%)\n",
      "Pairs written (passing filters):            82 (100.0%)\n",
      "\n",
      "Total basepairs processed:        30,515 bp\n",
      "  Read 1:        24,190 bp\n",
      "  Read 2:         6,325 bp\n",
      "Total written (filtered):         28,617 bp (93.8%)\n",
      "  Read 1:        22,552 bp\n",
      "  Read 2:         6,065 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 82 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "20\t81\t0.0\t2\t80 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1365_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1365_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1365_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1365_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1365_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1365_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1365_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1365_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,762 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.37 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,762\n",
      "  Read 1 with adapter:                  33,711 (99.8%)\n",
      "  Read 2 with adapter:                  33,371 (98.8%)\n",
      "Pairs written (passing filters):        33,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,859,518 bp\n",
      "  Read 1:     9,959,394 bp\n",
      "  Read 2:     9,900,124 bp\n",
      "Total written (filtered):     18,551,422 bp (93.4%)\n",
      "  Read 1:     9,285,263 bp\n",
      "  Read 2:     9,266,159 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33711 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t204\t0.0\t1\t0 200 4\n",
      "20\t33396\t0.0\t2\t32722 665 9\n",
      "21\t75\t0.0\t2\t0 75\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33371 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t141\t0.0\t1\t0 141\n",
      "19\t33173\t0.0\t1\t32400 773\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1366_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1366_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1366_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1366_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1366_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1366_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1366_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1366_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        26,354 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.80 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,354\n",
      "  Read 1 with adapter:                  26,330 (99.9%)\n",
      "  Read 2 with adapter:                  26,266 (99.7%)\n",
      "Pairs written (passing filters):        26,354 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,548,719 bp\n",
      "  Read 1:     7,774,304 bp\n",
      "  Read 2:     7,774,415 bp\n",
      "Total written (filtered):     14,523,254 bp (93.4%)\n",
      "  Read 1:     7,247,803 bp\n",
      "  Read 2:     7,275,451 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26330 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t176\t0.0\t1\t0 170 6\n",
      "20\t26066\t0.0\t2\t25557 507 2\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t126\t0.0\t1\t0 126\n",
      "19\t26104\t0.0\t1\t25520 584\n",
      "20\t36\t0.0\t1\t0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1367_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1367_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1367_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1367_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1367_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1367_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1367_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1367_187_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,943 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.54 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,943\n",
      "  Read 1 with adapter:                  54,875 (99.9%)\n",
      "  Read 2 with adapter:                  54,682 (99.5%)\n",
      "Pairs written (passing filters):        54,943 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,410,356 bp\n",
      "  Read 1:    16,206,848 bp\n",
      "  Read 2:    16,203,508 bp\n",
      "Total written (filtered):     30,274,258 bp (93.4%)\n",
      "  Read 1:    15,109,550 bp\n",
      "  Read 2:    15,164,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54875 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t345\t0.0\t1\t0 340 5\n",
      "20\t54363\t0.0\t2\t53223 1129 11\n",
      "21\t111\t0.0\t2\t0 108 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54682 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t277\t0.0\t1\t0 277\n",
      "19\t54286\t0.0\t1\t53123 1163\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1368_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1368_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1368_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1368_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1368_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1368_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1368_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1368_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           202 reads  @    249.0 µs/read;   0.24 M reads/minute\n",
      "Finished in 0.05 s (264 us/read; 0.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                202\n",
      "  Read 1 with adapter:                     200 (99.0%)\n",
      "  Read 2 with adapter:                      20 (9.9%)\n",
      "Pairs written (passing filters):           201 (99.5%)\n",
      "\n",
      "Total basepairs processed:        70,343 bp\n",
      "  Read 1:        59,417 bp\n",
      "  Read 2:        10,926 bp\n",
      "Total written (filtered):         65,673 bp (93.4%)\n",
      "  Read 1:        55,147 bp\n",
      "  Read 2:        10,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t3\t0.0\t1\t0 2 1\n",
      "20\t196\t0.0\t2\t190 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t20\t0.0\t1\t0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1369_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1369_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1369_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1369_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1369_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1369_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1369_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1369_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,026 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.12 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,026\n",
      "  Read 1 with adapter:                  55,989 (99.9%)\n",
      "  Read 2 with adapter:                  55,515 (99.1%)\n",
      "Pairs written (passing filters):        56,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,991,492 bp\n",
      "  Read 1:    16,525,056 bp\n",
      "  Read 2:    16,466,436 bp\n",
      "Total written (filtered):     30,817,006 bp (93.4%)\n",
      "  Read 1:    15,405,137 bp\n",
      "  Read 2:    15,411,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55989 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t511\t0.0\t1\t0 502 9\n",
      "20\t55320\t0.0\t2\t54245 1059 16\n",
      "21\t104\t0.0\t2\t0 101 3\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55515 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t320\t0.0\t1\t0 320\n",
      "19\t55093\t0.0\t1\t53803 1290\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1370_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1370_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1370_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1370_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1370_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1370_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1370_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1370_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,400 reads  @     74.0 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.95 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,400\n",
      "  Read 1 with adapter:                  53,344 (99.9%)\n",
      "  Read 2 with adapter:                  53,213 (99.6%)\n",
      "Pairs written (passing filters):        53,400 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,499,642 bp\n",
      "  Read 1:    15,747,679 bp\n",
      "  Read 2:    15,751,963 bp\n",
      "Total written (filtered):     29,422,193 bp (93.4%)\n",
      "  Read 1:    14,681,114 bp\n",
      "  Read 2:    14,741,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53344 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t459\t0.0\t1\t0 449 10\n",
      "20\t52734\t0.0\t2\t51619 1110 5\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t52874\t0.0\t1\t51638 1236\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1371_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1371_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1371_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1371_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1371_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-a2or2_zx/S1371_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1371_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kyi38g0t/d2ac9b47-14e1-49d7-b8d3-c08b31beffa5/data/S1371_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,569 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.19 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,569\n",
      "  Read 1 with adapter:                  55,524 (99.9%)\n",
      "  Read 2 with adapter:                  55,275 (99.5%)\n",
      "Pairs written (passing filters):        55,569 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,774,531 bp\n",
      "  Read 1:    16,389,817 bp\n",
      "  Read 2:    16,384,714 bp\n",
      "Total written (filtered):     30,614,479 bp (93.4%)\n",
      "  Read 1:    15,279,776 bp\n",
      "  Read 2:    15,334,703 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t581\t0.0\t1\t0 572 9\n",
      "20\t54767\t0.0\t2\t53591 1167 9\n",
      "21\t94\t0.0\t2\t0 94\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55275 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t339\t0.0\t1\t0 339\n",
      "19\t54811\t0.0\t1\t53461 1350\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: e-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: e-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-wle3564i')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: ca155b91-3408-4945-a072-ec6d61c478d8>"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.e import the fastq files for library e into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/Hirai-Lib1-manifest_18S.csv \\\n",
    "  --output-path e-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.e trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences e-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences e-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.e summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data e-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization e-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('e-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: e-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: e-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: e-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.e Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs e-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 267 \\\n",
    "  --p-trunc-len-r 222 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table e-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats e-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: e-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-pze9hqbu')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 474dc4cc-f1ea-4f31-963c-cecc484b199d>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.e view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table e-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization e-table.qzv \n",
    "q2.Visualization.load('e-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: e-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.e view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization e-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/Hirai-Lib2-manifest_18S.csv as PairedEndFastqManifestPhred33 to f-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1645_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1645_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1645_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1645_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1645_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1645_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1645_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1645_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       107,960 reads  @    100.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 10.84 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,960\n",
      "  Read 1 with adapter:                  92,230 (85.4%)\n",
      "  Read 2 with adapter:                  91,799 (85.0%)\n",
      "Pairs written (passing filters):       107,960 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,683,854 bp\n",
      "  Read 1:    31,841,836 bp\n",
      "  Read 2:    31,842,018 bp\n",
      "Total written (filtered):     60,095,762 bp (94.4%)\n",
      "  Read 1:    29,997,642 bp\n",
      "  Read 2:    30,098,120 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 92230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t570\t0.0\t1\t0 557 13\n",
      "20\t91462\t0.0\t2\t89508 1913 41\n",
      "21\t132\t0.0\t2\t0 130 2\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 91799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t444\t0.0\t1\t0 444\n",
      "19\t91194\t0.0\t1\t88940 2254\n",
      "20\t161\t0.0\t1\t0 161\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1646_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1646_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1646_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1646_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1646_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1646_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1646_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1646_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11       113,710 reads  @     97.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 11.07 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,710\n",
      "  Read 1 with adapter:                  86,226 (75.8%)\n",
      "  Read 2 with adapter:                  85,938 (75.6%)\n",
      "Pairs written (passing filters):       113,710 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,077,317 bp\n",
      "  Read 1:    33,539,712 bp\n",
      "  Read 2:    33,537,605 bp\n",
      "Total written (filtered):     63,720,497 bp (95.0%)\n",
      "  Read 1:    31,815,494 bp\n",
      "  Read 2:    31,905,003 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 86226 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t481\t0.0\t1\t0 473 8\n",
      "20\t85558\t0.0\t2\t83680 1827 51\n",
      "21\t131\t0.0\t2\t0 126 5\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85938 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t369\t0.0\t1\t0 369\n",
      "19\t85420\t0.0\t1\t82976 2444\n",
      "20\t149\t0.0\t1\t0 149\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1647_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1647_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1647_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1647_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1647_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1647_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1647_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1647_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       100,818 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 9.07 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,818\n",
      "  Read 1 with adapter:                  95,190 (94.4%)\n",
      "  Read 2 with adapter:                  94,757 (94.0%)\n",
      "Pairs written (passing filters):       100,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,465,864 bp\n",
      "  Read 1:    29,736,111 bp\n",
      "  Read 2:    29,729,753 bp\n",
      "Total written (filtered):     55,762,367 bp (93.8%)\n",
      "  Read 1:    27,832,685 bp\n",
      "  Read 2:    27,929,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 95190 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t563\t0.0\t1\t0 547 16\n",
      "20\t94402\t0.0\t2\t92254 2098 50\n",
      "21\t153\t0.0\t2\t0 149 4\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94757 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t462\t0.0\t1\t0 462\n",
      "19\t94145\t0.0\t1\t91606 2539\n",
      "20\t150\t0.0\t1\t0 150\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1648_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1648_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1648_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1648_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1648_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1648_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1648_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1648_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       100,449 reads  @     98.1 µs/read;   0.61 M reads/minute\n",
      "Finished in 9.85 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,449\n",
      "  Read 1 with adapter:                  79,972 (79.6%)\n",
      "  Read 2 with adapter:                  79,595 (79.2%)\n",
      "Pairs written (passing filters):       100,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,261,810 bp\n",
      "  Read 1:    29,631,497 bp\n",
      "  Read 2:    29,630,313 bp\n",
      "Total written (filtered):     56,150,669 bp (94.8%)\n",
      "  Read 1:    28,032,335 bp\n",
      "  Read 2:    28,118,334 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 79972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t463\t0.0\t1\t0 449 14\n",
      "20\t79314\t0.0\t2\t77633 1636 45\n",
      "21\t129\t0.0\t2\t0 126 3\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79595 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t515\t0.0\t1\t0 515\n",
      "19\t78891\t0.0\t1\t76677 2214\n",
      "20\t189\t0.0\t1\t0 189\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1649_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1649_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1649_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1649_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1649_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1649_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1649_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1649_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11       113,540 reads  @    100.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 11.43 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,540\n",
      "  Read 1 with adapter:                  84,512 (74.4%)\n",
      "  Read 2 with adapter:                  84,129 (74.1%)\n",
      "Pairs written (passing filters):       113,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,975,739 bp\n",
      "  Read 1:    33,491,986 bp\n",
      "  Read 2:    33,483,753 bp\n",
      "Total written (filtered):     63,687,337 bp (95.1%)\n",
      "  Read 1:    31,801,754 bp\n",
      "  Read 2:    31,885,583 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 84512 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t467\t0.0\t1\t0 446 21\n",
      "20\t83842\t0.0\t2\t81975 1818 49\n",
      "21\t154\t0.0\t2\t0 148 6\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 84129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t398\t0.0\t1\t0 398\n",
      "19\t83614\t0.0\t1\t81179 2435\n",
      "20\t117\t0.0\t1\t0 117\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1650_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1650_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1650_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1650_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1650_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1650_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1650_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1650_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:12       113,640 reads  @    106.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 12.05 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,640\n",
      "  Read 1 with adapter:                  75,715 (66.6%)\n",
      "  Read 2 with adapter:                  75,369 (66.3%)\n",
      "Pairs written (passing filters):       113,640 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,012,170 bp\n",
      "  Read 1:    33,517,058 bp\n",
      "  Read 2:    33,495,112 bp\n",
      "Total written (filtered):     64,066,294 bp (95.6%)\n",
      "  Read 1:    32,002,978 bp\n",
      "  Read 2:    32,063,316 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t420\t0.0\t1\t0 409 11\n",
      "20\t75117\t0.0\t2\t73483 1575 59\n",
      "21\t124\t0.0\t2\t0 123 1\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75369 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t343\t0.0\t1\t0 343\n",
      "19\t74898\t0.0\t1\t72831 2067\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1651_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1651_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1651_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1651_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1651_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1651_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1651_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1651_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11       117,232 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 11.65 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,232\n",
      "  Read 1 with adapter:                  80,933 (69.0%)\n",
      "  Read 2 with adapter:                  80,651 (68.8%)\n",
      "Pairs written (passing filters):       117,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,144,441 bp\n",
      "  Read 1:    34,581,139 bp\n",
      "  Read 2:    34,563,302 bp\n",
      "Total written (filtered):     65,993,932 bp (95.4%)\n",
      "  Read 1:    32,962,746 bp\n",
      "  Read 2:    33,031,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t453\t0.0\t1\t0 435 18\n",
      "20\t80286\t0.0\t2\t78402 1831 53\n",
      "21\t130\t0.0\t2\t0 125 5\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t401\t0.0\t1\t0 401\n",
      "19\t80102\t0.0\t1\t78180 1922\n",
      "20\t148\t0.0\t1\t0 148\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1652_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1652_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1652_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1652_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1652_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1652_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1652_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1652_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:10       101,365 reads  @    106.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 10.74 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,365\n",
      "  Read 1 with adapter:                  77,528 (76.5%)\n",
      "  Read 2 with adapter:                  76,252 (75.2%)\n",
      "Pairs written (passing filters):       101,365 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,732,441 bp\n",
      "  Read 1:    29,900,122 bp\n",
      "  Read 2:    29,832,319 bp\n",
      "Total written (filtered):     56,733,608 bp (95.0%)\n",
      "  Read 1:    28,349,818 bp\n",
      "  Read 2:    28,383,790 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t418\t0.0\t1\t0 411 7\n",
      "20\t76935\t0.0\t2\t75044 1835 56\n",
      "21\t120\t0.0\t2\t0 119 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 76252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t358\t0.0\t1\t0 358\n",
      "19\t75795\t0.0\t1\t74140 1655\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1653_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1653_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1653_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1653_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1653_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1653_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1653_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1653_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        77,040 reads  @     98.2 µs/read;   0.61 M reads/minute\n",
      "Finished in 7.57 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,040\n",
      "  Read 1 with adapter:                  66,500 (86.3%)\n",
      "  Read 2 with adapter:                  66,260 (86.0%)\n",
      "Pairs written (passing filters):        77,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,448,838 bp\n",
      "  Read 1:    22,726,241 bp\n",
      "  Read 2:    22,722,597 bp\n",
      "Total written (filtered):     42,860,344 bp (94.3%)\n",
      "  Read 1:    21,396,459 bp\n",
      "  Read 2:    21,463,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t380\t0.0\t1\t0 375 5\n",
      "20\t65980\t0.0\t2\t64633 1320 27\n",
      "21\t78\t0.0\t2\t0 75 3\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66260 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t336\t0.0\t1\t0 336\n",
      "19\t65816\t0.0\t1\t64204 1612\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1654_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1654_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1654_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1654_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1654_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1654_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1654_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1654_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:08        84,442 reads  @     95.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 8.07 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,442\n",
      "  Read 1 with adapter:                  65,334 (77.4%)\n",
      "  Read 2 with adapter:                  65,125 (77.1%)\n",
      "Pairs written (passing filters):        84,442 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,812,455 bp\n",
      "  Read 1:    24,909,620 bp\n",
      "  Read 2:    24,902,835 bp\n",
      "Total written (filtered):     47,268,862 bp (94.9%)\n",
      "  Read 1:    23,603,199 bp\n",
      "  Read 2:    23,665,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t411\t0.0\t1\t0 400 11\n",
      "20\t64750\t0.0\t2\t63322 1397 31\n",
      "21\t110\t0.0\t2\t0 106 4\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t338\t0.0\t1\t0 338\n",
      "19\t64652\t0.0\t1\t63096 1556\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1655_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1655_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1655_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1655_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1655_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1655_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1655_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1655_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        78,077 reads  @     96.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 7.51 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,077\n",
      "  Read 1 with adapter:                  66,610 (85.3%)\n",
      "  Read 2 with adapter:                  66,296 (84.9%)\n",
      "Pairs written (passing filters):        78,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,048,025 bp\n",
      "  Read 1:    23,031,604 bp\n",
      "  Read 2:    23,016,421 bp\n",
      "Total written (filtered):     43,456,634 bp (94.4%)\n",
      "  Read 1:    21,699,637 bp\n",
      "  Read 2:    21,756,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66610 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t414\t0.0\t1\t0 396 18\n",
      "20\t66016\t0.0\t2\t64494 1488 34\n",
      "21\t119\t0.0\t2\t0 115 4\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t331\t0.0\t1\t0 331\n",
      "19\t65834\t0.0\t1\t64166 1668\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1656_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1656_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1656_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1656_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1656_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1656_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1656_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1656_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        69,760 reads  @     90.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 6.30 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,760\n",
      "  Read 1 with adapter:                  55,742 (79.9%)\n",
      "  Read 2 with adapter:                  55,461 (79.5%)\n",
      "Pairs written (passing filters):        69,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,157,128 bp\n",
      "  Read 1:    20,578,363 bp\n",
      "  Read 2:    20,578,765 bp\n",
      "Total written (filtered):     38,988,882 bp (94.7%)\n",
      "  Read 1:    19,463,667 bp\n",
      "  Read 2:    19,525,215 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55742 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t283\t0.0\t1\t0 275 8\n",
      "20\t55307\t0.0\t2\t54149 1128 30\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55461 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t328\t0.0\t1\t0 328\n",
      "19\t55014\t0.0\t1\t53535 1479\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1657_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1657_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1657_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1657_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1657_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1657_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1657_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1657_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:08        80,787 reads  @    106.7 µs/read;   0.56 M reads/minute\n",
      "Finished in 8.62 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,787\n",
      "  Read 1 with adapter:                  55,633 (68.9%)\n",
      "  Read 2 with adapter:                  55,429 (68.6%)\n",
      "Pairs written (passing filters):        80,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,652,147 bp\n",
      "  Read 1:    23,831,785 bp\n",
      "  Read 2:    23,820,362 bp\n",
      "Total written (filtered):     45,486,710 bp (95.5%)\n",
      "  Read 1:    22,719,315 bp\n",
      "  Read 2:    22,767,395 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55633 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t317\t0.0\t1\t0 310 7\n",
      "20\t55180\t0.0\t2\t53919 1222 39\n",
      "21\t93\t0.0\t2\t0 92 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55429 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t287\t0.0\t1\t0 287\n",
      "19\t55039\t0.0\t1\t53655 1384\n",
      "20\t103\t0.0\t1\t0 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1658_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1658_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1658_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1658_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1658_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1658_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1658_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1658_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        68,455 reads  @     96.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 6.60 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,455\n",
      "  Read 1 with adapter:                  49,143 (71.8%)\n",
      "  Read 2 with adapter:                  48,890 (71.4%)\n",
      "Pairs written (passing filters):        68,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,365,455 bp\n",
      "  Read 1:    20,193,902 bp\n",
      "  Read 2:    20,171,553 bp\n",
      "Total written (filtered):     38,453,960 bp (95.3%)\n",
      "  Read 1:    19,211,200 bp\n",
      "  Read 2:    19,242,760 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49143 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t287\t0.0\t1\t0 281 6\n",
      "20\t48713\t0.0\t2\t47626 1050 37\n",
      "21\t105\t0.0\t2\t0 104 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48890 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t48553\t0.0\t1\t47404 1149\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1659_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1659_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1659_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1659_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1659_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1659_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1659_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1659_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        70,412 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.70 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,412\n",
      "  Read 1 with adapter:                  69,810 (99.1%)\n",
      "  Read 2 with adapter:                  69,592 (98.8%)\n",
      "Pairs written (passing filters):        70,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,526,152 bp\n",
      "  Read 1:    20,771,024 bp\n",
      "  Read 2:    20,755,128 bp\n",
      "Total written (filtered):     38,808,208 bp (93.5%)\n",
      "  Read 1:    19,375,130 bp\n",
      "  Read 2:    19,433,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t465\t0.0\t1\t0 456 9\n",
      "20\t69171\t0.0\t2\t67608 1512 51\n",
      "21\t121\t0.0\t2\t0 120 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69592 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t351\t0.0\t1\t0 351\n",
      "19\t69088\t0.0\t1\t67557 1531\n",
      "20\t153\t0.0\t1\t0 153\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1660_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1660_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1660_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1660_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1660_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1660_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1660_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1660_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8=-------] 00:00:08        79,759 reads  @    106.7 µs/read;   0.56 M reads/minute\n",
      "Finished in 8.51 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,759\n",
      "  Read 1 with adapter:                  56,351 (70.7%)\n",
      "  Read 2 with adapter:                  55,494 (69.6%)\n",
      "Pairs written (passing filters):        79,759 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,007,046 bp\n",
      "  Read 1:    23,528,124 bp\n",
      "  Read 2:    23,478,922 bp\n",
      "Total written (filtered):     44,826,046 bp (95.4%)\n",
      "  Read 1:    22,401,319 bp\n",
      "  Read 2:    22,424,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t335\t0.0\t1\t0 331 4\n",
      "20\t55869\t0.0\t2\t54610 1228 31\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55494 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t286\t0.0\t1\t0 286\n",
      "19\t55113\t0.0\t1\t53968 1145\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1661_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1661_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1661_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1661_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1661_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1661_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1661_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1661_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       104,455 reads  @     99.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 10.35 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,455\n",
      "  Read 1 with adapter:                  75,817 (72.6%)\n",
      "  Read 2 with adapter:                  75,347 (72.1%)\n",
      "Pairs written (passing filters):       104,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,616,444 bp\n",
      "  Read 1:    30,812,140 bp\n",
      "  Read 2:    30,804,304 bp\n",
      "Total written (filtered):     58,668,939 bp (95.2%)\n",
      "  Read 1:    29,295,980 bp\n",
      "  Read 2:    29,372,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t355\t0.0\t1\t0 344 11\n",
      "20\t75285\t0.0\t2\t73709 1533 43\n",
      "21\t127\t0.0\t2\t0 121 6\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75347 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t381\t0.0\t1\t0 381\n",
      "19\t74833\t0.0\t1\t72748 2085\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1662_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1662_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1662_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1662_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1662_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1662_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1662_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1662_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       102,287 reads  @     95.2 µs/read;   0.63 M reads/minute\n",
      "Finished in 9.74 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            102,287\n",
      "  Read 1 with adapter:                  86,310 (84.4%)\n",
      "  Read 2 with adapter:                  85,835 (83.9%)\n",
      "Pairs written (passing filters):       102,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,342,898 bp\n",
      "  Read 1:    30,174,093 bp\n",
      "  Read 2:    30,168,805 bp\n",
      "Total written (filtered):     56,986,362 bp (94.4%)\n",
      "  Read 1:    28,448,177 bp\n",
      "  Read 2:    28,538,185 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 86310 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t466\t0.0\t1\t0 450 16\n",
      "20\t85653\t0.0\t2\t83778 1827 48\n",
      "21\t132\t0.0\t2\t0 128 4\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t410\t0.0\t1\t0 410\n",
      "19\t85260\t0.0\t1\t82910 2350\n",
      "20\t165\t0.0\t1\t0 165\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1663_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1663_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1663_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1663_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1663_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1663_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1663_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1663_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       107,311 reads  @     98.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 10.60 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,311\n",
      "  Read 1 with adapter:                  85,327 (79.5%)\n",
      "  Read 2 with adapter:                  84,875 (79.1%)\n",
      "Pairs written (passing filters):       107,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,300,587 bp\n",
      "  Read 1:    31,654,990 bp\n",
      "  Read 2:    31,645,597 bp\n",
      "Total written (filtered):     59,982,006 bp (94.8%)\n",
      "  Read 1:    29,948,774 bp\n",
      "  Read 2:    30,033,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 85327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t548\t0.0\t1\t0 533 15\n",
      "20\t84557\t0.0\t2\t82616 1882 59\n",
      "21\t140\t0.0\t2\t0 138 2\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 84875 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t419\t0.0\t1\t0 419\n",
      "19\t84297\t0.0\t1\t82011 2286\n",
      "20\t159\t0.0\t1\t0 159\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1664_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1664_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1664_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1664_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1664_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1664_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1664_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1664_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        93,340 reads  @     99.2 µs/read;   0.60 M reads/minute\n",
      "Finished in 9.27 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,340\n",
      "  Read 1 with adapter:                  70,121 (75.1%)\n",
      "  Read 2 with adapter:                  69,694 (74.7%)\n",
      "Pairs written (passing filters):        93,340 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,066,750 bp\n",
      "  Read 1:    27,533,089 bp\n",
      "  Read 2:    27,533,661 bp\n",
      "Total written (filtered):     52,340,657 bp (95.0%)\n",
      "  Read 1:    26,130,873 bp\n",
      "  Read 2:    26,209,784 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70121 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t389\t0.0\t1\t0 382 7\n",
      "20\t69536\t0.0\t2\t67969 1526 41\n",
      "21\t127\t0.0\t2\t0 121 6\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t448\t0.0\t1\t0 448\n",
      "19\t69107\t0.0\t1\t67068 2039\n",
      "20\t139\t0.0\t1\t0 139\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1665_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1665_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1665_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1665_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1665_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1665_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1665_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1665_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        97,645 reads  @     89.5 µs/read;   0.67 M reads/minute\n",
      "Finished in 8.74 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,645\n",
      "  Read 1 with adapter:                  74,244 (76.0%)\n",
      "  Read 2 with adapter:                  73,799 (75.6%)\n",
      "Pairs written (passing filters):        97,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,591,852 bp\n",
      "  Read 1:    28,800,747 bp\n",
      "  Read 2:    28,791,105 bp\n",
      "Total written (filtered):     54,704,762 bp (95.0%)\n",
      "  Read 1:    27,315,599 bp\n",
      "  Read 2:    27,389,163 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74244 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t418\t0.0\t1\t0 410 8\n",
      "20\t73639\t0.0\t2\t72029 1568 42\n",
      "21\t130\t0.0\t2\t0 127 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 73799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t372\t0.0\t1\t0 372\n",
      "19\t73294\t0.0\t1\t71102 2192\n",
      "20\t133\t0.0\t1\t0 133\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1666_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1666_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1666_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1666_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1666_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1666_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1666_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1666_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:10       108,368 reads  @     94.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 10.28 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,368\n",
      "  Read 1 with adapter:                  81,200 (74.9%)\n",
      "  Read 2 with adapter:                  80,777 (74.5%)\n",
      "Pairs written (passing filters):       108,368 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,905,610 bp\n",
      "  Read 1:    31,963,436 bp\n",
      "  Read 2:    31,942,174 bp\n",
      "Total written (filtered):     60,747,421 bp (95.1%)\n",
      "  Read 1:    30,339,750 bp\n",
      "  Read 2:    30,407,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t490\t0.0\t1\t0 483 7\n",
      "20\t80518\t0.0\t2\t78773 1693 52\n",
      "21\t124\t0.0\t2\t0 122 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t404\t0.0\t1\t0 404\n",
      "19\t80229\t0.0\t1\t78115 2114\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1667_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1667_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1667_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1667_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1667_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1667_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1667_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1667_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       103,262 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 9.78 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,262\n",
      "  Read 1 with adapter:                  65,842 (63.8%)\n",
      "  Read 2 with adapter:                  65,432 (63.4%)\n",
      "Pairs written (passing filters):       103,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,889,671 bp\n",
      "  Read 1:    30,460,023 bp\n",
      "  Read 2:    30,429,648 bp\n",
      "Total written (filtered):     58,329,771 bp (95.8%)\n",
      "  Read 1:    29,143,127 bp\n",
      "  Read 2:    29,186,644 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65842 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t361\t0.0\t1\t0 351 10\n",
      "20\t65316\t0.0\t2\t63755 1527 34\n",
      "21\t116\t0.0\t2\t0 112 4\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t325\t0.0\t1\t0 325\n",
      "19\t64986\t0.0\t1\t63226 1760\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1668_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1668_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1668_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1668_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1668_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1668_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1668_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1668_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       109,407 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 9.82 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,407\n",
      "  Read 1 with adapter:                  93,587 (85.5%)\n",
      "  Read 2 with adapter:                  90,844 (83.0%)\n",
      "Pairs written (passing filters):       109,407 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,400,789 bp\n",
      "  Read 1:    32,271,784 bp\n",
      "  Read 2:    32,129,005 bp\n",
      "Total written (filtered):     60,803,657 bp (94.4%)\n",
      "  Read 1:    30,400,358 bp\n",
      "  Read 2:    30,403,299 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t521\t0.0\t1\t0 511 10\n",
      "20\t92820\t0.0\t2\t90692 2056 72\n",
      "21\t157\t0.0\t2\t0 148 9\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 90844 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t461\t0.0\t1\t0 461\n",
      "19\t90252\t0.0\t1\t87768 2484\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1669_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1669_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1669_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1669_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1669_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1669_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1669_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1669_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        97,411 reads  @     97.5 µs/read;   0.62 M reads/minute\n",
      "Finished in 9.50 s (98 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,411\n",
      "  Read 1 with adapter:                  80,813 (83.0%)\n",
      "  Read 2 with adapter:                  80,565 (82.7%)\n",
      "Pairs written (passing filters):        97,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,464,809 bp\n",
      "  Read 1:    28,732,443 bp\n",
      "  Read 2:    28,732,366 bp\n",
      "Total written (filtered):     54,318,305 bp (94.5%)\n",
      "  Read 1:    27,116,444 bp\n",
      "  Read 2:    27,201,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80813 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t442\t0.0\t1\t0 430 12\n",
      "20\t80177\t0.0\t2\t78319 1813 45\n",
      "21\t139\t0.0\t2\t0 139\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t359\t0.0\t1\t0 359\n",
      "19\t80077\t0.0\t1\t78224 1853\n",
      "20\t129\t0.0\t1\t0 129\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1670_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1670_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1670_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1670_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1670_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1670_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1670_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1670_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       107,960 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 10.73 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,960\n",
      "  Read 1 with adapter:                  80,560 (74.6%)\n",
      "  Read 2 with adapter:                  80,296 (74.4%)\n",
      "Pairs written (passing filters):       107,960 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,682,998 bp\n",
      "  Read 1:    31,846,904 bp\n",
      "  Read 2:    31,836,094 bp\n",
      "Total written (filtered):     60,546,858 bp (95.1%)\n",
      "  Read 1:    30,236,112 bp\n",
      "  Read 2:    30,310,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80560 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t567\t0.0\t1\t0 561 6\n",
      "20\t79821\t0.0\t2\t78028 1745 48\n",
      "21\t109\t0.0\t2\t0 106 3\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t428\t0.0\t1\t0 428\n",
      "19\t79716\t0.0\t1\t77942 1774\n",
      "20\t152\t0.0\t1\t0 152\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1671_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1671_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1671_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1671_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1671_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1671_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1671_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1671_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       100,095 reads  @     93.9 µs/read;   0.64 M reads/minute\n",
      "Finished in 9.40 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,095\n",
      "  Read 1 with adapter:                  70,401 (70.3%)\n",
      "  Read 2 with adapter:                  70,221 (70.2%)\n",
      "Pairs written (passing filters):       100,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,047,625 bp\n",
      "  Read 1:    29,526,292 bp\n",
      "  Read 2:    29,521,333 bp\n",
      "Total written (filtered):     56,305,965 bp (95.4%)\n",
      "  Read 1:    28,118,602 bp\n",
      "  Read 2:    28,187,363 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 70401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t506\t0.0\t1\t0 496 10\n",
      "20\t69710\t0.0\t2\t68202 1464 44\n",
      "21\t134\t0.0\t2\t0 131 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70221 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t345\t0.0\t1\t0 345\n",
      "19\t69760\t0.0\t1\t68188 1572\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1672_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1672_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1672_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1672_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1672_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1672_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1672_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1672_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:08        93,141 reads  @     94.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 8.81 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,141\n",
      "  Read 1 with adapter:                  81,543 (87.5%)\n",
      "  Read 2 with adapter:                  81,261 (87.2%)\n",
      "Pairs written (passing filters):        93,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,949,363 bp\n",
      "  Read 1:    27,474,706 bp\n",
      "  Read 2:    27,474,657 bp\n",
      "Total written (filtered):     51,775,216 bp (94.2%)\n",
      "  Read 1:    25,844,210 bp\n",
      "  Read 2:    25,931,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81543 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t502\t0.0\t1\t0 496 6\n",
      "20\t80868\t0.0\t2\t79023 1792 53\n",
      "21\t124\t0.0\t2\t0 119 5\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t491\t0.0\t1\t0 491\n",
      "19\t80587\t0.0\t1\t78623 1964\n",
      "20\t183\t0.0\t1\t0 183\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1673_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1673_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1673_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1673_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1673_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1673_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1673_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1673_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:11       111,519 reads  @    102.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 11.46 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,519\n",
      "  Read 1 with adapter:                  75,782 (68.0%)\n",
      "  Read 2 with adapter:                  75,569 (67.8%)\n",
      "Pairs written (passing filters):       111,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,779,376 bp\n",
      "  Read 1:    32,894,690 bp\n",
      "  Read 2:    32,884,686 bp\n",
      "Total written (filtered):     62,828,573 bp (95.5%)\n",
      "  Read 1:    31,379,458 bp\n",
      "  Read 2:    31,449,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t517\t0.0\t1\t0 504 13\n",
      "20\t75109\t0.0\t2\t73364 1699 46\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t376\t0.0\t1\t0 376\n",
      "19\t75057\t0.0\t1\t73361 1696\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1674_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1674_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1674_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1674_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1674_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1674_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1674_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1674_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:07        78,951 reads  @     92.4 µs/read;   0.65 M reads/minute\n",
      "Finished in 7.30 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,951\n",
      "  Read 1 with adapter:                  52,225 (66.1%)\n",
      "  Read 2 with adapter:                  52,044 (65.9%)\n",
      "Pairs written (passing filters):        78,951 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,563,877 bp\n",
      "  Read 1:    23,287,924 bp\n",
      "  Read 2:    23,275,953 bp\n",
      "Total written (filtered):     44,530,927 bp (95.6%)\n",
      "  Read 1:    22,243,632 bp\n",
      "  Read 2:    22,287,295 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t348\t0.0\t1\t0 340 8\n",
      "20\t51741\t0.0\t2\t50598 1112 31\n",
      "21\t92\t0.0\t2\t0 90 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t261\t0.0\t1\t0 261\n",
      "19\t51700\t0.0\t1\t50636 1064\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1675_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1675_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1675_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1675_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1675_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1675_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1675_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1675_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        94,457 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 9.38 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,457\n",
      "  Read 1 with adapter:                  61,603 (65.2%)\n",
      "  Read 2 with adapter:                  61,423 (65.0%)\n",
      "Pairs written (passing filters):        94,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,711,566 bp\n",
      "  Read 1:    27,863,777 bp\n",
      "  Read 2:    27,847,789 bp\n",
      "Total written (filtered):     53,312,895 bp (95.7%)\n",
      "  Read 1:    26,631,982 bp\n",
      "  Read 2:    26,680,913 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61603 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t374\t0.0\t1\t0 364 10\n",
      "20\t61075\t0.0\t2\t59610 1420 45\n",
      "21\t107\t0.0\t2\t0 103 4\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t284\t0.0\t1\t0 284\n",
      "19\t61016\t0.0\t1\t59731 1285\n",
      "20\t123\t0.0\t1\t0 123\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1676_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1676_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1676_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1676_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1676_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1676_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1676_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1676_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        90,773 reads  @     93.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 8.47 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,773\n",
      "  Read 1 with adapter:                  72,441 (79.8%)\n",
      "  Read 2 with adapter:                  71,194 (78.4%)\n",
      "Pairs written (passing filters):        90,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,481,006 bp\n",
      "  Read 1:    26,775,706 bp\n",
      "  Read 2:    26,705,300 bp\n",
      "Total written (filtered):     50,680,058 bp (94.8%)\n",
      "  Read 1:    25,327,184 bp\n",
      "  Read 2:    25,352,874 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 72441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t476\t0.0\t1\t0 468 8\n",
      "20\t71756\t0.0\t2\t70030 1688 38\n",
      "21\t132\t0.0\t2\t0 129 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 71194 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t377\t0.0\t1\t0 377\n",
      "19\t70700\t0.0\t1\t69377 1323\n",
      "20\t117\t0.0\t1\t0 117\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1677_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1677_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1677_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1677_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1677_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1677_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1677_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1677_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            65 reads  @    543.0 µs/read;   0.11 M reads/minute\n",
      "Finished in 0.04 s (597 us/read; 0.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 65\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            65 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,061 bp\n",
      "  Read 1:        13,211 bp\n",
      "  Read 2:        17,850 bp\n",
      "Total written (filtered):         31,061 bp (100.0%)\n",
      "  Read 1:        13,211 bp\n",
      "  Read 2:        17,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1693_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1693_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1693_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1693_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1693_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1693_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1693_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1693_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       107,672 reads  @     96.7 µs/read;   0.62 M reads/minute\n",
      "Finished in 10.42 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,672\n",
      "  Read 1 with adapter:                  91,972 (85.4%)\n",
      "  Read 2 with adapter:                  91,707 (85.2%)\n",
      "Pairs written (passing filters):       107,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,510,287 bp\n",
      "  Read 1:    31,751,970 bp\n",
      "  Read 2:    31,758,317 bp\n",
      "Total written (filtered):     59,928,972 bp (94.4%)\n",
      "  Read 1:    29,912,830 bp\n",
      "  Read 2:    30,016,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 91972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t519\t0.0\t1\t0 505 14\n",
      "20\t91222\t0.0\t2\t89052 2104 66\n",
      "21\t139\t0.0\t2\t0 138 1\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 91707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t422\t0.0\t1\t0 422\n",
      "19\t91121\t0.0\t1\t89013 2108\n",
      "20\t164\t0.0\t1\t0 164\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1694_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1694_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1694_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1694_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1694_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1694_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1694_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1694_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       106,168 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 10.06 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,168\n",
      "  Read 1 with adapter:                  83,325 (78.5%)\n",
      "  Read 2 with adapter:                  83,087 (78.3%)\n",
      "Pairs written (passing filters):       106,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,616,177 bp\n",
      "  Read 1:    31,302,783 bp\n",
      "  Read 2:    31,313,394 bp\n",
      "Total written (filtered):     59,371,557 bp (94.8%)\n",
      "  Read 1:    29,636,573 bp\n",
      "  Read 2:    29,734,984 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 83325 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t559\t0.0\t1\t0 545 14\n",
      "20\t82490\t0.0\t2\t80580 1858 52\n",
      "21\t183\t0.0\t2\t0 182 1\n",
      "22\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 83087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t404\t0.0\t1\t0 404\n",
      "19\t82522\t0.0\t1\t80693 1829\n",
      "20\t161\t0.0\t1\t0 161\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1695_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1695_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1695_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1695_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1695_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1695_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1695_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1695_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       104,432 reads  @     87.1 µs/read;   0.69 M reads/minute\n",
      "Finished in 9.10 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,432\n",
      "  Read 1 with adapter:                  97,571 (93.4%)\n",
      "  Read 2 with adapter:                  97,290 (93.2%)\n",
      "Pairs written (passing filters):       104,432 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,601,955 bp\n",
      "  Read 1:    30,801,689 bp\n",
      "  Read 2:    30,800,266 bp\n",
      "Total written (filtered):     57,802,688 bp (93.8%)\n",
      "  Read 1:    28,850,619 bp\n",
      "  Read 2:    28,952,069 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 97571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t668\t0.0\t1\t0 649 19\n",
      "20\t96570\t0.0\t2\t94281 2232 57\n",
      "21\t216\t0.0\t2\t0 211 5\n",
      "22\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 97290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t487\t0.0\t1\t0 487\n",
      "19\t96629\t0.0\t1\t94439 2190\n",
      "20\t174\t0.0\t1\t0 174\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1696_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1696_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1696_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1696_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1696_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1696_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1696_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1696_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        93,637 reads  @     92.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 8.70 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,637\n",
      "  Read 1 with adapter:                  81,021 (86.5%)\n",
      "  Read 2 with adapter:                  80,808 (86.3%)\n",
      "Pairs written (passing filters):        93,637 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,231,950 bp\n",
      "  Read 1:    27,610,035 bp\n",
      "  Read 2:    27,621,915 bp\n",
      "Total written (filtered):     52,076,842 bp (94.3%)\n",
      "  Read 1:    25,989,910 bp\n",
      "  Read 2:    26,086,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81021 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t531\t0.0\t1\t0 517 14\n",
      "20\t80217\t0.0\t2\t78298 1869 50\n",
      "21\t182\t0.0\t2\t0 175 7\n",
      "22\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t541\t0.0\t1\t0 541\n",
      "19\t80095\t0.0\t1\t78139 1956\n",
      "20\t172\t0.0\t1\t0 172\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1697_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1697_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1697_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1697_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1697_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1697_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1697_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1697_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:09       105,071 reads  @     93.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 9.82 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,071\n",
      "  Read 1 with adapter:                  87,167 (83.0%)\n",
      "  Read 2 with adapter:                  86,919 (82.7%)\n",
      "Pairs written (passing filters):       105,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,983,209 bp\n",
      "  Read 1:    30,992,575 bp\n",
      "  Read 2:    30,990,634 bp\n",
      "Total written (filtered):     58,589,076 bp (94.5%)\n",
      "  Read 1:    29,249,575 bp\n",
      "  Read 2:    29,339,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 87167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t589\t0.0\t1\t0 575 14\n",
      "20\t86303\t0.0\t2\t84001 2245 57\n",
      "21\t189\t0.0\t2\t0 183 6\n",
      "22\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 86919 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t498\t0.0\t1\t0 498\n",
      "19\t86251\t0.0\t1\t84063 2188\n",
      "20\t170\t0.0\t1\t0 170\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1698_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1698_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1698_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1698_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1698_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1698_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1698_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1698_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        68,302 reads  @     94.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 6.46 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,302\n",
      "  Read 1 with adapter:                  58,295 (85.3%)\n",
      "  Read 2 with adapter:                  58,143 (85.1%)\n",
      "Pairs written (passing filters):        68,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,273,279 bp\n",
      "  Read 1:    20,138,994 bp\n",
      "  Read 2:    20,134,285 bp\n",
      "Total written (filtered):     38,003,080 bp (94.4%)\n",
      "  Read 1:    18,973,322 bp\n",
      "  Read 2:    19,029,758 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58295 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t413\t0.0\t1\t0 407 6\n",
      "20\t57702\t0.0\t2\t56489 1171 42\n",
      "21\t123\t0.0\t2\t0 123\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58143 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t287\t0.0\t1\t0 287\n",
      "19\t57759\t0.0\t1\t56609 1150\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1699_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1699_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1699_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1699_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1699_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1699_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1699_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1699_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        94,939 reads  @     91.1 µs/read;   0.66 M reads/minute\n",
      "Finished in 8.65 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,939\n",
      "  Read 1 with adapter:                  77,617 (81.8%)\n",
      "  Read 2 with adapter:                  77,407 (81.5%)\n",
      "Pairs written (passing filters):        94,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,983,457 bp\n",
      "  Read 1:    27,988,839 bp\n",
      "  Read 2:    27,994,618 bp\n",
      "Total written (filtered):     52,960,972 bp (94.6%)\n",
      "  Read 1:    26,436,852 bp\n",
      "  Read 2:    26,524,120 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77617 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t551\t0.0\t1\t0 535 16\n",
      "20\t76801\t0.0\t2\t74850 1892 59\n",
      "21\t168\t0.0\t2\t0 167 1\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77407 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t403\t0.0\t1\t0 403\n",
      "19\t76836\t0.0\t1\t75149 1687\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1700_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1700_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1700_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1700_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1700_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1700_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1700_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1700_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        99,134 reads  @     97.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 9.63 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,134\n",
      "  Read 1 with adapter:                  80,599 (81.3%)\n",
      "  Read 2 with adapter:                  79,379 (80.1%)\n",
      "Pairs written (passing filters):        99,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,406,456 bp\n",
      "  Read 1:    29,234,810 bp\n",
      "  Read 2:    29,171,646 bp\n",
      "Total written (filtered):     55,286,813 bp (94.7%)\n",
      "  Read 1:    27,623,124 bp\n",
      "  Read 2:    27,663,689 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80599 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t529\t0.0\t1\t0 506 23\n",
      "20\t79829\t0.0\t2\t77813 1960 56\n",
      "21\t167\t0.0\t2\t0 162 5\n",
      "22\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79379 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t370\t0.0\t1\t0 370\n",
      "19\t78883\t0.0\t1\t77371 1512\n",
      "20\t126\t0.0\t1\t0 126\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1701_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1701_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1701_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1701_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1701_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1701_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1701_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1701_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        93,527 reads  @    100.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 9.39 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,527\n",
      "  Read 1 with adapter:                  78,162 (83.6%)\n",
      "  Read 2 with adapter:                  77,771 (83.2%)\n",
      "Pairs written (passing filters):        93,527 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,172,558 bp\n",
      "  Read 1:    27,587,391 bp\n",
      "  Read 2:    27,585,167 bp\n",
      "Total written (filtered):     52,132,515 bp (94.5%)\n",
      "  Read 1:    26,024,760 bp\n",
      "  Read 2:    26,107,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 78162 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t770\t0.0\t1\t0 752 18\n",
      "20\t77208\t0.0\t2\t75400 1761 47\n",
      "21\t115\t0.0\t2\t0 113 2\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t383\t0.0\t1\t0 383\n",
      "19\t77242\t0.0\t1\t75144 2098\n",
      "20\t146\t0.0\t1\t0 146\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1702_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1702_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1702_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1702_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1702_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1702_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1702_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1702_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       101,023 reads  @     93.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 9.45 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,023\n",
      "  Read 1 with adapter:                  86,771 (85.9%)\n",
      "  Read 2 with adapter:                  86,365 (85.5%)\n",
      "Pairs written (passing filters):       101,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,592,999 bp\n",
      "  Read 1:    29,800,082 bp\n",
      "  Read 2:    29,792,917 bp\n",
      "Total written (filtered):     56,217,572 bp (94.3%)\n",
      "  Read 1:    28,065,323 bp\n",
      "  Read 2:    28,152,249 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 86771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t865\t0.0\t1\t0 843 22\n",
      "20\t85666\t0.0\t2\t83620 1998 48\n",
      "21\t164\t0.0\t2\t0 162 2\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 86365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t438\t0.0\t1\t0 438\n",
      "19\t85756\t0.0\t1\t83075 2681\n",
      "20\t171\t0.0\t1\t0 171\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1703_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1703_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1703_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1703_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1703_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1703_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1703_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1703_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:09        92,764 reads  @     97.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 9.02 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,764\n",
      "  Read 1 with adapter:                  79,029 (85.2%)\n",
      "  Read 2 with adapter:                  78,412 (84.5%)\n",
      "Pairs written (passing filters):        92,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,686,098 bp\n",
      "  Read 1:    27,359,300 bp\n",
      "  Read 2:    27,326,798 bp\n",
      "Total written (filtered):     51,616,537 bp (94.4%)\n",
      "  Read 1:    25,779,344 bp\n",
      "  Read 2:    25,837,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 79029 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t760\t0.0\t1\t0 738 22\n",
      "20\t78076\t0.0\t2\t76129 1898 49\n",
      "21\t130\t0.0\t2\t0 127 3\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 78412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t372\t0.0\t1\t0 372\n",
      "19\t77891\t0.0\t1\t75634 2257\n",
      "20\t149\t0.0\t1\t0 149\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1704_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1704_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1704_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1704_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1704_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1704_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1704_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1704_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        73,441 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 6.56 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,441\n",
      "  Read 1 with adapter:                  62,125 (84.6%)\n",
      "  Read 2 with adapter:                  61,794 (84.1%)\n",
      "Pairs written (passing filters):        73,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,327,265 bp\n",
      "  Read 1:    21,663,795 bp\n",
      "  Read 2:    21,663,470 bp\n",
      "Total written (filtered):     40,911,408 bp (94.4%)\n",
      "  Read 1:    20,421,735 bp\n",
      "  Read 2:    20,489,673 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t576\t0.0\t1\t0 562 14\n",
      "20\t61384\t0.0\t2\t60029 1332 23\n",
      "21\t98\t0.0\t2\t0 95 3\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t414\t0.0\t1\t0 414\n",
      "19\t61255\t0.0\t1\t59358 1897\n",
      "20\t125\t0.0\t1\t0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1705_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1705_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1705_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1705_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1705_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1705_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1705_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1705_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       106,435 reads  @     96.7 µs/read;   0.62 M reads/minute\n",
      "Finished in 10.29 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,435\n",
      "  Read 1 with adapter:                  92,794 (87.2%)\n",
      "  Read 2 with adapter:                  92,334 (86.8%)\n",
      "Pairs written (passing filters):       106,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,777,776 bp\n",
      "  Read 1:    31,395,328 bp\n",
      "  Read 2:    31,382,448 bp\n",
      "Total written (filtered):     59,168,281 bp (94.3%)\n",
      "  Read 1:    29,539,860 bp\n",
      "  Read 2:    29,628,421 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 92794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t861\t0.0\t1\t0 838 23\n",
      "20\t91727\t0.0\t2\t89576 2110 41\n",
      "21\t134\t0.0\t2\t0 133 1\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 92334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t514\t0.0\t1\t0 514\n",
      "19\t91625\t0.0\t1\t88344 3281\n",
      "20\t195\t0.0\t1\t0 195\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1706_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1706_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1706_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1706_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1706_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1706_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1706_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1706_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        59,261 reads  @     92.6 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.49 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,261\n",
      "  Read 1 with adapter:                  46,833 (79.0%)\n",
      "  Read 2 with adapter:                  46,555 (78.6%)\n",
      "Pairs written (passing filters):        59,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,941,238 bp\n",
      "  Read 1:    17,477,266 bp\n",
      "  Read 2:    17,463,972 bp\n",
      "Total written (filtered):     33,120,432 bp (94.8%)\n",
      "  Read 1:    16,540,918 bp\n",
      "  Read 2:    16,579,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46833 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t406\t0.0\t1\t0 397 9\n",
      "20\t46328\t0.0\t2\t45422 874 32\n",
      "21\t68\t0.0\t2\t0 68\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46555 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t180\t0.0\t1\t0 180\n",
      "19\t46282\t0.0\t1\t44880 1402\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1707_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1707_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1707_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1707_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1707_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1707_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1707_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1707_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       109,019 reads  @     91.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 9.96 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,019\n",
      "  Read 1 with adapter:                  88,999 (81.6%)\n",
      "  Read 2 with adapter:                  88,677 (81.3%)\n",
      "Pairs written (passing filters):       109,019 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,294,603 bp\n",
      "  Read 1:    32,152,146 bp\n",
      "  Read 2:    32,142,457 bp\n",
      "Total written (filtered):     60,830,624 bp (94.6%)\n",
      "  Read 1:    30,372,782 bp\n",
      "  Read 2:    30,457,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 88999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t782\t0.0\t1\t0 764 18\n",
      "20\t88003\t0.0\t2\t85945 2010 48\n",
      "21\t158\t0.0\t2\t0 153 5\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 88677 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t423\t0.0\t1\t0 423\n",
      "19\t88079\t0.0\t1\t85909 2170\n",
      "20\t175\t0.0\t1\t0 175\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1708_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1708_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1708_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1708_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1708_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1708_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1708_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1708_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        88,870 reads  @     94.0 µs/read;   0.64 M reads/minute\n",
      "Finished in 8.35 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,870\n",
      "  Read 1 with adapter:                  81,051 (91.2%)\n",
      "  Read 2 with adapter:                  79,320 (89.3%)\n",
      "Pairs written (passing filters):        88,870 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,332,404 bp\n",
      "  Read 1:    26,215,152 bp\n",
      "  Read 2:    26,117,252 bp\n",
      "Total written (filtered):     49,205,088 bp (94.0%)\n",
      "  Read 1:    24,594,717 bp\n",
      "  Read 2:    24,610,371 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t766\t0.0\t1\t0 752 14\n",
      "20\t80075\t0.0\t2\t78207 1815 53\n",
      "21\t143\t0.0\t2\t0 143\n",
      "22\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t334\t0.0\t1\t0 334\n",
      "19\t78851\t0.0\t1\t76923 1928\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1709_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1709_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1709_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1709_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1709_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1709_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1709_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1709_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:09        97,345 reads  @     96.7 µs/read;   0.62 M reads/minute\n",
      "Finished in 9.42 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,345\n",
      "  Read 1 with adapter:                  74,463 (76.5%)\n",
      "  Read 2 with adapter:                  74,230 (76.3%)\n",
      "Pairs written (passing filters):        97,345 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,422,715 bp\n",
      "  Read 1:    28,711,379 bp\n",
      "  Read 2:    28,711,336 bp\n",
      "Total written (filtered):     54,523,615 bp (95.0%)\n",
      "  Read 1:    27,222,412 bp\n",
      "  Read 2:    27,301,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 74463 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t440\t0.0\t1\t0 433 7\n",
      "20\t73852\t0.0\t2\t72201 1608 43\n",
      "21\t119\t0.0\t2\t0 117 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t361\t0.0\t1\t0 361\n",
      "19\t73745\t0.0\t1\t71950 1795\n",
      "20\t124\t0.0\t1\t0 124\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1710_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1710_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1710_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1710_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1710_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1710_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1710_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1710_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        72,950 reads  @     94.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 6.92 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,950\n",
      "  Read 1 with adapter:                  58,380 (80.0%)\n",
      "  Read 2 with adapter:                  58,209 (79.8%)\n",
      "Pairs written (passing filters):        72,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,027,147 bp\n",
      "  Read 1:    21,511,038 bp\n",
      "  Read 2:    21,516,109 bp\n",
      "Total written (filtered):     40,753,995 bp (94.7%)\n",
      "  Read 1:    20,343,671 bp\n",
      "  Read 2:    20,410,324 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t402\t0.0\t1\t0 385 17\n",
      "20\t57800\t0.0\t2\t56474 1293 33\n",
      "21\t115\t0.0\t2\t0 112 3\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t284\t0.0\t1\t0 284\n",
      "19\t57827\t0.0\t1\t56443 1384\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1711_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1711_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1711_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1711_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1711_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1711_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1711_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1711_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        84,072 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 7.96 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,072\n",
      "  Read 1 with adapter:                  71,054 (84.5%)\n",
      "  Read 2 with adapter:                  70,883 (84.3%)\n",
      "Pairs written (passing filters):        84,072 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,586,968 bp\n",
      "  Read 1:    24,792,357 bp\n",
      "  Read 2:    24,794,611 bp\n",
      "Total written (filtered):     46,819,659 bp (94.4%)\n",
      "  Read 1:    23,371,616 bp\n",
      "  Read 2:    23,448,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 71054 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t501\t0.0\t1\t0 487 14\n",
      "20\t70330\t0.0\t2\t68591 1700 39\n",
      "21\t148\t0.0\t2\t0 146 2\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 70883 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t355\t0.0\t1\t0 355\n",
      "19\t70382\t0.0\t1\t68771 1611\n",
      "20\t146\t0.0\t1\t0 146\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1712_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1712_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1712_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1712_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1712_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1712_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1712_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1712_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        85,413 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 7.85 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,413\n",
      "  Read 1 with adapter:                  71,715 (84.0%)\n",
      "  Read 2 with adapter:                  71,487 (83.7%)\n",
      "Pairs written (passing filters):        85,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,384,949 bp\n",
      "  Read 1:    25,189,522 bp\n",
      "  Read 2:    25,195,427 bp\n",
      "Total written (filtered):     47,593,001 bp (94.5%)\n",
      "  Read 1:    23,755,512 bp\n",
      "  Read 2:    23,837,489 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 71715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t470\t0.0\t1\t0 461 9\n",
      "20\t71030\t0.0\t2\t69442 1549 39\n",
      "21\t138\t0.0\t2\t0 133 5\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 71487 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t489\t0.0\t1\t0 489\n",
      "19\t70824\t0.0\t1\t69177 1647\n",
      "20\t174\t0.0\t1\t0 174\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1713_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1713_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1713_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1713_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1713_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1713_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1713_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1713_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        85,803 reads  @     85.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 7.32 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,803\n",
      "  Read 1 with adapter:                  68,605 (80.0%)\n",
      "  Read 2 with adapter:                  68,499 (79.8%)\n",
      "Pairs written (passing filters):        85,803 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,612,672 bp\n",
      "  Read 1:    25,305,712 bp\n",
      "  Read 2:    25,306,960 bp\n",
      "Total written (filtered):     47,939,675 bp (94.7%)\n",
      "  Read 1:    23,933,889 bp\n",
      "  Read 2:    24,005,786 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 68605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t429\t0.0\t1\t0 423 6\n",
      "20\t67981\t0.0\t2\t66481 1457 43\n",
      "21\t130\t0.0\t2\t0 128 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 68499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t397\t0.0\t1\t0 397\n",
      "19\t68012\t0.0\t1\t66602 1410\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1714_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1714_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1714_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1714_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1714_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1714_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1714_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1714_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       114,158 reads  @     86.3 µs/read;   0.69 M reads/minute\n",
      "Finished in 9.86 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,158\n",
      "  Read 1 with adapter:                  91,891 (80.5%)\n",
      "  Read 2 with adapter:                  91,657 (80.3%)\n",
      "Pairs written (passing filters):       114,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,325,932 bp\n",
      "  Read 1:    33,665,406 bp\n",
      "  Read 2:    33,660,526 bp\n",
      "Total written (filtered):     63,747,046 bp (94.7%)\n",
      "  Read 1:    31,827,732 bp\n",
      "  Read 2:    31,919,314 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 91891 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t644\t0.0\t1\t0 630 14\n",
      "20\t90995\t0.0\t2\t88892 2058 45\n",
      "21\t161\t0.0\t2\t0 159 2\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 91657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t455\t0.0\t1\t0 455\n",
      "19\t91018\t0.0\t1\t89135 1883\n",
      "20\t184\t0.0\t1\t0 184\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1715_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1715_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1715_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1715_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1715_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1715_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1715_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1715_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:08        97,354 reads  @     88.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 8.65 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,354\n",
      "  Read 1 with adapter:                  76,412 (78.5%)\n",
      "  Read 2 with adapter:                  76,126 (78.2%)\n",
      "Pairs written (passing filters):        97,351 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,406,058 bp\n",
      "  Read 1:    28,716,637 bp\n",
      "  Read 2:    28,689,421 bp\n",
      "Total written (filtered):     54,431,129 bp (94.8%)\n",
      "  Read 1:    27,187,867 bp\n",
      "  Read 2:    27,243,262 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 76412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t468\t0.0\t1\t0 453 15\n",
      "20\t75730\t0.0\t2\t73991 1689 50\n",
      "21\t147\t0.0\t2\t0 144 3\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 76126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t397\t0.0\t1\t0 397\n",
      "19\t75567\t0.0\t1\t73913 1654\n",
      "20\t162\t0.0\t1\t0 162\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1716_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1716_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1716_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1716_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1716_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1716_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1716_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1716_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        93,135 reads  @     85.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 7.93 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,135\n",
      "  Read 1 with adapter:                  75,338 (80.9%)\n",
      "  Read 2 with adapter:                  74,197 (79.7%)\n",
      "Pairs written (passing filters):        93,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,867,928 bp\n",
      "  Read 1:    27,456,838 bp\n",
      "  Read 2:    27,411,090 bp\n",
      "Total written (filtered):     51,951,960 bp (94.7%)\n",
      "  Read 1:    25,950,365 bp\n",
      "  Read 2:    26,001,595 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t436\t0.0\t1\t0 428 8\n",
      "20\t74699\t0.0\t2\t73034 1613 52\n",
      "21\t149\t0.0\t2\t0 147 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 74197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t376\t0.0\t1\t0 376\n",
      "19\t73693\t0.0\t1\t72120 1573\n",
      "20\t128\t0.0\t1\t0 128\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1717_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1717_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1717_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1717_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1717_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1717_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1717_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1717_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        90,630 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 8.68 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,630\n",
      "  Read 1 with adapter:                  77,713 (85.7%)\n",
      "  Read 2 with adapter:                  77,473 (85.5%)\n",
      "Pairs written (passing filters):        90,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,467,287 bp\n",
      "  Read 1:    26,734,257 bp\n",
      "  Read 2:    26,733,030 bp\n",
      "Total written (filtered):     50,441,504 bp (94.3%)\n",
      "  Read 1:    25,180,214 bp\n",
      "  Read 2:    25,261,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77713 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t389\t0.0\t1\t0 381 8\n",
      "20\t77149\t0.0\t2\t75534 1572 43\n",
      "21\t114\t0.0\t2\t0 109 5\n",
      "22\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t357\t0.0\t1\t0 357\n",
      "19\t77006\t0.0\t1\t75113 1893\n",
      "20\t110\t0.0\t1\t0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1718_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1718_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1718_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1718_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1718_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1718_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1718_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1718_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        98,124 reads  @     94.0 µs/read;   0.64 M reads/minute\n",
      "Finished in 9.22 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,124\n",
      "  Read 1 with adapter:                  83,073 (84.7%)\n",
      "  Read 2 with adapter:                  82,804 (84.4%)\n",
      "Pairs written (passing filters):        98,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,885,195 bp\n",
      "  Read 1:    28,944,304 bp\n",
      "  Read 2:    28,940,891 bp\n",
      "Total written (filtered):     54,651,000 bp (94.4%)\n",
      "  Read 1:    27,283,131 bp\n",
      "  Read 2:    27,367,869 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 83073 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t538\t0.0\t1\t0 525 13\n",
      "20\t82300\t0.0\t2\t80485 1761 54\n",
      "21\t139\t0.0\t2\t0 136 3\n",
      "22\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 82804 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t394\t0.0\t1\t0 394\n",
      "19\t82270\t0.0\t1\t80171 2099\n",
      "20\t140\t0.0\t1\t0 140\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1719_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1719_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1719_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1719_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1719_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1719_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1719_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1719_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        85,116 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 7.66 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,116\n",
      "  Read 1 with adapter:                  72,479 (85.2%)\n",
      "  Read 2 with adapter:                  72,218 (84.8%)\n",
      "Pairs written (passing filters):        85,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,210,149 bp\n",
      "  Read 1:    25,108,154 bp\n",
      "  Read 2:    25,101,995 bp\n",
      "Total written (filtered):     47,388,940 bp (94.4%)\n",
      "  Read 1:    23,658,886 bp\n",
      "  Read 2:    23,730,054 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 72479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t480\t0.0\t1\t0 468 12\n",
      "20\t71816\t0.0\t2\t70177 1590 49\n",
      "21\t126\t0.0\t2\t0 122 4\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t344\t0.0\t1\t0 344\n",
      "19\t71731\t0.0\t1\t70010 1721\n",
      "20\t143\t0.0\t1\t0 143\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1720_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1720_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1720_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1720_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1720_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1720_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1720_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1720_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        59,674 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.48 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,674\n",
      "  Read 1 with adapter:                  56,287 (94.3%)\n",
      "  Read 2 with adapter:                  56,135 (94.1%)\n",
      "Pairs written (passing filters):        59,674 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,204,866 bp\n",
      "  Read 1:    17,602,408 bp\n",
      "  Read 2:    17,602,458 bp\n",
      "Total written (filtered):     33,013,047 bp (93.8%)\n",
      "  Read 1:    16,476,913 bp\n",
      "  Read 2:    16,536,134 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t363\t0.0\t1\t0 354 9\n",
      "20\t55794\t0.0\t2\t54541 1217 36\n",
      "21\t90\t0.0\t2\t0 85 5\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t361\t0.0\t1\t0 361\n",
      "19\t55654\t0.0\t1\t54212 1442\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1721_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1721_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1721_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1721_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1721_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1721_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1721_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1721_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:08        91,265 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 8.65 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,265\n",
      "  Read 1 with adapter:                  77,773 (85.2%)\n",
      "  Read 2 with adapter:                  77,564 (85.0%)\n",
      "Pairs written (passing filters):        91,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,833,451 bp\n",
      "  Read 1:    26,920,651 bp\n",
      "  Read 2:    26,912,800 bp\n",
      "Total written (filtered):     50,804,810 bp (94.4%)\n",
      "  Read 1:    25,365,463 bp\n",
      "  Read 2:    25,439,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 77773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t479\t0.0\t1\t0 469 10\n",
      "20\t77095\t0.0\t2\t75335 1703 57\n",
      "21\t131\t0.0\t2\t0 129 2\n",
      "22\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 77564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t405\t0.0\t1\t0 405\n",
      "19\t77017\t0.0\t1\t75055 1962\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1722_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1722_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1722_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1722_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1722_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1722_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1722_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1722_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,224 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 6.70 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,224\n",
      "  Read 1 with adapter:                  66,819 (82.3%)\n",
      "  Read 2 with adapter:                  66,561 (81.9%)\n",
      "Pairs written (passing filters):        81,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,902,431 bp\n",
      "  Read 1:    23,959,806 bp\n",
      "  Read 2:    23,942,625 bp\n",
      "Total written (filtered):     45,301,820 bp (94.6%)\n",
      "  Read 1:    22,623,687 bp\n",
      "  Read 2:    22,678,133 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66819 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t413\t0.0\t1\t0 407 6\n",
      "20\t66227\t0.0\t2\t64755 1426 46\n",
      "21\t118\t0.0\t2\t0 114 4\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t286\t0.0\t1\t0 286\n",
      "19\t66156\t0.0\t1\t64676 1480\n",
      "20\t119\t0.0\t1\t0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1723_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1723_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1723_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1723_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1723_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1723_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1723_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1723_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        99,193 reads  @     96.0 µs/read;   0.63 M reads/minute\n",
      "Finished in 9.53 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,193\n",
      "  Read 1 with adapter:                  62,919 (63.4%)\n",
      "  Read 2 with adapter:                  62,692 (63.2%)\n",
      "Pairs written (passing filters):        99,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,492,952 bp\n",
      "  Read 1:    29,256,749 bp\n",
      "  Read 2:    29,236,203 bp\n",
      "Total written (filtered):     56,043,844 bp (95.8%)\n",
      "  Read 1:    27,998,598 bp\n",
      "  Read 2:    28,045,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62919 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t363\t0.0\t1\t0 355 8\n",
      "20\t62395\t0.0\t2\t60959 1414 22\n",
      "21\t108\t0.0\t2\t0 106 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t333\t0.0\t1\t0 333\n",
      "19\t62217\t0.0\t1\t60843 1374\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1724_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1724_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1724_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1724_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1724_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1724_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1724_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1724_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        84,135 reads  @     91.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 7.71 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,135\n",
      "  Read 1 with adapter:                  67,998 (80.8%)\n",
      "  Read 2 with adapter:                  66,844 (79.4%)\n",
      "Pairs written (passing filters):        84,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,570,747 bp\n",
      "  Read 1:    24,816,316 bp\n",
      "  Read 2:    24,754,431 bp\n",
      "Total written (filtered):     46,941,172 bp (94.7%)\n",
      "  Read 1:    23,456,572 bp\n",
      "  Read 2:    23,484,600 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67998 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t403\t0.0\t1\t0 385 18\n",
      "20\t67402\t0.0\t2\t65943 1404 55\n",
      "21\t119\t0.0\t2\t0 118 1\n",
      "22\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66844 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t323\t0.0\t1\t0 323\n",
      "19\t66403\t0.0\t1\t64993 1410\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1725_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1725_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1725_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1725_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1725_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1725_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1725_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1725_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            32 reads  @   1161.7 µs/read;   0.05 M reads/minute\n",
      "Finished in 0.04 s (1260 us/read; 0.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 32\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            32 (100.0%)\n",
      "\n",
      "Total basepairs processed:        13,110 bp\n",
      "  Read 1:         4,715 bp\n",
      "  Read 2:         8,395 bp\n",
      "Total written (filtered):         13,110 bp (100.0%)\n",
      "  Read 1:         4,715 bp\n",
      "  Read 2:         8,395 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1741_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1741_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1741_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1741_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1741_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1741_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1741_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1741_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           305 reads  @    198.5 µs/read;   0.30 M reads/minute\n",
      "Finished in 0.06 s (209 us/read; 0.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                305\n",
      "  Read 1 with adapter:                     305 (100.0%)\n",
      "  Read 2 with adapter:                      17 (5.6%)\n",
      "Pairs written (passing filters):           305 (100.0%)\n",
      "\n",
      "Total basepairs processed:       104,870 bp\n",
      "  Read 1:        89,974 bp\n",
      "  Read 2:        14,896 bp\n",
      "Total written (filtered):         98,434 bp (93.9%)\n",
      "  Read 1:        83,878 bp\n",
      "  Read 2:        14,556 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t301\t0.0\t2\t293 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 17 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t17\t0.0\t1\t0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1742_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1742_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1742_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1742_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1742_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1742_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1742_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1742_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,276 reads  @     78.6 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.80 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,276\n",
      "  Read 1 with adapter:                  48,229 (99.9%)\n",
      "  Read 2 with adapter:                  48,013 (99.5%)\n",
      "Pairs written (passing filters):        48,276 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,474,983 bp\n",
      "  Read 1:    14,240,926 bp\n",
      "  Read 2:    14,234,057 bp\n",
      "Total written (filtered):     26,598,555 bp (93.4%)\n",
      "  Read 1:    13,276,587 bp\n",
      "  Read 2:    13,321,968 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48229 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t318\t0.0\t1\t0 310 8\n",
      "20\t47796\t0.0\t2\t46834 928 34\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48013 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t238\t0.0\t1\t0 238\n",
      "19\t47695\t0.0\t1\t46692 1003\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1743_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1743_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1743_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1743_137_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1743_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1743_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1743_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1743_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,108 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.22 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,108\n",
      "  Read 1 with adapter:                  65,037 (99.9%)\n",
      "  Read 2 with adapter:                  64,903 (99.7%)\n",
      "Pairs written (passing filters):        65,108 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,411,203 bp\n",
      "  Read 1:    19,204,613 bp\n",
      "  Read 2:    19,206,590 bp\n",
      "Total written (filtered):     35,877,756 bp (93.4%)\n",
      "  Read 1:    17,904,159 bp\n",
      "  Read 2:    17,973,597 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t414\t0.0\t1\t0 406 8\n",
      "20\t64452\t0.0\t2\t63042 1372 38\n",
      "21\t106\t0.0\t2\t0 101 5\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t294\t0.0\t1\t0 294\n",
      "19\t64479\t0.0\t1\t63170 1309\n",
      "20\t130\t0.0\t1\t0 130\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1744_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1744_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1744_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1744_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1744_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1744_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1744_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1744_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        73,503 reads  @     79.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 5.84 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,503\n",
      "  Read 1 with adapter:                  73,431 (99.9%)\n",
      "  Read 2 with adapter:                  73,272 (99.7%)\n",
      "Pairs written (passing filters):        73,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,364,109 bp\n",
      "  Read 1:    21,683,094 bp\n",
      "  Read 2:    21,681,015 bp\n",
      "Total written (filtered):     40,503,730 bp (93.4%)\n",
      "  Read 1:    20,214,741 bp\n",
      "  Read 2:    20,288,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73431 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t472\t0.0\t1\t0 465 7\n",
      "20\t72736\t0.0\t2\t71219 1481 36\n",
      "21\t153\t0.0\t2\t0 149 4\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 73272 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t344\t0.0\t1\t0 344\n",
      "19\t72726\t0.0\t1\t71377 1349\n",
      "20\t202\t0.0\t1\t0 202\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1745_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1745_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1745_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1745_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1745_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1745_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1745_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1745_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           196 reads  @    261.4 µs/read;   0.23 M reads/minute\n",
      "Finished in 0.05 s (278 us/read; 0.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                196\n",
      "  Read 1 with adapter:                     194 (99.0%)\n",
      "  Read 2 with adapter:                      26 (13.3%)\n",
      "Pairs written (passing filters):           196 (100.0%)\n",
      "\n",
      "Total basepairs processed:        71,370 bp\n",
      "  Read 1:        57,820 bp\n",
      "  Read 2:        13,550 bp\n",
      "Total written (filtered):         66,970 bp (93.8%)\n",
      "  Read 1:        53,940 bp\n",
      "  Read 2:        13,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 194 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t192\t0.0\t2\t186 6\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t26\t0.0\t1\t0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1746_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1746_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1746_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1746_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1746_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1746_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1746_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1746_143_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,189 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 6.58 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,189\n",
      "  Read 1 with adapter:                  81,111 (99.9%)\n",
      "  Read 2 with adapter:                  80,834 (99.6%)\n",
      "Pairs written (passing filters):        81,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,886,479 bp\n",
      "  Read 1:    23,950,620 bp\n",
      "  Read 2:    23,935,859 bp\n",
      "Total written (filtered):     44,728,967 bp (93.4%)\n",
      "  Read 1:    22,328,731 bp\n",
      "  Read 2:    22,400,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81111 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t480\t0.0\t1\t0 468 12\n",
      "20\t80412\t0.0\t2\t78567 1793 52\n",
      "21\t145\t0.0\t2\t0 141 4\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t390\t0.0\t1\t0 390\n",
      "19\t80277\t0.0\t1\t78517 1760\n",
      "20\t167\t0.0\t1\t0 167\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1747_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1747_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1747_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1747_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1747_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1747_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1747_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1747_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,967 reads  @     76.9 µs/read;   0.78 M reads/minute\n",
      "Finished in 6.31 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,967\n",
      "  Read 1 with adapter:                  81,890 (99.9%)\n",
      "  Read 2 with adapter:                  81,742 (99.7%)\n",
      "Pairs written (passing filters):        81,967 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,358,659 bp\n",
      "  Read 1:    24,179,620 bp\n",
      "  Read 2:    24,179,039 bp\n",
      "Total written (filtered):     45,168,369 bp (93.4%)\n",
      "  Read 1:    22,542,192 bp\n",
      "  Read 2:    22,626,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81890 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t535\t0.0\t1\t0 522 13\n",
      "20\t81134\t0.0\t2\t79588 1505 41\n",
      "21\t135\t0.0\t2\t0 132 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81742 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t372\t0.0\t1\t0 372\n",
      "19\t81234\t0.0\t1\t79743 1491\n",
      "20\t136\t0.0\t1\t0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1748_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1748_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1748_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1748_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1748_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1748_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1748_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1748_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,288 reads  @     81.9 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.37 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,288\n",
      "  Read 1 with adapter:                  53,223 (99.9%)\n",
      "  Read 2 with adapter:                  53,106 (99.7%)\n",
      "Pairs written (passing filters):        53,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,437,115 bp\n",
      "  Read 1:    15,719,770 bp\n",
      "  Read 2:    15,717,345 bp\n",
      "Total written (filtered):     29,363,965 bp (93.4%)\n",
      "  Read 1:    14,655,489 bp\n",
      "  Read 2:    14,708,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t315\t0.0\t1\t0 307 8\n",
      "20\t52756\t0.0\t2\t51648 1080 28\n",
      "21\t96\t0.0\t2\t0 93 3\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t280\t0.0\t1\t0 280\n",
      "19\t52691\t0.0\t1\t51713 978\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1749_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1749_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1749_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1749_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1749_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1749_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1749_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1749_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00           276 reads  @    221.9 µs/read;   0.27 M reads/minute\n",
      "Finished in 0.06 s (233 us/read; 0.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                276\n",
      "  Read 1 with adapter:                     276 (100.0%)\n",
      "  Read 2 with adapter:                      23 (8.3%)\n",
      "Pairs written (passing filters):           276 (100.0%)\n",
      "\n",
      "Total basepairs processed:        98,412 bp\n",
      "  Read 1:        81,420 bp\n",
      "  Read 2:        16,992 bp\n",
      "Total written (filtered):         92,435 bp (93.9%)\n",
      "  Read 1:        75,902 bp\n",
      "  Read 2:        16,533 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 276 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t274\t0.0\t2\t266 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 23 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t22\t0.0\t1\t0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1750_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1750_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1750_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1750_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1750_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1750_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1750_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1750_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,075 reads  @     84.4 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.81 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,075\n",
      "  Read 1 with adapter:                  45,016 (99.9%)\n",
      "  Read 2 with adapter:                  44,748 (99.3%)\n",
      "Pairs written (passing filters):        45,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,580,158 bp\n",
      "  Read 1:    13,296,631 bp\n",
      "  Read 2:    13,283,527 bp\n",
      "Total written (filtered):     24,829,931 bp (93.4%)\n",
      "  Read 1:    12,396,456 bp\n",
      "  Read 2:    12,433,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t252\t0.0\t1\t0 251 1\n",
      "20\t44634\t0.0\t2\t43640 968 26\n",
      "21\t77\t0.0\t2\t0 75 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t231\t0.0\t1\t0 231\n",
      "19\t44446\t0.0\t1\t43164 1282\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1751_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1751_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1751_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1751_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1751_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1751_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1751_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1751_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        58,687 reads  @     86.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 5.06 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,687\n",
      "  Read 1 with adapter:                  58,611 (99.9%)\n",
      "  Read 2 with adapter:                  58,440 (99.6%)\n",
      "Pairs written (passing filters):        58,687 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,624,478 bp\n",
      "  Read 1:    17,311,839 bp\n",
      "  Read 2:    17,312,639 bp\n",
      "Total written (filtered):     32,342,304 bp (93.4%)\n",
      "  Read 1:    16,139,849 bp\n",
      "  Read 2:    16,202,455 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 58611 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t365\t0.0\t1\t0 352 13\n",
      "20\t58095\t0.0\t2\t56784 1283 28\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t264\t0.0\t1\t0 264\n",
      "19\t58088\t0.0\t1\t56482 1606\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1752_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1752_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1752_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1752_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1752_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1752_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1752_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1752_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,676 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.31 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,676\n",
      "  Read 1 with adapter:                  69,591 (99.9%)\n",
      "  Read 2 with adapter:                  69,310 (99.5%)\n",
      "Pairs written (passing filters):        69,676 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,106,835 bp\n",
      "  Read 1:    20,554,214 bp\n",
      "  Read 2:    20,552,621 bp\n",
      "Total written (filtered):     38,398,616 bp (93.4%)\n",
      "  Read 1:    19,162,669 bp\n",
      "  Read 2:    19,235,947 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 69591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t399\t0.0\t1\t0 394 5\n",
      "20\t69043\t0.0\t2\t67579 1425 39\n",
      "21\t90\t0.0\t2\t0 87 3\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 69310 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t371\t0.0\t1\t0 371\n",
      "19\t68784\t0.0\t1\t66919 1865\n",
      "20\t155\t0.0\t1\t0 155\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1753_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1753_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1753_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1753_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1753_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1753_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1753_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1753_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           226 reads  @    260.0 µs/read;   0.23 M reads/minute\n",
      "Finished in 0.06 s (274 us/read; 0.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                226\n",
      "  Read 1 with adapter:                     225 (99.6%)\n",
      "  Read 2 with adapter:                      31 (13.7%)\n",
      "Pairs written (passing filters):           225 (99.6%)\n",
      "\n",
      "Total basepairs processed:        84,916 bp\n",
      "  Read 1:        66,669 bp\n",
      "  Read 2:        18,247 bp\n",
      "Total written (filtered):         79,523 bp (93.6%)\n",
      "  Read 1:        61,896 bp\n",
      "  Read 2:        17,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t223\t0.0\t2\t214 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t31\t0.0\t1\t0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1754_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1754_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1754_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1754_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1754_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1754_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1754_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1754_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,007 reads  @     80.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.40 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,007\n",
      "  Read 1 with adapter:                  66,949 (99.9%)\n",
      "  Read 2 with adapter:                  66,572 (99.4%)\n",
      "Pairs written (passing filters):        67,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,521,232 bp\n",
      "  Read 1:    19,766,939 bp\n",
      "  Read 2:    19,754,293 bp\n",
      "Total written (filtered):     36,917,907 bp (93.4%)\n",
      "  Read 1:    18,428,273 bp\n",
      "  Read 2:    18,489,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t451\t0.0\t1\t0 443 8\n",
      "20\t66327\t0.0\t2\t64977 1303 47\n",
      "21\t117\t0.0\t2\t0 116 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t326\t0.0\t1\t0 326\n",
      "19\t66129\t0.0\t1\t64489 1640\n",
      "20\t117\t0.0\t1\t0 117\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1755_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1755_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1755_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1755_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1755_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1755_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1755_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1755_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,743 reads  @     79.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.07 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,743\n",
      "  Read 1 with adapter:                  63,701 (99.9%)\n",
      "  Read 2 with adapter:                  63,475 (99.6%)\n",
      "Pairs written (passing filters):        63,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,607,618 bp\n",
      "  Read 1:    18,803,933 bp\n",
      "  Read 2:    18,803,685 bp\n",
      "Total written (filtered):     35,127,996 bp (93.4%)\n",
      "  Read 1:    17,530,193 bp\n",
      "  Read 2:    17,597,803 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63701 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t399\t0.0\t1\t0 394 5\n",
      "20\t63154\t0.0\t2\t61888 1224 42\n",
      "21\t97\t0.0\t2\t0 94 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63475 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t264\t0.0\t1\t0 264\n",
      "19\t63090\t0.0\t1\t61607 1483\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1756_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1756_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1756_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1756_163_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1756_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1756_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1756_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1756_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        60,921 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 5.03 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,921\n",
      "  Read 1 with adapter:                  60,864 (99.9%)\n",
      "  Read 2 with adapter:                  60,617 (99.5%)\n",
      "Pairs written (passing filters):        60,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,940,989 bp\n",
      "  Read 1:    17,971,606 bp\n",
      "  Read 2:    17,969,383 bp\n",
      "Total written (filtered):     33,572,452 bp (93.4%)\n",
      "  Read 1:    16,754,617 bp\n",
      "  Read 2:    16,817,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 60864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t418\t0.0\t1\t0 410 8\n",
      "20\t60282\t0.0\t2\t59080 1173 29\n",
      "21\t109\t0.0\t2\t0 107 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 60617 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t302\t0.0\t1\t0 302\n",
      "19\t60188\t0.0\t1\t58614 1574\n",
      "20\t127\t0.0\t1\t0 127\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1757_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1757_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1757_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1757_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1757_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1757_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1757_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1757_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           284 reads  @    170.5 µs/read;   0.35 M reads/minute\n",
      "Finished in 0.05 s (181 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                284\n",
      "  Read 1 with adapter:                     283 (99.6%)\n",
      "  Read 2 with adapter:                      19 (6.7%)\n",
      "Pairs written (passing filters):           284 (100.0%)\n",
      "\n",
      "Total basepairs processed:        98,358 bp\n",
      "  Read 1:        83,543 bp\n",
      "  Read 2:        14,815 bp\n",
      "Total written (filtered):         92,318 bp (93.9%)\n",
      "  Read 1:        77,883 bp\n",
      "  Read 2:        14,435 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t281\t0.0\t2\t275 5 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 19 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t19\t0.0\t1\t0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1758_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1758_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1758_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1758_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1758_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1758_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1758_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1758_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,369 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.48 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,369\n",
      "  Read 1 with adapter:                  57,315 (99.9%)\n",
      "  Read 2 with adapter:                  57,153 (99.6%)\n",
      "Pairs written (passing filters):        57,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,837,939 bp\n",
      "  Read 1:    16,923,203 bp\n",
      "  Read 2:    16,914,736 bp\n",
      "Total written (filtered):     31,606,114 bp (93.4%)\n",
      "  Read 1:    15,777,091 bp\n",
      "  Read 2:    15,829,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 57315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t342\t0.0\t1\t0 334 8\n",
      "20\t56825\t0.0\t2\t55575 1223 27\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 57153 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t307\t0.0\t1\t0 307\n",
      "19\t56733\t0.0\t1\t55504 1229\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1759_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1759_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1759_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1759_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1759_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1759_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1759_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1759_169_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,203 reads  @     81.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 5.33 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,203\n",
      "  Read 1 with adapter:                  65,136 (99.9%)\n",
      "  Read 2 with adapter:                  64,993 (99.7%)\n",
      "Pairs written (passing filters):        65,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,469,160 bp\n",
      "  Read 1:    19,234,618 bp\n",
      "  Read 2:    19,234,542 bp\n",
      "Total written (filtered):     35,932,011 bp (93.4%)\n",
      "  Read 1:    17,932,127 bp\n",
      "  Read 2:    17,999,884 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65136 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t396\t0.0\t1\t0 392 4\n",
      "20\t64558\t0.0\t2\t63033 1487 38\n",
      "21\t113\t0.0\t2\t0 111 2\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64993 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t340\t0.0\t1\t0 340\n",
      "19\t64522\t0.0\t1\t63106 1416\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1760_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1760_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1760_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1760_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1760_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1760_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1760_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1760_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        75,220 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 5.65 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,220\n",
      "  Read 1 with adapter:                  75,142 (99.9%)\n",
      "  Read 2 with adapter:                  75,033 (99.8%)\n",
      "Pairs written (passing filters):        75,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,376,343 bp\n",
      "  Read 1:    22,189,298 bp\n",
      "  Read 2:    22,187,045 bp\n",
      "Total written (filtered):     41,448,337 bp (93.4%)\n",
      "  Read 1:    20,686,710 bp\n",
      "  Read 2:    20,761,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t448\t0.0\t1\t0 443 5\n",
      "20\t74481\t0.0\t2\t72976 1465 40\n",
      "21\t134\t0.0\t2\t0 133 1\n",
      "22\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t403\t0.0\t1\t0 403\n",
      "19\t74436\t0.0\t1\t73068 1368\n",
      "20\t194\t0.0\t1\t0 194\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1761_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1761_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1761_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1761_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1761_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1761_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1761_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1761_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           196 reads  @    246.4 µs/read;   0.24 M reads/minute\n",
      "Finished in 0.05 s (263 us/read; 0.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                196\n",
      "  Read 1 with adapter:                     194 (99.0%)\n",
      "  Read 2 with adapter:                      20 (10.2%)\n",
      "Pairs written (passing filters):           196 (100.0%)\n",
      "\n",
      "Total basepairs processed:        70,339 bp\n",
      "  Read 1:        57,820 bp\n",
      "  Read 2:        12,519 bp\n",
      "Total written (filtered):         66,063 bp (93.9%)\n",
      "  Read 1:        53,944 bp\n",
      "  Read 2:        12,119 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 194 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t190\t0.0\t2\t186 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 20 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t20\t0.0\t1\t0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1762_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1762_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1762_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1762_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1762_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1762_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1762_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1762_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,564 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.80 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,564\n",
      "  Read 1 with adapter:                  66,488 (99.9%)\n",
      "  Read 2 with adapter:                  66,328 (99.6%)\n",
      "Pairs written (passing filters):        66,564 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,262,819 bp\n",
      "  Read 1:    19,636,350 bp\n",
      "  Read 2:    19,626,469 bp\n",
      "Total written (filtered):     36,673,197 bp (93.4%)\n",
      "  Read 1:    18,306,751 bp\n",
      "  Read 2:    18,366,446 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t364\t0.0\t1\t0 352 12\n",
      "20\t65912\t0.0\t2\t64231 1637 44\n",
      "21\t157\t0.0\t2\t0 152 5\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t323\t0.0\t1\t0 323\n",
      "19\t65891\t0.0\t1\t64392 1499\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1763_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1763_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1763_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1763_177_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1763_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1763_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1763_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1763_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,979 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 6.12 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,979\n",
      "  Read 1 with adapter:                  81,900 (99.9%)\n",
      "  Read 2 with adapter:                  81,767 (99.7%)\n",
      "Pairs written (passing filters):        81,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,367,402 bp\n",
      "  Read 1:    24,183,633 bp\n",
      "  Read 2:    24,183,769 bp\n",
      "Total written (filtered):     45,176,299 bp (93.4%)\n",
      "  Read 1:    22,545,883 bp\n",
      "  Read 2:    22,630,416 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t461\t0.0\t1\t0 455 6\n",
      "20\t81190\t0.0\t2\t79672 1465 53\n",
      "21\t183\t0.0\t2\t0 179 4\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t373\t0.0\t1\t0 373\n",
      "19\t81241\t0.0\t1\t79726 1515\n",
      "20\t153\t0.0\t1\t0 153\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1764_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1764_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1764_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1764_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1764_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1764_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1764_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1764_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,888 reads  @     73.5 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.77 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,888\n",
      "  Read 1 with adapter:                  64,822 (99.9%)\n",
      "  Read 2 with adapter:                  64,715 (99.7%)\n",
      "Pairs written (passing filters):        64,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,282,843 bp\n",
      "  Read 1:    19,141,937 bp\n",
      "  Read 2:    19,140,906 bp\n",
      "Total written (filtered):     35,757,148 bp (93.4%)\n",
      "  Read 1:    17,845,621 bp\n",
      "  Read 2:    17,911,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t340\t0.0\t1\t0 331 9\n",
      "20\t64253\t0.0\t2\t63122 1104 27\n",
      "21\t170\t0.0\t2\t0 166 4\n",
      "22\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t360\t0.0\t1\t0 360\n",
      "19\t64201\t0.0\t1\t62954 1247\n",
      "20\t154\t0.0\t1\t0 154\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1765_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1765_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1765_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1765_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1765_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1765_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1765_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1765_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           186 reads  @    245.4 µs/read;   0.24 M reads/minute\n",
      "Finished in 0.05 s (261 us/read; 0.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                186\n",
      "  Read 1 with adapter:                     184 (98.9%)\n",
      "  Read 2 with adapter:                      13 (7.0%)\n",
      "Pairs written (passing filters):           186 (100.0%)\n",
      "\n",
      "Total basepairs processed:        63,186 bp\n",
      "  Read 1:        54,870 bp\n",
      "  Read 2:         8,316 bp\n",
      "Total written (filtered):         59,247 bp (93.8%)\n",
      "  Read 1:        51,191 bp\n",
      "  Read 2:         8,056 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t183\t0.0\t2\t180 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1766_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1766_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1766_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1766_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1766_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1766_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1766_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1766_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,504 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.94 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,504\n",
      "  Read 1 with adapter:                  48,428 (99.8%)\n",
      "  Read 2 with adapter:                  48,245 (99.5%)\n",
      "Pairs written (passing filters):        48,504 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,607,234 bp\n",
      "  Read 1:    14,308,668 bp\n",
      "  Read 2:    14,298,566 bp\n",
      "Total written (filtered):     26,722,369 bp (93.4%)\n",
      "  Read 1:    13,340,315 bp\n",
      "  Read 2:    13,382,054 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t331\t0.0\t1\t0 324 7\n",
      "20\t47947\t0.0\t2\t46852 1060 35\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t245\t0.0\t1\t0 245\n",
      "19\t47898\t0.0\t1\t46813 1085\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1767_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1767_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1767_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1767_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1767_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1767_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1767_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1767_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,565 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.96 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,565\n",
      "  Read 1 with adapter:                  48,514 (99.9%)\n",
      "  Read 2 with adapter:                  48,411 (99.7%)\n",
      "Pairs written (passing filters):        48,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,652,447 bp\n",
      "  Read 1:    14,326,167 bp\n",
      "  Read 2:    14,326,280 bp\n",
      "Total written (filtered):     26,762,685 bp (93.4%)\n",
      "  Read 1:    13,356,077 bp\n",
      "  Read 2:    13,406,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48514 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t332\t0.0\t1\t0 320 12\n",
      "20\t48004\t0.0\t2\t46954 1025 25\n",
      "21\t114\t0.0\t2\t0 113 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t48106\t0.0\t1\t47101 1005\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1768_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1768_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1768_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1768_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1768_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1768_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1768_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1768_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        62,266 reads  @     83.3 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.19 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,266\n",
      "  Read 1 with adapter:                  62,152 (99.8%)\n",
      "  Read 2 with adapter:                  61,951 (99.5%)\n",
      "Pairs written (passing filters):        62,266 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,732,962 bp\n",
      "  Read 1:    18,368,210 bp\n",
      "  Read 2:    18,364,752 bp\n",
      "Total written (filtered):     34,313,251 bp (93.4%)\n",
      "  Read 1:    17,125,438 bp\n",
      "  Read 2:    17,187,813 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t419\t0.0\t1\t0 407 12\n",
      "20\t61539\t0.0\t2\t60121 1384 34\n",
      "21\t117\t0.0\t2\t0 115 2\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t298\t0.0\t1\t0 298\n",
      "19\t61485\t0.0\t1\t60007 1478\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1769_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1769_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1769_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1769_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1769_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1769_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1769_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1769_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           200 reads  @    289.4 µs/read;   0.21 M reads/minute\n",
      "Finished in 0.06 s (306 us/read; 0.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                200\n",
      "  Read 1 with adapter:                     200 (100.0%)\n",
      "  Read 2 with adapter:                      27 (13.5%)\n",
      "Pairs written (passing filters):           200 (100.0%)\n",
      "\n",
      "Total basepairs processed:        74,426 bp\n",
      "  Read 1:        59,000 bp\n",
      "  Read 2:        15,426 bp\n",
      "Total written (filtered):         69,886 bp (93.9%)\n",
      "  Read 1:        55,000 bp\n",
      "  Read 2:        14,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t200\t0.0\t2\t197 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t27\t0.0\t1\t0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1770_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1770_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1770_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1770_191_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1770_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1770_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1770_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1770_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,826 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.63 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,826\n",
      "  Read 1 with adapter:                  64,751 (99.9%)\n",
      "  Read 2 with adapter:                  64,545 (99.6%)\n",
      "Pairs written (passing filters):        64,826 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,233,117 bp\n",
      "  Read 1:    19,123,320 bp\n",
      "  Read 2:    19,109,797 bp\n",
      "Total written (filtered):     35,712,362 bp (93.4%)\n",
      "  Read 1:    17,828,720 bp\n",
      "  Read 2:    17,883,642 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t583\t0.0\t1\t0 572 11\n",
      "20\t63979\t0.0\t2\t62607 1340 32\n",
      "21\t131\t0.0\t2\t0 126 5\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64545 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t349\t0.0\t1\t0 349\n",
      "19\t64047\t0.0\t1\t62675 1372\n",
      "20\t149\t0.0\t1\t0 149\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1771_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1771_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1771_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1771_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1771_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1771_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1771_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1771_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,060 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.66 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,060\n",
      "  Read 1 with adapter:                  59,969 (99.8%)\n",
      "  Read 2 with adapter:                  59,822 (99.6%)\n",
      "Pairs written (passing filters):        60,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,433,120 bp\n",
      "  Read 1:    17,715,608 bp\n",
      "  Read 2:    17,717,512 bp\n",
      "Total written (filtered):     33,097,723 bp (93.4%)\n",
      "  Read 1:    16,516,654 bp\n",
      "  Read 2:    16,581,069 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t578\t0.0\t1\t0 565 13\n",
      "20\t59215\t0.0\t2\t57947 1232 36\n",
      "21\t104\t0.0\t2\t0 102 2\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t296\t0.0\t1\t0 296\n",
      "19\t59405\t0.0\t1\t58090 1315\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1772_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1772_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1772_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1772_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1772_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1772_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1772_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1772_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,718 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.76 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,718\n",
      "  Read 1 with adapter:                  47,667 (99.9%)\n",
      "  Read 2 with adapter:                  47,499 (99.5%)\n",
      "Pairs written (passing filters):        47,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,151,845 bp\n",
      "  Read 1:    14,075,895 bp\n",
      "  Read 2:    14,075,950 bp\n",
      "Total written (filtered):     26,296,499 bp (93.4%)\n",
      "  Read 1:    13,122,867 bp\n",
      "  Read 2:    13,173,632 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47667 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t437\t0.0\t1\t0 426 11\n",
      "20\t47094\t0.0\t2\t46000 1065 29\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t47124\t0.0\t1\t45918 1206\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1773_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1773_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1773_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1773_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1773_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-btv2ny_v/1773_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1773_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-f7jfu71p/97d2e28c-3f7d-4530-9c3d-116c62ca0478/data/1773_197_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: f-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: f-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-ousbxesc')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e9195c65-f4e0-4351-9d8e-91fa76ef1d40>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.f import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/Hirai-Lib2-manifest_18S.csv \\\n",
    "  --output-path f-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.f trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences f-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences f-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.f summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data f-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization f-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('f-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: f-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: f-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: f-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.f Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs f-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 265 \\\n",
    "  --p-trunc-len-r 222 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table f-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats f-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: f-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_uucrio6')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 9bdec81a-b86f-47fe-9502-b6b33bd7cb2d>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.f view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table f-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization f-table.qzv \n",
    "q2.Visualization.load('f-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: f-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.f view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization f-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P2107-Lib1-manifest_18S.csv as PairedEndFastqManifestPhred33 to g-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1945_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1945_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1945_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1945_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1945_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1945_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1945_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1945_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,478 reads  @     74.5 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.47 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,478\n",
      "  Read 1 with adapter:                  46,440 (99.9%)\n",
      "  Read 2 with adapter:                  46,306 (99.6%)\n",
      "Pairs written (passing filters):        46,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,420,427 bp\n",
      "  Read 1:    13,709,987 bp\n",
      "  Read 2:    13,710,440 bp\n",
      "Total written (filtered):     25,612,103 bp (93.4%)\n",
      "  Read 1:    12,781,331 bp\n",
      "  Read 2:    12,830,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t221\t0.0\t1\t0 220 1\n",
      "20\t46128\t0.0\t2\t45453 658 17\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t46024\t0.0\t1\t45352 672\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1946_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1946_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1946_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1946_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1946_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1946_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1946_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1946_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,618 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.72 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,618\n",
      "  Read 1 with adapter:                  38,580 (99.9%)\n",
      "  Read 2 with adapter:                  38,524 (99.8%)\n",
      "Pairs written (passing filters):        38,618 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,783,328 bp\n",
      "  Read 1:    11,391,794 bp\n",
      "  Read 2:    11,391,534 bp\n",
      "Total written (filtered):     21,279,976 bp (93.4%)\n",
      "  Read 1:    10,620,309 bp\n",
      "  Read 2:    10,659,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38580 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t191\t0.0\t1\t0 189 2\n",
      "20\t38278\t0.0\t2\t37645 621 12\n",
      "21\t78\t0.0\t2\t0 76 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t166\t0.0\t1\t0 166\n",
      "19\t38281\t0.0\t1\t37661 620\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1947_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1947_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1947_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1947_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1947_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1947_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1947_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1947_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,418 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.47 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,418\n",
      "  Read 1 with adapter:                  34,356 (99.8%)\n",
      "  Read 2 with adapter:                  34,314 (99.7%)\n",
      "Pairs written (passing filters):        34,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,306,073 bp\n",
      "  Read 1:    10,153,011 bp\n",
      "  Read 2:    10,153,062 bp\n",
      "Total written (filtered):     18,967,199 bp (93.4%)\n",
      "  Read 1:     9,466,010 bp\n",
      "  Read 2:     9,501,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t200\t0.0\t1\t0 194 6\n",
      "20\t34066\t0.0\t2\t33545 510 11\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t157\t0.0\t1\t0 157\n",
      "19\t34093\t0.0\t1\t33549 544\n",
      "20\t64\t0.0\t1\t0 64\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1948_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1948_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1948_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1948_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1948_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1948_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1948_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1948_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,736 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.34 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,736\n",
      "  Read 1 with adapter:                  32,706 (99.9%)\n",
      "  Read 2 with adapter:                  32,605 (99.6%)\n",
      "Pairs written (passing filters):        32,736 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,314,208 bp\n",
      "  Read 1:     9,657,112 bp\n",
      "  Read 2:     9,657,096 bp\n",
      "Total written (filtered):     18,040,834 bp (93.4%)\n",
      "  Read 1:     9,003,102 bp\n",
      "  Read 2:     9,037,732 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t173\t0.0\t1\t0 169 4\n",
      "20\t32453\t0.0\t2\t31866 581 6\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t201\t0.0\t1\t0 201\n",
      "19\t32334\t0.0\t1\t31748 586\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1949_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1949_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1949_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1949_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1949_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1949_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1949_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1949_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,495 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.65 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,495\n",
      "  Read 1 with adapter:                  49,418 (99.8%)\n",
      "  Read 2 with adapter:                  49,327 (99.7%)\n",
      "Pairs written (passing filters):        49,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,198,396 bp\n",
      "  Read 1:    14,599,160 bp\n",
      "  Read 2:    14,599,236 bp\n",
      "Total written (filtered):     27,273,141 bp (93.4%)\n",
      "  Read 1:    13,610,972 bp\n",
      "  Read 2:    13,662,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49418 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t247\t0.0\t1\t0 244 3\n",
      "20\t49072\t0.0\t2\t48235 803 34\n",
      "21\t67\t0.0\t2\t0 64 3\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t224\t0.0\t1\t0 224\n",
      "19\t49025\t0.0\t1\t48144 881\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1950_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1950_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1950_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1950_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1950_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1950_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1950_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1950_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,769 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.79 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,769\n",
      "  Read 1 with adapter:                  35,722 (99.9%)\n",
      "  Read 2 with adapter:                  35,604 (99.5%)\n",
      "Pairs written (passing filters):        35,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,089,969 bp\n",
      "  Read 1:    10,550,527 bp\n",
      "  Read 2:    10,539,442 bp\n",
      "Total written (filtered):     19,699,288 bp (93.4%)\n",
      "  Read 1:     9,836,238 bp\n",
      "  Read 2:     9,863,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t200\t0.0\t1\t0 199 1\n",
      "20\t35443\t0.0\t2\t34867 562 14\n",
      "21\t57\t0.0\t2\t0 55 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35604 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t138\t0.0\t1\t0 138\n",
      "19\t35412\t0.0\t1\t34800 612\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1951_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1951_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1951_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1951_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1951_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1951_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1951_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1951_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,138 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.74 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,138\n",
      "  Read 1 with adapter:                  37,073 (99.8%)\n",
      "  Read 2 with adapter:                  37,010 (99.7%)\n",
      "Pairs written (passing filters):        37,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,906,465 bp\n",
      "  Read 1:    10,955,470 bp\n",
      "  Read 2:    10,950,995 bp\n",
      "Total written (filtered):     20,462,054 bp (93.4%)\n",
      "  Read 1:    10,214,145 bp\n",
      "  Read 2:    10,247,909 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37073 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t184\t0.0\t1\t0 178 6\n",
      "20\t36814\t0.0\t2\t36188 617 9\n",
      "21\t57\t0.0\t2\t0 57\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37010 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t36778\t0.0\t1\t36156 622\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1952_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1952_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1952_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1952_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1952_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1952_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1952_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1952_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,764 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.14 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,764\n",
      "  Read 1 with adapter:                  41,724 (99.9%)\n",
      "  Read 2 with adapter:                  41,302 (98.9%)\n",
      "Pairs written (passing filters):        41,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,617,226 bp\n",
      "  Read 1:    12,318,462 bp\n",
      "  Read 2:    12,298,764 bp\n",
      "Total written (filtered):     22,998,234 bp (93.4%)\n",
      "  Read 1:    11,484,101 bp\n",
      "  Read 2:    11,514,133 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41724 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t230\t0.0\t1\t0 224 6\n",
      "20\t41385\t0.0\t2\t40690 673 22\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t41061\t0.0\t1\t40449 612\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1953_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1953_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1953_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1953_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1953_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1953_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1953_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1953_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,314 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.49 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,314\n",
      "  Read 1 with adapter:                  34,299 (100.0%)\n",
      "  Read 2 with adapter:                  34,212 (99.7%)\n",
      "Pairs written (passing filters):        34,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,243,676 bp\n",
      "  Read 1:    10,122,356 bp\n",
      "  Read 2:    10,121,320 bp\n",
      "Total written (filtered):     18,907,872 bp (93.4%)\n",
      "  Read 1:     9,436,489 bp\n",
      "  Read 2:     9,471,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34299 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t176\t0.0\t1\t0 170 6\n",
      "20\t34040\t0.0\t2\t33534 493 13\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34212 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t33983\t0.0\t1\t33464 519\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1954_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1954_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1954_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1954_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1954_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1954_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1954_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1954_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        28,546 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.95 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,546\n",
      "  Read 1 with adapter:                  28,531 (99.9%)\n",
      "  Read 2 with adapter:                  28,466 (99.7%)\n",
      "Pairs written (passing filters):        28,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,841,613 bp\n",
      "  Read 1:     8,421,065 bp\n",
      "  Read 2:     8,420,548 bp\n",
      "Total written (filtered):     15,730,348 bp (93.4%)\n",
      "  Read 1:     7,850,555 bp\n",
      "  Read 2:     7,879,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t185\t0.0\t1\t0 179 6\n",
      "20\t28258\t0.0\t2\t27784 465 9\n",
      "21\t69\t0.0\t2\t0 67 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t141\t0.0\t1\t0 141\n",
      "19\t28283\t0.0\t1\t27794 489\n",
      "20\t42\t0.0\t1\t0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1955_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1955_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1955_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1955_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1955_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1955_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1955_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1955_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,201 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.13 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,201\n",
      "  Read 1 with adapter:                  31,184 (99.9%)\n",
      "  Read 2 with adapter:                  31,112 (99.7%)\n",
      "Pairs written (passing filters):        31,201 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,408,263 bp\n",
      "  Read 1:     9,204,124 bp\n",
      "  Read 2:     9,204,139 bp\n",
      "Total written (filtered):     17,193,613 bp (93.4%)\n",
      "  Read 1:     8,580,553 bp\n",
      "  Read 2:     8,613,060 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t166\t0.0\t1\t0 162 4\n",
      "20\t30939\t0.0\t2\t30424 501 14\n",
      "21\t53\t0.0\t2\t0 51 2\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t115\t0.0\t1\t0 115\n",
      "19\t30931\t0.0\t1\t30434 497\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1956_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1956_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1956_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1956_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1956_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1956_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1956_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1956_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        25,967 reads  @     77.4 µs/read;   0.78 M reads/minute\n",
      "Finished in 2.02 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,967\n",
      "  Read 1 with adapter:                  25,943 (99.9%)\n",
      "  Read 2 with adapter:                  25,806 (99.4%)\n",
      "Pairs written (passing filters):        25,967 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,311,583 bp\n",
      "  Read 1:     7,660,004 bp\n",
      "  Read 2:     7,651,579 bp\n",
      "Total written (filtered):     14,302,563 bp (93.4%)\n",
      "  Read 1:     7,141,216 bp\n",
      "  Read 2:     7,161,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 25943 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t130\t0.0\t1\t0 128 2\n",
      "20\t25753\t0.0\t2\t25324 419 10\n",
      "21\t42\t0.0\t2\t0 40 2\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t139\t0.0\t1\t0 139\n",
      "19\t25610\t0.0\t1\t25143 467\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1957_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1957_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1957_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1957_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1957_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1957_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1957_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1957_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,213 reads  @     75.3 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.43 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,213\n",
      "  Read 1 with adapter:                  32,189 (99.9%)\n",
      "  Read 2 with adapter:                  32,109 (99.7%)\n",
      "Pairs written (passing filters):        32,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,004,456 bp\n",
      "  Read 1:     9,502,262 bp\n",
      "  Read 2:     9,502,194 bp\n",
      "Total written (filtered):     17,750,805 bp (93.4%)\n",
      "  Read 1:     8,858,596 bp\n",
      "  Read 2:     8,892,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t183\t0.0\t1\t0 178 5\n",
      "20\t31922\t0.0\t2\t31396 518 8\n",
      "21\t59\t0.0\t2\t0 57 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32109 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t151\t0.0\t1\t0 151\n",
      "19\t31893\t0.0\t1\t31332 561\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1958_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1958_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1958_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1958_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1958_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1958_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1958_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1958_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,156 reads  @     67.0 µs/read;   0.90 M reads/minute\n",
      "Finished in 1.82 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,156\n",
      "  Read 1 with adapter:                  27,141 (99.9%)\n",
      "  Read 2 with adapter:                  27,049 (99.6%)\n",
      "Pairs written (passing filters):        27,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,017,009 bp\n",
      "  Read 1:     8,011,018 bp\n",
      "  Read 2:     8,005,991 bp\n",
      "Total written (filtered):     14,960,407 bp (93.4%)\n",
      "  Read 1:     7,468,276 bp\n",
      "  Read 2:     7,492,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27141 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t149\t0.0\t1\t0 148 1\n",
      "20\t26915\t0.0\t2\t26457 450 8\n",
      "21\t47\t0.0\t2\t0 46 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t126\t0.0\t1\t0 126\n",
      "19\t26868\t0.0\t1\t26434 434\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1959_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1959_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1959_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1959_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1959_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1959_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1959_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1959_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,194 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.28 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,194\n",
      "  Read 1 with adapter:                  32,156 (99.9%)\n",
      "  Read 2 with adapter:                  32,074 (99.6%)\n",
      "Pairs written (passing filters):        32,194 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,990,024 bp\n",
      "  Read 1:     9,497,215 bp\n",
      "  Read 2:     9,492,809 bp\n",
      "Total written (filtered):     17,737,771 bp (93.4%)\n",
      "  Read 1:     8,854,247 bp\n",
      "  Read 2:     8,883,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32156 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t209\t0.0\t1\t0 206 3\n",
      "20\t31864\t0.0\t2\t31311 538 15\n",
      "21\t57\t0.0\t2\t0 52 5\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t166\t0.0\t1\t0 166\n",
      "19\t31863\t0.0\t1\t31417 446\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1960_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1960_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1960_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1960_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1960_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1960_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1960_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1960_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        35,768 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.68 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,768\n",
      "  Read 1 with adapter:                  35,738 (99.9%)\n",
      "  Read 2 with adapter:                  35,328 (98.8%)\n",
      "Pairs written (passing filters):        35,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,080,903 bp\n",
      "  Read 1:    10,550,889 bp\n",
      "  Read 2:    10,530,014 bp\n",
      "Total written (filtered):     19,695,158 bp (93.4%)\n",
      "  Read 1:     9,836,280 bp\n",
      "  Read 2:     9,858,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t222\t0.0\t1\t0 219 3\n",
      "20\t35417\t0.0\t2\t34884 517 16\n",
      "21\t67\t0.0\t2\t0 61 6\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t35100\t0.0\t1\t34646 454\n",
      "20\t66\t0.0\t1\t0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1961_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1961_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1961_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1961_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1961_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1961_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1961_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1961_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,168 reads  @     71.0 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.00 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,168\n",
      "  Read 1 with adapter:                  42,140 (99.9%)\n",
      "  Read 2 with adapter:                  42,010 (99.6%)\n",
      "Pairs written (passing filters):        42,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,877,001 bp\n",
      "  Read 1:    12,439,321 bp\n",
      "  Read 2:    12,437,680 bp\n",
      "Total written (filtered):     23,236,274 bp (93.4%)\n",
      "  Read 1:    11,596,650 bp\n",
      "  Read 2:    11,639,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t219\t0.0\t1\t0 217 2\n",
      "20\t41829\t0.0\t2\t41136 681 12\n",
      "21\t62\t0.0\t2\t0 59 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42010 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t201\t0.0\t1\t0 201\n",
      "19\t41742\t0.0\t1\t41069 673\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1962_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1962_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1962_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1962_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1962_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1962_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1962_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1962_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,064 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.90 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,064\n",
      "  Read 1 with adapter:                  40,026 (99.9%)\n",
      "  Read 2 with adapter:                  39,903 (99.6%)\n",
      "Pairs written (passing filters):        40,064 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,636,251 bp\n",
      "  Read 1:    11,818,624 bp\n",
      "  Read 2:    11,817,627 bp\n",
      "Total written (filtered):     22,077,801 bp (93.4%)\n",
      "  Read 1:    11,018,216 bp\n",
      "  Read 2:    11,059,585 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t191\t0.0\t1\t0 186 5\n",
      "20\t39756\t0.0\t2\t39052 690 14\n",
      "21\t51\t0.0\t2\t0 47 4\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t39646\t0.0\t1\t38952 694\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1963_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1963_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1963_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1963_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1963_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1963_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1963_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1963_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,405 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.39 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,405\n",
      "  Read 1 with adapter:                  31,378 (99.9%)\n",
      "  Read 2 with adapter:                  31,302 (99.7%)\n",
      "Pairs written (passing filters):        31,405 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,527,620 bp\n",
      "  Read 1:     9,264,284 bp\n",
      "  Read 2:     9,263,336 bp\n",
      "Total written (filtered):     17,305,455 bp (93.4%)\n",
      "  Read 1:     8,636,775 bp\n",
      "  Read 2:     8,668,680 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t140\t0.0\t1\t0 137 3\n",
      "20\t31152\t0.0\t2\t30603 531 18\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t144\t0.0\t1\t0 144\n",
      "19\t31096\t0.0\t1\t30562 534\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1964_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1964_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1964_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1964_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1964_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1964_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1964_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1964_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,635 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 1.87 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,635\n",
      "  Read 1 with adapter:                  24,615 (99.9%)\n",
      "  Read 2 with adapter:                  24,524 (99.5%)\n",
      "Pairs written (passing filters):        24,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,534,235 bp\n",
      "  Read 1:     7,266,924 bp\n",
      "  Read 2:     7,267,311 bp\n",
      "Total written (filtered):     13,576,114 bp (93.4%)\n",
      "  Read 1:     6,774,687 bp\n",
      "  Read 2:     6,801,427 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24615 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t109\t0.0\t1\t0 105 4\n",
      "20\t24452\t0.0\t2\t24015 426 11\n",
      "21\t38\t0.0\t2\t0 36 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t140\t0.0\t1\t0 140\n",
      "19\t24316\t0.0\t1\t23823 493\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1965_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1965_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1965_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1965_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1965_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1965_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1965_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1965_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,355 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.30 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,355\n",
      "  Read 1 with adapter:                  30,336 (99.9%)\n",
      "  Read 2 with adapter:                  30,233 (99.6%)\n",
      "Pairs written (passing filters):        30,355 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,907,438 bp\n",
      "  Read 1:     8,954,491 bp\n",
      "  Read 2:     8,952,947 bp\n",
      "Total written (filtered):     16,726,493 bp (93.4%)\n",
      "  Read 1:     8,347,852 bp\n",
      "  Read 2:     8,378,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t156\t0.0\t1\t0 153 3\n",
      "20\t30092\t0.0\t2\t29522 562 8\n",
      "21\t61\t0.0\t2\t0 59 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t176\t0.0\t1\t0 176\n",
      "19\t30002\t0.0\t1\t29432 570\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1966_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1966_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1966_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1966_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1966_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1966_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1966_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1966_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        30,240 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.08 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,240\n",
      "  Read 1 with adapter:                  30,218 (99.9%)\n",
      "  Read 2 with adapter:                  30,122 (99.6%)\n",
      "Pairs written (passing filters):        30,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,836,477 bp\n",
      "  Read 1:     8,920,510 bp\n",
      "  Read 2:     8,915,967 bp\n",
      "Total written (filtered):     16,659,976 bp (93.4%)\n",
      "  Read 1:     8,316,250 bp\n",
      "  Read 2:     8,343,726 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t173\t0.0\t1\t0 169 4\n",
      "20\t29963\t0.0\t2\t29412 541 10\n",
      "21\t51\t0.0\t2\t0 51\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t133\t0.0\t1\t0 133\n",
      "19\t29933\t0.0\t1\t29457 476\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1967_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1967_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1967_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1967_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1967_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1967_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1967_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1967_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,867 reads  @     64.3 µs/read;   0.93 M reads/minute\n",
      "Finished in 1.67 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,867\n",
      "  Read 1 with adapter:                  25,850 (99.9%)\n",
      "  Read 2 with adapter:                  25,755 (99.6%)\n",
      "Pairs written (passing filters):        25,867 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,257,238 bp\n",
      "  Read 1:     7,630,758 bp\n",
      "  Read 2:     7,626,480 bp\n",
      "Total written (filtered):     14,251,054 bp (93.4%)\n",
      "  Read 1:     7,113,816 bp\n",
      "  Read 2:     7,137,238 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 25850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t115\t0.0\t1\t0 112 3\n",
      "20\t25670\t0.0\t2\t25220 439 11\n",
      "21\t37\t0.0\t2\t0 37\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t148\t0.0\t1\t0 148\n",
      "19\t25562\t0.0\t1\t25162 400\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1968_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1968_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1968_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1968_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1968_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1968_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1968_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1968_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,334 reads  @     73.6 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.24 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,334\n",
      "  Read 1 with adapter:                  30,291 (99.9%)\n",
      "  Read 2 with adapter:                  29,898 (98.6%)\n",
      "Pairs written (passing filters):        30,334 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,874,577 bp\n",
      "  Read 1:     8,948,333 bp\n",
      "  Read 2:     8,926,244 bp\n",
      "Total written (filtered):     16,700,910 bp (93.4%)\n",
      "  Read 1:     8,342,614 bp\n",
      "  Read 2:     8,358,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t162\t0.0\t1\t0 158 4\n",
      "20\t30063\t0.0\t2\t29554 498 11\n",
      "21\t43\t0.0\t2\t0 42 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t29692\t0.0\t1\t29254 438\n",
      "20\t46\t0.0\t1\t0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1969_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1969_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1969_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1969_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1969_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1969_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1969_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1969_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,975 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.30 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,975\n",
      "  Read 1 with adapter:                  47,933 (99.9%)\n",
      "  Read 2 with adapter:                  47,845 (99.7%)\n",
      "Pairs written (passing filters):        47,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,304,360 bp\n",
      "  Read 1:    14,151,803 bp\n",
      "  Read 2:    14,152,557 bp\n",
      "Total written (filtered):     26,436,935 bp (93.4%)\n",
      "  Read 1:    13,193,295 bp\n",
      "  Read 2:    13,243,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t238\t0.0\t1\t0 237 1\n",
      "20\t47604\t0.0\t2\t46828 757 19\n",
      "21\t68\t0.0\t2\t0 66 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t47547\t0.0\t1\t46740 807\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1970_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1970_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1970_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1970_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1970_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1970_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1970_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1970_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,708 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.30 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,708\n",
      "  Read 1 with adapter:                  33,680 (99.9%)\n",
      "  Read 2 with adapter:                  33,602 (99.7%)\n",
      "Pairs written (passing filters):        33,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,887,418 bp\n",
      "  Read 1:     9,943,591 bp\n",
      "  Read 2:     9,943,827 bp\n",
      "Total written (filtered):     18,575,624 bp (93.4%)\n",
      "  Read 1:     9,270,143 bp\n",
      "  Read 2:     9,305,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33680 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t204\t0.0\t1\t0 200 4\n",
      "20\t33399\t0.0\t2\t32827 559 13\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t159\t0.0\t1\t0 159\n",
      "19\t33376\t0.0\t1\t32862 514\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1971_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1971_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1971_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1971_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1971_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1971_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1971_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1971_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        36,853 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.02 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,853\n",
      "  Read 1 with adapter:                  36,832 (99.9%)\n",
      "  Read 2 with adapter:                  36,753 (99.7%)\n",
      "Pairs written (passing filters):        36,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,742,632 bp\n",
      "  Read 1:    10,871,626 bp\n",
      "  Read 2:    10,871,006 bp\n",
      "Total written (filtered):     20,307,934 bp (93.4%)\n",
      "  Read 1:    10,135,134 bp\n",
      "  Read 2:    10,172,800 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t240\t0.0\t1\t0 230 10\n",
      "20\t36485\t0.0\t2\t35875 596 14\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t36512\t0.0\t1\t36006 506\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1972_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1972_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1972_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1972_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1972_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1972_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1972_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1972_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        29,593 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.31 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,593\n",
      "  Read 1 with adapter:                  29,567 (99.9%)\n",
      "  Read 2 with adapter:                  29,465 (99.6%)\n",
      "Pairs written (passing filters):        29,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,459,304 bp\n",
      "  Read 1:     8,729,665 bp\n",
      "  Read 2:     8,729,639 bp\n",
      "Total written (filtered):     16,308,375 bp (93.4%)\n",
      "  Read 1:     8,138,462 bp\n",
      "  Read 2:     8,169,913 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29567 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t194\t0.0\t1\t0 187 7\n",
      "20\t29302\t0.0\t2\t28823 461 18\n",
      "21\t49\t0.0\t2\t0 47 2\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29465 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t29226\t0.0\t1\t28758 468\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1973_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1973_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1973_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1973_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1973_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1973_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1973_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1973_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,947 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.75 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,947\n",
      "  Read 1 with adapter:                  36,921 (99.9%)\n",
      "  Read 2 with adapter:                  36,854 (99.7%)\n",
      "Pairs written (passing filters):        36,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,797,995 bp\n",
      "  Read 1:    10,899,125 bp\n",
      "  Read 2:    10,898,870 bp\n",
      "Total written (filtered):     20,359,657 bp (93.4%)\n",
      "  Read 1:    10,160,887 bp\n",
      "  Read 2:    10,198,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36921 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t235\t0.0\t1\t0 231 4\n",
      "20\t36602\t0.0\t2\t35981 613 8\n",
      "21\t59\t0.0\t2\t0 54 5\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t197\t0.0\t1\t0 197\n",
      "19\t36586\t0.0\t1\t36031 555\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1974_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1974_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1974_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1974_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1974_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1974_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1974_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1974_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,713 reads  @     76.3 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.35 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,713\n",
      "  Read 1 with adapter:                  30,693 (99.9%)\n",
      "  Read 2 with adapter:                  30,607 (99.7%)\n",
      "Pairs written (passing filters):        30,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,115,590 bp\n",
      "  Read 1:     9,060,053 bp\n",
      "  Read 2:     9,055,537 bp\n",
      "Total written (filtered):     16,920,415 bp (93.4%)\n",
      "  Read 1:     8,446,342 bp\n",
      "  Read 2:     8,474,073 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t220\t0.0\t1\t0 211 9\n",
      "20\t30387\t0.0\t2\t29843 534 10\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t129\t0.0\t1\t0 129\n",
      "19\t30418\t0.0\t1\t29908 510\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1975_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1975_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1975_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1975_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1975_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1975_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1975_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1975_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        34,342 reads  @     88.5 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.04 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,342\n",
      "  Read 1 with adapter:                  34,314 (99.9%)\n",
      "  Read 2 with adapter:                  34,234 (99.7%)\n",
      "Pairs written (passing filters):        34,342 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,255,068 bp\n",
      "  Read 1:    10,130,886 bp\n",
      "  Read 2:    10,124,182 bp\n",
      "Total written (filtered):     18,918,580 bp (93.4%)\n",
      "  Read 1:     9,444,759 bp\n",
      "  Read 2:     9,473,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t212\t0.0\t1\t0 208 4\n",
      "20\t34039\t0.0\t2\t33477 547 15\n",
      "21\t43\t0.0\t2\t0 43\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34234 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t155\t0.0\t1\t0 155\n",
      "19\t34009\t0.0\t1\t33518 491\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1976_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1976_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1976_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1976_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1976_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1976_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1976_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1976_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,232 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.49 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,232\n",
      "  Read 1 with adapter:                  34,210 (99.9%)\n",
      "  Read 2 with adapter:                  33,834 (98.8%)\n",
      "Pairs written (passing filters):        34,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,175,881 bp\n",
      "  Read 1:    10,098,433 bp\n",
      "  Read 2:    10,077,448 bp\n",
      "Total written (filtered):     18,849,068 bp (93.4%)\n",
      "  Read 1:     9,414,377 bp\n",
      "  Read 2:     9,434,691 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t211\t0.0\t1\t0 206 5\n",
      "20\t33930\t0.0\t2\t33401 514 15\n",
      "21\t51\t0.0\t2\t0 47 4\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t146\t0.0\t1\t0 146\n",
      "19\t33631\t0.0\t1\t33190 441\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1977_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1977_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1977_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1977_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1977_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1977_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1977_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1977_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,080 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.21 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,080\n",
      "  Read 1 with adapter:                  46,001 (99.8%)\n",
      "  Read 2 with adapter:                  45,920 (99.7%)\n",
      "Pairs written (passing filters):        46,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,181,645 bp\n",
      "  Read 1:    13,589,639 bp\n",
      "  Read 2:    13,592,006 bp\n",
      "Total written (filtered):     25,389,415 bp (93.4%)\n",
      "  Read 1:    12,669,738 bp\n",
      "  Read 2:    12,719,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t205\t0.0\t1\t0 203 2\n",
      "20\t45696\t0.0\t2\t44908 768 20\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t236\t0.0\t1\t0 236\n",
      "19\t45599\t0.0\t1\t44834 765\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1978_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1978_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1978_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1978_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1978_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1978_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1978_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1978_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        34,561 reads  @     69.5 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.41 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,561\n",
      "  Read 1 with adapter:                  34,530 (99.9%)\n",
      "  Read 2 with adapter:                  34,425 (99.6%)\n",
      "Pairs written (passing filters):        34,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,388,761 bp\n",
      "  Read 1:    10,193,507 bp\n",
      "  Read 2:    10,195,254 bp\n",
      "Total written (filtered):     19,044,299 bp (93.4%)\n",
      "  Read 1:     9,503,029 bp\n",
      "  Read 2:     9,541,270 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34530 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t193\t0.0\t1\t0 185 8\n",
      "20\t34261\t0.0\t2\t33657 585 19\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34425 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t164\t0.0\t1\t0 164\n",
      "19\t34188\t0.0\t1\t33604 584\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1979_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1979_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1979_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1979_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1979_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1979_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1979_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1979_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,858 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.95 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,858\n",
      "  Read 1 with adapter:                  40,789 (99.8%)\n",
      "  Read 2 with adapter:                  40,737 (99.7%)\n",
      "Pairs written (passing filters):        40,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,103,661 bp\n",
      "  Read 1:    12,051,468 bp\n",
      "  Read 2:    12,052,193 bp\n",
      "Total written (filtered):     22,514,068 bp (93.4%)\n",
      "  Read 1:    11,235,779 bp\n",
      "  Read 2:    11,278,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40789 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t179\t0.0\t1\t0 177 2\n",
      "20\t40520\t0.0\t2\t39815 692 13\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t40516\t0.0\t1\t39856 660\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1980_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1980_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1980_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1980_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1980_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1980_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1980_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1980_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,517 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 2.36 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,517\n",
      "  Read 1 with adapter:                  30,412 (99.7%)\n",
      "  Read 2 with adapter:                  30,411 (99.7%)\n",
      "Pairs written (passing filters):        30,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,991,947 bp\n",
      "  Read 1:     8,989,738 bp\n",
      "  Read 2:     9,002,209 bp\n",
      "Total written (filtered):     16,806,112 bp (93.4%)\n",
      "  Read 1:     8,381,596 bp\n",
      "  Read 2:     8,424,516 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t165\t0.0\t1\t0 157 8\n",
      "20\t30173\t0.0\t2\t29686 478 9\n",
      "21\t53\t0.0\t2\t0 53\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t170\t0.0\t1\t0 170\n",
      "19\t30187\t0.0\t1\t29664 523\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1981_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1981_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1981_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1981_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1981_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1981_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1981_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1981_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        39,378 reads  @     90.8 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.58 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,378\n",
      "  Read 1 with adapter:                  39,330 (99.9%)\n",
      "  Read 2 with adapter:                  39,279 (99.7%)\n",
      "Pairs written (passing filters):        39,378 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,231,121 bp\n",
      "  Read 1:    11,615,489 bp\n",
      "  Read 2:    11,615,632 bp\n",
      "Total written (filtered):     21,698,519 bp (93.4%)\n",
      "  Read 1:    10,829,017 bp\n",
      "  Read 2:    10,869,502 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39330 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t234\t0.0\t1\t0 226 8\n",
      "20\t38981\t0.0\t2\t38272 695 14\n",
      "21\t88\t0.0\t2\t0 83 5\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t226\t0.0\t1\t0 226\n",
      "19\t38998\t0.0\t1\t38292 706\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1982_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1982_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1982_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1982_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1982_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1982_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1982_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1982_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,025 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.16 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,025\n",
      "  Read 1 with adapter:                  40,980 (99.9%)\n",
      "  Read 2 with adapter:                  40,851 (99.6%)\n",
      "Pairs written (passing filters):        41,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,195,146 bp\n",
      "  Read 1:    12,100,381 bp\n",
      "  Read 2:    12,094,765 bp\n",
      "Total written (filtered):     22,599,642 bp (93.4%)\n",
      "  Read 1:    11,280,926 bp\n",
      "  Read 2:    11,318,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t226\t0.0\t1\t0 220 6\n",
      "20\t40654\t0.0\t2\t39895 748 11\n",
      "21\t63\t0.0\t2\t0 59 4\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40851 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t40587\t0.0\t1\t39921 666\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1983_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1983_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1983_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1983_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1983_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1983_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1983_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1983_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,865 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 2.99 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,865\n",
      "  Read 1 with adapter:                  35,762 (99.7%)\n",
      "  Read 2 with adapter:                  35,690 (99.5%)\n",
      "Pairs written (passing filters):        35,865 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,149,533 bp\n",
      "  Read 1:    10,578,468 bp\n",
      "  Read 2:    10,571,065 bp\n",
      "Total written (filtered):     19,756,426 bp (93.4%)\n",
      "  Read 1:     9,863,342 bp\n",
      "  Read 2:     9,893,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t185\t0.0\t1\t0 183 2\n",
      "20\t35478\t0.0\t2\t34773 685 20\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t35451\t0.0\t1\t34821 630\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1984_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1984_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1984_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1984_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1984_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1984_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1984_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1984_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        40,788 reads  @     95.1 µs/read;   0.63 M reads/minute\n",
      "Finished in 3.88 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,788\n",
      "  Read 1 with adapter:                  40,751 (99.9%)\n",
      "  Read 2 with adapter:                  40,329 (98.9%)\n",
      "Pairs written (passing filters):        40,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,040,581 bp\n",
      "  Read 1:    12,031,337 bp\n",
      "  Read 2:    12,009,244 bp\n",
      "Total written (filtered):     22,459,598 bp (93.4%)\n",
      "  Read 1:    11,216,474 bp\n",
      "  Read 2:    11,243,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t227\t0.0\t1\t0 224 3\n",
      "20\t40437\t0.0\t2\t39720 698 19\n",
      "21\t52\t0.0\t2\t0 52\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40329 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t40044\t0.0\t1\t39462 582\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1993_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1993_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1993_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1993_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1993_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1993_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1993_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1993_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,785 reads  @     70.9 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.68 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,785\n",
      "  Read 1 with adapter:                  51,526 (99.5%)\n",
      "  Read 2 with adapter:                  51,429 (99.3%)\n",
      "Pairs written (passing filters):        51,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,547,652 bp\n",
      "  Read 1:    15,271,618 bp\n",
      "  Read 2:    15,276,034 bp\n",
      "Total written (filtered):     28,540,224 bp (93.4%)\n",
      "  Read 1:    14,241,202 bp\n",
      "  Read 2:    14,299,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t261\t0.0\t1\t0 258 3\n",
      "20\t51103\t0.0\t2\t50222 859 22\n",
      "21\t115\t0.0\t2\t0 113 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51429 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t51098\t0.0\t1\t50273 825\n",
      "20\t96\t0.0\t1\t0 96\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1994_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1994_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1994_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1994_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1994_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1994_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1994_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1994_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,991 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.29 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,991\n",
      "  Read 1 with adapter:                  46,897 (99.8%)\n",
      "  Read 2 with adapter:                  46,786 (99.6%)\n",
      "Pairs written (passing filters):        46,991 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,722,541 bp\n",
      "  Read 1:    13,861,301 bp\n",
      "  Read 2:    13,861,240 bp\n",
      "Total written (filtered):     25,895,989 bp (93.4%)\n",
      "  Read 1:    12,923,528 bp\n",
      "  Read 2:    12,972,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t307\t0.0\t1\t0 298 9\n",
      "20\t46455\t0.0\t2\t45659 779 17\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46786 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t238\t0.0\t1\t0 238\n",
      "19\t46465\t0.0\t1\t45743 722\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1995_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1995_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1995_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1995_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1995_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1995_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1995_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1995_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,179 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.87 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,179\n",
      "  Read 1 with adapter:                  42,077 (99.8%)\n",
      "  Read 2 with adapter:                  41,992 (99.6%)\n",
      "Pairs written (passing filters):        42,179 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,883,314 bp\n",
      "  Read 1:    12,440,816 bp\n",
      "  Read 2:    12,442,498 bp\n",
      "Total written (filtered):     23,244,215 bp (93.4%)\n",
      "  Read 1:    11,599,432 bp\n",
      "  Read 2:    11,644,783 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t283\t0.0\t1\t0 274 9\n",
      "20\t41654\t0.0\t2\t40891 748 15\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t41697\t0.0\t1\t41032 665\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1996_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1996_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1996_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1996_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1996_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1996_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1996_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1996_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,906 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.43 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,906\n",
      "  Read 1 with adapter:                  33,857 (99.9%)\n",
      "  Read 2 with adapter:                  33,757 (99.6%)\n",
      "Pairs written (passing filters):        33,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,004,087 bp\n",
      "  Read 1:    10,001,844 bp\n",
      "  Read 2:    10,002,243 bp\n",
      "Total written (filtered):     18,685,815 bp (93.4%)\n",
      "  Read 1:     9,324,840 bp\n",
      "  Read 2:     9,360,975 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33857 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t231\t0.0\t1\t0 229 2\n",
      "20\t33514\t0.0\t2\t32922 572 20\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33757 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t206\t0.0\t1\t0 206\n",
      "19\t33460\t0.0\t1\t32877 583\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1997_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1997_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1997_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1997_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1997_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1997_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1997_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1997_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,780 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.81 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,780\n",
      "  Read 1 with adapter:                  37,730 (99.9%)\n",
      "  Read 2 with adapter:                  37,646 (99.6%)\n",
      "Pairs written (passing filters):        37,780 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,288,817 bp\n",
      "  Read 1:    11,144,591 bp\n",
      "  Read 2:    11,144,226 bp\n",
      "Total written (filtered):     20,819,230 bp (93.4%)\n",
      "  Read 1:    10,390,147 bp\n",
      "  Read 2:    10,429,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t257\t0.0\t1\t0 250 7\n",
      "20\t37371\t0.0\t2\t36672 684 15\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t206\t0.0\t1\t0 206\n",
      "19\t37365\t0.0\t1\t36769 596\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1998_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1998_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1998_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1998_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1998_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1998_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1998_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1998_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        34,042 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.57 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,042\n",
      "  Read 1 with adapter:                  33,988 (99.8%)\n",
      "  Read 2 with adapter:                  33,875 (99.5%)\n",
      "Pairs written (passing filters):        34,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,071,325 bp\n",
      "  Read 1:    10,042,132 bp\n",
      "  Read 2:    10,029,193 bp\n",
      "Total written (filtered):     18,748,162 bp (93.4%)\n",
      "  Read 1:     9,362,485 bp\n",
      "  Read 2:     9,385,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t232\t0.0\t1\t0 225 7\n",
      "20\t33631\t0.0\t2\t33036 574 21\n",
      "21\t83\t0.0\t2\t0 81 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33875 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t33638\t0.0\t1\t33108 530\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1999_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1999_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1999_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1999_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1999_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S1999_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1999_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S1999_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,607 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.49 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,607\n",
      "  Read 1 with adapter:                  35,526 (99.8%)\n",
      "  Read 2 with adapter:                  35,403 (99.4%)\n",
      "Pairs written (passing filters):        35,607 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,002,018 bp\n",
      "  Read 1:    10,503,351 bp\n",
      "  Read 2:    10,498,667 bp\n",
      "Total written (filtered):     19,619,096 bp (93.4%)\n",
      "  Read 1:     9,792,987 bp\n",
      "  Read 2:     9,826,109 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t249\t0.0\t1\t0 245 4\n",
      "20\t35164\t0.0\t2\t34526 616 22\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t169\t0.0\t1\t0 169\n",
      "19\t35164\t0.0\t1\t34629 535\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2000_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2000_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2000_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2000_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2000_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2000_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2000_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2000_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,746 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.43 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,746\n",
      "  Read 1 with adapter:                  33,693 (99.8%)\n",
      "  Read 2 with adapter:                  33,336 (98.8%)\n",
      "Pairs written (passing filters):        33,746 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,888,480 bp\n",
      "  Read 1:     9,952,763 bp\n",
      "  Read 2:     9,935,717 bp\n",
      "Total written (filtered):     18,581,469 bp (93.4%)\n",
      "  Read 1:     9,279,037 bp\n",
      "  Read 2:     9,302,432 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t214\t0.0\t1\t0 210 4\n",
      "20\t33381\t0.0\t2\t32798 571 12\n",
      "21\t68\t0.0\t2\t0 64 4\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t161\t0.0\t1\t0 161\n",
      "19\t33113\t0.0\t1\t32691 422\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2001_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2001_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2001_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2001_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2001_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2001_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2001_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2001_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,097 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.67 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,097\n",
      "  Read 1 with adapter:                  39,058 (99.9%)\n",
      "  Read 2 with adapter:                  38,914 (99.5%)\n",
      "Pairs written (passing filters):        39,097 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,064,788 bp\n",
      "  Read 1:    11,532,896 bp\n",
      "  Read 2:    11,531,892 bp\n",
      "Total written (filtered):     21,544,648 bp (93.4%)\n",
      "  Read 1:    10,751,990 bp\n",
      "  Read 2:    10,792,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t319\t0.0\t1\t0 313 6\n",
      "20\t38648\t0.0\t2\t38000 631 17\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t194\t0.0\t1\t0 194\n",
      "19\t38658\t0.0\t1\t37959 699\n",
      "20\t62\t0.0\t1\t0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2002_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2002_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2002_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2002_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2002_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2002_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2002_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2002_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,157 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.61 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,157\n",
      "  Read 1 with adapter:                  38,123 (99.9%)\n",
      "  Read 2 with adapter:                  38,000 (99.6%)\n",
      "Pairs written (passing filters):        38,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,510,990 bp\n",
      "  Read 1:    11,255,530 bp\n",
      "  Read 2:    11,255,460 bp\n",
      "Total written (filtered):     21,026,617 bp (93.4%)\n",
      "  Read 1:    10,493,316 bp\n",
      "  Read 2:    10,533,301 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t336\t0.0\t1\t0 328 8\n",
      "20\t37684\t0.0\t2\t37069 595 20\n",
      "21\t72\t0.0\t2\t0 71 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t37733\t0.0\t1\t37059 674\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2003_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2003_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2003_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2003_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2003_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2003_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2003_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2003_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,650 reads  @     73.7 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.78 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,650\n",
      "  Read 1 with adapter:                  37,606 (99.9%)\n",
      "  Read 2 with adapter:                  37,527 (99.7%)\n",
      "Pairs written (passing filters):        37,649 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,212,117 bp\n",
      "  Read 1:    11,106,063 bp\n",
      "  Read 2:    11,106,054 bp\n",
      "Total written (filtered):     20,747,079 bp (93.4%)\n",
      "  Read 1:    10,354,224 bp\n",
      "  Read 2:    10,392,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t350\t0.0\t1\t0 342 8\n",
      "20\t37162\t0.0\t2\t36507 639 16\n",
      "21\t71\t0.0\t2\t0 68 3\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37527 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t163\t0.0\t1\t0 163\n",
      "19\t37291\t0.0\t1\t36629 662\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2004_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2004_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2004_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2004_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2004_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2004_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2004_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2004_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        29,680 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.56 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,680\n",
      "  Read 1 with adapter:                  29,644 (99.9%)\n",
      "  Read 2 with adapter:                  29,481 (99.3%)\n",
      "Pairs written (passing filters):        29,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,500,118 bp\n",
      "  Read 1:     8,754,562 bp\n",
      "  Read 2:     8,745,556 bp\n",
      "Total written (filtered):     16,347,366 bp (93.4%)\n",
      "  Read 1:     8,161,858 bp\n",
      "  Read 2:     8,185,508 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t246\t0.0\t1\t0 240 6\n",
      "20\t29315\t0.0\t2\t28768 539 8\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t29252\t0.0\t1\t28690 562\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2005_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2005_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2005_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2005_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2005_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2005_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2005_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2005_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,613 reads  @     78.6 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.04 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,613\n",
      "  Read 1 with adapter:                  38,569 (99.9%)\n",
      "  Read 2 with adapter:                  38,455 (99.6%)\n",
      "Pairs written (passing filters):        38,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,779,876 bp\n",
      "  Read 1:    11,389,744 bp\n",
      "  Read 2:    11,390,132 bp\n",
      "Total written (filtered):     21,278,243 bp (93.4%)\n",
      "  Read 1:    10,618,635 bp\n",
      "  Read 2:    10,659,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t347\t0.0\t1\t0 338 9\n",
      "20\t38123\t0.0\t2\t37472 630 21\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38455 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t38190\t0.0\t1\t37407 783\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2006_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2006_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2006_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2006_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2006_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2006_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2006_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2006_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,223 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.18 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,223\n",
      "  Read 1 with adapter:                  32,187 (99.9%)\n",
      "  Read 2 with adapter:                  32,074 (99.5%)\n",
      "Pairs written (passing filters):        32,223 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,005,791 bp\n",
      "  Read 1:     9,504,608 bp\n",
      "  Read 2:     9,501,183 bp\n",
      "Total written (filtered):     17,752,922 bp (93.4%)\n",
      "  Read 1:     8,861,069 bp\n",
      "  Read 2:     8,891,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t267\t0.0\t1\t0 263 4\n",
      "20\t31848\t0.0\t2\t31322 517 9\n",
      "21\t46\t0.0\t2\t0 44 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t132\t0.0\t1\t0 132\n",
      "19\t31886\t0.0\t1\t31275 611\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2007_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2007_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2007_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2007_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2007_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2007_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2007_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2007_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,592 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.52 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,592\n",
      "  Read 1 with adapter:                  35,532 (99.8%)\n",
      "  Read 2 with adapter:                  35,442 (99.6%)\n",
      "Pairs written (passing filters):        35,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,987,138 bp\n",
      "  Read 1:    10,498,046 bp\n",
      "  Read 2:    10,489,092 bp\n",
      "Total written (filtered):     19,603,420 bp (93.4%)\n",
      "  Read 1:     9,787,649 bp\n",
      "  Read 2:     9,815,771 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t294\t0.0\t1\t0 287 7\n",
      "20\t35161\t0.0\t2\t34522 623 16\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t153\t0.0\t1\t0 153\n",
      "19\t35213\t0.0\t1\t34620 593\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2008_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2008_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2008_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2008_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2008_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2008_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2008_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2008_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,477 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.16 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,477\n",
      "  Read 1 with adapter:                  29,426 (99.8%)\n",
      "  Read 2 with adapter:                  29,042 (98.5%)\n",
      "Pairs written (passing filters):        29,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,369,530 bp\n",
      "  Read 1:     8,694,467 bp\n",
      "  Read 2:     8,675,063 bp\n",
      "Total written (filtered):     16,229,444 bp (93.4%)\n",
      "  Read 1:     8,106,107 bp\n",
      "  Read 2:     8,123,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t221\t0.0\t1\t0 217 4\n",
      "20\t29131\t0.0\t2\t28712 405 14\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29042 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t127\t0.0\t1\t0 127\n",
      "19\t28860\t0.0\t1\t28419 441\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2009_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2009_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2009_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2009_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2009_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2009_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2009_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2009_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,523 reads  @     71.9 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.63 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,523\n",
      "  Read 1 with adapter:                  36,117 (98.9%)\n",
      "  Read 2 with adapter:                  36,167 (99.0%)\n",
      "Pairs written (passing filters):        36,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,534,318 bp\n",
      "  Read 1:    10,761,301 bp\n",
      "  Read 2:    10,773,017 bp\n",
      "Total written (filtered):     20,125,038 bp (93.5%)\n",
      "  Read 1:    10,039,088 bp\n",
      "  Read 2:    10,085,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36117 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t203\t0.0\t1\t0 201 2\n",
      "20\t35835\t0.0\t2\t35194 626 15\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t161\t0.0\t1\t0 161\n",
      "19\t35951\t0.0\t1\t35375 576\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2010_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2010_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2010_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2010_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2010_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2010_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2010_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2010_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        41,158 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.06 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,158\n",
      "  Read 1 with adapter:                  40,988 (99.6%)\n",
      "  Read 2 with adapter:                  40,969 (99.5%)\n",
      "Pairs written (passing filters):        41,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,271,599 bp\n",
      "  Read 1:    12,130,823 bp\n",
      "  Read 2:    12,140,776 bp\n",
      "Total written (filtered):     22,673,693 bp (93.4%)\n",
      "  Read 1:    11,311,215 bp\n",
      "  Read 2:    11,362,478 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t235\t0.0\t1\t0 231 4\n",
      "20\t40645\t0.0\t2\t39913 715 17\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t40686\t0.0\t1\t39997 689\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2011_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2011_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2011_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2011_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2011_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2011_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2011_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2011_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,272 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.53 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,272\n",
      "  Read 1 with adapter:                  33,147 (99.6%)\n",
      "  Read 2 with adapter:                  33,148 (99.6%)\n",
      "Pairs written (passing filters):        33,272 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,623,568 bp\n",
      "  Read 1:     9,808,714 bp\n",
      "  Read 2:     9,814,854 bp\n",
      "Total written (filtered):     18,331,045 bp (93.4%)\n",
      "  Read 1:     9,145,922 bp\n",
      "  Read 2:     9,185,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33147 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t222\t0.0\t1\t0 213 9\n",
      "20\t32833\t0.0\t2\t32264 558 11\n",
      "21\t74\t0.0\t2\t0 72 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33148 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t170\t0.0\t1\t0 170\n",
      "19\t32889\t0.0\t1\t32362 527\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2012_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2012_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2012_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2012_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2012_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2012_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2012_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2012_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        27,724 reads  @     79.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.21 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,724\n",
      "  Read 1 with adapter:                  27,649 (99.7%)\n",
      "  Read 2 with adapter:                  27,586 (99.5%)\n",
      "Pairs written (passing filters):        27,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,354,568 bp\n",
      "  Read 1:     8,176,210 bp\n",
      "  Read 2:     8,178,358 bp\n",
      "Total written (filtered):     15,277,725 bp (93.4%)\n",
      "  Read 1:     7,623,367 bp\n",
      "  Read 2:     7,654,358 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27649 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t196\t0.0\t1\t0 193 3\n",
      "20\t27377\t0.0\t2\t26866 497 14\n",
      "21\t53\t0.0\t2\t0 51 2\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27586 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t178\t0.0\t1\t0 178\n",
      "19\t27364\t0.0\t1\t26891 473\n",
      "20\t44\t0.0\t1\t0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2013_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2013_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2013_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2013_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2013_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2013_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2013_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2013_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,769 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 2.69 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,769\n",
      "  Read 1 with adapter:                  35,582 (99.5%)\n",
      "  Read 2 with adapter:                  35,636 (99.6%)\n",
      "Pairs written (passing filters):        35,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,090,868 bp\n",
      "  Read 1:    10,539,336 bp\n",
      "  Read 2:    10,551,532 bp\n",
      "Total written (filtered):     19,702,431 bp (93.4%)\n",
      "  Read 1:     9,827,861 bp\n",
      "  Read 2:     9,874,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35582 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t219\t0.0\t1\t0 216 3\n",
      "20\t35285\t0.0\t2\t34635 633 17\n",
      "21\t54\t0.0\t2\t0 51 3\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35636 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t35388\t0.0\t1\t34846 542\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2014_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2014_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2014_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2014_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2014_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2014_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2014_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2014_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,171 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.52 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,171\n",
      "  Read 1 with adapter:                  35,996 (99.5%)\n",
      "  Read 2 with adapter:                  35,994 (99.5%)\n",
      "Pairs written (passing filters):        36,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,327,160 bp\n",
      "  Read 1:    10,661,869 bp\n",
      "  Read 2:    10,665,291 bp\n",
      "Total written (filtered):     19,923,565 bp (93.4%)\n",
      "  Read 1:     9,942,099 bp\n",
      "  Read 2:     9,981,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t230\t0.0\t1\t0 227 3\n",
      "20\t35665\t0.0\t2\t35012 628 25\n",
      "21\t62\t0.0\t2\t0 62\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t35735\t0.0\t1\t35146 589\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2015_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2015_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2015_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2015_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2015_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2015_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2015_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2015_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,980 reads  @     66.5 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.13 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,980\n",
      "  Read 1 with adapter:                  31,957 (99.9%)\n",
      "  Read 2 with adapter:                  31,855 (99.6%)\n",
      "Pairs written (passing filters):        31,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,864,260 bp\n",
      "  Read 1:     9,433,824 bp\n",
      "  Read 2:     9,430,436 bp\n",
      "Total written (filtered):     17,620,123 bp (93.4%)\n",
      "  Read 1:     8,794,828 bp\n",
      "  Read 2:     8,825,295 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31957 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t195\t0.0\t1\t0 190 5\n",
      "20\t31690\t0.0\t2\t31098 575 17\n",
      "21\t53\t0.0\t2\t0 51 2\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t164\t0.0\t1\t0 164\n",
      "19\t31631\t0.0\t1\t31097 534\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2016_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2016_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2016_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2016_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2016_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2016_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2016_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2016_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        35,305 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.69 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,305\n",
      "  Read 1 with adapter:                  34,580 (97.9%)\n",
      "  Read 2 with adapter:                  34,318 (97.2%)\n",
      "Pairs written (passing filters):        35,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,798,756 bp\n",
      "  Read 1:    10,404,550 bp\n",
      "  Read 2:    10,394,206 bp\n",
      "Total written (filtered):     19,455,369 bp (93.5%)\n",
      "  Read 1:     9,713,079 bp\n",
      "  Read 2:     9,742,290 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34580 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t190\t0.0\t1\t0 187 3\n",
      "20\t34316\t0.0\t2\t33721 579 16\n",
      "21\t47\t0.0\t2\t0 46 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t179\t0.0\t1\t0 179\n",
      "19\t34086\t0.0\t1\t33639 447\n",
      "20\t53\t0.0\t1\t0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2017_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2017_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2017_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2017_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2017_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2017_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2017_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2017_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        27,950 reads  @     72.4 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.03 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,950\n",
      "  Read 1 with adapter:                  27,929 (99.9%)\n",
      "  Read 2 with adapter:                  27,855 (99.7%)\n",
      "Pairs written (passing filters):        27,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,489,950 bp\n",
      "  Read 1:     8,245,014 bp\n",
      "  Read 2:     8,244,936 bp\n",
      "Total written (filtered):     15,402,287 bp (93.4%)\n",
      "  Read 1:     7,686,504 bp\n",
      "  Read 2:     7,715,783 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t123\t0.0\t1\t0 122 1\n",
      "20\t27756\t0.0\t2\t27299 443 14\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t129\t0.0\t1\t0 129\n",
      "19\t27689\t0.0\t1\t27298 391\n",
      "20\t37\t0.0\t1\t0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2018_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2018_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2018_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2018_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2018_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2018_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2018_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2018_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,578 reads  @     67.3 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.06 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,578\n",
      "  Read 1 with adapter:                  30,561 (99.9%)\n",
      "  Read 2 with adapter:                  30,481 (99.7%)\n",
      "Pairs written (passing filters):        30,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,040,285 bp\n",
      "  Read 1:     9,020,218 bp\n",
      "  Read 2:     9,020,067 bp\n",
      "Total written (filtered):     16,850,132 bp (93.4%)\n",
      "  Read 1:     8,409,091 bp\n",
      "  Read 2:     8,441,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t191\t0.0\t1\t0 188 3\n",
      "20\t30273\t0.0\t2\t29756 498 19\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t170\t0.0\t1\t0 170\n",
      "19\t30254\t0.0\t1\t29748 506\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2019_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2019_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2019_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2019_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2019_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2019_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2019_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2019_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,457 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.15 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,457\n",
      "  Read 1 with adapter:                  30,437 (99.9%)\n",
      "  Read 2 with adapter:                  30,373 (99.7%)\n",
      "Pairs written (passing filters):        30,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,968,708 bp\n",
      "  Read 1:     8,984,559 bp\n",
      "  Read 2:     8,984,149 bp\n",
      "Total written (filtered):     16,783,048 bp (93.4%)\n",
      "  Read 1:     8,375,911 bp\n",
      "  Read 2:     8,407,137 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30437 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t183\t0.0\t1\t0 180 3\n",
      "20\t30187\t0.0\t2\t29625 551 11\n",
      "21\t39\t0.0\t2\t0 38 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30373 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t118\t0.0\t1\t0 118\n",
      "19\t30212\t0.0\t1\t29724 488\n",
      "20\t43\t0.0\t1\t0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2020_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2020_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2020_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2020_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2020_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2020_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2020_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2020_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,168 reads  @     77.8 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.35 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,168\n",
      "  Read 1 with adapter:                  30,146 (99.9%)\n",
      "  Read 2 with adapter:                  30,065 (99.7%)\n",
      "Pairs written (passing filters):        30,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,799,055 bp\n",
      "  Read 1:     8,899,505 bp\n",
      "  Read 2:     8,899,550 bp\n",
      "Total written (filtered):     16,625,169 bp (93.4%)\n",
      "  Read 1:     8,296,693 bp\n",
      "  Read 2:     8,328,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30146 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t177\t0.0\t1\t0 173 4\n",
      "20\t29893\t0.0\t2\t29378 500 15\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t29790\t0.0\t1\t29238 552\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2021_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2021_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2021_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2021_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2021_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2021_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2021_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2021_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,231 reads  @     77.3 µs/read;   0.78 M reads/minute\n",
      "Finished in 2.26 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,231\n",
      "  Read 1 with adapter:                  29,212 (99.9%)\n",
      "  Read 2 with adapter:                  29,132 (99.7%)\n",
      "Pairs written (passing filters):        29,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,245,344 bp\n",
      "  Read 1:     8,623,135 bp\n",
      "  Read 2:     8,622,209 bp\n",
      "Total written (filtered):     16,107,815 bp (93.4%)\n",
      "  Read 1:     8,039,019 bp\n",
      "  Read 2:     8,068,796 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29212 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t186\t0.0\t1\t0 184 2\n",
      "20\t28953\t0.0\t2\t28441 495 17\n",
      "21\t52\t0.0\t2\t0 50 2\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t150\t0.0\t1\t0 150\n",
      "19\t28927\t0.0\t1\t28439 488\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2022_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2022_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2022_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2022_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2022_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2022_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2022_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2022_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:01        28,140 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 1.98 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,140\n",
      "  Read 1 with adapter:                  28,121 (99.9%)\n",
      "  Read 2 with adapter:                  28,035 (99.6%)\n",
      "Pairs written (passing filters):        28,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,598,674 bp\n",
      "  Read 1:     8,301,002 bp\n",
      "  Read 2:     8,297,672 bp\n",
      "Total written (filtered):     15,503,736 bp (93.4%)\n",
      "  Read 1:     7,738,654 bp\n",
      "  Read 2:     7,765,082 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28121 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t139\t0.0\t1\t0 138 1\n",
      "20\t27898\t0.0\t2\t27400 486 12\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28035 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t127\t0.0\t1\t0 127\n",
      "19\t27856\t0.0\t1\t27394 462\n",
      "20\t52\t0.0\t1\t0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2023_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2023_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2023_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2023_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2023_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2023_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2023_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2023_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,056 reads  @     70.5 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.26 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,056\n",
      "  Read 1 with adapter:                  32,026 (99.9%)\n",
      "  Read 2 with adapter:                  31,928 (99.6%)\n",
      "Pairs written (passing filters):        32,055 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,903,749 bp\n",
      "  Read 1:     9,456,130 bp\n",
      "  Read 2:     9,447,619 bp\n",
      "Total written (filtered):     17,656,536 bp (93.4%)\n",
      "  Read 1:     8,815,444 bp\n",
      "  Read 2:     8,841,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t175\t0.0\t1\t0 170 5\n",
      "20\t31773\t0.0\t2\t31206 554 13\n",
      "21\t58\t0.0\t2\t0 55 3\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t31713\t0.0\t1\t31241 472\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2024_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2024_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2024_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2024_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2024_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2024_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2024_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2024_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,971 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.04 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,971\n",
      "  Read 1 with adapter:                  28,947 (99.9%)\n",
      "  Read 2 with adapter:                  28,589 (98.7%)\n",
      "Pairs written (passing filters):        28,971 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,073,497 bp\n",
      "  Read 1:     8,546,297 bp\n",
      "  Read 2:     8,527,200 bp\n",
      "Total written (filtered):     15,951,521 bp (93.4%)\n",
      "  Read 1:     7,967,453 bp\n",
      "  Read 2:     7,984,068 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28947 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t149\t0.0\t1\t0 148 1\n",
      "20\t28736\t0.0\t2\t28242 481 13\n",
      "21\t39\t0.0\t2\t0 37 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t114\t0.0\t1\t0 114\n",
      "19\t28420\t0.0\t1\t28019 401\n",
      "20\t55\t0.0\t1\t0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2025_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2025_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2025_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2025_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2025_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2025_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2025_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2025_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,180 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.03 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,180\n",
      "  Read 1 with adapter:                  42,138 (99.9%)\n",
      "  Read 2 with adapter:                  42,000 (99.6%)\n",
      "Pairs written (passing filters):        42,180 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,884,273 bp\n",
      "  Read 1:    12,442,382 bp\n",
      "  Read 2:    12,441,891 bp\n",
      "Total written (filtered):     23,243,803 bp (93.4%)\n",
      "  Read 1:    11,599,781 bp\n",
      "  Read 2:    11,644,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t245\t0.0\t1\t0 242 3\n",
      "20\t41773\t0.0\t2\t41155 605 13\n",
      "21\t78\t0.0\t2\t0 74 4\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t41701\t0.0\t1\t41068 633\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2026_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2026_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2026_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2026_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2026_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2026_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2026_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2026_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,669 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.40 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,669\n",
      "  Read 1 with adapter:                  33,637 (99.9%)\n",
      "  Read 2 with adapter:                  33,551 (99.6%)\n",
      "Pairs written (passing filters):        33,669 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,862,346 bp\n",
      "  Read 1:     9,930,623 bp\n",
      "  Read 2:     9,931,723 bp\n",
      "Total written (filtered):     18,552,369 bp (93.4%)\n",
      "  Read 1:     9,257,983 bp\n",
      "  Read 2:     9,294,386 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t186\t0.0\t1\t0 179 7\n",
      "20\t33348\t0.0\t2\t32786 550 12\n",
      "21\t68\t0.0\t2\t0 65 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33551 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t33305\t0.0\t1\t32713 592\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2027_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2027_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2027_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2027_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2027_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2027_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2027_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2027_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,632 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.04 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,632\n",
      "  Read 1 with adapter:                  41,591 (99.9%)\n",
      "  Read 2 with adapter:                  41,503 (99.7%)\n",
      "Pairs written (passing filters):        41,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,560,514 bp\n",
      "  Read 1:    12,279,421 bp\n",
      "  Read 2:    12,281,093 bp\n",
      "Total written (filtered):     22,940,442 bp (93.4%)\n",
      "  Read 1:    11,447,784 bp\n",
      "  Read 2:    11,492,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t263\t0.0\t1\t0 262 1\n",
      "20\t41217\t0.0\t2\t40530 674 13\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41503 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t186\t0.0\t1\t0 186\n",
      "19\t41253\t0.0\t1\t40555 698\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2028_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2028_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2028_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2028_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2028_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2028_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2028_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2028_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        28,543 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 2.27 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,543\n",
      "  Read 1 with adapter:                  28,516 (99.9%)\n",
      "  Read 2 with adapter:                  28,423 (99.6%)\n",
      "Pairs written (passing filters):        28,543 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,837,865 bp\n",
      "  Read 1:     8,417,937 bp\n",
      "  Read 2:     8,419,928 bp\n",
      "Total written (filtered):     15,727,711 bp (93.4%)\n",
      "  Read 1:     7,847,707 bp\n",
      "  Read 2:     7,880,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t166\t0.0\t1\t0 164 2\n",
      "20\t28260\t0.0\t2\t27813 433 14\n",
      "21\t60\t0.0\t2\t0 53 7\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t28170\t0.0\t1\t27645 525\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2029_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2029_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2029_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2029_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2029_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2029_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2029_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2029_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,641 reads  @     78.6 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.20 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,641\n",
      "  Read 1 with adapter:                  40,596 (99.9%)\n",
      "  Read 2 with adapter:                  40,504 (99.7%)\n",
      "Pairs written (passing filters):        40,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,975,493 bp\n",
      "  Read 1:    11,987,410 bp\n",
      "  Read 2:    11,988,083 bp\n",
      "Total written (filtered):     22,394,289 bp (93.4%)\n",
      "  Read 1:    11,175,596 bp\n",
      "  Read 2:    11,218,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t229\t0.0\t1\t0 223 6\n",
      "20\t40247\t0.0\t2\t39533 695 19\n",
      "21\t81\t0.0\t2\t0 62 19\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t267\t0.0\t1\t0 267\n",
      "19\t40156\t0.0\t1\t39425 731\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2030_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2030_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2030_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2030_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2030_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2030_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2030_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2030_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,047 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 2.86 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,047\n",
      "  Read 1 with adapter:                  36,025 (99.9%)\n",
      "  Read 2 with adapter:                  35,911 (99.6%)\n",
      "Pairs written (passing filters):        36,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,261,522 bp\n",
      "  Read 1:    10,633,255 bp\n",
      "  Read 2:    10,628,267 bp\n",
      "Total written (filtered):     19,858,914 bp (93.4%)\n",
      "  Read 1:     9,912,839 bp\n",
      "  Read 2:     9,946,075 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t219\t0.0\t1\t0 213 6\n",
      "20\t35669\t0.0\t2\t35012 641 16\n",
      "21\t91\t0.0\t2\t0 88 3\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35911 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t184\t0.0\t1\t0 184\n",
      "19\t35660\t0.0\t1\t35019 641\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2031_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2031_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2031_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2031_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2031_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2031_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2031_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2031_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,033 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.17 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,033\n",
      "  Read 1 with adapter:                  37,980 (99.9%)\n",
      "  Read 2 with adapter:                  37,867 (99.6%)\n",
      "Pairs written (passing filters):        38,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,430,295 bp\n",
      "  Read 1:    11,217,146 bp\n",
      "  Read 2:    11,213,149 bp\n",
      "Total written (filtered):     20,951,492 bp (93.4%)\n",
      "  Read 1:    10,457,696 bp\n",
      "  Read 2:    10,493,796 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t237\t0.0\t1\t0 232 5\n",
      "20\t37639\t0.0\t2\t37045 584 10\n",
      "21\t69\t0.0\t2\t0 66 3\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37867 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t190\t0.0\t1\t0 190\n",
      "19\t37607\t0.0\t1\t37070 537\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2032_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2032_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2032_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2032_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2032_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2032_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2032_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2032_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,470 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.35 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,470\n",
      "  Read 1 with adapter:                  37,436 (99.9%)\n",
      "  Read 2 with adapter:                  37,008 (98.8%)\n",
      "Pairs written (passing filters):        37,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,083,870 bp\n",
      "  Read 1:    11,050,584 bp\n",
      "  Read 2:    11,033,286 bp\n",
      "Total written (filtered):     20,632,247 bp (93.4%)\n",
      "  Read 1:    10,302,007 bp\n",
      "  Read 2:    10,330,240 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t236\t0.0\t1\t0 233 3\n",
      "20\t37091\t0.0\t2\t36476 601 14\n",
      "21\t65\t0.0\t2\t0 64 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t170\t0.0\t1\t0 170\n",
      "19\t36774\t0.0\t1\t36268 506\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2041_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2041_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2041_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2041_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2041_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2041_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2041_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2041_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,979 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 0.18 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,979\n",
      "  Read 1 with adapter:                   1,976 (99.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         1,978 (99.9%)\n",
      "\n",
      "Total basepairs processed:       922,136 bp\n",
      "  Read 1:       583,805 bp\n",
      "  Read 2:       338,331 bp\n",
      "Total written (filtered):        882,345 bp (95.7%)\n",
      "  Read 1:       544,014 bp\n",
      "  Read 2:       338,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 1976 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t11\t0.0\t1\t0 11\n",
      "20\t1959\t0.0\t2\t1939 19 1\n",
      "21\t5\t0.0\t2\t0 5\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2042_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2042_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2042_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2042_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2042_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2042_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2042_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2042_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,801 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.04 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,801\n",
      "  Read 1 with adapter:                  28,770 (99.9%)\n",
      "  Read 2 with adapter:                  28,661 (99.5%)\n",
      "Pairs written (passing filters):        28,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,986,307 bp\n",
      "  Read 1:     8,496,004 bp\n",
      "  Read 2:     8,490,303 bp\n",
      "Total written (filtered):     15,866,571 bp (93.4%)\n",
      "  Read 1:     7,920,735 bp\n",
      "  Read 2:     7,945,836 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28770 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t187\t0.0\t1\t0 182 5\n",
      "20\t28508\t0.0\t2\t28020 478 10\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t150\t0.0\t1\t0 150\n",
      "19\t28453\t0.0\t1\t28031 422\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2043_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2043_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2043_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2043_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2043_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2043_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2043_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2043_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,232 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.20 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,232\n",
      "  Read 1 with adapter:                  33,199 (99.9%)\n",
      "  Read 2 with adapter:                  33,146 (99.7%)\n",
      "Pairs written (passing filters):        33,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,606,588 bp\n",
      "  Read 1:     9,803,173 bp\n",
      "  Read 2:     9,803,415 bp\n",
      "Total written (filtered):     18,313,042 bp (93.4%)\n",
      "  Read 1:     9,139,308 bp\n",
      "  Read 2:     9,173,734 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t188\t0.0\t1\t0 185 3\n",
      "20\t32913\t0.0\t2\t32365 538 10\n",
      "21\t63\t0.0\t2\t0 62 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33146 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t149\t0.0\t1\t0 149\n",
      "19\t32941\t0.0\t1\t32464 477\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2044_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2044_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2044_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2044_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2044_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2044_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2044_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2044_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,317 reads  @     66.4 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.28 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,317\n",
      "  Read 1 with adapter:                  34,284 (99.9%)\n",
      "  Read 2 with adapter:                  34,224 (99.7%)\n",
      "Pairs written (passing filters):        34,317 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,245,855 bp\n",
      "  Read 1:    10,123,501 bp\n",
      "  Read 2:    10,122,354 bp\n",
      "Total written (filtered):     18,910,150 bp (93.4%)\n",
      "  Read 1:     9,437,954 bp\n",
      "  Read 2:     9,472,196 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34284 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t196\t0.0\t1\t0 193 3\n",
      "20\t33995\t0.0\t2\t33431 551 13\n",
      "21\t55\t0.0\t2\t0 53 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34224 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t33956\t0.0\t1\t33528 428\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2045_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2045_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2045_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2045_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2045_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2045_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2045_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2045_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         4,600 reads  @     67.6 µs/read;   0.89 M reads/minute\n",
      "Finished in 0.31 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              4,600\n",
      "  Read 1 with adapter:                   4,597 (99.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         4,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,143,238 bp\n",
      "  Read 1:     1,356,999 bp\n",
      "  Read 2:       786,239 bp\n",
      "Total written (filtered):      2,051,308 bp (95.7%)\n",
      "  Read 1:     1,265,069 bp\n",
      "  Read 2:       786,239 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 4597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t28\t0.0\t1\t0 26 2\n",
      "20\t4553\t0.0\t2\t4488 62 3\n",
      "21\t10\t0.0\t2\t0 10\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2046_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2046_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2046_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2046_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2046_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2046_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2046_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2046_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        38,844 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.75 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,844\n",
      "  Read 1 with adapter:                  38,812 (99.9%)\n",
      "  Read 2 with adapter:                  38,703 (99.6%)\n",
      "Pairs written (passing filters):        38,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,911,394 bp\n",
      "  Read 1:    11,458,202 bp\n",
      "  Read 2:    11,453,192 bp\n",
      "Total written (filtered):     21,400,063 bp (93.4%)\n",
      "  Read 1:    10,682,089 bp\n",
      "  Read 2:    10,717,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t228\t0.0\t1\t0 224 4\n",
      "20\t38471\t0.0\t2\t37863 595 13\n",
      "21\t77\t0.0\t2\t0 73 4\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t38438\t0.0\t1\t37879 559\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2047_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2047_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2047_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2047_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2047_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2047_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2047_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2047_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,917 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.39 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,917\n",
      "  Read 1 with adapter:                  34,895 (99.9%)\n",
      "  Read 2 with adapter:                  34,806 (99.7%)\n",
      "Pairs written (passing filters):        34,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,600,999 bp\n",
      "  Read 1:    10,300,501 bp\n",
      "  Read 2:    10,300,498 bp\n",
      "Total written (filtered):     19,241,960 bp (93.4%)\n",
      "  Read 1:     9,602,695 bp\n",
      "  Read 2:     9,639,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34895 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t184\t0.0\t1\t0 180 4\n",
      "20\t34617\t0.0\t2\t34118 491 8\n",
      "21\t66\t0.0\t2\t0 66\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t149\t0.0\t1\t0 149\n",
      "19\t34589\t0.0\t1\t34131 458\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2048_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2048_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2048_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2048_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2048_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2048_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2048_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2048_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        25,606 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 1.81 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,606\n",
      "  Read 1 with adapter:                  25,591 (99.9%)\n",
      "  Read 2 with adapter:                  25,536 (99.7%)\n",
      "Pairs written (passing filters):        25,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,105,339 bp\n",
      "  Read 1:     7,553,488 bp\n",
      "  Read 2:     7,551,851 bp\n",
      "Total written (filtered):     14,108,505 bp (93.4%)\n",
      "  Read 1:     7,041,763 bp\n",
      "  Read 2:     7,066,742 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 25591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t161\t0.0\t1\t0 159 2\n",
      "20\t25366\t0.0\t2\t24977 371 18\n",
      "21\t38\t0.0\t2\t0 38\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 25536 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t152\t0.0\t1\t0 152\n",
      "19\t25307\t0.0\t1\t24985 322\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2049_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2049_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2049_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2049_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2049_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2049_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2049_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2049_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        22,699 reads  @     57.3 µs/read;   1.05 M reads/minute\n",
      "Finished in 1.30 s (57 us/read; 1.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,699\n",
      "  Read 1 with adapter:                  22,678 (99.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        22,698 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,600,876 bp\n",
      "  Read 1:     6,696,195 bp\n",
      "  Read 2:     3,904,681 bp\n",
      "Total written (filtered):     10,147,097 bp (95.7%)\n",
      "  Read 1:     6,242,416 bp\n",
      "  Read 2:     3,904,681 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 22678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t113\t0.0\t1\t0 108 5\n",
      "20\t22504\t0.0\t2\t22175 321 8\n",
      "21\t45\t0.0\t2\t0 43 2\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2050_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2050_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2050_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2050_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2050_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2050_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2050_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2050_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,564 reads  @     71.0 µs/read;   0.84 M reads/minute\n",
      "Finished in 1.75 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,564\n",
      "  Read 1 with adapter:                  24,544 (99.9%)\n",
      "  Read 2 with adapter:                  24,444 (99.5%)\n",
      "Pairs written (passing filters):        24,564 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,487,520 bp\n",
      "  Read 1:     7,246,149 bp\n",
      "  Read 2:     7,241,371 bp\n",
      "Total written (filtered):     13,532,370 bp (93.4%)\n",
      "  Read 1:     6,755,348 bp\n",
      "  Read 2:     6,777,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t129\t0.0\t1\t0 127 2\n",
      "20\t24345\t0.0\t2\t23930 410 5\n",
      "21\t54\t0.0\t2\t0 52 2\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24444 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t130\t0.0\t1\t0 130\n",
      "19\t24271\t0.0\t1\t23808 463\n",
      "20\t43\t0.0\t1\t0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2051_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2051_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2051_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2051_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2051_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2051_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2051_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2051_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,402 reads  @     66.6 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.23 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,402\n",
      "  Read 1 with adapter:                  33,387 (100.0%)\n",
      "  Read 2 with adapter:                  33,293 (99.7%)\n",
      "Pairs written (passing filters):        33,402 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,707,118 bp\n",
      "  Read 1:     9,853,581 bp\n",
      "  Read 2:     9,853,537 bp\n",
      "Total written (filtered):     18,407,029 bp (93.4%)\n",
      "  Read 1:     9,185,948 bp\n",
      "  Read 2:     9,221,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t169\t0.0\t1\t0 166 3\n",
      "20\t33130\t0.0\t2\t32548 573 9\n",
      "21\t54\t0.0\t2\t0 53 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t160\t0.0\t1\t0 160\n",
      "19\t33084\t0.0\t1\t32524 560\n",
      "20\t49\t0.0\t1\t0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2052_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2052_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2052_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2052_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2052_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2052_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2052_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2052_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,403 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.26 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,403\n",
      "  Read 1 with adapter:                  30,372 (99.9%)\n",
      "  Read 2 with adapter:                  30,283 (99.6%)\n",
      "Pairs written (passing filters):        30,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,935,367 bp\n",
      "  Read 1:     8,968,864 bp\n",
      "  Read 2:     8,966,503 bp\n",
      "Total written (filtered):     16,752,707 bp (93.4%)\n",
      "  Read 1:     8,361,509 bp\n",
      "  Read 2:     8,391,198 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t164\t0.0\t1\t0 158 6\n",
      "20\t30122\t0.0\t2\t29607 500 15\n",
      "21\t57\t0.0\t2\t0 56 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t147\t0.0\t1\t0 147\n",
      "19\t30061\t0.0\t1\t29583 478\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2053_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2053_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2053_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2053_185_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2053_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2053_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2053_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2053_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           677 reads  @    134.6 µs/read;   0.45 M reads/minute\n",
      "Finished in 0.09 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                677\n",
      "  Read 1 with adapter:                     676 (99.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):           677 (100.0%)\n",
      "\n",
      "Total basepairs processed:       281,886 bp\n",
      "  Read 1:       199,714 bp\n",
      "  Read 2:        82,172 bp\n",
      "Total written (filtered):        268,369 bp (95.2%)\n",
      "  Read 1:       186,197 bp\n",
      "  Read 2:        82,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t671\t0.0\t2\t649 22\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2054_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2054_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2054_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2054_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2054_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2054_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2054_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2054_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,727 reads  @     63.0 µs/read;   0.95 M reads/minute\n",
      "Finished in 2.13 s (63 us/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,727\n",
      "  Read 1 with adapter:                  33,706 (99.9%)\n",
      "  Read 2 with adapter:                  33,607 (99.6%)\n",
      "Pairs written (passing filters):        33,727 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,895,386 bp\n",
      "  Read 1:     9,949,456 bp\n",
      "  Read 2:     9,945,930 bp\n",
      "Total written (filtered):     18,582,964 bp (93.4%)\n",
      "  Read 1:     9,275,449 bp\n",
      "  Read 2:     9,307,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t196\t0.0\t1\t0 195 1\n",
      "20\t33424\t0.0\t2\t32960 450 14\n",
      "21\t61\t0.0\t2\t0 59 2\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t175\t0.0\t1\t0 175\n",
      "19\t33375\t0.0\t1\t32825 550\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2055_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2055_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2055_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2055_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2055_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2055_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2055_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2055_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,953 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.19 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,953\n",
      "  Read 1 with adapter:                  31,936 (99.9%)\n",
      "  Read 2 with adapter:                  31,864 (99.7%)\n",
      "Pairs written (passing filters):        31,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,851,995 bp\n",
      "  Read 1:     9,425,947 bp\n",
      "  Read 2:     9,426,048 bp\n",
      "Total written (filtered):     17,608,073 bp (93.4%)\n",
      "  Read 1:     8,787,363 bp\n",
      "  Read 2:     8,820,710 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31936 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t213\t0.0\t1\t0 212 1\n",
      "20\t31642\t0.0\t2\t31142 484 16\n",
      "21\t53\t0.0\t2\t0 51 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t130\t0.0\t1\t0 130\n",
      "19\t31682\t0.0\t1\t31168 514\n",
      "20\t52\t0.0\t1\t0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2056_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2056_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2056_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2056_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2056_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2056_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2056_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2056_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        30,593 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.12 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,593\n",
      "  Read 1 with adapter:                  30,568 (99.9%)\n",
      "  Read 2 with adapter:                  30,434 (99.5%)\n",
      "Pairs written (passing filters):        30,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,048,865 bp\n",
      "  Read 1:     9,024,930 bp\n",
      "  Read 2:     9,023,935 bp\n",
      "Total written (filtered):     16,859,447 bp (93.4%)\n",
      "  Read 1:     8,413,682 bp\n",
      "  Read 2:     8,445,765 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t184\t0.0\t1\t0 179 5\n",
      "20\t30299\t0.0\t2\t29843 443 13\n",
      "21\t62\t0.0\t2\t0 60 2\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t159\t0.0\t1\t0 159\n",
      "19\t30192\t0.0\t1\t29724 468\n",
      "20\t83\t0.0\t1\t0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2057_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2057_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2057_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2057_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2057_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2057_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2057_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2057_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         5,021 reads  @     63.3 µs/read;   0.95 M reads/minute\n",
      "Finished in 0.32 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              5,021\n",
      "  Read 1 with adapter:                   5,017 (99.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         5,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,317,472 bp\n",
      "  Read 1:     1,481,193 bp\n",
      "  Read 2:       836,279 bp\n",
      "Total written (filtered):      2,217,155 bp (95.7%)\n",
      "  Read 1:     1,380,876 bp\n",
      "  Read 2:       836,279 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 5017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t31\t0.0\t1\t0 31\n",
      "20\t4976\t0.0\t2\t4894 82\n",
      "21\t8\t0.0\t2\t0 8\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2058_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2058_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2058_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2058_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2058_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2058_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2058_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2058_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,729 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.10 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,729\n",
      "  Read 1 with adapter:                  29,701 (99.9%)\n",
      "  Read 2 with adapter:                  29,633 (99.7%)\n",
      "Pairs written (passing filters):        29,729 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,535,742 bp\n",
      "  Read 1:     8,768,854 bp\n",
      "  Read 2:     8,766,888 bp\n",
      "Total written (filtered):     16,378,925 bp (93.4%)\n",
      "  Read 1:     8,174,949 bp\n",
      "  Read 2:     8,203,976 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29701 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t174\t0.0\t1\t0 170 4\n",
      "20\t29446\t0.0\t2\t28948 489 9\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29633 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t166\t0.0\t1\t0 166\n",
      "19\t29416\t0.0\t1\t28953 463\n",
      "20\t51\t0.0\t1\t0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2059_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2059_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2059_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2059_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2059_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2059_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2059_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2059_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,547 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.23 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,547\n",
      "  Read 1 with adapter:                  30,472 (99.8%)\n",
      "  Read 2 with adapter:                  30,452 (99.7%)\n",
      "Pairs written (passing filters):        30,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,014,412 bp\n",
      "  Read 1:     9,003,071 bp\n",
      "  Read 2:     9,011,341 bp\n",
      "Total written (filtered):     16,826,591 bp (93.4%)\n",
      "  Read 1:     8,393,743 bp\n",
      "  Read 2:     8,432,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30472 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t180\t0.0\t1\t0 177 3\n",
      "20\t30217\t0.0\t2\t29752 457 8\n",
      "21\t54\t0.0\t2\t0 54\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t158\t0.0\t1\t0 158\n",
      "19\t30231\t0.0\t1\t29828 403\n",
      "20\t63\t0.0\t1\t0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2060_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2060_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2060_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2060_199_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2060_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2060_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2060_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2060_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,860 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.49 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,860\n",
      "  Read 1 with adapter:                  34,832 (99.9%)\n",
      "  Read 2 with adapter:                  34,755 (99.7%)\n",
      "Pairs written (passing filters):        34,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,566,086 bp\n",
      "  Read 1:    10,282,721 bp\n",
      "  Read 2:    10,283,365 bp\n",
      "Total written (filtered):     19,209,349 bp (93.4%)\n",
      "  Read 1:     9,586,225 bp\n",
      "  Read 2:     9,623,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t205\t0.0\t1\t0 203 2\n",
      "20\t34536\t0.0\t2\t34029 497 10\n",
      "21\t53\t0.0\t2\t0 52 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 34755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t34489\t0.0\t1\t34045 444\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2061_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2061_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2061_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2061_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2061_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2061_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2061_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2061_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,220 reads  @     55.7 µs/read;   1.08 M reads/minute\n",
      "Finished in 0.63 s (56 us/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,220\n",
      "  Read 1 with adapter:                  11,211 (99.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        11,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,709,374 bp\n",
      "  Read 1:     3,309,898 bp\n",
      "  Read 2:     1,399,476 bp\n",
      "Total written (filtered):      4,484,909 bp (95.2%)\n",
      "  Read 1:     3,085,433 bp\n",
      "  Read 2:     1,399,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 11211 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t55\t0.0\t1\t0 52 3\n",
      "20\t11132\t0.0\t2\t10964 167 1\n",
      "21\t19\t0.0\t2\t0 18 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2062_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2062_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2062_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2062_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2062_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2062_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2062_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2062_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,853 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.35 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,853\n",
      "  Read 1 with adapter:                  35,819 (99.9%)\n",
      "  Read 2 with adapter:                  35,738 (99.7%)\n",
      "Pairs written (passing filters):        35,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,149,711 bp\n",
      "  Read 1:    10,576,518 bp\n",
      "  Read 2:    10,573,193 bp\n",
      "Total written (filtered):     19,754,505 bp (93.4%)\n",
      "  Read 1:     9,860,219 bp\n",
      "  Read 2:     9,894,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35819 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t177\t0.0\t1\t0 176 1\n",
      "20\t35539\t0.0\t2\t35029 496 14\n",
      "21\t78\t0.0\t2\t0 78\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t35475\t0.0\t1\t34992 483\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2063_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2063_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2063_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2063_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2063_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2063_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2063_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2063_205_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        37,150 reads  @     63.9 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.38 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,150\n",
      "  Read 1 with adapter:                  37,126 (99.9%)\n",
      "  Read 2 with adapter:                  37,055 (99.7%)\n",
      "Pairs written (passing filters):        37,150 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,918,326 bp\n",
      "  Read 1:    10,959,100 bp\n",
      "  Read 2:    10,959,226 bp\n",
      "Total written (filtered):     20,471,950 bp (93.4%)\n",
      "  Read 1:    10,216,667 bp\n",
      "  Read 2:    10,255,283 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 37126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t198\t0.0\t1\t0 189 9\n",
      "20\t36812\t0.0\t2\t36153 642 17\n",
      "21\t85\t0.0\t2\t0 78 7\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 37055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t169\t0.0\t1\t0 169\n",
      "19\t36819\t0.0\t1\t36255 564\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2064_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2064_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2064_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2064_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2064_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2064_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2064_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2064_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,792 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.32 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,792\n",
      "  Read 1 with adapter:                  32,724 (99.8%)\n",
      "  Read 2 with adapter:                  32,711 (99.8%)\n",
      "Pairs written (passing filters):        32,792 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,341,391 bp\n",
      "  Read 1:     9,668,500 bp\n",
      "  Read 2:     9,672,891 bp\n",
      "Total written (filtered):     18,065,575 bp (93.4%)\n",
      "  Read 1:     9,014,072 bp\n",
      "  Read 2:     9,051,503 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32724 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t150\t0.0\t1\t0 143 7\n",
      "20\t32462\t0.0\t2\t31987 466 9\n",
      "21\t82\t0.0\t2\t0 81 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32711 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t32450\t0.0\t1\t32030 420\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2065_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2065_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2065_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2065_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2065_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2065_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2065_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2065_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           971 reads  @    107.9 µs/read;   0.56 M reads/minute\n",
      "Finished in 0.11 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                971\n",
      "  Read 1 with adapter:                     964 (99.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):           970 (99.9%)\n",
      "\n",
      "Total basepairs processed:       446,007 bp\n",
      "  Read 1:       286,445 bp\n",
      "  Read 2:       159,562 bp\n",
      "Total written (filtered):        426,460 bp (95.6%)\n",
      "  Read 1:       266,898 bp\n",
      "  Read 2:       159,562 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 964 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t10\t0.0\t1\t0 10\n",
      "20\t952\t0.0\t2\t932 19 1\n",
      "21\t2\t0.0\t2\t0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2066_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2066_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2066_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2066_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2066_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2066_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2066_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2066_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        26,635 reads  @     67.1 µs/read;   0.89 M reads/minute\n",
      "Finished in 1.79 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,635\n",
      "  Read 1 with adapter:                  26,608 (99.9%)\n",
      "  Read 2 with adapter:                  26,526 (99.6%)\n",
      "Pairs written (passing filters):        26,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,710,576 bp\n",
      "  Read 1:     7,857,051 bp\n",
      "  Read 2:     7,853,525 bp\n",
      "Total written (filtered):     14,674,604 bp (93.4%)\n",
      "  Read 1:     7,324,980 bp\n",
      "  Read 2:     7,349,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 26608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t154\t0.0\t1\t0 153 1\n",
      "20\t26363\t0.0\t2\t25943 412 8\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 26526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t131\t0.0\t1\t0 131\n",
      "19\t26357\t0.0\t1\t25948 409\n",
      "20\t38\t0.0\t1\t0 38\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2067_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2067_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2067_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2067_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2067_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2067_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2067_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2067_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,820 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 1.68 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,820\n",
      "  Read 1 with adapter:                  24,804 (99.9%)\n",
      "  Read 2 with adapter:                  24,730 (99.6%)\n",
      "Pairs written (passing filters):        24,820 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,643,770 bp\n",
      "  Read 1:     7,321,892 bp\n",
      "  Read 2:     7,321,878 bp\n",
      "Total written (filtered):     13,677,992 bp (93.4%)\n",
      "  Read 1:     6,825,917 bp\n",
      "  Read 2:     6,852,075 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24804 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t162\t0.0\t1\t0 160 2\n",
      "20\t24551\t0.0\t2\t24169 372 10\n",
      "21\t61\t0.0\t2\t0 57 4\n",
      "22\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t111\t0.0\t1\t0 111\n",
      "19\t24575\t0.0\t1\t24212 363\n",
      "20\t44\t0.0\t1\t0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2068_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2068_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2068_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2068_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2068_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2068_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2068_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2068_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,571 reads  @     77.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.37 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,571\n",
      "  Read 1 with adapter:                  30,539 (99.9%)\n",
      "  Read 2 with adapter:                  30,475 (99.7%)\n",
      "Pairs written (passing filters):        30,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,035,865 bp\n",
      "  Read 1:     9,017,922 bp\n",
      "  Read 2:     9,017,943 bp\n",
      "Total written (filtered):     16,846,239 bp (93.4%)\n",
      "  Read 1:     8,407,235 bp\n",
      "  Read 2:     8,439,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30539 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t186\t0.0\t1\t0 182 4\n",
      "20\t30262\t0.0\t2\t29782 470 10\n",
      "21\t69\t0.0\t2\t0 66 3\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30475 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t154\t0.0\t1\t0 154\n",
      "19\t30253\t0.0\t1\t29832 421\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2069_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2069_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2069_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2069_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2069_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2069_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2069_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2069_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,684 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 0.20 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,684\n",
      "  Read 1 with adapter:                   2,683 (100.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         2,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,232,945 bp\n",
      "  Read 1:       791,780 bp\n",
      "  Read 2:       441,165 bp\n",
      "Total written (filtered):      1,179,293 bp (95.6%)\n",
      "  Read 1:       738,128 bp\n",
      "  Read 2:       441,165 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 2683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t17\t0.0\t1\t0 16 1\n",
      "20\t2659\t0.0\t2\t2613 45 1\n",
      "21\t5\t0.0\t2\t0 5\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2070_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2070_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2070_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2070_219_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2070_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2070_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2070_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2070_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,102 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.85 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,102\n",
      "  Read 1 with adapter:                  27,072 (99.9%)\n",
      "  Read 2 with adapter:                  27,018 (99.7%)\n",
      "Pairs written (passing filters):        27,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,986,198 bp\n",
      "  Read 1:     7,994,460 bp\n",
      "  Read 2:     7,991,738 bp\n",
      "Total written (filtered):     14,931,691 bp (93.4%)\n",
      "  Read 1:     7,453,221 bp\n",
      "  Read 2:     7,478,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27072 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t252\t0.0\t1\t0 249 3\n",
      "20\t26755\t0.0\t2\t26367 378 10\n",
      "21\t43\t0.0\t2\t0 43\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27018 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t135\t0.0\t1\t0 135\n",
      "19\t26822\t0.0\t1\t26448 374\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2071_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2071_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2071_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2071_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2071_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2071_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2071_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2071_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,947 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 1.17 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,947\n",
      "  Read 1 with adapter:                  16,917 (99.8%)\n",
      "  Read 2 with adapter:                  16,899 (99.7%)\n",
      "Pairs written (passing filters):        16,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,995,362 bp\n",
      "  Read 1:     4,996,009 bp\n",
      "  Read 2:     4,999,353 bp\n",
      "Total written (filtered):      9,336,120 bp (93.4%)\n",
      "  Read 1:     4,657,791 bp\n",
      "  Read 2:     4,678,329 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 16917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t148\t0.0\t1\t0 147 1\n",
      "20\t16719\t0.0\t2\t16490 223 6\n",
      "21\t30\t0.0\t2\t0 29 1\n",
      "22\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 16899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t79\t0.0\t1\t0 79\n",
      "19\t16798\t0.0\t1\t16566 232\n",
      "20\t22\t0.0\t1\t0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2072_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2072_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2072_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2072_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2072_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2072_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2072_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2072_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        24,553 reads  @     66.1 µs/read;   0.91 M reads/minute\n",
      "Finished in 1.63 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,553\n",
      "  Read 1 with adapter:                  24,528 (99.9%)\n",
      "  Read 2 with adapter:                  24,453 (99.6%)\n",
      "Pairs written (passing filters):        24,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,484,740 bp\n",
      "  Read 1:     7,242,847 bp\n",
      "  Read 2:     7,241,893 bp\n",
      "Total written (filtered):     13,529,810 bp (93.4%)\n",
      "  Read 1:     6,752,450 bp\n",
      "  Read 2:     6,777,360 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 24528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t212\t0.0\t1\t0 206 6\n",
      "20\t24253\t0.0\t2\t23911 337 5\n",
      "21\t37\t0.0\t2\t0 37\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 24453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t132\t0.0\t1\t0 132\n",
      "19\t24263\t0.0\t1\t23903 360\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2073_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2073_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2073_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2073_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2073_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2073_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2073_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2073_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00         2,804 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 0.20 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,804\n",
      "  Read 1 with adapter:                   2,793 (99.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         2,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,297,961 bp\n",
      "  Read 1:       826,398 bp\n",
      "  Read 2:       471,563 bp\n",
      "Total written (filtered):      1,242,113 bp (95.7%)\n",
      "  Read 1:       770,550 bp\n",
      "  Read 2:       471,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 2793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t2\t0.0\t1\t0 0 2\n",
      "19\t17\t0.0\t1\t0 16 1\n",
      "20\t2766\t0.0\t2\t2707 59\n",
      "21\t7\t0.0\t2\t0 7\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2074_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2074_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2074_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2074_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2074_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2074_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2074_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2074_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,363 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 1.93 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,363\n",
      "  Read 1 with adapter:                  27,245 (99.6%)\n",
      "  Read 2 with adapter:                  27,193 (99.4%)\n",
      "Pairs written (passing filters):        27,363 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,136,900 bp\n",
      "  Read 1:     8,069,389 bp\n",
      "  Read 2:     8,067,511 bp\n",
      "Total written (filtered):     15,075,536 bp (93.4%)\n",
      "  Read 1:     7,524,610 bp\n",
      "  Read 2:     7,550,926 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 27245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t162\t0.0\t1\t0 160 2\n",
      "20\t27025\t0.0\t2\t26572 435 18\n",
      "21\t35\t0.0\t2\t0 35\n",
      "22\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 27193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t127\t0.0\t1\t0 127\n",
      "19\t27021\t0.0\t1\t26539 482\n",
      "20\t45\t0.0\t1\t0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2075_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2075_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2075_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2075_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2075_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2075_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2075_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2075_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,418 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.28 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,418\n",
      "  Read 1 with adapter:                  31,332 (99.7%)\n",
      "  Read 2 with adapter:                  31,276 (99.5%)\n",
      "Pairs written (passing filters):        31,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,532,393 bp\n",
      "  Read 1:     9,264,098 bp\n",
      "  Read 2:     9,268,295 bp\n",
      "Total written (filtered):     17,311,710 bp (93.4%)\n",
      "  Read 1:     8,637,577 bp\n",
      "  Read 2:     8,674,133 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31332 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t178\t0.0\t1\t0 177 1\n",
      "20\t31052\t0.0\t2\t30550 489 13\n",
      "21\t61\t0.0\t2\t0 59 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31276 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t131\t0.0\t1\t0 131\n",
      "19\t31096\t0.0\t1\t30593 503\n",
      "20\t49\t0.0\t1\t0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2076_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2076_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2076_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2076_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2076_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2076_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2076_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2076_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,380 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.24 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,380\n",
      "  Read 1 with adapter:                  29,264 (99.6%)\n",
      "  Read 2 with adapter:                  29,223 (99.5%)\n",
      "Pairs written (passing filters):        29,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,330,023 bp\n",
      "  Read 1:     8,664,247 bp\n",
      "  Read 2:     8,665,776 bp\n",
      "Total written (filtered):     16,189,759 bp (93.4%)\n",
      "  Read 1:     8,079,124 bp\n",
      "  Read 2:     8,110,635 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 29264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t198\t0.0\t1\t0 197 1\n",
      "20\t28970\t0.0\t2\t28464 491 15\n",
      "21\t55\t0.0\t2\t0 54 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 29223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t171\t0.0\t1\t0 171\n",
      "19\t28977\t0.0\t1\t28572 405\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2077_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2077_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2077_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2077_233_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2077_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2077_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2077_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2077_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,872 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 0.26 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,872\n",
      "  Read 1 with adapter:                   3,871 (100.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         3,872 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,621,891 bp\n",
      "  Read 1:     1,142,237 bp\n",
      "  Read 2:       479,654 bp\n",
      "Total written (filtered):      1,544,473 bp (95.2%)\n",
      "  Read 1:     1,064,819 bp\n",
      "  Read 2:       479,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 3871 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t17\t0.0\t1\t0 17\n",
      "20\t3841\t0.0\t2\t3782 56 3\n",
      "21\t11\t0.0\t2\t0 11\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2078_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2078_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2078_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2078_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2078_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2078_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2078_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2078_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,005 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.74 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,005\n",
      "  Read 1 with adapter:                  38,973 (99.9%)\n",
      "  Read 2 with adapter:                  38,874 (99.7%)\n",
      "Pairs written (passing filters):        39,005 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,007,285 bp\n",
      "  Read 1:    11,506,467 bp\n",
      "  Read 2:    11,500,818 bp\n",
      "Total written (filtered):     21,489,470 bp (93.4%)\n",
      "  Read 1:    10,727,150 bp\n",
      "  Read 2:    10,762,320 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38973 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t220\t0.0\t1\t0 215 5\n",
      "20\t38666\t0.0\t2\t38064 588 14\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t38636\t0.0\t1\t38023 613\n",
      "20\t65\t0.0\t1\t0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2079_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2079_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2079_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2079_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2079_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2079_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2079_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2079_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,363 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.19 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,363\n",
      "  Read 1 with adapter:                  30,343 (99.9%)\n",
      "  Read 2 with adapter:                  30,263 (99.7%)\n",
      "Pairs written (passing filters):        30,363 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,913,861 bp\n",
      "  Read 1:     8,957,076 bp\n",
      "  Read 2:     8,956,785 bp\n",
      "Total written (filtered):     16,732,158 bp (93.4%)\n",
      "  Read 1:     8,350,269 bp\n",
      "  Read 2:     8,381,889 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 30343 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t195\t0.0\t1\t0 192 3\n",
      "20\t30034\t0.0\t2\t29433 589 12\n",
      "21\t58\t0.0\t2\t0 57 1\n",
      "22\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 30263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t157\t0.0\t1\t0 157\n",
      "19\t30050\t0.0\t1\t29460 590\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2080_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2080_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2080_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2080_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2080_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-zckhhp6o/S2080_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2080_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-vx5ue5lh/a3b108f0-2bda-475f-819c-be7bac8ef7e3/data/S2080_239_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,623 reads  @     85.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.78 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,623\n",
      "  Read 1 with adapter:                  32,602 (99.9%)\n",
      "  Read 2 with adapter:                  32,480 (99.6%)\n",
      "Pairs written (passing filters):        32,623 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,244,913 bp\n",
      "  Read 1:     9,623,352 bp\n",
      "  Read 2:     9,621,561 bp\n",
      "Total written (filtered):     17,976,027 bp (93.4%)\n",
      "  Read 1:     8,971,445 bp\n",
      "  Read 2:     9,004,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t199\t0.0\t1\t0 197 2\n",
      "20\t32304\t0.0\t2\t31787 499 18\n",
      "21\t60\t0.0\t2\t0 60\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t32191\t0.0\t1\t31718 473\n",
      "20\t74\t0.0\t1\t0 74\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: g-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: g-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-m2zb81qm')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b08e7896-74ec-4353-a375-7b590659ea24>"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.g import the fastq files for library g into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/P2107-Lib1-manifest_18S.csv \\\n",
    "  --output-path g-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.g trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences g-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences g-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.g summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data g-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization g-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('g-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: g-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: g-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: g-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.g Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs g-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 255 \\\n",
    "  --p-trunc-len-r 205 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table g-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats g-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: g-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0tpqteuw')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c54e9a66-c7cb-4831-9b39-1563e58ea4f5>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.g view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table g-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization g-table.qzv \n",
    "q2.Visualization.load('g-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: g-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.g view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization g-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/ENP-Lib-manifest_18S.csv as PairedEndFastqManifestPhred33 to h-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2345_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2345_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2345_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2345_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2345_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2345_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2345_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2345_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       141,001 reads  @     87.1 µs/read;   0.69 M reads/minute\n",
      "Finished in 12.28 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            141,001\n",
      "  Read 1 with adapter:                 140,945 (100.0%)\n",
      "  Read 2 with adapter:                 140,094 (99.4%)\n",
      "Pairs written (passing filters):       141,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    83,179,170 bp\n",
      "  Read 1:    41,590,972 bp\n",
      "  Read 2:    41,588,198 bp\n",
      "Total written (filtered):     77,699,302 bp (93.4%)\n",
      "  Read 1:    38,772,494 bp\n",
      "  Read 2:    38,926,808 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 140945 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t713\t0.0\t1\t0 693 20\n",
      "20\t139944\t0.0\t2\t137618 2282 44\n",
      "21\t201\t0.0\t2\t0 199 2\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 140094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t629\t0.0\t1\t0 629\n",
      "19\t139232\t0.0\t1\t135832 3400\n",
      "20\t233\t0.0\t1\t0 233\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2346_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2346_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2346_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2346_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2346_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2346_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2346_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2346_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       123,007 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 10.19 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,007\n",
      "  Read 1 with adapter:                 122,978 (100.0%)\n",
      "  Read 2 with adapter:                 122,302 (99.4%)\n",
      "Pairs written (passing filters):       123,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,568,404 bp\n",
      "  Read 1:    36,286,120 bp\n",
      "  Read 2:    36,282,284 bp\n",
      "Total written (filtered):     67,785,750 bp (93.4%)\n",
      "  Read 1:    33,826,901 bp\n",
      "  Read 2:    33,958,849 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 122978 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t603\t0.0\t1\t0 597 6\n",
      "20\t122102\t0.0\t2\t120019 2053 30\n",
      "21\t188\t0.0\t2\t0 184 4\n",
      "22\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 122302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t519\t0.0\t1\t0 519\n",
      "19\t121567\t0.0\t1\t118126 3441\n",
      "20\t216\t0.0\t1\t0 216\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2347_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2347_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2347_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2347_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2347_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2347_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2347_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2347_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       137,585 reads  @     88.4 µs/read;   0.68 M reads/minute\n",
      "Finished in 12.16 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,585\n",
      "  Read 1 with adapter:                 137,512 (99.9%)\n",
      "  Read 2 with adapter:                 136,874 (99.5%)\n",
      "Pairs written (passing filters):       137,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,164,488 bp\n",
      "  Read 1:    40,582,892 bp\n",
      "  Read 2:    40,581,596 bp\n",
      "Total written (filtered):     75,814,191 bp (93.4%)\n",
      "  Read 1:    37,833,099 bp\n",
      "  Read 2:    37,981,092 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 137512 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t1\t0 0 34\n",
      "19\t711\t0.0\t1\t0 698 13\n",
      "20\t136503\t0.0\t2\t134199 2275 29\n",
      "21\t196\t0.0\t2\t0 193 3\n",
      "22\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 136874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t608\t0.0\t1\t0 608\n",
      "19\t136036\t0.0\t1\t132502 3534\n",
      "20\t230\t0.0\t1\t0 230\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2348_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2348_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2348_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2348_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2348_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2348_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2348_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2348_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       110,588 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 9.55 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,588\n",
      "  Read 1 with adapter:                 110,552 (100.0%)\n",
      "  Read 2 with adapter:                 109,937 (99.4%)\n",
      "Pairs written (passing filters):       110,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,236,620 bp\n",
      "  Read 1:    32,622,465 bp\n",
      "  Read 2:    32,614,155 bp\n",
      "Total written (filtered):     60,937,567 bp (93.4%)\n",
      "  Read 1:    30,411,792 bp\n",
      "  Read 2:    30,525,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 110552 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t605\t0.0\t1\t0 597 8\n",
      "20\t109700\t0.0\t2\t107920 1754 26\n",
      "21\t160\t0.0\t2\t0 157 3\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109937 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t681\t0.0\t1\t0 681\n",
      "19\t108998\t0.0\t1\t106068 2930\n",
      "20\t258\t0.0\t1\t0 258\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2349_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2349_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2349_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2349_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2349_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2349_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2349_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2349_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[       8<---] 00:00:23       250,326 reads  @     92.4 µs/read;   0.65 M reads/minute\n",
      "Finished in 23.13 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            250,326\n",
      "  Read 1 with adapter:                 250,233 (100.0%)\n",
      "  Read 2 with adapter:                 249,202 (99.6%)\n",
      "Pairs written (passing filters):       250,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:   147,669,005 bp\n",
      "  Read 1:    73,840,218 bp\n",
      "  Read 2:    73,828,787 bp\n",
      "Total written (filtered):    137,926,715 bp (93.4%)\n",
      "  Read 1:    68,831,987 bp\n",
      "  Read 2:    69,094,728 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 250233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t59\t0.0\t1\t0 0 59\n",
      "19\t1311\t0.0\t1\t0 1280 31\n",
      "20\t248373\t0.0\t2\t244163 4166 44\n",
      "21\t380\t0.0\t2\t0 371 9\n",
      "22\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 249202 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1169\t0.0\t1\t0 1169\n",
      "19\t247643\t0.0\t1\t238940 8703\n",
      "20\t390\t0.0\t1\t0 390\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2350_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2350_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2350_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2350_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2350_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2350_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2350_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2350_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:12       123,589 reads  @     99.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 12.31 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,589\n",
      "  Read 1 with adapter:                 123,551 (100.0%)\n",
      "  Read 2 with adapter:                 122,301 (99.0%)\n",
      "Pairs written (passing filters):       123,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,818,942 bp\n",
      "  Read 1:    36,457,582 bp\n",
      "  Read 2:    36,361,360 bp\n",
      "Total written (filtered):     68,024,855 bp (93.4%)\n",
      "  Read 1:    33,986,924 bp\n",
      "  Read 2:    34,037,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 123551 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t630\t0.0\t1\t0 617 13\n",
      "20\t122605\t0.0\t2\t120542 2041 22\n",
      "21\t216\t0.0\t2\t0 210 6\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 122301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t496\t0.0\t1\t0 496\n",
      "19\t121599\t0.0\t1\t117516 4083\n",
      "20\t206\t0.0\t1\t0 206\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2351_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2351_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2351_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2351_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2351_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2351_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2351_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2351_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11       118,759 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 11.25 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,759\n",
      "  Read 1 with adapter:                 118,687 (99.9%)\n",
      "  Read 2 with adapter:                 118,106 (99.5%)\n",
      "Pairs written (passing filters):       118,759 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,027,134 bp\n",
      "  Read 1:    35,031,853 bp\n",
      "  Read 2:    34,995,281 bp\n",
      "Total written (filtered):     65,410,122 bp (93.4%)\n",
      "  Read 1:    32,658,530 bp\n",
      "  Read 2:    32,751,592 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 118687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t673\t0.0\t1\t0 661 12\n",
      "20\t117724\t0.0\t2\t115562 2139 23\n",
      "21\t216\t0.0\t2\t0 215 1\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 118106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t582\t0.0\t1\t0 582\n",
      "19\t117267\t0.0\t1\t114802 2465\n",
      "20\t257\t0.0\t1\t0 257\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2352_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2352_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2352_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2352_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2352_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2352_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2352_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2352_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        87,605 reads  @     93.6 µs/read;   0.64 M reads/minute\n",
      "Finished in 8.20 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,605\n",
      "  Read 1 with adapter:                  87,579 (100.0%)\n",
      "  Read 2 with adapter:                  85,131 (97.2%)\n",
      "Pairs written (passing filters):        87,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,545,089 bp\n",
      "  Read 1:    25,842,755 bp\n",
      "  Read 2:    25,702,334 bp\n",
      "Total written (filtered):     48,176,577 bp (93.5%)\n",
      "  Read 1:    24,091,481 bp\n",
      "  Read 2:    24,085,096 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 87579 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t491\t0.0\t1\t0 481 10\n",
      "20\t86883\t0.0\t2\t85261 1594 28\n",
      "21\t137\t0.0\t2\t0 136 1\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t395\t0.0\t1\t0 395\n",
      "19\t84592\t0.0\t1\t83036 1556\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2353_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2353_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2353_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2353_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2353_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2353_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2353_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2353_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       108,441 reads  @     94.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 10.29 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,441\n",
      "  Read 1 with adapter:                 108,409 (100.0%)\n",
      "  Read 2 with adapter:                 107,829 (99.4%)\n",
      "Pairs written (passing filters):       108,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,974,765 bp\n",
      "  Read 1:    31,988,476 bp\n",
      "  Read 2:    31,986,289 bp\n",
      "Total written (filtered):     59,758,455 bp (93.4%)\n",
      "  Read 1:    29,820,613 bp\n",
      "  Read 2:    29,937,842 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 108409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t550\t0.0\t1\t0 543 7\n",
      "20\t107632\t0.0\t2\t105939 1674 19\n",
      "21\t145\t0.0\t2\t0 140 5\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 107829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t476\t0.0\t1\t0 476\n",
      "19\t107181\t0.0\t1\t104819 2362\n",
      "20\t172\t0.0\t1\t0 172\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2354_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2354_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2354_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2354_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2354_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2354_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2354_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2354_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        95,343 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 7.55 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,343\n",
      "  Read 1 with adapter:                  95,319 (100.0%)\n",
      "  Read 2 with adapter:                  94,891 (99.5%)\n",
      "Pairs written (passing filters):        95,343 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,248,088 bp\n",
      "  Read 1:    28,125,714 bp\n",
      "  Read 2:    28,122,374 bp\n",
      "Total written (filtered):     52,539,302 bp (93.4%)\n",
      "  Read 1:    26,219,613 bp\n",
      "  Read 2:    26,319,689 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 95319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t537\t0.0\t1\t0 523 14\n",
      "20\t94525\t0.0\t2\t92932 1573 20\n",
      "21\t172\t0.0\t2\t0 170 2\n",
      "22\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94891 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t429\t0.0\t1\t0 429\n",
      "19\t94277\t0.0\t1\t92109 2168\n",
      "20\t185\t0.0\t1\t0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2355_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2355_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2355_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2355_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2355_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2355_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2355_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2355_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       101,995 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 8.82 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,995\n",
      "  Read 1 with adapter:                 101,970 (100.0%)\n",
      "  Read 2 with adapter:                 101,500 (99.5%)\n",
      "Pairs written (passing filters):       101,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,173,353 bp\n",
      "  Read 1:    30,088,317 bp\n",
      "  Read 2:    30,085,036 bp\n",
      "Total written (filtered):     56,206,073 bp (93.4%)\n",
      "  Read 1:    28,049,275 bp\n",
      "  Read 2:    28,156,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 101970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t596\t0.0\t1\t0 588 8\n",
      "20\t101110\t0.0\t2\t99391 1701 18\n",
      "21\t170\t0.0\t2\t0 168 2\n",
      "22\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 101500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t462\t0.0\t1\t0 462\n",
      "19\t100838\t0.0\t1\t98681 2157\n",
      "20\t200\t0.0\t1\t0 200\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2356_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2356_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2356_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2356_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2356_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2356_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2356_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2356_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:14       139,323 reads  @    101.7 µs/read;   0.59 M reads/minute\n",
      "Finished in 14.18 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            139,323\n",
      "  Read 1 with adapter:                 139,268 (100.0%)\n",
      "  Read 2 with adapter:                 138,681 (99.5%)\n",
      "Pairs written (passing filters):       139,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,197,142 bp\n",
      "  Read 1:    41,098,591 bp\n",
      "  Read 2:    41,098,551 bp\n",
      "Total written (filtered):     76,777,951 bp (93.4%)\n",
      "  Read 1:    38,313,388 bp\n",
      "  Read 2:    38,464,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 139268 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t968\t0.0\t1\t0 940 28\n",
      "20\t137660\t0.0\t2\t134393 3213 54\n",
      "21\t257\t0.0\t2\t0 249 8\n",
      "22\t330\t0.0\t2\t0 0 330\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 138681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1317\t0.0\t1\t0 1317\n",
      "19\t136998\t0.0\t1\t133422 3576\n",
      "20\t366\t0.0\t1\t0 366\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2357_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2357_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2357_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2357_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2357_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2357_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2357_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2357_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:16       167,571 reads  @     98.3 µs/read;   0.61 M reads/minute\n",
      "Finished in 16.48 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,571\n",
      "  Read 1 with adapter:                 167,521 (100.0%)\n",
      "  Read 2 with adapter:                 166,746 (99.5%)\n",
      "Pairs written (passing filters):       167,566 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,856,710 bp\n",
      "  Read 1:    49,433,140 bp\n",
      "  Read 2:    49,423,570 bp\n",
      "Total written (filtered):     92,337,774 bp (93.4%)\n",
      "  Read 1:    46,081,864 bp\n",
      "  Read 2:    46,255,910 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 167521 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t919\t0.0\t1\t0 907 12\n",
      "20\t166185\t0.0\t2\t163410 2733 42\n",
      "21\t266\t0.0\t2\t0 261 5\n",
      "22\t109\t0.0\t2\t0 0 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 166746 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t832\t0.0\t1\t0 832\n",
      "19\t165596\t0.0\t1\t161083 4513\n",
      "20\t318\t0.0\t1\t0 318\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2358_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2358_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2358_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2358_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2358_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2358_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2358_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2358_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       117,951 reads  @     86.9 µs/read;   0.69 M reads/minute\n",
      "Finished in 10.25 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,951\n",
      "  Read 1 with adapter:                 117,904 (100.0%)\n",
      "  Read 2 with adapter:                 116,881 (99.1%)\n",
      "Pairs written (passing filters):       117,951 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,534,712 bp\n",
      "  Read 1:    34,795,403 bp\n",
      "  Read 2:    34,739,309 bp\n",
      "Total written (filtered):     64,956,572 bp (93.4%)\n",
      "  Read 1:    32,437,690 bp\n",
      "  Read 2:    32,518,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 117904 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t709\t0.0\t1\t0 700 9\n",
      "20\t116871\t0.0\t2\t114910 1936 25\n",
      "21\t202\t0.0\t2\t0 200 2\n",
      "22\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 116881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t530\t0.0\t1\t0 530\n",
      "19\t116133\t0.0\t1\t112952 3181\n",
      "20\t218\t0.0\t1\t0 218\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2359_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2359_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2359_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2359_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2359_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2359_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2359_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2359_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[      8=----] 00:00:15       134,908 reads  @    112.7 µs/read;   0.53 M reads/minute\n",
      "Finished in 15.21 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,908\n",
      "  Read 1 with adapter:                 134,773 (99.9%)\n",
      "  Read 2 with adapter:                 133,713 (99.1%)\n",
      "Pairs written (passing filters):       134,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,518,374 bp\n",
      "  Read 1:    39,794,664 bp\n",
      "  Read 2:    39,723,710 bp\n",
      "Total written (filtered):     74,283,352 bp (93.4%)\n",
      "  Read 1:    37,099,565 bp\n",
      "  Read 2:    37,183,787 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 134773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t918\t0.0\t1\t0 897 21\n",
      "20\t133365\t0.0\t2\t129986 3335 44\n",
      "21\t247\t0.0\t2\t0 240 7\n",
      "22\t199\t0.0\t2\t0 0 199\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 133713 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t925\t0.0\t1\t0 925\n",
      "19\t132487\t0.0\t1\t128996 3491\n",
      "20\t301\t0.0\t1\t0 301\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2360_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2360_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2360_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2360_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2360_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2360_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2360_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2360_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:13       112,095 reads  @    121.4 µs/read;   0.49 M reads/minute\n",
      "Finished in 13.61 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,095\n",
      "  Read 1 with adapter:                 112,056 (100.0%)\n",
      "  Read 2 with adapter:                 109,814 (98.0%)\n",
      "Pairs written (passing filters):       112,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,011,119 bp\n",
      "  Read 1:    33,066,476 bp\n",
      "  Read 2:    32,944,643 bp\n",
      "Total written (filtered):     61,684,356 bp (93.4%)\n",
      "  Read 1:    30,825,692 bp\n",
      "  Read 2:    30,858,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 112056 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t769\t0.0\t1\t0 752 17\n",
      "20\t110911\t0.0\t2\t108406 2471 34\n",
      "21\t195\t0.0\t2\t0 190 5\n",
      "22\t150\t0.0\t2\t0 0 150\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109814 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t726\t0.0\t1\t0 726\n",
      "19\t108849\t0.0\t1\t106192 2657\n",
      "20\t239\t0.0\t1\t0 239\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2361_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2361_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2361_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2361_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2361_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2361_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2361_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2361_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       130,971 reads  @     94.0 µs/read;   0.64 M reads/minute\n",
      "Finished in 12.31 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,971\n",
      "  Read 1 with adapter:                 130,946 (100.0%)\n",
      "  Read 2 with adapter:                 130,048 (99.3%)\n",
      "Pairs written (passing filters):       130,970 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,264,688 bp\n",
      "  Read 1:    38,636,405 bp\n",
      "  Read 2:    38,628,283 bp\n",
      "Total written (filtered):     72,175,256 bp (93.4%)\n",
      "  Read 1:    36,017,507 bp\n",
      "  Read 2:    36,157,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 130946 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t593\t0.0\t1\t0 579 14\n",
      "20\t130077\t0.0\t2\t127962 2098 17\n",
      "21\t192\t0.0\t2\t0 190 2\n",
      "22\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 130048 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t601\t0.0\t1\t0 601\n",
      "19\t129224\t0.0\t1\t126068 3156\n",
      "20\t223\t0.0\t1\t0 223\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2362_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2362_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2362_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2362_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2362_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2362_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2362_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2362_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[      8=----] 00:00:15       138,610 reads  @    111.4 µs/read;   0.54 M reads/minute\n",
      "Finished in 15.45 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,610\n",
      "  Read 1 with adapter:                 138,568 (100.0%)\n",
      "  Read 2 with adapter:                 138,003 (99.6%)\n",
      "Pairs written (passing filters):       138,610 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,772,369 bp\n",
      "  Read 1:    40,888,774 bp\n",
      "  Read 2:    40,883,595 bp\n",
      "Total written (filtered):     76,379,761 bp (93.4%)\n",
      "  Read 1:    38,117,485 bp\n",
      "  Read 2:    38,262,276 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 138568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t862\t0.0\t1\t0 832 30\n",
      "20\t137102\t0.0\t2\t133557 3499 46\n",
      "21\t249\t0.0\t2\t0 239 10\n",
      "22\t313\t0.0\t2\t0 0 313\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 138003 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1056\t0.0\t1\t0 1056\n",
      "19\t136629\t0.0\t1\t132934 3695\n",
      "20\t318\t0.0\t1\t0 318\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2363_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2363_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2363_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2363_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2363_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2363_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2363_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2363_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       136,929 reads  @     88.6 µs/read;   0.68 M reads/minute\n",
      "Finished in 12.14 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,929\n",
      "  Read 1 with adapter:                 136,884 (100.0%)\n",
      "  Read 2 with adapter:                 136,069 (99.4%)\n",
      "Pairs written (passing filters):       136,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,776,814 bp\n",
      "  Read 1:    40,393,443 bp\n",
      "  Read 2:    40,383,371 bp\n",
      "Total written (filtered):     75,454,558 bp (93.4%)\n",
      "  Read 1:    37,656,098 bp\n",
      "  Read 2:    37,798,460 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 136884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t709\t0.0\t1\t0 698 11\n",
      "20\t135782\t0.0\t2\t133176 2571 35\n",
      "21\t220\t0.0\t2\t0 217 3\n",
      "22\t125\t0.0\t2\t0 0 125\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 136069 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t638\t0.0\t1\t0 638\n",
      "19\t135193\t0.0\t1\t131807 3386\n",
      "20\t238\t0.0\t1\t0 238\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2364_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2364_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2364_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2364_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2364_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2364_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2364_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2364_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        88,719 reads  @     81.5 µs/read;   0.74 M reads/minute\n",
      "Finished in 7.23 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,719\n",
      "  Read 1 with adapter:                  88,686 (100.0%)\n",
      "  Read 2 with adapter:                  88,206 (99.4%)\n",
      "Pairs written (passing filters):        88,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,342,890 bp\n",
      "  Read 1:    26,172,076 bp\n",
      "  Read 2:    26,170,814 bp\n",
      "Total written (filtered):     48,893,871 bp (93.4%)\n",
      "  Read 1:    24,398,620 bp\n",
      "  Read 2:    24,495,251 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 88686 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t486\t0.0\t1\t0 479 7\n",
      "20\t87957\t0.0\t2\t86358 1574 25\n",
      "21\t152\t0.0\t2\t0 149 3\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 88206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t552\t0.0\t1\t0 552\n",
      "19\t87453\t0.0\t1\t85230 2223\n",
      "20\t201\t0.0\t1\t0 201\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2365_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2365_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2365_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2365_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2365_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2365_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2365_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2365_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       109,040 reads  @     80.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 8.80 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,040\n",
      "  Read 1 with adapter:                 108,997 (100.0%)\n",
      "  Read 2 with adapter:                 108,362 (99.4%)\n",
      "Pairs written (passing filters):       109,031 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,321,041 bp\n",
      "  Read 1:    32,166,551 bp\n",
      "  Read 2:    32,154,490 bp\n",
      "Total written (filtered):     60,080,382 bp (93.4%)\n",
      "  Read 1:    29,984,455 bp\n",
      "  Read 2:    30,095,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 108997 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t571\t0.0\t1\t0 555 16\n",
      "20\t108127\t0.0\t2\t106206 1901 20\n",
      "21\t178\t0.0\t2\t0 172 6\n",
      "22\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 108362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t505\t0.0\t1\t0 505\n",
      "19\t107667\t0.0\t1\t104329 3338\n",
      "20\t190\t0.0\t1\t0 190\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2366_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2366_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2366_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2366_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2366_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2366_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2366_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2366_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       114,167 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 9.98 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,167\n",
      "  Read 1 with adapter:                 114,100 (99.9%)\n",
      "  Read 2 with adapter:                 112,040 (98.1%)\n",
      "Pairs written (passing filters):       114,166 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,236,536 bp\n",
      "  Read 1:    33,679,029 bp\n",
      "  Read 2:    33,557,507 bp\n",
      "Total written (filtered):     62,826,130 bp (93.4%)\n",
      "  Read 1:    31,396,801 bp\n",
      "  Read 2:    31,429,329 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 114100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t637\t0.0\t1\t0 616 21\n",
      "20\t112994\t0.0\t2\t110439 2528 27\n",
      "21\t192\t0.0\t2\t0 185 7\n",
      "22\t238\t0.0\t2\t0 0 238\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 112040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t788\t0.0\t1\t0 788\n",
      "19\t111046\t0.0\t1\t106523 4523\n",
      "20\t206\t0.0\t1\t0 206\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2367_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2367_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2367_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2367_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2367_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2367_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2367_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2367_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       125,097 reads  @     87.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 10.98 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,097\n",
      "  Read 1 with adapter:                 125,060 (100.0%)\n",
      "  Read 2 with adapter:                 124,227 (99.3%)\n",
      "Pairs written (passing filters):       125,097 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,763,723 bp\n",
      "  Read 1:    36,902,822 bp\n",
      "  Read 2:    36,860,901 bp\n",
      "Total written (filtered):     68,903,001 bp (93.4%)\n",
      "  Read 1:    34,402,068 bp\n",
      "  Read 2:    34,500,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 125060 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t688\t0.0\t1\t0 671 17\n",
      "20\t124083\t0.0\t2\t121727 2318 38\n",
      "21\t196\t0.0\t2\t0 191 5\n",
      "22\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 124227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t595\t0.0\t1\t0 595\n",
      "19\t123382\t0.0\t1\t120638 2744\n",
      "20\t250\t0.0\t1\t0 250\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2368_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2368_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2368_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2368_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2368_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2368_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2368_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2368_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       115,433 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 9.53 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,433\n",
      "  Read 1 with adapter:                 115,406 (100.0%)\n",
      "  Read 2 with adapter:                 109,114 (94.5%)\n",
      "Pairs written (passing filters):       115,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,730,283 bp\n",
      "  Read 1:    34,052,690 bp\n",
      "  Read 2:    33,677,593 bp\n",
      "Total written (filtered):     63,349,629 bp (93.5%)\n",
      "  Read 1:    31,744,890 bp\n",
      "  Read 2:    31,604,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 115406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t593\t0.0\t1\t0 582 11\n",
      "20\t114522\t0.0\t2\t112443 2052 27\n",
      "21\t197\t0.0\t2\t0 193 4\n",
      "22\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 109114 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t500\t0.0\t1\t0 500\n",
      "19\t108426\t0.0\t1\t106021 2405\n",
      "20\t188\t0.0\t1\t0 188\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2369_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2369_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2369_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2369_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2369_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2369_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2369_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2369_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11       132,145 reads  @     88.4 µs/read;   0.68 M reads/minute\n",
      "Finished in 11.69 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,145\n",
      "  Read 1 with adapter:                 132,100 (100.0%)\n",
      "  Read 2 with adapter:                 131,480 (99.5%)\n",
      "Pairs written (passing filters):       132,145 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,958,791 bp\n",
      "  Read 1:    38,979,893 bp\n",
      "  Read 2:    38,978,898 bp\n",
      "Total written (filtered):     72,819,413 bp (93.4%)\n",
      "  Read 1:    36,338,290 bp\n",
      "  Read 2:    36,481,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 132100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t722\t0.0\t1\t0 715 7\n",
      "20\t131091\t0.0\t2\t128980 2092 19\n",
      "21\t201\t0.0\t2\t0 199 2\n",
      "22\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 131480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t557\t0.0\t1\t0 557\n",
      "19\t130711\t0.0\t1\t128005 2706\n",
      "20\t212\t0.0\t1\t0 212\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2370_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2370_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2370_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2370_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2370_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2370_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2370_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2370_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       134,048 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 10.89 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,048\n",
      "  Read 1 with adapter:                 134,013 (100.0%)\n",
      "  Read 2 with adapter:                 133,356 (99.5%)\n",
      "Pairs written (passing filters):       134,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,082,909 bp\n",
      "  Read 1:    39,543,671 bp\n",
      "  Read 2:    39,539,238 bp\n",
      "Total written (filtered):     73,869,734 bp (93.4%)\n",
      "  Read 1:    36,863,912 bp\n",
      "  Read 2:    37,005,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 134013 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t820\t0.0\t1\t0 804 16\n",
      "20\t132865\t0.0\t2\t130650 2188 27\n",
      "21\t225\t0.0\t2\t0 220 5\n",
      "22\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 133356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t611\t0.0\t1\t0 611\n",
      "19\t132482\t0.0\t1\t129710 2772\n",
      "20\t263\t0.0\t1\t0 263\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2371_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2371_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2371_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2371_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2371_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2371_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2371_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2371_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       148,856 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 12.59 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,856\n",
      "  Read 1 with adapter:                 148,809 (100.0%)\n",
      "  Read 2 with adapter:                 148,342 (99.7%)\n",
      "Pairs written (passing filters):       148,856 (100.0%)\n",
      "\n",
      "Total basepairs processed:    87,818,924 bp\n",
      "  Read 1:    43,912,007 bp\n",
      "  Read 2:    43,906,917 bp\n",
      "Total written (filtered):     82,025,223 bp (93.4%)\n",
      "  Read 1:    40,936,403 bp\n",
      "  Read 2:    41,088,820 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 148809 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t906\t0.0\t1\t0 887 19\n",
      "20\t147561\t0.0\t2\t144965 2555 41\n",
      "21\t226\t0.0\t2\t0 220 6\n",
      "22\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 148342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t710\t0.0\t1\t0 710\n",
      "19\t147323\t0.0\t1\t144584 2739\n",
      "20\t309\t0.0\t1\t0 309\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2372_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2372_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2372_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2372_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2372_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2372_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2372_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2372_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        94,313 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 6.90 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,313\n",
      "  Read 1 with adapter:                  94,293 (100.0%)\n",
      "  Read 2 with adapter:                  93,961 (99.6%)\n",
      "Pairs written (passing filters):        94,313 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,643,490 bp\n",
      "  Read 1:    27,822,065 bp\n",
      "  Read 2:    27,821,425 bp\n",
      "Total written (filtered):     51,973,159 bp (93.4%)\n",
      "  Read 1:    25,936,596 bp\n",
      "  Read 2:    26,036,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 94293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t618\t0.0\t1\t0 604 14\n",
      "20\t93439\t0.0\t2\t91898 1525 16\n",
      "21\t157\t0.0\t2\t0 155 2\n",
      "22\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 93961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t587\t0.0\t1\t0 587\n",
      "19\t93184\t0.0\t1\t91361 1823\n",
      "20\t190\t0.0\t1\t0 190\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2373_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2373_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2373_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2373_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2373_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2373_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2373_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2373_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11       146,880 reads  @     78.8 µs/read;   0.76 M reads/minute\n",
      "Finished in 11.57 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            146,880\n",
      "  Read 1 with adapter:                 146,845 (100.0%)\n",
      "  Read 2 with adapter:                 146,333 (99.6%)\n",
      "Pairs written (passing filters):       146,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,653,413 bp\n",
      "  Read 1:    43,329,341 bp\n",
      "  Read 2:    43,324,072 bp\n",
      "Total written (filtered):     80,935,876 bp (93.4%)\n",
      "  Read 1:    40,391,603 bp\n",
      "  Read 2:    40,544,273 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 146845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t866\t0.0\t1\t0 846 20\n",
      "20\t145627\t0.0\t2\t143097 2497 33\n",
      "21\t251\t0.0\t2\t0 246 5\n",
      "22\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 146333 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t802\t0.0\t1\t0 802\n",
      "19\t145257\t0.0\t1\t142363 2894\n",
      "20\t274\t0.0\t1\t0 274\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2374_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2374_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2374_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2374_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2374_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2374_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2374_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2374_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       105,441 reads  @     79.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 8.38 s (79 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,441\n",
      "  Read 1 with adapter:                 105,402 (100.0%)\n",
      "  Read 2 with adapter:                 104,914 (99.5%)\n",
      "Pairs written (passing filters):       105,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,190,773 bp\n",
      "  Read 1:    31,105,060 bp\n",
      "  Read 2:    31,085,713 bp\n",
      "Total written (filtered):     58,089,737 bp (93.4%)\n",
      "  Read 1:    28,997,141 bp\n",
      "  Read 2:    29,092,596 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 105402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t596\t0.0\t1\t0 586 10\n",
      "20\t104577\t0.0\t2\t102675 1865 37\n",
      "21\t158\t0.0\t2\t0 158\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 104914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t434\t0.0\t1\t0 434\n",
      "19\t104295\t0.0\t1\t102213 2082\n",
      "20\t185\t0.0\t1\t0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2375_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2375_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2375_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2375_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2375_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2375_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2375_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2375_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       120,229 reads  @     87.3 µs/read;   0.69 M reads/minute\n",
      "Finished in 10.50 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,229\n",
      "  Read 1 with adapter:                 120,190 (100.0%)\n",
      "  Read 2 with adapter:                 119,710 (99.6%)\n",
      "Pairs written (passing filters):       120,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,903,980 bp\n",
      "  Read 1:    35,466,250 bp\n",
      "  Read 2:    35,437,730 bp\n",
      "Total written (filtered):     66,226,518 bp (93.4%)\n",
      "  Read 1:    33,062,959 bp\n",
      "  Read 2:    33,163,559 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 120190 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t744\t0.0\t1\t0 737 7\n",
      "20\t119197\t0.0\t2\t116938 2224 35\n",
      "21\t183\t0.0\t2\t0 180 3\n",
      "22\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 119710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t567\t0.0\t1\t0 567\n",
      "19\t118895\t0.0\t1\t116792 2103\n",
      "20\t248\t0.0\t1\t0 248\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2376_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2376_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2376_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2376_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2376_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2376_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2376_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2376_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       109,434 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 8.32 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,434\n",
      "  Read 1 with adapter:                 109,404 (100.0%)\n",
      "  Read 2 with adapter:                 105,462 (96.4%)\n",
      "Pairs written (passing filters):       109,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    64,323,206 bp\n",
      "  Read 1:    32,282,047 bp\n",
      "  Read 2:    32,041,159 bp\n",
      "Total written (filtered):     60,132,128 bp (93.5%)\n",
      "  Read 1:    30,094,441 bp\n",
      "  Read 2:    30,037,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 109404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t663\t0.0\t1\t0 651 12\n",
      "20\t108516\t0.0\t2\t106518 1966 32\n",
      "21\t149\t0.0\t2\t0 145 4\n",
      "22\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 105462 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t476\t0.0\t1\t0 476\n",
      "19\t104816\t0.0\t1\t103262 1554\n",
      "20\t170\t0.0\t1\t0 170\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2377_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2377_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2377_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2377_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2377_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2377_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2377_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2377_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11       138,352 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 11.11 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,352\n",
      "  Read 1 with adapter:                 138,309 (100.0%)\n",
      "  Read 2 with adapter:                 137,602 (99.5%)\n",
      "Pairs written (passing filters):       138,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,621,260 bp\n",
      "  Read 1:    40,811,255 bp\n",
      "  Read 2:    40,810,005 bp\n",
      "Total written (filtered):     76,241,422 bp (93.4%)\n",
      "  Read 1:    38,045,475 bp\n",
      "  Read 2:    38,195,947 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 138309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t667\t0.0\t1\t0 651 16\n",
      "20\t137351\t0.0\t2\t135217 2108 26\n",
      "21\t199\t0.0\t2\t0 195 4\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 137602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t629\t0.0\t1\t0 629\n",
      "19\t136724\t0.0\t1\t133831 2893\n",
      "20\t249\t0.0\t1\t0 249\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2378_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2378_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2378_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2378_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2378_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2378_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2378_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2378_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       118,427 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 9.14 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,427\n",
      "  Read 1 with adapter:                 118,365 (99.9%)\n",
      "  Read 2 with adapter:                 117,906 (99.6%)\n",
      "Pairs written (passing filters):       118,427 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,864,815 bp\n",
      "  Read 1:    34,930,718 bp\n",
      "  Read 2:    34,934,097 bp\n",
      "Total written (filtered):     65,257,950 bp (93.4%)\n",
      "  Read 1:    32,563,747 bp\n",
      "  Read 2:    32,694,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 118365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t612\t0.0\t1\t0 601 11\n",
      "20\t117463\t0.0\t2\t115395 2049 19\n",
      "21\t181\t0.0\t2\t0 178 3\n",
      "22\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 117906 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t564\t0.0\t1\t0 564\n",
      "19\t117098\t0.0\t1\t114569 2529\n",
      "20\t244\t0.0\t1\t0 244\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2379_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2379_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2379_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2379_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2379_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2379_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2379_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2379_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       101,154 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 9.70 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,154\n",
      "  Read 1 with adapter:                 101,077 (99.9%)\n",
      "  Read 2 with adapter:                 100,401 (99.3%)\n",
      "Pairs written (passing filters):       101,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,659,986 bp\n",
      "  Read 1:    29,832,677 bp\n",
      "  Read 2:    29,827,309 bp\n",
      "Total written (filtered):     55,731,443 bp (93.4%)\n",
      "  Read 1:    27,811,312 bp\n",
      "  Read 2:    27,920,131 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 101077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t653\t0.0\t1\t0 639 14\n",
      "20\t100051\t0.0\t2\t97619 2390 42\n",
      "21\t148\t0.0\t2\t0 144 4\n",
      "22\t195\t0.0\t2\t0 0 195\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 100401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t722\t0.0\t1\t0 722\n",
      "19\t99398\t0.0\t1\t96375 3023\n",
      "20\t281\t0.0\t1\t0 281\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2380_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2380_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2380_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2380_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2380_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2380_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2380_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2380_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        94,877 reads  @     87.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 8.34 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,877\n",
      "  Read 1 with adapter:                  94,841 (100.0%)\n",
      "  Read 2 with adapter:                  94,577 (99.7%)\n",
      "Pairs written (passing filters):        94,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,972,949 bp\n",
      "  Read 1:    27,985,892 bp\n",
      "  Read 2:    27,987,057 bp\n",
      "Total written (filtered):     52,279,992 bp (93.4%)\n",
      "  Read 1:    26,089,251 bp\n",
      "  Read 2:    26,190,741 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 94841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t655\t0.0\t1\t0 637 18\n",
      "20\t93821\t0.0\t2\t91747 2052 22\n",
      "21\t166\t0.0\t2\t0 163 3\n",
      "22\t177\t0.0\t2\t0 0 177\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94577 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t877\t0.0\t1\t0 877\n",
      "19\t93470\t0.0\t1\t91522 1948\n",
      "20\t230\t0.0\t1\t0 230\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2381_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2381_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2381_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2381_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2381_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2381_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2381_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2381_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       120,056 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 9.77 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            120,056\n",
      "  Read 1 with adapter:                 120,039 (100.0%)\n",
      "  Read 2 with adapter:                 119,550 (99.6%)\n",
      "Pairs written (passing filters):       120,053 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,825,432 bp\n",
      "  Read 1:    35,415,573 bp\n",
      "  Read 2:    35,409,859 bp\n",
      "Total written (filtered):     66,153,143 bp (93.4%)\n",
      "  Read 1:    33,014,266 bp\n",
      "  Read 2:    33,138,877 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 120039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t635\t0.0\t1\t0 624 11\n",
      "20\t119094\t0.0\t2\t116933 2126 35\n",
      "21\t167\t0.0\t2\t0 159 8\n",
      "22\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 119550 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t711\t0.0\t1\t0 711\n",
      "19\t118596\t0.0\t1\t115396 3200\n",
      "20\t243\t0.0\t1\t0 243\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2382_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2382_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2382_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2382_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2382_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2382_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2382_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2382_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       123,561 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 10.81 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            123,561\n",
      "  Read 1 with adapter:                 123,505 (100.0%)\n",
      "  Read 2 with adapter:                 122,500 (99.1%)\n",
      "Pairs written (passing filters):       123,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    72,858,703 bp\n",
      "  Read 1:    36,448,869 bp\n",
      "  Read 2:    36,409,834 bp\n",
      "Total written (filtered):     68,061,743 bp (93.4%)\n",
      "  Read 1:    33,978,882 bp\n",
      "  Read 2:    34,082,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 123505 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t51\t0.0\t1\t0 0 51\n",
      "19\t759\t0.0\t1\t0 739 20\n",
      "20\t122202\t0.0\t2\t119586 2565 51\n",
      "21\t238\t0.0\t2\t0 233 5\n",
      "22\t255\t0.0\t2\t0 0 255\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 122500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t791\t0.0\t1\t0 791\n",
      "19\t121445\t0.0\t1\t117808 3637\n",
      "20\t264\t0.0\t1\t0 264\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2383_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2383_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2383_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2383_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2383_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2383_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2383_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2383_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        99,614 reads  @     81.5 µs/read;   0.74 M reads/minute\n",
      "Finished in 8.12 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,614\n",
      "  Read 1 with adapter:                  99,522 (99.9%)\n",
      "  Read 2 with adapter:                  98,896 (99.3%)\n",
      "Pairs written (passing filters):        99,613 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,725,306 bp\n",
      "  Read 1:    29,384,466 bp\n",
      "  Read 2:    29,340,840 bp\n",
      "Total written (filtered):     54,856,133 bp (93.4%)\n",
      "  Read 1:    27,394,061 bp\n",
      "  Read 2:    27,462,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 99522 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t561\t0.0\t1\t0 545 16\n",
      "20\t98717\t0.0\t2\t96666 2017 34\n",
      "21\t153\t0.0\t2\t0 148 5\n",
      "22\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 98896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t474\t0.0\t1\t0 474\n",
      "19\t98204\t0.0\t1\t95919 2285\n",
      "20\t218\t0.0\t1\t0 218\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2384_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2384_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2384_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2384_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2384_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2384_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2384_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2384_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[      8=----] 00:00:13       133,703 reads  @    104.0 µs/read;   0.58 M reads/minute\n",
      "Finished in 13.91 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            133,703\n",
      "  Read 1 with adapter:                 133,620 (99.9%)\n",
      "  Read 2 with adapter:                 131,863 (98.6%)\n",
      "Pairs written (passing filters):       133,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,777,982 bp\n",
      "  Read 1:    39,433,232 bp\n",
      "  Read 2:    39,344,750 bp\n",
      "Total written (filtered):     73,601,354 bp (93.4%)\n",
      "  Read 1:    36,761,235 bp\n",
      "  Read 2:    36,840,119 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 133620 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t66\t0.0\t1\t0 0 66\n",
      "19\t956\t0.0\t1\t0 929 27\n",
      "20\t132128\t0.0\t2\t128765 3313 50\n",
      "21\t255\t0.0\t2\t0 251 4\n",
      "22\t215\t0.0\t2\t0 0 215\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 131863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t988\t0.0\t1\t0 988\n",
      "19\t130653\t0.0\t1\t127589 3064\n",
      "20\t222\t0.0\t1\t0 222\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2393_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2393_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2393_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2393_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2393_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2393_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2393_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2393_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:15       185,787 reads  @     85.9 µs/read;   0.70 M reads/minute\n",
      "Finished in 15.97 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            185,787\n",
      "  Read 1 with adapter:                 185,326 (99.8%)\n",
      "  Read 2 with adapter:                 184,561 (99.3%)\n",
      "Pairs written (passing filters):       185,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,604,574 bp\n",
      "  Read 1:    54,805,876 bp\n",
      "  Read 2:    54,798,698 bp\n",
      "Total written (filtered):    102,392,170 bp (93.4%)\n",
      "  Read 1:    51,099,612 bp\n",
      "  Read 2:    51,292,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 185326 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t41\t0.0\t1\t0 0 41\n",
      "19\t1033\t0.0\t1\t0 1017 16\n",
      "20\t183780\t0.0\t2\t180696 3052 32\n",
      "21\t360\t0.0\t2\t0 349 11\n",
      "22\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 184561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t847\t0.0\t1\t0 847\n",
      "19\t183386\t0.0\t1\t179546 3840\n",
      "20\t328\t0.0\t1\t0 328\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2394_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2394_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2394_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2394_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2394_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2394_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2394_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2394_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       133,676 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 10.31 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            133,676\n",
      "  Read 1 with adapter:                 133,533 (99.9%)\n",
      "  Read 2 with adapter:                 132,942 (99.5%)\n",
      "Pairs written (passing filters):       133,676 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,863,093 bp\n",
      "  Read 1:    39,434,209 bp\n",
      "  Read 2:    39,428,884 bp\n",
      "Total written (filtered):     73,667,301 bp (93.4%)\n",
      "  Read 1:    36,763,992 bp\n",
      "  Read 2:    36,903,309 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 133533 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t862\t0.0\t1\t0 841 21\n",
      "20\t132228\t0.0\t2\t129813 2382 33\n",
      "21\t291\t0.0\t2\t0 279 12\n",
      "22\t108\t0.0\t2\t0 0 108\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 132942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t564\t0.0\t1\t0 564\n",
      "19\t132137\t0.0\t1\t129405 2732\n",
      "20\t241\t0.0\t1\t0 241\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2395_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2395_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2395_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2395_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2395_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2395_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2395_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2395_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11       134,451 reads  @     84.2 µs/read;   0.71 M reads/minute\n",
      "Finished in 11.33 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,451\n",
      "  Read 1 with adapter:                 134,275 (99.9%)\n",
      "  Read 2 with adapter:                 133,800 (99.5%)\n",
      "Pairs written (passing filters):       134,451 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,319,883 bp\n",
      "  Read 1:    39,661,741 bp\n",
      "  Read 2:    39,658,142 bp\n",
      "Total written (filtered):     74,093,024 bp (93.4%)\n",
      "  Read 1:    36,976,702 bp\n",
      "  Read 2:    37,116,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 134275 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t40\t0.0\t1\t0 0 40\n",
      "19\t874\t0.0\t1\t0 854 20\n",
      "20\t132967\t0.0\t2\t130412 2518 37\n",
      "21\t295\t0.0\t2\t0 287 8\n",
      "22\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 133800 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t645\t0.0\t1\t0 645\n",
      "19\t132890\t0.0\t1\t130230 2660\n",
      "20\t265\t0.0\t1\t0 265\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2396_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2396_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2396_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2396_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2396_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2396_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2396_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2396_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       130,748 reads  @     95.9 µs/read;   0.63 M reads/minute\n",
      "Finished in 12.54 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,748\n",
      "  Read 1 with adapter:                 130,641 (99.9%)\n",
      "  Read 2 with adapter:                 130,243 (99.6%)\n",
      "Pairs written (passing filters):       130,748 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,133,650 bp\n",
      "  Read 1:    38,569,807 bp\n",
      "  Read 2:    38,563,843 bp\n",
      "Total written (filtered):     72,047,124 bp (93.4%)\n",
      "  Read 1:    35,957,455 bp\n",
      "  Read 2:    36,089,669 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 130641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t53\t0.0\t1\t0 0 53\n",
      "19\t838\t0.0\t1\t0 824 14\n",
      "20\t129377\t0.0\t2\t126979 2355 43\n",
      "21\t270\t0.0\t2\t0 265 5\n",
      "22\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 130243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t770\t0.0\t1\t0 770\n",
      "19\t129146\t0.0\t1\t126470 2676\n",
      "20\t327\t0.0\t1\t0 327\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2397_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2397_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2397_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2397_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2397_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2397_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2397_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2397_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12       145,501 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 12.80 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            145,501\n",
      "  Read 1 with adapter:                 145,441 (100.0%)\n",
      "  Read 2 with adapter:                 144,820 (99.5%)\n",
      "Pairs written (passing filters):       145,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    85,839,932 bp\n",
      "  Read 1:    42,922,099 bp\n",
      "  Read 2:    42,917,833 bp\n",
      "Total written (filtered):     80,179,688 bp (93.4%)\n",
      "  Read 1:    40,012,957 bp\n",
      "  Read 2:    40,166,731 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 145441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t881\t0.0\t1\t0 865 16\n",
      "20\t144137\t0.0\t2\t141402 2701 34\n",
      "21\t268\t0.0\t2\t0 261 7\n",
      "22\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 144820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t718\t0.0\t1\t0 718\n",
      "19\t143862\t0.0\t1\t140299 3563\n",
      "20\t240\t0.0\t1\t0 240\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2398_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2398_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2398_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2398_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2398_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2398_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2398_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2398_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       119,561 reads  @     87.1 µs/read;   0.69 M reads/minute\n",
      "Finished in 10.42 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            119,561\n",
      "  Read 1 with adapter:                 119,491 (99.9%)\n",
      "  Read 2 with adapter:                 118,573 (99.2%)\n",
      "Pairs written (passing filters):       119,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,449,619 bp\n",
      "  Read 1:    35,270,229 bp\n",
      "  Read 2:    35,179,390 bp\n",
      "Total written (filtered):     65,807,551 bp (93.4%)\n",
      "  Read 1:    32,880,729 bp\n",
      "  Read 2:    32,926,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 119491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t697\t0.0\t1\t0 689 8\n",
      "20\t118392\t0.0\t2\t116192 2166 34\n",
      "21\t235\t0.0\t2\t0 230 5\n",
      "22\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 118573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t534\t0.0\t1\t0 534\n",
      "19\t117824\t0.0\t1\t115195 2629\n",
      "20\t215\t0.0\t1\t0 215\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2399_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2399_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2399_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2399_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2399_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2399_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2399_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2399_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11       125,038 reads  @     91.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 11.45 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,038\n",
      "  Read 1 with adapter:                 124,826 (99.8%)\n",
      "  Read 2 with adapter:                 124,318 (99.4%)\n",
      "Pairs written (passing filters):       125,037 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,740,955 bp\n",
      "  Read 1:    36,885,372 bp\n",
      "  Read 2:    36,855,583 bp\n",
      "Total written (filtered):     68,882,978 bp (93.4%)\n",
      "  Read 1:    34,389,077 bp\n",
      "  Read 2:    34,493,901 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 124826 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t822\t0.0\t1\t0 799 23\n",
      "20\t123639\t0.0\t2\t121154 2451 34\n",
      "21\t252\t0.0\t2\t0 248 4\n",
      "22\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 124318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t591\t0.0\t1\t0 591\n",
      "19\t123496\t0.0\t1\t121184 2312\n",
      "20\t231\t0.0\t1\t0 231\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2400_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2400_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2400_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2400_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2400_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2400_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2400_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2400_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       110,643 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 9.55 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,643\n",
      "  Read 1 with adapter:                 110,560 (99.9%)\n",
      "  Read 2 with adapter:                 108,126 (97.7%)\n",
      "Pairs written (passing filters):       110,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,131,311 bp\n",
      "  Read 1:    32,638,242 bp\n",
      "  Read 2:    32,493,069 bp\n",
      "Total written (filtered):     60,866,358 bp (93.5%)\n",
      "  Read 1:    30,427,365 bp\n",
      "  Read 2:    30,438,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 110560 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t651\t0.0\t1\t0 632 19\n",
      "20\t109547\t0.0\t2\t107242 2262 43\n",
      "21\t252\t0.0\t2\t0 248 4\n",
      "22\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 108126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t501\t0.0\t1\t0 501\n",
      "19\t107442\t0.0\t1\t105782 1660\n",
      "20\t183\t0.0\t1\t0 183\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2401_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2401_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2401_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2401_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2401_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2401_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2401_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2401_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:14       148,138 reads  @     98.5 µs/read;   0.61 M reads/minute\n",
      "Finished in 14.60 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,138\n",
      "  Read 1 with adapter:                 148,074 (100.0%)\n",
      "  Read 2 with adapter:                 147,147 (99.3%)\n",
      "Pairs written (passing filters):       148,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    87,394,532 bp\n",
      "  Read 1:    43,698,697 bp\n",
      "  Read 2:    43,695,835 bp\n",
      "Total written (filtered):     81,638,608 bp (93.4%)\n",
      "  Read 1:    40,738,082 bp\n",
      "  Read 2:    40,900,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 148074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t1240\t0.0\t1\t0 1210 30\n",
      "20\t146447\t0.0\t2\t143938 2474 35\n",
      "21\t255\t0.0\t2\t0 251 4\n",
      "22\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 147147 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t723\t0.0\t1\t0 723\n",
      "19\t146185\t0.0\t1\t142337 3848\n",
      "20\t239\t0.0\t1\t0 239\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2402_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2402_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2402_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2402_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2402_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2402_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2402_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2402_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:10       131,060 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 10.90 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,060\n",
      "  Read 1 with adapter:                 131,017 (100.0%)\n",
      "  Read 2 with adapter:                 130,305 (99.4%)\n",
      "Pairs written (passing filters):       131,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,322,659 bp\n",
      "  Read 1:    38,661,956 bp\n",
      "  Read 2:    38,660,703 bp\n",
      "Total written (filtered):     72,227,687 bp (93.4%)\n",
      "  Read 1:    36,042,429 bp\n",
      "  Read 2:    36,185,258 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 131017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t1108\t0.0\t1\t0 1093 15\n",
      "20\t129595\t0.0\t2\t127275 2297 23\n",
      "21\t209\t0.0\t2\t0 203 6\n",
      "22\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 130305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t584\t0.0\t1\t0 584\n",
      "19\t129487\t0.0\t1\t125814 3673\n",
      "20\t234\t0.0\t1\t0 234\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2403_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2403_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2403_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2403_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2403_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2403_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2403_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2403_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:14       147,383 reads  @     99.5 µs/read;   0.60 M reads/minute\n",
      "Finished in 14.68 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            147,383\n",
      "  Read 1 with adapter:                 147,304 (99.9%)\n",
      "  Read 2 with adapter:                 146,670 (99.5%)\n",
      "Pairs written (passing filters):       147,383 (100.0%)\n",
      "\n",
      "Total basepairs processed:    86,948,717 bp\n",
      "  Read 1:    43,475,209 bp\n",
      "  Read 2:    43,473,508 bp\n",
      "Total written (filtered):     81,217,169 bp (93.4%)\n",
      "  Read 1:    40,529,986 bp\n",
      "  Read 2:    40,687,183 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 147304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t42\t0.0\t1\t0 0 42\n",
      "19\t1211\t0.0\t1\t0 1184 27\n",
      "20\t145696\t0.0\t2\t143085 2579 32\n",
      "21\t272\t0.0\t2\t0 265 7\n",
      "22\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 146670 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t694\t0.0\t1\t0 694\n",
      "19\t145687\t0.0\t1\t141868 3819\n",
      "20\t289\t0.0\t1\t0 289\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2404_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2404_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2404_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2404_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2404_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2404_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2404_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2404_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:12       107,222 reads  @    112.4 µs/read;   0.53 M reads/minute\n",
      "Finished in 12.06 s (112 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,222\n",
      "  Read 1 with adapter:                 107,145 (99.9%)\n",
      "  Read 2 with adapter:                 106,616 (99.4%)\n",
      "Pairs written (passing filters):       107,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,252,587 bp\n",
      "  Read 1:    31,625,230 bp\n",
      "  Read 2:    31,627,357 bp\n",
      "Total written (filtered):     59,085,175 bp (93.4%)\n",
      "  Read 1:    29,482,893 bp\n",
      "  Read 2:    29,602,282 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 107145 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t1092\t0.0\t1\t0 1060 32\n",
      "20\t105616\t0.0\t2\t102950 2620 46\n",
      "21\t189\t0.0\t2\t0 179 10\n",
      "22\t209\t0.0\t2\t0 0 209\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 106616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t986\t0.0\t1\t0 986\n",
      "19\t105273\t0.0\t1\t101909 3364\n",
      "20\t357\t0.0\t1\t0 357\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2405_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2405_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2405_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2405_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2405_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2405_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2405_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2405_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:12       136,745 reads  @     90.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 12.31 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,745\n",
      "  Read 1 with adapter:                 136,687 (100.0%)\n",
      "  Read 2 with adapter:                 136,004 (99.5%)\n",
      "Pairs written (passing filters):       136,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:    80,667,038 bp\n",
      "  Read 1:    40,338,349 bp\n",
      "  Read 2:    40,328,689 bp\n",
      "Total written (filtered):     75,347,302 bp (93.4%)\n",
      "  Read 1:    37,602,233 bp\n",
      "  Read 2:    37,745,069 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 136687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t36\t0.0\t1\t0 0 36\n",
      "19\t1237\t0.0\t1\t0 1209 28\n",
      "20\t135097\t0.0\t2\t132582 2483 32\n",
      "21\t249\t0.0\t2\t0 239 10\n",
      "22\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 136004 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t700\t0.0\t1\t0 700\n",
      "19\t135060\t0.0\t1\t129701 5359\n",
      "20\t244\t0.0\t1\t0 244\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2406_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2406_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2406_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2406_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2406_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2406_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2406_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2406_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       124,130 reads  @     84.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 10.43 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            124,130\n",
      "  Read 1 with adapter:                 124,068 (100.0%)\n",
      "  Read 2 with adapter:                 122,553 (98.7%)\n",
      "Pairs written (passing filters):       124,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,167,379 bp\n",
      "  Read 1:    36,616,551 bp\n",
      "  Read 2:    36,550,828 bp\n",
      "Total written (filtered):     68,358,632 bp (93.4%)\n",
      "  Read 1:    34,135,987 bp\n",
      "  Read 2:    34,222,645 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 124068 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t1119\t0.0\t1\t0 1091 28\n",
      "20\t122618\t0.0\t2\t120441 2140 37\n",
      "21\t211\t0.0\t2\t0 207 4\n",
      "22\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 122553 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t537\t0.0\t1\t0 537\n",
      "19\t121803\t0.0\t1\t116434 5369\n",
      "20\t213\t0.0\t1\t0 213\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2407_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2407_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2407_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2407_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2407_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2407_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2407_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2407_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[       8<---] 00:00:17       155,815 reads  @    110.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 17.18 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            155,815\n",
      "  Read 1 with adapter:                 155,704 (99.9%)\n",
      "  Read 2 with adapter:                 154,674 (99.3%)\n",
      "Pairs written (passing filters):       155,814 (100.0%)\n",
      "\n",
      "Total basepairs processed:    91,841,961 bp\n",
      "  Read 1:    45,954,263 bp\n",
      "  Read 2:    45,887,698 bp\n",
      "Total written (filtered):     85,790,537 bp (93.4%)\n",
      "  Read 1:    42,840,966 bp\n",
      "  Read 2:    42,949,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 155704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t51\t0.0\t1\t0 0 51\n",
      "19\t1580\t0.0\t1\t0 1539 41\n",
      "20\t153610\t0.0\t2\t149717 3835 58\n",
      "21\t302\t0.0\t2\t0 294 8\n",
      "22\t161\t0.0\t2\t0 0 161\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 154674 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1030\t0.0\t1\t0 1030\n",
      "19\t153293\t0.0\t1\t149076 4217\n",
      "20\t351\t0.0\t1\t0 351\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2408_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2408_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2408_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2408_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2408_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2408_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2408_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2408_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:14       138,061 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 14.86 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,061\n",
      "  Read 1 with adapter:                 137,993 (100.0%)\n",
      "  Read 2 with adapter:                 133,864 (97.0%)\n",
      "Pairs written (passing filters):       138,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,201,722 bp\n",
      "  Read 1:    40,723,796 bp\n",
      "  Read 2:    40,477,926 bp\n",
      "Total written (filtered):     75,899,900 bp (93.5%)\n",
      "  Read 1:    37,964,834 bp\n",
      "  Read 2:    37,935,066 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 137993 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t1423\t0.0\t1\t0 1384 39\n",
      "20\t136075\t0.0\t2\t132707 3309 59\n",
      "21\t289\t0.0\t2\t0 280 9\n",
      "22\t162\t0.0\t2\t0 0 162\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 133864 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t801\t0.0\t1\t0 801\n",
      "19\t132818\t0.0\t1\t129599 3219\n",
      "20\t245\t0.0\t1\t0 245\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2409_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2409_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2409_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2409_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2409_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2409_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2409_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2409_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10       140,457 reads  @     76.6 µs/read;   0.78 M reads/minute\n",
      "Finished in 10.77 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            140,457\n",
      "  Read 1 with adapter:                 140,077 (99.7%)\n",
      "  Read 2 with adapter:                 139,592 (99.4%)\n",
      "Pairs written (passing filters):       140,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,856,097 bp\n",
      "  Read 1:    41,425,420 bp\n",
      "  Read 2:    41,430,677 bp\n",
      "Total written (filtered):     77,403,248 bp (93.4%)\n",
      "  Read 1:    38,624,396 bp\n",
      "  Read 2:    38,778,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 140077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t38\t0.0\t1\t0 0 38\n",
      "19\t760\t0.0\t1\t0 746 14\n",
      "20\t139010\t0.0\t2\t136708 2271 31\n",
      "21\t218\t0.0\t2\t0 212 6\n",
      "22\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 139592 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t638\t0.0\t1\t0 638\n",
      "19\t138739\t0.0\t1\t135878 2861\n",
      "20\t215\t0.0\t1\t0 215\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2410_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2410_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2410_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2410_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2410_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2410_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2410_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2410_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:13       137,846 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 13.22 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            137,846\n",
      "  Read 1 with adapter:                 137,129 (99.5%)\n",
      "  Read 2 with adapter:                 136,773 (99.2%)\n",
      "Pairs written (passing filters):       137,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,288,762 bp\n",
      "  Read 1:    40,640,643 bp\n",
      "  Read 2:    40,648,119 bp\n",
      "Total written (filtered):     75,948,263 bp (93.4%)\n",
      "  Read 1:    37,898,085 bp\n",
      "  Read 2:    38,050,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 137129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t64\t0.0\t1\t0 0 64\n",
      "19\t1225\t0.0\t1\t0 1191 34\n",
      "20\t135319\t0.0\t2\t131886 3378 55\n",
      "21\t261\t0.0\t2\t0 256 5\n",
      "22\t260\t0.0\t2\t0 0 260\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 136773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1076\t0.0\t1\t0 1076\n",
      "19\t135367\t0.0\t1\t131757 3610\n",
      "20\t330\t0.0\t1\t0 330\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2411_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2411_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2411_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2411_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2411_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2411_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2411_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2411_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       129,257 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 9.88 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,257\n",
      "  Read 1 with adapter:                 128,781 (99.6%)\n",
      "  Read 2 with adapter:                 128,345 (99.3%)\n",
      "Pairs written (passing filters):       129,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,252,569 bp\n",
      "  Read 1:    38,126,138 bp\n",
      "  Read 2:    38,126,431 bp\n",
      "Total written (filtered):     71,239,340 bp (93.4%)\n",
      "  Read 1:    35,551,087 bp\n",
      "  Read 2:    35,688,253 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 128781 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t45\t0.0\t1\t0 0 45\n",
      "19\t881\t0.0\t1\t0 866 15\n",
      "20\t127551\t0.0\t2\t125070 2440 41\n",
      "21\t206\t0.0\t2\t0 198 8\n",
      "22\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 128345 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t614\t0.0\t1\t0 614\n",
      "19\t127494\t0.0\t1\t124828 2666\n",
      "20\t237\t0.0\t1\t0 237\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2412_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2412_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2412_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2412_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2412_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2412_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2412_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2412_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        92,295 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 6.73 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,295\n",
      "  Read 1 with adapter:                  92,146 (99.8%)\n",
      "  Read 2 with adapter:                  91,889 (99.6%)\n",
      "Pairs written (passing filters):        92,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,450,095 bp\n",
      "  Read 1:    27,223,506 bp\n",
      "  Read 2:    27,226,589 bp\n",
      "Total written (filtered):     50,862,042 bp (93.4%)\n",
      "  Read 1:    25,380,995 bp\n",
      "  Read 2:    25,481,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 92146 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t578\t0.0\t1\t0 564 14\n",
      "20\t91338\t0.0\t2\t89624 1694 20\n",
      "21\t151\t0.0\t2\t0 149 2\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 91889 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t572\t0.0\t1\t0 572\n",
      "19\t91094\t0.0\t1\t89253 1841\n",
      "20\t223\t0.0\t1\t0 223\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2413_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2413_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2413_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2413_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2413_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2413_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2413_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2413_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       127,496 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 9.26 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,496\n",
      "  Read 1 with adapter:                 127,245 (99.8%)\n",
      "  Read 2 with adapter:                 127,036 (99.6%)\n",
      "Pairs written (passing filters):       127,489 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,205,535 bp\n",
      "  Read 1:    37,600,640 bp\n",
      "  Read 2:    37,604,895 bp\n",
      "Total written (filtered):     70,245,937 bp (93.4%)\n",
      "  Read 1:    35,054,316 bp\n",
      "  Read 2:    35,191,621 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 127245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t45\t0.0\t1\t0 0 45\n",
      "19\t774\t0.0\t1\t0 761 13\n",
      "20\t126143\t0.0\t2\t123902 2211 30\n",
      "21\t203\t0.0\t2\t0 196 7\n",
      "22\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 127036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t625\t0.0\t1\t0 625\n",
      "19\t126196\t0.0\t1\t123776 2420\n",
      "20\t215\t0.0\t1\t0 215\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2414_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2414_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2414_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2414_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2414_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2414_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2414_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2414_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       108,399 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 8.32 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            108,399\n",
      "  Read 1 with adapter:                 106,370 (98.1%)\n",
      "  Read 2 with adapter:                 105,283 (97.1%)\n",
      "Pairs written (passing filters):       108,399 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,908,626 bp\n",
      "  Read 1:    31,973,724 bp\n",
      "  Read 2:    31,934,902 bp\n",
      "Total written (filtered):     59,781,668 bp (93.5%)\n",
      "  Read 1:    29,846,695 bp\n",
      "  Read 2:    29,934,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 106370 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t790\t0.0\t1\t0 770 20\n",
      "20\t105214\t0.0\t2\t102887 2291 36\n",
      "21\t141\t0.0\t2\t0 137 4\n",
      "22\t182\t0.0\t2\t0 0 182\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 105283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t658\t0.0\t1\t0 658\n",
      "19\t104415\t0.0\t1\t101359 3056\n",
      "20\t210\t0.0\t1\t0 210\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2415_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2415_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2415_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2415_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2415_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2415_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2415_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2415_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       130,234 reads  @     79.9 µs/read;   0.75 M reads/minute\n",
      "Finished in 10.41 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,234\n",
      "  Read 1 with adapter:                 129,926 (99.8%)\n",
      "  Read 2 with adapter:                 129,457 (99.4%)\n",
      "Pairs written (passing filters):       130,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,797,624 bp\n",
      "  Read 1:    38,410,907 bp\n",
      "  Read 2:    38,386,717 bp\n",
      "Total written (filtered):     71,740,331 bp (93.4%)\n",
      "  Read 1:    35,812,969 bp\n",
      "  Read 2:    35,927,362 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 129926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t780\t0.0\t1\t0 768 12\n",
      "20\t128866\t0.0\t2\t126457 2382 27\n",
      "21\t190\t0.0\t2\t0 186 4\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 129457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t587\t0.0\t1\t0 587\n",
      "19\t128611\t0.0\t1\t126249 2362\n",
      "20\t259\t0.0\t1\t0 259\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2416_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2416_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2416_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2416_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2416_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2416_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2416_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2416_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       115,309 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 8.61 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            115,309\n",
      "  Read 1 with adapter:                 115,115 (99.8%)\n",
      "  Read 2 with adapter:                 111,822 (97.0%)\n",
      "Pairs written (passing filters):       115,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,832,535 bp\n",
      "  Read 1:    34,012,137 bp\n",
      "  Read 2:    33,820,398 bp\n",
      "Total written (filtered):     63,406,439 bp (93.5%)\n",
      "  Read 1:    31,710,330 bp\n",
      "  Read 2:    31,696,109 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 115115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t730\t0.0\t1\t0 720 10\n",
      "20\t114095\t0.0\t2\t112048 2019 28\n",
      "21\t187\t0.0\t2\t0 183 4\n",
      "22\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 111822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t541\t0.0\t1\t0 541\n",
      "19\t111069\t0.0\t1\t109250 1819\n",
      "20\t212\t0.0\t1\t0 212\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2417_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2417_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2417_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2417_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2417_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2417_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2417_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2417_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       112,753 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 8.78 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,753\n",
      "  Read 1 with adapter:                 112,697 (100.0%)\n",
      "  Read 2 with adapter:                 112,102 (99.4%)\n",
      "Pairs written (passing filters):       112,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,518,259 bp\n",
      "  Read 1:    33,257,628 bp\n",
      "  Read 2:    33,260,631 bp\n",
      "Total written (filtered):     62,134,732 bp (93.4%)\n",
      "  Read 1:    31,003,985 bp\n",
      "  Read 2:    31,130,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 112697 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t549\t0.0\t1\t0 541 8\n",
      "20\t111912\t0.0\t2\t110104 1786 22\n",
      "21\t168\t0.0\t2\t0 166 2\n",
      "22\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 112102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t521\t0.0\t1\t0 521\n",
      "19\t111390\t0.0\t1\t108836 2554\n",
      "20\t191\t0.0\t1\t0 191\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2418_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2418_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2418_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2418_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2418_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2418_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2418_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2418_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08       118,987 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 8.69 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,987\n",
      "  Read 1 with adapter:                 118,937 (100.0%)\n",
      "  Read 2 with adapter:                 118,334 (99.5%)\n",
      "Pairs written (passing filters):       118,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,196,018 bp\n",
      "  Read 1:    35,100,272 bp\n",
      "  Read 2:    35,095,746 bp\n",
      "Total written (filtered):     65,569,738 bp (93.4%)\n",
      "  Read 1:    32,721,963 bp\n",
      "  Read 2:    32,847,775 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 118937 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t707\t0.0\t1\t0 692 15\n",
      "20\t117940\t0.0\t2\t115845 2063 32\n",
      "21\t172\t0.0\t2\t0 167 5\n",
      "22\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 118334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t583\t0.0\t1\t0 583\n",
      "19\t117543\t0.0\t1\t114801 2742\n",
      "20\t208\t0.0\t1\t0 208\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2419_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2419_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2419_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2419_133_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2419_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2419_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2419_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2419_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:13       182,403 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 13.84 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            182,403\n",
      "  Read 1 with adapter:                 182,317 (100.0%)\n",
      "  Read 2 with adapter:                 181,622 (99.6%)\n",
      "Pairs written (passing filters):       182,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:   107,608,046 bp\n",
      "  Read 1:    53,805,160 bp\n",
      "  Read 2:    53,802,886 bp\n",
      "Total written (filtered):    100,511,934 bp (93.4%)\n",
      "  Read 1:    50,159,395 bp\n",
      "  Read 2:    50,352,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 182317 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t1055\t0.0\t1\t0 1037 18\n",
      "20\t180789\t0.0\t2\t177619 3126 44\n",
      "21\t290\t0.0\t2\t0 288 2\n",
      "22\t139\t0.0\t2\t0 0 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 181622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t809\t0.0\t1\t0 809\n",
      "19\t180475\t0.0\t1\t176659 3816\n",
      "20\t338\t0.0\t1\t0 338\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2420_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2420_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2420_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2420_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2420_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2420_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2420_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2420_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        96,815 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 6.46 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,815\n",
      "  Read 1 with adapter:                  96,778 (100.0%)\n",
      "  Read 2 with adapter:                  96,443 (99.6%)\n",
      "Pairs written (passing filters):        96,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,119,759 bp\n",
      "  Read 1:    28,559,770 bp\n",
      "  Read 2:    28,559,989 bp\n",
      "Total written (filtered):     53,352,422 bp (93.4%)\n",
      "  Read 1:    26,624,524 bp\n",
      "  Read 2:    26,727,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 96778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t525\t0.0\t1\t0 514 11\n",
      "20\t96023\t0.0\t2\t94386 1612 25\n",
      "21\t149\t0.0\t2\t0 148 1\n",
      "22\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 96443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t528\t0.0\t1\t0 528\n",
      "19\t95713\t0.0\t1\t93660 2053\n",
      "20\t202\t0.0\t1\t0 202\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2421_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2421_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2421_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2421_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2421_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2421_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2421_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2421_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       130,384 reads  @     70.6 µs/read;   0.85 M reads/minute\n",
      "Finished in 9.21 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,384\n",
      "  Read 1 with adapter:                 130,351 (100.0%)\n",
      "  Read 2 with adapter:                 129,851 (99.6%)\n",
      "Pairs written (passing filters):       130,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,917,446 bp\n",
      "  Read 1:    38,462,622 bp\n",
      "  Read 2:    38,454,824 bp\n",
      "Total written (filtered):     71,842,238 bp (93.4%)\n",
      "  Read 1:    35,854,154 bp\n",
      "  Read 2:    35,988,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 130351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t742\t0.0\t1\t0 730 12\n",
      "20\t129328\t0.0\t2\t127147 2154 27\n",
      "21\t177\t0.0\t2\t0 172 5\n",
      "22\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 129851 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t651\t0.0\t1\t0 651\n",
      "19\t128978\t0.0\t1\t125660 3318\n",
      "20\t222\t0.0\t1\t0 222\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2422_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2422_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2422_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2422_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2422_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2422_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2422_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2422_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07       118,702 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 7.94 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,702\n",
      "  Read 1 with adapter:                 118,671 (100.0%)\n",
      "  Read 2 with adapter:                 117,956 (99.4%)\n",
      "Pairs written (passing filters):       118,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    70,008,654 bp\n",
      "  Read 1:    35,016,373 bp\n",
      "  Read 2:    34,992,281 bp\n",
      "Total written (filtered):     65,394,753 bp (93.4%)\n",
      "  Read 1:    32,643,316 bp\n",
      "  Read 2:    32,751,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 118671 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t682\t0.0\t1\t0 668 14\n",
      "20\t117672\t0.0\t2\t115638 2003 31\n",
      "21\t199\t0.0\t2\t0 193 6\n",
      "22\t89\t0.0\t2\t0 0 89\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 117956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t517\t0.0\t1\t0 517\n",
      "19\t117242\t0.0\t1\t114444 2798\n",
      "20\t197\t0.0\t1\t0 197\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2423_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2423_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2423_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2423_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2423_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2423_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2423_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2423_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       125,011 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 9.97 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,011\n",
      "  Read 1 with adapter:                 124,964 (100.0%)\n",
      "  Read 2 with adapter:                 124,256 (99.4%)\n",
      "Pairs written (passing filters):       125,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,704,933 bp\n",
      "  Read 1:    36,877,343 bp\n",
      "  Read 2:    36,827,590 bp\n",
      "Total written (filtered):     68,845,034 bp (93.4%)\n",
      "  Read 1:    34,377,951 bp\n",
      "  Read 2:    34,467,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 124964 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t691\t0.0\t1\t0 670 21\n",
      "20\t123988\t0.0\t2\t121696 2264 28\n",
      "21\t201\t0.0\t2\t0 198 3\n",
      "22\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 124256 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t606\t0.0\t1\t0 606\n",
      "19\t123401\t0.0\t1\t120971 2430\n",
      "20\t249\t0.0\t1\t0 249\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2424_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2424_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2424_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2424_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2424_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2424_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2424_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2424_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08       125,646 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 8.96 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,646\n",
      "  Read 1 with adapter:                 125,605 (100.0%)\n",
      "  Read 2 with adapter:                 121,647 (96.8%)\n",
      "Pairs written (passing filters):       125,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    73,894,979 bp\n",
      "  Read 1:    37,065,116 bp\n",
      "  Read 2:    36,829,863 bp\n",
      "Total written (filtered):     69,072,313 bp (93.5%)\n",
      "  Read 1:    34,553,425 bp\n",
      "  Read 2:    34,518,888 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 125605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t675\t0.0\t1\t0 661 14\n",
      "20\t124664\t0.0\t2\t122645 1986 33\n",
      "21\t182\t0.0\t2\t0 177 5\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 121647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t500\t0.0\t1\t0 500\n",
      "19\t120965\t0.0\t1\t118924 2041\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2425_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2425_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2425_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2425_145_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2425_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2425_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2425_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2425_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10       141,068 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 10.76 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            141,068\n",
      "  Read 1 with adapter:                 140,961 (99.9%)\n",
      "  Read 2 with adapter:                 140,045 (99.3%)\n",
      "Pairs written (passing filters):       141,068 (100.0%)\n",
      "\n",
      "Total basepairs processed:    83,207,601 bp\n",
      "  Read 1:    41,599,990 bp\n",
      "  Read 2:    41,607,611 bp\n",
      "Total written (filtered):     77,728,414 bp (93.4%)\n",
      "  Read 1:    38,781,261 bp\n",
      "  Read 2:    38,947,153 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 140961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t48\t0.0\t1\t0 0 48\n",
      "19\t827\t0.0\t1\t0 809 18\n",
      "20\t139742\t0.0\t2\t137394 2306 42\n",
      "21\t256\t0.0\t2\t0 251 5\n",
      "22\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 140045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t653\t0.0\t1\t0 653\n",
      "19\t139136\t0.0\t1\t135512 3624\n",
      "20\t256\t0.0\t1\t0 256\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2426_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2426_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2426_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2426_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2426_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2426_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2426_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2426_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09       134,029 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 9.82 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            134,029\n",
      "  Read 1 with adapter:                 133,953 (99.9%)\n",
      "  Read 2 with adapter:                 133,338 (99.5%)\n",
      "Pairs written (passing filters):       134,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,069,050 bp\n",
      "  Read 1:    39,533,169 bp\n",
      "  Read 2:    39,535,881 bp\n",
      "Total written (filtered):     73,857,338 bp (93.4%)\n",
      "  Read 1:    36,854,519 bp\n",
      "  Read 2:    37,002,819 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 133953 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t54\t0.0\t1\t0 0 54\n",
      "19\t753\t0.0\t1\t0 737 16\n",
      "20\t132802\t0.0\t2\t130440 2328 34\n",
      "21\t237\t0.0\t2\t0 235 2\n",
      "22\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 133338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t603\t0.0\t1\t0 603\n",
      "19\t132492\t0.0\t1\t129015 3477\n",
      "20\t243\t0.0\t1\t0 243\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2427_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2427_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2427_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2427_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2427_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2427_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2427_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2427_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:12       132,204 reads  @     92.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 12.28 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,204\n",
      "  Read 1 with adapter:                 132,106 (99.9%)\n",
      "  Read 2 with adapter:                 131,387 (99.4%)\n",
      "Pairs written (passing filters):       132,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    77,973,342 bp\n",
      "  Read 1:    38,989,683 bp\n",
      "  Read 2:    38,983,659 bp\n",
      "Total written (filtered):     72,835,715 bp (93.4%)\n",
      "  Read 1:    36,347,712 bp\n",
      "  Read 2:    36,488,003 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 132106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t46\t0.0\t1\t0 0 46\n",
      "19\t979\t0.0\t1\t0 949 30\n",
      "20\t130461\t0.0\t2\t126969 3437 55\n",
      "21\t318\t0.0\t2\t0 311 7\n",
      "22\t302\t0.0\t2\t0 0 302\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 131387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1008\t0.0\t1\t0 1008\n",
      "19\t130068\t0.0\t1\t125859 4209\n",
      "20\t311\t0.0\t1\t0 311\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2428_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2428_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2428_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2428_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2428_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2428_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2428_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2428_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        97,792 reads  @     97.7 µs/read;   0.61 M reads/minute\n",
      "Finished in 9.56 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             97,792\n",
      "  Read 1 with adapter:                  97,726 (99.9%)\n",
      "  Read 2 with adapter:                  97,273 (99.5%)\n",
      "Pairs written (passing filters):        97,792 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,685,266 bp\n",
      "  Read 1:    28,842,941 bp\n",
      "  Read 2:    28,842,325 bp\n",
      "Total written (filtered):     53,883,275 bp (93.4%)\n",
      "  Read 1:    26,888,578 bp\n",
      "  Read 2:    26,994,697 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 97726 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t680\t0.0\t1\t0 665 15\n",
      "20\t96588\t0.0\t2\t94034 2501 53\n",
      "21\t197\t0.0\t2\t0 192 5\n",
      "22\t212\t0.0\t2\t0 0 212\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 97273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t848\t0.0\t1\t0 848\n",
      "19\t96136\t0.0\t1\t93094 3042\n",
      "20\t289\t0.0\t1\t0 289\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2429_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2429_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2429_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2429_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2429_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2429_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2429_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2429_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11       142,988 reads  @     78.3 µs/read;   0.77 M reads/minute\n",
      "Finished in 11.19 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            142,988\n",
      "  Read 1 with adapter:                 142,930 (100.0%)\n",
      "  Read 2 with adapter:                 142,195 (99.4%)\n",
      "Pairs written (passing filters):       142,978 (100.0%)\n",
      "\n",
      "Total basepairs processed:    84,350,097 bp\n",
      "  Read 1:    42,179,889 bp\n",
      "  Read 2:    42,170,208 bp\n",
      "Total written (filtered):     78,787,937 bp (93.4%)\n",
      "  Read 1:    39,318,957 bp\n",
      "  Read 2:    39,468,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 142930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t49\t0.0\t1\t0 0 49\n",
      "19\t854\t0.0\t1\t0 832 22\n",
      "20\t141624\t0.0\t2\t139217 2378 29\n",
      "21\t272\t0.0\t2\t0 270 2\n",
      "22\t131\t0.0\t2\t0 0 131\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 142195 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t766\t0.0\t1\t0 766\n",
      "19\t141140\t0.0\t1\t136180 4960\n",
      "20\t289\t0.0\t1\t0 289\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2430_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2430_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2430_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2430_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2430_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2430_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2430_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2430_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11       133,129 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 11.25 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            133,129\n",
      "  Read 1 with adapter:                 133,064 (100.0%)\n",
      "  Read 2 with adapter:                 132,192 (99.3%)\n",
      "Pairs written (passing filters):       133,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:    78,508,043 bp\n",
      "  Read 1:    39,270,210 bp\n",
      "  Read 2:    39,237,833 bp\n",
      "Total written (filtered):     73,335,942 bp (93.4%)\n",
      "  Read 1:    36,609,166 bp\n",
      "  Read 2:    36,726,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 133064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t55\t0.0\t1\t0 0 55\n",
      "19\t908\t0.0\t1\t0 888 20\n",
      "20\t131580\t0.0\t2\t128854 2686 40\n",
      "21\t260\t0.0\t2\t0 254 6\n",
      "22\t261\t0.0\t2\t0 0 261\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 132192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t899\t0.0\t1\t0 899\n",
      "19\t130985\t0.0\t1\t127002 3983\n",
      "20\t308\t0.0\t1\t0 308\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2431_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2431_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2431_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2431_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2431_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2431_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2431_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2431_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09       104,763 reads  @     89.8 µs/read;   0.67 M reads/minute\n",
      "Finished in 9.41 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            104,763\n",
      "  Read 1 with adapter:                 104,691 (99.9%)\n",
      "  Read 2 with adapter:                 103,885 (99.2%)\n",
      "Pairs written (passing filters):       104,763 (100.0%)\n",
      "\n",
      "Total basepairs processed:    61,750,421 bp\n",
      "  Read 1:    30,902,129 bp\n",
      "  Read 2:    30,848,292 bp\n",
      "Total written (filtered):     57,683,516 bp (93.4%)\n",
      "  Read 1:    28,808,663 bp\n",
      "  Read 2:    28,874,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 104691 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t32\t0.0\t1\t0 0 32\n",
      "19\t695\t0.0\t1\t0 675 20\n",
      "20\t103666\t0.0\t2\t101406 2225 35\n",
      "21\t191\t0.0\t2\t0 181 10\n",
      "22\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 103885 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t579\t0.0\t1\t0 579\n",
      "19\t103103\t0.0\t1\t100360 2743\n",
      "20\t203\t0.0\t1\t0 203\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2432_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2432_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2432_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2432_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2432_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2432_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2432_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2432_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:11       111,362 reads  @    104.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 11.59 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,362\n",
      "  Read 1 with adapter:                 111,290 (99.9%)\n",
      "  Read 2 with adapter:                 108,517 (97.4%)\n",
      "Pairs written (passing filters):       111,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,519,142 bp\n",
      "  Read 1:    32,840,957 bp\n",
      "  Read 2:    32,678,185 bp\n",
      "Total written (filtered):     61,232,336 bp (93.5%)\n",
      "  Read 1:    30,615,476 bp\n",
      "  Read 2:    30,616,860 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 111290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t31\t0.0\t1\t0 0 31\n",
      "19\t751\t0.0\t1\t0 723 28\n",
      "20\t110160\t0.0\t2\t107364 2755 41\n",
      "21\t202\t0.0\t2\t0 197 5\n",
      "22\t146\t0.0\t2\t0 0 146\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 108517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t723\t0.0\t1\t0 723\n",
      "19\t107569\t0.0\t1\t104814 2755\n",
      "20\t225\t0.0\t1\t0 225\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2446_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2446_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2446_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2446_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2446_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2446_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2446_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2446_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2461_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2461_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2461_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2461_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2461_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2461_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2461_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2461_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2469_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2469_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2469_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2469_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2469_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2469_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2469_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2469_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2470_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2470_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2470_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2470_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2470_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2470_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2470_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2470_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2471_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2471_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2471_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2471_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2471_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2471_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2471_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2471_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2472_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2472_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2472_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2472_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2472_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2472_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2472_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2472_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2473_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2473_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2473_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2473_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2473_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2473_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2473_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2473_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2477_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2477_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2477_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2477_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2477_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2477_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2477_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2477_175_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2478_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2478_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2478_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2478_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2478_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2478_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2478_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2478_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2480_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2480_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2480_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2480_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2480_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-_0khf7pd/S2480_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2480_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9que063v/c855a551-a7ea-4e8c-afe0-3d1c051f220d/data/S2480_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: h-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: h-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'q2' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-3-0e67edce0140>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;31m#3. summarize and view\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msystem\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'qiime demux summarize    --i-data h-demux-pairedend-trimmed.qza    --o-visualization h-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0mq2\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mVisualization\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'h-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'q2' is not defined"
     ]
    }
   ],
   "source": [
    "#1.h import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/ENP-Lib-manifest_18S.csv \\\n",
    "  --output-path h-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2. trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences h-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences h-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3. summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data h-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization h-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('h-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: h-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: h-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: h-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.h Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs h-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 274 \\\n",
    "  --p-trunc-len-r 196 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table h-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats h-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: h-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9u4mwzak')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c8555c18-e5f5-428f-ae01-d5bfe3c68194>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.h view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table h-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization h-table.qzv \n",
    "q2.Visualization.load('h-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: h-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.h view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization h-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/GAK-manifest_18S.csv as PairedEndFastqManifestPhred33 to i-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3122_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3122_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3122_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3122_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3122_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3122_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3122_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3122_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07       117,604 reads  @     67.8 µs/read;   0.89 M reads/minute\n",
      "Finished in 7.97 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            117,604\n",
      "  Read 1 with adapter:                 117,551 (100.0%)\n",
      "  Read 2 with adapter:                 116,770 (99.3%)\n",
      "Pairs written (passing filters):       117,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:    69,268,756 bp\n",
      "  Read 1:    34,693,180 bp\n",
      "  Read 2:    34,575,576 bp\n",
      "Total written (filtered):     64,699,566 bp (93.4%)\n",
      "  Read 1:    32,342,380 bp\n",
      "  Read 2:    32,357,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 117551 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t466\t0.0\t1\t0 457 9\n",
      "20\t116778\t0.0\t2\t115238 1525 15\n",
      "21\t276\t0.0\t2\t0 267 9\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 116770 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t500\t0.0\t1\t0 500\n",
      "19\t116010\t0.0\t1\t113991 2019\n",
      "20\t260\t0.0\t1\t0 260\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3123_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3123_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3123_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3123_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3123_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3123_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3123_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3123_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        86,625 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 6.20 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,625\n",
      "  Read 1 with adapter:                  86,585 (100.0%)\n",
      "  Read 2 with adapter:                  85,954 (99.2%)\n",
      "Pairs written (passing filters):        86,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,108,750 bp\n",
      "  Read 1:    25,641,000 bp\n",
      "  Read 2:    25,467,750 bp\n",
      "Total written (filtered):     47,744,104 bp (93.4%)\n",
      "  Read 1:    23,909,346 bp\n",
      "  Read 2:    23,834,758 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 86585 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t413\t0.0\t1\t0 408 5\n",
      "20\t85752\t0.0\t2\t84926 817 9\n",
      "21\t393\t0.0\t2\t0 388 5\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t377\t0.0\t1\t0 377\n",
      "19\t85334\t0.0\t1\t83669 1665\n",
      "20\t243\t0.0\t1\t0 243\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3124_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3124_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3124_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3124_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3124_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3124_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3124_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3124_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,454 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.90 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,454\n",
      "  Read 1 with adapter:                  67,432 (100.0%)\n",
      "  Read 2 with adapter:                  66,878 (99.1%)\n",
      "Pairs written (passing filters):        67,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,595,498 bp\n",
      "  Read 1:    19,764,022 bp\n",
      "  Read 2:    19,831,476 bp\n",
      "Total written (filtered):     36,976,532 bp (93.4%)\n",
      "  Read 1:    18,415,586 bp\n",
      "  Read 2:    18,560,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 67432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t352\t0.0\t1\t0 348 4\n",
      "20\t66869\t0.0\t2\t66214 641 14\n",
      "21\t186\t0.0\t2\t0 183 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66878 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t309\t0.0\t1\t0 309\n",
      "19\t66412\t0.0\t1\t65162 1250\n",
      "20\t157\t0.0\t1\t0 157\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3125_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3125_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3125_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3125_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3125_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3125_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3125_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3125_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,882 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.76 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,882\n",
      "  Read 1 with adapter:                  62,845 (99.9%)\n",
      "  Read 2 with adapter:                  62,476 (99.4%)\n",
      "Pairs written (passing filters):        62,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,974,616 bp\n",
      "  Read 1:    18,487,308 bp\n",
      "  Read 2:    18,487,308 bp\n",
      "Total written (filtered):     34,531,175 bp (93.4%)\n",
      "  Read 1:    17,230,682 bp\n",
      "  Read 2:    17,300,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 62845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t44\t0.0\t1\t0 0 44\n",
      "19\t386\t0.0\t1\t0 384 2\n",
      "20\t62221\t0.0\t2\t60869 1328 24\n",
      "21\t188\t0.0\t2\t0 182 6\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62476 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t385\t0.0\t1\t0 385\n",
      "19\t61935\t0.0\t1\t60779 1156\n",
      "20\t156\t0.0\t1\t0 156\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3126_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3126_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3126_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3126_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3126_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3126_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3126_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3126_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        72,751 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 5.29 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,751\n",
      "  Read 1 with adapter:                  72,721 (100.0%)\n",
      "  Read 2 with adapter:                  72,225 (99.3%)\n",
      "Pairs written (passing filters):        72,751 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,850,339 bp\n",
      "  Read 1:    21,461,545 bp\n",
      "  Read 2:    21,388,794 bp\n",
      "Total written (filtered):     40,024,066 bp (93.4%)\n",
      "  Read 1:    20,007,370 bp\n",
      "  Read 2:    20,016,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 72721 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t330\t0.0\t1\t0 319 11\n",
      "20\t72246\t0.0\t2\t71547 688 11\n",
      "21\t121\t0.0\t2\t0 118 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t344\t0.0\t1\t0 344\n",
      "19\t71714\t0.0\t1\t70417 1297\n",
      "20\t167\t0.0\t1\t0 167\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3127_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3127_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3127_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3127_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3127_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3127_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3127_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3127_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,498 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.70 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,498\n",
      "  Read 1 with adapter:                  54,473 (100.0%)\n",
      "  Read 2 with adapter:                  54,157 (99.4%)\n",
      "Pairs written (passing filters):        54,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,153,820 bp\n",
      "  Read 1:    16,131,408 bp\n",
      "  Read 2:    16,022,412 bp\n",
      "Total written (filtered):     30,035,637 bp (93.4%)\n",
      "  Read 1:    15,042,093 bp\n",
      "  Read 2:    14,993,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t235\t0.0\t1\t0 234 1\n",
      "20\t54096\t0.0\t2\t53633 455 8\n",
      "21\t122\t0.0\t2\t0 121 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t236\t0.0\t1\t0 236\n",
      "19\t53800\t0.0\t1\t52907 893\n",
      "20\t121\t0.0\t1\t0 121\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3128_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3128_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3128_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3128_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3128_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3128_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3128_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3128_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:05        83,790 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 5.99 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,790\n",
      "  Read 1 with adapter:                  83,739 (99.9%)\n",
      "  Read 2 with adapter:                  83,383 (99.5%)\n",
      "Pairs written (passing filters):        83,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,352,310 bp\n",
      "  Read 1:    24,550,470 bp\n",
      "  Read 2:    24,801,840 bp\n",
      "Total written (filtered):     46,093,884 bp (93.4%)\n",
      "  Read 1:    22,875,921 bp\n",
      "  Read 2:    23,217,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 83739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t361\t0.0\t1\t0 355 6\n",
      "20\t83173\t0.0\t2\t82367 788 18\n",
      "21\t172\t0.0\t2\t0 170 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 83383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t662\t0.0\t1\t0 662\n",
      "19\t82459\t0.0\t1\t81286 1173\n",
      "20\t262\t0.0\t1\t0 262\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3129_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3129_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3129_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3129_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3129_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3129_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3129_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3129_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        95,516 reads  @     67.3 µs/read;   0.89 M reads/minute\n",
      "Finished in 6.44 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,516\n",
      "  Read 1 with adapter:                  95,474 (100.0%)\n",
      "  Read 2 with adapter:                  94,995 (99.5%)\n",
      "Pairs written (passing filters):        95,516 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,258,924 bp\n",
      "  Read 1:    28,177,220 bp\n",
      "  Read 2:    28,081,704 bp\n",
      "Total written (filtered):     52,545,046 bp (93.4%)\n",
      "  Read 1:    26,267,982 bp\n",
      "  Read 2:    26,277,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 95474 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t391\t0.0\t1\t0 380 11\n",
      "20\t94850\t0.0\t2\t93574 1258 18\n",
      "21\t201\t0.0\t2\t0 191 10\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94995 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t461\t0.0\t1\t0 461\n",
      "19\t94338\t0.0\t1\t92802 1536\n",
      "20\t196\t0.0\t1\t0 196\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3130_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3130_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3130_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3130_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3130_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3130_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3130_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3130_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        73,257 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 5.15 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,257\n",
      "  Read 1 with adapter:                  73,209 (99.9%)\n",
      "  Read 2 with adapter:                  72,749 (99.3%)\n",
      "Pairs written (passing filters):        73,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,221,630 bp\n",
      "  Read 1:    21,684,072 bp\n",
      "  Read 2:    21,537,558 bp\n",
      "Total written (filtered):     40,375,371 bp (93.4%)\n",
      "  Read 1:    20,219,878 bp\n",
      "  Read 2:    20,155,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 73209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t301\t0.0\t1\t0 299 2\n",
      "20\t72524\t0.0\t2\t71819 686 19\n",
      "21\t357\t0.0\t2\t0 354 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t336\t0.0\t1\t0 336\n",
      "19\t72243\t0.0\t1\t71011 1232\n",
      "20\t170\t0.0\t1\t0 170\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3131_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3131_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3131_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3131_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3131_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3131_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3131_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3131_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,226 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.53 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,226\n",
      "  Read 1 with adapter:                  63,200 (100.0%)\n",
      "  Read 2 with adapter:                  62,737 (99.2%)\n",
      "Pairs written (passing filters):        63,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,113,662 bp\n",
      "  Read 1:    18,525,218 bp\n",
      "  Read 2:    18,588,444 bp\n",
      "Total written (filtered):     34,658,051 bp (93.4%)\n",
      "  Read 1:    17,261,429 bp\n",
      "  Read 2:    17,396,622 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t335\t0.0\t1\t0 331 4\n",
      "20\t62689\t0.0\t2\t62074 608 7\n",
      "21\t148\t0.0\t2\t0 148\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t325\t0.0\t1\t0 325\n",
      "19\t62268\t0.0\t1\t61059 1209\n",
      "20\t144\t0.0\t1\t0 144\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3132_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3132_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3132_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3132_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3132_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3132_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3132_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3132_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        75,976 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.74 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,976\n",
      "  Read 1 with adapter:                  75,951 (100.0%)\n",
      "  Read 2 with adapter:                  75,482 (99.3%)\n",
      "Pairs written (passing filters):        75,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,673,888 bp\n",
      "  Read 1:    22,336,944 bp\n",
      "  Read 2:    22,336,944 bp\n",
      "Total written (filtered):     41,721,401 bp (93.4%)\n",
      "  Read 1:    20,818,230 bp\n",
      "  Read 2:    20,903,171 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 75951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t43\t0.0\t1\t0 0 43\n",
      "19\t469\t0.0\t1\t0 464 5\n",
      "20\t75198\t0.0\t2\t73642 1519 37\n",
      "21\t233\t0.0\t2\t0 227 6\n",
      "22\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 75482 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t567\t0.0\t1\t0 567\n",
      "19\t74733\t0.0\t1\t73493 1240\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3133_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3133_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3133_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3133_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3133_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3133_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3133_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3133_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        94,940 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 6.79 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,940\n",
      "  Read 1 with adapter:                  94,888 (99.9%)\n",
      "  Read 2 with adapter:                  94,261 (99.3%)\n",
      "Pairs written (passing filters):        94,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,919,660 bp\n",
      "  Read 1:    28,007,300 bp\n",
      "  Read 2:    27,912,360 bp\n",
      "Total written (filtered):     52,231,448 bp (93.4%)\n",
      "  Read 1:    26,109,843 bp\n",
      "  Read 2:    26,121,605 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 94888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t476\t0.0\t1\t0 468 8\n",
      "20\t94166\t0.0\t2\t93208 945 13\n",
      "21\t219\t0.0\t2\t0 214 5\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 94261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t411\t0.0\t1\t0 411\n",
      "19\t93643\t0.0\t1\t91996 1647\n",
      "20\t207\t0.0\t1\t0 207\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3134_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3134_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3134_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3134_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3134_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3134_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3134_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3134_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        73,015 reads  @     67.3 µs/read;   0.89 M reads/minute\n",
      "Finished in 4.92 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,015\n",
      "  Read 1 with adapter:                  72,987 (100.0%)\n",
      "  Read 2 with adapter:                  72,616 (99.5%)\n",
      "Pairs written (passing filters):        73,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,078,850 bp\n",
      "  Read 1:    21,612,440 bp\n",
      "  Read 2:    21,466,410 bp\n",
      "Total written (filtered):     40,239,702 bp (93.4%)\n",
      "  Read 1:    20,152,841 bp\n",
      "  Read 2:    20,086,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 72987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t304\t0.0\t1\t0 302 2\n",
      "20\t72472\t0.0\t2\t71780 683 9\n",
      "21\t187\t0.0\t2\t0 186 1\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 72616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t313\t0.0\t1\t0 313\n",
      "19\t72145\t0.0\t1\t71019 1126\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3135_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3135_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3135_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3135_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3135_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3135_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3135_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3135_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        82,532 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 5.75 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,532\n",
      "  Read 1 with adapter:                  82,488 (99.9%)\n",
      "  Read 2 with adapter:                  82,054 (99.4%)\n",
      "Pairs written (passing filters):        82,532 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,611,348 bp\n",
      "  Read 1:    24,181,876 bp\n",
      "  Read 2:    24,429,472 bp\n",
      "Total written (filtered):     45,403,335 bp (93.4%)\n",
      "  Read 1:    22,532,364 bp\n",
      "  Read 2:    22,870,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 82488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t37\t0.0\t1\t0 0 37\n",
      "19\t366\t0.0\t1\t0 357 9\n",
      "20\t81895\t0.0\t2\t81032 850 13\n",
      "21\t188\t0.0\t2\t0 185 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 82054 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t814\t0.0\t1\t0 814\n",
      "19\t80951\t0.0\t1\t79683 1268\n",
      "20\t289\t0.0\t1\t0 289\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3136_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3136_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3136_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3136_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3136_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3136_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3136_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3136_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,902 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.43 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,902\n",
      "  Read 1 with adapter:                  63,874 (100.0%)\n",
      "  Read 2 with adapter:                  63,477 (99.3%)\n",
      "Pairs written (passing filters):        63,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,638,278 bp\n",
      "  Read 1:    18,851,090 bp\n",
      "  Read 2:    18,787,188 bp\n",
      "Total written (filtered):     35,155,000 bp (93.4%)\n",
      "  Read 1:    17,573,763 bp\n",
      "  Read 2:    17,581,237 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t257\t0.0\t1\t0 249 8\n",
      "20\t63442\t0.0\t2\t62659 768 15\n",
      "21\t146\t0.0\t2\t0 143 3\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63477 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t284\t0.0\t1\t0 284\n",
      "19\t63021\t0.0\t1\t61927 1094\n",
      "20\t172\t0.0\t1\t0 172\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3137_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3137_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3137_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3137_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3137_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3137_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3137_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3137_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10       150,139 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 10.35 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            150,139\n",
      "  Read 1 with adapter:                 150,051 (99.9%)\n",
      "  Read 2 with adapter:                 149,051 (99.3%)\n",
      "Pairs written (passing filters):       150,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    88,582,010 bp\n",
      "  Read 1:    44,441,144 bp\n",
      "  Read 2:    44,140,866 bp\n",
      "Total written (filtered):     82,749,461 bp (93.4%)\n",
      "  Read 1:    41,440,254 bp\n",
      "  Read 2:    41,309,207 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 150051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t50\t0.0\t1\t0 0 50\n",
      "19\t653\t0.0\t1\t0 643 10\n",
      "20\t148732\t0.0\t2\t147269 1436 27\n",
      "21\t609\t0.0\t2\t0 600 9\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 149051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t690\t0.0\t1\t0 690\n",
      "19\t147981\t0.0\t1\t145309 2672\n",
      "20\t380\t0.0\t1\t0 380\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3138_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3138_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3138_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3138_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3138_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3138_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3138_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3138_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,256 reads  @     74.5 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.05 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,256\n",
      "  Read 1 with adapter:                  54,231 (100.0%)\n",
      "  Read 2 with adapter:                  53,779 (99.1%)\n",
      "Pairs written (passing filters):        54,256 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,848,272 bp\n",
      "  Read 1:    15,897,008 bp\n",
      "  Read 2:    15,951,264 bp\n",
      "Total written (filtered):     29,742,188 bp (93.4%)\n",
      "  Read 1:    14,812,594 bp\n",
      "  Read 2:    14,929,594 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t303\t0.0\t1\t0 296 7\n",
      "20\t53783\t0.0\t2\t53283 491 9\n",
      "21\t121\t0.0\t2\t0 119 2\n",
      "22\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53779 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t266\t0.0\t1\t0 266\n",
      "19\t53378\t0.0\t1\t52365 1013\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3139_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3139_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3139_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3139_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3139_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3139_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3139_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3139_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,041 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.08 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,041\n",
      "  Read 1 with adapter:                  41,024 (100.0%)\n",
      "  Read 2 with adapter:                  40,782 (99.4%)\n",
      "Pairs written (passing filters):        41,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,132,108 bp\n",
      "  Read 1:    12,066,054 bp\n",
      "  Read 2:    12,066,054 bp\n",
      "Total written (filtered):     22,537,114 bp (93.4%)\n",
      "  Read 1:    11,245,772 bp\n",
      "  Read 2:    11,291,342 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41024 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t247\t0.0\t1\t0 246 1\n",
      "20\t40658\t0.0\t2\t39763 876 19\n",
      "21\t89\t0.0\t2\t0 85 4\n",
      "22\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t40446\t0.0\t1\t39759 687\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3140_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3140_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3140_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3140_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3140_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3140_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3140_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3140_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        66,561 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.76 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,561\n",
      "  Read 1 with adapter:                  66,527 (99.9%)\n",
      "  Read 2 with adapter:                  66,072 (99.3%)\n",
      "Pairs written (passing filters):        66,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,204,429 bp\n",
      "  Read 1:    19,635,495 bp\n",
      "  Read 2:    19,568,934 bp\n",
      "Total written (filtered):     36,618,845 bp (93.4%)\n",
      "  Read 1:    18,305,126 bp\n",
      "  Read 2:    18,313,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66527 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t284\t0.0\t1\t0 275 9\n",
      "20\t66049\t0.0\t2\t65402 633 14\n",
      "21\t163\t0.0\t2\t0 161 2\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66072 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t295\t0.0\t1\t0 295\n",
      "19\t65635\t0.0\t1\t64464 1171\n",
      "20\t142\t0.0\t1\t0 142\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3141_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3141_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3141_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3141_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3141_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3141_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3141_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3141_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,522 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.44 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,522\n",
      "  Read 1 with adapter:                  50,500 (100.0%)\n",
      "  Read 2 with adapter:                  50,246 (99.5%)\n",
      "Pairs written (passing filters):        50,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,807,980 bp\n",
      "  Read 1:    14,954,512 bp\n",
      "  Read 2:    14,853,468 bp\n",
      "Total written (filtered):     27,843,519 bp (93.4%)\n",
      "  Read 1:    13,944,620 bp\n",
      "  Read 2:    13,898,899 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t220\t0.0\t1\t0 217 3\n",
      "20\t50142\t0.0\t2\t49678 449 15\n",
      "21\t124\t0.0\t2\t0 122 2\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50246 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t49927\t0.0\t1\t49103 824\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3142_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3142_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3142_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3142_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3142_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-p6nsxmdm/S3142_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3142_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-edpg_1ss/eb04efbd-d576-4144-a74c-df6d82ecd8d9/data/S3142_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        65,774 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.73 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,774\n",
      "  Read 1 with adapter:                  65,735 (99.9%)\n",
      "  Read 2 with adapter:                  65,430 (99.5%)\n",
      "Pairs written (passing filters):        65,774 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,740,886 bp\n",
      "  Read 1:    19,271,782 bp\n",
      "  Read 2:    19,469,104 bp\n",
      "Total written (filtered):     36,183,513 bp (93.4%)\n",
      "  Read 1:    17,957,226 bp\n",
      "  Read 2:    18,226,287 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 65735 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t253\t0.0\t1\t0 251 2\n",
      "20\t65315\t0.0\t2\t64686 617 12\n",
      "21\t145\t0.0\t2\t0 142 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 65430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t541\t0.0\t1\t0 541\n",
      "19\t64701\t0.0\t1\t63703 998\n",
      "20\t188\t0.0\t1\t0 188\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: i-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: i-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'q2' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-7-0e78c3b44d8a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;31m#3.i summarize and view\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msystem\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'qiime demux summarize    --i-data i-demux-pairedend-trimmed.qza    --o-visualization i-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0mq2\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mVisualization\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'i-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'q2' is not defined"
     ]
    }
   ],
   "source": [
    "#1.i import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/18S/GAK-manifest_18S.csv \\\n",
    "  --output-path i-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.i trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences i-paired-end-demux.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences i-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.i summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data i-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization i-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('i-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: i-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: i-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: i-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.i Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs i-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 273 \\\n",
    "  --p-trunc-len-r 253 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table i-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats i-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: i-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kcjrn724')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b1f7f3e6-88cc-40d1-80fb-a13c6051d210>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.i view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table i-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization i-table.qzv \n",
    "q2.Visualization.load('i-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: i-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.i view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization i-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Merge the separate sequencing runs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#merge \n",
    "!qiime feature-table merge \\\n",
    "  --i-tables a-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables b-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables c-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables d-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables e-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables f-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables g-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables h-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables i-paired-end-demux_table-dada2.qza \\\n",
    "  --o-merged-table 18S-paired-end-demux_table-dada2.qza\n",
    "!qiime feature-table merge-seqs \\\n",
    "  --i-data a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-merged-data 18S-paired-end-demux_rep-seqs-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported 18S-paired-end-demux_table-dada2.qza as BIOMV210DirFmt to directory 18S-exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools export \\\n",
    "  --input-path 18S-paired-end-demux_table-dada2.qza \\\n",
    "  --output-path 18S-exported-feature-table\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-tjifdf72')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1b0a9bb3-9150-44a6-85e4-d0f538a0ccb7>"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data 18S-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization 18S-paired-end-demux_rep-seqs-dada2.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-paired-end-demux_rep-seqs-dada2.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i 18S-exported-feature-table/feature-table.biom -o 18S_97vsearch_OTUtable.tsv --to-tsv\n",
    "!biom convert -i 18S-exported-feature-table/feature-table.biom -o 18S-exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\"\n",
    "!biom convert -i 18S-exported-feature-table/feature-table.biom -o 18S-exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1z7v0glc')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1b0a9bb3-9150-44a6-85e4-d0f538a0ccb7>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('18S-paired-end-demux_rep-seqs-dada2.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
