{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Jupyter notebook with code for importing demultiplexed data from the COI marker. \n",
    "For each library (10 total, a-j)\n",
    "1) import\n",
    "2) trim primers\n",
    "3) summarize and view trimmed data\n",
    "4) generate ESVs with quality trimming parameters based on (3)\n",
    "5) summarize and view ESV table\n",
    "6) tabulate and view ESV sequences\n",
    "\n",
    "Then: \n",
    "7) merge ESV table and sequences for all libraries \n",
    "8) vsearch cluster (97%) \n",
    "9) tabulate and view sequences, download as .fasta (GUI download) \n",
    "10) summarize and view OTU table \n",
    "11) export OTU table as biom \n",
    "12) convert biom to tsv, json, and hdf formates "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "a: P1604 Library 1 \n",
    "b: P1604 Library 2\n",
    "c: P1604 Library 3\n",
    "d: BB Library 1\n",
    "e: Hirai Library 1\n",
    "f: Hirai Library 2\n",
    "g: P2107 Library 1\n",
    "h: ENP Library 1\n",
    "i: GAK Library 1\n",
    "j: DS5B Library 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#0: import the qiime2 python module to allow for inline visualizations\n",
    "import qiime2 as q2\n",
    "#which version of qiime2 and python are we using?\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib1-leray_manifest_run1_grid.csv as PairedEndFastqManifestPhred33 to a-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S001_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S001_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,393 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.68 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,393\n",
      "  Read 1 with adapter:                  32,365 (99.9%)\n",
      "  Read 2 with adapter:                  32,122 (99.2%)\n",
      "Pairs written (passing filters):        32,393 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,107,367 bp\n",
      "  Read 1:     9,555,543 bp\n",
      "  Read 2:     9,551,824 bp\n",
      "Total written (filtered):     17,431,725 bp (91.2%)\n",
      "  Read 1:     8,713,815 bp\n",
      "  Read 2:     8,717,910 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t288\t0.0\t2\t0 277 11\n",
      "26\t31637\t0.0\t2\t30730 876 31\n",
      "27\t290\t0.0\t2\t0 275 15\n",
      "28\t134\t0.0\t2\t0 0 134\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t1379\t0.0\t2\t0 1347 32\n",
      "26\t30527\t0.0\t2\t29836 673 18\n",
      "27\t163\t0.0\t2\t0 159 4\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S002_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S002_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,557 reads  @     82.1 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.73 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,557\n",
      "  Read 1 with adapter:                  57,517 (99.9%)\n",
      "  Read 2 with adapter:                  57,309 (99.6%)\n",
      "Pairs written (passing filters):        57,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,955,633 bp\n",
      "  Read 1:    16,978,797 bp\n",
      "  Read 2:    16,976,836 bp\n",
      "Total written (filtered):     30,970,955 bp (91.2%)\n",
      "  Read 1:    15,483,296 bp\n",
      "  Read 2:    15,487,659 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t526\t0.0\t2\t0 512 14\n",
      "26\t56440\t0.0\t2\t55015 1406 19\n",
      "27\t341\t0.0\t2\t0 330 11\n",
      "28\t166\t0.0\t2\t0 0 166\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t1041\t0.0\t2\t0 1006 35\n",
      "26\t55949\t0.0\t2\t54468 1423 58\n",
      "27\t254\t0.0\t2\t0 235 19\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S003_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S003_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,039 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.74 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,039\n",
      "  Read 1 with adapter:                  68,994 (99.9%)\n",
      "  Read 2 with adapter:                  68,479 (99.2%)\n",
      "Pairs written (passing filters):        69,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,731,067 bp\n",
      "  Read 1:    20,365,377 bp\n",
      "  Read 2:    20,365,690 bp\n",
      "Total written (filtered):     37,158,126 bp (91.2%)\n",
      "  Read 1:    18,571,565 bp\n",
      "  Read 2:    18,586,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t729\t0.0\t2\t0 708 21\n",
      "26\t67632\t0.0\t2\t66043 1557 32\n",
      "27\t385\t0.0\t2\t0 371 14\n",
      "28\t202\t0.0\t2\t0 0 202\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t1542\t0.0\t2\t0 1504 38\n",
      "26\t66589\t0.0\t2\t65050 1475 64\n",
      "27\t287\t0.0\t2\t0 278 9\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S004_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S004_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,547 reads  @     86.8 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.00 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,547\n",
      "  Read 1 with adapter:                  34,522 (99.9%)\n",
      "  Read 2 with adapter:                  33,737 (97.7%)\n",
      "Pairs written (passing filters):        34,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,267,932 bp\n",
      "  Read 1:    10,191,101 bp\n",
      "  Read 2:    10,076,831 bp\n",
      "Total written (filtered):     18,493,406 bp (91.2%)\n",
      "  Read 1:     9,293,443 bp\n",
      "  Read 2:     9,199,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34522 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t374\t0.0\t2\t0 362 12\n",
      "26\t33903\t0.0\t2\t33049 836 18\n",
      "27\t183\t0.0\t2\t0 171 12\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t416\t0.0\t2\t0 397 19\n",
      "26\t33128\t0.0\t2\t32112 962 54\n",
      "27\t148\t0.0\t2\t0 139 9\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S005_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S005_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:07        73,292 reads  @     96.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 7.04 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,292\n",
      "  Read 1 with adapter:                  73,230 (99.9%)\n",
      "  Read 2 with adapter:                  68,191 (93.0%)\n",
      "Pairs written (passing filters):        73,290 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,370,803 bp\n",
      "  Read 1:    21,619,064 bp\n",
      "  Read 2:    20,751,739 bp\n",
      "Total written (filtered):     38,694,944 bp (91.3%)\n",
      "  Read 1:    19,715,089 bp\n",
      "  Read 2:    18,979,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t898\t0.0\t2\t0 867 31\n",
      "26\t71889\t0.0\t2\t70098 1754 37\n",
      "27\t271\t0.0\t2\t0 267 4\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t1314\t0.0\t2\t0 1251 63\n",
      "26\t66501\t0.0\t2\t64477 1916 108\n",
      "27\t248\t0.0\t2\t0 226 22\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S006_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S006_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        73,240 reads  @     86.8 µs/read;   0.69 M reads/minute\n",
      "Finished in 6.36 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,240\n",
      "  Read 1 with adapter:                  73,159 (99.9%)\n",
      "  Read 2 with adapter:                  72,895 (99.5%)\n",
      "Pairs written (passing filters):        73,239 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,170,391 bp\n",
      "  Read 1:    21,603,403 bp\n",
      "  Read 2:    21,566,988 bp\n",
      "Total written (filtered):     39,374,532 bp (91.2%)\n",
      "  Read 1:    19,701,553 bp\n",
      "  Read 2:    19,672,979 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t923\t0.0\t2\t0 898 25\n",
      "26\t71748\t0.0\t2\t69884 1832 32\n",
      "27\t302\t0.0\t2\t0 291 11\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72895 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t82\t0.0\t2\t0 0 82\n",
      "25\t1447\t0.0\t2\t0 1408 39\n",
      "26\t71068\t0.0\t2\t69116 1869 83\n",
      "27\t246\t0.0\t2\t0 237 9\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S007_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S007_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        72,146 reads  @     89.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 6.42 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,146\n",
      "  Read 1 with adapter:                  72,068 (99.9%)\n",
      "  Read 2 with adapter:                  72,005 (99.8%)\n",
      "Pairs written (passing filters):        72,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,557,841 bp\n",
      "  Read 1:    21,278,823 bp\n",
      "  Read 2:    21,279,018 bp\n",
      "Total written (filtered):     38,812,603 bp (91.2%)\n",
      "  Read 1:    19,405,303 bp\n",
      "  Read 2:    19,407,300 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72068 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t872\t0.0\t2\t0 843 29\n",
      "26\t70524\t0.0\t2\t68636 1860 28\n",
      "27\t368\t0.0\t2\t0 348 20\n",
      "28\t216\t0.0\t2\t0 0 216\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72005 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t792\t0.0\t2\t0 758 34\n",
      "26\t70797\t0.0\t2\t68826 1898 73\n",
      "27\t312\t0.0\t2\t0 292 20\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S008_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S008_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,056 reads  @     92.5 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.83 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,056\n",
      "  Read 1 with adapter:                  62,981 (99.9%)\n",
      "  Read 2 with adapter:                  62,934 (99.8%)\n",
      "Pairs written (passing filters):        63,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,196,828 bp\n",
      "  Read 1:    18,596,411 bp\n",
      "  Read 2:    18,600,417 bp\n",
      "Total written (filtered):     33,923,493 bp (91.2%)\n",
      "  Read 1:    16,958,911 bp\n",
      "  Read 2:    16,964,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t636\t0.0\t2\t0 617 19\n",
      "26\t61698\t0.0\t2\t60090 1566 42\n",
      "27\t412\t0.0\t2\t0 396 16\n",
      "28\t172\t0.0\t2\t0 0 172\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62934 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t677\t0.0\t2\t0 651 26\n",
      "26\t61915\t0.0\t2\t60408 1457 50\n",
      "27\t252\t0.0\t2\t0 241 11\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S009_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S009_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,359 reads  @     84.7 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.59 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,359\n",
      "  Read 1 with adapter:                  42,324 (99.9%)\n",
      "  Read 2 with adapter:                  42,000 (99.2%)\n",
      "Pairs written (passing filters):        42,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,986,505 bp\n",
      "  Read 1:    12,495,588 bp\n",
      "  Read 2:    12,490,917 bp\n",
      "Total written (filtered):     22,795,704 bp (91.2%)\n",
      "  Read 1:    11,395,041 bp\n",
      "  Read 2:    11,400,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t304\t0.0\t2\t0 292 12\n",
      "26\t41663\t0.0\t2\t40625 1014 24\n",
      "27\t175\t0.0\t2\t0 168 7\n",
      "28\t154\t0.0\t2\t0 0 154\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t1868\t0.0\t2\t0 1814 54\n",
      "26\t39879\t0.0\t2\t38963 894 22\n",
      "27\t144\t0.0\t2\t0 137 7\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S010_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S010_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,219 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.11 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,219\n",
      "  Read 1 with adapter:                  59,180 (99.9%)\n",
      "  Read 2 with adapter:                  59,039 (99.7%)\n",
      "Pairs written (passing filters):        59,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,937,638 bp\n",
      "  Read 1:    17,469,140 bp\n",
      "  Read 2:    17,468,498 bp\n",
      "Total written (filtered):     31,864,726 bp (91.2%)\n",
      "  Read 1:    15,930,406 bp\n",
      "  Read 2:    15,934,320 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t441\t0.0\t2\t0 425 16\n",
      "26\t58261\t0.0\t2\t56930 1313 18\n",
      "27\t261\t0.0\t2\t0 254 7\n",
      "28\t167\t0.0\t2\t0 0 167\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t996\t0.0\t2\t0 971 25\n",
      "26\t57754\t0.0\t2\t56435 1276 43\n",
      "27\t230\t0.0\t2\t0 209 21\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S011_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S011_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,187 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.36 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,187\n",
      "  Read 1 with adapter:                  48,150 (99.9%)\n",
      "  Read 2 with adapter:                  48,039 (99.7%)\n",
      "Pairs written (passing filters):        48,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,428,516 bp\n",
      "  Read 1:    14,214,601 bp\n",
      "  Read 2:    14,213,915 bp\n",
      "Total written (filtered):     25,928,360 bp (91.2%)\n",
      "  Read 1:    12,962,713 bp\n",
      "  Read 2:    12,965,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48150 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t420\t0.0\t2\t0 404 16\n",
      "26\t47351\t0.0\t2\t46243 1088 20\n",
      "27\t218\t0.0\t2\t0 217 1\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t1139\t0.0\t2\t0 1107 32\n",
      "26\t46671\t0.0\t2\t45559 1081 31\n",
      "27\t182\t0.0\t2\t0 174 8\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S012_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S012_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,308 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.63 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,308\n",
      "  Read 1 with adapter:                  40,966 (99.2%)\n",
      "  Read 2 with adapter:                  40,406 (97.8%)\n",
      "Pairs written (passing filters):        41,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,244,733 bp\n",
      "  Read 1:    12,185,401 bp\n",
      "  Read 2:    12,059,332 bp\n",
      "Total written (filtered):     22,129,198 bp (91.3%)\n",
      "  Read 1:    11,120,068 bp\n",
      "  Read 2:    11,009,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40966 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t353\t0.0\t2\t0 342 11\n",
      "26\t40333\t0.0\t2\t39294 1024 15\n",
      "27\t163\t0.0\t2\t0 152 11\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t509\t0.0\t2\t0 485 24\n",
      "26\t39665\t0.0\t2\t38406 1178 81\n",
      "27\t193\t0.0\t2\t0 182 11\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S013_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S013_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        81,229 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 7.89 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,229\n",
      "  Read 1 with adapter:                  81,120 (99.9%)\n",
      "  Read 2 with adapter:                  76,307 (93.9%)\n",
      "Pairs written (passing filters):        81,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,095,166 bp\n",
      "  Read 1:    23,961,272 bp\n",
      "  Read 2:    23,133,894 bp\n",
      "Total written (filtered):     43,003,473 bp (91.3%)\n",
      "  Read 1:    21,852,595 bp\n",
      "  Read 2:    21,150,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81120 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t825\t0.0\t2\t0 791 34\n",
      "26\t79823\t0.0\t2\t77683 2110 30\n",
      "27\t314\t0.0\t2\t0 301 13\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 76307 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t1472\t0.0\t2\t0 1425 47\n",
      "26\t74421\t0.0\t2\t72022 2262 137\n",
      "27\t287\t0.0\t2\t0 268 19\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S014_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S014_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        50,951 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.88 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,951\n",
      "  Read 1 with adapter:                  50,915 (99.9%)\n",
      "  Read 2 with adapter:                  50,763 (99.6%)\n",
      "Pairs written (passing filters):        50,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,034,833 bp\n",
      "  Read 1:    15,029,291 bp\n",
      "  Read 2:    15,005,542 bp\n",
      "Total written (filtered):     27,391,620 bp (91.2%)\n",
      "  Read 1:    13,705,003 bp\n",
      "  Read 2:    13,686,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t499\t0.0\t2\t0 484 15\n",
      "26\t50120\t0.0\t2\t48760 1343 17\n",
      "27\t186\t0.0\t2\t0 181 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t993\t0.0\t2\t0 947 46\n",
      "26\t49518\t0.0\t2\t48218 1247 53\n",
      "27\t164\t0.0\t2\t0 161 3\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S015_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S015_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,420 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.76 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,420\n",
      "  Read 1 with adapter:                  57,365 (99.9%)\n",
      "  Read 2 with adapter:                  57,296 (99.8%)\n",
      "Pairs written (passing filters):        57,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,874,877 bp\n",
      "  Read 1:    16,938,524 bp\n",
      "  Read 2:    16,936,353 bp\n",
      "Total written (filtered):     30,893,918 bp (91.2%)\n",
      "  Read 1:    15,447,241 bp\n",
      "  Read 2:    15,446,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t567\t0.0\t2\t0 545 22\n",
      "26\t56400\t0.0\t2\t54844 1529 27\n",
      "27\t244\t0.0\t2\t0 231 13\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t622\t0.0\t2\t0 588 34\n",
      "26\t56274\t0.0\t2\t54574 1623 77\n",
      "27\t287\t0.0\t2\t0 271 16\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S016_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S016_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,126 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.82 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,126\n",
      "  Read 1 with adapter:                  55,081 (99.9%)\n",
      "  Read 2 with adapter:                  55,058 (99.9%)\n",
      "Pairs written (passing filters):        55,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,523,930 bp\n",
      "  Read 1:    16,262,074 bp\n",
      "  Read 2:    16,261,856 bp\n",
      "Total written (filtered):     29,660,878 bp (91.2%)\n",
      "  Read 1:    14,830,183 bp\n",
      "  Read 2:    14,830,695 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55081 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t527\t0.0\t2\t0 516 11\n",
      "26\t54197\t0.0\t2\t52715 1449 33\n",
      "27\t246\t0.0\t2\t0 239 7\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t558\t0.0\t2\t0 544 14\n",
      "26\t54199\t0.0\t2\t52825 1333 41\n",
      "27\t223\t0.0\t2\t0 214 9\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S017_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S017_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,055 reads  @     86.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.08 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,055\n",
      "  Read 1 with adapter:                  46,927 (99.7%)\n",
      "  Read 2 with adapter:                  46,618 (99.1%)\n",
      "Pairs written (passing filters):        47,055 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,750,894 bp\n",
      "  Read 1:    13,876,162 bp\n",
      "  Read 2:    13,874,732 bp\n",
      "Total written (filtered):     25,320,683 bp (91.2%)\n",
      "  Read 1:    12,656,038 bp\n",
      "  Read 2:    12,664,645 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46927 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t402\t0.0\t2\t0 386 16\n",
      "26\t46132\t0.0\t2\t44861 1249 22\n",
      "27\t210\t0.0\t2\t0 200 10\n",
      "28\t145\t0.0\t2\t0 0 145\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t2120\t0.0\t2\t0 2059 61\n",
      "26\t44186\t0.0\t2\t42972 1169 45\n",
      "27\t237\t0.0\t2\t0 233 4\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S018_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S018_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,486 reads  @     76.9 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.66 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,486\n",
      "  Read 1 with adapter:                  47,253 (99.5%)\n",
      "  Read 2 with adapter:                  47,271 (99.5%)\n",
      "Pairs written (passing filters):        47,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,004,961 bp\n",
      "  Read 1:    13,998,814 bp\n",
      "  Read 2:    14,006,147 bp\n",
      "Total written (filtered):     25,548,007 bp (91.2%)\n",
      "  Read 1:    12,770,192 bp\n",
      "  Read 2:    12,777,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47253 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t335\t0.0\t2\t0 331 4\n",
      "26\t46545\t0.0\t2\t45482 1051 12\n",
      "27\t223\t0.0\t2\t0 214 9\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t882\t0.0\t2\t0 852 30\n",
      "26\t46105\t0.0\t2\t44818 1222 65\n",
      "27\t232\t0.0\t2\t0 218 14\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S019_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S019_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,412 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.29 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,412\n",
      "  Read 1 with adapter:                  31,287 (99.6%)\n",
      "  Read 2 with adapter:                  31,245 (99.5%)\n",
      "Pairs written (passing filters):        31,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,523,040 bp\n",
      "  Read 1:     9,261,342 bp\n",
      "  Read 2:     9,261,698 bp\n",
      "Total written (filtered):     16,897,757 bp (91.2%)\n",
      "  Read 1:     8,447,838 bp\n",
      "  Read 2:     8,449,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t225\t0.0\t2\t0 218 7\n",
      "26\t30841\t0.0\t2\t30136 696 9\n",
      "27\t115\t0.0\t2\t0 112 3\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t688\t0.0\t2\t0 665 23\n",
      "26\t30388\t0.0\t2\t29651 700 37\n",
      "27\t137\t0.0\t2\t0 129 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S020_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S020_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,191 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.09 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,191\n",
      "  Read 1 with adapter:                  40,065 (99.7%)\n",
      "  Read 2 with adapter:                  39,907 (99.3%)\n",
      "Pairs written (passing filters):        40,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,693,493 bp\n",
      "  Read 1:    11,851,145 bp\n",
      "  Read 2:    11,842,348 bp\n",
      "Total written (filtered):     21,614,563 bp (91.2%)\n",
      "  Read 1:    10,809,377 bp\n",
      "  Read 2:    10,805,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t326\t0.0\t2\t0 313 13\n",
      "26\t39393\t0.0\t2\t38378 999 16\n",
      "27\t172\t0.0\t2\t0 165 7\n",
      "28\t145\t0.0\t2\t0 0 145\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39907 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t541\t0.0\t2\t0 518 23\n",
      "26\t39120\t0.0\t2\t37686 1335 99\n",
      "27\t195\t0.0\t2\t0 175 20\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S021_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S021_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,366 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.80 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,366\n",
      "  Read 1 with adapter:                  46,232 (99.7%)\n",
      "  Read 2 with adapter:                  45,671 (98.5%)\n",
      "Pairs written (passing filters):        46,366 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,248,937 bp\n",
      "  Read 1:    13,673,276 bp\n",
      "  Read 2:    13,575,661 bp\n",
      "Total written (filtered):     24,860,178 bp (91.2%)\n",
      "  Read 1:    12,471,232 bp\n",
      "  Read 2:    12,388,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t376\t0.0\t2\t0 361 15\n",
      "26\t45501\t0.0\t2\t44308 1168 25\n",
      "27\t202\t0.0\t2\t0 199 3\n",
      "28\t123\t0.0\t2\t0 0 123\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45671 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t882\t0.0\t2\t0 839 43\n",
      "26\t44499\t0.0\t2\t42804 1559 136\n",
      "27\t213\t0.0\t2\t0 188 25\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S022_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S022_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,214 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.66 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,214\n",
      "  Read 1 with adapter:                  49,149 (99.9%)\n",
      "  Read 2 with adapter:                  48,738 (99.0%)\n",
      "Pairs written (passing filters):        49,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,950,792 bp\n",
      "  Read 1:    14,515,921 bp\n",
      "  Read 2:    14,434,871 bp\n",
      "Total written (filtered):     26,406,817 bp (91.2%)\n",
      "  Read 1:    13,238,328 bp\n",
      "  Read 2:    13,168,489 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t469\t0.0\t2\t0 455 14\n",
      "26\t48413\t0.0\t2\t47178 1217 18\n",
      "27\t162\t0.0\t2\t0 155 7\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t965\t0.0\t2\t0 925 40\n",
      "26\t47475\t0.0\t2\t46058 1340 77\n",
      "27\t177\t0.0\t2\t0 171 6\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S023_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S023_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,939 reads  @     76.5 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.51 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,939\n",
      "  Read 1 with adapter:                  58,822 (99.8%)\n",
      "  Read 2 with adapter:                  58,802 (99.8%)\n",
      "Pairs written (passing filters):        58,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,763,809 bp\n",
      "  Read 1:    17,378,988 bp\n",
      "  Read 2:    17,384,821 bp\n",
      "Total written (filtered):     31,706,360 bp (91.2%)\n",
      "  Read 1:    15,850,027 bp\n",
      "  Read 2:    15,856,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t620\t0.0\t2\t0 603 17\n",
      "26\t57936\t0.0\t2\t56365 1548 23\n",
      "27\t187\t0.0\t2\t0 183 4\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t661\t0.0\t2\t0 637 24\n",
      "26\t57755\t0.0\t2\t55917 1757 81\n",
      "27\t267\t0.0\t2\t0 257 10\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S024_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S024_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,524 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.76 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,524\n",
      "  Read 1 with adapter:                  50,397 (99.7%)\n",
      "  Read 2 with adapter:                  50,443 (99.8%)\n",
      "Pairs written (passing filters):        50,524 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,798,261 bp\n",
      "  Read 1:    14,894,438 bp\n",
      "  Read 2:    14,903,823 bp\n",
      "Total written (filtered):     27,177,043 bp (91.2%)\n",
      "  Read 1:    13,584,360 bp\n",
      "  Read 2:    13,592,683 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t490\t0.0\t2\t0 468 22\n",
      "26\t49615\t0.0\t2\t48194 1395 26\n",
      "27\t170\t0.0\t2\t0 165 5\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t593\t0.0\t2\t0 570 23\n",
      "26\t49542\t0.0\t2\t48138 1335 69\n",
      "27\t201\t0.0\t2\t0 194 7\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S025_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S025_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        20,219 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 1.67 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,219\n",
      "  Read 1 with adapter:                  20,198 (99.9%)\n",
      "  Read 2 with adapter:                  20,016 (99.0%)\n",
      "Pairs written (passing filters):        20,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,925,969 bp\n",
      "  Read 1:     5,964,065 bp\n",
      "  Read 2:     5,961,904 bp\n",
      "Total written (filtered):     10,881,311 bp (91.2%)\n",
      "  Read 1:     5,438,995 bp\n",
      "  Read 2:     5,442,316 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t192\t0.0\t2\t0 179 13\n",
      "26\t19884\t0.0\t2\t19115 740 29\n",
      "27\t70\t0.0\t2\t0 68 2\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t918\t0.0\t2\t0 897 21\n",
      "26\t18941\t0.0\t2\t18430 499 12\n",
      "27\t80\t0.0\t2\t0 75 5\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S026_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S026_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        68,110 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 4.61 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,110\n",
      "  Read 1 with adapter:                  68,057 (99.9%)\n",
      "  Read 2 with adapter:                  67,974 (99.8%)\n",
      "Pairs written (passing filters):        68,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,178,468 bp\n",
      "  Read 1:    20,091,620 bp\n",
      "  Read 2:    20,086,848 bp\n",
      "Total written (filtered):     36,643,070 bp (91.2%)\n",
      "  Read 1:    18,322,517 bp\n",
      "  Read 2:    18,320,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t616\t0.0\t2\t0 597 19\n",
      "26\t67101\t0.0\t2\t65112 1951 38\n",
      "27\t231\t0.0\t2\t0 225 6\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t84\t0.0\t2\t0 0 84\n",
      "25\t1328\t0.0\t2\t0 1284 44\n",
      "26\t66193\t0.0\t2\t64302 1826 65\n",
      "27\t271\t0.0\t2\t0 259 12\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S027_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S027_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,021 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.55 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,021\n",
      "  Read 1 with adapter:                  59,987 (99.9%)\n",
      "  Read 2 with adapter:                  59,850 (99.7%)\n",
      "Pairs written (passing filters):        60,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,488 bp\n",
      "  Read 1:    17,705,793 bp\n",
      "  Read 2:    17,704,695 bp\n",
      "Total written (filtered):     32,296,070 bp (91.2%)\n",
      "  Read 1:    16,146,229 bp\n",
      "  Read 2:    16,149,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t438\t0.0\t2\t0 427 11\n",
      "26\t59201\t0.0\t2\t57734 1445 22\n",
      "27\t212\t0.0\t2\t0 205 7\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t1415\t0.0\t2\t0 1380 35\n",
      "26\t58123\t0.0\t2\t56692 1387 44\n",
      "27\t231\t0.0\t2\t0 224 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S028_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S028_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,305 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.20 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,305\n",
      "  Read 1 with adapter:                  53,273 (99.9%)\n",
      "  Read 2 with adapter:                  52,961 (99.4%)\n",
      "Pairs written (passing filters):        53,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,419,020 bp\n",
      "  Read 1:    15,724,783 bp\n",
      "  Read 2:    15,694,237 bp\n",
      "Total written (filtered):     28,657,477 bp (91.2%)\n",
      "  Read 1:    14,339,729 bp\n",
      "  Read 2:    14,317,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t423\t0.0\t2\t0 405 18\n",
      "26\t52492\t0.0\t2\t51141 1323 28\n",
      "27\t217\t0.0\t2\t0 209 8\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t720\t0.0\t2\t0 700 20\n",
      "26\t51921\t0.0\t2\t50381 1456 84\n",
      "27\t265\t0.0\t2\t0 255 10\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S029_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S029_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,305 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.20 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,305\n",
      "  Read 1 with adapter:                  53,273 (99.9%)\n",
      "  Read 2 with adapter:                  52,961 (99.4%)\n",
      "Pairs written (passing filters):        53,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,419,020 bp\n",
      "  Read 1:    15,724,783 bp\n",
      "  Read 2:    15,694,237 bp\n",
      "Total written (filtered):     28,657,477 bp (91.2%)\n",
      "  Read 1:    14,339,729 bp\n",
      "  Read 2:    14,317,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t423\t0.0\t2\t0 405 18\n",
      "26\t52492\t0.0\t2\t51141 1323 28\n",
      "27\t217\t0.0\t2\t0 209 8\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t720\t0.0\t2\t0 700 20\n",
      "26\t51921\t0.0\t2\t50381 1456 84\n",
      "27\t265\t0.0\t2\t0 255 10\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S030_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S030_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,443 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.16 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,443\n",
      "  Read 1 with adapter:                  67,398 (99.9%)\n",
      "  Read 2 with adapter:                  67,227 (99.7%)\n",
      "Pairs written (passing filters):        67,443 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,770,902 bp\n",
      "  Read 1:    19,895,185 bp\n",
      "  Read 2:    19,875,717 bp\n",
      "Total written (filtered):     36,272,147 bp (91.2%)\n",
      "  Read 1:    18,142,988 bp\n",
      "  Read 2:    18,129,159 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67398 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t549\t0.0\t2\t0 528 21\n",
      "26\t66411\t0.0\t2\t64756 1624 31\n",
      "27\t262\t0.0\t2\t0 249 13\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t1457\t0.0\t2\t0 1414 43\n",
      "26\t65411\t0.0\t2\t63748 1603 60\n",
      "27\t261\t0.0\t2\t0 251 10\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S031_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S031_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,148 reads  @     92.0 µs/read;   0.65 M reads/minute\n",
      "Finished in 2.59 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,148\n",
      "  Read 1 with adapter:                  28,129 (99.9%)\n",
      "  Read 2 with adapter:                  28,028 (99.6%)\n",
      "Pairs written (passing filters):        28,148 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,604,448 bp\n",
      "  Read 1:     8,303,378 bp\n",
      "  Read 2:     8,301,070 bp\n",
      "Total written (filtered):     15,144,623 bp (91.2%)\n",
      "  Read 1:     7,572,079 bp\n",
      "  Read 2:     7,572,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t264\t0.0\t2\t0 256 8\n",
      "26\t27678\t0.0\t2\t26770 892 16\n",
      "27\t101\t0.0\t2\t0 99 2\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28028 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t327\t0.0\t2\t0 311 16\n",
      "26\t27518\t0.0\t2\t26326 1112 80\n",
      "27\t145\t0.0\t2\t0 133 12\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S032_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S032_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,776 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 5.01 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,776\n",
      "  Read 1 with adapter:                  66,730 (99.9%)\n",
      "  Read 2 with adapter:                  66,683 (99.9%)\n",
      "Pairs written (passing filters):        66,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,397,377 bp\n",
      "  Read 1:    19,698,812 bp\n",
      "  Read 2:    19,698,565 bp\n",
      "Total written (filtered):     35,929,380 bp (91.2%)\n",
      "  Read 1:    17,964,115 bp\n",
      "  Read 2:    17,965,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t594\t0.0\t2\t0 569 25\n",
      "26\t65786\t0.0\t2\t64002 1755 29\n",
      "27\t225\t0.0\t2\t0 215 10\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t708\t0.0\t2\t0 683 25\n",
      "26\t65597\t0.0\t2\t63982 1578 37\n",
      "27\t254\t0.0\t2\t0 247 7\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S033_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S033_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,390 reads  @     80.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.74 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,390\n",
      "  Read 1 with adapter:                  46,355 (99.9%)\n",
      "  Read 2 with adapter:                  46,026 (99.2%)\n",
      "Pairs written (passing filters):        46,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,364,714 bp\n",
      "  Read 1:    13,684,915 bp\n",
      "  Read 2:    13,679,799 bp\n",
      "Total written (filtered):     24,965,142 bp (91.2%)\n",
      "  Read 1:    12,479,971 bp\n",
      "  Read 2:    12,485,171 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t488\t0.0\t2\t0 471 17\n",
      "26\t45602\t0.0\t2\t44117 1449 36\n",
      "27\t180\t0.0\t2\t0 173 7\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t2182\t0.0\t2\t0 2127 55\n",
      "26\t43563\t0.0\t2\t42519 1017 27\n",
      "27\t192\t0.0\t2\t0 181 11\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S034_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S034_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,163 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.60 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,163\n",
      "  Read 1 with adapter:                  63,085 (99.9%)\n",
      "  Read 2 with adapter:                  63,057 (99.8%)\n",
      "Pairs written (passing filters):        63,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,262,409 bp\n",
      "  Read 1:    18,631,273 bp\n",
      "  Read 2:    18,631,136 bp\n",
      "Total written (filtered):     33,983,975 bp (91.2%)\n",
      "  Read 1:    16,991,384 bp\n",
      "  Read 2:    16,992,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t578\t0.0\t2\t0 558 20\n",
      "26\t62199\t0.0\t2\t60614 1552 33\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t1174\t0.0\t2\t0 1145 29\n",
      "26\t61530\t0.0\t2\t59934 1531 65\n",
      "27\t281\t0.0\t2\t0 269 12\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S035_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S035_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,974 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.95 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,974\n",
      "  Read 1 with adapter:                  60,899 (99.9%)\n",
      "  Read 2 with adapter:                  60,868 (99.8%)\n",
      "Pairs written (passing filters):        60,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,970,814 bp\n",
      "  Read 1:    17,985,154 bp\n",
      "  Read 2:    17,985,660 bp\n",
      "Total written (filtered):     32,806,358 bp (91.2%)\n",
      "  Read 1:    16,402,011 bp\n",
      "  Read 2:    16,404,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t570\t0.0\t2\t0 547 23\n",
      "26\t59957\t0.0\t2\t58298 1624 35\n",
      "27\t253\t0.0\t2\t0 244 9\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t1408\t0.0\t2\t0 1381 27\n",
      "26\t59108\t0.0\t2\t57655 1423 30\n",
      "27\t259\t0.0\t2\t0 250 9\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S036_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S036_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,938 reads  @     84.8 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.73 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,938\n",
      "  Read 1 with adapter:                  43,854 (99.8%)\n",
      "  Read 2 with adapter:                  43,656 (99.4%)\n",
      "Pairs written (passing filters):        43,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,900,364 bp\n",
      "  Read 1:    12,959,857 bp\n",
      "  Read 2:    12,940,507 bp\n",
      "Total written (filtered):     23,625,665 bp (91.2%)\n",
      "  Read 1:    11,819,819 bp\n",
      "  Read 2:    11,805,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t407\t0.0\t2\t0 396 11\n",
      "26\t43186\t0.0\t2\t42037 1132 17\n",
      "27\t181\t0.0\t2\t0 173 8\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t579\t0.0\t2\t0 549 30\n",
      "26\t42833\t0.0\t2\t41439 1319 75\n",
      "27\t200\t0.0\t2\t0 185 15\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S037_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S037_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,320 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.16 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,320\n",
      "  Read 1 with adapter:                  35,241 (99.8%)\n",
      "  Read 2 with adapter:                  34,362 (97.3%)\n",
      "Pairs written (passing filters):        35,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,678,343 bp\n",
      "  Read 1:    10,417,144 bp\n",
      "  Read 2:    10,261,199 bp\n",
      "Total written (filtered):     18,869,417 bp (91.3%)\n",
      "  Read 1:     9,501,014 bp\n",
      "  Read 2:     9,368,403 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t333\t0.0\t2\t0 319 14\n",
      "26\t34707\t0.0\t2\t33658 1029 20\n",
      "27\t145\t0.0\t2\t0 138 7\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t701\t0.0\t2\t0 669 32\n",
      "26\t33455\t0.0\t2\t32311 1087 57\n",
      "27\t147\t0.0\t2\t0 136 11\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S038_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S038_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        68,497 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.49 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,497\n",
      "  Read 1 with adapter:                  68,420 (99.9%)\n",
      "  Read 2 with adapter:                  68,228 (99.6%)\n",
      "Pairs written (passing filters):        68,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,382,365 bp\n",
      "  Read 1:    20,204,720 bp\n",
      "  Read 2:    20,177,645 bp\n",
      "Total written (filtered):     36,830,574 bp (91.2%)\n",
      "  Read 1:    18,425,545 bp\n",
      "  Read 2:    18,405,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68420 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t664\t0.0\t2\t0 641 23\n",
      "26\t67365\t0.0\t2\t65574 1760 31\n",
      "27\t273\t0.0\t2\t0 264 9\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t91\t0.0\t2\t0 0 91\n",
      "25\t1497\t0.0\t2\t0 1455 42\n",
      "26\t66292\t0.0\t2\t64476 1754 62\n",
      "27\t329\t0.0\t2\t0 316 13\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S039_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S039_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,193 reads  @     80.8 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.60 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,193\n",
      "  Read 1 with adapter:                  69,111 (99.9%)\n",
      "  Read 2 with adapter:                  69,031 (99.8%)\n",
      "Pairs written (passing filters):        69,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,811,644 bp\n",
      "  Read 1:    20,410,442 bp\n",
      "  Read 2:    20,401,202 bp\n",
      "Total written (filtered):     37,220,785 bp (91.2%)\n",
      "  Read 1:    18,613,849 bp\n",
      "  Read 2:    18,606,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69111 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t618\t0.0\t2\t0 606 12\n",
      "26\t68089\t0.0\t2\t66296 1746 47\n",
      "27\t275\t0.0\t2\t0 265 10\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t802\t0.0\t2\t0 776 26\n",
      "26\t67822\t0.0\t2\t65843 1874 105\n",
      "27\t336\t0.0\t2\t0 319 17\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S040_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S040_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,414 reads  @    140.3 µs/read;   0.43 M reads/minute\n",
      "Finished in 0.34 s (141 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,414\n",
      "  Read 1 with adapter:                   2,374 (98.3%)\n",
      "  Read 2 with adapter:                   2,373 (98.3%)\n",
      "Pairs written (passing filters):         2,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,423,006 bp\n",
      "  Read 1:       710,878 bp\n",
      "  Read 2:       712,128 bp\n",
      "Total written (filtered):      1,299,620 bp (91.3%)\n",
      "  Read 1:       649,166 bp\n",
      "  Read 2:       650,454 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 2374 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t25\t0.0\t2\t0 24 1\n",
      "26\t2336\t0.0\t2\t2253 82 1\n",
      "27\t5\t0.0\t2\t0 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 2373 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t36\t0.0\t2\t0 34 2\n",
      "26\t2320\t0.0\t2\t2233 86 1\n",
      "27\t10\t0.0\t2\t0 10\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S041_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S041_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S041_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S041_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S041_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S041_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S041_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S041_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,089 reads  @     80.0 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.33 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,089\n",
      "  Read 1 with adapter:                  29,068 (99.9%)\n",
      "  Read 2 with adapter:                  28,858 (99.2%)\n",
      "Pairs written (passing filters):        29,089 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,158,681 bp\n",
      "  Read 1:     8,580,915 bp\n",
      "  Read 2:     8,577,766 bp\n",
      "Total written (filtered):     15,653,916 bp (91.2%)\n",
      "  Read 1:     7,825,086 bp\n",
      "  Read 2:     7,828,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29068 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t292\t0.0\t2\t0 283 9\n",
      "26\t28467\t0.0\t2\t27650 794 23\n",
      "27\t173\t0.0\t2\t0 160 13\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t1422\t0.0\t2\t0 1389 33\n",
      "26\t27275\t0.0\t2\t26616 642 17\n",
      "27\t112\t0.0\t2\t0 110 2\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S042_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S042_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S042_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S042_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S042_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S042_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S042_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S042_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,202 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.69 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,202\n",
      "  Read 1 with adapter:                  50,171 (99.9%)\n",
      "  Read 2 with adapter:                  50,076 (99.7%)\n",
      "Pairs written (passing filters):        50,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,618,012 bp\n",
      "  Read 1:    14,809,583 bp\n",
      "  Read 2:    14,808,429 bp\n",
      "Total written (filtered):     27,012,354 bp (91.2%)\n",
      "  Read 1:    13,505,198 bp\n",
      "  Read 2:    13,507,156 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t460\t0.0\t2\t0 448 12\n",
      "26\t49311\t0.0\t2\t48241 1054 16\n",
      "27\t241\t0.0\t2\t0 236 5\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t855\t0.0\t2\t0 835 20\n",
      "26\t48956\t0.0\t2\t47784 1128 44\n",
      "27\t222\t0.0\t2\t0 213 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S043_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S043_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S043_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S043_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S043_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S043_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S043_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S043_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07       100,648 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 7.56 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,648\n",
      "  Read 1 with adapter:                 100,577 (99.9%)\n",
      "  Read 2 with adapter:                 100,409 (99.8%)\n",
      "Pairs written (passing filters):       100,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,381,312 bp\n",
      "  Read 1:    29,690,783 bp\n",
      "  Read 2:    29,690,529 bp\n",
      "Total written (filtered):     54,158,050 bp (91.2%)\n",
      "  Read 1:    27,076,054 bp\n",
      "  Read 2:    27,081,996 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 100577 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t989\t0.0\t2\t0 955 34\n",
      "26\t98879\t0.0\t2\t96546 2287 46\n",
      "27\t450\t0.0\t2\t0 433 17\n",
      "28\t196\t0.0\t2\t0 0 196\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 100409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t104\t0.0\t2\t0 0 104\n",
      "25\t2365\t0.0\t2\t0 2300 65\n",
      "26\t97483\t0.0\t2\t95250 2161 72\n",
      "27\t442\t0.0\t2\t0 430 12\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S044_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S044_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S044_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S044_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S044_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S044_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S044_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S044_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,321 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.97 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,321\n",
      "  Read 1 with adapter:                  46,282 (99.9%)\n",
      "  Read 2 with adapter:                  45,744 (98.8%)\n",
      "Pairs written (passing filters):        46,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,247,994 bp\n",
      "  Read 1:    13,664,306 bp\n",
      "  Read 2:    13,583,688 bp\n",
      "Total written (filtered):     24,855,948 bp (91.2%)\n",
      "  Read 1:    12,461,191 bp\n",
      "  Read 2:    12,394,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46282 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t474\t0.0\t2\t0 461 13\n",
      "26\t45514\t0.0\t2\t44381 1116 17\n",
      "27\t223\t0.0\t2\t0 213 10\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t588\t0.0\t2\t0 561 27\n",
      "26\t44903\t0.0\t2\t43645 1198 60\n",
      "27\t211\t0.0\t2\t0 196 15\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S045_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S045_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S045_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S045_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S045_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S045_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S045_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S045_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,667 reads  @     91.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.84 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,667\n",
      "  Read 1 with adapter:                  52,641 (100.0%)\n",
      "  Read 2 with adapter:                  45,637 (86.7%)\n",
      "Pairs written (passing filters):        52,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,855,874 bp\n",
      "  Read 1:    15,536,450 bp\n",
      "  Read 2:    14,319,424 bp\n",
      "Total written (filtered):     27,301,502 bp (91.4%)\n",
      "  Read 1:    14,167,927 bp\n",
      "  Read 2:    13,133,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t578\t0.0\t2\t0 556 22\n",
      "26\t51816\t0.0\t2\t50574 1228 14\n",
      "27\t180\t0.0\t2\t0 176 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t871\t0.0\t2\t0 847 24\n",
      "26\t44507\t0.0\t2\t43214 1228 65\n",
      "27\t188\t0.0\t2\t0 179 9\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S046_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S046_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S046_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S046_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S046_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S046_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S046_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S046_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,360 reads  @     85.9 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.24 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,360\n",
      "  Read 1 with adapter:                  49,325 (99.9%)\n",
      "  Read 2 with adapter:                  49,160 (99.6%)\n",
      "Pairs written (passing filters):        49,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,099,179 bp\n",
      "  Read 1:    14,560,706 bp\n",
      "  Read 2:    14,538,473 bp\n",
      "Total written (filtered):     26,539,589 bp (91.2%)\n",
      "  Read 1:    13,278,352 bp\n",
      "  Read 2:    13,261,237 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49325 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t549\t0.0\t2\t0 532 17\n",
      "26\t48547\t0.0\t2\t47409 1122 16\n",
      "27\t158\t0.0\t2\t0 154 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49160 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t1079\t0.0\t2\t0 1051 28\n",
      "26\t47795\t0.0\t2\t46586 1158 51\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S047_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S047_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S047_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S047_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S047_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S047_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S047_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S047_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,840 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.78 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,840\n",
      "  Read 1 with adapter:                  54,793 (99.9%)\n",
      "  Read 2 with adapter:                  54,739 (99.8%)\n",
      "Pairs written (passing filters):        54,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,352,352 bp\n",
      "  Read 1:    16,176,947 bp\n",
      "  Read 2:    16,175,405 bp\n",
      "Total written (filtered):     29,504,761 bp (91.2%)\n",
      "  Read 1:    14,752,192 bp\n",
      "  Read 2:    14,752,569 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t533\t0.0\t2\t0 521 12\n",
      "26\t53852\t0.0\t2\t52542 1291 19\n",
      "27\t263\t0.0\t2\t0 258 5\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t606\t0.0\t2\t0 587 19\n",
      "26\t53799\t0.0\t2\t52463 1272 64\n",
      "27\t256\t0.0\t2\t0 248 8\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S048_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S048_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S048_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S048_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S048_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S048_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S048_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S048_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        53,720 reads  @     99.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.35 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,720\n",
      "  Read 1 with adapter:                  53,683 (99.9%)\n",
      "  Read 2 with adapter:                  53,644 (99.9%)\n",
      "Pairs written (passing filters):        53,720 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,693,849 bp\n",
      "  Read 1:    15,846,833 bp\n",
      "  Read 2:    15,847,016 bp\n",
      "Total written (filtered):     28,903,989 bp (91.2%)\n",
      "  Read 1:    14,451,327 bp\n",
      "  Read 2:    14,452,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t645\t0.0\t2\t0 622 23\n",
      "26\t52612\t0.0\t2\t51123 1453 36\n",
      "27\t263\t0.0\t2\t0 258 5\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t589\t0.0\t2\t0 561 28\n",
      "26\t52728\t0.0\t2\t51320 1362 46\n",
      "27\t231\t0.0\t2\t0 215 16\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S049_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S049_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S049_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S049_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S049_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S049_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S049_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S049_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        33,711 reads  @     96.6 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.26 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,711\n",
      "  Read 1 with adapter:                  33,679 (99.9%)\n",
      "  Read 2 with adapter:                  33,439 (99.2%)\n",
      "Pairs written (passing filters):        33,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,881,163 bp\n",
      "  Read 1:     9,940,572 bp\n",
      "  Read 2:     9,940,591 bp\n",
      "Total written (filtered):     18,137,433 bp (91.2%)\n",
      "  Read 1:     9,064,770 bp\n",
      "  Read 2:     9,072,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t240\t0.0\t2\t0 226 14\n",
      "26\t33123\t0.0\t2\t32090 1011 22\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t144\t0.0\t2\t0 0 144\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33439 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t1594\t0.0\t2\t0 1561 33\n",
      "26\t31629\t0.0\t2\t30918 692 19\n",
      "27\t132\t0.0\t2\t0 129 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S050_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S050_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S050_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S050_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S050_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S050_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S050_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S050_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,642 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.71 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,642\n",
      "  Read 1 with adapter:                  61,597 (99.9%)\n",
      "  Read 2 with adapter:                  61,532 (99.8%)\n",
      "Pairs written (passing filters):        61,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,363,311 bp\n",
      "  Read 1:    18,181,070 bp\n",
      "  Read 2:    18,182,241 bp\n",
      "Total written (filtered):     33,162,818 bp (91.2%)\n",
      "  Read 1:    16,579,462 bp\n",
      "  Read 2:    16,583,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t369\t0.0\t2\t0 356 13\n",
      "26\t60829\t0.0\t2\t59609 1207 13\n",
      "27\t219\t0.0\t2\t0 213 6\n",
      "28\t149\t0.0\t2\t0 0 149\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1114\t0.0\t2\t0 1081 33\n",
      "26\t60084\t0.0\t2\t58802 1225 57\n",
      "27\t265\t0.0\t2\t0 254 11\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S051_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S051_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S051_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S051_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S051_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S051_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S051_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S051_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,977 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.68 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,977\n",
      "  Read 1 with adapter:                  58,931 (99.9%)\n",
      "  Read 2 with adapter:                  58,887 (99.8%)\n",
      "Pairs written (passing filters):        58,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,791,286 bp\n",
      "  Read 1:    17,395,411 bp\n",
      "  Read 2:    17,395,875 bp\n",
      "Total written (filtered):     31,729,238 bp (91.2%)\n",
      "  Read 1:    15,863,139 bp\n",
      "  Read 2:    15,866,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58931 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t417\t0.0\t2\t0 403 14\n",
      "26\t58111\t0.0\t2\t56807 1283 21\n",
      "27\t235\t0.0\t2\t0 229 6\n",
      "28\t146\t0.0\t2\t0 0 146\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t1439\t0.0\t2\t0 1393 46\n",
      "26\t57148\t0.0\t2\t55946 1158 44\n",
      "27\t243\t0.0\t2\t0 234 9\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S052_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S052_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S052_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S052_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S052_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S052_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S052_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S052_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        51,244 reads  @    106.4 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.46 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,244\n",
      "  Read 1 with adapter:                  51,109 (99.7%)\n",
      "  Read 2 with adapter:                  50,512 (98.6%)\n",
      "Pairs written (passing filters):        51,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,106,857 bp\n",
      "  Read 1:    15,092,209 bp\n",
      "  Read 2:    15,014,648 bp\n",
      "Total written (filtered):     27,465,186 bp (91.2%)\n",
      "  Read 1:    13,763,401 bp\n",
      "  Read 2:    13,701,785 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51109 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t387\t0.0\t2\t0 380 7\n",
      "26\t50379\t0.0\t2\t49218 1151 10\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50512 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t629\t0.0\t2\t0 598 31\n",
      "26\t49592\t0.0\t2\t48242 1273 77\n",
      "27\t234\t0.0\t2\t0 218 16\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S053_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S053_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S053_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S053_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S053_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S053_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S053_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S053_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        68,624 reads  @    112.5 µs/read;   0.53 M reads/minute\n",
      "Finished in 7.73 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,624\n",
      "  Read 1 with adapter:                  68,366 (99.6%)\n",
      "  Read 2 with adapter:                  63,077 (91.9%)\n",
      "Pairs written (passing filters):        68,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,483,854 bp\n",
      "  Read 1:    20,185,585 bp\n",
      "  Read 2:    19,298,269 bp\n",
      "Total written (filtered):     36,067,654 bp (91.3%)\n",
      "  Read 1:    18,408,288 bp\n",
      "  Read 2:    17,659,366 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t536\t0.0\t2\t0 521 15\n",
      "26\t67446\t0.0\t2\t65815 1606 25\n",
      "27\t255\t0.0\t2\t0 245 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1257\t0.0\t2\t0 1220 37\n",
      "26\t61484\t0.0\t2\t59832 1574 78\n",
      "27\t230\t0.0\t2\t0 213 17\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S054_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S054_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S054_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S054_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S054_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S054_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S054_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S054_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        65,038 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.46 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,038\n",
      "  Read 1 with adapter:                  64,791 (99.6%)\n",
      "  Read 2 with adapter:                  64,780 (99.6%)\n",
      "Pairs written (passing filters):        65,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,286,601 bp\n",
      "  Read 1:    19,133,842 bp\n",
      "  Read 2:    19,152,759 bp\n",
      "Total written (filtered):     34,918,394 bp (91.2%)\n",
      "  Read 1:    17,448,772 bp\n",
      "  Read 2:    17,469,622 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t573\t0.0\t2\t0 560 13\n",
      "26\t63876\t0.0\t2\t62308 1537 31\n",
      "27\t242\t0.0\t2\t0 233 9\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t1305\t0.0\t2\t0 1271 34\n",
      "26\t63117\t0.0\t2\t61647 1422 48\n",
      "27\t242\t0.0\t2\t0 233 9\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S055_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S055_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S055_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S055_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S055_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S055_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S055_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S055_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,827 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.96 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,827\n",
      "  Read 1 with adapter:                  65,589 (99.6%)\n",
      "  Read 2 with adapter:                  65,718 (99.8%)\n",
      "Pairs written (passing filters):        65,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,784,842 bp\n",
      "  Read 1:    19,367,591 bp\n",
      "  Read 2:    19,417,251 bp\n",
      "Total written (filtered):     35,371,519 bp (91.2%)\n",
      "  Read 1:    17,662,467 bp\n",
      "  Read 2:    17,709,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t529\t0.0\t2\t0 518 11\n",
      "26\t64646\t0.0\t2\t63017 1605 24\n",
      "27\t273\t0.0\t2\t0 259 14\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t797\t0.0\t2\t0 763 34\n",
      "26\t64468\t0.0\t2\t62662 1754 52\n",
      "27\t358\t0.0\t2\t0 344 14\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S056_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S056_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S056_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S056_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S056_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S056_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S056_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S056_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        75,333 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 7.41 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,333\n",
      "  Read 1 with adapter:                  74,991 (99.5%)\n",
      "  Read 2 with adapter:                  75,216 (99.8%)\n",
      "Pairs written (passing filters):        75,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,364,372 bp\n",
      "  Read 1:    22,142,751 bp\n",
      "  Read 2:    22,221,621 bp\n",
      "Total written (filtered):     40,459,820 bp (91.2%)\n",
      "  Read 1:    20,193,212 bp\n",
      "  Read 2:    20,266,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t654\t0.0\t2\t0 631 23\n",
      "26\t73877\t0.0\t2\t71919 1924 34\n",
      "27\t301\t0.0\t2\t0 292 9\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t841\t0.0\t2\t0 811 30\n",
      "26\t73951\t0.0\t2\t72187 1706 58\n",
      "27\t298\t0.0\t2\t0 292 6\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S057_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S057_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S057_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S057_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S057_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S057_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S057_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S057_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,702 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.50 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,702\n",
      "  Read 1 with adapter:                  55,657 (99.9%)\n",
      "  Read 2 with adapter:                  55,189 (99.1%)\n",
      "Pairs written (passing filters):        55,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,856,311 bp\n",
      "  Read 1:    16,431,388 bp\n",
      "  Read 2:    16,424,923 bp\n",
      "Total written (filtered):     29,976,676 bp (91.2%)\n",
      "  Read 1:    14,984,313 bp\n",
      "  Read 2:    14,992,363 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t421\t0.0\t2\t0 406 15\n",
      "26\t54839\t0.0\t2\t53482 1333 24\n",
      "27\t212\t0.0\t2\t0 206 6\n",
      "28\t143\t0.0\t2\t0 0 143\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t2482\t0.0\t2\t0 2424 58\n",
      "26\t52306\t0.0\t2\t51002 1263 41\n",
      "27\t298\t0.0\t2\t0 280 18\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S058_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S058_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S058_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S058_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S058_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S058_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S058_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S058_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,712 reads  @     92.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.80 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,712\n",
      "  Read 1 with adapter:                  51,679 (99.9%)\n",
      "  Read 2 with adapter:                  51,572 (99.7%)\n",
      "Pairs written (passing filters):        51,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,507,595 bp\n",
      "  Read 1:    15,254,469 bp\n",
      "  Read 2:    15,253,126 bp\n",
      "Total written (filtered):     27,823,840 bp (91.2%)\n",
      "  Read 1:    13,910,851 bp\n",
      "  Read 2:    13,912,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t416\t0.0\t2\t0 402 14\n",
      "26\t50861\t0.0\t2\t49790 1058 13\n",
      "27\t264\t0.0\t2\t0 257 7\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t903\t0.0\t2\t0 882 21\n",
      "26\t50394\t0.0\t2\t49159 1184 51\n",
      "27\t226\t0.0\t2\t0 217 9\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S059_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S059_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S059_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S059_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S059_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S059_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S059_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S059_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,424 reads  @     84.9 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.63 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,424\n",
      "  Read 1 with adapter:                  54,391 (99.9%)\n",
      "  Read 2 with adapter:                  54,321 (99.8%)\n",
      "Pairs written (passing filters):        54,424 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,108,571 bp\n",
      "  Read 1:    16,054,406 bp\n",
      "  Read 2:    16,054,165 bp\n",
      "Total written (filtered):     29,283,261 bp (91.2%)\n",
      "  Read 1:    14,640,182 bp\n",
      "  Read 2:    14,643,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t392\t0.0\t2\t0 378 14\n",
      "26\t53601\t0.0\t2\t52570 1013 18\n",
      "27\t238\t0.0\t2\t0 231 7\n",
      "28\t133\t0.0\t2\t0 0 133\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t1371\t0.0\t2\t0 1332 39\n",
      "26\t52664\t0.0\t2\t51388 1226 50\n",
      "27\t221\t0.0\t2\t0 209 12\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S060_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S060_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S060_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S060_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S060_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S060_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S060_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S060_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,063 reads  @     89.4 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.85 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,063\n",
      "  Read 1 with adapter:                  43,037 (99.9%)\n",
      "  Read 2 with adapter:                  42,788 (99.4%)\n",
      "Pairs written (passing filters):        43,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,384,317 bp\n",
      "  Read 1:    12,703,200 bp\n",
      "  Read 2:    12,681,117 bp\n",
      "Total written (filtered):     23,153,169 bp (91.2%)\n",
      "  Read 1:    11,584,163 bp\n",
      "  Read 2:    11,569,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t286\t0.0\t2\t0 275 11\n",
      "26\t42438\t0.0\t2\t41562 862 14\n",
      "27\t169\t0.0\t2\t0 163 6\n",
      "28\t120\t0.0\t2\t0 0 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42788 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t525\t0.0\t2\t0 504 21\n",
      "26\t42033\t0.0\t2\t40867 1096 70\n",
      "27\t188\t0.0\t2\t0 174 14\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S061_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S061_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S061_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S061_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S061_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S061_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S061_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S061_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,251 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.67 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,251\n",
      "  Read 1 with adapter:                  57,193 (99.9%)\n",
      "  Read 2 with adapter:                  56,626 (98.9%)\n",
      "Pairs written (passing filters):        57,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,682,939 bp\n",
      "  Read 1:    16,883,510 bp\n",
      "  Read 2:    16,799,429 bp\n",
      "Total written (filtered):     30,724,554 bp (91.2%)\n",
      "  Read 1:    15,396,477 bp\n",
      "  Read 2:    15,328,077 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t431\t0.0\t2\t0 416 15\n",
      "26\t56356\t0.0\t2\t55044 1291 21\n",
      "27\t246\t0.0\t2\t0 242 4\n",
      "28\t130\t0.0\t2\t0 0 130\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1066\t0.0\t2\t0 1022 44\n",
      "26\t55221\t0.0\t2\t53441 1664 116\n",
      "27\t252\t0.0\t2\t0 236 16\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S062_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S062_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S062_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S062_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S062_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S062_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S062_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S062_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,513 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.91 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,513\n",
      "  Read 1 with adapter:                  64,480 (99.9%)\n",
      "  Read 2 with adapter:                  63,941 (99.1%)\n",
      "Pairs written (passing filters):        64,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,963,887 bp\n",
      "  Read 1:    19,029,827 bp\n",
      "  Read 2:    18,934,060 bp\n",
      "Total written (filtered):     34,625,503 bp (91.2%)\n",
      "  Read 1:    17,352,794 bp\n",
      "  Read 2:    17,272,709 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t486\t0.0\t2\t0 469 17\n",
      "26\t63682\t0.0\t2\t62297 1361 24\n",
      "27\t188\t0.0\t2\t0 185 3\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63941 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t1392\t0.0\t2\t0 1348 44\n",
      "26\t62106\t0.0\t2\t60469 1573 64\n",
      "27\t233\t0.0\t2\t0 219 14\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S063_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S063_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S063_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S063_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S063_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S063_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S063_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S063_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,375 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 5.09 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,375\n",
      "  Read 1 with adapter:                  67,314 (99.9%)\n",
      "  Read 2 with adapter:                  67,239 (99.8%)\n",
      "Pairs written (passing filters):        67,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,746,356 bp\n",
      "  Read 1:    19,873,639 bp\n",
      "  Read 2:    19,872,717 bp\n",
      "Total written (filtered):     36,248,787 bp (91.2%)\n",
      "  Read 1:    18,123,807 bp\n",
      "  Read 2:    18,124,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t550\t0.0\t2\t0 529 21\n",
      "26\t66449\t0.0\t2\t64878 1543 28\n",
      "27\t204\t0.0\t2\t0 201 3\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t784\t0.0\t2\t0 759 25\n",
      "26\t66038\t0.0\t2\t64229 1729 80\n",
      "27\t287\t0.0\t2\t0 278 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S064_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S064_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S064_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S064_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S064_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S064_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S064_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S064_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        72,830 reads  @     78.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 5.71 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,830\n",
      "  Read 1 with adapter:                  72,779 (99.9%)\n",
      "  Read 2 with adapter:                  72,724 (99.9%)\n",
      "Pairs written (passing filters):        72,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,967,981 bp\n",
      "  Read 1:    21,483,754 bp\n",
      "  Read 2:    21,484,227 bp\n",
      "Total written (filtered):     39,185,741 bp (91.2%)\n",
      "  Read 1:    19,591,837 bp\n",
      "  Read 2:    19,593,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72779 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t657\t0.0\t2\t0 632 25\n",
      "26\t71747\t0.0\t2\t69830 1886 31\n",
      "27\t226\t0.0\t2\t0 220 6\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72724 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t835\t0.0\t2\t0 804 31\n",
      "26\t71428\t0.0\t2\t69557 1799 72\n",
      "27\t284\t0.0\t2\t0 273 11\n",
      "28\t101\t0.0\t2\t0 0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S065_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S065_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S065_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S065_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S065_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S065_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S065_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S065_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,841 reads  @     79.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.36 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,841\n",
      "  Read 1 with adapter:                  54,406 (99.2%)\n",
      "  Read 2 with adapter:                  54,293 (99.0%)\n",
      "Pairs written (passing filters):        54,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,285,776 bp\n",
      "  Read 1:    16,115,741 bp\n",
      "  Read 2:    16,170,035 bp\n",
      "Total written (filtered):     29,462,108 bp (91.3%)\n",
      "  Read 1:    14,701,361 bp\n",
      "  Read 2:    14,760,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t468\t0.0\t2\t0 445 23\n",
      "26\t53590\t0.0\t2\t52034 1525 31\n",
      "27\t216\t0.0\t2\t0 211 5\n",
      "28\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t98\t0.0\t2\t0 0 98\n",
      "25\t2548\t0.0\t2\t0 2463 85\n",
      "26\t51328\t0.0\t2\t49909 1369 50\n",
      "27\t224\t0.0\t2\t0 213 11\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S066_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S066_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S066_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S066_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S066_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S066_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S066_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S066_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S067_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S067_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S067_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S067_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S067_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S067_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S067_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S067_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        59,890 reads  @     87.3 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.23 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,890\n",
      "  Read 1 with adapter:                  59,783 (99.8%)\n",
      "  Read 2 with adapter:                  59,772 (99.8%)\n",
      "Pairs written (passing filters):        59,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,323,806 bp\n",
      "  Read 1:    17,657,191 bp\n",
      "  Read 2:    17,666,615 bp\n",
      "Total written (filtered):     32,216,738 bp (91.2%)\n",
      "  Read 1:    16,102,883 bp\n",
      "  Read 2:    16,113,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t443\t0.0\t2\t0 431 12\n",
      "26\t58928\t0.0\t2\t57542 1363 23\n",
      "27\t247\t0.0\t2\t0 237 10\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t1507\t0.0\t2\t0 1469 38\n",
      "26\t57887\t0.0\t2\t56631 1212 44\n",
      "27\t289\t0.0\t2\t0 280 9\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S068_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S068_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S068_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S068_135_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S068_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S068_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S068_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S068_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        40,854 reads  @    100.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.10 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,854\n",
      "  Read 1 with adapter:                  40,774 (99.8%)\n",
      "  Read 2 with adapter:                  40,511 (99.2%)\n",
      "Pairs written (passing filters):        40,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,054,421 bp\n",
      "  Read 1:    12,044,084 bp\n",
      "  Read 2:    12,010,337 bp\n",
      "Total written (filtered):     21,941,403 bp (91.2%)\n",
      "  Read 1:    10,983,950 bp\n",
      "  Read 2:    10,957,453 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40774 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t293\t0.0\t2\t0 287 6\n",
      "26\t40203\t0.0\t2\t39257 921 25\n",
      "27\t165\t0.0\t2\t0 162 3\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t532\t0.0\t2\t0 525 7\n",
      "26\t39766\t0.0\t2\t38708 987 71\n",
      "27\t168\t0.0\t2\t0 160 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S069_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S069_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S069_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S069_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S069_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S069_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S069_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S069_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,083 reads  @     93.7 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.17 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,083\n",
      "  Read 1 with adapter:                  54,990 (99.8%)\n",
      "  Read 2 with adapter:                  54,033 (98.1%)\n",
      "Pairs written (passing filters):        55,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,324,159 bp\n",
      "  Read 1:    16,239,479 bp\n",
      "  Read 2:    16,084,680 bp\n",
      "Total written (filtered):     29,490,481 bp (91.2%)\n",
      "  Read 1:    14,809,757 bp\n",
      "  Read 2:    14,680,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t402\t0.0\t2\t0 393 9\n",
      "26\t54207\t0.0\t2\t52938 1246 23\n",
      "27\t230\t0.0\t2\t0 226 4\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1036\t0.0\t2\t0 1006 30\n",
      "26\t52680\t0.0\t2\t51171 1427 82\n",
      "27\t216\t0.0\t2\t0 201 15\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S070_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S070_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S070_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S070_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S070_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S070_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S070_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S070_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        68,046 reads  @    100.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.85 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,046\n",
      "  Read 1 with adapter:                  67,942 (99.8%)\n",
      "  Read 2 with adapter:                  67,838 (99.7%)\n",
      "Pairs written (passing filters):        68,046 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,120,745 bp\n",
      "  Read 1:    20,063,966 bp\n",
      "  Read 2:    20,056,779 bp\n",
      "Total written (filtered):     36,591,736 bp (91.2%)\n",
      "  Read 1:    18,297,531 bp\n",
      "  Read 2:    18,294,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t536\t0.0\t2\t0 525 11\n",
      "26\t66922\t0.0\t2\t65412 1482 28\n",
      "27\t281\t0.0\t2\t0 272 9\n",
      "28\t151\t0.0\t2\t0 0 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t100\t0.0\t2\t0 0 100\n",
      "25\t1391\t0.0\t2\t0 1351 40\n",
      "26\t66000\t0.0\t2\t64347 1595 58\n",
      "27\t317\t0.0\t2\t0 304 13\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S071_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S071_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S071_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S071_141_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S071_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S071_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S071_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S071_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        82,493 reads  @     94.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 7.80 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,493\n",
      "  Read 1 with adapter:                  82,252 (99.7%)\n",
      "  Read 2 with adapter:                  82,327 (99.8%)\n",
      "Pairs written (passing filters):        82,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,639,002 bp\n",
      "  Read 1:    24,308,066 bp\n",
      "  Read 2:    24,330,936 bp\n",
      "Total written (filtered):     44,360,743 bp (91.2%)\n",
      "  Read 1:    22,169,711 bp\n",
      "  Read 2:    22,191,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 82252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t688\t0.0\t2\t0 666 22\n",
      "26\t81051\t0.0\t2\t79065 1952 34\n",
      "27\t323\t0.0\t2\t0 315 8\n",
      "28\t137\t0.0\t2\t0 0 137\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 82327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t923\t0.0\t2\t0 893 30\n",
      "26\t80915\t0.0\t2\t78805 2026 84\n",
      "27\t377\t0.0\t2\t0 361 16\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S072_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S072_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S072_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S072_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S072_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S072_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S072_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S072_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:18       204,052 reads  @     90.4 µs/read;   0.66 M reads/minute\n",
      "Finished in 18.44 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            204,052\n",
      "  Read 1 with adapter:                 202,937 (99.5%)\n",
      "  Read 2 with adapter:                 203,762 (99.9%)\n",
      "Pairs written (passing filters):       204,052 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,238,092 bp\n",
      "  Read 1:    60,044,742 bp\n",
      "  Read 2:    60,193,350 bp\n",
      "Total written (filtered):    109,666,113 bp (91.2%)\n",
      "  Read 1:    54,769,253 bp\n",
      "  Read 2:    54,896,860 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 202937 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t117\t0.0\t2\t0 0 117\n",
      "25\t1740\t0.0\t2\t0 1683 57\n",
      "26\t200188\t0.0\t2\t194922 5179 87\n",
      "27\t683\t0.0\t2\t0 664 19\n",
      "28\t209\t0.0\t2\t0 0 209\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 203762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t177\t0.0\t2\t0 0 177\n",
      "25\t2208\t0.0\t2\t0 2144 64\n",
      "26\t200322\t0.0\t2\t195518 4663 141\n",
      "27\t870\t0.0\t2\t0 838 32\n",
      "28\t185\t0.0\t2\t0 0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S073_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S073_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S073_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S073_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S073_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S073_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S073_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S073_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,666 reads  @     97.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.87 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,666\n",
      "  Read 1 with adapter:                  49,607 (99.9%)\n",
      "  Read 2 with adapter:                  49,266 (99.2%)\n",
      "Pairs written (passing filters):        49,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,292,850 bp\n",
      "  Read 1:    14,647,431 bp\n",
      "  Read 2:    14,645,419 bp\n",
      "Total written (filtered):     26,724,677 bp (91.2%)\n",
      "  Read 1:    13,358,106 bp\n",
      "  Read 2:    13,366,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t565\t0.0\t2\t0 544 21\n",
      "26\t48858\t0.0\t2\t47400 1434 24\n",
      "27\t120\t0.0\t2\t0 114 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t87\t0.0\t2\t0 0 87\n",
      "25\t2214\t0.0\t2\t0 2159 55\n",
      "26\t46692\t0.0\t2\t45603 1053 36\n",
      "27\t226\t0.0\t2\t0 217 9\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S074_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S074_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S074_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S074_147_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S074_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S074_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S074_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S074_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        60,305 reads  @     86.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.22 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,305\n",
      "  Read 1 with adapter:                  60,230 (99.9%)\n",
      "  Read 2 with adapter:                  60,215 (99.9%)\n",
      "Pairs written (passing filters):        60,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,575,654 bp\n",
      "  Read 1:    17,787,359 bp\n",
      "  Read 2:    17,788,295 bp\n",
      "Total written (filtered):     32,445,583 bp (91.2%)\n",
      "  Read 1:    16,221,955 bp\n",
      "  Read 2:    16,223,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t731\t0.0\t2\t0 703 28\n",
      "26\t59249\t0.0\t2\t57679 1543 27\n",
      "27\t153\t0.0\t2\t0 148 5\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t1157\t0.0\t2\t0 1139 18\n",
      "26\t58711\t0.0\t2\t57327 1337 47\n",
      "27\t284\t0.0\t2\t0 274 10\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S075_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S075_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S075_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S075_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S075_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S075_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S075_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S075_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,695 reads  @     86.7 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.35 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,695\n",
      "  Read 1 with adapter:                  61,632 (99.9%)\n",
      "  Read 2 with adapter:                  61,597 (99.8%)\n",
      "Pairs written (passing filters):        61,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,395,627 bp\n",
      "  Read 1:    18,198,071 bp\n",
      "  Read 2:    18,197,556 bp\n",
      "Total written (filtered):     33,193,542 bp (91.2%)\n",
      "  Read 1:    16,596,129 bp\n",
      "  Read 2:    16,597,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t713\t0.0\t2\t0 696 17\n",
      "26\t60657\t0.0\t2\t59017 1608 32\n",
      "27\t145\t0.0\t2\t0 138 7\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1591\t0.0\t2\t0 1560 31\n",
      "26\t59635\t0.0\t2\t58200 1370 65\n",
      "27\t294\t0.0\t2\t0 279 15\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S076_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S076_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S076_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S076_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S076_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S076_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S076_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S076_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,215 reads  @     90.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.99 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,215\n",
      "  Read 1 with adapter:                  55,149 (99.9%)\n",
      "  Read 2 with adapter:                  54,783 (99.2%)\n",
      "Pairs written (passing filters):        55,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,535,638 bp\n",
      "  Read 1:    16,287,003 bp\n",
      "  Read 2:    16,248,635 bp\n",
      "Total written (filtered):     29,678,406 bp (91.2%)\n",
      "  Read 1:    14,853,581 bp\n",
      "  Read 2:    14,824,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t647\t0.0\t2\t0 625 22\n",
      "26\t54270\t0.0\t2\t52907 1329 34\n",
      "27\t145\t0.0\t2\t0 141 4\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t731\t0.0\t2\t0 699 32\n",
      "26\t53771\t0.0\t2\t52181 1467 123\n",
      "27\t227\t0.0\t2\t0 214 13\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S077_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S077_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S077_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S077_153_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S077_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S077_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S077_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S077_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,305 reads  @     92.5 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.67 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,305\n",
      "  Read 1 with adapter:                  61,243 (99.9%)\n",
      "  Read 2 with adapter:                  60,243 (98.3%)\n",
      "Pairs written (passing filters):        61,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,002,358 bp\n",
      "  Read 1:    18,082,846 bp\n",
      "  Read 2:    17,919,512 bp\n",
      "Total written (filtered):     32,845,229 bp (91.2%)\n",
      "  Read 1:    16,491,000 bp\n",
      "  Read 2:    16,354,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t716\t0.0\t2\t0 699 17\n",
      "26\t60217\t0.0\t2\t58615 1577 25\n",
      "27\t188\t0.0\t2\t0 173 15\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t1198\t0.0\t2\t0 1150 48\n",
      "26\t58725\t0.0\t2\t56772 1814 139\n",
      "27\t245\t0.0\t2\t0 227 18\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S078_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S078_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S078_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S078_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S078_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S078_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S078_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S078_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,289 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.73 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,289\n",
      "  Read 1 with adapter:                  33,255 (99.9%)\n",
      "  Read 2 with adapter:                  33,157 (99.6%)\n",
      "Pairs written (passing filters):        33,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,625,793 bp\n",
      "  Read 1:     9,819,894 bp\n",
      "  Read 2:     9,805,899 bp\n",
      "Total written (filtered):     17,899,997 bp (91.2%)\n",
      "  Read 1:     8,955,524 bp\n",
      "  Read 2:     8,944,473 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t394\t0.0\t2\t0 384 10\n",
      "26\t32700\t0.0\t2\t31767 922 11\n",
      "27\t104\t0.0\t2\t0 101 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t760\t0.0\t2\t0 730 30\n",
      "26\t32181\t0.0\t2\t31277 870 34\n",
      "27\t160\t0.0\t2\t0 151 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S079_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S079_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S079_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S079_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S079_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S079_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S079_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S079_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        84,683 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 7.04 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,683\n",
      "  Read 1 with adapter:                  84,601 (99.9%)\n",
      "  Read 2 with adapter:                  84,502 (99.8%)\n",
      "Pairs written (passing filters):        84,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,950,367 bp\n",
      "  Read 1:    24,977,387 bp\n",
      "  Read 2:    24,972,980 bp\n",
      "Total written (filtered):     45,555,092 bp (91.2%)\n",
      "  Read 1:    22,778,528 bp\n",
      "  Read 2:    22,776,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t1032\t0.0\t2\t0 999 33\n",
      "26\t83165\t0.0\t2\t80809 2309 47\n",
      "27\t251\t0.0\t2\t0 241 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 84502 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t981\t0.0\t2\t0 948 33\n",
      "26\t82991\t0.0\t2\t80713 2187 91\n",
      "27\t435\t0.0\t2\t0 417 18\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S081_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S081_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S081_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S081_159_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S081_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S081_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S081_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S081_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           465 reads  @    290.7 µs/read;   0.21 M reads/minute\n",
      "Finished in 0.14 s (298 us/read; 0.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                465\n",
      "  Read 1 with adapter:                     453 (97.4%)\n",
      "  Read 2 with adapter:                     450 (96.8%)\n",
      "Pairs written (passing filters):           465 (100.0%)\n",
      "\n",
      "Total basepairs processed:       273,930 bp\n",
      "  Read 1:       136,997 bp\n",
      "  Read 2:       136,933 bp\n",
      "Total written (filtered):        250,487 bp (91.4%)\n",
      "  Read 1:       125,230 bp\n",
      "  Read 2:       125,257 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t11\t0.0\t2\t0 11\n",
      "26\t440\t0.0\t2\t422 17 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 450 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t28\t0.0\t2\t0 28\n",
      "26\t419\t0.0\t2\t407 12\n",
      "27\t2\t0.0\t2\t0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S084_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S084_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S084_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S084_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S084_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S084_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S084_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S084_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,080 reads  @     80.8 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.13 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,080\n",
      "  Read 1 with adapter:                  51,038 (99.9%)\n",
      "  Read 2 with adapter:                  51,003 (99.8%)\n",
      "Pairs written (passing filters):        51,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,135,840 bp\n",
      "  Read 1:    15,068,330 bp\n",
      "  Read 2:    15,067,510 bp\n",
      "Total written (filtered):     27,483,005 bp (91.2%)\n",
      "  Read 1:    13,741,243 bp\n",
      "  Read 2:    13,741,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51038 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t472\t0.0\t2\t0 456 16\n",
      "26\t50032\t0.0\t2\t48921 1095 16\n",
      "27\t333\t0.0\t2\t0 324 9\n",
      "28\t160\t0.0\t2\t0 0 160\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51003 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t494\t0.0\t2\t0 478 16\n",
      "26\t50251\t0.0\t2\t49052 1140 59\n",
      "27\t184\t0.0\t2\t0 180 4\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S085_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S085_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S085_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S085_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S085_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S085_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S085_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S085_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,589 reads  @     86.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.59 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,589\n",
      "  Read 1 with adapter:                  64,542 (99.9%)\n",
      "  Read 2 with adapter:                  64,430 (99.8%)\n",
      "Pairs written (passing filters):        64,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,101,632 bp\n",
      "  Read 1:    19,053,250 bp\n",
      "  Read 2:    19,048,382 bp\n",
      "Total written (filtered):     34,748,734 bp (91.2%)\n",
      "  Read 1:    17,375,085 bp\n",
      "  Read 2:    17,373,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64542 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t606\t0.0\t2\t0 595 11\n",
      "26\t63284\t0.0\t2\t61908 1347 29\n",
      "27\t389\t0.0\t2\t0 379 10\n",
      "28\t204\t0.0\t2\t0 0 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t637\t0.0\t2\t0 615 22\n",
      "26\t63495\t0.0\t2\t61955 1479 61\n",
      "27\t250\t0.0\t2\t0 235 15\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S086_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S086_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S086_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S086_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S086_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S086_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S086_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S086_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,149 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.11 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,149\n",
      "  Read 1 with adapter:                  61,090 (99.9%)\n",
      "  Read 2 with adapter:                  61,000 (99.8%)\n",
      "Pairs written (passing filters):        61,149 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,068,555 bp\n",
      "  Read 1:    18,037,969 bp\n",
      "  Read 2:    18,030,586 bp\n",
      "Total written (filtered):     32,894,600 bp (91.2%)\n",
      "  Read 1:    16,449,654 bp\n",
      "  Read 2:    16,444,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61090 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t643\t0.0\t2\t0 629 14\n",
      "26\t59866\t0.0\t2\t58608 1226 32\n",
      "27\t324\t0.0\t2\t0 311 13\n",
      "28\t202\t0.0\t2\t0 0 202\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t587\t0.0\t2\t0 564 23\n",
      "26\t60069\t0.0\t2\t58421 1567 81\n",
      "27\t277\t0.0\t2\t0 261 16\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S087_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S087_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S087_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S087_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S087_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S087_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S087_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S087_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        67,304 reads  @     92.5 µs/read;   0.65 M reads/minute\n",
      "Finished in 6.23 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,304\n",
      "  Read 1 with adapter:                  67,239 (99.9%)\n",
      "  Read 2 with adapter:                  66,757 (99.2%)\n",
      "Pairs written (passing filters):        67,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,643,732 bp\n",
      "  Read 1:    19,853,331 bp\n",
      "  Read 2:    19,790,401 bp\n",
      "Total written (filtered):     36,160,674 bp (91.2%)\n",
      "  Read 1:    18,105,532 bp\n",
      "  Read 2:    18,055,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t830\t0.0\t2\t0 814 16\n",
      "26\t65919\t0.0\t2\t64382 1516 21\n",
      "27\t345\t0.0\t2\t0 337 8\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66757 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t635\t0.0\t2\t0 612 23\n",
      "26\t65802\t0.0\t2\t64210 1547 45\n",
      "27\t264\t0.0\t2\t0 257 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S088_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S088_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S088_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S088_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S088_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S088_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S088_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S088_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        58,816 reads  @    112.8 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.64 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,816\n",
      "  Read 1 with adapter:                  58,761 (99.9%)\n",
      "  Read 2 with adapter:                  58,727 (99.8%)\n",
      "Pairs written (passing filters):        58,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,694,597 bp\n",
      "  Read 1:    17,350,231 bp\n",
      "  Read 2:    17,344,366 bp\n",
      "Total written (filtered):     31,640,677 bp (91.2%)\n",
      "  Read 1:    15,822,803 bp\n",
      "  Read 2:    15,817,874 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58761 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t572\t0.0\t2\t0 554 18\n",
      "26\t57910\t0.0\t2\t56405 1474 31\n",
      "27\t180\t0.0\t2\t0 170 10\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58727 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t623\t0.0\t2\t0 605 18\n",
      "26\t57753\t0.0\t2\t56413 1299 41\n",
      "27\t245\t0.0\t2\t0 236 9\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S089_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S089_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S089_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S089_171_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S089_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S089_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S089_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S089_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        62,219 reads  @    106.2 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.61 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,219\n",
      "  Read 1 with adapter:                  62,174 (99.9%)\n",
      "  Read 2 with adapter:                  62,126 (99.9%)\n",
      "Pairs written (passing filters):        62,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,705,203 bp\n",
      "  Read 1:    18,354,004 bp\n",
      "  Read 2:    18,351,199 bp\n",
      "Total written (filtered):     33,474,233 bp (91.2%)\n",
      "  Read 1:    16,737,911 bp\n",
      "  Read 2:    16,736,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t605\t0.0\t2\t0 585 20\n",
      "26\t61283\t0.0\t2\t59779 1482 22\n",
      "27\t194\t0.0\t2\t0 188 6\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t654\t0.0\t2\t0 632 22\n",
      "26\t61124\t0.0\t2\t59605 1457 62\n",
      "27\t273\t0.0\t2\t0 259 14\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S090_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S090_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S090_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S090_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S090_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S090_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S090_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S090_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        74,513 reads  @    113.3 µs/read;   0.53 M reads/minute\n",
      "Finished in 8.45 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,513\n",
      "  Read 1 with adapter:                  74,426 (99.9%)\n",
      "  Read 2 with adapter:                  74,416 (99.9%)\n",
      "Pairs written (passing filters):        74,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,958,439 bp\n",
      "  Read 1:    21,980,649 bp\n",
      "  Read 2:    21,977,790 bp\n",
      "Total written (filtered):     40,089,255 bp (91.2%)\n",
      "  Read 1:    20,045,873 bp\n",
      "  Read 2:    20,043,382 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t734\t0.0\t2\t0 709 25\n",
      "26\t73233\t0.0\t2\t71421 1788 24\n",
      "27\t280\t0.0\t2\t0 273 7\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t708\t0.0\t2\t0 680 28\n",
      "26\t73266\t0.0\t2\t71406 1807 53\n",
      "27\t332\t0.0\t2\t0 320 12\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S091_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S091_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S091_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S091_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S091_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S091_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S091_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S091_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        66,840 reads  @    110.7 µs/read;   0.54 M reads/minute\n",
      "Finished in 7.40 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,840\n",
      "  Read 1 with adapter:                  66,785 (99.9%)\n",
      "  Read 2 with adapter:                  66,667 (99.7%)\n",
      "Pairs written (passing filters):        66,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,433,803 bp\n",
      "  Read 1:    19,717,639 bp\n",
      "  Read 2:    19,716,164 bp\n",
      "Total written (filtered):     35,964,584 bp (91.2%)\n",
      "  Read 1:    17,981,379 bp\n",
      "  Read 2:    17,983,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t625\t0.0\t2\t0 608 17\n",
      "26\t65660\t0.0\t2\t64046 1587 27\n",
      "27\t313\t0.0\t2\t0 300 13\n",
      "28\t134\t0.0\t2\t0 0 134\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66667 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t646\t0.0\t2\t0 626 20\n",
      "26\t65656\t0.0\t2\t63921 1665 70\n",
      "27\t263\t0.0\t2\t0 248 15\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S092_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S092_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S092_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S092_177_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S092_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S092_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S092_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S092_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        52,811 reads  @     90.8 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.80 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,811\n",
      "  Read 1 with adapter:                  52,723 (99.8%)\n",
      "  Read 2 with adapter:                  52,729 (99.8%)\n",
      "Pairs written (passing filters):        52,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,153,521 bp\n",
      "  Read 1:    15,575,586 bp\n",
      "  Read 2:    15,577,935 bp\n",
      "Total written (filtered):     28,412,072 bp (91.2%)\n",
      "  Read 1:    14,204,749 bp\n",
      "  Read 2:    14,207,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52723 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t424\t0.0\t2\t0 406 18\n",
      "26\t51860\t0.0\t2\t50632 1208 20\n",
      "27\t259\t0.0\t2\t0 252 7\n",
      "28\t141\t0.0\t2\t0 0 141\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52729 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t544\t0.0\t2\t0 509 35\n",
      "26\t51885\t0.0\t2\t50588 1226 71\n",
      "27\t174\t0.0\t2\t0 165 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S093_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S093_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S093_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S093_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S093_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S093_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S093_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S093_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,877 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.03 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,877\n",
      "  Read 1 with adapter:                  45,768 (99.8%)\n",
      "  Read 2 with adapter:                  45,794 (99.8%)\n",
      "Pairs written (passing filters):        45,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,059,715 bp\n",
      "  Read 1:    13,528,658 bp\n",
      "  Read 2:    13,531,057 bp\n",
      "Total written (filtered):     24,679,426 bp (91.2%)\n",
      "  Read 1:    12,338,657 bp\n",
      "  Read 2:    12,340,769 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t356\t0.0\t2\t0 348 8\n",
      "26\t45054\t0.0\t2\t44061 973 20\n",
      "27\t183\t0.0\t2\t0 179 4\n",
      "28\t139\t0.0\t2\t0 0 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t483\t0.0\t2\t0 462 21\n",
      "26\t45095\t0.0\t2\t44008 1034 53\n",
      "27\t177\t0.0\t2\t0 166 11\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S094_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S094_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S094_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S094_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S094_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S094_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S094_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S094_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,222 reads  @     89.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.56 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,222\n",
      "  Read 1 with adapter:                  51,017 (99.6%)\n",
      "  Read 2 with adapter:                  51,139 (99.8%)\n",
      "Pairs written (passing filters):        51,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,208,156 bp\n",
      "  Read 1:    15,101,548 bp\n",
      "  Read 2:    15,106,608 bp\n",
      "Total written (filtered):     27,552,302 bp (91.2%)\n",
      "  Read 1:    13,775,010 bp\n",
      "  Read 2:    13,777,292 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t408\t0.0\t2\t0 391 17\n",
      "26\t50134\t0.0\t2\t48995 1115 24\n",
      "27\t266\t0.0\t2\t0 262 4\n",
      "28\t164\t0.0\t2\t0 0 164\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t452\t0.0\t2\t0 437 15\n",
      "26\t50403\t0.0\t2\t49248 1118 37\n",
      "27\t230\t0.0\t2\t0 219 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S095_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S095_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S095_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S095_183_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S095_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S095_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S095_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S095_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        52,396 reads  @     94.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.96 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,396\n",
      "  Read 1 with adapter:                  52,306 (99.8%)\n",
      "  Read 2 with adapter:                  52,291 (99.8%)\n",
      "Pairs written (passing filters):        52,396 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,908,476 bp\n",
      "  Read 1:    15,453,445 bp\n",
      "  Read 2:    15,455,031 bp\n",
      "Total written (filtered):     28,189,284 bp (91.2%)\n",
      "  Read 1:    14,093,529 bp\n",
      "  Read 2:    14,095,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t418\t0.0\t2\t0 400 18\n",
      "26\t51523\t0.0\t2\t50258 1243 22\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t127\t0.0\t2\t0 0 127\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t489\t0.0\t2\t0 472 17\n",
      "26\t51557\t0.0\t2\t50296 1229 32\n",
      "27\t211\t0.0\t2\t0 201 10\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S096_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S096_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S096_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S096_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S096_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S096_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S096_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S096_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        69,663 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 6.86 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,663\n",
      "  Read 1 with adapter:                  69,610 (99.9%)\n",
      "  Read 2 with adapter:                  69,558 (99.8%)\n",
      "Pairs written (passing filters):        69,663 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,095,502 bp\n",
      "  Read 1:    20,549,151 bp\n",
      "  Read 2:    20,546,351 bp\n",
      "Total written (filtered):     37,477,860 bp (91.2%)\n",
      "  Read 1:    18,739,629 bp\n",
      "  Read 2:    18,738,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69610 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t623\t0.0\t2\t0 605 18\n",
      "26\t68605\t0.0\t2\t66823 1760 22\n",
      "27\t251\t0.0\t2\t0 242 9\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69558 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t656\t0.0\t2\t0 634 22\n",
      "26\t68515\t0.0\t2\t66821 1637 57\n",
      "27\t306\t0.0\t2\t0 295 11\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S097_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S097_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S097_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S097_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S097_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S097_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S097_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S097_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        61,246 reads  @    108.9 µs/read;   0.55 M reads/minute\n",
      "Finished in 6.67 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,246\n",
      "  Read 1 with adapter:                  61,196 (99.9%)\n",
      "  Read 2 with adapter:                  61,154 (99.8%)\n",
      "Pairs written (passing filters):        61,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,128,125 bp\n",
      "  Read 1:    18,064,697 bp\n",
      "  Read 2:    18,063,428 bp\n",
      "Total written (filtered):     32,947,726 bp (91.2%)\n",
      "  Read 1:    16,473,918 bp\n",
      "  Read 2:    16,473,808 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61196 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t526\t0.0\t2\t0 512 14\n",
      "26\t60368\t0.0\t2\t58954 1390 24\n",
      "27\t203\t0.0\t2\t0 199 4\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t618\t0.0\t2\t0 595 23\n",
      "26\t60218\t0.0\t2\t58682 1479 57\n",
      "27\t262\t0.0\t2\t0 253 9\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S098_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S098_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S098_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S098_189_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S098_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S098_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S098_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S098_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        92,211 reads  @     92.2 µs/read;   0.65 M reads/minute\n",
      "Finished in 8.50 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,211\n",
      "  Read 1 with adapter:                  92,138 (99.9%)\n",
      "  Read 2 with adapter:                  92,047 (99.8%)\n",
      "Pairs written (passing filters):        92,211 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,397,490 bp\n",
      "  Read 1:    27,200,427 bp\n",
      "  Read 2:    27,197,063 bp\n",
      "Total written (filtered):     49,609,379 bp (91.2%)\n",
      "  Read 1:    24,805,155 bp\n",
      "  Read 2:    24,804,224 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 92138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t784\t0.0\t2\t0 754 30\n",
      "26\t90775\t0.0\t2\t88018 2693 64\n",
      "27\t386\t0.0\t2\t0 373 13\n",
      "28\t117\t0.0\t2\t0 0 117\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 92047 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t939\t0.0\t2\t0 896 43\n",
      "26\t90461\t0.0\t2\t87678 2678 105\n",
      "27\t470\t0.0\t2\t0 441 29\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S099_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S099_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S099_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S099_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S099_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S099_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S099_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S099_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        75,259 reads  @    100.9 µs/read;   0.59 M reads/minute\n",
      "Finished in 7.60 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,259\n",
      "  Read 1 with adapter:                  75,209 (99.9%)\n",
      "  Read 2 with adapter:                  75,102 (99.8%)\n",
      "Pairs written (passing filters):        75,259 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,400,580 bp\n",
      "  Read 1:    22,200,841 bp\n",
      "  Read 2:    22,199,739 bp\n",
      "Total written (filtered):     40,493,093 bp (91.2%)\n",
      "  Read 1:    20,245,710 bp\n",
      "  Read 2:    20,247,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 75209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t622\t0.0\t2\t0 599 23\n",
      "26\t74192\t0.0\t2\t72398 1759 35\n",
      "27\t275\t0.0\t2\t0 267 8\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t657\t0.0\t2\t0 635 22\n",
      "26\t74002\t0.0\t2\t72114 1807 81\n",
      "27\t337\t0.0\t2\t0 314 23\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S100_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S100_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S100_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S100_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S100_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S100_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S100_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S100_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        54,315 reads  @    100.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.47 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,315\n",
      "  Read 1 with adapter:                  54,263 (99.9%)\n",
      "  Read 2 with adapter:                  54,210 (99.8%)\n",
      "Pairs written (passing filters):        54,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,044,538 bp\n",
      "  Read 1:    16,022,503 bp\n",
      "  Read 2:    16,022,035 bp\n",
      "Total written (filtered):     29,224,644 bp (91.2%)\n",
      "  Read 1:    14,611,711 bp\n",
      "  Read 2:    14,612,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t504\t0.0\t2\t0 490 14\n",
      "26\t53342\t0.0\t2\t51904 1413 25\n",
      "27\t260\t0.0\t2\t0 252 8\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t544\t0.0\t2\t0 514 30\n",
      "26\t53321\t0.0\t2\t51859 1393 69\n",
      "27\t272\t0.0\t2\t0 261 11\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S101_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S101_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S101_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S101_195_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S101_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S101_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S101_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S101_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,317 reads  @     90.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.55 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,317\n",
      "  Read 1 with adapter:                  50,278 (99.9%)\n",
      "  Read 2 with adapter:                  50,203 (99.8%)\n",
      "Pairs written (passing filters):        50,317 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,683,750 bp\n",
      "  Read 1:    14,843,223 bp\n",
      "  Read 2:    14,840,527 bp\n",
      "Total written (filtered):     27,071,560 bp (91.2%)\n",
      "  Read 1:    13,535,985 bp\n",
      "  Read 2:    13,535,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t394\t0.0\t2\t0 381 13\n",
      "26\t49481\t0.0\t2\t48392 1068 21\n",
      "27\t270\t0.0\t2\t0 266 4\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50203 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t511\t0.0\t2\t0 495 16\n",
      "26\t49425\t0.0\t2\t48103 1264 58\n",
      "27\t233\t0.0\t2\t0 224 9\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S102_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S102_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S102_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S102_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S102_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S102_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S102_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S102_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        69,938 reads  @     85.8 µs/read;   0.70 M reads/minute\n",
      "Finished in 6.00 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,938\n",
      "  Read 1 with adapter:                  69,884 (99.9%)\n",
      "  Read 2 with adapter:                  69,777 (99.8%)\n",
      "Pairs written (passing filters):        69,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,255,627 bp\n",
      "  Read 1:    20,630,569 bp\n",
      "  Read 2:    20,625,058 bp\n",
      "Total written (filtered):     37,624,713 bp (91.2%)\n",
      "  Read 1:    18,813,472 bp\n",
      "  Read 2:    18,811,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t532\t0.0\t2\t0 522 10\n",
      "26\t68760\t0.0\t2\t67238 1494 28\n",
      "27\t359\t0.0\t2\t0 347 12\n",
      "28\t188\t0.0\t2\t0 0 188\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t598\t0.0\t2\t0 577 21\n",
      "26\t68859\t0.0\t2\t67030 1728 101\n",
      "27\t275\t0.0\t2\t0 262 13\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S103_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S103_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S103_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S103_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S103_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S103_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S103_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S103_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,204 reads  @     93.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.99 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,204\n",
      "  Read 1 with adapter:                  64,158 (99.9%)\n",
      "  Read 2 with adapter:                  64,033 (99.7%)\n",
      "Pairs written (passing filters):        64,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,878,540 bp\n",
      "  Read 1:    18,939,272 bp\n",
      "  Read 2:    18,939,268 bp\n",
      "Total written (filtered):     34,545,955 bp (91.2%)\n",
      "  Read 1:    17,271,138 bp\n",
      "  Read 2:    17,274,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t533\t0.0\t2\t0 515 18\n",
      "26\t63139\t0.0\t2\t61714 1413 12\n",
      "27\t285\t0.0\t2\t0 280 5\n",
      "28\t169\t0.0\t2\t0 0 169\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t592\t0.0\t2\t0 564 28\n",
      "26\t63145\t0.0\t2\t61513 1575 57\n",
      "27\t251\t0.0\t2\t0 241 10\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S104_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S104_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S104_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S104_201_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S104_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S104_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S104_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S104_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        78,979 reads  @    103.0 µs/read;   0.58 M reads/minute\n",
      "Finished in 8.14 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,979\n",
      "  Read 1 with adapter:                  78,917 (99.9%)\n",
      "  Read 2 with adapter:                  78,836 (99.8%)\n",
      "Pairs written (passing filters):        78,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,595,053 bp\n",
      "  Read 1:    23,298,381 bp\n",
      "  Read 2:    23,296,672 bp\n",
      "Total written (filtered):     42,494,219 bp (91.2%)\n",
      "  Read 1:    21,246,789 bp\n",
      "  Read 2:    21,247,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 78917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t793\t0.0\t2\t0 769 24\n",
      "26\t77573\t0.0\t2\t75861 1693 19\n",
      "27\t331\t0.0\t2\t0 325 6\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t789\t0.0\t2\t0 751 38\n",
      "26\t77595\t0.0\t2\t75621 1882 92\n",
      "27\t369\t0.0\t2\t0 342 27\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S105_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S105_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S105_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S105_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S105_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S105_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S105_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S105_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,455 reads  @     90.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.75 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,455\n",
      "  Read 1 with adapter:                  63,412 (99.9%)\n",
      "  Read 2 with adapter:                  63,341 (99.8%)\n",
      "Pairs written (passing filters):        63,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,434,407 bp\n",
      "  Read 1:    18,719,197 bp\n",
      "  Read 2:    18,715,210 bp\n",
      "Total written (filtered):     34,139,648 bp (91.2%)\n",
      "  Read 1:    17,070,931 bp\n",
      "  Read 2:    17,068,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t686\t0.0\t2\t0 672 14\n",
      "26\t62413\t0.0\t2\t60944 1445 24\n",
      "27\t206\t0.0\t2\t0 200 6\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63341 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t681\t0.0\t2\t0 658 23\n",
      "26\t62320\t0.0\t2\t60633 1593 94\n",
      "27\t228\t0.0\t2\t0 224 4\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S106_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S106_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S106_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S106_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S106_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S106_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S106_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S106_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,956 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.59 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,956\n",
      "  Read 1 with adapter:                  66,885 (99.9%)\n",
      "  Read 2 with adapter:                  66,839 (99.8%)\n",
      "Pairs written (passing filters):        66,956 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,500,853 bp\n",
      "  Read 1:    19,751,516 bp\n",
      "  Read 2:    19,749,337 bp\n",
      "Total written (filtered):     36,024,842 bp (91.2%)\n",
      "  Read 1:    18,013,025 bp\n",
      "  Read 2:    18,011,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66885 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t732\t0.0\t2\t0 715 17\n",
      "26\t65826\t0.0\t2\t64154 1641 31\n",
      "27\t221\t0.0\t2\t0 209 12\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66839 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t669\t0.0\t2\t0 640 29\n",
      "26\t65746\t0.0\t2\t63908 1749 89\n",
      "27\t289\t0.0\t2\t0 274 15\n",
      "28\t89\t0.0\t2\t0 0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S107_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S107_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S107_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S107_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S107_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S107_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S107_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S107_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        17,625 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 1.40 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,625\n",
      "  Read 1 with adapter:                  17,611 (99.9%)\n",
      "  Read 2 with adapter:                  17,598 (99.8%)\n",
      "Pairs written (passing filters):        17,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,398,170 bp\n",
      "  Read 1:     5,199,367 bp\n",
      "  Read 2:     5,198,803 bp\n",
      "Total written (filtered):      9,482,930 bp (91.2%)\n",
      "  Read 1:     4,741,585 bp\n",
      "  Read 2:     4,741,345 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17611 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t210\t0.0\t2\t0 204 6\n",
      "26\t17272\t0.0\t2\t16767 496 9\n",
      "27\t84\t0.0\t2\t0 80 4\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t197\t0.0\t2\t0 188 9\n",
      "26\t17279\t0.0\t2\t16759 498 22\n",
      "27\t81\t0.0\t2\t0 77 4\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S108_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S108_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S108_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S108_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S108_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S108_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S108_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S108_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,322 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.11 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,322\n",
      "  Read 1 with adapter:                  39,017 (99.2%)\n",
      "  Read 2 with adapter:                  39,267 (99.9%)\n",
      "Pairs written (passing filters):        39,322 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,125,237 bp\n",
      "  Read 1:    11,526,151 bp\n",
      "  Read 2:    11,599,086 bp\n",
      "Total written (filtered):     21,090,243 bp (91.2%)\n",
      "  Read 1:    10,511,839 bp\n",
      "  Read 2:    10,578,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t347\t0.0\t2\t0 333 14\n",
      "26\t38443\t0.0\t2\t37311 1118 14\n",
      "27\t137\t0.0\t2\t0 136 1\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39267 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t455\t0.0\t2\t0 435 20\n",
      "26\t38559\t0.0\t2\t37469 1050 40\n",
      "27\t143\t0.0\t2\t0 138 5\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S109_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S109_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S109_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S109_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S109_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S109_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S109_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S109_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,381 reads  @     77.7 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.83 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,381\n",
      "  Read 1 with adapter:                  36,271 (99.7%)\n",
      "  Read 2 with adapter:                  36,337 (99.9%)\n",
      "Pairs written (passing filters):        36,381 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,441,947 bp\n",
      "  Read 1:    10,710,449 bp\n",
      "  Read 2:    10,731,498 bp\n",
      "Total written (filtered):     19,554,661 bp (91.2%)\n",
      "  Read 1:     9,767,638 bp\n",
      "  Read 2:     9,787,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t349\t0.0\t2\t0 335 14\n",
      "26\t35759\t0.0\t2\t34849 886 24\n",
      "27\t112\t0.0\t2\t0 111 1\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t408\t0.0\t2\t0 385 23\n",
      "26\t35739\t0.0\t2\t34925 789 25\n",
      "27\t127\t0.0\t2\t0 122 5\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S110_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S110_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S110_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S110_213_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S110_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S110_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S110_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S110_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10       129,515 reads  @     84.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 10.89 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,515\n",
      "  Read 1 with adapter:                 129,419 (99.9%)\n",
      "  Read 2 with adapter:                 129,298 (99.8%)\n",
      "Pairs written (passing filters):       129,515 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,397,180 bp\n",
      "  Read 1:    38,202,137 bp\n",
      "  Read 2:    38,195,043 bp\n",
      "Total written (filtered):     69,671,669 bp (91.2%)\n",
      "  Read 1:    34,837,502 bp\n",
      "  Read 2:    34,834,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 129419 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t1010\t0.0\t2\t0 984 26\n",
      "26\t127664\t0.0\t2\t124770 2846 48\n",
      "27\t451\t0.0\t2\t0 437 14\n",
      "28\t222\t0.0\t2\t0 0 222\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 129298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t119\t0.0\t2\t0 0 119\n",
      "25\t1220\t0.0\t2\t0 1181 39\n",
      "26\t127403\t0.0\t2\t124471 2836 96\n",
      "27\t526\t0.0\t2\t0 503 23\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S111_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S111_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S111_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S111_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S111_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S111_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S111_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S111_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S112_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S112_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S112_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S112_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S112_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S112_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S112_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S112_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        79,498 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 7.53 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,498\n",
      "  Read 1 with adapter:                  79,436 (99.9%)\n",
      "  Read 2 with adapter:                  79,383 (99.9%)\n",
      "Pairs written (passing filters):        79,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,899,908 bp\n",
      "  Read 1:    23,449,880 bp\n",
      "  Read 2:    23,450,028 bp\n",
      "Total written (filtered):     42,771,283 bp (91.2%)\n",
      "  Read 1:    21,384,620 bp\n",
      "  Read 2:    21,386,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t593\t0.0\t2\t0 577 16\n",
      "26\t78334\t0.0\t2\t76506 1789 39\n",
      "27\t281\t0.0\t2\t0 274 7\n",
      "28\t173\t0.0\t2\t0 0 173\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 79383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t836\t0.0\t2\t0 809 27\n",
      "26\t78098\t0.0\t2\t76407 1661 30\n",
      "27\t331\t0.0\t2\t0 324 7\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S113_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S113_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S113_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S113_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S113_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S113_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S113_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S113_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        84,583 reads  @     88.2 µs/read;   0.68 M reads/minute\n",
      "Finished in 7.46 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,583\n",
      "  Read 1 with adapter:                  84,542 (100.0%)\n",
      "  Read 2 with adapter:                  84,455 (99.8%)\n",
      "Pairs written (passing filters):        84,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,898,140 bp\n",
      "  Read 1:    24,951,439 bp\n",
      "  Read 2:    24,946,701 bp\n",
      "Total written (filtered):     45,504,937 bp (91.2%)\n",
      "  Read 1:    22,753,509 bp\n",
      "  Read 2:    22,751,428 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84542 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t658\t0.0\t2\t0 639 19\n",
      "26\t83368\t0.0\t2\t81617 1723 28\n",
      "27\t308\t0.0\t2\t0 300 8\n",
      "28\t151\t0.0\t2\t0 0 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 84455 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t814\t0.0\t2\t0 797 17\n",
      "26\t83213\t0.0\t2\t81305 1844 64\n",
      "27\t347\t0.0\t2\t0 334 13\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S114_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S114_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S114_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S114_221_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S114_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S114_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S114_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S114_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        74,360 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 6.48 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,360\n",
      "  Read 1 with adapter:                  74,302 (99.9%)\n",
      "  Read 2 with adapter:                  74,241 (99.8%)\n",
      "Pairs written (passing filters):        74,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,867,545 bp\n",
      "  Read 1:    21,935,192 bp\n",
      "  Read 2:    21,932,353 bp\n",
      "Total written (filtered):     40,006,104 bp (91.2%)\n",
      "  Read 1:    20,003,588 bp\n",
      "  Read 2:    20,002,516 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t587\t0.0\t2\t0 568 19\n",
      "26\t73329\t0.0\t2\t71688 1615 26\n",
      "27\t253\t0.0\t2\t0 240 13\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t702\t0.0\t2\t0 681 21\n",
      "26\t73137\t0.0\t2\t71428 1665 44\n",
      "27\t315\t0.0\t2\t0 300 15\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S115_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S115_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S115_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S115_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S115_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S115_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S115_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S115_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        72,433 reads  @     85.6 µs/read;   0.70 M reads/minute\n",
      "Finished in 6.21 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,433\n",
      "  Read 1 with adapter:                  72,382 (99.9%)\n",
      "  Read 2 with adapter:                  72,309 (99.8%)\n",
      "Pairs written (passing filters):        72,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,734,495 bp\n",
      "  Read 1:    21,367,240 bp\n",
      "  Read 2:    21,367,255 bp\n",
      "Total written (filtered):     38,973,306 bp (91.2%)\n",
      "  Read 1:    19,485,644 bp\n",
      "  Read 2:    19,487,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72382 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t584\t0.0\t2\t0 565 19\n",
      "26\t71482\t0.0\t2\t69804 1655 23\n",
      "27\t208\t0.0\t2\t0 202 6\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t716\t0.0\t2\t0 703 13\n",
      "26\t71191\t0.0\t2\t69517 1612 62\n",
      "27\t293\t0.0\t2\t0 282 11\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S116_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S116_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S116_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S116_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S116_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S116_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S116_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S116_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,131 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 5.69 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,131\n",
      "  Read 1 with adapter:                  65,678 (99.3%)\n",
      "  Read 2 with adapter:                  66,025 (99.8%)\n",
      "Pairs written (passing filters):        66,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,947,360 bp\n",
      "  Read 1:    19,443,748 bp\n",
      "  Read 2:    19,503,612 bp\n",
      "Total written (filtered):     35,523,896 bp (91.2%)\n",
      "  Read 1:    17,736,542 bp\n",
      "  Read 2:    17,787,354 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t594\t0.0\t2\t0 584 10\n",
      "26\t64812\t0.0\t2\t63156 1637 19\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t657\t0.0\t2\t0 638 19\n",
      "26\t65010\t0.0\t2\t63541 1427 42\n",
      "27\t263\t0.0\t2\t0 253 10\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S117_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S117_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S117_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S117_227_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S117_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S117_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S117_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S117_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        62,168 reads  @     84.3 µs/read;   0.71 M reads/minute\n",
      "Finished in 5.24 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,168\n",
      "  Read 1 with adapter:                  62,110 (99.9%)\n",
      "  Read 2 with adapter:                  62,085 (99.9%)\n",
      "Pairs written (passing filters):        62,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,673,000 bp\n",
      "  Read 1:    18,336,271 bp\n",
      "  Read 2:    18,336,729 bp\n",
      "Total written (filtered):     33,444,604 bp (91.2%)\n",
      "  Read 1:    16,721,687 bp\n",
      "  Read 2:    16,722,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t480\t0.0\t2\t0 465 15\n",
      "26\t61328\t0.0\t2\t59889 1417 22\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t591\t0.0\t2\t0 569 22\n",
      "26\t61193\t0.0\t2\t59684 1433 76\n",
      "27\t249\t0.0\t2\t0 236 13\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S118_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S118_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S118_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S118_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S118_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S118_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S118_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S118_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        79,734 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 6.48 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,734\n",
      "  Read 1 with adapter:                  79,657 (99.9%)\n",
      "  Read 2 with adapter:                  79,609 (99.8%)\n",
      "Pairs written (passing filters):        79,734 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,035,147 bp\n",
      "  Read 1:    23,517,450 bp\n",
      "  Read 2:    23,517,697 bp\n",
      "Total written (filtered):     42,895,079 bp (91.2%)\n",
      "  Read 1:    21,446,742 bp\n",
      "  Read 2:    21,448,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t690\t0.0\t2\t0 673 17\n",
      "26\t78552\t0.0\t2\t76647 1878 27\n",
      "27\t274\t0.0\t2\t0 261 13\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 79609 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t781\t0.0\t2\t0 759 22\n",
      "26\t78377\t0.0\t2\t76440 1874 63\n",
      "27\t355\t0.0\t2\t0 343 12\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S119_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S119_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S119_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S119_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S119_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S119_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S119_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S119_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        87,641 reads  @     79.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 6.97 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,641\n",
      "  Read 1 with adapter:                  87,568 (99.9%)\n",
      "  Read 2 with adapter:                  87,489 (99.8%)\n",
      "Pairs written (passing filters):        87,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,700,500 bp\n",
      "  Read 1:    25,848,792 bp\n",
      "  Read 2:    25,851,708 bp\n",
      "Total written (filtered):     47,149,880 bp (91.2%)\n",
      "  Read 1:    23,572,432 bp\n",
      "  Read 2:    23,577,448 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 87568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t757\t0.0\t2\t0 735 22\n",
      "26\t86377\t0.0\t2\t84293 2044 40\n",
      "27\t315\t0.0\t2\t0 303 12\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 87489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t769\t0.0\t2\t0 750 19\n",
      "26\t86235\t0.0\t2\t84138 2026 71\n",
      "27\t387\t0.0\t2\t0 374 13\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S120_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S120_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S120_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S120_233_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S120_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S120_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S120_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S120_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,940 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.80 s (77 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,940\n",
      "  Read 1 with adapter:                  61,876 (99.9%)\n",
      "  Read 2 with adapter:                  61,803 (99.8%)\n",
      "Pairs written (passing filters):        61,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,530,798 bp\n",
      "  Read 1:    18,270,519 bp\n",
      "  Read 2:    18,260,279 bp\n",
      "Total written (filtered):     33,316,045 bp (91.2%)\n",
      "  Read 1:    16,662,221 bp\n",
      "  Read 2:    16,653,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t725\t0.0\t2\t0 699 26\n",
      "26\t60838\t0.0\t2\t59214 1596 28\n",
      "27\t163\t0.0\t2\t0 151 12\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t636\t0.0\t2\t0 623 13\n",
      "26\t60820\t0.0\t2\t59220 1539 61\n",
      "27\t281\t0.0\t2\t0 266 15\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S121_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S121_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S121_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S121_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S121_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S121_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S121_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S121_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        74,940 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 5.62 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,940\n",
      "  Read 1 with adapter:                  74,868 (99.9%)\n",
      "  Read 2 with adapter:                  74,837 (99.9%)\n",
      "Pairs written (passing filters):        74,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,209,135 bp\n",
      "  Read 1:    22,106,818 bp\n",
      "  Read 2:    22,102,317 bp\n",
      "Total written (filtered):     40,317,984 bp (91.2%)\n",
      "  Read 1:    20,160,879 bp\n",
      "  Read 2:    20,157,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t833\t0.0\t2\t0 812 21\n",
      "26\t73733\t0.0\t2\t71927 1785 21\n",
      "27\t184\t0.0\t2\t0 176 8\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74837 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t796\t0.0\t2\t0 763 33\n",
      "26\t73648\t0.0\t2\t71888 1695 65\n",
      "27\t324\t0.0\t2\t0 314 10\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S122_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S122_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S122_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S122_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S122_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-j0_ziqbl/S122_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S122_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-u6_bil41/4bac5890-fc8f-4be6-8665-5535a1712d96/data/S122_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        85,904 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 6.70 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,904\n",
      "  Read 1 with adapter:                  85,799 (99.9%)\n",
      "  Read 2 with adapter:                  85,738 (99.8%)\n",
      "Pairs written (passing filters):        85,904 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,675,309 bp\n",
      "  Read 1:    25,338,831 bp\n",
      "  Read 2:    25,336,478 bp\n",
      "Total written (filtered):     46,216,532 bp (91.2%)\n",
      "  Read 1:    23,108,716 bp\n",
      "  Read 2:    23,107,816 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 85799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t958\t0.0\t2\t0 932 26\n",
      "26\t84422\t0.0\t2\t82233 2154 35\n",
      "27\t259\t0.0\t2\t0 245 14\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 85738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t816\t0.0\t2\t0 791 25\n",
      "26\t84458\t0.0\t2\t82290 2078 90\n",
      "27\t358\t0.0\t2\t0 339 19\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: a-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: a-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-i3b076jb')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 24bf734b-b37c-4949-ace4-fbc2291580bb>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.a import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib1-leray_manifest_run1_grid.csv \\\n",
    "  --output-path a-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.a trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences a-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences a-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.a summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data a-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization a-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('a-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: a-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: a-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: a-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.a Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs a-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 260 \\\n",
    "  --p-trunc-len-r 220 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table a-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats a-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: a-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-ndxc_om5')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 4a7e4a54-88ef-4efa-bda7-7986b466f848>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.a view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table a-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization a-table.qzv \n",
    "q2.Visualization.load('a-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: a-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.a view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization a-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib2-leray_manifest_run2_v1.csv as PairedEndFastqManifestPhred33 to b-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S251_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S251_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S251_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S251_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S251_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S251_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S251_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S251_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,869 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.16 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,869\n",
      "  Read 1 with adapter:                  46,812 (99.9%)\n",
      "  Read 2 with adapter:                  46,776 (99.8%)\n",
      "Pairs written (passing filters):        46,869 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,650,425 bp\n",
      "  Read 1:    13,825,108 bp\n",
      "  Read 2:    13,825,317 bp\n",
      "Total written (filtered):     25,217,847 bp (91.2%)\n",
      "  Read 1:    12,608,002 bp\n",
      "  Read 2:    12,609,845 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t303\t0.0\t2\t0 286 17\n",
      "26\t46212\t0.0\t2\t44741 1430 41\n",
      "27\t185\t0.0\t2\t0 175 10\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t763\t0.0\t2\t0 742 21\n",
      "26\t45750\t0.0\t2\t44816 902 32\n",
      "27\t167\t0.0\t2\t0 161 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S252_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S252_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S252_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S252_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S252_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S252_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S252_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S252_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,127 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.89 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,127\n",
      "  Read 1 with adapter:                  62,050 (99.9%)\n",
      "  Read 2 with adapter:                  61,983 (99.8%)\n",
      "Pairs written (passing filters):        62,127 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,651,992 bp\n",
      "  Read 1:    18,326,787 bp\n",
      "  Read 2:    18,325,205 bp\n",
      "Total written (filtered):     33,428,109 bp (91.2%)\n",
      "  Read 1:    16,713,544 bp\n",
      "  Read 2:    16,714,565 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t457\t0.0\t2\t0 442 15\n",
      "26\t61180\t0.0\t2\t59245 1887 48\n",
      "27\t262\t0.0\t2\t0 253 9\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61983 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t1032\t0.0\t2\t0 999 33\n",
      "26\t60577\t0.0\t2\t59179 1351 47\n",
      "27\t278\t0.0\t2\t0 265 13\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S253_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S253_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S253_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S253_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S253_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S253_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S253_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S253_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,830 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.00 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,830\n",
      "  Read 1 with adapter:                  53,755 (99.9%)\n",
      "  Read 2 with adapter:                  53,702 (99.8%)\n",
      "Pairs written (passing filters):        53,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,756,516 bp\n",
      "  Read 1:    15,879,630 bp\n",
      "  Read 2:    15,876,886 bp\n",
      "Total written (filtered):     28,963,519 bp (91.2%)\n",
      "  Read 1:    14,482,038 bp\n",
      "  Read 2:    14,481,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t387\t0.0\t2\t0 377 10\n",
      "26\t52980\t0.0\t2\t51261 1672 47\n",
      "27\t243\t0.0\t2\t0 239 4\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t919\t0.0\t2\t0 899 20\n",
      "26\t52454\t0.0\t2\t51262 1161 31\n",
      "27\t230\t0.0\t2\t0 217 13\n",
      "28\t10\t0.0\t2\t0 0 10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S254_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S254_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S254_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S254_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S254_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S254_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S254_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S254_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,257 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.96 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,257\n",
      "  Read 1 with adapter:                  34,215 (99.9%)\n",
      "  Read 2 with adapter:                  34,087 (99.5%)\n",
      "Pairs written (passing filters):        34,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,198,540 bp\n",
      "  Read 1:    10,105,561 bp\n",
      "  Read 2:    10,092,979 bp\n",
      "Total written (filtered):     18,423,313 bp (91.2%)\n",
      "  Read 1:     9,216,124 bp\n",
      "  Read 2:     9,207,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t301\t0.0\t2\t0 288 13\n",
      "26\t33735\t0.0\t2\t32651 1050 34\n",
      "27\t130\t0.0\t2\t0 128 2\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t550\t0.0\t2\t0 531 19\n",
      "26\t33359\t0.0\t2\t32493 833 33\n",
      "27\t138\t0.0\t2\t0 129 9\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S255_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S255_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S255_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S255_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S255_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S255_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S255_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S255_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,440 reads  @     93.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.68 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,440\n",
      "  Read 1 with adapter:                  39,385 (99.9%)\n",
      "  Read 2 with adapter:                  37,218 (94.4%)\n",
      "Pairs written (passing filters):        39,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,805,159 bp\n",
      "  Read 1:    11,633,658 bp\n",
      "  Read 2:    11,171,501 bp\n",
      "Total written (filtered):     20,814,368 bp (91.3%)\n",
      "  Read 1:    10,609,788 bp\n",
      "  Read 2:    10,204,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t309\t0.0\t2\t0 297 12\n",
      "26\t38865\t0.0\t2\t37669 1170 26\n",
      "27\t133\t0.0\t2\t0 129 4\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t789\t0.0\t2\t0 758 31\n",
      "26\t36209\t0.0\t2\t35329 854 26\n",
      "27\t150\t0.0\t2\t0 143 7\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S256_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S256_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S256_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S256_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S256_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S256_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S256_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S256_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,723 reads  @     96.8 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.62 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,723\n",
      "  Read 1 with adapter:                  47,650 (99.8%)\n",
      "  Read 2 with adapter:                  47,529 (99.6%)\n",
      "Pairs written (passing filters):        47,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,143,456 bp\n",
      "  Read 1:    14,077,353 bp\n",
      "  Read 2:    14,066,103 bp\n",
      "Total written (filtered):     25,669,355 bp (91.2%)\n",
      "  Read 1:    12,838,167 bp\n",
      "  Read 2:    12,831,188 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t422\t0.0\t2\t0 413 9\n",
      "26\t46995\t0.0\t2\t45523 1432 40\n",
      "27\t164\t0.0\t2\t0 160 4\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47529 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t93\t0.0\t2\t0 0 93\n",
      "25\t845\t0.0\t2\t0 826 19\n",
      "26\t46404\t0.0\t2\t45316 1051 37\n",
      "27\t182\t0.0\t2\t0 175 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S257_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S257_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S257_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S257_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S257_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S257_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S257_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S257_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        56,870 reads  @     88.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 5.06 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,870\n",
      "  Read 1 with adapter:                  56,790 (99.9%)\n",
      "  Read 2 with adapter:                  56,782 (99.8%)\n",
      "Pairs written (passing filters):        56,870 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,547,054 bp\n",
      "  Read 1:    16,774,963 bp\n",
      "  Read 2:    16,772,091 bp\n",
      "Total written (filtered):     30,595,322 bp (91.2%)\n",
      "  Read 1:    15,298,629 bp\n",
      "  Read 2:    15,296,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56790 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t397\t0.0\t2\t0 391 6\n",
      "26\t56127\t0.0\t2\t54507 1581 39\n",
      "27\t173\t0.0\t2\t0 165 8\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t1015\t0.0\t2\t0 992 23\n",
      "26\t55460\t0.0\t2\t54269 1164 27\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S258_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S258_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S258_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S258_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S258_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S258_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S258_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S258_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,189 reads  @     89.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.92 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,189\n",
      "  Read 1 with adapter:                  55,119 (99.9%)\n",
      "  Read 2 with adapter:                  55,078 (99.8%)\n",
      "Pairs written (passing filters):        55,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,560,239 bp\n",
      "  Read 1:    16,280,447 bp\n",
      "  Read 2:    16,279,792 bp\n",
      "Total written (filtered):     29,695,976 bp (91.2%)\n",
      "  Read 1:    14,847,484 bp\n",
      "  Read 2:    14,848,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55119 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t397\t0.0\t2\t0 390 7\n",
      "26\t54439\t0.0\t2\t52850 1548 41\n",
      "27\t188\t0.0\t2\t0 181 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t840\t0.0\t2\t0 821 19\n",
      "26\t53974\t0.0\t2\t52822 1118 34\n",
      "27\t180\t0.0\t2\t0 173 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S259_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S259_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S259_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S259_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S259_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S259_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S259_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S259_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,113 reads  @     85.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.71 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,113\n",
      "  Read 1 with adapter:                  55,045 (99.9%)\n",
      "  Read 2 with adapter:                  54,966 (99.7%)\n",
      "Pairs written (passing filters):        55,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,515,219 bp\n",
      "  Read 1:    16,257,941 bp\n",
      "  Read 2:    16,257,278 bp\n",
      "Total written (filtered):     29,655,857 bp (91.2%)\n",
      "  Read 1:    14,826,879 bp\n",
      "  Read 2:    14,828,978 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t420\t0.0\t2\t0 409 11\n",
      "26\t54303\t0.0\t2\t52554 1688 61\n",
      "27\t228\t0.0\t2\t0 224 4\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54966 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t98\t0.0\t2\t0 0 98\n",
      "25\t927\t0.0\t2\t0 902 25\n",
      "26\t53675\t0.0\t2\t52575 1059 41\n",
      "27\t225\t0.0\t2\t0 213 12\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S260_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S260_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S260_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S260_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S260_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S260_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S260_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S260_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,246 reads  @     91.2 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.95 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,246\n",
      "  Read 1 with adapter:                  43,169 (99.8%)\n",
      "  Read 2 with adapter:                  43,137 (99.7%)\n",
      "Pairs written (passing filters):        43,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,512,748 bp\n",
      "  Read 1:    12,757,270 bp\n",
      "  Read 2:    12,755,478 bp\n",
      "Total written (filtered):     23,269,602 bp (91.2%)\n",
      "  Read 1:    11,635,024 bp\n",
      "  Read 2:    11,634,578 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t370\t0.0\t2\t0 357 13\n",
      "26\t42583\t0.0\t2\t41141 1400 42\n",
      "27\t150\t0.0\t2\t0 146 4\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t715\t0.0\t2\t0 703 12\n",
      "26\t42163\t0.0\t2\t41151 983 29\n",
      "27\t181\t0.0\t2\t0 175 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S261_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S261_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S261_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S261_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S261_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S261_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S261_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S261_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,797 reads  @     86.8 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.15 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,797\n",
      "  Read 1 with adapter:                  47,718 (99.8%)\n",
      "  Read 2 with adapter:                  47,719 (99.8%)\n",
      "Pairs written (passing filters):        47,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,196,437 bp\n",
      "  Read 1:    14,099,812 bp\n",
      "  Read 2:    14,096,625 bp\n",
      "Total written (filtered):     25,715,498 bp (91.2%)\n",
      "  Read 1:    12,859,296 bp\n",
      "  Read 2:    12,856,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t403\t0.0\t2\t0 390 13\n",
      "26\t47047\t0.0\t2\t45401 1613 33\n",
      "27\t169\t0.0\t2\t0 161 8\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47719 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t609\t0.0\t2\t0 589 20\n",
      "26\t46866\t0.0\t2\t45851 992 23\n",
      "27\t183\t0.0\t2\t0 174 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S262_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S262_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S262_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S262_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S262_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S262_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S262_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S262_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,787 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.67 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,787\n",
      "  Read 1 with adapter:                  45,732 (99.9%)\n",
      "  Read 2 with adapter:                  45,518 (99.4%)\n",
      "Pairs written (passing filters):        45,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,988,157 bp\n",
      "  Read 1:    13,507,158 bp\n",
      "  Read 2:    13,480,999 bp\n",
      "Total written (filtered):     24,615,930 bp (91.2%)\n",
      "  Read 1:    12,317,925 bp\n",
      "  Read 2:    12,298,005 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t345\t0.0\t2\t0 336 9\n",
      "26\t45107\t0.0\t2\t43729 1334 44\n",
      "27\t191\t0.0\t2\t0 186 5\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t556\t0.0\t2\t0 532 24\n",
      "26\t44777\t0.0\t2\t43790 936 51\n",
      "27\t148\t0.0\t2\t0 145 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S263_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S263_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S263_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S263_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S263_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S263_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S263_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S263_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,933 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.26 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,933\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                  47,538 (99.2%)\n",
      "Pairs written (passing filters):        47,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,154,146 bp\n",
      "  Read 1:    14,077,073 bp\n",
      "  Read 2:    14,077,073 bp\n",
      "Total written (filtered):     26,918,845 bp (95.6%)\n",
      "  Read 1:    14,077,073 bp\n",
      "  Read 2:    12,841,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47538 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t774\t0.0\t2\t0 751 23\n",
      "26\t46519\t0.0\t2\t45433 1035 51\n",
      "27\t175\t0.0\t2\t0 165 10\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S264_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S264_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S264_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S264_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S264_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S264_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S264_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S264_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,209 reads  @     92.6 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.47 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,209\n",
      "  Read 1 with adapter:                  48,145 (99.9%)\n",
      "  Read 2 with adapter:                  48,059 (99.7%)\n",
      "Pairs written (passing filters):        48,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,423,171 bp\n",
      "  Read 1:    14,221,650 bp\n",
      "  Read 2:    14,201,521 bp\n",
      "Total written (filtered):     25,922,109 bp (91.2%)\n",
      "  Read 1:    12,969,480 bp\n",
      "  Read 2:    12,952,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48145 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t395\t0.0\t2\t0 386 9\n",
      "26\t47477\t0.0\t2\t45748 1679 50\n",
      "27\t165\t0.0\t2\t0 157 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t717\t0.0\t2\t0 705 12\n",
      "26\t47068\t0.0\t2\t45952 1093 23\n",
      "27\t189\t0.0\t2\t0 180 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S265_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S265_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S265_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S265_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S265_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S265_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S265_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S265_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,096 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.82 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,096\n",
      "  Read 1 with adapter:                  46,020 (99.8%)\n",
      "  Read 2 with adapter:                  45,996 (99.8%)\n",
      "Pairs written (passing filters):        46,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,192,507 bp\n",
      "  Read 1:    13,597,758 bp\n",
      "  Read 2:    13,594,749 bp\n",
      "Total written (filtered):     24,801,112 bp (91.2%)\n",
      "  Read 1:    12,401,521 bp\n",
      "  Read 2:    12,399,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46020 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t415\t0.0\t2\t0 397 18\n",
      "26\t45417\t0.0\t2\t43901 1469 47\n",
      "27\t116\t0.0\t2\t0 110 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t860\t0.0\t2\t0 834 26\n",
      "26\t44868\t0.0\t2\t43836 994 38\n",
      "27\t202\t0.0\t2\t0 194 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S266_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S266_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S266_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S266_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S266_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S266_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S266_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S266_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,616 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.15 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,616\n",
      "  Read 1 with adapter:                  52,547 (99.9%)\n",
      "  Read 2 with adapter:                  52,493 (99.8%)\n",
      "Pairs written (passing filters):        52,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,042,444 bp\n",
      "  Read 1:    15,521,543 bp\n",
      "  Read 2:    15,520,901 bp\n",
      "Total written (filtered):     28,312,224 bp (91.2%)\n",
      "  Read 1:    14,155,525 bp\n",
      "  Read 2:    14,156,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t419\t0.0\t2\t0 409 10\n",
      "26\t51863\t0.0\t2\t50262 1550 51\n",
      "27\t195\t0.0\t2\t0 188 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t734\t0.0\t2\t0 717 17\n",
      "26\t51498\t0.0\t2\t50381 1079 38\n",
      "27\t160\t0.0\t2\t0 154 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S267_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S267_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S267_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S267_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S267_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S267_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S267_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S267_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,888 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 5.40 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,888\n",
      "  Read 1 with adapter:                  65,717 (99.7%)\n",
      "  Read 2 with adapter:                  65,744 (99.8%)\n",
      "Pairs written (passing filters):        65,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,866,280 bp\n",
      "  Read 1:    19,430,148 bp\n",
      "  Read 2:    19,436,132 bp\n",
      "Total written (filtered):     35,449,271 bp (91.2%)\n",
      "  Read 1:    17,721,547 bp\n",
      "  Read 2:    17,727,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65717 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t439\t0.0\t2\t0 422 17\n",
      "26\t64868\t0.0\t2\t62768 2054 46\n",
      "27\t254\t0.0\t2\t0 246 8\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t95\t0.0\t2\t0 0 95\n",
      "25\t1098\t0.0\t2\t0 1065 33\n",
      "26\t64215\t0.0\t2\t62630 1533 52\n",
      "27\t320\t0.0\t2\t0 309 11\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S268_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S268_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S268_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S268_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S268_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S268_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S268_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S268_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,840 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.23 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,840\n",
      "  Read 1 with adapter:                  55,598 (99.6%)\n",
      "  Read 2 with adapter:                  55,745 (99.8%)\n",
      "Pairs written (passing filters):        55,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,934,106 bp\n",
      "  Read 1:    16,463,440 bp\n",
      "  Read 2:    16,470,666 bp\n",
      "Total written (filtered):     30,039,868 bp (91.2%)\n",
      "  Read 1:    15,017,875 bp\n",
      "  Read 2:    15,021,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t397\t0.0\t2\t0 381 16\n",
      "26\t54803\t0.0\t2\t53084 1679 40\n",
      "27\t246\t0.0\t2\t0 237 9\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55745 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t80\t0.0\t2\t0 0 80\n",
      "25\t776\t0.0\t2\t0 759 17\n",
      "26\t54656\t0.0\t2\t53351 1258 47\n",
      "27\t227\t0.0\t2\t0 214 13\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S269_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S269_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S269_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S269_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S269_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S269_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S269_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S269_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,171 reads  @     76.5 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.23 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,171\n",
      "  Read 1 with adapter:                  41,992 (99.6%)\n",
      "  Read 2 with adapter:                  42,102 (99.8%)\n",
      "Pairs written (passing filters):        42,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,871,331 bp\n",
      "  Read 1:    12,434,517 bp\n",
      "  Read 2:    12,436,814 bp\n",
      "Total written (filtered):     22,685,385 bp (91.2%)\n",
      "  Read 1:    11,342,732 bp\n",
      "  Read 2:    11,342,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t275\t0.0\t2\t0 269 6\n",
      "26\t41456\t0.0\t2\t40169 1244 43\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t551\t0.0\t2\t0 542 9\n",
      "26\t41354\t0.0\t2\t40525 802 27\n",
      "27\t150\t0.0\t2\t0 141 9\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S270_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S270_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S270_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S270_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S270_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S270_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S270_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S270_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,855 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.34 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,855\n",
      "  Read 1 with adapter:                  40,750 (99.7%)\n",
      "  Read 2 with adapter:                  40,714 (99.7%)\n",
      "Pairs written (passing filters):        40,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,095,183 bp\n",
      "  Read 1:    12,050,009 bp\n",
      "  Read 2:    12,045,174 bp\n",
      "Total written (filtered):     21,977,700 bp (91.2%)\n",
      "  Read 1:    10,990,526 bp\n",
      "  Read 2:    10,987,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40750 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t306\t0.0\t2\t0 296 10\n",
      "26\t40144\t0.0\t2\t38865 1238 41\n",
      "27\t167\t0.0\t2\t0 163 4\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40714 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t622\t0.0\t2\t0 609 13\n",
      "26\t39899\t0.0\t2\t38887 956 56\n",
      "27\t148\t0.0\t2\t0 142 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S271_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S271_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S271_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S271_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S271_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S271_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S271_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S271_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,916 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.89 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,916\n",
      "  Read 1 with adapter:                  46,826 (99.8%)\n",
      "  Read 2 with adapter:                  46,304 (98.7%)\n",
      "Pairs written (passing filters):        46,915 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,570,307 bp\n",
      "  Read 1:    13,838,345 bp\n",
      "  Read 2:    13,731,962 bp\n",
      "Total written (filtered):     25,149,486 bp (91.2%)\n",
      "  Read 1:    12,620,563 bp\n",
      "  Read 2:    12,528,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46826 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t308\t0.0\t2\t0 299 9\n",
      "26\t46195\t0.0\t2\t44763 1396 36\n",
      "27\t185\t0.0\t2\t0 176 9\n",
      "28\t109\t0.0\t2\t0 0 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t926\t0.0\t2\t0 897 29\n",
      "26\t45158\t0.0\t2\t44108 999 51\n",
      "27\t163\t0.0\t2\t0 156 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S272_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S272_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S272_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S272_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S272_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S272_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S272_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S272_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,323 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.47 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,323\n",
      "  Read 1 with adapter:                  42,241 (99.8%)\n",
      "  Read 2 with adapter:                  42,211 (99.7%)\n",
      "Pairs written (passing filters):        42,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,961,469 bp\n",
      "  Read 1:    12,483,218 bp\n",
      "  Read 2:    12,478,251 bp\n",
      "Total written (filtered):     22,765,699 bp (91.2%)\n",
      "  Read 1:    11,384,348 bp\n",
      "  Read 2:    11,381,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t280\t0.0\t2\t0 273 7\n",
      "26\t41659\t0.0\t2\t40359 1267 33\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42211 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t652\t0.0\t2\t0 637 15\n",
      "26\t41349\t0.0\t2\t40402 909 38\n",
      "27\t154\t0.0\t2\t0 147 7\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S273_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S273_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S273_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S273_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S273_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S273_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S273_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S273_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,309 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.96 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,309\n",
      "  Read 1 with adapter:                  49,223 (99.8%)\n",
      "  Read 2 with adapter:                  49,181 (99.7%)\n",
      "Pairs written (passing filters):        49,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,086,434 bp\n",
      "  Read 1:    14,545,134 bp\n",
      "  Read 2:    14,541,300 bp\n",
      "Total written (filtered):     26,528,877 bp (91.2%)\n",
      "  Read 1:    13,265,406 bp\n",
      "  Read 2:    13,263,471 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t377\t0.0\t2\t0 371 6\n",
      "26\t48499\t0.0\t2\t47003 1460 36\n",
      "27\t227\t0.0\t2\t0 215 12\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t80\t0.0\t2\t0 0 80\n",
      "25\t946\t0.0\t2\t0 907 39\n",
      "26\t47945\t0.0\t2\t46889 1018 38\n",
      "27\t191\t0.0\t2\t0 181 10\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S274_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S274_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S274_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S274_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S274_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S274_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S274_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S274_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,472 reads  @     78.9 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.38 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,472\n",
      "  Read 1 with adapter:                  55,061 (99.3%)\n",
      "  Read 2 with adapter:                  55,372 (99.8%)\n",
      "Pairs written (passing filters):        55,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,689,376 bp\n",
      "  Read 1:    16,326,076 bp\n",
      "  Read 2:    16,363,300 bp\n",
      "Total written (filtered):     29,819,151 bp (91.2%)\n",
      "  Read 1:    14,894,751 bp\n",
      "  Read 2:    14,924,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t469\t0.0\t2\t0 450 19\n",
      "26\t54333\t0.0\t2\t52540 1747 46\n",
      "27\t162\t0.0\t2\t0 158 4\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t872\t0.0\t2\t0 831 41\n",
      "26\t54205\t0.0\t2\t52910 1232 63\n",
      "27\t218\t0.0\t2\t0 211 7\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S275_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S275_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S275_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S275_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S275_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S275_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S275_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S275_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        26,476 reads  @     80.0 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.12 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,476\n",
      "  Read 1 with adapter:                  26,437 (99.9%)\n",
      "  Read 2 with adapter:                  26,424 (99.8%)\n",
      "Pairs written (passing filters):        26,476 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,619,359 bp\n",
      "  Read 1:     7,809,385 bp\n",
      "  Read 2:     7,809,974 bp\n",
      "Total written (filtered):     14,245,317 bp (91.2%)\n",
      "  Read 1:     7,122,012 bp\n",
      "  Read 2:     7,123,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26437 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t149\t0.0\t2\t0 143 6\n",
      "26\t26115\t0.0\t2\t25078 1004 33\n",
      "27\t98\t0.0\t2\t0 95 3\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t407\t0.0\t2\t0 400 7\n",
      "26\t25875\t0.0\t2\t25299 548 28\n",
      "27\t108\t0.0\t2\t0 105 3\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S276_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S276_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S276_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S276_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S276_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S276_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S276_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S276_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,056 reads  @     80.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.76 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,056\n",
      "  Read 1 with adapter:                  58,972 (99.9%)\n",
      "  Read 2 with adapter:                  58,925 (99.8%)\n",
      "Pairs written (passing filters):        59,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,840,117 bp\n",
      "  Read 1:    17,421,238 bp\n",
      "  Read 2:    17,418,879 bp\n",
      "Total written (filtered):     31,775,741 bp (91.2%)\n",
      "  Read 1:    15,888,143 bp\n",
      "  Read 2:    15,887,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t432\t0.0\t2\t0 424 8\n",
      "26\t58254\t0.0\t2\t56555 1658 41\n",
      "27\t165\t0.0\t2\t0 159 6\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58925 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t862\t0.0\t2\t0 828 34\n",
      "26\t57748\t0.0\t2\t56501 1198 49\n",
      "27\t237\t0.0\t2\t0 231 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S277_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S277_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S277_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S277_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S277_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S277_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S277_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S277_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,078 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.75 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,078\n",
      "  Read 1 with adapter:                  47,017 (99.9%)\n",
      "  Read 2 with adapter:                  47,025 (99.9%)\n",
      "Pairs written (passing filters):        47,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,772,890 bp\n",
      "  Read 1:    13,887,603 bp\n",
      "  Read 2:    13,885,287 bp\n",
      "Total written (filtered):     25,328,447 bp (91.2%)\n",
      "  Read 1:    12,665,288 bp\n",
      "  Read 2:    12,663,159 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t316\t0.0\t2\t0 307 9\n",
      "26\t46500\t0.0\t2\t45044 1416 40\n",
      "27\t117\t0.0\t2\t0 114 3\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t614\t0.0\t2\t0 599 15\n",
      "26\t46224\t0.0\t2\t45289 911 24\n",
      "27\t138\t0.0\t2\t0 135 3\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S278_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S278_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S278_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S278_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S278_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S278_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S278_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S278_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,786 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.80 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,786\n",
      "  Read 1 with adapter:                  45,730 (99.9%)\n",
      "  Read 2 with adapter:                  45,668 (99.7%)\n",
      "Pairs written (passing filters):        45,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,007,989 bp\n",
      "  Read 1:    13,506,860 bp\n",
      "  Read 2:    13,501,129 bp\n",
      "Total written (filtered):     24,632,333 bp (91.2%)\n",
      "  Read 1:    12,317,991 bp\n",
      "  Read 2:    12,314,342 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t316\t0.0\t2\t0 308 8\n",
      "26\t45186\t0.0\t2\t43854 1300 32\n",
      "27\t139\t0.0\t2\t0 136 3\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45668 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t682\t0.0\t2\t0 656 26\n",
      "26\t44742\t0.0\t2\t43669 1020 53\n",
      "27\t183\t0.0\t2\t0 173 10\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S279_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S279_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S279_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S279_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S279_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S279_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S279_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S279_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,468 reads  @     77.6 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.61 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,468\n",
      "  Read 1 with adapter:                  46,404 (99.9%)\n",
      "  Read 2 with adapter:                  46,001 (99.0%)\n",
      "Pairs written (passing filters):        46,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,334,355 bp\n",
      "  Read 1:    13,707,762 bp\n",
      "  Read 2:    13,626,593 bp\n",
      "Total written (filtered):     24,932,732 bp (91.2%)\n",
      "  Read 1:    12,501,369 bp\n",
      "  Read 2:    12,431,363 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t301\t0.0\t2\t0 289 12\n",
      "26\t45889\t0.0\t2\t44560 1297 32\n",
      "27\t130\t0.0\t2\t0 124 6\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t847\t0.0\t2\t0 820 27\n",
      "26\t44954\t0.0\t2\t44032 888 34\n",
      "27\t121\t0.0\t2\t0 119 2\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S280_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S280_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S280_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S280_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S280_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S280_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S280_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S280_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,579 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.74 s (75 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,579\n",
      "  Read 1 with adapter:                  49,507 (99.9%)\n",
      "  Read 2 with adapter:                  49,486 (99.8%)\n",
      "Pairs written (passing filters):        49,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,245,247 bp\n",
      "  Read 1:    14,625,143 bp\n",
      "  Read 2:    14,620,104 bp\n",
      "Total written (filtered):     26,672,046 bp (91.2%)\n",
      "  Read 1:    13,337,950 bp\n",
      "  Read 2:    13,334,096 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49507 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t307\t0.0\t2\t0 303 4\n",
      "26\t48926\t0.0\t2\t47369 1523 34\n",
      "27\t138\t0.0\t2\t0 132 6\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49486 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t688\t0.0\t2\t0 671 17\n",
      "26\t48610\t0.0\t2\t47542 1031 37\n",
      "27\t144\t0.0\t2\t0 134 10\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S281_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S281_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S281_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S281_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S281_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S281_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S281_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S281_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        34,550 reads  @     95.1 µs/read;   0.63 M reads/minute\n",
      "Finished in 3.29 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,550\n",
      "  Read 1 with adapter:                  34,506 (99.9%)\n",
      "  Read 2 with adapter:                  34,433 (99.7%)\n",
      "Pairs written (passing filters):        34,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,375,879 bp\n",
      "  Read 1:    10,192,245 bp\n",
      "  Read 2:    10,183,634 bp\n",
      "Total written (filtered):     18,584,019 bp (91.2%)\n",
      "  Read 1:     9,295,044 bp\n",
      "  Read 2:     9,288,975 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t218\t0.0\t2\t0 212 6\n",
      "26\t34066\t0.0\t2\t32893 1142 31\n",
      "27\t105\t0.0\t2\t0 104 1\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t612\t0.0\t2\t0 590 22\n",
      "26\t33659\t0.0\t2\t32614 991 54\n",
      "27\t111\t0.0\t2\t0 106 5\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S282_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S282_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S282_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S282_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S282_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S282_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S282_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S282_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,891 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.41 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,891\n",
      "  Read 1 with adapter:                  62,812 (99.9%)\n",
      "  Read 2 with adapter:                  62,772 (99.8%)\n",
      "Pairs written (passing filters):        62,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,104,906 bp\n",
      "  Read 1:    18,552,560 bp\n",
      "  Read 2:    18,552,346 bp\n",
      "Total written (filtered):     33,840,857 bp (91.2%)\n",
      "  Read 1:    16,919,779 bp\n",
      "  Read 2:    16,921,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t431\t0.0\t2\t0 417 14\n",
      "26\t62184\t0.0\t2\t60127 2006 51\n",
      "27\t130\t0.0\t2\t0 126 4\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t930\t0.0\t2\t0 909 21\n",
      "26\t61536\t0.0\t2\t60134 1354 48\n",
      "27\t242\t0.0\t2\t0 231 11\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S283_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S283_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S283_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S283_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S283_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S283_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S283_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S283_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,750 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.58 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,750\n",
      "  Read 1 with adapter:                  45,706 (99.9%)\n",
      "  Read 2 with adapter:                  45,639 (99.8%)\n",
      "Pairs written (passing filters):        45,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,990,534 bp\n",
      "  Read 1:    13,494,815 bp\n",
      "  Read 2:    13,495,719 bp\n",
      "Total written (filtered):     24,616,636 bp (91.2%)\n",
      "  Read 1:    12,306,903 bp\n",
      "  Read 2:    12,309,733 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t567\t0.0\t2\t0 556 11\n",
      "26\t44947\t0.0\t2\t43419 1485 43\n",
      "27\t133\t0.0\t2\t0 125 8\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t704\t0.0\t2\t0 689 15\n",
      "26\t44669\t0.0\t2\t43816 839 14\n",
      "27\t200\t0.0\t2\t0 197 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S284_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S284_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S284_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S284_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S284_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S284_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S284_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S284_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,158 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.88 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,158\n",
      "  Read 1 with adapter:                  55,078 (99.9%)\n",
      "  Read 2 with adapter:                  55,067 (99.8%)\n",
      "Pairs written (passing filters):        55,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,541,629 bp\n",
      "  Read 1:    16,271,228 bp\n",
      "  Read 2:    16,270,401 bp\n",
      "Total written (filtered):     29,679,126 bp (91.2%)\n",
      "  Read 1:    14,839,692 bp\n",
      "  Read 2:    14,839,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t640\t0.0\t2\t0 626 14\n",
      "26\t54262\t0.0\t2\t52531 1686 45\n",
      "27\t120\t0.0\t2\t0 117 3\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t82\t0.0\t2\t0 0 82\n",
      "25\t842\t0.0\t2\t0 812 30\n",
      "26\t53915\t0.0\t2\t52819 1074 22\n",
      "27\t225\t0.0\t2\t0 220 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S285_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S285_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S285_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S285_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S285_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S285_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S285_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S285_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,576 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.70 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,576\n",
      "  Read 1 with adapter:                  51,506 (99.9%)\n",
      "  Read 2 with adapter:                  51,481 (99.8%)\n",
      "Pairs written (passing filters):        51,576 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,426,233 bp\n",
      "  Read 1:    15,214,587 bp\n",
      "  Read 2:    15,211,646 bp\n",
      "Total written (filtered):     27,749,723 bp (91.2%)\n",
      "  Read 1:    13,875,928 bp\n",
      "  Read 2:    13,873,795 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t657\t0.0\t2\t0 640 17\n",
      "26\t50634\t0.0\t2\t48866 1725 43\n",
      "27\t142\t0.0\t2\t0 138 4\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t790\t0.0\t2\t0 768 22\n",
      "26\t50391\t0.0\t2\t49315 1055 21\n",
      "27\t225\t0.0\t2\t0 218 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S286_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S286_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S286_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S286_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S286_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S286_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S286_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S286_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,574 reads  @    128.5 µs/read;   0.47 M reads/minute\n",
      "Finished in 0.20 s (130 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,574\n",
      "  Read 1 with adapter:                   1,569 (99.7%)\n",
      "  Read 2 with adapter:                   1,505 (95.6%)\n",
      "Pairs written (passing filters):         1,574 (100.0%)\n",
      "\n",
      "Total basepairs processed:       921,916 bp\n",
      "  Read 1:       464,119 bp\n",
      "  Read 2:       457,797 bp\n",
      "Total written (filtered):        842,024 bp (91.3%)\n",
      "  Read 1:       423,342 bp\n",
      "  Read 2:       418,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t26\t0.0\t2\t0 24 2\n",
      "26\t1532\t0.0\t2\t1450 79 3\n",
      "27\t9\t0.0\t2\t0 9\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1505 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t31\t0.0\t2\t0 30 1\n",
      "26\t1455\t0.0\t2\t1385 64 6\n",
      "27\t6\t0.0\t2\t0 5 1\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S287_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S287_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S287_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S287_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S287_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S287_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S287_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S287_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,194 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.10 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,194\n",
      "  Read 1 with adapter:                  40,157 (99.9%)\n",
      "  Read 2 with adapter:                  40,095 (99.8%)\n",
      "Pairs written (passing filters):        40,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,703,625 bp\n",
      "  Read 1:    11,856,763 bp\n",
      "  Read 2:    11,846,862 bp\n",
      "Total written (filtered):     21,617,744 bp (91.2%)\n",
      "  Read 1:    10,812,602 bp\n",
      "  Read 2:    10,805,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t427\t0.0\t2\t0 412 15\n",
      "26\t39494\t0.0\t2\t38291 1181 22\n",
      "27\t139\t0.0\t2\t0 135 4\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40095 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t789\t0.0\t2\t0 762 27\n",
      "26\t39077\t0.0\t2\t38244 805 28\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S288_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S288_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S288_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S288_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S288_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S288_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S288_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S288_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,062 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.82 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,062\n",
      "  Read 1 with adapter:                  51,991 (99.9%)\n",
      "  Read 2 with adapter:                  51,716 (99.3%)\n",
      "Pairs written (passing filters):        52,059 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,660,240 bp\n",
      "  Read 1:    15,357,717 bp\n",
      "  Read 2:    15,302,523 bp\n",
      "Total written (filtered):     27,964,355 bp (91.2%)\n",
      "  Read 1:    14,005,708 bp\n",
      "  Read 2:    13,958,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t702\t0.0\t2\t0 674 28\n",
      "26\t51067\t0.0\t2\t49238 1779 50\n",
      "27\t154\t0.0\t2\t0 150 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t859\t0.0\t2\t0 839 20\n",
      "26\t50545\t0.0\t2\t49262 1253 30\n",
      "27\t201\t0.0\t2\t0 196 5\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S289_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S289_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S289_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S289_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S289_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S289_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S289_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S289_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        57,389 reads  @     91.5 µs/read;   0.66 M reads/minute\n",
      "Finished in 5.26 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,389\n",
      "  Read 1 with adapter:                  57,293 (99.8%)\n",
      "  Read 2 with adapter:                  57,287 (99.8%)\n",
      "Pairs written (passing filters):        57,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,853,108 bp\n",
      "  Read 1:    16,926,673 bp\n",
      "  Read 2:    16,926,435 bp\n",
      "Total written (filtered):     30,875,625 bp (91.2%)\n",
      "  Read 1:    15,437,617 bp\n",
      "  Read 2:    15,438,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t716\t0.0\t2\t0 693 23\n",
      "26\t56369\t0.0\t2\t54501 1830 38\n",
      "27\t142\t0.0\t2\t0 135 7\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t1076\t0.0\t2\t0 1041 35\n",
      "26\t55919\t0.0\t2\t54640 1244 35\n",
      "27\t191\t0.0\t2\t0 184 7\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S290_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S290_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S290_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S290_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S290_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S290_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S290_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S290_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            71 reads  @    567.7 µs/read;   0.11 M reads/minute\n",
      "Finished in 0.04 s (607 us/read; 0.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 71\n",
      "  Read 1 with adapter:                      60 (84.5%)\n",
      "  Read 2 with adapter:                      60 (84.5%)\n",
      "Pairs written (passing filters):            71 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,890 bp\n",
      "  Read 1:        20,945 bp\n",
      "  Read 2:        20,945 bp\n",
      "Total written (filtered):         38,776 bp (92.6%)\n",
      "  Read 1:        19,388 bp\n",
      "  Read 2:        19,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t56\t0.0\t2\t53 2 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t56\t0.0\t2\t54 2\n",
      "27\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S291_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S291_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S291_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S291_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S291_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S291_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S291_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S291_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,972 reads  @     91.0 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.46 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,972\n",
      "  Read 1 with adapter:                  37,944 (99.9%)\n",
      "  Read 2 with adapter:                  37,899 (99.8%)\n",
      "Pairs written (passing filters):        37,972 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,402,398 bp\n",
      "  Read 1:    11,201,690 bp\n",
      "  Read 2:    11,200,708 bp\n",
      "Total written (filtered):     20,430,964 bp (91.2%)\n",
      "  Read 1:    10,215,143 bp\n",
      "  Read 2:    10,215,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37944 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t229\t0.0\t2\t0 223 6\n",
      "26\t37485\t0.0\t2\t36255 1192 38\n",
      "27\t156\t0.0\t2\t0 151 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t571\t0.0\t2\t0 559 12\n",
      "26\t37129\t0.0\t2\t36385 710 34\n",
      "27\t138\t0.0\t2\t0 132 6\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S292_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S292_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S292_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S292_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S292_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S292_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S292_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S292_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,460 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.34 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,460\n",
      "  Read 1 with adapter:                  52,397 (99.9%)\n",
      "  Read 2 with adapter:                  52,355 (99.8%)\n",
      "Pairs written (passing filters):        52,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,949,326 bp\n",
      "  Read 1:    15,475,695 bp\n",
      "  Read 2:    15,473,631 bp\n",
      "Total written (filtered):     28,226,577 bp (91.2%)\n",
      "  Read 1:    14,113,399 bp\n",
      "  Read 2:    14,113,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t343\t0.0\t2\t0 331 12\n",
      "26\t51743\t0.0\t2\t50190 1513 40\n",
      "27\t197\t0.0\t2\t0 191 6\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t843\t0.0\t2\t0 813 30\n",
      "26\t51235\t0.0\t2\t50134 1051 50\n",
      "27\t192\t0.0\t2\t0 187 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S293_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S293_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S293_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S293_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S293_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S293_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S293_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S293_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,289 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.61 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,289\n",
      "  Read 1 with adapter:                  45,227 (99.9%)\n",
      "  Read 2 with adapter:                  45,227 (99.9%)\n",
      "Pairs written (passing filters):        45,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,370 bp\n",
      "  Read 1:    13,360,082 bp\n",
      "  Read 2:    13,357,288 bp\n",
      "Total written (filtered):     24,366,270 bp (91.2%)\n",
      "  Read 1:    12,184,239 bp\n",
      "  Read 2:    12,182,031 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t336\t0.0\t2\t0 326 10\n",
      "26\t44633\t0.0\t2\t43279 1317 37\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t676\t0.0\t2\t0 659 17\n",
      "26\t44330\t0.0\t2\t43446 861 23\n",
      "27\t151\t0.0\t2\t0 147 4\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S294_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S294_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S294_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S294_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S294_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S294_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S294_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S294_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,661 reads  @     93.6 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.62 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,661\n",
      "  Read 1 with adapter:                  38,625 (99.9%)\n",
      "  Read 2 with adapter:                  38,459 (99.5%)\n",
      "Pairs written (passing filters):        38,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,790,159 bp\n",
      "  Read 1:    11,404,084 bp\n",
      "  Read 2:    11,386,075 bp\n",
      "Total written (filtered):     20,786,536 bp (91.2%)\n",
      "  Read 1:    10,399,908 bp\n",
      "  Read 2:    10,386,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38625 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t265\t0.0\t2\t0 257 8\n",
      "26\t38162\t0.0\t2\t36979 1149 34\n",
      "27\t133\t0.0\t2\t0 130 3\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t568\t0.0\t2\t0 557 11\n",
      "26\t37687\t0.0\t2\t36843 811 33\n",
      "27\t155\t0.0\t2\t0 152 3\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S295_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S295_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S295_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S295_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S295_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S295_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S295_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S295_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        41,494 reads  @    110.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 4.58 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,494\n",
      "  Read 1 with adapter:                  41,439 (99.9%)\n",
      "  Read 2 with adapter:                  39,034 (94.1%)\n",
      "Pairs written (passing filters):        41,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,960,452 bp\n",
      "  Read 1:    12,240,389 bp\n",
      "  Read 2:    11,720,063 bp\n",
      "Total written (filtered):     21,868,502 bp (91.3%)\n",
      "  Read 1:    11,162,582 bp\n",
      "  Read 2:    10,705,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41439 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t331\t0.0\t2\t0 317 14\n",
      "26\t40905\t0.0\t2\t39614 1263 28\n",
      "27\t132\t0.0\t2\t0 128 4\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t762\t0.0\t2\t0 739 23\n",
      "26\t38076\t0.0\t2\t37246 800 30\n",
      "27\t131\t0.0\t2\t0 127 4\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S296_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S296_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S296_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S296_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S296_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S296_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S296_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S296_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,814 reads  @    111.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 4.88 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,814\n",
      "  Read 1 with adapter:                  43,753 (99.9%)\n",
      "  Read 2 with adapter:                  43,650 (99.6%)\n",
      "Pairs written (passing filters):        43,812 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,834,630 bp\n",
      "  Read 1:    12,924,656 bp\n",
      "  Read 2:    12,909,974 bp\n",
      "Total written (filtered):     23,562,480 bp (91.2%)\n",
      "  Read 1:    11,786,747 bp\n",
      "  Read 2:    11,775,733 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t395\t0.0\t2\t0 386 9\n",
      "26\t43144\t0.0\t2\t41724 1371 49\n",
      "27\t140\t0.0\t2\t0 136 4\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t724\t0.0\t2\t0 706 18\n",
      "26\t42655\t0.0\t2\t41679 941 35\n",
      "27\t189\t0.0\t2\t0 183 6\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S297_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S297_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S297_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S297_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S297_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S297_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S297_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S297_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        49,274 reads  @    100.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.95 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,274\n",
      "  Read 1 with adapter:                  49,204 (99.9%)\n",
      "  Read 2 with adapter:                  49,181 (99.8%)\n",
      "Pairs written (passing filters):        49,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,068,066 bp\n",
      "  Read 1:    14,535,679 bp\n",
      "  Read 2:    14,532,387 bp\n",
      "Total written (filtered):     26,511,187 bp (91.2%)\n",
      "  Read 1:    13,256,558 bp\n",
      "  Read 2:    13,254,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t377\t0.0\t2\t0 362 15\n",
      "26\t48602\t0.0\t2\t47042 1527 33\n",
      "27\t144\t0.0\t2\t0 136 8\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t1026\t0.0\t2\t0 994 32\n",
      "26\t47871\t0.0\t2\t46739 1104 28\n",
      "27\t194\t0.0\t2\t0 186 8\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S298_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S298_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S298_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S298_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S298_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S298_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S298_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S298_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,281 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.38 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,281\n",
      "  Read 1 with adapter:                  48,223 (99.9%)\n",
      "  Read 2 with adapter:                  48,198 (99.8%)\n",
      "Pairs written (passing filters):        48,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,484,038 bp\n",
      "  Read 1:    14,242,599 bp\n",
      "  Read 2:    14,241,439 bp\n",
      "Total written (filtered):     25,977,894 bp (91.2%)\n",
      "  Read 1:    12,988,935 bp\n",
      "  Read 2:    12,988,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t354\t0.0\t2\t0 344 10\n",
      "26\t47643\t0.0\t2\t46202 1398 43\n",
      "27\t148\t0.0\t2\t0 143 5\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t752\t0.0\t2\t0 734 18\n",
      "26\t47227\t0.0\t2\t46268 934 25\n",
      "27\t146\t0.0\t2\t0 141 5\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S299_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S299_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S299_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S299_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S299_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S299_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S299_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S299_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,914 reads  @     93.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.92 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,914\n",
      "  Read 1 with adapter:                  41,841 (99.8%)\n",
      "  Read 2 with adapter:                  41,831 (99.8%)\n",
      "Pairs written (passing filters):        41,914 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,724,849 bp\n",
      "  Read 1:    12,361,234 bp\n",
      "  Read 2:    12,363,615 bp\n",
      "Total written (filtered):     22,549,939 bp (91.2%)\n",
      "  Read 1:    11,273,397 bp\n",
      "  Read 2:    11,276,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t321\t0.0\t2\t0 310 11\n",
      "26\t41220\t0.0\t2\t39816 1372 32\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t635\t0.0\t2\t0 623 12\n",
      "26\t40958\t0.0\t2\t40130 795 33\n",
      "27\t162\t0.0\t2\t0 154 8\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S300_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S300_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S300_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S300_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S300_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S300_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S300_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S300_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        62,071 reads  @     86.9 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.40 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,071\n",
      "  Read 1 with adapter:                  61,940 (99.8%)\n",
      "  Read 2 with adapter:                  61,885 (99.7%)\n",
      "Pairs written (passing filters):        62,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,608,322 bp\n",
      "  Read 1:    18,300,356 bp\n",
      "  Read 2:    18,307,966 bp\n",
      "Total written (filtered):     33,389,879 bp (91.2%)\n",
      "  Read 1:    16,690,016 bp\n",
      "  Read 2:    16,699,863 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t435\t0.0\t2\t0 423 12\n",
      "26\t61132\t0.0\t2\t59055 2005 72\n",
      "27\t263\t0.0\t2\t0 248 15\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61885 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t90\t0.0\t2\t0 0 90\n",
      "25\t1032\t0.0\t2\t0 1008 24\n",
      "26\t60472\t0.0\t2\t58973 1434 65\n",
      "27\t277\t0.0\t2\t0 268 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S301_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S301_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S301_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S301_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S301_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S301_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S301_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S301_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,457 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.16 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,457\n",
      "  Read 1 with adapter:                  52,355 (99.8%)\n",
      "  Read 2 with adapter:                  52,336 (99.8%)\n",
      "Pairs written (passing filters):        52,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,942,322 bp\n",
      "  Read 1:    15,469,464 bp\n",
      "  Read 2:    15,472,858 bp\n",
      "Total written (filtered):     28,221,075 bp (91.2%)\n",
      "  Read 1:    14,108,189 bp\n",
      "  Read 2:    14,112,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t358\t0.0\t2\t0 348 10\n",
      "26\t51601\t0.0\t2\t49802 1755 44\n",
      "27\t257\t0.0\t2\t0 248 9\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t858\t0.0\t2\t0 836 22\n",
      "26\t51185\t0.0\t2\t50101 1053 31\n",
      "27\t220\t0.0\t2\t0 213 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S302_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S302_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S302_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S302_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S302_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S302_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S302_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S302_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,891 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.44 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,891\n",
      "  Read 1 with adapter:                  45,825 (99.9%)\n",
      "  Read 2 with adapter:                  45,641 (99.5%)\n",
      "Pairs written (passing filters):        45,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,052,303 bp\n",
      "  Read 1:    13,535,309 bp\n",
      "  Read 2:    13,516,994 bp\n",
      "Total written (filtered):     24,674,515 bp (91.2%)\n",
      "  Read 1:    12,343,524 bp\n",
      "  Read 2:    12,330,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t261\t0.0\t2\t0 248 13\n",
      "26\t45247\t0.0\t2\t43850 1368 29\n",
      "27\t211\t0.0\t2\t0 205 6\n",
      "28\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t716\t0.0\t2\t0 702 14\n",
      "26\t44702\t0.0\t2\t43736 924 42\n",
      "27\t157\t0.0\t2\t0 150 7\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S303_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S303_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S303_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S303_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S303_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S303_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S303_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S303_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,902 reads  @     81.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.83 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,902\n",
      "  Read 1 with adapter:                  46,785 (99.8%)\n",
      "  Read 2 with adapter:                  46,442 (99.0%)\n",
      "Pairs written (passing filters):        46,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,589,426 bp\n",
      "  Read 1:    13,825,372 bp\n",
      "  Read 2:    13,764,054 bp\n",
      "Total written (filtered):     25,166,435 bp (91.2%)\n",
      "  Read 1:    12,608,986 bp\n",
      "  Read 2:    12,557,449 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t323\t0.0\t2\t0 309 14\n",
      "26\t46165\t0.0\t2\t44675 1446 44\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t919\t0.0\t2\t0 896 23\n",
      "26\t45292\t0.0\t2\t44331 935 26\n",
      "27\t154\t0.0\t2\t0 146 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S304_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S304_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S304_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S304_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S304_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S304_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S304_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S304_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,124 reads  @     83.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.62 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,124\n",
      "  Read 1 with adapter:                  42,967 (99.6%)\n",
      "  Read 2 with adapter:                  42,980 (99.7%)\n",
      "Pairs written (passing filters):        43,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,406,526 bp\n",
      "  Read 1:    12,698,317 bp\n",
      "  Read 2:    12,708,209 bp\n",
      "Total written (filtered):     23,172,647 bp (91.2%)\n",
      "  Read 1:    11,581,191 bp\n",
      "  Read 2:    11,591,456 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t316\t0.0\t2\t0 304 12\n",
      "26\t42352\t0.0\t2\t41031 1288 33\n",
      "27\t206\t0.0\t2\t0 197 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t770\t0.0\t2\t0 748 22\n",
      "26\t41961\t0.0\t2\t40965 960 36\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S305_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S305_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S305_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S305_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S305_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S305_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S305_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S305_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,232 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.15 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,232\n",
      "  Read 1 with adapter:                  53,085 (99.7%)\n",
      "  Read 2 with adapter:                  53,092 (99.7%)\n",
      "Pairs written (passing filters):        53,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,386,302 bp\n",
      "  Read 1:    15,686,331 bp\n",
      "  Read 2:    15,699,971 bp\n",
      "Total written (filtered):     28,627,195 bp (91.2%)\n",
      "  Read 1:    14,306,442 bp\n",
      "  Read 2:    14,320,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t476\t0.0\t2\t0 459 17\n",
      "26\t52399\t0.0\t2\t50501 1832 66\n",
      "27\t153\t0.0\t2\t0 149 4\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53092 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t123\t0.0\t2\t0 0 123\n",
      "25\t1158\t0.0\t2\t0 1127 31\n",
      "26\t51589\t0.0\t2\t50301 1248 40\n",
      "27\t214\t0.0\t2\t0 203 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S306_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S306_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S306_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S306_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S306_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S306_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S306_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S306_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,786 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.31 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,786\n",
      "  Read 1 with adapter:                  55,518 (99.5%)\n",
      "  Read 2 with adapter:                  55,694 (99.8%)\n",
      "Pairs written (passing filters):        55,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,862,309 bp\n",
      "  Read 1:    16,405,724 bp\n",
      "  Read 2:    16,456,585 bp\n",
      "Total written (filtered):     29,971,629 bp (91.2%)\n",
      "  Read 1:    14,962,359 bp\n",
      "  Read 2:    15,009,270 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t373\t0.0\t2\t0 361 12\n",
      "26\t54821\t0.0\t2\t53303 1474 44\n",
      "27\t246\t0.0\t2\t0 242 4\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t811\t0.0\t2\t0 797 14\n",
      "26\t54618\t0.0\t2\t53535 1027 56\n",
      "27\t168\t0.0\t2\t0 161 7\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S307_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S307_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S307_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S307_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S307_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S307_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S307_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S307_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,513 reads  @     88.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.58 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,513\n",
      "  Read 1 with adapter:                  51,436 (99.9%)\n",
      "  Read 2 with adapter:                  51,401 (99.8%)\n",
      "Pairs written (passing filters):        51,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,391,135 bp\n",
      "  Read 1:    15,195,766 bp\n",
      "  Read 2:    15,195,369 bp\n",
      "Total written (filtered):     27,718,112 bp (91.2%)\n",
      "  Read 1:    13,858,444 bp\n",
      "  Read 2:    13,859,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t344\t0.0\t2\t0 337 7\n",
      "26\t50802\t0.0\t2\t49154 1606 42\n",
      "27\t182\t0.0\t2\t0 178 4\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t795\t0.0\t2\t0 767 28\n",
      "26\t50361\t0.0\t2\t49310 1008 43\n",
      "27\t172\t0.0\t2\t0 167 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S308_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S308_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S308_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S308_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S308_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S308_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S308_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S308_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,438 reads  @     87.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.86 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,438\n",
      "  Read 1 with adapter:                  55,360 (99.9%)\n",
      "  Read 2 with adapter:                  55,328 (99.8%)\n",
      "Pairs written (passing filters):        55,438 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,706,228 bp\n",
      "  Read 1:    16,353,867 bp\n",
      "  Read 2:    16,352,361 bp\n",
      "Total written (filtered):     29,829,189 bp (91.2%)\n",
      "  Read 1:    14,914,533 bp\n",
      "  Read 2:    14,914,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55360 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t376\t0.0\t2\t0 365 11\n",
      "26\t54643\t0.0\t2\t52930 1665 48\n",
      "27\t196\t0.0\t2\t0 189 7\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t77\t0.0\t2\t0 0 77\n",
      "25\t895\t0.0\t2\t0 862 33\n",
      "26\t54132\t0.0\t2\t52806 1284 42\n",
      "27\t222\t0.0\t2\t0 215 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S309_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S309_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S309_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S309_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S309_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S309_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S309_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S309_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,179 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.65 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,179\n",
      "  Read 1 with adapter:                  43,099 (99.8%)\n",
      "  Read 2 with adapter:                  43,081 (99.8%)\n",
      "Pairs written (passing filters):        43,179 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,469,344 bp\n",
      "  Read 1:    12,737,186 bp\n",
      "  Read 2:    12,732,158 bp\n",
      "Total written (filtered):     23,229,208 bp (91.2%)\n",
      "  Read 1:    11,616,604 bp\n",
      "  Read 2:    11,612,604 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43099 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t262\t0.0\t2\t0 255 7\n",
      "26\t42573\t0.0\t2\t41217 1325 31\n",
      "27\t170\t0.0\t2\t0 166 4\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43081 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t618\t0.0\t2\t0 606 12\n",
      "26\t42232\t0.0\t2\t41372 835 25\n",
      "27\t172\t0.0\t2\t0 163 9\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S310_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S310_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S310_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S310_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S310_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S310_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S310_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S310_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,392 reads  @     83.3 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.45 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,392\n",
      "  Read 1 with adapter:                  41,322 (99.8%)\n",
      "  Read 2 with adapter:                  41,253 (99.7%)\n",
      "Pairs written (passing filters):        41,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,409,584 bp\n",
      "  Read 1:    12,208,505 bp\n",
      "  Read 2:    12,201,079 bp\n",
      "Total written (filtered):     22,263,188 bp (91.2%)\n",
      "  Read 1:    11,134,098 bp\n",
      "  Read 2:    11,129,090 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t283\t0.0\t2\t0 275 8\n",
      "26\t40757\t0.0\t2\t39467 1258 32\n",
      "27\t174\t0.0\t2\t0 169 5\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41253 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t623\t0.0\t2\t0 606 17\n",
      "26\t40418\t0.0\t2\t39486 890 42\n",
      "27\t150\t0.0\t2\t0 141 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S311_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S311_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S311_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S311_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S311_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S311_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S311_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S311_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,850 reads  @     85.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.17 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,850\n",
      "  Read 1 with adapter:                  48,780 (99.9%)\n",
      "  Read 2 with adapter:                  48,201 (98.7%)\n",
      "Pairs written (passing filters):        48,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,705,818 bp\n",
      "  Read 1:    14,410,215 bp\n",
      "  Read 2:    14,295,603 bp\n",
      "Total written (filtered):     26,185,161 bp (91.2%)\n",
      "  Read 1:    13,141,879 bp\n",
      "  Read 2:    13,043,282 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t300\t0.0\t2\t0 285 15\n",
      "26\t48153\t0.0\t2\t46609 1507 37\n",
      "27\t166\t0.0\t2\t0 159 7\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t977\t0.0\t2\t0 947 30\n",
      "26\t46974\t0.0\t2\t45916 1018 40\n",
      "27\t188\t0.0\t2\t0 175 13\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S312_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S312_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S312_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S312_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S312_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S312_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S312_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S312_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,837 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.21 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,837\n",
      "  Read 1 with adapter:                  47,762 (99.8%)\n",
      "  Read 2 with adapter:                  47,718 (99.8%)\n",
      "Pairs written (passing filters):        47,835 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,215,558 bp\n",
      "  Read 1:    14,110,294 bp\n",
      "  Read 2:    14,105,264 bp\n",
      "Total written (filtered):     25,733,228 bp (91.2%)\n",
      "  Read 1:    12,867,921 bp\n",
      "  Read 2:    12,865,307 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t327\t0.0\t2\t0 314 13\n",
      "26\t47119\t0.0\t2\t45614 1465 40\n",
      "27\t178\t0.0\t2\t0 171 7\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t770\t0.0\t2\t0 747 23\n",
      "26\t46710\t0.0\t2\t45644 1022 44\n",
      "27\t173\t0.0\t2\t0 166 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S313_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S313_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S313_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S313_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S313_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S313_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S313_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S313_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,060 reads  @     84.3 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.48 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,060\n",
      "  Read 1 with adapter:                  52,959 (99.8%)\n",
      "  Read 2 with adapter:                  52,947 (99.8%)\n",
      "Pairs written (passing filters):        53,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,299,771 bp\n",
      "  Read 1:    15,650,829 bp\n",
      "  Read 2:    15,648,942 bp\n",
      "Total written (filtered):     28,547,305 bp (91.2%)\n",
      "  Read 1:    14,273,971 bp\n",
      "  Read 2:    14,273,334 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52959 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t393\t0.0\t2\t0 378 15\n",
      "26\t52256\t0.0\t2\t50637 1578 41\n",
      "27\t191\t0.0\t2\t0 187 4\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52947 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t1078\t0.0\t2\t0 1054 24\n",
      "26\t51559\t0.0\t2\t50398 1128 33\n",
      "27\t212\t0.0\t2\t0 207 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S314_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S314_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S314_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S314_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S314_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S314_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S314_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S314_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,187 reads  @     71.9 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.12 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,187\n",
      "  Read 1 with adapter:                  57,100 (99.8%)\n",
      "  Read 2 with adapter:                  57,083 (99.8%)\n",
      "Pairs written (passing filters):        57,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,739,169 bp\n",
      "  Read 1:    16,870,143 bp\n",
      "  Read 2:    16,869,026 bp\n",
      "Total written (filtered):     30,771,537 bp (91.2%)\n",
      "  Read 1:    15,385,820 bp\n",
      "  Read 2:    15,385,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t479\t0.0\t2\t0 461 18\n",
      "26\t56364\t0.0\t2\t54581 1741 42\n",
      "27\t180\t0.0\t2\t0 177 3\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57083 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t79\t0.0\t2\t0 0 79\n",
      "25\t891\t0.0\t2\t0 870 21\n",
      "26\t55925\t0.0\t2\t54679 1205 41\n",
      "27\t176\t0.0\t2\t0 171 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S315_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S315_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S315_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S315_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S315_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S315_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S315_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S315_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,259 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.10 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,259\n",
      "  Read 1 with adapter:                  45,854 (99.1%)\n",
      "  Read 2 with adapter:                  46,189 (99.8%)\n",
      "Pairs written (passing filters):        46,259 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,265,604 bp\n",
      "  Read 1:    13,619,261 bp\n",
      "  Read 2:    13,646,343 bp\n",
      "Total written (filtered):     24,873,155 bp (91.2%)\n",
      "  Read 1:    12,427,093 bp\n",
      "  Read 2:    12,446,062 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t326\t0.0\t2\t0 311 15\n",
      "26\t45247\t0.0\t2\t43883 1321 43\n",
      "27\t164\t0.0\t2\t0 156 8\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t720\t0.0\t2\t0 706 14\n",
      "26\t45207\t0.0\t2\t44320 861 26\n",
      "27\t201\t0.0\t2\t0 194 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S316_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S316_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S316_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S316_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S316_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S316_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S316_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S316_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,078 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.31 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,078\n",
      "  Read 1 with adapter:                  45,006 (99.8%)\n",
      "  Read 2 with adapter:                  44,978 (99.8%)\n",
      "Pairs written (passing filters):        45,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,591,866 bp\n",
      "  Read 1:    13,295,524 bp\n",
      "  Read 2:    13,296,342 bp\n",
      "Total written (filtered):     24,253,040 bp (91.2%)\n",
      "  Read 1:    12,125,494 bp\n",
      "  Read 2:    12,127,546 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t357\t0.0\t2\t0 347 10\n",
      "26\t44414\t0.0\t2\t43087 1295 32\n",
      "27\t131\t0.0\t2\t0 123 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44978 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t749\t0.0\t2\t0 731 18\n",
      "26\t43983\t0.0\t2\t43027 922 34\n",
      "27\t179\t0.0\t2\t0 170 9\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S317_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S317_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S317_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S317_133_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S317_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S317_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S317_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S317_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,274 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.37 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,274\n",
      "  Read 1 with adapter:                  48,180 (99.8%)\n",
      "  Read 2 with adapter:                  48,182 (99.8%)\n",
      "Pairs written (passing filters):        48,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,474,831 bp\n",
      "  Read 1:    14,236,197 bp\n",
      "  Read 2:    14,238,634 bp\n",
      "Total written (filtered):     25,970,188 bp (91.2%)\n",
      "  Read 1:    12,983,625 bp\n",
      "  Read 2:    12,986,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t321\t0.0\t2\t0 308 13\n",
      "26\t47635\t0.0\t2\t46128 1468 39\n",
      "27\t151\t0.0\t2\t0 146 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t786\t0.0\t2\t0 772 14\n",
      "26\t47128\t0.0\t2\t46135 970 23\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S318_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S318_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S318_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S318_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S318_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S318_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S318_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S318_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,681 reads  @     74.1 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.42 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,681\n",
      "  Read 1 with adapter:                  32,592 (99.7%)\n",
      "  Read 2 with adapter:                  32,594 (99.7%)\n",
      "Pairs written (passing filters):        32,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,271,096 bp\n",
      "  Read 1:     9,634,378 bp\n",
      "  Read 2:     9,636,718 bp\n",
      "Total written (filtered):     17,576,780 bp (91.2%)\n",
      "  Read 1:     8,787,035 bp\n",
      "  Read 2:     8,789,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32592 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t269\t0.0\t2\t0 260 9\n",
      "26\t32110\t0.0\t2\t31058 1023 29\n",
      "27\t142\t0.0\t2\t0 136 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32594 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t518\t0.0\t2\t0 502 16\n",
      "26\t31882\t0.0\t2\t31124 728 30\n",
      "27\t117\t0.0\t2\t0 113 4\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S319_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S319_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S319_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S319_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S319_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S319_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S319_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S319_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,331 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.63 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,331\n",
      "  Read 1 with adapter:                  39,261 (99.8%)\n",
      "  Read 2 with adapter:                  38,893 (98.9%)\n",
      "Pairs written (passing filters):        39,331 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,131,739 bp\n",
      "  Read 1:    11,598,798 bp\n",
      "  Read 2:    11,532,941 bp\n",
      "Total written (filtered):     21,100,728 bp (91.2%)\n",
      "  Read 1:    10,578,203 bp\n",
      "  Read 2:    10,522,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t296\t0.0\t2\t0 293 3\n",
      "26\t38834\t0.0\t2\t37730 1070 34\n",
      "27\t89\t0.0\t2\t0 87 2\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38893 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t836\t0.0\t2\t0 816 20\n",
      "26\t37868\t0.0\t2\t37095 752 21\n",
      "27\t92\t0.0\t2\t0 86 6\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S320_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S320_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S320_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S320_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S320_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S320_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S320_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S320_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,311 reads  @     65.1 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.50 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,311\n",
      "  Read 1 with adapter:                  38,246 (99.8%)\n",
      "  Read 2 with adapter:                  38,169 (99.6%)\n",
      "Pairs written (passing filters):        38,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,595,996 bp\n",
      "  Read 1:    11,301,261 bp\n",
      "  Read 2:    11,294,735 bp\n",
      "Total written (filtered):     20,609,833 bp (91.2%)\n",
      "  Read 1:    10,306,891 bp\n",
      "  Read 2:    10,302,942 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38246 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t266\t0.0\t2\t0 256 10\n",
      "26\t37751\t0.0\t2\t36641 1076 34\n",
      "27\t142\t0.0\t2\t0 133 9\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t660\t0.0\t2\t0 646 14\n",
      "26\t37309\t0.0\t2\t36552 717 40\n",
      "27\t145\t0.0\t2\t0 140 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S321_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S321_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S321_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S321_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S321_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S321_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S321_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S321_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,827 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.93 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,827\n",
      "  Read 1 with adapter:                  56,707 (99.8%)\n",
      "  Read 2 with adapter:                  56,695 (99.8%)\n",
      "Pairs written (passing filters):        56,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,514,863 bp\n",
      "  Read 1:    16,757,621 bp\n",
      "  Read 2:    16,757,242 bp\n",
      "Total written (filtered):     30,567,643 bp (91.2%)\n",
      "  Read 1:    15,283,278 bp\n",
      "  Read 2:    15,284,365 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t423\t0.0\t2\t0 411 12\n",
      "26\t55914\t0.0\t2\t54085 1794 35\n",
      "27\t212\t0.0\t2\t0 203 9\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56695 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t102\t0.0\t2\t0 0 102\n",
      "25\t1201\t0.0\t2\t0 1168 33\n",
      "26\t55190\t0.0\t2\t53933 1215 42\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S322_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S322_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S322_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S322_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S322_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S322_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S322_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S322_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,287 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.31 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,287\n",
      "  Read 1 with adapter:                  62,201 (99.9%)\n",
      "  Read 2 with adapter:                  62,139 (99.8%)\n",
      "Pairs written (passing filters):        62,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,747,239 bp\n",
      "  Read 1:    18,374,143 bp\n",
      "  Read 2:    18,373,096 bp\n",
      "Total written (filtered):     33,515,556 bp (91.2%)\n",
      "  Read 1:    16,757,252 bp\n",
      "  Read 2:    16,758,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t534\t0.0\t2\t0 517 17\n",
      "26\t61402\t0.0\t2\t59404 1935 63\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t924\t0.0\t2\t0 897 27\n",
      "26\t60867\t0.0\t2\t59399 1439 29\n",
      "27\t262\t0.0\t2\t0 257 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S323_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S323_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S323_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S323_145_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S323_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S323_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S323_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S323_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,039 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.24 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,039\n",
      "  Read 1 with adapter:                  44,982 (99.9%)\n",
      "  Read 2 with adapter:                  44,933 (99.8%)\n",
      "Pairs written (passing filters):        45,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,571,297 bp\n",
      "  Read 1:    13,285,885 bp\n",
      "  Read 2:    13,285,412 bp\n",
      "Total written (filtered):     24,234,730 bp (91.2%)\n",
      "  Read 1:    12,116,903 bp\n",
      "  Read 2:    12,117,827 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t638\t0.0\t2\t0 621 17\n",
      "26\t44169\t0.0\t2\t42637 1490 42\n",
      "27\t114\t0.0\t2\t0 109 5\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t714\t0.0\t2\t0 695 19\n",
      "26\t43989\t0.0\t2\t43113 847 29\n",
      "27\t163\t0.0\t2\t0 158 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S324_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S324_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S324_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S324_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S324_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S324_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S324_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S324_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,353 reads  @     69.8 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.17 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,353\n",
      "  Read 1 with adapter:                  45,290 (99.9%)\n",
      "  Read 2 with adapter:                  45,236 (99.7%)\n",
      "Pairs written (passing filters):        45,353 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,757,024 bp\n",
      "  Read 1:    13,378,788 bp\n",
      "  Read 2:    13,378,236 bp\n",
      "Total written (filtered):     24,404,642 bp (91.2%)\n",
      "  Read 1:    12,201,817 bp\n",
      "  Read 2:    12,202,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t700\t0.0\t2\t0 681 19\n",
      "26\t44379\t0.0\t2\t42845 1496 38\n",
      "27\t139\t0.0\t2\t0 132 7\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t814\t0.0\t2\t0 791 23\n",
      "26\t44155\t0.0\t2\t43090 1028 37\n",
      "27\t197\t0.0\t2\t0 193 4\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S325_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S325_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S325_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S325_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S325_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S325_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S325_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S325_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,901 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.60 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,901\n",
      "  Read 1 with adapter:                  51,845 (99.9%)\n",
      "  Read 2 with adapter:                  51,787 (99.8%)\n",
      "Pairs written (passing filters):        51,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,614,417 bp\n",
      "  Read 1:    15,309,074 bp\n",
      "  Read 2:    15,305,343 bp\n",
      "Total written (filtered):     27,921,062 bp (91.2%)\n",
      "  Read 1:    13,961,437 bp\n",
      "  Read 2:    13,959,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t504\t0.0\t2\t0 489 15\n",
      "26\t51140\t0.0\t2\t49495 1598 47\n",
      "27\t147\t0.0\t2\t0 142 5\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51787 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t805\t0.0\t2\t0 778 27\n",
      "26\t50688\t0.0\t2\t49573 1085 30\n",
      "27\t203\t0.0\t2\t0 202 1\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S326_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S326_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S326_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S326_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S326_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S326_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S326_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S326_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,933 reads  @    111.7 µs/read;   0.54 M reads/minute\n",
      "Finished in 0.44 s (112 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,933\n",
      "  Read 1 with adapter:                   3,904 (99.3%)\n",
      "  Read 2 with adapter:                   3,740 (95.1%)\n",
      "Pairs written (passing filters):         3,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,310,635 bp\n",
      "  Read 1:     1,160,048 bp\n",
      "  Read 2:     1,150,587 bp\n",
      "Total written (filtered):      2,111,947 bp (91.4%)\n",
      "  Read 1:     1,058,553 bp\n",
      "  Read 2:     1,053,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 3904 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t44\t0.0\t2\t0 43 1\n",
      "26\t3807\t0.0\t2\t3629 176 2\n",
      "27\t47\t0.0\t2\t0 45 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 3740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t60\t0.0\t2\t0 56 4\n",
      "26\t3655\t0.0\t2\t3519 127 9\n",
      "27\t9\t0.0\t2\t0 8 1\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S327_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S327_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S327_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S327_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S327_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S327_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S327_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S327_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,007 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.36 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,007\n",
      "  Read 1 with adapter:                  45,914 (99.8%)\n",
      "  Read 2 with adapter:                  45,732 (99.4%)\n",
      "Pairs written (passing filters):        46,005 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,096,983 bp\n",
      "  Read 1:    13,568,196 bp\n",
      "  Read 2:    13,528,787 bp\n",
      "Total written (filtered):     24,714,793 bp (91.2%)\n",
      "  Read 1:    12,374,191 bp\n",
      "  Read 2:    12,340,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t531\t0.0\t2\t0 511 20\n",
      "26\t45129\t0.0\t2\t43708 1385 36\n",
      "27\t158\t0.0\t2\t0 148 10\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t936\t0.0\t2\t0 912 24\n",
      "26\t44550\t0.0\t2\t43618 899 33\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S328_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S328_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S328_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S328_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S328_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S328_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S328_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S328_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,317 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 1.92 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,317\n",
      "  Read 1 with adapter:                  27,278 (99.9%)\n",
      "  Read 2 with adapter:                  27,097 (99.2%)\n",
      "Pairs written (passing filters):        27,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,084,197 bp\n",
      "  Read 1:     8,058,035 bp\n",
      "  Read 2:     8,026,162 bp\n",
      "Total written (filtered):     14,671,016 bp (91.2%)\n",
      "  Read 1:     7,348,899 bp\n",
      "  Read 2:     7,322,117 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t446\t0.0\t2\t0 435 11\n",
      "26\t26711\t0.0\t2\t25692 987 32\n",
      "27\t85\t0.0\t2\t0 84 1\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27097 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t534\t0.0\t2\t0 519 15\n",
      "26\t26376\t0.0\t2\t25711 642 23\n",
      "27\t113\t0.0\t2\t0 110 3\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S329_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S329_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S329_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S329_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S329_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S329_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S329_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S329_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,768 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.10 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,768\n",
      "  Read 1 with adapter:                  47,698 (99.9%)\n",
      "  Read 2 with adapter:                  47,665 (99.8%)\n",
      "Pairs written (passing filters):        47,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,177,601 bp\n",
      "  Read 1:    14,089,015 bp\n",
      "  Read 2:    14,088,586 bp\n",
      "Total written (filtered):     25,699,519 bp (91.2%)\n",
      "  Read 1:    12,849,313 bp\n",
      "  Read 2:    12,850,206 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47698 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t583\t0.0\t2\t0 567 16\n",
      "26\t46953\t0.0\t2\t45481 1418 54\n",
      "27\t123\t0.0\t2\t0 119 4\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t950\t0.0\t2\t0 933 17\n",
      "26\t46464\t0.0\t2\t45487 946 31\n",
      "27\t166\t0.0\t2\t0 163 3\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S330_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S330_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S330_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S330_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S330_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S330_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S330_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S330_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            13 reads  @   2519.2 µs/read;   0.02 M reads/minute\n",
      "Finished in 0.04 s (2714 us/read; 0.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 13\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            13 (100.0%)\n",
      "\n",
      "Total basepairs processed:         7,670 bp\n",
      "  Read 1:         3,835 bp\n",
      "  Read 2:         3,835 bp\n",
      "Total written (filtered):          7,670 bp (100.0%)\n",
      "  Read 1:         3,835 bp\n",
      "  Read 2:         3,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S331_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S331_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S331_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S331_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S331_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S331_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S331_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S331_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        56,241 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.95 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,241\n",
      "  Read 1 with adapter:                  56,169 (99.9%)\n",
      "  Read 2 with adapter:                  56,185 (99.9%)\n",
      "Pairs written (passing filters):        56,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,180,291 bp\n",
      "  Read 1:    16,590,570 bp\n",
      "  Read 2:    16,589,721 bp\n",
      "Total written (filtered):     30,259,485 bp (91.2%)\n",
      "  Read 1:    15,129,992 bp\n",
      "  Read 2:    15,129,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t399\t0.0\t2\t0 388 11\n",
      "26\t55433\t0.0\t2\t53927 1469 37\n",
      "27\t232\t0.0\t2\t0 219 13\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t698\t0.0\t2\t0 681 17\n",
      "26\t55243\t0.0\t2\t54181 1030 32\n",
      "27\t168\t0.0\t2\t0 161 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S332_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S332_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S332_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S332_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S332_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S332_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S332_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S332_163_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,804 reads  @     70.9 µs/read;   0.85 M reads/minute\n",
      "Finished in 4.03 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,804\n",
      "  Read 1 with adapter:                  56,740 (99.9%)\n",
      "  Read 2 with adapter:                  56,703 (99.8%)\n",
      "Pairs written (passing filters):        56,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,509,015 bp\n",
      "  Read 1:    16,756,779 bp\n",
      "  Read 2:    16,752,236 bp\n",
      "Total written (filtered):     30,560,398 bp (91.2%)\n",
      "  Read 1:    15,281,649 bp\n",
      "  Read 2:    15,278,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t408\t0.0\t2\t0 397 11\n",
      "26\t55982\t0.0\t2\t54278 1660 44\n",
      "27\t206\t0.0\t2\t0 203 3\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t870\t0.0\t2\t0 848 22\n",
      "26\t55561\t0.0\t2\t54372 1142 47\n",
      "27\t197\t0.0\t2\t0 189 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S333_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S333_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S333_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S333_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S333_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S333_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S333_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S333_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,062 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.11 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,062\n",
      "  Read 1 with adapter:                  41,002 (99.9%)\n",
      "  Read 2 with adapter:                  40,988 (99.8%)\n",
      "Pairs written (passing filters):        41,062 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,221,789 bp\n",
      "  Read 1:    12,112,652 bp\n",
      "  Read 2:    12,109,137 bp\n",
      "Total written (filtered):     22,090,526 bp (91.2%)\n",
      "  Read 1:    11,046,661 bp\n",
      "  Read 2:    11,043,865 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41002 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t291\t0.0\t2\t0 282 9\n",
      "26\t40445\t0.0\t2\t39254 1156 35\n",
      "27\t158\t0.0\t2\t0 154 4\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t493\t0.0\t2\t0 479 14\n",
      "26\t40353\t0.0\t2\t39447 869 37\n",
      "27\t111\t0.0\t2\t0 104 7\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S334_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S334_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S334_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S334_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S334_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S334_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S334_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S334_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,550 reads  @     78.9 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.46 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,550\n",
      "  Read 1 with adapter:                  56,460 (99.8%)\n",
      "  Read 2 with adapter:                  56,453 (99.8%)\n",
      "Pairs written (passing filters):        56,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,362,219 bp\n",
      "  Read 1:    16,681,818 bp\n",
      "  Read 2:    16,680,401 bp\n",
      "Total written (filtered):     30,427,369 bp (91.2%)\n",
      "  Read 1:    15,214,031 bp\n",
      "  Read 2:    15,213,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56460 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t429\t0.0\t2\t0 420 9\n",
      "26\t55724\t0.0\t2\t53949 1736 39\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t868\t0.0\t2\t0 846 22\n",
      "26\t55323\t0.0\t2\t54207 1080 36\n",
      "27\t215\t0.0\t2\t0 204 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S335_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S335_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S335_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S335_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S335_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S335_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S335_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S335_169_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,654 reads  @     90.8 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.33 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,654\n",
      "  Read 1 with adapter:                  47,595 (99.9%)\n",
      "  Read 2 with adapter:                  47,510 (99.7%)\n",
      "Pairs written (passing filters):        47,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,107,506 bp\n",
      "  Read 1:    14,057,624 bp\n",
      "  Read 2:    14,049,882 bp\n",
      "Total written (filtered):     25,635,416 bp (91.2%)\n",
      "  Read 1:    12,820,412 bp\n",
      "  Read 2:    12,815,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47595 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t468\t0.0\t2\t0 457 11\n",
      "26\t46897\t0.0\t2\t45302 1546 49\n",
      "27\t154\t0.0\t2\t0 149 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t745\t0.0\t2\t0 732 13\n",
      "26\t46521\t0.0\t2\t45439 1054 28\n",
      "27\t178\t0.0\t2\t0 173 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S336_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S336_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S336_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S336_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S336_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S336_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S336_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S336_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        56,251 reads  @     92.5 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.21 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,251\n",
      "  Read 1 with adapter:                  56,164 (99.8%)\n",
      "  Read 2 with adapter:                  56,139 (99.8%)\n",
      "Pairs written (passing filters):        56,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,181,221 bp\n",
      "  Read 1:    16,592,992 bp\n",
      "  Read 2:    16,588,229 bp\n",
      "Total written (filtered):     30,262,506 bp (91.2%)\n",
      "  Read 1:    15,133,025 bp\n",
      "  Read 2:    15,129,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t542\t0.0\t2\t0 528 14\n",
      "26\t55349\t0.0\t2\t53604 1698 47\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t950\t0.0\t2\t0 920 30\n",
      "26\t54866\t0.0\t2\t53564 1255 47\n",
      "27\t230\t0.0\t2\t0 223 7\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S337_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S337_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S337_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S337_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S337_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S337_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S337_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S337_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,327 reads  @     86.7 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.97 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,327\n",
      "  Read 1 with adapter:                  57,264 (99.9%)\n",
      "  Read 2 with adapter:                  57,185 (99.8%)\n",
      "Pairs written (passing filters):        57,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,819,334 bp\n",
      "  Read 1:    16,910,373 bp\n",
      "  Read 2:    16,908,961 bp\n",
      "Total written (filtered):     30,844,509 bp (91.2%)\n",
      "  Read 1:    15,421,790 bp\n",
      "  Read 2:    15,422,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t501\t0.0\t2\t0 491 10\n",
      "26\t56497\t0.0\t2\t54642 1808 47\n",
      "27\t184\t0.0\t2\t0 179 5\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t938\t0.0\t2\t0 916 22\n",
      "26\t55905\t0.0\t2\t54593 1262 50\n",
      "27\t239\t0.0\t2\t0 234 5\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S338_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S338_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S338_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S338_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S338_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S338_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S338_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S338_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        34,908 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.89 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,908\n",
      "  Read 1 with adapter:                  34,858 (99.9%)\n",
      "  Read 2 with adapter:                  34,842 (99.8%)\n",
      "Pairs written (passing filters):        34,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,595,071 bp\n",
      "  Read 1:    10,297,595 bp\n",
      "  Read 2:    10,297,476 bp\n",
      "Total written (filtered):     18,783,544 bp (91.2%)\n",
      "  Read 1:     9,391,463 bp\n",
      "  Read 2:     9,392,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t325\t0.0\t2\t0 316 9\n",
      "26\t34371\t0.0\t2\t33343 1011 17\n",
      "27\t107\t0.0\t2\t0 104 3\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34842 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t559\t0.0\t2\t0 540 19\n",
      "26\t34103\t0.0\t2\t33388 696 19\n",
      "27\t118\t0.0\t2\t0 110 8\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S339_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S339_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S339_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S339_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S339_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S339_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S339_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S339_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,931 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 1.32 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,931\n",
      "  Read 1 with adapter:                  16,883 (99.7%)\n",
      "  Read 2 with adapter:                  16,899 (99.8%)\n",
      "Pairs written (passing filters):        16,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,986,511 bp\n",
      "  Read 1:     4,993,149 bp\n",
      "  Read 2:     4,993,362 bp\n",
      "Total written (filtered):      9,108,400 bp (91.2%)\n",
      "  Read 1:     4,554,228 bp\n",
      "  Read 2:     4,554,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 16883 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t125\t0.0\t2\t0 122 3\n",
      "26\t16654\t0.0\t2\t16135 504 15\n",
      "27\t76\t0.0\t2\t0 72 4\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 16899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t245\t0.0\t2\t0 241 4\n",
      "26\t16564\t0.0\t2\t16209 342 13\n",
      "27\t67\t0.0\t2\t0 63 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S340_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S340_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S340_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S340_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S340_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S340_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S340_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S340_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,161 reads  @     77.6 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.74 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,161\n",
      "  Read 1 with adapter:                  47,887 (99.4%)\n",
      "  Read 2 with adapter:                  48,065 (99.8%)\n",
      "Pairs written (passing filters):        48,161 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,402,222 bp\n",
      "  Read 1:    14,197,907 bp\n",
      "  Read 2:    14,204,315 bp\n",
      "Total written (filtered):     25,908,210 bp (91.2%)\n",
      "  Read 1:    12,952,932 bp\n",
      "  Read 2:    12,955,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t345\t0.0\t2\t0 332 13\n",
      "26\t47227\t0.0\t2\t45691 1490 46\n",
      "27\t204\t0.0\t2\t0 197 7\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t771\t0.0\t2\t0 749 22\n",
      "26\t47037\t0.0\t2\t45928 1073 36\n",
      "27\t188\t0.0\t2\t0 181 7\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S341_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S341_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S341_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S341_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S341_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S341_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S341_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S341_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        49,098 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.55 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,098\n",
      "  Read 1 with adapter:                  48,832 (99.5%)\n",
      "  Read 2 with adapter:                  48,992 (99.8%)\n",
      "Pairs written (passing filters):        49,098 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,954,337 bp\n",
      "  Read 1:    14,475,487 bp\n",
      "  Read 2:    14,478,850 bp\n",
      "Total written (filtered):     26,411,541 bp (91.2%)\n",
      "  Read 1:    13,205,868 bp\n",
      "  Read 2:    13,205,673 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t328\t0.0\t2\t0 315 13\n",
      "26\t48200\t0.0\t2\t46659 1504 37\n",
      "27\t193\t0.0\t2\t0 185 8\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t696\t0.0\t2\t0 678 18\n",
      "26\t48058\t0.0\t2\t47022 994 42\n",
      "27\t183\t0.0\t2\t0 176 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S342_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S342_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S342_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S342_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S342_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S342_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S342_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S342_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,653 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.83 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,653\n",
      "  Read 1 with adapter:                  53,385 (99.5%)\n",
      "  Read 2 with adapter:                  53,561 (99.8%)\n",
      "Pairs written (passing filters):        53,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,643,042 bp\n",
      "  Read 1:    15,817,105 bp\n",
      "  Read 2:    15,825,937 bp\n",
      "Total written (filtered):     28,863,179 bp (91.2%)\n",
      "  Read 1:    14,429,107 bp\n",
      "  Read 2:    14,434,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t382\t0.0\t2\t0 371 11\n",
      "26\t52634\t0.0\t2\t50925 1663 46\n",
      "27\t232\t0.0\t2\t0 227 5\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t834\t0.0\t2\t0 807 27\n",
      "26\t52450\t0.0\t2\t51253 1163 34\n",
      "27\t209\t0.0\t2\t0 197 12\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S343_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S343_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S343_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S343_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S343_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S343_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S343_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S343_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,891 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.92 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,891\n",
      "  Read 1 with adapter:                  50,817 (99.9%)\n",
      "  Read 2 with adapter:                  50,786 (99.8%)\n",
      "Pairs written (passing filters):        50,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,019,647 bp\n",
      "  Read 1:    15,011,640 bp\n",
      "  Read 2:    15,008,007 bp\n",
      "Total written (filtered):     27,378,430 bp (91.2%)\n",
      "  Read 1:    13,690,155 bp\n",
      "  Read 2:    13,688,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t356\t0.0\t2\t0 344 12\n",
      "26\t50143\t0.0\t2\t48526 1584 33\n",
      "27\t198\t0.0\t2\t0 188 10\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50786 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t774\t0.0\t2\t0 749 25\n",
      "26\t49720\t0.0\t2\t48554 1138 28\n",
      "27\t210\t0.0\t2\t0 199 11\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S344_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S344_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S344_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S344_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S344_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S344_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S344_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S344_187_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,787 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.30 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,787\n",
      "  Read 1 with adapter:                  51,710 (99.9%)\n",
      "  Read 2 with adapter:                  51,681 (99.8%)\n",
      "Pairs written (passing filters):        51,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,551,016 bp\n",
      "  Read 1:    15,277,154 bp\n",
      "  Read 2:    15,273,862 bp\n",
      "Total written (filtered):     27,863,788 bp (91.2%)\n",
      "  Read 1:    13,932,822 bp\n",
      "  Read 2:    13,930,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t377\t0.0\t2\t0 364 13\n",
      "26\t51050\t0.0\t2\t49379 1631 40\n",
      "27\t173\t0.0\t2\t0 170 3\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t882\t0.0\t2\t0 853 29\n",
      "26\t50517\t0.0\t2\t49262 1221 34\n",
      "27\t200\t0.0\t2\t0 194 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S345_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S345_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S345_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S345_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S345_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S345_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S345_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S345_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,561 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.59 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,561\n",
      "  Read 1 with adapter:                  46,485 (99.8%)\n",
      "  Read 2 with adapter:                  46,453 (99.8%)\n",
      "Pairs written (passing filters):        46,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,468,822 bp\n",
      "  Read 1:    13,735,092 bp\n",
      "  Read 2:    13,733,730 bp\n",
      "Total written (filtered):     25,053,461 bp (91.2%)\n",
      "  Read 1:    12,526,782 bp\n",
      "  Read 2:    12,526,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t407\t0.0\t2\t0 398 9\n",
      "26\t45905\t0.0\t2\t44317 1549 39\n",
      "27\t123\t0.0\t2\t0 120 3\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t804\t0.0\t2\t0 774 30\n",
      "26\t45366\t0.0\t2\t44183 1151 32\n",
      "27\t205\t0.0\t2\t0 197 8\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S346_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S346_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S346_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S346_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S346_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S346_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S346_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S346_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,670 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.55 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,670\n",
      "  Read 1 with adapter:                  47,558 (99.8%)\n",
      "  Read 2 with adapter:                  47,573 (99.8%)\n",
      "Pairs written (passing filters):        47,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,120,413 bp\n",
      "  Read 1:    14,060,018 bp\n",
      "  Read 2:    14,060,395 bp\n",
      "Total written (filtered):     25,647,875 bp (91.2%)\n",
      "  Read 1:    12,823,623 bp\n",
      "  Read 2:    12,824,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47558 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t317\t0.0\t2\t0 306 11\n",
      "26\t46999\t0.0\t2\t45579 1379 41\n",
      "27\t176\t0.0\t2\t0 174 2\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t853\t0.0\t2\t0 833 20\n",
      "26\t46444\t0.0\t2\t45402 1003 39\n",
      "27\t166\t0.0\t2\t0 159 7\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S347_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S347_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S347_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S347_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S347_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S347_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S347_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S347_193_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,722 reads  @     75.1 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.59 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,722\n",
      "  Read 1 with adapter:                  47,687 (99.9%)\n",
      "  Read 2 with adapter:                  47,626 (99.8%)\n",
      "Pairs written (passing filters):        47,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,152,021 bp\n",
      "  Read 1:    14,077,450 bp\n",
      "  Read 2:    14,074,571 bp\n",
      "Total written (filtered):     25,674,786 bp (91.2%)\n",
      "  Read 1:    12,837,863 bp\n",
      "  Read 2:    12,836,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t591\t0.0\t2\t0 574 17\n",
      "26\t46777\t0.0\t2\t45264 1467 46\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t699\t0.0\t2\t0 684 15\n",
      "26\t46660\t0.0\t2\t45601 1027 32\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S348_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S348_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S348_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S348_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S348_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S348_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S348_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S348_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,458 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.05 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,458\n",
      "  Read 1 with adapter:                  53,391 (99.9%)\n",
      "  Read 2 with adapter:                  53,347 (99.8%)\n",
      "Pairs written (passing filters):        53,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,537,705 bp\n",
      "  Read 1:    15,770,093 bp\n",
      "  Read 2:    15,767,612 bp\n",
      "Total written (filtered):     28,763,553 bp (91.2%)\n",
      "  Read 1:    14,382,209 bp\n",
      "  Read 2:    14,381,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t653\t0.0\t2\t0 634 19\n",
      "26\t52393\t0.0\t2\t50650 1700 43\n",
      "27\t215\t0.0\t2\t0 209 6\n",
      "28\t104\t0.0\t2\t0 0 104\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53347 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t859\t0.0\t2\t0 819 40\n",
      "26\t52188\t0.0\t2\t50864 1286 38\n",
      "27\t227\t0.0\t2\t0 219 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S349_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S349_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S349_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S349_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S349_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S349_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S349_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S349_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        57,409 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.00 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,409\n",
      "  Read 1 with adapter:                  57,342 (99.9%)\n",
      "  Read 2 with adapter:                  57,273 (99.8%)\n",
      "Pairs written (passing filters):        57,409 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,864,239 bp\n",
      "  Read 1:    16,934,565 bp\n",
      "  Read 2:    16,929,674 bp\n",
      "Total written (filtered):     30,885,337 bp (91.2%)\n",
      "  Read 1:    15,443,930 bp\n",
      "  Read 2:    15,441,407 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t667\t0.0\t2\t0 641 26\n",
      "26\t56266\t0.0\t2\t54347 1875 44\n",
      "27\t272\t0.0\t2\t0 264 8\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t941\t0.0\t2\t0 918 23\n",
      "26\t56036\t0.0\t2\t54664 1339 33\n",
      "27\t230\t0.0\t2\t0 225 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S350_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S350_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S350_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S350_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S350_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S350_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S350_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S350_199_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,962 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.88 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,962\n",
      "  Read 1 with adapter:                  53,900 (99.9%)\n",
      "  Read 2 with adapter:                  53,862 (99.8%)\n",
      "Pairs written (passing filters):        53,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,834,974 bp\n",
      "  Read 1:    15,917,538 bp\n",
      "  Read 2:    15,917,436 bp\n",
      "Total written (filtered):     29,034,212 bp (91.2%)\n",
      "  Read 1:    14,516,321 bp\n",
      "  Read 2:    14,517,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t598\t0.0\t2\t0 578 20\n",
      "26\t52921\t0.0\t2\t51173 1696 52\n",
      "27\t235\t0.0\t2\t0 227 8\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53862 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t949\t0.0\t2\t0 920 29\n",
      "26\t52595\t0.0\t2\t51340 1221 34\n",
      "27\t230\t0.0\t2\t0 221 9\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S351_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S351_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S351_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S351_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S351_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S351_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S351_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S351_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,596 reads  @     70.7 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.79 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,596\n",
      "  Read 1 with adapter:                  53,533 (99.9%)\n",
      "  Read 2 with adapter:                  53,496 (99.8%)\n",
      "Pairs written (passing filters):        53,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,618,645 bp\n",
      "  Read 1:    15,810,616 bp\n",
      "  Read 2:    15,808,029 bp\n",
      "Total written (filtered):     28,836,647 bp (91.2%)\n",
      "  Read 1:    14,418,716 bp\n",
      "  Read 2:    14,417,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53533 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t393\t0.0\t2\t0 377 16\n",
      "26\t52766\t0.0\t2\t50994 1734 38\n",
      "27\t237\t0.0\t2\t0 229 8\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t861\t0.0\t2\t0 838 23\n",
      "26\t52358\t0.0\t2\t51193 1132 33\n",
      "27\t203\t0.0\t2\t0 192 11\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S352_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S352_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S352_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S352_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S352_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S352_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S352_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S352_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,386 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.32 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,386\n",
      "  Read 1 with adapter:                  45,336 (99.9%)\n",
      "  Read 2 with adapter:                  45,264 (99.7%)\n",
      "Pairs written (passing filters):        45,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,772,017 bp\n",
      "  Read 1:    13,388,863 bp\n",
      "  Read 2:    13,383,154 bp\n",
      "Total written (filtered):     24,417,169 bp (91.2%)\n",
      "  Read 1:    12,210,160 bp\n",
      "  Read 2:    12,207,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t335\t0.0\t2\t0 326 9\n",
      "26\t44734\t0.0\t2\t43252 1444 38\n",
      "27\t156\t0.0\t2\t0 153 3\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t66\t0.0\t2\t0 0 66\n",
      "25\t776\t0.0\t2\t0 751 25\n",
      "26\t44235\t0.0\t2\t43115 1071 49\n",
      "27\t185\t0.0\t2\t0 177 8\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S353_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S353_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S353_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S353_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S353_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S353_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S353_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S353_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,630 reads  @     73.9 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.74 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,630\n",
      "  Read 1 with adapter:                  50,564 (99.9%)\n",
      "  Read 2 with adapter:                  50,500 (99.7%)\n",
      "Pairs written (passing filters):        50,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,867,648 bp\n",
      "  Read 1:    14,935,684 bp\n",
      "  Read 2:    14,931,964 bp\n",
      "Total written (filtered):     27,240,784 bp (91.2%)\n",
      "  Read 1:    13,621,055 bp\n",
      "  Read 2:    13,619,729 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t387\t0.0\t2\t0 369 18\n",
      "26\t49849\t0.0\t2\t48144 1653 52\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t108\t0.0\t2\t0 0 108\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t77\t0.0\t2\t0 0 77\n",
      "25\t853\t0.0\t2\t0 822 31\n",
      "26\t49343\t0.0\t2\t48121 1184 38\n",
      "27\t212\t0.0\t2\t0 199 13\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S354_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S354_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S354_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S354_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S354_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S354_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S354_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S354_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,602 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.63 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,602\n",
      "  Read 1 with adapter:                  49,559 (99.9%)\n",
      "  Read 2 with adapter:                  49,511 (99.8%)\n",
      "Pairs written (passing filters):        49,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,262,435 bp\n",
      "  Read 1:    14,631,992 bp\n",
      "  Read 2:    14,630,443 bp\n",
      "Total written (filtered):     26,687,661 bp (91.2%)\n",
      "  Read 1:    13,343,725 bp\n",
      "  Read 2:    13,343,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49559 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t426\t0.0\t2\t0 410 16\n",
      "26\t48933\t0.0\t2\t47242 1645 46\n",
      "27\t129\t0.0\t2\t0 124 5\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t871\t0.0\t2\t0 854 17\n",
      "26\t48386\t0.0\t2\t47206 1144 36\n",
      "27\t184\t0.0\t2\t0 177 7\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S355_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S355_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S355_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S355_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S355_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S355_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S355_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S355_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,858 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.64 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,858\n",
      "  Read 1 with adapter:                  39,544 (99.2%)\n",
      "  Read 2 with adapter:                  39,802 (99.9%)\n",
      "Pairs written (passing filters):        39,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,442,570 bp\n",
      "  Read 1:    11,685,077 bp\n",
      "  Read 2:    11,757,493 bp\n",
      "Total written (filtered):     21,380,155 bp (91.2%)\n",
      "  Read 1:    10,656,971 bp\n",
      "  Read 2:    10,723,184 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t273\t0.0\t2\t0 264 9\n",
      "26\t39004\t0.0\t2\t37706 1260 38\n",
      "27\t171\t0.0\t2\t0 167 4\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t619\t0.0\t2\t0 606 13\n",
      "26\t38979\t0.0\t2\t38093 858 28\n",
      "27\t156\t0.0\t2\t0 149 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S356_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S356_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S356_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S356_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S356_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S356_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S356_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S356_211_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,413 reads  @     73.9 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.73 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,413\n",
      "  Read 1 with adapter:                  50,338 (99.9%)\n",
      "  Read 2 with adapter:                  50,289 (99.8%)\n",
      "Pairs written (passing filters):        50,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,736,540 bp\n",
      "  Read 1:    14,869,551 bp\n",
      "  Read 2:    14,866,989 bp\n",
      "Total written (filtered):     27,121,068 bp (91.2%)\n",
      "  Read 1:    13,560,839 bp\n",
      "  Read 2:    13,560,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t371\t0.0\t2\t0 355 16\n",
      "26\t49649\t0.0\t2\t47987 1613 49\n",
      "27\t205\t0.0\t2\t0 200 5\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50289 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t815\t0.0\t2\t0 789 26\n",
      "26\t49226\t0.0\t2\t48077 1122 27\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S357_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S357_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S357_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S357_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S357_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S357_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S357_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S357_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,708 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.77 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,708\n",
      "  Read 1 with adapter:                  53,605 (99.8%)\n",
      "  Read 2 with adapter:                  53,591 (99.8%)\n",
      "Pairs written (passing filters):        53,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,678,466 bp\n",
      "  Read 1:    15,839,890 bp\n",
      "  Read 2:    15,838,576 bp\n",
      "Total written (filtered):     28,892,156 bp (91.2%)\n",
      "  Read 1:    14,446,250 bp\n",
      "  Read 2:    14,445,906 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t390\t0.0\t2\t0 374 16\n",
      "26\t52908\t0.0\t2\t51119 1747 42\n",
      "27\t190\t0.0\t2\t0 184 6\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t806\t0.0\t2\t0 788 18\n",
      "26\t52534\t0.0\t2\t51340 1145 49\n",
      "27\t188\t0.0\t2\t0 180 8\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S358_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S358_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S358_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S358_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S358_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S358_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S358_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S358_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,422 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.12 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,422\n",
      "  Read 1 with adapter:                  59,286 (99.8%)\n",
      "  Read 2 with adapter:                  59,311 (99.8%)\n",
      "Pairs written (passing filters):        59,422 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,046,659 bp\n",
      "  Read 1:    17,518,914 bp\n",
      "  Read 2:    17,527,745 bp\n",
      "Total written (filtered):     31,964,151 bp (91.2%)\n",
      "  Read 1:    15,977,588 bp\n",
      "  Read 2:    15,986,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59286 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t445\t0.0\t2\t0 436 9\n",
      "26\t58477\t0.0\t2\t56482 1946 49\n",
      "27\t213\t0.0\t2\t0 207 6\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59311 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t85\t0.0\t2\t0 0 85\n",
      "25\t984\t0.0\t2\t0 965 19\n",
      "26\t58019\t0.0\t2\t56682 1286 51\n",
      "27\t196\t0.0\t2\t0 189 7\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S359_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S359_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S359_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S359_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S359_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S359_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S359_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S359_217_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,412 reads  @     67.4 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.20 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,412\n",
      "  Read 1 with adapter:                  47,354 (99.9%)\n",
      "  Read 2 with adapter:                  47,313 (99.8%)\n",
      "Pairs written (passing filters):        47,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,971,562 bp\n",
      "  Read 1:    13,986,282 bp\n",
      "  Read 2:    13,985,280 bp\n",
      "Total written (filtered):     25,511,019 bp (91.2%)\n",
      "  Read 1:    12,755,289 bp\n",
      "  Read 2:    12,755,730 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47354 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t367\t0.0\t2\t0 356 11\n",
      "26\t46773\t0.0\t2\t45194 1542 37\n",
      "27\t120\t0.0\t2\t0 117 3\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t742\t0.0\t2\t0 724 18\n",
      "26\t46346\t0.0\t2\t45346 980 20\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S360_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S360_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S360_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S360_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S360_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S360_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S360_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S360_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,655 reads  @     63.8 µs/read;   0.94 M reads/minute\n",
      "Finished in 3.75 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,655\n",
      "  Read 1 with adapter:                  58,552 (99.8%)\n",
      "  Read 2 with adapter:                  58,525 (99.8%)\n",
      "Pairs written (passing filters):        58,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,601,536 bp\n",
      "  Read 1:    17,302,765 bp\n",
      "  Read 2:    17,298,771 bp\n",
      "Total written (filtered):     31,558,471 bp (91.2%)\n",
      "  Read 1:    15,780,445 bp\n",
      "  Read 2:    15,778,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58552 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t403\t0.0\t2\t0 384 19\n",
      "26\t57806\t0.0\t2\t55821 1942 43\n",
      "27\t203\t0.0\t2\t0 198 5\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58525 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t998\t0.0\t2\t0 973 25\n",
      "26\t57217\t0.0\t2\t55811 1366 40\n",
      "27\t227\t0.0\t2\t0 213 14\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S361_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S361_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S361_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S361_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S361_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S361_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S361_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S361_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,304 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.67 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,304\n",
      "  Read 1 with adapter:                  54,204 (99.8%)\n",
      "  Read 2 with adapter:                  54,173 (99.8%)\n",
      "Pairs written (passing filters):        54,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,035,016 bp\n",
      "  Read 1:    16,019,662 bp\n",
      "  Read 2:    16,015,354 bp\n",
      "Total written (filtered):     29,218,130 bp (91.2%)\n",
      "  Read 1:    14,610,379 bp\n",
      "  Read 2:    14,607,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t372\t0.0\t2\t0 367 5\n",
      "26\t53462\t0.0\t2\t51539 1880 43\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54173 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t913\t0.0\t2\t0 887 26\n",
      "26\t52981\t0.0\t2\t51751 1193 37\n",
      "27\t184\t0.0\t2\t0 179 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S362_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S362_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S362_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S362_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S362_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S362_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S362_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S362_223_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,630 reads  @     66.0 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.48 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,630\n",
      "  Read 1 with adapter:                  52,547 (99.8%)\n",
      "  Read 2 with adapter:                  52,515 (99.8%)\n",
      "Pairs written (passing filters):        52,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,049,572 bp\n",
      "  Read 1:    15,524,964 bp\n",
      "  Read 2:    15,524,608 bp\n",
      "Total written (filtered):     28,319,158 bp (91.2%)\n",
      "  Read 1:    14,159,075 bp\n",
      "  Read 2:    14,160,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t489\t0.0\t2\t0 480 9\n",
      "26\t51847\t0.0\t2\t50134 1665 48\n",
      "27\t158\t0.0\t2\t0 152 6\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52515 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t914\t0.0\t2\t0 891 23\n",
      "26\t51345\t0.0\t2\t50238 1079 28\n",
      "27\t183\t0.0\t2\t0 178 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S363_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S363_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S363_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S363_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S363_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S363_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S363_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S363_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,276 reads  @     69.3 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.69 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,276\n",
      "  Read 1 with adapter:                  53,203 (99.9%)\n",
      "  Read 2 with adapter:                  53,140 (99.7%)\n",
      "Pairs written (passing filters):        53,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,428,236 bp\n",
      "  Read 1:    15,716,031 bp\n",
      "  Read 2:    15,712,205 bp\n",
      "Total written (filtered):     28,664,063 bp (91.2%)\n",
      "  Read 1:    14,332,804 bp\n",
      "  Read 2:    14,331,259 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53203 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t507\t0.0\t2\t0 497 10\n",
      "26\t52456\t0.0\t2\t50779 1620 57\n",
      "27\t165\t0.0\t2\t0 158 7\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t66\t0.0\t2\t0 0 66\n",
      "25\t752\t0.0\t2\t0 733 19\n",
      "26\t52134\t0.0\t2\t51112 1006 16\n",
      "27\t186\t0.0\t2\t0 180 6\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S364_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S364_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S364_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S364_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S364_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S364_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S364_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S364_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,285 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.26 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,285\n",
      "  Read 1 with adapter:                  58,216 (99.9%)\n",
      "  Read 2 with adapter:                  58,165 (99.8%)\n",
      "Pairs written (passing filters):        58,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,381,721 bp\n",
      "  Read 1:    17,192,992 bp\n",
      "  Read 2:    17,188,729 bp\n",
      "Total written (filtered):     31,356,849 bp (91.2%)\n",
      "  Read 1:    15,679,593 bp\n",
      "  Read 2:    15,677,256 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t484\t0.0\t2\t0 468 16\n",
      "26\t57453\t0.0\t2\t55460 1933 60\n",
      "27\t199\t0.0\t2\t0 192 7\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58165 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t930\t0.0\t2\t0 900 30\n",
      "26\t56891\t0.0\t2\t55461 1389 41\n",
      "27\t243\t0.0\t2\t0 236 7\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S365_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S365_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S365_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S365_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S365_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S365_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S365_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S365_229_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,367 reads  @     70.4 µs/read;   0.85 M reads/minute\n",
      "Finished in 4.18 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,367\n",
      "  Read 1 with adapter:                  59,298 (99.9%)\n",
      "  Read 2 with adapter:                  59,204 (99.7%)\n",
      "Pairs written (passing filters):        59,367 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,021,096 bp\n",
      "  Read 1:    17,512,704 bp\n",
      "  Read 2:    17,508,392 bp\n",
      "Total written (filtered):     31,941,214 bp (91.2%)\n",
      "  Read 1:    15,971,264 bp\n",
      "  Read 2:    15,969,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t550\t0.0\t2\t0 530 20\n",
      "26\t58453\t0.0\t2\t56499 1912 42\n",
      "27\t224\t0.0\t2\t0 219 5\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t938\t0.0\t2\t0 909 29\n",
      "26\t58002\t0.0\t2\t56686 1281 35\n",
      "27\t192\t0.0\t2\t0 186 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S366_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S366_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S366_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S366_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S366_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S366_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S366_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S366_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,236 reads  @     94.9 µs/read;   0.63 M reads/minute\n",
      "Finished in 0.12 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,236\n",
      "  Read 1 with adapter:                   1,235 (99.9%)\n",
      "  Read 2 with adapter:                   1,235 (99.9%)\n",
      "Pairs written (passing filters):         1,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:       729,240 bp\n",
      "  Read 1:       364,620 bp\n",
      "  Read 2:       364,620 bp\n",
      "Total written (filtered):        665,051 bp (91.2%)\n",
      "  Read 1:       332,520 bp\n",
      "  Read 2:       332,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t11\t0.0\t2\t0 11\n",
      "26\t1217\t0.0\t2\t1165 46 6\n",
      "27\t5\t0.0\t2\t0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t25\t0.0\t2\t0 23 2\n",
      "26\t1203\t0.0\t2\t1165 37 1\n",
      "27\t2\t0.0\t2\t0 2\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S367_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S367_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S367_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S367_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S367_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S367_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S367_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S367_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,511 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.65 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,511\n",
      "  Read 1 with adapter:                  51,407 (99.8%)\n",
      "  Read 2 with adapter:                  51,413 (99.8%)\n",
      "Pairs written (passing filters):        51,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,383,594 bp\n",
      "  Read 1:    15,189,773 bp\n",
      "  Read 2:    15,193,821 bp\n",
      "Total written (filtered):     27,711,370 bp (91.2%)\n",
      "  Read 1:    13,853,594 bp\n",
      "  Read 2:    13,857,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51407 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t712\t0.0\t2\t0 693 19\n",
      "26\t50380\t0.0\t2\t48642 1686 52\n",
      "27\t173\t0.0\t2\t0 168 5\n",
      "28\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t813\t0.0\t2\t0 788 25\n",
      "26\t50300\t0.0\t2\t49129 1144 27\n",
      "27\t220\t0.0\t2\t0 213 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S368_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S368_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S368_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S368_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S368_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S368_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S368_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S368_235_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,047 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.17 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,047\n",
      "  Read 1 with adapter:                  60,965 (99.9%)\n",
      "  Read 2 with adapter:                  60,926 (99.8%)\n",
      "Pairs written (passing filters):        61,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,012,491 bp\n",
      "  Read 1:    18,006,531 bp\n",
      "  Read 2:    18,005,960 bp\n",
      "Total written (filtered):     32,844,940 bp (91.2%)\n",
      "  Read 1:    16,422,104 bp\n",
      "  Read 2:    16,422,836 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t858\t0.0\t2\t0 827 31\n",
      "26\t59810\t0.0\t2\t57523 2219 68\n",
      "27\t199\t0.0\t2\t0 188 11\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t97\t0.0\t2\t0 0 97\n",
      "25\t1118\t0.0\t2\t0 1073 45\n",
      "26\t59400\t0.0\t2\t57765 1592 43\n",
      "27\t262\t0.0\t2\t0 255 7\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S369_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S369_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S369_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S369_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S369_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-2mli2637/S369_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S369_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-9529fskz/db6f779a-8853-46b1-a71b-956bf5f6bb8e/data/S369_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,748 reads  @     84.6 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.89 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,748\n",
      "  Read 1 with adapter:                  57,628 (99.8%)\n",
      "  Read 2 with adapter:                  57,607 (99.8%)\n",
      "Pairs written (passing filters):        57,748 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,059,910 bp\n",
      "  Read 1:    17,027,974 bp\n",
      "  Read 2:    17,031,936 bp\n",
      "Total written (filtered):     31,065,509 bp (91.2%)\n",
      "  Read 1:    15,530,409 bp\n",
      "  Read 2:    15,535,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t886\t0.0\t2\t0 859 27\n",
      "26\t56517\t0.0\t2\t54457 2005 55\n",
      "27\t151\t0.0\t2\t0 144 7\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t998\t0.0\t2\t0 974 24\n",
      "26\t56286\t0.0\t2\t55037 1211 38\n",
      "27\t218\t0.0\t2\t0 214 4\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: b-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: b-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-qh_wq3ow')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b9a208a2-da62-48cc-a347-26c8dd556905>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.b import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib2-leray_manifest_run2_v1.csv \\\n",
    "  --output-path b-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.b trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences b-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences b-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.b summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data b-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization b-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('b-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: b-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: b-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: b-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.b Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs b-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 265 \\\n",
    "  --p-trunc-len-r 135 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table b-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats b-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: b-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-knxi8rfk')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 9e0e2f69-260e-4680-b7c1-1ad82b9cfe16>"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.b view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table b-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization b-table.qzv \n",
    "q2.Visualization.load('b-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: b-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.b view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization b-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib3-leray_manifest_r3.csv as PairedEndFastqManifestPhred33 to c-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s651_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s651_148_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s651_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s651_148_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s651_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s651_148_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s651_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s651_148_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,266 reads  @     92.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.30 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,266\n",
      "  Read 1 with adapter:                  46,252 (100.0%)\n",
      "  Read 2 with adapter:                  45,940 (99.3%)\n",
      "Pairs written (passing filters):        46,266 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,291,190 bp\n",
      "  Read 1:    13,647,696 bp\n",
      "  Read 2:    13,643,494 bp\n",
      "Total written (filtered):     24,894,621 bp (91.2%)\n",
      "  Read 1:    12,445,335 bp\n",
      "  Read 2:    12,449,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t503\t0.0\t2\t0 487 16\n",
      "26\t45420\t0.0\t2\t44628 782 10\n",
      "27\t222\t0.0\t2\t0 221 1\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t447\t0.0\t2\t0 438 9\n",
      "26\t45265\t0.0\t2\t44407 824 34\n",
      "27\t173\t0.0\t2\t0 172 1\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s652_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s652_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s652_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s652_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s652_1_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s652_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s652_1_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s652_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,738 reads  @     98.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.78 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,738\n",
      "  Read 1 with adapter:                  48,721 (100.0%)\n",
      "  Read 2 with adapter:                  48,548 (99.6%)\n",
      "Pairs written (passing filters):        48,738 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,740,402 bp\n",
      "  Read 1:    14,376,991 bp\n",
      "  Read 2:    14,363,411 bp\n",
      "Total written (filtered):     26,211,818 bp (91.2%)\n",
      "  Read 1:    13,110,543 bp\n",
      "  Read 2:    13,101,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48721 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t617\t0.0\t2\t0 593 24\n",
      "26\t47760\t0.0\t2\t46576 1167 17\n",
      "27\t233\t0.0\t2\t0 226 7\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48548 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t410\t0.0\t2\t0 396 14\n",
      "26\t47836\t0.0\t2\t46675 1127 34\n",
      "27\t226\t0.0\t2\t0 221 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s653_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s653_150_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s653_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s653_150_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s653_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s653_150_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s653_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s653_150_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,346 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.38 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,346\n",
      "  Read 1 with adapter:                  44,326 (100.0%)\n",
      "  Read 2 with adapter:                  44,284 (99.9%)\n",
      "Pairs written (passing filters):        44,346 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,155,282 bp\n",
      "  Read 1:    13,081,138 bp\n",
      "  Read 2:    13,074,144 bp\n",
      "Total written (filtered):     23,851,912 bp (91.2%)\n",
      "  Read 1:    11,929,027 bp\n",
      "  Read 2:    11,922,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44326 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t562\t0.0\t2\t0 554 8\n",
      "26\t43511\t0.0\t2\t42648 850 13\n",
      "27\t205\t0.0\t2\t0 203 2\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44284 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t315\t0.0\t2\t0 309 6\n",
      "26\t43731\t0.0\t2\t42898 801 32\n",
      "27\t222\t0.0\t2\t0 217 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s654_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s654_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s654_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s654_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s654_3_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s654_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s654_3_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s654_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,599 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.78 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,599\n",
      "  Read 1 with adapter:                  52,576 (100.0%)\n",
      "  Read 2 with adapter:                  52,569 (99.9%)\n",
      "Pairs written (passing filters):        52,599 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,030,621 bp\n",
      "  Read 1:    15,515,143 bp\n",
      "  Read 2:    15,515,478 bp\n",
      "Total written (filtered):     28,297,408 bp (91.2%)\n",
      "  Read 1:    14,148,611 bp\n",
      "  Read 2:    14,148,797 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52576 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t687\t0.0\t2\t0 664 23\n",
      "26\t51588\t0.0\t2\t50502 1065 21\n",
      "27\t235\t0.0\t2\t0 227 8\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t356\t0.0\t2\t0 348 8\n",
      "26\t51932\t0.0\t2\t50694 1195 43\n",
      "27\t263\t0.0\t2\t0 254 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s655_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s655_152_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s655_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s655_152_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s655_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s655_152_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s655_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s655_152_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,431 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.01 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,431\n",
      "  Read 1 with adapter:                  54,406 (100.0%)\n",
      "  Read 2 with adapter:                  54,383 (99.9%)\n",
      "Pairs written (passing filters):        54,431 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,107,193 bp\n",
      "  Read 1:    16,054,379 bp\n",
      "  Read 2:    16,052,814 bp\n",
      "Total written (filtered):     29,279,433 bp (91.2%)\n",
      "  Read 1:    14,640,219 bp\n",
      "  Read 2:    14,639,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t659\t0.0\t2\t0 634 25\n",
      "26\t53436\t0.0\t2\t52402 1014 20\n",
      "27\t243\t0.0\t2\t0 238 5\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t573\t0.0\t2\t0 554 19\n",
      "26\t53506\t0.0\t2\t52437 1035 34\n",
      "27\t269\t0.0\t2\t0 265 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s656_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s656_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s656_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s656_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s656_5_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s656_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s656_5_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s656_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,641 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.68 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,641\n",
      "  Read 1 with adapter:                  50,624 (100.0%)\n",
      "  Read 2 with adapter:                  50,565 (99.8%)\n",
      "Pairs written (passing filters):        50,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,867,067 bp\n",
      "  Read 1:    14,938,450 bp\n",
      "  Read 2:    14,928,617 bp\n",
      "Total written (filtered):     27,236,702 bp (91.2%)\n",
      "  Read 1:    13,622,579 bp\n",
      "  Read 2:    13,614,123 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50624 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t610\t0.0\t2\t0 595 15\n",
      "26\t49708\t0.0\t2\t48690 1006 12\n",
      "27\t235\t0.0\t2\t0 228 7\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t443\t0.0\t2\t0 428 15\n",
      "26\t49823\t0.0\t2\t48869 925 29\n",
      "27\t279\t0.0\t2\t0 269 10\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s657_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s657_154_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s657_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s657_154_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s657_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s657_154_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s657_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s657_154_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,200 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.10 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,200\n",
      "  Read 1 with adapter:                  56,170 (99.9%)\n",
      "  Read 2 with adapter:                  56,130 (99.9%)\n",
      "Pairs written (passing filters):        56,200 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,151,548 bp\n",
      "  Read 1:    16,577,301 bp\n",
      "  Read 2:    16,574,247 bp\n",
      "Total written (filtered):     30,232,425 bp (91.2%)\n",
      "  Read 1:    15,117,315 bp\n",
      "  Read 2:    15,115,110 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56170 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t678\t0.0\t2\t0 648 30\n",
      "26\t55200\t0.0\t2\t54058 1129 13\n",
      "27\t228\t0.0\t2\t0 225 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56130 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t444\t0.0\t2\t0 432 12\n",
      "26\t55414\t0.0\t2\t54344 1048 22\n",
      "27\t247\t0.0\t2\t0 245 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s658_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s658_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s658_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s658_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s658_7_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s658_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s658_7_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s658_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,828 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.91 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,828\n",
      "  Read 1 with adapter:                  47,807 (100.0%)\n",
      "  Read 2 with adapter:                  47,787 (99.9%)\n",
      "Pairs written (passing filters):        47,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,217,071 bp\n",
      "  Read 1:    14,108,190 bp\n",
      "  Read 2:    14,108,881 bp\n",
      "Total written (filtered):     25,732,182 bp (91.2%)\n",
      "  Read 1:    12,865,613 bp\n",
      "  Read 2:    12,866,569 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t637\t0.0\t2\t0 608 29\n",
      "26\t46900\t0.0\t2\t45868 1017 15\n",
      "27\t212\t0.0\t2\t0 208 4\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47787 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t345\t0.0\t2\t0 337 8\n",
      "26\t47213\t0.0\t2\t46368 829 16\n",
      "27\t211\t0.0\t2\t0 207 4\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s659_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s659_156_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s659_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s659_156_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s659_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s659_156_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s659_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s659_156_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,810 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.15 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,810\n",
      "  Read 1 with adapter:                  40,784 (99.9%)\n",
      "  Read 2 with adapter:                  40,444 (99.1%)\n",
      "Pairs written (passing filters):        40,809 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,070,189 bp\n",
      "  Read 1:    12,036,780 bp\n",
      "  Read 2:    12,033,409 bp\n",
      "Total written (filtered):     21,958,636 bp (91.2%)\n",
      "  Read 1:    10,976,804 bp\n",
      "  Read 2:    10,981,832 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40784 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t594\t0.0\t2\t0 578 16\n",
      "26\t39924\t0.0\t2\t39023 881 20\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40444 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t408\t0.0\t2\t0 393 15\n",
      "26\t39820\t0.0\t2\t38990 793 37\n",
      "27\t200\t0.0\t2\t0 196 4\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s660_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s660_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s660_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s660_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s660_9_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s660_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s660_9_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s660_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,351 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.22 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,351\n",
      "  Read 1 with adapter:                  46,328 (100.0%)\n",
      "  Read 2 with adapter:                  46,298 (99.9%)\n",
      "Pairs written (passing filters):        46,349 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,342,504 bp\n",
      "  Read 1:    13,670,932 bp\n",
      "  Read 2:    13,671,572 bp\n",
      "Total written (filtered):     24,934,289 bp (91.2%)\n",
      "  Read 1:    12,466,878 bp\n",
      "  Read 2:    12,467,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t693\t0.0\t2\t0 672 21\n",
      "26\t45348\t0.0\t2\t44363 970 15\n",
      "27\t219\t0.0\t2\t0 214 5\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t363\t0.0\t2\t0 359 4\n",
      "26\t45633\t0.0\t2\t44636 962 35\n",
      "27\t274\t0.0\t2\t0 265 9\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s661_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s661_158_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s661_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s661_158_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s661_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s661_158_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s661_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s661_158_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,340 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.90 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,340\n",
      "  Read 1 with adapter:                  38,320 (99.9%)\n",
      "  Read 2 with adapter:                  38,294 (99.9%)\n",
      "Pairs written (passing filters):        38,339 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,611,737 bp\n",
      "  Read 1:    11,308,842 bp\n",
      "  Read 2:    11,302,895 bp\n",
      "Total written (filtered):     20,619,954 bp (91.2%)\n",
      "  Read 1:    10,312,861 bp\n",
      "  Read 2:    10,307,093 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t538\t0.0\t2\t0 513 25\n",
      "26\t37540\t0.0\t2\t36692 832 16\n",
      "27\t193\t0.0\t2\t0 189 4\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38294 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t292\t0.0\t2\t0 285 7\n",
      "26\t37805\t0.0\t2\t37064 720 21\n",
      "27\t189\t0.0\t2\t0 184 5\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s662_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s662_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s662_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s662_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s662_11_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s662_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s662_11_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s662_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,429 reads  @     78.3 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.09 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,429\n",
      "  Read 1 with adapter:                  39,383 (99.9%)\n",
      "  Read 2 with adapter:                  39,392 (99.9%)\n",
      "Pairs written (passing filters):        39,429 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,260,326 bp\n",
      "  Read 1:    11,630,167 bp\n",
      "  Read 2:    11,630,159 bp\n",
      "Total written (filtered):     21,212,579 bp (91.2%)\n",
      "  Read 1:    10,606,507 bp\n",
      "  Read 2:    10,606,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t508\t0.0\t2\t0 497 11\n",
      "26\t38602\t0.0\t2\t37722 860 20\n",
      "27\t208\t0.0\t2\t0 203 5\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39392 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t285\t0.0\t2\t0 279 6\n",
      "26\t38903\t0.0\t2\t38014 848 41\n",
      "27\t188\t0.0\t2\t0 180 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s663_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s663_160_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s663_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s663_160_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s663_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s663_160_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s663_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s663_160_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,996 reads  @     76.1 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.35 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,996\n",
      "  Read 1 with adapter:                  43,968 (99.9%)\n",
      "  Read 2 with adapter:                  43,964 (99.9%)\n",
      "Pairs written (passing filters):        43,996 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,954,805 bp\n",
      "  Read 1:    12,977,332 bp\n",
      "  Read 2:    12,977,473 bp\n",
      "Total written (filtered):     23,669,306 bp (91.2%)\n",
      "  Read 1:    11,834,646 bp\n",
      "  Read 2:    11,834,660 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43968 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t646\t0.0\t2\t0 622 24\n",
      "26\t43079\t0.0\t2\t42106 958 15\n",
      "27\t178\t0.0\t2\t0 173 5\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43964 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t466\t0.0\t2\t0 458 8\n",
      "26\t43235\t0.0\t2\t42367 844 24\n",
      "27\t239\t0.0\t2\t0 234 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s664_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s664_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s664_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s664_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s664_13_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s664_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s664_13_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s664_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,135 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.28 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,135\n",
      "  Read 1 with adapter:                  43,112 (99.9%)\n",
      "  Read 2 with adapter:                  43,051 (99.8%)\n",
      "Pairs written (passing filters):        43,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,436,380 bp\n",
      "  Read 1:    12,723,669 bp\n",
      "  Read 2:    12,712,711 bp\n",
      "Total written (filtered):     23,196,465 bp (91.2%)\n",
      "  Read 1:    11,603,154 bp\n",
      "  Read 2:    11,593,311 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t620\t0.0\t2\t0 601 19\n",
      "26\t42224\t0.0\t2\t41293 913 18\n",
      "27\t197\t0.0\t2\t0 192 5\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t366\t0.0\t2\t0 356 10\n",
      "26\t42467\t0.0\t2\t41618 819 30\n",
      "27\t201\t0.0\t2\t0 196 5\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s665_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s665_162_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s665_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s665_162_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s665_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s665_162_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s665_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s665_162_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,977 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 2.92 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,977\n",
      "  Read 1 with adapter:                  39,946 (99.9%)\n",
      "  Read 2 with adapter:                  39,926 (99.9%)\n",
      "Pairs written (passing filters):        39,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,578,558 bp\n",
      "  Read 1:    11,790,284 bp\n",
      "  Read 2:    11,788,274 bp\n",
      "Total written (filtered):     21,501,952 bp (91.2%)\n",
      "  Read 1:    10,752,113 bp\n",
      "  Read 2:    10,749,839 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39946 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t614\t0.0\t2\t0 598 16\n",
      "26\t39068\t0.0\t2\t38264 793 11\n",
      "27\t203\t0.0\t2\t0 198 5\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t334\t0.0\t2\t0 322 12\n",
      "26\t39395\t0.0\t2\t38612 759 24\n",
      "27\t183\t0.0\t2\t0 178 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s666_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s666_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s666_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s666_163_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s666_15_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s666_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s666_15_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s666_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,162 reads  @     75.4 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.11 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,162\n",
      "  Read 1 with adapter:                  41,135 (99.9%)\n",
      "  Read 2 with adapter:                  41,127 (99.9%)\n",
      "Pairs written (passing filters):        41,162 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,283,338 bp\n",
      "  Read 1:    12,141,022 bp\n",
      "  Read 2:    12,142,316 bp\n",
      "Total written (filtered):     22,145,065 bp (91.2%)\n",
      "  Read 1:    11,071,887 bp\n",
      "  Read 2:    11,073,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41135 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t578\t0.0\t2\t0 557 21\n",
      "26\t40268\t0.0\t2\t39351 904 13\n",
      "27\t211\t0.0\t2\t0 207 4\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41127 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t353\t0.0\t2\t0 344 9\n",
      "26\t40542\t0.0\t2\t39779 737 26\n",
      "27\t215\t0.0\t2\t0 212 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s667_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s667_164_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s667_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s667_164_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s667_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s667_164_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s667_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s667_164_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        50,186 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.03 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,186\n",
      "  Read 1 with adapter:                  49,651 (98.9%)\n",
      "  Read 2 with adapter:                  49,726 (99.1%)\n",
      "Pairs written (passing filters):        50,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,542,781 bp\n",
      "  Read 1:    14,745,440 bp\n",
      "  Read 2:    14,797,341 bp\n",
      "Total written (filtered):     26,959,591 bp (91.3%)\n",
      "  Read 1:    13,454,821 bp\n",
      "  Read 2:    13,504,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t527\t0.0\t2\t0 510 17\n",
      "26\t48859\t0.0\t2\t47899 947 13\n",
      "27\t202\t0.0\t2\t0 196 6\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49726 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t555\t0.0\t2\t0 537 18\n",
      "26\t48863\t0.0\t2\t47840 962 61\n",
      "27\t266\t0.0\t2\t0 255 11\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s668_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s668_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s668_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s668_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s668_17_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s668_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s668_17_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s668_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,008 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.26 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,008\n",
      "  Read 1 with adapter:                  55,497 (99.1%)\n",
      "  Read 2 with adapter:                  55,949 (99.9%)\n",
      "Pairs written (passing filters):        56,008 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,987,129 bp\n",
      "  Read 1:    16,466,402 bp\n",
      "  Read 2:    16,520,727 bp\n",
      "Total written (filtered):     30,090,051 bp (91.2%)\n",
      "  Read 1:    15,023,742 bp\n",
      "  Read 2:    15,066,309 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55497 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t565\t0.0\t2\t0 546 19\n",
      "26\t54641\t0.0\t2\t53549 1076 16\n",
      "27\t219\t0.0\t2\t0 211 8\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t428\t0.0\t2\t0 420 8\n",
      "26\t55299\t0.0\t2\t54138 1113 48\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s669_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s669_166_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s669_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s669_166_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s669_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s669_166_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s669_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s669_166_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,317 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.25 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,317\n",
      "  Read 1 with adapter:                  43,868 (99.0%)\n",
      "  Read 2 with adapter:                  44,261 (99.9%)\n",
      "Pairs written (passing filters):        44,317 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,090,030 bp\n",
      "  Read 1:    13,024,067 bp\n",
      "  Read 2:    13,065,963 bp\n",
      "Total written (filtered):     23,799,050 bp (91.2%)\n",
      "  Read 1:    11,883,758 bp\n",
      "  Read 2:    11,915,292 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t488\t0.0\t2\t0 468 20\n",
      "26\t43152\t0.0\t2\t42277 859 16\n",
      "27\t191\t0.0\t2\t0 183 8\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t295\t0.0\t2\t0 293 2\n",
      "26\t43741\t0.0\t2\t42859 864 18\n",
      "27\t206\t0.0\t2\t0 202 4\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s670_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s670_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s670_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s670_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s670_19_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s670_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s670_19_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s670_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,672 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.72 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,672\n",
      "  Read 1 with adapter:                  49,076 (98.8%)\n",
      "  Read 2 with adapter:                  49,622 (99.9%)\n",
      "Pairs written (passing filters):        49,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,239,871 bp\n",
      "  Read 1:    14,587,090 bp\n",
      "  Read 2:    14,652,781 bp\n",
      "Total written (filtered):     26,674,216 bp (91.2%)\n",
      "  Read 1:    13,311,452 bp\n",
      "  Read 2:    13,362,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t574\t0.0\t2\t0 552 22\n",
      "26\t48236\t0.0\t2\t47221 1004 11\n",
      "27\t212\t0.0\t2\t0 206 6\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t364\t0.0\t2\t0 351 13\n",
      "26\t49026\t0.0\t2\t47814 1164 48\n",
      "27\t211\t0.0\t2\t0 206 5\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s671_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s671_168_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s671_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s671_168_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s671_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s671_168_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s671_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s671_168_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,660 reads  @     75.3 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.19 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,660\n",
      "  Read 1 with adapter:                  54,916 (98.7%)\n",
      "  Read 2 with adapter:                  55,598 (99.9%)\n",
      "Pairs written (passing filters):        55,660 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,753,804 bp\n",
      "  Read 1:    16,338,040 bp\n",
      "  Read 2:    16,415,764 bp\n",
      "Total written (filtered):     29,881,178 bp (91.2%)\n",
      "  Read 1:    14,910,633 bp\n",
      "  Read 2:    14,970,545 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54916 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t699\t0.0\t2\t0 670 29\n",
      "26\t53888\t0.0\t2\t52669 1201 18\n",
      "27\t264\t0.0\t2\t0 261 3\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t632\t0.0\t2\t0 619 13\n",
      "26\t54586\t0.0\t2\t53374 1176 36\n",
      "27\t337\t0.0\t2\t0 327 10\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s672_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s672_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s672_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s672_169_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s672_21_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s672_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s672_21_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s672_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,571 reads  @     78.3 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.67 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,571\n",
      "  Read 1 with adapter:                  58,898 (98.9%)\n",
      "  Read 2 with adapter:                  59,475 (99.8%)\n",
      "Pairs written (passing filters):        59,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,060,993 bp\n",
      "  Read 1:    17,499,468 bp\n",
      "  Read 2:    17,561,525 bp\n",
      "Total written (filtered):     31,983,948 bp (91.2%)\n",
      "  Read 1:    15,968,495 bp\n",
      "  Read 2:    16,015,453 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t659\t0.0\t2\t0 639 20\n",
      "26\t57904\t0.0\t2\t56665 1225 14\n",
      "27\t270\t0.0\t2\t0 266 4\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59475 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t551\t0.0\t2\t0 538 13\n",
      "26\t58585\t0.0\t2\t57396 1152 37\n",
      "27\t305\t0.0\t2\t0 299 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s673_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s673_170_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s673_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s673_170_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s673_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s673_170_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s673_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s673_170_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,788 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.74 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,788\n",
      "  Read 1 with adapter:                  56,133 (98.8%)\n",
      "  Read 2 with adapter:                  56,687 (99.8%)\n",
      "Pairs written (passing filters):        56,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,456,638 bp\n",
      "  Read 1:    16,710,202 bp\n",
      "  Read 2:    16,746,436 bp\n",
      "Total written (filtered):     30,523,881 bp (91.2%)\n",
      "  Read 1:    15,251,083 bp\n",
      "  Read 2:    15,272,798 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t650\t0.0\t2\t0 623 27\n",
      "26\t55113\t0.0\t2\t53986 1108 19\n",
      "27\t301\t0.0\t2\t0 292 9\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t420\t0.0\t2\t0 411 9\n",
      "26\t56012\t0.0\t2\t54947 1033 32\n",
      "27\t226\t0.0\t2\t0 220 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s674_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s674_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s674_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s674_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s674_23_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s674_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s674_23_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s674_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,496 reads  @     94.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.78 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,496\n",
      "  Read 1 with adapter:                  50,164 (99.3%)\n",
      "  Read 2 with adapter:                  50,467 (99.9%)\n",
      "Pairs written (passing filters):        50,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,772,068 bp\n",
      "  Read 1:    14,876,484 bp\n",
      "  Read 2:    14,895,584 bp\n",
      "Total written (filtered):     27,156,155 bp (91.2%)\n",
      "  Read 1:    13,572,546 bp\n",
      "  Read 2:    13,583,609 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t586\t0.0\t2\t0 566 20\n",
      "26\t49261\t0.0\t2\t48250 996 15\n",
      "27\t250\t0.0\t2\t0 243 7\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50467 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t402\t0.0\t2\t0 388 14\n",
      "26\t49779\t0.0\t2\t48777 977 25\n",
      "27\t245\t0.0\t2\t0 238 7\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s675_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s675_172_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s675_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s675_172_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s675_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s675_172_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s675_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s675_172_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,196 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.81 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,196\n",
      "  Read 1 with adapter:                  44,168 (99.9%)\n",
      "  Read 2 with adapter:                  43,841 (99.2%)\n",
      "Pairs written (passing filters):        44,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,069,588 bp\n",
      "  Read 1:    13,037,269 bp\n",
      "  Read 2:    13,032,319 bp\n",
      "Total written (filtered):     23,781,907 bp (91.2%)\n",
      "  Read 1:    11,889,243 bp\n",
      "  Read 2:    11,892,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44168 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t620\t0.0\t2\t0 594 26\n",
      "26\t43242\t0.0\t2\t42423 807 12\n",
      "27\t206\t0.0\t2\t0 200 6\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t445\t0.0\t2\t0 429 16\n",
      "26\t43149\t0.0\t2\t42251 868 30\n",
      "27\t168\t0.0\t2\t0 166 2\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s676_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s676_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s676_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s676_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s676_25_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s676_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s676_25_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s676_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,880 reads  @     98.1 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.99 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,880\n",
      "  Read 1 with adapter:                  50,846 (99.9%)\n",
      "  Read 2 with adapter:                  50,777 (99.8%)\n",
      "Pairs written (passing filters):        50,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,015,439 bp\n",
      "  Read 1:    15,007,973 bp\n",
      "  Read 2:    15,007,466 bp\n",
      "Total written (filtered):     27,373,769 bp (91.2%)\n",
      "  Read 1:    13,686,367 bp\n",
      "  Read 2:    13,687,402 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50846 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t697\t0.0\t2\t0 671 26\n",
      "26\t49784\t0.0\t2\t48718 1049 17\n",
      "27\t291\t0.0\t2\t0 284 7\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t367\t0.0\t2\t0 360 7\n",
      "26\t50148\t0.0\t2\t49198 920 30\n",
      "27\t239\t0.0\t2\t0 233 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s677_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s677_174_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s677_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s677_174_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s677_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s677_174_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s677_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s677_174_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        40,921 reads  @     96.7 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.96 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,921\n",
      "  Read 1 with adapter:                  40,899 (99.9%)\n",
      "  Read 2 with adapter:                  40,799 (99.7%)\n",
      "Pairs written (passing filters):        40,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,133,666 bp\n",
      "  Read 1:    12,071,284 bp\n",
      "  Read 2:    12,062,382 bp\n",
      "Total written (filtered):     22,009,956 bp (91.2%)\n",
      "  Read 1:    11,008,245 bp\n",
      "  Read 2:    11,001,711 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t565\t0.0\t2\t0 547 18\n",
      "26\t40045\t0.0\t2\t39114 921 10\n",
      "27\t220\t0.0\t2\t0 210 10\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t292\t0.0\t2\t0 286 6\n",
      "26\t40263\t0.0\t2\t39464 784 15\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s678_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s678_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s678_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s678_175_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s678_27_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s678_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s678_27_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s678_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,335 reads  @     92.3 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.91 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,335\n",
      "  Read 1 with adapter:                  42,314 (100.0%)\n",
      "  Read 2 with adapter:                  42,304 (99.9%)\n",
      "Pairs written (passing filters):        42,335 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,977,148 bp\n",
      "  Read 1:    12,488,484 bp\n",
      "  Read 2:    12,488,664 bp\n",
      "Total written (filtered):     22,777,484 bp (91.2%)\n",
      "  Read 1:    11,388,653 bp\n",
      "  Read 2:    11,388,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t569\t0.0\t2\t0 548 21\n",
      "26\t41454\t0.0\t2\t40549 898 7\n",
      "27\t218\t0.0\t2\t0 214 4\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t306\t0.0\t2\t0 296 10\n",
      "26\t41747\t0.0\t2\t40894 829 24\n",
      "27\t219\t0.0\t2\t0 211 8\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s679_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s679_176_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s679_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s679_176_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s679_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s679_176_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s679_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s679_176_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        50,837 reads  @     85.9 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.37 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,837\n",
      "  Read 1 with adapter:                  50,801 (99.9%)\n",
      "  Read 2 with adapter:                  50,724 (99.8%)\n",
      "Pairs written (passing filters):        50,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,978,376 bp\n",
      "  Read 1:    14,996,625 bp\n",
      "  Read 2:    14,981,751 bp\n",
      "Total written (filtered):     27,339,411 bp (91.2%)\n",
      "  Read 1:    13,676,189 bp\n",
      "  Read 2:    13,663,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50801 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t729\t0.0\t2\t0 704 25\n",
      "26\t49705\t0.0\t2\t48622 1062 21\n",
      "27\t267\t0.0\t2\t0 259 8\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50724 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t599\t0.0\t2\t0 585 14\n",
      "26\t49812\t0.0\t2\t48839 949 24\n",
      "27\t246\t0.0\t2\t0 239 7\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s680_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s680_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s680_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s680_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s680_29_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s680_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s680_29_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s680_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,765 reads  @     93.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.90 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,765\n",
      "  Read 1 with adapter:                  41,752 (100.0%)\n",
      "  Read 2 with adapter:                  41,617 (99.6%)\n",
      "Pairs written (passing filters):        41,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,618,509 bp\n",
      "  Read 1:    12,320,656 bp\n",
      "  Read 2:    12,297,853 bp\n",
      "Total written (filtered):     22,451,467 bp (91.2%)\n",
      "  Read 1:    11,235,453 bp\n",
      "  Read 2:    11,216,014 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41752 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t482\t0.0\t2\t0 458 24\n",
      "26\t41075\t0.0\t2\t40245 820 10\n",
      "27\t153\t0.0\t2\t0 149 4\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41617 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t376\t0.0\t2\t0 365 11\n",
      "26\t41057\t0.0\t2\t40294 748 15\n",
      "27\t151\t0.0\t2\t0 146 5\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s681_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s681_178_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s681_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s681_178_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s681_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s681_178_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s681_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s681_178_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,248 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.34 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,248\n",
      "  Read 1 with adapter:                  45,222 (99.9%)\n",
      "  Read 2 with adapter:                  45,207 (99.9%)\n",
      "Pairs written (passing filters):        45,248 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,693,263 bp\n",
      "  Read 1:    13,347,771 bp\n",
      "  Read 2:    13,345,492 bp\n",
      "Total written (filtered):     24,342,720 bp (91.2%)\n",
      "  Read 1:    12,172,449 bp\n",
      "  Read 2:    12,170,271 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45222 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t664\t0.0\t2\t0 652 12\n",
      "26\t44290\t0.0\t2\t43290 983 17\n",
      "27\t206\t0.0\t2\t0 201 5\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45207 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t366\t0.0\t2\t0 354 12\n",
      "26\t44611\t0.0\t2\t43764 818 29\n",
      "27\t211\t0.0\t2\t0 206 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s682_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s682_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s682_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s682_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s682_31_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s682_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s682_31_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s682_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        41,870 reads  @    105.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.42 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,870\n",
      "  Read 1 with adapter:                  41,845 (99.9%)\n",
      "  Read 2 with adapter:                  41,840 (99.9%)\n",
      "Pairs written (passing filters):        41,870 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,701,454 bp\n",
      "  Read 1:    12,350,983 bp\n",
      "  Read 2:    12,350,471 bp\n",
      "Total written (filtered):     22,526,020 bp (91.2%)\n",
      "  Read 1:    11,263,257 bp\n",
      "  Read 2:    11,262,763 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t500\t0.0\t2\t0 484 16\n",
      "26\t41062\t0.0\t2\t40210 838 14\n",
      "27\t186\t0.0\t2\t0 180 6\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41840 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t334\t0.0\t2\t0 325 9\n",
      "26\t41296\t0.0\t2\t40575 710 11\n",
      "27\t166\t0.0\t2\t0 161 5\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s683_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s683_180_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s683_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s683_180_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s683_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s683_180_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s683_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s683_180_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,516 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.51 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,516\n",
      "  Read 1 with adapter:                  46,500 (100.0%)\n",
      "  Read 2 with adapter:                  46,202 (99.3%)\n",
      "Pairs written (passing filters):        46,516 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,438,985 bp\n",
      "  Read 1:    13,722,024 bp\n",
      "  Read 2:    13,716,961 bp\n",
      "Total written (filtered):     25,029,353 bp (91.2%)\n",
      "  Read 1:    12,513,326 bp\n",
      "  Read 2:    12,516,027 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t594\t0.0\t2\t0 572 22\n",
      "26\t45602\t0.0\t2\t44723 863 16\n",
      "27\t224\t0.0\t2\t0 216 8\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46202 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t496\t0.0\t2\t0 485 11\n",
      "26\t45488\t0.0\t2\t44605 847 36\n",
      "27\t190\t0.0\t2\t0 180 10\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s684_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s684_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s684_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s684_181_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s684_33_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s684_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s684_33_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s684_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,661 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.84 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,661\n",
      "  Read 1 with adapter:                  52,634 (99.9%)\n",
      "  Read 2 with adapter:                  52,600 (99.9%)\n",
      "Pairs written (passing filters):        52,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,068,201 bp\n",
      "  Read 1:    15,534,925 bp\n",
      "  Read 2:    15,533,276 bp\n",
      "Total written (filtered):     28,332,786 bp (91.2%)\n",
      "  Read 1:    14,166,879 bp\n",
      "  Read 2:    14,165,907 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52634 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t811\t0.0\t2\t0 781 30\n",
      "26\t51376\t0.0\t2\t50288 1065 23\n",
      "27\t373\t0.0\t2\t0 369 4\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t427\t0.0\t2\t0 419 8\n",
      "26\t51906\t0.0\t2\t50941 941 24\n",
      "27\t230\t0.0\t2\t0 226 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s685_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s685_182_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s685_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s685_182_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s685_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s685_182_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s685_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s685_182_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,917 reads  @     99.8 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.49 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,917\n",
      "  Read 1 with adapter:                  44,866 (99.9%)\n",
      "  Read 2 with adapter:                  44,850 (99.9%)\n",
      "Pairs written (passing filters):        44,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,491,262 bp\n",
      "  Read 1:    13,247,200 bp\n",
      "  Read 2:    13,244,062 bp\n",
      "Total written (filtered):     24,158,989 bp (91.2%)\n",
      "  Read 1:    12,080,948 bp\n",
      "  Read 2:    12,078,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44866 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t595\t0.0\t2\t0 581 14\n",
      "26\t43944\t0.0\t2\t43071 859 14\n",
      "27\t215\t0.0\t2\t0 213 2\n",
      "28\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t288\t0.0\t2\t0 276 12\n",
      "26\t44339\t0.0\t2\t43473 843 23\n",
      "27\t197\t0.0\t2\t0 191 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s686_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s686_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s686_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s686_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s686_35_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s686_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s686_35_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s686_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,484 reads  @     89.2 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.15 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,484\n",
      "  Read 1 with adapter:                  46,452 (99.9%)\n",
      "  Read 2 with adapter:                  46,452 (99.9%)\n",
      "Pairs written (passing filters):        46,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,424,423 bp\n",
      "  Read 1:    13,712,208 bp\n",
      "  Read 2:    13,712,215 bp\n",
      "Total written (filtered):     25,009,194 bp (91.2%)\n",
      "  Read 1:    12,504,624 bp\n",
      "  Read 2:    12,504,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t604\t0.0\t2\t0 575 29\n",
      "26\t45452\t0.0\t2\t44589 851 12\n",
      "27\t268\t0.0\t2\t0 262 6\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t311\t0.0\t2\t0 297 14\n",
      "26\t45878\t0.0\t2\t44939 905 34\n",
      "27\t230\t0.0\t2\t0 228 2\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s687_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s687_184_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s687_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s687_184_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s687_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s687_184_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s687_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s687_184_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        49,854 reads  @     99.9 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.98 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,854\n",
      "  Read 1 with adapter:                  49,739 (99.8%)\n",
      "  Read 2 with adapter:                  49,741 (99.8%)\n",
      "Pairs written (passing filters):        49,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,395,597 bp\n",
      "  Read 1:    14,703,207 bp\n",
      "  Read 2:    14,692,390 bp\n",
      "Total written (filtered):     26,809,794 bp (91.2%)\n",
      "  Read 1:    13,410,390 bp\n",
      "  Read 2:    13,399,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t779\t0.0\t2\t0 749 30\n",
      "26\t48544\t0.0\t2\t47339 1179 26\n",
      "27\t290\t0.0\t2\t0 281 9\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49741 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t624\t0.0\t2\t0 609 15\n",
      "26\t48728\t0.0\t2\t47655 1051 22\n",
      "27\t330\t0.0\t2\t0 320 10\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s688_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s688_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s688_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s688_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s688_37_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s688_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s688_37_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s688_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        52,718 reads  @     97.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.15 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,718\n",
      "  Read 1 with adapter:                  52,693 (100.0%)\n",
      "  Read 2 with adapter:                  52,643 (99.9%)\n",
      "Pairs written (passing filters):        52,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,092,913 bp\n",
      "  Read 1:    15,550,895 bp\n",
      "  Read 2:    15,542,018 bp\n",
      "Total written (filtered):     28,354,693 bp (91.2%)\n",
      "  Read 1:    14,181,215 bp\n",
      "  Read 2:    14,173,478 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t726\t0.0\t2\t0 700 26\n",
      "26\t51543\t0.0\t2\t50602 922 19\n",
      "27\t340\t0.0\t2\t0 332 8\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t465\t0.0\t2\t0 457 8\n",
      "26\t51882\t0.0\t2\t50873 976 33\n",
      "27\t229\t0.0\t2\t0 222 7\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s689_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s689_186_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s689_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s689_186_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s689_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s689_186_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s689_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s689_186_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           837 reads  @    221.1 µs/read;   0.27 M reads/minute\n",
      "Finished in 0.19 s (225 us/read; 0.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                837\n",
      "  Read 1 with adapter:                     831 (99.3%)\n",
      "  Read 2 with adapter:                     831 (99.3%)\n",
      "Pairs written (passing filters):           837 (100.0%)\n",
      "\n",
      "Total basepairs processed:       492,133 bp\n",
      "  Read 1:       246,381 bp\n",
      "  Read 2:       245,752 bp\n",
      "Total written (filtered):        448,924 bp (91.2%)\n",
      "  Read 1:       224,780 bp\n",
      "  Read 2:       224,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t15\t0.0\t2\t0 14 1\n",
      "26\t806\t0.0\t2\t791 14 1\n",
      "27\t6\t0.0\t2\t0 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t826\t0.0\t2\t801 24 1\n",
      "27\t2\t0.0\t2\t0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s690_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s690_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s690_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s690_187_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s690_39_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s690_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s690_39_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s690_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,409 reads  @     85.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.72 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,409\n",
      "  Read 1 with adapter:                  55,344 (99.9%)\n",
      "  Read 2 with adapter:                  55,387 (100.0%)\n",
      "Pairs written (passing filters):        55,409 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,689,285 bp\n",
      "  Read 1:    16,343,902 bp\n",
      "  Read 2:    16,345,383 bp\n",
      "Total written (filtered):     29,810,868 bp (91.2%)\n",
      "  Read 1:    14,905,363 bp\n",
      "  Read 2:    14,905,505 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55344 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t732\t0.0\t2\t0 699 33\n",
      "26\t54242\t0.0\t2\t53123 1100 19\n",
      "27\t285\t0.0\t2\t0 278 7\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t387\t0.0\t2\t0 371 16\n",
      "26\t54738\t0.0\t2\t53753 969 16\n",
      "27\t233\t0.0\t2\t0 228 5\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s691_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s691_188_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s691_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s691_188_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s691_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s691_188_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s691_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s691_188_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,336 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.76 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,336\n",
      "  Read 1 with adapter:                  42,318 (100.0%)\n",
      "  Read 2 with adapter:                  41,980 (99.2%)\n",
      "Pairs written (passing filters):        42,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,972,429 bp\n",
      "  Read 1:    12,488,723 bp\n",
      "  Read 2:    12,483,706 bp\n",
      "Total written (filtered):     22,781,175 bp (91.2%)\n",
      "  Read 1:    11,388,703 bp\n",
      "  Read 2:    11,392,472 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t469\t0.0\t2\t0 450 19\n",
      "26\t41597\t0.0\t2\t40803 782 12\n",
      "27\t183\t0.0\t2\t0 177 6\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t416\t0.0\t2\t0 404 12\n",
      "26\t41331\t0.0\t2\t40484 811 36\n",
      "27\t196\t0.0\t2\t0 190 6\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s692_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s692_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s692_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s692_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s692_41_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s692_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s692_41_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s692_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,999 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.08 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,999\n",
      "  Read 1 with adapter:                  44,984 (100.0%)\n",
      "  Read 2 with adapter:                  44,954 (99.9%)\n",
      "Pairs written (passing filters):        44,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,547,677 bp\n",
      "  Read 1:    13,274,537 bp\n",
      "  Read 2:    13,273,140 bp\n",
      "Total written (filtered):     24,209,674 bp (91.2%)\n",
      "  Read 1:    12,105,172 bp\n",
      "  Read 2:    12,104,502 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44984 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t457\t0.0\t2\t0 443 14\n",
      "26\t44262\t0.0\t2\t43409 841 12\n",
      "27\t208\t0.0\t2\t0 205 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t335\t0.0\t2\t0 329 6\n",
      "26\t44406\t0.0\t2\t43573 810 23\n",
      "27\t185\t0.0\t2\t0 183 2\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s693_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s693_190_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s693_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s693_190_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s693_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s693_190_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s693_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s693_190_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,712 reads  @     87.3 µs/read;   0.69 M reads/minute\n",
      "Finished in 2.69 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,712\n",
      "  Read 1 with adapter:                  30,699 (100.0%)\n",
      "  Read 2 with adapter:                  30,669 (99.9%)\n",
      "Pairs written (passing filters):        30,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,114,253 bp\n",
      "  Read 1:     9,059,835 bp\n",
      "  Read 2:     9,054,418 bp\n",
      "Total written (filtered):     16,518,892 bp (91.2%)\n",
      "  Read 1:     8,261,797 bp\n",
      "  Read 2:     8,257,095 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30699 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t308\t0.0\t2\t0 301 7\n",
      "26\t30225\t0.0\t2\t29567 645 13\n",
      "27\t132\t0.0\t2\t0 126 6\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30669 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t221\t0.0\t2\t0 218 3\n",
      "26\t30282\t0.0\t2\t29678 584 20\n",
      "27\t150\t0.0\t2\t0 145 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s694_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s694_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s694_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s694_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s694_43_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s694_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s694_43_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s694_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,060 reads  @     87.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.61 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,060\n",
      "  Read 1 with adapter:                  41,037 (99.9%)\n",
      "  Read 2 with adapter:                  41,023 (99.9%)\n",
      "Pairs written (passing filters):        41,059 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,224,699 bp\n",
      "  Read 1:    12,112,447 bp\n",
      "  Read 2:    12,112,252 bp\n",
      "Total written (filtered):     22,091,160 bp (91.2%)\n",
      "  Read 1:    11,045,399 bp\n",
      "  Read 2:    11,045,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t417\t0.0\t2\t0 401 16\n",
      "26\t40381\t0.0\t2\t39520 850 11\n",
      "27\t180\t0.0\t2\t0 178 2\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t287\t0.0\t2\t0 280 7\n",
      "26\t40516\t0.0\t2\t39658 829 29\n",
      "27\t196\t0.0\t2\t0 189 7\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s695_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s695_192_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s695_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s695_192_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s695_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s695_192_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s695_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s695_192_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,453 reads  @     84.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.57 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,453\n",
      "  Read 1 with adapter:                  42,433 (100.0%)\n",
      "  Read 2 with adapter:                  42,420 (99.9%)\n",
      "Pairs written (passing filters):        42,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,044,588 bp\n",
      "  Read 1:    12,523,243 bp\n",
      "  Read 2:    12,521,345 bp\n",
      "Total written (filtered):     22,838,816 bp (91.2%)\n",
      "  Read 1:    11,420,138 bp\n",
      "  Read 2:    11,418,678 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t434\t0.0\t2\t0 423 11\n",
      "26\t41737\t0.0\t2\t40851 869 17\n",
      "27\t187\t0.0\t2\t0 181 6\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42420 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t424\t0.0\t2\t0 409 15\n",
      "26\t41769\t0.0\t2\t40955 790 24\n",
      "27\t195\t0.0\t2\t0 192 3\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s696_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s696_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s696_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s696_193_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s696_45_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s696_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s696_45_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s696_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,707 reads  @     85.8 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.01 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,707\n",
      "  Read 1 with adapter:                  46,694 (100.0%)\n",
      "  Read 2 with adapter:                  46,616 (99.8%)\n",
      "Pairs written (passing filters):        46,707 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,546,169 bp\n",
      "  Read 1:    13,778,556 bp\n",
      "  Read 2:    13,767,613 bp\n",
      "Total written (filtered):     25,120,495 bp (91.2%)\n",
      "  Read 1:    12,564,695 bp\n",
      "  Read 2:    12,555,800 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t471\t0.0\t2\t0 451 20\n",
      "26\t45925\t0.0\t2\t44957 958 10\n",
      "27\t244\t0.0\t2\t0 243 1\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t401\t0.0\t2\t0 391 10\n",
      "26\t45969\t0.0\t2\t45062 875 32\n",
      "27\t214\t0.0\t2\t0 212 2\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s697_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s697_194_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s697_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s697_194_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s697_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s697_194_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s697_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s697_194_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,599 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.81 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,599\n",
      "  Read 1 with adapter:                  43,584 (100.0%)\n",
      "  Read 2 with adapter:                  43,552 (99.9%)\n",
      "Pairs written (passing filters):        43,599 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,720,004 bp\n",
      "  Read 1:    12,861,694 bp\n",
      "  Read 2:    12,858,310 bp\n",
      "Total written (filtered):     23,454,890 bp (91.2%)\n",
      "  Read 1:    11,728,769 bp\n",
      "  Read 2:    11,726,121 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43584 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t461\t0.0\t2\t0 446 15\n",
      "26\t42873\t0.0\t2\t41947 918 8\n",
      "27\t182\t0.0\t2\t0 178 4\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43552 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t352\t0.0\t2\t0 336 16\n",
      "26\t42976\t0.0\t2\t42190 766 20\n",
      "27\t195\t0.0\t2\t0 192 3\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s698_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s698_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s698_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s698_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s698_47_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s698_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s698_47_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s698_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,128 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.23 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,128\n",
      "  Read 1 with adapter:                  39,118 (100.0%)\n",
      "  Read 2 with adapter:                  39,110 (100.0%)\n",
      "Pairs written (passing filters):        39,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,084,549 bp\n",
      "  Read 1:    11,542,495 bp\n",
      "  Read 2:    11,542,054 bp\n",
      "Total written (filtered):     21,050,881 bp (91.2%)\n",
      "  Read 1:    10,525,588 bp\n",
      "  Read 2:    10,525,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t362\t0.0\t2\t0 349 13\n",
      "26\t38551\t0.0\t2\t37702 831 18\n",
      "27\t145\t0.0\t2\t0 141 4\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t271\t0.0\t2\t0 266 5\n",
      "26\t38633\t0.0\t2\t37902 713 18\n",
      "27\t176\t0.0\t2\t0 169 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s699_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s699_196_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s699_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s699_196_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s699_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s699_196_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s699_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s699_196_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        23,256 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 2.31 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,256\n",
      "  Read 1 with adapter:                  23,242 (99.9%)\n",
      "  Read 2 with adapter:                  23,088 (99.3%)\n",
      "Pairs written (passing filters):        23,256 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,718,292 bp\n",
      "  Read 1:     6,860,504 bp\n",
      "  Read 2:     6,857,788 bp\n",
      "Total written (filtered):     12,513,997 bp (91.2%)\n",
      "  Read 1:     6,256,310 bp\n",
      "  Read 2:     6,257,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23242 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t221\t0.0\t2\t0 210 11\n",
      "26\t22887\t0.0\t2\t22132 738 17\n",
      "27\t101\t0.0\t2\t0 101\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23088 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t296\t0.0\t2\t0 290 6\n",
      "26\t22640\t0.0\t2\t22122 506 12\n",
      "27\t131\t0.0\t2\t0 125 6\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s700_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s700_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s700_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s700_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s700_49_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s700_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s700_49_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s700_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        14,796 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 1.23 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,796\n",
      "  Read 1 with adapter:                  14,785 (99.9%)\n",
      "  Read 2 with adapter:                  14,764 (99.8%)\n",
      "Pairs written (passing filters):        14,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,728,541 bp\n",
      "  Read 1:     4,364,207 bp\n",
      "  Read 2:     4,364,334 bp\n",
      "Total written (filtered):      7,960,388 bp (91.2%)\n",
      "  Read 1:     3,979,874 bp\n",
      "  Read 2:     3,980,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 14785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t145\t0.0\t2\t0 135 10\n",
      "26\t14566\t0.0\t2\t14118 441 7\n",
      "27\t64\t0.0\t2\t0 61 3\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 14764 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t139\t0.0\t2\t0 135 4\n",
      "26\t14534\t0.0\t2\t14069 443 22\n",
      "27\t79\t0.0\t2\t0 76 3\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s701_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s701_198_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s701_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s701_198_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s701_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s701_198_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s701_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s701_198_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,081 reads  @     88.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.19 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,081\n",
      "  Read 1 with adapter:                  46,970 (99.8%)\n",
      "  Read 2 with adapter:                  46,740 (99.3%)\n",
      "Pairs written (passing filters):        47,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,746,781 bp\n",
      "  Read 1:    13,863,191 bp\n",
      "  Read 2:    13,883,590 bp\n",
      "Total written (filtered):     25,310,813 bp (91.2%)\n",
      "  Read 1:    12,642,226 bp\n",
      "  Read 2:    12,668,587 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t573\t0.0\t2\t0 554 19\n",
      "26\t46102\t0.0\t2\t45054 1022 26\n",
      "27\t204\t0.0\t2\t0 199 5\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t566\t0.0\t2\t0 547 19\n",
      "26\t45826\t0.0\t2\t44756 1013 57\n",
      "27\t255\t0.0\t2\t0 245 10\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s702_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s702_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s702_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s702_199_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s702_51_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s702_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s702_51_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s702_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,397 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.95 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,397\n",
      "  Read 1 with adapter:                  47,358 (99.9%)\n",
      "  Read 2 with adapter:                  47,237 (99.7%)\n",
      "Pairs written (passing filters):        47,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,950,395 bp\n",
      "  Read 1:    13,979,790 bp\n",
      "  Read 2:    13,970,605 bp\n",
      "Total written (filtered):     25,491,413 bp (91.2%)\n",
      "  Read 1:    12,748,815 bp\n",
      "  Read 2:    12,742,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t627\t0.0\t2\t0 597 30\n",
      "26\t46438\t0.0\t2\t45133 1284 21\n",
      "27\t216\t0.0\t2\t0 213 3\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47237 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t397\t0.0\t2\t0 388 9\n",
      "26\t46580\t0.0\t2\t45345 1200 35\n",
      "27\t218\t0.0\t2\t0 211 7\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s703_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s703_200_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s703_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s703_200_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s703_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s703_200_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s703_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s703_200_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,191 reads  @     79.8 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.69 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,191\n",
      "  Read 1 with adapter:                  46,171 (100.0%)\n",
      "  Read 2 with adapter:                  46,126 (99.9%)\n",
      "Pairs written (passing filters):        46,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,243,812 bp\n",
      "  Read 1:    13,625,696 bp\n",
      "  Read 2:    13,618,116 bp\n",
      "Total written (filtered):     24,844,532 bp (91.2%)\n",
      "  Read 1:    12,425,585 bp\n",
      "  Read 2:    12,418,947 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t577\t0.0\t2\t0 562 15\n",
      "26\t45281\t0.0\t2\t44251 1011 19\n",
      "27\t248\t0.0\t2\t0 241 7\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t303\t0.0\t2\t0 298 5\n",
      "26\t45581\t0.0\t2\t44604 958 19\n",
      "27\t224\t0.0\t2\t0 215 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s704_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s704_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s704_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s704_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s704_53_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s704_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s704_53_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s704_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,979 reads  @     77.0 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.54 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,979\n",
      "  Read 1 with adapter:                  45,950 (99.9%)\n",
      "  Read 2 with adapter:                  45,936 (99.9%)\n",
      "Pairs written (passing filters):        45,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,125,567 bp\n",
      "  Read 1:    13,562,359 bp\n",
      "  Read 2:    13,563,208 bp\n",
      "Total written (filtered):     24,737,000 bp (91.2%)\n",
      "  Read 1:    12,368,024 bp\n",
      "  Read 2:    12,368,976 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45950 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t574\t0.0\t2\t0 552 22\n",
      "26\t45137\t0.0\t2\t44120 993 24\n",
      "27\t185\t0.0\t2\t0 178 7\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45936 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t303\t0.0\t2\t0 295 8\n",
      "26\t45395\t0.0\t2\t44373 990 32\n",
      "27\t221\t0.0\t2\t0 217 4\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s705_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s705_202_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s705_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s705_202_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s705_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s705_202_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s705_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s705_202_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,138 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.51 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,138\n",
      "  Read 1 with adapter:                  56,105 (99.9%)\n",
      "  Read 2 with adapter:                  56,087 (99.9%)\n",
      "Pairs written (passing filters):        56,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,115,144 bp\n",
      "  Read 1:    16,556,429 bp\n",
      "  Read 2:    16,558,715 bp\n",
      "Total written (filtered):     30,198,852 bp (91.2%)\n",
      "  Read 1:    15,098,097 bp\n",
      "  Read 2:    15,100,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56105 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t708\t0.0\t2\t0 681 27\n",
      "26\t55078\t0.0\t2\t53807 1251 20\n",
      "27\t264\t0.0\t2\t0 256 8\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t638\t0.0\t2\t0 621 17\n",
      "26\t55054\t0.0\t2\t53888 1130 36\n",
      "27\t370\t0.0\t2\t0 360 10\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s706_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s706_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s706_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s706_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s706_55_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s706_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s706_55_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s706_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,782 reads  @     70.9 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.82 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,782\n",
      "  Read 1 with adapter:                  53,748 (99.9%)\n",
      "  Read 2 with adapter:                  53,688 (99.8%)\n",
      "Pairs written (passing filters):        53,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,718,622 bp\n",
      "  Read 1:    15,864,791 bp\n",
      "  Read 2:    15,853,831 bp\n",
      "Total written (filtered):     28,925,870 bp (91.2%)\n",
      "  Read 1:    14,467,727 bp\n",
      "  Read 2:    14,458,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t640\t0.0\t2\t0 617 23\n",
      "26\t52797\t0.0\t2\t51692 1091 14\n",
      "27\t254\t0.0\t2\t0 248 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53688 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t479\t0.0\t2\t0 468 11\n",
      "26\t52887\t0.0\t2\t51844 1009 34\n",
      "27\t301\t0.0\t2\t0 293 8\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s707_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s707_204_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s707_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s707_204_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s707_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s707_204_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s707_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s707_204_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,355 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.76 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,355\n",
      "  Read 1 with adapter:                  52,324 (99.9%)\n",
      "  Read 2 with adapter:                  52,302 (99.9%)\n",
      "Pairs written (passing filters):        52,355 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,883,449 bp\n",
      "  Read 1:    15,442,741 bp\n",
      "  Read 2:    15,440,708 bp\n",
      "Total written (filtered):     28,163,771 bp (91.2%)\n",
      "  Read 1:    14,082,690 bp\n",
      "  Read 2:    14,081,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t642\t0.0\t2\t0 629 13\n",
      "26\t51368\t0.0\t2\t50148 1191 29\n",
      "27\t251\t0.0\t2\t0 245 6\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t480\t0.0\t2\t0 465 15\n",
      "26\t51486\t0.0\t2\t50448 1013 25\n",
      "27\t301\t0.0\t2\t0 292 9\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s708_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s708_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s708_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s708_205_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s708_57_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s708_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s708_57_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s708_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,693 reads  @     76.4 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.34 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,693\n",
      "  Read 1 with adapter:                  43,662 (99.9%)\n",
      "  Read 2 with adapter:                  43,664 (99.9%)\n",
      "Pairs written (passing filters):        43,693 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,775,824 bp\n",
      "  Read 1:    12,886,719 bp\n",
      "  Read 2:    12,889,105 bp\n",
      "Total written (filtered):     23,505,825 bp (91.2%)\n",
      "  Read 1:    11,751,813 bp\n",
      "  Read 2:    11,754,012 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t566\t0.0\t2\t0 542 24\n",
      "26\t42820\t0.0\t2\t41778 1015 27\n",
      "27\t224\t0.0\t2\t0 219 5\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t355\t0.0\t2\t0 348 7\n",
      "26\t43069\t0.0\t2\t42286 764 19\n",
      "27\t212\t0.0\t2\t0 209 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s709_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s709_206_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s709_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s709_206_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s709_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s709_206_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s709_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s709_206_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,869 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.53 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,869\n",
      "  Read 1 with adapter:                  49,845 (100.0%)\n",
      "  Read 2 with adapter:                  49,432 (99.1%)\n",
      "Pairs written (passing filters):        49,869 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,415,213 bp\n",
      "  Read 1:    14,710,494 bp\n",
      "  Read 2:    14,704,719 bp\n",
      "Total written (filtered):     26,834,736 bp (91.2%)\n",
      "  Read 1:    13,414,875 bp\n",
      "  Read 2:    13,419,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t615\t0.0\t2\t0 601 14\n",
      "26\t48922\t0.0\t2\t47933 973 16\n",
      "27\t244\t0.0\t2\t0 234 10\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t533\t0.0\t2\t0 521 12\n",
      "26\t48632\t0.0\t2\t47509 1069 54\n",
      "27\t231\t0.0\t2\t0 222 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s710_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s710_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s710_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s710_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s710_59_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s710_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s710_59_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s710_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,411 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.20 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,411\n",
      "  Read 1 with adapter:                  53,381 (99.9%)\n",
      "  Read 2 with adapter:                  53,362 (99.9%)\n",
      "Pairs written (passing filters):        53,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,510,606 bp\n",
      "  Read 1:    15,755,765 bp\n",
      "  Read 2:    15,754,841 bp\n",
      "Total written (filtered):     28,735,828 bp (91.2%)\n",
      "  Read 1:    14,368,252 bp\n",
      "  Read 2:    14,367,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53381 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t660\t0.0\t2\t0 641 19\n",
      "26\t52371\t0.0\t2\t51256 1101 14\n",
      "27\t277\t0.0\t2\t0 265 12\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t448\t0.0\t2\t0 439 9\n",
      "26\t52555\t0.0\t2\t51368 1142 45\n",
      "27\t337\t0.0\t2\t0 326 11\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s711_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s711_208_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s711_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s711_208_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s711_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s711_208_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s711_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s711_208_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,728 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.18 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,728\n",
      "  Read 1 with adapter:                  42,699 (99.9%)\n",
      "  Read 2 with adapter:                  42,661 (99.8%)\n",
      "Pairs written (passing filters):        42,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,197,784 bp\n",
      "  Read 1:    12,603,404 bp\n",
      "  Read 2:    12,594,380 bp\n",
      "Total written (filtered):     22,978,822 bp (91.2%)\n",
      "  Read 1:    11,493,581 bp\n",
      "  Read 2:    11,485,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42699 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t616\t0.0\t2\t0 595 21\n",
      "26\t41782\t0.0\t2\t40867 901 14\n",
      "27\t241\t0.0\t2\t0 235 6\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t275\t0.0\t2\t0 266 9\n",
      "26\t42144\t0.0\t2\t41287 826 31\n",
      "27\t232\t0.0\t2\t0 220 12\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s712_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s712_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s712_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s712_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s712_61_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s712_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s712_61_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s712_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,652 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.88 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,652\n",
      "  Read 1 with adapter:                  47,628 (99.9%)\n",
      "  Read 2 with adapter:                  47,616 (99.9%)\n",
      "Pairs written (passing filters):        47,652 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,113,183 bp\n",
      "  Read 1:    14,056,750 bp\n",
      "  Read 2:    14,056,433 bp\n",
      "Total written (filtered):     25,637,210 bp (91.2%)\n",
      "  Read 1:    12,818,740 bp\n",
      "  Read 2:    12,818,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t616\t0.0\t2\t0 595 21\n",
      "26\t46680\t0.0\t2\t45677 983 20\n",
      "27\t282\t0.0\t2\t0 277 5\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t326\t0.0\t2\t0 316 10\n",
      "26\t46990\t0.0\t2\t45951 996 43\n",
      "27\t283\t0.0\t2\t0 271 12\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s713_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s713_210_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s713_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s713_210_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s713_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s713_210_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s713_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s713_210_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,612 reads  @     83.4 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.56 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,612\n",
      "  Read 1 with adapter:                  54,578 (99.9%)\n",
      "  Read 2 with adapter:                  54,572 (99.9%)\n",
      "Pairs written (passing filters):        54,612 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,218,797 bp\n",
      "  Read 1:    16,109,645 bp\n",
      "  Read 2:    16,109,152 bp\n",
      "Total written (filtered):     29,381,704 bp (91.2%)\n",
      "  Read 1:    14,691,022 bp\n",
      "  Read 2:    14,690,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54578 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t690\t0.0\t2\t0 666 24\n",
      "26\t53547\t0.0\t2\t52343 1191 13\n",
      "27\t277\t0.0\t2\t0 268 9\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t660\t0.0\t2\t0 643 17\n",
      "26\t53554\t0.0\t2\t52425 1090 39\n",
      "27\t318\t0.0\t2\t0 303 15\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s714_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s714_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s714_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s714_211_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s714_63_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s714_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s714_63_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s714_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,553 reads  @     87.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.08 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,553\n",
      "  Read 1 with adapter:                  46,534 (100.0%)\n",
      "  Read 2 with adapter:                  46,461 (99.8%)\n",
      "Pairs written (passing filters):        46,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,455,662 bp\n",
      "  Read 1:    13,732,825 bp\n",
      "  Read 2:    13,722,837 bp\n",
      "Total written (filtered):     25,038,238 bp (91.2%)\n",
      "  Read 1:    12,523,243 bp\n",
      "  Read 2:    12,514,995 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t576\t0.0\t2\t0 555 21\n",
      "26\t45657\t0.0\t2\t44714 928 15\n",
      "27\t240\t0.0\t2\t0 231 9\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46461 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t376\t0.0\t2\t0 368 8\n",
      "26\t45816\t0.0\t2\t44806 958 52\n",
      "27\t254\t0.0\t2\t0 245 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s715_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s715_212_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s715_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s715_212_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s715_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s715_212_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s715_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s715_212_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,877 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.41 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,877\n",
      "  Read 1 with adapter:                  53,849 (99.9%)\n",
      "  Read 2 with adapter:                  53,823 (99.9%)\n",
      "Pairs written (passing filters):        53,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,782,476 bp\n",
      "  Read 1:    15,893,005 bp\n",
      "  Read 2:    15,889,471 bp\n",
      "Total written (filtered):     28,983,548 bp (91.2%)\n",
      "  Read 1:    14,493,274 bp\n",
      "  Read 2:    14,490,274 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53849 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t680\t0.0\t2\t0 657 23\n",
      "26\t52798\t0.0\t2\t51648 1128 22\n",
      "27\t313\t0.0\t2\t0 307 6\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t416\t0.0\t2\t0 406 10\n",
      "26\t53137\t0.0\t2\t52118 992 27\n",
      "27\t249\t0.0\t2\t0 244 5\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s716_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s716_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s716_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s716_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s716_65_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s716_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s716_65_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s716_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,850 reads  @     80.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.18 s (81 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,850\n",
      "  Read 1 with adapter:                  51,824 (99.9%)\n",
      "  Read 2 with adapter:                  51,802 (99.9%)\n",
      "Pairs written (passing filters):        51,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,589,103 bp\n",
      "  Read 1:    15,294,084 bp\n",
      "  Read 2:    15,295,019 bp\n",
      "Total written (filtered):     27,895,360 bp (91.2%)\n",
      "  Read 1:    13,947,040 bp\n",
      "  Read 2:    13,948,320 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51824 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t657\t0.0\t2\t0 635 22\n",
      "26\t50819\t0.0\t2\t49703 1102 14\n",
      "27\t267\t0.0\t2\t0 263 4\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t404\t0.0\t2\t0 393 11\n",
      "26\t51106\t0.0\t2\t50125 949 32\n",
      "27\t273\t0.0\t2\t0 261 12\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s717_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s717_214_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s717_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s717_214_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s717_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s717_214_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s717_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s717_214_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,602 reads  @     75.6 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.38 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,602\n",
      "  Read 1 with adapter:                  44,578 (99.9%)\n",
      "  Read 2 with adapter:                  44,231 (99.2%)\n",
      "Pairs written (passing filters):        44,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,308,334 bp\n",
      "  Read 1:    13,156,377 bp\n",
      "  Read 2:    13,151,957 bp\n",
      "Total written (filtered):     23,999,791 bp (91.2%)\n",
      "  Read 1:    11,997,548 bp\n",
      "  Read 2:    12,002,243 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44578 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t436\t0.0\t2\t0 419 17\n",
      "26\t43891\t0.0\t2\t42913 964 14\n",
      "27\t197\t0.0\t2\t0 195 2\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t500\t0.0\t2\t0 484 16\n",
      "26\t43464\t0.0\t2\t42581 850 33\n",
      "27\t226\t0.0\t2\t0 217 9\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s718_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s718_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s718_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s718_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s718_67_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s718_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s718_67_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s718_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,029 reads  @     79.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.90 s (79 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,029\n",
      "  Read 1 with adapter:                  48,999 (99.9%)\n",
      "  Read 2 with adapter:                  48,975 (99.9%)\n",
      "Pairs written (passing filters):        49,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,923,139 bp\n",
      "  Read 1:    14,462,316 bp\n",
      "  Read 2:    14,460,823 bp\n",
      "Total written (filtered):     26,376,162 bp (91.2%)\n",
      "  Read 1:    13,188,558 bp\n",
      "  Read 2:    13,187,604 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t466\t0.0\t2\t0 441 25\n",
      "26\t48243\t0.0\t2\t47223 1007 13\n",
      "27\t238\t0.0\t2\t0 233 5\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48975 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t411\t0.0\t2\t0 404 7\n",
      "26\t48261\t0.0\t2\t47201 1020 40\n",
      "27\t282\t0.0\t2\t0 278 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s719_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s719_216_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s719_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s719_216_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s719_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s719_216_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s719_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s719_216_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,208 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.58 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,208\n",
      "  Read 1 with adapter:                  43,179 (99.9%)\n",
      "  Read 2 with adapter:                  43,157 (99.9%)\n",
      "Pairs written (passing filters):        43,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,485,094 bp\n",
      "  Read 1:    12,744,871 bp\n",
      "  Read 2:    12,740,223 bp\n",
      "Total written (filtered):     23,240,632 bp (91.2%)\n",
      "  Read 1:    11,622,385 bp\n",
      "  Read 2:    11,618,247 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t394\t0.0\t2\t0 376 18\n",
      "26\t42529\t0.0\t2\t41662 860 7\n",
      "27\t202\t0.0\t2\t0 193 9\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t340\t0.0\t2\t0 329 11\n",
      "26\t42556\t0.0\t2\t41741 800 15\n",
      "27\t236\t0.0\t2\t0 226 10\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s720_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s720_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s720_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s720_217_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s720_69_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s720_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s720_69_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s720_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,945 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.11 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,945\n",
      "  Read 1 with adapter:                  40,924 (99.9%)\n",
      "  Read 2 with adapter:                  40,922 (99.9%)\n",
      "Pairs written (passing filters):        40,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,155,675 bp\n",
      "  Read 1:    12,077,350 bp\n",
      "  Read 2:    12,078,325 bp\n",
      "Total written (filtered):     22,027,904 bp (91.2%)\n",
      "  Read 1:    11,013,476 bp\n",
      "  Read 2:    11,014,428 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40924 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t377\t0.0\t2\t0 363 14\n",
      "26\t40296\t0.0\t2\t39378 906 12\n",
      "27\t215\t0.0\t2\t0 209 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40922 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t312\t0.0\t2\t0 303 9\n",
      "26\t40369\t0.0\t2\t39424 914 31\n",
      "27\t225\t0.0\t2\t0 214 11\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s721_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s721_218_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s721_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s721_218_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s721_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s721_218_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s721_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s721_218_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,672 reads  @     82.1 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.08 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,672\n",
      "  Read 1 with adapter:                  49,642 (99.9%)\n",
      "  Read 2 with adapter:                  49,644 (99.9%)\n",
      "Pairs written (passing filters):        49,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,303,567 bp\n",
      "  Read 1:    14,651,299 bp\n",
      "  Read 2:    14,652,268 bp\n",
      "Total written (filtered):     26,722,647 bp (91.2%)\n",
      "  Read 1:    13,360,841 bp\n",
      "  Read 2:    13,361,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49642 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t495\t0.0\t2\t0 478 17\n",
      "26\t48849\t0.0\t2\t47872 967 10\n",
      "27\t235\t0.0\t2\t0 224 11\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t517\t0.0\t2\t0 504 13\n",
      "26\t48835\t0.0\t2\t47879 926 30\n",
      "27\t261\t0.0\t2\t0 255 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s722_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s722_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s722_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s722_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s722_71_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s722_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s722_71_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s722_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,945 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.30 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,945\n",
      "  Read 1 with adapter:                  52,902 (99.9%)\n",
      "  Read 2 with adapter:                  52,887 (99.9%)\n",
      "Pairs written (passing filters):        52,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,228,276 bp\n",
      "  Read 1:    15,616,692 bp\n",
      "  Read 2:    15,611,584 bp\n",
      "Total written (filtered):     28,478,217 bp (91.2%)\n",
      "  Read 1:    14,241,469 bp\n",
      "  Read 2:    14,236,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52902 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t503\t0.0\t2\t0 480 23\n",
      "26\t52088\t0.0\t2\t51000 1071 17\n",
      "27\t260\t0.0\t2\t0 254 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t485\t0.0\t2\t0 474 11\n",
      "26\t52088\t0.0\t2\t51081 981 26\n",
      "27\t273\t0.0\t2\t0 267 6\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s723_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s723_220_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s723_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s723_220_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s723_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s723_220_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s723_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s723_220_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,858 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.55 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,858\n",
      "  Read 1 with adapter:                  52,804 (99.9%)\n",
      "  Read 2 with adapter:                  52,766 (99.8%)\n",
      "Pairs written (passing filters):        52,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,174,039 bp\n",
      "  Read 1:    15,586,838 bp\n",
      "  Read 2:    15,587,201 bp\n",
      "Total written (filtered):     28,429,576 bp (91.2%)\n",
      "  Read 1:    14,214,092 bp\n",
      "  Read 2:    14,215,484 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52804 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t465\t0.0\t2\t0 441 24\n",
      "26\t51968\t0.0\t2\t50932 1026 10\n",
      "27\t315\t0.0\t2\t0 304 11\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t395\t0.0\t2\t0 387 8\n",
      "26\t52140\t0.0\t2\t51168 952 20\n",
      "27\t210\t0.0\t2\t0 204 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s724_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s724_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s724_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s724_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s724_73_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s724_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s724_73_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s724_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,419 reads  @     81.9 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.72 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,419\n",
      "  Read 1 with adapter:                  45,401 (100.0%)\n",
      "  Read 2 with adapter:                  45,387 (99.9%)\n",
      "Pairs written (passing filters):        45,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,795,801 bp\n",
      "  Read 1:    13,397,705 bp\n",
      "  Read 2:    13,398,096 bp\n",
      "Total written (filtered):     24,435,555 bp (91.2%)\n",
      "  Read 1:    12,217,377 bp\n",
      "  Read 2:    12,218,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t408\t0.0\t2\t0 398 10\n",
      "26\t44650\t0.0\t2\t43776 864 10\n",
      "27\t300\t0.0\t2\t0 292 8\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t362\t0.0\t2\t0 356 6\n",
      "26\t44773\t0.0\t2\t43947 804 22\n",
      "27\t222\t0.0\t2\t0 215 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s725_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s725_222_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s725_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s725_222_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s725_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s725_222_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s725_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s725_222_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,348 reads  @     93.4 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.96 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,348\n",
      "  Read 1 with adapter:                  42,322 (99.9%)\n",
      "  Read 2 with adapter:                  41,871 (98.9%)\n",
      "Pairs written (passing filters):        42,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,975,983 bp\n",
      "  Read 1:    12,491,004 bp\n",
      "  Read 2:    12,484,979 bp\n",
      "Total written (filtered):     22,787,307 bp (91.2%)\n",
      "  Read 1:    11,390,783 bp\n",
      "  Read 2:    11,396,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t446\t0.0\t2\t0 431 15\n",
      "26\t41578\t0.0\t2\t40732 835 11\n",
      "27\t213\t0.0\t2\t0 205 8\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41871 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t516\t0.0\t2\t0 498 18\n",
      "26\t41034\t0.0\t2\t40045 941 48\n",
      "27\t237\t0.0\t2\t0 229 8\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s726_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s726_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s726_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s726_223_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s726_75_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s726_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s726_75_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s726_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,942 reads  @     98.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.41 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,942\n",
      "  Read 1 with adapter:                  44,904 (99.9%)\n",
      "  Read 2 with adapter:                  44,780 (99.6%)\n",
      "Pairs written (passing filters):        44,942 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,506,457 bp\n",
      "  Read 1:    13,252,121 bp\n",
      "  Read 2:    13,254,336 bp\n",
      "Total written (filtered):     24,174,959 bp (91.2%)\n",
      "  Read 1:    12,084,787 bp\n",
      "  Read 2:    12,090,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44904 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t433\t0.0\t2\t0 413 20\n",
      "26\t44182\t0.0\t2\t43364 809 9\n",
      "27\t251\t0.0\t2\t0 245 6\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t345\t0.0\t2\t0 338 7\n",
      "26\t44181\t0.0\t2\t43287 869 25\n",
      "27\t231\t0.0\t2\t0 226 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s727_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s727_224_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s727_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s727_224_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s727_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s727_224_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s727_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s727_224_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        38,897 reads  @     93.1 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.62 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,897\n",
      "  Read 1 with adapter:                  38,871 (99.9%)\n",
      "  Read 2 with adapter:                  38,708 (99.5%)\n",
      "Pairs written (passing filters):        38,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,924,531 bp\n",
      "  Read 1:    11,470,908 bp\n",
      "  Read 2:    11,453,623 bp\n",
      "Total written (filtered):     20,907,730 bp (91.2%)\n",
      "  Read 1:    10,460,386 bp\n",
      "  Read 2:    10,447,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38871 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t400\t0.0\t2\t0 386 14\n",
      "26\t38183\t0.0\t2\t37409 764 10\n",
      "27\t232\t0.0\t2\t0 226 6\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38708 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t317\t0.0\t2\t0 315 2\n",
      "26\t38175\t0.0\t2\t37419 739 17\n",
      "27\t196\t0.0\t2\t0 194 2\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s728_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s728_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s728_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s728_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s728_77_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s728_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s728_77_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s728_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,592 reads  @     93.4 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.70 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,592\n",
      "  Read 1 with adapter:                  39,542 (99.9%)\n",
      "  Read 2 with adapter:                  39,555 (99.9%)\n",
      "Pairs written (passing filters):        39,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,352,261 bp\n",
      "  Read 1:    11,672,917 bp\n",
      "  Read 2:    11,679,344 bp\n",
      "Total written (filtered):     21,295,944 bp (91.2%)\n",
      "  Read 1:    10,644,977 bp\n",
      "  Read 2:    10,650,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39542 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t375\t0.0\t2\t0 362 13\n",
      "26\t38923\t0.0\t2\t38150 759 14\n",
      "27\t193\t0.0\t2\t0 190 3\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39555 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t264\t0.0\t2\t0 254 10\n",
      "26\t39074\t0.0\t2\t38303 737 34\n",
      "27\t191\t0.0\t2\t0 185 6\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s729_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s729_226_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s729_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s729_226_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s729_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s729_226_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s729_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s729_226_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,236 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.95 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,236\n",
      "  Read 1 with adapter:                  44,180 (99.9%)\n",
      "  Read 2 with adapter:                  44,125 (99.7%)\n",
      "Pairs written (passing filters):        44,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,073,514 bp\n",
      "  Read 1:    13,038,763 bp\n",
      "  Read 2:    13,034,751 bp\n",
      "Total written (filtered):     23,778,008 bp (91.2%)\n",
      "  Read 1:    11,890,235 bp\n",
      "  Read 2:    11,887,773 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t449\t0.0\t2\t0 433 16\n",
      "26\t43441\t0.0\t2\t42546 890 5\n",
      "27\t215\t0.0\t2\t0 211 4\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t503\t0.0\t2\t0 489 14\n",
      "26\t43354\t0.0\t2\t42530 812 12\n",
      "27\t201\t0.0\t2\t0 198 3\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s730_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s730_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s730_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s730_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s730_79_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s730_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s730_79_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s730_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,942 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.48 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,942\n",
      "  Read 1 with adapter:                  41,878 (99.8%)\n",
      "  Read 2 with adapter:                  41,733 (99.5%)\n",
      "Pairs written (passing filters):        41,942 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,700,416 bp\n",
      "  Read 1:    12,362,361 bp\n",
      "  Read 2:    12,338,055 bp\n",
      "Total written (filtered):     22,526,889 bp (91.2%)\n",
      "  Read 1:    11,273,818 bp\n",
      "  Read 2:    11,253,071 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41878 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t440\t0.0\t2\t0 429 11\n",
      "26\t41241\t0.0\t2\t40440 791 10\n",
      "27\t143\t0.0\t2\t0 139 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t415\t0.0\t2\t0 404 11\n",
      "26\t41007\t0.0\t2\t40225 763 19\n",
      "27\t233\t0.0\t2\t0 226 7\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s731_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s731_228_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s731_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s731_228_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s731_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s731_228_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s731_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s731_228_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,369 reads  @     82.2 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.90 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,369\n",
      "  Read 1 with adapter:                  47,339 (99.9%)\n",
      "  Read 2 with adapter:                  47,316 (99.9%)\n",
      "Pairs written (passing filters):        47,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,943,875 bp\n",
      "  Read 1:    13,972,792 bp\n",
      "  Read 2:    13,971,083 bp\n",
      "Total written (filtered):     25,483,286 bp (91.2%)\n",
      "  Read 1:    12,742,280 bp\n",
      "  Read 2:    12,741,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47339 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t525\t0.0\t2\t0 506 19\n",
      "26\t46529\t0.0\t2\t45643 877 9\n",
      "27\t239\t0.0\t2\t0 232 7\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47316 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t350\t0.0\t2\t0 345 5\n",
      "26\t46723\t0.0\t2\t45845 865 13\n",
      "27\t219\t0.0\t2\t0 216 3\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s732_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s732_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s732_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s732_229_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s732_81_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s732_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s732_81_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s732_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,520 reads  @     94.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 3.74 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,520\n",
      "  Read 1 with adapter:                  39,491 (99.9%)\n",
      "  Read 2 with adapter:                  39,475 (99.9%)\n",
      "Pairs written (passing filters):        39,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,314,793 bp\n",
      "  Read 1:    11,656,813 bp\n",
      "  Read 2:    11,657,980 bp\n",
      "Total written (filtered):     21,261,874 bp (91.2%)\n",
      "  Read 1:    10,630,178 bp\n",
      "  Read 2:    10,631,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t396\t0.0\t2\t0 379 17\n",
      "26\t38829\t0.0\t2\t38041 776 12\n",
      "27\t203\t0.0\t2\t0 198 5\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39475 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t310\t0.0\t2\t0 302 8\n",
      "26\t38912\t0.0\t2\t38147 741 24\n",
      "27\t198\t0.0\t2\t0 195 3\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s733_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s733_230_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s733_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s733_230_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s733_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s733_230_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s733_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s733_230_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,975 reads  @     94.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 4.43 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,975\n",
      "  Read 1 with adapter:                  46,948 (99.9%)\n",
      "  Read 2 with adapter:                  46,635 (99.3%)\n",
      "Pairs written (passing filters):        46,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,709,720 bp\n",
      "  Read 1:    13,857,436 bp\n",
      "  Read 2:    13,852,284 bp\n",
      "Total written (filtered):     25,277,025 bp (91.2%)\n",
      "  Read 1:    12,636,982 bp\n",
      "  Read 2:    12,640,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t435\t0.0\t2\t0 424 11\n",
      "26\t46246\t0.0\t2\t45349 885 12\n",
      "27\t205\t0.0\t2\t0 198 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46635 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t495\t0.0\t2\t0 484 11\n",
      "26\t45858\t0.0\t2\t44880 941 37\n",
      "27\t246\t0.0\t2\t0 241 5\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s734_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s734_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s734_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s734_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s734_83_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s734_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s734_83_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s734_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,875 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.23 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,875\n",
      "  Read 1 with adapter:                  48,850 (99.9%)\n",
      "  Read 2 with adapter:                  48,793 (99.8%)\n",
      "Pairs written (passing filters):        48,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,833,530 bp\n",
      "  Read 1:    14,417,843 bp\n",
      "  Read 2:    14,415,687 bp\n",
      "Total written (filtered):     26,295,165 bp (91.2%)\n",
      "  Read 1:    13,147,913 bp\n",
      "  Read 2:    13,147,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t458\t0.0\t2\t0 443 15\n",
      "26\t48048\t0.0\t2\t47112 923 13\n",
      "27\t284\t0.0\t2\t0 276 8\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t434\t0.0\t2\t0 428 6\n",
      "26\t48063\t0.0\t2\t47049 986 28\n",
      "27\t265\t0.0\t2\t0 256 9\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s735_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s735_232_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s735_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s735_232_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s735_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s735_232_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s735_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s735_232_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,259 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.88 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,259\n",
      "  Read 1 with adapter:                  43,235 (99.9%)\n",
      "  Read 2 with adapter:                  43,184 (99.8%)\n",
      "Pairs written (passing filters):        43,259 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,516,390 bp\n",
      "  Read 1:    12,760,348 bp\n",
      "  Read 2:    12,756,042 bp\n",
      "Total written (filtered):     23,269,644 bp (91.2%)\n",
      "  Read 1:    11,636,326 bp\n",
      "  Read 2:    11,633,318 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t364\t0.0\t2\t0 343 21\n",
      "26\t42588\t0.0\t2\t41701 874 13\n",
      "27\t190\t0.0\t2\t0 187 3\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t286\t0.0\t2\t0 280 6\n",
      "26\t42647\t0.0\t2\t41812 817 18\n",
      "27\t224\t0.0\t2\t0 219 5\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s736_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s736_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s736_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s736_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s736_85_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s736_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s736_85_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s736_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,394 reads  @     82.2 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.73 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,394\n",
      "  Read 1 with adapter:                  45,363 (99.9%)\n",
      "  Read 2 with adapter:                  45,365 (99.9%)\n",
      "Pairs written (passing filters):        45,394 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,780,145 bp\n",
      "  Read 1:    13,389,979 bp\n",
      "  Read 2:    13,390,166 bp\n",
      "Total written (filtered):     24,421,294 bp (91.2%)\n",
      "  Read 1:    12,210,535 bp\n",
      "  Read 2:    12,210,759 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t373\t0.0\t2\t0 357 16\n",
      "26\t44638\t0.0\t2\t43689 935 14\n",
      "27\t233\t0.0\t2\t0 227 6\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t313\t0.0\t2\t0 300 13\n",
      "26\t44781\t0.0\t2\t43800 938 43\n",
      "27\t248\t0.0\t2\t0 243 5\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s737_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s737_234_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s737_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s737_234_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s737_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s737_234_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s737_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s737_234_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        55,119 reads  @    103.9 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.73 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,119\n",
      "  Read 1 with adapter:                  55,073 (99.9%)\n",
      "  Read 2 with adapter:                  54,980 (99.7%)\n",
      "Pairs written (passing filters):        55,119 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,496,518 bp\n",
      "  Read 1:    16,257,969 bp\n",
      "  Read 2:    16,238,549 bp\n",
      "Total written (filtered):     29,635,516 bp (91.2%)\n",
      "  Read 1:    14,826,152 bp\n",
      "  Read 2:    14,809,364 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55073 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t477\t0.0\t2\t0 460 17\n",
      "26\t54204\t0.0\t2\t53155 1036 13\n",
      "27\t268\t0.0\t2\t0 256 12\n",
      "28\t94\t0.0\t2\t0 0 94\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t658\t0.0\t2\t0 650 8\n",
      "26\t53913\t0.0\t2\t52740 1140 33\n",
      "27\t343\t0.0\t2\t0 336 7\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s738_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s738_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s738_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s738_235_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s738_87_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s738_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s738_87_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s738_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        54,191 reads  @     94.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.11 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,191\n",
      "  Read 1 with adapter:                  54,158 (99.9%)\n",
      "  Read 2 with adapter:                  54,080 (99.8%)\n",
      "Pairs written (passing filters):        54,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,958,748 bp\n",
      "  Read 1:    15,985,751 bp\n",
      "  Read 2:    15,972,997 bp\n",
      "Total written (filtered):     29,144,798 bp (91.2%)\n",
      "  Read 1:    14,577,824 bp\n",
      "  Read 2:    14,566,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t477\t0.0\t2\t0 461 16\n",
      "26\t53347\t0.0\t2\t52350 985 12\n",
      "27\t268\t0.0\t2\t0 257 11\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t525\t0.0\t2\t0 510 15\n",
      "26\t53113\t0.0\t2\t52102 978 33\n",
      "27\t300\t0.0\t2\t0 295 5\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s739_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s739_236_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s739_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s739_236_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s739_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s739_236_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s739_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s739_236_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           834 reads  @    163.3 µs/read;   0.37 M reads/minute\n",
      "Finished in 0.14 s (166 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                834\n",
      "  Read 1 with adapter:                     830 (99.5%)\n",
      "  Read 2 with adapter:                     830 (99.5%)\n",
      "Pairs written (passing filters):           834 (100.0%)\n",
      "\n",
      "Total basepairs processed:       491,770 bp\n",
      "  Read 1:       246,028 bp\n",
      "  Read 2:       245,742 bp\n",
      "Total written (filtered):        448,624 bp (91.2%)\n",
      "  Read 1:       224,456 bp\n",
      "  Read 2:       224,168 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 830 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t11\t0.0\t2\t0 9 2\n",
      "26\t816\t0.0\t2\t793 23\n",
      "27\t3\t0.0\t2\t0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 830 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t9\t0.0\t2\t0 8 1\n",
      "26\t819\t0.0\t2\t801 17 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s740_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s740_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s740_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s740_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s740_89_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s740_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s740_89_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s740_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,678 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.78 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,678\n",
      "  Read 1 with adapter:                  45,657 (100.0%)\n",
      "  Read 2 with adapter:                  45,641 (99.9%)\n",
      "Pairs written (passing filters):        45,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,948,443 bp\n",
      "  Read 1:    13,473,732 bp\n",
      "  Read 2:    13,474,711 bp\n",
      "Total written (filtered):     24,575,067 bp (91.2%)\n",
      "  Read 1:    12,286,861 bp\n",
      "  Read 2:    12,288,206 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t449\t0.0\t2\t0 433 16\n",
      "26\t44938\t0.0\t2\t43955 964 19\n",
      "27\t210\t0.0\t2\t0 203 7\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t397\t0.0\t2\t0 391 6\n",
      "26\t44978\t0.0\t2\t44173 779 26\n",
      "27\t232\t0.0\t2\t0 225 7\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s741_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s741_238_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s741_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s741_238_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s741_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s741_238_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s741_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s741_238_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,558 reads  @     82.1 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.48 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,558\n",
      "  Read 1 with adapter:                  54,537 (100.0%)\n",
      "  Read 2 with adapter:                  54,106 (99.2%)\n",
      "Pairs written (passing filters):        54,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,181,704 bp\n",
      "  Read 1:    16,094,091 bp\n",
      "  Read 2:    16,087,613 bp\n",
      "Total written (filtered):     29,357,466 bp (91.2%)\n",
      "  Read 1:    14,676,352 bp\n",
      "  Read 2:    14,681,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54537 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t497\t0.0\t2\t0 481 16\n",
      "26\t53745\t0.0\t2\t52761 967 17\n",
      "27\t220\t0.0\t2\t0 215 5\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t529\t0.0\t2\t0 516 13\n",
      "26\t53231\t0.0\t2\t52188 999 44\n",
      "27\t304\t0.0\t2\t0 299 5\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s742_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s742_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s742_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s742_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s742_91_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s742_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s742_91_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s742_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,452 reads  @     80.7 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.40 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,452\n",
      "  Read 1 with adapter:                  54,423 (99.9%)\n",
      "  Read 2 with adapter:                  54,385 (99.9%)\n",
      "Pairs written (passing filters):        54,452 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,123,636 bp\n",
      "  Read 1:    16,062,675 bp\n",
      "  Read 2:    16,060,961 bp\n",
      "Total written (filtered):     29,295,021 bp (91.2%)\n",
      "  Read 1:    14,647,899 bp\n",
      "  Read 2:    14,647,122 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t499\t0.0\t2\t0 479 20\n",
      "26\t53632\t0.0\t2\t52563 1056 13\n",
      "27\t223\t0.0\t2\t0 215 8\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t417\t0.0\t2\t0 409 8\n",
      "26\t53674\t0.0\t2\t52581 1052 41\n",
      "27\t262\t0.0\t2\t0 256 6\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s743_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s743_240_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s743_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s743_240_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s743_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s743_240_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s743_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s743_240_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,955 reads  @     80.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.62 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,955\n",
      "  Read 1 with adapter:                  44,944 (100.0%)\n",
      "  Read 2 with adapter:                  44,896 (99.9%)\n",
      "Pairs written (passing filters):        44,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,516,490 bp\n",
      "  Read 1:    13,261,046 bp\n",
      "  Read 2:    13,255,444 bp\n",
      "Total written (filtered):     24,180,980 bp (91.2%)\n",
      "  Read 1:    12,092,776 bp\n",
      "  Read 2:    12,088,204 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44944 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t489\t0.0\t2\t0 473 16\n",
      "26\t44213\t0.0\t2\t43272 923 18\n",
      "27\t193\t0.0\t2\t0 190 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44896 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t318\t0.0\t2\t0 311 7\n",
      "26\t44295\t0.0\t2\t43444 817 34\n",
      "27\t268\t0.0\t2\t0 262 6\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s744_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s744_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s744_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s744_241_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s744_93_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s744_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s744_93_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s744_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,645 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.94 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,645\n",
      "  Read 1 with adapter:                  48,621 (100.0%)\n",
      "  Read 2 with adapter:                  48,597 (99.9%)\n",
      "Pairs written (passing filters):        48,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,698,489 bp\n",
      "  Read 1:    14,349,549 bp\n",
      "  Read 2:    14,348,940 bp\n",
      "Total written (filtered):     26,171,130 bp (91.2%)\n",
      "  Read 1:    13,085,604 bp\n",
      "  Read 2:    13,085,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48621 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t469\t0.0\t2\t0 456 13\n",
      "26\t47877\t0.0\t2\t46888 972 17\n",
      "27\t210\t0.0\t2\t0 206 4\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t366\t0.0\t2\t0 356 10\n",
      "26\t47946\t0.0\t2\t46905 1008 33\n",
      "27\t260\t0.0\t2\t0 253 7\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s745_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s745_242_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s745_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s745_242_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s745_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s745_242_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s745_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s745_242_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,607 reads  @     83.7 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.57 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,607\n",
      "  Read 1 with adapter:                  54,584 (100.0%)\n",
      "  Read 2 with adapter:                  54,544 (99.9%)\n",
      "Pairs written (passing filters):        54,607 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,211,937 bp\n",
      "  Read 1:    16,108,406 bp\n",
      "  Read 2:    16,103,531 bp\n",
      "Total written (filtered):     29,375,141 bp (91.2%)\n",
      "  Read 1:    14,689,467 bp\n",
      "  Read 2:    14,685,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54584 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t554\t0.0\t2\t0 530 24\n",
      "26\t53711\t0.0\t2\t52561 1133 17\n",
      "27\t237\t0.0\t2\t0 228 9\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t577\t0.0\t2\t0 559 18\n",
      "26\t53612\t0.0\t2\t52449 1126 37\n",
      "27\t304\t0.0\t2\t0 298 6\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s746_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s746_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s746_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s746_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s746_95_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s746_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s746_95_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s746_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,799 reads  @     78.9 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.17 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,799\n",
      "  Read 1 with adapter:                  52,772 (99.9%)\n",
      "  Read 2 with adapter:                  52,688 (99.8%)\n",
      "Pairs written (passing filters):        52,799 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,134,474 bp\n",
      "  Read 1:    15,575,012 bp\n",
      "  Read 2:    15,559,462 bp\n",
      "Total written (filtered):     28,392,919 bp (91.2%)\n",
      "  Read 1:    14,203,187 bp\n",
      "  Read 2:    14,189,732 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t476\t0.0\t2\t0 455 21\n",
      "26\t52023\t0.0\t2\t50957 1046 20\n",
      "27\t209\t0.0\t2\t0 208 1\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52688 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t480\t0.0\t2\t0 469 11\n",
      "26\t51841\t0.0\t2\t50781 1024 36\n",
      "27\t324\t0.0\t2\t0 316 8\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s747_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s747_244_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s747_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s747_244_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s747_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s747_244_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s747_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s747_244_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,221 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.41 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,221\n",
      "  Read 1 with adapter:                  54,196 (100.0%)\n",
      "  Read 2 with adapter:                  54,162 (99.9%)\n",
      "Pairs written (passing filters):        54,221 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,985,361 bp\n",
      "  Read 1:    15,994,474 bp\n",
      "  Read 2:    15,990,887 bp\n",
      "Total written (filtered):     29,168,486 bp (91.2%)\n",
      "  Read 1:    14,585,604 bp\n",
      "  Read 2:    14,582,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54196 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t516\t0.0\t2\t0 498 18\n",
      "26\t53384\t0.0\t2\t52280 1084 20\n",
      "27\t210\t0.0\t2\t0 207 3\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54162 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t431\t0.0\t2\t0 423 8\n",
      "26\t53450\t0.0\t2\t52531 900 19\n",
      "27\t250\t0.0\t2\t0 245 5\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s748_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s748_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s748_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s748_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s748_97_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s748_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s748_97_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s748_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,902 reads  @     84.7 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.81 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,902\n",
      "  Read 1 with adapter:                  44,832 (99.8%)\n",
      "  Read 2 with adapter:                  44,863 (99.9%)\n",
      "Pairs written (passing filters):        44,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,479,553 bp\n",
      "  Read 1:    13,233,896 bp\n",
      "  Read 2:    13,245,657 bp\n",
      "Total written (filtered):     24,147,816 bp (91.2%)\n",
      "  Read 1:    12,068,482 bp\n",
      "  Read 2:    12,079,334 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t449\t0.0\t2\t0 435 14\n",
      "26\t44128\t0.0\t2\t43123 986 19\n",
      "27\t187\t0.0\t2\t0 181 6\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t334\t0.0\t2\t0 326 8\n",
      "26\t44285\t0.0\t2\t43552 715 18\n",
      "27\t213\t0.0\t2\t0 207 6\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s749_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s749_246_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s749_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s749_246_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s749_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s749_246_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s749_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s749_246_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,451 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.93 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,451\n",
      "  Read 1 with adapter:                  64,423 (100.0%)\n",
      "  Read 2 with adapter:                  63,863 (99.1%)\n",
      "Pairs written (passing filters):        64,451 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,015,362 bp\n",
      "  Read 1:    19,012,437 bp\n",
      "  Read 2:    19,002,925 bp\n",
      "Total written (filtered):     34,680,814 bp (91.2%)\n",
      "  Read 1:    17,337,893 bp\n",
      "  Read 2:    17,342,921 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t778\t0.0\t2\t0 746 32\n",
      "26\t63252\t0.0\t2\t61661 1565 26\n",
      "27\t294\t0.0\t2\t0 280 14\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63863 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t768\t0.0\t2\t0 751 17\n",
      "26\t62663\t0.0\t2\t61186 1442 35\n",
      "27\t382\t0.0\t2\t0 377 5\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s751_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s751_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s751_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s751_247_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s751_99_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s751_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s751_99_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s751_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,748 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.51 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,748\n",
      "  Read 1 with adapter:                  42,733 (100.0%)\n",
      "  Read 2 with adapter:                  42,713 (99.9%)\n",
      "Pairs written (passing filters):        42,748 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,218,762 bp\n",
      "  Read 1:    12,609,810 bp\n",
      "  Read 2:    12,608,952 bp\n",
      "Total written (filtered):     22,997,568 bp (91.2%)\n",
      "  Read 1:    11,498,947 bp\n",
      "  Read 2:    11,498,621 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t508\t0.0\t2\t0 488 20\n",
      "26\t41938\t0.0\t2\t41144 785 9\n",
      "27\t189\t0.0\t2\t0 186 3\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42713 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t438\t0.0\t2\t0 428 10\n",
      "26\t42015\t0.0\t2\t41268 724 23\n",
      "27\t201\t0.0\t2\t0 194 7\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s752_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s752_248_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s752_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s752_248_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s752_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s752_248_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s752_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s752_248_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,037 reads  @     77.8 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.27 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,037\n",
      "  Read 1 with adapter:                  42,016 (100.0%)\n",
      "  Read 2 with adapter:                  42,004 (99.9%)\n",
      "Pairs written (passing filters):        42,037 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,800,091 bp\n",
      "  Read 1:    12,400,666 bp\n",
      "  Read 2:    12,399,425 bp\n",
      "Total written (filtered):     22,615,928 bp (91.2%)\n",
      "  Read 1:    11,308,498 bp\n",
      "  Read 2:    11,307,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t540\t0.0\t2\t0 514 26\n",
      "26\t41192\t0.0\t2\t40120 1056 16\n",
      "27\t196\t0.0\t2\t0 190 6\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42004 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t366\t0.0\t2\t0 351 15\n",
      "26\t41384\t0.0\t2\t40423 928 33\n",
      "27\t211\t0.0\t2\t0 202 9\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s753_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s753_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s753_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s753_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s753_101_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s753_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s753_101_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s753_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,132 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.78 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,132\n",
      "  Read 1 with adapter:                  49,109 (100.0%)\n",
      "  Read 2 with adapter:                  49,093 (99.9%)\n",
      "Pairs written (passing filters):        49,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,985,043 bp\n",
      "  Read 1:    14,493,253 bp\n",
      "  Read 2:    14,491,790 bp\n",
      "Total written (filtered):     26,432,198 bp (91.2%)\n",
      "  Read 1:    13,216,767 bp\n",
      "  Read 2:    13,215,431 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49109 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t562\t0.0\t2\t0 539 23\n",
      "26\t48292\t0.0\t2\t47384 891 17\n",
      "27\t196\t0.0\t2\t0 188 8\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t307\t0.0\t2\t0 295 12\n",
      "26\t48505\t0.0\t2\t47549 930 26\n",
      "27\t270\t0.0\t2\t0 268 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s754_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s754_250_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s754_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s754_250_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s754_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s754_250_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s754_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s754_250_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,338 reads  @     73.2 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.54 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,338\n",
      "  Read 1 with adapter:                  48,315 (100.0%)\n",
      "  Read 2 with adapter:                  48,300 (99.9%)\n",
      "Pairs written (passing filters):        48,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,517,601 bp\n",
      "  Read 1:    14,258,410 bp\n",
      "  Read 2:    14,259,191 bp\n",
      "Total written (filtered):     26,006,114 bp (91.2%)\n",
      "  Read 1:    13,002,623 bp\n",
      "  Read 2:    13,003,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t618\t0.0\t2\t0 595 23\n",
      "26\t47429\t0.0\t2\t46438 978 13\n",
      "27\t209\t0.0\t2\t0 202 7\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t369\t0.0\t2\t0 364 5\n",
      "26\t47615\t0.0\t2\t46734 858 23\n",
      "27\t295\t0.0\t2\t0 288 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s755_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s755_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s755_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s755_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s755_103_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s755_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s755_103_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s755_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,101 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.56 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,101\n",
      "  Read 1 with adapter:                  46,080 (100.0%)\n",
      "  Read 2 with adapter:                  46,058 (99.9%)\n",
      "Pairs written (passing filters):        46,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,193,692 bp\n",
      "  Read 1:    13,597,730 bp\n",
      "  Read 2:    13,595,962 bp\n",
      "Total written (filtered):     24,798,511 bp (91.2%)\n",
      "  Read 1:    12,400,059 bp\n",
      "  Read 2:    12,398,452 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t660\t0.0\t2\t0 631 29\n",
      "26\t45118\t0.0\t2\t44190 916 12\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t481\t0.0\t2\t0 472 9\n",
      "26\t45290\t0.0\t2\t44415 866 9\n",
      "27\t256\t0.0\t2\t0 251 5\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s756_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s756_252_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s756_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s756_252_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s756_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s756_252_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s756_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s756_252_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,040 reads  @     74.0 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.85 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,040\n",
      "  Read 1 with adapter:                  52,016 (100.0%)\n",
      "  Read 2 with adapter:                  51,991 (99.9%)\n",
      "Pairs written (passing filters):        52,038 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,700,163 bp\n",
      "  Read 1:    15,350,508 bp\n",
      "  Read 2:    15,349,655 bp\n",
      "Total written (filtered):     27,996,064 bp (91.2%)\n",
      "  Read 1:    13,998,537 bp\n",
      "  Read 2:    13,997,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t671\t0.0\t2\t0 650 21\n",
      "26\t51039\t0.0\t2\t49989 1026 24\n",
      "27\t226\t0.0\t2\t0 220 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t378\t0.0\t2\t0 371 7\n",
      "26\t51338\t0.0\t2\t50312 1000 26\n",
      "27\t244\t0.0\t2\t0 243 1\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s757_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s757_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s757_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s757_253_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s757_105_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s757_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s757_105_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s757_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,106 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.90 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,106\n",
      "  Read 1 with adapter:                  53,063 (99.9%)\n",
      "  Read 2 with adapter:                  53,060 (99.9%)\n",
      "Pairs written (passing filters):        53,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,329,849 bp\n",
      "  Read 1:    15,665,068 bp\n",
      "  Read 2:    15,664,781 bp\n",
      "Total written (filtered):     28,570,593 bp (91.2%)\n",
      "  Read 1:    14,285,852 bp\n",
      "  Read 2:    14,284,741 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53063 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t703\t0.0\t2\t0 689 14\n",
      "26\t52021\t0.0\t2\t50913 1095 13\n",
      "27\t245\t0.0\t2\t0 242 3\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53060 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t381\t0.0\t2\t0 370 11\n",
      "26\t52326\t0.0\t2\t51337 958 31\n",
      "27\t339\t0.0\t2\t0 329 10\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s758_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s758_254_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s758_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s758_254_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s758_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s758_254_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s758_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s758_254_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,430 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.70 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,430\n",
      "  Read 1 with adapter:                  48,396 (99.9%)\n",
      "  Read 2 with adapter:                  48,389 (99.9%)\n",
      "Pairs written (passing filters):        48,429 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,570,257 bp\n",
      "  Read 1:    14,284,279 bp\n",
      "  Read 2:    14,285,978 bp\n",
      "Total written (filtered):     26,054,137 bp (91.2%)\n",
      "  Read 1:    13,026,412 bp\n",
      "  Read 2:    13,027,725 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48396 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t669\t0.0\t2\t0 635 34\n",
      "26\t47430\t0.0\t2\t46288 1121 21\n",
      "27\t218\t0.0\t2\t0 208 10\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48389 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t376\t0.0\t2\t0 365 11\n",
      "26\t47709\t0.0\t2\t46701 981 27\n",
      "27\t274\t0.0\t2\t0 268 6\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s759_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s759_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s759_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s759_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s759_107_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s759_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s759_107_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s759_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,031 reads  @     71.7 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.45 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,031\n",
      "  Read 1 with adapter:                  47,456 (98.8%)\n",
      "  Read 2 with adapter:                  47,969 (99.9%)\n",
      "Pairs written (passing filters):        48,031 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,269,697 bp\n",
      "  Read 1:    14,106,063 bp\n",
      "  Read 2:    14,163,634 bp\n",
      "Total written (filtered):     25,789,392 bp (91.2%)\n",
      "  Read 1:    12,872,609 bp\n",
      "  Read 2:    12,916,783 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47456 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t566\t0.0\t2\t0 553 13\n",
      "26\t46649\t0.0\t2\t45705 930 14\n",
      "27\t182\t0.0\t2\t0 177 5\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t520\t0.0\t2\t0 506 14\n",
      "26\t47203\t0.0\t2\t46164 1009 30\n",
      "27\t219\t0.0\t2\t0 213 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s760_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s760_256_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s760_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s760_256_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s760_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s760_256_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s760_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s760_256_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,202 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.14 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,202\n",
      "  Read 1 with adapter:                  39,703 (98.8%)\n",
      "  Read 2 with adapter:                  40,158 (99.9%)\n",
      "Pairs written (passing filters):        40,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,661,342 bp\n",
      "  Read 1:    11,804,279 bp\n",
      "  Read 2:    11,857,063 bp\n",
      "Total written (filtered):     21,585,399 bp (91.2%)\n",
      "  Read 1:    10,772,296 bp\n",
      "  Read 2:    10,813,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t445\t0.0\t2\t0 424 21\n",
      "26\t39072\t0.0\t2\t38170 890 12\n",
      "27\t146\t0.0\t2\t0 145 1\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t310\t0.0\t2\t0 298 12\n",
      "26\t39632\t0.0\t2\t38810 786 36\n",
      "27\t198\t0.0\t2\t0 191 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s761_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s761_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s761_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s761_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s761_109_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s761_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s761_109_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s761_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,578 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.68 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,578\n",
      "  Read 1 with adapter:                  49,977 (98.8%)\n",
      "  Read 2 with adapter:                  50,530 (99.9%)\n",
      "Pairs written (passing filters):        50,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,772,031 bp\n",
      "  Read 1:    14,854,019 bp\n",
      "  Read 2:    14,918,012 bp\n",
      "Total written (filtered):     27,159,273 bp (91.2%)\n",
      "  Read 1:    13,554,977 bp\n",
      "  Read 2:    13,604,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49977 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t616\t0.0\t2\t0 586 30\n",
      "26\t49097\t0.0\t2\t48001 1080 16\n",
      "27\t216\t0.0\t2\t0 213 3\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50530 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t363\t0.0\t2\t0 354 9\n",
      "26\t49814\t0.0\t2\t48751 1028 35\n",
      "27\t331\t0.0\t2\t0 325 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s762_258_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s762_258_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s762_258_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s762_258_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,041 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.64 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,041\n",
      "  Read 1 with adapter:                  49,380 (98.7%)\n",
      "  Read 2 with adapter:                  49,994 (99.9%)\n",
      "Pairs written (passing filters):        50,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,450,329 bp\n",
      "  Read 1:    14,689,225 bp\n",
      "  Read 2:    14,761,104 bp\n",
      "Total written (filtered):     26,867,138 bp (91.2%)\n",
      "  Read 1:    13,405,759 bp\n",
      "  Read 2:    13,461,379 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t596\t0.0\t2\t0 578 18\n",
      "26\t48571\t0.0\t2\t47502 1052 17\n",
      "27\t172\t0.0\t2\t0 171 1\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t389\t0.0\t2\t0 383 6\n",
      "26\t49283\t0.0\t2\t48245 1007 31\n",
      "27\t302\t0.0\t2\t0 294 8\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s763_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s763_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s763_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s763_259_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s763_111_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s763_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s763_111_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s763_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,106 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.58 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,106\n",
      "  Read 1 with adapter:                  50,079 (99.9%)\n",
      "  Read 2 with adapter:                  50,040 (99.9%)\n",
      "Pairs written (passing filters):        50,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,557,285 bp\n",
      "  Read 1:    14,781,027 bp\n",
      "  Read 2:    14,776,258 bp\n",
      "Total written (filtered):     26,954,981 bp (91.2%)\n",
      "  Read 1:    13,479,471 bp\n",
      "  Read 2:    13,475,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t682\t0.0\t2\t0 653 29\n",
      "26\t49100\t0.0\t2\t48076 1002 22\n",
      "27\t230\t0.0\t2\t0 224 6\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t540\t0.0\t2\t0 522 18\n",
      "26\t49196\t0.0\t2\t48157 994 45\n",
      "27\t284\t0.0\t2\t0 277 7\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s764_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s764_260_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s764_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s764_260_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s764_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s764_260_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s764_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s764_260_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,329 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.81 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,329\n",
      "  Read 1 with adapter:                  53,305 (100.0%)\n",
      "  Read 2 with adapter:                  53,269 (99.9%)\n",
      "Pairs written (passing filters):        53,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,459,757 bp\n",
      "  Read 1:    15,731,742 bp\n",
      "  Read 2:    15,728,015 bp\n",
      "Total written (filtered):     28,689,519 bp (91.2%)\n",
      "  Read 1:    14,346,283 bp\n",
      "  Read 2:    14,343,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t758\t0.0\t2\t0 741 17\n",
      "26\t52249\t0.0\t2\t51196 1044 9\n",
      "27\t225\t0.0\t2\t0 216 9\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t412\t0.0\t2\t0 403 9\n",
      "26\t52595\t0.0\t2\t51472 1079 44\n",
      "27\t235\t0.0\t2\t0 231 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s765_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s765_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s765_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s765_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s765_113_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s765_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s765_113_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s765_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,525 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.75 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,525\n",
      "  Read 1 with adapter:                  52,499 (100.0%)\n",
      "  Read 2 with adapter:                  52,481 (99.9%)\n",
      "Pairs written (passing filters):        52,525 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,987,749 bp\n",
      "  Read 1:    15,494,588 bp\n",
      "  Read 2:    15,493,161 bp\n",
      "Total written (filtered):     28,258,833 bp (91.2%)\n",
      "  Read 1:    14,130,111 bp\n",
      "  Read 2:    14,128,722 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t740\t0.0\t2\t0 714 26\n",
      "26\t51435\t0.0\t2\t50436 986 13\n",
      "27\t257\t0.0\t2\t0 251 6\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t376\t0.0\t2\t0 364 12\n",
      "26\t51753\t0.0\t2\t50716 1005 32\n",
      "27\t335\t0.0\t2\t0 331 4\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s766_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s766_262_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s766_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s766_262_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s766_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s766_262_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s766_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s766_262_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,846 reads  @     73.9 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.61 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,846\n",
      "  Read 1 with adapter:                  48,826 (100.0%)\n",
      "  Read 2 with adapter:                  48,811 (99.9%)\n",
      "Pairs written (passing filters):        48,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,818,862 bp\n",
      "  Read 1:    14,409,552 bp\n",
      "  Read 2:    14,409,310 bp\n",
      "Total written (filtered):     26,280,748 bp (91.2%)\n",
      "  Read 1:    13,140,451 bp\n",
      "  Read 2:    13,140,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48826 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t633\t0.0\t2\t0 611 22\n",
      "26\t47900\t0.0\t2\t46869 1018 13\n",
      "27\t224\t0.0\t2\t0 212 12\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t354\t0.0\t2\t0 348 6\n",
      "26\t48110\t0.0\t2\t47072 1008 30\n",
      "27\t321\t0.0\t2\t0 314 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s767_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s767_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s767_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s767_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s767_115_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s767_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s767_115_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s767_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,832 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.33 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,832\n",
      "  Read 1 with adapter:                  45,806 (99.9%)\n",
      "  Read 2 with adapter:                  45,783 (99.9%)\n",
      "Pairs written (passing filters):        45,832 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,036,012 bp\n",
      "  Read 1:    13,520,250 bp\n",
      "  Read 2:    13,515,762 bp\n",
      "Total written (filtered):     24,655,377 bp (91.2%)\n",
      "  Read 1:    12,329,705 bp\n",
      "  Read 2:    12,325,672 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t678\t0.0\t2\t0 647 31\n",
      "26\t44814\t0.0\t2\t43838 965 11\n",
      "27\t245\t0.0\t2\t0 239 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t514\t0.0\t2\t0 497 17\n",
      "26\t44966\t0.0\t2\t44118 814 34\n",
      "27\t268\t0.0\t2\t0 264 4\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s768_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s768_264_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s768_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s768_264_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s768_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s768_264_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s768_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s768_264_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,312 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.52 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,312\n",
      "  Read 1 with adapter:                  46,292 (100.0%)\n",
      "  Read 2 with adapter:                  46,269 (99.9%)\n",
      "Pairs written (passing filters):        46,312 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,322,200 bp\n",
      "  Read 1:    13,661,901 bp\n",
      "  Read 2:    13,660,299 bp\n",
      "Total written (filtered):     24,916,256 bp (91.2%)\n",
      "  Read 1:    12,458,781 bp\n",
      "  Read 2:    12,457,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t708\t0.0\t2\t0 680 28\n",
      "26\t45300\t0.0\t2\t44399 886 15\n",
      "27\t212\t0.0\t2\t0 207 5\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t386\t0.0\t2\t0 374 12\n",
      "26\t45637\t0.0\t2\t44771 841 25\n",
      "27\t228\t0.0\t2\t0 224 4\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s769_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s769_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s769_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s769_265_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s769_117_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s769_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s769_117_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s769_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,133 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.65 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,133\n",
      "  Read 1 with adapter:                  49,104 (99.9%)\n",
      "  Read 2 with adapter:                  49,101 (99.9%)\n",
      "Pairs written (passing filters):        49,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,986,277 bp\n",
      "  Read 1:    14,493,511 bp\n",
      "  Read 2:    14,492,766 bp\n",
      "Total written (filtered):     26,433,463 bp (91.2%)\n",
      "  Read 1:    13,217,254 bp\n",
      "  Read 2:    13,216,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49104 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t692\t0.0\t2\t0 660 32\n",
      "26\t48113\t0.0\t2\t47067 1020 26\n",
      "27\t221\t0.0\t2\t0 212 9\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49101 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t357\t0.0\t2\t0 352 5\n",
      "26\t48419\t0.0\t2\t47458 943 18\n",
      "27\t294\t0.0\t2\t0 283 11\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s770_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s770_266_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s770_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s770_266_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s770_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s770_266_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s770_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s770_266_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,330 reads  @     76.0 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.68 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,330\n",
      "  Read 1 with adapter:                  48,308 (100.0%)\n",
      "  Read 2 with adapter:                  48,301 (99.9%)\n",
      "Pairs written (passing filters):        48,330 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,513,972 bp\n",
      "  Read 1:    14,256,643 bp\n",
      "  Read 2:    14,257,329 bp\n",
      "Total written (filtered):     26,002,669 bp (91.2%)\n",
      "  Read 1:    13,001,046 bp\n",
      "  Read 2:    13,001,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48308 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t698\t0.0\t2\t0 673 25\n",
      "26\t47274\t0.0\t2\t46259 999 16\n",
      "27\t269\t0.0\t2\t0 261 8\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t413\t0.0\t2\t0 409 4\n",
      "26\t47557\t0.0\t2\t46628 905 24\n",
      "27\t309\t0.0\t2\t0 300 9\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s771_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s771_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s771_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s771_267_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s771_119_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s771_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s771_119_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s771_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,672 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.79 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,672\n",
      "  Read 1 with adapter:                  47,659 (100.0%)\n",
      "  Read 2 with adapter:                  47,624 (99.9%)\n",
      "Pairs written (passing filters):        47,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,122,164 bp\n",
      "  Read 1:    14,063,063 bp\n",
      "  Read 2:    14,059,101 bp\n",
      "Total written (filtered):     25,645,409 bp (91.2%)\n",
      "  Read 1:    12,824,258 bp\n",
      "  Read 2:    12,821,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t561\t0.0\t2\t0 540 21\n",
      "26\t46829\t0.0\t2\t45884 926 19\n",
      "27\t210\t0.0\t2\t0 206 4\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47624 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t494\t0.0\t2\t0 490 4\n",
      "26\t46864\t0.0\t2\t45943 895 26\n",
      "27\t240\t0.0\t2\t0 235 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s772_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s772_268_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s772_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s772_268_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s772_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s772_268_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s772_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s772_268_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,677 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.72 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,677\n",
      "  Read 1 with adapter:                  49,662 (100.0%)\n",
      "  Read 2 with adapter:                  49,634 (99.9%)\n",
      "Pairs written (passing filters):        49,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,306,811 bp\n",
      "  Read 1:    14,654,300 bp\n",
      "  Read 2:    14,652,511 bp\n",
      "Total written (filtered):     26,725,595 bp (91.2%)\n",
      "  Read 1:    13,363,414 bp\n",
      "  Read 2:    13,362,181 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t585\t0.0\t2\t0 560 25\n",
      "26\t48763\t0.0\t2\t47741 997 25\n",
      "27\t237\t0.0\t2\t0 233 4\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49634 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t374\t0.0\t2\t0 365 9\n",
      "26\t49003\t0.0\t2\t48051 916 36\n",
      "27\t234\t0.0\t2\t0 229 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s773_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s773_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s773_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s773_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s773_121_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s773_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s773_121_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s773_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,188 reads  @     79.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.16 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,188\n",
      "  Read 1 with adapter:                  52,163 (100.0%)\n",
      "  Read 2 with adapter:                  52,122 (99.9%)\n",
      "Pairs written (passing filters):        52,188 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,787,016 bp\n",
      "  Read 1:    15,395,447 bp\n",
      "  Read 2:    15,391,569 bp\n",
      "Total written (filtered):     28,075,878 bp (91.2%)\n",
      "  Read 1:    14,039,401 bp\n",
      "  Read 2:    14,036,477 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52163 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t495\t0.0\t2\t0 478 17\n",
      "26\t51297\t0.0\t2\t50331 954 12\n",
      "27\t315\t0.0\t2\t0 306 9\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t346\t0.0\t2\t0 340 6\n",
      "26\t51470\t0.0\t2\t50536 913 21\n",
      "27\t282\t0.0\t2\t0 277 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s774_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s774_270_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s774_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s774_270_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s774_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s774_270_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s774_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s774_270_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,925 reads  @     84.7 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.15 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,925\n",
      "  Read 1 with adapter:                  48,903 (100.0%)\n",
      "  Read 2 with adapter:                  48,894 (99.9%)\n",
      "Pairs written (passing filters):        48,925 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,864,402 bp\n",
      "  Read 1:    14,431,563 bp\n",
      "  Read 2:    14,432,839 bp\n",
      "Total written (filtered):     26,321,958 bp (91.2%)\n",
      "  Read 1:    13,160,252 bp\n",
      "  Read 2:    13,161,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t503\t0.0\t2\t0 487 16\n",
      "26\t48036\t0.0\t2\t47031 987 18\n",
      "27\t320\t0.0\t2\t0 313 7\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48894 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t391\t0.0\t2\t0 375 16\n",
      "26\t48200\t0.0\t2\t47282 900 18\n",
      "27\t270\t0.0\t2\t0 266 4\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s775_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s775_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s775_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s775_271_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s775_123_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s775_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s775_123_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s775_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,146 reads  @     83.4 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.19 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,146\n",
      "  Read 1 with adapter:                  38,007 (99.6%)\n",
      "  Read 2 with adapter:                  38,114 (99.9%)\n",
      "Pairs written (passing filters):        38,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,470,351 bp\n",
      "  Read 1:    11,218,041 bp\n",
      "  Read 2:    11,252,310 bp\n",
      "Total written (filtered):     20,491,516 bp (91.2%)\n",
      "  Read 1:    10,230,010 bp\n",
      "  Read 2:    10,261,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t429\t0.0\t2\t0 417 12\n",
      "26\t37300\t0.0\t2\t36611 675 14\n",
      "27\t194\t0.0\t2\t0 186 8\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38114 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t385\t0.0\t2\t0 375 10\n",
      "26\t37493\t0.0\t2\t36776 696 21\n",
      "27\t163\t0.0\t2\t0 157 6\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s776_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s776_272_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s776_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s776_272_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s776_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s776_272_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s776_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s776_272_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        34,720 reads  @    103.5 µs/read;   0.58 M reads/minute\n",
      "Finished in 3.60 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,720\n",
      "  Read 1 with adapter:                  34,662 (99.8%)\n",
      "  Read 2 with adapter:                  34,653 (99.8%)\n",
      "Pairs written (passing filters):        34,720 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,470,048 bp\n",
      "  Read 1:    10,229,838 bp\n",
      "  Read 2:    10,240,210 bp\n",
      "Total written (filtered):     18,668,102 bp (91.2%)\n",
      "  Read 1:     9,328,798 bp\n",
      "  Read 2:     9,339,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t397\t0.0\t2\t0 379 18\n",
      "26\t34023\t0.0\t2\t33330 681 12\n",
      "27\t199\t0.0\t2\t0 193 6\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34653 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t270\t0.0\t2\t0 262 8\n",
      "26\t34167\t0.0\t2\t33464 686 17\n",
      "27\t190\t0.0\t2\t0 186 4\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s777_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s777_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s777_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s777_273_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s777_125_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s777_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s777_125_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s777_273_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,426 reads  @     87.0 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.26 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,426\n",
      "  Read 1 with adapter:                  37,363 (99.8%)\n",
      "  Read 2 with adapter:                  37,357 (99.8%)\n",
      "Pairs written (passing filters):        37,426 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,062,662 bp\n",
      "  Read 1:    11,028,758 bp\n",
      "  Read 2:    11,033,904 bp\n",
      "Total written (filtered):     20,120,173 bp (91.2%)\n",
      "  Read 1:    10,057,487 bp\n",
      "  Read 2:    10,062,686 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t436\t0.0\t2\t0 419 17\n",
      "26\t36650\t0.0\t2\t35929 705 16\n",
      "27\t233\t0.0\t2\t0 229 4\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37357 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t287\t0.0\t2\t0 282 5\n",
      "26\t36823\t0.0\t2\t36091 708 24\n",
      "27\t227\t0.0\t2\t0 223 4\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s778_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s778_274_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s778_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s778_274_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s778_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s778_274_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s778_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s778_274_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,680 reads  @     90.9 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.61 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,680\n",
      "  Read 1 with adapter:                  39,548 (99.7%)\n",
      "  Read 2 with adapter:                  39,633 (99.9%)\n",
      "Pairs written (passing filters):        39,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,378,524 bp\n",
      "  Read 1:    11,674,057 bp\n",
      "  Read 2:    11,704,467 bp\n",
      "Total written (filtered):     21,320,169 bp (91.2%)\n",
      "  Read 1:    10,646,062 bp\n",
      "  Read 2:    10,674,107 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39548 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t471\t0.0\t2\t0 459 12\n",
      "26\t38846\t0.0\t2\t38111 717 18\n",
      "27\t176\t0.0\t2\t0 171 5\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39633 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t314\t0.0\t2\t0 310 4\n",
      "26\t39064\t0.0\t2\t38284 762 18\n",
      "27\t210\t0.0\t2\t0 201 9\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s779_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s779_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s779_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s779_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s779_127_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s779_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s779_127_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s779_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,150 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.03 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,150\n",
      "  Read 1 with adapter:                  35,129 (99.9%)\n",
      "  Read 2 with adapter:                  35,112 (99.9%)\n",
      "Pairs written (passing filters):        35,150 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,735,989 bp\n",
      "  Read 1:    10,368,008 bp\n",
      "  Read 2:    10,367,981 bp\n",
      "Total written (filtered):     18,910,025 bp (91.2%)\n",
      "  Read 1:     9,454,802 bp\n",
      "  Read 2:     9,455,223 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t397\t0.0\t2\t0 388 9\n",
      "26\t34484\t0.0\t2\t33792 681 11\n",
      "27\t191\t0.0\t2\t0 189 2\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t369\t0.0\t2\t0 361 8\n",
      "26\t34507\t0.0\t2\t33819 677 11\n",
      "27\t181\t0.0\t2\t0 175 6\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s780_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s780_276_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s780_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s780_276_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s780_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s780_276_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s780_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s780_276_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,911 reads  @     84.9 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.31 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,911\n",
      "  Read 1 with adapter:                  38,900 (100.0%)\n",
      "  Read 2 with adapter:                  38,892 (100.0%)\n",
      "Pairs written (passing filters):        38,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,956,634 bp\n",
      "  Read 1:    11,478,724 bp\n",
      "  Read 2:    11,477,910 bp\n",
      "Total written (filtered):     20,934,408 bp (91.2%)\n",
      "  Read 1:    10,467,571 bp\n",
      "  Read 2:    10,466,837 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t448\t0.0\t2\t0 432 16\n",
      "26\t38207\t0.0\t2\t37511 682 14\n",
      "27\t201\t0.0\t2\t0 197 4\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38892 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t279\t0.0\t2\t0 272 7\n",
      "26\t38424\t0.0\t2\t37729 679 16\n",
      "27\t154\t0.0\t2\t0 151 3\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s781_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s781_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s781_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s781_277_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s781_129_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s781_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s781_129_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s781_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,891 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.54 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,891\n",
      "  Read 1 with adapter:                  39,873 (100.0%)\n",
      "  Read 2 with adapter:                  39,854 (99.9%)\n",
      "Pairs written (passing filters):        39,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,532,721 bp\n",
      "  Read 1:    11,766,340 bp\n",
      "  Read 2:    11,766,381 bp\n",
      "Total written (filtered):     21,460,133 bp (91.2%)\n",
      "  Read 1:    10,729,895 bp\n",
      "  Read 2:    10,730,238 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39873 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t492\t0.0\t2\t0 479 13\n",
      "26\t39098\t0.0\t2\t38349 739 10\n",
      "27\t235\t0.0\t2\t0 228 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t308\t0.0\t2\t0 295 13\n",
      "26\t39274\t0.0\t2\t38504 754 16\n",
      "27\t241\t0.0\t2\t0 236 5\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s782_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s782_278_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s782_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s782_278_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s782_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s782_278_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s782_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s782_278_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,584 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.14 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,584\n",
      "  Read 1 with adapter:                  41,561 (99.9%)\n",
      "  Read 2 with adapter:                  41,524 (99.9%)\n",
      "Pairs written (passing filters):        41,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,531,809 bp\n",
      "  Read 1:    12,265,276 bp\n",
      "  Read 2:    12,266,533 bp\n",
      "Total written (filtered):     22,371,811 bp (91.2%)\n",
      "  Read 1:    11,184,849 bp\n",
      "  Read 2:    11,186,962 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t483\t0.0\t2\t0 465 18\n",
      "26\t40748\t0.0\t2\t39878 857 13\n",
      "27\t228\t0.0\t2\t0 221 7\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t365\t0.0\t2\t0 351 14\n",
      "26\t40868\t0.0\t2\t40055 788 25\n",
      "27\t206\t0.0\t2\t0 205 1\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s783_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s783_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s783_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s783_279_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s783_131_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s783_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s783_131_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s783_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        46,081 reads  @    100.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.65 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,081\n",
      "  Read 1 with adapter:                  45,951 (99.7%)\n",
      "  Read 2 with adapter:                  46,045 (99.9%)\n",
      "Pairs written (passing filters):        46,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,166,792 bp\n",
      "  Read 1:    13,574,802 bp\n",
      "  Read 2:    13,591,990 bp\n",
      "Total written (filtered):     24,775,304 bp (91.2%)\n",
      "  Read 1:    12,380,241 bp\n",
      "  Read 2:    12,395,063 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t415\t0.0\t2\t0 401 14\n",
      "26\t45264\t0.0\t2\t44388 870 6\n",
      "27\t210\t0.0\t2\t0 208 2\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t482\t0.0\t2\t0 472 10\n",
      "26\t45265\t0.0\t2\t44392 852 21\n",
      "27\t249\t0.0\t2\t0 243 6\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s784_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s784_280_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s784_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s784_280_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s784_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s784_280_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s784_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s784_280_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,048 reads  @     90.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.70 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,048\n",
      "  Read 1 with adapter:                  40,996 (99.9%)\n",
      "  Read 2 with adapter:                  40,995 (99.9%)\n",
      "Pairs written (passing filters):        41,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,211,436 bp\n",
      "  Read 1:    12,104,195 bp\n",
      "  Read 2:    12,107,241 bp\n",
      "Total written (filtered):     22,079,824 bp (91.2%)\n",
      "  Read 1:    11,038,344 bp\n",
      "  Read 2:    11,041,480 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t316\t0.0\t2\t0 309 7\n",
      "26\t40401\t0.0\t2\t39655 739 7\n",
      "27\t239\t0.0\t2\t0 235 4\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40995 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t294\t0.0\t2\t0 278 16\n",
      "26\t40479\t0.0\t2\t39660 788 31\n",
      "27\t199\t0.0\t2\t0 195 4\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s785_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s785_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s785_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s785_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s785_133_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s785_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s785_133_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s785_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,210 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.70 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,210\n",
      "  Read 1 with adapter:                  47,097 (99.8%)\n",
      "  Read 2 with adapter:                  47,150 (99.9%)\n",
      "Pairs written (passing filters):        47,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,843,208 bp\n",
      "  Read 1:    13,919,293 bp\n",
      "  Read 2:    13,923,915 bp\n",
      "Total written (filtered):     25,392,910 bp (91.2%)\n",
      "  Read 1:    12,694,795 bp\n",
      "  Read 2:    12,698,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47097 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t406\t0.0\t2\t0 388 18\n",
      "26\t46338\t0.0\t2\t45417 910 11\n",
      "27\t252\t0.0\t2\t0 247 5\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47150 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t362\t0.0\t2\t0 354 8\n",
      "26\t46497\t0.0\t2\t45563 905 29\n",
      "27\t268\t0.0\t2\t0 264 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s786_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s786_282_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s786_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s786_282_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s786_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s786_282_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s786_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s786_282_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,175 reads  @     90.2 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.17 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,175\n",
      "  Read 1 with adapter:                  46,132 (99.9%)\n",
      "  Read 2 with adapter:                  46,140 (99.9%)\n",
      "Pairs written (passing filters):        46,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,239,692 bp\n",
      "  Read 1:    13,618,997 bp\n",
      "  Read 2:    13,620,695 bp\n",
      "Total written (filtered):     24,840,677 bp (91.2%)\n",
      "  Read 1:    12,419,508 bp\n",
      "  Read 2:    12,421,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t370\t0.0\t2\t0 354 16\n",
      "26\t45375\t0.0\t2\t44455 894 26\n",
      "27\t275\t0.0\t2\t0 264 11\n",
      "28\t94\t0.0\t2\t0 0 94\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t351\t0.0\t2\t0 344 7\n",
      "26\t45508\t0.0\t2\t44626 857 25\n",
      "27\t253\t0.0\t2\t0 245 8\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s787_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s787_283_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s787_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s787_283_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s787_135_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s787_283_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s787_135_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s787_283_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,137 reads  @    100.0 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.52 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,137\n",
      "  Read 1 with adapter:                  45,115 (100.0%)\n",
      "  Read 2 with adapter:                  45,093 (99.9%)\n",
      "Pairs written (passing filters):        45,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,626,866 bp\n",
      "  Read 1:    13,314,086 bp\n",
      "  Read 2:    13,312,780 bp\n",
      "Total written (filtered):     24,281,700 bp (91.2%)\n",
      "  Read 1:    12,141,114 bp\n",
      "  Read 2:    12,140,586 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t385\t0.0\t2\t0 372 13\n",
      "26\t44378\t0.0\t2\t43608 759 11\n",
      "27\t253\t0.0\t2\t0 247 6\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t462\t0.0\t2\t0 450 12\n",
      "26\t44355\t0.0\t2\t43459 867 29\n",
      "27\t238\t0.0\t2\t0 227 11\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s788_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s788_284_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s788_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s788_284_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s788_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s788_284_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s788_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s788_284_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,724 reads  @     91.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.21 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,724\n",
      "  Read 1 with adapter:                  45,701 (99.9%)\n",
      "  Read 2 with adapter:                  45,670 (99.9%)\n",
      "Pairs written (passing filters):        45,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,973,457 bp\n",
      "  Read 1:    13,488,193 bp\n",
      "  Read 2:    13,485,264 bp\n",
      "Total written (filtered):     24,598,021 bp (91.2%)\n",
      "  Read 1:    12,300,097 bp\n",
      "  Read 2:    12,297,924 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45701 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t404\t0.0\t2\t0 395 9\n",
      "26\t44973\t0.0\t2\t44233 728 12\n",
      "27\t266\t0.0\t2\t0 261 5\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45670 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t316\t0.0\t2\t0 300 16\n",
      "26\t45099\t0.0\t2\t44238 841 20\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s789_137_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s789_285_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s789_137_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s789_285_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s789_137_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s789_285_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s789_137_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s789_285_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,105 reads  @    122.5 µs/read;   0.49 M reads/minute\n",
      "Finished in 0.14 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,105\n",
      "  Read 1 with adapter:                   1,046 (94.7%)\n",
      "  Read 2 with adapter:                   1,035 (93.7%)\n",
      "Pairs written (passing filters):         1,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:       651,537 bp\n",
      "  Read 1:       325,718 bp\n",
      "  Read 2:       325,819 bp\n",
      "Total written (filtered):        597,417 bp (91.7%)\n",
      "  Read 1:       298,519 bp\n",
      "  Read 2:       298,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1046 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t11\t0.0\t2\t0 11\n",
      "26\t1018\t0.0\t2\t976 41 1\n",
      "27\t16\t0.0\t2\t0 5 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1035 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t9\t0.0\t2\t0 9\n",
      "26\t1010\t0.0\t2\t971 37 2\n",
      "27\t8\t0.0\t2\t0 8\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s790_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s790_286_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s790_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s790_286_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s790_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s790_286_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s790_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s790_286_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,380 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.84 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,380\n",
      "  Read 1 with adapter:                  48,365 (100.0%)\n",
      "  Read 2 with adapter:                  48,322 (99.9%)\n",
      "Pairs written (passing filters):        48,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,542,346 bp\n",
      "  Read 1:    14,271,525 bp\n",
      "  Read 2:    14,270,821 bp\n",
      "Total written (filtered):     26,028,819 bp (91.2%)\n",
      "  Read 1:    13,014,270 bp\n",
      "  Read 2:    13,014,549 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t482\t0.0\t2\t0 465 17\n",
      "26\t47600\t0.0\t2\t46610 975 15\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t371\t0.0\t2\t0 361 10\n",
      "26\t47633\t0.0\t2\t46640 958 35\n",
      "27\t285\t0.0\t2\t0 275 10\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s791_139_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s791_287_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s791_139_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s791_287_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s791_139_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s791_287_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s791_139_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s791_287_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,406 reads  @     84.9 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.20 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,406\n",
      "  Read 1 with adapter:                  49,377 (99.9%)\n",
      "  Read 2 with adapter:                  49,351 (99.9%)\n",
      "Pairs written (passing filters):        49,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,145,047 bp\n",
      "  Read 1:    14,574,746 bp\n",
      "  Read 2:    14,570,301 bp\n",
      "Total written (filtered):     26,578,485 bp (91.2%)\n",
      "  Read 1:    13,291,063 bp\n",
      "  Read 2:    13,287,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49377 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t447\t0.0\t2\t0 430 17\n",
      "26\t48617\t0.0\t2\t47633 966 18\n",
      "27\t230\t0.0\t2\t0 227 3\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t471\t0.0\t2\t0 465 6\n",
      "26\t48608\t0.0\t2\t47684 898 26\n",
      "27\t236\t0.0\t2\t0 230 6\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s792_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s792_288_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s792_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s792_288_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s792_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s792_288_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s792_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s792_288_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,011 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 2.70 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,011\n",
      "  Read 1 with adapter:                  29,997 (100.0%)\n",
      "  Read 2 with adapter:                  29,990 (99.9%)\n",
      "Pairs written (passing filters):        30,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,705,281 bp\n",
      "  Read 1:     8,852,983 bp\n",
      "  Read 2:     8,852,298 bp\n",
      "Total written (filtered):     16,145,834 bp (91.2%)\n",
      "  Read 1:     8,073,187 bp\n",
      "  Read 2:     8,072,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29997 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t269\t0.0\t2\t0 260 9\n",
      "26\t29568\t0.0\t2\t28867 680 21\n",
      "27\t109\t0.0\t2\t0 106 3\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t236\t0.0\t2\t0 229 7\n",
      "26\t29579\t0.0\t2\t28996 564 19\n",
      "27\t151\t0.0\t2\t0 148 3\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s793_141_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s793_289_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s793_141_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s793_289_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s793_141_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s793_289_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s793_141_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s793_289_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,841 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.15 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,841\n",
      "  Read 1 with adapter:                  49,814 (99.9%)\n",
      "  Read 2 with adapter:                  49,799 (99.9%)\n",
      "Pairs written (passing filters):        49,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,403,147 bp\n",
      "  Read 1:    14,703,066 bp\n",
      "  Read 2:    14,700,081 bp\n",
      "Total written (filtered):     26,813,480 bp (91.2%)\n",
      "  Read 1:    13,408,146 bp\n",
      "  Read 2:    13,405,334 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49814 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t510\t0.0\t2\t0 494 16\n",
      "26\t49004\t0.0\t2\t47840 1145 19\n",
      "27\t242\t0.0\t2\t0 235 7\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t373\t0.0\t2\t0 364 9\n",
      "26\t49059\t0.0\t2\t48041 980 38\n",
      "27\t332\t0.0\t2\t0 322 10\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s794_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s794_290_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s794_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s794_290_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s794_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s794_290_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s794_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s794_290_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,102 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.44 s (81 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,102\n",
      "  Read 1 with adapter:                  55,079 (100.0%)\n",
      "  Read 2 with adapter:                  55,069 (99.9%)\n",
      "Pairs written (passing filters):        55,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,509,419 bp\n",
      "  Read 1:    16,254,809 bp\n",
      "  Read 2:    16,254,610 bp\n",
      "Total written (filtered):     29,645,946 bp (91.2%)\n",
      "  Read 1:    14,823,021 bp\n",
      "  Read 2:    14,822,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t549\t0.0\t2\t0 535 14\n",
      "26\t54202\t0.0\t2\t52997 1182 23\n",
      "27\t241\t0.0\t2\t0 235 6\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55069 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t427\t0.0\t2\t0 414 13\n",
      "26\t54277\t0.0\t2\t53233 1015 29\n",
      "27\t328\t0.0\t2\t0 327 1\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s795_143_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s795_291_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s795_143_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s795_291_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s795_143_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s795_291_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s795_143_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s795_291_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,425 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.09 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,425\n",
      "  Read 1 with adapter:                  49,402 (100.0%)\n",
      "  Read 2 with adapter:                  49,376 (99.9%)\n",
      "Pairs written (passing filters):        49,425 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,155,428 bp\n",
      "  Read 1:    14,580,139 bp\n",
      "  Read 2:    14,575,289 bp\n",
      "Total written (filtered):     26,587,665 bp (91.2%)\n",
      "  Read 1:    13,295,911 bp\n",
      "  Read 2:    13,291,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49402 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t456\t0.0\t2\t0 437 19\n",
      "26\t48683\t0.0\t2\t47715 954 14\n",
      "27\t194\t0.0\t2\t0 186 8\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t488\t0.0\t2\t0 479 9\n",
      "26\t48587\t0.0\t2\t47596 960 31\n",
      "27\t263\t0.0\t2\t0 257 6\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s796_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s796_292_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s796_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s796_292_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s796_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s796_292_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s796_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s796_292_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,877 reads  @     77.8 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.19 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,877\n",
      "  Read 1 with adapter:                  53,860 (100.0%)\n",
      "  Read 2 with adapter:                  53,823 (99.9%)\n",
      "Pairs written (passing filters):        53,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,784,413 bp\n",
      "  Read 1:    15,893,692 bp\n",
      "  Read 2:    15,890,721 bp\n",
      "Total written (filtered):     28,985,112 bp (91.2%)\n",
      "  Read 1:    14,493,630 bp\n",
      "  Read 2:    14,491,482 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t550\t0.0\t2\t0 541 9\n",
      "26\t53041\t0.0\t2\t51937 1087 17\n",
      "27\t210\t0.0\t2\t0 203 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t412\t0.0\t2\t0 400 12\n",
      "26\t53118\t0.0\t2\t52082 1001 35\n",
      "27\t269\t0.0\t2\t0 264 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s797_145_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s797_293_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s797_145_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s797_293_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s797_145_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s797_293_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s797_145_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s797_293_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,737 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.34 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,737\n",
      "  Read 1 with adapter:                  54,711 (100.0%)\n",
      "  Read 2 with adapter:                  54,707 (99.9%)\n",
      "Pairs written (passing filters):        54,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,293,120 bp\n",
      "  Read 1:    16,146,960 bp\n",
      "  Read 2:    16,146,160 bp\n",
      "Total written (filtered):     29,448,526 bp (91.2%)\n",
      "  Read 1:    14,724,697 bp\n",
      "  Read 2:    14,723,829 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54711 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t561\t0.0\t2\t0 545 16\n",
      "26\t53815\t0.0\t2\t52698 1103 14\n",
      "27\t244\t0.0\t2\t0 236 8\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t423\t0.0\t2\t0 417 6\n",
      "26\t53888\t0.0\t2\t52827 1038 23\n",
      "27\t364\t0.0\t2\t0 353 11\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s798_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s798_294_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s798_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s798_294_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s798_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s798_294_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s798_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s798_294_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,881 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.96 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,881\n",
      "  Read 1 with adapter:                  48,804 (99.8%)\n",
      "  Read 2 with adapter:                  48,843 (99.9%)\n",
      "Pairs written (passing filters):        48,881 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,825,531 bp\n",
      "  Read 1:    14,406,800 bp\n",
      "  Read 2:    14,418,731 bp\n",
      "Total written (filtered):     26,287,065 bp (91.2%)\n",
      "  Read 1:    13,138,128 bp\n",
      "  Read 2:    13,148,937 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48804 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t475\t0.0\t2\t0 456 19\n",
      "26\t48053\t0.0\t2\t47125 909 19\n",
      "27\t205\t0.0\t2\t0 198 7\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48843 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t351\t0.0\t2\t0 340 11\n",
      "26\t48217\t0.0\t2\t47237 947 33\n",
      "27\t243\t0.0\t2\t0 238 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s799_147_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s799_295_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s799_147_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s799_295_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s799_147_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-y7p6h75d/s799_295_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s799_147_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-25gaw0lf/24639387-bb97-439b-bb22-526b05719acb/data/s799_295_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        27,387 reads  @     81.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.24 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,387\n",
      "  Read 1 with adapter:                  27,371 (99.9%)\n",
      "  Read 2 with adapter:                  27,139 (99.1%)\n",
      "Pairs written (passing filters):        27,387 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,154,137 bp\n",
      "  Read 1:     8,078,874 bp\n",
      "  Read 2:     8,075,263 bp\n",
      "Total written (filtered):     14,737,130 bp (91.2%)\n",
      "  Read 1:     7,367,342 bp\n",
      "  Read 2:     7,369,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27371 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t271\t0.0\t2\t0 263 8\n",
      "26\t26935\t0.0\t2\t26249 672 14\n",
      "27\t137\t0.0\t2\t0 134 3\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t298\t0.0\t2\t0 286 12\n",
      "26\t26646\t0.0\t2\t26041 588 17\n",
      "27\t171\t0.0\t2\t0 165 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: c-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: c-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-ny7455u5')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 371e1dd6-b7f0-44ab-a090-c540aecc7f66>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.c import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P1604-Lib3-leray_manifest_r3.csv \\\n",
    "  --output-path c-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.c trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences c-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences c-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.c summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data c-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization c-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('c-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: c-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: c-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: c-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.c Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs c-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 267 \\\n",
    "  --p-trunc-len-r 180 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table c-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats c-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: c-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-fks173wt')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 6fa72dd3-dffa-401e-97cc-b6c90f9090e0>"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.c view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table c-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization c-table.qzv \n",
    "q2.Visualization.load('c-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: c-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.c view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization c-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/BB-Lib1_leray_manifest_runbb_v2_grid.csv as PairedEndFastqManifestPhred33 to d-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S701_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S701_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S701_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S701_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S701_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S701_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S701_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S701_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        35,223 reads  @    102.5 µs/read;   0.59 M reads/minute\n",
      "Finished in 3.61 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,223\n",
      "  Read 1 with adapter:                  35,164 (99.8%)\n",
      "  Read 2 with adapter:                  34,841 (98.9%)\n",
      "Pairs written (passing filters):        35,223 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,765,985 bp\n",
      "  Read 1:    10,381,253 bp\n",
      "  Read 2:    10,384,732 bp\n",
      "Total written (filtered):     18,946,355 bp (91.2%)\n",
      "  Read 1:     9,467,242 bp\n",
      "  Read 2:     9,479,113 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t450\t0.0\t2\t0 436 14\n",
      "26\t34492\t0.0\t2\t33504 964 24\n",
      "27\t159\t0.0\t2\t0 156 3\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t378\t0.0\t2\t0 358 20\n",
      "26\t34302\t0.0\t2\t31767 2488 47\n",
      "27\t147\t0.0\t2\t0 134 13\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S702_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S702_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S702_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S702_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S702_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S702_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S702_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S702_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        73,432 reads  @     94.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 6.97 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,432\n",
      "  Read 1 with adapter:                  73,363 (99.9%)\n",
      "  Read 2 with adapter:                  73,210 (99.7%)\n",
      "Pairs written (passing filters):        73,432 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,306,684 bp\n",
      "  Read 1:    21,654,700 bp\n",
      "  Read 2:    21,651,984 bp\n",
      "Total written (filtered):     39,496,442 bp (91.2%)\n",
      "  Read 1:    19,747,618 bp\n",
      "  Read 2:    19,748,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t831\t0.0\t2\t0 816 15\n",
      "26\t72039\t0.0\t2\t70608 1405 26\n",
      "27\t379\t0.0\t2\t0 370 9\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 73210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t580\t0.0\t2\t0 536 44\n",
      "26\t72293\t0.0\t2\t66622 5468 203\n",
      "27\t298\t0.0\t2\t0 272 26\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S703_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S703_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S703_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S703_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S703_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S703_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S703_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S703_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,041 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.31 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,041\n",
      "  Read 1 with adapter:                  51,981 (99.9%)\n",
      "  Read 2 with adapter:                  51,952 (99.8%)\n",
      "Pairs written (passing filters):        52,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,694,719 bp\n",
      "  Read 1:    15,345,537 bp\n",
      "  Read 2:    15,349,182 bp\n",
      "Total written (filtered):     27,992,824 bp (91.2%)\n",
      "  Read 1:    13,994,280 bp\n",
      "  Read 2:    13,998,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t604\t0.0\t2\t0 584 20\n",
      "26\t51010\t0.0\t2\t49929 1063 18\n",
      "27\t283\t0.0\t2\t0 275 8\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t337\t0.0\t2\t0 313 24\n",
      "26\t51351\t0.0\t2\t47205 4002 144\n",
      "27\t237\t0.0\t2\t0 225 12\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S704_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S704_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S704_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S704_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S704_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S704_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S704_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S704_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,048 reads  @     84.1 µs/read;   0.71 M reads/minute\n",
      "Finished in 2.53 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,048\n",
      "  Read 1 with adapter:                  30,041 (100.0%)\n",
      "  Read 2 with adapter:                  29,768 (99.1%)\n",
      "Pairs written (passing filters):        30,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,693,590 bp\n",
      "  Read 1:     8,864,145 bp\n",
      "  Read 2:     8,829,445 bp\n",
      "Total written (filtered):     16,138,682 bp (91.2%)\n",
      "  Read 1:     8,083,152 bp\n",
      "  Read 2:     8,055,530 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30041 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t309\t0.0\t2\t0 295 14\n",
      "26\t29502\t0.0\t2\t28892 599 11\n",
      "27\t136\t0.0\t2\t0 131 5\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t196\t0.0\t2\t0 176 20\n",
      "26\t29400\t0.0\t2\t26643 2557 200\n",
      "27\t153\t0.0\t2\t0 135 18\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S706_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S706_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S706_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S706_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S706_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S706_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S706_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S706_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,560 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.62 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,560\n",
      "  Read 1 with adapter:                  43,536 (99.9%)\n",
      "  Read 2 with adapter:                  43,435 (99.7%)\n",
      "Pairs written (passing filters):        43,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,695,254 bp\n",
      "  Read 1:    12,849,807 bp\n",
      "  Read 2:    12,845,447 bp\n",
      "Total written (filtered):     23,434,295 bp (91.2%)\n",
      "  Read 1:    11,717,997 bp\n",
      "  Read 2:    11,716,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43536 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t472\t0.0\t2\t0 461 11\n",
      "26\t42759\t0.0\t2\t41896 849 14\n",
      "27\t228\t0.0\t2\t0 223 5\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t338\t0.0\t2\t0 318 20\n",
      "26\t42878\t0.0\t2\t39418 3329 131\n",
      "27\t189\t0.0\t2\t0 175 14\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S707_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S707_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S707_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S707_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S707_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S707_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S707_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S707_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,725 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.44 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,725\n",
      "  Read 1 with adapter:                  40,709 (100.0%)\n",
      "  Read 2 with adapter:                  40,606 (99.7%)\n",
      "Pairs written (passing filters):        40,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,020,789 bp\n",
      "  Read 1:    12,013,608 bp\n",
      "  Read 2:    12,007,181 bp\n",
      "Total written (filtered):     21,906,880 bp (91.2%)\n",
      "  Read 1:    10,955,343 bp\n",
      "  Read 2:    10,951,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t443\t0.0\t2\t0 436 7\n",
      "26\t39990\t0.0\t2\t39184 794 12\n",
      "27\t198\t0.0\t2\t0 187 11\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t307\t0.0\t2\t0 280 27\n",
      "26\t40107\t0.0\t2\t36894 3056 157\n",
      "27\t157\t0.0\t2\t0 152 5\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S708_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S708_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S708_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S708_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S708_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S708_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S708_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S708_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,153 reads  @     93.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.28 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,153\n",
      "  Read 1 with adapter:                  35,133 (99.9%)\n",
      "  Read 2 with adapter:                  35,089 (99.8%)\n",
      "Pairs written (passing filters):        35,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,738,524 bp\n",
      "  Read 1:    10,368,991 bp\n",
      "  Read 2:    10,369,533 bp\n",
      "Total written (filtered):     18,912,997 bp (91.2%)\n",
      "  Read 1:     9,455,710 bp\n",
      "  Read 2:     9,457,287 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t416\t0.0\t2\t0 403 13\n",
      "26\t34469\t0.0\t2\t33606 833 30\n",
      "27\t185\t0.0\t2\t0 175 10\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35089 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t257\t0.0\t2\t0 229 28\n",
      "26\t34647\t0.0\t2\t31988 2577 82\n",
      "27\t145\t0.0\t2\t0 133 12\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S709_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S709_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S709_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S709_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S709_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S709_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S709_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S709_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        26,844 reads  @    101.8 µs/read;   0.59 M reads/minute\n",
      "Finished in 2.73 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,844\n",
      "  Read 1 with adapter:                  26,820 (99.9%)\n",
      "  Read 2 with adapter:                  26,690 (99.4%)\n",
      "Pairs written (passing filters):        26,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,835,316 bp\n",
      "  Read 1:     7,918,557 bp\n",
      "  Read 2:     7,916,759 bp\n",
      "Total written (filtered):     14,444,456 bp (91.2%)\n",
      "  Read 1:     7,221,479 bp\n",
      "  Read 2:     7,222,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t381\t0.0\t2\t0 368 13\n",
      "26\t26255\t0.0\t2\t25573 670 12\n",
      "27\t133\t0.0\t2\t0 130 3\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t270\t0.0\t2\t0 250 20\n",
      "26\t26268\t0.0\t2\t24341 1888 39\n",
      "27\t124\t0.0\t2\t0 110 14\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S710_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S710_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S710_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S710_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S710_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S710_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S710_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S710_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        33,794 reads  @     98.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 3.31 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,794\n",
      "  Read 1 with adapter:                  33,767 (99.9%)\n",
      "  Read 2 with adapter:                  33,703 (99.7%)\n",
      "Pairs written (passing filters):        33,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,934,051 bp\n",
      "  Read 1:     9,967,947 bp\n",
      "  Read 2:     9,966,104 bp\n",
      "Total written (filtered):     18,180,016 bp (91.2%)\n",
      "  Read 1:     9,090,039 bp\n",
      "  Read 2:     9,089,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t498\t0.0\t2\t0 488 10\n",
      "26\t33042\t0.0\t2\t32270 758 14\n",
      "27\t167\t0.0\t2\t0 161 6\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t292\t0.0\t2\t0 272 20\n",
      "26\t33240\t0.0\t2\t30579 2552 109\n",
      "27\t149\t0.0\t2\t0 137 12\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S711_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S711_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S711_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S711_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S711_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S711_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S711_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S711_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        57,455 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.28 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,455\n",
      "  Read 1 with adapter:                  57,405 (99.9%)\n",
      "  Read 2 with adapter:                  57,351 (99.8%)\n",
      "Pairs written (passing filters):        57,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,896,387 bp\n",
      "  Read 1:    16,948,480 bp\n",
      "  Read 2:    16,947,907 bp\n",
      "Total written (filtered):     30,913,144 bp (91.2%)\n",
      "  Read 1:    15,456,227 bp\n",
      "  Read 2:    15,456,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57405 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t777\t0.0\t2\t0 754 23\n",
      "26\t56120\t0.0\t2\t54832 1260 28\n",
      "27\t388\t0.0\t2\t0 380 8\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t404\t0.0\t2\t0 374 30\n",
      "26\t56622\t0.0\t2\t52182 4273 167\n",
      "27\t298\t0.0\t2\t0 280 18\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S712_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S712_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S712_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S712_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S712_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S712_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S712_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S712_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,779 reads  @     84.8 µs/read;   0.71 M reads/minute\n",
      "Finished in 2.44 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,779\n",
      "  Read 1 with adapter:                  28,755 (99.9%)\n",
      "  Read 2 with adapter:                  27,924 (97.0%)\n",
      "Pairs written (passing filters):        28,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,791,818 bp\n",
      "  Read 1:     8,489,539 bp\n",
      "  Read 2:     8,302,279 bp\n",
      "Total written (filtered):     15,317,653 bp (91.2%)\n",
      "  Read 1:     7,741,389 bp\n",
      "  Read 2:     7,576,264 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t319\t0.0\t2\t0 306 13\n",
      "26\t28155\t0.0\t2\t27549 594 12\n",
      "27\t181\t0.0\t2\t0 173 8\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27924 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t155\t0.0\t2\t0 143 12\n",
      "26\t27607\t0.0\t2\t25013 2401 193\n",
      "27\t150\t0.0\t2\t0 127 23\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S714_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S714_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S714_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S714_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S714_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S714_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S714_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S714_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,472 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.05 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,472\n",
      "  Read 1 with adapter:                  39,448 (99.9%)\n",
      "  Read 2 with adapter:                  39,327 (99.6%)\n",
      "Pairs written (passing filters):        39,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,279,737 bp\n",
      "  Read 1:    11,644,211 bp\n",
      "  Read 2:    11,635,526 bp\n",
      "Total written (filtered):     21,231,712 bp (91.2%)\n",
      "  Read 1:    10,618,533 bp\n",
      "  Read 2:    10,613,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t500\t0.0\t2\t0 488 12\n",
      "26\t38651\t0.0\t2\t37852 790 9\n",
      "27\t229\t0.0\t2\t0 222 7\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t345\t0.0\t2\t0 315 30\n",
      "26\t38785\t0.0\t2\t35739 2909 137\n",
      "27\t144\t0.0\t2\t0 133 11\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S715_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S715_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S715_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S715_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S715_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S715_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S715_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S715_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,150 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 2.79 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,150\n",
      "  Read 1 with adapter:                  34,123 (99.9%)\n",
      "  Read 2 with adapter:                  34,009 (99.6%)\n",
      "Pairs written (passing filters):        34,150 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,143,301 bp\n",
      "  Read 1:    10,074,236 bp\n",
      "  Read 2:    10,069,065 bp\n",
      "Total written (filtered):     18,372,164 bp (91.2%)\n",
      "  Read 1:     9,187,218 bp\n",
      "  Read 2:     9,184,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t470\t0.0\t2\t0 462 8\n",
      "26\t33350\t0.0\t2\t32688 652 10\n",
      "27\t216\t0.0\t2\t0 206 10\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34009 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t264\t0.0\t2\t0 241 23\n",
      "26\t33606\t0.0\t2\t30836 2620 150\n",
      "27\t117\t0.0\t2\t0 108 9\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S716_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S716_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S716_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S716_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S716_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S716_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S716_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S716_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,935 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.39 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,935\n",
      "  Read 1 with adapter:                  37,910 (99.9%)\n",
      "  Read 2 with adapter:                  37,878 (99.8%)\n",
      "Pairs written (passing filters):        37,934 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,380,787 bp\n",
      "  Read 1:    11,190,442 bp\n",
      "  Read 2:    11,190,345 bp\n",
      "Total written (filtered):     20,410,410 bp (91.2%)\n",
      "  Read 1:    10,205,006 bp\n",
      "  Read 2:    10,205,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t477\t0.0\t2\t0 468 9\n",
      "26\t37104\t0.0\t2\t36347 739 18\n",
      "27\t249\t0.0\t2\t0 245 4\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37878 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t312\t0.0\t2\t0 289 23\n",
      "26\t37392\t0.0\t2\t34506 2788 98\n",
      "27\t140\t0.0\t2\t0 130 10\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S717_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S717_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S717_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S717_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S717_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S717_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S717_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S717_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,850 reads  @     96.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.64 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,850\n",
      "  Read 1 with adapter:                  37,766 (99.8%)\n",
      "  Read 2 with adapter:                  37,497 (99.1%)\n",
      "Pairs written (passing filters):        37,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,318,619 bp\n",
      "  Read 1:    11,157,887 bp\n",
      "  Read 2:    11,160,732 bp\n",
      "Total written (filtered):     20,362,213 bp (91.2%)\n",
      "  Read 1:    10,176,188 bp\n",
      "  Read 2:    10,186,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t395\t0.0\t2\t0 389 6\n",
      "26\t37166\t0.0\t2\t36313 832 21\n",
      "27\t156\t0.0\t2\t0 153 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37497 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t390\t0.0\t2\t0 353 37\n",
      "26\t36917\t0.0\t2\t34081 2737 99\n",
      "27\t169\t0.0\t2\t0 154 15\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S718_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S718_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S718_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S718_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S718_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S718_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S718_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S718_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,488 reads  @     92.7 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.66 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,488\n",
      "  Read 1 with adapter:                  39,352 (99.7%)\n",
      "  Read 2 with adapter:                  39,352 (99.7%)\n",
      "Pairs written (passing filters):        39,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,281,661 bp\n",
      "  Read 1:    11,635,196 bp\n",
      "  Read 2:    11,646,465 bp\n",
      "Total written (filtered):     21,235,609 bp (91.2%)\n",
      "  Read 1:    10,612,153 bp\n",
      "  Read 2:    10,623,456 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t369\t0.0\t2\t0 366 3\n",
      "26\t38717\t0.0\t2\t37911 791 15\n",
      "27\t192\t0.0\t2\t0 189 3\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t300\t0.0\t2\t0 279 21\n",
      "26\t38867\t0.0\t2\t35528 3135 204\n",
      "27\t161\t0.0\t2\t0 145 16\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S719_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S719_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S719_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S719_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S719_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S719_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S719_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S719_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,627 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.36 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,627\n",
      "  Read 1 with adapter:                  40,262 (99.1%)\n",
      "  Read 2 with adapter:                  40,514 (99.7%)\n",
      "Pairs written (passing filters):        40,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,929,872 bp\n",
      "  Read 1:    11,945,855 bp\n",
      "  Read 2:    11,984,017 bp\n",
      "Total written (filtered):     21,829,868 bp (91.2%)\n",
      "  Read 1:    10,899,128 bp\n",
      "  Read 2:    10,930,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t391\t0.0\t2\t0 383 8\n",
      "26\t39616\t0.0\t2\t38842 763 11\n",
      "27\t160\t0.0\t2\t0 155 5\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40514 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t259\t0.0\t2\t0 238 21\n",
      "26\t40052\t0.0\t2\t36744 3133 175\n",
      "27\t186\t0.0\t2\t0 163 23\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S720_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S720_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S720_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S720_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S720_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S720_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S720_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S720_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        26,720 reads  @     95.4 µs/read;   0.63 M reads/minute\n",
      "Finished in 2.55 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,720\n",
      "  Read 1 with adapter:                  26,608 (99.6%)\n",
      "  Read 2 with adapter:                  26,362 (98.7%)\n",
      "Pairs written (passing filters):        26,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,730,809 bp\n",
      "  Read 1:     7,870,966 bp\n",
      "  Read 2:     7,859,843 bp\n",
      "Total written (filtered):     14,353,465 bp (91.2%)\n",
      "  Read 1:     7,178,999 bp\n",
      "  Read 2:     7,174,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t266\t0.0\t2\t0 260 6\n",
      "26\t26179\t0.0\t2\t25622 548 9\n",
      "27\t118\t0.0\t2\t0 116 2\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t180\t0.0\t2\t0 165 15\n",
      "26\t26022\t0.0\t2\t23342 2430 250\n",
      "27\t147\t0.0\t2\t0 121 26\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S722_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S722_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S722_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S722_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S722_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S722_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S722_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S722_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,800 reads  @     83.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.17 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,800\n",
      "  Read 1 with adapter:                  37,745 (99.9%)\n",
      "  Read 2 with adapter:                  37,600 (99.5%)\n",
      "Pairs written (passing filters):        37,800 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,291,493 bp\n",
      "  Read 1:    11,147,617 bp\n",
      "  Read 2:    11,143,876 bp\n",
      "Total written (filtered):     20,332,913 bp (91.2%)\n",
      "  Read 1:    10,166,450 bp\n",
      "  Read 2:    10,166,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37745 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t418\t0.0\t2\t0 403 15\n",
      "26\t37092\t0.0\t2\t36253 830 9\n",
      "27\t183\t0.0\t2\t0 178 5\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t360\t0.0\t2\t0 329 31\n",
      "26\t37036\t0.0\t2\t33802 3010 224\n",
      "27\t175\t0.0\t2\t0 152 23\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S723_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S723_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S723_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S723_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S723_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S723_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S723_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S723_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,386 reads  @     75.3 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.12 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,386\n",
      "  Read 1 with adapter:                  41,361 (99.9%)\n",
      "  Read 2 with adapter:                  41,209 (99.6%)\n",
      "Pairs written (passing filters):        41,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,414,352 bp\n",
      "  Read 1:    12,208,206 bp\n",
      "  Read 2:    12,206,146 bp\n",
      "Total written (filtered):     22,267,941 bp (91.2%)\n",
      "  Read 1:    11,133,150 bp\n",
      "  Read 2:    11,134,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t459\t0.0\t2\t0 448 11\n",
      "26\t40719\t0.0\t2\t39762 935 22\n",
      "27\t149\t0.0\t2\t0 149\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t334\t0.0\t2\t0 302 32\n",
      "26\t40633\t0.0\t2\t37084 3361 188\n",
      "27\t185\t0.0\t2\t0 168 17\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S724_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S724_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S724_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S724_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S724_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S724_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S724_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S724_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        37,290 reads  @    113.1 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.22 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,290\n",
      "  Read 1 with adapter:                  37,245 (99.9%)\n",
      "  Read 2 with adapter:                  37,182 (99.7%)\n",
      "Pairs written (passing filters):        37,290 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,995,047 bp\n",
      "  Read 1:    10,997,111 bp\n",
      "  Read 2:    10,997,936 bp\n",
      "Total written (filtered):     20,060,166 bp (91.2%)\n",
      "  Read 1:    10,028,869 bp\n",
      "  Read 2:    10,031,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t342\t0.0\t2\t0 334 8\n",
      "26\t36677\t0.0\t2\t35858 806 13\n",
      "27\t158\t0.0\t2\t0 153 5\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t270\t0.0\t2\t0 246 24\n",
      "26\t36721\t0.0\t2\t33526 2995 200\n",
      "27\t165\t0.0\t2\t0 151 14\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S725_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S725_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S725_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S725_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S725_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S725_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S725_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S725_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        20,397 reads  @    139.2 µs/read;   0.43 M reads/minute\n",
      "Finished in 2.84 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,397\n",
      "  Read 1 with adapter:                  20,385 (99.9%)\n",
      "  Read 2 with adapter:                  20,251 (99.3%)\n",
      "Pairs written (passing filters):        20,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,031,681 bp\n",
      "  Read 1:     6,017,107 bp\n",
      "  Read 2:     6,014,574 bp\n",
      "Total written (filtered):     10,975,568 bp (91.2%)\n",
      "  Read 1:     5,487,335 bp\n",
      "  Read 2:     5,488,233 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t301\t0.0\t2\t0 292 9\n",
      "26\t19983\t0.0\t2\t19160 794 29\n",
      "27\t79\t0.0\t2\t0 76 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20251 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t259\t0.0\t2\t0 243 16\n",
      "26\t19881\t0.0\t2\t18392 1460 29\n",
      "27\t88\t0.0\t2\t0 87 1\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S726_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S726_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S726_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S726_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S726_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S726_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S726_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S726_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        37,413 reads  @     92.7 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.47 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,413\n",
      "  Read 1 with adapter:                  37,383 (99.9%)\n",
      "  Read 2 with adapter:                  37,246 (99.6%)\n",
      "Pairs written (passing filters):        37,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,065,028 bp\n",
      "  Read 1:    11,035,606 bp\n",
      "  Read 2:    11,029,422 bp\n",
      "Total written (filtered):     20,124,897 bp (91.2%)\n",
      "  Read 1:    10,063,764 bp\n",
      "  Read 2:    10,061,133 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t410\t0.0\t2\t0 397 13\n",
      "26\t36691\t0.0\t2\t35907 769 15\n",
      "27\t194\t0.0\t2\t0 189 5\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37246 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t298\t0.0\t2\t0 282 16\n",
      "26\t36748\t0.0\t2\t33871 2774 103\n",
      "27\t169\t0.0\t2\t0 152 17\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S727_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S727_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S727_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S727_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S727_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S727_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S727_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S727_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,496 reads  @     84.8 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.10 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,496\n",
      "  Read 1 with adapter:                  36,470 (99.9%)\n",
      "  Read 2 with adapter:                  36,428 (99.8%)\n",
      "Pairs written (passing filters):        36,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,528,833 bp\n",
      "  Read 1:    10,764,632 bp\n",
      "  Read 2:    10,764,201 bp\n",
      "Total written (filtered):     19,633,688 bp (91.2%)\n",
      "  Read 1:     9,816,553 bp\n",
      "  Read 2:     9,817,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36470 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t435\t0.0\t2\t0 423 12\n",
      "26\t35774\t0.0\t2\t35012 747 15\n",
      "27\t164\t0.0\t2\t0 160 4\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t216\t0.0\t2\t0 196 20\n",
      "26\t36027\t0.0\t2\t33315 2641 71\n",
      "27\t172\t0.0\t2\t0 157 15\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S728_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S728_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S728_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S728_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S728_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S728_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S728_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S728_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,461 reads  @    104.9 µs/read;   0.57 M reads/minute\n",
      "Finished in 2.99 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,461\n",
      "  Read 1 with adapter:                  28,433 (99.9%)\n",
      "  Read 2 with adapter:                  28,346 (99.6%)\n",
      "Pairs written (passing filters):        28,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,781,965 bp\n",
      "  Read 1:     8,395,050 bp\n",
      "  Read 2:     8,386,915 bp\n",
      "Total written (filtered):     15,305,899 bp (91.2%)\n",
      "  Read 1:     7,655,908 bp\n",
      "  Read 2:     7,649,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t330\t0.0\t2\t0 320 10\n",
      "26\t27891\t0.0\t2\t27292 588 11\n",
      "27\t150\t0.0\t2\t0 145 5\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t206\t0.0\t2\t0 189 17\n",
      "26\t28000\t0.0\t2\t25650 2237 113\n",
      "27\t126\t0.0\t2\t0 119 7\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S729_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S729_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S729_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S729_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S729_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S729_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S729_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S729_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S730_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S730_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S730_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S730_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S730_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S730_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S730_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S730_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,104 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.67 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,104\n",
      "  Read 1 with adapter:                  48,070 (99.9%)\n",
      "  Read 2 with adapter:                  47,833 (99.4%)\n",
      "Pairs written (passing filters):        48,103 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,344,525 bp\n",
      "  Read 1:    14,190,450 bp\n",
      "  Read 2:    14,154,075 bp\n",
      "Total written (filtered):     25,851,341 bp (91.2%)\n",
      "  Read 1:    12,940,786 bp\n",
      "  Read 2:    12,910,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48070 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t597\t0.0\t2\t0 585 12\n",
      "26\t47244\t0.0\t2\t46190 1035 19\n",
      "27\t190\t0.0\t2\t0 188 2\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47833 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t399\t0.0\t2\t0 369 30\n",
      "26\t47180\t0.0\t2\t43547 3508 125\n",
      "27\t191\t0.0\t2\t0 180 11\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S731_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S731_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S731_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S731_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S731_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S731_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S731_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S731_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        18,642 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 1.93 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,642\n",
      "  Read 1 with adapter:                  18,634 (100.0%)\n",
      "  Read 2 with adapter:                  18,539 (99.4%)\n",
      "Pairs written (passing filters):        18,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,996,091 bp\n",
      "  Read 1:     5,499,382 bp\n",
      "  Read 2:     5,496,709 bp\n",
      "Total written (filtered):     10,029,793 bp (91.2%)\n",
      "  Read 1:     5,015,039 bp\n",
      "  Read 2:     5,014,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 18634 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t247\t0.0\t2\t0 238 9\n",
      "26\t18262\t0.0\t2\t17711 532 19\n",
      "27\t100\t0.0\t2\t0 97 3\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 18539 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t156\t0.0\t2\t0 147 9\n",
      "26\t18282\t0.0\t2\t15814 2219 249\n",
      "27\t89\t0.0\t2\t0 72 17\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S732_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S732_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S732_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S732_57_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S732_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S732_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S732_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S732_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,235 reads  @     90.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.64 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,235\n",
      "  Read 1 with adapter:                  40,204 (99.9%)\n",
      "  Read 2 with adapter:                  40,193 (99.9%)\n",
      "Pairs written (passing filters):        40,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,737,381 bp\n",
      "  Read 1:    11,868,664 bp\n",
      "  Read 2:    11,868,717 bp\n",
      "Total written (filtered):     21,647,527 bp (91.2%)\n",
      "  Read 1:    10,823,677 bp\n",
      "  Read 2:    10,823,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t493\t0.0\t2\t0 486 7\n",
      "26\t39478\t0.0\t2\t38575 885 18\n",
      "27\t178\t0.0\t2\t0 170 8\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t297\t0.0\t2\t0 276 21\n",
      "26\t39720\t0.0\t2\t36821 2822 77\n",
      "27\t154\t0.0\t2\t0 145 9\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S733_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S733_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S733_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S733_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S733_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S733_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S733_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S733_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        30,949 reads  @    110.1 µs/read;   0.55 M reads/minute\n",
      "Finished in 3.41 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,949\n",
      "  Read 1 with adapter:                  30,933 (99.9%)\n",
      "  Read 2 with adapter:                  30,746 (99.3%)\n",
      "Pairs written (passing filters):        30,949 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,255,877 bp\n",
      "  Read 1:     9,129,398 bp\n",
      "  Read 2:     9,126,479 bp\n",
      "Total written (filtered):     16,652,604 bp (91.2%)\n",
      "  Read 1:     8,325,298 bp\n",
      "  Read 2:     8,327,306 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t408\t0.0\t2\t0 395 13\n",
      "26\t30289\t0.0\t2\t29455 810 24\n",
      "27\t146\t0.0\t2\t0 141 5\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30746 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t335\t0.0\t2\t0 308 27\n",
      "26\t30247\t0.0\t2\t28052 2154 41\n",
      "27\t140\t0.0\t2\t0 126 14\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S734_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S734_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S734_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S734_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S734_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S734_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S734_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S734_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        37,945 reads  @     96.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.65 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,945\n",
      "  Read 1 with adapter:                  37,929 (100.0%)\n",
      "  Read 2 with adapter:                  37,860 (99.8%)\n",
      "Pairs written (passing filters):        37,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,384,254 bp\n",
      "  Read 1:    11,193,601 bp\n",
      "  Read 2:    11,190,653 bp\n",
      "Total written (filtered):     20,414,081 bp (91.2%)\n",
      "  Read 1:    10,207,634 bp\n",
      "  Read 2:    10,206,447 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t473\t0.0\t2\t0 465 8\n",
      "26\t37110\t0.0\t2\t36335 759 16\n",
      "27\t286\t0.0\t2\t0 283 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t321\t0.0\t2\t0 302 19\n",
      "26\t37353\t0.0\t2\t34585 2695 73\n",
      "27\t177\t0.0\t2\t0 167 10\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S735_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S735_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S735_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S735_63_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S735_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S735_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S735_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S735_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        48,370 reads  @    103.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.02 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,370\n",
      "  Read 1 with adapter:                  48,346 (100.0%)\n",
      "  Read 2 with adapter:                  48,288 (99.8%)\n",
      "Pairs written (passing filters):        48,370 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,536,521 bp\n",
      "  Read 1:    14,268,861 bp\n",
      "  Read 2:    14,267,660 bp\n",
      "Total written (filtered):     26,024,164 bp (91.2%)\n",
      "  Read 1:    13,011,978 bp\n",
      "  Read 2:    13,012,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t546\t0.0\t2\t0 534 12\n",
      "26\t47379\t0.0\t2\t46407 954 18\n",
      "27\t341\t0.0\t2\t0 335 6\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48288 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t333\t0.0\t2\t0 298 35\n",
      "26\t47698\t0.0\t2\t44038 3541 119\n",
      "27\t155\t0.0\t2\t0 143 12\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S736_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S736_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S736_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S736_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S736_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S736_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S736_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S736_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        26,437 reads  @    114.5 µs/read;   0.52 M reads/minute\n",
      "Finished in 3.03 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,437\n",
      "  Read 1 with adapter:                  26,431 (100.0%)\n",
      "  Read 2 with adapter:                  26,154 (98.9%)\n",
      "Pairs written (passing filters):        26,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,559,397 bp\n",
      "  Read 1:     7,798,700 bp\n",
      "  Read 2:     7,760,697 bp\n",
      "Total written (filtered):     14,191,559 bp (91.2%)\n",
      "  Read 1:     7,110,834 bp\n",
      "  Read 2:     7,080,725 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26431 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t318\t0.0\t2\t0 309 9\n",
      "26\t25918\t0.0\t2\t25348 559 11\n",
      "27\t155\t0.0\t2\t0 151 4\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t176\t0.0\t2\t0 165 11\n",
      "26\t25823\t0.0\t2\t23585 2145 93\n",
      "27\t126\t0.0\t2\t0 114 12\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S738_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S738_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S738_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S738_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S738_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S738_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S738_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S738_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,449 reads  @    104.8 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.66 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,449\n",
      "  Read 1 with adapter:                  44,420 (99.9%)\n",
      "  Read 2 with adapter:                  44,320 (99.7%)\n",
      "Pairs written (passing filters):        44,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,216,211 bp\n",
      "  Read 1:    13,110,278 bp\n",
      "  Read 2:    13,105,933 bp\n",
      "Total written (filtered):     23,909,409 bp (91.2%)\n",
      "  Read 1:    11,955,633 bp\n",
      "  Read 2:    11,953,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44420 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t556\t0.0\t2\t0 539 17\n",
      "26\t43572\t0.0\t2\t42632 928 12\n",
      "27\t215\t0.0\t2\t0 211 4\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t353\t0.0\t2\t0 324 29\n",
      "26\t43763\t0.0\t2\t40464 3205 94\n",
      "27\t170\t0.0\t2\t0 156 14\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S739_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S739_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S739_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S739_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S739_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S739_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S739_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S739_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        44,459 reads  @    114.2 µs/read;   0.53 M reads/minute\n",
      "Finished in 5.08 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,459\n",
      "  Read 1 with adapter:                  44,441 (100.0%)\n",
      "  Read 2 with adapter:                  44,300 (99.6%)\n",
      "Pairs written (passing filters):        44,459 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,227,665 bp\n",
      "  Read 1:    13,115,167 bp\n",
      "  Read 2:    13,112,498 bp\n",
      "Total written (filtered):     23,920,738 bp (91.2%)\n",
      "  Read 1:    11,959,941 bp\n",
      "  Read 2:    11,960,797 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t547\t0.0\t2\t0 536 11\n",
      "26\t43579\t0.0\t2\t42662 905 12\n",
      "27\t223\t0.0\t2\t0 216 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t325\t0.0\t2\t0 295 30\n",
      "26\t43754\t0.0\t2\t40086 3499 169\n",
      "27\t184\t0.0\t2\t0 170 14\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S740_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S740_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S740_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S740_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S740_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S740_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S740_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S740_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        54,222 reads  @     95.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.19 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,222\n",
      "  Read 1 with adapter:                  54,200 (100.0%)\n",
      "  Read 2 with adapter:                  54,127 (99.8%)\n",
      "Pairs written (passing filters):        54,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,988,958 bp\n",
      "  Read 1:    15,995,183 bp\n",
      "  Read 2:    15,993,775 bp\n",
      "Total written (filtered):     29,172,722 bp (91.2%)\n",
      "  Read 1:    14,586,155 bp\n",
      "  Read 2:    14,586,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t636\t0.0\t2\t0 621 15\n",
      "26\t53130\t0.0\t2\t51930 1166 34\n",
      "27\t292\t0.0\t2\t0 289 3\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54127 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t395\t0.0\t2\t0 367 28\n",
      "26\t53447\t0.0\t2\t49386 3939 122\n",
      "27\t173\t0.0\t2\t0 157 16\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S741_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S741_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S741_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S741_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S741_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S741_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S741_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S741_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,028 reads  @     97.4 µs/read;   0.62 M reads/minute\n",
      "Finished in 2.83 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,028\n",
      "  Read 1 with adapter:                  29,011 (99.9%)\n",
      "  Read 2 with adapter:                  28,820 (99.3%)\n",
      "Pairs written (passing filters):        29,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,122,926 bp\n",
      "  Read 1:     8,562,961 bp\n",
      "  Read 2:     8,559,965 bp\n",
      "Total written (filtered):     15,619,525 bp (91.2%)\n",
      "  Read 1:     7,808,714 bp\n",
      "  Read 2:     7,810,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t287\t0.0\t2\t0 277 10\n",
      "26\t28498\t0.0\t2\t27593 882 23\n",
      "27\t152\t0.0\t2\t0 147 5\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t276\t0.0\t2\t0 248 28\n",
      "26\t28393\t0.0\t2\t26275 2069 49\n",
      "27\t132\t0.0\t2\t0 122 10\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S742_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S742_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S742_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S742_75_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S742_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S742_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S742_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S742_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,045 reads  @    105.9 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.24 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,045\n",
      "  Read 1 with adapter:                  40,021 (99.9%)\n",
      "  Read 2 with adapter:                  39,949 (99.8%)\n",
      "Pairs written (passing filters):        40,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,621,966 bp\n",
      "  Read 1:    11,812,785 bp\n",
      "  Read 2:    11,809,181 bp\n",
      "Total written (filtered):     21,542,917 bp (91.2%)\n",
      "  Read 1:    10,772,318 bp\n",
      "  Read 2:    10,770,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40021 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t413\t0.0\t2\t0 404 9\n",
      "26\t39309\t0.0\t2\t38364 931 14\n",
      "27\t208\t0.0\t2\t0 203 5\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t293\t0.0\t2\t0 269 24\n",
      "26\t39427\t0.0\t2\t36368 2957 102\n",
      "27\t209\t0.0\t2\t0 180 29\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S743_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S743_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S743_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S743_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S743_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S743_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S743_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S743_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,110 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.77 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,110\n",
      "  Read 1 with adapter:                  43,091 (100.0%)\n",
      "  Read 2 with adapter:                  42,999 (99.7%)\n",
      "Pairs written (passing filters):        43,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,431,241 bp\n",
      "  Read 1:    12,716,935 bp\n",
      "  Read 2:    12,714,306 bp\n",
      "Total written (filtered):     23,192,997 bp (91.2%)\n",
      "  Read 1:    11,596,619 bp\n",
      "  Read 2:    11,596,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43091 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t350\t0.0\t2\t0 342 8\n",
      "26\t42430\t0.0\t2\t41616 805 9\n",
      "27\t234\t0.0\t2\t0 230 4\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t284\t0.0\t2\t0 254 30\n",
      "26\t42494\t0.0\t2\t39227 3152 115\n",
      "27\t156\t0.0\t2\t0 150 6\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S744_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S744_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S744_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S744_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S744_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S744_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S744_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S744_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,792 reads  @     96.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 2.87 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,792\n",
      "  Read 1 with adapter:                  29,769 (99.9%)\n",
      "  Read 2 with adapter:                  29,366 (98.6%)\n",
      "Pairs written (passing filters):        29,789 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,519,625 bp\n",
      "  Read 1:     8,788,627 bp\n",
      "  Read 2:     8,730,998 bp\n",
      "Total written (filtered):     15,981,459 bp (91.2%)\n",
      "  Read 1:     8,013,922 bp\n",
      "  Read 2:     7,967,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29769 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t276\t0.0\t2\t0 266 10\n",
      "26\t29275\t0.0\t2\t28652 612 11\n",
      "27\t172\t0.0\t2\t0 167 5\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t191\t0.0\t2\t0 166 25\n",
      "26\t29017\t0.0\t2\t26479 2388 150\n",
      "27\t136\t0.0\t2\t0 121 15\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S746_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S746_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S746_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S746_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S746_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S746_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S746_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S746_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,920 reads  @    108.4 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.44 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,920\n",
      "  Read 1 with adapter:                  40,899 (99.9%)\n",
      "  Read 2 with adapter:                  40,791 (99.7%)\n",
      "Pairs written (passing filters):        40,920 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,133,300 bp\n",
      "  Read 1:    12,071,302 bp\n",
      "  Read 2:    12,061,998 bp\n",
      "Total written (filtered):     22,009,600 bp (91.2%)\n",
      "  Read 1:    11,008,002 bp\n",
      "  Read 2:    11,001,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t350\t0.0\t2\t0 337 13\n",
      "26\t40271\t0.0\t2\t39440 818 13\n",
      "27\t224\t0.0\t2\t0 220 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t345\t0.0\t2\t0 319 26\n",
      "26\t40245\t0.0\t2\t37042 3099 104\n",
      "27\t155\t0.0\t2\t0 143 12\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S747_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S747_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S747_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S747_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S747_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S747_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S747_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S747_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,213 reads  @    104.9 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.12 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,213\n",
      "  Read 1 with adapter:                  39,187 (99.9%)\n",
      "  Read 2 with adapter:                  39,110 (99.7%)\n",
      "Pairs written (passing filters):        39,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,131,791 bp\n",
      "  Read 1:    11,567,681 bp\n",
      "  Read 2:    11,564,110 bp\n",
      "Total written (filtered):     21,096,251 bp (91.2%)\n",
      "  Read 1:    10,548,921 bp\n",
      "  Read 2:    10,547,330 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t381\t0.0\t2\t0 373 8\n",
      "26\t38544\t0.0\t2\t37710 820 14\n",
      "27\t197\t0.0\t2\t0 192 5\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t287\t0.0\t2\t0 264 23\n",
      "26\t38619\t0.0\t2\t35596 2896 127\n",
      "27\t153\t0.0\t2\t0 141 12\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S748_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S748_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S748_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S748_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S748_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S748_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S748_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S748_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        35,687 reads  @    102.6 µs/read;   0.58 M reads/minute\n",
      "Finished in 3.66 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,687\n",
      "  Read 1 with adapter:                  35,664 (99.9%)\n",
      "  Read 2 with adapter:                  35,640 (99.9%)\n",
      "Pairs written (passing filters):        35,687 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,054,744 bp\n",
      "  Read 1:    10,527,176 bp\n",
      "  Read 2:    10,527,568 bp\n",
      "Total written (filtered):     19,200,982 bp (91.2%)\n",
      "  Read 1:     9,599,940 bp\n",
      "  Read 2:     9,601,042 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t322\t0.0\t2\t0 313 9\n",
      "26\t35092\t0.0\t2\t34299 774 19\n",
      "27\t178\t0.0\t2\t0 174 4\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35640 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t284\t0.0\t2\t0 260 24\n",
      "26\t35161\t0.0\t2\t32519 2581 61\n",
      "27\t160\t0.0\t2\t0 149 11\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S749_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S749_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S749_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S749_87_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S749_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S749_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S749_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S749_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        46,295 reads  @    105.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.88 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,295\n",
      "  Read 1 with adapter:                  46,279 (100.0%)\n",
      "  Read 2 with adapter:                  46,118 (99.6%)\n",
      "Pairs written (passing filters):        46,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,304,578 bp\n",
      "  Read 1:    13,655,678 bp\n",
      "  Read 2:    13,648,900 bp\n",
      "Total written (filtered):     24,902,636 bp (91.2%)\n",
      "  Read 1:    12,452,617 bp\n",
      "  Read 2:    12,450,019 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t522\t0.0\t2\t0 510 12\n",
      "26\t45432\t0.0\t2\t44281 1129 22\n",
      "27\t229\t0.0\t2\t0 219 10\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t495\t0.0\t2\t0 446 49\n",
      "26\t45296\t0.0\t2\t41113 3928 255\n",
      "27\t258\t0.0\t2\t0 231 27\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S750_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S750_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S750_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S750_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S750_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S750_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S750_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S750_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,821 reads  @     90.7 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.16 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,821\n",
      "  Read 1 with adapter:                  45,803 (100.0%)\n",
      "  Read 2 with adapter:                  45,734 (99.8%)\n",
      "Pairs written (passing filters):        45,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,032,043 bp\n",
      "  Read 1:    13,516,163 bp\n",
      "  Read 2:    13,515,880 bp\n",
      "Total written (filtered):     24,652,331 bp (91.2%)\n",
      "  Read 1:    12,325,459 bp\n",
      "  Read 2:    12,326,872 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t505\t0.0\t2\t0 491 14\n",
      "26\t44960\t0.0\t2\t43836 1102 22\n",
      "27\t277\t0.0\t2\t0 267 10\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45734 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t366\t0.0\t2\t0 337 29\n",
      "26\t45072\t0.0\t2\t41499 3439 134\n",
      "27\t238\t0.0\t2\t0 217 21\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S751_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S751_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S751_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S751_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S751_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S751_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S751_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S751_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           875 reads  @    161.4 µs/read;   0.37 M reads/minute\n",
      "Finished in 0.14 s (164 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                875\n",
      "  Read 1 with adapter:                     380 (43.4%)\n",
      "  Read 2 with adapter:                     821 (93.8%)\n",
      "Pairs written (passing filters):           875 (100.0%)\n",
      "\n",
      "Total basepairs processed:       515,038 bp\n",
      "  Read 1:       258,124 bp\n",
      "  Read 2:       256,914 bp\n",
      "Total written (filtered):        483,814 bp (93.9%)\n",
      "  Read 1:       248,237 bp\n",
      "  Read 2:       235,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t8\t0.0\t2\t0 7 1\n",
      "26\t357\t0.0\t2\t349 7 1\n",
      "27\t11\t0.0\t2\t0 4 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t14\t0.0\t2\t0 13 1\n",
      "26\t801\t0.0\t2\t712 86 3\n",
      "27\t3\t0.0\t2\t0 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S752_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S752_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S752_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S752_93_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S752_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S752_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S752_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S752_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,144 reads  @    109.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.71 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,144\n",
      "  Read 1 with adapter:                  42,927 (99.5%)\n",
      "  Read 2 with adapter:                  42,868 (99.4%)\n",
      "Pairs written (passing filters):        43,144 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,400,988 bp\n",
      "  Read 1:    12,677,782 bp\n",
      "  Read 2:    12,723,206 bp\n",
      "Total written (filtered):     23,170,853 bp (91.2%)\n",
      "  Read 1:    11,561,905 bp\n",
      "  Read 2:    11,608,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42927 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t501\t0.0\t2\t0 486 15\n",
      "26\t42121\t0.0\t2\t40792 1282 47\n",
      "27\t230\t0.0\t2\t0 227 3\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t469\t0.0\t2\t0 434 35\n",
      "26\t42178\t0.0\t2\t39102 3014 62\n",
      "27\t187\t0.0\t2\t0 177 10\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S753_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S753_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S753_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S753_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S753_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S753_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S753_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S753_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,560 reads  @     98.5 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.10 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,560\n",
      "  Read 1 with adapter:                  41,376 (99.6%)\n",
      "  Read 2 with adapter:                  41,471 (99.8%)\n",
      "Pairs written (passing filters):        41,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,474,020 bp\n",
      "  Read 1:    12,216,205 bp\n",
      "  Read 2:    12,257,815 bp\n",
      "Total written (filtered):     22,320,310 bp (91.2%)\n",
      "  Read 1:    11,140,611 bp\n",
      "  Read 2:    11,179,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t461\t0.0\t2\t0 449 12\n",
      "26\t40641\t0.0\t2\t39666 958 17\n",
      "27\t217\t0.0\t2\t0 210 7\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t309\t0.0\t2\t0 284 25\n",
      "26\t40943\t0.0\t2\t37748 3093 102\n",
      "27\t187\t0.0\t2\t0 167 20\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S754_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S754_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S754_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S754_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S754_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S754_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S754_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S754_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,227 reads  @     84.9 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.33 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,227\n",
      "  Read 1 with adapter:                  38,969 (99.3%)\n",
      "  Read 2 with adapter:                  39,167 (99.8%)\n",
      "Pairs written (passing filters):        39,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,079,214 bp\n",
      "  Read 1:    11,508,458 bp\n",
      "  Read 2:    11,570,756 bp\n",
      "Total written (filtered):     21,047,784 bp (91.2%)\n",
      "  Read 1:    10,495,317 bp\n",
      "  Read 2:    10,552,467 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t367\t0.0\t2\t0 355 12\n",
      "26\t38297\t0.0\t2\t37476 802 19\n",
      "27\t224\t0.0\t2\t0 220 4\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t234\t0.0\t2\t0 214 20\n",
      "26\t38725\t0.0\t2\t35769 2865 91\n",
      "27\t187\t0.0\t2\t0 171 16\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S755_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S755_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S755_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S755_99_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S755_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S755_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S755_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S755_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,010 reads  @     83.2 µs/read;   0.72 M reads/minute\n",
      "Finished in 2.50 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,010\n",
      "  Read 1 with adapter:                  29,956 (99.8%)\n",
      "  Read 2 with adapter:                  29,747 (99.1%)\n",
      "Pairs written (passing filters):        30,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,650,270 bp\n",
      "  Read 1:     8,842,186 bp\n",
      "  Read 2:     8,808,084 bp\n",
      "Total written (filtered):     16,098,089 bp (91.2%)\n",
      "  Read 1:     8,063,381 bp\n",
      "  Read 2:     8,034,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t306\t0.0\t2\t0 287 19\n",
      "26\t29427\t0.0\t2\t28793 620 14\n",
      "27\t139\t0.0\t2\t0 136 3\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t173\t0.0\t2\t0 156 17\n",
      "26\t29424\t0.0\t2\t26784 2496 144\n",
      "27\t137\t0.0\t2\t0 122 15\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S757_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S757_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S757_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S757_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S757_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S757_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S757_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S757_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,370 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.33 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,370\n",
      "  Read 1 with adapter:                  52,192 (99.7%)\n",
      "  Read 2 with adapter:                  52,271 (99.8%)\n",
      "Pairs written (passing filters):        52,370 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,856,920 bp\n",
      "  Read 1:    15,411,377 bp\n",
      "  Read 2:    15,445,543 bp\n",
      "Total written (filtered):     28,141,158 bp (91.2%)\n",
      "  Read 1:    14,054,475 bp\n",
      "  Read 2:    14,086,683 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t548\t0.0\t2\t0 535 13\n",
      "26\t51232\t0.0\t2\t50181 1031 20\n",
      "27\t294\t0.0\t2\t0 290 4\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t443\t0.0\t2\t0 410 33\n",
      "26\t51561\t0.0\t2\t47608 3828 125\n",
      "27\t241\t0.0\t2\t0 227 14\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S758_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S758_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S758_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S758_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S758_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S758_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S758_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S758_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,711 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.35 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,711\n",
      "  Read 1 with adapter:                  38,619 (99.8%)\n",
      "  Read 2 with adapter:                  38,637 (99.8%)\n",
      "Pairs written (passing filters):        38,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,818,761 bp\n",
      "  Read 1:    11,402,673 bp\n",
      "  Read 2:    11,416,088 bp\n",
      "Total written (filtered):     20,810,323 bp (91.2%)\n",
      "  Read 1:    10,398,656 bp\n",
      "  Read 2:    10,411,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38619 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t410\t0.0\t2\t0 404 6\n",
      "26\t37899\t0.0\t2\t37073 807 19\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t309\t0.0\t2\t0 288 21\n",
      "26\t38140\t0.0\t2\t35317 2740 83\n",
      "27\t156\t0.0\t2\t0 145 11\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S759_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S759_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S759_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S759_105_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S759_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S759_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S759_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S759_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,101 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.23 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,101\n",
      "  Read 1 with adapter:                  43,874 (99.5%)\n",
      "  Read 2 with adapter:                  44,050 (99.9%)\n",
      "Pairs written (passing filters):        44,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,971,138 bp\n",
      "  Read 1:    12,961,968 bp\n",
      "  Read 2:    13,009,170 bp\n",
      "Total written (filtered):     23,685,368 bp (91.2%)\n",
      "  Read 1:    11,821,395 bp\n",
      "  Read 2:    11,863,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t475\t0.0\t2\t0 458 17\n",
      "26\t43079\t0.0\t2\t42058 996 25\n",
      "27\t236\t0.0\t2\t0 226 10\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t343\t0.0\t2\t0 308 35\n",
      "26\t43467\t0.0\t2\t40118 3250 99\n",
      "27\t188\t0.0\t2\t0 183 5\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S760_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S760_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S760_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S760_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S760_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S760_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S760_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S760_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        36,849 reads  @    107.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 3.94 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,849\n",
      "  Read 1 with adapter:                  36,772 (99.8%)\n",
      "  Read 2 with adapter:                  36,556 (99.2%)\n",
      "Pairs written (passing filters):        36,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,727,071 bp\n",
      "  Read 1:    10,860,972 bp\n",
      "  Read 2:    10,866,099 bp\n",
      "Total written (filtered):     19,820,950 bp (91.2%)\n",
      "  Read 1:     9,905,082 bp\n",
      "  Read 2:     9,915,868 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t441\t0.0\t2\t0 431 10\n",
      "26\t36068\t0.0\t2\t35182 872 14\n",
      "27\t223\t0.0\t2\t0 219 4\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36556 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t407\t0.0\t2\t0 374 33\n",
      "26\t35922\t0.0\t2\t33213 2654 55\n",
      "27\t200\t0.0\t2\t0 184 16\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S761_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S761_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S761_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S761_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S761_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S761_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S761_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S761_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        35,414 reads  @    114.9 µs/read;   0.52 M reads/minute\n",
      "Finished in 4.07 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,414\n",
      "  Read 1 with adapter:                  35,315 (99.7%)\n",
      "  Read 2 with adapter:                  35,356 (99.8%)\n",
      "Pairs written (passing filters):        35,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,879,187 bp\n",
      "  Read 1:    10,433,887 bp\n",
      "  Read 2:    10,445,300 bp\n",
      "Total written (filtered):     19,042,021 bp (91.2%)\n",
      "  Read 1:     9,515,849 bp\n",
      "  Read 2:     9,526,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t409\t0.0\t2\t0 396 13\n",
      "26\t34625\t0.0\t2\t33739 863 23\n",
      "27\t225\t0.0\t2\t0 217 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t279\t0.0\t2\t0 263 16\n",
      "26\t34904\t0.0\t2\t32093 2707 104\n",
      "27\t147\t0.0\t2\t0 134 13\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S762_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S762_111_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S762_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S762_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        38,767 reads  @     98.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 3.84 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,767\n",
      "  Read 1 with adapter:                  38,715 (99.9%)\n",
      "  Read 2 with adapter:                  38,692 (99.8%)\n",
      "Pairs written (passing filters):        38,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,865,192 bp\n",
      "  Read 1:    11,432,045 bp\n",
      "  Read 2:    11,433,147 bp\n",
      "Total written (filtered):     20,852,769 bp (91.2%)\n",
      "  Read 1:    10,425,552 bp\n",
      "  Read 2:    10,427,217 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t423\t0.0\t2\t0 415 8\n",
      "26\t37962\t0.0\t2\t37002 943 17\n",
      "27\t250\t0.0\t2\t0 242 8\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t257\t0.0\t2\t0 237 20\n",
      "26\t38213\t0.0\t2\t35142 2971 100\n",
      "27\t205\t0.0\t2\t0 183 22\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S763_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S763_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S763_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S763_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S763_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S763_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S763_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S763_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,363 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.71 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,363\n",
      "  Read 1 with adapter:                  31,343 (99.9%)\n",
      "  Read 2 with adapter:                  30,609 (97.6%)\n",
      "Pairs written (passing filters):        31,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,334,127 bp\n",
      "  Read 1:     9,251,516 bp\n",
      "  Read 2:     9,082,611 bp\n",
      "Total written (filtered):     16,722,639 bp (91.2%)\n",
      "  Read 1:     8,435,818 bp\n",
      "  Read 2:     8,286,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31343 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t319\t0.0\t2\t0 312 7\n",
      "26\t30758\t0.0\t2\t30178 568 12\n",
      "27\t172\t0.0\t2\t0 169 3\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30609 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t189\t0.0\t2\t0 177 12\n",
      "26\t30250\t0.0\t2\t27523 2544 183\n",
      "27\t155\t0.0\t2\t0 138 17\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S765_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S765_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S765_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S765_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S765_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S765_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S765_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S765_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,327 reads  @     79.9 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.15 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,327\n",
      "  Read 1 with adapter:                  39,306 (99.9%)\n",
      "  Read 2 with adapter:                  39,139 (99.5%)\n",
      "Pairs written (passing filters):        39,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,193,528 bp\n",
      "  Read 1:    11,600,903 bp\n",
      "  Read 2:    11,592,625 bp\n",
      "Total written (filtered):     21,153,948 bp (91.2%)\n",
      "  Read 1:    10,578,770 bp\n",
      "  Read 2:    10,575,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t402\t0.0\t2\t0 397 5\n",
      "26\t38594\t0.0\t2\t37811 771 12\n",
      "27\t250\t0.0\t2\t0 244 6\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t323\t0.0\t2\t0 292 31\n",
      "26\t38629\t0.0\t2\t35536 2971 122\n",
      "27\t138\t0.0\t2\t0 129 9\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S766_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S766_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S766_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S766_117_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S766_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S766_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S766_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S766_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,203 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.41 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,203\n",
      "  Read 1 with adapter:                  41,179 (99.9%)\n",
      "  Read 2 with adapter:                  41,079 (99.7%)\n",
      "Pairs written (passing filters):        41,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,305,063 bp\n",
      "  Read 1:    12,154,862 bp\n",
      "  Read 2:    12,150,201 bp\n",
      "Total written (filtered):     22,166,512 bp (91.2%)\n",
      "  Read 1:    11,084,244 bp\n",
      "  Read 2:    11,082,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t443\t0.0\t2\t0 431 12\n",
      "26\t40353\t0.0\t2\t39544 799 10\n",
      "27\t291\t0.0\t2\t0 287 4\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t302\t0.0\t2\t0 280 22\n",
      "26\t40572\t0.0\t2\t37389 3057 126\n",
      "27\t173\t0.0\t2\t0 154 19\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S767_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S767_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S767_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S767_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S767_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S767_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S767_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S767_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,391 reads  @     91.4 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.06 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,391\n",
      "  Read 1 with adapter:                  44,362 (99.9%)\n",
      "  Read 2 with adapter:                  44,339 (99.9%)\n",
      "Pairs written (passing filters):        44,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,188,393 bp\n",
      "  Read 1:    13,093,759 bp\n",
      "  Read 2:    13,094,634 bp\n",
      "Total written (filtered):     23,882,445 bp (91.2%)\n",
      "  Read 1:    11,940,492 bp\n",
      "  Read 2:    11,941,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t488\t0.0\t2\t0 475 13\n",
      "26\t43509\t0.0\t2\t42557 941 11\n",
      "27\t291\t0.0\t2\t0 288 3\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44339 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t337\t0.0\t2\t0 300 37\n",
      "26\t43780\t0.0\t2\t40456 3208 116\n",
      "27\t168\t0.0\t2\t0 160 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S768_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S768_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S768_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S768_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S768_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S768_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S768_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S768_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        35,297 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.43 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,297\n",
      "  Read 1 with adapter:                  35,232 (99.8%)\n",
      "  Read 2 with adapter:                  35,044 (99.3%)\n",
      "Pairs written (passing filters):        35,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,812,249 bp\n",
      "  Read 1:    10,403,791 bp\n",
      "  Read 2:    10,408,458 bp\n",
      "Total written (filtered):     18,985,408 bp (91.2%)\n",
      "  Read 1:     9,487,890 bp\n",
      "  Read 2:     9,497,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t306\t0.0\t2\t0 298 8\n",
      "26\t34714\t0.0\t2\t33965 741 8\n",
      "27\t161\t0.0\t2\t0 158 3\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t345\t0.0\t2\t0 322 23\n",
      "26\t34523\t0.0\t2\t32056 2414 53\n",
      "27\t151\t0.0\t2\t0 147 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S769_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S769_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S769_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S769_123_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S769_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S769_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S769_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S769_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,490 reads  @     94.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.82 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,490\n",
      "  Read 1 with adapter:                  40,430 (99.9%)\n",
      "  Read 2 with adapter:                  40,432 (99.9%)\n",
      "Pairs written (passing filters):        40,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,880,545 bp\n",
      "  Read 1:    11,937,519 bp\n",
      "  Read 2:    11,943,026 bp\n",
      "Total written (filtered):     21,778,295 bp (91.2%)\n",
      "  Read 1:    10,886,403 bp\n",
      "  Read 2:    10,891,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t327\t0.0\t2\t0 315 12\n",
      "26\t39829\t0.0\t2\t39014 805 10\n",
      "27\t217\t0.0\t2\t0 210 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t312\t0.0\t2\t0 285 27\n",
      "26\t39890\t0.0\t2\t36794 3002 94\n",
      "27\t214\t0.0\t2\t0 195 19\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S770_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S770_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S770_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S770_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S770_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S770_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S770_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S770_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,003 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.37 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,003\n",
      "  Read 1 with adapter:                  40,956 (99.9%)\n",
      "  Read 2 with adapter:                  40,942 (99.9%)\n",
      "Pairs written (passing filters):        41,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,186,857 bp\n",
      "  Read 1:    12,091,471 bp\n",
      "  Read 2:    12,095,386 bp\n",
      "Total written (filtered):     22,057,504 bp (91.2%)\n",
      "  Read 1:    11,026,522 bp\n",
      "  Read 2:    11,030,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t300\t0.0\t2\t0 291 9\n",
      "26\t40316\t0.0\t2\t39466 840 10\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t246\t0.0\t2\t0 225 21\n",
      "26\t40526\t0.0\t2\t37602 2820 104\n",
      "27\t158\t0.0\t2\t0 140 18\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S771_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S771_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S771_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S771_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S771_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S771_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S771_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S771_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        27,386 reads  @    102.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 2.82 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,386\n",
      "  Read 1 with adapter:                  27,324 (99.8%)\n",
      "  Read 2 with adapter:                  27,165 (99.2%)\n",
      "Pairs written (passing filters):        27,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,132,991 bp\n",
      "  Read 1:     8,072,347 bp\n",
      "  Read 2:     8,060,644 bp\n",
      "Total written (filtered):     14,716,341 bp (91.2%)\n",
      "  Read 1:     7,361,935 bp\n",
      "  Read 2:     7,354,406 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t214\t0.0\t2\t0 206 8\n",
      "26\t26905\t0.0\t2\t26350 545 10\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27165 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t165\t0.0\t2\t0 151 14\n",
      "26\t26876\t0.0\t2\t24625 2126 125\n",
      "27\t111\t0.0\t2\t0 103 8\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S773_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S773_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S773_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S773_129_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S773_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S773_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S773_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S773_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,371 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.38 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,371\n",
      "  Read 1 with adapter:                  39,123 (99.4%)\n",
      "  Read 2 with adapter:                  39,306 (99.8%)\n",
      "Pairs written (passing filters):        39,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,183,696 bp\n",
      "  Read 1:    11,570,824 bp\n",
      "  Read 2:    11,612,872 bp\n",
      "Total written (filtered):     21,144,844 bp (91.2%)\n",
      "  Read 1:    10,553,761 bp\n",
      "  Read 2:    10,591,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t381\t0.0\t2\t0 372 9\n",
      "26\t38472\t0.0\t2\t37621 832 19\n",
      "27\t218\t0.0\t2\t0 207 11\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t337\t0.0\t2\t0 307 30\n",
      "26\t38765\t0.0\t2\t35827 2850 88\n",
      "27\t178\t0.0\t2\t0 166 12\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S774_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S774_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S774_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S774_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S774_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S774_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S774_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S774_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,603 reads  @     79.5 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.83 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,603\n",
      "  Read 1 with adapter:                  35,403 (99.4%)\n",
      "  Read 2 with adapter:                  35,535 (99.8%)\n",
      "Pairs written (passing filters):        35,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,968,741 bp\n",
      "  Read 1:    10,467,651 bp\n",
      "  Read 2:    10,501,090 bp\n",
      "Total written (filtered):     19,124,628 bp (91.2%)\n",
      "  Read 1:     9,547,384 bp\n",
      "  Read 2:     9,577,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t336\t0.0\t2\t0 326 10\n",
      "26\t34898\t0.0\t2\t33964 920 14\n",
      "27\t145\t0.0\t2\t0 142 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t267\t0.0\t2\t0 247 20\n",
      "26\t35065\t0.0\t2\t32324 2642 99\n",
      "27\t163\t0.0\t2\t0 152 11\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S775_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S775_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S775_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S775_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S775_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S775_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S775_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S775_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        59,879 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.33 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,879\n",
      "  Read 1 with adapter:                  59,815 (99.9%)\n",
      "  Read 2 with adapter:                  59,811 (99.9%)\n",
      "Pairs written (passing filters):        59,879 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,319,793 bp\n",
      "  Read 1:    17,657,770 bp\n",
      "  Read 2:    17,662,023 bp\n",
      "Total written (filtered):     32,209,765 bp (91.2%)\n",
      "  Read 1:    16,102,589 bp\n",
      "  Read 2:    16,107,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t474\t0.0\t2\t0 463 11\n",
      "26\t58876\t0.0\t2\t57645 1203 28\n",
      "27\t357\t0.0\t2\t0 341 16\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t481\t0.0\t2\t0 442 39\n",
      "26\t59057\t0.0\t2\t54675 4290 92\n",
      "27\t224\t0.0\t2\t0 207 17\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S776_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S776_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S776_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S776_135_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S776_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S776_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S776_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S776_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,407 reads  @    104.6 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.39 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,407\n",
      "  Read 1 with adapter:                  32,358 (99.8%)\n",
      "  Read 2 with adapter:                  32,179 (99.3%)\n",
      "Pairs written (passing filters):        32,407 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,114,269 bp\n",
      "  Read 1:     9,557,784 bp\n",
      "  Read 2:     9,556,485 bp\n",
      "Total written (filtered):     17,436,744 bp (91.2%)\n",
      "  Read 1:     8,716,680 bp\n",
      "  Read 2:     8,720,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t385\t0.0\t2\t0 372 13\n",
      "26\t31770\t0.0\t2\t30793 948 29\n",
      "27\t165\t0.0\t2\t0 154 11\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t391\t0.0\t2\t0 366 25\n",
      "26\t31575\t0.0\t2\t29154 2368 53\n",
      "27\t176\t0.0\t2\t0 160 16\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S777_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S777_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S777_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S777_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S777_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S777_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S777_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S777_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,683 reads  @     97.7 µs/read;   0.61 M reads/minute\n",
      "Finished in 3.59 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,683\n",
      "  Read 1 with adapter:                  36,646 (99.9%)\n",
      "  Read 2 with adapter:                  36,527 (99.6%)\n",
      "Pairs written (passing filters):        36,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,630,872 bp\n",
      "  Read 1:    10,820,292 bp\n",
      "  Read 2:    10,810,580 bp\n",
      "Total written (filtered):     19,728,540 bp (91.2%)\n",
      "  Read 1:     9,867,515 bp\n",
      "  Read 2:     9,861,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t307\t0.0\t2\t0 297 10\n",
      "26\t36061\t0.0\t2\t35310 738 13\n",
      "27\t204\t0.0\t2\t0 197 7\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36527 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t299\t0.0\t2\t0 274 25\n",
      "26\t36057\t0.0\t2\t33214 2737 106\n",
      "27\t146\t0.0\t2\t0 136 10\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S778_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S778_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S778_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S778_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S778_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S778_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S778_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S778_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        38,833 reads  @     91.1 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.54 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,833\n",
      "  Read 1 with adapter:                  38,811 (99.9%)\n",
      "  Read 2 with adapter:                  38,777 (99.9%)\n",
      "Pairs written (passing filters):        38,833 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,909,112 bp\n",
      "  Read 1:    11,455,300 bp\n",
      "  Read 2:    11,453,812 bp\n",
      "Total written (filtered):     20,891,905 bp (91.2%)\n",
      "  Read 1:    10,446,190 bp\n",
      "  Read 2:    10,445,715 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t344\t0.0\t2\t0 335 9\n",
      "26\t38155\t0.0\t2\t37312 824 19\n",
      "27\t212\t0.0\t2\t0 201 11\n",
      "28\t89\t0.0\t2\t0 0 89\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t268\t0.0\t2\t0 255 13\n",
      "26\t38313\t0.0\t2\t35338 2882 93\n",
      "27\t181\t0.0\t2\t0 166 15\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S779_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S779_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S779_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S779_141_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S779_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S779_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S779_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S779_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        29,665 reads  @    106.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.15 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,665\n",
      "  Read 1 with adapter:                  29,639 (99.9%)\n",
      "  Read 2 with adapter:                  29,481 (99.4%)\n",
      "Pairs written (passing filters):        29,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,488,829 bp\n",
      "  Read 1:     8,750,342 bp\n",
      "  Read 2:     8,738,487 bp\n",
      "Total written (filtered):     15,951,801 bp (91.2%)\n",
      "  Read 1:     7,979,787 bp\n",
      "  Read 2:     7,972,014 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t251\t0.0\t2\t0 245 6\n",
      "26\t29198\t0.0\t2\t28631 551 16\n",
      "27\t140\t0.0\t2\t0 137 3\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t188\t0.0\t2\t0 167 21\n",
      "26\t29136\t0.0\t2\t26555 2415 166\n",
      "27\t139\t0.0\t2\t0 121 18\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S781_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S781_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S781_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S781_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S781_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S781_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S781_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S781_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,897 reads  @     85.4 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.16 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,897\n",
      "  Read 1 with adapter:                  36,868 (99.9%)\n",
      "  Read 2 with adapter:                  36,686 (99.4%)\n",
      "Pairs written (passing filters):        36,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,741,770 bp\n",
      "  Read 1:    10,883,819 bp\n",
      "  Read 2:    10,857,951 bp\n",
      "Total written (filtered):     19,829,696 bp (91.2%)\n",
      "  Read 1:     9,925,444 bp\n",
      "  Read 2:     9,904,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t369\t0.0\t2\t0 363 6\n",
      "26\t36323\t0.0\t2\t35553 759 11\n",
      "27\t160\t0.0\t2\t0 152 8\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36686 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t323\t0.0\t2\t0 303 20\n",
      "26\t36195\t0.0\t2\t33514 2590 91\n",
      "27\t130\t0.0\t2\t0 121 9\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S782_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S782_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S782_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S782_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S782_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S782_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S782_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S782_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        68,313 reads  @     90.9 µs/read;   0.66 M reads/minute\n",
      "Finished in 6.21 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,313\n",
      "  Read 1 with adapter:                  68,261 (99.9%)\n",
      "  Read 2 with adapter:                  68,152 (99.8%)\n",
      "Pairs written (passing filters):        68,313 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,295,829 bp\n",
      "  Read 1:    20,148,308 bp\n",
      "  Read 2:    20,147,521 bp\n",
      "Total written (filtered):     36,749,586 bp (91.2%)\n",
      "  Read 1:    18,373,830 bp\n",
      "  Read 2:    18,375,756 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t636\t0.0\t2\t0 619 17\n",
      "26\t67233\t0.0\t2\t65778 1429 26\n",
      "27\t324\t0.0\t2\t0 320 4\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t497\t0.0\t2\t0 455 42\n",
      "26\t67332\t0.0\t2\t62256 4908 168\n",
      "27\t284\t0.0\t2\t0 265 19\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S783_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S783_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S783_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S783_147_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S783_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S783_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S783_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S783_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,217 reads  @     94.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.32 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,217\n",
      "  Read 1 with adapter:                  35,180 (99.9%)\n",
      "  Read 2 with adapter:                  35,161 (99.8%)\n",
      "Pairs written (passing filters):        35,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,774,708 bp\n",
      "  Read 1:    10,386,724 bp\n",
      "  Read 2:    10,387,984 bp\n",
      "Total written (filtered):     18,946,146 bp (91.2%)\n",
      "  Read 1:     9,472,211 bp\n",
      "  Read 2:     9,473,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t364\t0.0\t2\t0 353 11\n",
      "26\t34610\t0.0\t2\t33827 774 9\n",
      "27\t163\t0.0\t2\t0 156 7\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t299\t0.0\t2\t0 278 21\n",
      "26\t34667\t0.0\t2\t31974 2598 95\n",
      "27\t168\t0.0\t2\t0 149 19\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S784_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S784_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S784_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S784_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S784_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S784_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S784_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S784_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,765 reads  @    102.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.16 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,765\n",
      "  Read 1 with adapter:                  40,743 (99.9%)\n",
      "  Read 2 with adapter:                  40,521 (99.4%)\n",
      "Pairs written (passing filters):        40,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,047,607 bp\n",
      "  Read 1:    12,025,655 bp\n",
      "  Read 2:    12,021,952 bp\n",
      "Total written (filtered):     21,934,972 bp (91.2%)\n",
      "  Read 1:    10,966,311 bp\n",
      "  Read 2:    10,968,661 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40743 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t338\t0.0\t2\t0 330 8\n",
      "26\t40094\t0.0\t2\t39037 1021 36\n",
      "27\t198\t0.0\t2\t0 194 4\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40521 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t431\t0.0\t2\t0 403 28\n",
      "26\t39846\t0.0\t2\t36874 2906 66\n",
      "27\t216\t0.0\t2\t0 198 18\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S785_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S785_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S785_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S785_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S785_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S785_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S785_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S785_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,441 reads  @     91.0 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.78 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,441\n",
      "  Read 1 with adapter:                  41,421 (100.0%)\n",
      "  Read 2 with adapter:                  41,378 (99.8%)\n",
      "Pairs written (passing filters):        41,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,448,347 bp\n",
      "  Read 1:    12,224,999 bp\n",
      "  Read 2:    12,223,348 bp\n",
      "Total written (filtered):     22,295,797 bp (91.2%)\n",
      "  Read 1:    11,148,120 bp\n",
      "  Read 2:    11,147,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41421 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t341\t0.0\t2\t0 331 10\n",
      "26\t40779\t0.0\t2\t39962 800 17\n",
      "27\t244\t0.0\t2\t0 239 5\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t321\t0.0\t2\t0 299 22\n",
      "26\t40855\t0.0\t2\t37784 2994 77\n",
      "27\t184\t0.0\t2\t0 171 13\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S786_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S786_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S786_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S786_153_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S786_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S786_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S786_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S786_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,931 reads  @     88.3 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.53 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,931\n",
      "  Read 1 with adapter:                  39,912 (100.0%)\n",
      "  Read 2 with adapter:                  39,807 (99.7%)\n",
      "Pairs written (passing filters):        39,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,557,533 bp\n",
      "  Read 1:    11,779,627 bp\n",
      "  Read 2:    11,777,906 bp\n",
      "Total written (filtered):     21,484,866 bp (91.2%)\n",
      "  Read 1:    10,741,907 bp\n",
      "  Read 2:    10,742,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39912 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t305\t0.0\t2\t0 293 12\n",
      "26\t39292\t0.0\t2\t38509 771 12\n",
      "27\t249\t0.0\t2\t0 241 8\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t279\t0.0\t2\t0 253 26\n",
      "26\t39329\t0.0\t2\t35911 3256 162\n",
      "27\t126\t0.0\t2\t0 114 12\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S787_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S787_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S787_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S787_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S787_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S787_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S787_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S787_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        26,527 reads  @    104.8 µs/read;   0.57 M reads/minute\n",
      "Finished in 2.78 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,527\n",
      "  Read 1 with adapter:                  26,499 (99.9%)\n",
      "  Read 2 with adapter:                  26,133 (98.5%)\n",
      "Pairs written (passing filters):        26,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,595,566 bp\n",
      "  Read 1:     7,825,250 bp\n",
      "  Read 2:     7,770,316 bp\n",
      "Total written (filtered):     14,226,132 bp (91.2%)\n",
      "  Read 1:     7,135,238 bp\n",
      "  Read 2:     7,090,894 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t211\t0.0\t2\t0 206 5\n",
      "26\t26110\t0.0\t2\t25549 554 7\n",
      "27\t143\t0.0\t2\t0 140 3\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t167\t0.0\t2\t0 149 18\n",
      "26\t25818\t0.0\t2\t23472 2190 156\n",
      "27\t121\t0.0\t2\t0 107 14\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S789_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S789_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S789_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S789_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S789_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S789_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S789_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S789_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,472 reads  @    108.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.70 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,472\n",
      "  Read 1 with adapter:                  43,435 (99.9%)\n",
      "  Read 2 with adapter:                  43,337 (99.7%)\n",
      "Pairs written (passing filters):        43,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,638,802 bp\n",
      "  Read 1:    12,823,331 bp\n",
      "  Read 2:    12,815,471 bp\n",
      "Total written (filtered):     23,383,003 bp (91.2%)\n",
      "  Read 1:    11,694,151 bp\n",
      "  Read 2:    11,688,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t379\t0.0\t2\t0 368 11\n",
      "26\t42803\t0.0\t2\t41866 924 13\n",
      "27\t189\t0.0\t2\t0 184 5\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t359\t0.0\t2\t0 321 38\n",
      "26\t42745\t0.0\t2\t39395 3242 108\n",
      "27\t186\t0.0\t2\t0 173 13\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S790_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S790_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S790_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S790_159_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S790_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S790_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S790_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S790_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,047 reads  @    106.5 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.59 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,047\n",
      "  Read 1 with adapter:                  43,022 (99.9%)\n",
      "  Read 2 with adapter:                  42,948 (99.8%)\n",
      "Pairs written (passing filters):        43,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,393,862 bp\n",
      "  Read 1:    12,698,849 bp\n",
      "  Read 2:    12,695,013 bp\n",
      "Total written (filtered):     23,158,808 bp (91.2%)\n",
      "  Read 1:    11,580,317 bp\n",
      "  Read 2:    11,578,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t365\t0.0\t2\t0 358 7\n",
      "26\t42348\t0.0\t2\t41438 891 19\n",
      "27\t225\t0.0\t2\t0 218 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t318\t0.0\t2\t0 289 29\n",
      "26\t42427\t0.0\t2\t39042 3254 131\n",
      "27\t170\t0.0\t2\t0 155 15\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S791_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S791_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S791_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S791_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S791_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S791_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S791_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S791_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        40,836 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.02 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,836\n",
      "  Read 1 with adapter:                  40,815 (99.9%)\n",
      "  Read 2 with adapter:                  40,771 (99.8%)\n",
      "Pairs written (passing filters):        40,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,092,358 bp\n",
      "  Read 1:    12,046,579 bp\n",
      "  Read 2:    12,045,779 bp\n",
      "Total written (filtered):     21,971,234 bp (91.2%)\n",
      "  Read 1:    10,985,373 bp\n",
      "  Read 2:    10,985,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t327\t0.0\t2\t0 322 5\n",
      "26\t40170\t0.0\t2\t39277 883 10\n",
      "27\t205\t0.0\t2\t0 196 9\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t334\t0.0\t2\t0 298 36\n",
      "26\t40241\t0.0\t2\t36961 3142 138\n",
      "27\t126\t0.0\t2\t0 113 13\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S792_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S792_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S792_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S792_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S792_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S792_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S792_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S792_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        24,818 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 2.38 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,818\n",
      "  Read 1 with adapter:                  24,793 (99.9%)\n",
      "  Read 2 with adapter:                  24,663 (99.4%)\n",
      "Pairs written (passing filters):        24,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,638,173 bp\n",
      "  Read 1:     7,319,503 bp\n",
      "  Read 2:     7,318,670 bp\n",
      "Total written (filtered):     13,352,575 bp (91.2%)\n",
      "  Read 1:     6,674,976 bp\n",
      "  Read 2:     6,677,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t259\t0.0\t2\t0 251 8\n",
      "26\t24377\t0.0\t2\t23610 743 24\n",
      "27\t102\t0.0\t2\t0 98 4\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24663 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t273\t0.0\t2\t0 261 12\n",
      "26\t24261\t0.0\t2\t22549 1672 40\n",
      "27\t120\t0.0\t2\t0 113 7\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S793_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S793_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S793_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S793_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S793_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S793_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S793_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S793_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,494 reads  @     99.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.11 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,494\n",
      "  Read 1 with adapter:                  41,459 (99.9%)\n",
      "  Read 2 with adapter:                  41,385 (99.7%)\n",
      "Pairs written (passing filters):        41,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,473,065 bp\n",
      "  Read 1:    12,238,389 bp\n",
      "  Read 2:    12,234,676 bp\n",
      "Total written (filtered):     22,319,333 bp (91.2%)\n",
      "  Read 1:    11,160,594 bp\n",
      "  Read 2:    11,158,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t388\t0.0\t2\t0 377 11\n",
      "26\t40808\t0.0\t2\t39818 974 16\n",
      "27\t185\t0.0\t2\t0 179 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t278\t0.0\t2\t0 258 20\n",
      "26\t40875\t0.0\t2\t37551 3204 120\n",
      "27\t211\t0.0\t2\t0 182 29\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S794_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S794_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S794_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S794_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S794_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S794_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S794_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S794_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,159 reads  @     92.6 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.82 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,159\n",
      "  Read 1 with adapter:                  41,136 (99.9%)\n",
      "  Read 2 with adapter:                  41,023 (99.7%)\n",
      "Pairs written (passing filters):        41,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,275,640 bp\n",
      "  Read 1:    12,141,027 bp\n",
      "  Read 2:    12,134,613 bp\n",
      "Total written (filtered):     22,139,673 bp (91.2%)\n",
      "  Read 1:    11,071,575 bp\n",
      "  Read 2:    11,068,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41136 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t367\t0.0\t2\t0 356 11\n",
      "26\t40488\t0.0\t2\t39545 921 22\n",
      "27\t203\t0.0\t2\t0 196 7\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t271\t0.0\t2\t0 244 27\n",
      "26\t40552\t0.0\t2\t37008 3404 140\n",
      "27\t144\t0.0\t2\t0 127 17\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S795_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S795_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S795_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S795_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S795_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S795_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S795_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S795_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,684 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.46 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,684\n",
      "  Read 1 with adapter:                  32,655 (99.9%)\n",
      "  Read 2 with adapter:                  32,337 (98.9%)\n",
      "Pairs written (passing filters):        32,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,246,274 bp\n",
      "  Read 1:     9,639,352 bp\n",
      "  Read 2:     9,606,922 bp\n",
      "Total written (filtered):     17,555,790 bp (91.2%)\n",
      "  Read 1:     8,789,603 bp\n",
      "  Read 2:     8,766,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t255\t0.0\t2\t0 252 3\n",
      "26\t32207\t0.0\t2\t31535 658 14\n",
      "27\t163\t0.0\t2\t0 157 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t203\t0.0\t2\t0 180 23\n",
      "26\t31953\t0.0\t2\t28974 2778 201\n",
      "27\t166\t0.0\t2\t0 144 22\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S797_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S797_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S797_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S797_171_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S797_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S797_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S797_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S797_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        46,831 reads  @    101.5 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.76 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,831\n",
      "  Read 1 with adapter:                  46,790 (99.9%)\n",
      "  Read 2 with adapter:                  46,655 (99.6%)\n",
      "Pairs written (passing filters):        46,831 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,615,891 bp\n",
      "  Read 1:    13,812,117 bp\n",
      "  Read 2:    13,803,774 bp\n",
      "Total written (filtered):     25,186,585 bp (91.2%)\n",
      "  Read 1:    12,595,647 bp\n",
      "  Read 2:    12,590,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46790 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t399\t0.0\t2\t0 393 6\n",
      "26\t46085\t0.0\t2\t45202 858 25\n",
      "27\t227\t0.0\t2\t0 223 4\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t407\t0.0\t2\t0 373 34\n",
      "26\t46008\t0.0\t2\t42260 3603 145\n",
      "27\t187\t0.0\t2\t0 166 21\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S798_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S798_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S798_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S798_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S798_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S798_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S798_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S798_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,205 reads  @    100.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.64 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,205\n",
      "  Read 1 with adapter:                  46,163 (99.9%)\n",
      "  Read 2 with adapter:                  46,071 (99.7%)\n",
      "Pairs written (passing filters):        46,205 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,254,308 bp\n",
      "  Read 1:    13,627,464 bp\n",
      "  Read 2:    13,626,844 bp\n",
      "Total written (filtered):     24,856,417 bp (91.2%)\n",
      "  Read 1:    12,427,281 bp\n",
      "  Read 2:    12,429,136 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46163 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t387\t0.0\t2\t0 381 6\n",
      "26\t45431\t0.0\t2\t44493 925 13\n",
      "27\t254\t0.0\t2\t0 247 7\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46071 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t358\t0.0\t2\t0 323 35\n",
      "26\t45474\t0.0\t2\t41841 3473 160\n",
      "27\t186\t0.0\t2\t0 172 14\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S799_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S799_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S799_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S799_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S799_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S799_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S799_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S799_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,795 reads  @     95.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.66 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,795\n",
      "  Read 1 with adapter:                  48,760 (99.9%)\n",
      "  Read 2 with adapter:                  48,747 (99.9%)\n",
      "Pairs written (passing filters):        48,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,786,482 bp\n",
      "  Read 1:    14,392,684 bp\n",
      "  Read 2:    14,393,798 bp\n",
      "Total written (filtered):     26,251,628 bp (91.2%)\n",
      "  Read 1:    13,125,104 bp\n",
      "  Read 2:    13,126,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48760 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t456\t0.0\t2\t0 444 12\n",
      "26\t48035\t0.0\t2\t46967 1041 27\n",
      "27\t202\t0.0\t2\t0 197 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t344\t0.0\t2\t0 311 33\n",
      "26\t48173\t0.0\t2\t44395 3643 135\n",
      "27\t192\t0.0\t2\t0 180 12\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S800_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S800_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S800_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S800_177_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S800_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S800_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S800_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S800_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,511 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.91 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,511\n",
      "  Read 1 with adapter:                  42,484 (99.9%)\n",
      "  Read 2 with adapter:                  42,409 (99.8%)\n",
      "Pairs written (passing filters):        42,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,078,240 bp\n",
      "  Read 1:    12,538,954 bp\n",
      "  Read 2:    12,539,286 bp\n",
      "Total written (filtered):     22,871,232 bp (91.2%)\n",
      "  Read 1:    11,434,555 bp\n",
      "  Read 2:    11,436,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t471\t0.0\t2\t0 462 9\n",
      "26\t41711\t0.0\t2\t40633 1052 26\n",
      "27\t222\t0.0\t2\t0 214 8\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t305\t0.0\t2\t0 274 31\n",
      "26\t41818\t0.0\t2\t38427 3249 142\n",
      "27\t228\t0.0\t2\t0 209 19\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S801_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S801_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S801_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S801_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S801_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S801_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S801_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S801_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,092 reads  @     82.6 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.98 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,092\n",
      "  Read 1 with adapter:                  48,061 (99.9%)\n",
      "  Read 2 with adapter:                  47,996 (99.8%)\n",
      "Pairs written (passing filters):        48,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,372,926 bp\n",
      "  Read 1:    14,186,981 bp\n",
      "  Read 2:    14,185,945 bp\n",
      "Total written (filtered):     25,875,918 bp (91.2%)\n",
      "  Read 1:    12,937,613 bp\n",
      "  Read 2:    12,938,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t603\t0.0\t2\t0 584 19\n",
      "26\t47028\t0.0\t2\t45741 1252 35\n",
      "27\t339\t0.0\t2\t0 325 14\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t505\t0.0\t2\t0 461 44\n",
      "26\t47187\t0.0\t2\t43413 3648 126\n",
      "27\t239\t0.0\t2\t0 219 20\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S802_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S802_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S802_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S802_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S802_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S802_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S802_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S802_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,682 reads  @    102.4 µs/read;   0.59 M reads/minute\n",
      "Finished in 0.17 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,682\n",
      "  Read 1 with adapter:                   1,681 (99.9%)\n",
      "  Read 2 with adapter:                   1,675 (99.6%)\n",
      "Pairs written (passing filters):         1,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:       991,914 bp\n",
      "  Read 1:       496,188 bp\n",
      "  Read 2:       495,726 bp\n",
      "Total written (filtered):        904,669 bp (91.2%)\n",
      "  Read 1:       452,492 bp\n",
      "  Read 2:       452,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t23\t0.0\t2\t0 23\n",
      "26\t1643\t0.0\t2\t1588 55\n",
      "27\t9\t0.0\t2\t0 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t14\t0.0\t2\t0 10 4\n",
      "26\t1646\t0.0\t2\t1485 149 12\n",
      "27\t13\t0.0\t2\t0 13\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S803_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S803_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S803_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S803_183_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S803_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S803_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S803_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S803_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,787 reads  @    103.5 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.53 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,787\n",
      "  Read 1 with adapter:                  43,700 (99.8%)\n",
      "  Read 2 with adapter:                  43,646 (99.7%)\n",
      "Pairs written (passing filters):        43,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,808,908 bp\n",
      "  Read 1:    12,905,166 bp\n",
      "  Read 2:    12,903,742 bp\n",
      "Total written (filtered):     23,538,387 bp (91.2%)\n",
      "  Read 1:    11,769,200 bp\n",
      "  Read 2:    11,769,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43700 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t534\t0.0\t2\t0 521 13\n",
      "26\t42880\t0.0\t2\t42000 861 19\n",
      "27\t208\t0.0\t2\t0 201 7\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t436\t0.0\t2\t0 409 27\n",
      "26\t42962\t0.0\t2\t39690 3179 93\n",
      "27\t217\t0.0\t2\t0 199 18\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S804_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S804_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S804_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S804_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S804_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S804_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S804_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S804_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        77,761 reads  @    102.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 8.00 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,761\n",
      "  Read 1 with adapter:                  77,676 (99.9%)\n",
      "  Read 2 with adapter:                  77,598 (99.8%)\n",
      "Pairs written (passing filters):        77,761 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,861,569 bp\n",
      "  Read 1:    22,927,215 bp\n",
      "  Read 2:    22,934,354 bp\n",
      "Total written (filtered):     41,825,093 bp (91.2%)\n",
      "  Read 1:    20,908,064 bp\n",
      "  Read 2:    20,917,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 77676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t882\t0.0\t2\t0 855 27\n",
      "26\t76311\t0.0\t2\t74677 1610 24\n",
      "27\t381\t0.0\t2\t0 374 7\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 77598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t584\t0.0\t2\t0 536 48\n",
      "26\t76630\t0.0\t2\t70454 5924 252\n",
      "27\t335\t0.0\t2\t0 300 35\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S805_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S805_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S805_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S805_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S805_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S805_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S805_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S805_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,840 reads  @     93.0 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.08 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,840\n",
      "  Read 1 with adapter:                  43,808 (99.9%)\n",
      "  Read 2 with adapter:                  43,667 (99.6%)\n",
      "Pairs written (passing filters):        43,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,853,428 bp\n",
      "  Read 1:    12,929,609 bp\n",
      "  Read 2:    12,923,819 bp\n",
      "Total written (filtered):     23,579,207 bp (91.2%)\n",
      "  Read 1:    11,790,697 bp\n",
      "  Read 2:    11,788,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t459\t0.0\t2\t0 450 9\n",
      "26\t43024\t0.0\t2\t42084 925 15\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43667 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t266\t0.0\t2\t0 240 26\n",
      "26\t43145\t0.0\t2\t39269 3630 246\n",
      "27\t235\t0.0\t2\t0 208 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S806_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S806_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S806_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S806_189_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S806_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S806_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S806_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S806_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,235 reads  @     83.6 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.62 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,235\n",
      "  Read 1 with adapter:                  67,205 (100.0%)\n",
      "  Read 2 with adapter:                  66,696 (99.2%)\n",
      "Pairs written (passing filters):        67,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,569,293 bp\n",
      "  Read 1:    19,833,889 bp\n",
      "  Read 2:    19,735,404 bp\n",
      "Total written (filtered):     36,088,157 bp (91.2%)\n",
      "  Read 1:    18,086,656 bp\n",
      "  Read 2:    18,001,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t669\t0.0\t2\t0 647 22\n",
      "26\t66016\t0.0\t2\t64659 1335 22\n",
      "27\t356\t0.0\t2\t0 348 8\n",
      "28\t136\t0.0\t2\t0 0 136\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66696 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t486\t0.0\t2\t0 452 34\n",
      "26\t65857\t0.0\t2\t60829 4830 198\n",
      "27\t313\t0.0\t2\t0 285 28\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S807_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S807_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S807_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S807_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S807_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S807_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S807_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S807_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,146 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.79 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,146\n",
      "  Read 1 with adapter:                  42,122 (99.9%)\n",
      "  Read 2 with adapter:                  42,064 (99.8%)\n",
      "Pairs written (passing filters):        42,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,863,616 bp\n",
      "  Read 1:    12,432,652 bp\n",
      "  Read 2:    12,430,964 bp\n",
      "Total written (filtered):     22,675,170 bp (91.2%)\n",
      "  Read 1:    11,337,695 bp\n",
      "  Read 2:    11,337,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t496\t0.0\t2\t0 486 10\n",
      "26\t41292\t0.0\t2\t40463 809 20\n",
      "27\t279\t0.0\t2\t0 271 8\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t370\t0.0\t2\t0 345 25\n",
      "26\t41468\t0.0\t2\t38311 3059 98\n",
      "27\t201\t0.0\t2\t0 189 12\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S808_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S808_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S808_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S808_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S808_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S808_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S808_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S808_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,364 reads  @     83.3 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.28 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,364\n",
      "  Read 1 with adapter:                  63,324 (99.9%)\n",
      "  Read 2 with adapter:                  63,248 (99.8%)\n",
      "Pairs written (passing filters):        63,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,379,841 bp\n",
      "  Read 1:    18,691,495 bp\n",
      "  Read 2:    18,688,346 bp\n",
      "Total written (filtered):     34,089,378 bp (91.2%)\n",
      "  Read 1:    17,045,321 bp\n",
      "  Read 2:    17,044,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t798\t0.0\t2\t0 783 15\n",
      "26\t61995\t0.0\t2\t60753 1214 28\n",
      "27\t366\t0.0\t2\t0 354 12\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63248 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t450\t0.0\t2\t0 418 32\n",
      "26\t62484\t0.0\t2\t57576 4727 181\n",
      "27\t273\t0.0\t2\t0 247 26\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S809_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S809_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S809_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S809_195_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S809_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S809_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S809_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S809_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,854 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.38 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,854\n",
      "  Read 1 with adapter:                  49,807 (99.9%)\n",
      "  Read 2 with adapter:                  49,723 (99.7%)\n",
      "Pairs written (passing filters):        49,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,405,014 bp\n",
      "  Read 1:    14,706,285 bp\n",
      "  Read 2:    14,698,729 bp\n",
      "Total written (filtered):     26,817,573 bp (91.2%)\n",
      "  Read 1:    13,411,653 bp\n",
      "  Read 2:    13,405,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t697\t0.0\t2\t0 669 28\n",
      "26\t48752\t0.0\t2\t47603 1121 28\n",
      "27\t237\t0.0\t2\t0 229 8\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49723 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t331\t0.0\t2\t0 296 35\n",
      "26\t49037\t0.0\t2\t45065 3816 156\n",
      "27\t280\t0.0\t2\t0 260 20\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S810_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S810_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S810_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S810_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S810_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S810_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S810_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S810_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,177 reads  @     97.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.58 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,177\n",
      "  Read 1 with adapter:                  47,142 (99.9%)\n",
      "  Read 2 with adapter:                  47,017 (99.7%)\n",
      "Pairs written (passing filters):        47,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,823,873 bp\n",
      "  Read 1:    13,916,404 bp\n",
      "  Read 2:    13,907,469 bp\n",
      "Total written (filtered):     25,376,117 bp (91.2%)\n",
      "  Read 1:    12,690,975 bp\n",
      "  Read 2:    12,685,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t652\t0.0\t2\t0 625 27\n",
      "26\t46093\t0.0\t2\t44967 1105 21\n",
      "27\t257\t0.0\t2\t0 249 8\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t380\t0.0\t2\t0 340 40\n",
      "26\t46353\t0.0\t2\t42295 3879 179\n",
      "27\t215\t0.0\t2\t0 196 19\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S811_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S811_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S811_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S811_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S811_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S811_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S811_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S811_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        38,536 reads  @    107.9 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.16 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,536\n",
      "  Read 1 with adapter:                  38,503 (99.9%)\n",
      "  Read 2 with adapter:                  38,430 (99.7%)\n",
      "Pairs written (passing filters):        38,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,725,217 bp\n",
      "  Read 1:    11,366,940 bp\n",
      "  Read 2:    11,358,277 bp\n",
      "Total written (filtered):     20,725,355 bp (91.2%)\n",
      "  Read 1:    10,366,061 bp\n",
      "  Read 2:    10,359,294 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38503 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t388\t0.0\t2\t0 382 6\n",
      "26\t37876\t0.0\t2\t37076 788 12\n",
      "27\t177\t0.0\t2\t0 171 6\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t393\t0.0\t2\t0 353 40\n",
      "26\t37828\t0.0\t2\t34925 2798 105\n",
      "27\t186\t0.0\t2\t0 171 15\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S812_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S812_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S812_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S812_201_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S812_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S812_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S812_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S812_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,535 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.26 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,535\n",
      "  Read 1 with adapter:                  39,506 (99.9%)\n",
      "  Read 2 with adapter:                  39,466 (99.8%)\n",
      "Pairs written (passing filters):        39,535 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,323,548 bp\n",
      "  Read 1:    11,662,081 bp\n",
      "  Read 2:    11,661,467 bp\n",
      "Total written (filtered):     21,270,558 bp (91.2%)\n",
      "  Read 1:    10,635,097 bp\n",
      "  Read 2:    10,635,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t424\t0.0\t2\t0 417 7\n",
      "26\t38836\t0.0\t2\t37985 831 20\n",
      "27\t204\t0.0\t2\t0 198 6\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t318\t0.0\t2\t0 299 19\n",
      "26\t38928\t0.0\t2\t35920 2911 97\n",
      "27\t192\t0.0\t2\t0 176 16\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S813_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S813_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S813_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S813_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S813_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S813_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S813_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S813_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        39,785 reads  @     99.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 3.97 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,785\n",
      "  Read 1 with adapter:                  39,654 (99.7%)\n",
      "  Read 2 with adapter:                  39,707 (99.8%)\n",
      "Pairs written (passing filters):        39,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,454,133 bp\n",
      "  Read 1:    11,721,554 bp\n",
      "  Read 2:    11,732,579 bp\n",
      "Total written (filtered):     21,390,913 bp (91.2%)\n",
      "  Read 1:    10,690,698 bp\n",
      "  Read 2:    10,700,215 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t411\t0.0\t2\t0 401 10\n",
      "26\t38963\t0.0\t2\t38048 897 18\n",
      "27\t201\t0.0\t2\t0 195 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t270\t0.0\t2\t0 248 22\n",
      "26\t39171\t0.0\t2\t36155 2905 111\n",
      "27\t248\t0.0\t2\t0 221 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S814_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S814_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S814_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S814_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S814_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S814_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S814_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S814_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,411 reads  @     87.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.72 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,411\n",
      "  Read 1 with adapter:                  42,055 (99.2%)\n",
      "  Read 2 with adapter:                  42,137 (99.4%)\n",
      "Pairs written (passing filters):        42,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,933,462 bp\n",
      "  Read 1:    12,470,235 bp\n",
      "  Read 2:    12,463,227 bp\n",
      "Total written (filtered):     22,744,735 bp (91.2%)\n",
      "  Read 1:    11,376,923 bp\n",
      "  Read 2:    11,367,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t430\t0.0\t2\t0 420 10\n",
      "26\t41339\t0.0\t2\t40457 862 20\n",
      "27\t188\t0.0\t2\t0 178 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t352\t0.0\t2\t0 319 33\n",
      "26\t41535\t0.0\t2\t38376 3064 95\n",
      "27\t227\t0.0\t2\t0 212 15\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S815_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S815_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S815_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S815_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S815_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S815_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S815_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S815_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,009 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.73 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,009\n",
      "  Read 1 with adapter:                  41,991 (100.0%)\n",
      "  Read 2 with adapter:                  41,910 (99.8%)\n",
      "Pairs written (passing filters):        42,008 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,782,234 bp\n",
      "  Read 1:    12,392,633 bp\n",
      "  Read 2:    12,389,601 bp\n",
      "Total written (filtered):     22,600,928 bp (91.2%)\n",
      "  Read 1:    11,300,752 bp\n",
      "  Read 2:    11,300,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t483\t0.0\t2\t0 475 8\n",
      "26\t41162\t0.0\t2\t40370 779 13\n",
      "27\t251\t0.0\t2\t0 245 6\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t446\t0.0\t2\t0 409 37\n",
      "26\t41213\t0.0\t2\t37935 3147 131\n",
      "27\t223\t0.0\t2\t0 208 15\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S816_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S816_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S816_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S816_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S816_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S816_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S816_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S816_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,751 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.95 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,751\n",
      "  Read 1 with adapter:                  48,716 (99.9%)\n",
      "  Read 2 with adapter:                  48,660 (99.8%)\n",
      "Pairs written (passing filters):        48,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,760,809 bp\n",
      "  Read 1:    14,381,331 bp\n",
      "  Read 2:    14,379,478 bp\n",
      "Total written (filtered):     26,229,030 bp (91.2%)\n",
      "  Read 1:    13,114,674 bp\n",
      "  Read 2:    13,114,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t549\t0.0\t2\t0 531 18\n",
      "26\t47825\t0.0\t2\t46870 926 29\n",
      "27\t267\t0.0\t2\t0 262 5\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48660 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t321\t0.0\t2\t0 300 21\n",
      "26\t48067\t0.0\t2\t44305 3614 148\n",
      "27\t205\t0.0\t2\t0 183 22\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S817_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S817_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S817_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S817_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S817_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S817_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S817_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S817_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,849 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.45 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,849\n",
      "  Read 1 with adapter:                  38,834 (100.0%)\n",
      "  Read 2 with adapter:                  38,753 (99.8%)\n",
      "Pairs written (passing filters):        38,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,917,596 bp\n",
      "  Read 1:    11,460,439 bp\n",
      "  Read 2:    11,457,157 bp\n",
      "Total written (filtered):     20,900,513 bp (91.2%)\n",
      "  Read 1:    10,450,922 bp\n",
      "  Read 2:    10,449,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t445\t0.0\t2\t0 439 6\n",
      "26\t38098\t0.0\t2\t37383 705 10\n",
      "27\t212\t0.0\t2\t0 207 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t250\t0.0\t2\t0 216 34\n",
      "26\t38269\t0.0\t2\t35114 3025 130\n",
      "27\t202\t0.0\t2\t0 186 16\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S818_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S818_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S818_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S818_213_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S818_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S818_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S818_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S818_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,612 reads  @     91.2 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.98 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,612\n",
      "  Read 1 with adapter:                  43,588 (99.9%)\n",
      "  Read 2 with adapter:                  43,513 (99.8%)\n",
      "Pairs written (passing filters):        43,612 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,728,438 bp\n",
      "  Read 1:    12,865,120 bp\n",
      "  Read 2:    12,863,318 bp\n",
      "Total written (filtered):     23,464,045 bp (91.2%)\n",
      "  Read 1:    11,731,986 bp\n",
      "  Read 2:    11,732,059 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43588 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t446\t0.0\t2\t0 441 5\n",
      "26\t42835\t0.0\t2\t41968 855 12\n",
      "27\t250\t0.0\t2\t0 246 4\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43513 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t326\t0.0\t2\t0 296 30\n",
      "26\t42954\t0.0\t2\t39475 3325 154\n",
      "27\t191\t0.0\t2\t0 175 16\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S819_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S819_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S819_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S819_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S819_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S819_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S819_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S819_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,668 reads  @     97.4 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.26 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,668\n",
      "  Read 1 with adapter:                  43,639 (99.9%)\n",
      "  Read 2 with adapter:                  43,481 (99.6%)\n",
      "Pairs written (passing filters):        43,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,753,657 bp\n",
      "  Read 1:    12,881,651 bp\n",
      "  Read 2:    12,872,006 bp\n",
      "Total written (filtered):     23,489,150 bp (91.2%)\n",
      "  Read 1:    11,747,398 bp\n",
      "  Read 2:    11,741,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t606\t0.0\t2\t0 590 16\n",
      "26\t42739\t0.0\t2\t41744 983 12\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t474\t0.0\t2\t0 440 34\n",
      "26\t42758\t0.0\t2\t39076 3474 208\n",
      "27\t216\t0.0\t2\t0 194 22\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S820_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S820_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S820_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S820_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S820_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S820_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S820_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S820_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,042 reads  @     95.3 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.29 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,042\n",
      "  Read 1 with adapter:                  45,011 (99.9%)\n",
      "  Read 2 with adapter:                  44,836 (99.5%)\n",
      "Pairs written (passing filters):        45,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,570,183 bp\n",
      "  Read 1:    13,286,069 bp\n",
      "  Read 2:    13,284,114 bp\n",
      "Total written (filtered):     24,234,545 bp (91.2%)\n",
      "  Read 1:    12,116,039 bp\n",
      "  Read 2:    12,118,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t589\t0.0\t2\t0 581 8\n",
      "26\t44068\t0.0\t2\t43179 874 15\n",
      "27\t255\t0.0\t2\t0 250 5\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t331\t0.0\t2\t0 301 30\n",
      "26\t44262\t0.0\t2\t40238 3759 265\n",
      "27\t223\t0.0\t2\t0 195 28\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S821_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S821_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S821_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S821_219_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S821_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S821_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S821_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S821_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,966 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.22 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,966\n",
      "  Read 1 with adapter:                  48,942 (100.0%)\n",
      "  Read 2 with adapter:                  48,722 (99.5%)\n",
      "Pairs written (passing filters):        48,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,882,651 bp\n",
      "  Read 1:    14,444,374 bp\n",
      "  Read 2:    14,438,277 bp\n",
      "Total written (filtered):     26,343,543 bp (91.2%)\n",
      "  Read 1:    13,172,007 bp\n",
      "  Read 2:    13,171,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t597\t0.0\t2\t0 576 21\n",
      "26\t47929\t0.0\t2\t46955 964 10\n",
      "27\t276\t0.0\t2\t0 269 7\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t341\t0.0\t2\t0 309 32\n",
      "26\t48060\t0.0\t2\t43653 4081 326\n",
      "27\t304\t0.0\t2\t0 277 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S822_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S822_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S822_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S822_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S822_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S822_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S822_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S822_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,054 reads  @     98.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.66 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,054\n",
      "  Read 1 with adapter:                  47,024 (99.9%)\n",
      "  Read 2 with adapter:                  46,706 (99.3%)\n",
      "Pairs written (passing filters):        47,054 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,722,824 bp\n",
      "  Read 1:    13,880,365 bp\n",
      "  Read 2:    13,842,459 bp\n",
      "Total written (filtered):     25,286,190 bp (91.2%)\n",
      "  Read 1:    12,658,009 bp\n",
      "  Read 2:    12,628,181 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47024 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t573\t0.0\t2\t0 556 17\n",
      "26\t46126\t0.0\t2\t45118 989 19\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t322\t0.0\t2\t0 288 34\n",
      "26\t46102\t0.0\t2\t42064 3820 218\n",
      "27\t260\t0.0\t2\t0 225 35\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S823_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S823_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S823_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S823_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S823_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S823_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S823_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S823_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,679 reads  @     96.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.40 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,679\n",
      "  Read 1 with adapter:                  45,651 (99.9%)\n",
      "  Read 2 with adapter:                  45,256 (99.1%)\n",
      "Pairs written (passing filters):        45,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,899,954 bp\n",
      "  Read 1:    13,475,149 bp\n",
      "  Read 2:    13,424,805 bp\n",
      "Total written (filtered):     24,536,893 bp (91.2%)\n",
      "  Read 1:    12,288,517 bp\n",
      "  Read 2:    12,248,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t525\t0.0\t2\t0 515 10\n",
      "26\t44863\t0.0\t2\t43947 895 21\n",
      "27\t203\t0.0\t2\t0 197 6\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45256 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t439\t0.0\t2\t0 390 49\n",
      "26\t44536\t0.0\t2\t40586 3700 250\n",
      "27\t254\t0.0\t2\t0 227 27\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S824_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S824_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S824_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S824_225_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S824_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S824_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S824_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S824_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,709 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.28 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,709\n",
      "  Read 1 with adapter:                  47,681 (99.9%)\n",
      "  Read 2 with adapter:                  47,587 (99.7%)\n",
      "Pairs written (passing filters):        47,709 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,145,419 bp\n",
      "  Read 1:    14,073,847 bp\n",
      "  Read 2:    14,071,572 bp\n",
      "Total written (filtered):     25,668,871 bp (91.2%)\n",
      "  Read 1:    12,834,482 bp\n",
      "  Read 2:    12,834,389 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t555\t0.0\t2\t0 538 17\n",
      "26\t46862\t0.0\t2\t45877 969 16\n",
      "27\t210\t0.0\t2\t0 200 10\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t344\t0.0\t2\t0 324 20\n",
      "26\t46948\t0.0\t2\t42838 3880 230\n",
      "27\t261\t0.0\t2\t0 227 34\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S825_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S825_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S825_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S825_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S825_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S825_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S825_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S825_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,017 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.74 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,017\n",
      "  Read 1 with adapter:                  58,990 (100.0%)\n",
      "  Read 2 with adapter:                  58,630 (99.3%)\n",
      "Pairs written (passing filters):        59,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,780,486 bp\n",
      "  Read 1:    17,409,999 bp\n",
      "  Read 2:    17,370,487 bp\n",
      "Total written (filtered):     31,723,024 bp (91.2%)\n",
      "  Read 1:    15,876,844 bp\n",
      "  Read 2:    15,846,180 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t742\t0.0\t2\t0 725 17\n",
      "26\t57997\t0.0\t2\t56476 1487 34\n",
      "27\t213\t0.0\t2\t0 207 6\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t427\t0.0\t2\t0 388 39\n",
      "26\t57820\t0.0\t2\t52726 4796 298\n",
      "27\t312\t0.0\t2\t0 283 29\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S826_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S826_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S826_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S826_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S826_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S826_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S826_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S826_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,947 reads  @    109.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.70 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,947\n",
      "  Read 1 with adapter:                  42,928 (100.0%)\n",
      "  Read 2 with adapter:                  42,793 (99.6%)\n",
      "Pairs written (passing filters):        42,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,334,152 bp\n",
      "  Read 1:    12,669,102 bp\n",
      "  Read 2:    12,665,050 bp\n",
      "Total written (filtered):     23,105,713 bp (91.2%)\n",
      "  Read 1:    11,553,188 bp\n",
      "  Read 2:    11,552,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t504\t0.0\t2\t0 493 11\n",
      "26\t42137\t0.0\t2\t41185 937 15\n",
      "27\t204\t0.0\t2\t0 199 5\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t316\t0.0\t2\t0 281 35\n",
      "26\t42234\t0.0\t2\t38545 3504 185\n",
      "27\t199\t0.0\t2\t0 182 17\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S827_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S827_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S827_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S827_231_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S827_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S827_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S827_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S827_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        40,584 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.00 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,584\n",
      "  Read 1 with adapter:                  40,315 (99.3%)\n",
      "  Read 2 with adapter:                  40,518 (99.8%)\n",
      "Pairs written (passing filters):        40,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,875,743 bp\n",
      "  Read 1:    11,906,919 bp\n",
      "  Read 2:    11,968,824 bp\n",
      "Total written (filtered):     21,774,591 bp (91.2%)\n",
      "  Read 1:    10,858,998 bp\n",
      "  Read 2:    10,915,593 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t537\t0.0\t2\t0 526 11\n",
      "26\t39493\t0.0\t2\t38457 1018 18\n",
      "27\t222\t0.0\t2\t0 212 10\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t473\t0.0\t2\t0 436 37\n",
      "26\t39768\t0.0\t2\t36593 3092 83\n",
      "27\t238\t0.0\t2\t0 218 20\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S828_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S828_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S828_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S828_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S828_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S828_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S828_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S828_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,011 reads  @     97.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.38 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,011\n",
      "  Read 1 with adapter:                  44,806 (99.5%)\n",
      "  Read 2 with adapter:                  44,930 (99.8%)\n",
      "Pairs written (passing filters):        45,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,508,002 bp\n",
      "  Read 1:    13,231,237 bp\n",
      "  Read 2:    13,276,765 bp\n",
      "Total written (filtered):     24,175,038 bp (91.2%)\n",
      "  Read 1:    12,066,375 bp\n",
      "  Read 2:    12,108,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t467\t0.0\t2\t0 447 20\n",
      "26\t43999\t0.0\t2\t43067 912 20\n",
      "27\t219\t0.0\t2\t0 214 5\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t314\t0.0\t2\t0 291 23\n",
      "26\t44343\t0.0\t2\t40907 3300 136\n",
      "27\t230\t0.0\t2\t0 216 14\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S829_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S829_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S829_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S829_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S829_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S829_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S829_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S829_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,074 reads  @     88.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.01 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,074\n",
      "  Read 1 with adapter:                  44,968 (99.8%)\n",
      "  Read 2 with adapter:                  44,990 (99.8%)\n",
      "Pairs written (passing filters):        45,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,568,763 bp\n",
      "  Read 1:    13,276,614 bp\n",
      "  Read 2:    13,292,149 bp\n",
      "Total written (filtered):     24,229,948 bp (91.2%)\n",
      "  Read 1:    12,107,444 bp\n",
      "  Read 2:    12,122,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44968 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t445\t0.0\t2\t0 437 8\n",
      "26\t44140\t0.0\t2\t43155 967 18\n",
      "27\t235\t0.0\t2\t0 227 8\n",
      "28\t127\t0.0\t2\t0 0 127\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t319\t0.0\t2\t0 300 19\n",
      "26\t44400\t0.0\t2\t41058 3227 115\n",
      "27\t238\t0.0\t2\t0 215 23\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S830_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S830_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S830_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S830_237_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S830_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S830_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S830_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S830_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,164 reads  @     94.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.66 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,164\n",
      "  Read 1 with adapter:                  48,876 (99.4%)\n",
      "  Read 2 with adapter:                  48,952 (99.6%)\n",
      "Pairs written (passing filters):        49,164 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,908,056 bp\n",
      "  Read 1:    14,436,220 bp\n",
      "  Read 2:    14,471,836 bp\n",
      "Total written (filtered):     26,364,800 bp (91.2%)\n",
      "  Read 1:    13,165,611 bp\n",
      "  Read 2:    13,199,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t552\t0.0\t2\t0 535 17\n",
      "26\t47955\t0.0\t2\t46912 1024 19\n",
      "27\t281\t0.0\t2\t0 273 8\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t356\t0.0\t2\t0 323 33\n",
      "26\t48314\t0.0\t2\t44663 3559 92\n",
      "27\t249\t0.0\t2\t0 226 23\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S831_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S831_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S831_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S831_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S831_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S831_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S831_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S831_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,559 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.87 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,559\n",
      "  Read 1 with adapter:                  46,532 (99.9%)\n",
      "  Read 2 with adapter:                  46,517 (99.9%)\n",
      "Pairs written (passing filters):        46,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,466,808 bp\n",
      "  Read 1:    13,733,960 bp\n",
      "  Read 2:    13,732,848 bp\n",
      "Total written (filtered):     25,048,052 bp (91.2%)\n",
      "  Read 1:    12,524,429 bp\n",
      "  Read 2:    12,523,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t560\t0.0\t2\t0 541 19\n",
      "26\t45668\t0.0\t2\t44635 1009 24\n",
      "27\t245\t0.0\t2\t0 234 11\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t475\t0.0\t2\t0 441 34\n",
      "26\t45763\t0.0\t2\t42485 3204 74\n",
      "27\t212\t0.0\t2\t0 202 10\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S832_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S832_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S832_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S832_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S832_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S832_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S832_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S832_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,893 reads  @     76.6 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.21 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,893\n",
      "  Read 1 with adapter:                  54,858 (99.9%)\n",
      "  Read 2 with adapter:                  54,823 (99.9%)\n",
      "Pairs written (passing filters):        54,893 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,385,641 bp\n",
      "  Read 1:    16,192,985 bp\n",
      "  Read 2:    16,192,656 bp\n",
      "Total written (filtered):     29,534,393 bp (91.2%)\n",
      "  Read 1:    14,767,058 bp\n",
      "  Read 2:    14,767,335 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t666\t0.0\t2\t0 653 13\n",
      "26\t53850\t0.0\t2\t52725 1107 18\n",
      "27\t279\t0.0\t2\t0 271 8\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t412\t0.0\t2\t0 382 30\n",
      "26\t54100\t0.0\t2\t50107 3876 117\n",
      "27\t235\t0.0\t2\t0 218 17\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S833_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S833_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S833_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S833_243_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S833_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S833_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S833_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S833_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,908 reads  @     86.6 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.67 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,908\n",
      "  Read 1 with adapter:                  53,881 (99.9%)\n",
      "  Read 2 with adapter:                  53,811 (99.8%)\n",
      "Pairs written (passing filters):        53,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,799,091 bp\n",
      "  Read 1:    15,902,552 bp\n",
      "  Read 2:    15,896,539 bp\n",
      "Total written (filtered):     28,999,472 bp (91.2%)\n",
      "  Read 1:    14,501,929 bp\n",
      "  Read 2:    14,497,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t628\t0.0\t2\t0 615 13\n",
      "26\t52863\t0.0\t2\t51730 1120 13\n",
      "27\t335\t0.0\t2\t0 323 12\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t406\t0.0\t2\t0 387 19\n",
      "26\t53074\t0.0\t2\t49128 3829 117\n",
      "27\t290\t0.0\t2\t0 266 24\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S834_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S834_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S834_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S834_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S834_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S834_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S834_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S834_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,632 reads  @     94.4 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.93 s (94 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,632\n",
      "  Read 1 with adapter:                  41,607 (99.9%)\n",
      "  Read 2 with adapter:                  41,573 (99.9%)\n",
      "Pairs written (passing filters):        41,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,560,015 bp\n",
      "  Read 1:    12,281,250 bp\n",
      "  Read 2:    12,278,765 bp\n",
      "Total written (filtered):     22,397,729 bp (91.2%)\n",
      "  Read 1:    11,199,773 bp\n",
      "  Read 2:    11,197,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t536\t0.0\t2\t0 526 10\n",
      "26\t40809\t0.0\t2\t39845 953 11\n",
      "27\t217\t0.0\t2\t0 212 5\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t327\t0.0\t2\t0 311 16\n",
      "26\t40992\t0.0\t2\t37892 3032 68\n",
      "27\t234\t0.0\t2\t0 212 22\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S835_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S835_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S835_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S835_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S835_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S835_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S835_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S835_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        54,666 reads  @    100.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.49 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,666\n",
      "  Read 1 with adapter:                  54,626 (99.9%)\n",
      "  Read 2 with adapter:                  54,606 (99.9%)\n",
      "Pairs written (passing filters):        54,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,249,083 bp\n",
      "  Read 1:    16,124,408 bp\n",
      "  Read 2:    16,124,675 bp\n",
      "Total written (filtered):     29,409,373 bp (91.2%)\n",
      "  Read 1:    14,704,164 bp\n",
      "  Read 2:    14,705,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t477\t0.0\t2\t0 462 15\n",
      "26\t53738\t0.0\t2\t52431 1289 18\n",
      "27\t257\t0.0\t2\t0 248 9\n",
      "28\t124\t0.0\t2\t0 0 124\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t535\t0.0\t2\t0 504 31\n",
      "26\t53726\t0.0\t2\t49530 4082 114\n",
      "27\t297\t0.0\t2\t0 279 18\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S836_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S836_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S836_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S836_249_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S836_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S836_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S836_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S836_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,212 reads  @     92.9 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.39 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,212\n",
      "  Read 1 with adapter:                  47,183 (99.9%)\n",
      "  Read 2 with adapter:                  47,137 (99.8%)\n",
      "Pairs written (passing filters):        47,212 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,852,851 bp\n",
      "  Read 1:    13,926,426 bp\n",
      "  Read 2:    13,926,425 bp\n",
      "Total written (filtered):     25,400,646 bp (91.2%)\n",
      "  Read 1:    12,699,673 bp\n",
      "  Read 2:    12,700,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47183 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t354\t0.0\t2\t0 347 7\n",
      "26\t46457\t0.0\t2\t45554 883 20\n",
      "27\t315\t0.0\t2\t0 307 8\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t321\t0.0\t2\t0 298 23\n",
      "26\t46577\t0.0\t2\t42859 3614 104\n",
      "27\t223\t0.0\t2\t0 203 20\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S837_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S837_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S837_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S837_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S837_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S837_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S837_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S837_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        89,775 reads  @     92.2 µs/read;   0.65 M reads/minute\n",
      "Finished in 8.28 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,775\n",
      "  Read 1 with adapter:                  89,690 (99.9%)\n",
      "  Read 2 with adapter:                  89,555 (99.8%)\n",
      "Pairs written (passing filters):        89,775 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,948,241 bp\n",
      "  Read 1:    26,476,127 bp\n",
      "  Read 2:    26,472,114 bp\n",
      "Total written (filtered):     48,287,854 bp (91.2%)\n",
      "  Read 1:    24,144,232 bp\n",
      "  Read 2:    24,143,622 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 89690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t683\t0.0\t2\t0 669 14\n",
      "26\t88341\t0.0\t2\t86425 1888 28\n",
      "27\t530\t0.0\t2\t0 521 9\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 89555 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t620\t0.0\t2\t0 549 71\n",
      "26\t88328\t0.0\t2\t81173 6847 308\n",
      "27\t392\t0.0\t2\t0 363 29\n",
      "28\t180\t0.0\t2\t0 0 180\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S838_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S838_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S838_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S838_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S838_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S838_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S838_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S838_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        50,477 reads  @    108.9 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.50 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,477\n",
      "  Read 1 with adapter:                  50,346 (99.7%)\n",
      "  Read 2 with adapter:                  50,366 (99.8%)\n",
      "Pairs written (passing filters):        50,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,762,693 bp\n",
      "  Read 1:    14,872,328 bp\n",
      "  Read 2:    14,890,365 bp\n",
      "Total written (filtered):     27,144,354 bp (91.2%)\n",
      "  Read 1:    13,563,401 bp\n",
      "  Read 2:    13,580,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t385\t0.0\t2\t0 379 6\n",
      "26\t49609\t0.0\t2\t48526 1065 18\n",
      "27\t292\t0.0\t2\t0 281 11\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t377\t0.0\t2\t0 343 34\n",
      "26\t49684\t0.0\t2\t45899 3671 114\n",
      "27\t261\t0.0\t2\t0 242 19\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S839_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S839_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S839_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S839_255_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S839_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S839_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S839_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S839_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,434 reads  @    113.1 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.91 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,434\n",
      "  Read 1 with adapter:                  43,363 (99.8%)\n",
      "  Read 2 with adapter:                  43,319 (99.7%)\n",
      "Pairs written (passing filters):        43,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,617,613 bp\n",
      "  Read 1:    12,808,062 bp\n",
      "  Read 2:    12,809,551 bp\n",
      "Total written (filtered):     23,364,146 bp (91.2%)\n",
      "  Read 1:    11,680,696 bp\n",
      "  Read 2:    11,683,450 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t411\t0.0\t2\t0 399 12\n",
      "26\t42630\t0.0\t2\t41733 882 15\n",
      "27\t257\t0.0\t2\t0 254 3\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t409\t0.0\t2\t0 364 45\n",
      "26\t42656\t0.0\t2\t39151 3347 158\n",
      "27\t212\t0.0\t2\t0 195 17\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S840_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S840_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S840_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S840_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S840_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S840_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S840_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S840_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[        8<--] 00:00:20       187,244 reads  @    108.7 µs/read;   0.55 M reads/minute\n",
      "Finished in 20.36 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,244\n",
      "  Read 1 with adapter:                 187,037 (99.9%)\n",
      "  Read 2 with adapter:                 186,996 (99.9%)\n",
      "Pairs written (passing filters):       187,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,456,621 bp\n",
      "  Read 1:    55,226,520 bp\n",
      "  Read 2:    55,230,101 bp\n",
      "Total written (filtered):    100,732,918 bp (91.2%)\n",
      "  Read 1:    50,364,365 bp\n",
      "  Read 2:    50,368,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 187037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t1880\t0.0\t2\t0 1844 36\n",
      "26\t183983\t0.0\t2\t180005 3917 61\n",
      "27\t923\t0.0\t2\t0 897 26\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 186996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t1300\t0.0\t2\t0 1178 122\n",
      "26\t184703\t0.0\t2\t170655 13608 440\n",
      "27\t862\t0.0\t2\t0 797 65\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S841_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S841_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S841_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S841_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S841_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S841_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S841_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S841_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        51,182 reads  @    107.1 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.49 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,182\n",
      "  Read 1 with adapter:                  51,149 (99.9%)\n",
      "  Read 2 with adapter:                  51,094 (99.8%)\n",
      "Pairs written (passing filters):        51,182 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,190,817 bp\n",
      "  Read 1:    15,096,853 bp\n",
      "  Read 2:    15,093,964 bp\n",
      "Total written (filtered):     27,532,601 bp (91.2%)\n",
      "  Read 1:    13,767,031 bp\n",
      "  Read 2:    13,765,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t455\t0.0\t2\t0 445 10\n",
      "26\t50300\t0.0\t2\t49192 1083 25\n",
      "27\t277\t0.0\t2\t0 262 15\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t322\t0.0\t2\t0 294 28\n",
      "26\t50482\t0.0\t2\t46567 3788 127\n",
      "27\t236\t0.0\t2\t0 219 17\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S842_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S842_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S842_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S842_261_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S842_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S842_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S842_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S842_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        50,286 reads  @    102.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.17 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,286\n",
      "  Read 1 with adapter:                  50,267 (100.0%)\n",
      "  Read 2 with adapter:                  50,220 (99.9%)\n",
      "Pairs written (passing filters):        50,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,667,052 bp\n",
      "  Read 1:    14,833,487 bp\n",
      "  Read 2:    14,833,565 bp\n",
      "Total written (filtered):     27,054,439 bp (91.2%)\n",
      "  Read 1:    13,526,514 bp\n",
      "  Read 2:    13,527,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50267 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t433\t0.0\t2\t0 424 9\n",
      "26\t49426\t0.0\t2\t48365 1045 16\n",
      "27\t268\t0.0\t2\t0 266 2\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50220 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t424\t0.0\t2\t0 394 30\n",
      "26\t49499\t0.0\t2\t45607 3767 125\n",
      "27\t206\t0.0\t2\t0 184 22\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S843_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S843_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S843_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S843_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S843_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S843_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S843_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S843_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,503 reads  @     92.6 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.49 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,503\n",
      "  Read 1 with adapter:                  48,479 (100.0%)\n",
      "  Read 2 with adapter:                  48,429 (99.8%)\n",
      "Pairs written (passing filters):        48,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,613,600 bp\n",
      "  Read 1:    14,308,072 bp\n",
      "  Read 2:    14,305,528 bp\n",
      "Total written (filtered):     26,094,276 bp (91.2%)\n",
      "  Read 1:    13,047,638 bp\n",
      "  Read 2:    13,046,638 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t393\t0.0\t2\t0 376 17\n",
      "26\t47741\t0.0\t2\t46737 984 20\n",
      "27\t245\t0.0\t2\t0 240 5\n",
      "28\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48429 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t474\t0.0\t2\t0 437 37\n",
      "26\t47700\t0.0\t2\t44102 3500 98\n",
      "27\t236\t0.0\t2\t0 212 24\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S844_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S844_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S844_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S844_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S844_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S844_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S844_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S844_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,205 reads  @    110.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 4.67 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,205\n",
      "  Read 1 with adapter:                  42,185 (100.0%)\n",
      "  Read 2 with adapter:                  42,132 (99.8%)\n",
      "Pairs written (passing filters):        42,205 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,897,779 bp\n",
      "  Read 1:    12,450,132 bp\n",
      "  Read 2:    12,447,647 bp\n",
      "Total written (filtered):     22,705,717 bp (91.2%)\n",
      "  Read 1:    11,353,337 bp\n",
      "  Read 2:    11,352,380 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t326\t0.0\t2\t0 320 6\n",
      "26\t41548\t0.0\t2\t40540 984 24\n",
      "27\t207\t0.0\t2\t0 198 9\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t335\t0.0\t2\t0 315 20\n",
      "26\t41588\t0.0\t2\t38196 3266 126\n",
      "27\t180\t0.0\t2\t0 168 12\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S845_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S845_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S845_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S845_267_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S845_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S845_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S845_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S845_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,301 reads  @     92.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.21 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,301\n",
      "  Read 1 with adapter:                  45,281 (100.0%)\n",
      "  Read 2 with adapter:                  45,093 (99.5%)\n",
      "Pairs written (passing filters):        45,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,716,038 bp\n",
      "  Read 1:    13,363,486 bp\n",
      "  Read 2:    13,352,552 bp\n",
      "Total written (filtered):     24,366,341 bp (91.2%)\n",
      "  Read 1:    12,186,184 bp\n",
      "  Read 2:    12,180,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45281 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t323\t0.0\t2\t0 307 16\n",
      "26\t44647\t0.0\t2\t43730 906 11\n",
      "27\t247\t0.0\t2\t0 238 9\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t277\t0.0\t2\t0 249 28\n",
      "26\t44564\t0.0\t2\t40362 3962 240\n",
      "27\t202\t0.0\t2\t0 179 23\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S846_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S846_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S846_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S846_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S846_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S846_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S846_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S846_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,644 reads  @     93.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 4.08 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,644\n",
      "  Read 1 with adapter:                  43,615 (99.9%)\n",
      "  Read 2 with adapter:                  43,531 (99.7%)\n",
      "Pairs written (passing filters):        43,644 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,748,231 bp\n",
      "  Read 1:    12,874,594 bp\n",
      "  Read 2:    12,873,637 bp\n",
      "Total written (filtered):     23,482,607 bp (91.2%)\n",
      "  Read 1:    11,740,656 bp\n",
      "  Read 2:    11,741,951 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43615 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t334\t0.0\t2\t0 327 7\n",
      "26\t42967\t0.0\t2\t42086 862 19\n",
      "27\t262\t0.0\t2\t0 257 5\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t319\t0.0\t2\t0 297 22\n",
      "26\t42960\t0.0\t2\t39664 3198 98\n",
      "27\t217\t0.0\t2\t0 198 19\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S847_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S847_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S847_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S847_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S847_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S847_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S847_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S847_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,362 reads  @    124.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.17 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,362\n",
      "  Read 1 with adapter:                  41,321 (99.9%)\n",
      "  Read 2 with adapter:                  41,272 (99.8%)\n",
      "Pairs written (passing filters):        41,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,395,715 bp\n",
      "  Read 1:    12,198,672 bp\n",
      "  Read 2:    12,197,043 bp\n",
      "Total written (filtered):     22,248,631 bp (91.2%)\n",
      "  Read 1:    11,124,442 bp\n",
      "  Read 2:    11,124,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t352\t0.0\t2\t0 342 10\n",
      "26\t40711\t0.0\t2\t39851 838 22\n",
      "27\t204\t0.0\t2\t0 198 6\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41272 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t420\t0.0\t2\t0 388 32\n",
      "26\t40617\t0.0\t2\t37385 3120 112\n",
      "27\t212\t0.0\t2\t0 192 20\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S848_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S848_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S848_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S848_273_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S848_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S848_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S848_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S848_273_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        64,922 reads  @    116.1 µs/read;   0.52 M reads/minute\n",
      "Finished in 7.54 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,922\n",
      "  Read 1 with adapter:                  64,874 (99.9%)\n",
      "  Read 2 with adapter:                  64,815 (99.8%)\n",
      "Pairs written (passing filters):        64,922 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,294,757 bp\n",
      "  Read 1:    19,146,758 bp\n",
      "  Read 2:    19,147,999 bp\n",
      "Total written (filtered):     34,923,084 bp (91.2%)\n",
      "  Read 1:    17,460,175 bp\n",
      "  Read 2:    17,462,909 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t535\t0.0\t2\t0 519 16\n",
      "26\t63927\t0.0\t2\t62555 1352 20\n",
      "27\t322\t0.0\t2\t0 310 12\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t465\t0.0\t2\t0 437 28\n",
      "26\t63978\t0.0\t2\t58913 4883 182\n",
      "27\t299\t0.0\t2\t0 280 19\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S849_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S849_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S849_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S849_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S849_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S849_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S849_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S849_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        56,558 reads  @    108.5 µs/read;   0.55 M reads/minute\n",
      "Finished in 6.14 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,558\n",
      "  Read 1 with adapter:                  56,534 (100.0%)\n",
      "  Read 2 with adapter:                  56,432 (99.8%)\n",
      "Pairs written (passing filters):        56,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,359,830 bp\n",
      "  Read 1:    16,682,927 bp\n",
      "  Read 2:    16,676,903 bp\n",
      "Total written (filtered):     30,422,884 bp (91.2%)\n",
      "  Read 1:    15,213,163 bp\n",
      "  Read 2:    15,209,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t486\t0.0\t2\t0 479 7\n",
      "26\t55664\t0.0\t2\t54607 1034 23\n",
      "27\t290\t0.0\t2\t0 286 4\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t380\t0.0\t2\t0 354 26\n",
      "26\t55694\t0.0\t2\t51340 4184 170\n",
      "27\t278\t0.0\t2\t0 257 21\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S850_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S850_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S850_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S850_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S850_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S850_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S850_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S850_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,232 reads  @    109.9 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.86 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,232\n",
      "  Read 1 with adapter:                  44,207 (99.9%)\n",
      "  Read 2 with adapter:                  44,154 (99.8%)\n",
      "Pairs written (passing filters):        44,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,094,735 bp\n",
      "  Read 1:    13,047,413 bp\n",
      "  Read 2:    13,047,322 bp\n",
      "Total written (filtered):     23,797,559 bp (91.2%)\n",
      "  Read 1:    11,898,137 bp\n",
      "  Read 2:    11,899,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44207 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t385\t0.0\t2\t0 376 9\n",
      "26\t43555\t0.0\t2\t42652 886 17\n",
      "27\t191\t0.0\t2\t0 191\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t340\t0.0\t2\t0 309 31\n",
      "26\t43569\t0.0\t2\t40211 3239 119\n",
      "27\t214\t0.0\t2\t0 196 18\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S851_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S851_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S851_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S851_279_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S851_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S851_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S851_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S851_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        50,263 reads  @    103.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.22 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,263\n",
      "  Read 1 with adapter:                  50,242 (100.0%)\n",
      "  Read 2 with adapter:                  50,074 (99.6%)\n",
      "Pairs written (passing filters):        50,263 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,650,892 bp\n",
      "  Read 1:    14,827,101 bp\n",
      "  Read 2:    14,823,791 bp\n",
      "Total written (filtered):     27,042,857 bp (91.2%)\n",
      "  Read 1:    13,521,009 bp\n",
      "  Read 2:    13,521,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50242 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t578\t0.0\t2\t0 562 16\n",
      "26\t49320\t0.0\t2\t48044 1255 21\n",
      "27\t250\t0.0\t2\t0 240 10\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t378\t0.0\t2\t0 341 37\n",
      "26\t49309\t0.0\t2\t44506 4515 288\n",
      "27\t305\t0.0\t2\t0 282 23\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S852_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S852_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S852_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S852_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S852_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-w67gwcuz/S852_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S852_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hpd1qhgw/f41ae676-1ede-4124-95dc-142b11d40161/data/S852_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,604 reads  @     95.3 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.25 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,604\n",
      "  Read 1 with adapter:                  44,572 (99.9%)\n",
      "  Read 2 with adapter:                  44,520 (99.8%)\n",
      "Pairs written (passing filters):        44,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,313,960 bp\n",
      "  Read 1:    13,158,150 bp\n",
      "  Read 2:    13,155,810 bp\n",
      "Total written (filtered):     23,997,856 bp (91.2%)\n",
      "  Read 1:    11,999,501 bp\n",
      "  Read 2:    11,998,355 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t581\t0.0\t2\t0 567 14\n",
      "26\t43593\t0.0\t2\t42457 1116 20\n",
      "27\t306\t0.0\t2\t0 295 11\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44520 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t345\t0.0\t2\t0 317 28\n",
      "26\t43873\t0.0\t2\t40270 3474 129\n",
      "27\t248\t0.0\t2\t0 224 24\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: d-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: d-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_d_kyk_j')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e89599a0-c77f-4f33-9d0c-a73ab6a05972>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.d import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/BB-Lib1_leray_manifest_runbb_v2_grid.csv \\\n",
    "  --output-path d-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.d trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences d-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences d-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.d summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data d-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization d-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('d-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: d-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: d-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: d-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.d Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs d-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 258 \\\n",
    "  --p-trunc-len-r 168 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table d-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats d-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: d-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-twdl3923')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 01c55a58-e80c-496a-890b-b75236dcde61>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.d view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table d-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization d-table.qzv \n",
    "q2.Visualization.load('d-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: d-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.d view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization d-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/Hirai-Lib1-manifest_COI.csv as PairedEndFastqManifestPhred33 to e-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1100_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1100_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1100_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1100_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1100_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1100_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1100_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1100_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        19,765 reads  @    114.0 µs/read;   0.53 M reads/minute\n",
      "Finished in 2.25 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,765\n",
      "  Read 1 with adapter:                  19,726 (99.8%)\n",
      "  Read 2 with adapter:                  19,603 (99.2%)\n",
      "Pairs written (passing filters):        19,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,655,131 bp\n",
      "  Read 1:     5,826,704 bp\n",
      "  Read 2:     5,828,427 bp\n",
      "Total written (filtered):     10,632,748 bp (91.2%)\n",
      "  Read 1:     5,313,880 bp\n",
      "  Read 2:     5,318,868 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 19726 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t232\t0.0\t2\t0 226 6\n",
      "26\t19331\t0.0\t2\t18573 736 22\n",
      "27\t110\t0.0\t2\t0 102 8\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19603 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t197\t0.0\t2\t0 190 7\n",
      "26\t19304\t0.0\t2\t18885 397 22\n",
      "27\t82\t0.0\t2\t0 82\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1101_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1101_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1101_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1101_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1101_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1101_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1101_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1101_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        67,660 reads  @    105.6 µs/read;   0.57 M reads/minute\n",
      "Finished in 7.15 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,660\n",
      "  Read 1 with adapter:                  67,594 (99.9%)\n",
      "  Read 2 with adapter:                  67,298 (99.5%)\n",
      "Pairs written (passing filters):        67,660 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,889,588 bp\n",
      "  Read 1:    19,955,708 bp\n",
      "  Read 2:    19,933,880 bp\n",
      "Total written (filtered):     36,383,001 bp (91.2%)\n",
      "  Read 1:    18,198,652 bp\n",
      "  Read 2:    18,184,349 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67594 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t815\t0.0\t2\t0 792 23\n",
      "26\t66336\t0.0\t2\t64779 1535 22\n",
      "27\t343\t0.0\t2\t0 335 8\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t573\t0.0\t2\t0 551 22\n",
      "26\t66339\t0.0\t2\t64566 1667 106\n",
      "27\t324\t0.0\t2\t0 309 15\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1102_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1102_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1102_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1102_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1102_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1102_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1102_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1102_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        75,754 reads  @    115.4 µs/read;   0.52 M reads/minute\n",
      "Finished in 8.74 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,754\n",
      "  Read 1 with adapter:                  75,641 (99.9%)\n",
      "  Read 2 with adapter:                  75,674 (99.9%)\n",
      "Pairs written (passing filters):        75,754 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,675,162 bp\n",
      "  Read 1:    22,335,051 bp\n",
      "  Read 2:    22,340,111 bp\n",
      "Total written (filtered):     40,741,638 bp (91.2%)\n",
      "  Read 1:    20,368,906 bp\n",
      "  Read 2:    20,372,732 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 75641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t921\t0.0\t2\t0 891 30\n",
      "26\t74304\t0.0\t2\t72408 1864 32\n",
      "27\t328\t0.0\t2\t0 320 8\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75674 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t530\t0.0\t2\t0 510 20\n",
      "26\t74743\t0.0\t2\t72850 1832 61\n",
      "27\t341\t0.0\t2\t0 332 9\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1103_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1103_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1103_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1103_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1103_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1103_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1103_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1103_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,001 reads  @    102.6 µs/read;   0.58 M reads/minute\n",
      "Finished in 0.10 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,001\n",
      "  Read 1 with adapter:                   1,001 (100.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):           998 (99.7%)\n",
      "\n",
      "Total basepairs processed:       408,482 bp\n",
      "  Read 1:       295,294 bp\n",
      "  Read 2:       113,188 bp\n",
      "Total written (filtered):        381,651 bp (93.4%)\n",
      "  Read 1:       268,463 bp\n",
      "  Read 2:       113,188 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t9\t0.0\t2\t0 9\n",
      "26\t988\t0.0\t2\t955 32 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1104_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1104_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1104_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1104_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1104_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1104_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1104_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1104_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           409 reads  @    164.7 µs/read;   0.36 M reads/minute\n",
      "Finished in 0.07 s (170 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                409\n",
      "  Read 1 with adapter:                     409 (100.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):           409 (100.0%)\n",
      "\n",
      "Total basepairs processed:       150,978 bp\n",
      "  Read 1:       120,655 bp\n",
      "  Read 2:        30,323 bp\n",
      "Total written (filtered):        140,349 bp (93.0%)\n",
      "  Read 1:       110,026 bp\n",
      "  Read 2:        30,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t8\t0.0\t2\t0 7 1\n",
      "26\t396\t0.0\t2\t385 11\n",
      "27\t3\t0.0\t2\t0 3\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1105_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1105_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1105_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1105_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1105_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1105_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1105_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1105_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:10        96,772 reads  @    109.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 10.58 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,772\n",
      "  Read 1 with adapter:                  96,711 (99.9%)\n",
      "  Read 2 with adapter:                  96,408 (99.6%)\n",
      "Pairs written (passing filters):        96,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,043,421 bp\n",
      "  Read 1:    28,543,753 bp\n",
      "  Read 2:    28,499,668 bp\n",
      "Total written (filtered):     52,023,351 bp (91.2%)\n",
      "  Read 1:    26,029,990 bp\n",
      "  Read 2:    25,993,361 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 96711 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t1210\t0.0\t2\t0 1179 31\n",
      "26\t94933\t0.0\t2\t92752 2141 40\n",
      "27\t445\t0.0\t2\t0 431 14\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 96408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t846\t0.0\t2\t0 824 22\n",
      "26\t94982\t0.0\t2\t92608 2250 124\n",
      "27\t467\t0.0\t2\t0 443 24\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1106_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1106_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1106_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1106_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1106_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1106_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1106_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1106_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             4 reads  @   8648.5 µs/read;   0.01 M reads/minute\n",
      "Finished in 0.04 s (9270 us/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  4\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             4 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,889 bp\n",
      "  Read 1:         1,180 bp\n",
      "  Read 2:           709 bp\n",
      "Total written (filtered):          1,889 bp (100.0%)\n",
      "  Read 1:         1,180 bp\n",
      "  Read 2:           709 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1107_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1107_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1107_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1107_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1107_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1107_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1107_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1107_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        68,441 reads  @    111.6 µs/read;   0.54 M reads/minute\n",
      "Finished in 7.64 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,441\n",
      "  Read 1 with adapter:                  68,352 (99.9%)\n",
      "  Read 2 with adapter:                  68,350 (99.9%)\n",
      "Pairs written (passing filters):        68,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,366,913 bp\n",
      "  Read 1:    20,178,362 bp\n",
      "  Read 2:    20,188,551 bp\n",
      "Total written (filtered):     36,813,137 bp (91.2%)\n",
      "  Read 1:    18,401,504 bp\n",
      "  Read 2:    18,411,633 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t777\t0.0\t2\t0 752 25\n",
      "26\t67094\t0.0\t2\t65041 1995 58\n",
      "27\t371\t0.0\t2\t0 363 8\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68350 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t507\t0.0\t2\t0 493 14\n",
      "26\t67459\t0.0\t2\t65841 1567 51\n",
      "27\t331\t0.0\t2\t0 322 9\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1108_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1108_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1108_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1108_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1108_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1108_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1108_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1108_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        37,165 reads  @    109.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.07 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,165\n",
      "  Read 1 with adapter:                  37,133 (99.9%)\n",
      "  Read 2 with adapter:                  36,887 (99.3%)\n",
      "Pairs written (passing filters):        37,165 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,922,315 bp\n",
      "  Read 1:    10,962,793 bp\n",
      "  Read 2:    10,959,522 bp\n",
      "Total written (filtered):     19,998,305 bp (91.2%)\n",
      "  Read 1:     9,997,601 bp\n",
      "  Read 2:    10,000,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t538\t0.0\t2\t0 520 18\n",
      "26\t36312\t0.0\t2\t35160 1122 30\n",
      "27\t218\t0.0\t2\t0 212 6\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t402\t0.0\t2\t0 393 9\n",
      "26\t36277\t0.0\t2\t35446 809 22\n",
      "27\t178\t0.0\t2\t0 174 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1109_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1109_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1109_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1109_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1109_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1109_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1109_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1109_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        63,051 reads  @    104.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.60 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,051\n",
      "  Read 1 with adapter:                  62,976 (99.9%)\n",
      "  Read 2 with adapter:                  62,520 (99.2%)\n",
      "Pairs written (passing filters):        63,051 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,154,541 bp\n",
      "  Read 1:    18,596,305 bp\n",
      "  Read 2:    18,558,236 bp\n",
      "Total written (filtered):     33,892,148 bp (91.2%)\n",
      "  Read 1:    16,959,338 bp\n",
      "  Read 2:    16,932,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62976 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t912\t0.0\t2\t0 881 31\n",
      "26\t61517\t0.0\t2\t59639 1847 31\n",
      "27\t399\t0.0\t2\t0 387 12\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62520 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t508\t0.0\t2\t0 489 19\n",
      "26\t61578\t0.0\t2\t59684 1793 101\n",
      "27\t362\t0.0\t2\t0 347 15\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1110_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1110_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1110_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1110_21_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1110_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1110_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1110_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1110_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        79,491 reads  @    106.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 8.49 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,491\n",
      "  Read 1 with adapter:                  79,419 (99.9%)\n",
      "  Read 2 with adapter:                  79,330 (99.8%)\n",
      "Pairs written (passing filters):        79,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,893,248 bp\n",
      "  Read 1:    23,449,346 bp\n",
      "  Read 2:    23,443,902 bp\n",
      "Total written (filtered):     42,766,446 bp (91.2%)\n",
      "  Read 1:    21,385,024 bp\n",
      "  Read 2:    21,381,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79419 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t1077\t0.0\t2\t0 1041 36\n",
      "26\t77733\t0.0\t2\t75870 1836 27\n",
      "27\t457\t0.0\t2\t0 442 15\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 79330 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t527\t0.0\t2\t0 510 17\n",
      "26\t78350\t0.0\t2\t76368 1910 72\n",
      "27\t391\t0.0\t2\t0 375 16\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1111_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1111_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1111_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1111_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1111_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1111_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1111_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1111_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        64,562 reads  @    100.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.50 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,562\n",
      "  Read 1 with adapter:                  64,005 (99.1%)\n",
      "  Read 2 with adapter:                  63,709 (98.7%)\n",
      "Pairs written (passing filters):        64,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,978,137 bp\n",
      "  Read 1:    19,044,378 bp\n",
      "  Read 2:    18,933,759 bp\n",
      "Total written (filtered):     34,657,404 bp (91.3%)\n",
      "  Read 1:    17,380,038 bp\n",
      "  Read 2:    17,277,366 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64005 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t816\t0.0\t2\t0 784 32\n",
      "26\t62645\t0.0\t2\t61184 1429 32\n",
      "27\t396\t0.0\t2\t0 376 20\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t444\t0.0\t2\t0 425 19\n",
      "26\t62858\t0.0\t2\t60643 2028 187\n",
      "27\t327\t0.0\t2\t0 310 17\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1112_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1112_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1112_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1112_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1112_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1112_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1112_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1112_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        99,682 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 9.81 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             99,682\n",
      "  Read 1 with adapter:                  99,557 (99.9%)\n",
      "  Read 2 with adapter:                  81,620 (81.9%)\n",
      "Pairs written (passing filters):        99,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,694,438 bp\n",
      "  Read 1:    29,404,799 bp\n",
      "  Read 2:    26,289,639 bp\n",
      "Total written (filtered):     50,984,331 bp (91.5%)\n",
      "  Read 1:    26,816,431 bp\n",
      "  Read 2:    24,167,900 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 99557 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t1347\t0.0\t2\t0 1305 42\n",
      "26\t97346\t0.0\t2\t94646 2648 52\n",
      "27\t657\t0.0\t2\t0 628 29\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81620 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t891\t0.0\t2\t0 859 32\n",
      "26\t80151\t0.0\t2\t77385 2535 231\n",
      "27\t498\t0.0\t2\t0 475 23\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1113_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1113_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1113_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1113_27_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1113_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1113_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1113_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1113_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:08        84,838 reads  @    105.0 µs/read;   0.57 M reads/minute\n",
      "Finished in 8.91 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,838\n",
      "  Read 1 with adapter:                  84,749 (99.9%)\n",
      "  Read 2 with adapter:                  84,532 (99.6%)\n",
      "Pairs written (passing filters):        84,838 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,032,176 bp\n",
      "  Read 1:    25,026,476 bp\n",
      "  Read 2:    25,005,700 bp\n",
      "Total written (filtered):     45,631,769 bp (91.2%)\n",
      "  Read 1:    22,823,600 bp\n",
      "  Read 2:    22,808,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t1247\t0.0\t2\t0 1202 45\n",
      "26\t82793\t0.0\t2\t80737 2021 35\n",
      "27\t525\t0.0\t2\t0 502 23\n",
      "28\t123\t0.0\t2\t0 0 123\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 84532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t754\t0.0\t2\t0 723 31\n",
      "26\t83249\t0.0\t2\t81172 1978 99\n",
      "27\t457\t0.0\t2\t0 442 15\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1114_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1114_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1114_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1114_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1114_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1114_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1114_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1114_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        81,618 reads  @    101.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 8.29 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,618\n",
      "  Read 1 with adapter:                  81,513 (99.9%)\n",
      "  Read 2 with adapter:                  81,343 (99.7%)\n",
      "Pairs written (passing filters):        81,618 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,143,696 bp\n",
      "  Read 1:    24,076,163 bp\n",
      "  Read 2:    24,067,533 bp\n",
      "Total written (filtered):     43,910,133 bp (91.2%)\n",
      "  Read 1:    21,957,316 bp\n",
      "  Read 2:    21,952,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81513 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t1130\t0.0\t2\t0 1100 30\n",
      "26\t79703\t0.0\t2\t77757 1922 24\n",
      "27\t469\t0.0\t2\t0 455 14\n",
      "28\t148\t0.0\t2\t0 0 148\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81343 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t602\t0.0\t2\t0 586 16\n",
      "26\t80316\t0.0\t2\t78041 2129 146\n",
      "27\t366\t0.0\t2\t0 349 17\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1115_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1115_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1115_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1115_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1115_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1115_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1115_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1115_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        74,593 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 6.66 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,593\n",
      "  Read 1 with adapter:                  74,524 (99.9%)\n",
      "  Read 2 with adapter:                  74,508 (99.9%)\n",
      "Pairs written (passing filters):        74,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,004,149 bp\n",
      "  Read 1:    22,001,888 bp\n",
      "  Read 2:    22,002,261 bp\n",
      "Total written (filtered):     40,129,736 bp (91.2%)\n",
      "  Read 1:    20,064,484 bp\n",
      "  Read 2:    20,065,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t890\t0.0\t2\t0 869 21\n",
      "26\t72993\t0.0\t2\t71101 1861 31\n",
      "27\t456\t0.0\t2\t0 440 16\n",
      "28\t146\t0.0\t2\t0 0 146\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t574\t0.0\t2\t0 562 12\n",
      "26\t73522\t0.0\t2\t71813 1643 66\n",
      "27\t365\t0.0\t2\t0 349 16\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1116_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1116_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1116_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1116_33_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1116_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1116_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1116_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1116_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        74,044 reads  @    109.6 µs/read;   0.55 M reads/minute\n",
      "Finished in 8.12 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,044\n",
      "  Read 1 with adapter:                  73,921 (99.8%)\n",
      "  Read 2 with adapter:                  73,426 (99.2%)\n",
      "Pairs written (passing filters):        74,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,665,086 bp\n",
      "  Read 1:    21,831,912 bp\n",
      "  Read 2:    21,833,174 bp\n",
      "Total written (filtered):     39,834,839 bp (91.2%)\n",
      "  Read 1:    19,910,304 bp\n",
      "  Read 2:    19,924,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73921 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t790\t0.0\t2\t0 766 24\n",
      "26\t72679\t0.0\t2\t70761 1887 31\n",
      "27\t332\t0.0\t2\t0 320 12\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 73426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t799\t0.0\t2\t0 776 23\n",
      "26\t72184\t0.0\t2\t70512 1617 55\n",
      "27\t388\t0.0\t2\t0 377 11\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1117_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1117_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1117_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1117_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1117_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1117_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1117_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1117_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:11       107,426 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 11.56 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,426\n",
      "  Read 1 with adapter:                 107,086 (99.7%)\n",
      "  Read 2 with adapter:                 107,098 (99.7%)\n",
      "Pairs written (passing filters):       107,426 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,323,841 bp\n",
      "  Read 1:    31,647,960 bp\n",
      "  Read 2:    31,675,881 bp\n",
      "Total written (filtered):     57,755,893 bp (91.2%)\n",
      "  Read 1:    28,864,211 bp\n",
      "  Read 2:    28,891,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 107086 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t1126\t0.0\t2\t0 1094 32\n",
      "26\t105275\t0.0\t2\t102793 2443 39\n",
      "27\t483\t0.0\t2\t0 471 12\n",
      "28\t140\t0.0\t2\t0 0 140\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 107098 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t872\t0.0\t2\t0 840 32\n",
      "26\t105598\t0.0\t2\t102591 2824 183\n",
      "27\t517\t0.0\t2\t0 485 32\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1118_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1118_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1118_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1118_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1118_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1118_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1118_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1118_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,875 reads  @     83.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 6.87 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,875\n",
      "  Read 1 with adapter:                  81,739 (99.8%)\n",
      "  Read 2 with adapter:                  81,787 (99.9%)\n",
      "Pairs written (passing filters):        81,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,291,548 bp\n",
      "  Read 1:    24,142,407 bp\n",
      "  Read 2:    24,149,141 bp\n",
      "Total written (filtered):     44,040,201 bp (91.2%)\n",
      "  Read 1:    22,017,357 bp\n",
      "  Read 2:    22,022,844 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t782\t0.0\t2\t0 762 20\n",
      "26\t80356\t0.0\t2\t78576 1756 24\n",
      "27\t432\t0.0\t2\t0 423 9\n",
      "28\t131\t0.0\t2\t0 0 131\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81787 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t554\t0.0\t2\t0 534 20\n",
      "26\t80780\t0.0\t2\t78652 2013 115\n",
      "27\t397\t0.0\t2\t0 376 21\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1119_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1119_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1119_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1119_39_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1119_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1119_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1119_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1119_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        73,897 reads  @     91.0 µs/read;   0.66 M reads/minute\n",
      "Finished in 6.73 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,897\n",
      "  Read 1 with adapter:                  73,692 (99.7%)\n",
      "  Read 2 with adapter:                  73,380 (99.3%)\n",
      "Pairs written (passing filters):        73,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,522,949 bp\n",
      "  Read 1:    21,779,787 bp\n",
      "  Read 2:    21,743,162 bp\n",
      "Total written (filtered):     39,698,004 bp (91.2%)\n",
      "  Read 1:    19,862,602 bp\n",
      "  Read 2:    19,835,402 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t687\t0.0\t2\t0 667 20\n",
      "26\t72511\t0.0\t2\t70882 1605 24\n",
      "27\t353\t0.0\t2\t0 334 19\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 73380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t504\t0.0\t2\t0 486 18\n",
      "26\t72431\t0.0\t2\t69648 2509 274\n",
      "27\t374\t0.0\t2\t0 344 30\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1120_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1120_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1120_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1120_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1120_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1120_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1120_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1120_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        59,387 reads  @    102.5 µs/read;   0.59 M reads/minute\n",
      "Finished in 6.09 s (103 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,387\n",
      "  Read 1 with adapter:                  59,340 (99.9%)\n",
      "  Read 2 with adapter:                  56,881 (95.8%)\n",
      "Pairs written (passing filters):        59,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,603,262 bp\n",
      "  Read 1:    17,517,013 bp\n",
      "  Read 2:    17,086,249 bp\n",
      "Total written (filtered):     31,581,782 bp (91.3%)\n",
      "  Read 1:    15,974,132 bp\n",
      "  Read 2:    15,607,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59340 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t618\t0.0\t2\t0 602 16\n",
      "26\t58317\t0.0\t2\t56931 1358 28\n",
      "27\t324\t0.0\t2\t0 320 4\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t608\t0.0\t2\t0 569 39\n",
      "26\t55908\t0.0\t2\t53916 1803 189\n",
      "27\t321\t0.0\t2\t0 299 22\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1121_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1121_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1121_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1121_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1121_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1121_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1121_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1121_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        94,041 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 7.76 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,041\n",
      "  Read 1 with adapter:                  93,322 (99.2%)\n",
      "  Read 2 with adapter:                  93,626 (99.6%)\n",
      "Pairs written (passing filters):        94,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,380,489 bp\n",
      "  Read 1:    27,663,871 bp\n",
      "  Read 2:    27,716,618 bp\n",
      "Total written (filtered):     50,520,100 bp (91.2%)\n",
      "  Read 1:    25,237,404 bp\n",
      "  Read 2:    25,282,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 93322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t921\t0.0\t2\t0 900 21\n",
      "26\t91720\t0.0\t2\t89708 1985 27\n",
      "27\t468\t0.0\t2\t0 453 15\n",
      "28\t176\t0.0\t2\t0 0 176\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 93626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t774\t0.0\t2\t0 749 25\n",
      "26\t92370\t0.0\t2\t89600 2531 239\n",
      "27\t440\t0.0\t2\t0 404 36\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1122_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1122_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1122_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1122_45_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1122_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1122_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1122_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1122_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        82,131 reads  @     76.3 µs/read;   0.79 M reads/minute\n",
      "Finished in 6.27 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,131\n",
      "  Read 1 with adapter:                  81,952 (99.8%)\n",
      "  Read 2 with adapter:                  81,913 (99.7%)\n",
      "Pairs written (passing filters):        82,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,434,812 bp\n",
      "  Read 1:    24,214,585 bp\n",
      "  Read 2:    24,220,227 bp\n",
      "Total written (filtered):     44,174,639 bp (91.2%)\n",
      "  Read 1:    22,083,926 bp\n",
      "  Read 2:    22,090,713 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t796\t0.0\t2\t0 770 26\n",
      "26\t80531\t0.0\t2\t78655 1836 40\n",
      "27\t395\t0.0\t2\t0 382 13\n",
      "28\t192\t0.0\t2\t0 0 192\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81913 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t648\t0.0\t2\t0 622 26\n",
      "26\t80848\t0.0\t2\t78369 2317 162\n",
      "27\t334\t0.0\t2\t0 319 15\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1123_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1123_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1123_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1123_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1123_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1123_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1123_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1123_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:11       112,213 reads  @    105.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 11.83 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,213\n",
      "  Read 1 with adapter:                 112,075 (99.9%)\n",
      "  Read 2 with adapter:                 112,072 (99.9%)\n",
      "Pairs written (passing filters):       112,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,193,182 bp\n",
      "  Read 1:    33,094,214 bp\n",
      "  Read 2:    33,098,968 bp\n",
      "Total written (filtered):     60,366,223 bp (91.2%)\n",
      "  Read 1:    30,180,692 bp\n",
      "  Read 2:    30,185,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 112075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t1261\t0.0\t2\t0 1226 35\n",
      "26\t109930\t0.0\t2\t106524 3347 59\n",
      "27\t703\t0.0\t2\t0 673 30\n",
      "28\t123\t0.0\t2\t0 0 123\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 112072 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t1030\t0.0\t2\t0 994 36\n",
      "26\t110350\t0.0\t2\t106855 3323 172\n",
      "27\t581\t0.0\t2\t0 555 26\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1124_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1124_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1124_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1124_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1124_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1124_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1124_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1124_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         6,332 reads  @     95.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 0.61 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,332\n",
      "  Read 1 with adapter:                   6,305 (99.6%)\n",
      "  Read 2 with adapter:                   6,250 (98.7%)\n",
      "Pairs written (passing filters):         6,332 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,734,515 bp\n",
      "  Read 1:     1,867,936 bp\n",
      "  Read 2:     1,866,579 bp\n",
      "Total written (filtered):      3,408,116 bp (91.3%)\n",
      "  Read 1:     1,703,996 bp\n",
      "  Read 2:     1,704,120 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 6305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t69\t0.0\t2\t0 67 2\n",
      "26\t6175\t0.0\t2\t5880 285 10\n",
      "27\t39\t0.0\t2\t0 36 3\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 6250 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t77\t0.0\t2\t0 75 2\n",
      "26\t6132\t0.0\t2\t5985 140 7\n",
      "27\t30\t0.0\t2\t0 30\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1125_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1125_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1125_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1125_51_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1125_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1125_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1125_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1125_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        81,587 reads  @     78.4 µs/read;   0.76 M reads/minute\n",
      "Finished in 6.40 s (78 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,587\n",
      "  Read 1 with adapter:                  81,479 (99.9%)\n",
      "  Read 2 with adapter:                  81,440 (99.8%)\n",
      "Pairs written (passing filters):        81,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,124,010 bp\n",
      "  Read 1:    24,060,505 bp\n",
      "  Read 2:    24,063,505 bp\n",
      "Total written (filtered):     43,888,591 bp (91.2%)\n",
      "  Read 1:    21,942,220 bp\n",
      "  Read 2:    21,946,371 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t913\t0.0\t2\t0 881 32\n",
      "26\t79881\t0.0\t2\t78041 1809 31\n",
      "27\t462\t0.0\t2\t0 449 13\n",
      "28\t182\t0.0\t2\t0 0 182\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t640\t0.0\t2\t0 620 20\n",
      "26\t80382\t0.0\t2\t78325 1958 99\n",
      "27\t374\t0.0\t2\t0 354 20\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1126_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1126_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1126_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1126_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1126_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1126_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1126_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1126_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        77,863 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 6.45 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,863\n",
      "  Read 1 with adapter:                  77,776 (99.9%)\n",
      "  Read 2 with adapter:                  77,776 (99.9%)\n",
      "Pairs written (passing filters):        77,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,928,683 bp\n",
      "  Read 1:    22,964,375 bp\n",
      "  Read 2:    22,964,308 bp\n",
      "Total written (filtered):     41,884,767 bp (91.2%)\n",
      "  Read 1:    20,942,462 bp\n",
      "  Read 2:    20,942,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 77776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t891\t0.0\t2\t0 859 32\n",
      "26\t76240\t0.0\t2\t74486 1726 28\n",
      "27\t454\t0.0\t2\t0 443 11\n",
      "28\t139\t0.0\t2\t0 0 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 77776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t496\t0.0\t2\t0 485 11\n",
      "26\t76900\t0.0\t2\t75159 1695 46\n",
      "27\t353\t0.0\t2\t0 343 10\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1127_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1127_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1127_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1127_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1127_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1127_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1127_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1127_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:09        84,215 reads  @    112.9 µs/read;   0.53 M reads/minute\n",
      "Finished in 9.51 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,215\n",
      "  Read 1 with adapter:                  84,075 (99.8%)\n",
      "  Read 2 with adapter:                  83,646 (99.3%)\n",
      "Pairs written (passing filters):        84,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,599,466 bp\n",
      "  Read 1:    24,834,375 bp\n",
      "  Read 2:    24,765,091 bp\n",
      "Total written (filtered):     45,235,718 bp (91.2%)\n",
      "  Read 1:    22,645,308 bp\n",
      "  Read 2:    22,590,410 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84075 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t949\t0.0\t2\t0 933 16\n",
      "26\t82541\t0.0\t2\t80840 1668 33\n",
      "27\t441\t0.0\t2\t0 423 18\n",
      "28\t104\t0.0\t2\t0 0 104\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 83646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t569\t0.0\t2\t0 542 27\n",
      "26\t82605\t0.0\t2\t80037 2354 214\n",
      "27\t374\t0.0\t2\t0 350 24\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1128_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1128_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1128_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1128_57_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1128_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1128_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1128_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1128_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        60,832 reads  @    104.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.37 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,832\n",
      "  Read 1 with adapter:                  60,790 (99.9%)\n",
      "  Read 2 with adapter:                  54,479 (89.6%)\n",
      "Pairs written (passing filters):        60,832 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,765,191 bp\n",
      "  Read 1:    17,943,517 bp\n",
      "  Read 2:    16,821,674 bp\n",
      "Total written (filtered):     31,768,745 bp (91.4%)\n",
      "  Read 1:    16,363,284 bp\n",
      "  Read 2:    15,405,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60790 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t684\t0.0\t2\t0 665 19\n",
      "26\t59680\t0.0\t2\t58313 1343 24\n",
      "27\t351\t0.0\t2\t0 343 8\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t564\t0.0\t2\t0 544 20\n",
      "26\t53561\t0.0\t2\t51977 1482 102\n",
      "27\t309\t0.0\t2\t0 293 16\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1129_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1129_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1129_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1129_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1129_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1129_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1129_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1129_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10       112,192 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 10.16 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,192\n",
      "  Read 1 with adapter:                 112,122 (99.9%)\n",
      "  Read 2 with adapter:                 111,331 (99.2%)\n",
      "Pairs written (passing filters):       112,190 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,104,177 bp\n",
      "  Read 1:    33,092,243 bp\n",
      "  Read 2:    33,011,934 bp\n",
      "Total written (filtered):     60,294,810 bp (91.2%)\n",
      "  Read 1:    30,177,101 bp\n",
      "  Read 2:    30,117,709 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 112122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t1293\t0.0\t2\t0 1269 24\n",
      "26\t110060\t0.0\t2\t107608 2411 41\n",
      "27\t545\t0.0\t2\t0 531 14\n",
      "28\t157\t0.0\t2\t0 0 157\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 111331 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t1004\t0.0\t2\t0 957 47\n",
      "26\t109707\t0.0\t2\t106831 2709 167\n",
      "27\t437\t0.0\t2\t0 419 18\n",
      "28\t138\t0.0\t2\t0 0 138\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1130_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1130_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1130_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1130_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1130_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1130_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1130_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1130_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        10,791 reads  @    109.6 µs/read;   0.55 M reads/minute\n",
      "Finished in 1.18 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,791\n",
      "  Read 1 with adapter:                  10,782 (99.9%)\n",
      "  Read 2 with adapter:                  10,725 (99.4%)\n",
      "Pairs written (passing filters):        10,791 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,363,816 bp\n",
      "  Read 1:     3,183,341 bp\n",
      "  Read 2:     3,180,475 bp\n",
      "Total written (filtered):      5,804,706 bp (91.2%)\n",
      "  Read 1:     2,903,071 bp\n",
      "  Read 2:     2,901,635 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 10782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t143\t0.0\t2\t0 140 3\n",
      "26\t10560\t0.0\t2\t10053 494 13\n",
      "27\t49\t0.0\t2\t0 48 1\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 10725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t87\t0.0\t2\t0 85 2\n",
      "26\t10565\t0.0\t2\t9992 517 56\n",
      "27\t57\t0.0\t2\t0 52 5\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1131_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1131_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1131_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1131_63_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1131_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1131_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1131_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1131_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        86,572 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 7.42 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,572\n",
      "  Read 1 with adapter:                  86,510 (99.9%)\n",
      "  Read 2 with adapter:                  86,496 (99.9%)\n",
      "Pairs written (passing filters):        86,572 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,074,578 bp\n",
      "  Read 1:    25,537,629 bp\n",
      "  Read 2:    25,536,949 bp\n",
      "Total written (filtered):     46,577,114 bp (91.2%)\n",
      "  Read 1:    23,288,828 bp\n",
      "  Read 2:    23,288,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 86510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t1134\t0.0\t2\t0 1099 35\n",
      "26\t84684\t0.0\t2\t82353 2279 52\n",
      "27\t509\t0.0\t2\t0 488 21\n",
      "28\t133\t0.0\t2\t0 0 133\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 86496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t734\t0.0\t2\t0 709 25\n",
      "26\t85205\t0.0\t2\t83183 1975 47\n",
      "27\t445\t0.0\t2\t0 433 12\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1132_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1132_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1132_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1132_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1132_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1132_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1132_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1132_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,496 reads  @     86.7 µs/read;   0.69 M reads/minute\n",
      "Finished in 0.31 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,496\n",
      "  Read 1 with adapter:                   3,486 (99.7%)\n",
      "  Read 2 with adapter:                   3,455 (98.8%)\n",
      "Pairs written (passing filters):         3,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,061,914 bp\n",
      "  Read 1:     1,031,316 bp\n",
      "  Read 2:     1,030,598 bp\n",
      "Total written (filtered):      1,881,501 bp (91.3%)\n",
      "  Read 1:       940,687 bp\n",
      "  Read 2:       940,814 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 3486 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t45\t0.0\t2\t0 43 2\n",
      "26\t3403\t0.0\t2\t3265 132 6\n",
      "27\t22\t0.0\t2\t0 22\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 3455 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t62\t0.0\t2\t0 56 6\n",
      "26\t3371\t0.0\t2\t3271 98 2\n",
      "27\t20\t0.0\t2\t0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1148_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1148_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1148_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1148_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1148_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1148_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1148_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1148_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        33,191 reads  @    111.6 µs/read;   0.54 M reads/minute\n",
      "Finished in 3.71 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,191\n",
      "  Read 1 with adapter:                  33,092 (99.7%)\n",
      "  Read 2 with adapter:                  32,974 (99.3%)\n",
      "Pairs written (passing filters):        33,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,565,652 bp\n",
      "  Read 1:     9,777,697 bp\n",
      "  Read 2:     9,787,955 bp\n",
      "Total written (filtered):     17,848,243 bp (91.2%)\n",
      "  Read 1:     8,917,418 bp\n",
      "  Read 2:     8,930,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33092 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t367\t0.0\t2\t0 352 15\n",
      "26\t32459\t0.0\t2\t30999 1399 61\n",
      "27\t190\t0.0\t2\t0 176 14\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t356\t0.0\t2\t0 351 5\n",
      "26\t32447\t0.0\t2\t31730 698 19\n",
      "27\t128\t0.0\t2\t0 124 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1149_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1149_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1149_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1149_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1149_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1149_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1149_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1149_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:13       130,709 reads  @    101.1 µs/read;   0.59 M reads/minute\n",
      "Finished in 13.22 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            130,709\n",
      "  Read 1 with adapter:                 130,230 (99.6%)\n",
      "  Read 2 with adapter:                 130,080 (99.5%)\n",
      "Pairs written (passing filters):       130,709 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,972,885 bp\n",
      "  Read 1:    38,457,421 bp\n",
      "  Read 2:    38,515,464 bp\n",
      "Total written (filtered):     70,205,640 bp (91.2%)\n",
      "  Read 1:    35,071,824 bp\n",
      "  Read 2:    35,133,816 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 130230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1382\t0.0\t2\t0 1349 33\n",
      "26\t127855\t0.0\t2\t124468 3321 66\n",
      "27\t751\t0.0\t2\t0 718 33\n",
      "28\t183\t0.0\t2\t0 0 183\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 130080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t1094\t0.0\t2\t0 1072 22\n",
      "26\t128282\t0.0\t2\t124890 3229 163\n",
      "27\t582\t0.0\t2\t0 559 23\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1150_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1150_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1150_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1150_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1150_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1150_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1150_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1150_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:12       116,076 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 12.27 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,076\n",
      "  Read 1 with adapter:                 115,572 (99.6%)\n",
      "  Read 2 with adapter:                 115,949 (99.9%)\n",
      "Pairs written (passing filters):       116,076 (100.0%)\n",
      "\n",
      "Total basepairs processed:    68,369,704 bp\n",
      "  Read 1:    34,135,668 bp\n",
      "  Read 2:    34,234,036 bp\n",
      "Total written (filtered):     62,351,077 bp (91.2%)\n",
      "  Read 1:    31,131,453 bp\n",
      "  Read 2:    31,219,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 115572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t1300\t0.0\t2\t0 1257 43\n",
      "26\t113541\t0.0\t2\t110543 2949 49\n",
      "27\t587\t0.0\t2\t0 568 19\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 115949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t836\t0.0\t2\t0 810 26\n",
      "26\t114488\t0.0\t2\t111758 2641 89\n",
      "27\t532\t0.0\t2\t0 516 16\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1151_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1151_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1151_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1151_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1151_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1151_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1151_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1151_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,140 reads  @     89.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 0.10 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,140\n",
      "  Read 1 with adapter:                   1,128 (98.9%)\n",
      "  Read 2 with adapter:                       6 (0.5%)\n",
      "Pairs written (passing filters):         1,137 (99.7%)\n",
      "\n",
      "Total basepairs processed:       465,856 bp\n",
      "  Read 1:       333,328 bp\n",
      "  Read 2:       132,528 bp\n",
      "Total written (filtered):        435,570 bp (93.5%)\n",
      "  Read 1:       303,198 bp\n",
      "  Read 2:       132,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1128 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t15\t0.0\t2\t0 13 2\n",
      "26\t1101\t0.0\t2\t1079 22\n",
      "27\t10\t0.0\t2\t0 10\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 6 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t6\t0.0\t2\t6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1152_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1152_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1152_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1152_75_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1152_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1152_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1152_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1152_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        69,007 reads  @    110.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 7.61 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,007\n",
      "  Read 1 with adapter:                  68,773 (99.7%)\n",
      "  Read 2 with adapter:                  68,770 (99.7%)\n",
      "Pairs written (passing filters):        69,007 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,643,954 bp\n",
      "  Read 1:    20,307,505 bp\n",
      "  Read 2:    20,336,449 bp\n",
      "Total written (filtered):     37,068,498 bp (91.2%)\n",
      "  Read 1:    18,519,728 bp\n",
      "  Read 2:    18,548,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t767\t0.0\t2\t0 746 21\n",
      "26\t67511\t0.0\t2\t65881 1606 24\n",
      "27\t392\t0.0\t2\t0 385 7\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68770 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t725\t0.0\t2\t0 700 25\n",
      "26\t67616\t0.0\t2\t65731 1769 116\n",
      "27\t370\t0.0\t2\t0 360 10\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1153_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1153_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1153_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1153_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1153_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1153_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1153_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1153_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:11       105,482 reads  @    110.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 11.62 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            105,482\n",
      "  Read 1 with adapter:                 104,956 (99.5%)\n",
      "  Read 2 with adapter:                 105,191 (99.7%)\n",
      "Pairs written (passing filters):       105,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:    62,085,871 bp\n",
      "  Read 1:    31,005,554 bp\n",
      "  Read 2:    31,080,317 bp\n",
      "Total written (filtered):     56,623,197 bp (91.2%)\n",
      "  Read 1:    28,277,397 bp\n",
      "  Read 2:    28,345,800 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 104956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t1222\t0.0\t2\t0 1187 35\n",
      "26\t103110\t0.0\t2\t100615 2450 45\n",
      "27\t501\t0.0\t2\t0 483 18\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 105191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t931\t0.0\t2\t0 903 28\n",
      "26\t103705\t0.0\t2\t101272 2351 82\n",
      "27\t468\t0.0\t2\t0 459 9\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1154_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1154_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1154_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1154_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1154_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1154_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1154_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1154_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             5 reads  @   6699.2 µs/read;   0.01 M reads/minute\n",
      "Finished in 0.04 s (7197 us/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  5\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       5 (100.0%)\n",
      "Pairs written (passing filters):             5 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,597 bp\n",
      "  Read 1:           122 bp\n",
      "  Read 2:         1,475 bp\n",
      "Total written (filtered):          1,467 bp (91.9%)\n",
      "  Read 1:           122 bp\n",
      "  Read 2:         1,345 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 5 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t5\t0.0\t2\t5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1155_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1155_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1155_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1155_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1155_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1155_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1155_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1155_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8=-------] 00:00:08        76,987 reads  @    108.2 µs/read;   0.55 M reads/minute\n",
      "Finished in 8.34 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,987\n",
      "  Read 1 with adapter:                  76,618 (99.5%)\n",
      "  Read 2 with adapter:                  76,905 (99.9%)\n",
      "Pairs written (passing filters):        76,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,340,246 bp\n",
      "  Read 1:    22,629,668 bp\n",
      "  Read 2:    22,710,578 bp\n",
      "Total written (filtered):     41,349,098 bp (91.2%)\n",
      "  Read 1:    20,637,869 bp\n",
      "  Read 2:    20,711,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 76618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t841\t0.0\t2\t0 813 28\n",
      "26\t75186\t0.0\t2\t73282 1880 24\n",
      "27\t454\t0.0\t2\t0 440 14\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 76905 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t558\t0.0\t2\t0 537 21\n",
      "26\t75925\t0.0\t2\t74265 1598 62\n",
      "27\t359\t0.0\t2\t0 348 11\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1156_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1156_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1156_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1156_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1156_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1156_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1156_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1156_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        78,909 reads  @    103.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 8.20 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,909\n",
      "  Read 1 with adapter:                  78,659 (99.7%)\n",
      "  Read 2 with adapter:                  78,286 (99.2%)\n",
      "Pairs written (passing filters):        78,909 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,512,895 bp\n",
      "  Read 1:    23,244,432 bp\n",
      "  Read 2:    23,268,463 bp\n",
      "Total written (filtered):     42,433,130 bp (91.2%)\n",
      "  Read 1:    21,199,637 bp\n",
      "  Read 2:    21,233,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 78659 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t930\t0.0\t2\t0 898 32\n",
      "26\t77094\t0.0\t2\t75111 1952 31\n",
      "27\t483\t0.0\t2\t0 469 14\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78286 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t800\t0.0\t2\t0 785 15\n",
      "26\t77063\t0.0\t2\t75259 1745 59\n",
      "27\t344\t0.0\t2\t0 328 16\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1157_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1157_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1157_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1157_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1157_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1157_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1157_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1157_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        47,954 reads  @    115.4 µs/read;   0.52 M reads/minute\n",
      "Finished in 5.54 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,954\n",
      "  Read 1 with adapter:                  47,878 (99.8%)\n",
      "  Read 2 with adapter:                  47,643 (99.4%)\n",
      "Pairs written (passing filters):        47,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,264,013 bp\n",
      "  Read 1:    14,142,055 bp\n",
      "  Read 2:    14,121,958 bp\n",
      "Total written (filtered):     25,780,753 bp (91.2%)\n",
      "  Read 1:    12,897,404 bp\n",
      "  Read 2:    12,883,349 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47878 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t605\t0.0\t2\t0 575 30\n",
      "26\t46848\t0.0\t2\t45289 1521 38\n",
      "27\t310\t0.0\t2\t0 294 16\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t425\t0.0\t2\t0 405 20\n",
      "26\t46886\t0.0\t2\t45384 1441 61\n",
      "27\t280\t0.0\t2\t0 271 9\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1158_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1158_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1158_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1158_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1158_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1158_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1158_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1158_87_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:10       100,821 reads  @    108.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 10.92 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,821\n",
      "  Read 1 with adapter:                 100,658 (99.8%)\n",
      "  Read 2 with adapter:                 100,445 (99.6%)\n",
      "Pairs written (passing filters):       100,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,454,465 bp\n",
      "  Read 1:    29,727,763 bp\n",
      "  Read 2:    29,726,702 bp\n",
      "Total written (filtered):     54,226,322 bp (91.2%)\n",
      "  Read 1:    27,111,006 bp\n",
      "  Read 2:    27,115,316 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 100658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1107\t0.0\t2\t0 1084 23\n",
      "26\t98742\t0.0\t2\t96482 2224 36\n",
      "27\t626\t0.0\t2\t0 609 17\n",
      "28\t124\t0.0\t2\t0 0 124\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 100445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t648\t0.0\t2\t0 627 21\n",
      "26\t99291\t0.0\t2\t96815 2366 110\n",
      "27\t440\t0.0\t2\t0 434 6\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1159_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1159_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1159_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1159_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1159_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1159_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1159_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1159_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        59,865 reads  @    100.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.01 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,865\n",
      "  Read 1 with adapter:                  59,808 (99.9%)\n",
      "  Read 2 with adapter:                  59,176 (98.8%)\n",
      "Pairs written (passing filters):        59,864 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,218,258 bp\n",
      "  Read 1:    17,655,382 bp\n",
      "  Read 2:    17,562,876 bp\n",
      "Total written (filtered):     32,124,678 bp (91.2%)\n",
      "  Read 1:    16,100,339 bp\n",
      "  Read 2:    16,024,339 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t696\t0.0\t2\t0 675 21\n",
      "26\t58635\t0.0\t2\t57276 1336 23\n",
      "27\t392\t0.0\t2\t0 381 11\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t430\t0.0\t2\t0 412 18\n",
      "26\t58391\t0.0\t2\t56394 1815 182\n",
      "27\t283\t0.0\t2\t0 267 16\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1160_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1160_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1160_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1160_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1160_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1160_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1160_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1160_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:07        73,737 reads  @     96.9 µs/read;   0.62 M reads/minute\n",
      "Finished in 7.15 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,737\n",
      "  Read 1 with adapter:                  73,637 (99.9%)\n",
      "  Read 2 with adapter:                  57,841 (78.4%)\n",
      "Pairs written (passing filters):        73,737 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,740,790 bp\n",
      "  Read 1:    21,741,544 bp\n",
      "  Read 2:    18,999,246 bp\n",
      "Total written (filtered):     37,322,924 bp (91.6%)\n",
      "  Read 1:    19,827,242 bp\n",
      "  Read 2:    17,495,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t869\t0.0\t2\t0 848 21\n",
      "26\t72125\t0.0\t2\t70309 1781 35\n",
      "27\t529\t0.0\t2\t0 511 18\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t622\t0.0\t2\t0 593 29\n",
      "26\t56854\t0.0\t2\t54808 1875 171\n",
      "27\t314\t0.0\t2\t0 291 23\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1161_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1161_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1161_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1161_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1161_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1161_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1161_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1161_93_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:09        94,820 reads  @    104.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 9.88 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,820\n",
      "  Read 1 with adapter:                  94,688 (99.9%)\n",
      "  Read 2 with adapter:                  94,441 (99.6%)\n",
      "Pairs written (passing filters):        94,820 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,897,234 bp\n",
      "  Read 1:    27,959,570 bp\n",
      "  Read 2:    27,937,664 bp\n",
      "Total written (filtered):     50,980,548 bp (91.2%)\n",
      "  Read 1:    25,497,969 bp\n",
      "  Read 2:    25,482,579 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 94688 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t1044\t0.0\t2\t0 1012 32\n",
      "26\t92867\t0.0\t2\t90741 2092 34\n",
      "27\t589\t0.0\t2\t0 567 22\n",
      "28\t136\t0.0\t2\t0 0 136\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 94441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t832\t0.0\t2\t0 803 29\n",
      "26\t93068\t0.0\t2\t90562 2355 151\n",
      "27\t447\t0.0\t2\t0 422 25\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1162_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1162_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1162_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1162_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1162_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1162_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1162_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1162_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        78,250 reads  @    101.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 7.91 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,250\n",
      "  Read 1 with adapter:                  78,125 (99.8%)\n",
      "  Read 2 with adapter:                  78,057 (99.8%)\n",
      "Pairs written (passing filters):        78,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,147,361 bp\n",
      "  Read 1:    23,074,016 bp\n",
      "  Read 2:    23,073,345 bp\n",
      "Total written (filtered):     42,087,212 bp (91.2%)\n",
      "  Read 1:    21,043,107 bp\n",
      "  Read 2:    21,044,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 78125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t916\t0.0\t2\t0 888 28\n",
      "26\t76607\t0.0\t2\t74776 1790 41\n",
      "27\t433\t0.0\t2\t0 421 12\n",
      "28\t120\t0.0\t2\t0 0 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t598\t0.0\t2\t0 578 20\n",
      "26\t77031\t0.0\t2\t74957 1964 110\n",
      "27\t356\t0.0\t2\t0 342 14\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1163_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1163_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1163_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1163_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1163_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1163_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1163_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1163_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        88,482 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 7.74 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             88,482\n",
      "  Read 1 with adapter:                  88,412 (99.9%)\n",
      "  Read 2 with adapter:                  88,403 (99.9%)\n",
      "Pairs written (passing filters):        88,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,200,000 bp\n",
      "  Read 1:    26,100,644 bp\n",
      "  Read 2:    26,099,356 bp\n",
      "Total written (filtered):     47,603,188 bp (91.2%)\n",
      "  Read 1:    23,802,039 bp\n",
      "  Read 2:    23,801,149 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 88412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t948\t0.0\t2\t0 917 31\n",
      "26\t86675\t0.0\t2\t84588 2054 33\n",
      "27\t557\t0.0\t2\t0 538 19\n",
      "28\t187\t0.0\t2\t0 0 187\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 88403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t677\t0.0\t2\t0 653 24\n",
      "26\t87249\t0.0\t2\t85239 1952 58\n",
      "27\t416\t0.0\t2\t0 395 21\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1164_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1164_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1164_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1164_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1164_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1164_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1164_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1164_99_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:07        72,973 reads  @    107.2 µs/read;   0.56 M reads/minute\n",
      "Finished in 7.83 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,973\n",
      "  Read 1 with adapter:                  72,776 (99.7%)\n",
      "  Read 2 with adapter:                  72,425 (99.2%)\n",
      "Pairs written (passing filters):        72,973 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,021,851 bp\n",
      "  Read 1:    21,503,226 bp\n",
      "  Read 2:    21,518,625 bp\n",
      "Total written (filtered):     39,247,261 bp (91.2%)\n",
      "  Read 1:    19,611,229 bp\n",
      "  Read 2:    19,636,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t663\t0.0\t2\t0 648 15\n",
      "26\t71628\t0.0\t2\t69834 1766 28\n",
      "27\t394\t0.0\t2\t0 383 11\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72425 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t766\t0.0\t2\t0 738 28\n",
      "26\t71252\t0.0\t2\t69595 1597 60\n",
      "27\t341\t0.0\t2\t0 332 9\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1165_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1165_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1165_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1165_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1165_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1165_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1165_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1165_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:11       111,236 reads  @    104.6 µs/read;   0.57 M reads/minute\n",
      "Finished in 11.64 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            111,236\n",
      "  Read 1 with adapter:                 111,012 (99.8%)\n",
      "  Read 2 with adapter:                 110,882 (99.7%)\n",
      "Pairs written (passing filters):       111,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    65,582,533 bp\n",
      "  Read 1:    32,787,666 bp\n",
      "  Read 2:    32,794,867 bp\n",
      "Total written (filtered):     59,813,641 bp (91.2%)\n",
      "  Read 1:    29,901,454 bp\n",
      "  Read 2:    29,912,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 111012 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t942\t0.0\t2\t0 917 25\n",
      "26\t109255\t0.0\t2\t106730 2504 21\n",
      "27\t624\t0.0\t2\t0 603 21\n",
      "28\t150\t0.0\t2\t0 0 150\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 110882 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t846\t0.0\t2\t0 820 26\n",
      "26\t109390\t0.0\t2\t106719 2544 127\n",
      "27\t546\t0.0\t2\t0 522 24\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1166_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1166_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1166_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1166_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1166_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1166_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1166_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1166_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        93,227 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 7.73 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,227\n",
      "  Read 1 with adapter:                  93,006 (99.8%)\n",
      "  Read 2 with adapter:                  93,133 (99.9%)\n",
      "Pairs written (passing filters):        93,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,973,377 bp\n",
      "  Read 1:    27,474,586 bp\n",
      "  Read 2:    27,498,791 bp\n",
      "Total written (filtered):     50,134,090 bp (91.2%)\n",
      "  Read 1:    25,056,523 bp\n",
      "  Read 2:    25,077,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 93006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t776\t0.0\t2\t0 758 18\n",
      "26\t91547\t0.0\t2\t89549 1965 33\n",
      "27\t503\t0.0\t2\t0 488 15\n",
      "28\t135\t0.0\t2\t0 0 135\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 93133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t673\t0.0\t2\t0 661 12\n",
      "26\t91926\t0.0\t2\t89801 2045 80\n",
      "27\t445\t0.0\t2\t0 435 10\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1167_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1167_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1167_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1167_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1167_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1167_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1167_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1167_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        51,777 reads  @     90.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.66 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,777\n",
      "  Read 1 with adapter:                  51,723 (99.9%)\n",
      "  Read 2 with adapter:                  51,376 (99.2%)\n",
      "Pairs written (passing filters):        51,775 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,485,439 bp\n",
      "  Read 1:    15,268,224 bp\n",
      "  Read 2:    15,217,215 bp\n",
      "Total written (filtered):     27,804,470 bp (91.2%)\n",
      "  Read 1:    13,922,932 bp\n",
      "  Read 2:    13,881,538 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51723 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t488\t0.0\t2\t0 476 12\n",
      "26\t50810\t0.0\t2\t49642 1145 23\n",
      "27\t314\t0.0\t2\t0 304 10\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t359\t0.0\t2\t0 342 17\n",
      "26\t50733\t0.0\t2\t49177 1430 126\n",
      "27\t244\t0.0\t2\t0 236 8\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1168_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1168_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1168_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1168_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1168_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1168_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1168_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1168_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        60,094 reads  @    102.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 6.18 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,094\n",
      "  Read 1 with adapter:                  59,996 (99.8%)\n",
      "  Read 2 with adapter:                  58,444 (97.3%)\n",
      "Pairs written (passing filters):        60,093 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,165,362 bp\n",
      "  Read 1:    17,716,715 bp\n",
      "  Read 2:    17,448,647 bp\n",
      "Total written (filtered):     32,086,107 bp (91.2%)\n",
      "  Read 1:    16,156,694 bp\n",
      "  Read 2:    15,929,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t534\t0.0\t2\t0 519 15\n",
      "26\t59043\t0.0\t2\t57719 1306 18\n",
      "27\t352\t0.0\t2\t0 342 10\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58444 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t611\t0.0\t2\t0 591 20\n",
      "26\t57501\t0.0\t2\t55919 1489 93\n",
      "27\t299\t0.0\t2\t0 283 16\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1169_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1169_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1169_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1169_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1169_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1169_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1169_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1169_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        83,445 reads  @     80.0 µs/read;   0.75 M reads/minute\n",
      "Finished in 6.68 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,445\n",
      "  Read 1 with adapter:                  83,386 (99.9%)\n",
      "  Read 2 with adapter:                  83,273 (99.8%)\n",
      "Pairs written (passing filters):        83,445 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,218,421 bp\n",
      "  Read 1:    24,614,286 bp\n",
      "  Read 2:    24,604,135 bp\n",
      "Total written (filtered):     44,885,623 bp (91.2%)\n",
      "  Read 1:    22,446,224 bp\n",
      "  Read 2:    22,439,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 83386 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t662\t0.0\t2\t0 651 11\n",
      "26\t82078\t0.0\t2\t80204 1828 46\n",
      "27\t456\t0.0\t2\t0 444 12\n",
      "28\t153\t0.0\t2\t0 0 153\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 83273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t744\t0.0\t2\t0 723 21\n",
      "26\t82054\t0.0\t2\t80112 1824 118\n",
      "27\t420\t0.0\t2\t0 404 16\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1170_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1170_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1170_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1170_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1170_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1170_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1170_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1170_111_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        78,770 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 5.81 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,770\n",
      "  Read 1 with adapter:                  78,710 (99.9%)\n",
      "  Read 2 with adapter:                  78,614 (99.8%)\n",
      "Pairs written (passing filters):        78,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,465,902 bp\n",
      "  Read 1:    23,233,095 bp\n",
      "  Read 2:    23,232,807 bp\n",
      "Total written (filtered):     42,375,637 bp (91.2%)\n",
      "  Read 1:    21,186,571 bp\n",
      "  Read 2:    21,189,066 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 78710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t621\t0.0\t2\t0 606 15\n",
      "26\t77457\t0.0\t2\t75708 1717 32\n",
      "27\t431\t0.0\t2\t0 411 20\n",
      "28\t164\t0.0\t2\t0 0 164\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78614 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t630\t0.0\t2\t0 608 22\n",
      "26\t77569\t0.0\t2\t75545 1923 101\n",
      "27\t307\t0.0\t2\t0 296 11\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1171_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1171_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1171_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1171_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1171_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1171_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1171_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1171_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        98,184 reads  @     97.4 µs/read;   0.62 M reads/minute\n",
      "Finished in 9.56 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,184\n",
      "  Read 1 with adapter:                  97,961 (99.8%)\n",
      "  Read 2 with adapter:                  98,096 (99.9%)\n",
      "Pairs written (passing filters):        98,184 (100.0%)\n",
      "\n",
      "Total basepairs processed:    57,911,714 bp\n",
      "  Read 1:    28,948,099 bp\n",
      "  Read 2:    28,963,615 bp\n",
      "Total written (filtered):     52,814,907 bp (91.2%)\n",
      "  Read 1:    26,401,350 bp\n",
      "  Read 2:    26,413,557 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 97961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t978\t0.0\t2\t0 943 35\n",
      "26\t96176\t0.0\t2\t93287 2829 60\n",
      "27\t673\t0.0\t2\t0 647 26\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 98096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t907\t0.0\t2\t0 885 22\n",
      "26\t96622\t0.0\t2\t94003 2529 90\n",
      "27\t457\t0.0\t2\t0 445 12\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1172_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1172_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1172_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1172_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1172_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1172_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1172_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1172_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,846 reads  @     84.1 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.61 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,846\n",
      "  Read 1 with adapter:                  42,806 (99.9%)\n",
      "  Read 2 with adapter:                  42,499 (99.2%)\n",
      "Pairs written (passing filters):        42,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,271,879 bp\n",
      "  Read 1:    12,638,120 bp\n",
      "  Read 2:    12,633,759 bp\n",
      "Total written (filtered):     23,054,237 bp (91.2%)\n",
      "  Read 1:    11,525,194 bp\n",
      "  Read 2:    11,529,043 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t399\t0.0\t2\t0 386 13\n",
      "26\t42023\t0.0\t2\t40797 1196 30\n",
      "27\t243\t0.0\t2\t0 233 10\n",
      "28\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t448\t0.0\t2\t0 438 10\n",
      "26\t41828\t0.0\t2\t41011 799 18\n",
      "27\t188\t0.0\t2\t0 183 5\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1173_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1173_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1173_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1173_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1173_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1173_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1173_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1173_117_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,948 reads  @     82.3 µs/read;   0.73 M reads/minute\n",
      "Finished in 5.10 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,948\n",
      "  Read 1 with adapter:                  61,903 (99.9%)\n",
      "  Read 2 with adapter:                  61,837 (99.8%)\n",
      "Pairs written (passing filters):        61,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,541,275 bp\n",
      "  Read 1:    18,271,160 bp\n",
      "  Read 2:    18,270,115 bp\n",
      "Total written (filtered):     33,324,279 bp (91.2%)\n",
      "  Read 1:    16,661,732 bp\n",
      "  Read 2:    16,662,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t562\t0.0\t2\t0 540 22\n",
      "26\t60864\t0.0\t2\t59439 1395 30\n",
      "27\t338\t0.0\t2\t0 329 9\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61837 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t479\t0.0\t2\t0 464 15\n",
      "26\t61016\t0.0\t2\t59627 1331 58\n",
      "27\t315\t0.0\t2\t0 307 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1174_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1174_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1174_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1174_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1174_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1174_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1174_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1174_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,915 reads  @     85.7 µs/read;   0.70 M reads/minute\n",
      "Finished in 5.99 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,915\n",
      "  Read 1 with adapter:                  69,845 (99.9%)\n",
      "  Read 2 with adapter:                  69,842 (99.9%)\n",
      "Pairs written (passing filters):        69,915 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,242,321 bp\n",
      "  Read 1:    20,621,350 bp\n",
      "  Read 2:    20,620,971 bp\n",
      "Total written (filtered):     37,610,593 bp (91.2%)\n",
      "  Read 1:    18,805,361 bp\n",
      "  Read 2:    18,805,232 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t584\t0.0\t2\t0 568 16\n",
      "26\t68687\t0.0\t2\t67106 1555 26\n",
      "27\t429\t0.0\t2\t0 410 19\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69842 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t457\t0.0\t2\t0 445 12\n",
      "26\t69011\t0.0\t2\t67457 1504 50\n",
      "27\t340\t0.0\t2\t0 329 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1175_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1175_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1175_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1175_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1175_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1175_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1175_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1175_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:07        71,707 reads  @    111.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 8.00 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,707\n",
      "  Read 1 with adapter:                  71,603 (99.9%)\n",
      "  Read 2 with adapter:                  71,129 (99.2%)\n",
      "Pairs written (passing filters):        71,696 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,216,595 bp\n",
      "  Read 1:    21,147,874 bp\n",
      "  Read 2:    21,068,721 bp\n",
      "Total written (filtered):     38,502,844 bp (91.2%)\n",
      "  Read 1:    19,283,363 bp\n",
      "  Read 2:    19,219,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 71603 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t624\t0.0\t2\t0 602 22\n",
      "26\t70491\t0.0\t2\t68895 1574 22\n",
      "27\t362\t0.0\t2\t0 353 9\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 71129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t497\t0.0\t2\t0 475 22\n",
      "26\t70240\t0.0\t2\t67784 2237 219\n",
      "27\t313\t0.0\t2\t0 289 24\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1176_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1176_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1176_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1176_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1176_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1176_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1176_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1176_123_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        63,557 reads  @     99.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 6.34 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,557\n",
      "  Read 1 with adapter:                  63,491 (99.9%)\n",
      "  Read 2 with adapter:                  57,904 (91.1%)\n",
      "Pairs written (passing filters):        63,555 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,508,397 bp\n",
      "  Read 1:    18,745,670 bp\n",
      "  Read 2:    17,762,727 bp\n",
      "Total written (filtered):     33,352,065 bp (91.4%)\n",
      "  Read 1:    17,094,553 bp\n",
      "  Read 2:    16,257,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t609\t0.0\t2\t0 595 14\n",
      "26\t62442\t0.0\t2\t60963 1458 21\n",
      "27\t358\t0.0\t2\t0 347 11\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57904 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t580\t0.0\t2\t0 558 22\n",
      "26\t56992\t0.0\t2\t55177 1670 145\n",
      "27\t293\t0.0\t2\t0 277 16\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1177_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1177_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1177_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1177_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1177_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1177_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1177_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1177_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:07        79,490 reads  @     89.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 7.07 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,490\n",
      "  Read 1 with adapter:                  79,404 (99.9%)\n",
      "  Read 2 with adapter:                  78,870 (99.2%)\n",
      "Pairs written (passing filters):        79,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,830,342 bp\n",
      "  Read 1:    23,443,961 bp\n",
      "  Read 2:    23,386,381 bp\n",
      "Total written (filtered):     42,715,539 bp (91.2%)\n",
      "  Read 1:    21,379,539 bp\n",
      "  Read 2:    21,336,000 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t674\t0.0\t2\t0 659 15\n",
      "26\t78158\t0.0\t2\t76432 1705 21\n",
      "27\t408\t0.0\t2\t0 402 6\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78870 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t704\t0.0\t2\t0 674 30\n",
      "26\t77692\t0.0\t2\t75601 1968 123\n",
      "27\t339\t0.0\t2\t0 325 14\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1178_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1178_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1178_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1178_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1178_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1178_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1178_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1178_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        73,066 reads  @     87.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 6.41 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,066\n",
      "  Read 1 with adapter:                  72,931 (99.8%)\n",
      "  Read 2 with adapter:                  72,892 (99.8%)\n",
      "Pairs written (passing filters):        73,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,091,447 bp\n",
      "  Read 1:    21,543,676 bp\n",
      "  Read 2:    21,547,771 bp\n",
      "Total written (filtered):     39,300,386 bp (91.2%)\n",
      "  Read 1:    19,647,620 bp\n",
      "  Read 2:    19,652,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72931 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t702\t0.0\t2\t0 690 12\n",
      "26\t71703\t0.0\t2\t69893 1788 22\n",
      "27\t412\t0.0\t2\t0 401 11\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72892 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t606\t0.0\t2\t0 590 16\n",
      "26\t71866\t0.0\t2\t70012 1768 86\n",
      "27\t325\t0.0\t2\t0 314 11\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1179_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1179_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1179_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1179_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1179_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1179_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1179_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1179_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        67,502 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.90 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,502\n",
      "  Read 1 with adapter:                  67,376 (99.8%)\n",
      "  Read 2 with adapter:                  67,424 (99.9%)\n",
      "Pairs written (passing filters):        67,502 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,807,307 bp\n",
      "  Read 1:    19,894,874 bp\n",
      "  Read 2:    19,912,433 bp\n",
      "Total written (filtered):     36,302,802 bp (91.2%)\n",
      "  Read 1:    18,143,239 bp\n",
      "  Read 2:    18,159,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t653\t0.0\t2\t0 637 16\n",
      "26\t66169\t0.0\t2\t64412 1737 20\n",
      "27\t444\t0.0\t2\t0 434 10\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t569\t0.0\t2\t0 548 21\n",
      "26\t66424\t0.0\t2\t64781 1573 70\n",
      "27\t343\t0.0\t2\t0 333 10\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1180_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1180_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1180_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1180_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1180_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1180_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1180_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1180_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         6,792 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 0.52 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,792\n",
      "  Read 1 with adapter:                   6,776 (99.8%)\n",
      "  Read 2 with adapter:                   6,735 (99.2%)\n",
      "Pairs written (passing filters):         6,792 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,006,320 bp\n",
      "  Read 1:     2,003,389 bp\n",
      "  Read 2:     2,002,931 bp\n",
      "Total written (filtered):      3,655,020 bp (91.2%)\n",
      "  Read 1:     1,827,150 bp\n",
      "  Read 2:     1,827,870 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 6776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t68\t0.0\t2\t0 65 3\n",
      "26\t6621\t0.0\t2\t6356 256 9\n",
      "27\t39\t0.0\t2\t0 37 2\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 6735 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t101\t0.0\t2\t0 96 5\n",
      "26\t6572\t0.0\t2\t6377 190 5\n",
      "27\t56\t0.0\t2\t0 54 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1196_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1196_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1196_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1196_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1196_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1196_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1196_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1196_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        39,662 reads  @    100.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.00 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,662\n",
      "  Read 1 with adapter:                  33,490 (84.4%)\n",
      "  Read 2 with adapter:                  33,480 (84.4%)\n",
      "Pairs written (passing filters):        39,662 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,457,739 bp\n",
      "  Read 1:    11,696,179 bp\n",
      "  Read 2:    10,761,560 bp\n",
      "Total written (filtered):     20,716,818 bp (92.2%)\n",
      "  Read 1:    10,825,577 bp\n",
      "  Read 2:     9,891,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t374\t0.0\t2\t0 364 10\n",
      "26\t32871\t0.0\t2\t32061 787 23\n",
      "27\t182\t0.0\t2\t0 171 11\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t358\t0.0\t2\t0 341 17\n",
      "26\t32905\t0.0\t2\t32042 825 38\n",
      "27\t169\t0.0\t2\t0 166 3\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1197_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1197_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1197_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1197_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1197_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1197_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1197_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1197_135_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,600 reads  @    102.6 µs/read;   0.58 M reads/minute\n",
      "Finished in 3.35 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,600\n",
      "  Read 1 with adapter:                  31,767 (97.4%)\n",
      "  Read 2 with adapter:                  31,725 (97.3%)\n",
      "Pairs written (passing filters):        32,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,011,559 bp\n",
      "  Read 1:     9,615,754 bp\n",
      "  Read 2:     9,395,805 bp\n",
      "Total written (filtered):     17,360,127 bp (91.3%)\n",
      "  Read 1:     8,789,063 bp\n",
      "  Read 2:     8,571,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t357\t0.0\t2\t0 347 10\n",
      "26\t31168\t0.0\t2\t30436 715 17\n",
      "27\t179\t0.0\t2\t0 176 3\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t238\t0.0\t2\t0 228 10\n",
      "26\t31342\t0.0\t2\t30459 830 53\n",
      "27\t121\t0.0\t2\t0 114 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1198_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1198_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1198_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1198_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1198_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1198_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1198_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1198_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,278 reads  @    109.5 µs/read;   0.55 M reads/minute\n",
      "Finished in 3.54 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,278\n",
      "  Read 1 with adapter:                  31,731 (98.3%)\n",
      "  Read 2 with adapter:                  31,710 (98.2%)\n",
      "Pairs written (passing filters):        32,278 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,948,434 bp\n",
      "  Read 1:     9,516,996 bp\n",
      "  Read 2:     9,431,438 bp\n",
      "Total written (filtered):     17,299,214 bp (91.3%)\n",
      "  Read 1:     8,692,228 bp\n",
      "  Read 2:     8,606,986 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31731 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t392\t0.0\t2\t0 378 14\n",
      "26\t31169\t0.0\t2\t30402 757 10\n",
      "27\t130\t0.0\t2\t0 121 9\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t226\t0.0\t2\t0 219 7\n",
      "26\t31259\t0.0\t2\t30292 904 63\n",
      "27\t200\t0.0\t2\t0 189 11\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1199_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1199_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1199_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1199_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1199_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1199_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1199_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1199_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           292 reads  @    200.6 µs/read;   0.30 M reads/minute\n",
      "Finished in 0.06 s (209 us/read; 0.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                292\n",
      "  Read 1 with adapter:                       1 (0.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):           290 (99.3%)\n",
      "\n",
      "Total basepairs processed:       104,431 bp\n",
      "  Read 1:        86,140 bp\n",
      "  Read 2:        18,291 bp\n",
      "Total written (filtered):        103,815 bp (99.4%)\n",
      "  Read 1:        85,524 bp\n",
      "  Read 2:        18,291 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1200_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1200_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1200_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1200_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1200_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1200_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1200_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1200_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        37,569 reads  @    108.8 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.09 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,569\n",
      "  Read 1 with adapter:                  37,534 (99.9%)\n",
      "  Read 2 with adapter:                  37,510 (99.8%)\n",
      "Pairs written (passing filters):        37,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,160,001 bp\n",
      "  Read 1:    11,081,880 bp\n",
      "  Read 2:    11,078,121 bp\n",
      "Total written (filtered):     20,209,010 bp (91.2%)\n",
      "  Read 1:    10,105,989 bp\n",
      "  Read 2:    10,103,021 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t534\t0.0\t2\t0 517 17\n",
      "26\t36700\t0.0\t2\t35812 869 19\n",
      "27\t232\t0.0\t2\t0 226 6\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t401\t0.0\t2\t0 395 6\n",
      "26\t36839\t0.0\t2\t35938 864 37\n",
      "27\t231\t0.0\t2\t0 224 7\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1201_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1201_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1201_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1201_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1201_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1201_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1201_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1201_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,856 reads  @    108.5 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.76 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,856\n",
      "  Read 1 with adapter:                  43,808 (99.9%)\n",
      "  Read 2 with adapter:                  43,772 (99.8%)\n",
      "Pairs written (passing filters):        43,856 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,865,297 bp\n",
      "  Read 1:    12,935,994 bp\n",
      "  Read 2:    12,929,303 bp\n",
      "Total written (filtered):     23,588,751 bp (91.2%)\n",
      "  Read 1:    11,797,378 bp\n",
      "  Read 2:    11,791,373 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t596\t0.0\t2\t0 586 10\n",
      "26\t42926\t0.0\t2\t41878 1027 21\n",
      "27\t204\t0.0\t2\t0 203 1\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t321\t0.0\t2\t0 312 9\n",
      "26\t43222\t0.0\t2\t42106 1077 39\n",
      "27\t203\t0.0\t2\t0 199 4\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1202_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1202_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1202_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1202_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1202_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1202_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1202_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1202_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             4 reads  @   8555.2 µs/read;   0.01 M reads/minute\n",
      "Finished in 0.04 s (9183 us/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  4\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             4 (100.0%)\n",
      "\n",
      "Total basepairs processed:           997 bp\n",
      "  Read 1:           582 bp\n",
      "  Read 2:           415 bp\n",
      "Total written (filtered):            997 bp (100.0%)\n",
      "  Read 1:           582 bp\n",
      "  Read 2:           415 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1203_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1203_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1203_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1203_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1203_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1203_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1203_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1203_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,675 reads  @    111.8 µs/read;   0.54 M reads/minute\n",
      "Finished in 5.44 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,675\n",
      "  Read 1 with adapter:                  48,638 (99.9%)\n",
      "  Read 2 with adapter:                  48,583 (99.8%)\n",
      "Pairs written (passing filters):        48,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,716,028 bp\n",
      "  Read 1:    14,358,801 bp\n",
      "  Read 2:    14,357,227 bp\n",
      "Total written (filtered):     26,188,704 bp (91.2%)\n",
      "  Read 1:    13,094,521 bp\n",
      "  Read 2:    13,094,183 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48638 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t694\t0.0\t2\t0 669 25\n",
      "26\t47540\t0.0\t2\t46317 1203 20\n",
      "27\t290\t0.0\t2\t0 280 10\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48583 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t397\t0.0\t2\t0 385 12\n",
      "26\t47861\t0.0\t2\t46479 1317 65\n",
      "27\t279\t0.0\t2\t0 267 12\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1204_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1204_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1204_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1204_149_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1204_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1204_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1204_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1204_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        25,338 reads  @    103.4 µs/read;   0.58 M reads/minute\n",
      "Finished in 2.62 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,338\n",
      "  Read 1 with adapter:                  25,279 (99.8%)\n",
      "  Read 2 with adapter:                  25,300 (99.9%)\n",
      "Pairs written (passing filters):        25,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,939,670 bp\n",
      "  Read 1:     7,467,992 bp\n",
      "  Read 2:     7,471,678 bp\n",
      "Total written (filtered):     13,624,875 bp (91.2%)\n",
      "  Read 1:     6,810,854 bp\n",
      "  Read 2:     6,814,021 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 25279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t254\t0.0\t2\t0 244 10\n",
      "26\t24883\t0.0\t2\t24231 646 6\n",
      "27\t110\t0.0\t2\t0 105 5\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t268\t0.0\t2\t0 261 7\n",
      "26\t24875\t0.0\t2\t24217 626 32\n",
      "27\t129\t0.0\t2\t0 121 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1205_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1205_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1205_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1205_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1205_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1205_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1205_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1205_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,320 reads  @    105.9 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.48 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,320\n",
      "  Read 1 with adapter:                  42,206 (99.7%)\n",
      "  Read 2 with adapter:                  42,239 (99.8%)\n",
      "Pairs written (passing filters):        42,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,962,320 bp\n",
      "  Read 1:    12,481,297 bp\n",
      "  Read 2:    12,481,023 bp\n",
      "Total written (filtered):     22,767,031 bp (91.2%)\n",
      "  Read 1:    11,384,111 bp\n",
      "  Read 2:    11,382,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t486\t0.0\t2\t0 464 22\n",
      "26\t41404\t0.0\t2\t40089 1298 17\n",
      "27\t220\t0.0\t2\t0 218 2\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t351\t0.0\t2\t0 338 13\n",
      "26\t41632\t0.0\t2\t40251 1322 59\n",
      "27\t200\t0.0\t2\t0 189 11\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1206_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1206_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1206_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1206_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1206_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1206_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1206_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1206_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,555 reads  @    109.6 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.45 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,555\n",
      "  Read 1 with adapter:                  40,504 (99.9%)\n",
      "  Read 2 with adapter:                  40,488 (99.8%)\n",
      "Pairs written (passing filters):        40,555 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,921,154 bp\n",
      "  Read 1:    11,960,291 bp\n",
      "  Read 2:    11,960,863 bp\n",
      "Total written (filtered):     21,815,621 bp (91.2%)\n",
      "  Read 1:    10,907,375 bp\n",
      "  Read 2:    10,908,246 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t435\t0.0\t2\t0 411 24\n",
      "26\t39812\t0.0\t2\t38847 947 18\n",
      "27\t187\t0.0\t2\t0 179 8\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40488 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t299\t0.0\t2\t0 289 10\n",
      "26\t39946\t0.0\t2\t38864 1036 46\n",
      "27\t210\t0.0\t2\t0 203 7\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1207_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1207_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1207_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1207_155_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1207_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1207_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1207_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1207_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        31,919 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 3.18 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,919\n",
      "  Read 1 with adapter:                  31,823 (99.7%)\n",
      "  Read 2 with adapter:                  31,876 (99.9%)\n",
      "Pairs written (passing filters):        31,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,822,811 bp\n",
      "  Read 1:     9,406,948 bp\n",
      "  Read 2:     9,415,863 bp\n",
      "Total written (filtered):     17,166,816 bp (91.2%)\n",
      "  Read 1:     8,579,684 bp\n",
      "  Read 2:     8,587,132 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t321\t0.0\t2\t0 309 12\n",
      "26\t31315\t0.0\t2\t30580 725 10\n",
      "27\t127\t0.0\t2\t0 123 4\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t233\t0.0\t2\t0 223 10\n",
      "26\t31451\t0.0\t2\t30735 695 21\n",
      "27\t152\t0.0\t2\t0 146 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1208_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1208_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1208_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1208_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1208_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1208_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1208_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1208_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        21,678 reads  @    100.9 µs/read;   0.59 M reads/minute\n",
      "Finished in 2.19 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,678\n",
      "  Read 1 with adapter:                  21,651 (99.9%)\n",
      "  Read 2 with adapter:                  21,626 (99.8%)\n",
      "Pairs written (passing filters):        21,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,783,689 bp\n",
      "  Read 1:     6,393,625 bp\n",
      "  Read 2:     6,390,064 bp\n",
      "Total written (filtered):     11,658,548 bp (91.2%)\n",
      "  Read 1:     5,830,627 bp\n",
      "  Read 2:     5,827,921 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t330\t0.0\t2\t0 321 9\n",
      "26\t21162\t0.0\t2\t20670 484 8\n",
      "27\t117\t0.0\t2\t0 115 2\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t257\t0.0\t2\t0 248 9\n",
      "26\t21225\t0.0\t2\t20636 563 26\n",
      "27\t128\t0.0\t2\t0 126 2\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1209_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1209_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1209_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1209_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1209_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1209_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1209_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1209_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        42,017 reads  @    103.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.34 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,017\n",
      "  Read 1 with adapter:                  41,982 (99.9%)\n",
      "  Read 2 with adapter:                  41,928 (99.8%)\n",
      "Pairs written (passing filters):        42,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,784,731 bp\n",
      "  Read 1:    12,393,212 bp\n",
      "  Read 2:    12,391,519 bp\n",
      "Total written (filtered):     22,603,401 bp (91.2%)\n",
      "  Read 1:    11,301,904 bp\n",
      "  Read 2:    11,301,497 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t498\t0.0\t2\t0 485 13\n",
      "26\t41181\t0.0\t2\t40229 939 13\n",
      "27\t228\t0.0\t2\t0 222 6\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t327\t0.0\t2\t0 317 10\n",
      "26\t41345\t0.0\t2\t40274 1019 52\n",
      "27\t223\t0.0\t2\t0 210 13\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1210_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1210_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1210_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1210_161_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1210_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1210_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1210_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1210_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        34,308 reads  @    100.9 µs/read;   0.59 M reads/minute\n",
      "Finished in 3.46 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,308\n",
      "  Read 1 with adapter:                  34,282 (99.9%)\n",
      "  Read 2 with adapter:                  34,244 (99.8%)\n",
      "Pairs written (passing filters):        34,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,238,674 bp\n",
      "  Read 1:    10,119,655 bp\n",
      "  Read 2:    10,119,019 bp\n",
      "Total written (filtered):     18,457,152 bp (91.2%)\n",
      "  Read 1:     9,228,465 bp\n",
      "  Read 2:     9,228,687 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34282 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t395\t0.0\t2\t0 381 14\n",
      "26\t33648\t0.0\t2\t32863 771 14\n",
      "27\t161\t0.0\t2\t0 156 5\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34244 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t223\t0.0\t2\t0 218 5\n",
      "26\t33793\t0.0\t2\t32876 873 44\n",
      "27\t193\t0.0\t2\t0 187 6\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1211_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1211_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1211_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1211_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1211_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1211_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1211_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1211_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,099 reads  @     89.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.84 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,099\n",
      "  Read 1 with adapter:                  43,071 (99.9%)\n",
      "  Read 2 with adapter:                  43,039 (99.9%)\n",
      "Pairs written (passing filters):        43,099 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,427,012 bp\n",
      "  Read 1:    12,714,054 bp\n",
      "  Read 2:    12,712,958 bp\n",
      "Total written (filtered):     23,188,486 bp (91.2%)\n",
      "  Read 1:    11,594,414 bp\n",
      "  Read 2:    11,594,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43071 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t537\t0.0\t2\t0 520 17\n",
      "26\t42212\t0.0\t2\t41203 992 17\n",
      "27\t213\t0.0\t2\t0 204 9\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t342\t0.0\t2\t0 332 10\n",
      "26\t42413\t0.0\t2\t41284 1073 56\n",
      "27\t246\t0.0\t2\t0 233 13\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1212_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1212_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1212_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1212_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1212_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1212_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1212_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1212_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        28,040 reads  @    104.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 2.93 s (104 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,040\n",
      "  Read 1 with adapter:                  28,009 (99.9%)\n",
      "  Read 2 with adapter:                  27,971 (99.8%)\n",
      "Pairs written (passing filters):        28,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,536,369 bp\n",
      "  Read 1:     8,270,249 bp\n",
      "  Read 2:     8,266,120 bp\n",
      "Total written (filtered):     15,081,202 bp (91.2%)\n",
      "  Read 1:     7,542,189 bp\n",
      "  Read 2:     7,539,013 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28009 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t350\t0.0\t2\t0 346 4\n",
      "26\t27462\t0.0\t2\t26846 607 9\n",
      "27\t146\t0.0\t2\t0 142 4\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27971 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t273\t0.0\t2\t0 263 10\n",
      "26\t27533\t0.0\t2\t26812 681 40\n",
      "27\t148\t0.0\t2\t0 141 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1213_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1213_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1213_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1213_167_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1213_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1213_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1213_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1213_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        31,288 reads  @    103.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 3.25 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,288\n",
      "  Read 1 with adapter:                  31,270 (99.9%)\n",
      "  Read 2 with adapter:                  31,198 (99.7%)\n",
      "Pairs written (passing filters):        31,288 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,454,252 bp\n",
      "  Read 1:     9,229,068 bp\n",
      "  Read 2:     9,225,184 bp\n",
      "Total written (filtered):     16,830,311 bp (91.2%)\n",
      "  Read 1:     8,416,223 bp\n",
      "  Read 2:     8,414,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31270 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t393\t0.0\t2\t0 389 4\n",
      "26\t30651\t0.0\t2\t29974 668 9\n",
      "27\t154\t0.0\t2\t0 151 3\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t218\t0.0\t2\t0 211 7\n",
      "26\t30802\t0.0\t2\t29852 879 71\n",
      "27\t154\t0.0\t2\t0 145 9\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1214_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1214_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1214_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1214_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1214_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1214_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1214_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1214_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,141 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.22 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,141\n",
      "  Read 1 with adapter:                  48,079 (99.9%)\n",
      "  Read 2 with adapter:                  48,055 (99.8%)\n",
      "Pairs written (passing filters):        48,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,396,617 bp\n",
      "  Read 1:    14,197,174 bp\n",
      "  Read 2:    14,199,443 bp\n",
      "Total written (filtered):     25,897,349 bp (91.2%)\n",
      "  Read 1:    12,947,298 bp\n",
      "  Read 2:    12,950,051 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t595\t0.0\t2\t0 585 10\n",
      "26\t47048\t0.0\t2\t45963 1068 17\n",
      "27\t307\t0.0\t2\t0 299 8\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t347\t0.0\t2\t0 336 11\n",
      "26\t47375\t0.0\t2\t45938 1348 89\n",
      "27\t289\t0.0\t2\t0 275 14\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1215_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1215_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1215_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1215_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1215_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1215_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1215_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1215_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,542 reads  @     86.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.67 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,542\n",
      "  Read 1 with adapter:                  42,434 (99.7%)\n",
      "  Read 2 with adapter:                  42,397 (99.7%)\n",
      "Pairs written (passing filters):        42,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,065,812 bp\n",
      "  Read 1:    12,537,109 bp\n",
      "  Read 2:    12,528,703 bp\n",
      "Total written (filtered):     22,860,467 bp (91.2%)\n",
      "  Read 1:    11,433,992 bp\n",
      "  Read 2:    11,426,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t557\t0.0\t2\t0 544 13\n",
      "26\t41497\t0.0\t2\t40554 929 14\n",
      "27\t270\t0.0\t2\t0 265 5\n",
      "28\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t306\t0.0\t2\t0 297 9\n",
      "26\t41849\t0.0\t2\t40765 1025 59\n",
      "27\t204\t0.0\t2\t0 198 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1216_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1216_173_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1216_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1216_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1216_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1216_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        40,684 reads  @     98.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.02 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,684\n",
      "  Read 1 with adapter:                  40,367 (99.2%)\n",
      "  Read 2 with adapter:                  40,297 (99.0%)\n",
      "Pairs written (passing filters):        40,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,945,318 bp\n",
      "  Read 1:    11,984,257 bp\n",
      "  Read 2:    11,961,061 bp\n",
      "Total written (filtered):     21,848,442 bp (91.2%)\n",
      "  Read 1:    10,934,901 bp\n",
      "  Read 2:    10,913,541 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40367 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t450\t0.0\t2\t0 433 17\n",
      "26\t39643\t0.0\t2\t38750 877 16\n",
      "27\t212\t0.0\t2\t0 208 4\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40297 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t418\t0.0\t2\t0 408 10\n",
      "26\t39618\t0.0\t2\t38417 1122 79\n",
      "27\t226\t0.0\t2\t0 212 14\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1217_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1217_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1217_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1217_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1217_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1217_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,638 reads  @     80.2 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.66 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,638\n",
      "  Read 1 with adapter:                  45,607 (99.9%)\n",
      "  Read 2 with adapter:                  45,532 (99.8%)\n",
      "Pairs written (passing filters):        45,638 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,921,598 bp\n",
      "  Read 1:    13,462,100 bp\n",
      "  Read 2:    13,459,498 bp\n",
      "Total written (filtered):     24,552,190 bp (91.2%)\n",
      "  Read 1:    12,276,428 bp\n",
      "  Read 2:    12,275,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t464\t0.0\t2\t0 447 17\n",
      "26\t44792\t0.0\t2\t43781 993 18\n",
      "27\t260\t0.0\t2\t0 255 5\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t353\t0.0\t2\t0 343 10\n",
      "26\t44883\t0.0\t2\t43606 1199 78\n",
      "27\t275\t0.0\t2\t0 261 14\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1218_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1218_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1218_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1218_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1218_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1218_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,117 reads  @     74.3 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.88 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,117\n",
      "  Read 1 with adapter:                  52,031 (99.8%)\n",
      "  Read 2 with adapter:                  52,023 (99.8%)\n",
      "Pairs written (passing filters):        52,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,746,980 bp\n",
      "  Read 1:    15,374,507 bp\n",
      "  Read 2:    15,372,473 bp\n",
      "Total written (filtered):     28,041,648 bp (91.2%)\n",
      "  Read 1:    14,021,719 bp\n",
      "  Read 2:    14,019,929 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t475\t0.0\t2\t0 464 11\n",
      "26\t51139\t0.0\t2\t49874 1246 19\n",
      "27\t281\t0.0\t2\t0 273 8\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t454\t0.0\t2\t0 429 25\n",
      "26\t51183\t0.0\t2\t49705 1371 107\n",
      "27\t280\t0.0\t2\t0 268 12\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1219_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1219_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1219_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1219_179_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1219_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1219_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1219_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1219_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        27,811 reads  @    104.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 2.90 s (104 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,811\n",
      "  Read 1 with adapter:                  27,709 (99.6%)\n",
      "  Read 2 with adapter:                  27,683 (99.5%)\n",
      "Pairs written (passing filters):        27,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,373,804 bp\n",
      "  Read 1:     8,192,288 bp\n",
      "  Read 2:     8,181,516 bp\n",
      "Total written (filtered):     14,933,860 bp (91.2%)\n",
      "  Read 1:     7,471,966 bp\n",
      "  Read 2:     7,461,894 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t351\t0.0\t2\t0 341 10\n",
      "26\t27113\t0.0\t2\t26352 748 13\n",
      "27\t203\t0.0\t2\t0 196 7\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t264\t0.0\t2\t0 259 5\n",
      "26\t27254\t0.0\t2\t26514 707 33\n",
      "27\t142\t0.0\t2\t0 137 5\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1220_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1220_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1220_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1220_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1220_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1220_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1220_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1220_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,109 reads  @     76.4 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.38 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,109\n",
      "  Read 1 with adapter:                  43,766 (99.2%)\n",
      "  Read 2 with adapter:                  44,052 (99.9%)\n",
      "Pairs written (passing filters):        44,109 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,936,723 bp\n",
      "  Read 1:    12,926,989 bp\n",
      "  Read 2:    13,009,734 bp\n",
      "Total written (filtered):     23,653,798 bp (91.2%)\n",
      "  Read 1:    11,789,160 bp\n",
      "  Read 2:    11,864,638 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t482\t0.0\t2\t0 469 13\n",
      "26\t42919\t0.0\t2\t41880 1026 13\n",
      "27\t259\t0.0\t2\t0 254 5\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44052 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t476\t0.0\t2\t0 464 12\n",
      "26\t43339\t0.0\t2\t42313 989 37\n",
      "27\t182\t0.0\t2\t0 174 8\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1221_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1221_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1221_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1221_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1221_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1221_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1221_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1221_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,211 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.38 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,211\n",
      "  Read 1 with adapter:                  40,947 (99.4%)\n",
      "  Read 2 with adapter:                  41,162 (99.9%)\n",
      "Pairs written (passing filters):        41,211 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,254,432 bp\n",
      "  Read 1:    12,099,566 bp\n",
      "  Read 2:    12,154,866 bp\n",
      "Total written (filtered):     22,119,742 bp (91.2%)\n",
      "  Read 1:    11,035,002 bp\n",
      "  Read 2:    11,084,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40947 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t423\t0.0\t2\t0 410 13\n",
      "26\t40180\t0.0\t2\t39170 995 15\n",
      "27\t239\t0.0\t2\t0 232 7\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41162 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t303\t0.0\t2\t0 292 11\n",
      "26\t40604\t0.0\t2\t39623 945 36\n",
      "27\t233\t0.0\t2\t0 224 9\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1222_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1222_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1222_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1222_185_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1222_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1222_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1222_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1222_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,669 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.81 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,669\n",
      "  Read 1 with adapter:                  32,440 (99.3%)\n",
      "  Read 2 with adapter:                  32,603 (99.8%)\n",
      "Pairs written (passing filters):        32,669 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,225,093 bp\n",
      "  Read 1:     9,590,458 bp\n",
      "  Read 2:     9,634,635 bp\n",
      "Total written (filtered):     17,534,135 bp (91.2%)\n",
      "  Read 1:     8,747,110 bp\n",
      "  Read 2:     8,787,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t368\t0.0\t2\t0 358 10\n",
      "26\t31804\t0.0\t2\t31086 706 12\n",
      "27\t192\t0.0\t2\t0 186 6\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32603 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t244\t0.0\t2\t0 232 12\n",
      "26\t32142\t0.0\t2\t31370 745 27\n",
      "27\t202\t0.0\t2\t0 189 13\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1223_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1223_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1223_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1223_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1223_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1223_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1223_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1223_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        65,716 reads  @    107.3 µs/read;   0.56 M reads/minute\n",
      "Finished in 7.06 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,716\n",
      "  Read 1 with adapter:                  65,439 (99.6%)\n",
      "  Read 2 with adapter:                  65,625 (99.9%)\n",
      "Pairs written (passing filters):        65,716 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,712,399 bp\n",
      "  Read 1:    19,328,776 bp\n",
      "  Read 2:    19,383,623 bp\n",
      "Total written (filtered):     35,305,139 bp (91.2%)\n",
      "  Read 1:    17,627,622 bp\n",
      "  Read 2:    17,677,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65439 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t717\t0.0\t2\t0 696 21\n",
      "26\t64241\t0.0\t2\t62704 1517 20\n",
      "27\t337\t0.0\t2\t0 327 10\n",
      "28\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65625 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t507\t0.0\t2\t0 492 15\n",
      "26\t64728\t0.0\t2\t63146 1527 55\n",
      "27\t313\t0.0\t2\t0 301 12\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1224_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1224_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1224_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1224_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1224_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1224_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1224_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1224_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,941 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.44 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,941\n",
      "  Read 1 with adapter:                  42,881 (99.9%)\n",
      "  Read 2 with adapter:                  42,858 (99.8%)\n",
      "Pairs written (passing filters):        42,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,322,269 bp\n",
      "  Read 1:    12,662,810 bp\n",
      "  Read 2:    12,659,459 bp\n",
      "Total written (filtered):     23,093,442 bp (91.2%)\n",
      "  Read 1:    11,548,044 bp\n",
      "  Read 2:    11,545,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t511\t0.0\t2\t0 498 13\n",
      "26\t41987\t0.0\t2\t40833 1140 14\n",
      "27\t323\t0.0\t2\t0 314 9\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t433\t0.0\t2\t0 424 9\n",
      "26\t42185\t0.0\t2\t41232 929 24\n",
      "27\t198\t0.0\t2\t0 194 4\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1225_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1225_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1225_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1225_191_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1225_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1225_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1225_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1225_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,967 reads  @     88.1 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.88 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,967\n",
      "  Read 1 with adapter:                  43,914 (99.9%)\n",
      "  Read 2 with adapter:                  43,872 (99.8%)\n",
      "Pairs written (passing filters):        43,967 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,933,871 bp\n",
      "  Read 1:    12,966,014 bp\n",
      "  Read 2:    12,967,857 bp\n",
      "Total written (filtered):     23,651,631 bp (91.2%)\n",
      "  Read 1:    11,824,376 bp\n",
      "  Read 2:    11,827,255 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t479\t0.0\t2\t0 471 8\n",
      "26\t43087\t0.0\t2\t42132 943 12\n",
      "27\t267\t0.0\t2\t0 262 5\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43872 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t332\t0.0\t2\t0 320 12\n",
      "26\t43289\t0.0\t2\t42198 1036 55\n",
      "27\t192\t0.0\t2\t0 188 4\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1226_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1226_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1226_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1226_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1226_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1226_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1226_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1226_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,448 reads  @     90.6 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.48 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,448\n",
      "  Read 1 with adapter:                  38,390 (99.8%)\n",
      "  Read 2 with adapter:                  38,379 (99.8%)\n",
      "Pairs written (passing filters):        38,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,680,800 bp\n",
      "  Read 1:    11,339,694 bp\n",
      "  Read 2:    11,341,106 bp\n",
      "Total written (filtered):     20,684,855 bp (91.2%)\n",
      "  Read 1:    10,341,544 bp\n",
      "  Read 2:    10,343,311 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38390 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t462\t0.0\t2\t0 447 15\n",
      "26\t37502\t0.0\t2\t36513 974 15\n",
      "27\t308\t0.0\t2\t0 294 14\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38379 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t326\t0.0\t2\t0 308 18\n",
      "26\t37787\t0.0\t2\t36844 914 29\n",
      "27\t213\t0.0\t2\t0 209 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1227_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1227_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1227_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1227_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1227_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1227_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1227_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1227_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        25,467 reads  @     88.6 µs/read;   0.68 M reads/minute\n",
      "Finished in 2.26 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,467\n",
      "  Read 1 with adapter:                  25,441 (99.9%)\n",
      "  Read 2 with adapter:                  25,440 (99.9%)\n",
      "Pairs written (passing filters):        25,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,023,721 bp\n",
      "  Read 1:     7,510,966 bp\n",
      "  Read 2:     7,512,755 bp\n",
      "Total written (filtered):     13,700,978 bp (91.2%)\n",
      "  Read 1:     6,849,609 bp\n",
      "  Read 2:     6,851,369 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 25441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t323\t0.0\t2\t0 316 7\n",
      "26\t24904\t0.0\t2\t24237 655 12\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t219\t0.0\t2\t0 207 12\n",
      "26\t25045\t0.0\t2\t24443 587 15\n",
      "27\t147\t0.0\t2\t0 144 3\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1228_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1228_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1228_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1228_197_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1228_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-6hy6rv06/S1228_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1228_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hxyij69f/ec26dfa1-41be-4961-a5e8-dccdb4e93eb3/data/S1228_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,724 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 0.30 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,724\n",
      "  Read 1 with adapter:                   3,694 (99.2%)\n",
      "  Read 2 with adapter:                   3,696 (99.2%)\n",
      "Pairs written (passing filters):         3,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,192,915 bp\n",
      "  Read 1:     1,097,853 bp\n",
      "  Read 2:     1,095,062 bp\n",
      "Total written (filtered):      2,000,763 bp (91.2%)\n",
      "  Read 1:     1,001,786 bp\n",
      "  Read 2:       998,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 3694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t31\t0.0\t2\t0 31\n",
      "26\t3618\t0.0\t2\t3502 112 4\n",
      "27\t32\t0.0\t2\t0 29 3\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 3696 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t38\t0.0\t2\t0 37 1\n",
      "26\t3632\t0.0\t2\t3509 118 5\n",
      "27\t25\t0.0\t2\t0 24 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: e-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: e-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-ekbwvf1i')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b642d41b-fa9e-4796-9633-7a63f755fb88>"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.e import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/Hirai-Lib1-manifest_COI.csv \\\n",
    "  --output-path e-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.e trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences e-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences e-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.e summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data e-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization e-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('e-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: e-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: e-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: e-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.e Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs e-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 265 \\\n",
    "  --p-trunc-len-r 222 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table e-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats e-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: e-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0nlf7wj2')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c129c3b0-abb0-4892-85f7-8105f4ec474d>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.e view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table e-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization e-table.qzv \n",
    "q2.Visualization.load('e-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: e-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.e view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization e-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/Hirai-Lib2-manifest_COI.csv as PairedEndFastqManifestPhred33 to f-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1501_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1501_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1501_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1501_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1501_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1501_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        30,777 reads  @    122.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 3.77 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,777\n",
      "  Read 1 with adapter:                  22,211 (72.2%)\n",
      "  Read 2 with adapter:                  22,083 (71.8%)\n",
      "Pairs written (passing filters):        30,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,152,039 bp\n",
      "  Read 1:     9,075,379 bp\n",
      "  Read 2:     9,076,660 bp\n",
      "Total written (filtered):     17,000,485 bp (93.7%)\n",
      "  Read 1:     8,497,823 bp\n",
      "  Read 2:     8,502,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22211 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t264\t0.0\t2\t0 255 9\n",
      "26\t21680\t0.0\t2\t20849 803 28\n",
      "27\t164\t0.0\t2\t0 159 5\n",
      "28\t94\t0.0\t2\t0 0 94\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22083 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t241\t0.0\t2\t0 232 9\n",
      "26\t21733\t0.0\t2\t21252 471 10\n",
      "27\t77\t0.0\t2\t0 73 4\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1502_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1502_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1502_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1502_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1502_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1502_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1502_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1502_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        63,462 reads  @    114.2 µs/read;   0.53 M reads/minute\n",
      "Finished in 7.25 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,462\n",
      "  Read 1 with adapter:                  52,257 (82.3%)\n",
      "  Read 2 with adapter:                  52,105 (82.1%)\n",
      "Pairs written (passing filters):        63,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,426,225 bp\n",
      "  Read 1:    18,715,032 bp\n",
      "  Read 2:    18,711,193 bp\n",
      "Total written (filtered):     34,713,234 bp (92.8%)\n",
      "  Read 1:    17,356,584 bp\n",
      "  Read 2:    17,356,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52257 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t598\t0.0\t2\t0 588 10\n",
      "26\t51262\t0.0\t2\t50059 1175 28\n",
      "27\t286\t0.0\t2\t0 266 20\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52105 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t409\t0.0\t2\t0 393 16\n",
      "26\t51427\t0.0\t2\t50214 1169 44\n",
      "27\t232\t0.0\t2\t0 220 12\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1503_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1503_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1503_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1503_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1503_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1503_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1503_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1503_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        50,239 reads  @    112.0 µs/read;   0.54 M reads/minute\n",
      "Finished in 5.63 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,239\n",
      "  Read 1 with adapter:                  39,372 (78.4%)\n",
      "  Read 2 with adapter:                  39,369 (78.4%)\n",
      "Pairs written (passing filters):        50,239 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,635,675 bp\n",
      "  Read 1:    14,818,105 bp\n",
      "  Read 2:    14,817,570 bp\n",
      "Total written (filtered):     27,588,605 bp (93.1%)\n",
      "  Read 1:    13,794,571 bp\n",
      "  Read 2:    13,794,034 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t434\t0.0\t2\t0 423 11\n",
      "26\t38644\t0.0\t2\t37764 863 17\n",
      "27\t208\t0.0\t2\t0 200 8\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39369 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t250\t0.0\t2\t0 240 10\n",
      "26\t38896\t0.0\t2\t38018 844 34\n",
      "27\t170\t0.0\t2\t0 163 7\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1504_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1504_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1504_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1504_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1504_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1504_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1504_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1504_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        44,249 reads  @    115.0 µs/read;   0.52 M reads/minute\n",
      "Finished in 5.09 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,249\n",
      "  Read 1 with adapter:                  39,446 (89.1%)\n",
      "  Read 2 with adapter:                  38,984 (88.1%)\n",
      "Pairs written (passing filters):        44,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,008,756 bp\n",
      "  Read 1:    13,051,843 bp\n",
      "  Read 2:    12,956,913 bp\n",
      "Total written (filtered):     23,965,226 bp (92.1%)\n",
      "  Read 1:    12,021,789 bp\n",
      "  Read 2:    11,943,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39446 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t460\t0.0\t2\t0 449 11\n",
      "26\t38657\t0.0\t2\t37901 733 23\n",
      "27\t205\t0.0\t2\t0 198 7\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38984 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t311\t0.0\t2\t0 290 21\n",
      "26\t38454\t0.0\t2\t37278 1081 95\n",
      "27\t167\t0.0\t2\t0 157 10\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1505_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1505_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1505_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1505_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1505_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1505_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1505_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1505_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        46,162 reads  @    112.7 µs/read;   0.53 M reads/minute\n",
      "Finished in 5.20 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,162\n",
      "  Read 1 with adapter:                  39,666 (85.9%)\n",
      "  Read 2 with adapter:                  36,794 (79.7%)\n",
      "Pairs written (passing filters):        46,162 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,672,132 bp\n",
      "  Read 1:    13,617,289 bp\n",
      "  Read 2:    13,054,843 bp\n",
      "Total written (filtered):     24,684,542 bp (92.5%)\n",
      "  Read 1:    12,586,149 bp\n",
      "  Read 2:    12,098,393 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t472\t0.0\t2\t0 456 16\n",
      "26\t38881\t0.0\t2\t38088 775 18\n",
      "27\t250\t0.0\t2\t0 240 10\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t398\t0.0\t2\t0 377 21\n",
      "26\t36147\t0.0\t2\t35049 1017 81\n",
      "27\t214\t0.0\t2\t0 198 16\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1506_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1506_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1506_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1506_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1506_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1506_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1506_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1506_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        58,624 reads  @    117.7 µs/read;   0.51 M reads/minute\n",
      "Finished in 6.90 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,624\n",
      "  Read 1 with adapter:                  46,526 (79.4%)\n",
      "  Read 2 with adapter:                  46,451 (79.2%)\n",
      "Pairs written (passing filters):        58,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,562,189 bp\n",
      "  Read 1:    17,289,912 bp\n",
      "  Read 2:    17,272,277 bp\n",
      "Total written (filtered):     32,145,198 bp (93.0%)\n",
      "  Read 1:    16,080,471 bp\n",
      "  Read 2:    16,064,727 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t586\t0.0\t2\t0 568 18\n",
      "26\t45568\t0.0\t2\t44440 1097 31\n",
      "27\t273\t0.0\t2\t0 256 17\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46451 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t435\t0.0\t2\t0 418 17\n",
      "26\t45723\t0.0\t2\t44484 1175 64\n",
      "27\t235\t0.0\t2\t0 226 9\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1507_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1507_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1507_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1507_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1507_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1507_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1507_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1507_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        59,403 reads  @    115.9 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.89 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,403\n",
      "  Read 1 with adapter:                  47,609 (80.1%)\n",
      "  Read 2 with adapter:                  47,616 (80.2%)\n",
      "Pairs written (passing filters):        59,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,039,915 bp\n",
      "  Read 1:    17,522,754 bp\n",
      "  Read 2:    17,517,161 bp\n",
      "Total written (filtered):     32,564,286 bp (92.9%)\n",
      "  Read 1:    16,285,026 bp\n",
      "  Read 2:    16,279,260 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47609 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t565\t0.0\t2\t0 550 15\n",
      "26\t46641\t0.0\t2\t45489 1128 24\n",
      "27\t271\t0.0\t2\t0 259 12\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t398\t0.0\t2\t0 382 16\n",
      "26\t46940\t0.0\t2\t45730 1169 41\n",
      "27\t217\t0.0\t2\t0 209 8\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1508_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1508_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1508_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1508_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1508_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1508_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1508_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1508_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,879 reads  @    121.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 4.99 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,879\n",
      "  Read 1 with adapter:                  29,894 (73.1%)\n",
      "  Read 2 with adapter:                  29,894 (73.1%)\n",
      "Pairs written (passing filters):        40,879 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,105,112 bp\n",
      "  Read 1:    12,056,865 bp\n",
      "  Read 2:    12,048,247 bp\n",
      "Total written (filtered):     22,550,698 bp (93.6%)\n",
      "  Read 1:    11,279,618 bp\n",
      "  Read 2:    11,271,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29894 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t334\t0.0\t2\t0 326 8\n",
      "26\t29264\t0.0\t2\t28371 870 23\n",
      "27\t203\t0.0\t2\t0 195 8\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29894 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t224\t0.0\t2\t0 220 4\n",
      "26\t29509\t0.0\t2\t28815 672 22\n",
      "27\t119\t0.0\t2\t0 117 2\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1509_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1509_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1509_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1509_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1509_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1509_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1509_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1509_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        29,630 reads  @    125.4 µs/read;   0.48 M reads/minute\n",
      "Finished in 3.72 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,630\n",
      "  Read 1 with adapter:                  22,044 (74.4%)\n",
      "  Read 2 with adapter:                  21,903 (73.9%)\n",
      "Pairs written (passing filters):        29,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,478,102 bp\n",
      "  Read 1:     8,740,014 bp\n",
      "  Read 2:     8,738,088 bp\n",
      "Total written (filtered):     16,335,698 bp (93.5%)\n",
      "  Read 1:     8,166,963 bp\n",
      "  Read 2:     8,168,735 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t300\t0.0\t2\t0 295 5\n",
      "26\t21519\t0.0\t2\t20888 612 19\n",
      "27\t171\t0.0\t2\t0 164 7\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t218\t0.0\t2\t0 212 6\n",
      "26\t21576\t0.0\t2\t21129 429 18\n",
      "27\t89\t0.0\t2\t0 86 3\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1510_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1510_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1510_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1510_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1510_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1510_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1510_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1510_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        41,713 reads  @    113.2 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.73 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,713\n",
      "  Read 1 with adapter:                  31,161 (74.7%)\n",
      "  Read 2 with adapter:                  31,034 (74.4%)\n",
      "Pairs written (passing filters):        41,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,597,769 bp\n",
      "  Read 1:    12,304,774 bp\n",
      "  Read 2:    12,292,995 bp\n",
      "Total written (filtered):     22,980,934 bp (93.4%)\n",
      "  Read 1:    11,494,734 bp\n",
      "  Read 2:    11,486,200 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t420\t0.0\t2\t0 406 14\n",
      "26\t30489\t0.0\t2\t29796 683 10\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t241\t0.0\t2\t0 235 6\n",
      "26\t30642\t0.0\t2\t29934 672 36\n",
      "27\t118\t0.0\t2\t0 116 2\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1511_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1511_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1511_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1511_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1511_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1511_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1511_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1511_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        22,955 reads  @    107.1 µs/read;   0.56 M reads/minute\n",
      "Finished in 2.46 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,955\n",
      "  Read 1 with adapter:                  11,940 (52.0%)\n",
      "  Read 2 with adapter:                  11,888 (51.8%)\n",
      "Pairs written (passing filters):        22,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,538,038 bp\n",
      "  Read 1:     6,771,583 bp\n",
      "  Read 2:     6,766,455 bp\n",
      "Total written (filtered):     12,918,567 bp (95.4%)\n",
      "  Read 1:     6,461,170 bp\n",
      "  Read 2:     6,457,397 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 11940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t142\t0.0\t2\t0 139 3\n",
      "26\t11681\t0.0\t2\t11392 282 7\n",
      "27\t95\t0.0\t2\t0 91 4\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 11888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t89\t0.0\t2\t0 89\n",
      "26\t11730\t0.0\t2\t11459 261 10\n",
      "27\t59\t0.0\t2\t0 56 3\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1512_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1512_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1512_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1512_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1512_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1512_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1512_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1512_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,867 reads  @    108.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.75 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,867\n",
      "  Read 1 with adapter:                  35,264 (80.4%)\n",
      "  Read 2 with adapter:                  35,255 (80.4%)\n",
      "Pairs written (passing filters):        43,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,832,138 bp\n",
      "  Read 1:    12,940,450 bp\n",
      "  Read 2:    12,891,688 bp\n",
      "Total written (filtered):     23,997,780 bp (92.9%)\n",
      "  Read 1:    12,022,696 bp\n",
      "  Read 2:    11,975,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t437\t0.0\t2\t0 428 9\n",
      "26\t34520\t0.0\t2\t33683 820 17\n",
      "27\t239\t0.0\t2\t0 229 10\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t224\t0.0\t2\t0 215 9\n",
      "26\t34821\t0.0\t2\t33579 1144 98\n",
      "27\t174\t0.0\t2\t0 156 18\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1513_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1513_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1513_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1513_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1513_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1513_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1513_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1513_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,827 reads  @    121.5 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.33 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,827\n",
      "  Read 1 with adapter:                  32,577 (74.3%)\n",
      "  Read 2 with adapter:                  29,941 (68.3%)\n",
      "Pairs written (passing filters):        43,826 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,366,973 bp\n",
      "  Read 1:    12,928,939 bp\n",
      "  Read 2:    12,438,034 bp\n",
      "Total written (filtered):     23,741,570 bp (93.6%)\n",
      "  Read 1:    12,081,859 bp\n",
      "  Read 2:    11,659,711 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32577 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t458\t0.0\t2\t0 444 14\n",
      "26\t31830\t0.0\t2\t31072 739 19\n",
      "27\t197\t0.0\t2\t0 188 9\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29941 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t315\t0.0\t2\t0 300 15\n",
      "26\t29434\t0.0\t2\t28561 823 50\n",
      "27\t156\t0.0\t2\t0 147 9\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1514_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1514_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1514_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1514_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1514_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1514_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1514_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1514_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,155 reads  @    108.8 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.27 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,155\n",
      "  Read 1 with adapter:                  21,959 (56.1%)\n",
      "  Read 2 with adapter:                  21,967 (56.1%)\n",
      "Pairs written (passing filters):        39,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,092,700 bp\n",
      "  Read 1:    11,550,712 bp\n",
      "  Read 2:    11,541,988 bp\n",
      "Total written (filtered):     21,950,731 bp (95.1%)\n",
      "  Read 1:    10,979,853 bp\n",
      "  Read 2:    10,970,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21959 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t290\t0.0\t2\t0 284 6\n",
      "26\t21458\t0.0\t2\t21016 431 11\n",
      "27\t139\t0.0\t2\t0 134 5\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t168\t0.0\t2\t0 159 9\n",
      "26\t21667\t0.0\t2\t21125 508 34\n",
      "27\t76\t0.0\t2\t0 73 3\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1515_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1515_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1515_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1515_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1515_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1515_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1515_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1515_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        34,541 reads  @    109.2 µs/read;   0.55 M reads/minute\n",
      "Finished in 3.78 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,541\n",
      "  Read 1 with adapter:                  19,327 (56.0%)\n",
      "  Read 2 with adapter:                  19,319 (55.9%)\n",
      "Pairs written (passing filters):        34,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,371,319 bp\n",
      "  Read 1:    10,189,135 bp\n",
      "  Read 2:    10,182,184 bp\n",
      "Total written (filtered):     19,366,622 bp (95.1%)\n",
      "  Read 1:     9,686,697 bp\n",
      "  Read 2:     9,679,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 19327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t239\t0.0\t2\t0 235 4\n",
      "26\t18917\t0.0\t2\t18420 488 9\n",
      "27\t103\t0.0\t2\t0 100 3\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t145\t0.0\t2\t0 141 4\n",
      "26\t19053\t0.0\t2\t18572 457 24\n",
      "27\t112\t0.0\t2\t0 108 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1516_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1516_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1516_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1516_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1516_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1516_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1516_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1516_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,464 reads  @    115.9 µs/read;   0.52 M reads/minute\n",
      "Finished in 4.93 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,464\n",
      "  Read 1 with adapter:                  23,867 (56.2%)\n",
      "  Read 2 with adapter:                  23,881 (56.2%)\n",
      "Pairs written (passing filters):        42,464 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,037,691 bp\n",
      "  Read 1:    12,526,353 bp\n",
      "  Read 2:    12,511,338 bp\n",
      "Total written (filtered):     23,796,466 bp (95.0%)\n",
      "  Read 1:    11,905,946 bp\n",
      "  Read 2:    11,890,520 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23867 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t335\t0.0\t2\t0 328 7\n",
      "26\t23324\t0.0\t2\t22763 550 11\n",
      "27\t144\t0.0\t2\t0 142 2\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t190\t0.0\t2\t0 184 6\n",
      "26\t23577\t0.0\t2\t23014 543 20\n",
      "27\t94\t0.0\t2\t0 93 1\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1517_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1517_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1517_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1517_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1517_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1517_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1517_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1517_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        54,218 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.74 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,218\n",
      "  Read 1 with adapter:                  46,001 (84.8%)\n",
      "  Read 2 with adapter:                  45,700 (84.3%)\n",
      "Pairs written (passing filters):        54,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,979,626 bp\n",
      "  Read 1:    15,991,248 bp\n",
      "  Read 2:    15,988,378 bp\n",
      "Total written (filtered):     29,595,408 bp (92.5%)\n",
      "  Read 1:    14,795,262 bp\n",
      "  Read 2:    14,800,146 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t434\t0.0\t2\t0 424 10\n",
      "26\t45186\t0.0\t2\t44186 980 20\n",
      "27\t292\t0.0\t2\t0 280 12\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45700 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t478\t0.0\t2\t0 453 25\n",
      "26\t44971\t0.0\t2\t43764 1135 72\n",
      "27\t169\t0.0\t2\t0 160 9\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1518_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1518_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1518_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1518_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1518_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1518_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1518_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1518_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        56,236 reads  @    113.4 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.38 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,236\n",
      "  Read 1 with adapter:                  45,029 (80.1%)\n",
      "  Read 2 with adapter:                  44,941 (79.9%)\n",
      "Pairs written (passing filters):        56,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,165,549 bp\n",
      "  Read 1:    16,586,800 bp\n",
      "  Read 2:    16,578,749 bp\n",
      "Total written (filtered):     30,826,649 bp (92.9%)\n",
      "  Read 1:    15,416,247 bp\n",
      "  Read 2:    15,410,402 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45029 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t500\t0.0\t2\t0 487 13\n",
      "26\t44226\t0.0\t2\t43209 990 27\n",
      "27\t219\t0.0\t2\t0 216 3\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44941 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t378\t0.0\t2\t0 360 18\n",
      "26\t44284\t0.0\t2\t43065 1156 63\n",
      "27\t231\t0.0\t2\t0 219 12\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1519_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1519_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1519_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1519_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1519_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1519_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1519_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1519_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        55,961 reads  @    125.3 µs/read;   0.48 M reads/minute\n",
      "Finished in 7.01 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,961\n",
      "  Read 1 with adapter:                  43,943 (78.5%)\n",
      "  Read 2 with adapter:                  43,966 (78.6%)\n",
      "Pairs written (passing filters):        55,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,006,519 bp\n",
      "  Read 1:    16,503,720 bp\n",
      "  Read 2:    16,502,799 bp\n",
      "Total written (filtered):     30,721,026 bp (93.1%)\n",
      "  Read 1:    15,361,286 bp\n",
      "  Read 2:    15,359,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43943 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t431\t0.0\t2\t0 418 13\n",
      "26\t43174\t0.0\t2\t42069 1084 21\n",
      "27\t225\t0.0\t2\t0 216 9\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43966 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t281\t0.0\t2\t0 266 15\n",
      "26\t43449\t0.0\t2\t42322 1070 57\n",
      "27\t184\t0.0\t2\t0 176 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1520_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1520_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1520_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1520_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1520_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1520_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1520_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1520_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,731 reads  @    125.4 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.49 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,731\n",
      "  Read 1 with adapter:                  36,291 (83.0%)\n",
      "  Read 2 with adapter:                  35,504 (81.2%)\n",
      "Pairs written (passing filters):        43,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,636,222 bp\n",
      "  Read 1:    12,899,345 bp\n",
      "  Read 2:    12,736,877 bp\n",
      "Total written (filtered):     23,766,920 bp (92.7%)\n",
      "  Read 1:    11,953,131 bp\n",
      "  Read 2:    11,813,789 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t348\t0.0\t2\t0 342 6\n",
      "26\t35678\t0.0\t2\t34867 795 16\n",
      "27\t188\t0.0\t2\t0 177 11\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t205\t0.0\t2\t0 188 17\n",
      "26\t35096\t0.0\t2\t33833 1131 132\n",
      "27\t173\t0.0\t2\t0 156 17\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1521_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1521_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1521_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1521_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1521_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1521_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1521_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1521_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,042 reads  @    100.2 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.61 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,042\n",
      "  Read 1 with adapter:                  40,038 (87.0%)\n",
      "  Read 2 with adapter:                  34,613 (75.2%)\n",
      "Pairs written (passing filters):        46,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,168,318 bp\n",
      "  Read 1:    13,574,942 bp\n",
      "  Read 2:    12,593,376 bp\n",
      "Total written (filtered):     24,226,703 bp (92.6%)\n",
      "  Read 1:    12,533,097 bp\n",
      "  Read 2:    11,693,606 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40038 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t364\t0.0\t2\t0 350 14\n",
      "26\t39380\t0.0\t2\t38533 832 15\n",
      "27\t194\t0.0\t2\t0 185 9\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34613 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t363\t0.0\t2\t0 351 12\n",
      "26\t34023\t0.0\t2\t32868 1053 102\n",
      "27\t203\t0.0\t2\t0 181 22\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1522_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1522_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1522_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1522_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1522_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1522_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1522_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1522_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        51,743 reads  @    115.3 µs/read;   0.52 M reads/minute\n",
      "Finished in 5.97 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,743\n",
      "  Read 1 with adapter:                  41,744 (80.7%)\n",
      "  Read 2 with adapter:                  41,736 (80.7%)\n",
      "Pairs written (passing filters):        51,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,510,158 bp\n",
      "  Read 1:    15,255,428 bp\n",
      "  Read 2:    15,254,730 bp\n",
      "Total written (filtered):     28,339,891 bp (92.9%)\n",
      "  Read 1:    14,170,145 bp\n",
      "  Read 2:    14,169,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t405\t0.0\t2\t0 399 6\n",
      "26\t41036\t0.0\t2\t40138 881 17\n",
      "27\t178\t0.0\t2\t0 172 6\n",
      "28\t104\t0.0\t2\t0 0 104\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41736 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t369\t0.0\t2\t0 352 17\n",
      "26\t41152\t0.0\t2\t39902 1173 77\n",
      "27\t177\t0.0\t2\t0 168 9\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1523_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1523_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1523_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1523_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1523_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1523_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1523_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1523_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        65,545 reads  @    106.1 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.96 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,545\n",
      "  Read 1 with adapter:                  65,215 (99.5%)\n",
      "  Read 2 with adapter:                  65,139 (99.4%)\n",
      "Pairs written (passing filters):        65,545 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,657,813 bp\n",
      "  Read 1:    19,330,990 bp\n",
      "  Read 2:    19,326,823 bp\n",
      "Total written (filtered):     35,268,848 bp (91.2%)\n",
      "  Read 1:    17,635,511 bp\n",
      "  Read 2:    17,633,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t602\t0.0\t2\t0 586 16\n",
      "26\t64173\t0.0\t2\t62669 1461 43\n",
      "27\t285\t0.0\t2\t0 269 16\n",
      "28\t129\t0.0\t2\t0 0 129\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t471\t0.0\t2\t0 454 17\n",
      "26\t64311\t0.0\t2\t62052 2076 183\n",
      "27\t287\t0.0\t2\t0 265 22\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1524_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1524_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1524_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1524_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1524_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1524_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1524_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1524_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        54,531 reads  @    103.5 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.65 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,531\n",
      "  Read 1 with adapter:                  43,066 (79.0%)\n",
      "  Read 2 with adapter:                  43,110 (79.1%)\n",
      "Pairs written (passing filters):        54,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,144,838 bp\n",
      "  Read 1:    16,078,610 bp\n",
      "  Read 2:    16,066,228 bp\n",
      "Total written (filtered):     29,904,539 bp (93.0%)\n",
      "  Read 1:    14,959,042 bp\n",
      "  Read 2:    14,945,497 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43066 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t478\t0.0\t2\t0 467 11\n",
      "26\t42282\t0.0\t2\t41237 1016 29\n",
      "27\t210\t0.0\t2\t0 199 11\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t351\t0.0\t2\t0 337 14\n",
      "26\t42512\t0.0\t2\t41227 1202 83\n",
      "27\t212\t0.0\t2\t0 196 16\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1525_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1525_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1525_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1525_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1525_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1525_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1525_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1525_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        16,167 reads  @    116.0 µs/read;   0.52 M reads/minute\n",
      "Finished in 1.88 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,167\n",
      "  Read 1 with adapter:                   7,140 (44.2%)\n",
      "  Read 2 with adapter:                   7,107 (44.0%)\n",
      "Pairs written (passing filters):        16,167 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,536,265 bp\n",
      "  Read 1:     4,767,873 bp\n",
      "  Read 2:     4,768,392 bp\n",
      "Total written (filtered):      9,165,847 bp (96.1%)\n",
      "  Read 1:     4,582,179 bp\n",
      "  Read 2:     4,583,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 7140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t91\t0.0\t2\t0 84 7\n",
      "26\t6931\t0.0\t2\t6619 301 11\n",
      "27\t75\t0.0\t2\t0 71 4\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 7107 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t86\t0.0\t2\t0 84 2\n",
      "26\t6979\t0.0\t2\t6846 129 4\n",
      "27\t36\t0.0\t2\t0 35 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1526_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1526_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1526_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1526_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1526_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1526_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1526_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1526_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        55,139 reads  @    121.6 µs/read;   0.49 M reads/minute\n",
      "Finished in 6.71 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,139\n",
      "  Read 1 with adapter:                  45,164 (81.9%)\n",
      "  Read 2 with adapter:                  45,047 (81.7%)\n",
      "Pairs written (passing filters):        55,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,514,207 bp\n",
      "  Read 1:    16,262,923 bp\n",
      "  Read 2:    16,251,284 bp\n",
      "Total written (filtered):     30,169,119 bp (92.8%)\n",
      "  Read 1:    15,088,894 bp\n",
      "  Read 2:    15,080,225 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t553\t0.0\t2\t0 537 16\n",
      "26\t44274\t0.0\t2\t43272 974 28\n",
      "27\t256\t0.0\t2\t0 251 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45047 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t334\t0.0\t2\t0 325 9\n",
      "26\t44499\t0.0\t2\t43490 960 49\n",
      "27\t169\t0.0\t2\t0 165 4\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1527_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1527_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1527_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1527_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1527_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1527_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1527_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1527_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        53,200 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.50 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,200\n",
      "  Read 1 with adapter:                  43,507 (81.8%)\n",
      "  Read 2 with adapter:                  43,534 (81.8%)\n",
      "Pairs written (passing filters):        53,200 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,385,340 bp\n",
      "  Read 1:    15,692,912 bp\n",
      "  Read 2:    15,692,428 bp\n",
      "Total written (filtered):     29,122,462 bp (92.8%)\n",
      "  Read 1:    14,561,847 bp\n",
      "  Read 2:    14,560,615 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43507 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t522\t0.0\t2\t0 512 10\n",
      "26\t42621\t0.0\t2\t41691 912 18\n",
      "27\t235\t0.0\t2\t0 232 3\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t285\t0.0\t2\t0 278 7\n",
      "26\t43032\t0.0\t2\t42120 880 32\n",
      "27\t196\t0.0\t2\t0 190 6\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1528_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1528_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1528_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1528_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1528_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1528_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1528_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1528_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,162 reads  @    122.8 µs/read;   0.49 M reads/minute\n",
      "Finished in 4.94 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,162\n",
      "  Read 1 with adapter:                  34,860 (86.8%)\n",
      "  Read 2 with adapter:                  34,494 (85.9%)\n",
      "Pairs written (passing filters):        40,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,621,238 bp\n",
      "  Read 1:    11,843,961 bp\n",
      "  Read 2:    11,777,277 bp\n",
      "Total written (filtered):     21,816,893 bp (92.4%)\n",
      "  Read 1:    10,936,412 bp\n",
      "  Read 2:    10,880,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t450\t0.0\t2\t0 443 7\n",
      "26\t34119\t0.0\t2\t33236 869 14\n",
      "27\t200\t0.0\t2\t0 189 11\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34494 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t259\t0.0\t2\t0 243 16\n",
      "26\t34025\t0.0\t2\t32977 988 60\n",
      "27\t165\t0.0\t2\t0 155 10\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1529_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1529_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1529_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1529_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1529_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1529_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1529_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1529_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        45,565 reads  @    122.1 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.57 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,565\n",
      "  Read 1 with adapter:                  40,676 (89.3%)\n",
      "  Read 2 with adapter:                  38,071 (83.6%)\n",
      "Pairs written (passing filters):        45,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,361,740 bp\n",
      "  Read 1:    13,437,597 bp\n",
      "  Read 2:    12,924,143 bp\n",
      "Total written (filtered):     24,314,569 bp (92.2%)\n",
      "  Read 1:    12,380,073 bp\n",
      "  Read 2:    11,934,496 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t479\t0.0\t2\t0 466 13\n",
      "26\t39820\t0.0\t2\t38897 898 25\n",
      "27\t251\t0.0\t2\t0 244 7\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38071 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t390\t0.0\t2\t0 371 19\n",
      "26\t37455\t0.0\t2\t36474 921 60\n",
      "27\t181\t0.0\t2\t0 168 13\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1530_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1530_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1530_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1530_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1530_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1530_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1530_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1530_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        47,848 reads  @    103.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.97 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,848\n",
      "  Read 1 with adapter:                  39,212 (82.0%)\n",
      "  Read 2 with adapter:                  39,204 (81.9%)\n",
      "Pairs written (passing filters):        47,848 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,223,833 bp\n",
      "  Read 1:    14,114,074 bp\n",
      "  Read 2:    14,109,759 bp\n",
      "Total written (filtered):     26,185,237 bp (92.8%)\n",
      "  Read 1:    13,094,645 bp\n",
      "  Read 2:    13,090,592 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39212 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t463\t0.0\t2\t0 446 17\n",
      "26\t38405\t0.0\t2\t37537 850 18\n",
      "27\t224\t0.0\t2\t0 219 5\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t321\t0.0\t2\t0 315 6\n",
      "26\t38680\t0.0\t2\t37759 884 37\n",
      "27\t174\t0.0\t2\t0 167 7\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1531_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1531_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1531_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1531_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1531_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1531_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1531_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1531_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        12,115 reads  @    129.0 µs/read;   0.47 M reads/minute\n",
      "Finished in 1.57 s (129 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,115\n",
      "  Read 1 with adapter:                  10,020 (82.7%)\n",
      "  Read 2 with adapter:                   9,993 (82.5%)\n",
      "Pairs written (passing filters):        12,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,140,719 bp\n",
      "  Read 1:     3,573,116 bp\n",
      "  Read 2:     3,567,603 bp\n",
      "Total written (filtered):      6,620,376 bp (92.7%)\n",
      "  Read 1:     3,312,576 bp\n",
      "  Read 2:     3,307,800 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 10020 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t116\t0.0\t2\t0 108 8\n",
      "26\t9793\t0.0\t2\t9332 440 21\n",
      "27\t70\t0.0\t2\t0 65 5\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 9993 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t83\t0.0\t2\t0 74 9\n",
      "26\t9838\t0.0\t2\t9037 734 67\n",
      "27\t56\t0.0\t2\t0 49 7\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1532_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1532_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1532_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1532_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1532_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1532_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1532_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1532_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        52,717 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.24 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,717\n",
      "  Read 1 with adapter:                  45,968 (87.2%)\n",
      "  Read 2 with adapter:                  45,976 (87.2%)\n",
      "Pairs written (passing filters):        52,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,094,887 bp\n",
      "  Read 1:    15,550,651 bp\n",
      "  Read 2:    15,544,236 bp\n",
      "Total written (filtered):     28,704,693 bp (92.3%)\n",
      "  Read 1:    14,355,649 bp\n",
      "  Read 2:    14,349,044 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45968 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t553\t0.0\t2\t0 545 8\n",
      "26\t45014\t0.0\t2\t43993 997 24\n",
      "27\t303\t0.0\t2\t0 295 8\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45976 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t414\t0.0\t2\t0 401 13\n",
      "26\t45322\t0.0\t2\t44456 839 27\n",
      "27\t206\t0.0\t2\t0 197 9\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1533_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1533_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1533_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1533_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1533_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1533_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1533_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1533_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1549_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1549_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1549_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1549_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1549_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1549_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1549_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1549_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        31,776 reads  @    116.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 3.72 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,776\n",
      "  Read 1 with adapter:                  17,461 (55.0%)\n",
      "  Read 2 with adapter:                  17,409 (54.8%)\n",
      "Pairs written (passing filters):        31,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,736,179 bp\n",
      "  Read 1:     9,364,317 bp\n",
      "  Read 2:     9,371,862 bp\n",
      "Total written (filtered):     17,829,595 bp (95.2%)\n",
      "  Read 1:     8,910,294 bp\n",
      "  Read 2:     8,919,301 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17461 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t211\t0.0\t2\t0 198 13\n",
      "26\t17035\t0.0\t2\t16172 830 33\n",
      "27\t138\t0.0\t2\t0 125 13\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t186\t0.0\t2\t0 181 5\n",
      "26\t17106\t0.0\t2\t16637 439 30\n",
      "27\t101\t0.0\t2\t0 99 2\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1550_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1550_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1550_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1550_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1550_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1550_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1550_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1550_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,943 reads  @    101.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.85 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,943\n",
      "  Read 1 with adapter:                  22,170 (46.2%)\n",
      "  Read 2 with adapter:                  22,176 (46.3%)\n",
      "Pairs written (passing filters):        47,943 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,269,424 bp\n",
      "  Read 1:    14,130,743 bp\n",
      "  Read 2:    14,138,681 bp\n",
      "Total written (filtered):     27,116,590 bp (95.9%)\n",
      "  Read 1:    13,554,405 bp\n",
      "  Read 2:    13,562,185 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22170 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t265\t0.0\t2\t0 259 6\n",
      "26\t21721\t0.0\t2\t21188 522 11\n",
      "27\t145\t0.0\t2\t0 141 4\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t186\t0.0\t2\t0 181 5\n",
      "26\t21869\t0.0\t2\t21417 430 22\n",
      "27\t108\t0.0\t2\t0 106 2\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1551_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1551_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1551_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1551_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1551_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1551_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1551_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1551_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        46,013 reads  @    105.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.85 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,013\n",
      "  Read 1 with adapter:                  21,814 (47.4%)\n",
      "  Read 2 with adapter:                  21,887 (47.6%)\n",
      "Pairs written (passing filters):        46,013 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,131,128 bp\n",
      "  Read 1:    13,558,311 bp\n",
      "  Read 2:    13,572,817 bp\n",
      "Total written (filtered):     25,995,008 bp (95.8%)\n",
      "  Read 1:    12,991,213 bp\n",
      "  Read 2:    13,003,795 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21814 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t250\t0.0\t2\t0 244 6\n",
      "26\t21396\t0.0\t2\t20889 490 17\n",
      "27\t124\t0.0\t2\t0 117 7\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t148\t0.0\t2\t0 145 3\n",
      "26\t21631\t0.0\t2\t21187 426 18\n",
      "27\t84\t0.0\t2\t0 81 3\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1552_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1552_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1552_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1552_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1552_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1552_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1552_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1552_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        33,115 reads  @    111.0 µs/read;   0.54 M reads/minute\n",
      "Finished in 3.68 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,115\n",
      "  Read 1 with adapter:                  19,940 (60.2%)\n",
      "  Read 2 with adapter:                  19,662 (59.4%)\n",
      "Pairs written (passing filters):        33,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,446,063 bp\n",
      "  Read 1:     9,750,049 bp\n",
      "  Read 2:     9,696,014 bp\n",
      "Total written (filtered):     18,412,456 bp (94.7%)\n",
      "  Read 1:     9,227,622 bp\n",
      "  Read 2:     9,184,834 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 19940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t220\t0.0\t2\t0 210 10\n",
      "26\t19571\t0.0\t2\t19122 439 10\n",
      "27\t106\t0.0\t2\t0 101 5\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t140\t0.0\t2\t0 135 5\n",
      "26\t19402\t0.0\t2\t18842 534 26\n",
      "27\t100\t0.0\t2\t0 93 7\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1553_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1553_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1553_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1553_75_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1553_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1553_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1553_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1553_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,864 reads  @    107.1 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.70 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,864\n",
      "  Read 1 with adapter:                  27,715 (63.2%)\n",
      "  Read 2 with adapter:                  26,136 (59.6%)\n",
      "Pairs written (passing filters):        43,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,535,537 bp\n",
      "  Read 1:    12,931,978 bp\n",
      "  Read 2:    12,603,559 bp\n",
      "Total written (filtered):     24,135,277 bp (94.5%)\n",
      "  Read 1:    12,211,135 bp\n",
      "  Read 2:    11,924,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t267\t0.0\t2\t0 260 7\n",
      "26\t27198\t0.0\t2\t26554 626 18\n",
      "27\t199\t0.0\t2\t0 193 6\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26136 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t284\t0.0\t2\t0 268 16\n",
      "26\t25674\t0.0\t2\t24983 644 47\n",
      "27\t159\t0.0\t2\t0 149 10\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1554_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1554_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1554_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1554_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1554_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1554_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1554_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1554_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        56,060 reads  @    114.3 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.41 s (114 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,060\n",
      "  Read 1 with adapter:                  34,375 (61.3%)\n",
      "  Read 2 with adapter:                  34,428 (61.4%)\n",
      "Pairs written (passing filters):        56,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,026,422 bp\n",
      "  Read 1:    16,508,474 bp\n",
      "  Read 2:    16,517,948 bp\n",
      "Total written (filtered):     31,237,742 bp (94.6%)\n",
      "  Read 1:    15,614,827 bp\n",
      "  Read 2:    15,622,915 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34375 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t377\t0.0\t2\t0 369 8\n",
      "26\t33729\t0.0\t2\t32853 856 20\n",
      "27\t208\t0.0\t2\t0 195 13\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t274\t0.0\t2\t0 267 7\n",
      "26\t33958\t0.0\t2\t33148 785 25\n",
      "27\t165\t0.0\t2\t0 159 6\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1555_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1555_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1555_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1555_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1555_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1555_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1555_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1555_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,785 reads  @    107.9 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.27 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,785\n",
      "  Read 1 with adapter:                  33,628 (68.9%)\n",
      "  Read 2 with adapter:                  33,761 (69.2%)\n",
      "Pairs written (passing filters):        48,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,746,274 bp\n",
      "  Read 1:    14,359,243 bp\n",
      "  Read 2:    14,387,031 bp\n",
      "Total written (filtered):     26,994,271 bp (93.9%)\n",
      "  Read 1:    13,484,965 bp\n",
      "  Read 2:    13,509,306 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t411\t0.0\t2\t0 397 14\n",
      "26\t32905\t0.0\t2\t32081 801 23\n",
      "27\t195\t0.0\t2\t0 187 8\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33761 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t253\t0.0\t2\t0 245 8\n",
      "26\t33312\t0.0\t2\t32514 764 34\n",
      "27\t160\t0.0\t2\t0 149 11\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1556_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1556_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1556_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1556_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1556_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1556_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1556_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1556_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,250 reads  @    105.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.09 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,250\n",
      "  Read 1 with adapter:                  20,739 (43.0%)\n",
      "  Read 2 with adapter:                  20,758 (43.0%)\n",
      "Pairs written (passing filters):        48,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,435,557 bp\n",
      "  Read 1:    14,227,825 bp\n",
      "  Read 2:    14,207,732 bp\n",
      "Total written (filtered):     27,356,647 bp (96.2%)\n",
      "  Read 1:    13,688,612 bp\n",
      "  Read 2:    13,668,035 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t232\t0.0\t2\t0 228 4\n",
      "26\t20293\t0.0\t2\t19736 549 8\n",
      "27\t149\t0.0\t2\t0 145 4\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20758 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t151\t0.0\t2\t0 148 3\n",
      "26\t20482\t0.0\t2\t20016 453 13\n",
      "27\t102\t0.0\t2\t0 96 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1557_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1557_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1557_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1557_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1557_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1557_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1557_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1557_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,102 reads  @    117.0 µs/read;   0.51 M reads/minute\n",
      "Finished in 5.05 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,102\n",
      "  Read 1 with adapter:                  29,893 (69.4%)\n",
      "  Read 2 with adapter:                  29,749 (69.0%)\n",
      "Pairs written (passing filters):        43,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,420,200 bp\n",
      "  Read 1:    12,709,752 bp\n",
      "  Read 2:    12,710,448 bp\n",
      "Total written (filtered):     23,869,855 bp (93.9%)\n",
      "  Read 1:    11,932,705 bp\n",
      "  Read 2:    11,937,150 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29893 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t376\t0.0\t2\t0 370 6\n",
      "26\t29296\t0.0\t2\t28523 750 23\n",
      "27\t165\t0.0\t2\t0 161 4\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t321\t0.0\t2\t0 312 9\n",
      "26\t29247\t0.0\t2\t28524 698 25\n",
      "27\t145\t0.0\t2\t0 139 6\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1558_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1558_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1558_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1558_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1558_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1558_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1558_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1558_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        35,702 reads  @    120.9 µs/read;   0.50 M reads/minute\n",
      "Finished in 4.32 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,702\n",
      "  Read 1 with adapter:                  23,856 (66.8%)\n",
      "  Read 2 with adapter:                  23,778 (66.6%)\n",
      "Pairs written (passing filters):        35,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,051,055 bp\n",
      "  Read 1:    10,527,810 bp\n",
      "  Read 2:    10,523,245 bp\n",
      "Total written (filtered):     19,812,689 bp (94.1%)\n",
      "  Read 1:     9,907,601 bp\n",
      "  Read 2:     9,905,088 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t300\t0.0\t2\t0 287 13\n",
      "26\t23318\t0.0\t2\t22631 673 14\n",
      "27\t179\t0.0\t2\t0 176 3\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t188\t0.0\t2\t0 183 5\n",
      "26\t23470\t0.0\t2\t22791 654 25\n",
      "27\t91\t0.0\t2\t0 85 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1559_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1559_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1559_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1559_87_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1559_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1559_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1559_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1559_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,920 reads  @    112.5 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.61 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,920\n",
      "  Read 1 with adapter:                  26,825 (65.6%)\n",
      "  Read 2 with adapter:                  26,752 (65.4%)\n",
      "Pairs written (passing filters):        40,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,127,170 bp\n",
      "  Read 1:    12,067,094 bp\n",
      "  Read 2:    12,060,076 bp\n",
      "Total written (filtered):     22,734,041 bp (94.2%)\n",
      "  Read 1:    11,369,454 bp\n",
      "  Read 2:    11,364,587 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t316\t0.0\t2\t0 304 12\n",
      "26\t26283\t0.0\t2\t25672 597 14\n",
      "27\t163\t0.0\t2\t0 157 6\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26752 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t191\t0.0\t2\t0 187 4\n",
      "26\t26424\t0.0\t2\t25738 648 38\n",
      "27\t102\t0.0\t2\t0 99 3\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1560_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1560_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1560_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1560_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1560_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1560_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1560_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1560_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        34,666 reads  @    106.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 3.70 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,666\n",
      "  Read 1 with adapter:                  21,532 (62.1%)\n",
      "  Read 2 with adapter:                  21,331 (61.5%)\n",
      "Pairs written (passing filters):        34,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,417,081 bp\n",
      "  Read 1:    10,225,225 bp\n",
      "  Read 2:    10,191,856 bp\n",
      "Total written (filtered):     19,301,327 bp (94.5%)\n",
      "  Read 1:     9,664,060 bp\n",
      "  Read 2:     9,637,267 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t223\t0.0\t2\t0 218 5\n",
      "26\t21101\t0.0\t2\t20580 508 13\n",
      "27\t168\t0.0\t2\t0 161 7\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21331 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t141\t0.0\t2\t0 133 8\n",
      "26\t21061\t0.0\t2\t20317 680 64\n",
      "27\t110\t0.0\t2\t0 101 9\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1561_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1561_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1561_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1561_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1561_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1561_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1561_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1561_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        35,611 reads  @    120.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 4.31 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,611\n",
      "  Read 1 with adapter:                  23,440 (65.8%)\n",
      "  Read 2 with adapter:                  21,526 (60.4%)\n",
      "Pairs written (passing filters):        35,611 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,641,004 bp\n",
      "  Read 1:    10,501,268 bp\n",
      "  Read 2:    10,139,736 bp\n",
      "Total written (filtered):     19,472,093 bp (94.3%)\n",
      "  Read 1:     9,891,890 bp\n",
      "  Read 2:     9,580,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23440 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t267\t0.0\t2\t0 265 2\n",
      "26\t22957\t0.0\t2\t22392 556 9\n",
      "27\t167\t0.0\t2\t0 156 11\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t248\t0.0\t2\t0 243 5\n",
      "26\t21151\t0.0\t2\t20407 696 48\n",
      "27\t105\t0.0\t2\t0 101 4\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1562_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1562_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1562_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1562_93_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1562_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1562_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1562_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1562_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        51,012 reads  @    109.2 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.58 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,012\n",
      "  Read 1 with adapter:                  41,054 (80.5%)\n",
      "  Read 2 with adapter:                  41,041 (80.5%)\n",
      "Pairs written (passing filters):        51,012 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,078,901 bp\n",
      "  Read 1:    15,042,191 bp\n",
      "  Read 2:    15,036,710 bp\n",
      "Total written (filtered):     27,944,485 bp (92.9%)\n",
      "  Read 1:    13,974,768 bp\n",
      "  Read 2:    13,969,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41054 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t443\t0.0\t2\t0 433 10\n",
      "26\t40189\t0.0\t2\t39198 970 21\n",
      "27\t306\t0.0\t2\t0 297 9\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41041 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t339\t0.0\t2\t0 323 16\n",
      "26\t40465\t0.0\t2\t39203 1178 84\n",
      "27\t156\t0.0\t2\t0 152 4\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1563_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1563_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1563_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1563_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1563_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1563_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1563_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1563_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        49,158 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.29 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,158\n",
      "  Read 1 with adapter:                  25,554 (52.0%)\n",
      "  Read 2 with adapter:                  25,590 (52.1%)\n",
      "Pairs written (passing filters):        49,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,989,252 bp\n",
      "  Read 1:    14,495,883 bp\n",
      "  Read 2:    14,493,369 bp\n",
      "Total written (filtered):     27,659,337 bp (95.4%)\n",
      "  Read 1:    13,831,228 bp\n",
      "  Read 2:    13,828,109 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 25554 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t307\t0.0\t2\t0 294 13\n",
      "26\t24995\t0.0\t2\t24382 602 11\n",
      "27\t193\t0.0\t2\t0 187 6\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t205\t0.0\t2\t0 201 4\n",
      "26\t25241\t0.0\t2\t24570 638 33\n",
      "27\t123\t0.0\t2\t0 115 8\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1564_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1564_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1564_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1564_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1564_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1564_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1564_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1564_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        45,384 reads  @    116.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 5.31 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,384\n",
      "  Read 1 with adapter:                  24,263 (53.5%)\n",
      "  Read 2 with adapter:                  24,283 (53.5%)\n",
      "Pairs written (passing filters):        45,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,747,468 bp\n",
      "  Read 1:    13,384,810 bp\n",
      "  Read 2:    13,362,658 bp\n",
      "Total written (filtered):     25,485,421 bp (95.3%)\n",
      "  Read 1:    12,754,043 bp\n",
      "  Read 2:    12,731,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t286\t0.0\t2\t0 275 11\n",
      "26\t23780\t0.0\t2\t23186 576 18\n",
      "27\t147\t0.0\t2\t0 142 5\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24283 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t174\t0.0\t2\t0 170 4\n",
      "26\t24002\t0.0\t2\t23473 510 19\n",
      "27\t90\t0.0\t2\t0 88 2\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1565_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1565_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1565_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1565_99_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1565_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1565_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1565_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1565_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        50,612 reads  @    106.2 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.38 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,612\n",
      "  Read 1 with adapter:                  32,888 (65.0%)\n",
      "  Read 2 with adapter:                  32,672 (64.6%)\n",
      "Pairs written (passing filters):        50,612 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,852,955 bp\n",
      "  Read 1:    14,926,548 bp\n",
      "  Read 2:    14,926,407 bp\n",
      "Total written (filtered):     28,148,553 bp (94.3%)\n",
      "  Read 1:    14,071,449 bp\n",
      "  Read 2:    14,077,104 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t270\t0.0\t2\t0 267 3\n",
      "26\t32341\t0.0\t2\t31535 796 10\n",
      "27\t221\t0.0\t2\t0 216 5\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32672 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t331\t0.0\t2\t0 323 8\n",
      "26\t32182\t0.0\t2\t31407 739 36\n",
      "27\t108\t0.0\t2\t0 106 2\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1566_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1566_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1566_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1566_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1566_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1566_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1566_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1566_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        42,914 reads  @    102.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.41 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,914\n",
      "  Read 1 with adapter:                  18,853 (43.9%)\n",
      "  Read 2 with adapter:                  18,850 (43.9%)\n",
      "Pairs written (passing filters):        42,914 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,306,272 bp\n",
      "  Read 1:    12,651,254 bp\n",
      "  Read 2:    12,655,018 bp\n",
      "Total written (filtered):     24,326,103 bp (96.1%)\n",
      "  Read 1:    12,161,137 bp\n",
      "  Read 2:    12,164,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 18853 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t164\t0.0\t2\t0 159 5\n",
      "26\t18568\t0.0\t2\t18160 403 5\n",
      "27\t95\t0.0\t2\t0 90 5\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 18850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t149\t0.0\t2\t0 140 9\n",
      "26\t18591\t0.0\t2\t18188 386 17\n",
      "27\t91\t0.0\t2\t0 87 4\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1567_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1567_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1567_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1567_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1567_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1567_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1567_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1567_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        50,448 reads  @    120.3 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.07 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,448\n",
      "  Read 1 with adapter:                  36,735 (72.8%)\n",
      "  Read 2 with adapter:                  36,823 (73.0%)\n",
      "Pairs written (passing filters):        50,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,746,734 bp\n",
      "  Read 1:    14,869,614 bp\n",
      "  Read 2:    14,877,120 bp\n",
      "Total written (filtered):     27,834,280 bp (93.6%)\n",
      "  Read 1:    13,914,487 bp\n",
      "  Read 2:    13,919,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36735 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t328\t0.0\t2\t0 320 8\n",
      "26\t36086\t0.0\t2\t35198 873 15\n",
      "27\t225\t0.0\t2\t0 213 12\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t260\t0.0\t2\t0 246 14\n",
      "26\t36351\t0.0\t2\t35443 882 26\n",
      "27\t167\t0.0\t2\t0 162 5\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1568_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1568_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1568_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1568_105_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1568_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1568_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1568_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1568_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        33,181 reads  @    110.2 µs/read;   0.54 M reads/minute\n",
      "Finished in 3.66 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,181\n",
      "  Read 1 with adapter:                  16,248 (49.0%)\n",
      "  Read 2 with adapter:                  15,909 (47.9%)\n",
      "Pairs written (passing filters):        33,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,483,192 bp\n",
      "  Read 1:     9,781,807 bp\n",
      "  Read 2:     9,701,385 bp\n",
      "Total written (filtered):     18,645,536 bp (95.7%)\n",
      "  Read 1:     9,357,748 bp\n",
      "  Read 2:     9,287,788 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 16248 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t145\t0.0\t2\t0 143 2\n",
      "26\t15967\t0.0\t2\t15636 326 5\n",
      "27\t97\t0.0\t2\t0 96 1\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15909 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t122\t0.0\t2\t0 120 2\n",
      "26\t15709\t0.0\t2\t15231 446 32\n",
      "27\t71\t0.0\t2\t0 66 5\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1569_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1569_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1569_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1569_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1569_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1569_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1569_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1569_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,217 reads  @     98.5 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.06 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,217\n",
      "  Read 1 with adapter:                  29,769 (72.2%)\n",
      "  Read 2 with adapter:                  27,116 (65.8%)\n",
      "Pairs written (passing filters):        41,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,804,840 bp\n",
      "  Read 1:    12,150,076 bp\n",
      "  Read 2:    11,654,764 bp\n",
      "Total written (filtered):     22,325,320 bp (93.8%)\n",
      "  Read 1:    11,375,394 bp\n",
      "  Read 2:    10,949,926 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29769 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t242\t0.0\t2\t0 234 8\n",
      "26\t29232\t0.0\t2\t28574 647 11\n",
      "27\t186\t0.0\t2\t0 183 3\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27116 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t298\t0.0\t2\t0 288 10\n",
      "26\t26660\t0.0\t2\t25959 659 42\n",
      "27\t136\t0.0\t2\t0 132 4\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1570_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1570_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1570_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1570_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1570_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1570_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1570_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1570_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        38,831 reads  @     98.1 µs/read;   0.61 M reads/minute\n",
      "Finished in 3.82 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,831\n",
      "  Read 1 with adapter:                  15,058 (38.8%)\n",
      "  Read 2 with adapter:                  15,082 (38.8%)\n",
      "Pairs written (passing filters):        38,831 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,900,390 bp\n",
      "  Read 1:    11,449,822 bp\n",
      "  Read 2:    11,450,568 bp\n",
      "Total written (filtered):     22,116,804 bp (96.6%)\n",
      "  Read 1:    11,058,291 bp\n",
      "  Read 2:    11,058,513 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t132\t0.0\t2\t0 130 2\n",
      "26\t14790\t0.0\t2\t14450 335 5\n",
      "27\t93\t0.0\t2\t0 90 3\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15082 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t131\t0.0\t2\t0 127 4\n",
      "26\t14878\t0.0\t2\t14546 322 10\n",
      "27\t60\t0.0\t2\t0 57 3\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1571_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1571_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1571_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1571_111_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1571_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1571_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1571_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1571_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,102 reads  @     95.4 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.69 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,102\n",
      "  Read 1 with adapter:                  27,305 (55.6%)\n",
      "  Read 2 with adapter:                  27,305 (55.6%)\n",
      "Pairs written (passing filters):        49,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,956,404 bp\n",
      "  Read 1:    14,482,645 bp\n",
      "  Read 2:    14,473,759 bp\n",
      "Total written (filtered):     27,536,036 bp (95.1%)\n",
      "  Read 1:    13,772,128 bp\n",
      "  Read 2:    13,763,908 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t228\t0.0\t2\t0 225 3\n",
      "26\t26848\t0.0\t2\t26281 554 13\n",
      "27\t169\t0.0\t2\t0 163 6\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27305 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t213\t0.0\t2\t0 207 6\n",
      "26\t26954\t0.0\t2\t26316 598 40\n",
      "27\t98\t0.0\t2\t0 92 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1572_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1572_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1572_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1572_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1572_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1572_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1572_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1572_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,589 reads  @     86.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.02 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,589\n",
      "  Read 1 with adapter:                  15,653 (33.6%)\n",
      "  Read 2 with adapter:                  15,741 (33.8%)\n",
      "Pairs written (passing filters):        46,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,449,208 bp\n",
      "  Read 1:    13,729,292 bp\n",
      "  Read 2:    13,719,916 bp\n",
      "Total written (filtered):     26,632,964 bp (97.0%)\n",
      "  Read 1:    13,322,299 bp\n",
      "  Read 2:    13,310,665 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15653 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t128\t0.0\t2\t0 123 5\n",
      "26\t15392\t0.0\t2\t15096 289 7\n",
      "27\t95\t0.0\t2\t0 92 3\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15741 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t106\t0.0\t2\t0 104 2\n",
      "26\t15546\t0.0\t2\t15258 275 13\n",
      "27\t79\t0.0\t2\t0 75 4\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1573_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1573_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1573_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1573_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1573_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1573_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1573_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1573_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,538 reads  @    120.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 4.87 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,538\n",
      "  Read 1 with adapter:                  39,181 (96.7%)\n",
      "  Read 2 with adapter:                  38,948 (96.1%)\n",
      "Pairs written (passing filters):        40,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,910,867 bp\n",
      "  Read 1:    11,956,591 bp\n",
      "  Read 2:    11,954,276 bp\n",
      "Total written (filtered):     21,879,680 bp (91.5%)\n",
      "  Read 1:    10,937,801 bp\n",
      "  Read 2:    10,941,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t378\t0.0\t2\t0 364 14\n",
      "26\t38409\t0.0\t2\t37193 1177 39\n",
      "27\t274\t0.0\t2\t0 262 12\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t425\t0.0\t2\t0 409 16\n",
      "26\t38313\t0.0\t2\t37410 886 17\n",
      "27\t154\t0.0\t2\t0 147 7\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1574_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1574_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1574_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1574_117_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1574_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1574_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1574_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1574_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,522 reads  @    123.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.12 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,522\n",
      "  Read 1 with adapter:                  36,066 (86.9%)\n",
      "  Read 2 with adapter:                  35,970 (86.6%)\n",
      "Pairs written (passing filters):        41,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,481,671 bp\n",
      "  Read 1:    12,244,833 bp\n",
      "  Read 2:    12,236,838 bp\n",
      "Total written (filtered):     22,608,942 bp (92.4%)\n",
      "  Read 1:    11,307,166 bp\n",
      "  Read 2:    11,301,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36066 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t370\t0.0\t2\t0 363 7\n",
      "26\t35398\t0.0\t2\t34447 926 25\n",
      "27\t219\t0.0\t2\t0 213 6\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t314\t0.0\t2\t0 304 10\n",
      "26\t35492\t0.0\t2\t34629 837 26\n",
      "27\t132\t0.0\t2\t0 123 9\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1575_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1575_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1575_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1575_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1575_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1575_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1575_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1575_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        47,041 reads  @    109.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.13 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,041\n",
      "  Read 1 with adapter:                  44,250 (94.1%)\n",
      "  Read 2 with adapter:                  44,310 (94.2%)\n",
      "Pairs written (passing filters):        47,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,747,969 bp\n",
      "  Read 1:    13,872,518 bp\n",
      "  Read 2:    13,875,451 bp\n",
      "Total written (filtered):     25,445,199 bp (91.7%)\n",
      "  Read 1:    12,721,712 bp\n",
      "  Read 2:    12,723,487 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44250 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t411\t0.0\t2\t0 396 15\n",
      "26\t43469\t0.0\t2\t42381 1067 21\n",
      "27\t250\t0.0\t2\t0 238 12\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44310 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t301\t0.0\t2\t0 291 10\n",
      "26\t43771\t0.0\t2\t42781 956 34\n",
      "27\t207\t0.0\t2\t0 205 2\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1576_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1576_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1576_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1576_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1576_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1576_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1576_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1576_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        36,955 reads  @    123.4 µs/read;   0.49 M reads/minute\n",
      "Finished in 4.57 s (124 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,955\n",
      "  Read 1 with adapter:                  31,874 (86.3%)\n",
      "  Read 2 with adapter:                  31,513 (85.3%)\n",
      "Pairs written (passing filters):        36,952 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,727,943 bp\n",
      "  Read 1:    10,898,592 bp\n",
      "  Read 2:    10,829,351 bp\n",
      "Total written (filtered):     20,079,133 bp (92.4%)\n",
      "  Read 1:    10,069,077 bp\n",
      "  Read 2:    10,010,056 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t348\t0.0\t2\t0 339 9\n",
      "26\t31222\t0.0\t2\t30464 738 20\n",
      "27\t220\t0.0\t2\t0 197 23\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31513 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t228\t0.0\t2\t0 215 13\n",
      "26\t31090\t0.0\t2\t30095 917 78\n",
      "27\t149\t0.0\t2\t0 140 9\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1577_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1577_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1577_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1577_123_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1577_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1577_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1577_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1577_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        42,076 reads  @    121.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.13 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,076\n",
      "  Read 1 with adapter:                  40,773 (96.9%)\n",
      "  Read 2 with adapter:                  38,304 (91.0%)\n",
      "Pairs written (passing filters):        42,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,335,559 bp\n",
      "  Read 1:    12,407,741 bp\n",
      "  Read 2:    11,927,818 bp\n",
      "Total written (filtered):     22,279,488 bp (91.6%)\n",
      "  Read 1:    11,347,359 bp\n",
      "  Read 2:    10,932,129 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40773 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t407\t0.0\t2\t0 392 15\n",
      "26\t40019\t0.0\t2\t38999 990 30\n",
      "27\t242\t0.0\t2\t0 230 12\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t453\t0.0\t2\t0 434 19\n",
      "26\t37605\t0.0\t2\t36466 1073 66\n",
      "27\t210\t0.0\t2\t0 201 9\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1578_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1578_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1578_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1578_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1578_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1578_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1578_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1578_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        55,482 reads  @    105.3 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.84 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,482\n",
      "  Read 1 with adapter:                  54,053 (97.4%)\n",
      "  Read 2 with adapter:                  54,002 (97.3%)\n",
      "Pairs written (passing filters):        55,481 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,717,258 bp\n",
      "  Read 1:    16,364,092 bp\n",
      "  Read 2:    16,353,166 bp\n",
      "Total written (filtered):     29,907,743 bp (91.4%)\n",
      "  Read 1:    14,958,383 bp\n",
      "  Read 2:    14,949,360 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54053 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t497\t0.0\t2\t0 479 18\n",
      "26\t53074\t0.0\t2\t51674 1365 35\n",
      "27\t329\t0.0\t2\t0 316 13\n",
      "28\t134\t0.0\t2\t0 0 134\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54002 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t507\t0.0\t2\t0 482 25\n",
      "26\t53188\t0.0\t2\t51771 1358 59\n",
      "27\t245\t0.0\t2\t0 235 10\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1579_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1579_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1579_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1579_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1579_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1579_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1579_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1579_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        44,805 reads  @    102.4 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.59 s (103 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,805\n",
      "  Read 1 with adapter:                  43,970 (98.1%)\n",
      "  Read 2 with adapter:                  43,909 (98.0%)\n",
      "Pairs written (passing filters):        44,805 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,421,880 bp\n",
      "  Read 1:    13,212,493 bp\n",
      "  Read 2:    13,209,387 bp\n",
      "Total written (filtered):     24,137,160 bp (91.4%)\n",
      "  Read 1:    12,069,297 bp\n",
      "  Read 2:    12,067,863 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t407\t0.0\t2\t0 395 12\n",
      "26\t43200\t0.0\t2\t42288 895 17\n",
      "27\t259\t0.0\t2\t0 252 7\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43909 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t356\t0.0\t2\t0 343 13\n",
      "26\t43308\t0.0\t2\t42349 914 45\n",
      "27\t196\t0.0\t2\t0 193 3\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1580_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1580_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1580_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1580_129_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1580_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1580_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1580_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1580_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,520 reads  @    107.5 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.58 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,520\n",
      "  Read 1 with adapter:                  41,423 (97.4%)\n",
      "  Read 2 with adapter:                  41,518 (97.6%)\n",
      "Pairs written (passing filters):        42,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,077,950 bp\n",
      "  Read 1:    12,537,634 bp\n",
      "  Read 2:    12,540,316 bp\n",
      "Total written (filtered):     22,921,759 bp (91.4%)\n",
      "  Read 1:    11,460,733 bp\n",
      "  Read 2:    11,461,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t403\t0.0\t2\t0 386 17\n",
      "26\t40705\t0.0\t2\t39755 930 20\n",
      "27\t244\t0.0\t2\t0 232 12\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t378\t0.0\t2\t0 363 15\n",
      "26\t40910\t0.0\t2\t39895 974 41\n",
      "27\t192\t0.0\t2\t0 186 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1581_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1581_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1581_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1581_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1581_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1581_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1581_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1581_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            64 reads  @    523.1 µs/read;   0.11 M reads/minute\n",
      "Finished in 0.04 s (563 us/read; 0.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 64\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            64 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,674 bp\n",
      "  Read 1:         9,883 bp\n",
      "  Read 2:        17,791 bp\n",
      "Total written (filtered):         27,674 bp (100.0%)\n",
      "  Read 1:         9,883 bp\n",
      "  Read 2:        17,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1597_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1597_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1597_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1597_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1597_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1597_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1597_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1597_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        57,429 reads  @    114.4 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.57 s (114 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,429\n",
      "  Read 1 with adapter:                  57,238 (99.7%)\n",
      "  Read 2 with adapter:                  57,335 (99.8%)\n",
      "Pairs written (passing filters):        57,429 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,848,150 bp\n",
      "  Read 1:    16,917,867 bp\n",
      "  Read 2:    16,930,283 bp\n",
      "Total written (filtered):     30,869,727 bp (91.2%)\n",
      "  Read 1:    15,429,865 bp\n",
      "  Read 2:    15,439,862 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57238 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t681\t0.0\t2\t0 664 17\n",
      "26\t56074\t0.0\t2\t54657 1382 35\n",
      "27\t351\t0.0\t2\t0 338 13\n",
      "28\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57335 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t616\t0.0\t2\t0 596 20\n",
      "26\t56355\t0.0\t2\t54934 1379 42\n",
      "27\t301\t0.0\t2\t0 287 14\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1598_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1598_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1598_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1598_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1598_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1598_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1598_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1598_135_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        55,403 reads  @    115.7 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.41 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,403\n",
      "  Read 1 with adapter:                  55,285 (99.8%)\n",
      "  Read 2 with adapter:                  55,329 (99.9%)\n",
      "Pairs written (passing filters):        55,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,676,716 bp\n",
      "  Read 1:    16,336,575 bp\n",
      "  Read 2:    16,340,141 bp\n",
      "Total written (filtered):     29,801,214 bp (91.2%)\n",
      "  Read 1:    14,899,410 bp\n",
      "  Read 2:    14,901,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55285 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t644\t0.0\t2\t0 627 17\n",
      "26\t54218\t0.0\t2\t52860 1323 35\n",
      "27\t323\t0.0\t2\t0 315 8\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55329 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t463\t0.0\t2\t0 451 12\n",
      "26\t54573\t0.0\t2\t53120 1377 76\n",
      "27\t244\t0.0\t2\t0 231 13\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1599_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1599_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1599_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1599_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1599_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1599_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1599_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1599_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,131 reads  @    120.2 µs/read;   0.50 M reads/minute\n",
      "Finished in 5.79 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,131\n",
      "  Read 1 with adapter:                  48,066 (99.9%)\n",
      "  Read 2 with adapter:                  48,030 (99.8%)\n",
      "Pairs written (passing filters):        48,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,390,787 bp\n",
      "  Read 1:    14,196,675 bp\n",
      "  Read 2:    14,194,112 bp\n",
      "Total written (filtered):     25,892,530 bp (91.2%)\n",
      "  Read 1:    12,947,172 bp\n",
      "  Read 2:    12,945,358 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48066 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t547\t0.0\t2\t0 521 26\n",
      "26\t47172\t0.0\t2\t45964 1185 23\n",
      "27\t248\t0.0\t2\t0 242 6\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48030 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t328\t0.0\t2\t0 305 23\n",
      "26\t47372\t0.0\t2\t45869 1398 105\n",
      "27\t274\t0.0\t2\t0 258 16\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1600_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1600_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1600_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1600_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1600_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1600_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1600_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1600_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        47,566 reads  @    110.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 5.26 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,566\n",
      "  Read 1 with adapter:                  47,490 (99.8%)\n",
      "  Read 2 with adapter:                  47,509 (99.9%)\n",
      "Pairs written (passing filters):        47,566 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,056,811 bp\n",
      "  Read 1:    14,029,816 bp\n",
      "  Read 2:    14,026,995 bp\n",
      "Total written (filtered):     25,586,925 bp (91.2%)\n",
      "  Read 1:    12,795,004 bp\n",
      "  Read 2:    12,791,921 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t559\t0.0\t2\t0 539 20\n",
      "26\t46438\t0.0\t2\t44933 1465 40\n",
      "27\t259\t0.0\t2\t0 253 6\n",
      "28\t210\t0.0\t2\t0 0 210\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t446\t0.0\t2\t0 427 19\n",
      "26\t46777\t0.0\t2\t45333 1393 51\n",
      "27\t234\t0.0\t2\t0 228 6\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1601_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1601_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1601_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1601_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1601_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1601_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1601_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1601_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8=--------] 00:00:05        51,373 reads  @    107.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.50 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,373\n",
      "  Read 1 with adapter:                  51,280 (99.8%)\n",
      "  Read 2 with adapter:                  51,298 (99.9%)\n",
      "Pairs written (passing filters):        51,373 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,305,005 bp\n",
      "  Read 1:    15,154,399 bp\n",
      "  Read 2:    15,150,606 bp\n",
      "Total written (filtered):     27,638,426 bp (91.2%)\n",
      "  Read 1:    13,821,282 bp\n",
      "  Read 2:    13,817,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51280 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t647\t0.0\t2\t0 626 21\n",
      "26\t50137\t0.0\t2\t48804 1296 37\n",
      "27\t396\t0.0\t2\t0 391 5\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t542\t0.0\t2\t0 528 14\n",
      "26\t50422\t0.0\t2\t49139 1235 48\n",
      "27\t280\t0.0\t2\t0 270 10\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1602_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1602_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1602_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1602_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1602_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1602_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1602_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1602_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        58,162 reads  @    119.7 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.97 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,162\n",
      "  Read 1 with adapter:                  58,049 (99.8%)\n",
      "  Read 2 with adapter:                  58,051 (99.8%)\n",
      "Pairs written (passing filters):        58,162 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,305,812 bp\n",
      "  Read 1:    17,153,927 bp\n",
      "  Read 2:    17,151,885 bp\n",
      "Total written (filtered):     31,287,728 bp (91.2%)\n",
      "  Read 1:    15,644,977 bp\n",
      "  Read 2:    15,642,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58049 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t763\t0.0\t2\t0 740 23\n",
      "26\t56801\t0.0\t2\t55103 1667 31\n",
      "27\t371\t0.0\t2\t0 351 20\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t465\t0.0\t2\t0 448 17\n",
      "26\t57258\t0.0\t2\t55609 1583 66\n",
      "27\t275\t0.0\t2\t0 260 15\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1603_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1603_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1603_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1603_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1603_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1603_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1603_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1603_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        63,189 reads  @    110.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 6.99 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,189\n",
      "  Read 1 with adapter:                  62,993 (99.7%)\n",
      "  Read 2 with adapter:                  63,112 (99.9%)\n",
      "Pairs written (passing filters):        63,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,278,312 bp\n",
      "  Read 1:    18,639,987 bp\n",
      "  Read 2:    18,638,325 bp\n",
      "Total written (filtered):     33,999,871 bp (91.2%)\n",
      "  Read 1:    17,002,424 bp\n",
      "  Read 2:    16,997,447 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62993 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t845\t0.0\t2\t0 818 27\n",
      "26\t61566\t0.0\t2\t59828 1695 43\n",
      "27\t354\t0.0\t2\t0 339 15\n",
      "28\t173\t0.0\t2\t0 0 173\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t492\t0.0\t2\t0 473 19\n",
      "26\t62135\t0.0\t2\t60416 1653 66\n",
      "27\t400\t0.0\t2\t0 389 11\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1604_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1604_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1604_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1604_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1604_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1604_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1604_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1604_147_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        52,480 reads  @    117.7 µs/read;   0.51 M reads/minute\n",
      "Finished in 6.18 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,480\n",
      "  Read 1 with adapter:                  52,400 (99.8%)\n",
      "  Read 2 with adapter:                  52,414 (99.9%)\n",
      "Pairs written (passing filters):        52,480 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,960,882 bp\n",
      "  Read 1:    15,481,107 bp\n",
      "  Read 2:    15,479,775 bp\n",
      "Total written (filtered):     28,235,898 bp (91.2%)\n",
      "  Read 1:    14,118,824 bp\n",
      "  Read 2:    14,117,074 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t616\t0.0\t2\t0 596 20\n",
      "26\t51284\t0.0\t2\t49770 1481 33\n",
      "27\t357\t0.0\t2\t0 344 13\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52414 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t405\t0.0\t2\t0 394 11\n",
      "26\t51675\t0.0\t2\t50030 1559 86\n",
      "27\t250\t0.0\t2\t0 238 12\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1605_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1605_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1605_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1605_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1605_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1605_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1605_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1605_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        44,532 reads  @    116.0 µs/read;   0.52 M reads/minute\n",
      "Finished in 5.17 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,532\n",
      "  Read 1 with adapter:                  44,432 (99.8%)\n",
      "  Read 2 with adapter:                  44,472 (99.9%)\n",
      "Pairs written (passing filters):        44,532 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,268,753 bp\n",
      "  Read 1:    13,133,197 bp\n",
      "  Read 2:    13,135,556 bp\n",
      "Total written (filtered):     23,957,679 bp (91.2%)\n",
      "  Read 1:    11,978,204 bp\n",
      "  Read 2:    11,979,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t472\t0.0\t2\t0 457 15\n",
      "26\t43710\t0.0\t2\t42753 943 14\n",
      "27\t199\t0.0\t2\t0 190 9\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44472 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t434\t0.0\t2\t0 415 19\n",
      "26\t43762\t0.0\t2\t42720 985 57\n",
      "27\t221\t0.0\t2\t0 205 16\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1606_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1606_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1606_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1606_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1606_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1606_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1606_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1606_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,588 reads  @    112.5 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.57 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,588\n",
      "  Read 1 with adapter:                  40,511 (99.8%)\n",
      "  Read 2 with adapter:                  40,523 (99.8%)\n",
      "Pairs written (passing filters):        40,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,941,825 bp\n",
      "  Read 1:    11,971,036 bp\n",
      "  Read 2:    11,970,789 bp\n",
      "Total written (filtered):     21,835,104 bp (91.2%)\n",
      "  Read 1:    10,917,773 bp\n",
      "  Read 2:    10,917,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t394\t0.0\t2\t0 381 13\n",
      "26\t39787\t0.0\t2\t38763 1006 18\n",
      "27\t209\t0.0\t2\t0 203 6\n",
      "28\t101\t0.0\t2\t0 0 101\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40523 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t326\t0.0\t2\t0 316 10\n",
      "26\t39983\t0.0\t2\t38945 979 59\n",
      "27\t166\t0.0\t2\t0 163 3\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1607_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1607_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1607_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1607_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1607_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1607_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1607_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1607_153_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        62,925 reads  @    110.1 µs/read;   0.54 M reads/minute\n",
      "Finished in 6.93 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,925\n",
      "  Read 1 with adapter:                  62,588 (99.5%)\n",
      "  Read 2 with adapter:                  62,801 (99.8%)\n",
      "Pairs written (passing filters):        62,925 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,108,227 bp\n",
      "  Read 1:    18,551,584 bp\n",
      "  Read 2:    18,556,643 bp\n",
      "Total written (filtered):     33,848,294 bp (91.2%)\n",
      "  Read 1:    16,924,396 bp\n",
      "  Read 2:    16,923,898 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62588 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t601\t0.0\t2\t0 585 16\n",
      "26\t61520\t0.0\t2\t59983 1509 28\n",
      "27\t329\t0.0\t2\t0 314 15\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62801 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t471\t0.0\t2\t0 454 17\n",
      "26\t61934\t0.0\t2\t60363 1513 58\n",
      "27\t326\t0.0\t2\t0 310 16\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1608_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1608_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1608_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1608_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1608_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1608_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1608_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1608_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        44,580 reads  @    103.4 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.61 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,580\n",
      "  Read 1 with adapter:                  44,296 (99.4%)\n",
      "  Read 2 with adapter:                  44,526 (99.9%)\n",
      "Pairs written (passing filters):        44,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,288,829 bp\n",
      "  Read 1:    13,138,590 bp\n",
      "  Read 2:    13,150,239 bp\n",
      "Total written (filtered):     23,979,697 bp (91.2%)\n",
      "  Read 1:    11,987,020 bp\n",
      "  Read 2:    11,992,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t403\t0.0\t2\t0 390 13\n",
      "26\t43599\t0.0\t2\t42525 1050 24\n",
      "27\t223\t0.0\t2\t0 211 12\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44526 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t344\t0.0\t2\t0 337 7\n",
      "26\t43898\t0.0\t2\t42835 1023 40\n",
      "27\t230\t0.0\t2\t0 220 10\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1609_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1609_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1609_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1609_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1609_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1609_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1609_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1609_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        44,102 reads  @    125.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.55 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,102\n",
      "  Read 1 with adapter:                  44,027 (99.8%)\n",
      "  Read 2 with adapter:                  44,019 (99.8%)\n",
      "Pairs written (passing filters):        44,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,004,225 bp\n",
      "  Read 1:    13,006,746 bp\n",
      "  Read 2:    12,997,479 bp\n",
      "Total written (filtered):     23,715,502 bp (91.2%)\n",
      "  Read 1:    11,862,297 bp\n",
      "  Read 2:    11,853,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44027 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t547\t0.0\t2\t0 533 14\n",
      "26\t43187\t0.0\t2\t42076 1078 33\n",
      "27\t208\t0.0\t2\t0 197 11\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44019 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t491\t0.0\t2\t0 478 13\n",
      "26\t43236\t0.0\t2\t42081 1098 57\n",
      "27\t229\t0.0\t2\t0 225 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1610_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1610_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1610_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1610_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1610_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1610_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1610_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1610_159_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,646 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.72 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,646\n",
      "  Read 1 with adapter:                  45,574 (99.8%)\n",
      "  Read 2 with adapter:                  45,569 (99.8%)\n",
      "Pairs written (passing filters):        45,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,923,613 bp\n",
      "  Read 1:    13,460,566 bp\n",
      "  Read 2:    13,463,047 bp\n",
      "Total written (filtered):     24,554,099 bp (91.2%)\n",
      "  Read 1:    12,275,753 bp\n",
      "  Read 2:    12,278,346 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45574 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t493\t0.0\t2\t0 474 19\n",
      "26\t44736\t0.0\t2\t43741 972 23\n",
      "27\t248\t0.0\t2\t0 244 4\n",
      "28\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t359\t0.0\t2\t0 350 9\n",
      "26\t44965\t0.0\t2\t43757 1148 60\n",
      "27\t180\t0.0\t2\t0 170 10\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1611_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1611_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1611_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1611_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1611_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1611_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1611_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1611_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        38,875 reads  @    113.8 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.43 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,875\n",
      "  Read 1 with adapter:                  38,801 (99.8%)\n",
      "  Read 2 with adapter:                  38,825 (99.9%)\n",
      "Pairs written (passing filters):        38,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,930,485 bp\n",
      "  Read 1:    11,464,533 bp\n",
      "  Read 2:    11,465,952 bp\n",
      "Total written (filtered):     20,912,475 bp (91.2%)\n",
      "  Read 1:    10,455,932 bp\n",
      "  Read 2:    10,456,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38801 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t487\t0.0\t2\t0 464 23\n",
      "26\t38030\t0.0\t2\t37021 978 31\n",
      "27\t206\t0.0\t2\t0 199 7\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t278\t0.0\t2\t0 269 9\n",
      "26\t38273\t0.0\t2\t37176 1046 51\n",
      "27\t251\t0.0\t2\t0 241 10\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1612_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1612_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1612_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1612_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1612_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1612_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1612_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1612_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        45,836 reads  @    146.0 µs/read;   0.41 M reads/minute\n",
      "Finished in 6.69 s (146 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,836\n",
      "  Read 1 with adapter:                  45,764 (99.8%)\n",
      "  Read 2 with adapter:                  45,785 (99.9%)\n",
      "Pairs written (passing filters):        45,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,039,178 bp\n",
      "  Read 1:    13,518,258 bp\n",
      "  Read 2:    13,520,920 bp\n",
      "Total written (filtered):     24,659,125 bp (91.2%)\n",
      "  Read 1:    12,328,608 bp\n",
      "  Read 2:    12,330,517 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45764 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t565\t0.0\t2\t0 550 15\n",
      "26\t44851\t0.0\t2\t43630 1196 25\n",
      "27\t213\t0.0\t2\t0 208 5\n",
      "28\t102\t0.0\t2\t0 0 102\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t352\t0.0\t2\t0 333 19\n",
      "26\t45094\t0.0\t2\t43789 1242 63\n",
      "27\t281\t0.0\t2\t0 264 17\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1613_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1613_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1613_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1613_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1613_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1613_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1613_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1613_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[   8<-------] 00:00:06        61,715 reads  @    109.9 µs/read;   0.55 M reads/minute\n",
      "Finished in 6.79 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,715\n",
      "  Read 1 with adapter:                  61,630 (99.9%)\n",
      "  Read 2 with adapter:                  61,621 (99.8%)\n",
      "Pairs written (passing filters):        61,715 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,405,384 bp\n",
      "  Read 1:    18,203,273 bp\n",
      "  Read 2:    18,202,111 bp\n",
      "Total written (filtered):     33,201,307 bp (91.2%)\n",
      "  Read 1:    16,601,102 bp\n",
      "  Read 2:    16,600,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t770\t0.0\t2\t0 759 11\n",
      "26\t60296\t0.0\t2\t58745 1516 35\n",
      "27\t431\t0.0\t2\t0 419 12\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61621 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t635\t0.0\t2\t0 605 30\n",
      "26\t60569\t0.0\t2\t58719 1733 117\n",
      "27\t283\t0.0\t2\t0 270 13\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1614_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1614_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1614_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1614_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1614_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1614_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1614_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1614_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        59,654 reads  @    112.9 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.74 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,654\n",
      "  Read 1 with adapter:                  59,565 (99.9%)\n",
      "  Read 2 with adapter:                  59,530 (99.8%)\n",
      "Pairs written (passing filters):        59,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,188,774 bp\n",
      "  Read 1:    17,597,410 bp\n",
      "  Read 2:    17,591,364 bp\n",
      "Total written (filtered):     32,092,791 bp (91.2%)\n",
      "  Read 1:    16,049,077 bp\n",
      "  Read 2:    16,043,714 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59565 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t787\t0.0\t2\t0 760 27\n",
      "26\t58323\t0.0\t2\t56828 1463 32\n",
      "27\t336\t0.0\t2\t0 330 6\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59530 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t442\t0.0\t2\t0 430 12\n",
      "26\t58732\t0.0\t2\t56967 1671 94\n",
      "27\t308\t0.0\t2\t0 294 14\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1615_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1615_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1615_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1615_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1615_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1615_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1615_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1615_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        67,253 reads  @    122.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 8.27 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,253\n",
      "  Read 1 with adapter:                  67,144 (99.8%)\n",
      "  Read 2 with adapter:                  67,125 (99.8%)\n",
      "Pairs written (passing filters):        67,253 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,671,102 bp\n",
      "  Read 1:    19,837,862 bp\n",
      "  Read 2:    19,833,240 bp\n",
      "Total written (filtered):     36,180,487 bp (91.2%)\n",
      "  Read 1:    18,092,464 bp\n",
      "  Read 2:    18,088,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67144 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t964\t0.0\t2\t0 933 31\n",
      "26\t65593\t0.0\t2\t63845 1720 28\n",
      "27\t392\t0.0\t2\t0 385 7\n",
      "28\t154\t0.0\t2\t0 0 154\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t499\t0.0\t2\t0 462 37\n",
      "26\t66128\t0.0\t2\t64053 1960 115\n",
      "27\t418\t0.0\t2\t0 395 23\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1616_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1616_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1616_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1616_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1616_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1616_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1616_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1616_171_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        60,594 reads  @    119.5 µs/read;   0.50 M reads/minute\n",
      "Finished in 7.25 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,594\n",
      "  Read 1 with adapter:                  60,509 (99.9%)\n",
      "  Read 2 with adapter:                  60,510 (99.9%)\n",
      "Pairs written (passing filters):        60,594 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,747,327 bp\n",
      "  Read 1:    17,872,708 bp\n",
      "  Read 2:    17,874,619 bp\n",
      "Total written (filtered):     32,601,272 bp (91.2%)\n",
      "  Read 1:    16,299,813 bp\n",
      "  Read 2:    16,301,459 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60509 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t785\t0.0\t2\t0 756 29\n",
      "26\t59242\t0.0\t2\t57632 1572 38\n",
      "27\t330\t0.0\t2\t0 320 10\n",
      "28\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t463\t0.0\t2\t0 435 28\n",
      "26\t59643\t0.0\t2\t57805 1757 81\n",
      "27\t353\t0.0\t2\t0 326 27\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1617_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1617_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1617_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1617_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1617_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1617_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1617_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1617_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        56,314 reads  @    102.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.79 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,314\n",
      "  Read 1 with adapter:                  56,230 (99.9%)\n",
      "  Read 2 with adapter:                  56,206 (99.8%)\n",
      "Pairs written (passing filters):        56,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,220,410 bp\n",
      "  Read 1:    16,612,184 bp\n",
      "  Read 2:    16,608,226 bp\n",
      "Total written (filtered):     30,297,271 bp (91.2%)\n",
      "  Read 1:    15,150,092 bp\n",
      "  Read 2:    15,147,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t537\t0.0\t2\t0 514 23\n",
      "26\t55152\t0.0\t2\t53766 1355 31\n",
      "27\t333\t0.0\t2\t0 323 10\n",
      "28\t183\t0.0\t2\t0 0 183\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t607\t0.0\t2\t0 580 27\n",
      "26\t55236\t0.0\t2\t53539 1582 115\n",
      "27\t308\t0.0\t2\t0 289 19\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1618_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1618_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1618_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1618_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1618_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1618_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1618_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1618_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        63,109 reads  @    118.2 µs/read;   0.51 M reads/minute\n",
      "Finished in 7.46 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,109\n",
      "  Read 1 with adapter:                  63,019 (99.9%)\n",
      "  Read 2 with adapter:                  63,011 (99.8%)\n",
      "Pairs written (passing filters):        63,109 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,229,573 bp\n",
      "  Read 1:    18,616,032 bp\n",
      "  Read 2:    18,613,541 bp\n",
      "Total written (filtered):     33,953,014 bp (91.2%)\n",
      "  Read 1:    16,977,613 bp\n",
      "  Read 2:    16,975,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63019 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t629\t0.0\t2\t0 604 25\n",
      "26\t61918\t0.0\t2\t60358 1521 39\n",
      "27\t290\t0.0\t2\t0 283 7\n",
      "28\t157\t0.0\t2\t0 0 157\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t477\t0.0\t2\t0 448 29\n",
      "26\t62175\t0.0\t2\t60298 1773 104\n",
      "27\t311\t0.0\t2\t0 298 13\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1619_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1619_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1619_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1619_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1619_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1619_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1619_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1619_177_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:07        70,651 reads  @    108.6 µs/read;   0.55 M reads/minute\n",
      "Finished in 7.68 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,651\n",
      "  Read 1 with adapter:                  70,534 (99.8%)\n",
      "  Read 2 with adapter:                  70,474 (99.7%)\n",
      "Pairs written (passing filters):        70,650 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,670,565 bp\n",
      "  Read 1:    20,839,238 bp\n",
      "  Read 2:    20,831,327 bp\n",
      "Total written (filtered):     38,004,167 bp (91.2%)\n",
      "  Read 1:    19,005,176 bp\n",
      "  Read 2:    18,998,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 70534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t704\t0.0\t2\t0 686 18\n",
      "26\t69273\t0.0\t2\t67508 1727 38\n",
      "27\t373\t0.0\t2\t0 351 22\n",
      "28\t152\t0.0\t2\t0 0 152\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 70474 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t484\t0.0\t2\t0 452 32\n",
      "26\t69496\t0.0\t2\t67073 2237 186\n",
      "27\t408\t0.0\t2\t0 388 20\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1620_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1620_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1620_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1620_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1620_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1620_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1620_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1620_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        52,296 reads  @    101.8 µs/read;   0.59 M reads/minute\n",
      "Finished in 5.32 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,296\n",
      "  Read 1 with adapter:                  52,228 (99.9%)\n",
      "  Read 2 with adapter:                  52,219 (99.9%)\n",
      "Pairs written (passing filters):        52,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,849,385 bp\n",
      "  Read 1:    15,427,053 bp\n",
      "  Read 2:    15,422,332 bp\n",
      "Total written (filtered):     28,134,107 bp (91.2%)\n",
      "  Read 1:    14,069,361 bp\n",
      "  Read 2:    14,064,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t601\t0.0\t2\t0 581 20\n",
      "26\t51265\t0.0\t2\t49998 1242 25\n",
      "27\t239\t0.0\t2\t0 231 8\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52219 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t408\t0.0\t2\t0 394 14\n",
      "26\t51463\t0.0\t2\t50074 1310 79\n",
      "27\t296\t0.0\t2\t0 287 9\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1621_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1621_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1621_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1621_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1621_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1621_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1621_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1621_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        60,076 reads  @    113.8 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.84 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,076\n",
      "  Read 1 with adapter:                  59,703 (99.4%)\n",
      "  Read 2 with adapter:                  59,980 (99.8%)\n",
      "Pairs written (passing filters):        60,076 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,364,237 bp\n",
      "  Read 1:    17,645,990 bp\n",
      "  Read 2:    17,718,247 bp\n",
      "Total written (filtered):     32,252,917 bp (91.2%)\n",
      "  Read 1:    16,093,774 bp\n",
      "  Read 2:    16,159,143 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t667\t0.0\t2\t0 654 13\n",
      "26\t58496\t0.0\t2\t57079 1384 33\n",
      "27\t399\t0.0\t2\t0 387 12\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t641\t0.0\t2\t0 619 22\n",
      "26\t58997\t0.0\t2\t57694 1259 44\n",
      "27\t259\t0.0\t2\t0 252 7\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1622_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1622_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1622_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1622_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1622_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1622_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1622_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1622_183_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        58,083 reads  @    119.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.92 s (119 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,083\n",
      "  Read 1 with adapter:                  57,768 (99.5%)\n",
      "  Read 2 with adapter:                  57,997 (99.9%)\n",
      "Pairs written (passing filters):        58,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,212,043 bp\n",
      "  Read 1:    17,080,301 bp\n",
      "  Read 2:    17,131,742 bp\n",
      "Total written (filtered):     31,202,405 bp (91.2%)\n",
      "  Read 1:    15,578,445 bp\n",
      "  Read 2:    15,623,960 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t618\t0.0\t2\t0 604 14\n",
      "26\t56663\t0.0\t2\t55275 1352 36\n",
      "27\t376\t0.0\t2\t0 359 17\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57997 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t427\t0.0\t2\t0 414 13\n",
      "26\t57248\t0.0\t2\t55862 1325 61\n",
      "27\t261\t0.0\t2\t0 247 14\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1623_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1623_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1623_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1623_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1623_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1623_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1623_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1623_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        58,076 reads  @    107.3 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.23 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,076\n",
      "  Read 1 with adapter:                  57,692 (99.3%)\n",
      "  Read 2 with adapter:                  58,008 (99.9%)\n",
      "Pairs written (passing filters):        58,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,185,446 bp\n",
      "  Read 1:    17,055,574 bp\n",
      "  Read 2:    17,129,872 bp\n",
      "Total written (filtered):     31,177,157 bp (91.2%)\n",
      "  Read 1:    15,555,374 bp\n",
      "  Read 2:    15,621,783 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t602\t0.0\t2\t0 581 21\n",
      "26\t56583\t0.0\t2\t55175 1376 32\n",
      "27\t365\t0.0\t2\t0 349 16\n",
      "28\t115\t0.0\t2\t0 0 115\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58008 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t421\t0.0\t2\t0 415 6\n",
      "26\t57233\t0.0\t2\t55909 1270 54\n",
      "27\t310\t0.0\t2\t0 301 9\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1624_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1624_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1624_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1624_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1624_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1624_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1624_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1624_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        62,305 reads  @    121.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 7.56 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,305\n",
      "  Read 1 with adapter:                  61,928 (99.4%)\n",
      "  Read 2 with adapter:                  62,225 (99.9%)\n",
      "Pairs written (passing filters):        62,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,684,662 bp\n",
      "  Read 1:    18,310,044 bp\n",
      "  Read 2:    18,374,618 bp\n",
      "Total written (filtered):     33,456,928 bp (91.2%)\n",
      "  Read 1:    16,700,037 bp\n",
      "  Read 2:    16,756,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t656\t0.0\t2\t0 636 20\n",
      "26\t60741\t0.0\t2\t58987 1711 43\n",
      "27\t367\t0.0\t2\t0 345 22\n",
      "28\t124\t0.0\t2\t0 0 124\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t523\t0.0\t2\t0 505 18\n",
      "26\t61278\t0.0\t2\t59565 1647 66\n",
      "27\t332\t0.0\t2\t0 319 13\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1625_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1625_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1625_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1625_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1625_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1625_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1625_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1625_189_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        51,803 reads  @    125.2 µs/read;   0.48 M reads/minute\n",
      "Finished in 6.49 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,803\n",
      "  Read 1 with adapter:                  51,708 (99.8%)\n",
      "  Read 2 with adapter:                  51,738 (99.9%)\n",
      "Pairs written (passing filters):        51,803 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,554,512 bp\n",
      "  Read 1:    15,276,848 bp\n",
      "  Read 2:    15,277,664 bp\n",
      "Total written (filtered):     27,865,173 bp (91.2%)\n",
      "  Read 1:    13,932,465 bp\n",
      "  Read 2:    13,932,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51708 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t572\t0.0\t2\t0 559 13\n",
      "26\t50667\t0.0\t2\t49289 1340 38\n",
      "27\t335\t0.0\t2\t0 321 14\n",
      "28\t120\t0.0\t2\t0 0 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t498\t0.0\t2\t0 482 16\n",
      "26\t50930\t0.0\t2\t49779 1103 48\n",
      "27\t246\t0.0\t2\t0 236 10\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1626_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1626_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1626_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1626_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1626_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1626_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1626_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1626_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        68,863 reads  @    101.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 7.00 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,863\n",
      "  Read 1 with adapter:                  68,771 (99.9%)\n",
      "  Read 2 with adapter:                  68,763 (99.9%)\n",
      "Pairs written (passing filters):        68,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,623,468 bp\n",
      "  Read 1:    20,312,664 bp\n",
      "  Read 2:    20,310,804 bp\n",
      "Total written (filtered):     37,047,711 bp (91.2%)\n",
      "  Read 1:    18,524,550 bp\n",
      "  Read 2:    18,523,161 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t717\t0.0\t2\t0 694 23\n",
      "26\t67374\t0.0\t2\t65769 1574 31\n",
      "27\t443\t0.0\t2\t0 430 13\n",
      "28\t204\t0.0\t2\t0 0 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t539\t0.0\t2\t0 525 14\n",
      "26\t67821\t0.0\t2\t66277 1489 55\n",
      "27\t346\t0.0\t2\t0 341 5\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1627_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1627_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1627_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1627_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1627_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1627_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1627_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1627_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:07        72,003 reads  @    109.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 7.87 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,003\n",
      "  Read 1 with adapter:                  71,884 (99.8%)\n",
      "  Read 2 with adapter:                  71,917 (99.9%)\n",
      "Pairs written (passing filters):        72,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,475,298 bp\n",
      "  Read 1:    21,236,771 bp\n",
      "  Read 2:    21,238,527 bp\n",
      "Total written (filtered):     38,736,611 bp (91.2%)\n",
      "  Read 1:    19,367,893 bp\n",
      "  Read 2:    19,368,718 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 71884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t824\t0.0\t2\t0 790 34\n",
      "26\t70364\t0.0\t2\t68587 1726 51\n",
      "27\t502\t0.0\t2\t0 489 13\n",
      "28\t151\t0.0\t2\t0 0 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 71917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t536\t0.0\t2\t0 515 21\n",
      "26\t70841\t0.0\t2\t69157 1624 60\n",
      "27\t433\t0.0\t2\t0 417 16\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1628_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1628_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1628_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1628_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1628_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1628_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1628_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1628_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:05        52,516 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 5.10 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,516\n",
      "  Read 1 with adapter:                  52,417 (99.8%)\n",
      "  Read 2 with adapter:                  52,476 (99.9%)\n",
      "Pairs written (passing filters):        52,516 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,976,958 bp\n",
      "  Read 1:    15,485,486 bp\n",
      "  Read 2:    15,491,472 bp\n",
      "Total written (filtered):     28,249,948 bp (91.2%)\n",
      "  Read 1:    14,122,764 bp\n",
      "  Read 2:    14,127,184 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52417 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t591\t0.0\t2\t0 575 16\n",
      "26\t51349\t0.0\t2\t49972 1338 39\n",
      "27\t371\t0.0\t2\t0 359 12\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52476 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t430\t0.0\t2\t0 416 14\n",
      "26\t51680\t0.0\t2\t50391 1247 42\n",
      "27\t310\t0.0\t2\t0 300 10\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1629_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1629_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1629_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1629_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1629_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-5fbz7xe_/1629_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1629_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-167js7nc/31bc23a2-c704-47b3-9813-ea24c2de8df9/data/1629_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            41 reads  @    734.4 µs/read;   0.08 M reads/minute\n",
      "Finished in 0.03 s (792 us/read; 0.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 41\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            41 (100.0%)\n",
      "\n",
      "Total basepairs processed:         8,991 bp\n",
      "  Read 1:         5,325 bp\n",
      "  Read 2:         3,666 bp\n",
      "Total written (filtered):          8,991 bp (100.0%)\n",
      "  Read 1:         5,325 bp\n",
      "  Read 2:         3,666 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: f-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: f-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1zpr1jvn')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b642d41b-fa9e-4796-9633-7a63f755fb88>"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.f import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/Hirai-Lib2-manifest_COI.csv \\\n",
    "  --output-path f-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.f trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences f-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences f-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.f summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data e-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization f-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('e-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: f-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: f-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: f-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.f Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs f-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 267 \\\n",
    "  --p-trunc-len-r 222 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table f-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats f-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: f-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-4sg_tylu')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 274bcef0-a1fe-4205-bf7f-a471bf35fa99>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.f view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table f-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization f-table.qzv \n",
    "q2.Visualization.load('f-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: f-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.f view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization f-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P2107-Lib1-manifest_COI.csv as PairedEndFastqManifestPhred33 to g-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1801_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1801_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1801_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1801_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1801_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1801_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1801_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1801_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        10,357 reads  @    169.0 µs/read;   0.35 M reads/minute\n",
      "Finished in 1.75 s (169 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,357\n",
      "  Read 1 with adapter:                  10,346 (99.9%)\n",
      "  Read 2 with adapter:                  10,286 (99.3%)\n",
      "Pairs written (passing filters):        10,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,109,254 bp\n",
      "  Read 1:     3,054,853 bp\n",
      "  Read 2:     3,054,401 bp\n",
      "Total written (filtered):      5,572,990 bp (91.2%)\n",
      "  Read 1:     2,785,944 bp\n",
      "  Read 2:     2,787,046 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 10346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t127\t0.0\t2\t0 116 11\n",
      "26\t10169\t0.0\t2\t9880 279 10\n",
      "27\t36\t0.0\t2\t0 34 2\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 10286 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t118\t0.0\t2\t0 113 5\n",
      "26\t10120\t0.0\t2\t9935 178 7\n",
      "27\t39\t0.0\t2\t0 38 1\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1802_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1802_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1802_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1802_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1802_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1802_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1802_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1802_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        29,804 reads  @    127.4 µs/read;   0.47 M reads/minute\n",
      "Finished in 3.80 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,804\n",
      "  Read 1 with adapter:                  29,774 (99.9%)\n",
      "  Read 2 with adapter:                  29,740 (99.8%)\n",
      "Pairs written (passing filters):        29,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,583,986 bp\n",
      "  Read 1:     8,792,006 bp\n",
      "  Read 2:     8,791,980 bp\n",
      "Total written (filtered):     16,036,848 bp (91.2%)\n",
      "  Read 1:     8,018,026 bp\n",
      "  Read 2:     8,018,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29774 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t317\t0.0\t2\t0 311 6\n",
      "26\t29275\t0.0\t2\t28708 561 6\n",
      "27\t135\t0.0\t2\t0 124 11\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t230\t0.0\t2\t0 223 7\n",
      "26\t29362\t0.0\t2\t28773 561 28\n",
      "27\t116\t0.0\t2\t0 107 9\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1803_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1803_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1803_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1803_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1803_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1803_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1803_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1803_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        30,947 reads  @    126.3 µs/read;   0.48 M reads/minute\n",
      "Finished in 3.91 s (126 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,947\n",
      "  Read 1 with adapter:                  30,917 (99.9%)\n",
      "  Read 2 with adapter:                  30,879 (99.8%)\n",
      "Pairs written (passing filters):        30,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,258,010 bp\n",
      "  Read 1:     9,128,840 bp\n",
      "  Read 2:     9,129,170 bp\n",
      "Total written (filtered):     16,651,666 bp (91.2%)\n",
      "  Read 1:     8,325,275 bp\n",
      "  Read 2:     8,326,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t382\t0.0\t2\t0 367 15\n",
      "26\t30383\t0.0\t2\t29777 589 17\n",
      "27\t127\t0.0\t2\t0 120 7\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30879 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t230\t0.0\t2\t0 220 10\n",
      "26\t30493\t0.0\t2\t29926 537 30\n",
      "27\t97\t0.0\t2\t0 95 2\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1804_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1804_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1804_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1804_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1804_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1804_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1804_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1804_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,920 reads  @    166.7 µs/read;   0.36 M reads/minute\n",
      "Finished in 3.82 s (167 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,920\n",
      "  Read 1 with adapter:                  22,890 (99.9%)\n",
      "  Read 2 with adapter:                  22,600 (98.6%)\n",
      "Pairs written (passing filters):        22,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,464,285 bp\n",
      "  Read 1:     6,759,803 bp\n",
      "  Read 2:     6,704,482 bp\n",
      "Total written (filtered):     12,279,586 bp (91.2%)\n",
      "  Read 1:     6,162,654 bp\n",
      "  Read 2:     6,116,932 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22890 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t260\t0.0\t2\t0 255 5\n",
      "26\t22498\t0.0\t2\t22087 404 7\n",
      "27\t92\t0.0\t2\t0 84 8\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t158\t0.0\t2\t0 154 4\n",
      "26\t22322\t0.0\t2\t21800 488 34\n",
      "27\t96\t0.0\t2\t0 94 2\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1805_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1805_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1805_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1805_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1805_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1805_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1805_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1805_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             8 reads  @   4796.0 µs/read;   0.01 M reads/minute\n",
      "Finished in 0.04 s (5302 us/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  8\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             8 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,979 bp\n",
      "  Read 1:         1,435 bp\n",
      "  Read 2:           544 bp\n",
      "Total written (filtered):          1,979 bp (100.0%)\n",
      "  Read 1:         1,435 bp\n",
      "  Read 2:           544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1806_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1806_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1806_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1806_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1806_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1806_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1806_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1806_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        34,837 reads  @    152.5 µs/read;   0.39 M reads/minute\n",
      "Finished in 5.32 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,837\n",
      "  Read 1 with adapter:                  34,807 (99.9%)\n",
      "  Read 2 with adapter:                  34,592 (99.3%)\n",
      "Pairs written (passing filters):        34,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,529,062 bp\n",
      "  Read 1:    10,275,842 bp\n",
      "  Read 2:    10,253,220 bp\n",
      "Total written (filtered):     18,724,908 bp (91.2%)\n",
      "  Read 1:     9,370,941 bp\n",
      "  Read 2:     9,353,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t382\t0.0\t2\t0 370 12\n",
      "26\t34140\t0.0\t2\t33549 576 15\n",
      "27\t205\t0.0\t2\t0 196 9\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34592 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t323\t0.0\t2\t0 311 12\n",
      "26\t34076\t0.0\t2\t33398 644 34\n",
      "27\t142\t0.0\t2\t0 133 9\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1807_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1807_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1807_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1807_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1807_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1807_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1807_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1807_13_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01         8,479 reads  @    146.7 µs/read;   0.41 M reads/minute\n",
      "Finished in 1.25 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,479\n",
      "  Read 1 with adapter:                   8,320 (98.1%)\n",
      "  Read 2 with adapter:                   8,303 (97.9%)\n",
      "Pairs written (passing filters):         8,479 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,941,360 bp\n",
      "  Read 1:     2,469,038 bp\n",
      "  Read 2:     2,472,322 bp\n",
      "Total written (filtered):      4,509,202 bp (91.3%)\n",
      "  Read 1:     2,252,735 bp\n",
      "  Read 2:     2,256,467 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 8320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t82\t0.0\t2\t0 78 4\n",
      "26\t8166\t0.0\t2\t8004 156 6\n",
      "27\t55\t0.0\t2\t0 52 3\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 8303 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t63\t0.0\t2\t0 62 1\n",
      "26\t8198\t0.0\t2\t8019 172 7\n",
      "27\t36\t0.0\t2\t0 35 1\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1808_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1808_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1808_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1808_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1808_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1808_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1808_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1808_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        17,875 reads  @    166.0 µs/read;   0.36 M reads/minute\n",
      "Finished in 2.97 s (166 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,875\n",
      "  Read 1 with adapter:                  17,860 (99.9%)\n",
      "  Read 2 with adapter:                  17,835 (99.8%)\n",
      "Pairs written (passing filters):        17,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,545,719 bp\n",
      "  Read 1:     5,272,663 bp\n",
      "  Read 2:     5,273,056 bp\n",
      "Total written (filtered):      9,617,845 bp (91.2%)\n",
      "  Read 1:     4,808,428 bp\n",
      "  Read 2:     4,809,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t210\t0.0\t2\t0 202 8\n",
      "26\t17553\t0.0\t2\t17138 397 18\n",
      "27\t69\t0.0\t2\t0 67 2\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t148\t0.0\t2\t0 143 5\n",
      "26\t17601\t0.0\t2\t17284 303 14\n",
      "27\t71\t0.0\t2\t0 70 1\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1809_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1809_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1809_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1809_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1809_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1809_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1809_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1809_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        17,455 reads  @    138.7 µs/read;   0.43 M reads/minute\n",
      "Finished in 2.43 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,455\n",
      "  Read 1 with adapter:                  17,437 (99.9%)\n",
      "  Read 2 with adapter:                  17,354 (99.4%)\n",
      "Pairs written (passing filters):        17,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,295,631 bp\n",
      "  Read 1:     5,147,878 bp\n",
      "  Read 2:     5,147,753 bp\n",
      "Total written (filtered):      9,391,085 bp (91.2%)\n",
      "  Read 1:     4,694,695 bp\n",
      "  Read 2:     4,696,390 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17437 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t251\t0.0\t2\t0 232 19\n",
      "26\t17092\t0.0\t2\t16683 396 13\n",
      "27\t72\t0.0\t2\t0 72\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17354 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t191\t0.0\t2\t0 179 12\n",
      "26\t17064\t0.0\t2\t16754 296 14\n",
      "27\t69\t0.0\t2\t0 67 2\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1810_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1810_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1810_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1810_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1810_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1810_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1810_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1810_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        21,625 reads  @    136.3 µs/read;   0.44 M reads/minute\n",
      "Finished in 2.95 s (136 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,625\n",
      "  Read 1 with adapter:                  21,600 (99.9%)\n",
      "  Read 2 with adapter:                  21,577 (99.8%)\n",
      "Pairs written (passing filters):        21,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,758,356 bp\n",
      "  Read 1:     6,379,367 bp\n",
      "  Read 2:     6,378,989 bp\n",
      "Total written (filtered):     11,636,004 bp (91.2%)\n",
      "  Read 1:     5,817,933 bp\n",
      "  Read 2:     5,818,071 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t273\t0.0\t2\t0 264 9\n",
      "26\t21206\t0.0\t2\t20791 408 7\n",
      "27\t87\t0.0\t2\t0 82 5\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21577 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t176\t0.0\t2\t0 169 7\n",
      "26\t21304\t0.0\t2\t20884 398 22\n",
      "27\t78\t0.0\t2\t0 73 5\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1811_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1811_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1811_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1811_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1811_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1811_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1811_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1811_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,834 reads  @    167.9 µs/read;   0.36 M reads/minute\n",
      "Finished in 3.67 s (168 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,834\n",
      "  Read 1 with adapter:                  21,817 (99.9%)\n",
      "  Read 2 with adapter:                  21,791 (99.8%)\n",
      "Pairs written (passing filters):        21,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,881,403 bp\n",
      "  Read 1:     6,440,870 bp\n",
      "  Read 2:     6,440,533 bp\n",
      "Total written (filtered):     11,747,810 bp (91.2%)\n",
      "  Read 1:     5,873,780 bp\n",
      "  Read 2:     5,874,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t283\t0.0\t2\t0 269 14\n",
      "26\t21382\t0.0\t2\t20948 422 12\n",
      "27\t117\t0.0\t2\t0 111 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t175\t0.0\t2\t0 167 8\n",
      "26\t21483\t0.0\t2\t21035 417 31\n",
      "27\t110\t0.0\t2\t0 103 7\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1812_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1812_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1812_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1812_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1812_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1812_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1812_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1812_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        20,258 reads  @    170.3 µs/read;   0.35 M reads/minute\n",
      "Finished in 3.45 s (171 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,258\n",
      "  Read 1 with adapter:                  20,196 (99.7%)\n",
      "  Read 2 with adapter:                  19,936 (98.4%)\n",
      "Pairs written (passing filters):        20,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,897,429 bp\n",
      "  Read 1:     5,975,548 bp\n",
      "  Read 2:     5,921,881 bp\n",
      "Total written (filtered):     10,853,001 bp (91.2%)\n",
      "  Read 1:     5,449,440 bp\n",
      "  Read 2:     5,403,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20196 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t237\t0.0\t2\t0 227 10\n",
      "26\t19787\t0.0\t2\t19386 391 10\n",
      "27\t127\t0.0\t2\t0 124 3\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19936 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t110\t0.0\t2\t0 102 8\n",
      "26\t19733\t0.0\t2\t19221 466 46\n",
      "27\t72\t0.0\t2\t0 66 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1813_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1813_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1813_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1813_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1813_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1813_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1813_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1813_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1814_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1814_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1814_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1814_27_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1814_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1814_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1814_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1814_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,938 reads  @    153.2 µs/read;   0.39 M reads/minute\n",
      "Finished in 3.36 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,938\n",
      "  Read 1 with adapter:                  21,916 (99.9%)\n",
      "  Read 2 with adapter:                  21,868 (99.7%)\n",
      "Pairs written (passing filters):        21,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,937,781 bp\n",
      "  Read 1:     6,471,441 bp\n",
      "  Read 2:     6,466,340 bp\n",
      "Total written (filtered):     11,799,612 bp (91.2%)\n",
      "  Read 1:     5,901,780 bp\n",
      "  Read 2:     5,897,832 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21916 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t292\t0.0\t2\t0 278 14\n",
      "26\t21461\t0.0\t2\t21042 408 11\n",
      "27\t117\t0.0\t2\t0 109 8\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t184\t0.0\t2\t0 176 8\n",
      "26\t21569\t0.0\t2\t21113 431 25\n",
      "27\t102\t0.0\t2\t0 100 2\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1815_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1815_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1815_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1815_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1815_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1815_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1815_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1815_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,626 reads  @    175.9 µs/read;   0.34 M reads/minute\n",
      "Finished in 3.81 s (176 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,626\n",
      "  Read 1 with adapter:                  21,600 (99.9%)\n",
      "  Read 2 with adapter:                  21,564 (99.7%)\n",
      "Pairs written (passing filters):        21,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,757,322 bp\n",
      "  Read 1:     6,379,383 bp\n",
      "  Read 2:     6,377,939 bp\n",
      "Total written (filtered):     11,635,226 bp (91.2%)\n",
      "  Read 1:     5,817,891 bp\n",
      "  Read 2:     5,817,335 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t275\t0.0\t2\t0 265 10\n",
      "26\t21173\t0.0\t2\t20703 448 22\n",
      "27\t113\t0.0\t2\t0 109 4\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t160\t0.0\t2\t0 152 8\n",
      "26\t21306\t0.0\t2\t20900 390 16\n",
      "27\t80\t0.0\t2\t0 73 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1816_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1816_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1816_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1816_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1816_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1816_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1816_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1816_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        20,220 reads  @    138.1 µs/read;   0.43 M reads/minute\n",
      "Finished in 2.80 s (138 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,220\n",
      "  Read 1 with adapter:                  20,208 (99.9%)\n",
      "  Read 2 with adapter:                  20,187 (99.8%)\n",
      "Pairs written (passing filters):        20,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,929,671 bp\n",
      "  Read 1:     5,964,893 bp\n",
      "  Read 2:     5,964,778 bp\n",
      "Total written (filtered):     10,879,566 bp (91.2%)\n",
      "  Read 1:     5,439,589 bp\n",
      "  Read 2:     5,439,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20208 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t267\t0.0\t2\t0 250 17\n",
      "26\t19772\t0.0\t2\t19370 389 13\n",
      "27\t119\t0.0\t2\t0 111 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t151\t0.0\t2\t0 148 3\n",
      "26\t19941\t0.0\t2\t19596 337 8\n",
      "27\t80\t0.0\t2\t0 79 1\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1817_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1817_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1817_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1817_33_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1817_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1817_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1817_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1817_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,375 reads  @    154.5 µs/read;   0.39 M reads/minute\n",
      "Finished in 3.46 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,375\n",
      "  Read 1 with adapter:                  22,273 (99.5%)\n",
      "  Read 2 with adapter:                  22,206 (99.2%)\n",
      "Pairs written (passing filters):        22,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,194,786 bp\n",
      "  Read 1:     6,596,439 bp\n",
      "  Read 2:     6,598,347 bp\n",
      "Total written (filtered):     12,038,568 bp (91.2%)\n",
      "  Read 1:     6,017,424 bp\n",
      "  Read 2:     6,021,144 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t225\t0.0\t2\t0 217 8\n",
      "26\t21904\t0.0\t2\t21462 433 9\n",
      "27\t110\t0.0\t2\t0 105 5\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t246\t0.0\t2\t0 237 9\n",
      "26\t21837\t0.0\t2\t21467 353 17\n",
      "27\t101\t0.0\t2\t0 97 4\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1818_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1818_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1818_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1818_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1818_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1818_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1818_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1818_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        31,948 reads  @    141.0 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.51 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,948\n",
      "  Read 1 with adapter:                  31,836 (99.6%)\n",
      "  Read 2 with adapter:                  31,747 (99.4%)\n",
      "Pairs written (passing filters):        31,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,835,696 bp\n",
      "  Read 1:     9,420,330 bp\n",
      "  Read 2:     9,415,366 bp\n",
      "Total written (filtered):     17,182,673 bp (91.2%)\n",
      "  Read 1:     8,592,641 bp\n",
      "  Read 2:     8,590,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t286\t0.0\t2\t0 277 9\n",
      "26\t31324\t0.0\t2\t30764 549 11\n",
      "27\t169\t0.0\t2\t0 160 9\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t252\t0.0\t2\t0 244 8\n",
      "26\t31336\t0.0\t2\t30632 660 44\n",
      "27\t118\t0.0\t2\t0 112 6\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1819_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1819_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1819_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1819_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1819_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1819_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1819_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1819_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        30,525 reads  @    166.6 µs/read;   0.36 M reads/minute\n",
      "Finished in 5.09 s (167 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,525\n",
      "  Read 1 with adapter:                  30,492 (99.9%)\n",
      "  Read 2 with adapter:                  30,475 (99.8%)\n",
      "Pairs written (passing filters):        30,525 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,008,944 bp\n",
      "  Read 1:     9,004,537 bp\n",
      "  Read 2:     9,004,407 bp\n",
      "Total written (filtered):     16,423,955 bp (91.2%)\n",
      "  Read 1:     8,211,821 bp\n",
      "  Read 2:     8,212,134 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t295\t0.0\t2\t0 286 9\n",
      "26\t30007\t0.0\t2\t29472 529 6\n",
      "27\t133\t0.0\t2\t0 128 5\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30475 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t225\t0.0\t2\t0 214 11\n",
      "26\t30102\t0.0\t2\t29504 569 29\n",
      "27\t120\t0.0\t2\t0 113 7\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1820_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1820_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1820_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1820_39_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1820_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1820_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1820_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1820_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        17,807 reads  @    195.2 µs/read;   0.31 M reads/minute\n",
      "Finished in 3.48 s (195 us/read; 0.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,807\n",
      "  Read 1 with adapter:                  17,785 (99.9%)\n",
      "  Read 2 with adapter:                  17,653 (99.1%)\n",
      "Pairs written (passing filters):        17,806 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,485,621 bp\n",
      "  Read 1:     5,252,162 bp\n",
      "  Read 2:     5,233,459 bp\n",
      "Total written (filtered):      9,564,071 bp (91.2%)\n",
      "  Read 1:     4,789,574 bp\n",
      "  Read 2:     4,774,497 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t192\t0.0\t2\t0 185 7\n",
      "26\t17493\t0.0\t2\t17194 294 5\n",
      "27\t71\t0.0\t2\t0 66 5\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17653 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t114\t0.0\t2\t0 101 13\n",
      "26\t17443\t0.0\t2\t16904 494 45\n",
      "27\t70\t0.0\t2\t0 66 4\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1821_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1821_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1821_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1821_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1821_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1821_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1821_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1821_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             1 reads  @  48476.0 µs/read;   0.00 M reads/minute\n",
      "Finished in 0.05 s (52112 us/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  1\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           368 bp\n",
      "  Read 1:           115 bp\n",
      "  Read 2:           253 bp\n",
      "Total written (filtered):            368 bp (100.0%)\n",
      "  Read 1:           115 bp\n",
      "  Read 2:           253 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1822_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1822_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1822_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1822_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1822_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1822_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1822_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1822_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        24,110 reads  @    147.5 µs/read;   0.41 M reads/minute\n",
      "Finished in 3.56 s (148 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,110\n",
      "  Read 1 with adapter:                  23,988 (99.5%)\n",
      "  Read 2 with adapter:                  24,040 (99.7%)\n",
      "Pairs written (passing filters):        24,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,216,288 bp\n",
      "  Read 1:     7,107,315 bp\n",
      "  Read 2:     7,108,973 bp\n",
      "Total written (filtered):     12,967,706 bp (91.2%)\n",
      "  Read 1:     6,483,678 bp\n",
      "  Read 2:     6,484,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t218\t0.0\t2\t0 216 2\n",
      "26\t23603\t0.0\t2\t23209 386 8\n",
      "27\t143\t0.0\t2\t0 138 5\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t192\t0.0\t2\t0 184 8\n",
      "26\t23729\t0.0\t2\t23223 477 29\n",
      "27\t109\t0.0\t2\t0 103 6\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1823_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1823_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1823_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1823_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1823_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1823_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1823_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1823_45_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,234 reads  @    154.6 µs/read;   0.39 M reads/minute\n",
      "Finished in 3.44 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,234\n",
      "  Read 1 with adapter:                  22,198 (99.8%)\n",
      "  Read 2 with adapter:                  22,174 (99.7%)\n",
      "Pairs written (passing filters):        22,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,112,778 bp\n",
      "  Read 1:     6,557,542 bp\n",
      "  Read 2:     6,555,236 bp\n",
      "Total written (filtered):     11,959,252 bp (91.2%)\n",
      "  Read 1:     5,980,481 bp\n",
      "  Read 2:     5,978,771 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t215\t0.0\t2\t0 208 7\n",
      "26\t21849\t0.0\t2\t21396 442 11\n",
      "27\t88\t0.0\t2\t0 86 2\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t161\t0.0\t2\t0 155 6\n",
      "26\t21893\t0.0\t2\t21415 454 24\n",
      "27\t102\t0.0\t2\t0 96 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1824_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1824_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1824_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1824_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1824_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1824_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1824_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1824_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        23,959 reads  @    136.0 µs/read;   0.44 M reads/minute\n",
      "Finished in 3.26 s (136 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,959\n",
      "  Read 1 with adapter:                  23,930 (99.9%)\n",
      "  Read 2 with adapter:                  23,888 (99.7%)\n",
      "Pairs written (passing filters):        23,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,134,882 bp\n",
      "  Read 1:     7,067,364 bp\n",
      "  Read 2:     7,067,518 bp\n",
      "Total written (filtered):     12,891,672 bp (91.2%)\n",
      "  Read 1:     6,445,202 bp\n",
      "  Read 2:     6,446,470 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t189\t0.0\t2\t0 181 8\n",
      "26\t23567\t0.0\t2\t23119 441 7\n",
      "27\t133\t0.0\t2\t0 129 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t156\t0.0\t2\t0 151 5\n",
      "26\t23624\t0.0\t2\t23120 476 28\n",
      "27\t76\t0.0\t2\t0 66 10\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1825_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1825_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1825_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1825_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1825_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1825_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1825_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1825_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,153 reads  @    156.4 µs/read;   0.38 M reads/minute\n",
      "Finished in 0.34 s (158 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,153\n",
      "  Read 1 with adapter:                   2,148 (99.8%)\n",
      "  Read 2 with adapter:                   2,138 (99.3%)\n",
      "Pairs written (passing filters):         2,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,270,037 bp\n",
      "  Read 1:       635,134 bp\n",
      "  Read 2:       634,903 bp\n",
      "Total written (filtered):      1,158,604 bp (91.2%)\n",
      "  Read 1:       579,280 bp\n",
      "  Read 2:       579,324 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 2148 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t25\t0.0\t2\t0 24 1\n",
      "26\t2092\t0.0\t2\t2023 66 3\n",
      "27\t23\t0.0\t2\t0 23\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 2138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t19\t0.0\t2\t0 19\n",
      "26\t2106\t0.0\t2\t2071 30 5\n",
      "27\t12\t0.0\t2\t0 10 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1826_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1826_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1826_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1826_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1826_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1826_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1826_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1826_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:03        29,346 reads  @    128.6 µs/read;   0.47 M reads/minute\n",
      "Finished in 3.78 s (129 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,346\n",
      "  Read 1 with adapter:                  29,314 (99.9%)\n",
      "  Read 2 with adapter:                  29,214 (99.6%)\n",
      "Pairs written (passing filters):        29,346 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,313,027 bp\n",
      "  Read 1:     8,656,536 bp\n",
      "  Read 2:     8,656,491 bp\n",
      "Total written (filtered):     15,791,513 bp (91.2%)\n",
      "  Read 1:     7,894,464 bp\n",
      "  Read 2:     7,897,049 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t290\t0.0\t2\t0 281 9\n",
      "26\t28792\t0.0\t2\t28287 495 10\n",
      "27\t178\t0.0\t2\t0 176 2\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t242\t0.0\t2\t0 227 15\n",
      "26\t28841\t0.0\t2\t28263 543 35\n",
      "27\t106\t0.0\t2\t0 100 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1827_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1827_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1827_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1827_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1827_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1827_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1827_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1827_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        30,130 reads  @    134.7 µs/read;   0.45 M reads/minute\n",
      "Finished in 4.06 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,130\n",
      "  Read 1 with adapter:                  30,103 (99.9%)\n",
      "  Read 2 with adapter:                  29,877 (99.2%)\n",
      "Pairs written (passing filters):        30,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,776,203 bp\n",
      "  Read 1:     8,888,338 bp\n",
      "  Read 2:     8,887,865 bp\n",
      "Total written (filtered):     16,216,902 bp (91.2%)\n",
      "  Read 1:     8,105,777 bp\n",
      "  Read 2:     8,111,125 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30103 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t337\t0.0\t2\t0 328 9\n",
      "26\t29518\t0.0\t2\t28987 519 12\n",
      "27\t176\t0.0\t2\t0 173 3\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29877 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t197\t0.0\t2\t0 195 2\n",
      "26\t29532\t0.0\t2\t28982 521 29\n",
      "27\t125\t0.0\t2\t0 118 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1828_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1828_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1828_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1828_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1828_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1828_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1828_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1828_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        32,673 reads  @    143.8 µs/read;   0.42 M reads/minute\n",
      "Finished in 4.70 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,673\n",
      "  Read 1 with adapter:                  32,648 (99.9%)\n",
      "  Read 2 with adapter:                  29,884 (91.5%)\n",
      "Pairs written (passing filters):        32,673 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,764,384 bp\n",
      "  Read 1:     9,638,137 bp\n",
      "  Read 2:     9,126,247 bp\n",
      "Total written (filtered):     17,138,677 bp (91.3%)\n",
      "  Read 1:     8,789,324 bp\n",
      "  Read 2:     8,349,353 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32648 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t307\t0.0\t2\t0 300 7\n",
      "26\t32075\t0.0\t2\t31535 523 17\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t216\t0.0\t2\t0 203 13\n",
      "26\t29527\t0.0\t2\t28891 601 35\n",
      "27\t108\t0.0\t2\t0 106 2\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1829_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1829_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1829_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1829_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1829_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1829_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1829_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1829_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            21 reads  @   1601.3 µs/read;   0.04 M reads/minute\n",
      "Finished in 0.04 s (1795 us/read; 0.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 21\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            21 (100.0%)\n",
      "\n",
      "Total basepairs processed:         2,880 bp\n",
      "  Read 1:         1,610 bp\n",
      "  Read 2:         1,270 bp\n",
      "Total written (filtered):          2,880 bp (100.0%)\n",
      "  Read 1:         1,610 bp\n",
      "  Read 2:         1,270 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1830_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1830_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1830_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1830_59_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1830_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1830_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1830_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1830_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        30,722 reads  @    153.8 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.73 s (154 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,722\n",
      "  Read 1 with adapter:                  30,699 (99.9%)\n",
      "  Read 2 with adapter:                  30,648 (99.8%)\n",
      "Pairs written (passing filters):        30,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,121,720 bp\n",
      "  Read 1:     9,061,894 bp\n",
      "  Read 2:     9,059,826 bp\n",
      "Total written (filtered):     16,526,921 bp (91.2%)\n",
      "  Read 1:     8,263,819 bp\n",
      "  Read 2:     8,263,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30699 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t354\t0.0\t2\t0 338 16\n",
      "26\t30108\t0.0\t2\t29513 582 13\n",
      "27\t155\t0.0\t2\t0 150 5\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30648 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t268\t0.0\t2\t0 258 10\n",
      "26\t30217\t0.0\t2\t29569 609 39\n",
      "27\t138\t0.0\t2\t0 131 7\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1831_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1831_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1831_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1831_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1831_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1831_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1831_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1831_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,335 reads  @    184.2 µs/read;   0.33 M reads/minute\n",
      "Finished in 0.43 s (186 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,335\n",
      "  Read 1 with adapter:                   2,333 (99.9%)\n",
      "  Read 2 with adapter:                   2,321 (99.4%)\n",
      "Pairs written (passing filters):         2,335 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,377,203 bp\n",
      "  Read 1:       688,825 bp\n",
      "  Read 2:       688,378 bp\n",
      "Total written (filtered):      1,256,184 bp (91.2%)\n",
      "  Read 1:       628,156 bp\n",
      "  Read 2:       628,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 2333 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t23\t0.0\t2\t0 22 1\n",
      "26\t2283\t0.0\t2\t2202 81\n",
      "27\t12\t0.0\t2\t0 11 1\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 2321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t12\t0.0\t2\t0 10 2\n",
      "26\t2297\t0.0\t2\t2182 104 11\n",
      "27\t8\t0.0\t2\t0 8\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1832_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1832_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1832_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1832_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1832_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1832_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1832_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1832_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        39,768 reads  @    137.2 µs/read;   0.44 M reads/minute\n",
      "Finished in 5.48 s (138 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,768\n",
      "  Read 1 with adapter:                  39,737 (99.9%)\n",
      "  Read 2 with adapter:                  39,666 (99.7%)\n",
      "Pairs written (passing filters):        39,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,461,748 bp\n",
      "  Read 1:    11,730,854 bp\n",
      "  Read 2:    11,730,894 bp\n",
      "Total written (filtered):     21,397,506 bp (91.2%)\n",
      "  Read 1:    10,697,836 bp\n",
      "  Read 2:    10,699,670 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t436\t0.0\t2\t0 421 15\n",
      "26\t38973\t0.0\t2\t38233 726 14\n",
      "27\t244\t0.0\t2\t0 237 7\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t281\t0.0\t2\t0 275 6\n",
      "26\t39201\t0.0\t2\t38421 746 34\n",
      "27\t131\t0.0\t2\t0 129 2\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1833_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1833_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1833_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1833_65_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1833_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1833_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1833_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1833_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        15,254 reads  @    172.7 µs/read;   0.35 M reads/minute\n",
      "Finished in 2.65 s (173 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,254\n",
      "  Read 1 with adapter:                  15,239 (99.9%)\n",
      "  Read 2 with adapter:                  15,163 (99.4%)\n",
      "Pairs written (passing filters):        15,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,997,679 bp\n",
      "  Read 1:     4,499,033 bp\n",
      "  Read 2:     4,498,646 bp\n",
      "Total written (filtered):      8,207,355 bp (91.2%)\n",
      "  Read 1:     4,102,869 bp\n",
      "  Read 2:     4,104,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t180\t0.0\t2\t0 172 8\n",
      "26\t14934\t0.0\t2\t14594 334 6\n",
      "27\t76\t0.0\t2\t0 70 6\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15163 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t151\t0.0\t2\t0 150 1\n",
      "26\t14922\t0.0\t2\t14661 249 12\n",
      "27\t75\t0.0\t2\t0 72 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1834_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1834_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1834_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1834_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1834_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1834_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1834_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1834_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        29,784 reads  @    141.3 µs/read;   0.42 M reads/minute\n",
      "Finished in 4.21 s (141 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,784\n",
      "  Read 1 with adapter:                  29,751 (99.9%)\n",
      "  Read 2 with adapter:                  29,649 (99.5%)\n",
      "Pairs written (passing filters):        29,784 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,570,574 bp\n",
      "  Read 1:     8,785,739 bp\n",
      "  Read 2:     8,784,835 bp\n",
      "Total written (filtered):     16,026,407 bp (91.2%)\n",
      "  Read 1:     8,012,357 bp\n",
      "  Read 2:     8,014,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t363\t0.0\t2\t0 356 7\n",
      "26\t29156\t0.0\t2\t28637 506 13\n",
      "27\t193\t0.0\t2\t0 190 3\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29649 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t219\t0.0\t2\t0 211 8\n",
      "26\t29277\t0.0\t2\t28736 517 24\n",
      "27\t136\t0.0\t2\t0 126 10\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1835_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1835_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1835_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1835_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1835_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1835_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1835_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1835_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,341 reads  @    179.9 µs/read;   0.33 M reads/minute\n",
      "Finished in 4.92 s (180 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,341\n",
      "  Read 1 with adapter:                  27,287 (99.8%)\n",
      "  Read 2 with adapter:                  27,264 (99.7%)\n",
      "Pairs written (passing filters):        27,341 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,128,332 bp\n",
      "  Read 1:     8,062,952 bp\n",
      "  Read 2:     8,065,380 bp\n",
      "Total written (filtered):     14,710,164 bp (91.2%)\n",
      "  Read 1:     7,353,572 bp\n",
      "  Read 2:     7,356,592 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t324\t0.0\t2\t0 320 4\n",
      "26\t26729\t0.0\t2\t26204 512 13\n",
      "27\t170\t0.0\t2\t0 165 5\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t175\t0.0\t2\t0 173 2\n",
      "26\t26974\t0.0\t2\t26473 477 24\n",
      "27\t91\t0.0\t2\t0 90 1\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1836_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1836_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1836_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1836_71_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1836_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1836_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1836_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1836_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,681 reads  @    168.3 µs/read;   0.36 M reads/minute\n",
      "Finished in 3.65 s (169 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,681\n",
      "  Read 1 with adapter:                  21,656 (99.9%)\n",
      "  Read 2 with adapter:                  21,227 (97.9%)\n",
      "Pairs written (passing filters):        21,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,716,300 bp\n",
      "  Read 1:     6,395,390 bp\n",
      "  Read 2:     6,320,910 bp\n",
      "Total written (filtered):     11,601,465 bp (91.2%)\n",
      "  Read 1:     5,832,426 bp\n",
      "  Read 2:     5,769,039 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t257\t0.0\t2\t0 246 11\n",
      "26\t21231\t0.0\t2\t20831 391 9\n",
      "27\t119\t0.0\t2\t0 111 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t138\t0.0\t2\t0 133 5\n",
      "26\t20970\t0.0\t2\t20501 441 28\n",
      "27\t107\t0.0\t2\t0 101 6\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1837_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1837_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1837_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1837_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1837_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1837_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1837_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1837_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             2 reads  @  18226.6 µs/read;   0.00 M reads/minute\n",
      "Finished in 0.04 s (19686 us/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  2\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             2 (100.0%)\n",
      "\n",
      "Total basepairs processed:           276 bp\n",
      "  Read 1:           213 bp\n",
      "  Read 2:            63 bp\n",
      "Total written (filtered):            276 bp (100.0%)\n",
      "  Read 1:           213 bp\n",
      "  Read 2:            63 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1838_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1838_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1838_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1838_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1838_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1838_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1838_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1838_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        31,490 reads  @    137.6 µs/read;   0.44 M reads/minute\n",
      "Finished in 4.34 s (138 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,490\n",
      "  Read 1 with adapter:                  31,441 (99.8%)\n",
      "  Read 2 with adapter:                  31,351 (99.6%)\n",
      "Pairs written (passing filters):        31,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,564,585 bp\n",
      "  Read 1:     9,287,371 bp\n",
      "  Read 2:     9,277,214 bp\n",
      "Total written (filtered):     16,932,333 bp (91.2%)\n",
      "  Read 1:     8,470,085 bp\n",
      "  Read 2:     8,462,248 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t405\t0.0\t2\t0 394 11\n",
      "26\t30786\t0.0\t2\t30230 549 7\n",
      "27\t207\t0.0\t2\t0 203 4\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t295\t0.0\t2\t0 284 11\n",
      "26\t30907\t0.0\t2\t30361 517 29\n",
      "27\t127\t0.0\t2\t0 119 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1839_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1839_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1839_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1839_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1839_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1839_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1839_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1839_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00         1,999 reads  @    153.4 µs/read;   0.39 M reads/minute\n",
      "Finished in 0.31 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,999\n",
      "  Read 1 with adapter:                   1,985 (99.3%)\n",
      "  Read 2 with adapter:                   1,976 (98.8%)\n",
      "Pairs written (passing filters):         1,999 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,177,597 bp\n",
      "  Read 1:       589,703 bp\n",
      "  Read 2:       587,894 bp\n",
      "Total written (filtered):      1,074,636 bp (91.3%)\n",
      "  Read 1:       538,107 bp\n",
      "  Read 2:       536,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t27\t0.0\t2\t0 26 1\n",
      "26\t1940\t0.0\t2\t1881 57 2\n",
      "27\t11\t0.0\t2\t0 9 2\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1976 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t13\t0.0\t2\t0 10 3\n",
      "26\t1952\t0.0\t2\t1897 52 3\n",
      "27\t8\t0.0\t2\t0 7 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1840_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1840_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1840_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1840_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1840_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1840_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1840_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1840_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:07        37,782 reads  @    195.1 µs/read;   0.31 M reads/minute\n",
      "Finished in 7.38 s (195 us/read; 0.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,782\n",
      "  Read 1 with adapter:                  37,733 (99.9%)\n",
      "  Read 2 with adapter:                  37,704 (99.8%)\n",
      "Pairs written (passing filters):        37,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,289,816 bp\n",
      "  Read 1:    11,144,515 bp\n",
      "  Read 2:    11,145,301 bp\n",
      "Total written (filtered):     20,328,827 bp (91.2%)\n",
      "  Read 1:    10,163,678 bp\n",
      "  Read 2:    10,165,149 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t508\t0.0\t2\t0 491 17\n",
      "26\t36948\t0.0\t2\t36100 831 17\n",
      "27\t195\t0.0\t2\t0 186 9\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t332\t0.0\t2\t0 320 12\n",
      "26\t37172\t0.0\t2\t36401 737 34\n",
      "27\t156\t0.0\t2\t0 152 4\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1849_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1849_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1849_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1849_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1849_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1849_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1849_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1849_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        11,246 reads  @    142.5 µs/read;   0.42 M reads/minute\n",
      "Finished in 1.61 s (143 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,246\n",
      "  Read 1 with adapter:                  11,215 (99.7%)\n",
      "  Read 2 with adapter:                  11,171 (99.3%)\n",
      "Pairs written (passing filters):        11,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,628,637 bp\n",
      "  Read 1:     3,312,101 bp\n",
      "  Read 2:     3,316,536 bp\n",
      "Total written (filtered):      6,046,749 bp (91.2%)\n",
      "  Read 1:     3,020,598 bp\n",
      "  Read 2:     3,026,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 11215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t120\t0.0\t2\t0 114 6\n",
      "26\t11034\t0.0\t2\t10708 317 9\n",
      "27\t45\t0.0\t2\t0 43 2\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 11171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t115\t0.0\t2\t0 113 2\n",
      "26\t10983\t0.0\t2\t10778 192 13\n",
      "27\t60\t0.0\t2\t0 57 3\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1850_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1850_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1850_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1850_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1850_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1850_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1850_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1850_83_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,818 reads  @    149.0 µs/read;   0.40 M reads/minute\n",
      "Finished in 5.49 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,818\n",
      "  Read 1 with adapter:                  36,695 (99.7%)\n",
      "  Read 2 with adapter:                  36,752 (99.8%)\n",
      "Pairs written (passing filters):        36,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,696,342 bp\n",
      "  Read 1:    10,835,519 bp\n",
      "  Read 2:    10,860,823 bp\n",
      "Total written (filtered):     19,787,004 bp (91.2%)\n",
      "  Read 1:     9,881,636 bp\n",
      "  Read 2:     9,905,368 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36695 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t414\t0.0\t2\t0 397 17\n",
      "26\t36026\t0.0\t2\t35248 765 13\n",
      "27\t207\t0.0\t2\t0 198 9\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36752 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t275\t0.0\t2\t0 258 17\n",
      "26\t36276\t0.0\t2\t35577 656 43\n",
      "27\t148\t0.0\t2\t0 141 7\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1851_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1851_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1851_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1851_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1851_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1851_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1851_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1851_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        31,626 reads  @    118.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 3.76 s (119 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,626\n",
      "  Read 1 with adapter:                  31,600 (99.9%)\n",
      "  Read 2 with adapter:                  31,564 (99.8%)\n",
      "Pairs written (passing filters):        31,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,655,813 bp\n",
      "  Read 1:     9,326,464 bp\n",
      "  Read 2:     9,329,349 bp\n",
      "Total written (filtered):     17,013,830 bp (91.2%)\n",
      "  Read 1:     8,505,124 bp\n",
      "  Read 2:     8,508,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t364\t0.0\t2\t0 356 8\n",
      "26\t31089\t0.0\t2\t30411 669 9\n",
      "27\t126\t0.0\t2\t0 123 3\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t207\t0.0\t2\t0 196 11\n",
      "26\t31189\t0.0\t2\t30572 590 27\n",
      "27\t94\t0.0\t2\t0 90 4\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1852_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1852_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1852_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1852_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1852_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1852_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1852_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1852_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        23,412 reads  @    155.1 µs/read;   0.39 M reads/minute\n",
      "Finished in 3.64 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,412\n",
      "  Read 1 with adapter:                  23,378 (99.9%)\n",
      "  Read 2 with adapter:                  23,301 (99.5%)\n",
      "Pairs written (passing filters):        23,410 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,796,869 bp\n",
      "  Read 1:     6,900,745 bp\n",
      "  Read 2:     6,896,124 bp\n",
      "Total written (filtered):     12,582,770 bp (91.2%)\n",
      "  Read 1:     6,292,459 bp\n",
      "  Read 2:     6,290,311 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t220\t0.0\t2\t0 211 9\n",
      "26\t22995\t0.0\t2\t22583 404 8\n",
      "27\t122\t0.0\t2\t0 114 8\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t140\t0.0\t2\t0 133 7\n",
      "26\t23031\t0.0\t2\t22538 465 28\n",
      "27\t97\t0.0\t2\t0 90 7\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1853_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1853_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1853_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1853_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1853_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1853_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1853_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1853_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8<----------] 00:00:00            30 reads  @   1422.9 µs/read;   0.04 M reads/minute\n",
      "Finished in 0.05 s (1534 us/read; 0.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 30\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            30 (100.0%)\n",
      "\n",
      "Total basepairs processed:         7,513 bp\n",
      "  Read 1:         2,894 bp\n",
      "  Read 2:         4,619 bp\n",
      "Total written (filtered):          7,513 bp (100.0%)\n",
      "  Read 1:         2,894 bp\n",
      "  Read 2:         4,619 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1854_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1854_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1854_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1854_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1854_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1854_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1854_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1854_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        32,254 reads  @    133.4 µs/read;   0.45 M reads/minute\n",
      "Finished in 4.31 s (134 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,254\n",
      "  Read 1 with adapter:                  32,161 (99.7%)\n",
      "  Read 2 with adapter:                  31,996 (99.2%)\n",
      "Pairs written (passing filters):        32,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,987,536 bp\n",
      "  Read 1:     9,499,324 bp\n",
      "  Read 2:     9,488,212 bp\n",
      "Total written (filtered):     17,319,596 bp (91.2%)\n",
      "  Read 1:     8,663,209 bp\n",
      "  Read 2:     8,656,387 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t321\t0.0\t2\t0 308 13\n",
      "26\t31593\t0.0\t2\t31034 542 17\n",
      "27\t184\t0.0\t2\t0 174 10\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t259\t0.0\t2\t0 248 11\n",
      "26\t31561\t0.0\t2\t30937 589 35\n",
      "27\t124\t0.0\t2\t0 116 8\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1855_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1855_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1855_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1855_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1855_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1855_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1855_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1855_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,303 reads  @    160.6 µs/read;   0.37 M reads/minute\n",
      "Finished in 4.23 s (161 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,303\n",
      "  Read 1 with adapter:                  26,224 (99.7%)\n",
      "  Read 2 with adapter:                  26,113 (99.3%)\n",
      "Pairs written (passing filters):        26,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,503,987 bp\n",
      "  Read 1:     7,746,798 bp\n",
      "  Read 2:     7,757,189 bp\n",
      "Total written (filtered):     14,143,395 bp (91.2%)\n",
      "  Read 1:     7,065,090 bp\n",
      "  Read 2:     7,078,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26224 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t294\t0.0\t2\t0 289 5\n",
      "26\t25742\t0.0\t2\t25205 526 11\n",
      "27\t146\t0.0\t2\t0 133 13\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t181\t0.0\t2\t0 175 6\n",
      "26\t25788\t0.0\t2\t25306 464 18\n",
      "27\t129\t0.0\t2\t0 120 9\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1856_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1856_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1856_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1856_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1856_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1856_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1856_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1856_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,244 reads  @    149.4 µs/read;   0.40 M reads/minute\n",
      "Finished in 4.08 s (150 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,244\n",
      "  Read 1 with adapter:                  27,172 (99.7%)\n",
      "  Read 2 with adapter:                  27,187 (99.8%)\n",
      "Pairs written (passing filters):        27,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,057,807 bp\n",
      "  Read 1:     8,020,899 bp\n",
      "  Read 2:     8,036,908 bp\n",
      "Total written (filtered):     14,644,748 bp (91.2%)\n",
      "  Read 1:     7,314,614 bp\n",
      "  Read 2:     7,330,134 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27172 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t330\t0.0\t2\t0 312 18\n",
      "26\t26695\t0.0\t2\t26128 554 13\n",
      "27\t111\t0.0\t2\t0 101 10\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t206\t0.0\t2\t0 199 7\n",
      "26\t26848\t0.0\t2\t26349 478 21\n",
      "27\t100\t0.0\t2\t0 94 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1857_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1857_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1857_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1857_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1857_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1857_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1857_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1857_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        25,942 reads  @    134.0 µs/read;   0.45 M reads/minute\n",
      "Finished in 3.48 s (134 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,942\n",
      "  Read 1 with adapter:                  25,923 (99.9%)\n",
      "  Read 2 with adapter:                  25,751 (99.3%)\n",
      "Pairs written (passing filters):        25,942 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,302,905 bp\n",
      "  Read 1:     7,652,626 bp\n",
      "  Read 2:     7,650,279 bp\n",
      "Total written (filtered):     13,959,724 bp (91.2%)\n",
      "  Read 1:     6,978,821 bp\n",
      "  Read 2:     6,980,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 25923 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t313\t0.0\t2\t0 301 12\n",
      "26\t25452\t0.0\t2\t24921 526 5\n",
      "27\t124\t0.0\t2\t0 116 8\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t279\t0.0\t2\t0 269 10\n",
      "26\t25328\t0.0\t2\t24895 410 23\n",
      "27\t91\t0.0\t2\t0 91\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1858_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1858_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1858_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1858_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1858_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1858_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1858_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1858_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        18,513 reads  @    162.0 µs/read;   0.37 M reads/minute\n",
      "Finished in 3.00 s (162 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,513\n",
      "  Read 1 with adapter:                  18,496 (99.9%)\n",
      "  Read 2 with adapter:                  18,469 (99.8%)\n",
      "Pairs written (passing filters):        18,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,922,234 bp\n",
      "  Read 1:     5,461,116 bp\n",
      "  Read 2:     5,461,118 bp\n",
      "Total written (filtered):      9,961,250 bp (91.2%)\n",
      "  Read 1:     4,980,309 bp\n",
      "  Read 2:     4,980,941 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 18496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t192\t0.0\t2\t0 186 6\n",
      "26\t18187\t0.0\t2\t17795 380 12\n",
      "27\t91\t0.0\t2\t0 85 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 18469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t150\t0.0\t2\t0 147 3\n",
      "26\t18183\t0.0\t2\t17789 378 16\n",
      "27\t99\t0.0\t2\t0 98 1\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1859_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1859_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1859_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1859_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1859_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1859_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1859_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1859_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,526 reads  @    148.9 µs/read;   0.40 M reads/minute\n",
      "Finished in 4.10 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,526\n",
      "  Read 1 with adapter:                  27,492 (99.9%)\n",
      "  Read 2 with adapter:                  27,478 (99.8%)\n",
      "Pairs written (passing filters):        27,526 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,238,784 bp\n",
      "  Read 1:     8,118,627 bp\n",
      "  Read 2:     8,120,157 bp\n",
      "Total written (filtered):     14,809,751 bp (91.2%)\n",
      "  Read 1:     7,403,947 bp\n",
      "  Read 2:     7,405,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t318\t0.0\t2\t0 305 13\n",
      "26\t26978\t0.0\t2\t26438 532 8\n",
      "27\t146\t0.0\t2\t0 138 8\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27478 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t194\t0.0\t2\t0 185 9\n",
      "26\t27153\t0.0\t2\t26593 520 40\n",
      "27\t115\t0.0\t2\t0 107 8\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1860_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1860_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1860_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1860_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1860_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1860_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1860_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1860_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        20,346 reads  @    144.8 µs/read;   0.41 M reads/minute\n",
      "Finished in 2.95 s (145 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,346\n",
      "  Read 1 with adapter:                  20,321 (99.9%)\n",
      "  Read 2 with adapter:                  19,989 (98.2%)\n",
      "Pairs written (passing filters):        20,335 (99.9%)\n",
      "\n",
      "Total basepairs processed:    11,939,494 bp\n",
      "  Read 1:     6,000,579 bp\n",
      "  Read 2:     5,938,915 bp\n",
      "Total written (filtered):     10,888,547 bp (91.2%)\n",
      "  Read 1:     5,469,310 bp\n",
      "  Read 2:     5,419,237 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t214\t0.0\t2\t0 208 6\n",
      "26\t19940\t0.0\t2\t19553 380 7\n",
      "27\t120\t0.0\t2\t0 116 4\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19989 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t113\t0.0\t2\t0 102 11\n",
      "26\t19793\t0.0\t2\t19328 435 30\n",
      "27\t65\t0.0\t2\t0 62 3\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1861_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1861_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1861_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1861_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1861_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1861_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1861_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1861_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             4 reads  @   7701.3 µs/read;   0.01 M reads/minute\n",
      "Finished in 0.03 s (8370 us/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  4\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             4 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,386 bp\n",
      "  Read 1:           796 bp\n",
      "  Read 2:           590 bp\n",
      "Total written (filtered):          1,386 bp (100.0%)\n",
      "  Read 1:           796 bp\n",
      "  Read 2:           590 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1862_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1862_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1862_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1862_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1862_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1862_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1862_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1862_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,686 reads  @    146.0 µs/read;   0.41 M reads/minute\n",
      "Finished in 4.05 s (146 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,686\n",
      "  Read 1 with adapter:                  27,663 (99.9%)\n",
      "  Read 2 with adapter:                  27,595 (99.7%)\n",
      "Pairs written (passing filters):        27,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,327,922 bp\n",
      "  Read 1:     8,166,503 bp\n",
      "  Read 2:     8,161,419 bp\n",
      "Total written (filtered):     14,891,475 bp (91.2%)\n",
      "  Read 1:     7,447,409 bp\n",
      "  Read 2:     7,444,066 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27663 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t323\t0.0\t2\t0 318 5\n",
      "26\t27144\t0.0\t2\t26605 530 9\n",
      "27\t157\t0.0\t2\t0 144 13\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27595 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t247\t0.0\t2\t0 240 7\n",
      "26\t27200\t0.0\t2\t26655 519 26\n",
      "27\t130\t0.0\t2\t0 126 4\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1863_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1863_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1863_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1863_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1863_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1863_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1863_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1863_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:06        41,617 reads  @    149.5 µs/read;   0.40 M reads/minute\n",
      "Finished in 6.23 s (150 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,617\n",
      "  Read 1 with adapter:                  41,570 (99.9%)\n",
      "  Read 2 with adapter:                  41,536 (99.8%)\n",
      "Pairs written (passing filters):        41,617 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,549,912 bp\n",
      "  Read 1:    12,275,440 bp\n",
      "  Read 2:    12,274,472 bp\n",
      "Total written (filtered):     22,389,427 bp (91.2%)\n",
      "  Read 1:    11,194,766 bp\n",
      "  Read 2:    11,194,661 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41570 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t471\t0.0\t2\t0 452 19\n",
      "26\t40777\t0.0\t2\t39947 817 13\n",
      "27\t243\t0.0\t2\t0 235 8\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41536 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t346\t0.0\t2\t0 329 17\n",
      "26\t40949\t0.0\t2\t40179 740 30\n",
      "27\t189\t0.0\t2\t0 180 9\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1864_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1864_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1864_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1864_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1864_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1864_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1864_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1864_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        29,721 reads  @    147.1 µs/read;   0.41 M reads/minute\n",
      "Finished in 4.38 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,721\n",
      "  Read 1 with adapter:                  29,693 (99.9%)\n",
      "  Read 2 with adapter:                  29,649 (99.8%)\n",
      "Pairs written (passing filters):        29,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,533,852 bp\n",
      "  Read 1:     8,767,251 bp\n",
      "  Read 2:     8,766,601 bp\n",
      "Total written (filtered):     15,991,108 bp (91.2%)\n",
      "  Read 1:     7,995,295 bp\n",
      "  Read 2:     7,995,813 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29693 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t323\t0.0\t2\t0 317 6\n",
      "26\t29105\t0.0\t2\t28523 571 11\n",
      "27\t189\t0.0\t2\t0 182 7\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29649 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t219\t0.0\t2\t0 212 7\n",
      "26\t29287\t0.0\t2\t28741 527 19\n",
      "27\t101\t0.0\t2\t0 95 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1865_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1865_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1865_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1865_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1865_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1865_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1865_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1865_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        33,123 reads  @    142.9 µs/read;   0.42 M reads/minute\n",
      "Finished in 4.74 s (143 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,123\n",
      "  Read 1 with adapter:                  33,024 (99.7%)\n",
      "  Read 2 with adapter:                  32,886 (99.3%)\n",
      "Pairs written (passing filters):        33,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,527,388 bp\n",
      "  Read 1:     9,758,841 bp\n",
      "  Read 2:     9,768,547 bp\n",
      "Total written (filtered):     17,813,932 bp (91.2%)\n",
      "  Read 1:     8,900,214 bp\n",
      "  Read 2:     8,913,718 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33024 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t255\t0.0\t2\t0 250 5\n",
      "26\t32509\t0.0\t2\t31932 564 13\n",
      "27\t218\t0.0\t2\t0 210 8\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32886 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t346\t0.0\t2\t0 333 13\n",
      "26\t32360\t0.0\t2\t31779 552 29\n",
      "27\t153\t0.0\t2\t0 147 6\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1866_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1866_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1866_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1866_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1866_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1866_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1866_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1866_115_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        32,391 reads  @    139.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.53 s (140 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,391\n",
      "  Read 1 with adapter:                  32,336 (99.8%)\n",
      "  Read 2 with adapter:                  32,155 (99.3%)\n",
      "Pairs written (passing filters):        32,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,095,794 bp\n",
      "  Read 1:     9,552,899 bp\n",
      "  Read 2:     9,542,895 bp\n",
      "Total written (filtered):     17,419,099 bp (91.2%)\n",
      "  Read 1:     8,712,144 bp\n",
      "  Read 2:     8,706,955 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t243\t0.0\t2\t0 233 10\n",
      "26\t31850\t0.0\t2\t31320 516 14\n",
      "27\t180\t0.0\t2\t0 173 7\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32155 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t247\t0.0\t2\t0 232 15\n",
      "26\t31740\t0.0\t2\t31090 614 36\n",
      "27\t127\t0.0\t2\t0 123 4\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1867_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1867_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1867_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1867_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1867_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1867_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1867_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1867_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        29,052 reads  @    153.9 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.47 s (154 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,052\n",
      "  Read 1 with adapter:                  28,900 (99.5%)\n",
      "  Read 2 with adapter:                  28,993 (99.8%)\n",
      "Pairs written (passing filters):        29,052 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,120,401 bp\n",
      "  Read 1:     8,550,274 bp\n",
      "  Read 2:     8,570,127 bp\n",
      "Total written (filtered):     15,615,188 bp (91.2%)\n",
      "  Read 1:     7,798,863 bp\n",
      "  Read 2:     7,816,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t209\t0.0\t2\t0 202 7\n",
      "26\t28492\t0.0\t2\t27989 493 10\n",
      "27\t146\t0.0\t2\t0 143 3\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28993 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t185\t0.0\t2\t0 179 6\n",
      "26\t28642\t0.0\t2\t28102 517 23\n",
      "27\t135\t0.0\t2\t0 126 9\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1868_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1868_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1868_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1868_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1868_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1868_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1868_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1868_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,276 reads  @    161.9 µs/read;   0.37 M reads/minute\n",
      "Finished in 3.45 s (162 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,276\n",
      "  Read 1 with adapter:                  21,179 (99.5%)\n",
      "  Read 2 with adapter:                  21,065 (99.0%)\n",
      "Pairs written (passing filters):        21,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,512,530 bp\n",
      "  Read 1:     6,265,803 bp\n",
      "  Read 2:     6,246,727 bp\n",
      "Total written (filtered):     11,413,989 bp (91.2%)\n",
      "  Read 1:     5,714,923 bp\n",
      "  Read 2:     5,699,066 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t187\t0.0\t2\t0 179 8\n",
      "26\t20848\t0.0\t2\t20460 379 9\n",
      "27\t116\t0.0\t2\t0 114 2\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t139\t0.0\t2\t0 131 8\n",
      "26\t20813\t0.0\t2\t20278 496 39\n",
      "27\t80\t0.0\t2\t0 76 4\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1869_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1869_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1869_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1869_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1869_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1869_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1869_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1869_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             1 reads  @  32878.9 µs/read;   0.00 M reads/minute\n",
      "Finished in 0.04 s (35979 us/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  1\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           204 bp\n",
      "  Read 1:            16 bp\n",
      "  Read 2:           188 bp\n",
      "Total written (filtered):            204 bp (100.0%)\n",
      "  Read 1:            16 bp\n",
      "  Read 2:           188 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1870_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1870_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1870_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1870_123_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1870_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1870_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1870_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1870_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        33,141 reads  @    143.8 µs/read;   0.42 M reads/minute\n",
      "Finished in 4.77 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,141\n",
      "  Read 1 with adapter:                  33,084 (99.8%)\n",
      "  Read 2 with adapter:                  33,022 (99.6%)\n",
      "Pairs written (passing filters):        33,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,545,392 bp\n",
      "  Read 1:     9,773,059 bp\n",
      "  Read 2:     9,772,333 bp\n",
      "Total written (filtered):     17,826,546 bp (91.2%)\n",
      "  Read 1:     8,912,656 bp\n",
      "  Read 2:     8,913,890 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33084 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t290\t0.0\t2\t0 279 11\n",
      "26\t32537\t0.0\t2\t31954 570 13\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t267\t0.0\t2\t0 262 5\n",
      "26\t32566\t0.0\t2\t31975 566 25\n",
      "27\t168\t0.0\t2\t0 162 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1871_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1871_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1871_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1871_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1871_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1871_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1871_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1871_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        29,673 reads  @    134.9 µs/read;   0.44 M reads/minute\n",
      "Finished in 4.01 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,673\n",
      "  Read 1 with adapter:                  29,615 (99.8%)\n",
      "  Read 2 with adapter:                  29,612 (99.8%)\n",
      "Pairs written (passing filters):        29,673 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,498,203 bp\n",
      "  Read 1:     8,748,048 bp\n",
      "  Read 2:     8,750,155 bp\n",
      "Total written (filtered):     15,958,416 bp (91.2%)\n",
      "  Read 1:     7,978,123 bp\n",
      "  Read 2:     7,980,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29615 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t269\t0.0\t2\t0 254 15\n",
      "26\t29152\t0.0\t2\t28537 599 16\n",
      "27\t128\t0.0\t2\t0 126 2\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29612 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t206\t0.0\t2\t0 205 1\n",
      "26\t29244\t0.0\t2\t28641 572 31\n",
      "27\t140\t0.0\t2\t0 132 8\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1872_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1872_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1872_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1872_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1872_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1872_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1872_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1872_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        31,237 reads  @    140.5 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.39 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,237\n",
      "  Read 1 with adapter:                  31,185 (99.8%)\n",
      "  Read 2 with adapter:                  31,166 (99.8%)\n",
      "Pairs written (passing filters):        31,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,425,471 bp\n",
      "  Read 1:     9,210,787 bp\n",
      "  Read 2:     9,214,684 bp\n",
      "Total written (filtered):     16,804,445 bp (91.2%)\n",
      "  Read 1:     8,400,005 bp\n",
      "  Read 2:     8,404,440 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t245\t0.0\t2\t0 236 9\n",
      "26\t30709\t0.0\t2\t30101 590 18\n",
      "27\t189\t0.0\t2\t0 179 10\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31166 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t250\t0.0\t2\t0 242 8\n",
      "26\t30748\t0.0\t2\t30212 511 25\n",
      "27\t118\t0.0\t2\t0 114 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1873_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1873_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1873_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1873_129_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1873_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1873_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1873_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1873_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,952 reads  @    166.8 µs/read;   0.36 M reads/minute\n",
      "Finished in 3.83 s (167 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,952\n",
      "  Read 1 with adapter:                  22,926 (99.9%)\n",
      "  Read 2 with adapter:                  22,769 (99.2%)\n",
      "Pairs written (passing filters):        22,952 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,538,298 bp\n",
      "  Read 1:     6,770,200 bp\n",
      "  Read 2:     6,768,098 bp\n",
      "Total written (filtered):     12,350,325 bp (91.2%)\n",
      "  Read 1:     6,174,096 bp\n",
      "  Read 2:     6,176,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t206\t0.0\t2\t0 199 7\n",
      "26\t22516\t0.0\t2\t21982 520 14\n",
      "27\t146\t0.0\t2\t0 137 9\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22769 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t246\t0.0\t2\t0 243 3\n",
      "26\t22376\t0.0\t2\t21945 409 22\n",
      "27\t133\t0.0\t2\t0 130 3\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1874_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1874_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1874_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1874_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1874_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1874_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1874_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1874_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,077 reads  @    128.0 µs/read;   0.47 M reads/minute\n",
      "Finished in 5.26 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,077\n",
      "  Read 1 with adapter:                  41,043 (99.9%)\n",
      "  Read 2 with adapter:                  40,838 (99.4%)\n",
      "Pairs written (passing filters):        41,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,233,967 bp\n",
      "  Read 1:    12,116,913 bp\n",
      "  Read 2:    12,117,054 bp\n",
      "Total written (filtered):     22,105,237 bp (91.2%)\n",
      "  Read 1:    11,049,863 bp\n",
      "  Read 2:    11,055,374 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41043 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t376\t0.0\t2\t0 368 8\n",
      "26\t40337\t0.0\t2\t39576 738 23\n",
      "27\t272\t0.0\t2\t0 266 6\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t300\t0.0\t2\t0 291 9\n",
      "26\t40332\t0.0\t2\t39547 747 38\n",
      "27\t152\t0.0\t2\t0 146 6\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1875_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1875_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1875_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1875_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1875_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1875_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1875_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1875_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        36,579 reads  @    122.0 µs/read;   0.49 M reads/minute\n",
      "Finished in 4.47 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,579\n",
      "  Read 1 with adapter:                  36,537 (99.9%)\n",
      "  Read 2 with adapter:                  36,113 (98.7%)\n",
      "Pairs written (passing filters):        36,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,578,969 bp\n",
      "  Read 1:    10,789,061 bp\n",
      "  Read 2:    10,789,908 bp\n",
      "Total written (filtered):     19,690,146 bp (91.2%)\n",
      "  Read 1:     9,839,111 bp\n",
      "  Read 2:     9,851,035 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36537 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t314\t0.0\t2\t0 306 8\n",
      "26\t35914\t0.0\t2\t35257 644 13\n",
      "27\t232\t0.0\t2\t0 219 13\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t245\t0.0\t2\t0 235 10\n",
      "26\t35683\t0.0\t2\t35028 622 33\n",
      "27\t166\t0.0\t2\t0 157 9\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1876_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1876_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1876_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1876_135_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1876_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1876_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1876_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1876_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        27,331 reads  @    138.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 3.80 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,331\n",
      "  Read 1 with adapter:                  27,290 (99.8%)\n",
      "  Read 2 with adapter:                  25,425 (93.0%)\n",
      "Pairs written (passing filters):        27,331 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,779,798 bp\n",
      "  Read 1:     8,062,128 bp\n",
      "  Read 2:     7,717,670 bp\n",
      "Total written (filtered):     14,409,318 bp (91.3%)\n",
      "  Read 1:     7,352,610 bp\n",
      "  Read 2:     7,056,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t222\t0.0\t2\t0 211 11\n",
      "26\t26886\t0.0\t2\t26383 495 8\n",
      "27\t140\t0.0\t2\t0 134 6\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25425 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t184\t0.0\t2\t0 176 8\n",
      "26\t25127\t0.0\t2\t24491 589 47\n",
      "27\t88\t0.0\t2\t0 82 6\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1877_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1877_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1877_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1877_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1877_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1877_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1877_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1877_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            13 reads  @   3145.0 µs/read;   0.02 M reads/minute\n",
      "Finished in 0.04 s (3381 us/read; 0.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 13\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            13 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,668 bp\n",
      "  Read 1:           804 bp\n",
      "  Read 2:           864 bp\n",
      "Total written (filtered):          1,668 bp (100.0%)\n",
      "  Read 1:           804 bp\n",
      "  Read 2:           864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1878_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1878_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1878_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1878_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1878_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1878_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1878_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1878_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,507 reads  @    104.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 2.99 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,507\n",
      "  Read 1 with adapter:                  28,481 (99.9%)\n",
      "  Read 2 with adapter:                  28,436 (99.8%)\n",
      "Pairs written (passing filters):        28,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,815,727 bp\n",
      "  Read 1:     8,409,010 bp\n",
      "  Read 2:     8,406,717 bp\n",
      "Total written (filtered):     15,336,063 bp (91.2%)\n",
      "  Read 1:     7,668,584 bp\n",
      "  Read 2:     7,667,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t281\t0.0\t2\t0 274 7\n",
      "26\t28002\t0.0\t2\t27448 538 16\n",
      "27\t143\t0.0\t2\t0 139 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t267\t0.0\t2\t0 260 7\n",
      "26\t27992\t0.0\t2\t27427 535 30\n",
      "27\t149\t0.0\t2\t0 141 8\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1879_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1879_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1879_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1879_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1879_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1879_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1879_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1879_141_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        26,482 reads  @    148.7 µs/read;   0.40 M reads/minute\n",
      "Finished in 3.94 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,482\n",
      "  Read 1 with adapter:                  26,445 (99.9%)\n",
      "  Read 2 with adapter:                  26,380 (99.6%)\n",
      "Pairs written (passing filters):        26,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,623,129 bp\n",
      "  Read 1:     7,811,879 bp\n",
      "  Read 2:     7,811,250 bp\n",
      "Total written (filtered):     14,249,732 bp (91.2%)\n",
      "  Read 1:     7,124,322 bp\n",
      "  Read 2:     7,125,410 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t250\t0.0\t2\t0 238 12\n",
      "26\t25982\t0.0\t2\t25559 418 5\n",
      "27\t157\t0.0\t2\t0 147 10\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t195\t0.0\t2\t0 188 7\n",
      "26\t26041\t0.0\t2\t25482 521 38\n",
      "27\t109\t0.0\t2\t0 107 2\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1880_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1880_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1880_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1880_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1880_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1880_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1880_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1880_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        25,294 reads  @    140.9 µs/read;   0.43 M reads/minute\n",
      "Finished in 3.57 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,294\n",
      "  Read 1 with adapter:                  25,271 (99.9%)\n",
      "  Read 2 with adapter:                  25,231 (99.8%)\n",
      "Pairs written (passing filters):        25,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,921,913 bp\n",
      "  Read 1:     7,460,464 bp\n",
      "  Read 2:     7,461,449 bp\n",
      "Total written (filtered):     13,608,955 bp (91.2%)\n",
      "  Read 1:     6,803,451 bp\n",
      "  Read 2:     6,805,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 25271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t263\t0.0\t2\t0 259 4\n",
      "26\t24792\t0.0\t2\t24339 444 9\n",
      "27\t178\t0.0\t2\t0 169 9\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 25231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t196\t0.0\t2\t0 186 10\n",
      "26\t24900\t0.0\t2\t24408 466 26\n",
      "27\t99\t0.0\t2\t0 96 3\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1881_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1881_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1881_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1881_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1881_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1881_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1881_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1881_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        15,718 reads  @    172.7 µs/read;   0.35 M reads/minute\n",
      "Finished in 2.72 s (173 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,718\n",
      "  Read 1 with adapter:                  15,703 (99.9%)\n",
      "  Read 2 with adapter:                  15,607 (99.3%)\n",
      "Pairs written (passing filters):        15,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,272,240 bp\n",
      "  Read 1:     4,636,805 bp\n",
      "  Read 2:     4,635,435 bp\n",
      "Total written (filtered):      8,458,277 bp (91.2%)\n",
      "  Read 1:     4,228,513 bp\n",
      "  Read 2:     4,229,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t138\t0.0\t2\t0 133 5\n",
      "26\t15436\t0.0\t2\t15086 341 9\n",
      "27\t82\t0.0\t2\t0 80 2\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t157\t0.0\t2\t0 153 4\n",
      "26\t15383\t0.0\t2\t15120 254 9\n",
      "27\t52\t0.0\t2\t0 51 1\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1882_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1882_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1882_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1882_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1882_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1882_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1882_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1882_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        24,614 reads  @    134.5 µs/read;   0.45 M reads/minute\n",
      "Finished in 3.31 s (135 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,614\n",
      "  Read 1 with adapter:                  24,589 (99.9%)\n",
      "  Read 2 with adapter:                  24,476 (99.4%)\n",
      "Pairs written (passing filters):        24,614 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,520,562 bp\n",
      "  Read 1:     7,260,202 bp\n",
      "  Read 2:     7,260,360 bp\n",
      "Total written (filtered):     13,244,972 bp (91.2%)\n",
      "  Read 1:     6,620,907 bp\n",
      "  Read 2:     6,624,065 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t203\t0.0\t2\t0 199 4\n",
      "26\t24197\t0.0\t2\t23721 462 14\n",
      "27\t158\t0.0\t2\t0 149 9\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24476 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t185\t0.0\t2\t0 177 8\n",
      "26\t24159\t0.0\t2\t23735 403 21\n",
      "27\t116\t0.0\t2\t0 109 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1883_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1883_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1883_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1883_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1883_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1883_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1883_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1883_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        31,367 reads  @    190.1 µs/read;   0.32 M reads/minute\n",
      "Finished in 5.97 s (190 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,367\n",
      "  Read 1 with adapter:                  31,333 (99.9%)\n",
      "  Read 2 with adapter:                  31,258 (99.7%)\n",
      "Pairs written (passing filters):        31,367 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,504,922 bp\n",
      "  Read 1:     9,252,165 bp\n",
      "  Read 2:     9,252,757 bp\n",
      "Total written (filtered):     16,877,641 bp (91.2%)\n",
      "  Read 1:     8,437,543 bp\n",
      "  Read 2:     8,440,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31333 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t260\t0.0\t2\t0 250 10\n",
      "26\t30871\t0.0\t2\t30240 617 14\n",
      "27\t152\t0.0\t2\t0 146 6\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31258 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t197\t0.0\t2\t0 187 10\n",
      "26\t30904\t0.0\t2\t30209 657 38\n",
      "27\t118\t0.0\t2\t0 112 6\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1884_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1884_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1884_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1884_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1884_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1884_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1884_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1884_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,655 reads  @    173.4 µs/read;   0.35 M reads/minute\n",
      "Finished in 3.76 s (174 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,655\n",
      "  Read 1 with adapter:                  21,629 (99.9%)\n",
      "  Read 2 with adapter:                  21,236 (98.1%)\n",
      "Pairs written (passing filters):        21,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,708,199 bp\n",
      "  Read 1:     6,387,299 bp\n",
      "  Read 2:     6,320,900 bp\n",
      "Total written (filtered):     11,593,742 bp (91.2%)\n",
      "  Read 1:     5,824,968 bp\n",
      "  Read 2:     5,768,774 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21629 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t153\t0.0\t2\t0 149 4\n",
      "26\t21332\t0.0\t2\t20960 364 8\n",
      "27\t94\t0.0\t2\t0 93 1\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t150\t0.0\t2\t0 139 11\n",
      "26\t20946\t0.0\t2\t20380 521 45\n",
      "27\t124\t0.0\t2\t0 120 4\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1885_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1885_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1885_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1885_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1885_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1885_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1885_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1885_153_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00             2 reads  @  25252.1 µs/read;   0.00 M reads/minute\n",
      "Finished in 0.06 s (28665 us/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                  2\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):             2 (100.0%)\n",
      "\n",
      "Total basepairs processed:           188 bp\n",
      "  Read 1:           128 bp\n",
      "  Read 2:            60 bp\n",
      "Total written (filtered):            188 bp (100.0%)\n",
      "  Read 1:           128 bp\n",
      "  Read 2:            60 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1886_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1886_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1886_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1886_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1886_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1886_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1886_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1886_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        31,462 reads  @    146.5 µs/read;   0.41 M reads/minute\n",
      "Finished in 4.61 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,462\n",
      "  Read 1 with adapter:                  31,434 (99.9%)\n",
      "  Read 2 with adapter:                  31,336 (99.6%)\n",
      "Pairs written (passing filters):        31,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,553,493 bp\n",
      "  Read 1:     9,281,180 bp\n",
      "  Read 2:     9,272,313 bp\n",
      "Total written (filtered):     16,921,641 bp (91.2%)\n",
      "  Read 1:     8,463,957 bp\n",
      "  Read 2:     8,457,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t259\t0.0\t2\t0 248 11\n",
      "26\t30974\t0.0\t2\t30452 513 9\n",
      "27\t156\t0.0\t2\t0 152 4\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t253\t0.0\t2\t0 240 13\n",
      "26\t30926\t0.0\t2\t30305 585 36\n",
      "27\t120\t0.0\t2\t0 117 3\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1887_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1887_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1887_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1887_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1887_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1887_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1887_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1887_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01         9,828 reads  @    147.2 µs/read;   0.41 M reads/minute\n",
      "Finished in 1.45 s (148 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,828\n",
      "  Read 1 with adapter:                   9,784 (99.6%)\n",
      "  Read 2 with adapter:                   9,794 (99.7%)\n",
      "Pairs written (passing filters):         9,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,797,167 bp\n",
      "  Read 1:     2,899,100 bp\n",
      "  Read 2:     2,898,067 bp\n",
      "Total written (filtered):      5,288,173 bp (91.2%)\n",
      "  Read 1:     2,644,711 bp\n",
      "  Read 2:     2,643,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 9784 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t83\t0.0\t2\t0 80 3\n",
      "26\t9622\t0.0\t2\t9375 243 4\n",
      "27\t50\t0.0\t2\t0 48 2\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 9794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t101\t0.0\t2\t0 100 1\n",
      "26\t9625\t0.0\t2\t9397 217 11\n",
      "27\t58\t0.0\t2\t0 55 3\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1888_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1888_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1888_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1888_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1888_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1888_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1888_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1888_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        31,302 reads  @    188.5 µs/read;   0.32 M reads/minute\n",
      "Finished in 5.91 s (189 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,302\n",
      "  Read 1 with adapter:                  31,261 (99.9%)\n",
      "  Read 2 with adapter:                  31,243 (99.8%)\n",
      "Pairs written (passing filters):        31,302 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,466,870 bp\n",
      "  Read 1:     9,232,826 bp\n",
      "  Read 2:     9,234,044 bp\n",
      "Total written (filtered):     16,841,904 bp (91.2%)\n",
      "  Read 1:     8,420,058 bp\n",
      "  Read 2:     8,421,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t260\t0.0\t2\t0 249 11\n",
      "26\t30793\t0.0\t2\t30147 629 17\n",
      "27\t154\t0.0\t2\t0 147 7\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t274\t0.0\t2\t0 264 10\n",
      "26\t30797\t0.0\t2\t30151 607 39\n",
      "27\t138\t0.0\t2\t0 129 9\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1897_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1897_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1897_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1897_161_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1897_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1897_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1897_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1897_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,300 reads  @    151.8 µs/read;   0.40 M reads/minute\n",
      "Finished in 5.51 s (152 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,300\n",
      "  Read 1 with adapter:                  36,264 (99.9%)\n",
      "  Read 2 with adapter:                  36,225 (99.8%)\n",
      "Pairs written (passing filters):        36,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,415,390 bp\n",
      "  Read 1:    10,708,231 bp\n",
      "  Read 2:    10,707,159 bp\n",
      "Total written (filtered):     19,531,187 bp (91.2%)\n",
      "  Read 1:     9,765,686 bp\n",
      "  Read 2:     9,765,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t450\t0.0\t2\t0 436 14\n",
      "26\t35653\t0.0\t2\t35001 645 7\n",
      "27\t131\t0.0\t2\t0 126 5\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t387\t0.0\t2\t0 375 12\n",
      "26\t35594\t0.0\t2\t34961 606 27\n",
      "27\t213\t0.0\t2\t0 204 9\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1898_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1898_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1898_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1898_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1898_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1898_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1898_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1898_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,804 reads  @    155.7 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.18 s (156 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,804\n",
      "  Read 1 with adapter:                  26,784 (99.9%)\n",
      "  Read 2 with adapter:                  26,747 (99.8%)\n",
      "Pairs written (passing filters):        26,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,813,518 bp\n",
      "  Read 1:     7,907,167 bp\n",
      "  Read 2:     7,906,351 bp\n",
      "Total written (filtered):     14,421,934 bp (91.2%)\n",
      "  Read 1:     7,210,940 bp\n",
      "  Read 2:     7,210,994 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26784 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t306\t0.0\t2\t0 290 16\n",
      "26\t26317\t0.0\t2\t25814 495 8\n",
      "27\t125\t0.0\t2\t0 119 6\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t201\t0.0\t2\t0 195 6\n",
      "26\t26406\t0.0\t2\t25858 520 28\n",
      "27\t120\t0.0\t2\t0 115 5\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1899_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1899_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1899_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1899_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1899_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1899_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1899_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1899_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        35,130 reads  @    145.2 µs/read;   0.41 M reads/minute\n",
      "Finished in 5.10 s (145 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,130\n",
      "  Read 1 with adapter:                  35,108 (99.9%)\n",
      "  Read 2 with adapter:                  35,025 (99.7%)\n",
      "Pairs written (passing filters):        35,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,725,893 bp\n",
      "  Read 1:    10,362,824 bp\n",
      "  Read 2:    10,363,069 bp\n",
      "Total written (filtered):     18,902,808 bp (91.2%)\n",
      "  Read 1:     9,450,410 bp\n",
      "  Read 2:     9,452,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35108 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t497\t0.0\t2\t0 481 16\n",
      "26\t34456\t0.0\t2\t33784 662 10\n",
      "27\t131\t0.0\t2\t0 123 8\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t261\t0.0\t2\t0 248 13\n",
      "26\t34502\t0.0\t2\t33656 796 50\n",
      "27\t182\t0.0\t2\t0 178 4\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1900_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1900_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1900_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1900_167_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1900_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1900_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1900_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1900_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        31,802 reads  @    157.9 µs/read;   0.38 M reads/minute\n",
      "Finished in 5.03 s (158 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,802\n",
      "  Read 1 with adapter:                  31,766 (99.9%)\n",
      "  Read 2 with adapter:                  31,606 (99.4%)\n",
      "Pairs written (passing filters):        31,802 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,761,429 bp\n",
      "  Read 1:     9,380,337 bp\n",
      "  Read 2:     9,381,092 bp\n",
      "Total written (filtered):     17,113,980 bp (91.2%)\n",
      "  Read 1:     8,554,565 bp\n",
      "  Read 2:     8,559,415 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t334\t0.0\t2\t0 325 9\n",
      "26\t31225\t0.0\t2\t30635 584 6\n",
      "27\t156\t0.0\t2\t0 151 5\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t235\t0.0\t2\t0 226 9\n",
      "26\t31195\t0.0\t2\t30661 503 31\n",
      "27\t144\t0.0\t2\t0 139 5\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1901_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1901_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1901_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1901_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1901_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1901_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1901_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1901_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        32,699 reads  @    145.0 µs/read;   0.41 M reads/minute\n",
      "Finished in 4.75 s (145 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,699\n",
      "  Read 1 with adapter:                  32,670 (99.9%)\n",
      "  Read 2 with adapter:                  32,625 (99.8%)\n",
      "Pairs written (passing filters):        32,699 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,291,540 bp\n",
      "  Read 1:     9,645,932 bp\n",
      "  Read 2:     9,645,608 bp\n",
      "Total written (filtered):     17,594,131 bp (91.2%)\n",
      "  Read 1:     8,796,609 bp\n",
      "  Read 2:     8,797,522 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32670 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t389\t0.0\t2\t0 381 8\n",
      "26\t32006\t0.0\t2\t31385 609 12\n",
      "27\t198\t0.0\t2\t0 188 10\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32625 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t352\t0.0\t2\t0 334 18\n",
      "26\t32075\t0.0\t2\t31433 608 34\n",
      "27\t152\t0.0\t2\t0 145 7\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1902_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1902_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1902_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1902_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1902_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1902_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1902_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1902_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        35,958 reads  @    153.3 µs/read;   0.39 M reads/minute\n",
      "Finished in 5.52 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,958\n",
      "  Read 1 with adapter:                  35,923 (99.9%)\n",
      "  Read 2 with adapter:                  35,874 (99.8%)\n",
      "Pairs written (passing filters):        35,958 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,213,377 bp\n",
      "  Read 1:    10,607,333 bp\n",
      "  Read 2:    10,606,044 bp\n",
      "Total written (filtered):     19,346,868 bp (91.2%)\n",
      "  Read 1:     9,673,474 bp\n",
      "  Read 2:     9,673,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35923 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t423\t0.0\t2\t0 410 13\n",
      "26\t35192\t0.0\t2\t34532 646 14\n",
      "27\t232\t0.0\t2\t0 224 8\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t258\t0.0\t2\t0 252 6\n",
      "26\t35412\t0.0\t2\t34706 661 45\n",
      "27\t156\t0.0\t2\t0 149 7\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1903_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1903_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1903_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1903_173_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1903_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1903_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1903_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1903_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        11,576 reads  @    156.4 µs/read;   0.38 M reads/minute\n",
      "Finished in 1.82 s (157 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,576\n",
      "  Read 1 with adapter:                  11,511 (99.4%)\n",
      "  Read 2 with adapter:                  11,514 (99.5%)\n",
      "Pairs written (passing filters):        11,576 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,822,874 bp\n",
      "  Read 1:     3,414,659 bp\n",
      "  Read 2:     3,408,215 bp\n",
      "Total written (filtered):      6,224,304 bp (91.2%)\n",
      "  Read 1:     3,115,447 bp\n",
      "  Read 2:     3,108,857 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 11511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t160\t0.0\t2\t0 154 6\n",
      "26\t11257\t0.0\t2\t11045 208 4\n",
      "27\t62\t0.0\t2\t0 60 2\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 11514 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t90\t0.0\t2\t0 86 4\n",
      "26\t11331\t0.0\t2\t11113 200 18\n",
      "27\t82\t0.0\t2\t0 80 2\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1904_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1904_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1904_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1904_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1904_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1904_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1904_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1904_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        50,376 reads  @    170.0 µs/read;   0.35 M reads/minute\n",
      "Finished in 8.57 s (170 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,376\n",
      "  Read 1 with adapter:                  50,335 (99.9%)\n",
      "  Read 2 with adapter:                  50,197 (99.6%)\n",
      "Pairs written (passing filters):        50,376 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,719,262 bp\n",
      "  Read 1:    14,860,632 bp\n",
      "  Read 2:    14,858,630 bp\n",
      "Total written (filtered):     27,105,886 bp (91.2%)\n",
      "  Read 1:    13,552,285 bp\n",
      "  Read 2:    13,553,601 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50335 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t654\t0.0\t2\t0 628 26\n",
      "26\t49317\t0.0\t2\t48286 1005 26\n",
      "27\t269\t0.0\t2\t0 261 8\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t412\t0.0\t2\t0 397 15\n",
      "26\t49451\t0.0\t2\t48411 981 59\n",
      "27\t269\t0.0\t2\t0 257 12\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1905_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1905_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1905_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1905_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1905_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1905_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1905_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1905_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        27,246 reads  @    135.5 µs/read;   0.44 M reads/minute\n",
      "Finished in 3.70 s (136 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,246\n",
      "  Read 1 with adapter:                  27,217 (99.9%)\n",
      "  Read 2 with adapter:                  27,175 (99.7%)\n",
      "Pairs written (passing filters):        27,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,073,646 bp\n",
      "  Read 1:     8,037,433 bp\n",
      "  Read 2:     8,036,213 bp\n",
      "Total written (filtered):     14,659,812 bp (91.2%)\n",
      "  Read 1:     7,330,015 bp\n",
      "  Read 2:     7,329,797 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27217 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t310\t0.0\t2\t0 298 12\n",
      "26\t26798\t0.0\t2\t26351 440 7\n",
      "27\t92\t0.0\t2\t0 91 1\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27175 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t285\t0.0\t2\t0 270 15\n",
      "26\t26731\t0.0\t2\t26186 513 32\n",
      "27\t103\t0.0\t2\t0 100 3\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1906_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1906_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1906_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1906_179_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1906_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1906_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1906_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1906_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        20,124 reads  @    168.5 µs/read;   0.36 M reads/minute\n",
      "Finished in 3.40 s (169 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,124\n",
      "  Read 1 with adapter:                  20,076 (99.8%)\n",
      "  Read 2 with adapter:                  20,074 (99.8%)\n",
      "Pairs written (passing filters):        20,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,868,780 bp\n",
      "  Read 1:     5,933,911 bp\n",
      "  Read 2:     5,934,869 bp\n",
      "Total written (filtered):     10,825,043 bp (91.2%)\n",
      "  Read 1:     5,412,059 bp\n",
      "  Read 2:     5,412,984 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t198\t0.0\t2\t0 195 3\n",
      "26\t19789\t0.0\t2\t19421 362 6\n",
      "27\t72\t0.0\t2\t0 69 3\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t154\t0.0\t2\t0 150 4\n",
      "26\t19786\t0.0\t2\t19394 374 18\n",
      "27\t97\t0.0\t2\t0 93 4\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1907_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1907_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1907_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1907_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1907_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1907_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1907_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1907_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,196 reads  @    159.0 µs/read;   0.38 M reads/minute\n",
      "Finished in 4.17 s (159 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,196\n",
      "  Read 1 with adapter:                  26,130 (99.7%)\n",
      "  Read 2 with adapter:                  26,139 (99.8%)\n",
      "Pairs written (passing filters):        26,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,451,294 bp\n",
      "  Read 1:     7,723,886 bp\n",
      "  Read 2:     7,727,408 bp\n",
      "Total written (filtered):     14,092,442 bp (91.2%)\n",
      "  Read 1:     7,044,604 bp\n",
      "  Read 2:     7,047,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26130 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t256\t0.0\t2\t0 250 6\n",
      "26\t25704\t0.0\t2\t25187 508 9\n",
      "27\t126\t0.0\t2\t0 120 6\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t164\t0.0\t2\t0 158 6\n",
      "26\t25819\t0.0\t2\t25332 453 34\n",
      "27\t132\t0.0\t2\t0 129 3\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1908_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1908_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1908_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1908_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1908_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1908_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1908_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1908_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,884 reads  @    172.0 µs/read;   0.35 M reads/minute\n",
      "Finished in 3.94 s (172 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,884\n",
      "  Read 1 with adapter:                  22,853 (99.9%)\n",
      "  Read 2 with adapter:                  22,812 (99.7%)\n",
      "Pairs written (passing filters):        22,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,499,906 bp\n",
      "  Read 1:     6,749,572 bp\n",
      "  Read 2:     6,750,334 bp\n",
      "Total written (filtered):     12,312,691 bp (91.2%)\n",
      "  Read 1:     6,155,433 bp\n",
      "  Read 2:     6,157,258 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22853 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t201\t0.0\t2\t0 196 5\n",
      "26\t22499\t0.0\t2\t22070 425 4\n",
      "27\t116\t0.0\t2\t0 113 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t161\t0.0\t2\t0 157 4\n",
      "26\t22521\t0.0\t2\t22099 396 26\n",
      "27\t99\t0.0\t2\t0 95 4\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1909_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1909_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1909_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1909_185_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1909_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1909_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1909_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1909_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        24,020 reads  @    129.8 µs/read;   0.46 M reads/minute\n",
      "Finished in 3.12 s (130 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,020\n",
      "  Read 1 with adapter:                  23,992 (99.9%)\n",
      "  Read 2 with adapter:                  23,969 (99.8%)\n",
      "Pairs written (passing filters):        24,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,171,154 bp\n",
      "  Read 1:     7,085,273 bp\n",
      "  Read 2:     7,085,881 bp\n",
      "Total written (filtered):     12,924,407 bp (91.2%)\n",
      "  Read 1:     6,461,614 bp\n",
      "  Read 2:     6,462,793 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t286\t0.0\t2\t0 283 3\n",
      "26\t23542\t0.0\t2\t23162 375 5\n",
      "27\t131\t0.0\t2\t0 128 3\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t218\t0.0\t2\t0 212 6\n",
      "26\t23607\t0.0\t2\t23116 452 39\n",
      "27\t100\t0.0\t2\t0 96 4\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1910_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1910_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1910_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1910_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1910_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1910_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1910_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1910_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        24,137 reads  @    143.6 µs/read;   0.42 M reads/minute\n",
      "Finished in 3.47 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,137\n",
      "  Read 1 with adapter:                  24,111 (99.9%)\n",
      "  Read 2 with adapter:                  24,103 (99.9%)\n",
      "Pairs written (passing filters):        24,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,239,754 bp\n",
      "  Read 1:     7,120,137 bp\n",
      "  Read 2:     7,119,617 bp\n",
      "Total written (filtered):     12,986,361 bp (91.2%)\n",
      "  Read 1:     6,493,381 bp\n",
      "  Read 2:     6,492,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24111 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t274\t0.0\t2\t0 267 7\n",
      "26\t23672\t0.0\t2\t23221 437 14\n",
      "27\t118\t0.0\t2\t0 112 6\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24103 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t168\t0.0\t2\t0 162 6\n",
      "26\t23796\t0.0\t2\t23330 441 25\n",
      "27\t123\t0.0\t2\t0 115 8\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1911_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1911_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1911_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1911_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1911_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1911_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1911_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1911_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,311 reads  @    180.5 µs/read;   0.33 M reads/minute\n",
      "Finished in 4.75 s (181 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,311\n",
      "  Read 1 with adapter:                  26,281 (99.9%)\n",
      "  Read 2 with adapter:                  26,239 (99.7%)\n",
      "Pairs written (passing filters):        26,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,516,887 bp\n",
      "  Read 1:     7,761,314 bp\n",
      "  Read 2:     7,755,573 bp\n",
      "Total written (filtered):     14,151,509 bp (91.2%)\n",
      "  Read 1:     7,078,147 bp\n",
      "  Read 2:     7,073,362 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26281 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t303\t0.0\t2\t0 298 5\n",
      "26\t25792\t0.0\t2\t25317 463 12\n",
      "27\t144\t0.0\t2\t0 135 9\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t176\t0.0\t2\t0 169 7\n",
      "26\t25900\t0.0\t2\t25374 498 28\n",
      "27\t129\t0.0\t2\t0 124 5\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1912_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1912_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1912_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1912_191_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1912_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1912_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1912_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1912_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        30,719 reads  @    152.9 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.70 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,719\n",
      "  Read 1 with adapter:                  30,675 (99.9%)\n",
      "  Read 2 with adapter:                  30,630 (99.7%)\n",
      "Pairs written (passing filters):        30,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,122,329 bp\n",
      "  Read 1:     9,060,734 bp\n",
      "  Read 2:     9,061,595 bp\n",
      "Total written (filtered):     16,528,531 bp (91.2%)\n",
      "  Read 1:     8,263,304 bp\n",
      "  Read 2:     8,265,227 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t361\t0.0\t2\t0 352 9\n",
      "26\t30072\t0.0\t2\t29540 516 16\n",
      "27\t167\t0.0\t2\t0 157 10\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t230\t0.0\t2\t0 218 12\n",
      "26\t30203\t0.0\t2\t29544 614 45\n",
      "27\t144\t0.0\t2\t0 139 5\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1913_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1913_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1913_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1913_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1913_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1913_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1913_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1913_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        30,261 reads  @    159.2 µs/read;   0.38 M reads/minute\n",
      "Finished in 4.82 s (159 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,261\n",
      "  Read 1 with adapter:                  30,241 (99.9%)\n",
      "  Read 2 with adapter:                  30,192 (99.8%)\n",
      "Pairs written (passing filters):        30,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,852,903 bp\n",
      "  Read 1:     8,926,839 bp\n",
      "  Read 2:     8,926,064 bp\n",
      "Total written (filtered):     16,281,948 bp (91.2%)\n",
      "  Read 1:     8,140,717 bp\n",
      "  Read 2:     8,141,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t364\t0.0\t2\t0 355 9\n",
      "26\t29630\t0.0\t2\t29151 469 10\n",
      "27\t182\t0.0\t2\t0 178 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t318\t0.0\t2\t0 308 10\n",
      "26\t29676\t0.0\t2\t29060 585 31\n",
      "27\t173\t0.0\t2\t0 164 9\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1914_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1914_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1914_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1914_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1914_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1914_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1914_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1914_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        22,829 reads  @    130.3 µs/read;   0.46 M reads/minute\n",
      "Finished in 2.98 s (131 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,829\n",
      "  Read 1 with adapter:                  22,812 (99.9%)\n",
      "  Read 2 with adapter:                  22,737 (99.6%)\n",
      "Pairs written (passing filters):        22,829 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,467,016 bp\n",
      "  Read 1:     6,734,069 bp\n",
      "  Read 2:     6,732,947 bp\n",
      "Total written (filtered):     12,282,892 bp (91.2%)\n",
      "  Read 1:     6,141,090 bp\n",
      "  Read 2:     6,141,802 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t268\t0.0\t2\t0 259 9\n",
      "26\t22386\t0.0\t2\t21988 384 14\n",
      "27\t121\t0.0\t2\t0 116 5\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t156\t0.0\t2\t0 147 9\n",
      "26\t22431\t0.0\t2\t21929 463 39\n",
      "27\t121\t0.0\t2\t0 115 6\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1915_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1915_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1915_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1915_197_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1915_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1915_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1915_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1915_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        30,389 reads  @    186.4 µs/read;   0.32 M reads/minute\n",
      "Finished in 5.67 s (187 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,389\n",
      "  Read 1 with adapter:                  30,363 (99.9%)\n",
      "  Read 2 with adapter:                  30,315 (99.8%)\n",
      "Pairs written (passing filters):        30,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,928,651 bp\n",
      "  Read 1:     8,964,709 bp\n",
      "  Read 2:     8,963,942 bp\n",
      "Total written (filtered):     16,351,137 bp (91.2%)\n",
      "  Read 1:     8,175,376 bp\n",
      "  Read 2:     8,175,761 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t367\t0.0\t2\t0 354 13\n",
      "26\t29763\t0.0\t2\t29230 533\n",
      "27\t168\t0.0\t2\t0 162 6\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t218\t0.0\t2\t0 207 11\n",
      "26\t29865\t0.0\t2\t29209 609 47\n",
      "27\t183\t0.0\t2\t0 167 16\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1916_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1916_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1916_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1916_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1916_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1916_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1916_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1916_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        23,813 reads  @    149.6 µs/read;   0.40 M reads/minute\n",
      "Finished in 3.57 s (150 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,813\n",
      "  Read 1 with adapter:                  23,796 (99.9%)\n",
      "  Read 2 with adapter:                  23,751 (99.7%)\n",
      "Pairs written (passing filters):        23,813 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,048,184 bp\n",
      "  Read 1:     7,024,684 bp\n",
      "  Read 2:     7,023,500 bp\n",
      "Total written (filtered):     12,812,088 bp (91.2%)\n",
      "  Read 1:     6,406,072 bp\n",
      "  Read 2:     6,406,016 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23796 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t254\t0.0\t2\t0 244 10\n",
      "26\t23367\t0.0\t2\t22941 419 7\n",
      "27\t116\t0.0\t2\t0 110 6\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t163\t0.0\t2\t0 155 8\n",
      "26\t23453\t0.0\t2\t22965 457 31\n",
      "27\t101\t0.0\t2\t0 96 5\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1917_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1917_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1917_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1917_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1917_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1917_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1917_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1917_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,483 reads  @    176.6 µs/read;   0.34 M reads/minute\n",
      "Finished in 4.86 s (177 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,483\n",
      "  Read 1 with adapter:                  27,459 (99.9%)\n",
      "  Read 2 with adapter:                  27,400 (99.7%)\n",
      "Pairs written (passing filters):        27,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,212,180 bp\n",
      "  Read 1:     8,107,357 bp\n",
      "  Read 2:     8,104,823 bp\n",
      "Total written (filtered):     14,786,117 bp (91.2%)\n",
      "  Read 1:     7,393,564 bp\n",
      "  Read 2:     7,392,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t281\t0.0\t2\t0 272 9\n",
      "26\t27020\t0.0\t2\t26518 493 9\n",
      "27\t132\t0.0\t2\t0 128 4\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t266\t0.0\t2\t0 253 13\n",
      "26\t26968\t0.0\t2\t26432 505 31\n",
      "27\t148\t0.0\t2\t0 142 6\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1918_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1918_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1918_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1918_203_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1918_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1918_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1918_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1918_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,714 reads  @    155.0 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.15 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,714\n",
      "  Read 1 with adapter:                  26,696 (99.9%)\n",
      "  Read 2 with adapter:                  26,562 (99.4%)\n",
      "Pairs written (passing filters):        26,714 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,760,338 bp\n",
      "  Read 1:     7,880,206 bp\n",
      "  Read 2:     7,880,132 bp\n",
      "Total written (filtered):     14,375,718 bp (91.2%)\n",
      "  Read 1:     7,186,139 bp\n",
      "  Read 2:     7,189,579 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26696 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t225\t0.0\t2\t0 219 6\n",
      "26\t26277\t0.0\t2\t25833 436 8\n",
      "27\t160\t0.0\t2\t0 157 3\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26562 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t179\t0.0\t2\t0 176 3\n",
      "26\t26247\t0.0\t2\t25734 490 23\n",
      "27\t128\t0.0\t2\t0 123 5\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1919_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1919_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1919_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1919_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1919_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1919_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1919_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1919_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        28,151 reads  @    168.2 µs/read;   0.36 M reads/minute\n",
      "Finished in 4.75 s (169 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,151\n",
      "  Read 1 with adapter:                  28,063 (99.7%)\n",
      "  Read 2 with adapter:                  28,081 (99.8%)\n",
      "Pairs written (passing filters):        28,151 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,608,502 bp\n",
      "  Read 1:     8,304,250 bp\n",
      "  Read 2:     8,304,252 bp\n",
      "Total written (filtered):     15,148,887 bp (91.2%)\n",
      "  Read 1:     7,574,725 bp\n",
      "  Read 2:     7,574,162 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28063 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t336\t0.0\t2\t0 327 9\n",
      "26\t27518\t0.0\t2\t26907 604 7\n",
      "27\t151\t0.0\t2\t0 144 7\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28081 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t214\t0.0\t2\t0 206 8\n",
      "26\t27660\t0.0\t2\t27060 571 29\n",
      "27\t188\t0.0\t2\t0 180 8\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1920_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1920_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1920_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1920_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1920_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1920_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1920_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1920_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,666 reads  @    147.0 µs/read;   0.41 M reads/minute\n",
      "Finished in 3.35 s (148 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,666\n",
      "  Read 1 with adapter:                  22,643 (99.9%)\n",
      "  Read 2 with adapter:                  22,595 (99.7%)\n",
      "Pairs written (passing filters):        22,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,372,749 bp\n",
      "  Read 1:     6,686,319 bp\n",
      "  Read 2:     6,686,430 bp\n",
      "Total written (filtered):     12,196,645 bp (91.2%)\n",
      "  Read 1:     6,097,652 bp\n",
      "  Read 2:     6,098,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t214\t0.0\t2\t0 208 6\n",
      "26\t22276\t0.0\t2\t21876 390 10\n",
      "27\t123\t0.0\t2\t0 116 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22595 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t170\t0.0\t2\t0 165 5\n",
      "26\t22294\t0.0\t2\t21873 398 23\n",
      "27\t97\t0.0\t2\t0 94 3\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1921_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1921_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1921_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1921_209_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1921_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1921_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1921_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1921_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        28,235 reads  @    138.6 µs/read;   0.43 M reads/minute\n",
      "Finished in 3.92 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,235\n",
      "  Read 1 with adapter:                  28,177 (99.8%)\n",
      "  Read 2 with adapter:                  28,161 (99.7%)\n",
      "Pairs written (passing filters):        28,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,650,091 bp\n",
      "  Read 1:     8,321,565 bp\n",
      "  Read 2:     8,328,526 bp\n",
      "Total written (filtered):     15,185,501 bp (91.2%)\n",
      "  Read 1:     7,588,998 bp\n",
      "  Read 2:     7,596,503 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28177 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t293\t0.0\t2\t0 278 15\n",
      "26\t27660\t0.0\t2\t27172 478 10\n",
      "27\t174\t0.0\t2\t0 167 7\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t299\t0.0\t2\t0 289 10\n",
      "26\t27706\t0.0\t2\t27218 463 25\n",
      "27\t144\t0.0\t2\t0 135 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1922_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1922_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1922_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1922_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1922_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1922_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1922_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1922_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        26,412 reads  @    155.7 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.12 s (156 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,412\n",
      "  Read 1 with adapter:                  26,378 (99.9%)\n",
      "  Read 2 with adapter:                  26,287 (99.5%)\n",
      "Pairs written (passing filters):        26,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,579,776 bp\n",
      "  Read 1:     7,788,454 bp\n",
      "  Read 2:     7,791,322 bp\n",
      "Total written (filtered):     14,210,597 bp (91.2%)\n",
      "  Read 1:     7,102,649 bp\n",
      "  Read 2:     7,107,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t241\t0.0\t2\t0 237 4\n",
      "26\t25887\t0.0\t2\t25442 432 13\n",
      "27\t198\t0.0\t2\t0 192 6\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t195\t0.0\t2\t0 183 12\n",
      "26\t25972\t0.0\t2\t25446 497 29\n",
      "27\t93\t0.0\t2\t0 89 4\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1923_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1923_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1923_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1923_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1923_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1923_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1923_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1923_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        29,635 reads  @    151.0 µs/read;   0.40 M reads/minute\n",
      "Finished in 4.48 s (151 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,635\n",
      "  Read 1 with adapter:                  29,597 (99.9%)\n",
      "  Read 2 with adapter:                  29,550 (99.7%)\n",
      "Pairs written (passing filters):        29,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,478,923 bp\n",
      "  Read 1:     8,738,318 bp\n",
      "  Read 2:     8,740,605 bp\n",
      "Total written (filtered):     15,941,217 bp (91.2%)\n",
      "  Read 1:     7,968,857 bp\n",
      "  Read 2:     7,972,360 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t321\t0.0\t2\t0 315 6\n",
      "26\t29025\t0.0\t2\t28497 512 16\n",
      "27\t190\t0.0\t2\t0 182 8\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29550 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t237\t0.0\t2\t0 227 10\n",
      "26\t29115\t0.0\t2\t28583 502 30\n",
      "27\t178\t0.0\t2\t0 169 9\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1924_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1924_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1924_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1924_215_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1924_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1924_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1924_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1924_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        28,350 reads  @    149.2 µs/read;   0.40 M reads/minute\n",
      "Finished in 4.23 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,350\n",
      "  Read 1 with adapter:                  28,214 (99.5%)\n",
      "  Read 2 with adapter:                  28,288 (99.8%)\n",
      "Pairs written (passing filters):        28,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,697,185 bp\n",
      "  Read 1:     8,333,948 bp\n",
      "  Read 2:     8,363,237 bp\n",
      "Total written (filtered):     15,228,244 bp (91.2%)\n",
      "  Read 1:     7,600,428 bp\n",
      "  Read 2:     7,627,816 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t249\t0.0\t2\t0 246 3\n",
      "26\t27771\t0.0\t2\t27243 518 10\n",
      "27\t139\t0.0\t2\t0 129 10\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28288 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t228\t0.0\t2\t0 219 9\n",
      "26\t27888\t0.0\t2\t27379 478 31\n",
      "27\t135\t0.0\t2\t0 128 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1925_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1925_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1925_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1925_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1925_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1925_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1925_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1925_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        23,875 reads  @    160.8 µs/read;   0.37 M reads/minute\n",
      "Finished in 3.84 s (161 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,875\n",
      "  Read 1 with adapter:                  23,851 (99.9%)\n",
      "  Read 2 with adapter:                  23,838 (99.8%)\n",
      "Pairs written (passing filters):        23,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,084,147 bp\n",
      "  Read 1:     7,041,254 bp\n",
      "  Read 2:     7,042,893 bp\n",
      "Total written (filtered):     12,844,365 bp (91.2%)\n",
      "  Read 1:     6,421,152 bp\n",
      "  Read 2:     6,423,213 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 23851 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t260\t0.0\t2\t0 245 15\n",
      "26\t23355\t0.0\t2\t22893 448 14\n",
      "27\t184\t0.0\t2\t0 176 8\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 23838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t230\t0.0\t2\t0 223 7\n",
      "26\t23457\t0.0\t2\t23045 401 11\n",
      "27\t136\t0.0\t2\t0 132 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1926_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1926_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1926_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1926_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1926_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1926_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1926_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1926_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,069 reads  @    142.4 µs/read;   0.42 M reads/minute\n",
      "Finished in 5.85 s (142 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,069\n",
      "  Read 1 with adapter:                  41,021 (99.9%)\n",
      "  Read 2 with adapter:                  40,994 (99.8%)\n",
      "Pairs written (passing filters):        41,069 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,227,321 bp\n",
      "  Read 1:    12,113,520 bp\n",
      "  Read 2:    12,113,801 bp\n",
      "Total written (filtered):     22,095,156 bp (91.2%)\n",
      "  Read 1:    11,047,089 bp\n",
      "  Read 2:    11,048,067 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41021 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t471\t0.0\t2\t0 453 18\n",
      "26\t40209\t0.0\t2\t39497 702 10\n",
      "27\t238\t0.0\t2\t0 230 8\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t292\t0.0\t2\t0 285 7\n",
      "26\t40505\t0.0\t2\t39765 693 47\n",
      "27\t164\t0.0\t2\t0 154 10\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1927_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1927_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1927_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1927_221_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1927_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1927_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1927_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1927_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        33,220 reads  @    161.2 µs/read;   0.37 M reads/minute\n",
      "Finished in 5.36 s (161 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,220\n",
      "  Read 1 with adapter:                  33,182 (99.9%)\n",
      "  Read 2 with adapter:                  33,143 (99.8%)\n",
      "Pairs written (passing filters):        33,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,595,435 bp\n",
      "  Read 1:     9,799,320 bp\n",
      "  Read 2:     9,796,115 bp\n",
      "Total written (filtered):     17,870,912 bp (91.2%)\n",
      "  Read 1:     8,936,537 bp\n",
      "  Read 2:     8,934,375 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t329\t0.0\t2\t0 322 7\n",
      "26\t32522\t0.0\t2\t31956 555 11\n",
      "27\t250\t0.0\t2\t0 241 9\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33143 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t210\t0.0\t2\t0 200 10\n",
      "26\t32725\t0.0\t2\t32075 611 39\n",
      "27\t156\t0.0\t2\t0 151 5\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1928_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1928_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1928_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1928_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1928_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1928_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1928_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1928_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        30,592 reads  @    152.7 µs/read;   0.39 M reads/minute\n",
      "Finished in 4.68 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,592\n",
      "  Read 1 with adapter:                  30,563 (99.9%)\n",
      "  Read 2 with adapter:                  30,358 (99.2%)\n",
      "Pairs written (passing filters):        30,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,048,465 bp\n",
      "  Read 1:     9,024,412 bp\n",
      "  Read 2:     9,024,053 bp\n",
      "Total written (filtered):     16,464,668 bp (91.2%)\n",
      "  Read 1:     8,229,828 bp\n",
      "  Read 2:     8,234,840 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30563 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t317\t0.0\t2\t0 305 12\n",
      "26\t29986\t0.0\t2\t29456 514 16\n",
      "27\t221\t0.0\t2\t0 210 11\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t252\t0.0\t2\t0 243 9\n",
      "26\t29929\t0.0\t2\t29366 540 23\n",
      "27\t129\t0.0\t2\t0 128 1\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1929_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1929_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1929_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1929_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1929_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1929_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1929_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1929_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        24,520 reads  @    177.4 µs/read;   0.34 M reads/minute\n",
      "Finished in 4.35 s (178 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,520\n",
      "  Read 1 with adapter:                  24,432 (99.6%)\n",
      "  Read 2 with adapter:                  24,449 (99.7%)\n",
      "Pairs written (passing filters):        24,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,457,657 bp\n",
      "  Read 1:     7,225,300 bp\n",
      "  Read 2:     7,232,357 bp\n",
      "Total written (filtered):     13,186,937 bp (91.2%)\n",
      "  Read 1:     6,590,107 bp\n",
      "  Read 2:     6,596,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t221\t0.0\t2\t0 212 9\n",
      "26\t24021\t0.0\t2\t23566 443 12\n",
      "27\t154\t0.0\t2\t0 144 10\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24449 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t265\t0.0\t2\t0 254 11\n",
      "26\t24038\t0.0\t2\t23613 406 19\n",
      "27\t122\t0.0\t2\t0 116 6\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1930_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1930_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1930_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1930_227_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1930_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1930_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1930_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1930_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        27,257 reads  @    157.3 µs/read;   0.38 M reads/minute\n",
      "Finished in 4.29 s (158 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,257\n",
      "  Read 1 with adapter:                  27,204 (99.8%)\n",
      "  Read 2 with adapter:                  27,123 (99.5%)\n",
      "Pairs written (passing filters):        27,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,073,885 bp\n",
      "  Read 1:     8,036,767 bp\n",
      "  Read 2:     8,037,118 bp\n",
      "Total written (filtered):     14,661,507 bp (91.2%)\n",
      "  Read 1:     7,329,525 bp\n",
      "  Read 2:     7,331,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t227\t0.0\t2\t0 221 6\n",
      "26\t26794\t0.0\t2\t26308 479 7\n",
      "27\t157\t0.0\t2\t0 152 5\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t186\t0.0\t2\t0 180 6\n",
      "26\t26808\t0.0\t2\t26246 533 29\n",
      "27\t114\t0.0\t2\t0 106 8\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1931_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1931_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1931_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1931_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1931_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1931_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1931_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1931_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        32,532 reads  @    194.3 µs/read;   0.31 M reads/minute\n",
      "Finished in 6.33 s (194 us/read; 0.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,532\n",
      "  Read 1 with adapter:                  32,426 (99.7%)\n",
      "  Read 2 with adapter:                  32,473 (99.8%)\n",
      "Pairs written (passing filters):        32,532 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,183,875 bp\n",
      "  Read 1:     9,587,939 bp\n",
      "  Read 2:     9,595,936 bp\n",
      "Total written (filtered):     17,496,573 bp (91.2%)\n",
      "  Read 1:     8,744,924 bp\n",
      "  Read 2:     8,751,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t253\t0.0\t2\t0 242 11\n",
      "26\t31966\t0.0\t2\t31336 617 13\n",
      "27\t162\t0.0\t2\t0 153 9\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t219\t0.0\t2\t0 208 11\n",
      "26\t32027\t0.0\t2\t31409 584 34\n",
      "27\t186\t0.0\t2\t0 175 11\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1932_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1932_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1932_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1932_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1932_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1932_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1932_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1932_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,620 reads  @    144.4 µs/read;   0.42 M reads/minute\n",
      "Finished in 5.29 s (145 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,620\n",
      "  Read 1 with adapter:                  36,516 (99.7%)\n",
      "  Read 2 with adapter:                  36,541 (99.8%)\n",
      "Pairs written (passing filters):        36,620 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,598,261 bp\n",
      "  Read 1:    10,796,615 bp\n",
      "  Read 2:    10,801,646 bp\n",
      "Total written (filtered):     19,698,882 bp (91.2%)\n",
      "  Read 1:     9,847,213 bp\n",
      "  Read 2:     9,851,669 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t277\t0.0\t2\t0 261 16\n",
      "26\t35980\t0.0\t2\t35333 631 16\n",
      "27\t183\t0.0\t2\t0 175 8\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36541 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t296\t0.0\t2\t0 283 13\n",
      "26\t36012\t0.0\t2\t35309 673 30\n",
      "27\t207\t0.0\t2\t0 193 14\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1933_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1933_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1933_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1933_233_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1933_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1933_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1933_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1933_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        31,296 reads  @    161.4 µs/read;   0.37 M reads/minute\n",
      "Finished in 5.06 s (162 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,296\n",
      "  Read 1 with adapter:                  31,217 (99.7%)\n",
      "  Read 2 with adapter:                  31,201 (99.7%)\n",
      "Pairs written (passing filters):        31,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,457,002 bp\n",
      "  Read 1:     9,229,107 bp\n",
      "  Read 2:     9,227,895 bp\n",
      "Total written (filtered):     16,834,394 bp (91.2%)\n",
      "  Read 1:     8,417,569 bp\n",
      "  Read 2:     8,416,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31217 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t294\t0.0\t2\t0 286 8\n",
      "26\t30723\t0.0\t2\t30128 587 8\n",
      "27\t154\t0.0\t2\t0 151 3\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t354\t0.0\t2\t0 346 8\n",
      "26\t30622\t0.0\t2\t29977 608 37\n",
      "27\t172\t0.0\t2\t0 165 7\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1934_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1934_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1934_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1934_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1934_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1934_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1934_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1934_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,307 reads  @    144.0 µs/read;   0.42 M reads/minute\n",
      "Finished in 5.24 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,307\n",
      "  Read 1 with adapter:                  36,259 (99.9%)\n",
      "  Read 2 with adapter:                  36,218 (99.8%)\n",
      "Pairs written (passing filters):        36,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,413,966 bp\n",
      "  Read 1:    10,705,077 bp\n",
      "  Read 2:    10,708,889 bp\n",
      "Total written (filtered):     19,529,727 bp (91.2%)\n",
      "  Read 1:     9,762,420 bp\n",
      "  Read 2:     9,767,307 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36259 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t306\t0.0\t2\t0 295 11\n",
      "26\t35719\t0.0\t2\t35099 604 16\n",
      "27\t183\t0.0\t2\t0 180 3\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t285\t0.0\t2\t0 277 8\n",
      "26\t35716\t0.0\t2\t35086 595 35\n",
      "27\t171\t0.0\t2\t0 161 10\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1935_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1935_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1935_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1935_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1935_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1935_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1935_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1935_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,531 reads  @    187.3 µs/read;   0.32 M reads/minute\n",
      "Finished in 0.66 s (188 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,531\n",
      "  Read 1 with adapter:                   3,054 (86.5%)\n",
      "  Read 2 with adapter:                   3,094 (87.6%)\n",
      "Pairs written (passing filters):         3,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,080,229 bp\n",
      "  Read 1:     1,039,800 bp\n",
      "  Read 2:     1,040,429 bp\n",
      "Total written (filtered):      1,920,381 bp (92.3%)\n",
      "  Read 1:       960,371 bp\n",
      "  Read 2:       960,010 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 3054 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t24\t0.0\t2\t0 20 4\n",
      "26\t2971\t0.0\t2\t2889 81 1\n",
      "27\t49\t0.0\t2\t0 25 24\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 3094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t23\t0.0\t2\t0 23\n",
      "26\t3043\t0.0\t2\t2967 75 1\n",
      "27\t14\t0.0\t2\t0 12 2\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1936_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1936_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1936_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1936_239_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1936_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-l65fas9j/S1936_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1936_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-_dpqxj_z/f9caf08f-ef4a-4d4e-959d-9d57e5c19e82/data/S1936_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        28,465 reads  @    208.7 µs/read;   0.29 M reads/minute\n",
      "Finished in 5.95 s (209 us/read; 0.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,465\n",
      "  Read 1 with adapter:                  28,433 (99.9%)\n",
      "  Read 2 with adapter:                  28,406 (99.8%)\n",
      "Pairs written (passing filters):        28,465 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,791,378 bp\n",
      "  Read 1:     8,396,325 bp\n",
      "  Read 2:     8,395,053 bp\n",
      "Total written (filtered):     15,313,701 bp (91.2%)\n",
      "  Read 1:     7,657,163 bp\n",
      "  Read 2:     7,656,538 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t296\t0.0\t2\t0 287 9\n",
      "26\t27941\t0.0\t2\t27359 564 18\n",
      "27\t144\t0.0\t2\t0 138 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t198\t0.0\t2\t0 196 2\n",
      "26\t28030\t0.0\t2\t27487 516 27\n",
      "27\t147\t0.0\t2\t0 144 3\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: g-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: g-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-tpvfqwue')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 09802584-0ebf-4243-8336-69277dbc7dce>"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.g import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/P2107-Lib1-manifest_COI.csv \\\n",
    "  --output-path g-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.g trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences g-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences g-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.g summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data g-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization g-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('g-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: g-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: g-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: g-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.g Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs g-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 224 \\\n",
    "  --p-trunc-len-r 167 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table g-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats g-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: g-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-a8t4nubo')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: f1746b21-66f0-4621-83bd-a8390f310468>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.g view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table g-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization g-table.qzv \n",
    "q2.Visualization.load('g-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: g-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.g view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization g-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/ENP-Lib-manifest_COI.csv as PairedEndFastqManifestPhred33 to h-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2201_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2201_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2201_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2201_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2201_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2201_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2201_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2201_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        14,623 reads  @    106.7 µs/read;   0.56 M reads/minute\n",
      "Finished in 1.56 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,623\n",
      "  Read 1 with adapter:                  14,591 (99.8%)\n",
      "  Read 2 with adapter:                  14,489 (99.1%)\n",
      "Pairs written (passing filters):        14,623 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,623,527 bp\n",
      "  Read 1:     4,312,255 bp\n",
      "  Read 2:     4,311,272 bp\n",
      "Total written (filtered):      7,867,555 bp (91.2%)\n",
      "  Read 1:     3,932,891 bp\n",
      "  Read 2:     3,934,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 14591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t180\t0.0\t2\t0 165 15\n",
      "26\t14268\t0.0\t2\t13410 788 70\n",
      "27\t80\t0.0\t2\t0 73 7\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 14489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t173\t0.0\t2\t0 171 2\n",
      "26\t14217\t0.0\t2\t13983 225 9\n",
      "27\t67\t0.0\t2\t0 66 1\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2202_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2202_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2202_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2202_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2202_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2202_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2202_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2202_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        54,209 reads  @     95.8 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.20 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,209\n",
      "  Read 1 with adapter:                  54,182 (100.0%)\n",
      "  Read 2 with adapter:                  54,067 (99.7%)\n",
      "Pairs written (passing filters):        54,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,978,586 bp\n",
      "  Read 1:    15,990,621 bp\n",
      "  Read 2:    15,987,965 bp\n",
      "Total written (filtered):     29,164,463 bp (91.2%)\n",
      "  Read 1:    14,582,080 bp\n",
      "  Read 2:    14,582,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t621\t0.0\t2\t0 605 16\n",
      "26\t53151\t0.0\t2\t52271 857 23\n",
      "27\t286\t0.0\t2\t0 280 6\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t371\t0.0\t2\t0 361 10\n",
      "26\t53416\t0.0\t2\t52150 1170 96\n",
      "27\t241\t0.0\t2\t0 230 11\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2203_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2203_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2203_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2203_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2203_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2203_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2203_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2203_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10       107,102 reads  @    100.9 µs/read;   0.59 M reads/minute\n",
      "Finished in 10.81 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            107,102\n",
      "  Read 1 with adapter:                 106,963 (99.9%)\n",
      "  Read 2 with adapter:                 106,887 (99.8%)\n",
      "Pairs written (passing filters):       107,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,177,466 bp\n",
      "  Read 1:    31,587,832 bp\n",
      "  Read 2:    31,589,634 bp\n",
      "Total written (filtered):     57,618,025 bp (91.2%)\n",
      "  Read 1:    28,807,202 bp\n",
      "  Read 2:    28,810,823 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 106963 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t1322\t0.0\t2\t0 1297 25\n",
      "26\t104742\t0.0\t2\t102793 1906 43\n",
      "27\t596\t0.0\t2\t0 568 28\n",
      "28\t231\t0.0\t2\t0 0 231\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 106887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t751\t0.0\t2\t0 728 23\n",
      "26\t105619\t0.0\t2\t102935 2515 169\n",
      "27\t398\t0.0\t2\t0 379 19\n",
      "28\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2204_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2204_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2204_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2204_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2204_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2204_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2204_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2204_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,514 reads  @     94.0 µs/read;   0.64 M reads/minute\n",
      "Finished in 2.69 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,514\n",
      "  Read 1 with adapter:                  28,469 (99.8%)\n",
      "  Read 2 with adapter:                  27,400 (96.1%)\n",
      "Pairs written (passing filters):        28,492 (99.9%)\n",
      "\n",
      "Total basepairs processed:    16,613,656 bp\n",
      "  Read 1:     8,410,774 bp\n",
      "  Read 2:     8,202,882 bp\n",
      "Total written (filtered):     15,155,220 bp (91.2%)\n",
      "  Read 1:     7,664,683 bp\n",
      "  Read 2:     7,490,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28469 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t307\t0.0\t2\t0 304 3\n",
      "26\t27886\t0.0\t2\t27501 369 16\n",
      "27\t242\t0.0\t2\t0 206 36\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t230\t0.0\t2\t0 191 39\n",
      "26\t26988\t0.0\t2\t25615 1172 201\n",
      "27\t129\t0.0\t2\t0 118 11\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2205_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2205_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2205_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2205_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2205_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2205_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2205_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2205_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        51,854 reads  @    106.5 µs/read;   0.56 M reads/minute\n",
      "Finished in 5.53 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,854\n",
      "  Read 1 with adapter:                  51,792 (99.9%)\n",
      "  Read 2 with adapter:                  50,430 (97.3%)\n",
      "Pairs written (passing filters):        51,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,373,728 bp\n",
      "  Read 1:    15,293,600 bp\n",
      "  Read 2:    15,080,128 bp\n",
      "Total written (filtered):     27,715,648 bp (91.2%)\n",
      "  Read 1:    13,946,420 bp\n",
      "  Read 2:    13,769,228 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51792 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t660\t0.0\t2\t0 652 8\n",
      "26\t50716\t0.0\t2\t49855 833 28\n",
      "27\t275\t0.0\t2\t0 259 16\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t554\t0.0\t2\t0 520 34\n",
      "26\t49571\t0.0\t2\t47485 1811 275\n",
      "27\t212\t0.0\t2\t0 188 24\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2206_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2206_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2206_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2206_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2206_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2206_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2206_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2206_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,912 reads  @     99.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.67 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,912\n",
      "  Read 1 with adapter:                  46,839 (99.8%)\n",
      "  Read 2 with adapter:                  46,733 (99.6%)\n",
      "Pairs written (passing filters):        46,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,658,442 bp\n",
      "  Read 1:    13,829,797 bp\n",
      "  Read 2:    13,828,645 bp\n",
      "Total written (filtered):     25,225,544 bp (91.2%)\n",
      "  Read 1:    12,611,894 bp\n",
      "  Read 2:    12,613,650 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46839 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t532\t0.0\t2\t0 524 8\n",
      "26\t45938\t0.0\t2\t45306 622 10\n",
      "27\t282\t0.0\t2\t0 264 18\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46733 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t352\t0.0\t2\t0 342 10\n",
      "26\t46114\t0.0\t2\t44960 1054 100\n",
      "27\t169\t0.0\t2\t0 167 2\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2207_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2207_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2207_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2207_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2207_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2207_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2207_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2207_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[          8<] 00:00:25       200,488 reads  @    127.9 µs/read;   0.47 M reads/minute\n",
      "Finished in 25.65 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            200,488\n",
      "  Read 1 with adapter:                 194,647 (97.1%)\n",
      "  Read 2 with adapter:                 193,892 (96.7%)\n",
      "Pairs written (passing filters):       200,469 (100.0%)\n",
      "\n",
      "Total basepairs processed:   115,960,975 bp\n",
      "  Read 1:    58,114,804 bp\n",
      "  Read 2:    57,846,171 bp\n",
      "Total written (filtered):    105,854,876 bp (91.3%)\n",
      "  Read 1:    53,049,959 bp\n",
      "  Read 2:    52,804,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 194647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t135\t0.0\t2\t0 0 135\n",
      "25\t2418\t0.0\t2\t0 2347 71\n",
      "26\t189859\t0.0\t2\t185931 3776 152\n",
      "27\t1733\t0.0\t2\t0 1598 135\n",
      "28\t502\t0.0\t2\t0 0 502\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 193892 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t1625\t0.0\t2\t0 1574 51\n",
      "26\t191284\t0.0\t2\t186299 4598 387\n",
      "27\t782\t0.0\t2\t0 752 30\n",
      "28\t125\t0.0\t2\t0 0 125\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2208_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2208_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2208_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2208_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2208_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2208_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2208_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2208_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        67,336 reads  @    117.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 7.94 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,336\n",
      "  Read 1 with adapter:                  67,170 (99.8%)\n",
      "  Read 2 with adapter:                  67,239 (99.9%)\n",
      "Pairs written (passing filters):        67,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,702,629 bp\n",
      "  Read 1:    19,841,707 bp\n",
      "  Read 2:    19,860,922 bp\n",
      "Total written (filtered):     36,208,622 bp (91.2%)\n",
      "  Read 1:    18,095,655 bp\n",
      "  Read 2:    18,112,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67170 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t903\t0.0\t2\t0 863 40\n",
      "26\t65681\t0.0\t2\t63331 2235 115\n",
      "27\t437\t0.0\t2\t0 400 37\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t597\t0.0\t2\t0 576 21\n",
      "26\t66277\t0.0\t2\t65118 1090 69\n",
      "27\t260\t0.0\t2\t0 255 5\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2209_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2209_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2209_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2209_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2209_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2209_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2209_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2209_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        55,435 reads  @    106.2 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.89 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,435\n",
      "  Read 1 with adapter:                  55,379 (99.9%)\n",
      "  Read 2 with adapter:                  54,967 (99.2%)\n",
      "Pairs written (passing filters):        55,435 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,698,177 bp\n",
      "  Read 1:    16,352,433 bp\n",
      "  Read 2:    16,345,744 bp\n",
      "Total written (filtered):     29,829,742 bp (91.2%)\n",
      "  Read 1:    14,912,706 bp\n",
      "  Read 2:    14,917,036 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55379 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t752\t0.0\t2\t0 719 33\n",
      "26\t54030\t0.0\t2\t52344 1609 77\n",
      "27\t345\t0.0\t2\t0 325 20\n",
      "28\t196\t0.0\t2\t0 0 196\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t659\t0.0\t2\t0 645 14\n",
      "26\t54026\t0.0\t2\t53094 901 31\n",
      "27\t207\t0.0\t2\t0 203 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2210_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2210_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2210_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2210_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2210_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2210_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2210_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2210_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        91,210 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 6.93 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,210\n",
      "  Read 1 with adapter:                  91,138 (99.9%)\n",
      "  Read 2 with adapter:                  90,834 (99.6%)\n",
      "Pairs written (passing filters):        91,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,805,689 bp\n",
      "  Read 1:    26,906,863 bp\n",
      "  Read 2:    26,898,826 bp\n",
      "Total written (filtered):     49,074,779 bp (91.2%)\n",
      "  Read 1:    24,537,452 bp\n",
      "  Read 2:    24,537,327 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 91138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t1237\t0.0\t2\t0 1211 26\n",
      "26\t88974\t0.0\t2\t87514 1434 26\n",
      "27\t490\t0.0\t2\t0 478 12\n",
      "28\t361\t0.0\t2\t0 0 361\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 90834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t780\t0.0\t2\t0 689 91\n",
      "26\t89496\t0.0\t2\t86245 2903 348\n",
      "27\t317\t0.0\t2\t0 297 20\n",
      "28\t190\t0.0\t2\t0 0 190\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2211_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2211_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2211_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2211_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2211_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2211_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2211_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2211_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,769 reads  @    106.9 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.68 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,769\n",
      "  Read 1 with adapter:                  43,651 (99.7%)\n",
      "  Read 2 with adapter:                  43,678 (99.8%)\n",
      "Pairs written (passing filters):        43,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,822,611 bp\n",
      "  Read 1:    12,911,565 bp\n",
      "  Read 2:    12,911,046 bp\n",
      "Total written (filtered):     23,552,484 bp (91.2%)\n",
      "  Read 1:    11,776,923 bp\n",
      "  Read 2:    11,775,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t604\t0.0\t2\t0 590 14\n",
      "26\t42679\t0.0\t2\t41867 797 15\n",
      "27\t236\t0.0\t2\t0 227 9\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t323\t0.0\t2\t0 310 13\n",
      "26\t43156\t0.0\t2\t42045 1036 75\n",
      "27\t158\t0.0\t2\t0 152 6\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2212_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2212_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2212_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2212_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2212_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2212_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2212_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2212_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        17,519 reads  @    115.1 µs/read;   0.52 M reads/minute\n",
      "Finished in 2.02 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,519\n",
      "  Read 1 with adapter:                  16,139 (92.1%)\n",
      "  Read 2 with adapter:                  17,349 (99.0%)\n",
      "Pairs written (passing filters):        17,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,331,668 bp\n",
      "  Read 1:     5,167,881 bp\n",
      "  Read 2:     5,163,787 bp\n",
      "Total written (filtered):      9,461,167 bp (91.6%)\n",
      "  Read 1:     4,748,348 bp\n",
      "  Read 2:     4,712,819 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 16139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t230\t0.0\t2\t0 222 8\n",
      "26\t15729\t0.0\t2\t15267 444 18\n",
      "27\t127\t0.0\t2\t0 91 36\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17349 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t169\t0.0\t2\t0 157 12\n",
      "26\t17033\t0.0\t2\t16210 724 99\n",
      "27\t99\t0.0\t2\t0 95 4\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2213_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2213_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2213_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2213_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2213_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2213_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2213_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2213_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,151 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 0.12 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,151\n",
      "  Read 1 with adapter:                   1,069 (92.9%)\n",
      "  Read 2 with adapter:                       1 (0.1%)\n",
      "Pairs written (passing filters):         1,147 (99.7%)\n",
      "\n",
      "Total basepairs processed:       457,327 bp\n",
      "  Read 1:       339,545 bp\n",
      "  Read 2:       117,782 bp\n",
      "Total written (filtered):        428,436 bp (93.7%)\n",
      "  Read 1:       310,680 bp\n",
      "  Read 2:       117,756 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1069 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t9\t0.0\t2\t0 7 2\n",
      "26\t1049\t0.0\t2\t1019 30\n",
      "27\t6\t0.0\t2\t0 6\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2214_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2214_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2214_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2214_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2214_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2214_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2214_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2214_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        52,361 reads  @    110.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 5.79 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,361\n",
      "  Read 1 with adapter:                  51,789 (98.9%)\n",
      "  Read 2 with adapter:                  51,981 (99.3%)\n",
      "Pairs written (passing filters):        52,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,878,810 bp\n",
      "  Read 1:    15,445,501 bp\n",
      "  Read 2:    15,433,309 bp\n",
      "Total written (filtered):     28,181,484 bp (91.3%)\n",
      "  Read 1:    14,099,459 bp\n",
      "  Read 2:    14,082,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51789 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t185\t0.0\t2\t0 0 185\n",
      "25\t743\t0.0\t2\t0 726 17\n",
      "26\t50361\t0.0\t2\t49261 1074 26\n",
      "27\t359\t0.0\t2\t0 342 17\n",
      "28\t141\t0.0\t2\t0 0 141\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t570\t0.0\t2\t0 555 15\n",
      "26\t51051\t0.0\t2\t49599 1337 115\n",
      "27\t272\t0.0\t2\t0 261 11\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2215_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2215_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2215_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2215_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2215_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2215_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2215_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2215_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        69,323 reads  @    103.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 7.19 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,323\n",
      "  Read 1 with adapter:                  68,658 (99.0%)\n",
      "  Read 2 with adapter:                  68,546 (98.9%)\n",
      "Pairs written (passing filters):        69,323 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,877,189 bp\n",
      "  Read 1:    20,449,418 bp\n",
      "  Read 2:    20,427,771 bp\n",
      "Total written (filtered):     37,310,800 bp (91.3%)\n",
      "  Read 1:    18,664,997 bp\n",
      "  Read 2:    18,645,803 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68658 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t1139\t0.0\t2\t0 1095 44\n",
      "26\t66956\t0.0\t2\t65059 1857 40\n",
      "27\t406\t0.0\t2\t0 385 21\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68546 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t670\t0.0\t2\t0 633 37\n",
      "26\t67381\t0.0\t2\t65048 2157 176\n",
      "27\t372\t0.0\t2\t0 359 13\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2216_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2216_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2216_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2216_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2216_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2216_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2216_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2216_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        75,787 reads  @    108.9 µs/read;   0.55 M reads/minute\n",
      "Finished in 8.25 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,787\n",
      "  Read 1 with adapter:                  75,575 (99.7%)\n",
      "  Read 2 with adapter:                  75,593 (99.7%)\n",
      "Pairs written (passing filters):        75,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,706,722 bp\n",
      "  Read 1:    22,356,153 bp\n",
      "  Read 2:    22,350,569 bp\n",
      "Total written (filtered):     40,777,279 bp (91.2%)\n",
      "  Read 1:    20,391,897 bp\n",
      "  Read 2:    20,385,382 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 75575 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t1202\t0.0\t2\t0 1165 37\n",
      "26\t73745\t0.0\t2\t71156 2512 77\n",
      "27\t448\t0.0\t2\t0 437 11\n",
      "28\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75593 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t657\t0.0\t2\t0 633 24\n",
      "26\t74488\t0.0\t2\t72306 2073 109\n",
      "27\t326\t0.0\t2\t0 316 10\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2217_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2217_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2217_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2217_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2217_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2217_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2217_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2217_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,039 reads  @     99.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.07 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,039\n",
      "  Read 1 with adapter:                  41,018 (99.9%)\n",
      "  Read 2 with adapter:                  40,691 (99.2%)\n",
      "Pairs written (passing filters):        41,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,207,245 bp\n",
      "  Read 1:    12,105,934 bp\n",
      "  Read 2:    12,101,311 bp\n",
      "Total written (filtered):     22,083,159 bp (91.2%)\n",
      "  Read 1:    11,039,620 bp\n",
      "  Read 2:    11,043,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41018 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t435\t0.0\t2\t0 427 8\n",
      "26\t40297\t0.0\t2\t39500 761 36\n",
      "27\t191\t0.0\t2\t0 184 7\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40691 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t381\t0.0\t2\t0 375 6\n",
      "26\t40097\t0.0\t2\t39265 782 50\n",
      "27\t167\t0.0\t2\t0 165 2\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2218_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2218_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2218_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2218_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2218_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2218_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2218_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2218_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        46,647 reads  @    108.6 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.07 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,647\n",
      "  Read 1 with adapter:                  46,486 (99.7%)\n",
      "  Read 2 with adapter:                  46,486 (99.7%)\n",
      "Pairs written (passing filters):        46,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,507,965 bp\n",
      "  Read 1:    13,752,990 bp\n",
      "  Read 2:    13,754,975 bp\n",
      "Total written (filtered):     25,091,190 bp (91.2%)\n",
      "  Read 1:    12,544,711 bp\n",
      "  Read 2:    12,546,479 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46486 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t636\t0.0\t2\t0 618 18\n",
      "26\t45524\t0.0\t2\t44152 1324 48\n",
      "27\t241\t0.0\t2\t0 228 13\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46486 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t440\t0.0\t2\t0 417 23\n",
      "26\t45741\t0.0\t2\t43800 1798 143\n",
      "27\t250\t0.0\t2\t0 229 21\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2219_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2219_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2219_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2219_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2219_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2219_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2219_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2219_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        56,648 reads  @    104.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.92 s (104 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,648\n",
      "  Read 1 with adapter:                  56,499 (99.7%)\n",
      "  Read 2 with adapter:                  56,482 (99.7%)\n",
      "Pairs written (passing filters):        56,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,414,996 bp\n",
      "  Read 1:    16,708,218 bp\n",
      "  Read 2:    16,706,778 bp\n",
      "Total written (filtered):     30,477,761 bp (91.2%)\n",
      "  Read 1:    15,239,417 bp\n",
      "  Read 2:    15,238,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t658\t0.0\t2\t0 642 16\n",
      "26\t55378\t0.0\t2\t54132 1218 28\n",
      "27\t325\t0.0\t2\t0 316 9\n",
      "28\t109\t0.0\t2\t0 0 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56482 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t417\t0.0\t2\t0 403 14\n",
      "26\t55732\t0.0\t2\t53987 1605 140\n",
      "27\t283\t0.0\t2\t0 265 18\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2220_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2220_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2220_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2220_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2220_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2220_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2220_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2220_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,248 reads  @     89.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 2.97 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,248\n",
      "  Read 1 with adapter:                  33,185 (99.8%)\n",
      "  Read 2 with adapter:                  32,712 (98.4%)\n",
      "Pairs written (passing filters):        33,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,561,839 bp\n",
      "  Read 1:     9,806,719 bp\n",
      "  Read 2:     9,755,120 bp\n",
      "Total written (filtered):     17,848,171 bp (91.2%)\n",
      "  Read 1:     8,943,467 bp\n",
      "  Read 2:     8,904,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t354\t0.0\t2\t0 348 6\n",
      "26\t32557\t0.0\t2\t31996 542 19\n",
      "27\t194\t0.0\t2\t0 154 40\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32712 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t236\t0.0\t2\t0 224 12\n",
      "26\t32290\t0.0\t2\t30843 1256 191\n",
      "27\t164\t0.0\t2\t0 137 27\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2221_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2221_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2221_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2221_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2221_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2221_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2221_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2221_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        37,858 reads  @    101.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 3.85 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,858\n",
      "  Read 1 with adapter:                  37,822 (99.9%)\n",
      "  Read 2 with adapter:                  36,257 (95.8%)\n",
      "Pairs written (passing filters):        37,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,071,495 bp\n",
      "  Read 1:    11,167,959 bp\n",
      "  Read 2:    10,903,536 bp\n",
      "Total written (filtered):     20,144,247 bp (91.3%)\n",
      "  Read 1:    10,183,217 bp\n",
      "  Read 2:     9,961,030 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t421\t0.0\t2\t0 413 8\n",
      "26\t36976\t0.0\t2\t36324 633 19\n",
      "27\t304\t0.0\t2\t0 271 33\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36257 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t371\t0.0\t2\t0 343 28\n",
      "26\t35650\t0.0\t2\t34100 1366 184\n",
      "27\t193\t0.0\t2\t0 163 30\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2222_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2222_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2222_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2222_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2222_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2222_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2222_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2222_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        59,646 reads  @     94.3 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.63 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,646\n",
      "  Read 1 with adapter:                  59,449 (99.7%)\n",
      "  Read 2 with adapter:                  59,282 (99.4%)\n",
      "Pairs written (passing filters):        59,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,166,890 bp\n",
      "  Read 1:    17,594,901 bp\n",
      "  Read 2:    17,571,989 bp\n",
      "Total written (filtered):     32,079,598 bp (91.2%)\n",
      "  Read 1:    16,048,720 bp\n",
      "  Read 2:    16,030,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59449 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t681\t0.0\t2\t0 662 19\n",
      "26\t57716\t0.0\t2\t56090 1572 54\n",
      "27\t756\t0.0\t2\t0 672 84\n",
      "28\t256\t0.0\t2\t0 0 256\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59282 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t654\t0.0\t2\t0 620 34\n",
      "26\t58158\t0.0\t2\t55711 2245 202\n",
      "27\t395\t0.0\t2\t0 364 31\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2223_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2223_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2223_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2223_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2223_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2223_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2223_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2223_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        64,787 reads  @    122.6 µs/read;   0.49 M reads/minute\n",
      "Finished in 7.95 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,787\n",
      "  Read 1 with adapter:                  64,697 (99.9%)\n",
      "  Read 2 with adapter:                  64,538 (99.6%)\n",
      "Pairs written (passing filters):        64,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,211,893 bp\n",
      "  Read 1:    19,111,753 bp\n",
      "  Read 2:    19,100,140 bp\n",
      "Total written (filtered):     34,852,186 bp (91.2%)\n",
      "  Read 1:    17,429,910 bp\n",
      "  Read 2:    17,422,276 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64697 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t714\t0.0\t2\t0 700 14\n",
      "26\t63567\t0.0\t2\t62509 1020 38\n",
      "27\t301\t0.0\t2\t0 293 8\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64538 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t445\t0.0\t2\t0 418 27\n",
      "26\t63739\t0.0\t2\t61944 1630 165\n",
      "27\t235\t0.0\t2\t0 220 15\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2224_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2224_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2224_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2224_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2224_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2224_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2224_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2224_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:10        91,356 reads  @    112.6 µs/read;   0.53 M reads/minute\n",
      "Finished in 10.29 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,356\n",
      "  Read 1 with adapter:                  91,007 (99.6%)\n",
      "  Read 2 with adapter:                  90,960 (99.6%)\n",
      "Pairs written (passing filters):        91,356 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,873,117 bp\n",
      "  Read 1:    26,937,014 bp\n",
      "  Read 2:    26,936,103 bp\n",
      "Total written (filtered):     49,142,412 bp (91.2%)\n",
      "  Read 1:    24,571,013 bp\n",
      "  Read 2:    24,571,399 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 91007 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t849\t0.0\t2\t0 830 19\n",
      "26\t89495\t0.0\t2\t88085 1371 39\n",
      "27\t510\t0.0\t2\t0 497 13\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 90960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t705\t0.0\t2\t0 641 64\n",
      "26\t89802\t0.0\t2\t86077 3293 432\n",
      "27\t309\t0.0\t2\t0 293 16\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2225_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2225_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2225_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2225_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2225_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2225_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2225_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2225_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01         8,929 reads  @    130.7 µs/read;   0.46 M reads/minute\n",
      "Finished in 1.17 s (131 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,929\n",
      "  Read 1 with adapter:                   8,903 (99.7%)\n",
      "  Read 2 with adapter:                   8,817 (98.7%)\n",
      "Pairs written (passing filters):         8,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,266,170 bp\n",
      "  Read 1:     2,633,689 bp\n",
      "  Read 2:     2,632,481 bp\n",
      "Total written (filtered):      4,805,540 bp (91.3%)\n",
      "  Read 1:     2,402,260 bp\n",
      "  Read 2:     2,403,280 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 8903 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t120\t0.0\t2\t0 111 9\n",
      "26\t8711\t0.0\t2\t8223 451 37\n",
      "27\t49\t0.0\t2\t0 46 3\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 8817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t81\t0.0\t2\t0 81\n",
      "26\t8689\t0.0\t2\t8549 130 10\n",
      "27\t42\t0.0\t2\t0 41 1\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2226_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2226_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2226_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2226_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2226_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2226_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2226_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2226_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:01:20       763,811 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 80.75 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            763,811\n",
      "  Read 1 with adapter:                 762,948 (99.9%)\n",
      "  Read 2 with adapter:                 757,228 (99.1%)\n",
      "Pairs written (passing filters):       763,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:   450,180,193 bp\n",
      "  Read 1:   225,281,977 bp\n",
      "  Read 2:   224,898,216 bp\n",
      "Total written (filtered):    410,661,076 bp (91.2%)\n",
      "  Read 1:   205,448,237 bp\n",
      "  Read 2:   205,212,839 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 762948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t470\t0.0\t2\t0 0 470\n",
      "25\t9232\t0.0\t2\t0 9053 179\n",
      "26\t746920\t0.0\t2\t733973 12644 303\n",
      "27\t5388\t0.0\t2\t0 5208 180\n",
      "28\t938\t0.0\t2\t0 0 938\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 757228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t317\t0.0\t2\t0 0 317\n",
      "25\t6222\t0.0\t2\t0 5890 332\n",
      "26\t747118\t0.0\t2\t726019 19126 1973\n",
      "27\t2837\t0.0\t2\t0 2705 132\n",
      "28\t734\t0.0\t2\t0 0 734\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2227_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2227_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2227_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2227_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2227_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2227_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2227_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2227_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:12       100,157 reads  @    119.9 µs/read;   0.50 M reads/minute\n",
      "Finished in 12.01 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,157\n",
      "  Read 1 with adapter:                 100,048 (99.9%)\n",
      "  Read 2 with adapter:                  99,949 (99.8%)\n",
      "Pairs written (passing filters):       100,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,083,860 bp\n",
      "  Read 1:    29,540,971 bp\n",
      "  Read 2:    29,542,889 bp\n",
      "Total written (filtered):     53,884,452 bp (91.2%)\n",
      "  Read 1:    26,939,991 bp\n",
      "  Read 2:    26,944,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 100048 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t1256\t0.0\t2\t0 1230 26\n",
      "26\t97798\t0.0\t2\t95878 1846 74\n",
      "27\t752\t0.0\t2\t0 706 46\n",
      "28\t180\t0.0\t2\t0 0 180\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 99949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t722\t0.0\t2\t0 693 29\n",
      "26\t98702\t0.0\t2\t96478 2077 147\n",
      "27\t422\t0.0\t2\t0 402 20\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2228_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2228_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2228_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2228_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2228_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2228_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2228_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2228_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        66,163 reads  @    121.4 µs/read;   0.49 M reads/minute\n",
      "Finished in 8.03 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,163\n",
      "  Read 1 with adapter:                  66,118 (99.9%)\n",
      "  Read 2 with adapter:                  64,590 (97.6%)\n",
      "Pairs written (passing filters):        66,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,742,976 bp\n",
      "  Read 1:    19,516,603 bp\n",
      "  Read 2:    19,226,373 bp\n",
      "Total written (filtered):     35,343,144 bp (91.2%)\n",
      "  Read 1:    17,796,003 bp\n",
      "  Read 2:    17,547,141 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t915\t0.0\t2\t0 895 20\n",
      "26\t64783\t0.0\t2\t63463 1281 39\n",
      "27\t365\t0.0\t2\t0 331 34\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t502\t0.0\t2\t0 460 42\n",
      "26\t63726\t0.0\t2\t61164 2289 273\n",
      "27\t246\t0.0\t2\t0 237 9\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2229_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2229_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2229_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2229_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2229_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2229_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2229_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2229_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        64,857 reads  @    120.3 µs/read;   0.50 M reads/minute\n",
      "Finished in 7.81 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,857\n",
      "  Read 1 with adapter:                  64,810 (99.9%)\n",
      "  Read 2 with adapter:                  62,095 (95.7%)\n",
      "Pairs written (passing filters):        64,847 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,793,214 bp\n",
      "  Read 1:    19,132,154 bp\n",
      "  Read 2:    18,661,060 bp\n",
      "Total written (filtered):     34,491,666 bp (91.3%)\n",
      "  Read 1:    17,444,693 bp\n",
      "  Read 2:    17,046,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t817\t0.0\t2\t0 799 18\n",
      "26\t63451\t0.0\t2\t62116 1300 35\n",
      "27\t382\t0.0\t2\t0 364 18\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62095 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t688\t0.0\t2\t0 665 23\n",
      "26\t61021\t0.0\t2\t58785 1992 244\n",
      "27\t327\t0.0\t2\t0 305 22\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2230_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2230_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2230_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2230_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2230_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2230_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2230_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2230_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:10        89,147 reads  @    115.4 µs/read;   0.52 M reads/minute\n",
      "Finished in 10.29 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,147\n",
      "  Read 1 with adapter:                  89,102 (99.9%)\n",
      "  Read 2 with adapter:                  88,960 (99.8%)\n",
      "Pairs written (passing filters):        89,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,585,592 bp\n",
      "  Read 1:    26,296,503 bp\n",
      "  Read 2:    26,289,089 bp\n",
      "Total written (filtered):     47,957,190 bp (91.2%)\n",
      "  Read 1:    23,980,675 bp\n",
      "  Read 2:    23,976,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 89102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t1120\t0.0\t2\t0 1102 18\n",
      "26\t87537\t0.0\t2\t86093 1416 28\n",
      "27\t370\t0.0\t2\t0 356 14\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 88960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t812\t0.0\t2\t0 789 23\n",
      "26\t87718\t0.0\t2\t85955 1625 138\n",
      "27\t286\t0.0\t2\t0 281 5\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2231_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2231_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2231_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2231_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2231_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2231_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2231_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2231_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        15,409 reads  @    126.2 µs/read;   0.48 M reads/minute\n",
      "Finished in 1.95 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,409\n",
      "  Read 1 with adapter:                  15,387 (99.9%)\n",
      "  Read 2 with adapter:                  15,339 (99.5%)\n",
      "Pairs written (passing filters):        15,409 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,087,927 bp\n",
      "  Read 1:     4,545,647 bp\n",
      "  Read 2:     4,542,280 bp\n",
      "Total written (filtered):      8,289,308 bp (91.2%)\n",
      "  Read 1:     4,145,781 bp\n",
      "  Read 2:     4,143,527 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t257\t0.0\t2\t0 238 19\n",
      "26\t15038\t0.0\t2\t14192 806 40\n",
      "27\t67\t0.0\t2\t0 66 1\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15339 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t144\t0.0\t2\t0 138 6\n",
      "26\t15109\t0.0\t2\t14420 636 53\n",
      "27\t65\t0.0\t2\t0 60 5\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2232_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2232_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2232_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2232_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2232_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2232_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2232_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2232_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,228 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.66 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,228\n",
      "  Read 1 with adapter:                  47,206 (100.0%)\n",
      "  Read 2 with adapter:                  47,164 (99.9%)\n",
      "Pairs written (passing filters):        47,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,861,762 bp\n",
      "  Read 1:    13,931,549 bp\n",
      "  Read 2:    13,930,213 bp\n",
      "Total written (filtered):     25,408,431 bp (91.2%)\n",
      "  Read 1:    12,704,354 bp\n",
      "  Read 2:    12,704,077 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47206 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t514\t0.0\t2\t0 504 10\n",
      "26\t46333\t0.0\t2\t45542 756 35\n",
      "27\t269\t0.0\t2\t0 264 5\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t346\t0.0\t2\t0 333 13\n",
      "26\t46587\t0.0\t2\t45803 736 48\n",
      "27\t180\t0.0\t2\t0 180\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2233_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2233_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2233_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2233_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2233_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2233_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2233_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2233_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        37,871 reads  @    127.8 µs/read;   0.47 M reads/minute\n",
      "Finished in 4.85 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,871\n",
      "  Read 1 with adapter:                  37,834 (99.9%)\n",
      "  Read 2 with adapter:                  37,657 (99.4%)\n",
      "Pairs written (passing filters):        37,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,339,657 bp\n",
      "  Read 1:    11,170,643 bp\n",
      "  Read 2:    11,169,014 bp\n",
      "Total written (filtered):     20,377,422 bp (91.2%)\n",
      "  Read 1:    10,187,175 bp\n",
      "  Read 2:    10,190,247 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t517\t0.0\t2\t0 499 18\n",
      "26\t36994\t0.0\t2\t35662 1249 83\n",
      "27\t221\t0.0\t2\t0 211 10\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t439\t0.0\t2\t0 432 7\n",
      "26\t37025\t0.0\t2\t36527 483 15\n",
      "27\t146\t0.0\t2\t0 146\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2234_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2234_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2234_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2234_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2234_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2234_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2234_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2234_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        59,497 reads  @    113.6 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.76 s (114 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,497\n",
      "  Read 1 with adapter:                  59,453 (99.9%)\n",
      "  Read 2 with adapter:                  59,143 (99.4%)\n",
      "Pairs written (passing filters):        59,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,077,595 bp\n",
      "  Read 1:    17,548,819 bp\n",
      "  Read 2:    17,528,776 bp\n",
      "Total written (filtered):     31,994,604 bp (91.2%)\n",
      "  Read 1:    16,003,339 bp\n",
      "  Read 2:    15,991,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t788\t0.0\t2\t0 767 21\n",
      "26\t58158\t0.0\t2\t56781 1355 22\n",
      "27\t372\t0.0\t2\t0 355 17\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59143 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t523\t0.0\t2\t0 510 13\n",
      "26\t58268\t0.0\t2\t56500 1624 144\n",
      "27\t244\t0.0\t2\t0 234 10\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2235_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2235_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2235_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2235_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2235_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2235_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2235_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2235_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:14       113,115 reads  @    124.0 µs/read;   0.48 M reads/minute\n",
      "Finished in 14.03 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            113,115\n",
      "  Read 1 with adapter:                 113,011 (99.9%)\n",
      "  Read 2 with adapter:                 112,947 (99.9%)\n",
      "Pairs written (passing filters):       113,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,725,519 bp\n",
      "  Read 1:    33,362,581 bp\n",
      "  Read 2:    33,362,938 bp\n",
      "Total written (filtered):     60,851,762 bp (91.2%)\n",
      "  Read 1:    30,425,134 bp\n",
      "  Read 2:    30,426,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 113011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t1682\t0.0\t2\t0 1626 56\n",
      "26\t110411\t0.0\t2\t107176 3151 84\n",
      "27\t693\t0.0\t2\t0 662 31\n",
      "28\t150\t0.0\t2\t0 0 150\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 112947 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t885\t0.0\t2\t0 864 21\n",
      "26\t111421\t0.0\t2\t108335 2944 142\n",
      "27\t529\t0.0\t2\t0 508 21\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2236_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2236_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2236_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2236_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2236_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2236_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2236_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2236_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        34,765 reads  @    132.5 µs/read;   0.45 M reads/minute\n",
      "Finished in 4.61 s (133 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,765\n",
      "  Read 1 with adapter:                  34,739 (99.9%)\n",
      "  Read 2 with adapter:                  33,533 (96.5%)\n",
      "Pairs written (passing filters):        34,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,284,544 bp\n",
      "  Read 1:    10,254,950 bp\n",
      "  Read 2:    10,029,594 bp\n",
      "Total written (filtered):     18,508,550 bp (91.2%)\n",
      "  Read 1:     9,350,694 bp\n",
      "  Read 2:     9,157,856 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t552\t0.0\t2\t0 538 14\n",
      "26\t33927\t0.0\t2\t33016 893 18\n",
      "27\t195\t0.0\t2\t0 191 4\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33533 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t279\t0.0\t2\t0 256 23\n",
      "26\t33075\t0.0\t2\t31521 1370 184\n",
      "27\t139\t0.0\t2\t0 126 13\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2237_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2237_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2237_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2237_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2237_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2237_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2237_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2237_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        62,447 reads  @    124.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 7.79 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,447\n",
      "  Read 1 with adapter:                  62,403 (99.9%)\n",
      "  Read 2 with adapter:                  59,400 (95.1%)\n",
      "Pairs written (passing filters):        62,445 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,309,762 bp\n",
      "  Read 1:    18,420,529 bp\n",
      "  Read 2:    17,889,233 bp\n",
      "Total written (filtered):     33,143,201 bp (91.3%)\n",
      "  Read 1:    16,797,979 bp\n",
      "  Read 2:    16,345,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t919\t0.0\t2\t0 899 20\n",
      "26\t60969\t0.0\t2\t59337 1592 40\n",
      "27\t349\t0.0\t2\t0 343 6\n",
      "28\t109\t0.0\t2\t0 0 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59400 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t747\t0.0\t2\t0 702 45\n",
      "26\t58254\t0.0\t2\t55602 2354 298\n",
      "27\t336\t0.0\t2\t0 307 29\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2238_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2238_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2238_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2238_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2238_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2238_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2238_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2238_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        64,306 reads  @    127.4 µs/read;   0.47 M reads/minute\n",
      "Finished in 8.20 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,306\n",
      "  Read 1 with adapter:                  64,276 (100.0%)\n",
      "  Read 2 with adapter:                  64,045 (99.6%)\n",
      "Pairs written (passing filters):        64,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,916,366 bp\n",
      "  Read 1:    18,969,307 bp\n",
      "  Read 2:    18,947,059 bp\n",
      "Total written (filtered):     34,580,142 bp (91.2%)\n",
      "  Read 1:    17,298,041 bp\n",
      "  Read 2:    17,282,101 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64276 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t954\t0.0\t2\t0 910 44\n",
      "26\t62782\t0.0\t2\t60838 1909 35\n",
      "27\t394\t0.0\t2\t0 379 15\n",
      "28\t101\t0.0\t2\t0 0 101\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t618\t0.0\t2\t0 596 22\n",
      "26\t63021\t0.0\t2\t60844 2029 148\n",
      "27\t306\t0.0\t2\t0 294 12\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2239_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2239_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2239_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2239_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2239_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2239_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2239_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2239_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        90,921 reads  @     95.3 µs/read;   0.63 M reads/minute\n",
      "Finished in 8.67 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             90,921\n",
      "  Read 1 with adapter:                  90,490 (99.5%)\n",
      "  Read 2 with adapter:                  90,140 (99.1%)\n",
      "Pairs written (passing filters):        90,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    53,595,570 bp\n",
      "  Read 1:    26,821,248 bp\n",
      "  Read 2:    26,774,322 bp\n",
      "Total written (filtered):     48,899,480 bp (91.2%)\n",
      "  Read 1:    24,468,564 bp\n",
      "  Read 2:    24,430,916 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 90490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t1264\t0.0\t2\t0 1215 49\n",
      "26\t88219\t0.0\t2\t85959 2155 105\n",
      "27\t538\t0.0\t2\t0 510 28\n",
      "28\t402\t0.0\t2\t0 0 402\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 90140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t768\t0.0\t2\t0 732 36\n",
      "26\t88726\t0.0\t2\t85112 3289 325\n",
      "27\t598\t0.0\t2\t0 544 54\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2240_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2240_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2240_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2240_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2240_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2240_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2240_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2240_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        31,450 reads  @    117.0 µs/read;   0.51 M reads/minute\n",
      "Finished in 3.69 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,450\n",
      "  Read 1 with adapter:                  31,380 (99.8%)\n",
      "  Read 2 with adapter:                  31,394 (99.8%)\n",
      "Pairs written (passing filters):        31,450 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,550,547 bp\n",
      "  Read 1:     9,273,711 bp\n",
      "  Read 2:     9,276,836 bp\n",
      "Total written (filtered):     16,918,861 bp (91.2%)\n",
      "  Read 1:     8,458,137 bp\n",
      "  Read 2:     8,460,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t497\t0.0\t2\t0 478 19\n",
      "26\t30664\t0.0\t2\t29744 900 20\n",
      "27\t167\t0.0\t2\t0 161 6\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31394 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t280\t0.0\t2\t0 273 7\n",
      "26\t30940\t0.0\t2\t30117 785 38\n",
      "27\t140\t0.0\t2\t0 131 9\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2249_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2249_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2249_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2249_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2249_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2249_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2249_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2249_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        16,325 reads  @    129.4 µs/read;   0.46 M reads/minute\n",
      "Finished in 2.12 s (130 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,325\n",
      "  Read 1 with adapter:                  16,269 (99.7%)\n",
      "  Read 2 with adapter:                  16,199 (99.2%)\n",
      "Pairs written (passing filters):        16,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,625,507 bp\n",
      "  Read 1:     4,811,541 bp\n",
      "  Read 2:     4,813,966 bp\n",
      "Total written (filtered):      8,781,429 bp (91.2%)\n",
      "  Read 1:     4,388,529 bp\n",
      "  Read 2:     4,392,900 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 16269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t192\t0.0\t2\t0 174 18\n",
      "26\t15914\t0.0\t2\t14891 960 63\n",
      "27\t80\t0.0\t2\t0 72 8\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 16199 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t179\t0.0\t2\t0 178 1\n",
      "26\t15943\t0.0\t2\t15682 251 10\n",
      "27\t59\t0.0\t2\t0 59\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2250_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2250_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2250_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2250_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2250_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2250_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2250_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2250_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        54,714 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.66 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,714\n",
      "  Read 1 with adapter:                  54,572 (99.7%)\n",
      "  Read 2 with adapter:                  54,528 (99.7%)\n",
      "Pairs written (passing filters):        54,714 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,257,519 bp\n",
      "  Read 1:    16,119,010 bp\n",
      "  Read 2:    16,138,509 bp\n",
      "Total written (filtered):     29,421,289 bp (91.2%)\n",
      "  Read 1:    14,700,323 bp\n",
      "  Read 2:    14,720,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t622\t0.0\t2\t0 615 7\n",
      "26\t53523\t0.0\t2\t52503 991 29\n",
      "27\t305\t0.0\t2\t0 295 10\n",
      "28\t94\t0.0\t2\t0 0 94\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54528 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t428\t0.0\t2\t0 411 17\n",
      "26\t53801\t0.0\t2\t52229 1440 132\n",
      "27\t259\t0.0\t2\t0 247 12\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2251_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2251_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2251_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2251_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2251_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2251_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2251_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2251_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        58,700 reads  @    110.7 µs/read;   0.54 M reads/minute\n",
      "Finished in 6.50 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,700\n",
      "  Read 1 with adapter:                  58,533 (99.7%)\n",
      "  Read 2 with adapter:                  58,578 (99.8%)\n",
      "Pairs written (passing filters):        58,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,613,902 bp\n",
      "  Read 1:    17,298,746 bp\n",
      "  Read 2:    17,315,156 bp\n",
      "Total written (filtered):     31,569,247 bp (91.2%)\n",
      "  Read 1:    15,776,979 bp\n",
      "  Read 2:    15,792,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58533 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t662\t0.0\t2\t0 645 17\n",
      "26\t57364\t0.0\t2\t56333 997 34\n",
      "27\t335\t0.0\t2\t0 321 14\n",
      "28\t145\t0.0\t2\t0 0 145\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58578 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t456\t0.0\t2\t0 438 18\n",
      "26\t57827\t0.0\t2\t56529 1204 94\n",
      "27\t234\t0.0\t2\t0 227 7\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2252_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2252_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2252_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2252_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2252_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2252_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2252_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2252_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        35,571 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.77 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,571\n",
      "  Read 1 with adapter:                  35,421 (99.6%)\n",
      "  Read 2 with adapter:                  34,985 (98.4%)\n",
      "Pairs written (passing filters):        35,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,912,707 bp\n",
      "  Read 1:    10,484,209 bp\n",
      "  Read 2:    10,428,498 bp\n",
      "Total written (filtered):     19,080,497 bp (91.2%)\n",
      "  Read 1:     9,561,552 bp\n",
      "  Read 2:     9,518,945 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35421 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t327\t0.0\t2\t0 319 8\n",
      "26\t34660\t0.0\t2\t34090 552 18\n",
      "27\t388\t0.0\t2\t0 293 95\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t260\t0.0\t2\t0 224 36\n",
      "26\t34533\t0.0\t2\t32927 1407 199\n",
      "27\t125\t0.0\t2\t0 118 7\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2253_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2253_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2253_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2253_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2253_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2253_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2253_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2253_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        12,294 reads  @    117.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 1.45 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,294\n",
      "  Read 1 with adapter:                  12,231 (99.5%)\n",
      "  Read 2 with adapter:                  10,787 (87.7%)\n",
      "Pairs written (passing filters):        12,291 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,985,380 bp\n",
      "  Read 1:     3,619,858 bp\n",
      "  Read 2:     3,365,522 bp\n",
      "Total written (filtered):      6,386,294 bp (91.4%)\n",
      "  Read 1:     3,301,133 bp\n",
      "  Read 2:     3,085,161 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 12231 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t154\t0.0\t2\t0 152 2\n",
      "26\t11950\t0.0\t2\t11756 186 8\n",
      "27\t92\t0.0\t2\t0 78 14\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 10787 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t162\t0.0\t2\t0 149 13\n",
      "26\t10555\t0.0\t2\t10153 346 56\n",
      "27\t51\t0.0\t2\t0 48 3\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2254_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2254_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2254_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2254_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2254_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2254_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2254_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2254_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        55,488 reads  @    103.4 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.74 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,488\n",
      "  Read 1 with adapter:                  55,265 (99.6%)\n",
      "  Read 2 with adapter:                  55,303 (99.7%)\n",
      "Pairs written (passing filters):        55,487 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,707,030 bp\n",
      "  Read 1:    16,349,228 bp\n",
      "  Read 2:    16,357,802 bp\n",
      "Total written (filtered):     29,832,157 bp (91.2%)\n",
      "  Read 1:    14,912,108 bp\n",
      "  Read 2:    14,920,049 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55265 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t569\t0.0\t2\t0 554 15\n",
      "26\t54174\t0.0\t2\t53264 884 26\n",
      "27\t386\t0.0\t2\t0 375 11\n",
      "28\t104\t0.0\t2\t0 0 104\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55303 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t443\t0.0\t2\t0 424 19\n",
      "26\t54553\t0.0\t2\t53348 1114 91\n",
      "27\t188\t0.0\t2\t0 180 8\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2255_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2255_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2255_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2255_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2255_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2255_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2255_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2255_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        66,478 reads  @    121.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 8.05 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,478\n",
      "  Read 1 with adapter:                  66,052 (99.4%)\n",
      "  Read 2 with adapter:                  66,117 (99.5%)\n",
      "Pairs written (passing filters):        66,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,179,874 bp\n",
      "  Read 1:    19,585,482 bp\n",
      "  Read 2:    19,594,392 bp\n",
      "Total written (filtered):     35,743,585 bp (91.2%)\n",
      "  Read 1:    17,867,982 bp\n",
      "  Read 2:    17,875,603 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66052 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t785\t0.0\t2\t0 757 28\n",
      "26\t64347\t0.0\t2\t62841 1442 64\n",
      "27\t663\t0.0\t2\t0 603 60\n",
      "28\t196\t0.0\t2\t0 0 196\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66117 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t607\t0.0\t2\t0 579 28\n",
      "26\t65139\t0.0\t2\t63329 1672 138\n",
      "27\t272\t0.0\t2\t0 252 20\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2256_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2256_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2256_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2256_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2256_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2256_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2256_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2256_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        53,215 reads  @    112.1 µs/read;   0.54 M reads/minute\n",
      "Finished in 5.97 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,215\n",
      "  Read 1 with adapter:                  52,982 (99.6%)\n",
      "  Read 2 with adapter:                  53,099 (99.8%)\n",
      "Pairs written (passing filters):        53,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,380,064 bp\n",
      "  Read 1:    15,683,409 bp\n",
      "  Read 2:    15,696,655 bp\n",
      "Total written (filtered):     28,622,262 bp (91.2%)\n",
      "  Read 1:    14,306,041 bp\n",
      "  Read 2:    14,316,221 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t637\t0.0\t2\t0 622 15\n",
      "26\t51829\t0.0\t2\t50549 1233 47\n",
      "27\t423\t0.0\t2\t0 380 43\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53099 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t393\t0.0\t2\t0 377 16\n",
      "26\t52429\t0.0\t2\t51415 962 52\n",
      "27\t201\t0.0\t2\t0 197 4\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2257_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2257_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2257_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2257_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2257_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2257_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2257_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2257_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        45,360 reads  @    100.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.55 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,360\n",
      "  Read 1 with adapter:                  45,293 (99.9%)\n",
      "  Read 2 with adapter:                  44,955 (99.1%)\n",
      "Pairs written (passing filters):        45,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,753,592 bp\n",
      "  Read 1:    13,379,128 bp\n",
      "  Read 2:    13,374,464 bp\n",
      "Total written (filtered):     24,407,494 bp (91.2%)\n",
      "  Read 1:    12,201,503 bp\n",
      "  Read 2:    12,205,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t483\t0.0\t2\t0 469 14\n",
      "26\t44373\t0.0\t2\t43442 895 36\n",
      "27\t288\t0.0\t2\t0 272 16\n",
      "28\t125\t0.0\t2\t0 0 125\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44955 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t543\t0.0\t2\t0 535 8\n",
      "26\t44166\t0.0\t2\t43290 839 37\n",
      "27\t170\t0.0\t2\t0 167 3\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2258_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2258_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2258_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2258_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2258_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2258_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2258_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2258_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,324 reads  @     88.5 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.57 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,324\n",
      "  Read 1 with adapter:                  40,301 (99.9%)\n",
      "  Read 2 with adapter:                  40,242 (99.8%)\n",
      "Pairs written (passing filters):        40,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,787,590 bp\n",
      "  Read 1:    11,894,932 bp\n",
      "  Read 2:    11,892,658 bp\n",
      "Total written (filtered):     21,693,591 bp (91.2%)\n",
      "  Read 1:    10,847,127 bp\n",
      "  Read 2:    10,846,464 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40301 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t495\t0.0\t2\t0 485 10\n",
      "26\t39405\t0.0\t2\t38519 858 28\n",
      "27\t244\t0.0\t2\t0 241 3\n",
      "28\t136\t0.0\t2\t0 0 136\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40242 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t289\t0.0\t2\t0 278 11\n",
      "26\t39751\t0.0\t2\t38690 976 85\n",
      "27\t137\t0.0\t2\t0 132 5\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2259_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2259_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2259_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2259_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2259_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2259_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2259_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2259_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        56,434 reads  @    115.0 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.49 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,434\n",
      "  Read 1 with adapter:                  56,344 (99.8%)\n",
      "  Read 2 with adapter:                  56,285 (99.7%)\n",
      "Pairs written (passing filters):        56,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,289,801 bp\n",
      "  Read 1:    16,643,595 bp\n",
      "  Read 2:    16,646,206 bp\n",
      "Total written (filtered):     30,361,692 bp (91.2%)\n",
      "  Read 1:    15,178,721 bp\n",
      "  Read 2:    15,182,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56344 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t624\t0.0\t2\t0 610 14\n",
      "26\t55188\t0.0\t2\t54222 942 24\n",
      "27\t378\t0.0\t2\t0 367 11\n",
      "28\t121\t0.0\t2\t0 0 121\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56285 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t426\t0.0\t2\t0 408 18\n",
      "26\t55570\t0.0\t2\t53959 1467 144\n",
      "27\t227\t0.0\t2\t0 205 22\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2260_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2260_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2260_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2260_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2260_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2260_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2260_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2260_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        26,990 reads  @    123.1 µs/read;   0.49 M reads/minute\n",
      "Finished in 3.33 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,990\n",
      "  Read 1 with adapter:                  26,952 (99.9%)\n",
      "  Read 2 with adapter:                  26,299 (97.4%)\n",
      "Pairs written (passing filters):        26,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,811,695 bp\n",
      "  Read 1:     7,960,503 bp\n",
      "  Read 2:     7,851,192 bp\n",
      "Total written (filtered):     14,426,295 bp (91.2%)\n",
      "  Read 1:     7,258,804 bp\n",
      "  Read 2:     7,167,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t355\t0.0\t2\t0 346 9\n",
      "26\t26354\t0.0\t2\t25605 730 19\n",
      "27\t180\t0.0\t2\t0 165 15\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26299 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t222\t0.0\t2\t0 209 13\n",
      "26\t25903\t0.0\t2\t24709 1059 135\n",
      "27\t151\t0.0\t2\t0 134 17\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2261_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2261_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2261_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2261_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2261_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2261_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2261_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2261_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        44,902 reads  @    112.4 µs/read;   0.53 M reads/minute\n",
      "Finished in 5.05 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,902\n",
      "  Read 1 with adapter:                  44,784 (99.7%)\n",
      "  Read 2 with adapter:                  42,197 (94.0%)\n",
      "Pairs written (passing filters):        44,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,015,911 bp\n",
      "  Read 1:    13,239,329 bp\n",
      "  Read 2:    12,776,582 bp\n",
      "Total written (filtered):     23,754,306 bp (91.3%)\n",
      "  Read 1:    12,074,627 bp\n",
      "  Read 2:    11,679,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44784 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t566\t0.0\t2\t0 552 14\n",
      "26\t43844\t0.0\t2\t43013 806 25\n",
      "27\t274\t0.0\t2\t0 263 11\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42197 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t479\t0.0\t2\t0 451 28\n",
      "26\t41419\t0.0\t2\t39808 1416 195\n",
      "27\t238\t0.0\t2\t0 213 25\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2262_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2262_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2262_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2262_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2262_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2262_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2262_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2262_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        73,709 reads  @    117.6 µs/read;   0.51 M reads/minute\n",
      "Finished in 8.67 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,709\n",
      "  Read 1 with adapter:                  73,530 (99.8%)\n",
      "  Read 2 with adapter:                  73,461 (99.7%)\n",
      "Pairs written (passing filters):        73,707 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,473,900 bp\n",
      "  Read 1:    21,741,623 bp\n",
      "  Read 2:    21,732,277 bp\n",
      "Total written (filtered):     39,652,152 bp (91.2%)\n",
      "  Read 1:    19,829,518 bp\n",
      "  Read 2:    19,822,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73530 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t893\t0.0\t2\t0 879 14\n",
      "26\t71940\t0.0\t2\t70572 1331 37\n",
      "27\t486\t0.0\t2\t0 462 24\n",
      "28\t154\t0.0\t2\t0 0 154\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 73461 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t696\t0.0\t2\t0 669 27\n",
      "26\t72348\t0.0\t2\t70473 1733 142\n",
      "27\t313\t0.0\t2\t0 308 5\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2263_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2263_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2263_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2263_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2263_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2263_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2263_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2263_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        50,892 reads  @    117.2 µs/read;   0.51 M reads/minute\n",
      "Finished in 5.97 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,892\n",
      "  Read 1 with adapter:                  50,788 (99.8%)\n",
      "  Read 2 with adapter:                  50,689 (99.6%)\n",
      "Pairs written (passing filters):        50,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,008,135 bp\n",
      "  Read 1:    15,007,253 bp\n",
      "  Read 2:    15,000,882 bp\n",
      "Total written (filtered):     27,370,186 bp (91.2%)\n",
      "  Read 1:    13,687,034 bp\n",
      "  Read 2:    13,683,152 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50788 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t694\t0.0\t2\t0 671 23\n",
      "26\t49652\t0.0\t2\t48319 1295 38\n",
      "27\t335\t0.0\t2\t0 329 6\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50689 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t449\t0.0\t2\t0 428 21\n",
      "26\t49950\t0.0\t2\t48397 1448 105\n",
      "27\t223\t0.0\t2\t0 214 9\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2264_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2264_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2264_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2264_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2264_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2264_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2264_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2264_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        66,011 reads  @    132.0 µs/read;   0.45 M reads/minute\n",
      "Finished in 8.71 s (132 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,011\n",
      "  Read 1 with adapter:                  64,761 (98.1%)\n",
      "  Read 2 with adapter:                  65,829 (99.7%)\n",
      "Pairs written (passing filters):        66,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,938,539 bp\n",
      "  Read 1:    19,468,123 bp\n",
      "  Read 2:    19,470,416 bp\n",
      "Total written (filtered):     35,544,206 bp (91.3%)\n",
      "  Read 1:    17,784,989 bp\n",
      "  Read 2:    17,759,217 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64761 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t1083\t0.0\t2\t0 1042 41\n",
      "26\t63204\t0.0\t2\t61156 2002 46\n",
      "27\t325\t0.0\t2\t0 310 15\n",
      "28\t101\t0.0\t2\t0 0 101\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t586\t0.0\t2\t0 563 23\n",
      "26\t64944\t0.0\t2\t63224 1650 70\n",
      "27\t243\t0.0\t2\t0 237 6\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2265_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2265_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2265_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2265_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2265_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2265_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2265_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2265_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        41,629 reads  @    100.1 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.17 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,629\n",
      "  Read 1 with adapter:                  41,524 (99.7%)\n",
      "  Read 2 with adapter:                  41,274 (99.1%)\n",
      "Pairs written (passing filters):        41,629 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,549,947 bp\n",
      "  Read 1:    12,274,822 bp\n",
      "  Read 2:    12,275,125 bp\n",
      "Total written (filtered):     22,397,533 bp (91.2%)\n",
      "  Read 1:    11,195,231 bp\n",
      "  Read 2:    11,202,302 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41524 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t324\t0.0\t2\t0 312 12\n",
      "26\t40897\t0.0\t2\t40202 677 18\n",
      "27\t219\t0.0\t2\t0 213 6\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41274 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t443\t0.0\t2\t0 437 6\n",
      "26\t40629\t0.0\t2\t39851 734 44\n",
      "27\t154\t0.0\t2\t0 152 2\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2266_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2266_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2266_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2266_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2266_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2266_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2266_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2266_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,971 reads  @    125.6 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.53 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,971\n",
      "  Read 1 with adapter:                  43,881 (99.8%)\n",
      "  Read 2 with adapter:                  43,800 (99.6%)\n",
      "Pairs written (passing filters):        43,971 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,926,991 bp\n",
      "  Read 1:    12,961,334 bp\n",
      "  Read 2:    12,965,657 bp\n",
      "Total written (filtered):     23,647,474 bp (91.2%)\n",
      "  Read 1:    11,820,509 bp\n",
      "  Read 2:    11,826,965 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t438\t0.0\t2\t0 433 5\n",
      "26\t43093\t0.0\t2\t41967 1101 25\n",
      "27\t283\t0.0\t2\t0 280 3\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43800 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t429\t0.0\t2\t0 402 27\n",
      "26\t43027\t0.0\t2\t41641 1285 101\n",
      "27\t279\t0.0\t2\t0 268 11\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2267_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2267_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2267_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2267_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2267_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2267_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2267_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2267_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        68,852 reads  @    117.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 8.12 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,852\n",
      "  Read 1 with adapter:                  61,540 (89.4%)\n",
      "  Read 2 with adapter:                  68,483 (99.5%)\n",
      "Pairs written (passing filters):        68,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,612,802 bp\n",
      "  Read 1:    20,304,657 bp\n",
      "  Read 2:    20,308,145 bp\n",
      "Total written (filtered):     37,232,356 bp (91.7%)\n",
      "  Read 1:    18,704,477 bp\n",
      "  Read 2:    18,527,879 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61540 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t578\t0.0\t2\t0 551 27\n",
      "26\t60373\t0.0\t2\t59090 1230 53\n",
      "27\t352\t0.0\t2\t0 336 16\n",
      "28\t210\t0.0\t2\t0 0 210\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68483 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t573\t0.0\t2\t0 553 20\n",
      "26\t67557\t0.0\t2\t65948 1511 98\n",
      "27\t293\t0.0\t2\t0 277 16\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2268_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2268_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2268_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2268_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2268_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2268_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2268_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2268_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        33,903 reads  @    100.6 µs/read;   0.60 M reads/minute\n",
      "Finished in 3.42 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,903\n",
      "  Read 1 with adapter:                  33,851 (99.8%)\n",
      "  Read 2 with adapter:                  33,117 (97.7%)\n",
      "Pairs written (passing filters):        33,899 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,883,839 bp\n",
      "  Read 1:     9,997,719 bp\n",
      "  Read 2:     9,886,120 bp\n",
      "Total written (filtered):     18,141,717 bp (91.2%)\n",
      "  Read 1:     9,116,524 bp\n",
      "  Read 2:     9,025,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33851 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t303\t0.0\t2\t0 298 5\n",
      "26\t33251\t0.0\t2\t32592 641 18\n",
      "27\t210\t0.0\t2\t0 182 28\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33117 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t275\t0.0\t2\t0 263 12\n",
      "26\t32644\t0.0\t2\t31506 1002 136\n",
      "27\t176\t0.0\t2\t0 158 18\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2269_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2269_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2269_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2269_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2269_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2269_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2269_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2269_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:09        83,189 reads  @    110.9 µs/read;   0.54 M reads/minute\n",
      "Finished in 9.23 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,189\n",
      "  Read 1 with adapter:                  83,001 (99.8%)\n",
      "  Read 2 with adapter:                  81,225 (97.6%)\n",
      "Pairs written (passing filters):        83,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,751,590 bp\n",
      "  Read 1:    24,529,324 bp\n",
      "  Read 2:    24,222,266 bp\n",
      "Total written (filtered):     44,481,161 bp (91.2%)\n",
      "  Read 1:    22,370,194 bp\n",
      "  Read 2:    22,110,967 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 83001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t722\t0.0\t2\t0 701 21\n",
      "26\t81092\t0.0\t2\t79641 1392 59\n",
      "27\t914\t0.0\t2\t0 802 112\n",
      "28\t230\t0.0\t2\t0 0 230\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 81225 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t905\t0.0\t2\t0 871 34\n",
      "26\t79867\t0.0\t2\t77409 2202 256\n",
      "27\t376\t0.0\t2\t0 342 34\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2270_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2270_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2270_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2270_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2270_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2270_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2270_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2270_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11       114,853 reads  @    100.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 11.53 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,853\n",
      "  Read 1 with adapter:                 114,581 (99.8%)\n",
      "  Read 2 with adapter:                 114,436 (99.6%)\n",
      "Pairs written (passing filters):       114,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    67,734,012 bp\n",
      "  Read 1:    33,873,889 bp\n",
      "  Read 2:    33,860,123 bp\n",
      "Total written (filtered):     61,778,400 bp (91.2%)\n",
      "  Read 1:    30,893,258 bp\n",
      "  Read 2:    30,885,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 114581 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t93\t0.0\t2\t0 0 93\n",
      "25\t1063\t0.0\t2\t0 1020 43\n",
      "26\t111172\t0.0\t2\t108264 2791 117\n",
      "27\t1732\t0.0\t2\t0 1527 205\n",
      "28\t521\t0.0\t2\t0 0 521\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 114436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t1151\t0.0\t2\t0 1107 44\n",
      "26\t112446\t0.0\t2\t109248 2977 221\n",
      "27\t662\t0.0\t2\t0 639 23\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2271_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2271_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2271_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2271_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2271_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2271_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2271_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2271_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        56,091 reads  @    121.1 µs/read;   0.50 M reads/minute\n",
      "Finished in 6.79 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,091\n",
      "  Read 1 with adapter:                  55,959 (99.8%)\n",
      "  Read 2 with adapter:                  55,898 (99.7%)\n",
      "Pairs written (passing filters):        56,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,076,442 bp\n",
      "  Read 1:    16,534,500 bp\n",
      "  Read 2:    16,541,942 bp\n",
      "Total written (filtered):     30,168,169 bp (91.2%)\n",
      "  Read 1:    15,079,650 bp\n",
      "  Read 2:    15,088,519 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55959 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t539\t0.0\t2\t0 530 9\n",
      "26\t55003\t0.0\t2\t54018 962 23\n",
      "27\t315\t0.0\t2\t0 306 9\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55898 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t464\t0.0\t2\t0 450 14\n",
      "26\t55144\t0.0\t2\t53733 1302 109\n",
      "27\t218\t0.0\t2\t0 210 8\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2272_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2272_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2272_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2272_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2272_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2272_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2272_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2272_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        49,084 reads  @    104.0 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.11 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,084\n",
      "  Read 1 with adapter:                  48,970 (99.8%)\n",
      "  Read 2 with adapter:                  48,992 (99.8%)\n",
      "Pairs written (passing filters):        49,084 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,951,464 bp\n",
      "  Read 1:    14,473,901 bp\n",
      "  Read 2:    14,477,563 bp\n",
      "Total written (filtered):     26,404,650 bp (91.2%)\n",
      "  Read 1:    13,200,700 bp\n",
      "  Read 2:    13,203,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48970 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t400\t0.0\t2\t0 389 11\n",
      "26\t48183\t0.0\t2\t47474 687 22\n",
      "27\t321\t0.0\t2\t0 308 13\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t386\t0.0\t2\t0 373 13\n",
      "26\t48373\t0.0\t2\t47234 1065 74\n",
      "27\t167\t0.0\t2\t0 160 7\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2273_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2273_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2273_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2273_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2273_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2273_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2273_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2273_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        66,810 reads  @    131.8 µs/read;   0.46 M reads/minute\n",
      "Finished in 8.81 s (132 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,810\n",
      "  Read 1 with adapter:                  66,637 (99.7%)\n",
      "  Read 2 with adapter:                  66,259 (99.2%)\n",
      "Pairs written (passing filters):        66,810 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,402,822 bp\n",
      "  Read 1:    19,701,245 bp\n",
      "  Read 2:    19,701,577 bp\n",
      "Total written (filtered):     35,947,685 bp (91.2%)\n",
      "  Read 1:    17,968,304 bp\n",
      "  Read 2:    17,979,381 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t676\t0.0\t2\t0 630 46\n",
      "26\t64988\t0.0\t2\t62897 1968 123\n",
      "27\t699\t0.0\t2\t0 627 72\n",
      "28\t226\t0.0\t2\t0 0 226\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66259 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t769\t0.0\t2\t0 755 14\n",
      "26\t65145\t0.0\t2\t63958 1112 75\n",
      "27\t259\t0.0\t2\t0 256 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2274_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2274_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2274_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2274_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2274_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2274_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2274_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2274_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        46,419 reads  @    104.6 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.86 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,419\n",
      "  Read 1 with adapter:                  46,352 (99.9%)\n",
      "  Read 2 with adapter:                  46,080 (99.3%)\n",
      "Pairs written (passing filters):        46,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,363,103 bp\n",
      "  Read 1:    13,692,158 bp\n",
      "  Read 2:    13,670,945 bp\n",
      "Total written (filtered):     24,959,989 bp (91.2%)\n",
      "  Read 1:    12,486,973 bp\n",
      "  Read 2:    12,473,016 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t437\t0.0\t2\t0 420 17\n",
      "26\t45447\t0.0\t2\t44547 873 27\n",
      "27\t370\t0.0\t2\t0 339 31\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t392\t0.0\t2\t0 375 17\n",
      "26\t45448\t0.0\t2\t44347 1012 89\n",
      "27\t175\t0.0\t2\t0 170 5\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2275_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2275_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2275_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2275_133_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2275_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2275_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2275_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2275_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        52,794 reads  @    117.4 µs/read;   0.51 M reads/minute\n",
      "Finished in 6.20 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,794\n",
      "  Read 1 with adapter:                  52,678 (99.8%)\n",
      "  Read 2 with adapter:                  52,643 (99.7%)\n",
      "Pairs written (passing filters):        52,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,142,862 bp\n",
      "  Read 1:    15,571,394 bp\n",
      "  Read 2:    15,571,468 bp\n",
      "Total written (filtered):     28,404,493 bp (91.2%)\n",
      "  Read 1:    14,201,625 bp\n",
      "  Read 2:    14,202,868 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t515\t0.0\t2\t0 496 19\n",
      "26\t51525\t0.0\t2\t50483 1007 35\n",
      "27\t496\t0.0\t2\t0 463 33\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t389\t0.0\t2\t0 374 15\n",
      "26\t51956\t0.0\t2\t50623 1234 99\n",
      "27\t237\t0.0\t2\t0 223 14\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2276_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2276_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2276_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2276_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2276_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2276_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2276_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2276_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,226 reads  @    108.8 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.38 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,226\n",
      "  Read 1 with adapter:                  39,398 (97.9%)\n",
      "  Read 2 with adapter:                  38,504 (95.7%)\n",
      "Pairs written (passing filters):        40,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,550,018 bp\n",
      "  Read 1:    11,859,114 bp\n",
      "  Read 2:    11,690,904 bp\n",
      "Total written (filtered):     21,523,850 bp (91.4%)\n",
      "  Read 1:    10,833,876 bp\n",
      "  Read 2:    10,689,974 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39398 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t447\t0.0\t2\t0 439 8\n",
      "26\t38641\t0.0\t2\t37958 669 14\n",
      "27\t275\t0.0\t2\t0 201 74\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38504 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t300\t0.0\t2\t0 274 26\n",
      "26\t37951\t0.0\t2\t36397 1359 195\n",
      "27\t144\t0.0\t2\t0 131 13\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2277_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2277_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2277_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2277_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2277_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2277_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2277_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2277_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        41,663 reads  @    121.2 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.05 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,663\n",
      "  Read 1 with adapter:                  41,481 (99.6%)\n",
      "  Read 2 with adapter:                  39,906 (95.8%)\n",
      "Pairs written (passing filters):        41,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,293,548 bp\n",
      "  Read 1:    12,287,296 bp\n",
      "  Read 2:    12,006,252 bp\n",
      "Total written (filtered):     22,172,898 bp (91.3%)\n",
      "  Read 1:    11,203,962 bp\n",
      "  Read 2:    10,968,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t471\t0.0\t2\t0 458 13\n",
      "26\t40610\t0.0\t2\t39901 689 20\n",
      "27\t299\t0.0\t2\t0 276 23\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39906 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t462\t0.0\t2\t0 435 27\n",
      "26\t39203\t0.0\t2\t37697 1334 172\n",
      "27\t216\t0.0\t2\t0 196 20\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2278_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2278_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2278_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2278_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2278_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2278_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2278_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2278_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        57,483 reads  @    105.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 6.07 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,483\n",
      "  Read 1 with adapter:                  57,410 (99.9%)\n",
      "  Read 2 with adapter:                  57,331 (99.7%)\n",
      "Pairs written (passing filters):        57,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,909,973 bp\n",
      "  Read 1:    16,956,513 bp\n",
      "  Read 2:    16,953,460 bp\n",
      "Total written (filtered):     30,927,350 bp (91.2%)\n",
      "  Read 1:    15,464,239 bp\n",
      "  Read 2:    15,463,111 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57410 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t619\t0.0\t2\t0 604 15\n",
      "26\t56464\t0.0\t2\t55538 904 22\n",
      "27\t269\t0.0\t2\t0 258 11\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57331 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t529\t0.0\t2\t0 501 28\n",
      "26\t56511\t0.0\t2\t55305 1116 90\n",
      "27\t162\t0.0\t2\t0 158 4\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2279_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2279_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2279_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2279_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2279_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2279_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2279_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2279_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        49,408 reads  @    104.4 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.16 s (104 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,408\n",
      "  Read 1 with adapter:                  49,106 (99.4%)\n",
      "  Read 2 with adapter:                  49,205 (99.6%)\n",
      "Pairs written (passing filters):        49,408 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,122,493 bp\n",
      "  Read 1:    14,553,321 bp\n",
      "  Read 2:    14,569,172 bp\n",
      "Total written (filtered):     26,566,732 bp (91.2%)\n",
      "  Read 1:    13,276,743 bp\n",
      "  Read 2:    13,289,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49106 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t477\t0.0\t2\t0 466 11\n",
      "26\t48306\t0.0\t2\t47539 746 21\n",
      "27\t247\t0.0\t2\t0 240 7\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t383\t0.0\t2\t0 361 22\n",
      "26\t48549\t0.0\t2\t47462 1013 74\n",
      "27\t214\t0.0\t2\t0 201 13\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2280_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2280_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2280_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2280_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2280_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2280_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2280_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2280_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,142 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.11 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,142\n",
      "  Read 1 with adapter:                  37,024 (99.7%)\n",
      "  Read 2 with adapter:                  37,084 (99.8%)\n",
      "Pairs written (passing filters):        37,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,903,753 bp\n",
      "  Read 1:    10,948,997 bp\n",
      "  Read 2:    10,954,756 bp\n",
      "Total written (filtered):     19,977,132 bp (91.2%)\n",
      "  Read 1:     9,986,424 bp\n",
      "  Read 2:     9,990,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37024 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t347\t0.0\t2\t0 340 7\n",
      "26\t36379\t0.0\t2\t35744 608 27\n",
      "27\t224\t0.0\t2\t0 218 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37084 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t259\t0.0\t2\t0 253 6\n",
      "26\t36676\t0.0\t2\t35885 735 56\n",
      "27\t115\t0.0\t2\t0 110 5\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2281_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2281_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2281_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2281_145_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2281_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2281_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2281_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2281_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        27,689 reads  @    125.7 µs/read;   0.48 M reads/minute\n",
      "Finished in 3.48 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,689\n",
      "  Read 1 with adapter:                  27,636 (99.8%)\n",
      "  Read 2 with adapter:                  27,473 (99.2%)\n",
      "Pairs written (passing filters):        27,689 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,330,882 bp\n",
      "  Read 1:     8,166,971 bp\n",
      "  Read 2:     8,163,911 bp\n",
      "Total written (filtered):     14,898,356 bp (91.2%)\n",
      "  Read 1:     7,448,525 bp\n",
      "  Read 2:     7,449,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27636 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t289\t0.0\t2\t0 278 11\n",
      "26\t27155\t0.0\t2\t26209 879 67\n",
      "27\t125\t0.0\t2\t0 119 6\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t329\t0.0\t2\t0 323 6\n",
      "26\t27004\t0.0\t2\t26611 379 14\n",
      "27\t109\t0.0\t2\t0 109\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2282_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2282_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2282_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2282_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2282_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2282_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2282_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2282_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        55,670 reads  @    114.4 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.37 s (114 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,670\n",
      "  Read 1 with adapter:                  55,596 (99.9%)\n",
      "  Read 2 with adapter:                  55,463 (99.6%)\n",
      "Pairs written (passing filters):        55,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,834,198 bp\n",
      "  Read 1:    16,421,086 bp\n",
      "  Read 2:    16,413,112 bp\n",
      "Total written (filtered):     29,946,998 bp (91.2%)\n",
      "  Read 1:    14,975,689 bp\n",
      "  Read 2:    14,971,309 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t538\t0.0\t2\t0 517 21\n",
      "26\t54612\t0.0\t2\t53374 1199 39\n",
      "27\t337\t0.0\t2\t0 323 14\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55463 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t480\t0.0\t2\t0 464 16\n",
      "26\t54673\t0.0\t2\t53600 1026 47\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2283_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2283_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2283_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2283_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2283_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2283_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2283_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2283_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        53,247 reads  @    114.8 µs/read;   0.52 M reads/minute\n",
      "Finished in 6.12 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,247\n",
      "  Read 1 with adapter:                  53,211 (99.9%)\n",
      "  Read 2 with adapter:                  53,130 (99.8%)\n",
      "Pairs written (passing filters):        53,247 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,410,049 bp\n",
      "  Read 1:    15,706,631 bp\n",
      "  Read 2:    15,703,418 bp\n",
      "Total written (filtered):     28,645,579 bp (91.2%)\n",
      "  Read 1:    14,323,412 bp\n",
      "  Read 2:    14,322,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53211 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t580\t0.0\t2\t0 569 11\n",
      "26\t52267\t0.0\t2\t50782 1450 35\n",
      "27\t279\t0.0\t2\t0 270 9\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53130 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t413\t0.0\t2\t0 387 26\n",
      "26\t52396\t0.0\t2\t50614 1666 116\n",
      "27\t262\t0.0\t2\t0 250 12\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2284_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2284_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2284_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2284_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2284_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2284_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2284_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2284_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        28,648 reads  @    108.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 3.10 s (108 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,648\n",
      "  Read 1 with adapter:                  28,624 (99.9%)\n",
      "  Read 2 with adapter:                  24,861 (86.8%)\n",
      "Pairs written (passing filters):        28,632 (99.9%)\n",
      "\n",
      "Total basepairs processed:    16,179,616 bp\n",
      "  Read 1:     8,451,147 bp\n",
      "  Read 2:     7,728,469 bp\n",
      "Total written (filtered):     14,784,891 bp (91.4%)\n",
      "  Read 1:     7,702,756 bp\n",
      "  Read 2:     7,082,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28624 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t312\t0.0\t2\t0 301 11\n",
      "26\t28133\t0.0\t2\t27421 690 22\n",
      "27\t139\t0.0\t2\t0 133 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24861 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t181\t0.0\t2\t0 159 22\n",
      "26\t24527\t0.0\t2\t23196 1178 153\n",
      "27\t121\t0.0\t2\t0 112 9\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2285_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2285_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2285_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2285_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2285_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2285_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2285_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2285_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[--------->8 ] 00:00:29       245,220 reads  @    118.5 µs/read;   0.51 M reads/minute\n",
      "Finished in 29.06 s (119 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            245,220\n",
      "  Read 1 with adapter:                 245,080 (99.9%)\n",
      "  Read 2 with adapter:                 242,408 (98.9%)\n",
      "Pairs written (passing filters):       245,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,304,606 bp\n",
      "  Read 1:    72,336,324 bp\n",
      "  Read 2:    71,968,282 bp\n",
      "Total written (filtered):    131,631,088 bp (91.2%)\n",
      "  Read 1:    65,964,073 bp\n",
      "  Read 2:    65,667,015 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 245080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t90\t0.0\t2\t0 0 90\n",
      "25\t2331\t0.0\t2\t0 2257 74\n",
      "26\t241077\t0.0\t2\t235705 5209 163\n",
      "27\t1289\t0.0\t2\t0 1238 51\n",
      "28\t293\t0.0\t2\t0 0 293\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 242408 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t158\t0.0\t2\t0 0 158\n",
      "25\t2855\t0.0\t2\t0 2663 192\n",
      "26\t237759\t0.0\t2\t227451 9159 1149\n",
      "27\t1442\t0.0\t2\t0 1320 122\n",
      "28\t194\t0.0\t2\t0 0 194\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2286_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2286_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2286_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2286_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2286_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2286_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2286_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2286_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[      8<----] 00:00:15       127,363 reads  @    117.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 15.01 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            127,363\n",
      "  Read 1 with adapter:                 127,293 (99.9%)\n",
      "  Read 2 with adapter:                 126,951 (99.7%)\n",
      "Pairs written (passing filters):       127,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,104,708 bp\n",
      "  Read 1:    37,570,350 bp\n",
      "  Read 2:    37,534,358 bp\n",
      "Total written (filtered):     68,494,204 bp (91.2%)\n",
      "  Read 1:    34,260,069 bp\n",
      "  Read 2:    34,234,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 127293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t1254\t0.0\t2\t0 1210 44\n",
      "26\t125208\t0.0\t2\t122303 2823 82\n",
      "27\t617\t0.0\t2\t0 598 19\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 126951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t1220\t0.0\t2\t0 1174 46\n",
      "26\t124995\t0.0\t2\t121360 3402 233\n",
      "27\t571\t0.0\t2\t0 547 24\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2287_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2287_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2287_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2287_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2287_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2287_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2287_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2287_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        75,986 reads  @    105.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 8.02 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,986\n",
      "  Read 1 with adapter:                  75,702 (99.6%)\n",
      "  Read 2 with adapter:                  75,643 (99.5%)\n",
      "Pairs written (passing filters):        75,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,809,109 bp\n",
      "  Read 1:    22,415,340 bp\n",
      "  Read 2:    22,393,769 bp\n",
      "Total written (filtered):     40,874,677 bp (91.2%)\n",
      "  Read 1:    20,447,219 bp\n",
      "  Read 2:    20,427,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 75702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t750\t0.0\t2\t0 721 29\n",
      "26\t74342\t0.0\t2\t72581 1703 58\n",
      "27\t361\t0.0\t2\t0 348 13\n",
      "28\t189\t0.0\t2\t0 0 189\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t729\t0.0\t2\t0 708 21\n",
      "26\t74538\t0.0\t2\t72551 1871 116\n",
      "27\t302\t0.0\t2\t0 288 14\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2288_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2288_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2288_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2288_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2288_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2288_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2288_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2288_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        62,208 reads  @    123.8 µs/read;   0.48 M reads/minute\n",
      "Finished in 7.71 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,208\n",
      "  Read 1 with adapter:                  62,158 (99.9%)\n",
      "  Read 2 with adapter:                  62,108 (99.8%)\n",
      "Pairs written (passing filters):        62,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,697,697 bp\n",
      "  Read 1:    18,350,196 bp\n",
      "  Read 2:    18,347,501 bp\n",
      "Total written (filtered):     33,467,231 bp (91.2%)\n",
      "  Read 1:    16,734,323 bp\n",
      "  Read 2:    16,732,908 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t627\t0.0\t2\t0 614 13\n",
      "26\t61100\t0.0\t2\t59655 1403 42\n",
      "27\t314\t0.0\t2\t0 300 14\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62108 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t545\t0.0\t2\t0 530 15\n",
      "26\t61214\t0.0\t2\t59797 1352 65\n",
      "27\t280\t0.0\t2\t0 271 9\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2300_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2300_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2300_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2300_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2300_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2300_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2300_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2300_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2301_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2301_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2301_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2301_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2301_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2301_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2301_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2301_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2303_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2303_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2303_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2303_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2303_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2303_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2303_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2303_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2304_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2304_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2304_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2304_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2304_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2304_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2304_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2304_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2311_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2311_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2311_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2311_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2311_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2311_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2311_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2311_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2313_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2313_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2313_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2313_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2313_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2313_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2313_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2313_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2317_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2317_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2317_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2317_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2317_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2317_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2317_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2317_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2319_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2319_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2319_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2319_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2319_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2319_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2319_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2319_175_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2322_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2322_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2322_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2322_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2322_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2322_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2322_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2322_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2325_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2325_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2325_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2325_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2325_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2325_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2325_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2325_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2326_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2326_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2326_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2326_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2326_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2326_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2326_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2326_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2327_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2327_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2327_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2327_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2327_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2327_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2327_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2327_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2328_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2328_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2328_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2328_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2328_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2328_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2328_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2328_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2329_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2329_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2329_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2329_187_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2329_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2329_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2329_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2329_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2333_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2333_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2333_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2333_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2333_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2333_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2333_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2333_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2335_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2335_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2335_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2335_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2335_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-gld1o7mg/S2335_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2335_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-au4is_6e/cd960d6c-06c6-4407-a726-5ed6762140fb/data/S2335_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: h-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: h-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'q2' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-4-b132b98364fa>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;31m#3.h summarize and view\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msystem\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'qiime demux summarize    --i-data h-demux-pairedend-trimmed.qza    --o-visualization h-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0mq2\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mVisualization\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'h-demux-pairedend-trimmed.qzv'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'q2' is not defined"
     ]
    }
   ],
   "source": [
    "#1.h import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/ENP-Lib-manifest_COI.csv \\\n",
    "  --output-path h-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.h trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences h-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences h-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.h summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data h-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization h-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('h-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: h-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: h-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: h-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.h Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs h-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 213 \\\n",
    "  --p-trunc-len-r 155 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table h-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats h-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: h-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-87mjdbps')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e942caf5-ab05-44fd-9952-5d8869b63df2>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.h view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table h-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization h-table.qzv \n",
    "q2.Visualization.load('h-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: h-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.h view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization h-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/GAK-manifest_COI.csv as PairedEndFastqManifestPhred33 to i-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3101_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3101_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3101_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3101_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3101_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3101_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3101_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3101_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        49,215 reads  @    168.2 µs/read;   0.36 M reads/minute\n",
      "Finished in 8.29 s (168 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,215\n",
      "  Read 1 with adapter:                  49,180 (99.9%)\n",
      "  Read 2 with adapter:                  49,143 (99.9%)\n",
      "Pairs written (passing filters):        49,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,938,420 bp\n",
      "  Read 1:    14,469,210 bp\n",
      "  Read 2:    14,469,210 bp\n",
      "Total written (filtered):     26,383,015 bp (91.2%)\n",
      "  Read 1:    13,189,656 bp\n",
      "  Read 2:    13,193,359 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t384\t0.0\t2\t0 373 11\n",
      "26\t47963\t0.0\t2\t47343 600 20\n",
      "27\t344\t0.0\t2\t0 335 9\n",
      "28\t473\t0.0\t2\t0 0 473\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49143 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t129\t0.0\t2\t0 0 129\n",
      "25\t1796\t0.0\t2\t0 1769 27\n",
      "26\t47043\t0.0\t2\t46210 747 86\n",
      "27\t163\t0.0\t2\t0 152 11\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3102_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3102_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3102_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3102_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3102_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3102_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3102_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3102_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        54,822 reads  @    131.7 µs/read;   0.46 M reads/minute\n",
      "Finished in 7.22 s (132 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,822\n",
      "  Read 1 with adapter:                  54,771 (99.9%)\n",
      "  Read 2 with adapter:                  54,725 (99.8%)\n",
      "Pairs written (passing filters):        54,822 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,180,514 bp\n",
      "  Read 1:    16,062,846 bp\n",
      "  Read 2:    16,117,668 bp\n",
      "Total written (filtered):     29,335,172 bp (91.2%)\n",
      "  Read 1:    14,638,244 bp\n",
      "  Read 2:    14,696,928 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t347\t0.0\t2\t0 341 6\n",
      "26\t53634\t0.0\t2\t52871 744 19\n",
      "27\t565\t0.0\t2\t0 557 8\n",
      "28\t197\t0.0\t2\t0 0 197\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t172\t0.0\t2\t0 0 172\n",
      "25\t2047\t0.0\t2\t0 2023 24\n",
      "26\t52230\t0.0\t2\t51362 784 84\n",
      "27\t271\t0.0\t2\t0 249 22\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3103_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3103_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3103_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3103_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3103_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3103_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3103_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3103_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        51,201 reads  @    145.7 µs/read;   0.41 M reads/minute\n",
      "Finished in 7.46 s (146 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,201\n",
      "  Read 1 with adapter:                  51,163 (99.9%)\n",
      "  Read 2 with adapter:                  51,130 (99.9%)\n",
      "Pairs written (passing filters):        51,201 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,208,590 bp\n",
      "  Read 1:    15,155,496 bp\n",
      "  Read 2:    15,053,094 bp\n",
      "Total written (filtered):     27,547,446 bp (91.2%)\n",
      "  Read 1:    13,823,595 bp\n",
      "  Read 2:    13,723,851 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51163 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t870\t0.0\t2\t0 858 12\n",
      "26\t48602\t0.0\t2\t48104 474 24\n",
      "27\t709\t0.0\t2\t0 690 19\n",
      "28\t947\t0.0\t2\t0 0 947\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51130 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t368\t0.0\t2\t0 353 15\n",
      "26\t50498\t0.0\t2\t49654 754 90\n",
      "27\t229\t0.0\t2\t0 215 14\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3104_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3104_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3104_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3104_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3104_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3104_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3104_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3104_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        45,198 reads  @    121.7 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.50 s (122 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,198\n",
      "  Read 1 with adapter:                  45,154 (99.9%)\n",
      "  Read 2 with adapter:                  45,131 (99.9%)\n",
      "Pairs written (passing filters):        45,198 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,621,622 bp\n",
      "  Read 1:    13,333,410 bp\n",
      "  Read 2:    13,288,212 bp\n",
      "Total written (filtered):     24,275,778 bp (91.2%)\n",
      "  Read 1:    12,159,043 bp\n",
      "  Read 2:    12,116,735 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t338\t0.0\t2\t0 335 3\n",
      "26\t44222\t0.0\t2\t43626 587 9\n",
      "27\t403\t0.0\t2\t0 398 5\n",
      "28\t170\t0.0\t2\t0 0 170\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t145\t0.0\t2\t0 0 145\n",
      "25\t1889\t0.0\t2\t0 1862 27\n",
      "26\t42876\t0.0\t2\t42197 610 69\n",
      "27\t192\t0.0\t2\t0 184 8\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3105_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3105_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3105_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3105_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3105_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3105_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3105_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3105_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        37,198 reads  @    144.9 µs/read;   0.41 M reads/minute\n",
      "Finished in 5.39 s (145 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,198\n",
      "  Read 1 with adapter:                  37,148 (99.9%)\n",
      "  Read 2 with adapter:                  37,131 (99.8%)\n",
      "Pairs written (passing filters):        37,198 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,872,424 bp\n",
      "  Read 1:    10,936,212 bp\n",
      "  Read 2:    10,936,212 bp\n",
      "Total written (filtered):     19,942,734 bp (91.2%)\n",
      "  Read 1:     9,970,170 bp\n",
      "  Read 2:     9,972,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37148 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t282\t0.0\t2\t0 273 9\n",
      "26\t36364\t0.0\t2\t34494 1847 23\n",
      "27\t424\t0.0\t2\t0 415 9\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37131 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t133\t0.0\t2\t0 0 133\n",
      "25\t1695\t0.0\t2\t0 1667 28\n",
      "26\t35110\t0.0\t2\t34508 538 64\n",
      "27\t183\t0.0\t2\t0 177 6\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3106_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3106_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3106_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3106_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3106_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3106_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3106_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3106_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        53,599 reads  @    170.1 µs/read;   0.35 M reads/minute\n",
      "Finished in 9.12 s (170 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,599\n",
      "  Read 1 with adapter:                  53,564 (99.9%)\n",
      "  Read 2 with adapter:                  53,506 (99.8%)\n",
      "Pairs written (passing filters):        53,599 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,462,613 bp\n",
      "  Read 1:    15,704,507 bp\n",
      "  Read 2:    15,758,106 bp\n",
      "Total written (filtered):     28,679,746 bp (91.2%)\n",
      "  Read 1:    14,310,592 bp\n",
      "  Read 2:    14,369,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t471\t0.0\t2\t0 455 16\n",
      "26\t51932\t0.0\t2\t51236 668 28\n",
      "27\t476\t0.0\t2\t0 472 4\n",
      "28\t654\t0.0\t2\t0 0 654\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t172\t0.0\t2\t0 0 172\n",
      "25\t2101\t0.0\t2\t0 2059 42\n",
      "26\t51003\t0.0\t2\t50005 909 89\n",
      "27\t219\t0.0\t2\t0 208 11\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3107_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3107_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3107_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3107_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3107_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3107_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3107_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3107_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        54,020 reads  @    128.2 µs/read;   0.47 M reads/minute\n",
      "Finished in 6.93 s (128 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,020\n",
      "  Read 1 with adapter:                  53,967 (99.9%)\n",
      "  Read 2 with adapter:                  53,948 (99.9%)\n",
      "Pairs written (passing filters):        54,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,817,780 bp\n",
      "  Read 1:    15,827,860 bp\n",
      "  Read 2:    15,989,920 bp\n",
      "Total written (filtered):     29,011,017 bp (91.2%)\n",
      "  Read 1:    14,423,090 bp\n",
      "  Read 2:    14,587,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t349\t0.0\t2\t0 340 9\n",
      "26\t52154\t0.0\t2\t51496 634 24\n",
      "27\t859\t0.0\t2\t0 841 18\n",
      "28\t582\t0.0\t2\t0 0 582\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t958\t0.0\t2\t0 943 15\n",
      "26\t52580\t0.0\t2\t51798 705 77\n",
      "27\t353\t0.0\t2\t0 337 16\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3108_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3108_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3108_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3108_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3108_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3108_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3108_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3108_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        56,359 reads  @    110.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 6.20 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,359\n",
      "  Read 1 with adapter:                  56,304 (99.9%)\n",
      "  Read 2 with adapter:                  56,267 (99.8%)\n",
      "Pairs written (passing filters):        56,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,195,451 bp\n",
      "  Read 1:    16,569,546 bp\n",
      "  Read 2:    16,625,905 bp\n",
      "Total written (filtered):     30,268,220 bp (91.2%)\n",
      "  Read 1:    15,104,637 bp\n",
      "  Read 2:    15,163,583 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t564\t0.0\t2\t0 555 9\n",
      "26\t54588\t0.0\t2\t53898 671 19\n",
      "27\t607\t0.0\t2\t0 593 14\n",
      "28\t513\t0.0\t2\t0 0 513\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56267 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t887\t0.0\t2\t0 867 20\n",
      "26\t55032\t0.0\t2\t54186 752 94\n",
      "27\t297\t0.0\t2\t0 272 25\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3109_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3109_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3109_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3109_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3109_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3109_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3109_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3109_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        35,087 reads  @    146.5 µs/read;   0.41 M reads/minute\n",
      "Finished in 5.14 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,087\n",
      "  Read 1 with adapter:                  35,064 (99.9%)\n",
      "  Read 2 with adapter:                  34,967 (99.7%)\n",
      "Pairs written (passing filters):        35,087 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,631,156 bp\n",
      "  Read 1:    10,280,491 bp\n",
      "  Read 2:    10,350,665 bp\n",
      "Total written (filtered):     18,810,654 bp (91.2%)\n",
      "  Read 1:     9,368,514 bp\n",
      "  Read 2:     9,442,140 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t281\t0.0\t2\t0 273 8\n",
      "26\t34221\t0.0\t2\t33789 415 17\n",
      "27\t482\t0.0\t2\t0 477 5\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t830\t0.0\t2\t0 815 15\n",
      "26\t33794\t0.0\t2\t33232 494 68\n",
      "27\t245\t0.0\t2\t0 223 22\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3110_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3110_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3110_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3110_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3110_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3110_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3110_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3110_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        35,403 reads  @    168.8 µs/read;   0.36 M reads/minute\n",
      "Finished in 5.98 s (169 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,403\n",
      "  Read 1 with adapter:                  35,360 (99.9%)\n",
      "  Read 2 with adapter:                  35,320 (99.8%)\n",
      "Pairs written (passing filters):        35,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,923,173 bp\n",
      "  Read 1:    10,479,288 bp\n",
      "  Read 2:    10,443,885 bp\n",
      "Total written (filtered):     19,083,702 bp (91.2%)\n",
      "  Read 1:     9,557,693 bp\n",
      "  Read 2:     9,526,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35360 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t516\t0.0\t2\t0 506 10\n",
      "26\t33032\t0.0\t2\t32639 367 26\n",
      "27\t725\t0.0\t2\t0 706 19\n",
      "28\t1050\t0.0\t2\t0 0 1050\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t795\t0.0\t2\t0 774 21\n",
      "26\t34114\t0.0\t2\t33498 554 62\n",
      "27\t351\t0.0\t2\t0 288 63\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3111_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3111_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3111_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3111_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3111_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3111_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3111_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3111_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        64,143 reads  @    137.6 µs/read;   0.44 M reads/minute\n",
      "Finished in 8.83 s (138 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,143\n",
      "  Read 1 with adapter:                  64,080 (99.9%)\n",
      "  Read 2 with adapter:                  64,067 (99.9%)\n",
      "Pairs written (passing filters):        64,143 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,844,370 bp\n",
      "  Read 1:    18,922,185 bp\n",
      "  Read 2:    18,922,185 bp\n",
      "Total written (filtered):     34,512,161 bp (91.2%)\n",
      "  Read 1:    17,255,192 bp\n",
      "  Read 2:    17,256,969 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t521\t0.0\t2\t0 515 6\n",
      "26\t62207\t0.0\t2\t61337 843 27\n",
      "27\t1162\t0.0\t2\t0 1141 21\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t990\t0.0\t2\t0 954 36\n",
      "26\t62565\t0.0\t2\t61485 976 104\n",
      "27\t424\t0.0\t2\t0 391 33\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3112_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3112_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3112_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3112_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3112_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3112_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3112_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3112_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        36,497 reads  @    156.5 µs/read;   0.38 M reads/minute\n",
      "Finished in 5.72 s (157 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,497\n",
      "  Read 1 with adapter:                  36,455 (99.9%)\n",
      "  Read 2 with adapter:                  36,406 (99.8%)\n",
      "Pairs written (passing filters):        36,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,496,733 bp\n",
      "  Read 1:    10,730,118 bp\n",
      "  Read 2:    10,766,615 bp\n",
      "Total written (filtered):     19,602,091 bp (91.2%)\n",
      "  Read 1:     9,781,918 bp\n",
      "  Read 2:     9,820,173 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36455 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t1127\t0.0\t2\t0 1104 23\n",
      "26\t34216\t0.0\t2\t33793 401 22\n",
      "27\t383\t0.0\t2\t0 377 6\n",
      "28\t643\t0.0\t2\t0 0 643\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t590\t0.0\t2\t0 574 16\n",
      "26\t35335\t0.0\t2\t34718 534 83\n",
      "27\t298\t0.0\t2\t0 276 22\n",
      "28\t136\t0.0\t2\t0 0 136\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3113_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3113_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3113_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3113_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3113_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3113_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3113_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3113_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        59,468 reads  @    117.8 µs/read;   0.51 M reads/minute\n",
      "Finished in 7.01 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,468\n",
      "  Read 1 with adapter:                  59,411 (99.9%)\n",
      "  Read 2 with adapter:                  59,384 (99.9%)\n",
      "Pairs written (passing filters):        59,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,967,184 bp\n",
      "  Read 1:    17,424,124 bp\n",
      "  Read 2:    17,543,060 bp\n",
      "Total written (filtered):     31,877,550 bp (91.2%)\n",
      "  Read 1:    15,877,913 bp\n",
      "  Read 2:    15,999,637 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t690\t0.0\t2\t0 674 16\n",
      "26\t56958\t0.0\t2\t56251 676 31\n",
      "27\t1167\t0.0\t2\t0 1144 23\n",
      "28\t560\t0.0\t2\t0 0 560\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59384 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t930\t0.0\t2\t0 910 20\n",
      "26\t58023\t0.0\t2\t56959 958 106\n",
      "27\t341\t0.0\t2\t0 310 31\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3114_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3114_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3114_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3114_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3114_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3114_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3114_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3114_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        34,785 reads  @    115.6 µs/read;   0.52 M reads/minute\n",
      "Finished in 4.03 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,785\n",
      "  Read 1 with adapter:                  34,751 (99.9%)\n",
      "  Read 2 with adapter:                  34,713 (99.8%)\n",
      "Pairs written (passing filters):        34,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,523,150 bp\n",
      "  Read 1:    10,226,790 bp\n",
      "  Read 2:    10,296,360 bp\n",
      "Total written (filtered):     18,716,160 bp (91.2%)\n",
      "  Read 1:     9,322,055 bp\n",
      "  Read 2:     9,394,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34751 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t577\t0.0\t2\t0 562 15\n",
      "26\t33011\t0.0\t2\t32585 407 19\n",
      "27\t444\t0.0\t2\t0 433 11\n",
      "28\t695\t0.0\t2\t0 0 695\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34713 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t520\t0.0\t2\t0 511 9\n",
      "26\t33956\t0.0\t2\t33483 424 49\n",
      "27\t177\t0.0\t2\t0 162 15\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3115_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3115_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3115_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3115_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3115_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3115_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3115_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3115_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        33,645 reads  @    143.3 µs/read;   0.42 M reads/minute\n",
      "Finished in 4.83 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,645\n",
      "  Read 1 with adapter:                  33,619 (99.9%)\n",
      "  Read 2 with adapter:                  33,608 (99.9%)\n",
      "Pairs written (passing filters):        33,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,816,905 bp\n",
      "  Read 1:     9,891,630 bp\n",
      "  Read 2:     9,925,275 bp\n",
      "Total written (filtered):     18,068,848 bp (91.2%)\n",
      "  Read 1:     9,016,953 bp\n",
      "  Read 2:     9,051,895 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33619 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t315\t0.0\t2\t0 307 8\n",
      "26\t32652\t0.0\t2\t32270 366 16\n",
      "27\t342\t0.0\t2\t0 342\n",
      "28\t294\t0.0\t2\t0 0 294\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t578\t0.0\t2\t0 571 7\n",
      "26\t32819\t0.0\t2\t32353 416 50\n",
      "27\t172\t0.0\t2\t0 158 14\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3116_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3116_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3116_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3116_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3116_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3116_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3116_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3116_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        15,223 reads  @    148.3 µs/read;   0.40 M reads/minute\n",
      "Finished in 2.26 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,223\n",
      "  Read 1 with adapter:                  15,209 (99.9%)\n",
      "  Read 2 with adapter:                  15,166 (99.6%)\n",
      "Pairs written (passing filters):        15,223 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,951,124 bp\n",
      "  Read 1:     4,460,339 bp\n",
      "  Read 2:     4,490,785 bp\n",
      "Total written (filtered):      8,161,456 bp (91.2%)\n",
      "  Read 1:     4,064,694 bp\n",
      "  Read 2:     4,096,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t136\t0.0\t2\t0 134 2\n",
      "26\t14745\t0.0\t2\t14529 206 10\n",
      "27\t277\t0.0\t2\t0 272 5\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15166 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t408\t0.0\t2\t0 396 12\n",
      "26\t14607\t0.0\t2\t14312 260 35\n",
      "27\t115\t0.0\t2\t0 100 15\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3117_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3117_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3117_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3117_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3117_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3117_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3117_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3117_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        15,751 reads  @    163.8 µs/read;   0.37 M reads/minute\n",
      "Finished in 2.59 s (164 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             15,751\n",
      "  Read 1 with adapter:                  15,738 (99.9%)\n",
      "  Read 2 with adapter:                  15,722 (99.8%)\n",
      "Pairs written (passing filters):        15,751 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,308,841 bp\n",
      "  Read 1:     4,662,296 bp\n",
      "  Read 2:     4,646,545 bp\n",
      "Total written (filtered):      8,489,951 bp (91.2%)\n",
      "  Read 1:     4,251,852 bp\n",
      "  Read 2:     4,238,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 15738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t177\t0.0\t2\t0 172 5\n",
      "26\t14675\t0.0\t2\t14505 164 6\n",
      "27\t295\t0.0\t2\t0 288 7\n",
      "28\t580\t0.0\t2\t0 0 580\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 15722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t369\t0.0\t2\t0 363 6\n",
      "26\t15208\t0.0\t2\t14881 253 74\n",
      "27\t95\t0.0\t2\t0 82 13\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3118_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3118_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3118_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3118_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3118_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3118_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3118_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3118_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        21,258 reads  @    161.0 µs/read;   0.37 M reads/minute\n",
      "Finished in 3.43 s (161 us/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,258\n",
      "  Read 1 with adapter:                  21,230 (99.9%)\n",
      "  Read 2 with adapter:                  21,237 (99.9%)\n",
      "Pairs written (passing filters):        21,258 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,542,220 bp\n",
      "  Read 1:     6,271,110 bp\n",
      "  Read 2:     6,271,110 bp\n",
      "Total written (filtered):     11,437,956 bp (91.2%)\n",
      "  Read 1:     5,718,863 bp\n",
      "  Read 2:     5,719,093 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 21230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t178\t0.0\t2\t0 175 3\n",
      "26\t20638\t0.0\t2\t20372 257 9\n",
      "27\t359\t0.0\t2\t0 353 6\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 21237 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t305\t0.0\t2\t0 301 4\n",
      "26\t20741\t0.0\t2\t20397 308 36\n",
      "27\t142\t0.0\t2\t0 133 9\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3119_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3119_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3119_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3119_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3119_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3119_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3119_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3119_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        34,070 reads  @    139.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 4.77 s (140 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,070\n",
      "  Read 1 with adapter:                  34,034 (99.9%)\n",
      "  Read 2 with adapter:                  34,031 (99.9%)\n",
      "Pairs written (passing filters):        34,070 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,067,230 bp\n",
      "  Read 1:    10,016,580 bp\n",
      "  Read 2:    10,050,650 bp\n",
      "Total written (filtered):     18,297,435 bp (91.2%)\n",
      "  Read 1:     9,131,257 bp\n",
      "  Read 2:     9,166,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t121\t0.0\t2\t0 0 121\n",
      "25\t1240\t0.0\t2\t0 1220 20\n",
      "26\t31532\t0.0\t2\t31068 437 27\n",
      "27\t361\t0.0\t2\t0 349 12\n",
      "28\t780\t0.0\t2\t0 0 780\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t533\t0.0\t2\t0 521 12\n",
      "26\t33251\t0.0\t2\t32710 469 72\n",
      "27\t219\t0.0\t2\t0 196 23\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3120_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3120_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3120_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3120_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3120_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3120_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3120_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3120_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        22,148 reads  @    152.7 µs/read;   0.39 M reads/minute\n",
      "Finished in 3.39 s (153 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,148\n",
      "  Read 1 with adapter:                  22,126 (99.9%)\n",
      "  Read 2 with adapter:                  22,118 (99.9%)\n",
      "Pairs written (passing filters):        22,148 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,023,024 bp\n",
      "  Read 1:     6,489,364 bp\n",
      "  Read 2:     6,533,660 bp\n",
      "Total written (filtered):     11,872,524 bp (91.2%)\n",
      "  Read 1:     5,913,718 bp\n",
      "  Read 2:     5,958,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 22126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t235\t0.0\t2\t0 232 3\n",
      "26\t21370\t0.0\t2\t21122 241 7\n",
      "27\t389\t0.0\t2\t0 380 9\n",
      "28\t120\t0.0\t2\t0 0 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 22118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t344\t0.0\t2\t0 332 12\n",
      "26\t21616\t0.0\t2\t21246 337 33\n",
      "27\t130\t0.0\t2\t0 117 13\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3121_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3121_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3121_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3121_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3121_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4pk1xprb/S3121_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3121_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7gfwlll/deca10c0-b879-40d4-ba95-f11a1c9b55a5/data/S3121_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        19,664 reads  @    112.1 µs/read;   0.54 M reads/minute\n",
      "Finished in 2.21 s (112 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,664\n",
      "  Read 1 with adapter:                  19,644 (99.9%)\n",
      "  Read 2 with adapter:                  19,643 (99.9%)\n",
      "Pairs written (passing filters):        19,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,601,760 bp\n",
      "  Read 1:     5,781,216 bp\n",
      "  Read 2:     5,820,544 bp\n",
      "Total written (filtered):     10,579,768 bp (91.2%)\n",
      "  Read 1:     5,269,669 bp\n",
      "  Read 2:     5,310,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 19644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t307\t0.0\t2\t0 299 8\n",
      "26\t18630\t0.0\t2\t18403 221 6\n",
      "27\t228\t0.0\t2\t0 225 3\n",
      "28\t460\t0.0\t2\t0 0 460\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 19643 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t331\t0.0\t2\t0 324 7\n",
      "26\t19228\t0.0\t2\t18960 238 30\n",
      "27\t70\t0.0\t2\t0 66 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: i-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: i-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-q1fvgcd6')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: cca02e94-e7e5-442f-ab99-76da706341fd>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.i import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/GAK-manifest_COI.csv \\\n",
    "  --output-path i-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.i trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences i-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences i-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.i summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data i-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization i-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('i-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: i-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: i-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: i-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.i Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs i-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 268 \\\n",
    "  --p-trunc-len-r 238 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table i-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats i-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: i-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-w3hchu73')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 40b6d161-c78d-4288-8a67-9ee6fbcaa76a>"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.i view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table i-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization i-table.qzv \n",
    "q2.Visualization.load('i-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: i-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.i view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization i-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/DS5B-Lib-manifest_COI.csv as PairedEndFastqManifestPhred33 to j-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3001_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3001_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:14        81,381 reads  @    180.5 µs/read;   0.33 M reads/minute\n",
      "Finished in 14.69 s (181 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,381\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        81,381 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,598,194 bp\n",
      "  Read 1:    22,269,990 bp\n",
      "  Read 2:    22,328,204 bp\n",
      "Total written (filtered):     44,598,194 bp (100.0%)\n",
      "  Read 1:    22,269,990 bp\n",
      "  Read 2:    22,328,204 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3002_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3002_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:15        86,175 reads  @    185.3 µs/read;   0.32 M reads/minute\n",
      "Finished in 15.98 s (185 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,175\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        86,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,202,675 bp\n",
      "  Read 1:    23,563,344 bp\n",
      "  Read 2:    23,639,331 bp\n",
      "Total written (filtered):     47,202,675 bp (100.0%)\n",
      "  Read 1:    23,563,344 bp\n",
      "  Read 2:    23,639,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3003_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3003_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:15        82,286 reads  @    183.8 µs/read;   0.33 M reads/minute\n",
      "Finished in 15.13 s (184 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,286\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        82,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,093,300 bp\n",
      "  Read 1:    22,520,645 bp\n",
      "  Read 2:    22,572,655 bp\n",
      "Total written (filtered):     45,093,300 bp (100.0%)\n",
      "  Read 1:    22,520,645 bp\n",
      "  Read 2:    22,572,655 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3004_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3004_7_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:12        78,793 reads  @    155.4 µs/read;   0.39 M reads/minute\n",
      "Finished in 12.26 s (156 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,793\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        78,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,301,314 bp\n",
      "  Read 1:    21,644,591 bp\n",
      "  Read 2:    21,656,723 bp\n",
      "Total written (filtered):     43,301,314 bp (100.0%)\n",
      "  Read 1:    21,644,591 bp\n",
      "  Read 2:    21,656,723 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3005_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3005_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:12        63,656 reads  @    195.9 µs/read;   0.31 M reads/minute\n",
      "Finished in 12.48 s (196 us/read; 0.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,656\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        63,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,842,391 bp\n",
      "  Read 1:    16,903,098 bp\n",
      "  Read 2:    16,939,293 bp\n",
      "Total written (filtered):     33,842,391 bp (100.0%)\n",
      "  Read 1:    16,903,098 bp\n",
      "  Read 2:    16,939,293 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3006_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3006_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:13        65,626 reads  @    212.1 µs/read;   0.28 M reads/minute\n",
      "Finished in 13.93 s (212 us/read; 0.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,626\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        65,626 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,608,516 bp\n",
      "  Read 1:    17,772,235 bp\n",
      "  Read 2:    17,836,281 bp\n",
      "Total written (filtered):     35,608,516 bp (100.0%)\n",
      "  Read 1:    17,772,235 bp\n",
      "  Read 2:    17,836,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3007_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3007_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:14        93,563 reads  @    150.0 µs/read;   0.40 M reads/minute\n",
      "Finished in 14.03 s (150 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,563\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        93,563 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,186,461 bp\n",
      "  Read 1:    25,526,288 bp\n",
      "  Read 2:    25,660,173 bp\n",
      "Total written (filtered):     51,186,461 bp (100.0%)\n",
      "  Read 1:    25,526,288 bp\n",
      "  Read 2:    25,660,173 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3008_15_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3008_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:10        83,224 reads  @    131.5 µs/read;   0.46 M reads/minute\n",
      "Finished in 10.95 s (132 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,224\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        83,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,559,111 bp\n",
      "  Read 1:    22,732,597 bp\n",
      "  Read 2:    22,826,514 bp\n",
      "Total written (filtered):     45,559,111 bp (100.0%)\n",
      "  Read 1:    22,732,597 bp\n",
      "  Read 2:    22,826,514 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3009_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3009_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:11       110,321 reads  @    108.0 µs/read;   0.56 M reads/minute\n",
      "Finished in 11.91 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,321\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):       110,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    60,079,594 bp\n",
      "  Read 1:    29,945,973 bp\n",
      "  Read 2:    30,133,621 bp\n",
      "Total written (filtered):     60,079,594 bp (100.0%)\n",
      "  Read 1:    29,945,973 bp\n",
      "  Read 2:    30,133,621 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3010_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3010_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[       8=---] 00:00:17       151,584 reads  @    118.1 µs/read;   0.51 M reads/minute\n",
      "Finished in 17.91 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            151,584\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):       151,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,028,204 bp\n",
      "  Read 1:    30,997,012 bp\n",
      "  Read 2:    32,031,192 bp\n",
      "Total written (filtered):     63,028,204 bp (100.0%)\n",
      "  Read 1:    30,997,012 bp\n",
      "  Read 2:    32,031,192 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3011_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3011_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        71,580 reads  @    135.1 µs/read;   0.44 M reads/minute\n",
      "Finished in 9.67 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,580\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        71,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,953,420 bp\n",
      "  Read 1:    19,453,213 bp\n",
      "  Read 2:    19,500,207 bp\n",
      "Total written (filtered):     38,953,420 bp (100.0%)\n",
      "  Read 1:    19,453,213 bp\n",
      "  Read 2:    19,500,207 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3012_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3012_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11        81,650 reads  @    138.5 µs/read;   0.43 M reads/minute\n",
      "Finished in 11.31 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,650\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        81,650 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,101,099 bp\n",
      "  Read 1:    22,017,346 bp\n",
      "  Read 2:    22,083,753 bp\n",
      "Total written (filtered):     44,101,099 bp (100.0%)\n",
      "  Read 1:    22,017,346 bp\n",
      "  Read 2:    22,083,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3013_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3013_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        76,834 reads  @    124.1 µs/read;   0.48 M reads/minute\n",
      "Finished in 9.54 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,834\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        76,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,078,725 bp\n",
      "  Read 1:    20,989,771 bp\n",
      "  Read 2:    21,088,954 bp\n",
      "Total written (filtered):     42,078,725 bp (100.0%)\n",
      "  Read 1:    20,989,771 bp\n",
      "  Read 2:    21,088,954 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3014_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3014_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:09        82,526 reads  @    109.3 µs/read;   0.55 M reads/minute\n",
      "Finished in 9.03 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,526\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        82,526 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,113,059 bp\n",
      "  Read 1:    22,485,929 bp\n",
      "  Read 2:    22,627,130 bp\n",
      "Total written (filtered):     45,113,059 bp (100.0%)\n",
      "  Read 1:    22,485,929 bp\n",
      "  Read 2:    22,627,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3015_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3015_29_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:13        77,837 reads  @    175.7 µs/read;   0.34 M reads/minute\n",
      "Finished in 13.69 s (176 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,837\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        77,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,520,732 bp\n",
      "  Read 1:    21,210,274 bp\n",
      "  Read 2:    21,310,458 bp\n",
      "Total written (filtered):     42,520,732 bp (100.0%)\n",
      "  Read 1:    21,210,274 bp\n",
      "  Read 2:    21,310,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3016_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3016_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:13        91,683 reads  @    151.7 µs/read;   0.40 M reads/minute\n",
      "Finished in 13.92 s (152 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,683\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        91,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,118,893 bp\n",
      "  Read 1:    25,003,652 bp\n",
      "  Read 2:    25,115,241 bp\n",
      "Total written (filtered):     50,118,893 bp (100.0%)\n",
      "  Read 1:    25,003,652 bp\n",
      "  Read 2:    25,115,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3017_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3017_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        75,147 reads  @    147.2 µs/read;   0.41 M reads/minute\n",
      "Finished in 11.07 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,147\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        75,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,091,003 bp\n",
      "  Read 1:    20,517,607 bp\n",
      "  Read 2:    20,573,396 bp\n",
      "Total written (filtered):     41,091,003 bp (100.0%)\n",
      "  Read 1:    20,517,607 bp\n",
      "  Read 2:    20,573,396 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3018_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3018_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11        80,333 reads  @    141.0 µs/read;   0.43 M reads/minute\n",
      "Finished in 11.33 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,333\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        80,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,967,173 bp\n",
      "  Read 1:    21,966,647 bp\n",
      "  Read 2:    22,000,526 bp\n",
      "Total written (filtered):     43,967,173 bp (100.0%)\n",
      "  Read 1:    21,966,647 bp\n",
      "  Read 2:    22,000,526 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3019_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3019_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        77,063 reads  @    147.3 µs/read;   0.41 M reads/minute\n",
      "Finished in 11.36 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,063\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        77,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,219,797 bp\n",
      "  Read 1:    21,093,542 bp\n",
      "  Read 2:    21,126,255 bp\n",
      "Total written (filtered):     42,219,797 bp (100.0%)\n",
      "  Read 1:    21,093,542 bp\n",
      "  Read 2:    21,126,255 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3020_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3020_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        25,137 reads  @    115.3 µs/read;   0.52 M reads/minute\n",
      "Finished in 2.90 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,137\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        25,137 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,262,193 bp\n",
      "  Read 1:     6,623,708 bp\n",
      "  Read 2:     6,638,485 bp\n",
      "Total written (filtered):     13,262,193 bp (100.0%)\n",
      "  Read 1:     6,623,708 bp\n",
      "  Read 2:     6,638,485 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3021_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3021_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        77,488 reads  @    144.2 µs/read;   0.42 M reads/minute\n",
      "Finished in 11.18 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,488\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        77,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,372,170 bp\n",
      "  Read 1:    21,176,610 bp\n",
      "  Read 2:    21,195,560 bp\n",
      "Total written (filtered):     42,372,170 bp (100.0%)\n",
      "  Read 1:    21,176,610 bp\n",
      "  Read 2:    21,195,560 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3022_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3022_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:10        92,849 reads  @    113.2 µs/read;   0.53 M reads/minute\n",
      "Finished in 10.51 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,849\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        92,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,421,752 bp\n",
      "  Read 1:    25,123,937 bp\n",
      "  Read 2:    25,297,815 bp\n",
      "Total written (filtered):     50,421,752 bp (100.0%)\n",
      "  Read 1:    25,123,937 bp\n",
      "  Read 2:    25,297,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3023_45_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3023_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        64,118 reads  @    121.2 µs/read;   0.49 M reads/minute\n",
      "Finished in 7.78 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,118\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        64,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,623,937 bp\n",
      "  Read 1:    16,812,111 bp\n",
      "  Read 2:    16,811,826 bp\n",
      "Total written (filtered):     33,623,937 bp (100.0%)\n",
      "  Read 1:    16,812,111 bp\n",
      "  Read 2:    16,811,826 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3024_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3024_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        66,531 reads  @    131.7 µs/read;   0.46 M reads/minute\n",
      "Finished in 8.77 s (132 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,531\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        66,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,409,180 bp\n",
      "  Read 1:    18,188,736 bp\n",
      "  Read 2:    18,220,444 bp\n",
      "Total written (filtered):     36,409,180 bp (100.0%)\n",
      "  Read 1:    18,188,736 bp\n",
      "  Read 2:    18,220,444 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3025_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3025_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11        83,400 reads  @    139.4 µs/read;   0.43 M reads/minute\n",
      "Finished in 11.63 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             83,400\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        83,400 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,733,774 bp\n",
      "  Read 1:    22,849,427 bp\n",
      "  Read 2:    22,884,347 bp\n",
      "Total written (filtered):     45,733,774 bp (100.0%)\n",
      "  Read 1:    22,849,427 bp\n",
      "  Read 2:    22,884,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3026_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3026_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        53,072 reads  @    143.8 µs/read;   0.42 M reads/minute\n",
      "Finished in 7.64 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,072\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        53,072 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,027,043 bp\n",
      "  Read 1:    14,509,928 bp\n",
      "  Read 2:    14,517,115 bp\n",
      "Total written (filtered):     29,027,043 bp (100.0%)\n",
      "  Read 1:    14,509,928 bp\n",
      "  Read 2:    14,517,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3027_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3027_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:09        73,793 reads  @    125.6 µs/read;   0.48 M reads/minute\n",
      "Finished in 9.27 s (126 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,793\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        73,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,392,344 bp\n",
      "  Read 1:    20,147,459 bp\n",
      "  Read 2:    20,244,885 bp\n",
      "Total written (filtered):     40,392,344 bp (100.0%)\n",
      "  Read 1:    20,147,459 bp\n",
      "  Read 2:    20,244,885 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3028_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3028_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:13        94,868 reads  @    140.5 µs/read;   0.43 M reads/minute\n",
      "Finished in 13.33 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,868\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        94,868 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,611,875 bp\n",
      "  Read 1:    25,775,293 bp\n",
      "  Read 2:    25,836,582 bp\n",
      "Total written (filtered):     51,611,875 bp (100.0%)\n",
      "  Read 1:    25,775,293 bp\n",
      "  Read 2:    25,836,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3029_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3029_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:09        80,135 reads  @    119.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 9.61 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,135\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        80,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,928,974 bp\n",
      "  Read 1:    21,913,450 bp\n",
      "  Read 2:    22,015,524 bp\n",
      "Total written (filtered):     43,928,974 bp (100.0%)\n",
      "  Read 1:    21,913,450 bp\n",
      "  Read 2:    22,015,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3030_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3030_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[    8<------] 00:00:13        86,252 reads  @    156.6 µs/read;   0.38 M reads/minute\n",
      "Finished in 13.51 s (157 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,252\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        86,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,809,939 bp\n",
      "  Read 1:    23,386,069 bp\n",
      "  Read 2:    23,423,870 bp\n",
      "Total written (filtered):     46,809,939 bp (100.0%)\n",
      "  Read 1:    23,386,069 bp\n",
      "  Read 2:    23,423,870 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3031_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3031_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:10        81,122 reads  @    133.3 µs/read;   0.45 M reads/minute\n",
      "Finished in 10.81 s (133 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,122\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        81,122 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,341,132 bp\n",
      "  Read 1:    22,126,379 bp\n",
      "  Read 2:    22,214,753 bp\n",
      "Total written (filtered):     44,341,132 bp (100.0%)\n",
      "  Read 1:    22,126,379 bp\n",
      "  Read 2:    22,214,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3032_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3032_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        74,664 reads  @    151.4 µs/read;   0.40 M reads/minute\n",
      "Finished in 11.31 s (151 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,664\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        74,664 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,896,673 bp\n",
      "  Read 1:    20,421,204 bp\n",
      "  Read 2:    20,475,469 bp\n",
      "Total written (filtered):     40,896,673 bp (100.0%)\n",
      "  Read 1:    20,421,204 bp\n",
      "  Read 2:    20,475,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3033_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3033_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:10        80,340 reads  @    127.1 µs/read;   0.47 M reads/minute\n",
      "Finished in 10.21 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,340\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        80,340 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,909,345 bp\n",
      "  Read 1:    21,907,682 bp\n",
      "  Read 2:    22,001,663 bp\n",
      "Total written (filtered):     43,909,345 bp (100.0%)\n",
      "  Read 1:    21,907,682 bp\n",
      "  Read 2:    22,001,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3034_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3034_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09        68,742 reads  @    139.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 9.62 s (140 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,742\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        68,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,605,425 bp\n",
      "  Read 1:    18,781,498 bp\n",
      "  Read 2:    18,823,927 bp\n",
      "Total written (filtered):     37,605,425 bp (100.0%)\n",
      "  Read 1:    18,781,498 bp\n",
      "  Read 2:    18,823,927 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3035_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3035_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:14        93,788 reads  @    154.5 µs/read;   0.39 M reads/minute\n",
      "Finished in 14.49 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,788\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        93,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,247,337 bp\n",
      "  Read 1:    25,611,273 bp\n",
      "  Read 2:    25,636,064 bp\n",
      "Total written (filtered):     51,247,337 bp (100.0%)\n",
      "  Read 1:    25,611,273 bp\n",
      "  Read 2:    25,636,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3036_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3036_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        52,390 reads  @    148.3 µs/read;   0.40 M reads/minute\n",
      "Finished in 7.77 s (148 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,390\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        52,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,705,486 bp\n",
      "  Read 1:    14,343,899 bp\n",
      "  Read 2:    14,361,587 bp\n",
      "Total written (filtered):     28,705,486 bp (100.0%)\n",
      "  Read 1:    14,343,899 bp\n",
      "  Read 2:    14,361,587 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3037_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3037_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         7,606 reads  @    110.1 µs/read;   0.55 M reads/minute\n",
      "Finished in 0.84 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              7,606\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         7,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,748,540 bp\n",
      "  Read 1:     1,863,914 bp\n",
      "  Read 2:     1,884,626 bp\n",
      "Total written (filtered):      3,748,540 bp (100.0%)\n",
      "  Read 1:     1,863,914 bp\n",
      "  Read 2:     1,884,626 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3038_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3038_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,089 reads  @     85.9 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.50 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,089\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        29,089 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,783,532 bp\n",
      "  Read 1:     4,867,381 bp\n",
      "  Read 2:     4,916,151 bp\n",
      "Total written (filtered):      9,783,532 bp (100.0%)\n",
      "  Read 1:     4,867,381 bp\n",
      "  Read 2:     4,916,151 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3039_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3039_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,882 reads  @    123.1 µs/read;   0.49 M reads/minute\n",
      "Finished in 0.24 s (125 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,882\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):         1,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:       912,075 bp\n",
      "  Read 1:       454,325 bp\n",
      "  Read 2:       457,750 bp\n",
      "Total written (filtered):        912,075 bp (100.0%)\n",
      "  Read 1:       454,325 bp\n",
      "  Read 2:       457,750 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3040_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-98ntfznv/S3040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-6mghzwon/867c1e5c-66b2-47c0-a323-1cab2b46c452/data/S3040_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        10,773 reads  @    111.5 µs/read;   0.54 M reads/minute\n",
      "Finished in 1.21 s (112 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,773\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):        10,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,080,464 bp\n",
      "  Read 1:     2,518,079 bp\n",
      "  Read 2:     2,562,385 bp\n",
      "Total written (filtered):      5,080,464 bp (100.0%)\n",
      "  Read 1:     2,518,079 bp\n",
      "  Read 2:     2,562,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: j-demux-pairedend-trimmed.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: j-demux-pairedend-trimmed.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kpfts8_5')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 32e907ff-8ea7-454a-a784-e6b37c55b1ea>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#1.j import the fastq files for library 1 into a QIIME2 artifact\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /Users/stephaniematthews/Documents/Chapter3/qiime2-analyses/manifests_demultiplexed-data_into_qiime2/COI/DS5B-Lib-manifest_COI.csv \\\n",
    "  --output-path j-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "#2.j trim primers\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences j-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences j-demux-pairedend-trimmed.qza \\\n",
    "    --verbose\n",
    "#3.j summarize and view\n",
    "!qiime demux summarize \\\n",
    "  --i-data j-demux-pairedend-trimmed.qza \\\n",
    "  --o-visualization j-demux-pairedend-trimmed.qzv\n",
    "q2.Visualization.load('j-demux-pairedend-trimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: j-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: j-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: j-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#4.j Generate ESVs (Exact Sequence Variants). \n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs j-demux-pairedend-trimmed.qza \\\n",
    "  --p-trunc-len-f 265 \\\n",
    "  --p-trunc-len-r 230 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences j-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table j-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats j-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: j-table.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5hbwkfyc')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 7a21d3b0-5d2e-4fc3-9e58-2a8ee87ad2ec>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#5.j view information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table j-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization j-table.qzv \n",
    "q2.Visualization.load('j-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: j-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#6.j view sequences \n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  j-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization j-paired-end-demux_rep-seqs-dada2.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now merge the separate libraries..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#merge \n",
    "!qiime feature-table merge \\\n",
    "  --i-tables a-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables b-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables c-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables d-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables e-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables f-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables g-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables h-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables i-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables j-paired-end-demux_table-dada2.qza \\\n",
    "  --o-merged-table COI-paired-end-demux_table-dada2.qza\n",
    "!qiime feature-table merge-seqs \\\n",
    "  --i-data a-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data b-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data c-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data d-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data e-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data f-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data g-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data h-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data i-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data j-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-merged-data COI-paired-end-demux_rep-seqs-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2_vsearch97.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime vsearch cluster-features-de-novo \\\n",
    "  --i-table COI-paired-end-demux_table-dada2.qza\\\n",
    "  --i-sequences COI-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --p-perc-identity 0.97 \\\n",
    "  --o-clustered-table  COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --o-clustered-sequences COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported COI-paired-end-demux_table-dada2_vsearch97.qza as BIOMV210DirFmt to directory COI-exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --output-path COI-exported-feature-table\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-kdvm7fx4')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 436cdcc7-361c-4c18-be92-d49a4dc47dbe>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i COI-exported-feature-table/feature-table.biom -o COI_97vsearch_OTUtable.tsv --to-tsv\n",
    "!biom convert -i COI-exported-feature-table/feature-table.biom -o COI-exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\"\n",
    "!biom convert -i COI-exported-feature-table/feature-table.biom -o COI-exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported COI-paired-end-demux_table-dada2.qza as BIOMV210DirFmt to directory COI-exported-feature-table-dada2\u001b[0m\n",
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "##Export the dada2 ASVs\n",
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2.qza \\\n",
    "  --output-path COI-exported-feature-table-dada2\n",
    "\n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data COI-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_rep-seqs-dada2.qzv')\n",
    "\n",
    "!biom convert -i COI-exported-feature-table-dada2/feature-table.biom -o COI-dada2_OTUtable.tsv --to-tsv\n",
    "!biom convert -i COI-exported-feature-table-dada2/feature-table.biom -o COI-exported-feature-table-dada2/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\"\n",
    "!biom convert -i COI-exported-feature-table-dada2/feature-table.biom -o COI-exported-feature-table-dada2/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-jqtn8028')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 436cdcc7-361c-4c18-be92-d49a4dc47dbe>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
